<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Wang et al. BMC Biotechnology 2013, 13:71 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                              Open Access <br />  <br /> Systematic screening glycosylation- trafficking-associated gene knockouts Saccharomyces cerevisiae identifies mutants improved heterologous exocellulase activity host secretion <br /> Tzi-Yuan Wang1&#226;&#8364;&#160;, Chih-Jen Huang1,2,3&#226;&#8364;&#160;, Hsin-Liang Chen1&#226;&#8364;&#160;, Po-Chun Ho1, Huei-Mien Ke1,4, Hsing-Yi Cho2,3,5, <br /> Sz-Kai Ruan1, Kuo-Yen Hung1, Li Wang1, Ya-Wun Cai1, Huang-Mo Sung6, Wen-Hsiung Li1,7* Ming-Che Shih2,5* <br />  <br />  <br />   Abstract <br />   Background: strong fermentator, Saccharomyces cerevisiae potential excellent host   ethanol production consolidated bioprocessing. purpose, necessary transform cellulose genes <br />   yeast genome contains cellulose genes.  heterologous protein expression S. <br />   cerevisiae suffers hyper-glycosylation poor secretion.  need genetically engineer <br />   yeast reduce glycosylation strength increase secretion ability. <br />   Results: Saccharomyces cerevisiae gene-knockout strains screened improved extracellular activity   recombinant exocellulase (PCX) cellulose digesting fungus Phanerochaete chrysosporium. Knockout <br />   mutants 47 glycosylation-related genes 10 protein-trafficking-related genes transformed PCX <br />   expression construct screened extracellular cellulase activity. screened mutants   2-fold increase extracellular PCX activity comparison wild type. extracellular <br />   PCX activities glycosylation-related mnn10 pmt5 null mutants  respectively, 6 4 times higher <br />   wild type; extracellular PCX activities 9 protein-trafficking-related mutants, especially   chc1, clc1 vps21 null mutants, 1.5 times higher parental strains. Site-directed <br />   mutagenesis studies revealed degree N-glycosylation plays important role   heterologous cellulase activity S. cerevisiae. <br />   Conclusions: Systematic screening knockout mutants glycosylation- protein trafficking-associated genes <br />   S. cerevisiae revealed  (1) blocking Golgi endosome transport force S. cerevisiae export cellulases; <br />   (2)  glycosylation alter enzyme activity cellulases. systematic <br />   gene-knockout screening approach serve convenient means increasing extracellular activities   recombinant proteins expressed S. cerevisiae. <br />   Keywords: Cellulase production, Glycosylation, Protein secretion <br />  <br />  <br />  <br />  <br /> * Correspondence: whli@sinica.edu.tw; mcshih@gate.sinica.edu.tw <br /> &#226;&#8364;&#160; <br />  Equal contributors <br /> 1 <br />  Biodiversity Research Center, Academia Sinica, Taipei 115, Taiwan <br /> 2 <br />  Molecular Biological Agricultural Sciences Program, Taiwan International <br /> Graduate Program, National Chung-Hsing University &#226;&#8364;&#8220; Academia Sinica, <br /> Taipei 115, Taiwan list author information available end article <br />  <br />                                         &#194;&#169; 2013 Wang et al.; licensee BioMed Central  Open Access article distributed terms Creative <br />                                         Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution,                                         reproduction medium, provided original work properly cited. <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                      Page 2 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br /> Background                                                     [42-46] shown potential cellulases exist <br /> Bioethanol derived cellulosic biomass consti-         white-rot fungus. genome sequence tute environmentally-friendly replacement fossil        P. chrysosporium, selected putative GH7 <br /> fuel future [1-3]. kinds bioprocesses   genes [47] expression S. cerevisiae. proposed conversion biomass cellu-        instead secreting recombinant cellulase losic ethanol separate hydrolysis fermentation     extracellular space, transformants accu- <br /> (SHF), simultaneous saccharification fermentation          mulate heterologous cellulase inside cells. (SSF), simultaneous saccharification fermentation       activity screening, chose GH7 gene (SSCF), consolidated bioprocessing (CBP).            highest exocellulase activity, designated PCX,  CBP promising           investigation. potential high efficiency low cost [4]. achieve       studies shown possible improve <br /> CBP, microbe capable producing secretory          secretion efficiency recombinant proteins expressed <br /> cellulases needed digest cellulose suitable       yeast manipulating cellular protein trafficking <br /> carbon source produce ethanol fermentation.              glycosylation pathways  reviews, [48-52]). <br />    Saccharomyces cerevisiae widely-used            examine effects changes glycosylation- microorganism fermentation high             protein trafficking-associated genes extracellular <br /> ethanol conversion rate. past decade, number        PCX activity, transformed PCX cDNA cellulases discovered animal guts,       yeast knockout (KO) strains 47 glycosylation-related <br /> est fungi plants, successfully          10 protein trafficking-associated genes screen <br /> expressed Pichia pastoris [5-22]. patented       increased cellulase activity. results <br /> commercial extracellular cellulases derived          showed knockouts glycosylation-related <br /> Trichoderma species [23-26].  establish           genes (MNN10 PMT5) Golgi endosome <br /> CBP platform using S. cerevisiae, host needs         transport gene (VPS21) increased extracellular exo- <br /> modified express secrete recombinant cellulases         cellulase activity 4 6 fold. comparison, efficiently [1,4,27,28].                                  expressed cellulase genes    Post-translational modifications S. cerevisiae affect    fungi, endocellulase (EgIII) exocellulase functions recombinant proteins. example,            (CBHI) T. ressei, endocellulase (Cen1) N-glycosylation yeast high-mannose type           Irpex lacteus, beta-glucosidase (BglI) Asper- <br /> [29-31] N-glycosylation recombinant              gillus niger S. cerevisiae. <br /> cellulase reduce activity enhance non-productive binding cellulose [32].           Results discussion hand, Trichoderma reesei benefits              Low level secretion recombinant cellulases S. <br /> expression S. cerevisiae glycosylation gene (DPM1)        cerevisiae increase cellulase activity [33].         Exocellulase PCX P. chrysosporium, endocellulase <br /> understanding glycosylation affects properties         EgIII exocellulase CBHI T. ressei, foreign proteins yeast cells helpful       endocellulase Cen1  lacteus successfully <br /> velopment S. cerevisiae production host            expressed S. cerevisiae BY4741 strain.  heterologous proteins.                                         alpha factor signal <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span> used secretion <br />    way achieve CBP express secrete        signal, heterologously expressed cellulases essential cellulases, endocellulase,       secreted yeast cells. Lower cellulase ac- <br /> exocellulase beta-glucosidase, S. cerevisiae        tivity observed protein extracted super- <br /> direct digestion cellulosic materials [19,34,35]. Exo-      natant cell culture protein extracted cellulase considered rate-limiting enzyme         cell pellets (Figure 1). suggested  cellulosic degradation. importance           glycosylation reduces activity recombinant cellu- <br /> exocellulase activity demonstrated         lases reduce secretion ability yeast <br /> studies [24,26,36]. reported white-rot      [53].  Golgi endosome transportation <br /> fungus Phanerochaete chrysosporium showed                 pathway interfere protein exocytosis  <br /> cellulase production higher activities          cause Golgi-endoplasmic reticulum  <br /> Trichoderma spp. agricultural waste [37]. Unlike T.         sponsible degradation detoxification reesei, P. chrysosporium produces different types         heterologous proteins [54].  investigated CBHI-like cellobiohydrolases [38], secretes             mutations glycosylation pathway exocellulases, especially <span id='am-35' about='obo:IMR_0001698' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-36' property="obo:IAO_0000115" content="Any compound that contains a constituent sugar, in which the hydroxyl group attached to the first carbon is substituted by an alcoholic, phenolic, or other group. They are named specifically for the sugar contained, such as glucoside (glucose), pentoside (pentose), fructoside (fructose), etc. Upon hydrolysis, a sugar and nonsugar component (aglycone) are formed. (From Dorland, 28th ed; From Miall&apos;s Dictionary of Chemistry, 5th ed)" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0001698" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="glycoside" datatype="xsd:string"></span>glycoside</span> hydrolase         Golgi endosome trafficking pathway affected family 7 (GH7) exocellulases cellulosic degrad-         secretion heterologous proteins cellulase <br /> ation. Omics approaches [39-41] functional studies         activity. <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                                                 Page 3 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br />  Figure 1 Relative total activities recombinant cellulases expressed S. cerevisiae.   4-methylumbelliferyl-&#206;&#178;-D-cellobiose (4-MUC assay) <br />  exocellulase activity CBH relative specific activity marker (100%). (B) Dye-carboxylmethy cellulose (dye-CMC) assay  endoglucanase activity EgIII relative specific activity marker (100%). (C) p-nitrophenyl-b-D-glucopyranoside (pNPG) assay  beta-glucosidase activity BglI relative specific activity marker (100%). expression plasmid related information described  Methods section. 4-MUC, Dye-CMC pNPG assay methods described [21]. <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                       Page 4 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br /> Deglycosylation increases extracellular PCX cellulase          studies suggested mutations MNN activity                                                       PMT increase endocellulase production S. cerevisiae survey S. cerevisiae BY4741 gene knockout collec-         [48]. summary, results suggest knocking tions (Open Biosystems) identified 70 viable         O- N-glycosylation-related genes increase <br /> strains knockout mutations genes related            total cellulase activity recombinant extracellular <br /> glycosylation protein trafficking. successfully         cellulase PCX. <br /> transformed PCX coding sequence 57 single- <br /> gene-knockout strains; 57 genes included 47 <br />                                                                Blocking Golgi endosome transport increases <br /> glycosylation-related genes, including                                                                extracellular PCX cellulase activity <br /> major ALG, OST, MNN, PMT gene families, 10 <br />                                                                yeast cells, heterologously expressed proteins trafficking-associated genes involved Golgi endo- <br />                                                                transported endosomes degradation VPS vacuole transport (Table 1 <span id='am-16' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-17' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Figure 2).                                                                family proteins [58]. results showed used 96 plate screening method test candi- <br />                                                                extracellular PCX activity increased  1.5- <br /> <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span> transformants increased total extracellular <br />                                                                fold 96 screening) 9 10 Golgi endo- <br /> cellulase activities using 4-methylumbellifery-&#206;&#178;-D- <br />                                                                related gene knockout mutants, especially cellobioside (4-MUC) assay  Table 1, 96 screen- <br />                                                                chc1, clc1 vps21 null mutants (Table 1). super- <br /> ing 4-MUC assay). extracellular PCX activities                                                                natants 50-ml cell culture condensed  <br /> 22 47 glycosylation-related gene knockout strains <br />                                                                analyzed activities. vps21 null mutant Golgi endosome transport <br />                                                                highest extracellular PCX cellulase activity, 6.5 times <br /> pathway mutants increased 1.3-fold relative                                                                higher parental strain. Chc1p Clc1p <br /> expression wild-type strain. ensure                                                                subunits clathrin protein form <br /> screening results reliable, condensed <br />                                                                clathrin-coated vesicles intracellular protein trans- supernatants 50-ml cell culture reanalyzed <br />                                                                port, Vps21p CORVET complex  activities. Interestingly, extracellular PCX cel- <br />                                                                quired vesicle transport vacuole lulase expressed glycosylation-related mnn10                                                                endosome (Figure 2). recent study  <br /> pmt5 null mutants showed 6.0- 4.3-fold increases                                                                hibition vesicle sorting vacuoles increase <br /> cellulase activity, respectively, compared expression <br />                                                                endocellulase production S. cerevisiae [52].  wild-type strain  Table 1, condensed sample 4- <br />                                                                sults imply blocking endocytosis pathway MUC assay). tested NpaBGS beta-glucosidase <br />                                                                indirectly influence S. cerevisiae export PCX isolated Neocallimastix patriciarum W5 strain <br />                                                                vesicle exocytosis pathway.  total extracellu- <br /> [55] 40 57 single-gene-knockout BY4741 <br />                                                                lar proteins increased slightly vps21 null mu- <br /> strains obtained similar results (Table 1). <br />                                                                tant (Figure 3).  unknown factors   4-MUC analyses showed increase extra- <br />                                                                influence extracellular PCX activity. <br /> cellular PCX activity ALG OST null mutants <br /> (Table 1). known ALG OST family <br /> genes play roles initiation addition N-<span id='am-40' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-41' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="glycine" datatype="xsd:string"></span>gly</span>-   Multiple-gene-knockout strains cans proteins elongation mannose          improvement PCX activity secretion <br /> endoplasmic reticulum [56,57]. ALG8, ALG9, ALG12,               constructed double gene knockout strains DIE2, OST4 OST6 downstream glycosylation               MNN10, PMT5 VPS21 examine combin- <br /> pathway genes responsible N-mannose N-glu-             ation mutations increase cellulase ac- <br /> cose elongation. Extracellular PCX activity        tivity secretion PCX enzyme. generation <br /> creased MNN PMT mutants. Golgi, MNN              times single- double-gene knockout strains <br /> family proteins add N-mannose complete N-linked gly-        shorter parental BY4741 strain <br /> cosylation, PMT family genes encode enzymes          (Additional file 1). strains, highest extra- <br /> elongating oligomannoses assembling O-glycosylation        cellular PCX activity recorded incubation [31]. previous study indicated disruption    24 hours (Additional file 2). addition, growth rate MNN10 gene enhanced protein secretion yeast [53].           mutant important extracellular 4-MUC analysis showed increases extracellu-        PCX activity. example, vps21 null mutant lar PCX activity mnn2, mnn5, mnn9, mnn10                higher enzyme activity pmt5 null mutant mnn11 null mutants (Table 1). genes        pmt5 null mutant shorter doubling time <br /> stream glycosylation pathway genes responsible high        higher saturation OD600 (Additional file 1).  N-mannose glycosylation (adding 10 mannose           analyses compared secreted PCX activity polymers). Extracellular PCX activity increased    single- double-gene-knockout strains 24-hour cul- pmt1, pmt2, pmt3 pmt5 null mutants. Previous           tures (Figure 3). <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                                                 Page 5 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br /> Table 1 Screening extracellular PCX cellulase activity                   Table 1 Screening extracellular PCX cellulase activity knockout original S. cerevisiae BY4741 strains                    knockout original S. cerevisiae BY4741 strains 4-MUC assay (recombinant/original ratio)                                    4-MUC assay (recombinant/original ratio) (Continued) <br /> Gene null          ORF namea           96 screening Condensed sample   mnn10&#206;&#8221;::KanMX      YDR245W              2.5            6.0 (3.8) <br /> mutant                                 4-MUC assay       4-MUC assay <br />                                                                                                           c <br />                                                          (o/n OD600)        mnn11&#206;&#8221;::KanMX      YJL183W              2.0            4.5 (3.7) <br /> Protein-trafficking genes:                                                  mns1&#206;&#8221;::KanMX       YJR131Wd             0.5            NA <br />                                                                                                           d <br /> chc1&#206;&#8221;::KanMX       YGL206C             2.5              2.4 (2.8)           mnt2&#206;&#8221;::KanMX       YGL257C              1.2            NA <br /> clc1&#206;&#8221;::KanMX       YGR167W             3.8              2.6 (3.1)           mnt3&#206;&#8221;::KanMX       YIL014Wd,e           2.5            1.7 (5.2) <br />                              f                                                                            f <br /> nyv1&#206;&#8221;::KanMX       YLR093C             1.5              1.7 (5.7)           mrl1&#206;&#8221;::KanMX       YPR079W              1.6            NA <br /> pep12&#206;&#8221;::KanMX      YOR036W             1.9              2.6 (2.9)           ost3&#206;&#8221;::KanMX       YOR085Wc             1.1            NA <br />                                                                                                           c,f <br /> vam3&#206;&#8221;::KanMX       YOR106W             1.6              NA                  ost4&#206;&#8221;::KanMX       YDL232W              1.6            NA <br /> vps1&#206;&#8221;::KanMX       YKR001C             1.2              NA                  ost5&#206;&#8221;::KanMX       YGL226C-Ac           0.9            NA <br />                                                                                                               c,f <br /> vps15&#206;&#8221;::KanMX      YBR097W             1.9              1.3 (4.2)           ost6&#206;&#8221;::KanMX       YML019W              1.8            NA <br /> vps21&#206;&#8221;::KanMX      YOR089Cf            3.3              6.5 (6.1)           pmt1&#206;&#8221;::KanMX       YDL095Wc             3.1            3.2 (4.7) <br />                                                                                                           c <br /> vps33&#206;&#8221;::KanMX      YLR396C             1.5              0.9 (5.5)           pmt2&#206;&#8221;::KanMX       YAL023C              5.3            1.8 (4.7) <br /> ypt7&#206;&#8221;::KanMX       YML001Wb            1.7              1.0 (5.1)           pmt3&#206;&#8221;::KanMX       YOR321Wc             1.1            1.0 (4.4) <br />                                                                                                           c <br /> Glycosylation-related genes:                                                pmt5&#206;&#8221;::KanMX       YDL093W              2.1            4.3 (6.3) <br /> aga2&#206;&#8221;::KanMX       YGL032C             0.9              NA                  pmt6&#206;&#8221;::KanMX       YGR199Wc             0.8            NA <br />                              c                                                                            f <br /> alg3&#206;&#8221;::KanMX       YBL082C             0.5              NA                  ynd1&#206;&#8221;::KanMX       YER005W              1.1            NA <br /> alg5&#206;&#8221;::KanMX       YPL227Cc            0.7              NA                  yur1&#206;&#8221;::KanMX       YJL139Cf             2.0            2.8(4.5) <br />                                  c <br /> alg6&#206;&#8221;::KanMX       YOR002W             0.8              NA                  yjr061w&#206;&#8221;::KanMX YJR061W                 1.0            NA <br />                                                                             alg8&#206;&#8221;::KanMX       YOR067Cc            1.5              NA                    YGL038C, YLR240W, YGL095C knockout strains failed transformed <br />                                                                             PCX. <br /> alg9&#206;&#8221;::KanMX       YNL219Cc            1.5              NA                  b <br />                                                                               Enhancement Ctcel8A endocellulase activity [52]. <br />                                                                             Enhancementc decreased Cel8Aenz endocellulase activity [48]. <br /> alg12&#206;&#8221;::KanMX      YNR030Wc            1.7              NA                  e <br />                                                                               Detectable NpaBGS beta-glucosidase activity screened pNPG assay. <br />                              c                                              f <br /> anp1&#206;&#8221;::KanMX       YEL036C             1.8              NA                    null mutant grow SC-URA medium 1% cellobiose                                                                             NpaBGS beta-glucosidase gene functional inside. <br /> csg2&#206;&#8221;::KanMX       YBR036C             1.0              NA                  4-MUC activity ratio average PCX activity highest <br /> cwh41&#206;&#8221;::KanMX      YGL027C   c <br />                                        0.1              NA                  expressing recombinants (single-gene-KO strain) average original PCX <br />                                                                             activity (BY4741 strain). <br /> cwh43&#206;&#8221;::KanMX      YCR017C             1.3              NA <br />                                  c <br /> die2&#206;&#8221;::KanMX       YGR227W             1.5              NA <br />                                                                                data showed mnn10 vps21 null mu- <br /> gda1&#206;&#8221;::KanMX       YEL042W             1.6              NA <br />                                                                             tants higher extracellular PCX activity gnt1&#206;&#8221;::KanMX       YOR320C             1.2              NA                  double-gene knockout mutants. previous study <br /> gon7&#206;&#8221;::KanMX       YJL184W             0.1              NA                  [32] reported Mnn10 protein elongated N-gly- <br />                                  d,e <br /> gtb1&#206;&#8221;::KanMX       YDR221W             2.7              0.0 (12.6)          adding mannose recombinant proteins hoc1&#206;&#8221;::KanMX       YJR075W             0.9              NA                  reduce heterologous protein activity yeast. <br /> hor7&#206;&#8221;::KanMX       YMR251W 1.4                        NA <br />                                                                             contrast, Figure 3 shows lack MNN10 gene <br />                                                                              e., deglycosylation) increase total activity ktr1&#206;&#8221;::KanMX       YOR099Wd            0.8              NA <br />                                                                             extracellular PCX secretion extracellular pro- <br /> ktr2&#206;&#8221;::KanMX       YKR061Wd,f          0.9              NA                  teins. significantly total extracellular <br /> ktr3&#206;&#8221;::KanMX       YBR205Wd            1.0              NA                  protein abundance mnn10 null mutant                                  d <br /> ktr4&#206;&#8221;::KanMX       YBR199W             1.0              NA                  mutants examined, suggest yeast <br /> ktr5&#206;&#8221;::KanMX       YNL029Cc            1.0              NA                  lacking MNN10 gene suitable raise yield ktr6&#206;&#8221;::KanMX       YPL053C   d <br />                                        1.3              NA <br />                                                                             heterologous protein expressed S. cerevisiae host sys- <br />                                                                             tem (Figure 3). Second, PMT5p functions addition <br /> ktr7&#206;&#8221;::KanMX       YIL085Cd            1.0              NA <br />                                  d <br />                                                                             O-linked glycans amino acids. results consist- <br /> mnl1&#206;&#8221;::KanMX       YHR204W             1.2              NA                  ently showed PMT5 mutation, result <br /> mnn2&#206;&#8221;::KanMX       YBR015Cc            1.3              0.8 (3.8)           O-glycosylation, increase total activ- <br />                              f <br /> mnn5&#206;&#8221;::KanMX       YJL186W             1.9              3.0 (5.6)           ity extracellular PCX. Extracellular protein secretion <br /> mnn9&#206;&#8221;::KanMX       YPL050Cc,f          2.2              3.8 (5.5)           did increase significantly pmt5 null mutant,                                                                             total cellulase activity increased, suggesting  Wang et al. BMC Biotechnology 2013, 13:71                                                                                Page 6 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br />  Figure 2 Major gene families involved glycosylation protein transport pathways yeast. <br />  <br />  <br /> reduction O-glycosylation increases cellulase activity             steps glycosylation pathway. introduced affect protein secretion.  Vps21 key           PCX gene alg1-1 mutant (ALG1) contains <br /> player Golgi vacuole transport. results showed              defects asparagine-linked glycosylation [59]. ALG1 total extracellular PCX activity increased          gene functions elongating N-glycan formation yeast lacks Vps21 protein. double gene              glycans (MansGlcNac2) N-glyco- <br /> knockout strains, mnn10&#206;&#8221;vps21&#206;&#8221; knockout strain                sylation site protein (Figure 2; [60]). purified PCX slow growth produce PCX protein. double-                 alg1-1 mutant showed significant shift molec- <br /> gene knockout mutant regulate protein                      ular weight, remaining null mutants (e.g., <br /> secretion, extracellular PCX activity, cell growth.                mnn10&#206;&#8221;::KanMX) slightly different SDS-PAGE mobil-  mutants probably suitable                 ity patterns, vps21&#206;&#8221;::KanMX (Figure 4).  <br /> application cellulosic ethanol production.                 PCX activity alg1-1 mutant lower  pmt5&#206;&#8221;vps21&#206;&#8221; mnn10&#206;&#8221;pmt5&#206;&#8221; knockout                         knockout strains implying modified <br /> strains increased total extracellular cellulase activity          glycans played important role PCX activity showed slower growth rates single-gene                  (Additional file 3). Furthermore, MNN10 mutant knockout strains. results suggest increase             highest level extracellular protein secretion extracellular cellulase activity mnn10 null mu-             knockout strains. previous study suggested dis- <br /> tant mainly increase cellulase protein se-            ruption MNN10 gene increases protein secretion cretion, protein activity itself,              yeasts change structure glycoproteins <br /> increase pmt5 vps21 null mutants mainly                 yeast cell wall affects release secretory protein post-translational modifications                proteins media stage protein secre- <br /> disordered intracellular trafficking.               tion [53].  MNN10 gene essential sults suggest single-gene knockout strains                increase yeast tolerance ethanol-induced stress [61]. suitable bioethanol applications.                               pmt5 null mutant,  <br />                                                                       codes O-mannosyltransferase, resulted increase <br /> Protein modification differences recombinant                    extracellular PCX activity (Table 1). PMT family <br /> cellulases null mutants                                            proteins manage O-glycosylation sorting determin- increase extracellular cellulase activity results               ant cell surface delivery [62], protein O-glycosyl- changes glycosylation extracellular secre-             ation essential cell wall rigidity cell integrity tion. PCX potential N-glycosylation sites            protein modification S. cerevisiae T. reesei <br /> catalytic domain putative O-glycosylation sites              [63,64].  results point possibility cellulose-binding domain. assess effect              PMT5 mutant increase extracellular PCX <br /> glycosylation extracellular PCX activity, compared              activity play role pro- effects abolishing gene function earlier later          tein secretory process. <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                                                Page 7 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br />  Figure 3 Extracellular PCX activity 50 ml supernatants different hosts measured 4-MUC assay.   Total extracellular activity; <br />  (B) specific extracellular activity; (C) total extracellular proteins. WT: BY4741; &#206;&#8221;M: mnn10 null mutant;&#206;&#8221;V: vps21 null mutant; &#206;&#8221;P: pmt5 null <br />  mutant; &#206;&#8221;M&#206;&#8221;P: mnn10&#206;&#8221;pmt5&#206;&#8221; double knockout strain; &#206;&#8221;P&#206;&#8221;V: pmt5&#206;&#8221;vps21&#206;&#8221; double knockout strain; CBHI: exocellulase Trichoderma ressei. <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                                                Page 8 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br />  Figure 4 Extracellular tagged PCX different hosts. SDS-PAGE, left, 10 &#206;&#188;g lane, western blot, right, 1 &#206;&#188;g lane  extracellular tagged PCX different hosts. left right: lane 1, PCX BY4741 strain; lane 2, PCX mnn10 null mutant; <br />  lane 3, PCX vps21 null mutant; lane 4, PCX pmt5 null mutant; lane 5, PCX alg1-1 mutant. protein marker  Fermentas PageRuler Pre-stained Protein Ladder. <br />  <br />  <br />  <br /> N-glycosyation site-mutagenesis influences activity                  results suggest individual mutations recombinant cellulase                                                     PCX gene affect PCX activity. conducted site-directed mutagenesis putative                       extracted intracellular soluble proteins N-glycosylation sites PCX examine possible                      cell pellets evaluate secretary efficiencies effects extracellular PCX activity (Figure 5A,B).                    mutants - N208D, N208Q, N326D, N326Q, N442D, <br /> Peptide-N-Glycosidase F (PNGaseF) cleave high man-                      N442Q (Figure 5C). PCX extracellular intracellu- <br /> nose complex oligosaccharides N-linked glyco-                      lar soluble protein activities N208D, N326D, proteins. deglycosylated PCX proteins treated                      N442Q mutants lower mutants. <br /> PNGaseF (Additional file 4) showed statistically sig-                N442Q mutant showed significant reduc- <br /> nificant difference 4-MUC activity vps21                     tion extracellular intracellular activities. contrast, <br /> null mutant wild-type BY4741; previous                      N208Q N442D mutants showed higher intracellu- <br /> strains added N-glycans expressed PCX                      lar higher extracellular activity wild-type PCX null mutants (pmt5&#206;&#8221; mnn10&#206;&#8221;) tested.                       (Figure 5). Furthermore, mutated PCX gene results suggest mutations O- N-glyco-                  manipulated mnn10, pmt5, vps21 null mu- <br /> sylation sites influence extracellular PCX activity.                tants, extracellular activities mutants    Protein modeling cellobiohydrolase 58 based X-                     higher wide-type strain Figure 3 (data <br /> ray data (PDB code 1Z3V) revealed N208 site                     shown).  mutants suitable <br /> located near cellulose-binding tunnel                     hosts extracellular PCX protein production. short, sites (N326 N442) located PCX pro-                       combination N442D site-mutated PCX protein tein surface (Additional file 5). N-glycosylation                  vps21 null mutants exhibit highest extracellular <br /> interfere substrate binding PCX conformation                     PCX activity. <br /> structure influence PCX activity.   <br /> placed potential N-glycosylation <span id='am-28' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-29' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-30' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span>                       Comparison null mutants enhance different <br /> sites (positions N208, N326, N442) PCX                       cellulase protein production S. cerevisiae <br /> partate (D) glutamine (Q). mutated PCX genes                       study complements recent studies [48,52]. transformed yeast determine effects                      Kitagawa et al. (2011) reported 55 S. cerevisiae knockout extracellular PCX cellulases. extracellular activ-                   strains exhibited enhanced Ctcel8A endocellulase <br /> ities mutated PCX cellulases,                         activity. vps3&#206;&#8221; vps16&#206;&#8221; strains, dele- <br /> N442D mutant, lower wild-type                        tions genes encoding components class C core <br /> PCX cellulase (Figure 5). N442Q mutant showed                           vacuole/endosome tethering (CORVET) complex, significantly lower extracellular PCX activity. total                   exhibited enhanced beta-glucosidase activity Ctcel8A specific extracellular activities N442D mu-                      proteins. Vps3p Vps21p major components <br /> tant,  higher wild-type PCX.                     CORVET complex.  inhibition  Wang et al. BMC Biotechnology 2013, 13:71                                                                                        Page 9 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br />  Figure 5 Extracellular PCX activity improved site-mutagenesis putative N-glycosylation amino acids.   Total extracellular <br />  activity; (B) specific extracellular activity; (C) total activity extracellular intracellular soluble proteins. <br />  <br />  <br /> endocytosis pathway improve heterologous                   mutations ALG, PMT MNN family genes cellulase production. second study, Suzuki et al.                  improved Cel8Aenz production S. cerevisiae. <br /> (2012) screened enhancement endocellulase                    results results shown Table 1 indicate <br /> Cel8Aenz using 44 protein glycosylation mutants.                    mutations mnn9&#206;&#8221;, ost4&#206;&#8221;, ost6&#206;&#8221; cause <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                        Page 10 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br /> reduction N-glycosylation abilities yeast host        100 ml defined medium (PD) 1% rice straw increase heterologous celluase activities.      grown 25&#194;&#176;C shaking 170 rpm 5&#226;&#8364;&#8220;7 days.  O-glycosylation deficiency, seen           filtered cell culture mass ground liquid nitrogen, <br /> pmt1&#206;&#8221;, pmt2&#206;&#8221;, pmt5&#206;&#8221; mutants, improve heter-          total RNA extracted using hot acid phenol- <br /> ologous cellulase production.                                   chloroform method [66] quantified spectropho- <br />   screening method (adopted [21])                tometer. total RNA converted cDNA using <br /> quantify sensitively detect total activity exo-      SuperScript&#226;&#8222;&#162; II Reverse Transcriptase Kit (Invitrogen, <br /> glucanase easily, plate method [52]           Carlsbad, CA). reaction contained total RNA (4 &#206;&#188;g), <br /> sensitive apply exoglucanse screening.         oligo(dT) primer, BD SMART II oligo (Clontech, <br /> systemic analyses using yeast deletion strain collec-       Mountain View, CA, USA), 4 &#206;&#188;L 5&#195;&#8212; RT buffer, 10 mM <br /> tion allowed identify genes  mutated,           DTT, 0.5 mM deoxyribonucleotide <span id='am-23' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span>triphosphate</span> increase efficiency cellulase production.          10 U Superscript II RNase H Reverse Transcriptase <br />                                                                 (Invitrogen). PCX coding sequence cloned <br /> Conclusions                                                     pGEM-T easy plasmid (Promega, Madison, WI). <br /> Systematic screening analysis revealed S. <br /> cerevisiae glycosylation protein-trafficking pathway        Plasmid construction gene cloning <br /> gene-knockout mutants improve extracellular cellulase           pRS426-GD plasmid constructed GAPDH <br /> activity undergo functional changes heterol-         5&#226;&#8364;&#8482;-promoter (upstream 500 bp) alpha <br /> ogous protein synthesis yeast. known N-gly-       factor sequence 3&#226;&#8364;&#8482;-terminator region (downstream <br /> cosylation yeast high-mannose type        300 bp) S. cerevisiae genomic sequence increased N-glycosylation recombinant cellulases          pRS426 plasmid backbone cellulase gene expression <br /> duces activity [29,32-34]. screening gene-         (Additional file 6). tagged cellulase construct <br /> knockout mutants glycosylation pathway showed            cloned pGEM-T Easy Vector (Promega) activity recombinant PCX cellulase increased        confirmed diagnostic PCR sequencing. significantly mannose-glycosylation gene knock-         confirmed cellulase cDNAs cloned strains mnn9, mnn10, mnn11 mnt3.                expression vector pRS426-GD restriction enzyme <br /> Heterologous PCX cellulase activity affected         (EcoRI + SmaI) digestion denatured-renatured tech- high-mannose glycosylation pattern,           niques [67]. activity intracellular trafficking systems S. <br /> cerevisiae. Systematic screening knockouts intracel-      Knockout yeast strains <br /> lular trafficking-associated genes showed activity     yeast strain BY4741 (MATa his3&#206;&#8221;1 leu2&#206;&#8221;0 met15&#206;&#8221;0 extracellular PCX cellulase increased significantly     ura3&#206;&#8221;0) descendant S288C. single gene knock- intracellular trafficking-associated genes         strains (gene&#206;&#8221;::KanMX) Table 1 derived CHC1, CLC1, VPS21 inactivated. O-glycosyla-            BY4741 Open Biosystems Company cat# YSC1021 <br /> tion associated genes, PMT1, PMT2 PMT5,             [68]. selected genes listed Table 1. involved post-translational modification        alg1-1 (MATa alg1-1 ura3-52 mal gal2) (ATCC 208304) endoplasmic reticulum, influenced cell wall        ALG1 deficient mutant glycosylation study [59], <br /> rigidity secretion PCX cellulase protein,        required adding glycans (MansGlcNac2) <br /> potentially posing obstacle efficient use S.       directly protein N-glycosylation site  Figure 2). <br /> cerevisiae host consolidated bioethanol production      construct double gene-knockout strains,  systematic gene knockout screening approach         MATa-type MNN10, VPS21 PMT5 knockout strains proven useful test secretion capability    switched MAT&#206;&#177;-type introducing cellulase activity proteins heterologously expressed S.   pGAL-HO plasmid [69]. knockout strains cerevisiae. future, approach         MATa MAT&#206;&#177;, respectively, mated modified automatic high-throughput platform            create diploid hybrids, fur- <br /> speed screening process S. cerevisiae mutant     ther sporulated haploid segregants. segregants <br /> strains suitable use bioprocessing.          double-gene knockout clones selected PCR <br />                                                                 following manufacturer&#226;&#8364;&#8482;s instructions. <br /> Methods <br /> White-rot fungial cDNA preparation                              Yeast transformation <br /> Cells P. chrysosporium (ATCC 32629) grown           Yeast strains grown YPAD medium harvested <br /> 5 days PDA medium (diced potatoes 200.0 g/l, glucose         mid-log phase. Overnight yeast cultures used <br /> 20.0 g/l, agar 15.0 g/l) 25&#194;&#176;C [65]. thirty squares     prepare starting cultures OD600 = 0.1 com- <br /> (5 mm &#195;&#8212; 5 mm) mycelium mat inoculated              petent cells grown YPAD media 30&#194;&#176;C  Wang et al. BMC Biotechnology 2013, 13:71                                                                              Page 11 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br /> 250 rpm shaking OD600 = 0.8-1.0, harvested.     Protein purification <br /> plasmid transformation performed LiAc/SS           Cell lysates PCX transformants wild-type Carrier DNA/PEG method [70] successful trans-             knockout strains harvested centrifugation formants selected SC-URA plate.                     supernatants dialyzed binding buffer [8 M <br />                                                               <span id='am-77' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-78' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>urea</span>, 0.1 M NaH2PO4, 0.01 M Tris-Cl, pH 8.0]. Purifica- <br /> Detection PCX cellulase activity 4-MUC assay            tion achieved Ni Sepharose 6 Fast Flow column colonies PCX transformants strain            (GE Healthcare, Taipei, Taiwan). column washed randomly selected cultured 96 deep           wash buffer [8 M urea, 0.1 M NaH2PO4, 0.01 M <br /> plates 500 &#206;&#188;l SC-URA medium 30&#194;&#176;C 1&#226;&#8364;&#8220;3 days.        Tris-Cl, pH 6.3] recombinant PCXs cellulase activity screening, 50 &#206;&#188;l supernatants     eluted elution buffer [8 M urea, 0.1 M NaH2PO4, <br /> transferred 96 plates, mixed 1 mg/ml 4-        0.01 M Tris-Cl, pH 4.5]. <br /> MUC substrates (Sigma-Aldrich, St Louis, MO),  <br /> cubated 24 hours 30&#194;&#176;C suitable pH (Additional       Western blot <br /> file 7). OD365/OD450 (excitation/emission) reads          purified recombinant PCX wild-type knock- measured SpectraMax M2 Microplate Readers             strains run 8% SDS&#226;&#8364;&#8220;polyacrylamide gel indicate higher exocellulase activity transformants.    proteins transferred PVDF (GE <br /> addition, confirm 4-MUC screening results,          Healthcare) electroblotting buffer (25 mM Tris&#226;&#8364;&#8220;HCl, colonies higher 4-MUC activity     192 mM <span id='am-52' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-53' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span>glycine</span>, 20% methanol) 400 mA 1 h 4&#194;&#176;C. <br /> gene diluted OD600 = 0.1 grown 50 ml         membrane blocked non-fat milk buffer [5% <br /> SC-URA medium 30&#194;&#176;C 24 hours. 24 hours            blocking reagent TBS buffer (125 mM NaCl, 25 mM stationary phase, cell density satu-    Tris, pH 8.0)] 4&#194;&#176;C overnight. tag monoclonal <br /> rated strains slow growth rates. total    antibody (1:3000) used primary antibody, PCX enzyme activities stable stationary       AP-&#206;&#177;-mouse (goat) mouse antibody (1:2000) used <br /> phase (Additional file 2). condensed supernatants         secondary antibody. Detection performed assayed                                            NBT/BCIP (Roche) substrates. <br />    selected colonies highest expres- <br /> sion evaluate total extracellular cellulase activity   Site-directed mutagenesis PCX N-glycosylation residues condensed supernatants 4-MUC assay. S.           N-glycosylation sites predicted ExPASy tools [71]. <br /> cerevisiae, copy number pRS426 plasmid          putative N-glycosylation sites (Asn&#226;&#8364;&#8220;Xaa&#226;&#8364;&#8220;Ser/Thr 20 haploid cell. copy number variations         motif), Asn208, Asn326, Asn442, predicted did correlate total 4-MUC activity          mutated Quick Change II Site-Directed mutagen- different hosts. example, vps21 null mu-        esis Kit (Stratagene, USA); Asn208, Asn326, Asn442 <br /> tant similar relative copy numbers showed different      mutated Asp (D) Gln (Q). primers used <br /> 4-MUC activity (Additional file 8). avoid activity vari-   listed Additional file 9. <br /> ation different cellulase gene expression levels loss plasmid copies, screened selected          Protein structure modeling colonies highest activity ex-      Modeling PCX structure performed perimental culture analysis performed      program SWISS-MODEL (http://swissmodel.expasy.org/) biological replicates experiment.    [72]. template selected based sequence <br /> overnight cultures candidates diluted      alignment. P. chrysosporium cellobiohydrolase Cel7D <br /> OD600 = 0.1 grown 50 ml SC-URA medium 30&#194;&#176;C          (CBH58) sequence (UniProtKB code Q7LIJ0) showed 82% 24 hours. 50 ml supernatants dialyzed       sequence identity PCX. protein-<span id='am-11' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Lactose" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span>lactose</span> complex condensed 100-fold Vivaspin 20 10 K NMWL             structure CBH58 solved 2005 (PDB code 1Z3V) <br /> (MWCO) concentrator (Sartorius, Goettingen, Germany).         [73] defined template. model structure <br /> Enzyme activities measured 4-MUC assay            PCX (Additional file 5) presented molecu- <br /> [21]. soluble protein concentration measured          lar visualization PyMOL. Protein Assay reagent (500&#226;&#8364;&#8220;0006; Bio-Rad La- <br /> boratories, Hercules, CA, USA). reaction time                                                               Additional files 4-MUC assay 24 hours condensed super- <br /> natant hour cell pellet-extract 30&#194;&#176;C.        Additional file 1: Cell density doubling time knockout <br /> addition, chose total cellulase activity evaluate    strains PCX expression SC-URA medium. <br /> suitable strains CBP application strains       Additional file 2: Time course extracellular PCX activity higher cell density usually better total cellu-      candidate strains. overnight cultures candidates                                                                diluted OD600 = 0.1 grown 50 ml SC-URA medium 30&#194;&#176;C lase activity. <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                                                      Page 12 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br />  24 hours. milliliter supernatant candidate harvested       SIC, Academia Sinica, DNA sequencing. study funded  hour total activity extracellular PCX assayed using     National Science Council (NSC 96-3114-P-001-004-Y NSC 97-3114-P-001- <br />  4-MUC method. 426GD(WT) plasmid PCX gene            001) Academia Sinica, Taiwan. <br />  wild-type strain background control. <br />                                                                                Author details <br />  Additional file 3: Extracellular PCX activity 50 ml supernatants        1 <br />                                                                                 Biodiversity Research Center, Academia Sinica, Taipei 115, Taiwan. <br />  different hosts measured 4-MUC assay. WT: BY4741; &#206;&#8221;M: mnn10             2 <br />                                                                                 Molecular Biological Agricultural Sciences Program, Taiwan International <br />  null mutant; &#206;&#8221;V: vps21 null mutant; &#206;&#8221;P: pmt5 null mutant; alg1-1: ALG1        Graduate Program, National Chung-Hsing University &#226;&#8364;&#8220; Academia Sinica, <br />  deficient mutant; CBHI: exocellulase Trichoderma ressei.                   Taipei 115, Taiwan. 3Graduate Institute Biotechnology, National Chung- <br />  Additional file 4: PNGaseF-treated extracellular PCX activity              Hsing University, Taichung 402, Taiwan. 4Program Microbial Genomics, <br />  50 ml supernatants different hosts measured 4-MUC assay.             National Chung Hsing University &#226;&#8364;&#8220; Academia Sinica, Taipei 115, Taiwan. <br />                                                                                5 <br />  WT: BY4741; &#206;&#8221;M: mnn10 null mutant; &#206;&#8221;V: vps21 null mutant; &#206;&#8221;P: pmt5 null        Agricultural Biotechnology Research Center, Academia Sinica, Taipei 115, <br />  mutant; CBHI: exocellulase Trichoderma ressei. 1 mg extracellular      Taiwan. 6Department Life Sciences, National Cheng Kung University, <br />  total proteins treated 500 units PNGaseF (P0704, NEB), 0.1%         Tainan 701, Taiwan. 7Department Ecology Evolution, University  NP-40 20 &#206;&#188;l reaction volume 4 h 37&#194;&#176;C interacted       Chicago, Chicago 60637, USA. <br />  4-MUC substrate 24 h 30&#194;&#176;C. *P &lt; 0.05; ** P &lt; 0.01. <br />  Additional file 5: Model structure PCX.   Ribbon view PCX             Received: 27 December 2012 Accepted: 29 August 2013 <br />  model structure, beta-sandwich consists large          Published: 3 September 2013 <br />  antiparallel beta-sheets catalytic core. Cellulose substrate  represented yellow-colored ball stick model. (B) view     References <br />  cellulose-binding tunnel. possible N-glycosylation sites        1. Torney F, Moeller L, Scarpa  Wang K: Genetic engineering approaches  shown red-colored ball stick models.                                   improve bioethanol production maize. Current opinion  Additional file 6: Map pRS426-GD plasmid PCX.   alpha                   biotechnology 2007, 18(3):193&#226;&#8364;&#8220;199. <br />  signal peptide; (B) non-alpha signal peptide.                                 2. Galbe M, Sassner P, Wingren  Zacchi G: Process engineering economics <br />  Additional file 7: Extracellular PCX activity different pHs                bioethanol production. Advances biochemical engineering/ <br />  reaction temperatures measured 4-MUC assay.                                 biotechnology 2007, 108:303&#226;&#8364;&#8220;327. <br />                                                                                3. Gray KA, Zhao L, Emptage M: Bioethanol. Current opinion <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> <br />  Additional file 8: Relative copy number expression plasmids                biology 2006, 10(2):141&#226;&#8364;&#8220;146. <br />  total 4-MUC activity. copy number Ampicillin gene         4. van Zyl WH, Lynd LR, den Haan R, McBride JE: Consolidated bioprocessing <br />  pRS426GD plasmid normalized Adh1 gene BY4741                  bioethanol production using Saccharomyces cerevisiae. Advances  genome. yeast cells harvested genomic DNA                    biochemical engineering/biotechnology 2007, 108:205&#226;&#8364;&#8220;235. <br />  extracted Fast <span id='am-76' about='oboInOwl:id' typeof='owl:Thing'>ID</span> Genomic DNA Extraction Kit (Genetic ID, Fairfield, IA)   5. Ferrarese L, Trainotti L, Gattolin S, Casadoro G: Secretion, purification  following manufacturer&#226;&#8364;&#8482;s instructions. relative            activity recombinant pepper endo-beta-1,4-glucanases expressed <br />  gene normalized Act1 gene (&#206;&#8221;Ct) quantified            yeast Pichia pastoris. FEBS Lett 1998, 422(1):23&#226;&#8364;&#8220;26. <br />  &#206;&#8221;&#206;&#8221;Ct relative quantification method relative ratio            6. Zhao XH, Wang W, Wang FQ, Wei DZ: comparative study beta-1, 4- <br />  determined following guidelines issued Applied Biosystems.             endoglucanase (possessing beta-1, 4-exoglucanase activity) Bacillus <br />  amplification efficiency primer pair tested using 2-fold           subtilis LH expressed Pichia pastoris GS115 Escherichia coli Rosetta <br />  serial dilutions templates. Q-PCR relative copy number ratio           (DE3). Bioresour Technol 2012, 110:539&#226;&#8364;&#8220;545. <br />  determined &#206;&#8221;&#206;&#8221;Ct value using formula, relative copy         7. Xu  Escamilla-Trevino L, Zeng L, Lalgondar M, Bevan D, Winkel B, <br />  number ratio (Ampicillin/ADH1) = 2[&#226;&#710;&#8217;&#206;&#8221;&#206;&#8221;Ct], suggested Applied             Mohamed  Cheng CL, Shih MC, Poulton J, et al: Functional genomic <br />  Biosystems, amplification efficiency target gene           analysis Arabidopsis thaliana glycoside hydrolase family 1. Plant Mol <br />  reference gene approximately equal (Adh1: 95.4%, Act1: 87.7%,                Biol 2004, 55(3):343&#226;&#8364;&#8220;367. <br />  Ampicillin: 100%).                                                            8. Wonganu B, Pootanakit K, Boonyapakron K, Champreda V, Tanapongpipat S, <br />  Additional file 9: Primer sequences site-directed mutagenesis.                Eurwilaichitr L: Cloning, expression characterization                                                                                    thermotolerant endoglucanase Syncephalastrum racemosum <br />                                                                                    (BCC18080) Pichia pastoris. Protein Expr Purif 2008, 58(1):78&#226;&#8364;&#8220;86. <br /> Abbreviations                                                                  9. Thongekkaew J, Ikeda H, Masaki K, Iefuji H: acidic thermostable <br /> 4-MUC: 4-methyl umbelliferyl cellobiose; CBP: Consolidated bioprocess;             carboxymethyl cellulase yeast Cryptococcus sp. S-2: purification, <br /> CMC: Carboxylmethyl cellulose; dye-CMC: Dye-carboxylmethyl cellulose;              characterization improvement recombinant enzyme <br /> pNPG: p-nitrophenyl-&#206;&#178;-D-glucopyranoside; SDS&#226;&#8364;&#8220;PAGE: Sodium dodecyl                  production high cell-density fermentation Pichia pastoris. <br /> sulfate&#226;&#8364;&#8220;polyacrylamide gel electrophoresis; SHF: Separate hydrolysis           Protein Expr Purif 2008, 60(2):140&#226;&#8364;&#8220;146. <br /> fermentation; SSF: Simultaneous saccharification fermentation;             10. Shi H, Yin X, Wu M, Tang C, Zhang H, Li J: Cloning bioinformatics <br /> SSCF: Simultaneous saccharification fermentation.                           analysis endoglucanase gene (Aucel12A) Aspergillus usamii <br />                                                                                    functional expression Pichia pastoris. J Ind Microbiol Biotechnol <br /> Competing interests                                                                2012, 39(2):347&#226;&#8364;&#8220;357. authors declare competing interests.                     11. Samanta S, Basu  Halder UC, Sen SK: Characterization Trichoderma <br />                                                                                    reesei endoglucanase II expressed heterologously Pichia pastoris Authors&#226;&#8364;&#8482; contributions                                                             better biofinishing biostoning. J Microbiol 2012, 50(3):518&#226;&#8364;&#8220;525. <br /> TYW, HLC, CJH designed, coordinated, conducted experiments         12. Molhoj M, Ulvskov P, Dal Degan F: Characterization functional analyses, wrote manuscript draft. HMS, WHL MCS                  soluble form Brassica napus membrane-anchored endo-1,4-beta- <br /> designed supervised research revised manuscript. HYC, HMK          glucanase heterologously expressed Pichia pastoris. Plant Physiol YWC conducted fungal culture, PCX cloning mutagenesis. SKR             2001, 127(2):674&#226;&#8364;&#8220;684. KYH conducted yeast transformation cellulase activity assay.       13. Lindenmuth  McDonald KA: Production characterization PCH ILW conducted protein purification related analyses.       Acidothermus cellulolyticus endoglucanase Pichia pastoris. Protein Expr <br /> authors read approved final manuscript.                                    Purif 2011, 77(2):153&#226;&#8364;&#8220;158. <br />                                                                                14. Li J, Tang C, Shi H, Wu M: Cloning optimized expression neutral <br /> Acknowledgements                                                                   endoglucanase gene (ncel5A) Volvariella volvacea WX32 Pichia thank Ms. Wan-Ting Lin Ms. Olivia Yu-Chia Chen assistance;        pastoris. J Biosci Bioeng 2011, 111(5):537&#226;&#8364;&#8220;540. Dr. Kay-Hooi Khoo, Mr. Sz-Wei Wu (NRPGM Core Facilities Proteomics     15. Godbole S, Decker SR, Nieves RA, Adney WS, Vinzant TB, Baker JO, Thomas Glycomics, Taipei, Taiwan) Dr. Chi-Huey Wong thoughtful                SR, Himmel  Cloning expression Trichoderma reesei <br /> discussion glycoprotein analysis. thank Sequencing Core Facility,        cellobiohydrolase Pichia pastoris. Biotechnol Prog 1999, 15(5):828&#226;&#8364;&#8220;833. <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                                                               Page 13 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br /> 16. Generoso WC, Malago-Jr W, Pereira N Jr, Henrique-Silva F: Recombinant           39. Vanden Wymelenberg  Minges P, Sabat G, Martinez D, Aerts  Salamov  <br />     expression characterization endoglucanase III (cel12a)               Grigoriev  Shapiro H, Putnam N, Belinky P, et al: Computational analysis     Trichoderma harzianum (Hypocreaceae) yeast Pichia pastoris.                  Phanerochaete chrysosporium v2.0 genome database mass <br />     Genet Mol Res 2012, 11(2):1544&#226;&#8364;&#8220;1557.                                                spectrometry identification peptides ligninolytic cultures reveal <br /> 17. Escamilla-Trevino LL, Chen W, Card ML, Shih MC, Cheng CL, Poulton JE:               complex mixtures secreted proteins. Fungal Genet Biol 2006, 43(5):343&#226;&#8364;&#8220;356. <br />     Arabidopsis thaliana beta-Glucosidases BGLU45 BGLU46 hydrolyse              40. Vanden Wymelenberg  Gaskell J, Mozuch M, Sabat G, Ralph J, Skyba O, <br />     monolignol glucosides. Phytochemistry 2006, 67(15):1651&#226;&#8364;&#8220;1660.                       Mansfield SD, Blanchette RA, Martinez D, Grigoriev  et al: Comparative <br /> 18. Ahn YO, Zheng M, Bevan DR, Esen  Shiu SH, Benson J, Peng HP, Miller JT,           transcriptome secretome analysis wood decay fungi Postia <br />     Cheng CL, Poulton JE, et al: Functional genomic analysis Arabidopsis             placenta Phanerochaete chrysosporium. Appl Environ Microbiol 2010, <br />     thaliana glycoside hydrolase family 35. Phytochemistry 2007,                        76(11):3599&#226;&#8364;&#8220;3610. <br />     68(11):1510&#226;&#8364;&#8220;1520.                                                               41. Sato S, Feltus FA, Iyer P, Tien M: genome-level transcriptome 19. Baldrian P, Valaskova V: Degradation cellulose basidiomycetous                wood-degrading fungus Phanerochaete chrysosporium grown red <br />     fungi. FEMS microbiology reviews 2008, 32(3):501&#226;&#8364;&#8220;521.                               oak. Curr Genet 2009, 55(3):273&#226;&#8364;&#8220;286. <br /> 20. Watanabe H, Tokuda G: Cellulolytic systems insects. Annu Rev Entomol         42. Vallim MA, Janse BJ, Gaskell J, Pizzirani-Kleiner AA, Cullen D: Phanerochaete <br />     2010, 55:609&#226;&#8364;&#8220;632.                                                                   chrysosporium cellobiohydrolase cellobiose dehydrogenase <br /> 21. Wang TY, Chen HL, Lu  Chen YC, Sung HM, Mao CT, Cho HY, Ke HM, Hwa               transcripts wood. Appl Environ Microbiol 1998, 64(5):1924&#226;&#8364;&#8220;1928. <br />     TY, Ruan SK, et al: Functional characterization cellulases identified   43. Henriksson G, Nutt  Henriksson H, Pettersson B, Stahlberg J, Johansson G, <br />     cow rumen fungus Neocallimastix patriciarum W5 transcriptomic                Pettersson G: Endoglucanase 28 (Cel12A), new Phanerochaete <br />     secretomic analyses. Biotechnology biofuels 2011, 4(1):24.                  chrysosporium cellulase. Eur J Biochem 1999, 259(1&#226;&#8364;&#8220;2):88&#226;&#8364;&#8220;95. <br /> 22. Pham TH, Quyen DT, Nghiem NM, Vu TD: Cloning, expression, purification,         44. Huy ND, Kim SW, Park SM: Heterologous expression endo-1,4-beta- <br />     properties endoglucanase gene (glycosyl hydrolase family 12)              xylanaseC Phanerochaete chrysosporium Pichia pastoris. J Biosci <br />     Aspergillus niger VTCC-F021 Pichia pastoris. J Microbiol Biotechnol         Bioeng 2011, 111(6):654&#226;&#8364;&#8220;657. <br />     2011, 21(10):1012&#226;&#8364;&#8220;1020.                                                         45. Munoz IG, Ubhayasekera W, Henriksson H, Szabo  Pettersson G, Johansson <br /> 23. Gary JS: Trichoderma: review biology systematics genus.             G, Mowbray SL, Stahlberg J: Family 7 cellobiohydrolases     Mycol Res 1996, 100(8):923&#226;&#8364;&#8220;935.                                                     Phanerochaete chrysosporium: crystal structure catalytic module 24. Teeri TT, Koivula  Linder M, Wohlfahrt G, Divne C, Jones TA: Trichoderma          Cel7D (CBH58) 1.32 resolution homology models     reesei cellobiohydrolases: efficient crystalline cellulose?               isozymes. Journal molecular biology 2001, 314(5):1097&#226;&#8364;&#8220;1111. <br />     Biochem Soc Trans 1998, 26(2):173&#226;&#8364;&#8220;178.                                          46. Wymelenberg AV, Denman S, Dietrich D, Bassett J, Yu X, Atalla R, Predki P, <br /> 25. Montenecourt BS: Trichoderma reesei cellulases. Trends Biotechnol 1983,             Rudsander U, Teeri TT, Cullen D: Transcript analysis genes encoding     1(5):156&#226;&#8364;&#8220;161.                                                                       family 61 endoglucanase putative membrane-anchored family 9 <br /> 26. Sinnott ML: cellobiohydrolases Trichoderma reesei: review               glycosyl hydrolase Phanerochaete chrysosporium. Appl Environ <br />     indirect direct evidence function just glycosidic             Microbiol 2002, 68(11):5765&#226;&#8364;&#8220;5768. <br />     bond hydrolysis. Biochem Soc Trans 1998, 26(2):160&#226;&#8364;&#8220;164.                         47. Wang TY, Chen HL, Li WH, Sung HM, Shih MC: Omics applications 27. Muller S, Sandal T, Kamp-Hansen P, Dalboge H: Comparison expression              biofuel research. Biocatalysis Biomolecular Engineering. Edited Hou <br />     systems yeasts Saccharomyces cerevisiae, Hansenula polymorpha,               CT, Shaw JF. New York: John Wiley &amp; Sons,  2010:265&#226;&#8364;&#8220;276. <br />     Klyveromyces lactis. Schizosaccharomyces pombe Yarrowia lipolytica.         48. Suzuki H, Imaeda T, Kitagawa T, Kohda K: Deglycosylation cellulosomal <br />     Cloning novel promoters Yarrowia lipolytica. Yeast 1998,                enzyme enhances cellulosome assembly Saccharomyces cerevisiae. <br />     14(14):1267&#226;&#8364;&#8220;1283.                                                                   J Biotechnol 2012, 157(1):64&#226;&#8364;&#8220;70. <br /> 28. Aho S, Arffman  Korhola M: Saccharomyces cerevisiae mutants selected          49. Hou J, Tyo KEJ, Liu ZH, Petranovic D, Nielsen J: Metabolic engineering     increased production Trichoderma reesei cellulases. Appl Microbiol           recombinant protein secretion Saccharomyces cerevisiae. FEMS Yeast <br />     Biotechnol 1996, 46(1):36&#226;&#8364;&#8220;45.                                                       Res 2012, 12(5):491&#226;&#8364;&#8220;510. <br /> 29. Qin Y, Wei X, Liu X, Wang T, Qu Y: Purification characterization         50. Idiris  Tohda H, Kumagai H, Takegawa K: Engineering protein <br />     recombinant endoglucanase Trichoderma reesei expressed                        secretion yeast: strategies impact protein production. <br />     Saccharomyces cerevisiae higher glycosylation stability.                   Appl Microbiol Biotechnol 2010, 86(2):403&#226;&#8364;&#8220;417. <br />     Protein Expr Purif 2008, 58(1):162&#226;&#8364;&#8220;167.                                         51. Mattanovich D, Branduardi P, Dato L, Gasser B, Sauer M, Porro D: <br /> 30. Wildt S, Gerngross TU: humanization N-glycosylation pathways              Recombinant protein production yeasts. Methods Mol Biol 2012, <br />     yeast. Nat Rev Microbiol 2005, 3(2):119&#226;&#8364;&#8220;128.                                        824:329&#226;&#8364;&#8220;358. <br /> 31. Jigami Y: Yeast glycobiology application. Biosci Biotechnol Biochem     52. Kitagawa T, Kohda K, Tokuhiro K, Hoshida H, Akada R, Takahashi H, Lmaeda <br />     2008, 72(3):637&#226;&#8364;&#8220;648.                                                                T: Identification genes enhance cellulase protein production 32. Jeoh T, Michener W, Himmel  Decker SR, Adney WS: Implications                 yeast. J Biotechnol 2011, 151(2):194&#226;&#8364;&#8220;203. <br />     cellobiohydrolase glycosylation use biomass conversion.                  53. Bartkeviciute D, Sasnauskas K: Disruption MNN10 gene enhances <br />     Biotechnology biofuels 2008, 1(1):10.                                           protein secretion Kluyveromyces lactis Saccharomyces cerevisiae. <br /> 33. Kruszewska JS, Butterweck AH, Kurzatkowski W, Migdalski  Kubicek CP,              FEMS Yeast Res 2004, 4(8):833&#226;&#8364;&#8220;840. <br />     Palamarczyk G: Overexpression Saccharomyces cerevisiae                   54. Nunes Bastos R: Functional dissection alternative secretory pathways     mannosylphosphodolichol synthase-encoding gene Trichoderma reesei                yeast S. cerevisiae. University Helsinki: Dissertationes Biocentri Viikki <br />     results increased level protein secretion abnormal cell                Universitatis Helsingiensis; 2008. <br />     ultrastructure. Appl Environ Microbiol 1999, 65(6):2382&#226;&#8364;&#8220;2387.                   55. Chen HL, Chen YC, Lu MYJ, Chang JJ, Wang HTC, Ke HM, Wang TY, Ruan SK, <br /> 34. Lynd LR, Weimer PJ, van Zyl WH, Pretorius  Microbial cellulose                   Wang TY, Hung KY, et al: highly efficient &#195;&#376;-glucosidase buffalo <br />     utilization: fundamentals biotechnology. Microbiol Mol Biol Rev 2002,           rumen fungus Neocallimastix patriciarum W5. Biotechnol Biofuels 2012, 5(1):24. <br />     66(3):506&#226;&#8364;&#8220;577. table contents.                                               56. Helenius  Aebi M: Roles N-linked glycans endoplasmic <br /> 35. Van Rensburg P, Van Zyl WH, Pretorius  Engineering yeast efficient           reticulum. Annu Rev Biochem 2004, 73:1019&#226;&#8364;&#8220;1049. <br />     cellulose degradation. Yeast 1998, 14(1):67&#226;&#8364;&#8220;76.                                 57. Lehle L, Strahl S, Tanner W: Protein glycosylation, conserved yeast 36. Liu YS, Baker JO, Zeng Y, Himmel  Haas T, Ding SY: Cellobiohydrolase             man: model organism helps elucidate congenital human diseases. <br />     hydrolyzes crystalline cellulose hydrophobic faces. J Biol Chem 2011,            Angew Chem Int Ed Engl 2006, 45(41):6802&#226;&#8364;&#8220;6818. <br />     286(13):11195&#226;&#8364;&#8220;11201.                                                            58. Toikkanen J: Functional studies components secretory pathway 37. Khan MH, Ali S, Fakhru&#226;&#8364;&#8482;l-Razi  Alam  Use fungi bioconversion         Saccharomyces cerevisiae, Ph.D. thesis. Helsinki, Finland: University Helsinki; <br />     rice straw cellulase enzyme. Journal environmental science           1999. <br />     health 2007, 42(4):381&#226;&#8364;&#8220;386.                                                59. Huffaker TC, Robbins PW: Temperature-sensitive yeast mutants deficient <br /> 38. Broda P, Birch PR, Brooks PR, Sims PF: Lignocellulose degradation                asparagine-linked glycosylation. J Biol Chem 1982, 257(6):3203&#226;&#8364;&#8220;3210. <br />     Phanerochaete chrysosporium: gene families gene expression            60. Herscovics  Orlean P: Glycoprotein biosynthesis yeast. FASEB J 1993, <br />     complex process. Mol Microbiol 1996, 19(5):923&#226;&#8364;&#8220;932.                                 7(6):540&#226;&#8364;&#8220;550. <br />  Wang et al. BMC Biotechnology 2013, 13:71                                                                                                     Page 14 14 <br /> http://www.biomedcentral.com/1472-6750/13/71 <br />  <br />  <br />  <br />  <br /> 61. Teixeira MC, Raposo LR, Mira NP, Lourenco AB, Sa-Correia  Genome-wide <br />     identification Saccharomyces cerevisiae genes required maximal <br />     tolerance ethanol. Appl Environ Microbiol 2009, 75(18):5761&#226;&#8364;&#8220;5772. <br /> 62. Proszynski TJ, Simons K, Bagnat M: O-glycosylation sorting <br />     determinant cell surface delivery yeast. Mol Biol Cell 2004, <br />     15(4):1533&#226;&#8364;&#8220;1543. <br /> 63. Gentzsch M, Tanner W: PM <br /> </body></html>