<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  Developmentally Essential Protein Flightless Nuclear Receptor <br />                Coactivator Actin Binding Activity <br />  <br />  <br />             Hormone-activated nuclear receptors (NR) activate transcription recruiting multiple coactivator com- <br />           plexes promoters target genes. important coactivator complex includes p160 coactivator (e.g., <br />           GRIP1, SRC-1, ACTR) binds directly activated NR, histone acetyltransferase p300 CBP,           arginine-specific histone methyltransferase CARM1. previously demonstrated coactivator <br />           function CARM1 depends methyltransferase activity additional unknown proteins           bind CARM1. study yeast hybrid screen proteins bind CARM1 identified protein <br />           Flightless (Fli , essential roles Drosophila mouse development. Fli bound CARM1, <br />           GRIP1, NRs cooperated synergistically CARM1 GRIP1 enhance NR function. Fli bound <br />           poorly did cooperate PRMT1, CARM1-related protein arginine methyltransferase           functions NR coactivator. synergy GRIP1, CARM1, Fli required methyltransferase <br />           activity CARM1. C-terminal AD1 (binding site p300/CBP) AD2 (binding site CARM1) <br />           activation domains GRIP1 contributed synergy stringently required N-terminal <br />           region GRIP1, binding site Fli  Endogenous Fli recruited estrogen-regulated <br />           pS2 gene promoter MCF-7 cells response hormone, reduction endogenous Fli levels           small interfering RNA reduced hormone-stimulated gene expression endogenous estrogen receptor.           fragment Fli related actin binding protein gelsolin enhanced estrogen receptor activity,           mutations reduced actin binding reduced coactivator function Fli fragment. data <br />           suggest Fli facilitate interaction p160 coactivator complex coactivators           coactivator complexes containing actin actin-like proteins. <br />  <br />  <br />    nuclear hormone receptors (NRs) family hor-                  driven developmental processes mice lacking functional <br /> mone-activated transcriptional activator proteins,                  p160 genes (24, 67, 68). <br /> clude receptors steroid hormones, <span id='am-84' about='obo:IMR_0100042' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000966 obo:IMR_0000001'><span id='am-85' property="oboInOwl:id" content="IMR:0100042" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="thyroid hormone" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="MeSH:D013963" datatype="xsd:string"></span>thyroid hormone</span>,                      Coactivators generally types functional domains, <br /> <span id='am-20' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span>vitamin</span> D, retinoids (3, 44, 62). NRs bind directly                   e., anchor coactivator promoter proteins specific enhancer elements associ-                 called activation domains transmit activating signal <br /> ated promoters target genes activate              transcription machinery. Activation domains repress transcription. Transcriptional activation involves               propagate activating signal catalyzing covalent modifi- <br /> recruitment various coactivator proteins cause local                 cations histones proteins transcription <br /> remodeling chromatin promoter region help                   machinery facilitating transcription initiation recruit RNA polymerase II (25, 28, 46).                                      direct protein-protein interactions anchor coactiva- <br />    coactivators exist function complexes                tors promoter recruit activate components multiple subunits collaborate accomplish single step               transcription machinery. p160 coactivators teth- transcriptional activation perform multiple functions             ered promoter NR box motifs <br /> cooperate synergistic manner. example, relatively               consensus sequence LXXLL  L represents characterized functional coactivator complex organized               leucine X represents <span id='am-56' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-57' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-61' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span>) (19, 27, 60, 63). p160 coactivators, include SRC-1, GRIP1/                    motifs form amphipathic alpha-helices insert TIF2, pCIP/ACTR/RAC3/AIB1/TRAM1. functional                          conserved hydrophobic groove located AF-2 activation physiological importance p160 coactivators NR                   function ligand binding domains hormone-acti- <br /> diated transcriptional activation established               vated NRs (15, 48). NR Box motifs located approxi- <br /> transient-transfection assays (25, 35, 49), vitro transcription           mately middle 1,400-amino-acid polypeptide <br /> systems (40), loss NR function cells injected p160                chain p160 coactivators. <span id='am-6' about='oboInOwl:date' typeof='owl:Thing'>date</span> conserved activation <br /> antibodies (60), specific deficiencies nuclear receptor-              domains located near C terminus p160 coactivators                                                                              identified binding sites additional coactivators: <br />                                                                              AD1 binds histone acetyltransferase p300 CBP (12, 60, <br />   * Corresponding author. Mailing address: Department Pathology, <br /> HMR 301, University Southern California, 2011 Zonal Ave., <br />                                                                              63), AD2 binds histone methyltransferase CARM1 <br /> Los Angeles, CA 90089-9092. Phone: (323) 442-1289. Fax: (323) 442-           (11). recruitment coactivator function CARM1 3049. E-mail: stallcup@usc.edu.                                              p300  CBP) apparently depends presence p160 <br />  <br />                                                                       2103 <br />  2104      LEE ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br /> coactivators (10, 11, 37);  refer sec-                  GRIP1N (amino acids 5 765), GRIP1 M (amino acids 730 1121), GRIP1C <br /> ondary coactivators. addition AD1 AD2,                      (amino acids 1122 1462) (11), GRIP1&#226;&#338;&#172;AD1 (lacking amino acids 1057                                                                                 1109), GRIP1&#226;&#338;&#172;AD2 (lacking amino acids 1122 1462) (42), GRIP1&#226;&#338;&#172;N (lacking undefined activation domains p160 coactivators;                     amino acids 1 562) (J. H. Kim, H. Li, M. R. Stallcup, submitted example, N-terminal region highly conserved                     publication), rat PRMT1 (32), CARM1(E267Q) Glu267 changed p160 coactivators (2) presently                      Gln (35). following mammalian cell expression vectors cytomegalovirus <br /> known role coactivator function.                                             promoters used: pCMV-p300 encoding length p300 (51)    CARM1 belongs protein arginine methyltransferase                      pcDNA3.FliI containing SalI-BamHI cDNA fragment encoding length <br />                                                                                 human Fli inserted XhoI BamHI sites pcDNA3. leucine- <br /> family, characterized highly conserved arginine-                  rich repeat (LRR) region (amino acids 1 494) gelsolin-like region (amino <br /> specific methyltransferase domain (71). mem-                    acids 495 1268) human Fli amplified PCR flanking MfeI bers mammalian family proteins methylates differ-                   SalI sites, digested enzymes, inserted EcoRI ent set protein substrates (23, 57). example, CARM1                      XhoI sites pSG5.Flag; pSG5.Flag derived pSG5.Flag-TEF4 (5). <br /> methylates histone H3 (11), p300/CBP (69), RNA                          Mammalian expression vectors pSG5.flag-Gel (human Fli amino acids <br />                                                                                 495 827), pSG5.flag-Gel G602S), pSG5.flag-Gel E585K) binding protein HuR (36), PRMT1 methylates histone                        generated PCR site-directed mutagenesis human Fli  <br /> H4 RNA binding proteins distinct HuR (57).                     serted EcoRI XhoI sites. expression vector HA-tagged recruitment promoters <span id='am-73' about='obo:IMR_0100033' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000966 obo:IMR_0000001'><span id='am-74' property="oboInOwl:hasDbXref" content="CHEBI:26764" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="steroid hormone" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:id" content="IMR:0100033" datatype="xsd:string"></span>steroid hormone</span>-                      BAF53 (50) kindly provided M. Cole (Princeton University). <br /> inducible genes, CARM1 methylates histone H3 local                          Mammalian expression vectors pHE0 encoding human estrogen receptor <br /> promoter region process transcriptional acti-                 (ER) &#226;&#65533;&#163; pCMX.hTR&#226;&#65533;&#164;1 encoding human thyroid hormone receptor (TR) &#226;&#65533;&#164;1 <br />                                                                                 described previously, reporter plasmids MMTV(ERE)-LUC vation (41). transient-transfection assays CARM1,                    ER, MMTV(TRE)-LUC TR, GK1 Gal4 (11). Mammalian expression <br /> PRMT1, cooperates synergistically p300  CBP)                        vector pM.LRR encoding Gal4 DNA-binding domain (DBD) fused p160 coactivators enhance transcriptional activation NRs                  LRR region Fli (amino acids 1 494) constructed inserting (35). binding interface CARM1 p160 pro-                         PCR-amplified cDNA fragment flanked EcoRI SalI sites anal- <br /> teins involves undefined surface CARM1 methyl-                        ogous sites pM (Clontech). <br />                                                                                    Bacterial expression vectors encoding <span id='am-25' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-26' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-36' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span>glutathione</span> S-transferase (GST) fused <br /> transferase domain C-terminal AD2 domain p160                        GRIP1 fragments described previously: GRIP1N (amino acids 5 765) <br /> coactivators (11, 58). Transmission activating signal                 (30); GRIP1 M (amino acids 730 1121) (29); GRIP1C (amino acids 1122 transcription machinery CARM1 involves methyl-                       1462) (42). GST fused ER encoded pGEX5X-1.HEG0 (65). Yeast <br /> transferase activity (11, 35, 41) additional domains,               expression vector pGBT9.CARM1 encoding Gal4 DBD fused length unique C-terminal N-terminal domains,                     CARM1 (amino acids 3 608) contained EcoRI insert encoding CARM1. <br />                                                                                    Protein-protein interaction assays. following procedures performed <br /> involved methyltransferase activity (58). result sug-               described previously: yeast hybrid screening (11); production GST <br /> gests CARM1 transmits activating signal coop-                   fusion proteins bacteria binding labeled proteins translated vitro erative mechanisms, e., protein methylation protein-pro-                  GST fusion proteins bound <span id='am-68' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-69' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span>glutathione</span>-agarose beads (32); coimmuno- <br /> tein interactions.                                                              precipitation immunoblot experiments (35) anti-Flag (Sigma F3165), <br />    propose additional, presently undefined, pro-                   anti-ER anti-TR (Santa Cruz), anti-HA (Roche 3F10) antibodies (1 &#226;&#65533;&#174;g <br />                                                                                 antibody immunoprecipitation 1:1,000 dilutions immunoblots). <br /> teins likely participate transcriptional activation                      Cell culture transient-transfection assays. Reporter gene activation  protein-protein interactions GRIP1 CARM1.                      says performed transient transfection CV-1 MCF-7 cells identified Flightless (Fli  yeast hybrid                   dishes described previously (35). indicated, 20 nM estradiol (E2) screen (22) proteins interact CARM1                      2,5,5&#226;&#172;&#732;-triiodo-L-thyronine (T3) added transfected cultures cooperate coactivators. Drosophila, par-                40 h growth. <br />                                                                                    <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>Peptide</span> antibody production Fli  peptide representing amino acids <br /> tial loss function mutations fliI result loss flight              1232 1251 unique portion gelsolin-like region Flightless ability, severer mutations lead impaired cellularization               (human) synthesized Tufts University Microchemical Core, rabbit gastrulation embryo (8). Fli highly conserved                 antiserum peptide (&#226;&#65533;&#163;-Fli-C) generated Zymed. <br /> protein diverse vertebrate invertebrate species (7,                      ChIP assay. Chromatin immunoprecipitation (ChIP) assays ware performed <br /> 8). human gene FLII mapped region human                          according previously described protocols (41, 43, 56) minor modifica- <br />                                                                                 tions. MCF-7 cells cultured 4 days phenol red-free Dulbecco&#226;&#8364;&#8482;s <br /> chromosome 17p, associated Smith-Magenis syn-                     modified Eagle medium supplemented 5% charcoal-dextran-stripped fetal <br /> drome, genetic disease causing developmental behav-                       bovine serum. Cells approximately 90 95% confluency treated 100 <br /> ioral abnormalities (13). clear                     nM E2 vehicle 45 min <span id='am-63' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-64' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>formaldehyde</span> cross-linking chromatin <br /> Smith-Magenis syndrome haploinsufficiency FLII,                    preparation. Immunoprecipitation sonicated chromatin solutions  <br /> homozygous disruption orthologous Fliih gene mice                     ducted overnight incubation 4&#194;&#176;C appropriate antibody.                                                                                 cross-linking immunoprecipitated chromatin reversed, DNA purifica- <br /> leads early failure embryonic development (6). Fli                 tion achieved phenol-chloroform extraction ethanol precipitation. actin binding protein associated                  purified DNA dissolved 50 &#226;&#65533;&#174;l 10 mM Tris-HCl, pH 8.0&#226;&#8364;&#8220;1 mM cytoskeleton nucleus, exact physiological                 EDTA, 1 &#226;&#65533;&#174;l used PCR analysis. ChIP reimmunopre- <br /> functions elucidated (16, 17, 26). iden-                 cipitation  IP) assay, immunoprecipitated fractions eluted incu- <br /> tified Fli CARM1 binding protein, investigated                    bating 30 min 37&#194;&#176;C IP elution buffer 10 mM dithiothreitol (43).                                                                                 eluates diluted 50-fold dilution buffer (1% Triton X-100, 2 mM EDTA, <br /> ability bind collaborate CARM1 NR                      150 mM NaCl, 20 mM Tris-HCl, pH 8.1) reprecipitated specific <br /> coactivators mammalian cells.                                                antibodies. Antibodies used ChIP assays obtained Santa Cruz <br />                                                                                 Biotechnology (normal mouse immunoglobulin G [IgG], normal rabbit IgG; ER, <br />                                                                                 AIB1, p300, &#226;&#65533;&#163;-Fli-N Fli LRR region) Upstate Biotechnology <br />                        MATERIALS METHODS                                    (CARM1). primers used detecting portions pS2 gene PCR    Plasmids. Mammalian cell expression vector pSG5.HA (11), simian    listed 5&#226;&#172;&#732; 3&#226;&#172;&#732; <span id='am-48' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-49' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-55' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> numbers representing distance <br /> virus 40 T7 promoters, used express proteins N-terminal      transcription start site: human pS2 promoter region (&#226;&#171;&#186;31 hemagglutinin (HA) epitope tag. Plasmids encoding following proteins   &#226;&#171;&#186;353), GGCAGGCTCTGTTTGCTTAAAGAGCG GGCCATCTCTCAC <br /> previously constructed pSG5.HA: mouse CARM1, mouse GRIP1,                  TATGAATCACTTCTGC; pS2 coding region (&#226;&#171;&#185;871 &#226;&#171;&#185; 1200), <br />  VOL. 24, 2004                                                                        NUCLEAR RECEPTOR COACTIVATOR Flightless                        2105 <br />  <br /> TGCCAGCTGTGGGGAGCTGAATAACTT CAGTTCGTTCTGTACA <br /> CCGAGGCCACT. <br />    siRNA. Small interfering RNA (siRNA) experiments performed accord- <br /> ing previously published methods (55) minor modifications. Single- <br /> stranded RNAs synthesized University Southern California Norris <br /> Comprehensive Cancer Center Microchemical Core Laboratory. Equimolar  <br /> centrations complementary RNAs incubated annealing buffer <br /> (100 mM potassium acetate, 30 mM HEPES-KOH, pH 7.4, 2 mM magne- <br /> sium acetate) 90&#194;&#176;C 1 min 37&#194;&#176;C 1 h form duplex. siRNA <br /> duplexes transfected MCF-7 cells using Lipofectamine 2000 Oligofectamine (Invitrogen) according manufacturer&#226;&#8364;&#8482;s protocol. siRNA sequences used (listed 5&#226;&#172;&#732; 3&#226;&#172;&#732;) follows: si-FliA, GCUGGAAC <br /> ACUUGUCUGUGdTdT CACAGACAAGUGUUCCAGCdTdT; si-FliB, <br /> CAACCUGACCACGCUUCAUdTdT AUGAAGCGUGGUCAGGUUG <br /> dTdT; si-CARM1, GCAGUCCUUCAUCAUCACCdTdT GGUGAUGAU <br /> GAAGGACUGCdTdT; si-AIB1 (55), AGACUCCUUAGGACCGCUUdTdT AAGCGGUCCUAAGGAGUCUdTdT. <br />  <br />  <br />                                  RESULTS <br />    Identification Fli CARM1 binding protein yeast hybrid screening. Bait plasmid pGBT9-CARM1, encoding length CARM1 fused Gal4 DBD, 17-day-old <br /> mouse embryo cDNA library Gal4 activation domain fusion <br /> vector pGAD10 sequentially transformed yeast <br /> strain Hf7c, contains yeast his3 gene lacZ gene control Gal4 response elements. Selection 1.1 <br /> &#226;&#171;&#187; 106 transformed yeast cells agar plates lacking histidine <br /> resulted growth 126 colonies, 25 clones                       FIG. 1. Binding CARM1 Fli vitro vivo.   subsequently confirmed positive testing expres-                      LRR region (vertical stripes) gelsolin-like motifs (black boxes) <br /> sion &#226;&#65533;&#164;-galactosidase. 25 clones encoded                              Fli protein indicated. (B) vitro binding assay. [35S]Fli                                                                                    protein synthesized vitro incubated GST GST-CARM1 <br /> CARM1, consistent previous structural func-                      immobilized glutathione-agarose beads; bound proteins tional studies indicating protein arginine methyltrans-                   eluted analyzed sodium dodecyl sulfate-polyacrylamide gel <br /> ferases exist homodimers (45, 71) indicates                       electrophoresis autofluorography. portion vitro trans- <br /> validity bait yeast hybrid screening condi-                     lated Fli protein incubation beads shown left <br />                                                                                    (Input, 10%). position 145-kDa length Fli protein tions identification protein partners CARM1.                             indicated. (C) Coimmunoprecipitation assay. pSG5.Flag-FliI(LRR)    positive cDNA clones, encoded                            pSG5.Flag-FliI(gelsolin) (2.5 &#226;&#65533;&#174;g) transiently transfected N-terminal fragment 1,268-amino acid mouse Fli pro-                       pSG5.HA-CARM1 pSG5.HA-PRMT1 (2.5 &#226;&#65533;&#174;g) Cos-7 <br /> tein (extending codon 480). used length human                         cells. Immunoprecipitation (IP) performed transfected-cell ex- <br /> cDNA clone (8) studies described  N-termi-                       tracts antibodies Flag epitope, precipitated <br />                                                                                    proteins analyzed immunoblotting (W) anti-HA antibod- <br /> nal region Fli contains LRR domain consisting 16                        ies (upper panel). sample transfected-cell extract tandem copies 23-amino-acid leucine-rich motif (Fig. 1A)                      immunoprecipitation (2% volume used immunoprecipi- <br /> (8, 31). LRR domains implicated protein-protein                       tation shown lane 3) shown lane 1. relative expression interactions. central C-terminal regions Fli                      HA-CARM1 HA-PRMT1 immunoprecipitation shown <br />                                                                                    lower panel. Results representative independent <br /> polypeptide chain homologous gelsolin family                         experiments. LRR, LRR fragment Fli  GEL, gelsolin-like frag- <br /> actin binding proteins (34). Fli binds actin colocalizes                     ment Fli  IgG, position immunoglobulin heavy chain actin cytoskeleton certain conditions                       immunoblot. localize nucleus conditions (16, 17). studies suggest Fli nuclear function addition involved signal-regulated remodeling                     (lane 2), length Fli enriched coupling proteins actin actin cytoskel-                   smaller bands input CARM1-bound fractions <br /> eton. identified Fli hybrid screen                    compared. data suggest Fli binds directly ability bind NR coactivator CARM1, confirmed                         CARM1. <br /> binding methods investigated ability Fli                     achieve efficient expression test proper- <br /> cooperate CARM1 NR coactivators enhanc-                          ties specific Fli subdomains, Fli subsequently divided <br /> ing transcriptional activation NRs.                                             separate fragments: amino acids 1 494, containing <br />    Specific binding Fli CARM1 NRs vitro                      LRR region; amino acids 495 1268, containing vivo. Fli protein translated vitro tested ability              gelsolin-like domains. test interactions vivo, Fli bind GST-CARM1 immobilized glutathione-agarose beads                            fragments fused Flag epitope expressed transient <br /> (Fig. 1B). reported previously (17), production                         transfection Cos-7 cells HA epitope-tagged <br /> length (145-kDa) Fli relatively inefficient                   CARM1 coimmunoprecipitation experiments  <br /> (lane 1). relatively weak specific binding                        formed transfected-cell extracts using antibodies length Fli GST-CARM1 (lane 3), compared GST                         Flag epitope subsequently analyzing im- <br />  2106     LEE ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br /> munoprecipitates immunoblot antibodies HA epitope (Fig. 1C, upper panel). CARM1 ex- <br /> pressed Fli fragment, detected immunoprecipitates (lane 3);  CARM1 coimmuno- <br /> precipitated N-terminal LRR domain Fli (lane 4) C-terminal gelsolin-like domain Fli (lane 7), <br /> suggesting presence direct indirect <br /> CARM1 binding sites Fli protein. <br />    test specificity interaction Fli CARM1, member protein arginine methyltrans- <br /> ferase family, PRMT1 (38), substituted CARM1. substantial homology CARM1 PRMT1 320-amino-acid methyltransferase domains (71), proteins function NR coactivators cooperation p160 coactivators (32). PRMT1 specifically coprecipitated fragments Fli (Fig. 1C, upper panel, lanes 5, 6, 8) weakly CARM1 (lanes 4 7). <br /> Immunoblots performed anti-HA antibodies samples transfected-cell extracts taken immunoprecipita- <br /> tion showed CARM1 PRMT1 expressed sim- <br /> ilar levels (Fig. 1C, lower panel). <br />    Fli binds NR coactivator CARM1, tested Fli bind NRs. Unlike length Fli (Fig. <br /> 1B), 35S-labeled LRR gelsolin-like domains Fli synthesized efficiently vitro (Fig. 2A, lanes 1 5). LRR gelsolin-like fragments did bind GST (lanes 2 6) bound weakly GST-ER fusion protein absence E2 (lanes 3 7). interaction Fli fragments GST-ER strongly enhanced E2 (lanes 4 8). Coimmunoprecipitation experiments used test interactions Cos-7 cells transiently expressing <br /> ER Flag-tagged Fli fragment. Fli frag- <br /> ments ER expressed  anti-Flag antibodies did          FIG. 2. Binding NRs Fli vitro vivo.   vitro <br /> precipitate ER (Fig. 2B, upper panel, lanes 1 2).       binding ER. LRR gelsolin-like fragments Fli trans- <br />                                                                     lated vitro tested binding bead-bound GST ER expressed LRR fragment gelso-           GST-ER absence presence E2. (B) Coimmunoprecipita- <br /> lin-like fragment Fli  ER coprecipitated anti-Flag      tion ER. Flag-tagged LRR gelsolin-like fragments Fli (2.5 <br /> antibody (lanes 3 6).  contrast binding    &#226;&#65533;&#174;g plasmid) coexpressed ER (2.5 &#226;&#65533;&#174;g plasmid) Cos-7 <br /> vitro (Fig. 2A), association ER Fli frag-     cells presence absence E2. Immunoprecipitation (IP) ments vivo occurred presence absence E2 (Fig.       anti-Flag antibody followed immunoblotting (W) anti-ER <br />                                                                     antibody (upper panel). Expression ER transfected-cell ex- <br /> 2B, upper panel, lanes 3 6). binding Fli          tracts immunoprecipitation (input) examined immuno- <br /> fragments ER hormone dependent vitro           blotting anti-ER antibodies (lower panel). (C) Coimmunoprecipi- <br /> vivo, additional cellular factors stabilize interaction     tation TR CARM1. Coimmunoprecipitation Fli ER Fli vivo, especially absence          fragments TR (2.5 &#226;&#65533;&#174;g plasmid) HA-tagged CARM1 (2.5 &#226;&#65533;&#174;g <br />                                                                     plasmid) tested panel B presence absence <br /> hormone. Alternatively, overexpression proteins       T3, anti-TR (lanes 1 6) anti-HA (lanes 7 8) transient transfections cause hormone-independent         used immunoprecipitation, Fli fragments detected association.                                                        subsequent immunoblot (W) anti-Flag antibodies. Expression <br />    Interaction Fli fragments NR, TR,      LRR gelsolin-like fragments Fli transfected-cell tested coimmunoprecipitation; case, antibodies      extracts examined immunoprecipitation (input) immu- <br />                                                                     noblot anti-Flag antibodies (lower panel). TR used immunoprecipitation, antibodies Flag epitope tag used visualize copre- <br /> cipitated Fli fragments. ER, TR coprecipitated LRR fragment gelsolin-like fragment Fli       test endogenous Fli protein recruited NR- hormone-independent manner (Fig. 2C, upper panel,              regulated promoter response appropriate hormone. <br /> lanes 1 6). association Fli fragments          purpose, antibodies Fli  commer- <br /> CARM1 used positive control (lanes 7 8).               cially available mouse monoclonal LRR region    Recruitment endogenous Fli protein steroid hor-        custom rabbit peptide-directed antiserum mone-regulated promoter response hormone. asso-           unique portion gelsolin-like region, shown im- <br /> ciation Fli NRs NR coactivators suggests      munoblot immunoprecipitation assays recognize specif- Fli participate NR-dependent transcriptional      ically Fli protein (data shown; Fig. 7B). <br /> regulation process.  ChIP assays performed        Endogenous Fli protein detected MCF-7 breast cancer <br />  VOL. 24, 2004                                                           NUCLEAR RECEPTOR COACTIVATOR Flightless                      2107 <br />  <br />  <br /> cells (data shown), contain estrogen-inducible <br /> pS2 gene. MCF-7 cells treated 100 nM E2 left untreated 45 min analyzed ChIP <br /> analysis, ER protein recruited pS2 promoter hor- <br /> mone-dependent manner (Fig. 3A B). Similarly, hormone- <br /> dependent recruitment Fli protein pS2 promoter <br /> region observed 0.5 &#226;&#65533;&#174;g mouse monoclonal <br /> antibody (Fig. 3A) 5 &#226;&#65533;&#174;l rabbit antiserum (Fig. 3B) Fli used;  twice anti-Fli antibody used, Fli protein appeared  <br /> stitutively associated pS2 promoter (Fig. 3A, lanes, B, shown). results suggest small <br /> amounts Fli protein associated pS2 promoter absence hormone promoter occupancy Fli increased E2 treatment. Normal IgG served negative control immunoprecipitations demonstrate specificity Fli antibodies (Fig. 3A B). PCR primers recognizing portion coding sequence <br /> (nucleotides 871 1200, relative transcription start site) pS2 gene used test chromatin fractions <br /> immunoprecipitated antibodies ER Fli  amplification products observed (Fig. 3B, panel), indicating recruitment ER Fli specific pS2 promoter, coding region pS2 gene.  ER Fli appear transcription <br /> initiation complex appear travel RNA <br /> polymerase progresses coding region. recruitment Fli protein pS2 promoter binding Fli ER coactivators, GRIP1 CARM1. <br />    ChIP data indicate ER Fli recruited pS2 promoters cell culture response hormone, indicate Fli ER molecules present simultaneously individual pS2 promoter. <br /> ChIP IP assays used answer question:  ChIP protocol performed anti-ER antibody; precipitated ER-containing chromatin fractions                  FIG. 3. Estrogen-dependent recruitment endogenous Fli eluted antibody-bead complex dithiothreitol,            endogenous pS2 promoter MCF-7 cells.  B) ChIP assay. <br /> diluted, reimmunoprecipitated using different          MCF-7 cells incubated E2 hormone 45 min. <br /> antibodies Fli antibodies known ER-            Extracted chromatin incubated antibodies indi- <br />                                                                       cated proteins, DNA purified immunoprecipitated (IP) <br /> associated coactivator. DNA recovered second             fractions analyzed PCR primers pS2 promoter immunoprecipitated pellet analyzed PCR                coding region. Mouse monoclonal antibody Fli LRR <br /> primers pS2 promoter (Fig. 3C).         region (&#226;&#65533;&#163;-Fli-N) used 0.5 1 &#226;&#65533;&#174;g assay, rabbit antiserum <br /> analysis, ER shown simultaneously localized           peptide gelsolin-like region Fli (&#226;&#65533;&#163;-Fli-C)                                                                       used 5 &#226;&#65533;&#174;l assay, antibodies used 1 &#226;&#65533;&#174;g dividual hormone-activated pS2 promoters Fli        assay, including normal mouse rabbit IgG. results presented known ER coactivators, AIB1, CARM1,            representative seven independent experiments. (C) Simul- <br /> p300. ChIP IP procedure performed                 taneous binding pS2 promoter ER coactivators. anti-Fli antibody immunoprecipitation,       Chromatin hormone-treated untreated MCF-7 cells im- <br />                                                                       munoprecipitated anti-ER antibody; immunoprecipitates <br /> Fli shown coincide ER, AIB1, CARM1               eluted beads immunoprecipitated individual hormone-activated pS2 promoters, CARM1              antibodies specified proteins. Amounts antibodies used: <br /> localized Fli hormone-independent promoters            0.5 &#226;&#65533;&#174;g &#226;&#65533;&#163;-Fli-N, 5 &#226;&#65533;&#174;l &#226;&#65533;&#163;-Fli-C, 1 &#226;&#65533;&#174;g antibodies. <br /> (Fig. 3D). high levels Fli antibodies used     DNA purified immunoprecipitated fractions analyzed                                                                       PCR primers pS2 promoter. (D) Simultaneous binding experiment (Fig. 3D), hormone-independent binding            pS2 promoter Fli ER coactivators. MCF-7 cell Fli CARM1 pS2 promoter observed               chromatin (E2 treated untreated) immunoprecipitated experiment consistent results observed high          1 &#226;&#65533;&#174;g &#226;&#65533;&#163;-Fli-N; immunoprecipitates eluted beads levels Fli antibodies standard ChIP assays (Fig. 3A).         immunoprecipitated 1 &#226;&#65533;&#174;g antibodies                                                                       specified proteins. DNA purified immunoprecipitated frac- ChIP IP results indicate endogenous Fli                                                                       tions analyzed PCR primers pS2 promoter. large ER-associated coactivator complex assembled hormone-activated pS2 promoter. <br />  2108     LEE ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Fli secondary coactivator NRs.   Dependence Fli coactivator activity GRIP1 ER. CV-1 cells transfected MMTV(ERE)-LUC reporter plasmid (250 ng) expression vectors encoding ER (100 ng), GRIP1 (250 ng), CARM1 (500 ng), Fli (500 ng) indicated  Transfected cells grown E2, luciferase activities transfected-cell extracts determined. (B) Increasing coactivator activity increasing Fli  Transfections performed panel 200 ng CARM1 vector 200 600 ng Fli vector used. (C) Partial coactivator function Fli LRR domain. Transfections performed MMTV(TRE)-LUC reporter plasmid (250 ng) expression vectors encoding TR (100 ng), GRIP1 (250 ng), CARM1 (500 ng), length Fli LRR gelsolin-like fragment (500 ng). Transfected cells grown T3 harvesting luciferase assays. <br /> Similar results obtained ER MMTV(ERE)-LUC reporter plasmid (data shown). (D) Autonomous activation function Fli LRR domain. CV-1 cells transfected GK1 reporter plasmid (250 ng) encoding luciferase controlled Gal4 response elements expression vectors encoding Gal4 DBD Gal4 DBD fused LRR domain (250 ng) ER (500 ng). Transfected cells grown E2 indicated. Results portrayed panels D representative independent experiments. <br />  <br />  <br />  <br />  <br />    Fli secondary coactivator NRs. Fli binds          reporter gene activity. contrast high-NR conditions, <br /> NRs NR coactivator CARM1 Fli                     use lower levels NR expression vectors cruited pS2 promoter response hormone activation,             transfection results stringent coactivator  tested ability Fli serve coactivator NRs,          quirements. low-NR conditions, NR effects e., enhance ability NRs activate expression              little reporter gene expression; instead, specific combi- <br /> reporter genes containing NR-responsive enhancer elements.                 nations coactivators required NR previously defined different sets transient-                hormone efficient reporter gene expression. <br /> transfection conditions testing function NR coacti-                CV-1 cells transfected high levels ER <br /> vators (32, 35). standard, relatively high levels NR               expression vector (100 ng well dish), GRIP1 <br /> expression vector transfected CV-1 cells, NR            enhanced ability ER activate reporter gene expression <br /> exhibits hormone-dependent activation suitable reporter               presence E2 (Fig. 4A, assays 1 3). Addition gene. effects individual coactivators                 CARM1 Fli GRIP1 enhanced reporter <br /> pairs coactivators observed, addition              gene expression (assays 4 5). ability CARM1 coactivator fails elicit enhancement               Fli function coactivators depended presence  VOL. 24, 2004                                                         NUCLEAR RECEPTOR COACTIVATOR Flightless                  2109 <br />  <br />  ER GRIP1 (assays 6 9).  previously shown        previously low-NR conditions (35), TR did CARM1 (10, 11), Fli functioned secondary coactiva-       activate reporter gene coexpression GRIP1 TR <br /> tor; e., ability enhance NR function depended       caused little activation (Fig. 5A, assays 1 3). Coex- <br /> coexpression p160 coactivator GRIP1.            pression GRIP1 coactivator (CARM1, p300, <br /> presence ER GRIP1, degree enhancement              Fli  Fli fragments) caused little enhancement <br /> reporter gene activity Fli roughly proportional     TR-mediated reporter gene expression (assays 4 8).  Fli expression vector transfected (Fig. 4B).       coexpression GRIP1 CARM1 p300 similar high-NR conditions, Fli functioned second-      Fli resulted dramatic synergistic enhancement TR <br /> ary coactivator TR (Fig. 4C, assays 1 5 8 9).        function (assays 9 10). domains Fli required separate LRR gelsolin-like domains Fli      GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli coactivator synergy (assays 10 bound CARM1 NRs, tested ability         12). <br /> Fli fragments replace length Fli coactivators.      GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli combination synergis- <br /> LRR fragment exhibited weak GRIP1-dependent coactivator           tic, substitution p300 PRMT1 CARM1 failed pro- <br /> activity, compared length Fli  gelsolin-like    duce synergy (Fig. 5B, assays 8 10). synergy fragment coactivator activity (Fig. 4C, assays 6 7       CARM1 Fli completely dependent coexpres- 10 11). worth noting fragments        sion GRIP1 (assay 11), confirming roles CARM1 Fli expressed higher levels transient transfec-    Fli secondary coactivators.  Fli specifically cooper- <br /> tions length Fli (data shown); taken   ated CARM1 coactivator did cooperate account, length Fli powerful coac-       p300 PRMT1. functional cooperation Fli tivator LRR fragment Fli   efficient     CARM1 PRMT1 correlates binding specificity <br /> coactivator function Fli depends LRR         Fli CARM1 PRMT1 (Fig. 1) sug- <br /> gelsolin-like domains.                                              gests binding CARM1 Fli important <br />    Autonomous activation function LRR domain Fli        functional cooperation. p300 PRMT1 coactivators recruited promoter,          failed cooperate functionally Fli (Fig. 5B), transmit activating signal transcription machinery    capable functioning synergistically combina- enzymatic activity (e.g., various histone modifications)      tions coactivators: p300 cooperated synergistically protein-protein contacts downstream compo-             CARM1 GRIP1 (Fig. 5A, assay 9) (35) PRMT1 nents transcription machinery, coactiva-       functioned synergistically CARM1 GRIP1 (Fig. 5C, <br /> tors, chromatin components, basal transcription factors,         assay 11) (32).  low-NR transient-transfection RNA polymerase II. cases, signal           revealed functional collaborations specific combina- <br /> transduction domains coactivators autonomous ac-         tions coactivators: coexpressed GRIP1, CARM1 <br /> tivation function activate reporter gene        cooperated p300 (Fig. 5A) (35), Fli (Fig. 5A C), <br /> activation function fused Gal4 DBD fusion pro-        PRMT1 (Fig. 5C) (32), Fli did cooperate <br /> tein expressed reporter gene controlled Gal4        p300 (Fig. 5B) PRMT1 (Fig. 5B), PRMT1 did sponse elements (10, 32, 58). LRR domain Fli          cooperate p300 (35). fused Gal4 DBD, enhanced expression Gal4-               test role CARM1&#226;&#8364;&#8482;s protein arginine methyl- <br /> responsive reporter gene (Fig. 4D, assays 1 2). Coexpres-       transferase activity GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli synergy, sion ER substantially enhanced activity hormone-       CARM1(E267Q) mutant substituted CARM1 independent manner (assays 3 4), corresponds           low-NR transient-transfection  CARM1(E267Q) lacks hormone-independent binding Fli ER coimmu-           protein methyltransferase activity retains ability bind <br /> noprecipitation studies (Fig. 2B). results suggest   GRIP1, substitution mutant wild-type CARM1 <br /> autonomous activation function Fli contribute       completely eliminated GRIP1-CARM1-p300 coac- <br /> coactivator function Fli interaction         tivator synergy (35). E267Q mutation eliminated stream components transcription machinery; fur-         GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli synergy ER (Fig. 5C, assays 8 ther suggest functional relevance binding Fli    9) TR (data shown), indicating CARM1 <br /> NRs.                                                                methyltransferase activity essential coactivator syn- <br />    Coactivator synergy CARM1 Fli                ergy CARM1 Fli  failure PRMT1 CARM1 Fli functions secondary NR coactiva-          CARM1(E267Q) mutant cooperate Fli tor cooperation GRIP1 (Fig. 4) CARM1          lack expression improper folding Fli specifically bind (Fig. 1), tested   CARM1 mutant, PRMT1 exhibited synergy wild- <br /> CARM1 Fli cooperate coactivators NRs.           type CARM1 CARM1 mutant retained par- <br /> employed transient-transfection assay low lev-      tial synergistic activity PRMT1, mutant <br /> els NR expression vector, previously established        inactive combination Fli (Fig. 5C, assays 9 12). low-NR conditions characterized stringent          CARM1 methyltransferase activity essential requirements multiple coactivators facilitate obser-    synergistic cooperation CARM1 Fli im- <br /> vation synergistic interactions multiple coactivators      portant synergistic cooperation CARM1 PRMT1. <br /> (35). series experiments TR, transfected           Coactivator function Fli depends N-terminal <br /> cultures exposed thyroid hormone, previously       domain GRIP1. coactivator function Fli demonstrated coactivator activities depend pres-    completely dependent coexpression GRIP1, tested <br /> ence NR appropriate hormone. shown               Fli binds GRIP1 interaction <br />  2110     LEE ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />                                                                         explain functional coactivator relationship                                                                         GRIP1 Fli  fragments GRIP1 prepared                                                                         GST fusion proteins: GRIP1.N, amino acids 5 730, contain- <br />                                                                         ing basic helix-loop-helix (bHLH) ARNT-SIM <br />                                                                         (PAS) domains   LXXLL <br />                                                                         motifs; GRIP1.M, amino acids 730 1121, containing                                                                         LXXLL motif AD1 domain, binds CBP <br />                                                                         p300; GRIP1.C, amino acids 1122 1462, containing <br />                                                                         AD2 domain, binds CARM1. Fli LRR frag- <br />                                                                         ment translated vitro bound specifically immobilized <br />                                                                         GST-GRIP1.N (Fig. 6A, upper panel, lane 3), gelsolin- <br />                                                                         like fragment bound weakly GRIP1 N-terminal <br />                                                                         region (lane 8). weak background-level binding                                                                         Fli fragments observed GST (lanes 2 7), GST- <br />                                                                         GRIP1.M (lanes 4 9), GST-GRIP1.C (lanes 5 10). <br />                                                                         positive control, demonstrated CARM1 trans- <br />                                                                         lated vitro bound GST-GRIP1.C (Fig. 6A, lower right <br />                                                                         panel), shown previously (11). results suggest direct <br />                                                                         interaction LRR domain Fli N-ter- <br />                                                                         minal region GRIP1. <br />                                                                            test Fli GRIP1 interactions vivo, HA-tagged <br />                                                                         GRIP1 fragments expressed transient transfection                                                                         Cos-7 cells Flag-tagged Fli fragments. Immuno- <br />                                                                         precipitations performed anti-Flag antibodies,                                                                         coprecipitated GRIP1 fragments visualized immuno- <br />                                                                         blot antibodies HA epitiope. vitro <br />                                                                         binding studies (Fig. 6A), coimmunoprecipitation experi- <br />                                                                         ments demonstrated GRIP1 N-terminal region bound <br />                                                                         Fli LRR domain Fli gelsolin-like <br />                                                                         domain (Fig. 6B, lanes 5 8).  C-termi- <br />                                                                         nal region GRIP1 did bind Fli domain vitro <br />                                                                         (Fig. 6A), bound domains Fli vivo (Fig. 6B, <br />                                                                         lanes 7 10). middle domain GRIP1 did bind                                                                         Fli domain (lanes 6 9). combined results                                                                         vitro vivo binding studies suggest Fli                                                                         LRR domain binds directly N-terminal region                                                                         GRIP1, LRR gelsolin-like domains Fli                                                                         bind indirectly C-terminal region GRIP1.                                                                         CARM1 binds C-terminal region GRIP1                                                                         LRR gelsolin-like domains Fli bind                                                                         CARM1, propose binding Fli GRIP1 <br />                                                                         C-terminal region observed vivo mediated endog- <br />                                                                         enous CARM1, present relatively high levels                                                                         Cos-7 cells (data shown). <br />                                                                            Fli physically interacts NRs, CARM1,                                                                         GRIP1, functional dependence Fli GRIP1 (Fig. 4 <br />                                                                         5) led test observed binding Fli                                                                         N-terminal region GRIP1 (Fig. 6A B) important <br />                                                                         functional collaboration coactiva- <br />                                                                         tors. tested known functional domains <br />                                                                         GRIP1 required GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli coactivator <br />                                                                         synergy. Deletion mutants GRIP1 lacking N-terminal <br />                                                                         domain (&#226;&#338;&#172;N, lacking amino acids 1 562), AD1 domain <br />  <br />   FIG. 5. Synergistic coactivator function Fli CARM1. <br /> CV-1 cells transfected MMTV(TRE)-LUC (250 ng)  B) MMTV(ERE)-LUC (250 ng) (C) expression vectors encod- <br /> ing TR (1 ng) ER (1 ng), GRIP1 (250 ng), CARM1 wild type          PRMT1 p300. (C) Methyltransferase activity CARM1 <br /> E267Q mutant (500 ng), PRMT1 (500 ng), p300 (500 ng),          required synergy Fli  Transfected cells grown T3 <br /> length Fli LRR gelsolin-like fragment (500 ng).       B) E2 (C) harvest. Results panel repre- <br /> length Fli required synergy. (B) Fli synergy CARM1     sentative independent experiments. <br />  VOL. 24, 2004                                                            NUCLEAR RECEPTOR COACTIVATOR Flightless                          2111 <br />  <br />  <br />                                                                        (&#226;&#338;&#172;AD1, lacking amino acids 1057 1109), AD2 domain <br />                                                                        (&#226;&#338;&#172;AD2, lacking amino acids 1122 1462) coexpressed <br />                                                                        CARM1, Fli  low levels TR transient-transfec- <br />                                                                        tion assays. shown (Fig. 5), reporter gene activation <br />                                                                        hormone-activated TR conditions synergisti- <br />                                                                        cally enhanced wild-type GRIP1, CARM1, Fli                                                                         coactivators required efficient reporter gene ex- <br />                                                                        pression (Fig. 6C, assays 1 15). contrast, GRIP1&#226;&#338;&#172;N <br />                                                                        mutant activity coexpressed CARM1 Fli                                                                        (assay 16); GRIP1&#226;&#338;&#172;AD1 GRIP1&#226;&#338;&#172;AD2 mutants exhibited <br />                                                                        weak partial activity cooperation CARM1 Fli                                                                        (assays 17 18). mutants expressed levels <br />                                                                        equal exceeding wild-type GRIP1 (data                                                                        shown) (42).  GRIP1 N-terminal domain,                                                                        direct binding site Fli  absolutely required                                                                        GRIP1-CARM1-Fli coactivator synergy. strong require- <br />                                                                        ment N-terminal domain, essential                                                                        binding GRIP1 NR, p300, CARM1 (11, 29, 42), sug- <br />                                                                        gests interaction GRIP1 N-terminal  <br />                                                                        main Fli important recruitment Fli                                                                        coactivator complex functional synergy                                                                        GRIP1 Fli  partial requirement AD1 suggests <br />                                                                        endogenous p300 CBP, binds AD1,  <br />                                                                        tribute GRIP1&#226;&#8364;&#8220;CARM1&#226;&#8364;&#8220;Fli synergy.                                                                        AD2 domain important achieving GRIP1&#226;&#8364;&#8220; <br />                                                                        CARM1&#226;&#8364;&#8220;Fli synergy, fact absolutely  <br />                                                                        quired somewhat surprising, CARM1 binds                                                                        AD2 region GRIP1; fact CARM1 bind Fli                                                                        provide alternative mechanism recruiting CARM1 <br />                                                                        coactivator complex achieving partial <br />                                                                        coactivator synergy observed GRIP1&#226;&#338;&#172;AD2 mutant. <br />                                                                        interpretation supported fact AD2  <br />                                                                        main absolutely required synergy GRIP1, <br />                                                                        CARM1, p300 (35); e., Fli included                                                                        case, alternative binding site CARM1. <br />                                                                           siRNAs Fli suppress ER-mediated transcription.                                                                        test physiological relevance Fli coactivator, <br />                                                                        siRNAs directed LRR domain Fli  <br />                                                                        signed tested effects ER transcription. speci- <br />                                                                        ficity siRNAs tested cotransfecting                                                                        Cos-7 cells separate vectors encoding Flag-tagged <br />                                                                        LRR gelsolin-like domains Fli  immunoblot <br />                                                                        analysis transfected-cell extracts anti-Flag antibody, <br />  <br />  <br />  <br />  <br />                                                                        fragments GRIP1 fragments vivo. Coimmunoprecipitations                                                                        performed Fig. 1C. Cos-7 cells transfected                                                                        pSG5.Flag-FliI(LRR) pSG5.Flag-FliI(gelsolin) (3 &#226;&#65533;&#174;g) pSG5.HA <br />                                                                        plasmid encoding GRIP1N, GRIP1 M, GRIP1C (3 &#226;&#65533;&#174;g). Immunopre- <br />                                                                        cipitation (IP) performed anti-Flag antibodies, immunoblot- <br />                                                                        ting (W) performed anti-HA antibodies. Immunoblotting   FIG. 6. Coactivator activity Fli depends N-terminal      immunoprecipitated proteins shown upper panel, immuno- <br /> domain GRIP1.   LRR domain Fli binds GRIP1 N-             blotting cell extracts immunoprecipitation shown terminal region vitro. vitro binding assays performed    lower panel. (C) Requirement GRIP1 N-terminal region coactiva- Fig. 1B. Upper panel tests binding 35S-labeled Fli   tor synergy. CV-1 cells transiently transfected MMTV(TRE)- <br /> LRR gelsolin-like fragments GST GST fused GRIP1          LUC (250 ng) expression vectors encoding TR (1 ng), CARM1 (500 <br /> N-terminal (GRIP1.N, amino acids 5 765), middle (GRIP1.M,           ng), Fli (500 ng), wild type indicated mutant GRIP1 (250 <br /> amino acids 730 1121), C-terminal (GRIP1.C, amino acids 1122    ng): GRIP1&#226;&#338;&#172;N lacked amino acids 1 562, &#226;&#338;&#172;AD1 lacked amino acids 1462) fragments. Lower right panel shows binding 35S-labeled     1057 1109, &#226;&#338;&#172;AD2 lacked amino acids 1122 1462. Transfected <br /> CARM1 GST-GRIP1C. Lower left panel shows Coomassie-stained          cells grown T3 harvest. Results representative gel GST-GRIP1 fusion proteins used. (B) Binding Fli        independent experiments. <br />  2112     LEE ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />  siRNAs Fli completely partially inhibited expression LRR domain (Fig. 7A, lanes 2 4), siRNAs AIB1 CARM1 did influence expres- <br /> sion LRR (lanes 5 6). contrast, siRNAs Fli effect expression Fli gelsolin-like <br /> domain (lanes 8 9), confirming specificity Fli siRNAs. <br />     effects siRNAs endogenous pro- <br /> teins investigated transfecting MCF-7 cells monitoring expression levels endogenous pro- <br /> teins immunoblot. Endogenous Fli protein specifically <br /> reduced siRNAs Fli siRNAs AIB1 CARM1 (Fig. 7B, panel). Endogenous CARM1 <br /> expression inhibited siRNA CARM1 siRNAs (middle panel). endogenous actin level changed siRNAs.  siRNAs tested efficient specific reducing endogenous protein <br /> expression. <br />    effect siRNAs transcriptional activation endogenous ER investigated MCF-7 cells. MCF-7 cells transfected estrogen-responsive reporter plasmid, transcription activation endogenous ER deter- <br /> mined reporter analysis. estrogen-responsive reporter <br /> gene activation endogenous ER significantly enhanced E2 (Fig. 7C, assays 1 6). Cotransfection reporter <br /> plasmid Fli siRNA, CARM1 siRNA, AIB1 siRNA resulted loss of estrogen- <br /> dependent reporter gene expression (assays 6 14), indicating endogenous AIB1, CARM1, Fli required endogenous ER function. siRNAs Fli  <br /> producibly efficient inhibiting ER activity CARM1 AIB1 siRNAs (data shown). siRNA di- <br /> rected irrelevant sequence effect reporter <br /> gene activation E2 endogenous ER (data shown). data demonstrate importance Fli coactiva- <br /> tor action endogenous ER. <br />    Role actin binding activity Fli gelsolin-like domains coactivator function. gelsolin-like domain Fli important Fli coactivator function (Fig. 5A) bind actin (16, 17, 26, 39), questioned actin binding activity Fli important coactivator function. fact, actin binding proteins,  <br /> cluding supervillin gelsolin itself, shown bind enhance transcriptional activation NRs (28,            FIG. 7. Requirement Fli MCF-7 cells ER-dependent gene <br />                                                                       expression.   Specificity LRR-directed siRNAs. Cos-7 cells  12- <br /> 59). Actin actin-like proteins components number         culture dishes 1 ml medium  transfected nuclear complexes involved transcription regula-        pSG5.Flag vector encoding LRR gelsolin-like domain Fli (1 <br /> tion, including SWI/SNF <span id='am-37' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-38' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span>ATP</span>-dependent chromatin                &#226;&#65533;&#174;g) 5 &#226;&#65533;&#174;l 20 &#226;&#65533;&#174;M siRNAs. hours transfection, cell <br /> modeling complex (4, 52, 53)  Discussion). Like protein       extracts analyzed immunoblotting (W) anti-Flag antibody. <br />                                                                       Lane 1 shows extract cells transfected vector gelsolin, gelsolin-like domain Fli consists gel-      siRNA. (B) Reduction endogenous Fli protein level MCF-7 cells solin-like motifs, contains repeats         siRNAs Fli  indicated siRNAs transfected actin binding motif (8). coimmunoprecipitation assays,          panel MCF-7 cells, endogenous protein expression ana- gelsolin-like motifs Fli         lyzed immunoblotting (W) antibody indicated right. <br /> (Gel  amino acids 495 827) bind Brg- Brm-              (C) siRNAs Fli suppress gene activation endogenous ER                                                                       coactivators MCF-7 cells. MMTV(ERE)-LUC reporter plasmid (1 &#226;&#65533;&#174;g) <br /> associated factor 53 (BAF53), actin-like component          transfected MCF-7 cells 12 culture dishes containing 1 ml <br /> SWI/SNF complex (Fig. 8A, upper panel). tested                medium indicated siRNAs (5 10 &#226;&#65533;&#174;l 20 &#226;&#65533;&#174;M siRNA). expression Gel fragment Fli        transfection, MCF-7 cells maintained phenol red-free <br /> enhance  e., serve coactivator  inhibit  e., act   Dulbecco&#226;&#8364;&#8482;s modified Eagle medium 5% charcoal-dextran-treated <br />                                                                       fetal bovine serum 4 days. transfection siRNAs, dominant-negative mutant interfere action  ER        MCF-7 cells treated 20 nM E2 indicated. Luciferase activity <br /> function. Coexpression Gel fragment Fli ER         determined 48 h transfection. data shown representative transient-transfection assays enhanced estrogen-dependent          10 independent experiments. <br />  VOL. 24, 2004                                                                 NUCLEAR RECEPTOR COACTIVATOR Flightless                  2113 <br />  <br />  <br />                                                                             activity coactivator function Gel fragment                                                                             Fli  point mutations generated                                                                             putative actin binding repeats Gel  E585K <br />                                                                             mutant Gel designed according previously  <br />                                                                             ported mutation gelsolin protein defective actin <br />                                                                             binding (64); G602S mutation Gel corresponds                                                                             Drosophila fliI3 mutation, causes partial defect Dro- <br />                                                                             sophila muscle development results flightless phe- <br />                                                                             notype (18). coimmunoprecipitation assays, G/S muta- <br />                                                                             tion effect binding Gel BAF53, E/K <br />                                                                             mutation reduced Gel binding BAF53 (Fig. 8A, upper <br />                                                                             panel). contrast, mutants retained ability                                                                             wild-type Gel fragment bind ER (Fig. 8A, lower panel); <br />                                                                             retention ER-binding activity shows mutants <br />                                                                             folded properly surfaces Gel interact <br />                                                                             actin ER distinct. <br />                                                                                Gel wild-type mutant proteins compared <br />                                                                             ability serve coactivators ER. G/S mu- <br />                                                                             tant equivalent wild-type Gel fragment                                                                             ability enhance estrogen-dependent activation tran- <br />                                                                             siently transfected reporter gene ER;  coacti- <br />                                                                             vator function E/K mutant substantially reduced <br />                                                                             (Fig. 8B). reduced activity E/K mutant                                                                             improper folding diminished expression stability,                                                                             shown ability mutant bind ER normally (Fig. <br />                                                                             8A). specific, correlated loss actin binding (64)                                                                             binding actin-like BAF53 loss coactivator function                                                                             Gel mutants suggests binding actin actin-like pro- <br />                                                                             teins important coactivator function Gel                                                                             fragment  extension, coactivator function Fli  <br />  <br />  <br />                                                                                                      DISCUSSION <br />                                                                                Fli novel secondary coactivator NRs. Fli                                                                             function cooperation specific combinations NR <br />    FIG. 8. Role actin binding activity Fli gelsolin-like domains    coactivators enhance ability NRs activate transcrip- coactivator function.   Binding Fli Gel domain (wild type       tion NR-regulated reporter genes (Fig. 4 5). Fli mutants) actin-like molecule BAF53 ER. Vectors             identified binding CARM1 (Fig. 1), coding human Fli gelsolin-like Gel fragment (amino acids 495 827) point mutants (E585K G602S) (2.5 &#226;&#65533;&#174;g)                                                                             binds specifically vitro vivo GRIP1 (Fig. 6) transfected Cos-7 cells vectors encoding actin-like     NRs (Fig. 2). protein-protein  <br /> protein BAF53 (2.5 &#226;&#65533;&#174;g, upper panel) ER (2.5 &#226;&#65533;&#174;g, lower panel). Cell       teractions important coactivator function Fli  <br /> lysates prepared immunoprecipitated (IP) antibodies           Fli itself did enhance activity NRs HA ER. precipitated fractions analyzed immu-         required presence p160 coactivator (Fig. 4 5), noblotting (W) anti-Flag antibody. (B) Coactivator activity wild-type mutant Fli Gel fragments. pSG5.Flag vectors encod-        binding Fli NRs apparently sufficient Fli ing wild-type mutant Fli Gel fragments (0.5 &#226;&#65533;&#174;g) trans-          coactivator function.  ability ER enhance <br /> fected CV1 cells 0.1 &#226;&#65533;&#174;g ER expression vector 0.25 &#226;&#65533;&#174;g       autonomous activation function Fli LRR domain MMTV(ERE)-LUC reporter plasmid; E2 added transfec-             (Fig. 4D) ability gelsolin-like Gel fragment tion, luciferase activity determined 40 h transfection. data shown representative seven independent experiments. <br />                                                                             Fli enhance ER function (Fig. 8B) indicate existence                                                                             functional relevance ER-Fli interaction                                                                             suggest Fli NR interaction contribute                                                                             ability Fli serve NR coactivator. <br /> expression transient reporter gene (Fig. 8B, assays 1 4                Enhancement NR function Fli observed 13).                                                                    GRIP1 coexpressed Fli  addi- <br />    previous mutational studies gelsolin, puta-            tional coactivators CARM1 p300 coex- <br /> tive actin binding repeat   gelsolin-like         pressed (Fig. 4 5). dependence Fli coactivator <br /> motif   important binding                   function GRIP1 indicates Fli secondary coacti- <br /> G-actin, second repeats motif              vator binding Fli GRIP1 shown important binding F-actin (8).                essential coactivator function Fli  Fli Gel fragment Fli represents gelsolin-           bound N-terminal C-terminal regions like motifs Fli  test importance actin binding               GRIP1 coimmunoprecipitation experiments (Fig. 6B), Fli  2114     LEE ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br /> bound N-terminal region GRIP1 vitro (Fig.          Fli did function synergistically protein acetyl- <br /> 6A), suggesting direct interaction N-terminal region      transferase p300 PRMT1 (Fig. 5B C), GRIP1 indirect binding C-terminal region         member protein arginine methyltransferase family GRIP1. Furthermore, cooperative coactivator activity          methylates different protein substrates CARM1 (57). Fli CARM1 GRIP1 absolutely dependent               ability Fli bind strongly CARM1 weakly N-terminal region GRIP1 partially depen-         PRMT1 suggests binding CARM1 Fli dent C-terminal AD1 AD2 regions GRIP1 (Fig.            contributes selective functional synergy observed  <br /> 6C).  binding Fli N-terminal region          tween Fli CARM1 PRMT1. synergy  <br /> GRIP1 appears critical binding interaction po-        tween CARM1 Fli required methyltransferase activ- <br /> tentiates Fli recruitment target gene promoter.       ity CARM1 (Fig. 5C). target protein(s) definitive data present Fli        methylated CARM1 achieve coactivator synergy CARM1 binding necessary Fli coactivator function,       include histone H3, known methylated functional synergy coactivators (Fig. 5) sug-      CARM1 steroid hormone activation stably inte- <br /> gests binding interaction contribute.         grated genes (41), presently unknown target, <br /> contacts Fli multiple components tran-          component transcription machinery scription complex (NRs, GRIP1, CARM1) result            coactivator. CARM1 Fli apparently  efficient recruitment Fli promoter,          cruited promoter interaction GRIP1, <br /> stable coactivator complex, highly functional          CARM1 binds C-terminal AD2 domain GRIP1 (10, <br /> formation coactivator complex.                                11), Fli binds N-terminal region GRIP1 <br />    LRR gelsolin-like domains Fli           (Fig. 6). observation synergistic cooperation  <br /> quired coactivator function. LRR domain            tween CARM1 Fli indicate coactivators modest coactivator function cooperation            distinct complementary roles transcriptional <br /> GRIP1 high levels NRs (Fig. 4C), Fli domains       activation process.  mediate functions required absolutely low-NR conditions (Fig. 5A).           different regions GRIP1 contribute tran- <br /> binding studies suggest LRR domain primarily          scriptional activation process different mechanisms. <br /> solely responsible critical Fli GRIP1.N interaction        CARM1 methylates histones, p300/CBP, possibly (Fig. 6). appears binding Fli CARM1          proteins transcription initiation complex (41, 69) NRs contributes cooperative coactivator function      recruit additional coactivators (58). complemen- <br /> Fli LRR gelsolin-like domains partic-        tary role Fli involve specific protein (presently <br /> ipate binding interactions. autonomous transcrip-       unknown) interacts target autono- <br /> tional activation activity LRR domain explain         mous transcriptional activation domain LRR region domain exhibited modest coactivator function GRIP1         Fli  addition, ability Fli gelsolin-like domain high-NR conditions suggests activation      bind actin (16, 17) actin-like proteins important function play role Fli coactivator activity, presum-      coactivator function Fli (Fig. 8; additional <br /> ably protein-protein interactions compo-          discussion . <br /> nents transcription machinery.                                   cooperation Fli GRIP1 provides addi- <br />    proteins previously bind Fli          tional insight mechanism coactivator function LRR domain include FLAP1  called LRRFIP2)                   GRIP1. N-terminal 350 amino acids GRIP1 FLAP2  called TRIP, LRRFIP1, GCF2) (9). Evi-                members p160 coactivator family dence date suggests FLAP2 bind specific RNA              highly conserved region family (2), role sequences function specific DNA-binding            region p160 coactivator function unclear. basic helix- <br /> transcriptional repressor. Binding Fli transcriptional   loop-helix (bHLH) Arnt-Sim (PAS) motifs regulator FLAP2 interesting light  consistent      region. Recently, proteins bind  conclusions Fli function transcriptional   region p160 coactivators identified. coregulator NRs. Separate studies LRR             proteins, including TEF4 (5) MEF2C (14), DNA- <br /> domain Caenorhabditis elegans Fli homologue bind        binding transcriptional activators, apparently recruit Ras small GTPases (9). similar           p160 coactivators binding bHLH-PAS region; binding interaction higher organisms            p160 coactivators function coactivators differ- <br /> firmed, interaction C. elegans interesting implica-       ent classes transcriptional activator proteins addition tions possible regulation Fli coactivator activity       NRs. coactivators, including BAF57 (4) MMS19 <br /> signaling pathways involving specific small GTPases.                 (66), bind bHLH-PAS region p160 coactivators. <br />    Characterization CARM1 GRIP1 function identi-           findings suggest N-terminal bHLH-PAS <br /> fication interacting proteins. mechanisms            region contribute p160 coactivator function recruit- <br /> CARM1 GRIP1 enhance NR function involve ability              ing additional coactivators promoter, clear demonstra- CARM1 GRIP1 bind proteins transmit           tion relationship previously shown. transcriptional activation signal cooperate          fact Fli binds N-terminal region GRIP1 coactivators (10, 35, 42, 58).  data indicate       cooperates synergistically GRIP1 coactivator demon- coactivator function CARM1 involve ability       strates functional relevance binding Fli GRIP1 bind function synergistically Fli  synergis-    indicates role N-terminal region GRIP1 <br /> tic cooperation CARM1 Fli specific,          recruit Fli coactivator, just AD1 domain  VOL. 24, 2004                                                           NUCLEAR RECEPTOR COACTIVATOR Flightless                          2115 <br />  <br />  <br /> p160 coactivators recruits p300 CBP just AD2 <br /> domain p160 coactivators recruits CARM1.   <br /> sults demonstrate N-terminal region p160 coacti- <br /> vators serve additional activation signal transmis- <br /> sion domain, AD1 AD2. <br />    p160 coactivators appear serve scaffold proteins contact NRs, classes DNA-binding <br /> transcriptional activator proteins, directly appear  <br /> semble complex coactivators promoter (25, <br /> 46). specific complement coactivators required vary different promoters different cell types.           FIG. 9. Proposed model cooperative linking p160 SWI/ <br /> example, results different domains GRIP1         SNF coactivator complexes Fli protein. Fli protein bind important potentiate synergy different combina-            components p160 coactivator complex (p160 CARM1), <br /> tions coactivators. GRIP1 acts synergistically           contains histone acetylating (CBP/p300) methylating <br />                                                                       (CARM1) activities. Fli bind actin actin-like <br /> CARM1 p300, AD2 domain absolutely required,                protein BAF53, components ATP-dependent deletion AD1 domain causes partial loss          nucleosome-remodeling complex SWI/SNF.  possible mech- <br /> synergy (35). contrast, GRIP1 acts synergistically       anism contribution Fli coactivator facilitate <br /> CARM1 Fli  AD1 AD2 domains contribute              recruitment coordinate complementary chro- <br />                                                                       matin-remodeling activities SWI/SNF complex p160 absolutely required (Fig. 6C).  specific <br />                                                                       coactivator complex. H, hormone. <br /> domains p160 coactivators required vary depending complex secondary coactivators available assembled promoter. addition, AD2 direct binding site CARM1, reduced requirement            nisms, e., ATP-dependent nucleosome remodeling  AD2 domain Fli present CARM1 suggests              tone modifications, respectively (25, 46). Overexpression CARM1 recruited p160 coactivator indi-             BAF57 shown enhance NR function cell line rectly binding Fli  bound N-terminal          low levels endogenous BAF57 (4), p160- <br /> region p160 coactivator.                                       binding ability BAF57 important coactivator func- <br />    Implications actin binding properties Fli      tions BAF57 SWI/SNF complexes deter- <br /> coactivator function. gelsolin-like motifs Fli bind      mined. actin G (globular) F (polymerized) forms,             role unknown actin binding proteins regulated Fli observed immunofluorescence associate             recruitment SWI/SNF complexes chromatin previ- actin cytoskeleton certain cellular conditions         ously proposed (72). observed Fli (16, 17). association actin subcellular local-       binds actin (17) binds actin-like BAF53 coim- <br /> ization Fli appear regulated pathways involving         munoprecipitation experiments (Fig. 8A), p160  <br /> <span id='am-7' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-8' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span>phosphatidylinositol</span> 3-kinase Ras-related small GTPases;          activators (Fig. 6).  Fli  like BAF57, help se- example, cells grown low serum concentrations Fli         cure association SWI/SNF complex p160 primarily nuclear, higher serum concentrations, Fli      coactivator complex help coordinate comple- cytoplasmic compartment associated          mentary ATP-dependent nucleosome-remodeling activity cytoskeleton (17). observations consistent reg-        SWI/SNF complex histone acetylating (e.g., ulated dual roles Fli transcriptional coactivator   CBP p300) methylating (e.g., CARM1 nucleus cytoplasmic modulator cytoskeleton; fur-         PRMT1) activities p160 coactivator complex (Fig. 9). <br /> thermore, suggest Fli function communi-             Recently, actin binding protein, supervillin,  <br /> cation link cytoskeleton structure influence        ported coactivator NRs (59). actin binding <br /> transcription.                                                        motif supervillin, like Fli  highly homologous    addition role cytoskeleton structural protein,      gelsolin. addition, yeast hybrid screen CARM1 <br /> previous studies suggested roles actin transcription       binding proteins identified Fli profilin 1 <br /> process (21, 54). Actin actin-like proteins      (data shown), actin binding protein (72). nucleus association various complexes associated         possible role profilin 1 coactivator remains transcription machinery (1). actin actin-like        tested, speculate Fli  supervillin, profilin 1 protein BAF53 identified integral essential com-          represent class actin binding coactivators similar <br /> ponents SWI/SNF coactivator complexes, occur              mechanisms enhancing transcriptional activation associ- <br /> multiple forms contribute chromatin-remodeling             ation actin actin-like proteins. dual roles Fli process ATP-dependent nucleosome-remodeling activ-             actin cytoskeleton-associated protein transcriptional <br /> ity (53, 70, 72). component SWI/SNF com-               coactivator unprecedented. developmentally <br /> plexes, BAF57, (like Fli  bind ER N-            important protein, &#226;&#65533;&#164;-catenin, participates regulation terminal region p160 coactivators (4); interaction             cytoskeleton association cell adhesion molecules (47) BAF57 ER-assembled transcription initiation           functions coactivator nucleus (20, 28, 61), <br /> complex suggests possible mechanism coordinating            bind function synergistically CARM1 <br /> actions SWI/SNF p160 coactivator complexes,          coactivators NRs classes DNA-binding <br /> remodel chromatin different complementary mecha-               transcriptional activator proteins (33). true Fli   <br /> localization &#226;&#65533;&#164;-catenin cytoplasm nucleus                       <br />                                                                                      <br /> regulated specific signal transduction pathways.  Fli                    signal transduction pathway links                    <br /> cytoskeleton structure transcriptional regulation.                          <br />                                                                                     <br /> CARM1 Fli functional synergy transcriptional                          <br /> coactivators, tempting speculate CARM1-                       <br /> Fli interaction influence cytoplasmic function                           <br />                                                                                      <br /> Fli nuclear activity.                                                 <br />    <br /> </body></html>