<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 76,  8, pp. 4006-4010, August 1979 <br /> Genetics <br />  <br />  <br /> Unstable transpositions his4 yeast <br />     (DNA insertions/chromosome rearrangements/gene excision/histidine biosynthesis/aneuploidy) <br /> HELEN GREER* GERALD R. FINK <br /> Department Botany, Genetics Development, Cornell University, Ithaca, New York 14853 <br /> Communicated Adrian M. Srb, 15, 1979 <br />  <br /> ABSTRACT           Unstable transpositions yeast                     Selection Procedure Transpositions. selection selected his4C gene chromosome III                     transposition takes advantage unique properties serted chromosome XII. event associated                  yeast his4 region. his4 region, located chromosome <br /> generation recessive lethal mutation, resulting                  III, subdivided  B, C complementation groups <br /> integration his4C essential gene. Strains                code, respectively,  second, steps <br /> transpositions viable diploids aneuploids chromo-                histidine biosynthesis (7). steps catalyzed XII. event generates transpositions does lead reciprocally deletion chromosome III, implying               single, multifunctional polypeptide chain;  B, C, <br /> synthesis new copy his4C subsequent transposition                  regions function independently (8, 9). occurred. his4C transpositions unstable                   Transcription translation proceed direction rise C- segregants high frequency, result ei-             C. <br /> ther precise excision his4C gene (restoring function                transposed his4 strains selected demanding <br /> gene insertion occurred) chromosome loss.                    expression C gene diploid strain homozygous Transposable elements widespread phenomenon bac-                      nonsense mutations his4A, UGA UAA, prevent <br /> teria. Insertion sequences, transposons, plasmids, temperate               C gene expression polarity. selected relief <br /> phage share ability integrate sites ge-              polarity, C+ function, demanding growth minimal <br /> nome (1). Transposable elements shown regulate                    + histidinol medium. C gene codes histidinol  <br /> gene expression, act mutators, rearrange gene locations                 hydrogenase enzyme histidine pathway (converts histi- joining previously unlinked DNA regions transposition                    dinol histidine). use polar mutations eliminated <br /> (1). Knowledge role transposable elements eukar-                  certain classes revertants occur single polar <br />                                                                                mutation used (9). Suppressors obtained yotes scanty, evidence                 efficient suppressor yeast UAA UGA <br /> existence systems. maize, controlling elements                   mutations. Revertants wild type obtained  <br /> transpose different chromosomes affect gene regulation                  mutation reverts, present maintain chromosome structure (2, 3). Drosophila                    polarity. haploid strain carrying nonsense muta- <br /> sertional translocations cases transpositions               tions used selection, frame deletions polar <br /> unstable genotypes mediated insertion elements                  mutations restore C activity observed (unpublished <br /> (4, 5).                                                                        data). diploid strain used ensure events    paper existence unstable transpo-                generate recessive lethal mutations lost. <br /> sitions yeast, Saccharomyces cerevisiae, his4C                       Overnight cultures starting diploid strain, HG183, <br /> (C + ) gene (codes histidinol dehydrogenase enzyme                  homozygous double polar mutations his4-260 (UGA) histidine biosynthetic pathway). standard strains, C+               his4-39 (UAA) heterozygous leu2-1, washed, <br /> gene chromosome III. Rare transpositions                    concentrated 10-fold, plated minimal medium plus histidinol <br /> tected C+ gene inserted chromosome XII.                   leucine, subjected UV irradiation. Mutants integration event associated generation                  obtained frequency 1 X 10-9 surviving cell. recessive lethal mutation, presumably resulting inte-                  mutants, R2 R3, discussed paper. R2 <br /> gration C+ gene essential gene chromosome                    R3 remained Leu+. separate colonies, derived XII. Strains carrying C+ transposition viable diploids               streaking R2, gave different results tetrad analysis. aneuploids chromosome XII. event generates                      derivatives original R2 revertant referred transposition does lead reciprocally deletion                   R2 R2-B. <br /> chromosome III. C+ transpositions unstable rise C- segregants high frequency. C- segregants result                                             RESULTS precise excision C+ element loss                     C+ diploids <br /> entire chromosome carrying insertion.                                          Lethal Segregates 2:2. Tetrad analysis mutant R2                                                                                 R3 C+ diploids showed segregation lethal viable <br />             MATERIALS METHODS                                               spores meiosis. chromosomal marker heterozygous   Media Genetic Techniques. Complete medium, YEP                            diploid (+/-), segregate 2+:2- meiotic <br /> medium, minimal medium supplemented appropriate                        spores single tetrad. 60 tetrads R2  50 tetrads <br /> amino acids, genetic techniques mating strains, com-                     R2-B, 20 tetrads R3, great majority plementation recombination testing, tetrad analysis                     tetrads strain gave viable spores tetrad described (6).                                                        (Table 1). result indicates presence recessive lethal <br />                                                                                 mutation heterozygous C+ mutant diploids. publication costs article defrayed page             tetrads viable spores discussed later. <br /> charge payment. article marked &quot;ad- <br /> vertisement&quot; accordance 18 U. S. C. &#194;&#167;1734 solely indicate            *   Present address: Biology Department, Harvard University, Cam- fact.                                                                          bridge, MA 02138. <br />                                                                          4006 <br />                      Genetics: Greer Fink                                                                     Proc. Natl. Acad. Sci. USA 76 (1979)       4007 <br />  <br />                                                                  Table 1. C(+dplkoids C- segregants <br />                                                                  Spore viability viables:lethals ratio              Centromere linkage <br />                                                              2:2        4:0        3:1        1:3         0:4          +/+           +/- <br />                               R2         C+                44             7          1          8          0         17       22       5 <br />                                            C- (1)             8            26          1          0          0 <br />                                            C-(2)             37             7          1          3          0 <br />                                            C-(3)             24             2          1          3          0 <br />                               R2-B         C+                45             3          1          1          0          9       11      25 <br />                                            C- (1)             0            20          0          0          0 <br />                                            C-(2)              0            18          2          0          0 <br />                                            C-(3)              0            19          0          0          1 <br />  <br />                               R3           C+                12             6          0          0          2          5        3       4 <br />                                          C-(1)           0         9         1          0         0 <br />                                          C-(2)           1         9         0          0         0 <br />                               Spore viability initial C+ mutant diploids (R2 = 5997, R2-B = 6001, R3 = 5999)                             independently isolated spontaneous C- segregants (1, 2, 3) assayed YEP medium 30&apos;C. <br />                             ratio columns indicate number tetrads giving corresponding ratios viable:lethal spores. <br />                             Fewer tetrads R3 analyzed sporulation strain poor. Centromere linkage <br />                             C+ marker deduced recombination events C+ (letal) centromere-linked <br />                             marker Ieu2. spores tetrads C+ diploid (heterozygous leu2-1) showed 2:2 <br />                             segregation viable (Ci):lethal (C+) analyzed leu2 genotypes. numbers                             +/+, -/-, +/- columns indicate number tetrads 2 viable spores spores <br />                             Leu+, Leu-, Leu+ Leu-, respectively. <br />      Lethal Linked C+. striking observation                             regants longer contain recessive lethal mutation (Table 1). <br />  virtually cases viable spores C-, indicating                           loss C+ associated concomitant loss  C+ lethal phenotypes linked. Integration C+ gene                             lethality. retention loss lethal depends  essential gene produce lethal consistent                            mechanism C+ element lost dis- <br />  observation (Fig. 1A).                                                           cussed later. instability C+ explains     tetrad data provide evidence difference                            spore tetrads observed meiotic analysis C+ <br />  clonal population R2 based recombination events                              diploids. viable spores spore tetrads  C+ centromere linked marker leu2. lower                                    C&apos;, presumably result simultaneous loss C+ region <br />  percentage recombinant spores (tetratypes), closer C+                               lethal. <br />  centromere. absence linkage                                    C+ Region Longer Linked Chromosome III <br />  marker centromere characterized ratio                               Genes. Meiotic analysis shows C+ region linked parental ditype (PD) nonparental ditype (NPD)                                lethal mutation chromosome III genes. his4 <br /> tetratypes (TT). C+ R2-B closely centromere                                   17 map units leu2 chromosome III standard strains <br /> linked (9 PD:11 NPD: 25 TT), C+ R2 (17                                (Fig. 2). contrast, R2 R3 diploids, C+ region <br /> PD: 22 NPD:5 TT) R3 (5 PD: 3 NPD: 4 TT) centromere                                   segregated independently 1eu2 locus; 50% viable <br /> linked (Table 1).                                                                          spores  C-) Leu+ 50% Leu- (Table 2). Sim- <br />     C+ Unstable Loss C+ Restores Viability. C+                               ilarly, C+ gene linked mat (mating type) phenotype diploids unstable gives rise sponta-                               chromosome III. <br /> neously C- segregants high frequency. liquid culture                                 chromosomal aberrations transloca- <br /> grown single C+ clone produces 1-5% C- colonies. C-                                 tions inversions change linkage relationships sectors appear C+ clones. diploid C- seg-                             his4 chromosome III markers, transposition C+ <br />                                                                                            consistent genetic data. Reciprocal translocations <br />                                                                                            generate duplication-deficiency chromosomes                                                                                            abnormal pairing rise regular ratios                                                                                            4:0 0:4 spore viability (5). observe 2:2 (viabre:lethal) <br />                   his4       C+ (lethal)                                 C+ (lethal)       0:4. Inversions chromosome III rise            eu2 B C                                                                     tetrads viable lethal spores result                                                                  his4 <br />             +                   40~              Ieu2 B C <br />          0 <br />  <br />                                                     (crossed his4C-)                               thr4       mat   cdc1O Ieu2     his4    HMa <br />                   Meiosis                                Meiosis <br />                                                              4 <br />         2 viable (Cit: 2 lethal                      4 viable:O lethal <br />                                                  3 viable:1 lethal <br />          occasionally viable                         2 viable:2 lethal <br />            C+ aneuploids                                          B <br />    FIG. 1. Structure meiotic products    R2  R2-B, R3 C+ diploids, showing transposition insertion C+ different chromosome production recessive lethal <br /> mutation; (B) R2  R2-B, R3 C+ aneuploids. rare C+                                FIG. 2. Map chromosome III. his4 region shown sub- <br /> euploids crossed haploid, 5140-3A:his4C-716                               divided  B, C. relevant point deletion muta- <br /> Holl.                                                                                      tions shown. <br />  4008       Genetics: Greer Fink                                                     Proc. Natl. Acad. Sci. USA 76 (1979) <br />  <br />          Table 2. Genotypes C- spores C+ diploids             yielded tetrads segregated 2:2, 3:1, 4:0 viable:lethal <br />                    Expected                                           spores (Table 3). C+ region segregated manner <br />                                                                 expected chromosome carrying lethal allele <br />  Spores       unlinked      linked      R2       R2-B      R3       homologous chromosomes carrying wild-type alleles <br />                                                                       trisomic strain. tetrads viable spores gave <br />  Total           100         100         119        106      48       2C+:2C-, viable spores gave lC+:2C-,                                                                       viable spores gave OC+:2C-.  chromosome <br />  LEU2+           50           83         50          50      27       containing C+ lethal transposition spore  leu2-           50           17         69          56      21 <br />                                                                       normal chromosome cover  spore died. coincident <br />    Linkage Ieu2 C+ determined number         segregation C+ lethality aneuploid crosses fur- Leu+ Leu- spores total number tetrads analyzed     ther evidence transposition model. C+ diploids Table 1.                                         Viable C+ Progeny Unstable. Spontaneous C- segre- <br />                                                                       gants C+ aneuploids arose frequency 1-10%. single crossover inversion.  C+ allele         aneuploid strains C- ways:   Loss frequent meiotic products C-            extra C+ chromosome, restoring haploid strain  <br /> allele, tetrads arising meiosis crossover          crossed, viable spores tetrad; (ii) loss viable spores, C+. data         C+ gene precise excision C+ DNA, resulting inconsistent inversion hypothesis C-        aneuploid strain  crossed, spores appear tetrads.  tetrad analysis original   viable spores tetrad; (iii) imprecise excision C+ <br /> diploid revertants suggests kind transposition        gene, resulting aneuploid strain lethal deletion, <br /> his4 occurred.                                                     crossed haploid, 4:0, 3:1, 2:2    Original his4 Region Intact C+ Revertants.               viability. crossed, C- segregants restored 4:0 <br /> Transposition C+ segment chromosome           segregation pattern viability, suggesting extra <br /> result excision C gene chromosome III           chromosome lost C+ gene excised precisely <br /> subsequent transposition synthesis additional copy      (Table 3). C gene transposition extra C region.              Chromosome III Genes Haploid Gene C. <br /> excision model predicts deletion C region exist    Genetic analysis indicated C gene transposed. copies chromosome III,               C+ aneuploid strains crossed haploid (HG293-1B) <br /> &quot;synthesis&quot; model predicts chromosome III homolog           multiply marked chromosome III mutations intact his4 region original his4-260        thr4, mat, cdclO, leu2, his4A, B, C (Fig. 2). his4-39 double mutations.                                             genes transposed, &quot;+&quot; alleles <br />    structure his4 region R2  R2-B, R3         &quot;-&quot; allele present 2n + 1 strains, diploids determined genetic fine structure analysis         4+:0-, 3+:1-, 2+:2- meiotic segregation patterns viable spores. meiotic spores       gene. genes tested segregated 2+:2- gene C. <br /> germinate diploids, alleles ho-           C gene appears transposed. <br /> mologs chromosome III (leu2-1 LEU2 +, mat               Aneuploid Chromosome XII. Aneuploid mapping <br /> mat  appeared equal frequency spores,                takes advantage irregular meiotic segregation ratios <br /> suggesting his4 regions present diploid         caused presence extra chromosome. aneu- <br /> present haploid spore progeny. Recombination tests        ploid, n + 1, carrying wild-type alleles (+), crossed eu- <br /> performed viable spores           ploid, n, carrying recessive mutant alleles (-) R2  R2-B, R3 strains defined point mutations           different chromosomes. Genes chromosomes present deletions his4 region (Fig. 2). Point mutations            copies +/- configuration 2n + 1 strain <br /> his4A-506, 4A-25, 4A-1111, 4B-594, 4C-864, 4C-290                 usual 2+:2- meiotic segregation. Genes recombined viable spores  recombi-           chromosomes present copies +/+/- <br /> nants.  his4-260, his4-39, deletions 15, 30, 29 failed  recombinants spores.         Table 3. Spore viability C+ aneuploids C- segregants, <br /> pattern recombination identical shown his4-260 his4-39 double mutations used construct                                                     Viables:lethals <br /> initial diploid, HG183.  his4 region                                              4:0           3:1       2:2 <br /> spores intact, showing transposition did leave                    R2     C+                6           11        3 <br /> deletion C+ diploids.                                                              C-               14            6        0 <br /> C+ aneuploids                                                                    R2-B     C+               5            16        7 <br />     Viable C+ Progeny Rare. shown                                   C-(1)           17             3        0 <br />  mutant diploids undergo meiosis, viable spores                        C-(2)           34             4        2 <br />  C-.  1% tetrads dissected contained <br />  viable C+ spore. sections viable               R3       C+                3           17        0 <br />  C+ spores aneuploids (n + 1) chromosome                                C-               17            3        0 <br />  C+ gene integrated. C+ aneuploid spores               Spore viability C+ aneuploids, R2 = 5997-31B  C  survived lethal mutation resulted C+       his4-260,39 HOLI), R2-B = 6001RS1  C+/his4-260,39 HOLJ),  integration event recessive normal gene present        R3 = 5999-lB  C+/his4-260,39 Ieu2 HOLI), C- segregants  extra chromosome (Fig. 1).                                       aneuploids determined crossing strains ha- <br />     Viable C+ Progeny Aneuploids. Crosses viable           ploid, his4C-716 HOLl analyzing meiotic spores growth <br />                                                                       YEP medium 30&apos;C. Yeast spores occasionally fail germinate <br />  C+ progeny showed pattern spore viability indicating n          wild-type crosses. lethal spores C- segments <br />  + 1 aneuploidy. C+ progeny crossed haploid (n)         attributed background lethality characteristic yeast <br />  produce 2n + 1 trisomic strains. meiosis, strains      strains. <br />                Genetics: Greer Fink                                                           Proc. Natl. Acad. Sci. USA 76 (1979)          4009 <br />    configuration 4+:0-, 3+:1-, 2+:2- segrega-                markers chromosome. Genes ordered    tion. aberrant segregation recessive auxotropic marker             frequency different recombinant classes obtained    identifies extra chromosome. identify extra chro-                double crossover events frequent <br />    mosome C+ region inserted, crossed C+                 class. <br />    aneuploids set tester strains 17 linkage              C+ insertion R2 mapped cross HG576 = <br />    groups represented recessive auxotrophic marker             HG556-5C  C + ASP5 + RAD5 + GAL2 + /asp5 rad5 gal2 <br />    (Table 4). crosses different strains carrying asp5             his4-260,39 HOL1) X 6218-36D  asp5 rad5 GAL2 + <br />    chromosome XII), aberrant segregation observed 5075%                 hts4C-720 arg4 HOL1). 120 tetrads, 26 recombinant spores <br />    tetrads. genes 16 chromosomes segre-                C+ Asp- Rad-, 9 C+ Asp+ Rad-, 0 C+ <br />   gated 2+:2- virtually tetrads.  R2  R2-B, ard              Asp- Rad+.  order C+ asp5 rad5. C+  <br />    R3 aneuploid chromosome XII.                                   sertion R3 mapped cross HG583 = HG558-12B        Additional crosses chromosome XII markers rad5                   C + ASP5 + RAD5 + GAL2 + /asp5 rad5 GAL2 + his4C-720 <br />   gal2 confirmed chromosome XII aneuploidy. C+                        arg4 HOL1) X 6218-28A  asp5 GAL2+ his4C-720 arg4 <br />   Asp+ Rad+ Gal+ aneuploids crossed asp5 rad5 gal2                  HOL1). 100 tetrads, 20 recombinant spores C+ Asp+ <br />   haploid, simultaneous aberrant meiotic segregation ob-                   Rad-, 18 C+ Asp- Rad-, 2 C+ Asp- Rad+.   served genes.                                               order R3 C + asp5 rad5. <br />       Transposed C+ Mapped Relation asp5, rad5,                      transpositions, C+ insertion maps   gal2 Genes. known order genes chromosome XII                   centromere asp gene, consistent results <br />    centromere asp5 rad5 gal2 (10). constructed 2n + 1                    Table 1 indicate centromere linkage C+ <br />   strains chromosome XII markers config-                 insertions. <br />   uration +/ -- chromosomes, &quot;+&quot;                    C- Segregants Types. C+ heterozygous <br />                                                                                aneuploids C + ASP5 + RAD5 + GAL2 + /asp5 rad5 gal2               Table 4. Mapping aneuploid chromosome                            used determine types spontaneous mitotic C- <br />                                                                               segregants arise. Loss extra chromosome produce <br />                                 Number tetrads                             cells simultaneously C- Asp- Rad- Gal- loss                         R2           R2-B            R3                     C+ element yield C- Asp+ Rad + Gal+    Link-             4+:0-         4+:0            4+:0                        heterozygous &quot;+&quot; genes) loss C+    age                                                            adjacent DNA C- Asp- Rad- Gal+ C+ Asp- <br />   group     Gene 3+:1- 2+:2- 3+:1- 2+:2- 3+:1- 2+:2-                          Rad- Gal+. representative data shown Table 5. <br />          adel          0        31          1       26        0       29    types segregants obtained heterozygous <br />   II        Iys2         0        15         0        19        0       13    C+ clone tested: C- Asp+ Rad+ Gal+ C- Asp- Rad- Gal-. <br />   III      mat           0      105          0      109         0     121      C- segregants arise different mecha- <br />            his4          0       49          0       34                 42    nisms: loss just C+ element loss entire extra C+ <br />    IV      trpl          0       46          0       50        0        54    chromosome. observed imprecise excision <br />    V       ura3          0       15          0       24        0        15    event. addition, seen Asp- C+ mitotic se- <br />   VI       his2          1       26          0       26        0        39    gregant, indicating segregation events involve            cdc14         0       12          0       15        0        17    C+ element. <br />   VII      leul          0       29          0       31        0       30        C Segregants Analyzed  rule pos- <br />   VIII     arg4          1       26          0       26        0       39     sibility C- Asp+ Rad+ Gal+ segregants arose            thrl          1       29          0       31        1       30     type recombination event subsequent loss entire <br />   IX       lysI         0        18          0       18        0       23 <br />   X        ura2         0        19         0        11        0       22     extra chromosome XII, distinguish C- Asp+ Rad+ <br />   XI       metl4         1       19         0        11        3       22     Gal+ segregants C- Asp+ Rad+ Gal+/Asp- Rad- Gal- <br />             ural        1        18         0        18        0       23 <br />   XII       asp5       16        27        22       49       19        32          Table 5. C- segregants chromosome XII aneuploids <br />   XIII      lys7        0        19         0        11        0       17                          C- Asp+        C- Asp-          XIV       Iys9        0        12         0       12        0        17                         Rad+ Gal+      Rad- Gal-      clones <br />   XV        ade2        0       24          0       22        0        31 <br />             his3        0       15          0       18        0        15                R2          3              1          750 <br />   XVI       radl        0       12          0       10        0        14                              5              0          580 <br />   XVII met2             0       18          1       18        0       23                               7              2          650 <br />             arg8        0       15          0       24        0        15                             15              2          940 <br />                                                                                                        5              1          700 <br />     Determination aneuploid chromosome achieved                                         12 <br />  crossing R2  R2-B, R3 aneuploids, C+/his4-260,39 HOLI,                                                   0          820 <br />  set haploid tester strains chro-                            47              6         4440 <br />  mosomes marked mutation. tester strains  <br /> 6082-1C  met2 his3 ural his4 lysl -1), 6040-14D  metl4 adel ura2                     R3:            2             8           680 <br />  Iys7 arg4 trpl his4-38 his2), 6093-1C  canR leul Iys9 ade2 thrl),                                    4             6           750 <br /> 5464-7C  asp5 adel his2 his4-38 arg4 trpl cdc14), MC1049-9C                                         3             8           700 <br /> asp5 trpl ura3 his3 arg8), 6095-3C  ade2 radl-1). numbers                                     0             2           730 sum tetrads different crosses contained                                    1             3           250 <br /> particular mutation. lethal segregating cross                                 1             3           410 <br /> (C+ lethal) tetrads 3 viable spores. spored <br /> tetrads 3+ spores 4+:0- 3+:1-,                                        11            30          3520 aberrant types tetrads listed          C+ aneuploid strains (n + 1) heterozygous asp5, rad5, gal2 <br /> column. addition, tetrads showing 2+:1- assigned             constructed. independent cultures hetero- <br /> normal (2+:2-) aberrant (3+:1-). included 2+:2-       zygous C+ aneuploids analyzed. horizontal line numbers <br /> column. Occasionally marker addition asp5 shows aberrant          indicates number clones screened genotypes spon- <br /> tetrad, likely result gene conversion.               taneous C- segregants obtained cultures. <br />  4010     Genetics: Greer Fink                                                    Proc. Nati. Acad. Sci. USA 76 (1979) <br />  <br /> segregants. Crosses segregants Asp+ Rad+          vertants arise mutation vital gene chro- <br /> Gal+ haploid Asp+ Rad+ Gal+ progeny spores          mosome XII function altered perform his4C C- segregants chromosome XII         reaction. mechanism unlikely account results Asp+ Rad+ Gal+ Asp- Rad- Gal- spores C-              fails explain instability C+ revertants. <br /> segregants heterozygous. C- Asp+ Rad+ Gal+ segregants               evidence exists possibility C+ trans- R2  R2-B, R3 produce Asp- Rad- Gal-              positions inducing chromosomal aberrations. initial R2 <br /> spores.  tetrads C- segregants         mutant gave rise clonal types, C+ 4 viable:0 lethal, indicating simultaneous loss lethal   centromere linked chromosome XII loss C+.  excision C+ gene loss        type chromosomal rearrangement subsequent lethal concomitant chromosome loss confirmed          initial selection responsible event.  C- Asp+ Rad+ Gal+ segregants shows precise          C- segregants aneuploid strain shows 4:0 <br /> excision C+ transposition occurs.                             0:4 (viable:lethal) segregation patterns, indicate <br />                                                                      reciprocal translocation (unpublished data). addition, <br />                          DISCUSSION                                  preliminary evidence indicates transposed C+ strains obtained unstable transpositions his4C (histi-        mutagenic, giving rise wide spectrum spontaneous <br /> dinol dehydrogenase) gene yeast. transpositions        mutations unexpectedly high frequencies, 5 X 10-4   <br /> selected UV irradiation demanding expression          published data). <br /> C gene strain contained nonsense mutations          thank Jean DeLucia technical assistance. research upstream position   his4 region. independent       supported Jane Coffin Childs Memorial Fund Medical  <br /> mutants-R1, R2, R3, R4 obtained frequency              search Postdoctoral Fellowship  H.G.) National Institutes of 1 x 10-9 surviving cell. R2 R3 transpositions        Health Grant GM15408  G.R.F.). discussed paper. R4 restores C activity long frame deletion, causing recessive lethal      1. Bukhari, , Shapiro, J. <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Adhya, S., eds. (1977) DNA Insertion <br /> (unpublished data). R1 extensively analyzed,             Elements, Plasmids, Episomes (Cold Spring Harbor Lab- lethal transposition deletion event               oratory, Cold Spring Harbor, NY). <br /> generated.                                                            2. McClintock, B. (1950) Proc. Natl. Acad. Sci. USA 36, 344- <br />                                                                          355. <br />     R2 R3 diploids, expression C occurs      3. McClintock, B. (1961)  Nat. 95,265-277. <br /> transposition C gene chromosome XII. transposed-        4. Green, M. (1977) DNA Insertion Elements, Plasmids, gene integrated essential gene, causing          Episomes, eds. Bukhari, , Shapiro, J. &amp; Adhya, S. (Cold Spring <br /> cessive lethal. initial transposition C gene appears          Harbor Laboratory, Cold Spring Harbor, NY), pp. 437-445. occurred synthesis new copy subsequent            5. Perkins, D. (1974) Genetics 77,459-489. <br /> transposition excision followed transposition,      6. Fink, G. (1970) Methods Enzymol. 17A, 59-78. produced deletion original site.        7. Fink, G. (1966) Genetics 53,445-459. <br /> transposed gene unstable rise C- segregants        8. Bigelis, R., Keesey, J. &amp; Fink, G. (1977) ICN-UCLA <br /> </body></html>