<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Evolutionary Bioinformatics <br />  <br />  <br />                                                                                                              Open Access <br />                                                                                                        open access                                                                                                       thousands papers O r g nal Resea r c h <br />                                                                                                         http://www.la-press.com. <br />  <br />  <br /> Prediction Protein Essentiality Support Vector <br /> Machine Statistical Tests <br />  <br /> Chiou-Yi Hor, Chang-Biau Yang, Zih-Jie Yang Chiou-Ting Tseng <br /> Department Science Engineering, National Sun Yat-sen University Kaohsiung 80424, Taiwan. <br /> Corresponding author email: cbyang@cse.nsysu.edu.tw <br />  <br />  <br />  <br />  <br /> Abstract: Essential proteins include minimum required set proteins support cell life. Identifying essential proteins important understanding cellular processes organism.  identifying essential proteins experimentally extremely time- <br /> consuming labor-intensive. Alternative methods developed examine essential proteins. goals study: identifying important features building learning machines discriminating essential proteins. Data Saccharomy- <br /> ces cerevisiae Escherichia coli used. collected information variety sources. proposed modified <br /> backward feature selection method build support vector machines (SVM) predictors based selected features. evaluate performance, conducted cross-validations originally imbalanced data set sampling balanced data set. statistical tests applied performance associated obtained feature subsets confirm significance. data <br /> set, best values F-measure Matthews correlation coefficient (MCC) 0.549 0.495 imbalanced experiments. balanced experiment, best values F-measure MCC 0.770 0.545, respectively. second data set, best values F-measure MCC 0.421 0.407 imbalanced experiments. balanced experiment, best values F-measure MCC 0.718 0.448, respectively. experimental results selected features compact performance improved. Prediction conducted users following internet address: http://bio2.cse.nsysu.edu.tw/ <br /> esspredict.aspx. <br />  <br /> Keywords: support vector machine, feature selection, protein-protein interaction, essential protein, statistical test <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9 387&#226;&#8364;&#8220;416 <br />  <br /> doi: 10.4137/EBO.S11975 <br />  article available http://www.la-press.com. <br />  <br /> &#194;&#169; author(s), publisher licensee Libertas Academica  <br />  open access article published Creative Commons CC NC 3.0 license. <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                  387 <br />  Hor et&#194;&#160;al <br />  <br />  <br /> Introduction                                                preliminary properties Identifying essential proteins important       somewhat relevant essentiality.  <br /> standing cellular processes organism      information highly accessible proteins perform functions essen-       sequence information required derive tial proteins. essential protein removed,        properties.  included properties dysfunction cell death results.  stud-      study. topological properties, addition physi- <br /> ies conducted identify essential proteins.     cal interactions, incorporated variety interac- <br /> Experimental approaches identifying essential           tion information, including metabolic, transcriptional <br /> proteins include gene deletion,1 RNA interference,2         regulation, integrated functional, genomic  conditional knockouts.3  meth-            text interactions. experimental results revealed <br /> ods labor-intensive time-consuming.           features provide complement infor- <br /> alternative methods identifying essential proteins      mation essentiality identification provide necessary.                                              biological justification. <br />     essential protein classification problem               identify reduced feature subset, involves determining necessity protein sus-    crucial biological processes, previous studies <br /> taining cellular function life. methods        used feature selection techniques. advan- <br /> available identifying essential proteins, machine-      tages method include storage reduction, <br /> learning based methods promising approaches.            performance improvement data interpretation.12  studies conducted          accordance feature selection <br /> examine effectiveness technique. Chin4         procedure bound predictor, method <br /> proposed double-screening scheme constructed         roughly classified categories: filter, framework known hub analyzer (http://hub.iis.     wrapper, embedded. Filter methods pro- <br /> sinica.edu.tw/Hubba/index.php) rank &#194;&#173;proteins.      vide complete order available features terms <br /> Acencio Lemke5 used Waikato Environment            relevance measures. Methods Fisher <br /> Knowledge Analysis (WEKA)6 predict essen-           score,12 mutual information, minimal redundancy <br /> tial proteins. Hwang et&#194;&#160; al7 applied support vector      maximal relevance (mRMR),13 conditional <br /> machine (SVM) classify proteins.                    mutual information maximization (CMIM),14     Protein-protein interactions (PPIs) known     minimal relevant redundancy (mRR)15 belong significant characteristics protein function.     category. wrapper embedded methods studies attempted predict classify     involve selection process learning <br /> protein function8 analyze protein pheno-        &#194;&#173;algorithm. utilizes learning machine type9 studying interactions. previous study10 fur-     evaluate subsets features according  <br /> ther suggested essential proteins nonessential     formance measurements. example, sequential <br /> proteins discriminated means topological       backward forward feature selection12 falls properties derived PPI network. spite        &#194;&#173;category. Embedded methods directly perform superior properties,  analyzing PPI       feature selection learning process experimentally time-consuming. advent        usually specific given learning machines. Example <br /> yeast hybrid11 high-throughput techniques,        include C4.5,16 Classification Regression Trees used identify PPIs experi-         (CART),17 ID3.18 Additionally, research- <br /> ment, obtaining PPI information easier.          ers proposed information-gain based feature PPI network similar social network          selection method,19 examines effective- aspects, researchers apply social net-         ness classifier combination. <br /> work techniques analyzing PPI networks.               paper, used datasets. topological properties extensively        Saccharomyces cerevisiae. corresponding <br /> explored studied recent years.                       PPI data set Scere20070107, obtained <br />     Fundamental properties, sequence             DIP database. data set totally contains <br /> protein physiochemical ones, subjected           4873 proteins 17,166&#194;&#160;interactions. feature set <br /> detailed examination previous studies.          consisted features obtained extracted  <br /> 388	                                                                                    Evolutionary Bioinformatics 2013:9 <br />                                                                                             Essentiality prediction SVM <br />  <br />  methods proposed Acencio Lemke,5 Chin,4          &#194;&#173; atabase (SGD), located http://www. <br />                                                             D <br /> Hwang et&#194;&#160; al,7 Lin et&#194;&#160; al.20 second data set        yeastgenome.org/. data set used Escherichia coli, compiled         previous studies, obtained incor- Gustafson et&#194;&#160; al.21 data set totally contained       porated various related features experiments. <br /> 3569 proteins. associated network information              E. coli data set obtained Gustafson <br /> included physical, integrated functional, genomic       et&#194;&#160;al.21 contained 3569 proteins, 611 <br /> context interactions collected Hu et&#194;&#160;al.22      essential. availability coverage issues, data sets, propose modified sequen-           used information additional <br /> tial backward feature selection method selecting        networks: physical interaction (PI), integrated func- <br /> important features.                                         tional interaction, integrated PI genomic <br />      SVM models built using selected          context (GC) network. information collected <br /> feature subsets. study, SVM software LIB-       Hu et&#194;&#160;al.22 <br /> SVM23 adopted classification models.              data sets, ratio nonessential <br /> model applied imbalanced balanced           proteins essential proteins approximately 4:1 <br /> data sets. results compared          5:1, respectively. data imbalance inevi- <br /> previous studies statistical tests conducted    tably led biased fitting nonessential proteins <br /> examine significance. imbalanced S. &#194;&#173;cerevisiae     learning processes.  constructed <br /> data, best results F-measure MCC           balanced data set. Taking data set, 0.549 0.495, respectively, outperform         example, randomly selected 975 nonessential pro- <br /> best previous method7 results 0.354 0.36,       teins mixed essential proteins form <br /> respectively. obtained values 0.770 0.545     balanced data set. new data set, number <br /> F-measure MCC balanced data experiment,          nonessential data elements essential superior best previous method7        elements equal. <br /> 0.737 0.492, respectively. experiments exam- <br /> ining E. coli data set, best values F-measure   Bootstrap cross-validation MCC 0.421 0.407, respectively,          used bootstrap cross-validation (BCV) compare imbalanced data set. balanced experiment,        performance classifiers using k-fold cross- <br /> best values F-measure MCC 0.718            validation. Assume sample S&#194;&#160;=&#194;&#160;{(x1, y1), (x2, y2), &#226;&#8364;&#166;, <br /> 0.448, respectively. results similar    (xn, yn)} composed n observations, xi represents <br /> Gustafson et&#194;&#160; al,21 examined 29 features,        feature vector ith observation yi denotes method, seven features used       class <span id='am-36' about='rdfs:label' typeof='owl:Thing'>label</span> associated xi. bootstrap sample <br /> prediction. verify improvement                 {                                      } <br />                                                             Sb* = ( x1* , y1* ) , ( x *2 , y2* ) , &#226;&#8364;&#166;, ( x *n , yn* ) consists n <br /> statistically significant, performed bootstrap cross-    obser&#194;&#173;vations sampled S replacement, <br /> validation24 performance measures.                       1 # b # B, B constant 50                                                             200. sample Sb*, k-fold cross-&#194;&#173;validation Background                                                  carried  performance measure cb, error data set                                                rate, calculated Sb*. procedure repeated paper, used data sets experiments:       B times average performance measure <br /> S. cerevisiae E. coli. included PPI          CB&#194;&#160; =&#194;&#160; &#226;&#710;&#8216;Bb=1 cb/B evaluated B bootstrap <br /> network data. downloaded data set           samples. distribution bootstrap  <br /> DIP (http://dip.doe-mbi.ucla.edu/) website.25           formance measures approximately normal, original data set contained 4873 proteins 17166         confidence interval significance estimated <br /> interactions. comply previous studies,           accordingly. adopted largest connected component network data. total 4815 proteins,          Performance measures <br /> including 975 essential proteins 3840 nonessen-         study, performance measures included <br /> tial proteins. information protein essential-        precision, recall, F-measure (F1), Matthews cor- <br /> ity obtained Saccharomyces Genome              relation coefficient (MCC), percentage <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                         389 <br />  Hor et&#194;&#160;al <br />  <br />  essential proteins. formulas given             tion, desire protein. follows:                                                       select 2 proteins using ranking method  <br />                                                                W X.  use method B rank <br /> 1.	 Precision: TP&#226;&#8364;&#8240;/&#226;&#8364;&#8240;TP + FP <br />                                                                proteins. ranks W X 2 <br /> 2.	 Recall: TP&#226;&#8364;&#8240;/&#226;&#8364;&#8240;TP + FN <br />                                                                1, respectively.  iteration, X <br /> 3.	 F-measure: 2 &#195;&#8212; precision &#195;&#8212; recall&#226;&#8364;&#8240;/(&#226;&#8364;&#352;precision + recall) <br />                                                                finally selected. follows bit M [X, 1] set <br />                         TP &#195;&#8212; TN &#226;&#710;&#8217; FP &#195;&#8212; FN                      1,  M [W, 1], M [Y, 1] M   1], <br /> 4.	 MCC: <br />              (TP + FN )(TP + FP)(TN + FP)(TN + FN )            set 0. second iteration, 2 ranking <br />                                                                proteins   proteins W, X, <br /> 5.	     percentage        essential    protein: <br />                                                                Y selected, 4 <br />     Number real essential proteins n .               proteins ranking method   ranking <br />                         n                                      method B, select 2 proteins  <br />      essential protein represented           X Y.  bits corresponding <br /> positive observation. True positive (TP), true neg-            M [X, 2] M [Y, 2] set 1, ative (TN), false positive (FP), false negative            set 0 iteration. Finally, sum (FN) represent numbers true positive, true              bits protein, shown fourth column <br /> negative, false positive, false negative proteins,         Table&#194;&#160;3. <br /> &#194;&#173;respectively. value n denotes total num-                  issue bit <span id='am-3' about='xsd:string' typeof='owl:Thing'>string</span> implemen- <br />  ber predictions. addition, receiver operating           tation,  M sparse handled <br />  characteristic (ROC) curve18 area curve             &#194;&#173;classifiers. number proteins  (AUC) used evaluate classification                selected n/2, sum n/2&#194;&#160; bits  performance.                                                  close 0. experience, makes difficult <br />                                                                distinguish proteins. overcome Feature extraction                                             problem, protein, added score feature set used included sequence properties           n &#226;&#8364;&#8220; r sum bit string, r rank <br /> (S), <span id='am-4' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-5' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> occurrence average amino           protein ranking method B. study, <br /> acid PSSM; protein properties (P), cell cycle          used DMNC rank MNC rank B. metabolic process; topological properties (T),        example, n&#194;&#160;=&#194;&#160;4, values n &#226;&#8364;&#8220; r W, X, Y bit string double screening scheme           0, 2, 3, 1, respectively. summed ness centrality related physical interactions;    values bit string;  final scores properties (O), phyletic retention essential       0, 4, 4, 1. overall procedure given index. total 45&#194;&#160;groups 90 features         Procedure bit string implementation DSS. S. cerevisiae data set. E. coli data set, 35&#194;&#160;groups 80 features. names            Sequential backward feature <br /> sources shown Table&#194;&#160;1. Bit string double        selection method <br /> screening scheme presented.                                 SVM established tool data analysis     remaining features detailed                 shown useful various fields, Appendix.                                                      text summarization,27 intrusion detection,28 image <br />     Lin et&#194;&#160;al26 Chin4 proposed double screen-          coding.29 study, utilized SVM software <br /> ing scheme. used multiple ranking scores sort          developed Chang Lin, called LIBSVM.23 essential proteins. drawback protein          address data imbalance, propose modified <br /> does unique score.  propose bit           sequential backward feature selection method. <br /> string implementation incorporate prop-              data nonessential, choosing erties single score.                                    accuracy objective adopting conventional fea- <br />     example bit string implementation             ture ranking schemes favored negative data. shown Tables&#194;&#160;2 3. Suppose proteins,           features excluded, overall accuracy <br /> W, X, Y,  ranked. itera-          declined. number negative data &#194;&#173;elements <br />  <br />  <br /> 390	                                                                                       Evolutionary Bioinformatics 2013:9 <br />                                                                                              Essentiality prediction SVM <br />  <br />  <br /> Table 1. Protein features. <br />  <br /> <span id='am-35' about='oboInOwl:id' typeof='owl:Thing'>ID</span>    Property                                                           Type   Size   Sub-names      S. cere   E. coli <br /> 1     Amino acid occurrence           20 <br />                                                                               S      20     Y            &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 2     Average amino acid PSSM20                                               S      20     Y            &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 3     Average cysteine position20                                             S      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 4     Average distance cysteines20                               S      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 5     Average hydrophobic20                                                   S      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 6     Average hydrophobicity cysteine20                                S      4      1&#226;&#8364;&#166;4            &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 7     Cysteine count20                                                        S      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 8     Cysteine location20                                                     S      5      1&#226;&#8364;&#166;5            &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 9     Cysteine odd index20                                               S      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 10    Protein length20                                                        S      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 11    Cell cycle5                                                             P      1                     &#226;&#8364;&#162; <br /> 12    Cytoplasm5                                                              P      1                     &#226;&#8364;&#162; <br /> 13    Endoplasmic reticulum5                                                  P      1                     &#226;&#8364;&#162; <br /> 14    Metabolic process5                                                      P      1                     &#226;&#8364;&#162; <br /> 15    Mitochondrion5                                                          P      1                     &#226;&#8364;&#162; <br /> 16    Nucleus5                                                                P      1                     &#226;&#8364;&#162; <br /> 17    process5                                                          P      1                     &#226;&#8364;&#162; <br /> 18    localization5                                                     P      1                     &#226;&#8364;&#162; <br /> 19    Signal transduction5                                                    P      1                     &#226;&#8364;&#162; <br /> 20    Transport5                                                              P      1                     &#226;&#8364;&#162; <br /> 21    Transcription5                                                          P      1                     &#226;&#8364;&#162; <br /> 22    Betweenness centrality related interactions41                    T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 23    Betweenness centrality related metabolic interactions5               T      1                     &#226;&#8364;&#162; <br /> 24    Betweenness centrality related physical interactions5                T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 25    Betweenness centrality transcriptional regulation interactions5         T      1                     &#226;&#8364;&#162; <br /> 26    Bit string double screening scheme  paper]                      T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 27    Bottleneck8,41                                                          T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 28    Clique level7                                                           T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 29    Closeness centrality42                                                  T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 30    Clustering coefficient7                                                 T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 31    Degree related interactions43                                    T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 32    Degree related physical interactions5                                T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 33    Density maximum neighborhood component4                              T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 34    Edge percolated component9                                              T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 35    Indegree related metabolic interaction5                              T      1                     &#226;&#8364;&#162; <br /> 36    Indegree related transcriptional regulation5                         T      1                     &#226;&#8364;&#162; <br /> 37    Maximum neighborhood component4                                         T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 38    Neighbors&#226;&#8364;&#8482; intra-degree7                                                T      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 39    Outdegree related metabolic interaction5                             T      1                     &#226;&#8364;&#162; <br /> 40    Outdegree related transcriptional regulation interaction5            T      1                     &#226;&#8364;&#162; <br /> 41    Betweenness centrality related integrated functional interaction22   T      1                               &#226;&#8364;&#162; <br /> 42    Betweenness centrality related integrated PI GC network22        T      1                               &#226;&#8364;&#162; <br /> 43    Degree related integrated functional interaction22                   T      1                               &#226;&#8364;&#162; <br /> 44    Degree related integrated PI GC network22                        T      1                               &#226;&#8364;&#162; <br /> 45    Common function degree7                                                 O      1                     &#226;&#8364;&#162; <br /> 46    Essential index7                                                        O      1                     &#226;&#8364;&#162; <br /> 47    Identicalness5                                                          O      1                     &#226;&#8364;&#162; <br /> 48    Open reading frame length7                                              O      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 49    Phyletic retention21                                                    O      1                     &#226;&#8364;&#162;         &#226;&#8364;&#162; <br /> 50    Number paralagous genes21                                            O      1                               &#226;&#8364;&#162; <br /> 51    Codon Adaptation Index (CAI)21,44                                       O      1                               &#226;&#8364;&#162; <br /> 52    Codon Bias Index (CBI)21,44                                             O      1                               &#226;&#8364;&#162; <br />                                                                                                                 (Continued) <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                      391 <br />  Hor et&#194;&#160;al <br />  <br />  <br /> Table 1. (Continued) <br />  <br /> ID     Property                                                                          Type      Size     Sub-names          S. cere       E. coli <br /> 53     Frequency optimal codons         21,44 <br />                                                                                               O         1                                         &#226;&#8364;&#162; <br /> 54     Aromaticity score21,44                                                                 O         1                                         &#226;&#8364;&#162; <br /> 55     Leading strand circular chromosome21                                            O         1                                         &#226;&#8364;&#162; <br />        Total                                                                                            100                         90            80 <br /> Notes: S. cere E. coli mean Saccharomyces cerevisiae Escherichia coli datasets, respectively. topological features, particularly <br /> mentioned, related physical interactions. coverage availability issue, adopt different features S. cere E. coli datasets. example, interactions E. coli data set contain integrated functional, PI, GC network information S. cere include metabolic, <br /> transcriptional regulation PI network information. <br /> Abbreviations: GC, genomic context; PI, physical interactions. <br />  <br />  <br />  higher positive factors, true-                              fewer features compete &#194;&#173;positive rate decreased true-nega-                           features.  <br />  tive rate.  features selected  positive samples correctly classified dete-                       C(S)&#194;&#160;=&#194;&#160; wP * P(S)&#194;&#160;+&#194;&#160;wR * R(S)&#194;&#160;+&#194;&#160;wF * F (S) + wM <br />  riorating overall accuracy  sense,                                 * M(S) &#195;&#8212; (|S| - t) * u(|S| - t) * e, <br />  using accuracy guide feature <br />  selection, used composite score C objec-                            S denotes selected feature subset. |S| t <br />  tive &#194;&#173;function. composite score represented                          denote size S goal number features <br />  terms &#194;&#173;precision (P), recall (R), F-measure (F),                         specified user, respectively. unit step func- <br />  MCC (M) given C =&#194;&#160;wP *&#194;&#160;P&#194;&#160;+&#194;&#160;wR&#194;&#160;*&#194;&#160;R +&#194;&#160;wF                               tion u(|S| &#226;&#8364;&#8220; t)&#194;&#160;=&#194;&#160;0 |S| &#226;&#8364;&#8220; t # 0, u(|S| &#226;&#8364;&#8220; t)&#194;&#160;=&#194;&#160;1. <br />  * F + wM * M. adjustable weights, wP, wR, wF,                          Finally, threshold &#207;? adopted make ensure <br />  wM, used, leading compromise                              improvement feature changes  associated performance measures. additional pun-                             random process. value &#207;? estimated  ishment imposed C allow scores associated                             comparing average score difference feature <br />                                                                                  subsets sizes p p&#194;&#160; +&#194;&#160; 1&#194;&#160; preliminary run <br />                                                                                  given different values p. value e <br />                                                                                  denotes penalty score additional fea- <br />      Procedure: Bit String Implementation DSS <br />                                                                                  ture selected. score specified      input: P: proteins ranking, |P|&#194;&#160;=&#194;&#160;n <br />                                                                                  user slightly larger &#207;? encour- <br />      &#226;&#8364;&#402;&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8240;&#226;&#8364;&#8240;  ranking methods,  &#194;&#160;=&#194;&#160;m&#194;&#160;=&#194;&#160;2 <br />                                  n                                               age feature subsets smaller sizes. procedure <br />      &#226;&#8364;&#402;&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8240;M: bit matrix n &#195;&#8212; 2 , initialized 0 <br />                                                                                  feature selection described Procedure  <br />      output: Protein ranking scores R <br />                                                                                  ward feature selection. <br />      begin <br />      &#226;&#8364;&#402; =&#194;&#160;1 n R &#194;&#160;=&#194;&#160;0; <br />      &#226;&#8364;&#402; =&#194;&#160;1 &#239;&#163;&#176;n/2&#239;&#163;&#187;                                                     Experimental procedure results <br />      &#226;&#8364;&#402;&#226;&#8364;&#402; T1&#194;&#160;= 2i proteins ranked 1) P;                                comparison purposes, used feature selec- <br />      &#226;&#8364;&#402;&#226;&#8364;&#402; T2&#194;&#160;= proteins ranked 2) T1;                                tion methods: mRMR CMIM. S. cerevisiae <br />      &#226;&#8364;&#402;&#226;&#8364;&#402; foreach x &#226;&#710;&#710; T2 M [x, &#194;&#160;=&#194;&#160;1; <br />      &#226;&#8364;&#402; end <br />                                                                                  Table 2. Ranking different methods, smaller <br />      &#226;&#8364;&#402; =&#194;&#160;1 n                                                         numbers indicate higher ranks. <br />      &#226;&#8364;&#402;&#226;&#8364;&#402; foreach j&#194;&#160;=&#194;&#160;1 &#239;&#163;&#176;n/2&#239;&#163;&#187; R &#194;&#160;=&#194;&#160;R  <br />          +&#194;&#160;M   j];                                                             Protein                     Ranking method <br />      &#226;&#8364;&#402; end                                                                                                        (DMNC)                    B (MNC) <br />      &#226;&#8364;&#402; T&#194;&#160;=&#194;&#160;protein orders ranked m) ;                                       W                                1                           4 <br />      &#226;&#8364;&#402; foreach =&#194;&#160;1 n R &#194;&#160;=&#194;&#160;R &#194;&#160;+&#194;&#160;(n &#226;&#8364;&#8220; T  )                           X                                2                           2 <br />      end                                                                         Y                                3                           1 <br />                                                                                                                 4                           3 <br />  <br />  <br /> 392	                                                                                                                  Evolutionary Bioinformatics 2013:9 <br />                                                                                                   Essentiality prediction SVM <br />  <br />  <br />                                                           Table 3. Bit strings double screening method. <br />    Procedure: Backward Feature Selection <br />                                                           Protein          th iteration             Sum bit           n-r              Sum <br />    input : e: penalty score                                                                      string <br />                                                                            1st          2nd <br />    	       k: number folds cross-validation <br />    	       r: maximal number rounds retry           W                0            0             0                    0                0 <br />    	 S0: set available features,             X                1            1             2                    2                4 <br />                                                           Y                0            1             1                    3                4 <br />            |S0|&#194;&#160;=&#194;&#160;n                                                      0            0             0                    1                1 <br />    	       t: goal number selected features <br />    	 wP, wR, wF wM: weights perfor- <br />            mance measures                                 used Gustafson&#226;&#8364;&#8482;s feature set benchmark.    	       &#207;?: minimal improvement proceed              feature sets considerably effective various <br />    output: Selected feature subset S                      performance measures. <br />    data: 50% available data elements <br />    begin                                                  Stage 2: Tune SVM parameters <br />    &#226;&#8364;&#402; S&#194;&#160;=&#194;&#160;S0;                                              best performance <br />    &#226;&#8364;&#402; t&#194;&#160;$&#194;&#160;n stop m&#194;&#160;=&#194;&#160;n &#226;&#8364;&#8220; t;                   feature sets, ran SVM <br />    &#226;&#8364;&#402; p&#194;&#160;=&#194;&#160;1 m                                    software using feature sets Hwang Gustafson <br />    	       improved&#194;&#160;=&#194;&#160;False ;                             tuned SVM parameters achieve highest <br />    	       =&#194;&#160;1 r                              average performances. <br />    	&#226;&#8364;&#402; Conduct k-fold CV calculate C (S); <br />    	&#226;&#8364;&#402; foreach s &#226;&#710;&#710; S                                    Stage 3: Adopt best performances <br />    	&#226;&#8364;&#402;&#226;&#8364;&#402; Conduct k-fold CV calculate C                  reference performances <br />                  (S &#226;&#8364;&#8220; {s});                               determining best SVM parameters    	&#226;&#8364;&#402; end                                                 feature sets Hwang Gustafson, recorded    	&#226;&#8364;&#402; q&#194;&#160;=&#194;&#160;argmaxs &#226;&#710;&#710; S {C (S &#226;&#8364;&#8220; {s}) &#226;&#8364;&#8220; C (S)};              SVM parameters results. compare results <br />    	&#226;&#8364;&#402; C (S - {q}) &#226;&#8364;&#8220; C (S) $ &#207;?                     models, obtained using    	&#226;&#8364;&#402;&#226;&#8364;&#402; S&#194;&#160;=&#194;&#160;S - {q};                                       methods, mRMR CMIM, used SVM <br />    	&#226;&#8364;&#402;&#226;&#8364;&#402; improved&#194;&#160;=&#194;&#160;True;                                   software adjust cost parameters SVM    	&#226;&#8364;&#402;&#226;&#8364;&#402; break;                                             order achieve similar levels precision. <br />    	&#226;&#8364;&#402; end <br />      &#226;&#8364;&#402;&#226;&#8364;&#402; end                                               Stage 4: Perform feature selection <br />      &#226;&#8364;&#402;&#226;&#8364;&#402; improved stop output S            randomly chose 50% available data.     &#226;&#8364;&#402; end                                                  backward feature selection procedure applied    end                                                    selected data. beginning feature selec- <br />                                                           tion procedure, imposed penalty score cal- <br />                                                           culation.  procedure attempts achieve data set, results compared      highest score. subsequent runs, added penal- <br /> Acencio Lemke5 Hwang et&#194;&#160;al.7 E. coli      ties feature sizes score &#194;&#173;calculation. &#194;&#173;Subsets <br /> data set, compared results      smaller feature size slightly &#194;&#173;inferior Gustafson et&#194;&#160;al.21 <br />  <br /> Experimental procedure                                              Determining <br />                                                                benchmark feature sets <br />                                                                                                  Tuning SVM <br />                                                                                             parameters best <br />                                                                                                                           Adopting best <br />                                                                                                                          performances  overall procedure experiments illus- <br />                                                                                                 performances         reference performances <br />  <br />  <br /> trated Figure&#194;&#160;1 described follows.                                                  Performing <br />                                                                      Performing               conventional         Performing feature <br />                                                                   significance tests            bootstraping               selections <br /> Stage 1: Determine benchmark feature set                                                      cross-validations <br />  <br />  S. cerevisiae data set, used Hwang&#226;&#8364;&#8482;s fea-      Figure 1. Flowchart construction SVM models performance <br /> ture set benchmark. E. coli data set,   comparison. <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                           393 <br />  Hor et&#194;&#160;al <br />  <br />  <br /> performance selected. compare results             o&#194;&#173; pportunities exploit available feature combina- methods, used mRMR             tions achieve best performance. initial CMIM feature ranking methods chose subsets             run, obtained feature subset 18 size. Procedure backward feature selection.                        subsequent runs, value t                                                                decreased starting 17 (=&#194;&#160; 18 - 1) perfor- <br /> Stage 5: Perform 10-fold bootstrap                         mances significantly worse Hwang <br /> cross-validations                                              et&#194;&#160; al.7 order obtain feature subsets reduced data prepared balanced imbal-             sizes, parameters set: k&#194;&#160;=&#194;&#160;2, wP&#194;&#160;=&#194;&#160;1.03, wR&#194;&#160;=&#194;&#160;1, <br /> anced manners. data set, randomly                  wF&#194;&#160;=&#194;&#160;1, wM&#194;&#160;=&#194;&#160;1, &#207;?&#194;&#160;=&#194;&#160;0.005, e&#194;&#160;=&#194;&#160;0.01, r&#194;&#160;=&#194;&#160;5. rea- <br /> &#194;&#173;partitioned data 10 disjoint groups used         son setting wP&#194;&#160;=&#194;&#160;1.03 prevent true posi- <br />  feature subsets selected previous stage         tive rate decreasing  addition, e . &#207;? <br />  calculate various performance measures. data              allow procedure proceeded fewer <br />  prepared balanced imbalanced                 features. settings used encourage <br />  &#194;&#173;manners. 10-fold cross validation repeated           selection reduced feature subsets. <br />   10 times average performance measures                   setting t, executed procedure <br />   computed.  bootstrap sampling procedure           10 times different k-fold partitions obtained <br />   conducted 200 bootstrap samples produced,           10 feature subsets size. 10 <br />   &#194;&#173;including balanced imbalanced &#194;&#173;samples.            resultant feature subsets slightly different,    bootstrap sample partitioned              performed 5-fold cross-validation    10-fold cross validations performance measures          feature subsets compared performance <br />    &#194;&#173;calculated. Note models examined        scores. highest score finally <br />    sets data partitions conventional        preserved feature subset. <br />    bootstraping cross-validations.                             addition methods Hwang et&#194;&#160; al                                                                Gustafson et&#194;&#160;al,21 used mRMR13 CMIM14 <br /> Stage 6: Perform significance tests                            feature selection methods comparison. Using bootstrap cross validations carried              mRMR example, data used feature significance tests adopted accordingly.            selection procedure input mRMR pro- <br /> addition average values AUC, precision,              gram, produced ranking score fea- <br /> recall, F-measure, MCC, conducted statisti-           ture. feature score removed <br /> cal significance test performance measures.          subsequent 5-fold cross-validation Additionally, calculated ROC curves             preserved features performed calculate centage values imbalanced experiments.                     composite score C (S).  ith iteration, <br />                                                                features lowest ranking scores Backward feature selection                                     removed C (S) calculated. removal mRMR/CMIM feature ranking                                  cross-validation procedure repeated fea- used 50% available data elements fea-                ture preserved. entire process (including ran- <br /> ture selection. Taking S. cerevisiae data set        dom choice 50% data feature removal) example, (3840&#194;&#160;+&#194;&#160;975)&#194;&#160;&#195;&#8212;&#194;&#160;50% observations             executed 10 times feature subset randomly chosen Backward feature selec-                size highest score recorded. <br /> tion procedure. procedure,                  Table&#194;&#160;4&#194;&#160;shows selected feature subsets dif- <br /> formance scores calculated means k-fold             ferent sizes S. cerevisiae data. second column <br /> cross-&#194;&#173;validations. run, parameters           table lists selected features. Ni set follows: k&#194;&#160; =&#194;&#160; 2, wP&#194;&#160; =&#194;&#160; 1, wR&#194;&#160; =&#194;&#160; 1, wF&#194;&#160; =&#194;&#160; 1,    row represents feature subset size  4 # <br /> wM&#194;&#160;=&#194;&#160;1, &#207;?&#194;&#160;=&#194;&#160;0.005, e&#194;&#160;=&#194;&#160;0, t&#194;&#160;=&#194;&#160;0, r&#194;&#160;=&#194;&#160;5.          # 18, using backward feature <br /> associated weights equal, procedure sought            selection procedure. feature subset Ni, best compromise performance mea-                 bullet (&#226;&#8364;&#162;) mark column used <br /> sures. t&#194;&#160; =&#194;&#160; e&#194;&#160; =&#194;&#160; 0, meaning goal number           indicate feature included. selected features imposed, giving procedure            competent feature subsets selected CMIM  <br /> 394	                                                                                       Evolutionary Bioinformatics 2013:9 <br />                                        Table 4. Selected features S. cerevisiae data set. <br />  <br />                                            Feature                             N5    N6    N7   N8   N9   N10   N11   N12   N13   N14   N15   N16   N17   N18   m31   C32   TOT <br />                                       1    PR (phyletic retention)             &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     16 <br />                                       2    EI (essentiality index)             &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     16 <br />                                       3    Cytoplasm                           &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;           &#226;&#8364;&#162;     15 <br />                                       4    Nucleus                             &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;           &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     15 <br />                                       5    Occurrence                             &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;           &#226;&#8364;&#162;     13 <br />                                       6    Bit string DSS                         &#226;&#8364;&#162;     &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;           &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;                 12 <br />                                       7    Occurrence  W                                 &#226;&#8364;&#162;    &#226;&#8364;&#162;    &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;           &#226;&#8364;&#162;     12 <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	 <br />                                       8    Endoplasmic reticulum                                     &#226;&#8364;&#162;    &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;           11 <br />                                       9    process                                                                          &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     7 <br />                                       10   Occurrence  S                                                       &#226;&#8364;&#162;           &#226;&#8364;&#162;           &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     7 <br />                                       11   Occurrence  G                                                 &#226;&#8364;&#162;           &#226;&#8364;&#162;           &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;                 6 <br />                                       12   KLV (clique level)                                                         &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;           &#226;&#8364;&#162;                 &#226;&#8364;&#162;     &#226;&#8364;&#162;     6 <br />                                       13   Cell cycle                                                                 &#226;&#8364;&#162;           &#226;&#8364;&#162;           &#226;&#8364;&#162;                 &#226;&#8364;&#162;     &#226;&#8364;&#162;     5 <br />                                       14   Average hydrophobic                                                              &#226;&#8364;&#162;           &#226;&#8364;&#162;           &#226;&#8364;&#162;     &#226;&#8364;&#162;           &#226;&#8364;&#162;     5 <br />                                       15   Average PSSM  R                                         &#226;&#8364;&#162;     &#226;&#8364;&#162;                       &#226;&#8364;&#162;           &#226;&#8364;&#162;     &#226;&#8364;&#162;                 5 <br />                                       16   B.C. related PI                                                               &#226;&#8364;&#162;                       &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;           4 <br />                                       17   Occurrence  E                                                             &#226;&#8364;&#162;                 &#226;&#8364;&#162;                 &#226;&#8364;&#162;     &#226;&#8364;&#162;     4 <br />                                       18   Average PSSM  P                                         &#226;&#8364;&#162;     &#226;&#8364;&#162;                       &#226;&#8364;&#162;                             &#226;&#8364;&#162;     4 <br />                                       19   ID related T.R.                                                                                       &#226;&#8364;&#162;     &#226;&#8364;&#162;     &#226;&#8364;&#162;           3 <br />                                       20   B.C. T.R. interactions                                                                       &#226;&#8364;&#162;                 &#226;&#8364;&#162;     &#226;&#8364;&#162;           3 <br />                                       21   localization                                                                                       &#226;&#8364;&#162;     &#226;&#8364;&#162;           &#226;&#8364;&#162;     3 <br />                                       22   DMNC                                &#226;&#8364;&#162;     &#226;&#8364;&#162;                                                                                &#226;&#8364;&#162;     3 <br />                                       23   Average HYD C-2                                                     &#226;&#8364;&#162;           &#226;&#8364;&#162;           &#226;&#8364;&#162;                             3 <br />                                       24   Signal transduction                                                                                      &#226;&#8364;&#162;     &#226;&#8364;&#162;                 2 <br />                                       25   Edge percolated component                                                                                            &#226;&#8364;&#162;     &#226;&#8364;&#162;     2 <br />                                       26   Occurrence  P                                                                                                 &#226;&#8364;&#162;     &#226;&#8364;&#162;     2 <br />                                       27   Occurrence  T                                 &#226;&#8364;&#162;    &#226;&#8364;&#162;                                                                      2 <br />                                       28   Occurrence  Y                                                                                                 &#226;&#8364;&#162;     &#226;&#8364;&#162;     2 <br />                                       29   Average PSSM  Q                                                           &#226;&#8364;&#162;                                         &#226;&#8364;&#162;     2 <br />                                       30   Average PSSM  E                                                                 &#226;&#8364;&#162;           &#226;&#8364;&#162;                             2 <br />                                       31   CLC (clustering coefficient)                                                                                         &#226;&#8364;&#162;     &#226;&#8364;&#162;     2 <br />                                       32   FunK (common function degree)                                                                                        &#226;&#8364;&#162;     &#226;&#8364;&#162;     2 <br />                                       33   OD related T.R. interaction                                                                                       &#226;&#8364;&#162;           1 <br />                                       34   OD related M                                                                                                    &#226;&#8364;&#162;           1 <br />                                       35   ID related M                                                                                                    &#226;&#8364;&#162;           1 <br />                                       36   B.C. related M                                                                                                  &#226;&#8364;&#162;           1 <br />                                       37   Degree related PI                                                                                                       &#226;&#8364;&#162;     1 <br />                                       38   Metabolic process                                                                                                    &#226;&#8364;&#162;           1 <br />                                       39   Bottleneck                                                                                                           &#226;&#8364;&#162;           1 <br />                                       40   MNC                                                                                                                  &#226;&#8364;&#162;           1 <br />                                       41   Occurrence                                                                                                        &#226;&#8364;&#162;     1 <br />                                       42   Occurrence  C                                                                                                 &#226;&#8364;&#162;           1 <br />                                       43   Occurrence  D                                                                                                       &#226;&#8364;&#162;     1 <br />  <br />  <br />  <br />  <br /> 395 <br />                                                                                                                                                                                     Essentiality prediction SVM <br />  <br />  <br />  <br />  <br />                                                                                                                                                                       (Continued) <br />      Hor et&#194;&#160;al <br />  <br />  <br />                                                                                                                                                                                                                                                           mRMR denoted C32 m31, means <br />  <br />  <br />  <br />  <br />                                                              Abbreviations: DSS, double screening scheme; , amino acid; B.C., betweenness centrality; T.R., transcriptional regulation; HYD, hydrophobicity; PI, physical interaction; &#226;&#8364;&#166; Y, amino <br />                        TOT <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           32 31 features selected, respectively. <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                  1 <br />                                                                                                                                                                                                                                                           observed total 60 features                        C32 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           selected, represent prominent proteins used <br />                                   &#226;&#8364;&#162; <br />  <br />                                       &#226;&#8364;&#162; <br />  <br />  <br />  <br />  <br />                                                      &#226;&#8364;&#162; <br />                                                      &#226;&#8364;&#162; <br />                                                      &#226;&#8364;&#162; <br />                                                      &#226;&#8364;&#162; <br />                                                      &#226;&#8364;&#162; <br />                                                      &#226;&#8364;&#162; <br />                                                                                                                                                                                                                                                           identify essential proteins. <br />                        m31 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                               feature subsets selected, conduct <br />                                  &#226;&#8364;&#162; <br />  <br />                                      &#226;&#8364;&#162; <br />  <br />                                          &#226;&#8364;&#162; <br />  <br />                                              &#226;&#8364;&#162; <br />  <br />                                                  &#226;&#8364;&#162; <br />  <br />  <br />  <br />  <br />                                                         &#226;&#8364;&#162; <br />                                                                                                                                                                                                                                                           performance comparison cope                                                                                                                                                                                                                                                           randomness, used Hwang&#226;&#8364;&#8482;s method perform <br />                        N18 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           10 10-fold cross-validations.  true positive <br />                                                                                                                                                                                                                                                           rates false positive rates input diferent <br />                        N17 <br />  <br />  <br />  <br />  <br />                                                              acid abbreviation; M , metabolic interaction; OD, outdegree; ID, indegree; m31, mRMR31; C32, CMIM32; FunK, Common function degree; TOT, total. <br />                                                                                                                                                                                                                                                           software program calculate ROC curves AUC <br />                                                                                                                                                                                                                                                           values. study, software package used <br />                        N16 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           ROCR, developed Tobias et&#194;&#160; al.30,31 <br />                                                                                                                                                                                                                                                            reported performance measures, includ- <br />                        N15 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           ing AUC, F1, MCC, precision, recall values                                                  &#226;&#8364;&#162; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           ROC curves, averaged 10 10-fold cross- <br />                                                                                                                                                                                                                                                           validations. <br />                        N14 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                               S. cerevisiae data set, predictor                                              &#226;&#8364;&#162; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           Hwang&#226;&#8364;&#8482;s 10 features served benchmark                        N13 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           yielded distinguished results terms feature size <br />                                                                                                                                                                                                                                                           performance. Additionally, mRMR CMIM <br />                        N12 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           adopted comparison. <br />                                                                                                                                                                                                                                                               appled procedure E. coli data <br />                        N11 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           set. selected features shown Table&#194;&#160;5,                                                                                                                                                                                                                                                           total 43 features selected. table, feature <br />                        N10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           subsets selected CMIM mRMR denoted <br />                                                                                                                                                                                                                                                           C9 m13, meaning 9 13 features                        N9 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           selected, respectively. <br />                        N8 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           Bootstrap cross validations <br />                                                                                                                                                                                                                                                           bootstrapping stage, bootstrap <br />                        N7 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           sample, identical 10-fold partition employed <br />                                              &#226;&#8364;&#162; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           feature subsets carry cross-validations <br />                        N6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           compute various average performance measures. <br />                                                                                                                                                                                                                                                           procedure repeated 200 distinct boot- <br />                        N5 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           strap samples. order perform parametric signifi- <br />                                                                                                                                                                                                                                                           cance tests, evaluated distribution                                                                                                                                                                                                                                                           resultant performance measures normal                                                                                                                                                                                                                                                           variances obtained different feature sub- <br />                                  Average HYD C-1 <br />  <br />  <br />  <br />  <br />                                  Average PSSM  W <br />  <br />  <br />                                  CC (closeness centrality) <br />                                  Average PSSM  D <br />                                  Cysteine odd index <br />  <br />  <br />  <br />  <br />                                  Average PSSM  S <br />  <br />                                  Average PSSM  Y <br />                                  Average PSSM   <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           sets similar. Consequently, 200 results                                  Occurrence  M <br />                                  Occurrence  H <br />                                  Occurrence  K <br />  <br />  <br />  <br />  <br />                                  ORFL (ORF length) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           performance measure feature subset                                  Cysteine location-1 <br />                                  Average C position <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           subjected Kolmogorov-Smirno test.31 test <br /> Table 4. (Continued) <br />  <br />  <br />  <br />  <br />                                  Cysteine count <br />                                  Protein length <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           examines null hypothesis systematic differ- <br />                                                                                                                                                                                                                                                           ence exists standard normal distribution <br />                                  BC (B.C.) <br />                        Feature <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                           underlying distribution alterna- <br />                                                                                                                                                                                                                                                           tive asserts systematic difference.                                                                                                                                                                                                                                                           threshold set 0.05. P-value                                                                                                                                                                                                                                                           0.05, rejected null hypothesis. CMIM                                  44 <br />                                  45 <br />                                  46 <br />                                  47 <br />                                  48 <br />                                  49 <br />                                  50 <br />                                  51 <br />                                  52 <br />                                  53 <br />                                  54 <br />                                  55 <br />                                  56 <br />                                  57 <br />                                  58 <br />                                  59 <br />                                  60 <br />  <br />  <br />  <br />  <br />     396	                                                                                                                                                                                                                                                                               Evolutionary Bioinformatics 2013:9 <br />                                                                                                             Essentiality prediction SVM <br />  <br />  <br /> Table 5. Selected features E. coli data set. <br />  <br />       Feature                                     N4     N5    N6     N7     N8     N9     N10     N11     N12      N13     C9     m13      TOT <br /> 1     PR (phyletic retention)                     &#226;&#8364;&#162;      &#226;&#8364;&#162;     &#226;&#8364;&#162;      &#226;&#8364;&#162;      &#226;&#8364;&#162;      &#226;&#8364;&#162;      &#226;&#8364;&#162;       &#226;&#8364;&#162;       &#226;&#8364;&#162;        &#226;&#8364;&#162;       &#226;&#8364;&#162;      &#226;&#8364;&#162;        12 <br /> 2     Open reading frame length                          &#226;&#8364;&#162;            &#226;&#8364;&#162;      &#226;&#8364;&#162;      &#226;&#8364;&#162;      &#226;&#8364;&#162;       &#226;&#8364;&#162;       &#226;&#8364;&#162;        &#226;&#8364;&#162;                       8 <br /> 3     Average PSSM  C                      &#226;&#8364;&#162;      &#226;&#8364;&#162;     &#226;&#8364;&#162;      &#226;&#8364;&#162;      &#226;&#8364;&#162;                                      &#226;&#8364;&#162;                       6 <br /> 4     Degree related F                       &#226;&#8364;&#162;      &#226;&#8364;&#162;     &#226;&#8364;&#162;      &#226;&#8364;&#162;      &#226;&#8364;&#162;                                      &#226;&#8364;&#162;                       6 <br /> 5     Degree related                                                         &#226;&#8364;&#162;      &#226;&#8364;&#162;       &#226;&#8364;&#162;       &#226;&#8364;&#162;                &#226;&#8364;&#162;               5 <br /> 6     Degree related PI                                                          &#226;&#8364;&#162;      &#226;&#8364;&#162;       &#226;&#8364;&#162;       &#226;&#8364;&#162;                       &#226;&#8364;&#162;        5 <br /> 7     Average PSSM                                           &#226;&#8364;&#162;      &#226;&#8364;&#162;                                              &#226;&#8364;&#162;      &#226;&#8364;&#162;        4 <br /> 8     Average PSSM  R                                                        &#226;&#8364;&#162;      &#226;&#8364;&#162;       &#226;&#8364;&#162;       &#226;&#8364;&#162;                                4 <br /> 9     Average hydrophobic                                                           &#226;&#8364;&#162;      &#226;&#8364;&#162;       &#226;&#8364;&#162;       &#226;&#8364;&#162;                                4 <br /> 10    Bit string DSS PI                                                      &#226;&#8364;&#162;      &#226;&#8364;&#162;       &#226;&#8364;&#162;       &#226;&#8364;&#162;                                4 <br /> 11    Paralog count                                            &#226;&#8364;&#162;                                           &#226;&#8364;&#162;        &#226;&#8364;&#162;              &#226;&#8364;&#162;        4 <br /> 12    Occurrence  M                                                   &#226;&#8364;&#162;      &#226;&#8364;&#162;                      &#226;&#8364;&#162;                                3 <br /> 13    Occurrence  W                                                                 &#226;&#8364;&#162;                        &#226;&#8364;&#162;       &#226;&#8364;&#162;               3 <br /> 14    Occurrence  E                               &#226;&#8364;&#162;     &#226;&#8364;&#162;                                                                            2 <br /> 15    Occurrence  F                                                                         &#226;&#8364;&#162;                        &#226;&#8364;&#162;               2 <br /> 16    Occurrence  G                                                          &#226;&#8364;&#162;              &#226;&#8364;&#162;                                        2 <br /> 17    Occurrence                                                                                                   &#226;&#8364;&#162;      &#226;&#8364;&#162;        2 <br /> 18    Average PSSM  Y                                                 &#226;&#8364;&#162;                     &#226;&#8364;&#162;                                        2 <br /> 19    Cysteine location-4                                                                                                   &#226;&#8364;&#162;      &#226;&#8364;&#162;        2 <br /> 20    KLV (clique level) PI                                       &#226;&#8364;&#162;                                             &#226;&#8364;&#162;                       2 <br /> 21    Degree related PI GC                                                                  &#226;&#8364;&#162;       &#226;&#8364;&#162;                                2 <br /> 22    Strand bias                                                                                          &#226;&#8364;&#162;                       &#226;&#8364;&#162;        2 <br /> 23    Occurrence                                                                  &#226;&#8364;&#162;                                                1 <br /> 24    Occurrence  C                                                                                                  &#226;&#8364;&#162;               1 <br /> 25    Occurrence  H                                                                                                  &#226;&#8364;&#162;               1 <br /> 26    Occurrence  P                                     &#226;&#8364;&#162;                                                                            1 <br /> 27    Occurrence  S                                                                 &#226;&#8364;&#162;                                                1 <br /> 28    Average PSSM  N                                                                                        &#226;&#8364;&#162;                       1 <br /> 29    Average PSSM  G                                                                                        &#226;&#8364;&#162;                       1 <br /> 30    Average PSSM  K                                          &#226;&#8364;&#162;                                                                     1 <br /> 31    Average PSSM  F                                                 &#226;&#8364;&#162;                                                              1 <br /> 32    Average PSSM  T                                                                                        &#226;&#8364;&#162;                       1 <br /> 33    Average PSSM  V                                                                                        &#226;&#8364;&#162;                       1 <br /> 34    Average distance Cs                                                                     &#226;&#8364;&#162;                                1 <br /> 35    Average HYD C-2                                                                                        &#226;&#8364;&#162;                       1 <br /> 36    Cysteine location-1                                                                                                          &#226;&#8364;&#162;        1 <br /> 37    Cysteine location-5                                                                                                          &#226;&#8364;&#162;        1 <br /> 38    Cysteine odd index                                                                                                      &#226;&#8364;&#162;        1 <br /> 39    Protein length                              &#226;&#8364;&#162;                                                                                         1 <br /> 40    Bottleneck PI                                                                                                            &#226;&#8364;&#162;        1 <br /> 41    CC (closeness centrality) PI                                                                                             &#226;&#8364;&#162;        1 <br /> 42    MNC PI                                                                                                    &#226;&#8364;&#162;                       1 <br /> 43    B.C. related F                                                                                                      &#226;&#8364;&#162;        1 <br /> Abbreviations: C9, CMIM09; m13, mRMR13; TOT, total; DSS, double screening scheme; F , integrated functional interaction; , interactions. <br /> PI, physical interaction; HYD, hydrophobicity; , amino acid; &#226;&#8364;&#166; Y, amino acid abbreviation. <br />  <br />  <br /> mRMR, prominent values shown.                             observed AUC N8-imbalanced experiment. <br /> Figures&#194;&#160;2 3 illustrate results S. cerevisiae                     likely significant E. coli data sets, respectively, test                      difference normal distribution values recorded according feature subsets,                      distribution performance measure performance measures, experiment types.                         feature subset. E. coli data set, perfor- <br /> S. cerevisiae data set, lowest P-value, 0.186,                      mance measures normal exception  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                         397 <br />  Hor et&#194;&#160;al <br />  <br />  <br />                                                                                                  AUC <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                                              0.186 <br />                           0.0 <br />  <br />                                                                                                Precision <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                                                                                                                                          Imbalanced <br />                           0.2 <br />                                                                                            0.239                                         Balanced <br />                           0.0 <br />  <br />                                                                                                 Recall <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                               0.229 <br />                           0.0 <br />  <br />                                                                                                    F1 <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                     0.310 <br />                           0.0 <br />  <br />                                                                                                  MCC <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                                                                      0.373 <br />                           0.0 <br />                                   CMIM32 <br />  <br />                                            mRMR31 <br />  <br />                                                     Hwang <br />  <br />                                                             Acencio <br />  <br />  <br />  <br />  <br />                                                                                                      N10 <br />  <br />                                                                                                            N11 <br />  <br />                                                                                                                  N12 <br />  <br />                                                                                                                        N13 <br />  <br />                                                                                                                              N14 <br />  <br />                                                                                                                                    N15 <br />  <br />                                                                                                                                          N16 <br />  <br />                                                                                                                                                N17 <br />  <br />                                                                                                                                                        N18 <br />  <br />                                                                                                                                                              N90 <br />                                                                       N4 <br />  <br />                                                                            N5 <br />  <br />                                                                                 N6 <br />  <br />                                                                                      N7 <br />  <br />                                                                                           N8 <br />  <br />                                                                                                 N9 <br />  <br />  <br />  <br />  <br /> Figure 2. P-value normality test S. cerevisiae data set. <br />  <br />  recall values associated N12-balanced                                                  Performance comparison <br /> P-value&#194;&#160;=&#194;&#160;0.035), comparing results                                                   significance tests reliable. performance measures associated                                                section, compared experimental results model, listed confidence inter-                                              associated feature selection <br /> val information odds ratios,32 shown                                                methods previous studies. conciseness, Appendix.                                                                                        prominent results associated    certain performance measure, vari-                                                mRMR CMIM. observed feature sizes <br /> ances obtained various feature subsets quite                                                 identified methods relatively large. <br /> similar, used analysis variance33 (ANOVA)                                                   compare feature subsets smaller sizes, test examine differences existed                                                    comparison working principles detailed <br /> performance measures different feature subsets.                                                   Appendix.  variance obtained multiple <br /> experiments feature subset. Differences existed                                               S. cerevisiae <br /> according ANOVA.  measures                                                  Table&#194;&#160;6 lists average values performance compared associated benchmark                                                     measures associated variety feature subsets, <br /> calculate performance deviations. average devia-                                                 obtained 10 10-fold cross-validations <br /> tion corresponding type performance mea-                                                  imbalanced data. adjusted SVM cost param- <br /> sure evaluated using 95% confidence interval                                                 eters order achieve similar levels precision. covering 0 determine significance.                                                                rows results CMIM32 (32 &#194;&#173;features), <br />  <br /> 398	                                                                                                                                                 Evolutionary Bioinformatics 2013:9 <br />                                                                                                                              Essentiality prediction SVM <br />  <br />  <br />                                                                                         AUC <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                                                                      0.344 <br />                           0.2 <br />                           0.0 <br />  <br />                                                                                     Precision <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4                                                                         Imbalanced <br />                           0.2                                                                         Balanced                   0.270 <br />                           0.0 <br />  <br />                                                                                         Recall <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2 <br />                           0.0                                                                                                0.035 <br />  <br />  <br />                                                                                          F1 <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2                                                                      0.139 <br />                           0.0 <br />  <br />                                                                                         MCC <br />                           1.0 <br />                           0.8 <br />                           0.6 <br />                           0.4 <br />                           0.2 <br />                                                                                            0.141 <br />                           0.0 <br />                                   CMIM09 <br />  <br />  <br />                                            mRMR13 <br />  <br />  <br />                                                     Gustafson <br />  <br />  <br />  <br />  <br />                                                                                                                N10 <br />  <br />  <br />                                                                                                                      N11 <br />  <br />  <br />                                                                                                                            N12 <br />  <br />  <br />                                                                                                                                      N13 <br />  <br />  <br />                                                                                                                                            N80 <br />                                                                 N3 <br />  <br />  <br />                                                                          N4 <br />  <br />  <br />                                                                               N5 <br />  <br />  <br />                                                                                    N6 <br />  <br />  <br />                                                                                           N7 <br />  <br />  <br />                                                                                                  N8 <br />  <br />  <br />                                                                                                         N9 <br />  <br />  <br />  <br />  <br /> Figure 3. P-value normality test S. cerevisiae data set. <br />  <br />  <br /> mRMR31 (31 features), Hwang&#226;&#8364;&#8482;s (10 features),                                           significantly higher N8. values Acencio&#226;&#8364;&#8482;s (23 features). Values following items                                      performance measure column, best enclosed parentheses represent numbers                                      underlined. Values row results features. Results produced method listed                                     set 90 features. subsequent rows table. &#194;&#173;Significance tests                                      Based Table&#194;&#160; 6, CMIM32, mRMR31 carried bootstrap cross-validations                                      Hwang&#226;&#8364;&#8482;s predictors outperformed Acencio&#226;&#8364;&#8482;s 200 bootstrap &#194;&#173;samples. symbols,                                      performance measures. feature subsets, plus (+) minus (-), following                                         performance measures slightly higher numerical value represents significantly higher                                         Hwang&#226;&#8364;&#8482;s. N8, perfor- <br /> lower benchmark results. serve                                        mance difference Hwang&#226;&#8364;&#8482;s AUC, benchmarks marked star (*) symbols                                           remaining measure values higher Hwang&#226;&#8364;&#8482;s. <br /> clarity. example, recall N6 significantly                                   feature size exceeded 8, preci- <br /> higher Hwang, AUC sig-                                          sion &#194;&#173;values, improvement Hwang&#226;&#8364;&#8482;s  <br /> nificantly lower mRMR31 Hwang.                                           sistently significant cases. comparison feature subsets prefix &#226;&#8364;&#732;N&#226;&#8364;&#8482;,                                      mRMR, method performed nearly fourth symbols numerical values used                                         mRMR31 feature size 9 indicate significance neighboring                                          13 &#194;&#173;exception AUC values. fea- <br /> rows. example, N7, AUC significantly                                       ture size ranged 14 18, perfor- <br /> higher N6 recall value                                       mance difference model mRMR31. <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                                    399 <br />  400	 <br />                                                                                                                                                                                                                                      Hor et&#194;&#160;al <br />  <br />  <br />  <br />  <br />                                      Table 6. Performance comparison imbalanced S. cerevisiae data set. <br />  <br />                                                           AUC                                Precision                           Recall                             F-measure                          MCC <br />                                      CMIM32               0.825	    *	 (+)	 (+)              0.744	 *		       (+)                0.369	 *		       (+)               0.493	 *		       (+)               0.450	 *		       (+) <br />                                      mRMR31               0.821	   (-)	 &#226;&#8364;&#8240;&#226;&#8364;&#8240;*	 (+)             0.738		      &#226;&#8364;&#8240;*                     0.372		      *	 (+)                0.495		      *	 (+)                0.449		      &#226;&#8364;&#8240;*	 (+) <br />                                      Hwang(10)            0.775	   (-)	 (-)	 &#226;&#8364;&#8240;&#226;&#8364;&#8240;&#226;&#8364;&#8240;*            0.743	 (-)		      &#226;&#8364;&#8240;*                0.343	 (-)	 (-)	 *                 0.469	 (-)	 (-)	 *                 0.432	 (-)	 (-)	 &#226;&#8364;&#8240;* <br />                                      Acencio(23)          0.707	   (-)	 (-)	 (-)             0.675	 (-)	 (-)	 (-)                0.121	 (-)	 (-)	 (-)               0.204	 (-)	 (-)	 (-)               0.228	 (-)	 (-)	 (-) <br />                                      N4                   0.744	   (-)	 (-)	 (-)             0.782                               0.327	 (-)	 (-)	 (-)               0.461	 (-)	 (-)	 (-)               0.439	 (-)	 (-) <br />                                      N5                   0.727	   (-)	 (-)	 (-)             0.741	 (-)			        (-)            0.387	 (-)	 (-)		        (+)       0.509	 (-)	 (-)		         (+)      0.461	 (-)	 (-)		       (+) <br />                                      N6                   0.730	   (-)	 (-)	 (-)             0.752	 (-)                          0.395	 (-)	 (-)	 (+)               0.518	 (-)	 (-)                    0.472	 (-)	 (-) <br />                                      N7                   0.761	   (-)	 (-)		     (+)        0.767	 (-)                          0.386	 (-)	 (-)	 (+)               0.513	 (-)	 (-)	 (+)               0.473	 (-)	 (-) <br />                                      N8                   0.772	   (-)	 (-)                  0.755                               0.371	 (-)	 (-)		        (-)       0.498	 (-)	 (-)	 (+)	     (-)      0.457	 (-)	 (-)		       (-) <br />                                      N9                   0.782	   (-)	 (-)	 (+)             0.749                               0.382	 (-)		     (+)	    (+)       0.506	 (-)		     (+)               0.462	 (-)		     (+) <br />                                      N10                  0.781	   (-)	 (-)	 (+)             0.751                               0.399			         (+)               0.521			         (+)               0.474			         (+) <br />                                      N11                  0.786	   (-)	 (-)	 (+)             0.752                               0.402			         (+)               0.524			         (+)               0.476			         (+) <br />                                      N12                  0.798	   (-)	 (-)	 (+)             0.759                               0.409			         (+)               0.532			         (+)               0.485			         (+) <br />                                      N13                  0.789	   (-)	 (-)	 (+)             0.748                               0.433			         (+)	    (+)       0.549			         (+)               0.495			         (+) <br />                                      N14                  0.802	   (-)		     (+)             0.749                               0.397			         (+)	    (-)       0.519			         (+)	     (-)      0.471			         (+)	   (-) <br />                                      N15                  0.801	   (-)		     (+)             0.763                               0.406			         (+)               0.530			         (+)               0.485			         (+)	   (+) <br />                                      N16                  0.814	   (-)		     (+)	 (+)        0.762                               0.401			         (+)               0.525			         (+)               0.480			         (+) <br />                                      N17                  0.814	   (-)		     (+)             0.761                               0.407			         (+)               0.530			         (+)               0.484			         (+) <br />                                      N18                  0.811	   (-)		     (+)             0.751                               0.411			         (+)               0.531			         (+)               0.482			         (+) <br />                                      N90                  0.829	   (+)	 (+)	 (+)	 (+)        0.738		     (+)	 (+)                0.355		     (+)	 (+)	    (-)       0.479	 (+)	 (+)	 (+)	     (-)      0.438	 (+)	 (+)	 (+)	   (-) <br />                                                                                                                                                                                                                     7 <br />                                      Note: polynomial kernel function, values precision, recall MCC reported 0.77, 0.23, 0.36, respectively, original paper Hwang et al. <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9 <br />                                                                                     Essentiality prediction SVM <br />  <br />  prominent predictor CMIM32.           higher  Table&#194;&#160;9, AUC values, results achieved similar levels    effective predictor mRMR13, shows performance size features exceeded 14.        performance difference feature subsets <br /> Note number features CMIM32            balanced experiments. ROC analysis, <br /> mRMR31 32 31, higher              addition CMIM09, mRMR13, Gustafson&#226;&#8364;&#8482;s,                                                 used N4, N8, N11 N80 draw ROC <br />     Table&#194;&#160; 7&#194;&#160; shows average performance mea-          curves. observed performances sures balanced experiments S. cerevisiae        insufficient, middle feature sets. <br /> data set, obtained 10 10-fold <br /> &#194;&#173;cross-validations. feature subsets      ROC analysis <br />  size ranging 5 18, nearly performance   S. cerevisiae <br />  measures higher        Figure&#194;&#160; 4 illustrates average ROC curves  Hwang&#226;&#8364;&#8482;s. shows feature subsets        AUCs various feature subsets imbalanced <br />  sizes exceeding 5 good Hwang&#226;&#8364;&#8482;s.       data experiments. Apart competent <br />  Additionally, 12 features achieved    predictor CMIM32, AUC N5  significant improvement. Compared CMIM32            higher Acencio&#226;&#8364;&#8482;s, intersection  mRMR31, results showed similar levels         observed 0.5 horizontal axis. indicates <br />  performance size features exceeded 15.       N5 better predictor allowed <br />  results set 90 features shown   maximal false positive rate 0.5.  <br />  row, performance measures         trast, allowed false positive rate exceeded <br />  similar N14 N18.                        0.5, Acencio&#226;&#8364;&#8482;s better N5. Comparing <br />     Table&#194;&#160; 6, observe feature subsets      N9 Hwang&#226;&#8364;&#8482;s method, AUC values  N5, N7, N9, N13, N15, N16&#194;&#160;showed significant         &#194;&#173;similar. feature subsets sizes exceed- <br />  improvement performance smaller fea-       ing 8  shown figure), true positive <br />  ture sizes compared neighboring rows.        rates higher close Hwang&#226;&#8364;&#8482;s. <br />  Table&#194;&#160;7, significant subsets N5, N6, N9.     supported significance tests  addition, shown Tables&#194;&#160;6 7, models       Table&#194;&#160; 6 suggests feature subsets  performed equally CMIM32 mRMR31               sizes exceeding 8 achieved higher performance  feature size 16 17. used N5, N9,       AUC Hwang&#226;&#8364;&#8482;s predictor. <br />  N16 draw ROC curves.                                    Figure&#194;&#160;5 illustrates average ROC curves                                                            AUCs various feature subsets balanced <br /> E. coli                                                    data experiments. CMIM32 com- <br /> Tables&#194;&#160; 8 9&#194;&#160; shows average values          petent predictor. Additionally, N16 achieved <br /> performance measures associated variety          level AUC. feature subsets feature subsets, obtained 10 10-fold         sizes ranging 5 18  shown), true <br /> cross-validations imbalanced balanced experi-      positive rates higher close ments, respectively. rows results    Hwang&#226;&#8364;&#8482;s level.  N5, N6, &#226;&#8364;&#166;, N18 outperformed <br /> CMIM09 (9 features) Gustafson&#226;&#8364;&#8482;s (29 features).         performed equally various combina- <br />    Table&#194;&#160; 8&#194;&#160; shows Gustafson&#226;&#8364;&#8482;s predictors         tions true false positive rates balanced <br /> performed CMIM09&#194;&#160; performance measures             &#194;&#173;experiments. &#194;&#173;Similarly imbalanced data set, imbalanced experiments. feature sub-             features, higher AUC values.   <br /> sets, performance measures slightly higher         improvement AUC feature addition CMIM09. feature size exceeded                 significant imbalanced <br /> 6, improvement CMIM09 consis-                  &#194;&#173;experiments. noted ROC <br /> tently significant. compare Gustafson&#226;&#8364;&#8482;s method            curve AUC Acencio&#226;&#8364;&#8482;s predictor repro- method, performed            duced &#194;&#173;experiments slightly <br /> Gustafson&#226;&#8364;&#8482;s feature size 11. Note          different original values reported &#194;&#173;Acencio number features Gustafson&#226;&#8364;&#8482;s 29,           Lemke.5 <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                            401 <br />  402	 <br />                                                                                                                                                                                                                                Hor et&#194;&#160;al <br />  <br />  <br />  <br />  <br />                                      Table 7. Perforamnce comparison balanced S. cerevisiae data set. <br />  <br />                                                          AUC                               Precision                        Recall                            F-measure                         MCC <br />                                      CMIM32              0.842	 *		       (+)              0.772	 *                         0.766	 *		       (+)              0.769	 *		       (+)              0.540	 *		       (+) <br />                                      mRMR31              0.836		      *	 (+)               0.765		      *                   0.741		      *	 (+)               0.752		      *	 (+)               0.513		     *	 (+) <br />                                      Hwang(10)           0.822	 (-)	 (-)	 *                0.778			          *              0.720	 (-)	 (-)	 *                0.748	 (-)	 (-)	 *                0.516	 (-)	 (-)	 * <br />                                      Acencio(23)         0.768	 (-)	 (-)	 (-)              0.696	 (-)	 (-)	 (-)             0.734	 (-)	 (-)                   0.714	 (-)	 (-)	 (-)              0.414	 (-)	 (-)	 (-) <br />                                      N4                  0.811	 (-)	 (-)	 (-)	    (+)      0.777		     (-)		    (+)         0.716	 (-)	 (-)                   0.745	 (-)	 (-)		    (+)          0.512	 (-)	 (-)		    (+) <br />                                      N5                  0.824	 (-)	 (-)		        (+)      0.778		     (-)                  0.735	 (-)	 (-)		     (+)         0.756	 (-)	 (-)                   0.527	 (-)	 (-) <br />                                      N6                  0.827	 (-)	 (-)		        (+)      0.778		     (-)                  0.739	 (-)	 (-)                   0.758	 (-)	 (-)                   0.530	 (-)	 (-) <br />                                      N7                  0.831	 (-)	 (-)                   0.779                            0.733	 (-)	 (-)                   0.755	 (-)	 (-)                   0.526	 (-)	 (-) <br />                                      N8                  0.826	 (-)	 (-)                   0.786                            0.721	 (-)	 (-)                   0.752	 (-)	 (-)                   0.527	 (-)	 (-) <br />                                      N9                  0.833	 (-)	 (-)		        (+)      0.791                            0.735	 (-)	 (-)		     (+)         0.762	 (-)	 (-)                   0.541	 (-)	 (-) <br />                                      N10                 0.834	 (-)	 (-)                   0.789                            0.736	 (-)	 (-)                   0.761	 (-)	 (-)                   0.540	 (-)	 (-) <br />                                      N11                 0.831	 (-)	 (-)                   0.784                            0.737	 (-)	 (-)                   0.760	 (-)	 (-)                   0.535	 (-)	 (-) <br />                                      N12                 0.829	 (-)	 (-)                   0.779                            0.732                             0.755			         (+)              0.526			         (+) <br />                                      N13                 0.834	 (-)	 (-)                   0.788                            0.730	 (-)	 (-)                   0.758	 (-)	 (-)                   0.535	 (-)	 (-) <br />                                      N14                 0.836	 (-)	 (-)	 (+)              0.777                            0.743	 (-)	 (-)	 (+)              0.759	 (-)	 (-)	 (+)              0.530	 (-)	 (-) <br />                                      N15                 0.843	 (-)	 (-)	 (+)              0.784                            0.748	 (-)	 (-)	 (+)              0.766		     (-)	 (+)              0.542		     (-)	 (+) <br />                                      N16                 0.842			         (+)              0.777                            0.756			         (+)              0.767			         (+)              0.540			         (+) <br />                                      N17                 0.847			         (+)              0.778                            0.763			         (+)              0.770			         (+)              0.545			         (+) <br />                                      N18                 0.840	 (-)	 (-)	 (+)	    (-)      0.779		     (-)                  0.740	 (-)	 (-)	 (+)	 (-)         0.759	 (-)	 (-)                   0.531	 (-)	 (-) <br />                                      N90                 0.839	 (+)		     (+)              0.760                            0.753			         (+)              0.757			         (+)              0.516			         (+) <br />                                      Note: original paper Hwang et al7 values precision, recall, F-measure MCC reported 0.763, 0.713, 0.737, 0.492, respectively, polynomial kernel <br />                                      function. <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9 <br />                                                                                                                                                  Essentiality prediction SVM <br />  <br />  <br />                                                                                                                         E. coli <br />  <br />  <br />                                                                                      (-)	 (+) <br />  <br />  <br />  <br />  <br />                                                                                                              (+)	 (-) <br />                                                                                                                         imbalanced data set, Figure&#194;&#160;6 illustrates aver- <br />                                                                                      (-) <br />                                                                                      (-) <br />  <br />                                                                                      (-) <br />  <br />  <br />  <br />                                                                                                  (-) <br />                                                                                                                         age ROC curves AUCs various feature subsets. <br />                                                                                      * <br />                                                                                      0.413	 (+)	 (+)	 <br />  <br />  <br />  <br />  <br />                                                                                      0.339	 (+)	 (+)	 <br />                                                                                      0.403	 (+)	 (+) <br />  <br />  <br />                                                                                      0.401	 (+)	 (+) <br />                                                                                      0.401	 (+)	 (+) <br />                                                                                      0.393	 (+)	 (+) <br />                                                                                      0.400	 (+)	 (+) <br />                                                                                      0.400	 (+)	 (+) <br />                                                                                                                         shows curves similar 10% <br />                                                                                                  *	 <br />                                                                                      0.382	 *		 <br />  <br />  <br />                                                                                      0.391			 <br />                                                                                      0.407	 (+)		 <br />  <br />  <br />                                                                                      0.405	 (+)		 <br />                                                                                                                         horizontal range. indicates little <br />                                                                                      0.392	 (+)                         difference allowable false positive rate                                                                                      0.360		 <br />                                                                          MCC <br />  <br />  <br />  <br />  <br />                                                                                                                         10%. horizontal range 10%, <br />                                                                                                                         N80 highest performer, Gustafson N11 <br />                                                                                                                         secondary, N4 worst. contrast                                                                                                                         imbalanced data set, Figure&#194;&#160;7 corresponding <br />                                                                                      0.421	 (+)	 (+)	 (-)	 (+) <br />  <br />  <br />                                                                                      0.415	 (+)	 (+)	 (-)	 (+) <br />  <br />  <br />  <br />  <br />                                                                                                            (-) <br />                                                                                                                         balanced data set, N4 N8 best <br />                                                                                                       (-) <br />                                                                                                   *	 (-) <br />  <br />                                                                                                       (-) <br />  <br />  <br />                                                                                      0.400	 (+)	 (+)	 (-) <br />                                                                                      0.420	 (+)	 (+)	 * <br />  <br />  <br />  <br />  <br />                                                                                      0.352	 (+)	 (+)		                  performers. remaining predictors showed                                                                                      0.414	 (+)	 (+) <br />  <br />  <br />                                                                                      0.417	 (+)	 (+) <br />                                                                                      0.418	 (+)	 (+) <br />                                                                                      0.403	 (+)	 (+) <br />                                                                                      0.416	 (+)	 (+) <br />                                                                                      0.413	 (+)	 (+) <br />  <br />                                                                                                                         differences. <br />                                                                                      0.394	 *		 <br />  <br />  <br />                                                                                      0.404	 (+)		 <br />  <br />  <br />  <br />  <br />                                                                                                                         percentage analysis <br />                                                                                      0.370		 <br />  <br />  <br />  <br />  <br />                                                                                                                         S. cerevisiae <br />                                                                          F1 <br />  <br />  <br />  <br />  <br />                                                                                                                         Table&#194;&#160; 10&#194;&#160; shows average percentage infor- <br />                                                                                                                         mation imbalanced data set. &#206;&#184; prob- <br />                                                                                                                         ability defined ratio number truly <br />                                                                                           (-) <br />                                                                                           (+) <br />  <br />  <br />                                                                                                  (+) <br />  <br />  <br />                                                                                                        (-) <br />  <br />  <br />                                                                                                              (-) <br />  <br />  <br />  <br />  <br />                                                                                                                         predicted essential proteins ranked &#206;&#184; &#195;&#8212; <br />                                                                                                       (-)	 <br />  <br />  <br />  <br />                                                                                      0.288	 (+)	 (+)	 (-)	 <br />                                                                                                       (-) <br />                                                                                                   *	 (-) <br />  <br />  <br />  <br />  <br />                                                                                      0.275	 (+)	 (+)	 (-) <br />                                                                                      0.290	 (+)	 (+)	 * <br />  <br />  <br />  <br />  <br />                                                                                                                         975 proteins, total number true essen- <br />                                                                                      0.295	 (+)	 (+)		 <br />  <br />  <br />  <br />  <br />                                                                                      0.278	 (+)	 (+)		 <br />  <br />  <br />                                                                                      0.237	 (+)	 (+)		 <br />                                                                                      0.287	 (+)	 (+) <br />  <br />  <br />                                                                                      0.293	 (+)	 (+) <br />                                                                                      0.294	 (+)	 (+) <br />  <br />                                                                                      0.292	 (+)	 (+) <br />                                                                                      0.287	 (+)	 (+) <br />  <br />  <br />  <br />  <br />                                                                                                                         tial proteins 975. &#206;&#184; probability shows                                                                                      0.271	 *		 <br />  <br />  <br />                                                                                      0.280	 (+)		 <br />  <br />  <br />  <br />  <br />                                                                                                                         likelihood proteins essential user <br />                                                                                      0.250		 <br />  <br />  <br />  <br />  <br />                                                                                                                         decides choose specific number ranked <br />                                                                          Recall <br />  <br />  <br />  <br />  <br />                                                                                                                         candidates. slightly different precision <br />                                                                                                                         ranked candidates  denomina- <br />                                                                                                                         tor) necessary classified essential. <br />                                                                                                            (-) <br />  <br />  <br />  <br />  <br />                                                                                      0.677	 (+)	 (+)	 (+)	 (-) <br />  <br />  <br />  <br />  <br />                                                                                                                         CMIM32, mRMR31 Hwang&#226;&#8364;&#8482;s results       Table 8. Performance comparison imbalanced E. coli data set. <br />  <br />  <br />  <br />  <br />                                                                                                                         served benchmarks denoted star <br />                                                                                                        * <br />  <br />  <br />  <br />  <br />                                                                                      0.726				 <br />  <br />  <br />  <br />  <br />                                                                                                                         &#226;&#8364;&#732;*&#226;&#8364;&#8482; symbols table. minus symbol follow- <br />                                                                                                                         ing value represents value lower <br />                                                                                                  * <br />                                                                                      0.720			 <br />                                                                          Precision <br />  <br />  <br />  <br />  <br />                                                                                                                         benchmark results. <br />                                                                                      0.720	 * <br />                                                                                      0.713		 <br />  <br />  <br />  <br />  <br />                                                                                                                             mRMR31 Hwang&#226;&#8364;&#8482;s predictor                                                                                      0.725 <br />                                                                                      0.737 <br />                                                                                      0.742 <br />                                                                                      0.735 <br />                                                                                      0.742 <br />  <br />                                                                                      0.724 <br />                                                                                      0.732 <br />                                                                                      0.725 <br />                                                                                      0.733 <br />  <br />  <br />  <br />  <br />                                                                                                                         extremely effective 10% range.                                                                                                                         indicates predictors quite preferable <br />                                                                                                                         total number true essential proteins                                                                                      (-) <br />                                                                                      (-) <br />  <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />                                                                                      (-) <br />  <br />  <br />                                                                                                              (+) <br />                                                                                       * <br />  <br />  <br />  <br />  <br />                                                                                                                         known allowable ranked candidates                                                                                      0.711	 (+)	 (+)	 <br />                                                                                      0.691	 (-)	 (-)	 <br />                                                                                      0.690	 (-)	 (-)	 <br />  <br />  <br />  <br />  <br />                                                                                      0.716	 (+)	 (+)	 <br />  <br />  <br />  <br />  <br />                                                                                                                         10%. predictors outperformed <br />                                                                                                   *	 <br />                                                                                      0.701	 *		 <br />  <br />  <br />  <br />  <br />                                                                                      0.701			 <br />                                                                                      0.714			 <br />                                                                                      0.705			 <br />                                                                                      0.707			 <br />                                                                                      0.711			 <br />                                                                                      0.714			 <br />  <br />  <br />  <br />  <br />                                                                                                                         10%. CMIM32, predictors  <br />                                                                                      0.712	 (+) <br />                                                                                      0.714	 (+) <br />  <br />  <br />  <br />  <br />                                                                                                                         performed 30%.  N14 bet- <br />                                                                                      0.715		 <br />  <br />  <br />  <br />  <br />                                                                                                                         ter choice relatively effective                                                                          AUC <br />  <br />  <br />  <br />  <br />                                                                                                                         10%. Figure&#194;&#160; 8 depicts average percentage <br />                                                                                                                         curves. <br />                                                                                      Gustafson(29) <br />  <br />  <br />  <br />  <br />                                                                                                                         E. coli <br />                                                                                      mRMR13 <br />  <br />  <br />  <br />  <br />                                                                                                                         Table&#194;&#160;11&#194;&#160;shows average percentage informa- <br />                                                                                      CMIM09 <br />  <br />  <br />  <br />  <br />                                                                                                                         tion imbalanced data set. CMIM09, mRMR13, <br />                                                                                      N10 <br />  <br />                                                                                      N12 <br />                                                                                      N13 <br />                                                                                      N80 <br />                                                                                      N11 <br />                                                                                      N4 <br />                                                                                      N5 <br />                                                                                      N6 <br />                                                                                      N7 <br />                                                                                      N8 <br />                                                                                      N9 <br />  <br />  <br />  <br />  <br />                                                                                                                         Gustafson&#226;&#8364;&#8482;s results serve &#194;&#173;benchmarks. <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                                                        403 <br />  Hor et&#194;&#160;al <br />  <br />  <br /> Table 9. Performance comparison balanced E. coli data set. <br />  <br />                                     AUC                            Precision          Recall                                      F1                              MCC <br /> CMIM09                              0.767&#226;&#8364;&#8218; *&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                 0.720&#226;&#8364;&#8218; *           0.700&#226;&#8364;&#8218; *&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                              0.710&#226;&#8364;&#8218; *&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                  0.421&#226;&#8364;&#8218; * <br /> mRMR13                              0.762&#226;&#8364;&#8218; (-)&#226;&#8364;&#8218; *&#226;&#8364;&#8218; (-)             0.728&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; *        0.654&#226;&#8364;&#8218; (-)&#226;&#8364;&#8218; *&#226;&#8364;&#8218; (-)                          0.689&#226;&#8364;&#8218; (-)&#226;&#8364;&#8218; *&#226;&#8364;&#8218; (-)              0.396&#226;&#8364;&#8218;&#226;&#8364;&#8218; *&#226;&#8364;&#8218; (-) <br /> Gustafson(29)                       0.777&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)&#226;&#8364;&#8218; *                0.722&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; *      0.715&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)&#226;&#8364;&#8218; *                             0.719&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)&#226;&#8364;&#8218; *                 0.440&#226;&#8364;&#8218; (+)&#226;&#8364;&#8218; * <br /> N4                                  0.780&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                   0.733              0.701&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                                0.717&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                    0.446&#226;&#8364;&#8218; (+) <br /> N5                                  0.779&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                   0.730              0.706&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                                0.718&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                    0.445&#226;&#8364;&#8218; (+) <br /> N6                                  0.762&#226;&#8364;&#402;&#226;&#8364;&#402;&#226;&#8364;&#402;&#226;&#8364;&#8218; (-) (-)              0.735              0.663&#226;&#8364;&#402;&#226;&#8364;&#402;&#226;&#8364;&#402;&#226;&#8364;&#8218; (-) (-)                           0.696                           0.425 <br /> N7                                  0.783&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)            0.737              0.696&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                                0.716&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)             0.448&#226;&#8364;&#8218; (+) <br /> N8                                  0.781&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                   0.723              0.711&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                                0.717&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                    0.439&#226;&#8364;&#8218; (+) <br /> N9                                  0.782&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                   0.715              0.703&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                                0.709&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                    0.423 <br /> N10                                 0.781&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                   0.725              0.702&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                                0.713&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                    0.436&#226;&#8364;&#8218; (+) <br /> N11                                 0.777&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                   0.719              0.700&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                                0.709&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                    0.426 <br /> N12                                 0.776&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                   0.715              0.695&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                                0.705&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                    0.418 <br /> N13                                 0.776&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                   0.731              0.695&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                                0.712&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                    0.439&#226;&#8364;&#8218; (+) <br /> N80                                 0.769                          0.711              0.715&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)&#226;&#8364;&#8218;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                         0.713&#226;&#8364;&#402;&#226;&#8364;&#8218;&#226;&#8364;&#8218; (+)                    0.424 <br />  <br />  <br />  CMIM09 predictor effective                                              amino acid occurrence W, bit string double screen- entire range. predictors outper-                                    ing scheme, cytoplasm, endoplasmic reticulum, EI <br /> formed predictors 15%. N9                                           (essentiality index), nucleus, PR (phyletic reten- prominent relatively effective                                    tion) selected 10 times. entire range. Figure&#194;&#160;9 depicts average                                   features, EI PR, included centage curves.                                                                          feature subsets regarded                                                                                          important factors identifying essential proteins. <br /> Discussion                                                                               N9 N8 feature subsets cover inspecting S. cerevisiae feature subsets listed                                   8 features. prediction capability asso- Table&#194;&#160;4, observed prominent fea-                                     ciated feature subsets outperformed <br /> tures come diverse sources. includes                                    Hwang&#226;&#8364;&#8482;s results performance measures, sequence, protein, topology &#194;&#173;properties.                                       AUC percentage probability features, amino acid occurrence                                            low value. Furthermore, amino acids,                                                                                                                 1.0 <br />                        1.0 <br />  <br />  <br />  <br />  <br />                                                                                                                 0.8 <br />                        0.8 <br />  <br />  <br />  <br />  <br />                                                                                                                                                             CMIM32: 0.842 <br />                                                                                                                                                             mRMR31: 0.836 <br />                                                                                            True positive rate <br />  <br />  <br />  <br />  <br />                                                                                                                                                             Hwang: 0.822 <br />   True positive rate <br />  <br />  <br />  <br />  <br />                                                                CMIM32: 0.825                                                                                Acencio: 0.768 <br />                                                                                                                 0.6 <br />  <br />  <br />  <br />  <br />                                                                mRMR31: 0.821 <br />                        0.6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                             N5: 0.824 <br />                                                                Hwang: 0.775                                                                                 N9: 0.833 <br />                                                                Acencio: 0.707                                                                               N16: 0.842 <br />                                                                N5: 0.727 <br />                                                                N9: 0.782 <br />                                                                                                                 0.4 <br />                        0.4 <br />  <br />  <br />  <br />  <br />                                                                N16: 0.814 <br />                                                                                                                 0.2 <br />                        0.2 <br />  <br />  <br />  <br />  <br />                                                                                                                 0.0 <br />                        0.0 <br />  <br />  <br />  <br />  <br />                                                                                                                       0.0   0.2           0.4         0.6        0.8         1.0 <br />                              0.0   0.2      0.4       0.6           0.8         1.0 <br />                                          False positive rate                                                                           False positive rate <br />  <br /> Figure 4. average ROC curves AUCs imbalanced                             Figure 5. average ROC curves AUCs balanced <br /> S. cerevisiae data set.                                                                  S. cerevisiae data set. <br />  <br />  <br />  <br /> 404	                                                                                                                                            Evolutionary Bioinformatics 2013:9 <br />                                                                                                                       Essentiality prediction SVM <br />  <br />  <br />                                                                                           N9 N6, choosing additional feature,                         1.0 <br />  <br />  <br />  <br />  <br />                                                                                           significant higher N8 N5  <br />                                                                                           formance measures. <br />                         0.8 <br />  <br />  <br />  <br />  <br />                                                                                                E. coli features listed Table&#194;&#160;5,    True positive rate <br />  <br />  <br />  <br />  <br />                                                                 CMIM09: 0.708             second important features PR (phyletic <br />                         0.6 <br />  <br />  <br />  <br />  <br />                                                                 mRMR13: 0.709 <br />                                                                 Gustafson: 0.711 <br />                                                                                           retention) open reading frame length. rest <br />                                                                 N4: 0.691                 important features selected                                                                 N8: 0.705 <br />                                                                                           times included: average PSSM amino <br />                         0.4 <br />  <br />  <br />  <br />  <br />                                                                 N11: 0.714 <br />                                                                 N80: 0.716 <br />                                                                                           acid C degree related integrated functional <br />                                                                                           &#194;&#173;interactions. N5, N7, N8, N13 covered                         0.2 <br />  <br />  <br />  <br />  <br />                                                                                            features. 43 listed features, <br />                                                                                            21&#194;&#160; sequence-related features, amino acid <br />                         0.0 <br />  <br />  <br />  <br />  <br />                                                                                            occurrence average PSSM, selected.                               0.0   0.2      0.4       0.6          0.8            1.0 <br />                                                                                            data set, recommend feature subsets sizes <br />                                           False positive rate <br />                                                                                            exceeding 11 effectiveness Figure 6. average ROC curves AUCs imbalanced E. coli                       compactness feature size. <br /> data set. <br />                                                                                                experimental results data sets,                                                                                            conclude phyletic retention important relatively easy extract, included                                    feature identifying essential proteins. defined feature subsets. predictors built                                     number present ortholog organisms. Gustafson feature subsets 10 features,                                      et&#194;&#160; al study21 analyzed different organisms calcu- <br /> consistently superior Hwang&#226;&#8364;&#8482;s nearly perfor-                                     late phyletic retention E. coli S. cerevisiae <br /> mance measures. Interestingly, mRMR CMIM                                               data sets. sensible different species <br /> selected sequence-derived features,                                        associated different &#194;&#173;organisms. PSSM amino acid occurrence.                                        biological view, retention process long <br /> features good essentiality prediction terms                                     evolutionary periods suggests organisms relevance feature independence. analyzing                                        crucial certain cell &#194;&#173;functionality. inspect- <br /> Tables&#194;&#160;6 7, recommend using N16, N9,                                            ing 5 occurrences amino acids Tables&#194;&#160;4 <br /> N6. N16 performed nearly CMIM32                                              5, Tryptophan (W) <span id='am-11' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-12' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>Glycine</span> (G) mRMR31) compact feature size.                                          &#194;&#173;ranking features. amino <br />                                                                                            acids non-polar hydrophobic, hypoth- <br />                                                                                            esize essentiality related physi- <br />                         1.0 <br />  <br />  <br />  <br />  <br />                                                                                            cochemical properties features possessing <br />                                                                                            discrimination information captured                                                                                            ranking features. <br />                         0.8 <br />  <br />  <br />  <br />  <br />                                                                  CMIM09: 0.775 <br />                                                                                                study, compiled various interaction <br />    True positive rate <br />  <br />  <br />  <br />  <br />                                                                  mRMR13: 0.762 <br />                                                                  Gustafson: 0.777 <br />                                                                                            information including physical, metabolic, tran- <br />                         0.6 <br />  <br />  <br />  <br />  <br />                                                                  N4: 0.780 <br />                                                                  N8: 0.781                 scriptional regulation, integrated functional <br />                                                                  N11: 0.777 <br />                                                                  N80: 0.769                genomic context interactions. experimen- <br />                         0.4 <br />  <br />  <br />  <br />  <br />                                                                                            tal results revealed various properties,                                                                                            degrees, identified impor- <br />                         0.2 <br />  <br />  <br />  <br />  <br />                                                                                            tant features. implies interaction <br />                                                                                            &#194;&#173;information, limited physical interactions, <br />                                                                                             closely related essential properties. <br />                         0.0 <br />  <br />  <br />  <br />  <br />                               0.0   0.2      0.4       0.6           0.8            1.0     According literature, hubs networks, <br />                                           False positive rate                               possessing abundance interaction partners, Figure 7. average ROC curves AUCs balanced E. coli <br />                                                                                             important fact play central <br /> data set.                                                                                   roles mediating interactions numerous <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                               405 <br />  Hor et&#194;&#160;al <br />  <br />  <br /> Table 10. Percentage essential proteins imbalanced S. cerevisiae data. <br />  <br />                                 5%       10%        15%         20%      25%       30%      50%      75%       100% <br /> CMIM32                          0.939*- -    0.918*         0.910*          0.892*       0.870*        0.839*       0.743*       0.645*        0.582* <br /> mRMR31                          0.955&#226;&#8364;&#402; * -   0.905- *&#226;&#8364;&#8218; -    0.884-*         0.862-*      0.834-* -     0.820-*      0.740-*      0.641-*       0.572-* <br /> Hwang(10)                       0.959&#226;&#8364;&#402;&#226;&#8364;&#8218; *    0.918&#226;&#8364;&#402;&#226;&#8364;&#8218; *      0.871- - *      0.853- -*    0.843-&#226;&#8364;&#402; *     0.816- -*    0.720- - *   0.637- - *    0.563- - * <br /> Acencio(23)                     0.800- - -   0.741- - -     0.693- - -      0.661- - -   0.646- - -    0.625- - -   0.578- - -   0.519- - -    0.457- - - <br /> N4                              0.980        0.930          0.905-          0.877-       0.865-        0.850        0.727- -     0.632- - -    0.559- - - <br /> N5                              0.843- - -   0.861- - -     0.859- - -      0.852- - -   0.841- &#226;&#8364;&#8218;&#226;&#8364;&#8240; -   0.827-       0.751        0.641-        0.530- - - <br /> N6                              0.908- - -   0.894- - -     0.875- -        0.857- -     0.850-        0.834-       0.763        0.635- - -    0.526- - - <br /> N7                              0.861- - -   0.892- - -     0.897-          0.885-       0.854-        0.832-       0.770        0.645-        0.570- - <br /> N8                              0.892- - -   0.904- - -     0.895-          0.877-       0.868-        0.850        0.751        0.657         0.574- <br /> N9                              0.880- - -   0.911- &#226;&#8364;&#8218;&#226;&#8364;&#8240; -    0.895-          0.875-       0.860-        0.832-       0.753        0.665         0.585 <br /> N10                             0.882- - -   0.896- - -     0.893-          0.882-       0.858-        0.846        0.762        0.665         0.581- <br /> N11                             0.900- - -   0.900- - -     0.888-          0.875-       0.861-        0.856        0.769        0.667         0.580- <br /> N12                             0.941&#226;&#8364;&#8218; - -   0.924          0.899-          0.872-       0.866-        0.854        0.776        0.664         0.588 <br /> N13                             0.941&#226;&#8364;&#8218; - -   0.910- &#226;&#8364;&#8218;&#226;&#8364;&#8240; -    0.886-          0.870-       0.853-        0.840        0.781        0.672         0.578- <br /> N14                             0.949&#226;&#8364;&#8218; - -   0.932          0.916           0.897        0.867-        0.845        0.759        0.667         0.587 <br /> N15                             0.906- - -   0.894- - -     0.897-          0.884-       0.866-        0.851        0.776        0.672         0.584 <br /> N16                             0.933- - -   0.901- - -     0.895-          0.886-       0.864-        0.851        0.771        0.677         0.599 <br /> N17                             0.943&#226;&#8364;&#8218; - -   0.903- - -     0.879- -        0.871-       0.866-        0.856        0.777        0.679         0.595 <br /> N18                             0.937- - -   0.892- - -     0.880- -        0.870-       0.864-        0.854        0.778        0.683         0.595 <br /> N90                             0.939&#226;&#8364;&#8218; - -   0.911- &#226;&#8364;&#8218;&#226;&#8364;&#8240; -    0.884-          0.869-       0.856-        0.835-       0.728- -     0.639- -      0.572- <br />  <br />  <br />  connected proteins.  proteins involved                                       takes approximately 1&#194;&#160;minute LIBSVM soft- complex mediation processes likely                                        ware perform 2-fold cross-validation crucial cellular activity survival.                                             Power5+ processor IBM P595  &#194;&#173;Assuming <br />    feature selection proposed study, let                                  m&#194;&#160; =&#194;&#160; 90, t&#194;&#160; =&#194;&#160; 10 r&#194;&#160; =&#194;&#160; 5, total running time size available target selected features                                 4,000 20,000&#194;&#160; minutes. IBM P595 <br /> m t, maximal retry times r. num-                                       allows users manually submit processes ber SVM cross-validation times 1/2                                          order speed &#194;&#173;execution. (m&#194;&#160;+&#194;&#160;t&#194;&#160;+&#194;&#160;1) &#195;&#8212; (m - t) 1/2 (m&#194;&#160;+&#194;&#160;t&#194;&#160;+&#194;&#160;1) &#195;&#8212; (m &#226;&#8364;&#8220; t) &#195;&#8212; r.                                  example, invoke 10 SVM processes <br />                                                                                           simultaneously. Consequently, maximal 10-time <br />                                                                                           speed achieved total running time <br />                       1.0 <br />  <br />  <br />  <br />  <br />                                                                    CMIM32                 reduced. <br />                                                                    mRMR31 <br />                                                                    Hwang                      inspect Tables&#194;&#160; 4 5,                       0.9 <br />  <br />  <br />  <br />  <br />                                                                    Acencio <br />                                                                    N5                     features signifi- <br />                                                                    N9 <br />                                                                    N16                    cantly relevant selected.                       0.8 <br />    Ess. probability <br />  <br />  <br />  <br />  <br />                                                                                           features relatively easy remove  <br />                                                                                           ward feature selection procedure beginning <br />                       0.7 <br />  <br />  <br />  <br />  <br />                                                                                           stage. &#194;&#173;According authors&#226;&#8364;&#8482; experience,                                                                                           rounds retry r critical stage.                       0.6 <br />  <br />  <br />  <br />  <br />                                                                                           increasing number features removed, required <br />                                                                                           number retry increased identifying <br />                       0.5 <br />  <br />  <br />  <br />  <br />                                                                                           relatively competent features increas- <br />                                                                                           ingly difficult. number retry r accompanied <br />                       0.4 <br />  <br />  <br />  <br />  <br />                             0       20       40       60      80      100                 rest user-specified parameters                                                    (%)                                 minimal improvement &#207;? et&#194;&#160;al) set appropriately <br /> Figure 8. average percentage curves imbalanced <br />                                                                                           ensure feature selection procedure S. cerevisiae data set.                                                                   proceed. <br />  <br />  <br /> 406	                                                                                                                      Evolutionary Bioinformatics 2013:9 <br />                                                                                                                       Essentiality prediction SVM <br />  <br />  <br /> Table 11. Percentage essential proteins imbalanced E. coli experiment. <br />  <br />                                 5%        10%      15%          20%       25%      30% 50%           75% 100% <br /> CMIM09                          0.745*        0.775*       0.730*           0.730*        0.737*       0.727*       0.644*       0.542*      0.440* - <br /> mRMR13                          0.719-*       0.725-*      0.714-*          0.701-* -     0.690-* -    0.679-* -    0.614-* -    0.531-*     0.446 * <br /> Gustafson(29)                   0.706- -*     0.692- -*    0.705- -*        0.707- &#226;&#8364;&#8218; *    0.695- &#226;&#8364;&#8218; *   0.685- &#226;&#8364;&#8218; *   0.624- &#226;&#8364;&#8218; *   0.522- -*   0.436- -* <br /> N4                              0.610- - -    0.689- - -   0.765            0.760         0.749        0.752        0.649        0.534-      0.449 <br /> N5                              0.655- - -    0.705- -     0.747            0.747         0.745        0.743        0.653        0.535-      0.443&#226;&#8364;&#8218;&#226;&#8364;&#8240;&#226;&#8364;&#8240;- <br /> N6                              0.719-        0.723- -     0.717-           0.736         0.744        0.748        0.658        0.525- -    0.435- - - <br /> N7                              0.568- - -    0.700- -     0.713- -         0.730         0.748        0.762        0.655        0.540-      0.464 <br /> N8                              0.671- - -    0.703- -     0.723-           0.736         0.728-       0.731        0.652        0.535-      0.449 <br /> N9                              0.813         0.785        0.766            0.754         0.734-       0.726-       0.685        0.548       0.459 <br /> N10                             0.794         0.751-       0.728-           0.732         0.741        0.745        0.668        0.550       0.463 <br /> N11                             0.719-        0.721- -     0.734            0.742         0.748        0.748        0.668        0.539-      0.457 <br /> N12                             0.735-        0.738-       0.745            0.750         0.743        0.740        0.667        0.548       0.458 <br /> N13                             0.655- - -    0.672- - -   0.713- -         0.739         0.736-       0.732        0.668        0.548       0.457 <br /> N80                             0.674- - -    0.690- - -   0.703- - -       0.705-&#226;&#8364;&#8218;&#226;&#8364;&#8240;-     0.703-       0.691-       0.632-       0.529- -    0.452 <br />  <br />  <br />  <br /> Conclusion Future Work                                                               method effective term feature size study, incorporated protein proper-                                   performance. method took ties, including sequence, protein, topology,                                   feature size performance consideration properties. total 55&#194;&#160; groups 96                                       consequently resultant feature subset consider- <br /> &#194;&#173;features. features included data sets                                   ably compact. compared experimental results <br />  experiments: S. cerevisiae E. coli. used                                   carrying significance tests types  modified sequential backward feature selection iden-                                 performance &#194;&#173;measures.  provides poten- <br />  tify good feature subsets used SVM software                                     tial researcher essential proteins practical guide  tools classifying essential proteins. addition,                               feature method prominent. <br />  built SVM models imbalanced                                            imbalanced S. cerevisiae data experiment, <br />  balanced data sets. experimental results illus-                                  best results F-measure MCC 0.549 <br />  trate, features shown effective                                  0.495, respectively, associated  essentiality prediction. Feature &#194;&#173;subsets selected                               N13 predictor. contrast, performance <br />                                                                                          measures, achieved 0.77 0.545 balanced <br />                                                                                          data experiment, associated N17 <br />                                                                 CMIM09 <br />                                                                                          predictor. experimental results showed                                                                                          performance models better Hwang&#226;&#8364;&#8482;s <br />                       0.8 <br />  <br />  <br />  <br />  <br />                                                                 mRMR13 <br />                                                                 Gustafson <br />                                                                 N4                       selected 9 features. achieving <br />                                                                 N8 <br />                                                                                          higher accuracy main issue, recommend                       0.7 <br />    Ess. probability <br />  <br />  <br />  <br />  <br />                                                                 N11 <br />                                                                 N80                      N16&#194;&#160;model (16 features). prefers compact <br />                                                                                          feature set small size, suggest using N9&#194;&#160;model <br />                                                                                          (9 features). list important features. fea- <br />                       0.6 <br />  <br />  <br />  <br />  <br />                                                                                          tures crucial identifying essential proteins. <br />                                                                                              E. coli data set, best values F-measure <br />                       0.5 <br />  <br />  <br />  <br />  <br />                                                                                          MCC 0.421 0.407&#194;&#160; imbalanced <br />                                                                                          experiments. balanced experiment, best <br />                                                                                          values F-measure MCC 0.718 0.448, <br />                       0.4 <br />  <br />  <br />  <br />  <br />                             0    20      40       60       80     100                    respectively. best results associated <br />                                               (%)                                    N5 predictor. data set, Figure 9. average percentage curves imbalanced E. coli <br />                                                                                          predictors associated feature size 11 <br /> data set.                                                                                comparable Gustafsons&#226;&#8364;&#8482;. <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                               407 <br />  Hor et&#194;&#160;al <br />  <br />  <br />    possible methods                p&#194;&#173; rocess, signal transduction, transcription, trans- <br /> improving prediction capability. Features related         port process). protein sequence properties use-           feature sets obtained ful identifying essentiality. Furthermore,       Acencio Lemke.5 <br /> proteins similar primary structures pos- <br />                                                            e.	 Betweenness centrality related integrated func- <br /> sess similar functions, essentiality                                                                tional, PI GC network: values defined <br /> addressed sequence motif perspective.34 <br />                                                                identically mentioned addition approaches, performance <br />                                                                paths represented terms integrated improved incorporating tools                                                                functional, PI GC network interactions. <br /> constructing hybrid predictors.                                                             f.	 Degree related integrated functional, PI GC <br /> majority vote35 strategy combining classi- <br />                                                                network: values defined identically fiers. method represents simplest method <br />                                                                mentioned  paths categorical data fusion. &#194;&#173;According litera- <br />                                                                represented terms integrated functional, PI, <br /> ture,36 prerequisite improvement arises                                                                GC network interactions. fact individual classifier contain <br /> distinct information discrimination.             feature sets, collected negative effects imposed            network information Hu et&#194;&#160; al22  <br /> structed ensemble.                                         ducted calculations using iGraph software.37 <br />                                                            g.	 Maximum neighborhood component density <br /> Appendix                                                       maximum neighborhood component: maxi- <br /> Feature extraction                                             mum neighborhood component (MNC) density 	 Outdegree indegree related transcriptional          maximum neigh-borhood component (DMNC) <br />     regulation interaction: feature represents             properties proposed Lin et&#194;&#160;al26 Chin.4 <br />     number outgoing  incoming) links              protein  let N  set neighbors  <br />     gene g corresponding protein. Links           MNC connected component N  <br />     represented terms transcriptional regulation         maximum size, denoted M N . <br />     interactions.                                                  protein  number proteins b.	 Betweenness centrality transcriptional regulation          number edges M N   denoted ni     interactions: Let &#207;&#402;gi gj denote number short-       di, respectively. DMNC di /ni&#206;&#177; 1 # <br />     est paths gi gj. value &#207;&#402;gi gj (g)       &#206;&#177; # 2.  set &#206;&#177; 1.7. <br />     defined number shortest paths gi     h.	 Sequence features: used feature sets     gj passing g. Paths represented         Lin et&#194;&#160;al.20 Let L protein length, Fi     terms transcriptional regulation interactions.          occurrence number amino acid protein, <br /> c.	 Betweenness centrality related physical inter-          H   hydrophobic coefficient38 amino acid <br />     actions: value &#207;&#8222;gi gj (g) defined num-        Pi,j position jth occurrence amino <br />     ber shortest paths gi gj passing            acid  j) amino acid position j Sm,n <br />     g. <span id='am-2' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> similar previous       score row m column n PSSM.39 <br />        paths represented           sequence features listed follows. <br />     terms protein physical interactions.                	 1.&#226;&#8364;&#8218; Protein length: L <br /> d.	 Protein properties: Acencio Lemke5 discov-         	 2.&#226;&#8364;&#8218; Cysteine count: FC <br />     ered integration topological properties,   	 3.&#226;&#8364;&#8218; Amino acid occurrence: composition     cellular components, biological processes                  amino acid Fi /L, 1 # # 20. <br />     possess good capability predicting essential                                                F <br />                                                            	 4.&#226;&#8364;&#8218; Average cysteine position: &#226;&#710;&#8216; j C=1 PC , j / FC. <br />     proteins.  features contained cel-      	 5.&#226;&#8364;&#8218; Average distance cysteines: <br />     lular components (cytoplasm, endoplasmic reticu-                   F     F <br />     lum, mitochondrion, nucleus localization)              &#226;&#710;&#8216; xC=1 &#226;&#710;&#8216; yC=1 PC , x &#226;&#710;&#8217; PC , y <br />     biological processes (cell cycle, metabolic                      L &#195;&#8212; FC <br />  <br />  <br />  <br /> 408	                                                                                     Evolutionary Bioinformatics 2013:9 <br />                                                                                                         Essentiality prediction SVM <br />  <br />  <br /> 	&#226;&#8364;&#8218; 6.&#226;&#8364;&#8218; Cysteine odd index: FC mod 2.                         used informational odds ratios (IOR)32 	&#226;&#8364;&#8218; 7.&#226;&#8364;&#8218; Average hydrophobicity: &#226;&#710;&#8216; i20=1 Fi &#195;&#8212; H  ) / L            represent association essentiality <br /> 	&#226;&#8364;&#8218; 8.&#226;&#8364;&#8218; Average hydrophobicity cysteine:                    &#194;&#173;predictions. IOR measures        kth values average hydrophobic-                     likely protein essential learn- <br />        ity cysteines defined                   ing machine outputs essentiality non- <br />         &#226;&#710;&#8216; j C=1 H (  PC , j + k )) / FC . set k&#194;&#160;=&#194;&#160;-2, <br />           F <br />                                                                   essentiality. value 1.0&#194;&#160;indicates association <br />        -1, +1, +2.                                                essentiality predictions produced <br /> 	&#226;&#8364;&#8218; 9.&#226;&#8364;&#8218; Cysteine position distribution: 1 # d # &#207;?,             learning machines. confidence intervals        dth cysteine position distribution                 performance measures informational odds <br />                                                                   ratios corresponding prediction models               &#239;&#163;&#177;&#239;&#163;&#180;          PC , j d &#226;&#710;&#8217; 1        d            &#239;&#163;&#188;&#239;&#163;&#180; <br />                &#239;&#163;&#178;&#206;&#177; j &#206;&#177; j =       ,      &#238;&#8364;&#172; &#206;&#177; j &#238;&#8364;&#163; ,1 &#238;&#8364;&#163; j &#238;&#8364;&#163; FC &#239;&#163;&#189;      shown Tables&#194;&#160;12&#226;&#8364;&#8220;15. <br /> 	              &#239;&#163;&#180;&#239;&#163;&#179;          L      &#207;?           &#207;?             &#239;&#163;&#180;&#239;&#163;&#190; <br />                                        Fc                         Comparison feature <br /> 		       set &#207;?&#194;&#160;=&#194;&#160;5                                             selection methods <br /> 	 10.&#226;&#8364;&#8218; Average PSSM amino acid: average                    introduced feature selection meth- <br />                                    L <br />          PSSM residue &#226;&#710;&#8216; m =1 S m  / L .                 ods served benchmarks, mRMR13 	 Phyletic retention: Gustafson et&#194;&#160; al21 discovered             CMIM,14 theoretical methods. <br />     essential proteins generally             compared feature selection <br />     served nonessential proteins. Phyletic reten-            method feature subsets equal size     tion protein number organisms               selected. <br />     ortholog present. ortholog                Unlike methods select ranking fea- <br />     protein obtained Hwang et&#194;&#160;al.7                       tures based F-score mutual information j.	 Essential index:7 Essential index measures ratio          considering relationships features, mRMR <br />     essential proteins neighbors N  node         accommodates feature relevance respect     Essential index node defined p  ) /di ,          class label dependency selected features. <br />     p  number essential proteins             strategy combines maximal relevance <br />     N  di degree node                           minimal redundancy criteria. order k.	 Clique level:7 clique level protein defined    criteria consideration avoid <br />     maximal clique containing   cliques      exhaustive search, mRMR adopts incremen- <br />     sizes 3 10 taken consideration.         tal search approach.  rth selected feature <br /> l.	 Number paralagous genes: shown genes            satisfy <br />     likely essential dupli- <br />                                                                                         &#239;&#163;&#177;&#239;&#163;&#180;          1                              &#239;&#163;&#188;&#239;&#163;&#180; <br />     cate existed genome.21 feature               X r = arg max &#239;&#163;&#178; ( X j , Y ) -          &#226;&#710;&#8216;        ( X j , X )&#239;&#163;&#189; 	   (1) <br />                                                                         X j &#226;&#710;&#710;X - X r -1 &#239;&#163;&#180;         r - 1 X &#226;&#710;&#710;X r -1                 &#239;&#163;&#190;&#239;&#163;&#180; <br />     defined number genes present                                   &#239;&#163;&#179; <br />     genome. addition, BLASTP <br />     E-values 10&#226;&#710;&#8217;20 ratios            X feature set. Xi feature     larger gene smaller exceed 1.33.            Xr-1 r - 1 number selected features  <br /> m.	Open reading frame length: Gustafson et&#194;&#160; al21                  tained Xr-1. Xj feature selected. <br />     observed ancestral genes likely           (Xj, Y) mutual information quantifies     essential proteins generally larger           dependence  relevance) feature Xj     evolution. Consequently, open                  class label Y. means rth selected feature <br />     reading frame length indicate essentiality.               relevant class label possible <br />                                                                   possesses dependency averagely Confidence intervals performance                               selected features. <br /> measures informational odds ratios                                CMIM, feature Xj good (Y, Xj|Xi) performance measures multiplied                       large selected feature Xi. (Y, Xj|Xi) 100. confidence intervals set 95%.                    &#194;&#173;conditional mutual information quantifies <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                                      409 <br />  Hor et&#194;&#160;al <br />  <br />  <br /> Table 12. Confidence intervals performance measures (&#195;&#8212;100) informational odds ratios models produced imbalanced S. cerevisiae data set. <br />  <br />                  AUC                Precision         Recall            F1                 MCC                   IOR <br /> CMIM32           82.5&#194;&#160;&#194;&#177;&#194;&#160;1.2         74.4&#194;&#160;&#194;&#177;&#194;&#160;3.1        36.9&#194;&#160;&#194;&#177;&#194;&#160;4.5        49.3&#194;&#160;&#194;&#177;&#194;&#160;3.8         45.0&#194;&#160;&#194;&#177;&#194;&#160;3.6            5.2&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> mRMR31           82.1&#194;&#160;&#194;&#177;&#194;&#160;1.6         73.8&#194;&#160;&#194;&#177;&#194;&#160;3.2        37.2&#194;&#160;&#194;&#177;&#194;&#160;4.3        49.5&#194;&#160;&#194;&#177;&#194;&#160;3.6         44.9&#194;&#160;&#194;&#177;&#194;&#160;3.4            5.2&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> Hwang            77.5&#194;&#160;&#194;&#177;&#194;&#160;2.2         74.3&#194;&#160;&#194;&#177;&#194;&#160;3.7        34.3&#194;&#160;&#194;&#177;&#194;&#160;4.3        46.9&#194;&#160;&#194;&#177;&#194;&#160;4.0         43.2&#194;&#160;&#194;&#177;&#194;&#160;3.6            5.1&#194;&#160;&#194;&#177;&#194;&#160;0.4 <br /> Acencio          70.7&#194;&#160;&#194;&#177;&#194;&#160;3.4         67.5&#194;&#160;&#194;&#177;&#194;&#160;6.3        12.1&#194;&#160;&#194;&#177;&#194;&#160;5.5        20.4&#194;&#160;&#194;&#177;&#194;&#160;7.6         22.8&#194;&#160;&#194;&#177;&#194;&#160;6.0            3.7&#194;&#160;&#194;&#177;&#194;&#160;0.4 <br /> N4               74.4&#194;&#160;&#194;&#177;&#194;&#160;2.7         78.2&#194;&#160;&#194;&#177;&#194;&#160;3.7        32.7&#194;&#160;&#194;&#177;&#194;&#160;4.1        46.1&#194;&#160;&#194;&#177;&#194;&#160;4.1         43.9&#194;&#160;&#194;&#177;&#194;&#160;3.5            5.3&#194;&#160;&#194;&#177;&#194;&#160;0.4 <br /> N5               72.7&#194;&#160;&#194;&#177;&#194;&#160;3.6         74.1&#194;&#160;&#194;&#177;&#194;&#160;4.1        38.7&#194;&#160;&#194;&#177;&#194;&#160;4.7        50.9&#194;&#160;&#194;&#177;&#194;&#160;4.1         46.1&#194;&#160;&#194;&#177;&#194;&#160;3.8            5.3&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N6               73.0&#194;&#160;&#194;&#177;&#194;&#160;3.2         75.2&#194;&#160;&#194;&#177;&#194;&#160;4.2        39.5&#194;&#160;&#194;&#177;&#194;&#160;4.4        51.8&#194;&#160;&#194;&#177;&#194;&#160;3.8         47.2&#194;&#160;&#194;&#177;&#194;&#160;3.6            5.5&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N7               76.1&#194;&#160;&#194;&#177;&#194;&#160;2.4         76.7&#194;&#160;&#194;&#177;&#194;&#160;3.7        38.6&#194;&#160;&#194;&#177;&#194;&#160;4.4        51.3&#194;&#160;&#194;&#177;&#194;&#160;3.9         47.3&#194;&#160;&#194;&#177;&#194;&#160;3.6            5.5&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N8               77.2&#194;&#160;&#194;&#177;&#194;&#160;2.4         75.5&#194;&#160;&#194;&#177;&#194;&#160;3.4        37.1&#194;&#160;&#194;&#177;&#194;&#160;4.9        49.8&#194;&#160;&#194;&#177;&#194;&#160;4.3         45.7&#194;&#160;&#194;&#177;&#194;&#160;3.9            5.3&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N9               78.2&#194;&#160;&#194;&#177;&#194;&#160;2.4         74.9&#194;&#160;&#194;&#177;&#194;&#160;3.4        38.2&#194;&#160;&#194;&#177;&#194;&#160;4.5        50.6&#194;&#160;&#194;&#177;&#194;&#160;3.9         46.2&#194;&#160;&#194;&#177;&#194;&#160;3.6            5.4&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N10              78.1&#194;&#160;&#194;&#177;&#194;&#160;2.2         75.1&#194;&#160;&#194;&#177;&#194;&#160;3.5        39.9&#194;&#160;&#194;&#177;&#194;&#160;4.1        52.1&#194;&#160;&#194;&#177;&#194;&#160;3.6         47.4&#194;&#160;&#194;&#177;&#194;&#160;3.5            5.5&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N11              78.6&#194;&#160;&#194;&#177;&#194;&#160;2.1         75.2&#194;&#160;&#194;&#177;&#194;&#160;3.2        40.2&#194;&#160;&#194;&#177;&#194;&#160;4.2        52.4&#194;&#160;&#194;&#177;&#194;&#160;3.6         47.6&#194;&#160;&#194;&#177;&#194;&#160;3.4            5.5&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N12              79.8&#194;&#160;&#194;&#177;&#194;&#160;2.0         75.9&#194;&#160;&#194;&#177;&#194;&#160;3.2        40.9&#194;&#160;&#194;&#177;&#194;&#160;4.2        53.2&#194;&#160;&#194;&#177;&#194;&#160;3.6         48.5&#194;&#160;&#194;&#177;&#194;&#160;3.4            5.7&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N13              78.9&#194;&#160;&#194;&#177;&#194;&#160;1.9         74.8&#194;&#160;&#194;&#177;&#194;&#160;3.2        43.3&#194;&#160;&#194;&#177;&#194;&#160;4.3        54.9&#194;&#160;&#194;&#177;&#194;&#160;3.4         49.5&#194;&#160;&#194;&#177;&#194;&#160;3.4            5.8&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N14              80.2&#194;&#160;&#194;&#177;&#194;&#160;1.8         74.9&#194;&#160;&#194;&#177;&#194;&#160;3.2        39.7&#194;&#160;&#194;&#177;&#194;&#160;4.3        51.9&#194;&#160;&#194;&#177;&#194;&#160;3.5         47.1&#194;&#160;&#194;&#177;&#194;&#160;3.4            5.5&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N15              80.1&#194;&#160;&#194;&#177;&#194;&#160;1.9         76.3&#194;&#160;&#194;&#177;&#194;&#160;3.3        40.6&#194;&#160;&#194;&#177;&#194;&#160;4.2        53.0&#194;&#160;&#194;&#177;&#194;&#160;3.5         48.5&#194;&#160;&#194;&#177;&#194;&#160;3.5            5.7&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N16              81.4&#194;&#160;&#194;&#177;&#194;&#160;1.7         76.2&#194;&#160;&#194;&#177;&#194;&#160;3.2        40.1&#194;&#160;&#194;&#177;&#194;&#160;4.6        52.5&#194;&#160;&#194;&#177;&#194;&#160;3.8         48.0&#194;&#160;&#194;&#177;&#194;&#160;3.6            5.6&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N17              81.4&#194;&#160;&#194;&#177;&#194;&#160;1.7         76.1&#194;&#160;&#194;&#177;&#194;&#160;3.3        40.7&#194;&#160;&#194;&#177;&#194;&#160;4.5        53.0&#194;&#160;&#194;&#177;&#194;&#160;3.8         48.4&#194;&#160;&#194;&#177;&#194;&#160;3.6            5.7&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N18              81.1&#194;&#160;&#194;&#177;&#194;&#160;1.8         75.1&#194;&#160;&#194;&#177;&#194;&#160;3.2        41.1&#194;&#160;&#194;&#177;&#194;&#160;4.3        53.1&#194;&#160;&#194;&#177;&#194;&#160;3.6         48.2&#194;&#160;&#194;&#177;&#194;&#160;3.5            5.6&#194;&#160;&#194;&#177;&#194;&#160;0.5 <br /> N90              82.9&#194;&#160;&#194;&#177;&#194;&#160;1.0         73.8&#194;&#160;&#194;&#177;&#194;&#160;2.8        35.5&#194;&#160;&#194;&#177;&#194;&#160;4.5        47.9&#194;&#160;&#194;&#177;&#194;&#160;3.6         43.8&#194;&#160;&#194;&#177;&#194;&#160;3.4            5.1&#194;&#160;&#194;&#177;&#194;&#160;0.4 <br />  <br />  <br />  <br /> Table 13. Confidence intervals performance measures (&#195;&#8212;100) informational odds ratios models produced balanced S. cerevisiae data set. <br />  <br />                  AUC                Precision         Recall            F1                 MCC                   IOR <br /> CMIM32           84.2&#194;&#160;&#194;&#177;&#194;&#160;1.5         77.2&#194;&#160;&#194;&#177;&#194;&#160;2.2        76.6&#194;&#160;&#194;&#177;&#194;&#160;2.9        76.9&#194;&#160;&#194;&#177;&#194;&#160;2.1         54.0&#194;&#160;&#194;&#177;&#194;&#160;3.9            3.3&#194;&#160;&#194;&#177;&#194;&#160;0.4 <br /> mRMR31           83.6&#194;&#160;&#194;&#177;&#194;&#160;1.6         76.5&#194;&#160;&#194;&#177;&#194;&#160;2.4        74.1&#194;&#160;&#194;&#177;&#194;&#160;3.0        75.2&#194;&#160;&#194;&#177;&#194;&#160;2.1         51.3&#194;&#160;&#194;&#177;&#194;&#160;4.1            3.0&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> Hwang            82.2&#194;&#160;&#194;&#177;&#194;&#160;1.7         77.8&#194;&#160;&#194;&#177;&#194;&#160;2.6        72.0&#194;&#160;&#194;&#177;&#194;&#160;3.7        74.8&#194;&#160;&#194;&#177;&#194;&#160;2.3         51.6&#194;&#160;&#194;&#177;&#194;&#160;3.9            3.0&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> Acencio          76.8&#194;&#160;&#194;&#177;&#194;&#160;2.2         69.6&#194;&#160;&#194;&#177;&#194;&#160;2.4        73.4&#194;&#160;&#194;&#177;&#194;&#160;4.0        71.4&#194;&#160;&#194;&#177;&#194;&#160;2.3         41.4&#194;&#160;&#194;&#177;&#194;&#160;4.3            2.5&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N4               81.1&#194;&#160;&#194;&#177;&#194;&#160;1.8         77.7&#194;&#160;&#194;&#177;&#194;&#160;2.5        71.6&#194;&#160;&#194;&#177;&#194;&#160;3.6        74.5&#194;&#160;&#194;&#177;&#194;&#160;2.3         51.2&#194;&#160;&#194;&#177;&#194;&#160;3.9            3.0&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N5               82.4&#194;&#160;&#194;&#177;&#194;&#160;1.8         77.8&#194;&#160;&#194;&#177;&#194;&#160;2.6        73.5&#194;&#160;&#194;&#177;&#194;&#160;3.5        75.6&#194;&#160;&#194;&#177;&#194;&#160;2.2         52.7&#194;&#160;&#194;&#177;&#194;&#160;4.1            3.1&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N6               82.7&#194;&#160;&#194;&#177;&#194;&#160;1.8         77.8&#194;&#160;&#194;&#177;&#194;&#160;2.6        73.9&#194;&#160;&#194;&#177;&#194;&#160;3.6        75.8&#194;&#160;&#194;&#177;&#194;&#160;2.3         53.0&#194;&#160;&#194;&#177;&#194;&#160;4.1            3.1&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N7               83.1&#194;&#160;&#194;&#177;&#194;&#160;1.8         77.9&#194;&#160;&#194;&#177;&#194;&#160;2.5        73.3&#194;&#160;&#194;&#177;&#194;&#160;3.5        75.5&#194;&#160;&#194;&#177;&#194;&#160;2.2         52.6&#194;&#160;&#194;&#177;&#194;&#160;4.0            3.1&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N8               82.6&#194;&#160;&#194;&#177;&#194;&#160;1.8         78.6&#194;&#160;&#194;&#177;&#194;&#160;2.5        72.1&#194;&#160;&#194;&#177;&#194;&#160;3.5        75.2&#194;&#160;&#194;&#177;&#194;&#160;2.3         52.7&#194;&#160;&#194;&#177;&#194;&#160;4.0            3.1&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N9               83.3&#194;&#160;&#194;&#177;&#194;&#160;1.8         79.1&#194;&#160;&#194;&#177;&#194;&#160;2.4        73.5&#194;&#160;&#194;&#177;&#194;&#160;3.4        76.2&#194;&#160;&#194;&#177;&#194;&#160;2.2         54.1&#194;&#160;&#194;&#177;&#194;&#160;3.8            3.2&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N10              83.4&#194;&#160;&#194;&#177;&#194;&#160;1.7         78.9&#194;&#160;&#194;&#177;&#194;&#160;2.4        73.6&#194;&#160;&#194;&#177;&#194;&#160;3.3        76.1&#194;&#160;&#194;&#177;&#194;&#160;2.1         54.0&#194;&#160;&#194;&#177;&#194;&#160;3.8            3.2&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N11              83.1&#194;&#160;&#194;&#177;&#194;&#160;1.7         78.4&#194;&#160;&#194;&#177;&#194;&#160;2.4        73.7&#194;&#160;&#194;&#177;&#194;&#160;3.5        76.0&#194;&#160;&#194;&#177;&#194;&#160;2.2         53.5&#194;&#160;&#194;&#177;&#194;&#160;3.8            3.2&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N12              82.9&#194;&#160;&#194;&#177;&#194;&#160;1.8         77.9&#194;&#160;&#194;&#177;&#194;&#160;2.5        73.2&#194;&#160;&#194;&#177;&#194;&#160;3.1        75.5&#194;&#160;&#194;&#177;&#194;&#160;2.2         52.6&#194;&#160;&#194;&#177;&#194;&#160;4.2            3.1&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N13              83.4&#194;&#160;&#194;&#177;&#194;&#160;1.7         78.8&#194;&#160;&#194;&#177;&#194;&#160;2.4        73.0&#194;&#160;&#194;&#177;&#194;&#160;3.5        75.8&#194;&#160;&#194;&#177;&#194;&#160;2.2         53.5&#194;&#160;&#194;&#177;&#194;&#160;3.9            3.1&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N14              83.6&#194;&#160;&#194;&#177;&#194;&#160;1.6         77.7&#194;&#160;&#194;&#177;&#194;&#160;2.3        74.3&#194;&#160;&#194;&#177;&#194;&#160;3.4        75.9&#194;&#160;&#194;&#177;&#194;&#160;2.1         53.0&#194;&#160;&#194;&#177;&#194;&#160;3.8            3.2&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N15              84.3&#194;&#160;&#194;&#177;&#194;&#160;1.7         78.4&#194;&#160;&#194;&#177;&#194;&#160;2.4        74.8&#194;&#160;&#194;&#177;&#194;&#160;3.3        76.6&#194;&#160;&#194;&#177;&#194;&#160;2.1         54.2&#194;&#160;&#194;&#177;&#194;&#160;3.9            3.3&#194;&#160;&#194;&#177;&#194;&#160;0.4 <br /> N16              84.2&#194;&#160;&#194;&#177;&#194;&#160;1.6         77.7&#194;&#160;&#194;&#177;&#194;&#160;2.2        75.6&#194;&#160;&#194;&#177;&#194;&#160;3.1        76.7&#194;&#160;&#194;&#177;&#194;&#160;2.0         54.0&#194;&#160;&#194;&#177;&#194;&#160;3.7            3.3&#194;&#160;&#194;&#177;&#194;&#160;0.4 <br /> N17              84.7&#194;&#160;&#194;&#177;&#194;&#160;1.6         77.8&#194;&#160;&#194;&#177;&#194;&#160;2.3        76.3&#194;&#160;&#194;&#177;&#194;&#160;3.0        77.0&#194;&#160;&#194;&#177;&#194;&#160;2.0         54.5&#194;&#160;&#194;&#177;&#194;&#160;3.8            3.3&#194;&#160;&#194;&#177;&#194;&#160;0.4 <br /> N18              84.0&#194;&#160;&#194;&#177;&#194;&#160;1.6         77.9&#194;&#160;&#194;&#177;&#194;&#160;2.4        74.0&#194;&#160;&#194;&#177;&#194;&#160;3.3        75.9&#194;&#160;&#194;&#177;&#194;&#160;2.0         53.1&#194;&#160;&#194;&#177;&#194;&#160;3.8            3.1&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N90              83.9&#194;&#160;&#194;&#177;&#194;&#160;1.4         76.0&#194;&#160;&#194;&#177;&#194;&#160;2.0        75.3&#194;&#160;&#194;&#177;&#194;&#160;2.7        75.7&#194;&#160;&#194;&#177;&#194;&#160;1.8         51.6&#194;&#160;&#194;&#177;&#194;&#160;3.5            3.1&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br />  <br />  <br /> 410	                                                                                      Evolutionary Bioinformatics 2013:9 <br />                                                                                         Essentiality prediction SVM <br />  <br />  <br /> Table 14. Confidence intervals performance measures (&#195;&#8212;100) informational odds ratios models produced imbalanced E. coli data set. <br />                       AUC             Precision        Recall             F1               MCC                IOR <br /> CMIM09                70.1&#194;&#160;&#194;&#177;&#194;&#160;0.9      72.0&#194;&#160;&#194;&#177;&#194;&#160;1.4       27.1&#194;&#160;&#194;&#177;&#194;&#160;0.7         39.4&#194;&#160;&#194;&#177;&#194;&#160;0.9       38.2&#194;&#160;&#194;&#177;&#194;&#160;0.9         5.2&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> mRMR13                71.5&#194;&#160;&#194;&#177;&#194;&#160;2.4      71.3&#194;&#160;&#194;&#177;&#194;&#160;5.4       25.0&#194;&#160;&#194;&#177;&#194;&#160;5.8         37.0&#194;&#160;&#194;&#177;&#194;&#160;6.8       36.0&#194;&#160;&#194;&#177;&#194;&#160;5.8         4.7&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> Gustafson             71.1&#194;&#160;&#194;&#177;&#194;&#160;2.3      66.5&#194;&#160;&#194;&#177;&#194;&#160;4.6       25.5&#194;&#160;&#194;&#177;&#194;&#160;5.0         36.8&#194;&#160;&#194;&#177;&#194;&#160;5.2       34.7&#194;&#160;&#194;&#177;&#194;&#160;4.8         4.9&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N4                    69.1&#194;&#160;&#194;&#177;&#194;&#160;1.8      72.5&#194;&#160;&#194;&#177;&#194;&#160;1.0       28.0&#194;&#160;&#194;&#177;&#194;&#160;0.6         40.4&#194;&#160;&#194;&#177;&#194;&#160;0.7       39.1&#194;&#160;&#194;&#177;&#194;&#160;0.7         5.5&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N5                    69.0&#194;&#160;&#194;&#177;&#194;&#160;2.0      73.7&#194;&#160;&#194;&#177;&#194;&#160;1.4       29.5&#194;&#160;&#194;&#177;&#194;&#160;0.8         42.1&#194;&#160;&#194;&#177;&#194;&#160;0.9       40.7&#194;&#160;&#194;&#177;&#194;&#160;1.0         5.7&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N6                    70.1&#194;&#160;&#194;&#177;&#194;&#160;1.7      74.2&#194;&#160;&#194;&#177;&#194;&#160;1.4       28.7&#194;&#160;&#194;&#177;&#194;&#160;0.9         41.4&#194;&#160;&#194;&#177;&#194;&#160;1.1       40.3&#194;&#160;&#194;&#177;&#194;&#160;1.0         5.7&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N7                    71.4&#194;&#160;&#194;&#177;&#194;&#160;1.4      73.5&#194;&#160;&#194;&#177;&#194;&#160;1.3       27.5&#194;&#160;&#194;&#177;&#194;&#160;0.7         40.0&#194;&#160;&#194;&#177;&#194;&#160;0.9       39.2&#194;&#160;&#194;&#177;&#194;&#160;0.9         5.5&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N8                    70.5&#194;&#160;&#194;&#177;&#194;&#160;1.3      74.2&#194;&#160;&#194;&#177;&#194;&#160;1.1       28.8&#194;&#160;&#194;&#177;&#194;&#160;0.8         41.5&#194;&#160;&#194;&#177;&#194;&#160;0.9       40.5&#194;&#160;&#194;&#177;&#194;&#160;0.9         5.7&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N9                    70.7&#194;&#160;&#194;&#177;&#194;&#160;1.5      72.6&#194;&#160;&#194;&#177;&#194;&#160;1.4       29.3&#194;&#160;&#194;&#177;&#194;&#160;1.0         41.7&#194;&#160;&#194;&#177;&#194;&#160;1.2       40.1&#194;&#160;&#194;&#177;&#194;&#160;1.2         5.6&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N10                   71.1&#194;&#160;&#194;&#177;&#194;&#160;1.5      72.4&#194;&#160;&#194;&#177;&#194;&#160;1.6       29.4&#194;&#160;&#194;&#177;&#194;&#160;1.0         41.8&#194;&#160;&#194;&#177;&#194;&#160;1.1       40.1&#194;&#160;&#194;&#177;&#194;&#160;1.2         5.6&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N11                   71.4&#194;&#160;&#194;&#177;&#194;&#160;1.4      73.2&#194;&#160;&#194;&#177;&#194;&#160;1.3       27.8&#194;&#160;&#194;&#177;&#194;&#160;0.8         40.3&#194;&#160;&#194;&#177;&#194;&#160;1.0       39.3&#194;&#160;&#194;&#177;&#194;&#160;1.0         5.5&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N12                   71.2&#194;&#160;&#194;&#177;&#194;&#160;1.4      72.5&#194;&#160;&#194;&#177;&#194;&#160;1.9       29.2&#194;&#160;&#194;&#177;&#194;&#160;1.1         41.6&#194;&#160;&#194;&#177;&#194;&#160;1.2       40.0&#194;&#160;&#194;&#177;&#194;&#160;1.3         5.6&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N13                   71.4&#194;&#160;&#194;&#177;&#194;&#160;1.3      73.3&#194;&#160;&#194;&#177;&#194;&#160;1.8       28.7&#194;&#160;&#194;&#177;&#194;&#160;1.2         41.3&#194;&#160;&#194;&#177;&#194;&#160;1.4       40.0&#194;&#160;&#194;&#177;&#194;&#160;1.4         5.6&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br /> N80                   71.6&#194;&#160;&#194;&#177;&#194;&#160;0.9      67.7&#194;&#160;&#194;&#177;&#194;&#160;2.1       23.7&#194;&#160;&#194;&#177;&#194;&#160;1.3         35.2&#194;&#160;&#194;&#177;&#194;&#160;1.6       33.9&#194;&#160;&#194;&#177;&#194;&#160;1.6         4.9&#194;&#160;&#194;&#177;&#194;&#160;0.6 <br />  <br />  <br /> Table 15. Confidence intervals performance measures (&#195;&#8212;100) informational odds ratios models produced bal- <br /> anced E. coli data set. <br />                       AUC             Precision       Recall              F1               MCC                IOR <br /> CMIM09                76.7&#194;&#160;&#194;&#177;&#194;&#160;1.7      72.0&#194;&#160;&#194;&#177;&#194;&#160;2.4      70.0&#194;&#160;&#194;&#177;&#194;&#160;3.7          71.0&#194;&#160;&#194;&#177;&#194;&#160;2.2       42.1&#194;&#160;&#194;&#177;&#194;&#160;4.0         2.4&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> mRMR13                76.2&#194;&#160;&#194;&#177;&#194;&#160;2.0      72.8&#194;&#160;&#194;&#177;&#194;&#160;3.1      65.4&#194;&#160;&#194;&#177;&#194;&#160;6.4          68.9&#194;&#160;&#194;&#177;&#194;&#160;3.3       39.6&#194;&#160;&#194;&#177;&#194;&#160;3.9         2.2&#194;&#160;&#194;&#177;&#194;&#160;0.2 <br /> Gustafson             77.7&#194;&#160;&#194;&#177;&#194;&#160;2.6      72.2&#194;&#160;&#194;&#177;&#194;&#160;3.3      71.5&#194;&#160;&#194;&#177;&#194;&#160;4.0          71.9&#194;&#160;&#194;&#177;&#194;&#160;2.8       44.0&#194;&#160;&#194;&#177;&#194;&#160;5.5         2.6&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N4                    78.0&#194;&#160;&#194;&#177;&#194;&#160;1.6      73.3&#194;&#160;&#194;&#177;&#194;&#160;2.5      70.1&#194;&#160;&#194;&#177;&#194;&#160;2.7          71.7&#194;&#160;&#194;&#177;&#194;&#160;1.9       44.6&#194;&#160;&#194;&#177;&#194;&#160;3.8         2.6&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N5                    77.9&#194;&#160;&#194;&#177;&#194;&#160;1.7      73.0&#194;&#160;&#194;&#177;&#194;&#160;2.5      70.6&#194;&#160;&#194;&#177;&#194;&#160;2.8          71.8&#194;&#160;&#194;&#177;&#194;&#160;1.8       44.5&#194;&#160;&#194;&#177;&#194;&#160;3.8         2.6&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N6                    76.2&#194;&#160;&#194;&#177;&#194;&#160;1.7      73.5&#194;&#160;&#194;&#177;&#194;&#160;2.6      66.3&#194;&#160;&#194;&#177;&#194;&#160;4.3          69.6&#194;&#160;&#194;&#177;&#194;&#160;2.6       42.5&#194;&#160;&#194;&#177;&#194;&#160;4.1         2.4&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N7                    78.3&#194;&#160;&#194;&#177;&#194;&#160;1.7      73.7&#194;&#160;&#194;&#177;&#194;&#160;2.5      69.6&#194;&#160;&#194;&#177;&#194;&#160;2.7          71.6&#194;&#160;&#194;&#177;&#194;&#160;1.8       44.8&#194;&#160;&#194;&#177;&#194;&#160;3.6         2.6&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N8                    78.1&#194;&#160;&#194;&#177;&#194;&#160;1.7      72.3&#194;&#160;&#194;&#177;&#194;&#160;2.3      71.1&#194;&#160;&#194;&#177;&#194;&#160;3.3          71.7&#194;&#160;&#194;&#177;&#194;&#160;2.1       43.9&#194;&#160;&#194;&#177;&#194;&#160;3.8         2.5&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N9                    78.2&#194;&#160;&#194;&#177;&#194;&#160;1.6      71.5&#194;&#160;&#194;&#177;&#194;&#160;2.2      70.3&#194;&#160;&#194;&#177;&#194;&#160;4.1          70.9&#194;&#160;&#194;&#177;&#194;&#160;2.3       42.3&#194;&#160;&#194;&#177;&#194;&#160;3.8         2.4&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N10                   78.1&#194;&#160;&#194;&#177;&#194;&#160;1.6      72.5&#194;&#160;&#194;&#177;&#194;&#160;2.4      70.2&#194;&#160;&#194;&#177;&#194;&#160;3.3          71.3&#194;&#160;&#194;&#177;&#194;&#160;2.1       43.6&#194;&#160;&#194;&#177;&#194;&#160;3.9         2.5&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N11                   77.7&#194;&#160;&#194;&#177;&#194;&#160;1.7      71.9&#194;&#160;&#194;&#177;&#194;&#160;2.2      70.0&#194;&#160;&#194;&#177;&#194;&#160;3.3          70.9&#194;&#160;&#194;&#177;&#194;&#160;2.0       42.6&#194;&#160;&#194;&#177;&#194;&#160;3.6         2.5&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N12                   77.6&#194;&#160;&#194;&#177;&#194;&#160;1.9      71.5&#194;&#160;&#194;&#177;&#194;&#160;2.3      69.5&#194;&#160;&#194;&#177;&#194;&#160;4.5          70.5&#194;&#160;&#194;&#177;&#194;&#160;2.5       41.8&#194;&#160;&#194;&#177;&#194;&#160;4.0         2.4&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N13                   77.6&#194;&#160;&#194;&#177;&#194;&#160;1.7      73.1&#194;&#160;&#194;&#177;&#194;&#160;2.4      69.5&#194;&#160;&#194;&#177;&#194;&#160;3.0          71.2&#194;&#160;&#194;&#177;&#194;&#160;2.1       43.9&#194;&#160;&#194;&#177;&#194;&#160;3.9         2.5&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br /> N80                   76.9&#194;&#160;&#194;&#177;&#194;&#160;1.8      71.1&#194;&#160;&#194;&#177;&#194;&#160;2.4      71.5&#194;&#160;&#194;&#177;&#194;&#160;2.4          71.3&#194;&#160;&#194;&#177;&#194;&#160;1.8       42.4&#194;&#160;&#194;&#177;&#194;&#160;3.8         2.5&#194;&#160;&#194;&#177;&#194;&#160;0.3 <br />  <br />  <br /> Table 16. Performance comparison method vs. mRMR imbalanced S. cerevisiae data set sizes feature subsets, . symbol represents values significantly higher. <br />                 AUC                   Precision           Recall                  F-measure             MCC <br /> N4              0.744 0.762           0.782   0.756       0.327 0.331             0.461 0.461           0.439 0.430 <br /> N5              0.727 0.718           0.741   0.740       0.387 0.359             0.509 0.484           0.461 0.442 <br /> N6              0.730 0.753           0.752   0.753       0.395 . 0.333           0.518 . 0.462         0.472 . 0.430 <br /> N7              0.761 0.763           0.767   0.761       0.386 . 0.330           0.513 . 0.460         0.473 . 0.431 <br /> N8              0.772 . 0.771         0.755   0.757       0.371 . 0.326           0.498 . 0.456         0.457 . 0.427 <br /> N9              0.782 0.776           0.749   0.749       0.382 . 0.341           0.506 . 0.469         0.462 . 0.434 <br /> N10             0.781 . 0.778         0.751   0.752       0.399 . 0.340           0.521 . 0.469         0.474 . 0.434 <br /> N11             0.786 . 0.774         0.752   0.750       0.402 . 0.341           0.524 . 0.469         0.476 . 0.434 <br /> N12             0.798 . 0.781         0.759   0.757       0.409 . 0.334           0.532 . 0.463         0.485 . 0.432 <br /> N13             0.789 . 0.774         0.748   0.746       0.433 . 0.342           0.549 . 0.469         0.495 . 0.432 <br /> N14             0.802 . 0.775         0.749   0.750       0.397 . 0.340           0.519 . 0.468         0.471 . 0.433 <br /> N15             0.801 . 0.798         0.763   0.764       0.406 . 0.318           0.530 . 0.449         0.485 . 0.424 <br /> N16             0.814 . 0.799         0.762   0.762       0.401 . 0.318           0.525 . 0.449         0.480 . 0.423 <br /> N17             0.814 . 0.799         0.761   0.759       0.407 . 0.326           0.530 . 0.456         0.484 . 0.427 <br /> N18             0.811 . 0.797         0.751   0.749       0.411 . 0.342           0.531 . 0.469         0.482 . 0.434 <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                                 411 <br />  Hor et&#194;&#160;al <br />  <br />  <br /> Table 17. Performance comparison new method vs. CMIM imbalanced S. cerevisiae data set identical <br /> number features selected. <br />  <br />              AUC                       Precision               Recall               F1                       MCC <br /> N4           0.744 0.761               0.782     0.762         0.327 0.344          0.461 0.474              0.439 0.442 <br /> N5           0.727 0.735               0.741     0.738         0.387 0.371          0.509 0.494              0.461 0.449 <br /> N6           0.730 0.757               0.752     0.749         0.395 0.359          0.518 0.485              0.472 0.446 <br /> N7           0.761 0.779               0.767     0.763         0.386 . 0.339        0.513 . 0.470            0.473 . 0.439 <br /> N8           0.772 0.779               0.755     0.754         0.371 . 0.350        0.498 . 0.478            0.457 . 0.442 <br /> N9           0.782 0.776               0.749     0.750         0.382 . 0.357        0.506 . 0.483            0.462 . 0.445 <br /> N10          0.781 0.782               0.751     0.751         0.399 . 0.353        0.521 . 0.480            0.474 . 0.443 <br /> N11          0.786 0.786               0.752     0.752         0.402 . 0.363        0.524 . 0.490            0.476 . 0.450 <br /> N12          0.798 0.799               0.759     0.758         0.409 . 0.354        0.532 . 0.483            0.485 . 0.447 <br /> N13          0.789 0.797               0.748     0.750         0.433 . 0.360        0.549 . 0.487            0.495 . 0.447 <br /> N14          0.802 . 0.801             0.749     0.749         0.397 . 0.348        0.519 . 0.475            0.471 . 0.438 <br /> N15          0.801 . 0.797             0.763     0.760         0.406 . 0.330        0.530 . 0.460            0.485 . 0.430 <br /> N16          0.814 . 0.796             0.762     0.759         0.401 . 0.338        0.525 . 0.468            0.480 . 0.436 <br /> N17          0.814 . 0.795             0.761     0.756         0.407 . 0.339        0.530 . 0.469            0.484 . 0.435 <br /> N18          0.811 0.799               0.751     0.756         0.411 . 0.338        0.531 . 0.467            0.482 . 0.435 <br />  <br />  <br /> d&#194;&#173; iscrepancy features Xj Xi given               following paragraph, compare  class label Y. Consequently, feature selection pro-          method feature selection methods <br />  cedure carried incremental manner             feature subsets equal size selected. <br />  follows:                                                      ran SVM software Hwang&#226;&#8364;&#8482;s                                                                   Gustafson feature sets tune SVM param- <br />                 &#207;&#8230; (1) = arg max ( X j , Y )	           (2)      eters achieve highest average performances. <br /> 	                               Xj <br />                                                                   fairly compare methods given feature subsets <br />  <br /> 	                      Xj <br />                             { <br />         &#207;&#8230; (r) = arg max min (Y , X j | X &#207;&#8230; (k ) ) .	 <br />                             k &#226;&#8240;&#164; r -1 <br />                                                    }     (3) <br />                                                                   sizes obtained methods, mRMR <br />                                                                   CMIM, used SVM software                                                                   adjust cost parameters order achieve simi- v(k) denotes kth selected feature.                      lar levels precision. S. cerevisiae E. coli <br />                                                                   data set, feature numbers k 4 # k # 18  <br /> Table 18. Performance comparison method vs. mRMR balanced S. cerevisiae data set sizes feature subsets, . symbol indicates values significantly higher. <br />  <br />              AUC                       Precision               Recall               F-measure                MCC <br /> N4           0.811 0.815               0.777     0.770         0.716 0.725          0.745 0.747              0.512 0.510 <br /> N5           0.824 0.818               0.778     0.771         0.735 0.722          0.756 0.745              0.527 0.508 <br /> N6           0.827 0.814               0.778     0.775         0.739 0.709          0.758 0.740              0.530 0.504 <br /> N7           0.831 0.824               0.779     0.779         0.733 0.718          0.755 0.747              0.526 0.516 <br /> N8           0.826 0.827               0.786     0.781         0.721 0.721          0.752 0.750              0.527 0.521 <br /> N9           0.833 0.834               0.791     0.783         0.735 0.734          0.762 0.758              0.541 0.531 <br /> N10          0.834 0.835               0.789     0.783         0.736 0.733          0.761 0.757              0.540 0.531 <br /> N11          0.831 0.834               0.784     0.780         0.737 0.730          0.760 0.754              0.535 0.525 <br /> N12          0.829 0.834               0.779     0.778         0.732 0.734          0.755 0.755              0.526 . 0.525 <br /> N13          0.834 0.834               0.788     0.779         0.730 0.732          0.758 0.754              0.535 0.525 <br /> N14          0.836 0.832               0.777     0.777         0.743 0.731          0.759 0.753              0.530 0.522 <br /> N15          0.843 0.835               0.784     0.778         0.748 0.734          0.766 0.756              0.542 0.526 <br /> N16          0.842 . 0.836             0.777     0.777         0.756 . 0.735        0.767 . 0.755            0.540 . 0.525 <br /> N17          0.847 . 0.834             0.778     0.777         0.763 0.733          0.770 . 0.754            0.545 . 0.523 <br /> N18          0.840 0.835               0.779     0.778         0.740 0.735          0.759 0.756              0.531 0.526 <br />  <br />  <br />  <br />  <br /> 412	                                                                                         Evolutionary Bioinformatics 2013:9 <br />                                                                                        Essentiality prediction SVM <br />  <br />  <br /> Table 19. Performance comparison method vs. CMIM balanced S. cerevisiae data set identical number features selected. <br />  <br />                 AUC                   Precision          Recall                 F1                     MCC <br /> N4              0.811 0.813           0.777   0.777      0.716 0.724            0.745 0.749            0.512 0.517 <br /> N5              0.824 0.817           0.778   0.775      0.735 0.740            0.756 0.757            0.527 0.526 <br /> N6              0.827 0.821           0.778   0.777      0.739 0.742            0.758 0.759            0.530 0.529 <br /> N7              0.831 0.830           0.779   0.772      0.733 0.744            0.755 0.758            0.526 0.524 <br /> N8              0.826 0.833           0.786   0.776      0.721 0.738            0.752 0.756            0.527 0.525 <br /> N9              0.833 0.834           0.791   0.775      0.735 0.740            0.762 0.757            0.541 0.526 <br /> N10             0.834 0.835           0.789   0.776      0.736 0.739            0.761 0.757            0.540 0.527 <br /> N11             0.831 0.836           0.784   0.778      0.737 0.739            0.760 0.758            0.535 0.528 <br /> N12             0.829 0.838           0.779   0.779      0.732 0.742            0.755 0.760            0.526 0.532 <br /> N13             0.834 0.837           0.788   0.778      0.730 0.741            0.758 0.759            0.535 0.530 <br /> N14             0.836 0.836           0.777   0.777      0.743 0.743            0.759 0.759            0.530 0.530 <br /> N15             0.843 0.836           0.784   0.777      0.748 0.739            0.766 0.758            0.542 0.528 <br /> N16             0.842 . 0.837         0.777   0.776      0.756 0.741            0.767 . 0.758          0.540 . 0.528 <br /> N17             0.847 . 0.838         0.778   0.777      0.763 . 0.744          0.770 . 0.760          0.545 . 0.531 <br /> N18             0.840 0.837           0.779   0.778      0.740 0.746            0.759 0.762            0.531 0.533 <br />  <br />  <br />  <br /> Table 20. Performance comparison method vs. mRMR imbalanced E. coli data set identical numbers features selected. <br />  <br />                 AUC                   Precision          Recall                 F1                     MCC <br /> N4              0.691 0.651           0.725   0.678      0.280 0.269            0.404 0.385            0.391    0.363 <br /> N5              0.690 0.675           0.737   0.687      0.295 0.254            0.421 0.371            0.407    0.356 <br /> N6              0.701 0.681           0.742   0.708      0.287 0.220            0.414 0.336            0.403    0.338 <br /> N7              0.714 0.686           0.735   0.712      0.275 0.212            0.400 0.326            0.392    0.333 <br /> N8              0.705 0.692           0.742   0.713      0.288 0.209            0.415 0.323            0.405    0.330 <br /> N9              0.707 0.692           0.726   0.713      0.293 . 0.199          0.417 . 0.312          0.401    0.322 <br /> N10             0.711 0.697           0.724   0.703      0.294 . 0.193          0.418 . 0.302          0.401    0.313 <br /> N11             0.714&#194;&#160;,&#194;&#160;0.702         0.732   0.697      0.278 . 0.187          0.403 0.295            0.393    0.306 <br /> N12             0.712&#194;&#160;,&#194;&#160;0.704         0.725   0.683      0.292 0.192            0.416 0.300            0.400    0.305 <br /> N13             0.714&#194;&#160;,&#194;&#160;0.715         0.733   0.713      0.287 0.250            0.413 0.370            0.400    0.360 <br />  <br />  <br />  <br /> Table 21. Performance comparison method vs. CMIM imbalanced E. coli data set identical numbers features selected. <br />  <br />                 AUC                   Precision          Recall                 F1                     MCC <br /> N4              0.691 . 0.663         0.725   0.717      0.280 0.271            0.404 0.393            0.391 0.381 <br /> N5              0.690 0.686           0.737   0.710      0.295 . 0.264          0.421 . 0.385          0.407 . 0.373 <br /> N6              0.701 0.697           0.742   0.715      0.287 . 0.265          0.414 . 0.387          0.403 . 0.376 <br /> N7              0.714 0.693           0.735   0.711      0.275 . 0.261          0.400 . 0.382          0.392 . 0.371 <br /> N8              0.705 0.690           0.742   0.709      0.288 . 0.254          0.415 . 0.373          0.405 . 0.364 <br /> N9              0.707 0.701           0.726   0.720      0.293 . 0.271          0.417 . 0.394          0.401 . 0.382 <br /> N10             0.711 0.702           0.724   0.692      0.294 . 0.248          0.418 . 0.364          0.401 . 0.353 <br /> N11             0.714 0.698           0.732   0.690      0.278 . 0.247          0.403 . 0.363          0.393 . 0.351 <br /> N12             0.712 0.690           0.725   0.683      0.292 . 0.239          0.416 . 0.353          0.400 . 0.342 <br /> N13             0.714 0.688           0.733   0.678      0.287 . 0.236          0.413 . 0.349          0.400 . 0.337 <br />  <br />  <br />  <br />  <br /> Evolutionary Bioinformatics 2013:9	                                                                              413 <br />  Hor et&#194;&#160;al <br />  <br />  <br /> Table 22. Performance comparison method vs. mRMR balanced E. coli data set identical numbers features selected. <br />  <br />              AUC                     Precision               Recall              F1                       MCC <br /> N4           0.780 . 0.773           0.733   0.726           0.701 . 0.651       0.717 . 0.686            0.446 . 0.407 <br /> N5           0.779 0.772             0.730   0.720           0.706 . 0.654       0.718 . 0.684            0.445 . 0.401 <br /> N6           0.762 0.771             0.735   0.717           0.663 0.649         0.696 0.680              0.425 0.394 <br /> N7           0.783 . 0.768           0.737   0.716           0.696 0.649         0.716 . 0.680            0.448 . 0.394 <br /> N8           0.781 . 0.764           0.723   0.715           0.711 . 0.641       0.717 . 0.675            0.439 . 0.387 <br /> N9           0.782 . 0.764           0.715   0.713           0.703 0.643         0.709 . 0.675            0.423 . 0.386 <br /> N10          0.781 . 0.765           0.725   0.716           0.702 . 0.636       0.713 . 0.673            0.436 . 0.386 <br /> N11          0.777 . 0.766           0.719   0.720           0.700 0.643         0.709 . 0.678            0.426 0.394 <br /> N12          0.776 0.765             0.715   0.714           0.695 0.643         0.705 0.676              0.418 0.388 <br /> N13          0.776 . 0.762           0.731   0.728           0.695 . 0.654       0.712 . 0.689            0.439 . 0.396 <br />  <br />  <br />  <br /> 4&#194;&#160;# k #13, respectively, compliant             23, method performed significantly better Section V.                                                   mRMR numbers selected features <br />    S. cerevisiae data set imbalanced exper-      exceeded 7. experimental results showed iment, Tables&#194;&#160;16 17 performance com-               difference method CMIM parison method versus mRMR CMIM.                 N6&#194;&#160;AUC. <br /> method performed significantly better size             methods mRMR CMIM, feature subset exceeded 7. balanced experi-           relevance information redundancy taken <br /> ment, illustrated Tables&#194;&#160;18 19, method        consideration.  obtained feature <br /> significantly better number selected           subsets quite compact effective. <br /> features exceeded 16.                                            relevance appropriate <br />    E. coli data set imbalanced experi-           performance measures, classifica- <br /> ment, Tables&#194;&#160;20 21 performance com-                tion accuracy precision. method took parison method versus mRMR CMIM                      performance feature size consideration. configurations similar S. cerevisiae                Consequently, resultant feature subsets data set. method performed significantly better             effective performance measures size feature subset exceeded 9.               given equal number features precision balanced experiment, illustrated Table&#194;&#160; 22            values. <br />  <br />  <br /> Table 23. Performance comparison method vs. CMIM balanced E. coli data set identical numbers features selected. <br />  <br />              AUC                     Precision               Recall              F1                       MCC <br /> N4           0.780 0.769             0.733   0.719           0.701    0.696      0.717   0.707            0.446      0.424 <br /> N5           0.779 0.771             0.730   0.715           0.706    0.696      0.718   0.705            0.445      0.419 <br /> N6           0.762 , 0.771           0.735   0.716           0.663    0.684      0.696   0.699            0.425      0.413 <br /> N7           0.783 0.769             0.737   0.711           0.696    0.696      0.716   0.703            0.448      0.413 <br /> N8           0.781 0.767             0.723   0.709           0.711    0.697      0.717   0.703            0.439      0.412 <br /> N9           0.782 0.767             0.715   0.720           0.703    0.700      0.709   0.710            0.423      0.421 <br /> N10          0.781 0.767             0.725   0.705           0.702    0.702      0.713   0.704            0.436      0.409 <br /> N11          0.777 0.765             0.719   0.706           0.700    0.700      0.709   0.703            0.426      0.408 <br /> N12          0.776 0.764             0.715   0.703           0.695    0.698      0.705   0.700            0.418      0.404 <br /> N13          0.776 0.765             0.731   0.704           0.695    0.700      0.712   0.702            0.439      0.406 <br />  <br />  <br />  <br />  <br /> 414	                                                                                      Evolutionary Bioinformatics 2013:9 <br />                                                                                                                                  Essentiality prediction SVM <br />  <br />  <br /> Author Contributions                                                                	 6.	 Witten IH, Frank E, Hall MA. Data Mining:Practical Machine Learn- <br />                                                                                                             ing Tools Techniques Java Implementations. Burlington, MA: <br /> Conceived designed experiments: CBY, CYH.                                                           &#194;&#173;Morgann Kaufmann; 2000. <br /> Analysed data: CYH, ZJY. Wrote draft                               	 7.	 Hwang YC, Lin CC, Chang JY, Mori H, Juan HF, Huang HC. &#194;&#173;Predicting <br />                                                                                                              essential genes based network sequence analysis. Mol Biosyst. 2009; manuscript: CYH, ZJY, CTT. Contributed                                                            5(12):1672&#226;&#8364;&#8220;8. <br /> writing manuscript: CYH, CBY, ZJY. Agree                                     	 8.	 Przulj N, Wigle DA, Jurisica  Functional topology network protein <br />                                                                                                              interactions. Bioinformatics. 1998;20(3):340&#226;&#8364;&#8220;8. manuscript results conclusions: CYH, CBY.                                  	 9.	 Chin CS, Samanta MP. Global snapshot protein interaction network <br /> Jointly developed structure arguments                                                            percolation based approach. Bioinformatics. 2003;19(18):2413&#226;&#8364;&#8220;9. paper: CYH, CBY, ZJY, CTT. critical revi-                                  	10.	 Barab&#195;&#161;si AL, Oltvai ZN. Network biology: understanding cell&#226;&#8364;&#8482;s func- <br />                                                                                                              tional organization. Nat Rev Genet. 2004;5(2):101&#226;&#8364;&#8220;13. <br /> sions approved final version: CBY. authors                                   	11.	 Ito T, Chiba T, Ozawa R, Yoshida M, Hattori M, Sakaki Y. comprehen- <br /> reviewed approved final manuscript.                                                                  sive hybrid analysis explore yeast protein interactome. Proc Natl <br />                                                                                                              Acad Sci U S  2001;98(8):4569&#226;&#8364;&#8220;74. <br />                                                                                      	12.	 Lal TN, Chapelle O, Sch&#195;&#182;lkopf B. Feature Extraction: Foundations Funding                                                                                                      Applications. Guyon S, Gunn M, Nikravesh M, Zadeh LA. Berlin: Springer; <br />                                                                                                              2006. research work partially supported                                     	13.	 Peng H, Long F, Ding C. Feature selection based mutual information: <br /> National Science Council Taiwan contract                                                            criteria max-dependency, max-relevance, min-redundancy. IEEE <br /> NSC100-2221-E-110-050.                                                                                       Trans Pattern Anal Mach Intell. 2005;27(8):1226&#226;&#8364;&#8220;38. <br />                                                                                        	14.	 Fleuret F. Fast binary feature selection conditional mutual information. <br />                                                                                                              Journal Machine Learning Research. 2004;5:1531&#226;&#8364;&#8220;55. <br /> Competing Interests                                                                     	15.	 Sotoca JM, Pla SF. Supervised feature selection clustering using condi- <br />                                                                                                              tional mutual information-based distances. Pattern Recognition. 2010;43(6): <br /> Author(s) disclose potential conflicts                                                        2068&#226;&#8364;&#8220;81. <br />                                                                                          	16.	 Quinlan JR. C4.5: Programs Machine Learning. Morgan Kaufmann: <br /> Disclosures Ethics                                                                                       Burlington, MA; 1993. <br />                                                                                           	17.	 Breiman L, Friedman J, Stone CJ, Olshen RA. Classification Regres- requirement publication authors pro-                                                        sion Trees. London: Chapman Hall/CRC; 1984. <br /> vided signed confirmation compliance                                         	18.	 Tan PN, Steinbach M, Kumar V. Introduction Data Mining. Boston, MA: <br />                                                                                                              Addison-Wesley; 2005. <br /> ethical legal obligations including limited                                  	19.	 Wang G, Ma J, Yan S. IGF-bagging: Information gain based feature selec- <br /> compliance ICMJE authorship competing inter-                                                        tion bagging. International Journal Innovative Computing, Informa- <br /> ests guidelines, article consider-                                                 tion Control. 2011;7(11):6247&#226;&#8364;&#8220;59. <br />                                                                                              	20.	 Lin CY, Yang CB, Hor CY, Huang KS. Disulfide bonding state prediction <br /> ation publication published                                                       svm based protein types. Bio-Inspired Computing: Theories compliance legal ethical guidelines concerning                                                      Applications. 2010:1436&#226;&#8364;&#8220;42. <br />                                                                                               	21.	 Gustafson  Snitkin ES, Parker SC, DeLisi C, Kasif S. iden- <br /> human animal research participants  applicable),                                                      tification essential genes using targeted genome sequencing com- permission obtained reproduction                                                       parative analysis. BMC Genomics. 2006;7:265. copyrighted material. article subject                                       	22.	 Hu P, Janga SC, Babu M, et&#194;&#160; al. Global functional atlas Escherichia coli <br />                                                                                                              encompassing previously uncharacterized proteins. PLoS Biol. 2009;7(4):e96. <br /> blind, independent, expert peer review. reviewers                                          	23.	 Chang CC, Lin CJ. LIBSVM: library support vector machines. http:// <br /> reported competing interests. preliminary version                                                       www.csie.ntu.edu.tw/~cjlin/libsvm. Updated <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> April 1, 2013. Accessed <br />                                                                                                              date Feb 1, 2012. paper presented International Confer-                                        	24.	 Fu WJ, Carroll RJ, Wang S. Estimating misclassification error small sam- <br /> ence Machine Learning Applications.40                                                                 ples bootstrap cross-validation. Bioinformatics. 2005;21(9):1979&#226;&#8364;&#8220;86. <br />                                                                                                  	25.	 Salwinski L, Miller CS, Smith AJ, Pettit FK, Bowie JU, Eisenberg D.                                                                                                              database interacting proteins: 2004 update. Nucleic Acids Res. 2004;32: <br /> References                                                                                                   D449&#226;&#8364;&#8220;51. <br /> 1.	 Cadigan K <br /> </body></html>