<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Cell Metab. 2012 October 3; 16(4): 526&#226;&#8364;&#8220;537. doi:10.1016/j.cmet.2012.09.007. <br />  <br />  <br />  <br />                            Mitochondrial SKN-1/Nrf Mediates Conserved Starvation <br />                            Response <br />                            Jennifer Paek1, Jacqueline Y. Lo2, Sri Devi Narasimhan3, Tammy N. Nguyen1, Kira Glover- <br />                            Cutter3, Stacey Robida-Stubbs3, Takafumi Suzuki4, Masayuki Yamamoto4, T. Keith <br />                            Blackwell3, Sean P. Curran1,2,5,* <br />                            1Leonard Davis School Gerontology, University Southern California, Los Angeles, CA <br />  <br />                            90089, USA <br />                            2College Letters, Arts, Science&#226;&#8364;&#8221;Department Molecular Computational Biology, <br />                            University Southern California, Los Angeles, CA 90089, USA <br />                            3Joslin <br />                                  Diabetes Center, Harvard Stem Cell Institute, Department Genetics, Harvard <br />                            Medical School, Boston, MA 02114, USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            4Tohoku    University School Medicine, 2-1 Seiryo-machi, Aoba-ku, Sendai, Miyagi, 980-8575, <br />                            Japan <br />                            5KeckSchool Medicine&#226;&#8364;&#8221;Department Biochemistry Molecular Biology, University                            Southern California, Los Angeles, CA 90089, USA <br />  <br />                            SUMMARY <br />                                SKN-1/Nrf plays multiple essential roles development cellular homeostasis.                                demonstrate SKN-1 executes specific appropriate transcriptional response changes                                available nutrients, leading metabolic adaptation. isolated gain function (gf) alleles                                skn-1, affecting domain SKN-1 binds transcription factor MXL-3                                mitochondrial outer membrane protein PGAM-5. skn-1(gf) mutants perceive state                                starvation presence plentiful food. aberrant monitoring cellular nutritional <br />                                status leads altered survival response skn-1(gf) mutants transcriptionally activate <br />                                genes associated metabolism, adaptation starvation, aging, survival. triggered <br />                                starvation response conserved mice constitutively activated Nrf contribute                                tumorgenicity associated activating Nrf mutations mammalian somatic cells. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                findings delineate evolutionarily conserved metabolic axis SKN-1/Nrf, establishing <br />                                complexity pathway. <br />  <br />  <br />                            INTRODUCTION <br />                                            Transcription factors influence multiple pathways choose variety targets <br />                                            ensure appropriate cellular response. SKN-1/Nrf (NF-E2-related factor) responds                                            environmental endogenous stressors including electrophiles, pathogens, xenobiotics, <br />                                            plays additional essential roles regulating development cellular homeostasis <br />                                            (Bowerman et al., 1992; Wang et al., 2010). mammals, Kelch-like ECH-associated <br />                                            protein 1 (Keap1) negatively regulates Nrf sequestration cytoplasm  <br />  <br />                            &#194;&#169;2012 Elsevier  <br />                            * <br />                              Correspondence: spcurran@usc.eduhttp://dx.doi.org/10.1016/j.cmet.2012.09.007. <br />                            SUPPLEMENTAL INFORMATION Supplemental Information includes figures, seven tables, Supplemental Experimental <br />                            Procedures article online http://dx.doi.org/10.1016/j.cmet.2012.09.007. <br />                             Paek et al.                                                                                             Page 2 <br />  <br />  <br />                                           presentation proteasome degradation (Itoh et al., 1999). orthologous <br />                                           mechanism modulating SKN-1 activity orchestrated C. elegans WDR-23 (Choe et <br />                                           al., 2009). Mutations skn-1 wdr-23 shown influence cell organism <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           survival C. elegans (Bishop Guarente, 2007; Curran Ruvkun, 2007; Tullet et al., <br />                                           2008; Wang et al., 2010). SKN-1 activate repress expression number                                           transcripts basal stress-activated conditions  et al., 2005; Inoue et al., 2005; <br />                                           Bishop Guarente, 2007; Kell et al., 2007; Kahn et al., 2008; Tullet et al., 2008; Oliveira <br />                                           et al., 2009; Li et al., 2011). SKN-1 Nrf shown occupy consensus <br />                                           DNA elements promoters target genes, mechanism target selection                                           SKN-1/Nrf varying stress conditions understood (Blackwell et al., 1994; <br />                                           Moi et al., 1994; Oliveira et al., 2009) (Figure S1). <br />  <br />                                           breadth cellular pathways influenced SKN-1 requires sophisticated mechanism <br />                                           regulating appropriate transcriptional response importantly restraining                                           activation unnecessary transcripts. C. elegans SKN-1 essential embryonic <br />                                           development digestive required normal life span adaptation <br />                                           electrophile pathogen stress larvae adult animals (Bowerman et al., 1992; <br />                                           Bowerman et al., 1993; Maduro et al., 2001; Blackwell, 2003; Maduro et al., 2007; <br />                                           Lin et al., 2009). Previous studies skn-1 utilize loss function alleles activation                                           SKN-1 inhibition negative regulator wdr-23 (Choe et al., 2009; Hasegawa                                           Miwa, 2010). SKN-1 modulated insulin-like/IGF-1 pathway promotion <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           longevity (Tullet et al., 2008). <br />  <br />                                           ability organism balance energy demands nutritional supply paramount <br />                                           survival. conditions nutrient deprivation organisms induce expression                                           starvation response genes facilitate adaptation change available nutrients                                           reliance energy stores (Lakowski Hekimi, 1998; Longo, 1999).                                           gain function alleles skn-1, alter expression genes tied                                           metabolism, starvation adaptation, growth, reproduction. mutations disrupt                                           association SKN-1 mitochondria outer-membrane protein, PGAM-5, <br />                                           independent canonical wdr-23 pathway. SKN-1 C. elegans ortholog                                           mammalian Nrf, constitutive activation Nrf induces similar starvation <br />                                           adaptation response mice. conclude SKN-1/Nrf functions evolutionarily <br />                                           conserved pathway modulate metabolism adaptation starvation. <br />  <br />                               RESULTS <br />                               Identification Novel SKN-1 Regulatory Domain Interacts PGAM-5 MXL-3 <br />                                           identify novel regulators SKN-1 activation, mutagenized C. elegans harboring NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           established gst-4p::gfp reporter SKN-1 activity, hypothesizing                                           feasible identify genetic mutants disrupt specialized SKN-1 activation (Figure S2).                                           support idea, identified classes mutants range SKN-1 activity <br />                                            unpublished results). class mutants includes alleles lax120 lax188                                           dominant (Figure S2) mapped genetic position near center LG IV (Figure <br />                                           1A). RNA interference (RNAi) screen genes genomic region identified                                           single RNAi clone targeted T19E7.2/skn-1, suppress dominant gain  <br />                                           function phenotype (Figure 1B). Sequencing skn-1 locus identified single missense <br />                                           mutation lax120 lax188, changing serine 245 leucine <span id='am-58' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-59' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span>glutamic acid</span> 237                                           lysine, respectively. similar phenotypes close proximity mutations <br />                                           suggested presence functional protein domain previously uncharacterized <br />                                           region SKN-1A/C protein (Figure 1C). <br />  <br />  <br />  <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                             Page 3 <br />  <br />  <br />                                          determine region SKN-1 polypeptide facilitates interaction specific <br />                                          proteins, performed yeast hybrid (Y2H) screen using bait construct consisting                                          200 <span id='am-51' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-52' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> region centered lax120 lax188 mutations (Figure 1C). NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          identified 56 prey constructs initial screen sequenced plasmids identify <br />                                          potential protein interactors SKN-1 domain. enhance specificity                                          interaction domain retested 56 plasmids directed Y2H assay utilizing refined <br />                                          bait construct containing 100 amino acids new SKN-1 domain. second tier                                          screening eliminated 16 constructs failed retest. performed final test                                          remaining 40 prey plasmids refined bait construct contained 50 <br />                                          amino acids SKN-1 domain did contain upstream DIDLID <br />                                          sequence (Walker et al., 2000). Bioinformatic analysis new 50 amino acid domain <br />                                          reveals casein kinase 2 domain, disrupted lax188, multiple potential <br />                                          phosphorylation sites, enrichment charged residues (Figure S2). <br />  <br />                                          plasmids passed tiers screening. plasmids contained                                          pgam-5 locus, codes mitochondrial outer membrane protein. plasmids <br />                                          contained sequence nutrient-responsive transcription factor mxl-3;                                          plasmids coded conserved serine proteinase inhibitor K10D3.4; plasmid coded <br />                                          high-mobility group protein, HMG-1.1; plasmid coded protein                                          unknown function, D1025.1. identification multiple independent isolates                                          genes combined fact plasmids facilitated strongest interaction NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          skn-1 bait constructs suggests PGAM-5, MXL-3, K10D3.4 bona fide SKN-1 <br />                                          interactors (Figure S2). ability SKN-1A SKN-1C bind MXL-3 PGAM-5 <br />                                          confirmed vitro coimmunoprecipitation studies utilizing engineered carboxyl- <br />                                          terminal HA epitope tag SKN-1 (Figure S2). results indicate direct biochemical <br />                                          interaction MXL-3 PGAM-5 SKN-1. <br />  <br />                                          dominant mutations skn-1 alter association SKN-1 protein                                          new interactors. tested directly repeating Y2H assay 50 amino <br />                                          bait construct encoding lax188(E237K) lax120(S245L) mutation combination                                          isolated prey plasmids. Yeast harboring plasmids screened change                                          bait-prey interaction, measured X-&#206;&#177;-Gal activity resistance antibiotic <br />                                          aureobasidin  lax188 lax120 mutations decreased interaction SKN-1 <br />                                          domain PGAM-5 72% 75%, respectively (Figure 1D). Interestingly,                                          mutations resulted modest increase interaction MXL-3, HMG-1.1,                                          D1025.1. findings suggest SKN-1 domain interacts PGAM-5, <br />                                          presumably mitochondria, MXL-3, help SKN-1 specify <br />                                          transcriptional targets. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          PGAM-5 predicted phosphoglycerate mutase uses alternate catalytic activity                                          protein serine/threonine phosphatase (Takeda et al., 2009). interaction PGAM-5                                          mitochondria optimize position SKN-1 function sensor                                          mitochondrial function, stress, energy output, cellular metabolic status. serine 245 <br />                                          mutation lax120 combined fact PGAM-5 function serine/threonine <br />                                          phosphatase intriguing clue biochemical function SKN-1/ <br />                                          PGAM-5 interaction. Based data, lax120 lax188 mutations destabilize                                          association SKN-1 PGAM-5 decrease affinity SKN-1 biochemical <br />                                          substrate PGAM-5 catalytic activity (Figure S1). <br />  <br />                                          identify domain interacting proteins facilitate binding SKN-1,                                          created new Y2H prey constructs coding truncated versions binding proteins                                          tested ability interact SKN-1 domain (Figure S2). N-terminal domain                                          MXL-3 capable binding SKN-1 50-, 100-, 200-mer constructs. N <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                             Page 4 <br />  <br />  <br />                                           terminus MXL-3 highly charged, facilitate interaction binding <br />                                           equally highly charged 50-mer domain SKN-1 electrostatic interactions. <br />                                           truncated versions PGAM-5 revealed positive yeast-2-hybrid interaction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           SKN-1, potentially misfolding prey fusion protein. <br />  <br />                               Identification Mitochondria Associated SKN-1 <br />                                           mitochondrial pool SKN-1 previously identified. determine                                           endogenous SKN-1 associated organelle, isolated enriched <br />                                           mitochondrial fraction worm lysates differential centrifugation. identified <br />                                           SKN-1-specific band predicted molecular weight SKN-1A isoform                                           enriched mitochondria subcellular fraction mitochondrial marker <br />                                           PDHE1 (Brys et al., 2010; Greiss et al., 2008; Li et al., 2009) (Figures 2A S3).                                           strength function SKN-1 mitochondrial surface unclear, steady- <br />                                           state presence SKN-1 isolated mitochondria suggest association                                           relatively stable. notable skn-1(lax120) skn-1(lax188) gain function (gf) <br />                                           mutants display Sma (small) body-size phenotype (Figure S2), trait shared                                           mitochondria mutants reduced <span id='am-35' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-36' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span>ATP</span> production (Lee et al., 2003). Taken                                            observations suggest pool SKN-1  referred mito-SKN-1) present <br />                                           mitochondria, potentially facilitated interaction mitochondrial <br />                                           outer membrane protein PGAM-5. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Regulation SKN-1(gf) Independent WDR-23 GSK-3 Pathways <br />                                           data indicate mitochondria-associated pool SKN-1 modulated                                           WDR-23-independent manner. WDR-23 molecularly defined regulator SKN-1 <br />                                           activity, did pull wdr-23 prey constructs Y2H screen using                                           new SKN-1 protein domain. addition, directly tested wdr-23 prey vector                                           SKN-1 50-mer-bait construct detect measureable interaction (Figure S2).                                           confirm hypothesis 50-mer-bait regulated independently                                           WDR-23 measured activation gst-4p::gfp reporter skn-1(lax120)                                           skn-1(lax188) backgrounds wdr-23 (Figure 2D). support                                           independent mechanisms SKN-1 regulation observed synergistic activation                                           SKN-1 target gene gst-4 skn-1(gf) alleles combined wdr-23 inactivation <br />                                           (Figure 2E). <br />  <br />                                           WDR-23 modulates activity abundance SKN-1 sequestering SKN-1                                           delivering proteasome degradation. verification model measured                                           increase expression wdr-23 mutant backgrounds (Table 1). activation <br />                                           wdr-23 dependent skn-1, suggesting molecular feedback loop NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           response pathway identifying wdr-23 major mechanism terminating SKN-1 <br />                                           activity regardless mode activation. <br />  <br />                                           GSK-3 shown inhibit localization SKN-1 nucleus subsequently <br />                                           repress activation SKN-1 targets  et al., 2005). determine skn-1(gf) <br />                                           mutants responsive GSK-3 regulation, treated skn-1(lax120)                                           skn-1(lax188) mutant animals gsk-3 RNAi. animals fed gsk-3 RNAi clone                                           higher expression SKN-1 target gst-4 animals fed control RNAi clones (Figure <br />                                           S3). data indicate SKN-1 gain function mutants responsive                                           gsk-3 pathway suggesting mitochondrial pool SKN-1 activated mechanisms <br />                                           independent wdr-23 gsk-3. <br />  <br />  <br />  <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                            Page 5 <br />  <br />  <br />                               SKN-1(gf) Induces Expression Metabolism Starvation Response Genes <br />                                          model predicts activation mito-SKN-1 induce specific <br />                                          transcriptional response C. elegans. examined transcriptional profile NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          synchronous skn-1(lax120) skn-1(lax188) animals compared age-matched wild-type <br />                                          controls (Figure 3A; Tables 1 S1). functional terms highly represented                                          mito-SKN-1 activation mutants genes tied metabolism starvation response <br />                                          pathways followed genes involved growth, development, life span.                                          metabolism-related genes upregulated skn-1 gain function mutants previously <br />                                          identified upregulated SKN-1. Y42G9A.3 induced SKN-1 normal <br />                                          conditions zk742.4 induced SKN-1 response exposure arsenic t- <br />                                          BOOH. particular metabolism starvation adaption genes previously <br />                                          reported SKN-1 responsive following exposure environmental electrophile <br />                                          stress conditions. metabolism genes include tyr-5/Tyrosinase, F09C8.1/ <br />                                          Phospholipase, sqrd-1/mitochondrial oxidoreductase, Y4C6B.6/glucosylceramidase, acdh-6/ <br />                                          acyl <span id='am-46' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CoA</span> dehydrogenase, lips-3/lipase-related, aat-5/amino acid transporter catalytic <br />                                          subunit, atgp-2/amino acid transporter glycoprotein subunit, aat-2/ amino acid transporter <br />                                          catalytic subunit, lips-10/lipase related. addition, aging/dauer-related genes <br />                                          zk180.6/uncharacterized protein, R08E5.3/methyltrasnferase, wdr-23/SKN-1 negative <br />                                          regulator, previously identified SKN-1 targets, differentially <br />                                          expressed starvation conditions. expression profile genes altered                                          direction wild-type worms subjected starvation (Kang Avery, 2009). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Taken  results indicate worms harboring skn-1 gain function <br />                                          alleles inappropriately perceive state starvation. confirmed specificity                                          gain function alleles WDR-23 pathways comparing abundance mito- <br />                                          SKN-1 target mRNAs activated SKN-1(gf) strains wdr-23 <br />                                          regulation. identified synergistic change gene expression mito-SKN-1(gf)- <br />                                          dependent transcripts depleted wdr-23 RNAi (Table S2). Additivity                                          observed transcripts induced repressed mito-SKN-1 activation indicating                                          existence parallel pathways activate SKN-1. <br />  <br />                                          altered expression metabolism nutrient-mobilization genes notable skn-1 <br />                                          loss function mutants shown suppress longevity phenotype associated <br />                                          dietary restriction (Bishop Guarente, 2007). genes changed                                          significantly direction change reminiscent sensed state starvation                                          severe dietary restriction (Szewczyk et al., 2006; Baugh et al., 2009).                                          transcriptional analysis mito-SKN-1 mutants representative entire organism <br />                                          tissue specific, observed strongest mito-SKN-1 activation hypodermal <br />                                          intestinal tissues (Figure 1B), collectively comprise significant portion                                          total worm body. test starvation capable inducing SKN-1 transcriptional <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          response compared expression gst-4p::gfp reporter SKN-1 activity                                          presence absence food. Consistent activation observed skn-1(gf) mutants <br />                                          detected GFP expression hypodermal tissues animals removed                                          food (Figures 3B 3C). intestine hypodermis major sites <span id='am-10' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span>lipid</span> <br />                                          storage C. elegans, explain observed changes metabolism                                          starvation response transcripts. <br />  <br />                                          Recent work shown bacterial diet itself strong impact                                          physiology worm, cases reverse suppress genetic phenotypes <br />                                          (Soukas et al., 2009; Maier et al., 2010). HT115 RNAi strain E. coli K12 <br />                                          background standard C. elegans food source, OP50, E.coli B genotype. <br />                                          genetic differences K12 B genotypes alter nutritional <br />                                          composition food source &#226;&#8364;&#339;taste&#226;&#8364;? food worm measured                                          occupancy lawn bacteria (Watts Browse, 2002; Shtonda Avery, 2006; <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                               Page 6 <br />  <br />  <br />                                           Soukas et al., 2009; Watts, 2009). Worms HT115 diet spend time bacterial <br />                                           lawn experience oscillating periods caloric restriction. test E.coli B <br />                                           K12 diets influence SKN-1 activity grew gst-4p::gfp sensor strain NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           strains. HT115 diet observed GFP expression intestinal cells normally <br />                                           observed animals fed OP50 diet (Figures 3D&#226;&#8364;&#8220;3I). <br />  <br />                                           diet strong impact physiology, examined expression mito- <br />                                           SKN-1 transcripts animals fed OP50/E.coli B compared animals raised                                           HT115/E.coli K12 food source. identified mito-SKN-1 transcripts                                           differentially expressed fed E.coli K12 diet. Notably, identified switch                                           transcriptional repression OP50 activation HT115 genes involved aging                                           metabolic pathways (Table S2). changes obvious aging/dauer- <br />                                           related genes dao-4/novel protein, ZK180.6/uncharacterized protein, R05G9R.1/ <br />                                           mucin-2-like, metabolism/signaling-related genes mxl-3/Max-like <br />                                           transcription factor, acdh-6/acyl CoA dehydrogenase, lips-3/lipase related (Table S2). <br />  <br />                                           physical interaction SKN-1 MXL-3 help specify transcription                                           metabolism starvation genes skn-1(gf) background. test combined <br />                                           skn-1(gf) alleles loss function allele mxl-3(ok1947) examined transcript <br />                                           levels genes altered gain function background (Figure 1, Table S3).                                           genes tied metabolism aging/dauer regulatory pathways affected NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           absence mxl-3. mxl-3 null allele partially suppressed increased expression                                           acdh-6/acyl CoA dehydrogenase, lips-10/lipase-related, R01E6.5/Filaggrin-2-like, dao-4/ <br />                                           novel protein, dod-24/downstream DAF-16, dod-17/downstream DAF-16, ugt-11/ <br />                                           <span id='am-94' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-95' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span>UDP</span>-Glucuronosyl Transferase, C32H11.4/hydrolase, ins-35/insulin <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> genes.                                           mxl-3 deletion reversed decreased expression F09C8.1/Phospholipase                                           R08E5.3/methyltransferase skn-1(gf) background. Intriguingly, skn-1;mxl-3 <br />                                           double mutants synergized increased expression wdr-23, indicating concerted effort <br />                                           suppress gain function activity SKN-1. dysregulated activation SKN-1 <br />                                           results transcriptional profile indicative perceived state starvation. change                                           expression metabolism genes SKN-1(gf) mutants dependence mxl-3 <br />                                           introduces new regulatory axis organismal nutrient mobilization maintain energy <br />                                           homeostasis. <br />  <br />                               Perceived State Starvation Alters Metabolic Adaptation Capacity skn-1(gf) Mutants <br />                                           presence specific independent mechanisms SKN-1 activation predicts                                           unique physiological role pathways. identify specific <br />                                           function mito-SKN-1 characterized mito-SKN-1(gf) mutants reproduction                                           life-span phenotypes challenged ability adapt physiologic stress conditions. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           skn-1 signaling pathway functions multiple longevity regulatory cascades; <br />                                            clear gain function mutants aging phenotype <br />                                           based altered expression pro- anti-aging genes (Figure 3A, Table 1).                                           assessed effect SKN-1 activating mutations adult lifespan (Figures 4A S4). <br />                                           allele skn-1 significant effect rate aging compared wild-type <br />                                           animals. <br />  <br />                                           Food availability strong influence life span.  tested ability                                           mutants respond dietary restriction (Figures 4A S4). examined life span <br />                                           skn-1(lax120) skn-1(lax188) mutants range bacteria concentrations. <br />                                           skn-1(lax188) failed display lifespan extension phenotype bacterial dilution (bDR), <br />                                           skn-1(lax120) mutant animals showed attenuated response. results                                           consistent diminished L1 survival phenotype (Figure S5) skn-1(lax188) <br />                                           mutant support idea animals perceive reduction food supply <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                           Page 7 <br />  <br />  <br />                                          ad libitum conditions leading extreme starvation response bDR <br />                                          growth conditions. alternative mechanism caloric restriction C. elegans results                                          genetic mutations decrease pharyngeal pumping reduce ingestion E. coli <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          (Lakowski Hekimi, 1998).  measured pharyngeal pumping, feeding rates, <br />                                          bacterial clearance skn-1(gf) mutants statistically significant <br />                                          difference wild-type (Figures 4B S4, Table S4). <br />  <br />                                          skn-1 essential development skn-1 m-) embryos viable (Bowerman et al., <br />                                          1992).  lax120 lax188 mutations facilitate extended reproductive <br />                                          period hermaphrodites approximately 2 days (Figure 5A). Intriguingly, observed <br />                                          suppression reproductive senescence does come expense reduced brood size <br />                                          worms mutation yield similar number viable progeny compared                                          wild-type animals (Figure 5B). did notice low penetrance  10%) bag <br />                                          worms (Bag) phenotype, progeny hatch parent higher temperatures <br />                                          (unpublished observation) (Lakowski Hekimi, 1998). C. elegans require demanding <br />                                          supply nutrients reach reproductive maturity. removed food, <br />                                          hermaphrodites cease egg laying, allowing progeny hatch utero leading Bag <br />                                          phenotype (Dong et al., 2000). low penetrance Bag phenotype mito-SKN-1(gf) <br />                                          mutants consistent idea animals experience perceived state                                          starvation. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          transcriptional changes skn-1(gf) mutants reminiscent animal coping <br />                                          nutrient deprivation conditions. C. elegans survive extended periods starvation <br />                                          stages diapause&#226;&#8364;&#8221; larval stage 1 (L1) dauers, <br />                                          alternative larval stage (L3). skn-1(gf) worms demonstrated <br />                                          transcriptional changes associated metabolic profile similar response                                          starvation, directly tested ability mutants survive food                                          extended periods time. performed L1 survival assays previously reported (Baugh et <br />                                          al., 2009) assessed ability skn-1(lax120), skn-1(lax188), skn-1- <br />                                          overexpressing skn-1b/c::gfp worms withstand starvation 5, 7, 12 days room <br />                                          temperature (~21&#194;&#176;C) (Figures 5C S5, Table S5). 5 days, clear difference <br />                                          survival skn-1(gf) worms compared wild-type skn-1b/c::gfp worms. <br />                                           increasing &#226;&#8364;&#339;age,&#226;&#8364;? skn-1b/c::gfp worms start exhibit significantly <br />                                          increased survival wild-type worms. Intriguingly, stronger skn-1(gf) allele, <br />                                          skn-1(lax188), modest increase day 7, day 12 dramatically decreased <br />                                          survival. contrast, skn-1(lax120) mutants somewhat comparable wild-type <br />                                          worms assay. results suggest given skn-1(lax188) animals                                          experiencing starvation-like response, prolonged absence food detrimental                                          long-term survival. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Worms constantly sensing environment nutritional cues, cultured                                          regular growth media, crawl bacterial lawn feed.  L1-arrested <br />                                          animals typically recover diapause resume reproductive growth transfer <br />                                          regular growth media (Figure S5, dark blue bars). assessed ability                                          &#226;&#8364;&#339;starving&#226;&#8364;? skn-1(gf) mutants recover 8 days starvation regular media (Figures <br />                                          5D S5; Table S5). shown Figure 5E, 2 days incubation 20 degrees,                                          majority wild-type worms (&gt;80%) grow gravid adults. contrast, skn-1(gf) <br />                                          worms slower growth profile. particular, small fraction skn-1(gf) <br />                                          worms arrested L1s, rest population demonstrating asynchronous <br />                                          growth primarily comprising L2 L3 stage animals. Despite obvious lag                                          growth, worms eventually develop fertile adults (data shown). skn-1 <br />                                          transgenic worms markedly reduced growth rate able form gravid <br />                                          adults (~30% adults) 2 days. conclude presence food,  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                              Page 8 <br />  <br />  <br />                                           skn-1(gf) animals behave undergoing constant state starvation. support <br />                                            upregulation metabolic transcripts, inability survive <br />                                           extended starvation, slow recovery time despite plentiful levels food. Additionally, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           data indicate gf alleles behave similarly skn-1 overexpressing <br />                                           strain, suggesting increased abundance SKN-1 does necessarily correlate                                           increased function/activity. stressful survival conditions, food <br />                                           resources limited, SKN-1 activation provide protective role ensuring                                           animal cope extended periods starvation. <br />  <br />                               Constitutive Activation Mammalian Nrf Induces Starvation Response <br />                                           Cellular adaptation starvation conserved stress response (Hardie, 2011). light                                           evolutionary conservation SKN-1-like cap-n-collar transcription factors,                                           curious Nrf activation mediate starvation-like response mammals.                                           determine Nrf activation leads perceived state starvation, measured                                           transcriptional profile tissue established Keap1&#226;&#710;&#8217;/&#226;&#710;&#8217; knockout mouse line (Ohta et <br />                                           al., 2008) (Tables 2 S6). Despite fact tissue display dysregulation                                           transcripts Keap1-Nrf dependent pathways, curious detect                                           alteration starvation-response genes. Genes tied metabolism terms                                           significantly enriched Keap1&#226;&#710;&#8217;/&#226;&#710;&#8217; tissues, similar C. elegans (Table S7). <br />                                           support SKN-1/Nrf function maintaining metabolic homeostasis, discovered <br />                                           genes associated starvation responses significantly upregulated Keap1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           knockout mice compared wild-type litter mates (Table 2). confirmed                                           transcriptional changes identified microarray data additional qRT-PCR analysis.                                           hallmark state starvation activation energy sensors <span id='am-16' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-17' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>AMP</span>-activated <br />                                           protein kinase adenylate kinase (Hardie, 2011). catalytic domain AMPK, Prkaa2 <br />                                           Ak7, upregulated lung tissue Keap1 knockout mice 2.07-                                           3.04-fold, respectively. Importantly, levels <span id='am-75' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-76' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span>cAMP</span>-dependent protein kinase subunits, <br />                                           opposite effects energy homeostasis, remained unchanged. additional <br />                                           upregulation genes involved mitochondrial peroxisomal fatty acid <br />                                           oxidation, lipases, lipid transporters (Table 2). genes shown change <br />                                           direction mouse liver periods starvation (Bauer et al., 2004). <br />                                           Remarkably, activated Nrf lung tissue 300-fold increase Ces1g. <br />                                           Overexpression mouse esterase-x/carboxylesterase 1 (Ces1g) shown promote <br />                                           beta-oxidation (Ko et al., 2009). C. elegans Ces1g-like orthologs coded operon <br />                                           T28C12.4 T28C12.5 significantly upregulated skn-1(lax120) skn-1 <br />                                           (lax188) mutant backgrounds (Table S1). results suggest constitutively <br />                                           activated Nrf lung tissue Keap1 knockout mice induces perceived state                                           starvation, similar observed intestine hypodermis C. elegans mito- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           SKN-1(gf) mutants.  addition canonical electrophile stress responses,                                           SKN-1/Nrf functions evolutionarily conserved pathway modulate <br />                                           cellular metabolism. <br />  <br />                               DISCUSSION <br />                                           Transcription factors like SKN-1/Nrf, involved multiple aspects cell biology, <br />                                           capable initiating appropriate transcriptional response activated                                           specific cellular condition. specification accomplished posttranslational <br />                                           modification, changing association transcriptional activators repressors, <br />                                           sequestration away nucleus, degradation transcription factor (Wang et <br />                                           al., 2006; Wolff Dillin, 2006; Villeneuve et al., 2010). SKN-1/Nrf family                                           transcription factors established regulator cellular response electrophiles <br />                                           pathogens. characterizing skn-1(gf) alleles, identified novel <br />  <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                              Page 9 <br />  <br />  <br />                                          function SKN-1 regulation metabolism adaptation starvation. new <br />                                          function mediated pool SKN-1 sequestered                                          mitochondrial presumably outer membrane. idea distinct organelle population <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          SKN-1 adapted respond specific changes cellular environment new <br />                                          exciting way thinking mechanism organism utilizes directing                                          appropriate transcriptional response. <br />  <br />                                          Despite functional conservation Nrf SKN-1, clear KEAP1 ortholog C. <br />                                          elegans identified. results suggest despite little sequence identity <br />                                          KEAP1 WDR-23, proteins function evolutionarily conserved <br />                                          manner negatively regulate SKN-1 activity formation mitochondrial <br />                                          SKN-1 complex. data support model SKN-1 interact                                          PGAM-5 independently WDR-23, interaction Nrf PGAM5 dependent                                          KEAP1. results intriguing evolutionary standpoint imply                                          mechanism recruiting SKN-1/Nrf evolved twice, clearly demonstrating importance                                          mitochondrial localized SKN-1 population. WDR-23,  capable                                          binding PGAM-5 MXL-3, suggesting aspects coordinated control                                          pathways (Figure S1). Despite mechanistic difference, increased activity                                          mitochondrial SKN-1 gain function mutants activated Nrf KEAP1 <br />                                          knockout mutant results perceived state starvation consequently activation                                          metabolism starvation-response pathways. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          facilitates interaction SKN-1 mitochondria? data suggest                                          SKN-1 gain function mutants reduced interaction PGAM-5. PGAM-5                                          anchor SKN-1 mitochondria reduced interaction promotes <br />                                          activation. alternative hypothesis SKN-1 anchored mitochondrial <br />                                          outermembrane predicted transmembrane domain N terminus                                          SKN-1A isoform. Future studies directed uncover mechanisms coordinate <br />                                          SKN-1 association mitochondria subsequent activation/dissociation. <br />  <br />                                          aberrant regulation SKN-1 gain function mutants independent                                          glycogen synthase kinase pathway, GSK-3  et al., 2005). SKN-1 shown                                          regulated GSK-3 phosphorylation event occurs serine residues 300&#226;&#8364;&#8220;400 <br />                                          amino acid region SKN-1. lax120 mutation serine 245 SKN-1A protein. <br />                                          serine residue poor GSK-3 target lacks required priming serine <br />                                          residue GSK-3 function  et al., 2005). <br />  <br />                                          identified new SKN-1-interacting proteins refine appropriate SKN-1 <br />                                          transcriptional response. bZip transcription factor MXL-3 partners SKN-1 NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          specification response starvation. new highly charged 50 amino acid SKN-1 <br />                                          domain identified responsible binding MXL-3 facilitates                                          interaction mitochondrial phosphatase PGAM-5 serine proteinase inhibitor <br />                                          K10D3.4. expression pgam-5 mxl-3 clearly expressed worm nervous <br />                                          pharyngeal tissues (Figure S2). behavioral response limited nutrients                                          starvation controlled established neuroendocrine circuit (Avery et al., 1993; <br />                                          Baumeister et al., 2006; Bishop Guarente, 2007). findings incorporate mito-SKN-1, <br />                                          PGAM-5, MXL-3 behavioral circuit. tested mechanistic role                                          mxl-3 expression transcriptional targets altered skn-1 gain function <br />                                          mutants. support metabolic specificity pathway, expression <br />                                          metabolism transcripts altered SKN-1 influenced MXL-3. Recent <br />                                          examination SKN-1 function normal conditions identifies different overlapping <br />                                          sets genes regulated response different stresses (Oliveira et al., 2009). <br />                                          MXL-3 provide paradigm SKN-1 interact combinatorially  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                 Page 10 <br />  <br />  <br />                                            transcription factors regulate different subsets genes conditions variable <br />                                            availability nutrients. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            gain function alleles lead perceived state starvation similar <br />                                            transcriptional profiles, molecular physiological differences                                            alleles. identified allele-specific changes effect E. coli K12 diet                                            wdr-23 inactivation mito-SKN-1-dependent transcripts (signaling: ins-35, kin-15, <br />                                            kin-16; metabolism: ZK742.4, lips-10, aat-2; aging/dauer: ZK180.6 R05G9R.1) <br />                                            (Table S2). allele-specific changes particular based observation <br />                                            skn-1(lax120) mutant animals remain partially responsive dietary restriction <br />                                            skn-1(lax188) mutants  results specify unique properties                                            different mito-SKN-1(gf) mutants substantiate idea mito-SKN-1,                                            concert MXL-3, influence metabolism organismal response changes                                            available nutrients. <br />  <br />                                            Activating Nrf mutations identified numerous somatic tumors, especially <br />                                            lung cell carcinomas (Ohta et al., 2008; Shibata et al., 2008). tumors tend                                            extremely resistant <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> radiation-based therapies. shown                                            resistance phenotype partially dependent activation Nrf-dependent transcripts <br />                                            (Singh et al., 2008). Hyperproliferative tumor cells rapidly exhaust available energy <br />                                            supplies forcing necessity adapt nutritionally limiting environment (Jang et al., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            2011). lung canonically thought metabolic tissue does                                            major contribution maintaining energy homeostasis organism. lung  <br />                                             exceptionally active metabolically, fabricating lipids form                                            surfactants rich phospholipids sterols. activation mitochondrial lipid oxidation <br />                                            provide mechanism ensure adequate ATP levels continued cancer growth prior <br />                                            new vasculature provided angiogenesis. Nrf identified potential target                                            cancer treatment suppress cellular stress adaption mechanisms. results suggest                                            understanding different specific mechanisms Nrf activation likely important <br />                                            elucidating effects cell survival carcinogenesis (Mitsuishi et al., 2012).                                            characterization role mito-SKN-1/Nrf provide insights physiologic <br />                                            roles mitochondrial-specific population essential biological regulator                                            lead novel strategies combatting cancers resistant traditional therapies. <br />  <br />                               EXPERIMENTAL PROCEDURES <br />                               Standard Growth Conditions <br />                                            C. elegans raised standard 6 cm nematode growth media plates supplemented                                            streptomycin seeded Escherichia coli OP50. RNAi experiments NGM plates <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            containing 5 mM IPTG 100 ug ml&#226;&#710;&#8217;1 carbencillin seeded overnight cultures                                            double-stranded RNAi-expressing HT115 bacteria. Plates allowed induce overnight <br />                                            followed transfer age-synchronous populations C. elegans. <br />  <br />                               Strains Used Study <br />                                            Strains used study N2 Bristol (wild-type), CB4856 (Hw), CL2166 <br />                                            [gst-4p::gfp]III, SPC167[skn-1(lax120)IV;gst-4p::gfp], SPC168[skn-1(lax188) <br />                                            IV;gst-4p::gfp], SPC207[skn-1(lax120)IV], SPC227[skn-1(lax188)IV]. <br />  <br />                               Statistics <br />                                            Statistical analyses performed JMP 8 software. <br />  <br />  <br />  <br />  <br />                                                 Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                            Page 11 <br />  <br />  <br />                               Isolation Dominant SKN-1 Activation Mutants <br />                                             C. elegans strain harboring SKN-1 transcriptional reporter gst-4p::gfp                                             mutagenized Ethyl methanesulfonate (EMS). F1 generation worms increased GFP <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             expression, indicating SKN-1 activation, isolated characterization. F2 <br />                                             generation animals singled F3 progeny analyzed 100% penetrance                                             activated GFP phenotype. homozygous animals mated                                             unmutagenized parental strain. Successful matings yielded 100% F1 progeny expressing <br />                                             activated GFP phenotype confirmed dominant mutants. <br />  <br />                               Yeast-2-Hybrid <br />                                             Bait prey plasmids generated cloning 50-mer, 100-mer, 200-mer <br />                                             oligonucleotides skn-1(lax120) skn-1(lax188) mutations pLexA C. <br />                                             elegans cDNA library pACT2.2 (Addgene), respectively. Interaction tested                                             synthetic complete agar lacked leucine tryptophan supplemented X-&#206;&#177;- <br />                                             gal. Interactors identified transforming Y2HGold (Clontech) bait (pLexA- <br />                                             SKN-1) cDNA prey library. Positive clones grown absence                                             tryptophan sequenced. Sequenced clones retested individually. <br />  <br />                               Microarrays <br />                                             RNA processed using Affymetrix GeneChip 3&#226;&#8364;&#178; IVT express Kit. Mouse samples <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             hybridized Affymetrix MouseGene 1.0 ST array. knockout mice                                             wild-type littermates used studies. C. elegans samples hybridized                                             Affymetrix GeneChip C. elegans Genome Arrays. biologically unique wild-type <br />                                             samples compared biological replicates skn-1(lax120) biological <br />                                             replicates skn-1(lax188) utilized study. <br />  <br />                               L1 Arrest, Survival, Recovery Assays <br />                                             Following hypochlorite treatment, eggs seeded 24 plates (4 wells strain) <br />                                             S-Basal + <span id='am-89' about='obo:IMR_0200486' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-90' property="oboInOwl:id" content="IMR:0200486" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="Cholesterol" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="KEGG:C00187" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Cholesterol</span>. scoring viability, drop spotted <br />                                             coverslip. Movement used parameter score viability. Bromophenol blue <br />                                             staining used confirm dead worms. Animals scored 5, 7, 12 days. day 8 <br />                                             cohort animals placed OP50 starvation recovery assays. Animals                                             scored developmental progress wild-type animals reached reproductive <br />                                             maturity. <br />  <br />                               Life-Span Analysis <br />                                             Life-span experiments performed previously described (Curran Ruvkun, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             2007). strains egg prepped generate synchronous populations, 20&#226;&#8364;&#8220;30 L4 <br />                                             animals placed NGM plates seeded OP50 containing 50 &#206;&#188;g ml&#226;&#710;&#8217;1 5- <br />                                             fluoro-2&#226;&#8364;&#178;-deoxyuridine (FUdR) prevent progeny hatching. worms kept 20&#194;&#176;C <br />                                             scored days survival gentle prodding platinum wire. Dietary <br />                                             restriction lifespan experiments performed 25&#194;&#176;C plates containing ampicillin                                             inhibit bacterial growth. Worms transferred daily fresh plates                                             bacterial concentration. <br />  <br />                                             Additional methods Supplemental Information. <br />  <br />                               Supplementary Material <br />                                             Refer Web version PubMed Central supplementary material. <br />  <br />  <br />  <br />  <br />                                                  Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                            Page 12 <br />  <br />  <br />                               Acknowledgments <br />                                          thank Chandra Tucker, Duke University C. elegans Y2H library (Developed Maureen Barr&#226;&#8364;&#8482;s <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          laboratory). grateful Daniel Wai, Children&#226;&#8364;&#8482;s Hospital Los Angeles analysis Affymetrix microarray <br />                                          data. thank Drs. Caleb Finch, Alexander Soukas, Shanshan Pang critically reading manuscript <br />                                          members Curran laboratory technical support fruitful discussions. S.P.C. conceived                                          designed study. T.K.B. M.Y. collaborated SPC plan organize experiments. J.P., J.Y.L., S.D.N., <br />                                          T.N.N., K.G.C., S.R.S., T.S., S.P.C. conducted experiments. strains used study                                          obtained Caenorhabditis Genetics Center (CGC), supported National Institutes Health&#226;&#8364;&#8221; <br />                                          NCRR. Research study supported NIH R01GM62891 (T.K.B.) R00AG032308 (S.P.C.). S.P.C.                                          Ellison Medical Foundation Junior Scholar Aging. <br />  <br />  <br />                               REFERENCES <br />                                          JH, Blackwell TK. SKN-1 links C. elegans mesendodermal specification conserved oxidative <br />                                            stress response. Genes Dev. 2003; 17:1882&#226;&#8364;&#8220;1893. [PubMed: 12869585] <br />                                          JH, Vranas K, Lucke M, Inoue H, Hisamoto N, Matsumoto K, Blackwell TK. Regulation                                            Caenorhabditis elegans oxidative stress defense protein SKN-1 glycogen synthase kinase-3. <br />                                            Proc. Natl. Acad. Sci. USA. 2005; 102:16275&#226;&#8364;&#8220;16280. [PubMed: 16251270] <br />                                          Avery L, Bargmann CI, Horvitz HR. Caenorhabditis elegans unc-31 gene affects multiple nervous <br />                                            controlled functions. Genetics. 1993; 134:455&#226;&#8364;&#8220;464. [PubMed: 8325482] <br />                                          Bauer M, Hamm AC, Bonaus M, Jacob  Jaekel J, Schorle H, Pankratz MJ, Katzenberger JD. <br />                                            Starvation response mouse liver shows strong correlation life-span-prolonging processes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Physiol. Genomics. 2004; 17:230&#226;&#8364;&#8220;244. [PubMed: 14762175] <br />                                          Baugh LR, Demodena J, Sternberg PW. RNA Pol II accumulates promoters growth genes                                            developmental arrest. Science. 2009; 324:92&#226;&#8364;&#8220;94. [PubMed: 19251593] <br />                                          Baumeister R, Schaffitzel E, Hertweck M. Endocrine signaling Caenorhabditis elegans controls <br />                                            stress response longevity. J. Endocrinol. 2006; 190:191&#226;&#8364;&#8220;202. [PubMed: 16899554] <br />                                          Bishop NA, Guarente L. neurons mediate diet-restriction-induced longevity C. elegans. Nature. <br />                                            2007; 447:545&#226;&#8364;&#8220;549. [PubMed: 17538612] <br />                                          Blackwell TK, Bowerman B, Priess JR, Weintraub H. Formation monomeric DNA binding <br />                                            domain Skn-1 bZIP homeodomain elements. Science. 1994; 266:621&#226;&#8364;&#8220;628. [PubMed: <br />                                            7939715] <br />                                          Bowerman B, Draper BW, Mello CC, Priess JR. maternal gene skn-1 encodes protein                                            distributed unequally early C. elegans embryos. Cell. 1993; 74:443&#226;&#8364;&#8220;452. [PubMed: 8348611] <br />                                          Bowerman B, Eaton BA, Priess JR. skn-1, maternally expressed gene required specify fate                                              ventral blastomeres early. C. elegans embryo. Cell. 1992; 68:1061&#226;&#8364;&#8220;1075. <br />                                          Brys K, Castelein N, Matthijssens F, Vanfleteren JR, Braeckman BP. Disruption insulin signalling <br />                                              preserves bioenergetic competence mitochondria ageing Caenorhabditis elegans. BMC Biol. <br />                                              2010; 8:91. [PubMed: 20584279] <br />                                          Choe KP, Przybysz AJ, Strange K. WD40 repeat protein WDR-23 functions CUL4/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              DDB1 ubiquitin ligase regulate nuclear abundance activity SKN-1 Caenorhabditis <br />                                              elegans. Mol. Cell. Biol. 2009; 29:2704&#226;&#8364;&#8220;2715. [PubMed: 19273594] <br />                                          Curran SP, Ruvkun G. Lifespan regulation evolutionarily conserved genes essential viability. <br />                                              PLoS Genet. 2007; 3:e56. [PubMed: 17411345] <br />                                          Dong MQ, Chase D, Patikoglou GA, Koelle MR. Multiple RGS proteins alter neural G protein <br />                                              signaling allow. C. elegans rapidly change behavior fed. Genes Dev. 2000; 14:2003&#226;&#8364;&#8220; <br />                                              2014. <br />                                          Greiss S, Hall J, Ahmed S, Gartner  C. elegans SIR-2.1 trans-location linked proapoptotic <br />                                              pathway parallel cep-1/p53 DNA damage-induced apoptosis. Genes Dev. 2008; <br />                                              22:2831&#226;&#8364;&#8220;2842. [PubMed: 18923081] <br />                                          Hardie DG. Sensing energy nutrients AMP-activated protein kinase.  J. Clin. Nutr. 2011; <br />                                              93:891S&#226;&#8364;&#8220;6. [PubMed: 21325438] <br />  <br />  <br />  <br />  <br />                                                Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                    Page 13 <br />  <br />  <br />                                          Hasegawa K, Miwa J. Genetic cellular characterization Caenorhabditis elegans mutants <br />                                               abnormal regulation phase II enzymes. PLoS  2010; 5:e11194. [PubMed: <br />                                               20585349] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Inoue H, Hisamoto N, JH, Oliveira RP, Nishida E, Blackwell TK, Matsumoto K. C. elegans <br />                                               p38 MAPK pathway regulates nuclear localization transcription factor SKN-1 oxidative <br />                                               stress response. Genes Dev. 2005; 19:2278&#226;&#8364;&#8220;2283. [PubMed: 16166371] <br />                                          Itoh K, Wakabayashi N, Katoh Y, Ishii T, Igarashi K, Engel JD, Yamamoto M. Keap1 represses <br />                                               nuclear activation antioxidant responsive elements Nrf2 binding amino- <br />                                               terminal Neh2 domain. Genes Dev. 1999; 13:76&#226;&#8364;&#8220;86. [PubMed: 9887101] <br />                                          Jang T, Calaoagan JM, Kwon E, Samuelsson S, Recht L, Laderoute KR. 5&#226;&#8364;&#178;-AMP-activated protein <br />                                               kinase activity elevated early primary brain tumor development rat. Int. J. Cancer. <br />                                               2011; 128:2230&#226;&#8364;&#8220;2239. [PubMed: 20635388] <br />                                          Kahn NW, Rea SL, Moyle S, Kell  Johnson TE. Proteasomal dysfunction activates transcription <br />                                               factor SKN-1 produces selective oxidative-stress response Caenorhabditis elegans. <br />                                               Biochem. J. 2008; 409:205&#226;&#8364;&#8220;213. [PubMed: 17714076] <br />                                          Kang C, Avery L. Systemic regulation starvation response Caenorhabditis elegans. Genes Dev. <br />                                               2009; 23:12&#226;&#8364;&#8220;17. [PubMed: 19136622] <br />                                          Kell  Ventura N, Kahn N, Johnson TE. Activation SKN-1 novel kinases Caenorhabditis <br />                                               elegans. Free Radic. Biol. Med. 2007; 43:1560&#226;&#8364;&#8220;1566. [PubMed: 17964427] <br />                                          Ko KW, Erickson B, Lehner R. Es-x/Ces1 prevents triacylglycerol accumulation McArdle-RH7777 <br />                                               hepatocytes. Biochim. Biophys. Acta. 2009; 1791:1133&#226;&#8364;&#8220;1143. [PubMed: 19651238] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Lakowski B, Hekimi S. genetics caloric restriction Caenorhabditis elegans. Proc. Natl. Acad. <br />                                               Sci. USA. 1998; 95:13091&#226;&#8364;&#8220;13096. [PubMed: 9789046] <br />                                          Lee SS, Lee RY, Fraser AG, Kamath RS, Ahringer J, Ruvkun G. systematic RNAi screen identifies <br />                                               critical role mitochondria C. elegans longevity. Nat. Genet. 2003; 33:40&#226;&#8364;&#8220;48. [PubMed: <br />                                               12447374] <br />                                          Li J, Cai T, Wu P, Cui  Chen X, Hou J, Xie  Xue P, Shi L, Liu P, et al. Proteomic analysis                                               mitochondria Caenorhabditis elegans. Proteomics. 2009; 9:4539&#226;&#8364;&#8220;4553. [PubMed: 19670372] <br />                                          Li X, Matilainen O, Jin C, Glover-Cutter KM, Holmberg CI, Blackwell TK. Specific SKN-1/Nrf stress <br />                                               responses perturbations translation elongation proteasome activity. PLoS Genet. 2011; <br />                                               7:e1002119. [PubMed: 21695230] <br />                                          Lin KT, Broitman-Maduro G, Hung WW, Cervantes S, Maduro MF. Knockdown SKN-1                                               Wnt effector TCF/POP-1 reveals differences endomesoderm specification C. briggsae                                               compared C. elegans. Dev. Biol. 2009; 325:296&#226;&#8364;&#8220;306. [PubMed: 18977344] <br />                                          Longo VD. Mutations signal transduction proteins increase stress resistance longevity yeast, <br />                                               nematodes, fruit flies, mammalian neuronal cells. Neurobiol. Aging. 1999; 20:479&#226;&#8364;&#8220;486. <br />                                               [PubMed: 10638521] <br />                                          Maduro MF, Broitman-Maduro G, Mengarelli  Rothman JH. Maternal deployment embryonic <br />                                               SKN-1&#226;&#8364;&#8221;&gt;MED-1,2 cell specification pathway C. elegans. Dev. Biol. 2007; 301:590&#226;&#8364;&#8220;601. <br />                                               [PubMed: 16979152] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Maduro MF, Meneghini MD, Bowerman B, Broitman-Maduro G, Rothman JH. Restriction                                               mesendoderm single blastomere combined action SKN-1 GSK-3beta homolog <br />                                               mediated MED-1 -2 C. elegans. Mol. Cell. 2001; 7:475&#226;&#8364;&#8220;485. [PubMed: 11463373] <br />                                          Maier W, Adilov B, Regenass M, Alcedo J. neuromedin U receptor acts sensory                                               modulate food type-dependent effects C. elegans lifespan. PLoS Biol. 2010; 8:e1000376. <br />                                               [PubMed: 20520844] <br />                                          Mitsuishi Y, Taguchi K, Kawatani Y, Shibata T, Nukiwa T, Aburatani H, Yamamoto M, Motohashi H. <br />                                               Nrf2 redirects glucose glutamine anabolic pathways metabolic reprogramming. Cancer <br />                                               Cell. 2012; 22:66&#226;&#8364;&#8220;79. [PubMed: 22789539] <br />                                          Moi P, Chan K, Asunis  Cao  Kan YW. Isolation NF-E2-related factor 2 (Nrf2), NF-E2-like <br />                                               basic leucine zipper transcriptional activator binds tandem NF-E2/AP1 repeat                                               beta-globin locus control region. Proc. Natl. Acad. Sci. USA. 1994; 91:9926&#226;&#8364;&#8220;9930. [PubMed: <br />                                               7937919] <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                   Page 14 <br />  <br />  <br />                                          Ohta T, Iijima K, Miyamoto M, Nakahara  Tanaka H, Ohtsuji M, Suzuki T, Kobayashi  Yokota J, <br />                                              Sakiyama T, et al. Loss Keap1 function activates Nrf2 provides advantages lung cancer <br />                                              cell growth. Cancer Res. 2008; 68:1303&#226;&#8364;&#8220;1309. [PubMed: 18316592] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Oliveira RP, Porter Abate J, Dilks K, Landis J, Ashraf J, Murphy CT, Blackwell TK. Condition- <br />                                              adapted stress longevity gene regulation Caenorhabditis elegans SKN-1/Nrf. Aging Cell. <br />                                              2009; 8:524&#226;&#8364;&#8220;541. [PubMed: 19575768] <br />                                          Shibata T, Ohta T, Tong KI, Kokubu  Odogawa R, Tsuta K, Asamura H, Yamamoto M, Hirohashi <br />                                              S. Cancer related mutations NRF2 impair recognition Keap1-Cul3 E3 ligase promote <br />                                              malignancy. Proc. Natl. Acad. Sci. USA. 2008; 105:13568&#226;&#8364;&#8220;13573. [PubMed: 18757741] <br />                                          Shtonda BB, Avery L. Dietary choice behavior Caenorhabditis elegans. J. Exp. Biol. 2006; 209:89&#226;&#8364;&#8220; <br />                                              102. [PubMed: 16354781] <br />                                          Singh  Boldin-Adamsky S, Thimmulappa RK, Rath SK, Ashush H, Coulter J, Blackford  <br />                                              Goodman SN, Bunz F, Watson WH, et al. RNAi-mediated silencing nuclear factor erythroid-2- <br />                                              related factor 2 gene expression non-small cell lung cancer inhibits tumor growth increases <br />                                              efficacy chemotherapy. Cancer Res. 2008; 68:7975&#226;&#8364;&#8220;7984. [PubMed: 18829555] <br />                                          Soukas AA, Kane EA, Carr CE, Melo JA, Ruvkun G. Rictor/TORC2 regulates fat metabolism, <br />                                              feeding, growth, life span Caenorhabditis elegans. Genes Dev. 2009; 23:496&#226;&#8364;&#8220;511. <br />                                              [PubMed: 19240135] <br />                                          Szewczyk NJ, Udranszky IA, Kozak E, Sunga J, Kim SK, Jacobson LA, Conley CA. Delayed <br />                                              development lifespan extension features metabolic lifestyle alteration C. elegans                                              dietary restriction. J. Exp. Biol. 2006; 209:4129&#226;&#8364;&#8220;4139. [PubMed: 17023606] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Takeda K, Komuro Y, Hayakawa T, Oguchi H, Ishida Y, Murakami S, Noguchi T, Kinoshita H, <br />                                              Sekine Y, Iemura S, et al. Mitochondrial phosphoglycerate mutase 5 uses alternate catalytic <br />                                              activity protein serine/threonine phosphatase activate ASK1. Proc. Natl. Acad. Sci. USA. <br />                                              2009; 106:12301&#226;&#8364;&#8220;12305. [PubMed: 19590015] <br />                                          Tullet JM, Hertweck M, JH, Baker J, Hwang JY, Liu S, Oliveira RP, Baumeister R, Blackwell TK. <br />                                              Direct inhibition longevity-promoting factor SKN-1 insulin-like signaling C. elegans. <br />                                              Cell. 2008; 132:1025&#226;&#8364;&#8220;1038. [PubMed: 18358814] <br />                                          Villeneuve NF, Lau  Zhang DD. Regulation Nrf2-Keap1 antioxidant response ubiquitin <br />                                              proteasome  insight cullin-ring ubiquitin ligases. Antioxid. Redox Signal. 2010; <br />                                              13:1699&#226;&#8364;&#8220;1712. [PubMed: 20486766] <br />                                          Walker AK, R, Batchelder C, Kophengnavong T, Gronniger JT, Shi Y, Blackwell TK.                                              conserved transcription motif suggesting functional parallels Caenorhabditis elegans <br />                                              SKN-1 Cap&#226;&#8364;&#732;n&#226;&#8364;&#8482;Collar-related basic leucine zipper proteins. J. Biol. Chem. 2000; 275:22166&#226;&#8364;&#8220; <br />                                              22171. [PubMed: 10764775] <br />                                          Wang J, Robida-Stubbs S, Tullet JM, Rual JF, Vidal M, Blackwell TK. RNAi screening implicates                                              SKN-1-dependent transcriptional response stress resistance longevity deriving                                              translation inhibition. PLoS Genet. 2010; 6:6. <br />                                          Wang Y, Oh SW, Deplancke B, Luo J, Walhout AJ, Tissenbaum HA. C. elegans 14-3-3 proteins <br />                                              regulate life span interact SIR-2.1 DAF-16/FOXO. Mech. Ageing Dev. 2006; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              127:741&#226;&#8364;&#8220;747. [PubMed: 16860373] <br />                                          Watts JL. Fat synthesis adiposity regulation Caenorhabditis elegans. Trends Endocrinol. Metab. <br />                                              2009; 20:58&#226;&#8364;&#8220;65. [PubMed: 19181539] <br />                                          Watts JL, Browse J. Genetic dissection polyunsaturated fatty acid synthesis Caenorhabditis <br />                                              elegans. Proc. Natl. Acad. Sci. USA. 2002; 99:5854&#226;&#8364;&#8220;5859. [PubMed: 11972048] <br />                                          Wolff S, Dillin  trifecta aging Caenorhabditis elegans. Exp. Gerontol. 2006; 41:894&#226;&#8364;&#8220;903. <br />                                              [PubMed: 16919905] <br />  <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                      Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 1. Dominant SKN-1 Domain Functions Independently Canonical WDR-23 <br />                                          Pathway <br />                                            SNP mapping Hawaiian wild-type strain CB4856 assigned alleles lax120                                          lax188 LGIV B0273 C49H3. 576 RNAi clones covering genes                                          SNPs tested suppression dominant phenotype. <br />                                          (B) skn-1/T19E7.2 RNAi clone construct capable suppression                                          dominant phenotype. <br />                                          (C) lax120 lax188 encode single missense mutations skn-1 coding sequence. Y2H <br />                                          analysis performed bait constructs code 200, 100, 50 amino acid <br />                                          region mutated dominant skn-1 alleles. <br />                                          (D) skn-1(lax120) skn-1(lax188) mutations alter interaction SKN-1 50 amino <br />                                          acid domain Y2H targets. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                           Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 2. Identification Unique Mitochondria-Associated SKN-1 Activation Pathway <br />                                            SKN-1 enriched mitochondria fraction C. elegans. pL4440                                          skn-1 RNAi controls worm extracts. left panel, asterisks denote bands <br />                                          clearly decreased skn-1 RNAi, contrast 85 kD SKN-1 itself. <br />                                          (B) Partial purification achieved spinning mito pellet (mito), indicated <br />                                          presence remaining mitochondrial marker, PDHE1 (Greiss et al., 2008; Li et al., <br />                                          2011; Brys et al., 2010), cytosolic fraction (cyto), mitochondria fraction lacks <br />                                          cytoplasmic marker GAPDH. <br />                                          (C) Arrowhead indicates remaining PDHE1 removed stripping. <br />                                          (D) Synergistic activation SKN-1 transcriptional reporter (gst-4p::gfp)                                          canonical wdr-23 pathway dominant skn-1 mutations. Percentage represents fraction <br />                                          total animals, n, phenotype. <br />                                          (E) GFP score indicated genotypes based fluorescence intensity. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                       Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 3. SKN-1 Activated Starvation Changes Available Nutrients <br />                                            terms genes identified dysregulated transcriptional analysis lax120 <br />                                          lax188 mutants. <br />                                          (B) SKN-1 activity sensor worms grown food starvation conditions NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          imaged GFP expression indicative SKN-1 activation gst-4p::gfp reporter. <br />                                          (C) Quantification GFP levels observed described (B). <br />                                          (D  Indicated strains fed E. coli B strain OP50. <br />                                          (D)&#226;&#8364;&#8220;(F) E.coli K12 strain HT115 (G)&#226;&#8364;&#8220;  imaged GFP expression described <br />                                          (B). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                        Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 4. Activated Mito-SKN-1 Promotes Perceived State Starvation <br />                                            Bacterial dilution (bDR) life span; mean life span adult worms grown solid <br />                                          media concentration bacteria 25&#194;&#176;C. (B) Feeding rates assessed                                          pharyngeal pumping assays indicated strains. Error bars represent average +SD. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                         Page 19 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 5. Activated SKN-1 Promotes Extended Reproduction Developmental Delay                                          Response Food <br />                                            Nonmated reproductive span worms fed OP50 bacteria 20&#194;&#176;C. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          (B) Total progeny  . <br />                                          (C) L1 diapause survival measured movement staining bromophenol blue <br />                                          following 5, 7, 12 days absence food. <br />                                          (D) Growth development animals L1 diapause following feeding. <br />                                          (E) Percentage animals arrested early development recovery starvation. (* &lt; <br />                                          0.05, ** &lt; 0.01, *** &lt; 0.001). Error bars represent SEM. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cell Metab. Author manuscript; available PMC 2013 October 03. <br />                             Paek et al.                                                                                                                     Page 20 <br />  <br />  <br />  <br />  <br />                                                                                                 Table 1 <br />                            Activation Mito-SKN-1 Regulates Unique Transcriptional Response <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                               skn-1(lax120)                           skn-1(lax188) <br />  <br />                                                                               fold changeb     std devc     t-testd   fold changeb    std devc   t-testd <br />                                Stress         <span id='am-23' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-24' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span>Glutathione</span>       gcs-1                   4.48   1.00         0.001             4.00    0.95       &lt;0.001 <br />                                                                 gst-4                 21.31    1.06         &lt;0.001           22.16    1.01       &lt;0.001 <br />  <br />                                               Detox             ugt-11                8.75   0.96         0.001             4.61    1.05       &lt;0.001 <br />  <br />                                Aging/Dauer    Aging             wdr-23                2.81   1.01         0.001             2.85    0.98       &lt;0.001 <br />  <br />                                                                 dod-17                  5.00   0.94         0.002             4.38    1.01       &lt;0.001 <br />                                                                 dod-24                  7.69   1.15         0.005             7.49    1.20       &lt;0.001 <br />                                               Dauer             dod-6                   2.60   0.92           NS              2.33    1.22       0.004 <br />                                                                 C32H11.4              18.85    1.08         &lt;0.001           14.68    1.15       &lt;0.001 <br />                                               Unknown           dao-4                 &#226;&#710;&#8217;9.92    1.21         0.001            &#226;&#710;&#8217;2.37    1.09       0.001 <br />                                                                 R05G9R.1              &#226;&#710;&#8217;8.38    2.05         0.018            &#226;&#710;&#8217;4.49    1.11       &lt;0.001 <br />  <br />                                                                 zk180.6             &#226;&#710;&#8217;7.66    2.49         0.031            &#226;&#710;&#8217;3.78    1.09       &lt;0.001 <br />  <br />                                Metabolism     Amino Acid        tyr-5              &#226;&#710;&#8217;42.71    2.38         0.009          &#226;&#710;&#8217;48.67     1.01       &lt;0.001 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                 aat-5                   1.77   0.87           NS              1.51    1.12       0.004 <br />                                                                 atgp-2                  1.88   0.85         0.045             1.81    1.07       0.002 <br />                                                                 aat-2                   2.31   1.02         &lt;0.001            3.21    1.00       &lt;0.001 <br />                                                                 K07E1.1                 4.97   1.08         0.002             2.83    0.98       &lt;0.001 <br />                                               Lipid             F09C8.1              &#226;&#710;&#8217;14.06    1.89         0.009            &#226;&#710;&#8217;9.03    0.82       &lt;0.001 <br />                                                                 Y4C6B.6               &#226;&#710;&#8217;3.23    0.95         0.010            &#226;&#710;&#8217;3.20    0.96       &lt;0.001 <br />  <br />                                                                 acdh-6              &#226;&#710;&#8217;2.93    1.18         0.010            &#226;&#710;&#8217;2.89    1.00       &lt;0.001 <br />  <br />                                                                 lips-3              &#226;&#710;&#8217;2.37    0.96         0.010            &#226;&#710;&#8217;2.96    1.06       0.011 <br />  <br />                                                                 lips-10                 2.74   1.14         0.007             1.90    1.13       &lt;0.001 <br />                                                                 zk742.4                 9.36   1.08         0.001             7.80    1.09       &lt;0.001 <br />  <br />                                               Quinone           R08E5.3             &#226;&#710;&#8217;6.67    2.67         0.039            &#226;&#710;&#8217;4.67    0.94       &lt;0.001 <br />  <br />                                                                 sqrd-1                &#226;&#710;&#8217;6.18    2.40         0.040            &#226;&#710;&#8217;3.59    1.15       &lt;0.001 <br />  <br />                                               Unknown           Y42G9A.3              3.77   1.08         0.004             3.42    1.07       &lt;0.001 <br />  <br />                                Reproduction   Germline          bcc-1                   2.32   0.96         0.001             2.35    0.95       &lt;0.001 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                 glh-1                   2.42   0.80         0.007             2.24    1.14       &lt;0.001 <br />                                               Piwi/Paz          ppw-2                   2.75   0.65           NS              2.38    1.21       0.001 <br />  <br />                                Signaling      Insulin           ins-35                3.20   1.00         0.001             3.64    1.01       &lt;0.001 <br />  <br />                                               Kinase            kin-15                  3.61   0.90         0.003             1.52    0.91       0.002 <br />  <br />                                                                 kin-16                4.04   1.05         0.003             2.15    1.00       &lt;0.001 <br />  <br />                                               Transcription     mxl-3               &#226;&#710;&#8217;1.03    0.92           NS             &#226;&#710; <br /> </body></html>