<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Cell. 2008 October 24; 32(2): 259&#226;&#8364;&#8220;275. doi:10.1016/j.molcel.2008.10.002. <br />  <br />  <br />  <br />                            Atomic structure KEOPS complex: ancient protein <br />                            kinase-containing molecular machine <br />                            Daniel Y.L. Mao1,*, Dante Neculai1,*, Michael Downey1,2, Stephen Orlicky1, Yosr  Haffani1, <br />                            Derek F. Ceccarelli1, Jenny S.L. Ho3, Rachel K. Szilard1, Wei Zhang1,2, Cynthia S. Ho1, Leo <br />                            Wan1, Christophe Fares4, Sigrun Rumpel4, Igor Kurinov5, Cheryl H. Arrowsmith4, Daniel <br />                            Durocher1,2,&#194;&#167;, Frank Sicheri1,2,&#194;&#167; <br />                            1 Samuel Lunenfeld Research Institute, Mount Sinai Hospital, 600 University Avenue, Toronto, <br />  <br />                             Canada M5G 1X5 <br />                            2   Department Molecular Genetics, University Toronto, Toronto, Ontario, Canada <br />                            3Program Developmental Stem Cell Biology, Hospital Sick Children University <br />                            Toronto, 555 University Avenue, Toronto,  Canada, M5G 1X8 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            4Ontario Cancer Institute, Department Medical Biophysics, University Toronto, 101 College <br />                            Street, Toronto,  Canada, M5G 1L7 <br />                            5Cornell University, Department Chemistry <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Chemical</span> Biology, NE-CAT, Bldg. 436E, <br />                            Advanced Photon Source, 9700 S. Cass Ave., Argonne,  60439 <br />  <br />                            SUMMARY <br />                                  Kae1 universally conserved ATPase essential gene set bacteria. archaea <br />                                  eukaryotes, Kae1 embedded protein kinase-containing KEOPS complex. <br />                                  Mutation KEOPS subunits yeast leads striking telomere transcription defects                                  exact biochemical function KEOPS known. step elucidating function,                                  solved atomic structure archaea-derived KEOPS complexes involving Kae1, Bud32, Pcc1 <br />                                  Cgi121 subunits. studies suggest Kae1 regulated levels primordial <br />                                  protein kinase Bud32, itself regulated Cgi121.  Pcc1 appears function                                  dimerization module, suggesting KEOPS processive molecular machine. <br />                                  Lastly, Bud32 lacks conventional substrate-recognition infrastructure eukaryotic protein <br />                                  kinases including activation segment, Bud32 provide glimpse evolutionary history <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  protein kinase family. <br />  <br />  <br />                            INTRODUCTION <br />                                              telomere nucleoproteic structure forms end linear chromosomes.                                              eukaryotes, structure accomplishes critical functions.  telomeres recruit  <br />  <br />                            &#194;&#169; 2008 Elsevier  rights reserved. <br />                            &#194;&#167; <br />                              Address correspondence  Frank Sicheri, Ph.D., Centre Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai <br />                            Hospital, Room 1090, 600 University Avenue, Toronto,  CANADA M5G 1X5, Tel: 416-586-4800 ext. 8471, Fax: 416-586-8869, <br />                            sicheri@mshri ca. Daniel Durocher, Ph.D., Centre Systems Biology, Samuel Lunenfeld Research Institute, Mount Sinai <br />                            Hospital, Room 1073, 600 University Avenue, Toronto,  CANADA M5G 1X5, Tel: 416-586-4800 ext. 2544, Fax: 416-586-8869, <br />                            durocher@mshri ca. <br />                             authors contributed equally work <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service                            customers providing early version manuscript. manuscript undergo copyediting, typesetting, review                            resulting proof published final citable form. note production process errors                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Mao et al.                                                                                           Page 2 <br />  <br />  <br />                                         activate telomerase, reverse transcriptase polymerizes short TG-rich repeats 3&#226;&#8364;&#178; <br />                                         end chromosomes (Gilson Geli, 2007; Smogorzewska Lange, 2004; Verdun <br />                                         Karlseder, 2007). Second, telomeres protect chromosome ends nucleolytic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         degradation, unscheduled DNA repair activation DNA damage checkpoints.                                         activity, termed telomere capping, essential cell viability genomic integrity <br />                                         (Feldser et al., 2003; Gilson Geli, 2007; Verdun Karlseder, 2007). <br />  <br />                                         identify genes modulate telomere capping, screened viable gene deletion <br />                                         collection Saccharomyces cerevisiae suppressors temperature sensitivity                                         cdc13-1 (Downey et al., 2006), allele telomere capping protein Cdc13. screen <br />                                         identified Cgi121 protein promotes telomere uncapping triggered cdc13-1. <br />                                         Cgi121 member complex yeast termed KEOPS  EKC) complex <br />                                         (Downey et al., 2006; Kisseleva-Romanova et al., 2006). addition Cgi121,                                         recognizable domains motifs, KEOPS complex composed protein kinase, <br />                                         Bud32; putative ATPase, Kae1  ; proteins small size                                         detectable enzymatic function recognizable domains, Pcc1 Gon7. KEOPS <br />                                         complex conserved humans, exception Gon7 detected. <br />                                         Interestingly, human orthologues Pcc1 characterized cancer antigens, <br />                                         indicating upregulation KEOPS subunits associated cancer. <br />  <br />                                         addition promoting telomere uncapping, KEOPS complex profoundly affects <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         telomere length homeostasis required cell growth budding yeast.  <br />                                         deletion BUD32 GON7 results dramatic telomere shortening, drastic reduction                                         ability telomerase add new telomere sequences DNA double-strand breaks                                         severe growth defects. fact, component KEOPS promotes telomere uncapping <br />                                         elongation (Downey et al., 2006; Gatbonton et al., 2006; Shachar et al., 2008) indicating <br />                                         proteins function collectively unit. <br />  <br />                                         parallel, KEOPS/EKC complex identified novel transcription factor <br />                                         (Kisseleva-Romanova et al., 2006). Components KEOPS nuclear proteins                                         associate chromatin vivo important inducible gene transcription <br />                                         (Kisseleva-Romanova et al., 2006). clear KEOPS promotes transcription                                         KEOPS mutant yeast strains defective recruitment coactivators, SAGA <br />                                         Mediator, target genes. Collectively, studies indicate KEOPS promotes                                         access nucleases telomerase chromosome end access                                         transcriptional complexes chromatin. model explains seemingly disparate <br />                                         observations KEOPS regulates accessibility chromatin, modulating                                         higher order structure. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Remarkably, KEOPS component Kae1 universally conserved domains                                         life. Kae1 homolog present smallest bacterial genome, Mycoplasma genitalium, <br />                                         smallest archaeal genome, Nanoarchaeum equitans, essential gene                                         bacterial species tested (Arigoni et al., 1998). Kae1 ATPase ASKHA (acetate <span id='am-27' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; <br />                                         sugar kinase/Hsp70/actin) superfamily. proteins superfamily carry diverse <br />                                         functions, clear specific biochemical function Kae1 possesses                                         Pasteurella haemolytica Kae1 act O-sialoglycoprotein protease (Abdullah et al., <br />                                         1991) Pyrococcus abyssi Kae1 proposed apurinic (AP) endonuclease (Hecker <br />                                         et al., 2007). functions dramatically different, conserved function                                         Kae1 remains open question. subunits KEOPS identified                                         bacteria.  archaea, nearly complete KEOPS complex identified                                          . observation suggests Kae1-interacting proteins evolved                                         regulatory infrastructure modulates activity Kae1. Given importance <br />                                         KEOPS telomere biology gene transcription, evidence KEOPS  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                           Page 3 <br />  <br />  <br />                                           evolutionarily conserved molecular machine, determined atomic structure KEOPS <br />                                           complexes reconstituted archaeal proteins probed functional significance <br />                                           vitro vivo. findings validate importance KEOPS architecture <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           discovered provide framework deciphering functional relationships <br />                                           KEOPS subunits. <br />  <br />                               RESULTS <br />                               Architecture KEOPS complex <br />                                           identified homologs Pcc1 Cgi121 archaea iterative BLAST searches <br />                                           (Figures S3 &amp; S4). did identify Gon7 homolog archaea human <br />                                           genome (data shown). determined underlying architecture KEOPS <br />                                           complex using pull experiments recombinant forms archaeal Kae1 (residues <br />                                           1-328 Methanococcus jannaschii MJ1130); Pcc1 (Pyrococcus furiosus PF2011); Cgi121 <br />                                           (M. jannaschii MJ0187); Bud32 (residues 328-535 MJ1130). protein                                           expressed GST fusion Escherichia coli purified near homogeneity. GST <br />                                           tags used immobilize proteins <span id='am-34' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-35' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span>glutathione</span>-Sepharose resin cleaved TEV <br />                                           protease, followed purification, isolate untagged protein. GST-Pcc1 immobilized                                           <span id='am-76' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-77' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span>glutathione</span> resin selectively retained Kae1, Bud32 (Figure 1A) Cgi121 (Figure <br />                                           1B). contrast, GST-Cgi121 retained Bud32 Kae1 (Figure 1C) GST-Kae1 <br />                                           retained Bud32 (Figure 1D). Lastly, Bud32 Cgi121 bound immobilized GST-Pcc1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           presence Kae1 (Figure 1B) Kae1 bound immobilized GST-Cgi121                                           presence Bud32 (Figure 1C). data indicate KEOPS organized                                           linear fashion Cgi121 binds Bud32, binds Kae1, turn binds                                           Pcc1. model depicting architecture KEOPS complex shown Figure 1E. <br />  <br />                               X-ray NMR structure determination <br />                                           linear topology KEOPS enabled crystallize solve binary ternary <br />                                           subcomplexes X-ray crystallography means develop atomic model                                           KEOPS complex. solved total structures using X-ray NMR methods                                            Table 1 &amp; S1 data collection refinement statistics). Structure based sequence <br />                                           alignments Bud32, Kae1, Pcc1, Cgi121 shown Figures S1&#226;&#8364;&#8220;5. composite <br />                                           model monomer KEOPS complex generated subcomplex structures                                           shown Figure 1F. <br />  <br />                                           following sections structure functional characterizations                                           individual KEOPS subunits followed characterizations binary interacting <br />                                           pairs KEOPS subunits. clarity, residue numbering corresponds M. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           jannaschii proteins  Pcc1, numbers P. furiosus),  appropriate, <br />                                           residue numbers corresponding S. cerevisiae ortholog shown parentheses.                                           Table S3 Figures S1&#226;&#8364;&#8220;5 residue equivalence. <br />  <br />                               Individual subunit analyses <br />                                            Bud32 structure function crystal structure Bud32 reveals bilobal <br />                                           architecture, characteristic eukaryotic protein kinases, major atypical features <br />                                           (Figure 2A). N-lobe canonical protein kinases consists minimally stranded <br />                                           &#206;&#178;-sheet flanking helix &#206;&#177;C. features conserved primary (Figure S1)                                           atomic structures Bud32 (Figure 2A) exception strand &#206;&#178;1, present                                           disordered different crystal complexes analyzed. C-lobe Bud32 unusually <br />                                           small, consisting &#206;&#178;-strands (&#206;&#178;7 &amp; &#206;&#178;8) &#206;&#177;-helices (&#206;&#177;D, &#206;&#177;E, &#206;&#177;F, &#206;&#177;J &amp; &#206;&#177;R)                                           (&#206;&#177;D, &#206;&#177;E &amp; &#206;&#177;F) direct equivalents canonical protein kinase <br />                                           structures (Figure 2A). Strikingly, C-lobe Bud32 lacks infrastructure normally <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                             Page 4 <br />  <br />  <br />                                         responsible phospho-<span id='am-81' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-82' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span>acceptor</span> site recognition, including activation segment   <br />                                         loop), substituted direct link strand &#206;&#178;8 helix &#206;&#177;F (Figure 2A). <br />                                         Despite atypical structure, Bud32 retains conserved elements required <span id='am-51' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span>ATP</span> binding <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         catalysis (Figure 2A, Table S2) auto-phosphorylate vitro (Figure 2B) <br />                                         (Facchin et al., 2002) manner dependent integrity conserved catalytic <br />                                         elements. <br />  <br />                                         closest structural homologues Bud32 RIO kinases (Rio1 Rio2) (Figure <br />                                         2C, panel). Rio kinases exhibit abbreviated C-lobe absence  <br />                                         loop  comparison PKA, loop, Figure 2C, panel).                                         question protein kinase recognizes phosphorylates substrate absence <br />                                         loop remains open question. <br />  <br />                                         test biological importance unique features identified archaeal KEOPS structures, <br />                                         mutated residues budding yeast counterpart identified structure based sequence <br />                                         alignments. mutants introduced S. cerevisiae strains low copy <br />                                         plasmids context chromosomal copy KEOPS component deleted. <br />                                         assessed biological readouts KEOPS function: cell growth telomere length. <br />                                         Consistent Bud32 competent phospho-transfer vitro, ATP binding                                         catalytic residues required biological function yeast [Figure 2DE, Table S3 &amp; <br />                                         (Downey et al., 2006)].  catalytic cleft mutants K360A (K52A), D467A <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         (D182A), N456A (N166A), D451A (D161A), D451R (D161R), N456A/D467A (N166A/ <br />                                         D182) result cell growth defects telomere shortening. severity                                         phenotypes imparted BUD32 mutations parallel  parallel loss <br />                                         protein kinase activity vitro caused equivalent mutation archaeal Bud32 <br />                                         (Figure 2B). Interestingly, kinase inactivating mutants displayed phenotypes                                         strong bud32&#206;&#8221;, indicating phosphotransfer activity Bud32 sole <br />                                         function. Finally, catalytic cleft mutations tested destroy protein structure                                         mutant binds archaeal Cgi121 vitro (Figure S6A) mutant Bud32 expression levels                                         yeast similar wild type levels (Figure S6B). <br />  <br />                                         ii. Kae1 structure function solved Kae1 structures corresponding                                         species (M. jannaschi, T. acidophilum) different protein complexes (Kae1-Bud32, <br />                                         Kae1-Bud32-Cgi121 &amp; Kae1-Pcc1) <span id='am-62' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-63' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-69' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span>. structures Kae1                                         similar published structure P. furiosus Kae1 (PDB 2IVO, 2IVP <br />                                         2IVN) (1.20 &#195;&#8230;, 1.17 &#195;&#8230; &amp; 1.20 &#195;&#8230; C&#206;&#177; rmsd respectively). structure isolated <br />                                         Kae1 subunit described previously (Hecker et al., 2007), features                                         relevant study. Kae1 belongs actin-like ATPase domain superfamily                                         proteins, known ASKHA superfamily. ASKHA fold serves broad <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         functions: 1) conformational hydrolases (e.g. Hsp70, actin) 2) metabolite kinases <br />                                         (e.g. glycerol kinase). ASKHA fold consists subdomains (denoted &amp; II)                                         highly similar RNase H-like folds (Hurley, 1996). fold subdomain                                         characterized central strand &#206;&#178;-sheet flanking &#206;&#177;-helices. active site resides <br />                                         cleft region subdomains, sandwiched &#206;&#178;-sheets. ATP <br />                                         binding dependent extensive protein contacts <span id='am-46' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-47' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span>adenine</span> ring sugar. ATPase <br />                                         catalytic function dependent bound metal ions. Ribbons schematic representations <br />                                         Figures 3AB summarize conserved nucleotide metal <span id='am-21' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> binding infrastructure                                         Kae1. <br />  <br />                                         specific function ASKHA family members attributable presence                                         unique inserts core fold (Hurley, 1996). Kae1 possess unique inserts (denoted <br />                                         specific inserts 1 &amp; 2) comprise surface exposed &#206;&#177;-helices (&#206;&#177;4, &#206;&#177;5 &amp; &#206;&#177;6)                                         long intervening linker &#206;&#177;5 &#206;&#177;6 insert, C-terminal tail <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                             Page 5 <br />  <br />  <br />                                         adopting extended conformation interspersed short 310 helices second <br />                                         insert (Figure 3C). C-terminal tail threads catalytic cleft Kae1. Comparison <br />                                         Kae1 ASKHA family members, kinase hydrolase function, did NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         insight specific function. <br />  <br />                                         begin characterization Kae1, purified wild type site-directed mutants M. <br />                                         jannaschii Kae1 assessed ATP-binding, ATPase activity, DNA binding AP <br />                                         endonuclease activity. wild type Kae1 binds ATP Kd 27 &#206;&#188;M (Table <br />                                         S3), intracellular ATP levels. detected low reproducible <br />                                         ATPase activity affected KEOPS subunits (data shown)                                         depended presence metal ion binding residues (Figure S7A). did detect <br />                                         apurinic (AP) endonuclease DNA binding activity (Figure S7B&#226;&#8364;&#8220;D). Mutation residues <br />                                         coordinate ATP, metal ion, perturbed ATP-binding function Kae1 <br />                                         (Table S3). Kae1 mutations destroyed protein fold mutant proteins <br />                                         bind Pcc1 vitro (Figure S7E). assess contribution ATP- metal ion- <br />                                         binding Kae1 biological function, kae1 mutant yeast strains perturbed                                         function  kae1 mutants perturbed ATP-binding vitro disrupted                                         putative metal ion binding site, resulted phenocopy kae1&#206;&#8221; mutation yeast [near <br />                                         absence cell growth (Figure S7F)] despite wild type protein expression levels (Figure <br />                                         S6C). effect yeast growth precluded analysis telomere length <br />                                         phenotype.  able culture ATP-binding mutant (E176R)  NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         predicted, mutant short telomeres (Figure 3D). <br />  <br />                                         iii. Pcc1 structure function Pcc1 structure solved complex <br />                                         Kae1. Pcc1 structure consists stranded anti-parallel &#206;&#178;-sheet packed                                         &#206;&#177; helices face, solvent exposed (Figure 4A panel). fold <br />                                         shared 700 functionally diverse proteins (DALI score &gt;2.0). close structural <br />                                         homologue Pcc1 KH domain  score=6.7), functions versatile RNA, <br />                                         ssDNA RNA hairpin binder (Figure S8A). ability KH domain bind nucleic <br />                                         acids dependent signature motif (GXXG loop), located kink helix &#206;&#177;1 <br />                                         (Valverde et al., 2008), notably absent Pcc1 (Figure S8A). <br />  <br />                                         Pcc1 forms tightly interdigitated dimer contact area 555 &#195;&#8230;2 (1110 &#195;&#8230;2 buried <br />                                         surface area) (Figure 4AB). Dimerization involves anti-parallel arrangement                                         monomers giving rise continuous stranded anti-parallel &#206;&#178;-sheet packed &#206;&#177; <br />                                         helices face. Interestingly, mode dimerization widely used KH <br />                                         domains (Figure S8A).  Pcc1 dimerizes solution, concentrations tested <br />                                         (Figure S8B). Equilibrium centrifugation analysis purified Kae1-Pcc1 complex revealed <br />                                         2:2 binding stochiometry (Figure S8C), suggesting Pcc1 functions dimerization <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         module presence KEOPS subunits. attempts disrupt Pcc1 homo- <br />                                         dimerization mutation yielded insoluble protein (data shown), suggesting Pcc1                                         obligate dimer, notion supported conserved hydrophobic nature dimer <br />                                         interface (Figure 4B &amp; S9A). <br />  <br />                                         iv. Cgi121 structure solved structures human Cgi121                                         archaeal Cgi121 complex Bud32 Kae1 (Figures 4C &amp; 6A).                                         human archaeal Cgi121 structures similar (2.0&#195;&#8230; C&#206;&#177; rmsd), consisting                                         central anti-parallel &#206;&#178;-sheet abutted &#206;&#177; helices face. surveying                                         PDB, identified close structural ortholog P. furiosus (PDB 1ZD0). Aside                                         protein, did identify proteins similar fold. <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                              Page 6 <br />  <br />  <br />                               Kae1-Bud32 interaction <br />                                          Binding Bud32 Kae1 involves contact surface 1583 &#195;&#8230;2 (3167 &#195;&#8230;2 buried surface) <br />                                          (Figure 5A) highly conserved Kae1 Bud32 orthologs (Figure S9BC &amp; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          S10A). Bud32 engages Kae1 using forward projecting surfaces N- C- <br />                                          terminal lobes, clamps subdomain II Kae1. Overall, binding mode                                          Bud32 Kae1 observed study similar Kae1-Bud32 structure recently <br />                                          reported Hecker et al. (2008) (PDB 2VWB, 1.75 &#195;&#8230; C&#206;&#177; rmsd). Binding Bud32 does                                          greatly alter conformation Kae1 (1.2 &#195;&#8230; C&#206;&#177; rmsd) exception &#206;&#178;3-&#206;&#177;1 <br />                                          linker. reverse assessed, Bud32 structure absence <br />                                          Kae1. <br />  <br />                                          N-lobe structural elements Bud32 involved Kae1 binding include C-terminal <br />                                          end helix &#206;&#177;C linker strand &#206;&#178;3 (Figure 5B). C-lobe structural elements involved <br />                                          Kae1 binding include &#206;&#177;E-&#206;&#178;7 linker containing catalytic loop, helix &#206;&#177;F, helix &#206;&#177;R <br />                                          extended C-terminal tail. Kae1 structural elements involved Bud32 binding <br />                                          include &#206;&#178;3&#226;&#8364;&#178; strand, extended linker sequence helices &#206;&#177;5 &#206;&#177;6                                           Kae1-specific insert 1), helix &#206;&#177;1&#226;&#8364;&#178;, helix &#206;&#177;2&#226;&#8364;&#178;. contacts subdomain Kae1 <br />                                          mediated C-terminal tail Bud32 interacts C-terminal region                                          helix &#206;&#177;1 &#206;&#178;4-&#206;&#177;2 linker. <br />  <br />                                          Bud32-Kae1 interface composed mixture hydrophobic, electrostatic NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          hydrogen bonding interactions. Interesting interaction clusters include salt network <br />                                          Bud32 residues R380 K365 Kae1 residues E236 E233, Bud32 <br />                                          K489 Kae1 E148, catalytic base Bud32, D451, Kae1 R287. <br />                                          Hydrophobic interactions include network involving V486 V482 Bud32 L150 <br />                                          K193 Kae1 (Figure 5B panel). Conspicuously, R534 C-terminal tail                                          Bud32 forms hydrogen bond N132 Kae1 immediately adjacent metal ion <br />                                          binding site (Figure 5B panel). R534 interaction correlates large <br />                                          displacement &#206;&#178;3-&#206;&#177;1 linker subdomain Kae1, giving rise partial opening                                          ATP binding cleft reduction inter-subdomain contacts (Figure 5B                                          panel). <br />  <br />                                          assess functional significance Bud32-Kae1 interaction, probed                                          interaction surfaces site-directed mutagenesis. K365E, R380E, V486Y, K489E <br />                                          single site mutations Bud32 E233R E236K single site mutations Kae1 <br />                                          markedly reduced Bud32-Kae1 interaction vitro (Figure 5C) perturbing <br />                                          binding Cgi121 Pcc1 (Figures 5D). contrast, deletion C-terminal tail                                          Bud32 impact Bud32-Kae1 interaction (Figure S11A).                                          equivalent mutations yeast Kae1 Bud32 test functional relevance vivo. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          shown Figure S10B S6C, Kae1 E233R (E292R) E236K (E295K) mutations <br />                                          severely impaired growth budding yeast adversely affecting protein expression. <br />                                          Similarly, Bud32 single site mutations K489E (R204E), V486Y (V201Y) double <br />                                          site mutations K365E/R380E (K57E/R72E) impaired Bud32 function vivo (Figure S10C) <br />                                          adversely affecting protein expression (Figure S6B). results indicate                                          binding Bud32 Kae1 critical KEOPS function, conclusion reached                                          Hecker et al. (2008). catalytic base D451 (D161) Bud32 lies Kae1 <br />                                          contact surface, perturbation binding Kae1 explain potent vivo <br />                                          effect caused D451R (D161R) mutant relative Bud32 active site mutants <br />                                          (Figure 2D). Consistent notion, combination mutants targeting Bud32 <br />                                          catalytic residues contact residues Kae1 pronounced phenotypic effects <br />                                          approaching severity BUD32 deletion (Figure S10E, Table S3) adversely <br />                                          affecting protein expression (Figure S6B). Finally, tested importance Bud32 C- <br />                                          terminal tail biological function. Despite fact deletion tail did affect <br />  <br />  <br />                                                 Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                             Page 7 <br />  <br />  <br />                                          Bud32 binding Kae1 vitro, deletion yeast Bud32 severe growth defect <br />                                          (Figure S6B, Figure S11B). result suggests possible role Bud32 tail                                          regulating Kae1 function. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Kae1 substrate Bud32 <br />                                          Kae1 binds close catalytic cleft Bud32, explored possibility Kae1 <br />                                          substrate Bud32.  Bud32 phosphorylate Kae1 vitro (Figure S11C). <br />                                          Secondly, phosphorylation Kae1 Bud32 functionally relevant, acceptor sites                                          Kae1 conserved Kae1 proteins species containing Bud32 ortholog.                                          11 conserved S/T residues, T308 T316 M. jannaschii Kae1 localize Kae1- <br />                                          specific insert 2 (Figure S11D). insert notable meandering path                                          catalytic cleft Kae1 conservation archaeal eukaryotic Kae1 <br />                                          orthologs bacterial counterparts lack Bud32 orthologue (Figure S5).                                          extended surface exposed nature Kae1 insert 2 leaves accessible                                          active site Bud32. contrast, remaining 9 conserved S/T sites deeply buried <br />                                          fold Kae1 integrated secondary structure elements. <br />  <br />                                          Bud32 regulates Kae1 function phosphorylation sites Kae1-specific insert <br />                                          2, deleting insert pronounced effect vivo. Supporting idea, <br />                                          partial (1-372) deletion (1-364) C-terminal tail yeast Kae1 caused severe <br />                                          growth defect (Figure S11E).  T308A (T369A) T316A (T377A) mutant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          showed similar phenotype, adversely affecting protein expression (Figure S6C). <br />                                          result consistent notion T308 (T369) Kae1 Bud32 phospho- <br />                                          regulatory site.  data support possibility Kae1 biologically <br />                                          relevant substrate Bud32 phospho-acceptor site lies Kae1-specific <br />                                          insert 2.  deletion Kae1-specific insert 2 did abolish Bud32 <br />                                          phosphorylation Kae1 vitro (data shown), implying residues                                          targeted Bud32 vitro. <br />  <br />                               Bud32-Cgi121 interaction <br />                                          binding mode Cgi121 Bud32 revealed Bud32-Cgi121-Kae1 ternary <br />                                          structure (Figures 6A &amp; S12A). Cgi121 binds exclusively N-lobe Bud32,                                          surface composed helix &#206;&#177;C, adjacent strand &#206;&#178;4 &#206;&#178;2-&#206;&#178;3 linker (Figure 6B).                                          reciprocal binding surface Cgi121 composed helices &#206;&#177;2 &#206;&#177;8. total contact <br />                                          surface 700 &#195;&#8230;2 (1400 &#195;&#8230;2) relative inter-subunit contact surfaces                                          KEOPS complex, conserved (Figure S9CD). <br />  <br />                                          Cgi121-Bud32 interface composed combination hydrophobic, hydrogen <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          bonding, electrostatic interactions (Figure 6B). Hydrophobic interactions consist                                          clusters. formed L353 Y352 tip Bud32 &#206;&#178;2-&#206;&#178;3 loop, L134 <br />                                          Cgi121 helix &#206;&#177;8. second formed A389 V404 Bud32 A138 L139 <br />                                          Cgi121. Salt interactions link K392 Bud32 D135 Cgi121, D403 Bud32 <br />                                          K46 Cgi121. Lastly, Y352 Bud32 hydrogen bonds H46 Cgi121, K46 <br />                                          Cgi121 hydrogen bonds backbone carbonyl D403 Bud32. <br />  <br />                                          inter-subunit contacts characterized, Bud32 displays largest conformational <br />                                          change binding Cgi121 (1.38 &#195;&#8230; C&#206;&#177; rmsd comparison 0.9 &#195;&#8230; C&#206;&#177; rmsd Kae1). <br />                                          apparent conformational change consists 10 &#195;&#8230; shift &#206;&#178;2-&#206;&#178;3 loop (Figure <br />                                          S12B). shift appears stabilized involvement Y352 Bud32                                          hydrophobic hydrogen bond interactions Cgi121. N-lobe elements                                          Bud32 including helix &#206;&#177;C entire C-lobe appear minimally affected Cgi121 <br />                                          binding. <br />  <br />  <br />                                                 Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                              Page 8 <br />  <br />  <br />                               Cgi121 stimulates Bud32 activity <br />                                           examined impact Cgi121 mutations Cgi121-Bud32 interaction.                                           L134E, L139R A138R single site mutations effect Bud32 binding vitro <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           (Figure 6C) L134E/A138R double mutation perturbed binding severely. confirmed <br />                                           NMR analysis overall fold Cgi121 affected double mutation <br />                                           (Figure S12C). dispersed nature HSQC spectra produced double mutant <br />                                           consistent folded state close correspondence HSQC spectra <br />                                           produced wild type mutant Cgi121 proteins consistent absence gross <br />                                           structural changes. <br />  <br />                                           binding Cgi121 Bud32 N-lobe site centered helix &#206;&#177;C reminiscent                                           intermolecular mode Cdk (cyclin-dependent protein kinase) regulation cyclins <br />                                           (Jeffrey et al., 1995). case Cdks, binding cyclins reorients helix &#206;&#177;C                                           productive position supports catalysis. position &#206;&#177;C Bud32                                           outwardly affected Cgi121, large conformational changes &#206;&#178;2-&#206;&#178;3 loop apparent. <br />                                           led test Cgi121 influences catalytic activity Bud32. Cgi121 potent <br />                                           activator (&gt;20 fold) Bud32 auto-phosphorylation vitro (Figures 2B 6D).                                           addition, activator function dependent ability Cgi121 bind Bud32,                                           Cgi121 L134E/A138R double mutant did stimulate Bud32 kinase activity (Figure <br />                                           6D). investigate impact loss binding Cgi121 Bud32 vivo,                                           analogous double mutation (L172E/I176R) yeast Cgi121. shown Figure <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           6E, Cgi121 (L172E/I176R) strain displayed slow growth phenotype range                                           cgi121&#206;&#8221; mutation, strongly suggesting binding Bud32 central Cgi121&#226;&#8364;&#8482;s <br />                                           biological function. <br />  <br />                                           role Cgi121 regulating Bud32 protein kinase activity consistent                                           observation phenotypes imparted cgi121&#206;&#8221; mutation milder                                           caused bud32&#206;&#8221; kae1&#206;&#8221; mutations (Downey et al., 2006), closely resemble                                           phenotype Bud32 catalytic site mutants.  Cgi121 expressed lower <br />                                           levels members KEOPS (Ghaemmaghami et al., 2003). observation                                           reminiscent cyclin-Cdk paradigm activity Cdks regulated                                           proteosome-mediated destruction cyclins, Cdk levels remain fixed. similarity <br />                                           led test yeast Cgi121 Bud32 unstable proteins GAL1 promoter shut                                           experiments. observed Cgi121 highly unstable half life ~30 min                                           Bud32 remained stable experiment (Figure 6F). Consistent possibility <br />                                           binding Cgi121 Bud32 enhances stability vivo, expression level                                           Cgi121 (L172E/I176R) mutant unexpectedly low relative wild type protein <br />                                           (Figure S6D).  data suggest Cgi121 function positive <br />                                           regulator Bud32 function. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Kae1-Pcc1 interaction <br />                                           unable crystallize 2:2 stochiometric complex Pcc1 Kae1                                           serendipitously crystallized solved structure 2:1 complex resulting                                           purification Kae1 presence excess Pcc1 (Figure 7A). Pcc1 Kae1                                           undergo significant conformational changes response complex formation (0.73&#195;&#8230; &amp; <br />                                           1.20&#195;&#8230; C&#206;&#177; rmsd, respectively). Kae1-Pcc1 binding interface extensive (contact <br />                                           area: 922 &#195;&#8230;2; buried area: 1844 &#195;&#8230;2) conserved (Figure S9AB) Kae1-Bud32 <br />                                           binding interface. <br />  <br />                                           Kae1 binds exclusively monomer Pcc1 dimer. Kae1 helices &#206;&#177;1 &#206;&#177;2 pack <br />                                           helices &#206;&#177;1 &#206;&#177;2 Pcc1, giving rise tightly packed helix bundle (Figure <br />                                           S13A). Additional contacts Kae1 310-1 helix Pcc1 helix &#206;&#177;2. <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                            Page 9 <br />  <br />  <br />                                           binding interactions, primarily hydrophobic nature, augmented small number <br />                                           salt hydrogen bonds (Figure S13A). majority hydrophobic interactions involve <br />                                           small chain residues, including T89, A90, T93 Kae1 A75, V79, A26 Pcc1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           allowing close approach participating &#206;&#177; helices. Non-hydrophobic contacts <br />                                           include salt interaction E30 Pcc1 R92 Kae1. <br />  <br />                                           Consistent Pcc1 functioning dimerization module (Figure S8C), fully <br />                                           accessible surfaces Kae1 binding presented Pcc1 dimer structure. <br />                                           Superposition second Kae1 molecule free binding surface Pcc1 revealed <br />                                           contacts involving subdomains II juxtaposed Kae1 molecules (Figure S13B).                                           reason contacts accommodated small changes inter-subunit <br />                                           closure angle Kae1 ASKHA fold yield 2:2 stochiometric complex observed <br />                                           solution (Figure S8C). Extension modeling studies include Bud32 Cgi121                                           fully dimeric KEOPS complex revealed prohibitory contacts involving Cgi121                                           Bud32 subunits (Figure S13C). <br />  <br />                               Functional characterizations Kae1-Pcc1 interaction <br />                                           Introduction Kae1 T88R mutation, combination T92R R91E, <br />                                           perturbed binding Kae1 Pcc1, Bud32, vitro (Figures 7B). Mutation                                           small hydrophobic contact residues Pcc1 large polar residues (Pcc1 A75Y V79R), <br />                                           impaired binding Kae1, double mutant A75Y/V79R pronounced <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           effect (Figure 7C). charge reversal mutation E30R Pcc1 disrupted binding Kae1. <br />                                           Pcc1 mutations did destroy protein fold mutant easily purified (Figure <br />                                           7D) eluted behaved dimer purification (data shown). Analogous <br />                                           mutations yeast PCC1 KAE1 counterparts introduced                                           temperature sensitive pcc1-4 yeast strain (Kisseleva-Romanova et al., 2006). Pcc1 <br />                                           V79R (A83R) A75Y (T79R) mutations disrupt binding Kae1 vitro did                                           rescue pcc1-4 temperature sensitivity phenotype wild type Pcc1 did (Figure 7E, <br />                                           Table S3). Similarly, T88R (I123R) R91E (R126A) mutations Kae1 resulted                                           severe loss function vivo (Figure 7F). Note R91E (R126A) Kae1 mutant                                           expressed yeast (Figure S6C). data indicate Pcc1-Kae1 interaction                                           required biological function. <br />  <br />                                           test functional requirement Pcc1 dimerization vivo, Pcc1- <br />                                           Pcc1 fusion. close proximity N- C-termini adjacent protomers Pcc1 <br />                                           homodimer suggested Pcc1-Pcc1 fusion readily form intramolecular dimers.                                           observed wild type Pcc1, Pcc1-Pcc1 wild type fusion rescued pcc1-4 temperature <br />                                           sensitive growth phenotype (Figure S13D). Consistent need functional copies <br />                                           Kae1 binding surface Pcc1 dimer, disruption binding sites NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Pcc1-Pcc1 fusion introduction A75R (T79R), V79R (A83R) A75R (T79R)/A75R <br />                                           (T79R) mutations abolished ability Pcc1-Pcc1 fusion rescue pcc1-4 growth <br />                                           defect restrictive temperature (Figure S13D). data consistent Pcc1 <br />                                           functioning dimer suggests KEOPS holocomplex acts dimer                                           vivo.  yeast KEOPS subunits Kae1, Gon7, Pcc1, Cgi121 Bud32  <br />                                           migrate complex apparent MW ~300,000 Da (Downey et al., 2006); (Downey et <br />                                           al., 2006; Kisseleva-Romanova et al., 2006), larger predicted <br />                                           monomer mass 116,872 Da closely matches predicted dimer mass 233,744 <br />                                           Da. <br />  <br />                               DISCUSSION <br />                                           KEOPS complex ancient, protein kinase-containing molecular machine critical                                           telomere function gene transcription budding yeast. work provides framework <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                            Page 10 <br />  <br />  <br />                                           depth functional biochemical characterization providing atomic model                                           complex. report catalytic activity KEOPS subunits, Bud32 <br />                                           (protein kinase) Kae1 (ATPase), inter-subunit interaction surfaces, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           important overall function KEOPS complex vivo. challenge                                           future identify substrate(s) KEOPS order unravel molecular <br />                                           machine exerts profound impact diverse aspects cell biology. <br />  <br />                               Kae1: universal ASKHA-type ATPase <br />                                           Kae1 minimal gene set organisms. eukaryotes, nuclear encoded Kae1 <br />                                           paralog (Qri7 S. cerevisiae) localized mitochondria, itself bacterial symbiont <br />                                           (Ghaemmaghami et al., 2003; Reinders et al., 2006), underscoring ancient critical <br />                                           role Kae1 play. ASKHA type ATPases function primarily conformational <br />                                           hydrolases metabolite kinases.  Kae1 proposed glycoprotease <br />                                           AP endonuclease, strikingly different biochemical functions. studies M. <br />                                           jannaschii Kae1 revealed ATPase activity DNA binding AP endonuclease <br />                                           activities.  exact function Kae1 remains mysterious. <br />  <br />                               Bud32 regulator Kae1 <br />                                           striking archaeal species produce single protein encoding Bud32 Kae1, <br />                                           indicating intimate relationship proteins. Kae1 conserved species, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           tempting speculate Bud32  members KEOPS) evolved                                           modulate Kae1 function.  rule Kae1 modulates Bud32                                           Kae1 Bud32 collaborate modify substrates. fact, Kae1 recently proposed <br />                                           inhibitor Bud32 kinase activity (Hecker et al. 2008).  work <br />                                           suggests means Bud32 regulate Kae1, phosphorylating Kae1,                                           second binding Kae1. Dual regulation Kae1 Bud32 experimentally supported                                           observation bud32&#206;&#8221; mutation phenotypically severe disruption                                           Bud32 catalytic activity binding Kae1. fact, close phenocopy                                           bud32&#206;&#8221; mutation bud32 mutants occurs catalysis binding Kae1 <br />                                           impaired (Figure S10E, Table S3). <br />  <br />                                           ATPases ASKHA fold superfamily regulated intermolecular binding <br />                                           partners (Hurley, 1996; Mayer Bukau, 2005). example, low ATPase activity                                           E. coli chaperone DnaK enhanced &gt;1000 fold binding <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> substrate                                           chaperone regulator DnaJ (Liberek et al., 1991). Additionally, catalytic cycle                                           DnaK ASKHA family ATPases limited <span id='am-15' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-16' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span>ADP</span> exchange. necessitates                                           action exchange factors GrpE, case DnaK, opens nucleotide <br />                                           binding cleft (Harrison et al., 1997). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           does binding Bud32 influence Kae1 catalytic function? C-terminal tail                                           Bud32 binds catalytic cleft Kae1 directly adjacent metal ion coordinating <br />                                           center. leaves Bud32 tail placed influence ATPase activity binding                                           biological substrate. Alternatively Bud32 induced movement &#206;&#178;3-&#206;&#177;1 linker Kae1 <br />                                           creates open catalytic cleft potentiate nucleotide exchange. Investigating <br />                                           possibilities awaits better understanding biochemical function Kae1                                           key players Kae1&#226;&#8364;&#8482;s catalytic cycle.  detect Kae1 ATPase activity vitro, <br />                                           low levels suggest key factor missing reconstitution assays. <br />  <br />                               Bud32 Rio kinase: living protein kinase fossils? <br />                                           protein kinase superfamily diverse group enzymes phospho-transfer activity <br />                                           carried evolutionary conserved bilobal structure. Bud32, Rio <br />                                           paralogs, form minimal set protein kinases conserved archaeal eukaryotic <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                               Page 11 <br />  <br />  <br />                                           species (Kannan et al., 2007). general feature Rio Bud32 protein kinases                                           minimal C-lobe absence loop. role C-lobe loop substrate <br />                                           recognition raises question Rio Bud32 kinases recognize substrates. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           suggest higher order binding mode revealed Bud32-Kae1 structure                                           reflect unconventional kinase substrate targeting mechanism, involving                                           recognition tertiary fold substrate simple linear peptide motif.                                           approach substrate recognition employed infrequently canonical eukaryotic protein <br />                                           kinases visualized atomic resolution eIF2&#206;&#177; protein kinase PKR <br />                                           (Dar et al., 2005) splicing related kinase SRPK1 (Ngo et al., 2008).                                           kinase-substrate complex notably similar Bud32-Kae1 complex substrate <br />                                           SRPK1 binds face kinase domain manner bridges                                           kinase lobes. <br />  <br />                                           work identified Cgi121 positive regulator Bud32 kinase activity.                                           conclusion based stimulatory effect Cgi121 binding Bud32 Bud32 <br />                                           autocatalytic activity.  yeast Cgi121 substochiometric component KEOPS <br />                                           abundance regulated protein degradation. Genetic analysis CGI121                                           budding yeast consistent positive regulator KEOPS. posit                                           identification cellular conditions Cgi121 stabilized provide important <br />                                           clue core function KEOPS complex. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Pcc1: dimerization subunit <br />                                           study identifies Pcc1 dimeric KH domain-like protein. Highlighting functional <br />                                           importance Pcc1 KEOPS complex, deletion PCC1 yeast results phenotypes <br />                                           severe BUD32 KAE1 deletions (Kisseleva-Romanova et al., 2006). <br />                                            ability Pcc1 dimer bind Kae1 molecules needed                                           function vivo. data consistent studies place Pcc1 center <br />                                           dimeric KEOPS holocomplex. KEOPS function dimer remains open <br />                                           question. answer question KEOPS processive molecular <br />                                           machine.  dimerization multimerization common theme processive <br />                                           enzymes [e.g. helicases, kinesins, ubiquitin ligases polymerases (Lohman et al., 1998; <br />                                           Tang et al., 2007)] perform multiple catalytic cycles single substrate.                                           identification KEOPS substrate(s) provide invaluable tool decipher inner <br />                                           workings complex, regulation biological function. <br />  <br />                               EXPERIMENTAL PROCEDURES <br />                               Bud32 vitro kinase assays <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Bud32 purified vitro hexahistidine tag fusion protein (pProEx-HTa)                                           concentrated buffer containing 10 mM HEPES pH 7.5, 300 mM NaCl 2 mM <br />                                           dithiothreitol (DTT). Kinase assays performed using [&#206;&#179;-32P]-ATP, 30 &#206;&#188;M cold ATP, <br />                                           20 &#206;&#188;M MnCl2 0.2&#226;&#8364;&#8220;1 &#206;&#188;M Bud32 60&#194;&#176;C. Reactions resolved sodium <br />                                           dodecylsulfate polyacrylamide gel electrophoresis (SDS-PAGE) visualized                                           PhosphorImager (Molecular Dynamics). <br />  <br />                               GST pull assays <br />                                           GST-tagged Cgi121, Kae1, Pcc1 proteins expressed Escherichia coli BL21 (DE3) <br />                                           Codon Plus cells lysed buffer containing 20 mM HEPES pH 7.5, 300 mM NaCl                                           2 mM DTT. Clarified lysate mixed glutathione-Sepharose resin 20&#194;&#176;C                                           allowed bind 30 min. 15 &#206;&#188;l aliquots resin bound fusion proteins incubated  <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                                Page 12 <br />  <br />  <br />                                           molar equivalents purified prey proteins 30 min, washed 6 times using 100X bed <br />                                           volumes buffer resolved SDS-PAGE. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Plasmids <br />                                           description plasmids used Supplementary Experimental <br />                                           Procedures. <br />  <br />                               Telomere Length Determination <br />                                           Telomere length assays performed previously described (Downey et al., 2006). <br />  <br />                               Yeast Strains <br />                                           Table S4 list yeast strains. Generation yeast strains study                                           performed using standard genetic techniques. <br />  <br />                               Protein Expression, Crystallization Data Collection <br />                                           proteins described study purified using conventional affinity tag purification <br />                                           techniques  glutathione-Sepharose nickel chelate chromatography). Affinity                                           tags removed TEV protease cleavage indicated. Seleno-methionine substituted <br />                                           proteins prepared described previously (Tang et al., 2007). Heavy atom <br />                                           derivatizations carried soaking crystals mother liquor solutions containing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           saturated [Ta6Br12]Br2. x-ray diffraction experiments, crystals flash frozen                                           mother liquors supplemented 20&#226;&#8364;&#8220;35% (v/v) glycerol. x-ray data sets processed <br />                                           using software XDS (Kabsch, 1993). <br />  <br />                                           Cgi121 isolated structures length Homo sapiens Methanococcus <br />                                           jannaschii Cgi121 proteins expressed E. coli BL21 (DE3) CodonPlus cells                                           glutathione S-transferase (GST) fusion protein using pGEX2T plasmid modified                                           affinity tag cleavage Tobacco Etch Virus (TEV) protease. H. sapiens CGi121 <br />                                           crystals space group P62  b=93.56&#195;&#8230;, c=87.16&#195;&#8230;, &#206;&#177;=&#206;&#178;=90&#194;&#176;, &#206;&#179;=120&#194;&#176;)                                           molecules asymmetric unit obtained using hanging drop method mixing <br />                                           equal volumes 15 mg/mL protein solution 20% (w/v) PEG1500, 0.2 M calcium <br />                                           acetate, 0.1 M imidazole pH 8 4&#194;&#176;C. selenium SAD data set collected NE- <br />                                           CAT beamline 24-<span id='am-75' about='oboInOwl:id' typeof='owl:Thing'>ID</span>-C. NMR analyses M. jannaschii Cgi121 performed using                                           Bruker AVANCE 600 800 spectrometer 298K. NMR samples contained 1.6 <br />                                           mM 15N, 13C-labeled M. jannaschii CGI121 20 mM sodium phosphate pH 6.3 100 <br />                                           mM NaCl 10% D2O. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Kae1-Bud32 complex Methanococcus jannaschii protein MJ1130 (Kae1-Bud32 <br />                                           fusion) internal deletion amino acids (MJ1130 &#206;&#8221;R332-K333) expressed <br />                                           purified N-terminal GST-fusion protein human Cgi121. Crystals                                           space group P4122  b=148.73&#195;&#8230;, c=136.42&#195;&#8230;, &#206;&#177;=&#206;&#178;=&#206;&#179;=90&#194;&#176;) molecules                                           asymmetric unit obtained using hanging drop method mixing equal volumes <br />                                           6 mg/ml protein solution 10 mM HEPES pH 7.5, 10% PEG3000 11% <span id='am-10' about='obo:IMR_0200092' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0200092" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Spermine" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00750" datatype="xsd:string"></span>spermine</span> <br />                                           tetrahydrochloride 20&#194;&#176;C. tantalum SAD data set collected NE-CAT beamline <br />                                           24-ID-C. <br />  <br />                                           Kae1-Bud32-Cgi121 complex 1:1 molar complex M. jannaschii Cgi121                                           MJ1130 obtained mixing excess molar Cgi121 MJ1130 followed size <br />                                           exclusion chromatography remove excess Cgi121. Crystals space group P212121 <br />                                            76.99&#195;&#8230;, b=106.91&#195;&#8230;, c=209.99&#195;&#8230;, &#206;&#177;=&#206;&#178;=&#206;&#179;=90&#194;&#176;) 1:1 complexes Cgi121                                           MJ1130 asymmetric unit obtained using hanging drop method mixing <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                            Page 13 <br />  <br />  <br />                                           equal volumes 8 mg/mL protein complex 0.1 M MES pH 6.5 15% PEG3000. <br />                                           tantalum SAD data set collected NE-CAT beamline 24-ID-C. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Pcc1 structure length Pyrococcus furiosus Pcc1 protein (pfuPcc1) expressed <br />                                           purified N-terminal GST-fusion protein human Cgi121. Crystals space <br />                                           group P63  b=78.81&#195;&#8230;, c=60.02&#195;&#8230;, &#206;&#177;=&#206;&#178;=90&#194;&#176;, &#206;&#179;=120&#194;&#176;) molecule asymmetric <br />                                           unit obtained using hanging drop method mixing equal volumes 5 mg/mL <br />                                           protein solution 45% PEG300, 0.1 M Na/K phosphate pH 6.1 20&#194;&#176;C. selenium <br />                                           SAD data set collected CHESS East - F2 beamline processed using software <br />                                           XDS (Kabsch, 1993). <br />  <br />                                           Kae1-Pcc1 complex Thermoplasma acidophilum Kae1 protein (taKae1)                                           expressed purified N-terminal GST-fusion protein human Cgi121. 2:1 <br />                                           molar complex taKae1-pfuPcc1 obtained mixing 20 fold molar excess pfuPcc1 <br />                                           taKae1 followed size exclusion chromatography remove excess Pcc1. Crystals                                           space group P65  b=66.56&#195;&#8230;, c=435.56&#195;&#8230;, &#206;&#177;=&#206;&#178;=&#206;&#179;=90&#194;&#176;) 2:1 complexes                                           Pcc1:Kae1, respectively, asymmetric unit obtained using hanging drop <br />                                           method mixing equal volumes 5 mg/mL protein complex 0.2 M NaCl, 40% <br />                                           PEG300, 0.1 M HEPES pH 7.5 20&#194;&#176;C. native data set collected NE-CAT <br />                                           beamline 24-ID-E. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               X-ray Structure Determinations <br />                                           experimental phasing, heavy atoms positions located using SHELXD (Schneider <br />                                           Sheldrick, 2002) refined using SHARP/AUTOSHARP (Vonrhein et al., 2007). <br />                                           RESOLVE (Terwilliger, 2004) used density modification model autobuilding. <br />                                           Coot (Emsley Cowtan, 2004) used manual building electron density map <br />                                           inspection. Refinement carried using Refmac (Winn et al., 2003) Phenix (Zwart <br />                                           et al., 2008). HsCgi121 PfuPcc1 crystal structures solved selenium <br />                                           SAD method. MJ1130 crystal structure solved tantalum SAD method.                                           structure MjCgi121/MJ1130 complex solved combination tantalum SAD <br />                                           method molecular replacement using Phaser (Storoni et al., 2004).                                           resolution MjCgi121/MJ1130 complex analysis low (3.6 &#195;&#8230;) comparison                                           structure analyses study, high quality experimentally phased electron <br />                                           density maps (Figure S6A) conjunction knowledge M. jannaschii Cgi121 <br />                                           NMR structure allowed unambiguously model position Cgi121 respect                                           Bud32. <br />  <br />                                           crystal structure TaKae1-PfuPcc1 complex solved molecular replacement <br />                                           using aforementioned Kae1 Pcc1 structures search models Phaser (Storoni et <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           al., 2004). <br />  <br />                               MjCGI121 NMR resonance assignment structure calculation <br />                                           3D spectra measured non-uniform sampling (NUS) indirect <br />                                           dimensions  30% sampling) reconstructed using multidimensional <br />                                           decomposition (MDD) using MDDNMR (Jaravine Orekhov, 2006). spectra                                           processed using NMRPipe/NMRDraw (Delaglio et al., 1995) analyzed using Sparky <br />                                           (Goddard T. D.). Backbone assignments obtained using standard triple-resonance <br />                                           experiments (Bax Grzesiek, 1993). Aliphatic chain assignments obtained                                           3D HCCH-TOCSY aromatic chain assignments 2D (H&#206;&#178;)C&#206;&#178;(C&#206;&#179;C&#206;&#180;)H&#206;&#180;                                           (H&#206;&#178;)C&#206;&#178;(C&#206;&#179;C&#206;&#180;C&#206;&#181;)H&#206;&#181; experiments (Yamazaki et al., 1993). Distance restraints, derived                                           15N- 13C-edited NOESY-HSQC (using cross-relaxation mixing time 100 ms)                                           dihedral angle restraints, predicted chemical shifts software TALOS <br />  <br />  <br />                                                  Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                                        Page 14 <br />  <br />  <br />                                         (Cornilescu et al., 1999; Jaravine Orekhov, 2006), used input combined <br />                                         automated NOE assignment structure calculation program CYANA (Guntert, <br />                                         2004). Residual dipolar couplings (RDCs) gained difference NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         1HN-15N splittings 2D IPAP-1H-15N HSQC experiments measured isotropic <br />                                         anisotropic conditions (Ottiger et al., 1998). Anisotropic media prepared                                         addition filamentous phages concentration 10 mg/mL (Hansen et al., 1998; <br />                                         Zweckstetter Bax, 2001). final 20 structures lowest target function                                         final CYANA cycle used refinement presence 122 HN-RDCs                                         explicit solvent (Linge et al., 2003). Structure statistics 10 lowest energy structures <br />                                         refinement shown Table S1. <br />  <br />                               Supplementary Material <br />                                         Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                         thank Steve Bell help identifying archaeal KEOPS orthologs Domenico Libri strains/plasmids. <br />                                         Research conducted NE-CAT beamlines (Advanced Photon Source) supported National Center <br />                                         Research Resources (NIH, award RR 15301) U.S. Department Energy (Office Basic Energy <br />                                         Sciences, contract AC02-06CH11357). DD Canada Research Chair (Tier II). work funded                                         grants CIHR (MOP 79441) DD Canadian Cancer Society FS (Grant 017220). DYLM DN <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         supported NCIC CIHR post-doctoral fellowships, respectively. SR supported postdoctoral <br />                                         fellowship German Academic Exchange Service (DAAD). <br />  <br />                                         Coordinates deposited PDB (accession codes 3EN9, 3ENH, 3ENP, 3ENC, 3ENO, 2K8Y). <br />  <br />  <br />                               References <br />                                         Abdullah KM, Lo RY, Mellors  Cloning, nucleotide sequence, expression Pasteurella <br />                                           haemolytica A1 glycoprotease gene. J Bacteriol. 1991; 173:5597&#226;&#8364;&#8220;5603. [PubMed: 1885539] <br />                                         Arigoni F, Talabot F, Peitsch M, Edgerton MD, Meldrum E, Allet E, Fish R, Jamotte T, Curchod ML, <br />                                           Loferer H. genome-based approach identification essential bacterial genes. Nat <br />                                           Biotechnol. 1998; 16:851&#226;&#8364;&#8220;856. [PubMed: 9743119] <br />                                         Bax  Grzesiek S. Methodological Advances Protein Nmr. Accounts Chemical Research. 1993; <br />                                           26:131&#226;&#8364;&#8220;138. <br />                                         Cornilescu G, Delaglio F, Bax  Protein backbone angle restraints searching database                                           chemical shift sequence homology. J Biomol NMR. 1999; 13:289&#226;&#8364;&#8220;302. [PubMed: 10212987] <br />                                         Dar AC, Dever TE, Sicheri F. Higher-order substrate recognition eIF2alpha RNA-dependent <br />                                           protein kinase PKR. Cell. 2005; 122:887&#226;&#8364;&#8220;900. [PubMed: 16179258] <br />                                         Delaglio F, Grzesiek S, Vuister GW, Zhu G, Pfeifer J, Bax  NMRPipe: multidimensional spectral <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           processing based UNIX pipes. J Biomol NMR. 1995; 6:277&#226;&#8364;&#8220;293. [PubMed: 8520220] <br />                                         Downey M, Houlsworth R, Maringele L, Rollie  Brehme M, Galicia S, Guillard S, Partington M, <br />                                           Zubko MK, Krogan NJ, et al. genome-wide screen identifies evolutionarily conserved <br />                                           KEOPS complex telomere regulator. Cell. 2006; 124:1155&#226;&#8364;&#8220;1168. [PubMed: 16564010] <br />                                         Emsley P, Cowtan K. Coot: model-building tools molecular graphics. Acta crystallographica. 2004; <br />                                           60:2126&#226;&#8364;&#8220;2132. <br />                                         Facchin S, Lopreiato R, Stocchetto S, Arrigoni G, Cesaro L, Marin O, Carignani G, Pinna LA. <br />                                           Structure-function analysis yeast piD261/Bud32, atypical protein kinase essential normal <br />                                           cell life. Biochem J. 2002; 364:457&#226;&#8364;&#8220;463. [PubMed: 12023889] <br />                                         Feldser DM, Hackett JA, Greider CW. Telomere dysfunction initiation genome instability. <br />                                             Nat Rev Cancer. 2003; 3:623&#226;&#8364;&#8220;627. [PubMed: 12894250] <br />                                         Gatbonton T, Imbesi M, Nelson M, Akey JM, Ruderfer DM, Kruglyak L, Simon JA, Bedalov  <br />                                             Telomere length quantitative trait: genome-wide survey genetic mapping telomere <br />                                             length-control genes yeast. PLoS Genet. 2006; 2:e35. [PubMed: 16552446] <br />  <br />  <br />  <br />                                                 Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                                  Page 15 <br />  <br />  <br />                                         Ghaemmaghami S, Huh WK, Bower K, Howson RW, Belle  Dephoure N, O&#226;&#8364;&#8482;Shea EK, Weissman <br />                                              JS. Global analysis protein expression yeast. Nature. 2003; 425:737&#226;&#8364;&#8220;741. [PubMed: <br />                                              14562106] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Gilson E, Geli V. telomeres replicated. Nat Rev Mol Cell Biol. 2007; 8:825&#226;&#8364;&#8220;838. [PubMed: <br />                                              17885666] <br />                                         Goddard, TD.; KDG. SPARKY. Vol. 3. University California; San Francisco: <br />                                         Guntert P. Automated NMR structure calculation CYANA. Methods Mol Biol. 2004; 278:353&#226;&#8364;&#8220; <br />                                              378. [PubMed: 15318003] <br />                                         Hansen MR, Mueller L, Pardi  Tunable alignment macromolecules filamentous phage yields <br />                                              dipolar coupling interactions. Nature Structural Biology. 1998; 5:1065&#226;&#8364;&#8220;1074. <br />                                         Harrison CJ, Hayer-Hartl M, Di Liberto M, Hartl F, Kuriyan J. Crystal structure nucleotide <br />                                              exchange factor GrpE bound ATPase domain molecular chaperone DnaK. Science. <br />                                              1997; 276:431&#226;&#8364;&#8220;435. [PubMed: 9103205] <br />                                         Hecker  Leulliot N, Gadelle D, Graille M, Justome  Dorlet P, Brochier C, Quevillon-Cheruel S, <br />                                              Le Cam E, van Tilbeurgh H, et al. archaeal orthologue universal protein Kae1 <span id='am-70' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-71' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Iron" datatype="xsd:string"></span>iron</span> <br />                                              metalloprotein exhibits atypical DNA-binding properties apurinic-endonuclease activity <br />                                              vitro. Nucleic Acids Res. 2007; 35:6042&#226;&#8364;&#8220;6051. [PubMed: 17766251] <br />                                         Hurley JH. sugar kinase/heat shock protein 70/actin superfamily: implications conserved <br />                                              structure mechanism. Annu Rev Biophys Biomol Struct. 1996; 25:137&#226;&#8364;&#8220;162. [PubMed: <br />                                              8800467] <br />                                         Jaravine VA, Orekhov VY. Targeted acquisition real-time NMR spectroscopy. Journal NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              American Chemical Society. 2006; 128:13421&#226;&#8364;&#8220;13426. [PubMed: 17031954] <br />                                         Jeffrey PD, Russo AA, Polyak K, Gibbs E, Hurwitz J, Massague J, Pavletich NP. Mechanism CDK <br />                                              activation revealed structure cyclinA-CDK2 complex. Nature. 1995; 376:313&#226;&#8364;&#8220;320. <br />                                              [PubMed: 7630397] <br />                                         Kabsch W. Automatic processing rotation diffraction data crystals initially unknown <br />                                              symmetry cell constants. Journal Applied Crystallography. 1993; 26:795&#226;&#8364;&#8220;800. <br />                                         Kannan N, Taylor SS, Zhai Y, Venter JC, Manning G. Structural functional diversity                                              microbial kinome. PLoS Biol. 2007; 5:e17. [PubMed: 17355172] <br />                                         Kisseleva-Romanova E, Lopreiato R, Baudin-Baillieu  Rousselle JC, Ilan L, Hofmann K, Namane <br />                                               Mann C, Libri D. Yeast homolog cancer-testis antigen defines new transcription <br />                                              complex. Embo J. 2006; 25:3576&#226;&#8364;&#8220;3585. [PubMed: 16874308] <br />                                         Liberek K, Marszalek J, Ang D, Georgopoulos C, Zylicz M. Escherichia coli DnaJ GrpE heat <br />                                              shock proteins jointly stimulate ATPase activity DnaK. Proc Natl Acad Sci U S  1991; <br />                                              88:2874&#226;&#8364;&#8220;2878. [PubMed: 1826368] <br />                                         Linge JP, Williams MA, Spronk CA, Bonvin  Nilges M. Refinement protein structures                                              explicit solvent. Proteins. 2003; 50:496&#226;&#8364;&#8220;506. [PubMed: 12557191] <br />                                         Lohman TM, Thorn K, Vale RD. Staying track: common features DNA helicases                                              microtubule motors. Cell. 1998; 93:9&#226;&#8364;&#8220;12. [PubMed: 9546385] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Mayer MP, Bukau B. Hsp70 chaperones: cellular functions molecular mechanism. Cell Mol Life <br />                                              Sci. 2005; 62:670&#226;&#8364;&#8220;684. [PubMed: 15770419] <br />                                         Ngo JC, Giang K, Chakrabarti S, Ma CT, Huynh N, Hagopian JC, Dorrestein PC, Fu XD, Adams JA, <br />                                              Ghosh G. sliding docking interaction essential sequential processive phosphorylation <br />                                              SR protein SRPK1. Mol Cell. 2008; 29:563&#226;&#8364;&#8220;576. [PubMed: 18342604] <br />                                         Ottiger M, Delaglio F, Bax  Measurement J dipolar couplings simplified  <br />                                              dimensional NMR spectra. Journal Magnetic Resonance. 1998; 131:373&#226;&#8364;&#8220;378. [PubMed: <br />                                              9571116] <br />                                         Reinders J, Zahedi RP, Pfanner N, Meisinger C, Sickmann  complete yeast <br />                                              mitochondrial proteome: multidimensional separation techniques mitochondrial proteomics. J <br />                                              Proteome Res. 2006; 5:1543&#226;&#8364;&#8220;1554. [PubMed: 16823961] <br />                                         Schneider TR, Sheldrick GM. Substructure solution SHELXD. Acta crystallographica. 2002; <br />                                              58:1772&#226;&#8364;&#8220;1779. <br />                                         Shachar R, Ungar L, Kupiec M, Ruppin E, Sharan R. systems-level approach mapping                                              telomere length maintenance gene circuitry. Mol Syst Biol. 2008; 4:172. [PubMed: 18319724] <br />  <br />  <br />                                                 Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                               Page 16 <br />  <br />  <br />                                         Smogorzewska  Lange T. Regulation telomerase telomeric proteins. Annu Rev Biochem. <br />                                             2004; 73:177&#226;&#8364;&#8220;208. [PubMed: 15189140] <br />                                         Storoni LC, McCoy AJ, Read RJ. Likelihood-enhanced fast rotation functions. Acta crystallographica. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             2004; 60:432&#226;&#8364;&#8220;438. <br />                                         Tang X, Orlicky S, Lin  Willems  Neculai D, Ceccarelli D, Mercurio F, Shilton BH, Sicheri F, <br />                                             Tyers M. Suprafacial orientation SCFCdc4 dimer accommodates multiple geometries                                             substrate ubiquitination. Cell. 2007; 129:1165&#226;&#8364;&#8220;1176. [PubMed: 17574027] <br />                                         Terwilliger T. SOLVE RESOLVE: automated structure solution, density modification model <br />                                             building. Journal synchrotron radiation. 2004; 11:49&#226;&#8364;&#8220;52. [PubMed: 14646132] <br />                                         Valverde R, Edwards L, Regan L. Structure function KH domains. Febs J. 2008; 275:2712&#226;&#8364;&#8220; <br />                                             2726. [PubMed: 18422648] <br />                                         Verdun  Karlseder J. Replication protection telomeres. Nature. 2007; 447:924&#226;&#8364;&#8220;931. <br />                                             [PubMed: 17581575] <br />                                         Vonrhein C, Blanc E, Roversi P, Bricogne G. Automated structure solution autoSHARP. Methods <br />                                             molecular biology (Clifton, NJ). 2007; 364:215&#226;&#8364;&#8220;230. <br />                                         Winn MD, Murshudov GN, Papiz MZ. Macromolecular TLS refinement REFMAC moderate <br />                                             resolutions. Methods enzymology. 2003; 374:300&#226;&#8364;&#8220;321. [PubMed: 14696379] <br />                                         Yamazaki T, Forman-Kay JD, Kay LE. dimensional NMR experiments correlating 13C-beta <br />                                             1H-delta/epsilon chemical shifts aromatic residues 13C-labeled proteins scalar <br />                                             couplings. J Chem Soc. 1993; 115:11054&#226;&#8364;&#8220;11055. <br />                                         Zwart PH, Afonine PV, Grosse-Kunstleve RW, Hung LW, Ioerger TR, McCoy AJ, McKee E, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Moriarty NW, Read RJ, Sacchettini JC, et al. Automated structure solution PHENIX <br />                                             suite. Methods molecular biology (Clifton, NJ). 2008; 426:419&#226;&#8364;&#8220;435. <br />                                         Zweckstetter M, Bax  Characterization molecular alignment aqueous suspensions Pf1 <br />                                             bacteriophage. Journal Biomolecular Nmr. 2001; 20:365&#226;&#8364;&#8220;377. [PubMed: 11563559] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                     Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 1. Architecture KEOPS complex <br />                                          D) GST pull experiments purified KEOPS subunits. Proteins separated <br />                                         SDS-PAGE visualized Coomassie blue staining. <br />                                         (E) Architectural model KEOPS complex derived  D). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         (F) Composite model KEOPS complex based X-ray crystal structures single <br />                                         subunits, binary complexes ternary complex summarized Table 1. <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                            Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 2. Structure catalytic function Bud32 <br />                                           Ribbons representation Bud32 showing important catalytic residues. figures <br />                                         tables, residues archaeal ortholog listed appropriate,                                         S. cerevisiae ortholog bracketed. Residue labels colored according effect                                         indicated mutations growth yeast  color legend). N-lobe Bud32 colored <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         light green C-lobe colored dark green. metal ion ATP modeled                                         catalytic cleft based structure RIO1 (PDB 1ZP9). <br />                                         (B) Autophosphorylation analysis Bud32 wild type mutant proteins presence                                         absence Cgi121. Assays carried [&#206;&#179;-32P]-ATP 60&#194;&#176;C stopped                                         indicated time points. Products reactions separated SDS-PAGE 32P <br />                                         incorporation detected autoradiography. <br />                                         (C) Ribbons comparison Bud32 Rio1 (PDB 1ZP9; panel) PKA (PDB 2CPK; <br />                                         panel). Bud32 colored   Rio kinase shown red unique N- <br />                                         terminal lobe insertion (labeled &#226;&#8364;&#732;Rio insert&#226;&#8364;&#8482;) orange. PKA colored yellow  <br />                                         loop shown red. <br />                                         (D) Effect Bud32 catalytic cleft mutations growth yeast. bud32&#206;&#8221; yeast strain <br />                                         transformed centromeric low copy vector expressing wild type BUD32,                                         indicated Bud32 mutant vector control. Liquid cultures strain  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                      Page 19 <br />  <br />  <br />                                         spotted minimal medium series 10 fold dilutions. Growth yeast 30&#194;&#176;C                                         monitored 6 day period. Representative data shown. summary doubling times <br />                                         liquid culture selection mutants provided Table S3. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         (E) Bud32 catalytic mutants short telomeres. Terminal restriction fragment (TRF) <br />                                         analysis yeast strains described (D). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                           Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 3. Structure catalytic function Kae1 <br />                                           Ribbons representation Kae1 highlighting catalytic cleft residues. Subdomains                                         II colored dark light blue respectively. metal ion ADP modeled                                         cleft based structure P. abyssi Kae1 (PDB 2IVP). <br />                                         (B) Schematic catalytic cleft residues Kae1 coordinate ATP metal ion (based <br />                                         PDB 2IVP). <br />                                         (C) Kae1 characteristic inserts (colored pink red) distinguish                                         ASKHA fold proteins. Secondary structure elements unique inserts labeled. <br />                                         (D) kae1-E176R (E213R) mutant short telomeres. TRF analysis performed                                         kae1&#206;&#8221; strain transformed low copy vector encoding wild type Kae1 E176 <br />                                         (E213R) mutant. Telomeres wild type, bud32&#206;&#8221; cgi121&#206;&#8221; strains used                                         reference. <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                     Page 21 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 4. Structures Pcc1 Cgi121 <br />                                           Ribbons representation Pcc1 monomer   dimer  . Pcc1 protomers <br />                                         1 2 colored light dark orange respectively. <br />                                         (B) Stereo view conserved hydrophobic contacts Pcc1 dimer interface. <br />                                         (C) Ribbons representation human Cgi121 (left) M.jannaschii Cgi121 (right) <br />                                         (rmsd=1.92 &#195;&#8230;). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                           Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 5. Binding mode Bud32 Kae1 <br />                                           Ribbons representation Bud32-Kae1 complex. Kae1 Bud32 colored                                         Figures 2A 3A. Interface residues mutation gives rise growth phenotypes                                         yeast shown stick representation. <br />                                         (B) Stereo zoom view Kae1-Bud32 binding interface centered   Kae1- <br />                                         specific insert 1   C-terminal tail region Bud32. Figures colored  . <br />                                         (C,D) Mutation Kae1-Bud32 binding interface. Binding experiments                                         indicated GST fusion baits prey proteins performed Figure 1. (C) <br />                                         Examination Bud32-Kae1 interaction. Lane 1 (lower panel) loading control                                         Bud32. (D) Examination Cgi121-Bud32 interaction using Bud32 mutants                                           Pcc1-Kae1 interaction using Kae1 mutants  . <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                       Page 23 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 6. Binding mode Cgi121 Bud32 <br />                                           Ribbons representation Cgi121-Bud32-Kae1 complex. Bud32, Kae1 Cgi121 <br />                                         colored Figures 2A, 3A 4B, respectively. Residues mutation gives rise <br />                                         strong phenotypes yeast highlighted. <br />                                         (B) Stereo zoom view Bud32-Cgi121 binding interface showing chains                                         participate interaction. <br />                                         (C) vitro binding experiments indicated GST fusion baits purified prey <br />                                         proteins. <br />                                         (D) Disruption Cgi121 binding Bud32 impairs ability Cgi121 potentiate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Bud32 autophosphorylation activity vitro. Bud32 kinase reactions performed                                         Figure 2B. <br />                                         (E) Functional analysis yeast Bud32-Cgi121 binding interface mutations cell <br />                                         growth. cgi121&#206;&#8221; yeast transformed vector, vector encoding wild type <br />                                         Cgi121 L134E/A138R (L172E/I176R) double mutant. strains plated                                         serial dilution minimal medium described Figure 2D. <br />                                         (F) Analysis Cgi121 Bud32 stability vivo. Yeast encoding galactose-inducible <br />                                         3xHA-Cgi121 Bud32-FLAG proteins grown 4 hrs media containing 2% <br />                                         glucose 2% galactose. Cells grown galactose washed resuspended 2% <br />                                         glucose-containing media repress GAL1 promoter. Cells harvested times <br />                                         following glucose addition. &#206;&#177;-HA &#206;&#177;-FLAG immunoblots shown. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                             Mao et al.                                                                                        Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 7. Binding mode Pcc1 Kae1 <br />                                           Ribbons representation binding interface Pcc1 Kae1. Pcc1 Kae1 <br />                                         colored Figures 3 4, respectively. Residues mutation gives rise                                         phenotype yeast highlighted. Figure S13A zoom view Pcc1-Kae1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         binding interface participating chains. <br />                                         (B&#226;&#8364;&#8220;D) Mutational analysis Pcc1-Kae1 binding interface. Binding experiments                                         indicated GST fusion baits prey proteins performed Figure 1. (B) <br />                                         Examination various Kae1 proteins binding Pcc1 (upper panel) Bud32 (lower <br />                                         panel). (C) Examination various Pcc1 proteins binding Kae1. (D) Input gel shows <br />                                         equivalent quantities soluble wild type mutant Pcc1 proteins used binding <br />                                         reactions (C). <br />                                         (E) Analysis Kae1-Pcc1 interface mutations yeast growth. Left panel: Serial dilutions <br />                                         thermosensitive pcc1-4 strains harboring plasmid encoding wild type Pcc1, pcc1 mutants <br />                                         vector grown 37&#194;&#176;C (non-permissive) 23&#194;&#176;C (permissive) 3&#226;&#8364;&#8220;5 days. <br />                                         Right panel: kae1&#206;&#8221; yeast harboring plasmid encoding wild type Kae1, Kae1 mutants                                         vector grown 3&#226;&#8364;&#8220;6 days. <br />  <br />  <br />  <br />  <br />                                                Mol Cell. Author manuscript; available PMC 2011 20. <br />                                                               NIH-PA Author Manuscript                                                  NIH-PA Author Manuscript                                                NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                               Table 1 <br />                                                              Data collection refinement statistics. <br />  <br />                                                                                            mjBud32/Kae1                   mjBud32/Kae1/mjCgi121                hsCgi121                      pfuPcc1                     taKae1/pfuPcc1 <br />                                                                                                                                                                                                                                                                          Mao et al. <br />  <br />  <br />  <br />  <br />                                                               Wavelength (&#195;&#8230;)               0.97888                        0.97888                              0.9793                        0.979                       0.97935 <br />                                                               Resolution (&#195;&#8230;)               2.67                           3.6                                  2.48                          2.63                        3.02 <br />                                                               Rsym (%)                     5.78 (22.86)                   7.59(18.38)                          5.6(31.17)      <br /> </body></html>