<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Rpp1, essential protein subunit nuclear RNase P required processing precursor tRNA 35S precursor <br /> rRNA Saccharomyces cerevisiae <br />  <br />  <br />  gene essential protein subunit nuclear RNase P Saccharomyces cerevisiae cloned. gene protein, RPP1, identified virtue homology human scleroderma <br /> autoimmune antigen, Rpp30, copurifies human RNase P. Epitope-tagged Rpp1 association RNase P RNA related endoribonuclease, RNase MRP RNA, immunoprecipitates crude extracts cells. Depletion Rpp1 vivo leads accumulation precursor tRNAs unprocessed 5* 3* termini reveals rRNA processing defects described previously proteins associated RNase P RNase MRP. Immunoprecipitated complexes cleave yeast precursor <br /> tRNAs precursor rRNAs. <br /> [Key Words: Rpp1; essential protein subunit; nuclear RNase P; S. cerevisiae; precursor tRNA; precursor rRNA] <br /> Received June 5, 1997; revised version accepted July 25, 1997. <br />  <br />  <br />  <br />  <br /> Ribonuclease P (RNase P) ubiquitous endoribonucle-              limited. RNAse P isolated human cells copurifies <br /> ase consists protein RNA subunits. cleaves            RNA (H1; 340), proteins&#226;&#8364;&#8221;Rpp14, <br /> 58-terminal leader sequences precursor tRNAs (Darr et             Rpp20, Rpp25, Rpp30, Rpp38, Rpp40 (Eder et al. <br /> al. 1992; Altman et al. 1993). Escherichia coli RNase P           1997). Genetic approaches Saccharomyces cerevisiae known process precursors small, meta-               identified essential proteins, Pop1, Pop3, biologically stable RNAs vivo, 4.5S RNA                   Pop4, associate RNase P RNA (RPR1); (Bothwell et al. 1976), 10Sa RNA (Komine et al. 1994),               proteins associate RNA (NME1) related polycistronic mRNA histidine operon (Ali-               endoribonuclease, RNase mitochondrial RNA process- <br /> fano et al. 1994), small RNAs encoded bac-               ing (MRP) (Schmitt Clayton 1992; Lygerou et al. <br /> teriophage (Bothwell et al. 1974; Hartmann et al. 1995).             1994; Dichtl Tollervey 1997; Chu et al. 1997). Analy- eubacteria, RNA component RNase P                 sis temperature-sensitive alleles proteins catalytic vitro (Guerrier-Takada et al. 1983). eu-            depletion proteins yeast cells shown bacterial protein subunit basic protein &#226;&#710;&#188;14 kD           role tRNA processing rRNA <br /> serves essential cofactor vivo enhancing             processing. suggested tRNA rRNA <br /> catalytic efficiency substrate range holoen-              processing coordinated (Pace Burgin 1990; Clay- <br /> zyme (Liu Altman 1994; Gopalan et al. 1997).                  ton 1994; Morrissey Tollervey 1995; Lee et al. 1996). <br /> contrast, RNA components archaeal eukary-                    eukaryotes, coordination tRNA rRNA pro- <br /> otic RNase P catalytically active vitro            cessing mediated activity related <br /> absence respective protein subunits, despite                enzymes, RNase P RNase MRP (Morrissey Tol- structural homology eubacterial RNAs (Alt-              lervey 1995; Chamberlain et al. 1996b). RNase P es- <br /> man et al. 1995; Haas et al. 1996).                                  sential biosynthesis tRNAs (Lee et al. 1991)    RNA subunit nuclear RNase P                   appears role rRNA processing yeast characterized variety eukaryotic organ-               (Chamberlain et al. 1996b). RNase MRP related isms (Altman et al. 1993; Tranguch Engelke 1993;                 RNase P structural similarities RNA <br /> Chamberlain et al. 1996a; Eder et al. 1996), information             component (Forster Altman 1990; Schmitt et al. <br /> regarding protein subunits eukaryotic RNase P              1993). suggested RNase P ancestor <br />                                                                      RNase MRP (Morrissey Tollervey 1995). RNase <br /> 3 <br />  Corresponding author.                                               MRP described originally endonuclease E-MAIL sidney.altman@qm.yale.edu; FAX (203) 432-5713.                cleaves RNA primers mitochondrial DNA replication <br />  <br />  <br /> 2926      GENES <span id='am-13' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>; DEVELOPMENT 11:2926&#226;&#8364;&#8220;2937 &#194;&#169; 1997 Cold Spring Harbor Laboratory Press ISSN 0890-9369/97 $5.00 <br />                                                                               essential protein subunit RNase P yeast <br />  <br />  <br />  <br /> (Chang Clayton 1987; Stohl Clayton 1992).         <span id='am-44' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-45' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-50' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> sequence identity human Rpp 30 (Fig. <br /> cently, role nuclear processing precursor rRNA     1A). yeast gene named RPP1 RNase P <br /> (prRNA) established (Lygerou et al. 1996a).         Protein 1. <br /> RNase MRP does cleave precursor tRNAs (ptRNAs)             address putative protein encoded vitro, depletion NME1 RNA does affect          RPP1 essential gene, disrupted gene  <br /> tRNA processing vivo (Schmitt Clayton 1993; Ly-       placing LEU2 gene  Materials Meth- <br /> gerou et al. 1996a).  proteins associated       ods). heterozygous RPP1/rpp1::LEU2 strain (VS161) RNase MRP depleted inactivated muta-        sporulated subsequent tetrad analysis showed tion yeast, cleavage ptRNAs blocked (Lygerou et     2:2 segregation cell viability (Fig. 1B). viable <br /> al. 1994; Chu et al. 1997; Dichtl Tollervey 1997).       spores Leu&#226;&#710;&#8217;, indicating wild-type <br />    learn functions nuclear RNase P      RPP1 allele.  RPP1 essential gene S. vivo, report cloning functional char-     cerevisiae. <br /> acterization essential protein (Rpp1, 32.2 kD) com- <br /> ponent S. cerevisiae RNase P. Rpp1 homologous                                                              Construction epitope-tagged allele RPP1 human scleroderma autoimmune antigen, Rpp30, identified recently protein copurifies   Epitope-tagged proteins useful study subunit human RNase P recognized sera       function interactions large holoenzyme com- <br /> patients autoimmune disease referred      plexes. determine Rpp1 associates <br /> Th antisera (Eder et al. 1997). identify novel func-   RPR1 RNA RNase P activity, epitope-tagged <br /> tions RNase P yeast, strain S. cerevisiae    RPP1 strain S. cerevisiae (VS162) constructed <br /> conditionally expresses Rpp1 protein constructed.        (Table 1). DNA fragment encodes copies Using strain strain contains epitope-     c-myc epitope (3 &#195;&#8212; myc; TerBush Novick 1996) tagged RPP1 gene, demonstrate role Rpp1          fused frame 38 initiator ATG codon RPP1 processing tRNA ribosomal RNA precursors. Deple-         low-copy-number plasmid (pRS316), pRS316::3 &#195;&#8212; myc&#226;&#8364;&#8220; <br /> tion Rpp1 protein revealed global defects rRNA         RPP1. resulting strain grew identical rates processing. Depletion inactivation proteins associ-    wild-type cells suggesting 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 allele <br /> ated RNase P RNase MRP affect subset       fully functional (Fig. 2A, lanes 2,4). Immunoblots rRNA processing events (Lygerou et al. 1994;        protein extracts 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 cells using anti- <br /> Chu et al. 1997; Dichtl Tollervey 1997). Based    myc antibody (9E10) detected polypeptide 36 kD&#226;&#8364;&#8221; <br /> observed defects rRNA processing, suggest pos-       size consistent predicted sible functional interaction RNase P RNase           3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 fusion protein (Fig. 2B, lanes 2,4). Wild- <br /> MRP, small nucleolar ribonucleoprotein (RNP)       type haploids (VS162A VS162C) lacking c-myc <br /> complexes (snoRNP; review, Filipowicz Kiss       tag contain 36-kD protein. results 1993; Fournier Maxwell 1993; Maxwell Fournier        myc epitope-tagged Rpp1 protein migrates 1995), required processing prRNA.            36-kD polypeptide fully functional. <br />  <br />  <br />                                                              RNase P RNase MRP RNAs coprecipitate Results <br />                                                              3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 fusion protein essential yeast gene encodes homolog                                                              Rpp1 associated RNase P RNA <br /> human scleroderma autoimmune antigen, Rpp30 <br />                                                              determined immunoprecipitation experiments used combination biochemical genetic stud-       (Fig. 3). Extracts 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 RPP1 strains <br /> ies human S. cerevisiae cells characterize       incubated anti-myc monoclonal antibody <br /> protein subunit eukaryotic RNase P. employed         (9E10) RNA extracted immunoprecipi- <br /> computational sequence search yeast genes       tates  Materials Methods). RNA 38 end- <br /> amino acid sequence similarity biochemically identi-      labeled [32P]pCp analyzed denaturing gel <br /> fied human RNase P protein subunits (Eder et al. 1997).      electrophoresis (Fig. 3A,B). Immunoprecipitatd RNA determine proteins associated hu-       analyzed Northern hybridization confirm man RNase P essential components required           identity labeled RNAs (Fig. 3C,D). Mature RNase catalytic function (Eder et al. 1997), searched   P RNA major RNA species 3 &#195;&#8212; myc&#226;&#8364;&#8220; <br /> homologs Rpp14, Rpp20, Rpp25, Rpp30, Rpp38,           Rpp1 precipitate 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 cells Rpp40 genome S. cerevisiae (Goffeau et al. 1996)   control cell lysates. putative precursors performing BLAST search (blastp tblastn algo-       RNase P RNA (Lee et al. 1991) RNase MRP RNA rithms; Altschul et al. 1990) S. cerevisiae genome    immunoprecipitates 3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 <br /> database (Cherry et al. 1996). human scleroderma         control immunoprecipitates. Approximately <br /> autoimmune antigen, Rpp30, highest amino acid        equal levels RNase P RNase MRP RNAs (RPR1 <br /> sequence similarity predicted sequence. previ-        NME1, respectively) end-labeled RNA <br /> ously uncharacterized open reading frame (ORF),              derived immunoprecipitates washed <br /> YHR062c, right arm chromosome VIII         buffer contained 150 mM KCl (Fig. 3A).  <br /> potential encode protein 32.2 kD shares 23%       RNase P RNA major RNA species detected <br />  <br />  <br />                                                                                          GENES &amp; DEVELOPMENT              2927 <br />  Stolc Altman <br />  <br />  <br />  <br />  <br /> Figure 1. RPP1, essential yeast gene <br /> encodes 32.2-kD protein ortholog human scleroderma autoimmune antigen, <br /> Rpp30.   Predicted amino acid sequence S. cerevisiae Rpp1. protein  <br /> coded ORF YHR062c chromosome <br /> VIII alignment human <br /> scleroderma autoimune antigen Rpp30 shown. amino acid sequences numbered methionine resi- protein. Identical amino acids shaded similar amino acids boxed. (B) heterozygous diploid strain <br /> VS161, RPP1/rpp1::LEU2, sporulated dissections performed 20 tet- <br /> rads. spores  D) derived tetrads shown vertical <br /> rows. <br />  <br />  <br />  38 end-labeling RNA 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 immunopre-               association Rpp1 RNase P activity cipitates washed buffer contained            demonstrated immunoprecipitation. Immuno- <br /> 600 mM KCl (Fig. 3B).  possible achieve       precipiated  resuspended) 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 pellets ac- significant separation enzymes, physi-         curately cleaved radiolabeled ptRNASer (Fig. 4) cally functionally (Lygerou et al. 1996a;            ptRNATyr (data shown) vitro. conclude .                                                          Rpp1 protein component  tightly associated <br />  <br />  <br />  <br /> Table 1.   Strains S. cerevisiae used study <br />  <br /> Strain                                                         Genotype <br />  <br /> JN161       MATa/MATa ade2-1/ade2-1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 his4-260/his4-260 thr1-4/thr1-4 lys2nNheI/lys2nNheI <br /> VS161       MATa/MATa ade2-1/ade2-1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 his4-260/his4-260 thr1-4/thr1-4 lys2nNheI/lys2nNheI <br />              RPP1/rpp1::LEU2/RPP1 <br /> VS162       MATa/MATa ade2-1/ade2-1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 his4-260/his4-260 thr1-4/thr1-4 lys2nNheI/lys2nNheI <br />              RPP1/rpp1::LEU2/RPP1 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162A      MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI RPP1 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162B      MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI rpp1::LEU2 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162C      MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI RPP1 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162D      MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI rpp1::LEU2 + pRS316&#226;&#8364;&#8220;3xmyc::RPP1 <br /> VS162A8     MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI RPP1 <br /> VS162C8     MATa ade2-1 leu2-3,112 ura3-1 his4-260 thr1-4 lys2nNheI RPP1 <br /> NY1060      MATa/MATa GAL1/GAL1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 <br /> VS163       MATa/MATa GAL1/GAL1 leu2-3,112/leu2-3,112 ura3-1/ura3-1 RPP1/rpp1::LEU2 <br /> VS164       MATa GAL1 leu2-3,112 ura3-1 + pYCpGAL::rpp1 (URA3) <br /> VS165       MATa GAL1 leu2-3,112 ura3-1 + pYCpGAL (URA3) <br />  <br />  <br />  <br /> 2928       GENES &amp; DEVELOPMENT <br />                                                                                       essential protein subunit RNase P yeast <br />  <br />  <br />   catalytically active RNase P holoenzyme. Indirect <br /> supporting evidence conclusion comes ex- <br /> periments human homolog Rpp1, Rpp30, shown separable active holoen- <br /> zyme extensive biochemical purification (Eder et al. <br /> 1997). <br />  <br />  <br /> Construction conditional lethal allele RPP1 <br /> RPP1 placed control GAL10 pro- <br /> moter, allows expression gene culture <br /> medium contains galactose suppresses expres- <br /> sion culture medium contains glucose.  <br /> sulting strain, rpp1::LEU2&#226;&#8364;&#8220;pGAL::rpp1 (VS164), compared phenotype control strain RPP1&#226;&#8364;&#8220; <br /> pGAL (VS165). liquid culture contained galac- <br /> tose, difference growth rate GAL::rpp1 strain wild-type RPP1 strain. cultures transferred medium contained <br /> glucose, cell growth continued initially doubling <br /> time 2 hr. 12&#226;&#8364;&#8220;16 hr, growth rate  <br />  <br />  <br />  <br />                                                                     Figure 3. RNA subunits RNase P (RPR1) RNase <br />                                                                     MRP (NME1) coprecipitate 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1.   Immuno- <br />                                                                     precipitated RNAs extracted 9E10 Ab&#226;&#8364;&#8220;IgG&#226;&#8364;&#8220;agarose <br />                                                                     beads incubated protein extracts                                                                     spores (VS162A8, VS162B, VS162C8, VS162D; lanes 1&#226;&#8364;&#8220;4,  <br />                                                                     spectively). RNA extracted immunoprecipitated <br />                                                                     beads washed 150 mM KCl  Materials                                                                     Methods). RNA 38 end-labeled [58-32P]pCp,                                                                     fractionated 8% polyacrylamide/7 M <span id='am-61' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span>urea</span> gel. (B) Immu- <br />                                                                     noprecipitated RNAs derived immunoprecipi- <br />                                                                     tated beads (lanes 1&#226;&#8364;&#8220;4, respectively)                                                                     immunoprecipitated beads washed 600 mM KCl prior <br />                                                                     38 end-labeling RNA. (C) Immunoprecipitated RNAs <br />                                                                     derived immunoprecipitated beads                                                                     transferred positively charged nylon membrane (Boehringer <br />                                                                     Mannheim) electroblotting hybridized uniformly <br />                                                                     labeled DNA probe complementary RPR1 gene  Ma- <br /> Figure 2. 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 functional recognized 9E10         terials Methods). (Lanes 1&#226;&#8364;&#8220;4) Immunoprecipitated RNA <br /> antibody.   spores (VS162 D; lanes 1&#226;&#8364;&#8220;4, respec-        spores VS162A8, VS162B, VS162C8, VS162D, respec- <br /> tively) derived diploid strain VS162, dissected    tively; (lane 5) RNA supernatant immunoprecipi- <br /> rich medium plates (YPAD), replica-plated plates      tated extract derived spore VS162A8 centrifugation contained synthetic complete medium lacked         beads; (lane 6) RNA supernatant immunoprecipi- <br /> leucine (leu) <span id='am-56' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span> (ura), contained 5-FOA. (B) Immu-   tated extract derived spore VS162B centrifugation noblot protein extracts prepared spores            beads; (lane 7) RNase P (RPR1) RNase MRP (NME1) RNAs <br /> VS162A8, VS162B, VS162C8, VS162D, derived dip-         (0.001 pmoles  transcribed vitro. small arrows <br /> loid strain VS162. spores isogenic          point larger species, precursors mature <br /> VS162A8 VS162C8 spores pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc&#226;&#8364;&#8220;          RNase P RNA, RPR1 (Lee et al. 1991). larger arrows indi- <br /> Rpp1 plasmid. (Lanes 1&#226;&#8364;&#8220;4) VS162A8, VS162B, VS162C8,             cate RNase P (RPR1) RNase MRP (NME1) RNAs. (D) VS162D, respectively. arrowhead points 36-kD             C, membrane hybridized uniformly <br /> 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 fusion protein. upper band nonspecifi-        labeled DNA probe complementary NME1 gene (Schmitt <br /> cally reacting protein.                                             Clayton 1992). <br />  <br />  <br />                                                                                                  GENES &amp; DEVELOPMENT              2929 <br />  Stolc Altman <br />  <br />  <br />  <br />                                                                   GAL::rpp1 strain declined rapidly little <br />                                                                   growth 16 hr (Fig. 5A). <br />  <br />  <br />                                                                   Rpp1 required processing ptRNA <br />                                                                   determine effects Rpp1 depletion bio- <br />                                                                   synthesis metabolically stable RNAs, total RNA                                                                   isolated GAL::rpp1 strain (VS164) RPP1 <br />                                                                   strain (VS165) various times growth glucose- <br />                                                                   containing medium. RNA samples VS164                                                                   compared VS165 denaturing gel <br />                                                                   stained ethidium bromide Northern hybrid- <br /> Figure 4. RNase P activity coprecipitates 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1.      ization. <br /> IgG&#226;&#8364;&#8220;agarose pellets, bound immunoprecipitated           effect Rpp1 depletion accumulation RNase P RNA, derived spores VS162A8, VS162B, VS162C8,        precursor tRNAs shown Figure 5B. RNAs VS162D  Fig. 3B), incubated uniformly       began accumulate 7 hr transfer glucose- <br /> labeled precursor tRNASer 30 min 37&#194;&#176;C, frac-      containing medium. time, culture growth <br /> tionated 8% polyacrylamide/7 M urea gel  Materials <br />                                                                   slowed. abundance RNAs increased Methods). (Lane 1) Precursor tRNASer; (lanes 2&#226;&#8364;&#8220;5) immuno- <br />                                                                   time sizes appropriate ptRNAs. precipitated RNase P spores VS162A8, VS162B, <br /> VS162C8, V6S162D, respectively; (lane 6) glycerol gradient-   abundance mature tRNAs decreased accordingly. <br /> purified human RNase P, fraction F29 (Eder et al. 1997). Arrows   Analysis Northern hybridization using probes <br /> indicate ptRNA, accurately processed mature tRNA, 58          complementary tRNALeu3 (Fig. 5C), intervening <br /> leader sequence.                                                  sequence (IVS) (data shown) 58 leader sequence <br />  <br />  <br />  <br />  <br /> Figure 5. Rpp1 required processing ptRNA 5.8S rRNA vivo.   Growth strain VS164 (j), wild-type, isogenic strain <br /> VS165 (h), transfer galactose-containing glucose-containing medium T = 0. Cell den- <br /> sity measured times indicated cultures diluted glucose-containing  <br /> dium time point prevent nutritional  <br /> privation maintain exponetial growth. (B) <br /> RNA extracted VS164 following growth glucose-containing medium indicated <br /> times, fractionated 8% polyacryl- <br /> amide/7 M urea gel, stained ethidium <br /> bromide. positions 5.8S (L) RNA, 5.8S (S) <br /> RNA, 5S RNA, ptRNA, mature tRNA  <br /> dicated. (C) Total RNA extracted VS164 VS165 transferred positively charged ny- <br /> lon membrane (Boehringer Mannheim) electro- <br /> blotting hybridized g-32P-labeled oligo- <br /> <span id='am-36' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-37' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-40' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> complementary mature tRNALeu <br />  Materials Methods). position processing intermediates ptRNA indicated. PT primary transcript, extra sequences termini contains intron; IVS ptRNA contains  <br /> tron processed 58 38 <br /> ends; 58 38 spliced ptRNA unpro- <br /> cessed termini. (D) Steady-state levels RNase P RNA (RPR1) RNase MRP RNA <br /> (NME1) VS164 transfer galactose- <br /> containing glucose-containing medium.  band RPR1 panel putative precur- <br /> sor RPR1 RNA extra sequences termini (Lee et al. 1991). U6 snRNA levels detected oligo U6 (Table 2), internal <br /> control RNA levels (Brow Guthrie 1990). <br />  <br />  <br /> 2930      GENES &amp; DEVELOPMENT <br />                                                                               essential protein subunit RNase P yeast <br />  <br />  <br />  pre-tRNALeu3 (data shown), showed accu-        mature precursor RPR1 RNAs NME1 RNA <br /> mulation tRNA unprocessed termini.       reduced (Lygerou et al. 1996a; Chu et al. 1997; Dichtl growth GAL::rpp1 strain glucose 12 hr,   Tollervey 1997). data suggest 58 38 unprocessed ptRNALeu3 accumulated ap-           Rpp1 correlated maturation stability proximately level mature tRNA.       RPR1 stability NME1 vivo. RNAs level pre-tRNALeu3 unspliced processed       large RNP complex, alternatively, ends reduced correspondingly. results      Rpp1 shared RNP particles showed sequentially ordered removal 58       vivo. <br /> leader sequence, 38 trailing sequence, finally intron ptRNALeu3, impaired Rpp1-depleted cells. phenotype observed RNase P RNA mu-         Rpp1 required processing 35S prRNA <br /> tants, rpr1 rpr1 (T315DT307) (Lee et al. 1991; Cham-     S. cerevisiae eukaryotes, rRNA tran- <br /> berlain et al. 1996b), conclude Rpp1 essen-    scribed 35S precursor RNA contains tial protein subunit catalytically active RNase P     sequences rRNA molecules (18S, <br /> complex vivo.                                             5.8S, 25&#226;&#8364;&#8220;28S). Subsequent processing nucleotide <br />                                                              modifications involving endonucleolytic exonucleo- <br />                                                              lytic cleavages methylation generate mature rRNA <br /> Rpp1 required accumulation RNase P                                                               review, Eichler Craig 1994; Venema Tol- <br /> RNase MRP RNAs <br />                                                              lervey 1995; Tollervey 1996). examined fidelity investigated effect Rpp1 depletion      rRNA processing Rpp1-depleted cells determine steady-state levels RNase P RNase MRP             defects pathway similar described <br /> RNAs ascertain cells lacking Rpp1 shared pheno-        previously mutants affect RNase P RNase <br /> typic traits previously described conditional lethal    MRP (Shuai Warner 1991; Lindahl et al. 1992; <br /> mutants Pop1, Pop3 Pop4, proteins associate      Schmitt Clayton 1993; Chamberlain et al. 1996b; RNPs. Depletion Rpp1 results decrease      Lygerou et al. 1996a; Chu et al. 1997; Dichtl Toller- steady-state levels RNase P RNase MRP          vey 1997). rRNAs analyzed gels stained RNAs (Fig. 5D).  RPR1 RNA precursor does         ethidium bromide Northern analysis oligo- appear decrease extent mature          nucleotide probes (Table 2) detect various prRNA spe- <br /> RPR1 RNA, 30 hr Rpp1 depletion.        cies  Figs. 5B 6). <br /> Pop4 depletion (Chu et al. 1997), mature RPR1 RNA            depletion Rpp1 protein, observed mul- detectable 21 hr Rpp1 depletion. contrast,   tiple defects rRNA processing A0 A1 sites <br /> depletion Pop3 does affect steady-state levels     58 external transcribed sequence (58 ETS), RNAs, pop1-1 steady-state levels         A2 A3 sites internal transcribed sequence <br />  <br />  <br />  <br /> Table 2.   Oligonucleotides used study <br />  <br /> Oligo   1: 58-CAGCAGAGAGACCCGA-38 <br /> Oligo   2: 58-ACTATCTTAAAAGAAGAAGC-38 <br /> Oligo   3: 58-GAATTACCACGGTTATACC-38 <br /> Oligo   4: 58-GCACAGAAATCTCTCACC-38 <br /> Oligo   5: 58-ATGAAAACTCCACAGTG-38 <br /> Oligo   6: 58-CCAGTTACGAAAATTCTTG-38 <br /> Oligo   7: 58-CGCATTTCGCTGCGTTCTTCATCG-38 <br /> Oligo   8: 58-ACAGAATGTTTGAGAAGGAAATG-38 <br /> Oligo   U6: 58-TCATCCTTATGCAGGG-38 <br /> Oligo   mtRNA: 58-GCATCTTACGATACCTG-38 <br /> Oligo   ltRNA: 58-CCAAACAACCACTTATTTGTTGA-38 <br /> Oligo   itRNA: 58-CACAGTTAACTGCGGTC-38 <br /> Oligo   T7MRP: 58-GGGAATTCGAAATTAATACGACTCACTATAGAATCCATGACCAAAGAATCGTCA-38 <br /> Oligo   38 MRP: 58-TCCCCCGGGTGAATCCATGGACCAAGA-38 <br /> Oligo   T7ITS14: 58-GGGAATTCGAAATTAATACGACTCACTATAGACACTGTGGAGTTTTCATATC-38 <br /> Oligo   38 ITS14: 58-GGGGATCCTTAAAATTTCCAGTTACGAAAATTC-38 <br /> Oligo   T7ITS16: 58-GGGAATTCGAAATTAATACGACTCACTATAGGCCAAACGGTGAGAGATTTCTG-38 <br /> Oligo   38 ITS16: 58-GGGGATCCACTTTAAGAACATTGTTCGCCT-38 <br /> Oligo   MYC5: 58-GCGGAATGCTGGAACAGAAACTTATT-38 <br /> Oligo   T7ITS16: 58-GGGAATTCGAAATTAATACGACTCACTATAGGCCAAACGGTGAGAGATTTCTG-38 <br /> Oligo   38 ITS16: 58-GGGGATCCACTTTAAGAACATTGTTCGCCT-38 <br /> Oligo   MYC3: 58-ACCAGCATTCCCAAATCTTCTTCAGA-38 <br /> Oligo   5 GAL30: 58-GGGGATCCGCATTATAGAACCGAGAATGC-38 <br /> Oligo   3 GAL30: 58-GGTCTAGAGATCCCCAAATTTTTTTGTCT-38 <br />  <br />  <br />  <br />                                                                                          GENES &amp; DEVELOPMENT              2931 <br />  Stolc Altman <br />  <br />  <br />  <br /> (ITS1), E C2 sites internal       5.8S(L) rRNA increased (Fig. 5B). altered ratio  <br /> transcribed sequence 2 (ITS2) (Fig. 6). Analysis low-       tween 5.8S(L) 5.8S(S) suggests defect cleavage molecular-weight RNA showed synthesis 5.8S(S)          A3 site ITS1 processing sites <br /> rRNA reduced Rpp1-depleted strain,           required maturation 5.8S rRNA vivo (Schmitt <br />  <br />  <br />  <br />  <br />                                          Figure 6.  facing page legend.) <br />  <br />  <br /> 2932      GENES &amp; DEVELOPMENT <br />                                                                                          essential protein subunit RNase P yeast <br />  <br />  <br />  Clayton 1993; Henry et al. 1994). Figure 6D shows accumulation 7S precursor 5.8S rRNA 7 hr Rpp1 depletion. precursor predicted 58 end A2 site ITS1 38 end 38 end 5.8S rRNA. prediction proven <br /> correct, able conclude defect cleavage A3 site conditions used. <br />    addition, large precursors 18S rRNA ac- <br /> cumulated later times Rpp1-depleted cells (Fig. <br /> 6C,D). 24S rRNA precursor contains 58 <br /> ETS ITS1 sequences shows defects cleavage A0, A1, A2, A3 processing sites, 21S rRNA <br /> precursor shows defects cleavage processing A2 A3 processing sites ITS1 E site ITS2 (Fig. 6C,D). 17S 12S rRNA degradation <br /> intermediates represent sequences fragmented 58 <br /> ends 18S rRNA (Fig. 6C,D; Allmang et al. <br /> 1996a). Figure 6E shows depletion 7S(S) 7S(L) <br /> precursors 5.8S rRNA, depletion 27SA 27SB <br /> precursors 25S rRNA, accumulation 8S <br /> rRNA precursor 5.8S rRNA, contains 58                    Figure 7. Processing prRNA Rpp1 immunoprecipitates. <br /> extended sequences ITS1 38 extended se-                        Position ITS141 rRNA substrate relative 35S <br /> quences ITS2. intermediates indicate defects               rRNA precursor [58 end A2 site 38 end B1(L) site]. cleavage A3, E C2 processing sites            (B) Uniformly labeled rRNA transcript, ITS141  Materials ITS1 ITS2, respectively. probes detected             Methods), incubated 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 immunopre- moderate accumulation 35S rRNA primary tran-                 cipitates derived immunoprecipitates Fig. <br /> script probes 2&#226;&#8364;&#8220;7 detect 32S precursor rRNA              3A 2 hr 37&#194;&#176;C, fractionated 8% polyacryl- <br />                                                                       amide/7 M urea gel. (Lanes 1&#226;&#8364;&#8220;4) ITS141 rRNA, plus IgG&#226;&#8364;&#8220;agarose <br /> (Fig. 6C&#226;&#8364;&#8220;E; data shown). Despite defects                                                                       pellets bound immunoprecipitated RNase P steady-state levels 18S rRNA 25S rRNA                   RNase MRP RNAs Fig. 3A. (Lane 5) ITS141 rRNA. mained unchanged (Fig. 6B), suggesting delayed process-               arrow indicates position cleavage products ing 35S rRNA absence Rpp1.                               identical size ITS1 rRNA transcripts. site cleav- <br />    results Rpp1 required processing              age corresponds region A3 site internal 35S rRNA 58 ETS, ITS1, ITS2. Interestingly,             transcribed sequence 1 (ITS1) (Lygerou et al. 1996). processing reactions shown dependent snoRNPs, RNase MRP, RNase P, stable rRNA degradation intermediates ac- <br /> cumulate cells deficient snoRNP components                      RNP particles exhibit global defects prRNA <br /> (Shuai Warner 1991; Lindahl et al. 1992; Chamber-                 ptRNA processing Rpp1-depleted cells.  <br /> lain et al. 1996b; Venema Tollervey 1996).                terestingly, depletion snoRNP protein, Rrp5,  known proteins associate                  sults striking similarities Rpp1-depleted cells  <br />  <br />  <br /> Figure 6. Effects Rpp1 depletion processing 35S prRNA.   Schematic plan rRNA processing pathway (Venema Tollervey 1995). Steady-state levels rRNA, prRNA intermediates, aberrant rRNA species detected ethidium <br /> bromide staining Northern hybridization oligonucleotide probes 1&#226;&#8364;&#8220;7. (B) Total RNA extracted VS164 VS165 obtained transfer galactose-containing glucose-containing medium times indicated, separated l.2% agarose <br /> gel stained ethidium bromide. Arrows indicate 25S rRNA, 18S rRNA, 5.8S rRNA, tRNAs. (C&#226;&#8364;&#8220;E) gel shown B transferred positively charged nylon membrane (Boehringer Mannheim) capillary diffusion hybridized g-32-labeled <br /> oligonucleotide probes 4&#226;&#8364;&#8220;6  Materials Methods). Oligonucleotide 4 (C) complementary ITS1, 38 end 18S <br /> rRNA A2 site (position 157&#226;&#8364;&#8220;180), oligonucleotide 5 (D) complementary ITS1, A2 A3 sites (position <br /> 119&#226;&#8364;&#8220;236), oligonucleotide 8 (E) complementary ITS2, 38 end 5.8S rRNA C2 site (position &#226;&#710;&#8217;3 +21). 24S precursor rRNA predicted extend 58 end 58 ETS 38 end 5.8S rRNA, represents product 35S rRNA precursor cleaved ITS2 absence cleavage A0, A1, A2, A3 sites. 21S precursor rRNA <br /> appears comprised rRNA species predicted extend 58 end 18S rRNA 38 end 5.8S rRNA, 38 ends extend 38 38 end 5.8S rRNA, heterogeneous. 208S rRNA species predicted extend 58 end 18S rRNA region A3 site ITS1. 17S8 12S8 aberrant rRNA species represent stable <br /> degradation intermediates, fragmented 58 ends correspond sequences 18S rRNA 38 ends close 38 end 5.8S rRNA. 8S rRNA species predicted extend 58 end ITS1 38 extended ends 5.8S rRNA. 7S RNA precursor 5.8S rRNA, predicted extend region A2 site ITS1 38 end 5.8S rRNA. <br />  <br />  <br />                                                                                                     GENES &amp; DEVELOPMENT              2933 <br />  Stolc Altman <br />  <br />  <br />  <br /> respect defects prRNA processing (Venema Tol-      site), ascribed previously RNase MRP (Lygerou et al. <br /> lervey 1996; Discussion).  suggest     1996a).  contrast RNase P, RNase MRP <br /> RNase P interacts functionally RNase MRP            RNA rRNA processing activity separated snoRNPs processing rRNA           Rpp1 high-salt washes. Furthermore, hu- <br /> yeast.                                                       man Rpp30 does associate RNase MRP (N. Jar- <br />                                                              rous S. Altman, unpubl.) yeast RNase P <br />                                                              cleaves prRNA substrates directly vivo  <br /> Processing precursor rRNA Rpp1 <br />                                                              known. Proposals secondary structure ITS1 <br /> immunoprecipitates <br />                                                              based phylogenetic analysis <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> tested Rpp1 associated directly           mapping ITS1 (Van Nues et al. 1994; Allmang et al. <br /> prRNA processing activity vitro cleavage assay      1996b) computational folding ITS1.141 fragments 35S rRNA. assayed cleavage        prRNA substrate, common structural fea- <br /> vitro resuspending 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1-containing immu-          ture RNase P substrate  helical segment noprecipitates fragments 35S precursor       junction single-stranded region; Altman et al. 1995). <br /> rRNA, ITS1.603 (Chamberlain et al. 1996b) ITS1.141        prRNA substrates used study (similar Lygerou et al. 1996a). prRNA substrates     cleaved A3 site vitro reconstituted E. coli <br /> overlap ITS1  Fig. 7A, Materials Meth-       RNase P (V. Stolc S. Altman, unpubl.), ods). Rpp1 immunoprecipitates, contained          broader substrate specificity eukaryotic RNase P <br /> RNase P RNase MRP RNAs, cleaved ITS1.141 (Fig.           (Yuan Altman 1994).  vivo, yeast Rpp1 <br /> 7B) ITS1.603 (data shown) region A3    shared subunit large RNP complex  <br /> processing site. result consistent  does    taining RNase P RNase MRP, rigorously prove, observed defects processing     snoRNPs. 35S rRNA depletion Rpp1 protein vivo.          depletion Rpp1 vivo, defect  3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 immunoprecipitates           ptRNA processing, indication Rpp1 associated <br /> washed extensively high ionic buffer  Material      RNase P activity. Surprisingly, Methods) failed cleave prRNA substrates       rRNA processing defect characterized absence cleaved ptRNAs vitro (data shown). results      cleavage A0 A1 sites 58 ETS, suggest addition RNase MRP, direct     A2 A3 sites ITS1, E C2 sites role RNase P-containing complex rRNA pro-          ITS2 primary 35S rRNA transcript. knowl- <br /> cessing vivo.                                             edge, Rpp1 RNase P RNase MRP protein, <br />                                                              depletion simultaneously inhibits cleavages                                                              major processing sites 35S rRNA Discussion <br />                                                              required ptRNA processing.  processing cloned gene encoding protein subunit          prRNA sites A0, A1, A2, A3, E, C2 nuclear RNase P S. cerevisiae based homol-       coordinated RNase P, depletion RNase MRP <br /> ogy human Rpp30. Yeast Rpp1 homologous          RNA results defects A3 site (Schmitt <br /> human scleroderma autoimmune antigen, Rpp30,           Clayton 1993). described previously autoantigen copuri-         Apart RNase MRP, essential snoRNPs fies Rpp protein subunits hu-     involved directly mechanism rRNA process- <br /> man RNase P&#226;&#8364;&#8221;Rpp14, Rpp20, Rpp25, Rpp30, Rpp38,               ing  recent reviews, Eichler Craig 1994; Ven- Rpp40 (Eder et al. 1997). Using database        ema Tollervey 1995; Tollervey 1996). contrast searches (blastp tblastn algorithms), compared        Rpp1 phenotype, depletion- temperature-sensitive predicted amino acid sequences human Rpp pro-         alleles snoRNAs associated proteins <br /> teins S. cerevisiae genome database. analy-    defect ptRNA processing  ex- <br /> sis revealed Rpp1 yeast proteins   ception snoRNP protein Rrp5, single component amino acid sequence similarity human Rpp pro-         defective processing processing sites teins. second yeast protein Pop4, subunit     58 ETS, ITS1, ITS2. example, Gar1p, Nop1p, <br /> yeast RNase P RNase MRP (Chu et al. 1997),         Sof1p, snoRNA U3, U14, snR30-depleted related amino acid sequence previously unchar-    cells, strains lacking snR10 defective A0, A1, <br /> acterized human Rpp protein, Rpp29 (P. Eder, N. Jarrous,     A2 sites detectable inhibition A3 site S. Altman, unpubl.). yeast, protein, Rpp2,     (Tollervey 1987; Li et al. 1990; Hughes Ares 1991; <br /> shares amino acid similarity Rpp20 (V. Stolc S.     Tollervey et al. 1991; Girard et al. 1992; Jansen et al. <br /> Altman, unpubl.)                                             1993; Morrissey Tollervey 1993). prRNA pro- <br />    Rpp1 small basic protein predicted molecu-    cessing phenotype Rrp5-depleted strains differs lar mass 32.2 kD pI 9.76. does        Rpp1 defective maintaining accu- <br /> previously identified RNA-binding domains. vitro,         mulation 25S rRNA, 18S rRNA, relative levels Rpp1 remains associated mature RNase P RNA          rRNA intermediates (Venema Tollervey 1996).  RNase P activity, buffers high ionic          terestingly, Rpp1-depleted cells, cleavage A3 <br /> strength. Rpp1 associates RNase MRP RNA            site defective Pop1, Pop3, Pop4, RNase rRNA processing activity (cleavage A3          MRP (NME1) conditional mutants (Shaui Warner <br />  <br />  <br /> 2934      GENES &amp; DEVELOPMENT <br />                                                                                         essential protein subunit RNase P yeast <br />  <br />  <br />  <br /> 1981; Lindahl et al. 1992; Schmitt Clayton 1993;                 rRNA ITS1 substrate, ITS1.603 (Chamberlain et al. 1996b), oli- <br /> Lygerou et al. 1996a; Chu et al. 1997; Dichtl Toller-            gonucleotides T7ITS16 38 ITS16 used PCR amplify <br /> vey 1997).  Rpp1 shares rRNA pro-                 yeast genomic DNA amplified fragment subse- <br /> cessing defects known RNase P, RNase MRP,                   quently subcloned pUC19 way ITS1.141 <br />                                                                      fragment. NME-coding sequence amplified PCR <br /> snoRNPs, described protein                                                                      yeast genomic DNA using T7MRP 38 MRP oligo- <br /> ptRNA processing defect rRNA processing  <br />                                                                      nucleotides, subcloned EcoRI SmaI sites pUC19 <br /> fects A0, A1, A2, A3, E, C2 processing             generate pUCT7MRP. <br /> sites. <br />                                                                      Strain construction <br /> Materials methods                                                diploid strain heterozygous RPP1 deletion (VS161) <br /> Strains, media, general procedures                               transformed pRS316::3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 plasmid                                                                      sporulated. sporulation dissection, spores disrupted <br /> S. cerevisiae strains used work listed Table 1.   RPP1, harboring plasmid encoded 3 &#195;&#8212; myc&#226;&#8364;&#8220;Rpp1 fu- <br /> composition media appropriate nutrients plas-        sion protein, viable. haploid cells (VS162A8 mid maintenance S. cerevisiae growth handling tech-          VS162C8), derived sporulation single tetrad, vi- <br /> niques used described (Guthrie Fink 1991). Unless        able loss pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 plasmid stated  techniques manipulating DNA, RNA,          selection plates contain 5-fluoro-<span id='am-20' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-21' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span>orotic acid</span> (5-FOA): oligonucleotides performed according standard pro-       haploids Leu&#226;&#710;&#8217;, showing lost cedures (Sambrook et al. 1989). identities constructs     pRS316&#226;&#8364;&#8220;3 &#195;&#8212; &#226;&#8364;&#8220;RPP1 plasmid wild-type RPP1 allele. verified sequence analysis. Oligonucleotides used         haploids (VS162B VS162D) viable study listed Table 2.                                    pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1 plasmid selection                                                                      5-FOA plates Leu+, showing Gene disruption                                                      3 &#195;&#8212; myc&#226;&#8364;&#8220;tagged RPP1 allele. results obtained <br />                                                                      dissection additional tetrads. growth rate genomic clone spanning RPP1 locus identified using         haploid rpp1:LEU2 cells (VS162B VS162D), depend S. cerevisiae Genome Database (SGD) obtained        low-copy-number episomal plasmid encoding 3 &#195;&#8212; myc&#226;&#8364;&#8220; <br /> American Tissue Culture Collection (ATCC) cosmid 8025.            RPP1 (pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1) viability, identical Clone 8025 sequenced previously (Johnston et al. 1994).        wild-type haploid cells (VS162A VS162C). <br /> 1.4-kb BamHI&#226;&#8364;&#8220;XbaI fragment encoding RPP1 gene sub-             VS163 transformed pYCp&#226;&#8364;&#8220;GAL::rpp1 plasmid <br /> cloned cosmid 8025 pBluescript (SK) vector (Strata-        (VS164) control plasmid, pYCp&#226;&#8364;&#8220;GAL (VS165), sporulated <br /> gene) cut BamHI&#226;&#8364;&#8220;XbaI generate pRPP1SK. RPP1              germinated medium contained galactose. vi- <br /> gene disrupted replacing 0.6-kb NcoI&#226;&#8364;&#8220;NdeI fragment          able spores obtained independent tetrads LEU2 gene, deleting 65% coding se-           presence plasmid-encoded GAL::rpp1 fusion gene quence. rpp1::LEU2 construct integrated RPP1          verified ura&#226;&#710;&#8217; leu&#226;&#710;&#8217; plates containing galactose. <br /> genomic locus diploid strain JN161 (VS161) correct  <br /> placement allele verified PCR digest analysis: <br /> Oligonucleotides 5GAL30 3GAL30 used amplify              Immunoprecipitation <br /> genomic DNA isolated VS161, resulting PCR-ampli-            Extracts prepared lysis yeast cells (25 ml, <br /> fied fragment (2.7 kb) encoding LEU2 flanking sequences       OD600 = 0.5) glass beads (Guthrie Fink 1991). Extracts <br /> Rpp1 mapped restriction enzymes. heterozygous           clarified centrifugation times 15,000g 10 <br /> RPP1/rpp1::LEU2 strain (VS161) sporulated tetrad             min. final supernatants used immunoprecipitations <br /> analysis showed 2:2 segregation cell viability. Dissections    adding 4 &#194;&#181;g 9E10 antibody protein extract derived performed 20 tetrads. disruption analysis       spores VS162A8, VS162B, VS162C8, VS162D,  performed strain VS163, disrupted         cubated 2.5 hr, followed addition 50 &#194;&#181;l 100 mg/ml rpp1::LEU2 construct.                                                anti-mouse IgG  molecule) agarose (Sigma) IP150 <br />                                                                      buffer (150 mM KCl, 10 mM Tris-Cl (pH 7.9), 0.1% NP-40,                                                                      0.1% NaN3). Pellets washed times IP150 Construction plasmids <br />                                                                      IP600  composition IP150, 600 mM KCl). RNA generate myc-epitope-tagged RPP1, myc epitope            extracted immunoprecipitated beads adding <br /> mains (3 &#195;&#8212; myc) PCR-amplified 3 &#195;&#8212; myc&#226;&#8364;&#8220;SEC8             100 &#194;&#181;l IPR buffer (100 mM Tris-Cl pH 7.5, 100 mM EDTA <br /> construct (TerBush Novick 1996) using oligonucleotides           pH 8.0, 150 mM NaCl, 1% SDS), followed extractions <br /> MYC5 MYC3, cut BsmI. PCR fragment              phenol extraction phenol/chloroform/iso- subcloned BsmI site pRPP1SK plasmid gen-          amylalcohol solution (25:24:1). RNA precipitated erate p3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1SK. 1.5-kb BamHI&#226;&#8364;&#8220;XbaI fragment             ethanol. <br /> p3 &#195;&#8212; myc&#226;&#8364;&#8220;RPP1SK subcloned pRS316 plasmid gen- <br /> erate pRS316&#226;&#8364;&#8220;3 &#195;&#8212; myc::RPP1. coding sequence RPP1                                                                      Assays RNase P activity rRNA processing activity <br /> PCR-amplified pRPP1SK using oligonucleotides 5GAL30 3GAL30, cut BamHI XhoI, subcloned              IgG&#226;&#8364;&#8220;agarose pellets, bound immunoprecipitated <br /> modified pYCp33 vector contained GAL1&#226;&#8364;&#8220;10 promoter           RNase P RNase MRP RNAs, washed IP150 generate pYCp&#226;&#8364;&#8220;GAL::rpp1. generate rRNA ITS1 substrate,         incubated labeled ptRNASer (Drainas et al. 1989) 30 <br /> ITS1.141 (similar Lygerou et al. 1996a), oligonucleotides         min 37&#194;&#176;C 1&#195;&#8212; BB (10 mM HEPES pH 8.0, 400 mM <br /> T7ITS14 38 ITS14 used PCR-amplify yeast genomic          NH4OAc, 10 mM Mg2OAc, 5% glycerol). rRNA transcripts <br /> DNA 141-bp fragment cloned EcoRI           ITS1.141 ITS603 (Chamberlain et al. 1996b) incubated <br /> BamHI sites pUC19 (New England Biolabs). generate              IgG pellets bound immunoprecipitated <br />  <br />  <br />  <br />  <br /> RNase P RNase MRP RNAs (washed IP150) 2 hr          <br /> 37&#194;&#176;C 1&#195;&#8212; PMB (20 mM Tris-HCl pH 8.0, 10 mM MgCl2, 1 mM          <br /> EDTA, 50 mM KCl, 2 units RNasin, 50 mg/ml BSA).             <br /> IgG pellets, bound RNase P RNA, washed         <br /> IP600 incubated ptRNASer, ITS141, ITS603            tRNA substrate, ITS141, ITS603 rRNA sub-                <br /> strates labeled uniformly [32P]<span id='am-5' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span>GTP</span> (3000 mCi/              <br /> mmole, Amersham) purified 8% polyacrylamide/7 M               <br /> urea gel. assays performed 0.5 nM ptRNA, 3.2 nM            <br /> ITS141 RNA, 0.5 nM ITS603 RNA. RNA products               <br /> fractionated 8% polyacrylamide/7 M urea gel.                <br /> RNA analysis <br />                                                                             <br /> Total RNA isolated disruption cell pellets (25 ml,                <br /> OD600 = 0.5), resuspended AE buffer [50 mM NaAc pH 5.3,        <br /> 10 mM EDTA, 10 mM RNase inhibitor, vanadyl ribonucleoside                  <br /> complex (GIBCO BRL)] equal volume phenol chlo-                   <br /> roform, glass beads 65&#194;&#176;C 15 min. Phenol/chloroform             <br /> extractions repeated times 65&#194;&#176;C 15 min, fol-          <br /> lowed extraction chloroform 25&#194;&#176;C ethanol                 <br /> precipitation. Northern hybridization performed                <br /> scribed previously (Guerrier-Takada et al. 1995). oligo-              <br /> nucleotides end-labeled T4 polynucleotide kinase             <br /> (New England Biolabs) [g-32P]<span id='am-25' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-26' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span>ATP</span> (Amersham). NME1                    <br /> EcoRI&#226;&#8364;&#8220;SmaI DNA fragment pUCT7MRP EcoRI&#226;&#8364;&#8220;SmaI                      <br /> DNA fragment pScRNAP (Lee et al. 1991) labeled                  <br /> uniformly [32P]<span id='am-51' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-52' property="rdfs:label" content="dCTP" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span>dCTP</span> (Amersham).                                  <br /> </body></html>