<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Apr. 1992, p. 1879-1892                                                                   Vol. 12,  4 <br /> 0270-7306/92/041879-14$02.00/0 <br /> Copyright C) 1992, American Society Microbiology <br />  <br />  <br />         Presumptive Helicase (MOTi Gene Product) Affects Gene <br />            Expression Required Viability Yeast <br />                         Saccharomyces cerevisiae <br />                          JENNIFER L. DAVIS,t RIYO KUNISAWA, JEREMY THORNER* <br />                 Division Biochemistry Molecular Biology, Department Molecular Cell Biology, <br />                          Room 401, Barker Hall, University California, Berkeley, California 94720 <br />                                         Received 25 November 1991/Accepted 8 January 1992 <br />  <br />             Exposure haploid yeast cell mating pheromone induces transcription set genes. Induction          mediated cis-acting DNA sequence upstream pheromone-responsive genes.          STE12 gene product binds specifically sequence element required maximum levels          basal induced transcription, pheromone-responsive genes regulated identical manner.          investigate additional factors play role transcription genes, genetic screen used <br />          identify mutants able express pheromone-responsive genes constitutively absence Stel2.          way, identified recessive, single gene mutation (mot], modifier transcription) increases          basal level expression   pheromone-responsive genes. motl-l allele relaxes          requirement class upstream activating sequence enhances expression          gene previously thought involved mating pathway. Cells carrying mot1-l grow slowly 30&#194;&#176;C <br />          inviable 38&#194;&#176;C. MOT] gene cloned complementation temperature-sensitive <br />          lethality. Construction null allele confirmed MOTI essential gene. MOT] resides chromosome <br />          XVI encodes large protein 1,867 amino acids contains seven conserved domains          known putative helicases. product MOT] strikingly homologous Saccharomyces cerevisiae <br />          SNF2ISWI2 RAD54 gene products entire helicase region. <br />  <br />    Coordinate transcriptional regulation appropriate set            induced transcription pheromone-responsive genes (23, genes essential feature response cells              24), requirement Stel2 basal transcription extracellular signals. eukaryotes, concurrent expres-             absolute. Fields et al. (23) demonstrated phero- <br /> sion gene set involves specific upstream DNA                   mone-responsive genes produced markedly lower, sequence elements regulatory proteins bind              readily detectable, levels mRNA stel2 null strain elements interact complex manner general                    isogenic STE12+ control, stel2 <br /> transcription factors. useful investigating                strain, transcription pheromone-responsive genes regulation mating response pathway                     undetectable. addition, Errede Ammerer (19) <br /> unicellular eukaryote Saccharomyces cerevisiae. Haploid                   observed apparent composition Stel2-dependent <br /> cells yeast exist cell types  4Ta              DNA-binding complexes varied pheromone-respon- <br /> M Ta). cell type secretes small oligopeptide phero-                sive promoter examined. findings suggest mone (AL4Ta cells secrete factor; Ta cells secrete                   pheromone-responsive genes regulated identical factor) acts cells opposite mating type              manner additional factors exist differ- <br /> induce programmed set physiological changes, including               entially modulate transcription various members arrest cell growth G1 phase cell cycle,                  set genes. <br /> synthesis cell surface agglutinins, characteristic alter-            expand understanding components ations cell morphology. Collectively, changes                govern expression pheromone-responsive genes,  <br /> mit cell fusion diploid formation  recent review,           vised genetic screen identify factors operate <br /> reference 68).                                                            downstream  independently  Stel2.  <br />    Response mating pheromone involves transcrip-                   scribe isolation mutation increases basal-level <br /> tional activation   cases, repression) set            expression pheromone-responsive genes genes, termed pheromone-responsive genes, ini-                  absence Stel2 absence pheromone. tiate mating program (16, 68, 69). product                 isolated corresponding wild-type gene determined <br /> STE12 gene protein binds specifically (17, 19)              encodes unusually large, previously unidentified 8-bp upstream activating sequence (UAS)                   protein required cell growth viability. <br /> pheromone-responsive promoters copies orientation (44, 74). UAS, called phero- <br /> mone response element (PRE), consensus sequence                                 MATERIALS METHODS <br /> 5&apos;-ATGAAACA-3&apos; (44, 74). <br />    Stel2 protein required basal                 Strains, media genetic methods. yeast strains used <br />                                                                           study listed Table 1. Genetic manipulations                                                                           culture conditions described Sherman et al. <br />   * <br />     Corresponding author.                                                 (63) twice recommended levels nutritional <br />   t Present address: Somatic Therapy Corp., 850 Marina Village            supplements used synthetic media. Plates  <br /> Parkway, Alameda, CA 94501.                                               taining 0.02% 5-bromo-4-chloro-3-indolyl-,B-D-galactoside <br />                                                                    1879 <br />  1880        DAVIS ET AL.                                                                                                           MOL. CELL. BIOL. <br />  <br />                      TABLE 1. Yeast strains used                                                    TABLE 2. Gene probes used <br />    Strain                         Genotype                           Source        Gene                      Probe                       Reference(s) <br /> EG123            MA Ta leu2-3, 112 ura3-52 trpl-l his4-           S. Fields       CMD1          0.8-kb EcoRI-StyI fragment                 13 <br />                    519 canl-101                                                   DEDI          1.8-kb BamHI fragment                      70 <br /> SF167-5aa        EG123 stel2::LEU2                                S. Fields       FUSI          0.5-kb NheI-SphI fragment                  72 <br /> 1788             MATa leu2-3,112/leu2-3,112 ura3-               D. Levin        MFal          1.6-kb BamHI fragment                      6 <br />                    52/ura3-52 trpl-lltrpl-l his4-519/                             MFa2          1.1-kb HpaI fragment                       6 <br />                    his4-519 canl-101/canl-101                                     PKC1          23-base specific oligonucleotide           48 <br /> JD195b           MATa isogenic EG123                           work       STE2          14-base specific oligonucleotide           9, 52 <br /> JD209C           JD195 stel2::LEU2                                work       STE6          0.8-kb SpeI fragment                       45 <br /> JD100            SF167-5a motl-l                                  work       SST2          1.3-kb ClaI-NheI fragment                  12, 15 <br /> JD102Cd          AL4Ta motl-l leu2-3, 112 ura3-52                 work <br />                    trpl-1 his4-519 canl-101 <br /> JD22/23e         MATa motl-lImotl-l leu2-3, 112/                work <br />                    leu2-3,112 ura3-52/ura3-52 trpl-1/                                Plasmid constructions gene probes. recombinant <br />                    trpl-1 his4-519/his4-519 canl-1011                             DNA methods performed standard procedures (50). <br />                    canl-101                                                       Plasmid pLGA178 (29) digested URA3 marker <br /> JD215Af          MATa isogenic JD102C                          work       ApaI, converted flush ends incubation T4 <br /> JD215Bf          ALTa isogenic JD102C                          work <br /> JD227Af          MATa stel2::LEU2 isogenic                     work       DNA polymerase, ligated BglII fragment containing <br />                    JD102C                                                         TRPI marker, excised plasmid p330 (73), JD216Df          M4Ta isogenic JD227A                          work       converted blunt ends incubation deoxyribonucle- <br /> YPH4999          MATa ura3-52 lys2-801a&apos; ade2-101&quot;c               P. Hieter       otides Klenow fragment E. coli DNA polymerase <br />                    trpl-A63 his3-A200 leu2-AI                                      generate plasmid pLGA178T. Plasmid pLGA<span id='am-15' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-16' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>;178T  <br /> YPH5019          MATa/ot ura3-52/ura3-52 lys2-801a,/              P. Hieter       fers tryptophan, <span id='am-103' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-104' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span>, prototrophy recipient trpl <br />                    lys2-801am ade2-101oclade2-101oc                               ura3 yeast cells. construct pJD11, XhoI cassette <br />                    trpl-A63/trpl-A63 his3-A200/his3-                              containing tandem consensus PRE boxes (provided                    A200 leu2-Al/leu2-Al                                           F. Gimble) inserted XhoI site pLGA178T. <br />     Construction strain described (25). <br />   b Constructed                                                                                    pJD11 plasmid contains tandem head tail inserts <br />                      DNA-mediated transformation EG123 plasmid-         PRE consensus sequences noncoding <br /> borne copy HO gene (35, 61).                                               strand.) Plasmid pSTE12(URA3) contains Sacl fragment <br />   &apos; <br />     Constructed step gene replacement method (60) using stel2::LEU2 plasmid (pLB1367), generously provided G. Ammerer.                 bearing C-terminal 18 amino acids   d        haploid segregant derived second backcross strain JD100        STE12 coding sequence (19), isolated plasmid pLB1212 <br /> carrying pSTE12(URA3) EG123  Materials Methods).                       (generously provided G. Ammerer), inserted pRS316 <br />   e haploid segregants opposite mating type derived described        (65). Plasmid pSTE12(URA3) fully restores mating compe- <br /> footnote d mated. <br />   f Haploid segregants derived backcross JD102C SF167-Sa             tence stel2 strains (data shown). <br /> carrying pSTE12(URA3)  Materials Methods). Prior sporulation           DNA probes used study listed Table 2. final resulting diploid, cells grown nonselectively plated        cleavage indicated restriction enzymes, DNA frag- <br /> medium containing 5-fluoroorotic acid (4) identify cells lost   ments purified gel electrophoresis, recovered pSTE12(URA3). <br />   g Derivation strains described (65).            electroelution, labeled random-primer (hexanu- <br />                                                                                   cleotide primers; Pharmacia) labeling method (22)                                                                                   [al-32P]<span id='am-98' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-99' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="dCTP" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span>dCTP</span> (Amersham). Oligonucleotide probes la- <br /> (X-Gal) (Bachem) prepared using synthetic medium                          beled T4 polynucleotide kinase [y_- 3P]<span id='am-32' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-33' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span> (Amer- <br /> buffered pH 7.0 50 mM potassium phosphate. DNA-                           sham). Excess unincorporated <span id='am-97' about='rdfs:label' typeof='owl:Thing'>label</span> removed mediated transformation yeast cells performed                       probes using NICK columns (Pharmacia). Restriction <br /> lithium acetate procedure (39). Plasmid genomic DNAs                          enzymes DNA-modifying enzymes purchased isolated yeast cells method Hoffman                       Stratagene, Pharmacia, Boehringer-Mannheim, New <br /> Winston (38), plasmids introduced Escherichia                       England Biolabs used according manufacturers&apos; <br /> coli method Chung Miller (10).                                      recommendations. <br />   E. coli HB101 (5) used propagation                             Isolation suppressors stel2. Multiple cultures strain <br /> plasmids. Strains DH5aF&apos; (33) JM101 (51) used                        SF167-5a carrying pJD11 grown selective condi- propagation certain plasmids preparation                       tions  Trp) -2 x 108 cells ml treated single-stranded double-stranded DNAs sequenc-                          various amounts ethyl methanesulfonate (Kodak) accord- <br /> ing. Standard methods culture conditions used                            ing standard protocols (63). Mutagenized cells plated <br /> (50).                                                                             synthetic complete medium lacking Trp cell densities <br />    Chromosomal assignment determined hybridiza-                            yield isolated single colonies. Plates incubated tion MOTI probe intact yeast chromosomes                        17&#194;&#176;C  7 days), 30&#194;&#176;C  3 days), 37&#194;&#176;C  2 days) separated pulsed-field gel electrophoresis (75) trans-                     replica plated X-Gal-containing plates. indi- <br /> ferred nitrocellulose. Assignment chromosome XVI                            cator plates incubated 30&#194;&#176;C 1 2 days, confirmed   rehybridizing filter                          colonies visually inspected blue color. motl-l <br /> stripping reexposing film ensure complete removal                       mutation described isolated culture cells probe) gene probe (PEP4) known reside                         mutagenized 50% survival initially plated 30&#194;&#176;C. <br /> chromosome XVI (41) (ii) hybridizing MOTJ                              Plasmid pSTE12(URA3) introduced original <br /> probe ordered collection yeast genomic DNA                               motl-l mutant strain (JD100), backcrossed <br /> fragments  bacteriophage X vector) (55), generously                          times EG123; time, slowly growing, STE+ <br /> provided L. Riles M. Olson (Washington University,                         spores selected examined temperature sensi- <br /> St. Louis, Mo.).                                                                  tivity blue color X-Gal medium. observed  VOL. 12, 1992                                         PUTATIVE HELICASE AFFECTING GENE EXPRESSION GROWTH                                              1881 <br />  <br />                                                      TABLE 3. Effect motl-J gene expression <br />  <br />   Strain                              Relevant genotypea                              Plasmid (UAS)b&quot;-Galactosidase (U)&apos; <br />                                                                                                                              MF                       MF <br /> SF167-5a                        MATa stel2::LEU2 MOTI+                        pJDll (PRE)                             Und                        Und <br /> JD227A                          MATa stel2::LEU2 motl-1                       pJDll (PRE)                             1.1                        0.5 <br /> EG123                           MATa STE12+ MOT1+                             pJDll (PRE)                             21.3                       121.6 <br /> JD215B                          M4Ta STE12+ motl-l                            pJDll (PRE)                             48.5                       156.9 <br /> SF167-5a                        MATa stel2::LEU2 MOTI+                        pLGA&amp;178T                          Und                        Und <br /> JD227A                          MA Ta stel2::LEU2 motl-l                      pLGA178T                           0.3                        0.5 <br /> EG123                           MATa STE12+ MOT1+                             pLGA&amp;178T                          Und                        Und <br /> JD215B                          MATa STE12+ motl-l                            pLGA178T                           0.6                        0.5 <br />   strains isogenic STE12 MOTI loci. <br />   b <br />     pLGA178T cycl promoter-lacZ fusion lacking normal UAS cycl; pJD11, synthetic PRE elements inserted place UAS cycl. <br />   c   Values averages assays performed duplicate (range, c30%). Und, undetectable (&lt;0.1 U 6-h incubation); MF, factor mating pheromone. <br />  <br />  growth phenotypes (slow growth 30&#194;&#176;C lethal-                         stranded herring sperm DNA ml, 1 x Denhardt&apos;s solution <br /> ity 38&#194;&#176;C) transcriptional phenotype coseg-                       (50), 1% <span id='am-68' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-69' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span>, 1% sodium dodecyl sulfate (SDS) regated. final backcross SF167-Sa carrying pSTE12                           washed 60&#194;&#176;C 0.lx SSPE-0.1% SDS. hybridization <br /> (URA3) performed generate strains used                          oligonucleotide probes, buffer described  majority experiments presented                                         formamide HEPES, used 42&#194;&#176;C; blots <br />    Procedures phenotypic characterizations. p-galac-                       washed 2x SSPE-0.1% SDS 45&#194;&#176;C (STE2 probe) tosidase assays, cultures grown early exponential                          55&#194;&#176;C (PKCJ probe). filters used expose Kodak <br /> phase synthetic complete medium lacking Trp, buffered                        XAR-5 film, using DuPont Cronex Lightning-Plus intensi- <br /> pH 3.5, select maintenance lacZ reporter                             fying screen. necessary remove probe <br /> plasmid split portions. culture                        rehybridizing filter, filter stripped set, factor (Peninsula Laboratories) added                         (50) exposed film, using intensifying screen final concentration 2.5 ,M, treated                            20 h ensure removal radioactivity untreated cultures incubated 30&#194;&#176;C 60 min. Cells                        complete prior reuse. harvested assayed P-galactosidase described                           DNA sequence analysis. Restriction fragments (31, 58). Independent assays performed                           region containing MOTJ-complementing activity (8-kb <br /> duplicate transformants, data averaged;                           BglII segment) adjacent fragments subcloned range observed usually 10% greater                         polylinker regions M13mpl8 (54), M13mpl9 (54), 30%.                                                                          pRS316 (65). Nested deletions fragments pRS316    Bioassays factor production performed                           generated using exonuclease III-mung bean nuclease <br /> scribed Kuchler et al. (45).                                                    protocol supplied manufacturer enzymes (Strat- <br />    Quantitative mating assays performed mixing                             agene). single-stranded double-stranded templates <br /> -107 exponential-phase cells strain examined                          sequenced using Sequenase kit (United States -10&apos; exponential-phase cells tester strain                        Biochemical). Specific oligonucleotide primers synthe- <br /> opposite mating type, DC17 (MATTa hisl) DC14                             sized used necessary close gaps sequence <br /> (A4Ta hisl), nitrocellulose filter. Filters washed                       obtained deletion constructions confirm YPD incubated YPD plate 30&#194;&#176;C 5                        sequence junctions restriction fragments. 6 h. Cells washed filters resuspended                     <span id='am-48' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-49' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> sequences confirmed region water. Dilutions cells  case STE12+ matings)                       DNA sequencing strands.  35S]<span id='am-22' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="dATP" datatype="xsd:string"></span>dATP</span> cells  case stel2 matings) plated minimal                    Amersham. <br /> plates selective diploids. Mating frequency defined                        Nucleotide sequence accession number. GenBank ac- total number diploids produced divided total                         cession number MOTI M83224. <br /> number input haploids strain examined. <br />    Isolation poly + RNA hybridization analysis. Cul-                                                 RESULTS <br /> tures grown mid-exponential phase YPD (pH 4.0) split equal portions. half culture                          Isolation motl-] mutation. genetic screen used treated factor (Peninsula Laboratories) final                      identify mutants expressed pheromone-responsive <br /> concentration 2.5 p,M 30&#194;&#176;C 40 min; half                        genes absence Stel2 absence phero- placed ice period time. Total RNA                        mone. reporter plasmid (pJD11) 16 PREs prepared, poly + RNA isolated using oli-                              inserted place UAS cycl promoter-lacZ gene dT)-cellulose (Pharmacia), described (63).                         fusion  Materials Methods) introduced Poly + RNA (4 p,g) separated electrophoresis                              strain lacking functional STE12 gene (SF167-5a) (25). 1% agarose gels containing 2.2 M <span id='am-92' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-93' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span>                          amply documented level 3-galactosidase <br /> transferred Hybond-extraC membranes (Amersham)                               activity expressed appropriate promoter-lacZ fusions <br /> 20x SSPE (lx SSPE 0.18 M NaCl, 10 mM NaPO4, 1                               directly reflects transcriptional activity gene (31, 58). mM EDTA [pH 7.7]) (50). Molecular weight markers                              absence Stel2, expression lacZ driven PRE Bethesda Research Laboratories. Filters hybrid-                          elements extremely low absence ized DNA probes 42&#194;&#176;C buffer containing 50%                              presence factor (Table 3). agreement formamide, 6x SSPE, 50 mM N-2-hydroxyethylpiperazine-                              finding, SF167-Sa[pJD11] white X-Gal-containing <br /> N&apos;-2-ethanesulfonic acid (HEPES; pH 7.4), 40 p,g single-                        plates absence presence pheromone (data <br />  1882     DAVIS ET AL.                                                                                                   MOL. CELL. BIOL. <br />  shown). contrast, pJD11 introduced                   TABLE 4. Effect motl-l mating efficiency <br /> isogenic STE12+ strain (EG123) (25), level ,B-galac- <br /> tosidase activity produced 3 orders magnitude higher         Strain                  Relevant genotypea                   Mating <br />                                                                                                                                frequency&quot; stel2 strain absence pheromone <br /> (Table 3). Furthermore, activity increased nearly sixfold         SF167-5a           MATa stel2::LEU2 MOT1I                   &lt;2.6 x 10-7 cells exposed factor (Table 3). Correspond-        JD227A             MATa stel2::LEU2 motl-l                  &lt;2.4 x 10-7 <br /> ingly, EG123[pJD11] pale blue absence phero-        JD209              MATa stel2::LEU2 MOTIJ                   &lt;1.7 x 10-7 <br />                                                                   JD216D             MATot stel2::LEU2 motl-l                 &lt;2.0 x 10-7 <br /> mone dark blue presence factor X-Gal-          EG123              MATa STE12+ MOT]+                        0.59 <br /> containing plates (data shown).  expression           JD215B             AL4Ta STE12+ motl-l                      0.44 <br /> pJD11 strains regulated manner identical      JD195              MATa STE12+ MOT1+                        0.73 observed natural pheromone-responsive genes           JD215A             MATa STE12+ motl-l                       0.40 maximum basal induced levels transcription              strains isogenic AL4T, STE12, MOTI loci. <br /> required functional STE12 gene (23, 24).                          bTotal diploids input number haploid cells strain examined.    SF167-5a carrying pJD11 mutagenized ethyl             Materials Methods experimental details. <br /> methanesulfonate  Materials Methods), 50,000 <br /> surviving colonies screened blue color X-Gal- <br /> containing plates lacking factor. Candidate blue colonies <br /> (73 total) assayed vitro 3-galactosidase activity     motl mutation did rescue sterility stel2 null <br />  Materials Methods) identify mutants             mutants. <br /> specifically elevated expression PRE-driven lacZ gene         effect motl mutation expression ,B-galac- eliminate displayed blue color           tosidase pLGA178T, lacks UAS  Materials <br /> increased permeability X-Gal nonspecific effects.     Methods), examined (Table 3). Interestingly, 73 candidates tested, displayed increased         motl-l strains displayed readily measurable level vitro ,-galactosidase activity. rule mutants      expression, isogenic MOT] + cells did                                                                    effect independent presence Stel2 expression indicator gene elevated plas-        influenced pheromone treatment. relaxation mid-linked alterations, expression endogenous phero-           requirement UAS cycl-lacZ expression suggested <br /> mone-responsive genes examined remaining         motl alter expression genes candidates.                                                       addition pheromone-responsive genes  . <br />    factor structural genes, MFal MFa2 (6),         rule possibility,  variations gene required factor secretion, STE6 (45),        plasmid copy number explain apparent effects <br /> induced factor (6, 45) require Stel2 basal      observed gene expression different strain  induced expression (23, 24). Mutations increase          grounds, plasmid DNA recovered expression reporter gene generally            strains examined quantitative hybridization analysis <br /> elevate expression PRE-containing UAS elements               (67). Comparison plasmid copy number increase expression chromosomally located          various strains examined showed negligible differ- <br /> pheromone-responsive genes generate              ences  twofold) (data shown), confirming factor does parental strain. Pheromone produc-         different levels 3-galactosidase observed reflected <br /> tion judged factor bioassay measures      changes gene expression. factor synthesized secreted sur-            Effect mot] gene transcription. determine rounding medium (45). remaining candidates,          general effect motl gene expression, hybrid- isolate reproducibly displayed elevated factor produc-      ization analysis poly + RNA variety chromo- <br /> tion (data shown) addition elevated ,-galactosi-    somal genes performed (Fig. 1). pheromone-respon- <br /> dase activity. level production factor         sive genes expression severely reduced, tected size zone growth inhibition (halo)   completely eliminated, inactivation STE12 (23), includ- <br /> lawn A4 Tao cells AL4Ta mutant cells       ing MFa2 (6), STE2 (encoding factor receptor) (9, 52), <br /> spotted. mutant called motl  modifier tran-       STE6 (45), motl-l allele caused significantly  <br /> scription), allele described designated          creased expression  threefold) stel2 background <br /> motl-l.                                                           modest increase STE12- background (Fig. 1A <br />    Phenotypic characterization motl-1. STE12+        Table 3). stel2 strains, factor induction <br /> ground, basal expression pJD11 twofold higher      detectable motl cells. STE12+ strains, pheromone <br /> cells carrying motl-l allele (JD215B) wild-type       treatment elicited transcriptional induction motl-l <br /> cells (EG123) (Table 3). Following exposure pheromone,         MOTI + cells, levels motl cells appeared induced level strains equivalent. expected     slightly higher. effect motl expression basis scheme isolation, stel2          chromosomally located genes qualitatively background, motl-l strain (JD227A) carrying pJD11               effect PRE-driven reporter plasmid (Table <br /> showed readily detectable expression PRE-driven            3). rule possibility transcriptional effects <br /> lacZ gene, MOTI + stel2 parent (SF167-Sa) did         motl-l cell specific, expression AL4Ta cells (Table 3). Expression essentially unaffected           STE3, cell-specific gene encodes factor <br /> exposure factor, indicating motl-l does restore     receptor (31, 52), examined. STE3 expression similar <br /> response pheromone action does bypass require-      MFa2, STE2, STE6 sense detectable <br /> ment Stel2 maximum levels induction. point        basal transcription observed stel2 mutant cells (23). supported quantitative mating assays (Table        STE3 expression increased  <br /> 4). Regardless mating type, significant difference       fold ALTa motl-l cells compared isogenic ALTa <br /> mating efficiency observed MOT] + motl-l          MOTJ + cells (data shown), indicating transcrip- <br /> strains containing functional STE12 gene; conversely,       tional phenotype motl-l cell type specific. <br />                                                                                                                                         U, <br /> VOL. 12, 1992                                                  PUTATIVE HELICASE AFFECTING GENE EXPRESSION GROWTH                                       1883 <br />  <br />                               stel2::LEU2                          STE112+ <br />                                                                                                        Alf- T       T   1   2+ <br />                                      +                +             +         + <br />                                vWE-  E--                      vi    E   v   Es4                  MnoTl7-            STEI2+ <br />                                0 0 0                      0 00 <br />                                fE3           E                E:          E <br />                                                                                          MOT1&quot;+ stee2::LEU2 <br />                     o-MF:      -     -       +        +       -     -     +   + <br />         <br />                     MFa 2     lI <br />                                s                                  ----                                                 mort1-1 stSel 2::LEU2 <br />                     STE2          al AW ilO                                                                                             3v C3&apos; ::C <br />                                                                                           FIG. 2. Growth phenotypes motl-l cells. different <br />                     S TE6         ,   , <br />                                                  ;W   _                                 spores identical genotype, derived cross JD102C (MA Ta <br />                                                                                         motl-1) SF167-5a  Ta stel12::LEU2 [pSTE12(URA3)]}        B.                                                                               streaked YPD plate allowed grow 30&#194;&#176;C 3 days. <br />                        J <br />                     FUS1                                  _                             Cells plates resuspended water equal densities, <br />                                                                                         placed multiwelled template dish, spotted fresh <br />                     SSlEa 1                               M               -             YPD plates using pronged inoculator. plates incu- <br />                                                                                         bated 30&#194;&#176;C 3 days 38&#194;&#176;C 3 days, indicated. <br />                     3372214                                         <br />  <br />  <br />        c. <br />                                                                                         calmodulin) (13) (Fig. 1D; Fig. 5B) PKCJ <br />                     DED1        .   .e <br />                                        . <br />                                     .... <br />                                            .~~~~~k  .w: ::.... <br />                                                               (encoding..    sZprotein kinase C) (48)  Fig. 5A).                                                                         yeast <br />                                                               level CMDJ PKCJ transcript used                                                                                         normalize differences RNA loading. <br />        D. <br />                       CMD 1         _                   44E -                            Effect moti mutation growth. addition                                                                                         transcriptional phenotypes described  growth <br />    FIG. 1. Effect motl gene transcription. Poly + RNA (4 <br /> pug) strain loaded sample  relevant                    phenotypes observed motl mutants. Cells carrying <br /> genotype strain indicated lane; strains                       motl-l allele grew slowly 30&#194;&#176;C, doubling time listed Table 3. Gene probes  Table 2) designated                    3.5 h rich medium (YPD), inviable 38&#194;&#176;C left strip. filter used hybridizations                 (Fig. 2). growth phenotypes manifest  Materials Methods conditions). MF, mating factor;                      stei2 STE12+ background. temperature shift experi- <br /> -, treated factor; +, treated factor.   Phero-                      ment performed early-exponential-phase <br /> mone-responsive genes display Ste12-independent basal tran-                        motl-l cells shifted 30 38&#194;&#176;C examined scription; (B) pheromone-responsive genes require Stel2                        time points shift. uniform terminal cell <br /> basal transcription; (C) DED1, gene product thought mating pathway; (D) CMD1, used control                       morphology observed (data shown). Cells held uniformity RNA loading.                                                          38&#194;&#176;C 4 h retained nearly 100% viability shifted <br />                                                                                         30&#194;&#176;C.  longer periods 38&#194;&#176;C, viabil- <br />                                                                                         ity dropped rapidly progressively. example, 24 <br />                                                                                         h 38&#194;&#176;C, viability 0.1% MOT] + cells <br />    pheromone-responsive genes expression                                   subjected treatment. MOTI gene reduced undetectable levels inactivation STE12 (23),                           cloned  , expression MOTJ transcript including FUSJ (encoding protein required cell fusion)                            motl-l mutants analyzed 4 8 h shift (72), MFal (6), SST2 (encoding protein involved                                38&#194;&#176;C. 8 h, transcript present level <br /> recovery factor-induced G1 arrest) (12, 15),                                 nearly equivalent observed initially (data shown). <br /> motl-l mutation did lead detectable transcription                              inviability 38&#194;&#176;C likely lack stel2 background (Fig. 1B). results account                               mRNA mutant cells defect fact despite elevated expression certain                            function motl-l product 38&#194;&#176;C. <br /> mating genes, motl stel2 strains mating deficient                                Isolation sequence MOT) gene. motl-l <br /> (Table 4). Remarkably, STE12+ strains, motl-l                                haploid crossed normal haploid, resulting <br /> mutation detectable effect enhancing basal                               heterozygous diploid cells grew normally 30 38&#194;&#176;C transcription genes;  factor induction                             did display elevated expression pJD11 reporter genes equivalent motl MOTI + cells.                                 plasmid;  motl-l allele recessive mutation. <br />    Expression gene thought unrelated mating <br /> pathway, DEDJ  defines essential domain) (70), increased motl-l mutation stel2                                  s&apos;                                                                      3. <br /> STE12+ backgrounds (Fig. 1C), way observed MFa2, STE2, STE3, STE6 genes. DEDI tran- <br /> scription significantly affected factor treatment                                             II                                                                                                                                                                             I0 STE12+ cells, agreement fact 5&apos;-                                     -                                                            _   9 <br /> flanking region DEDJ gene lacks canonical PRE                                    kb         0              1           2    3       4   5        6 <br />  <br /> sequences (70). DEDJ gene product implicated mating process previously. does                               FIG. 3. Restriction map MOTI gene. Rectangular regions <br /> play role mating, data suggest expres-                          represent areas subjected DNA sequence analysis; <br />                                                                                         dashed lines represent regions sequenced. open <br /> sion genes outside pheromone response                                    reading frame (ORF) represented hatched rectangle; <br /> network affected motl-l mutation.                                             noncoding regions represented open rectangles. sites <br />   genes identified expression com-                                   enzyme indicated shown ClaI  second <br /> pletely unaltered motl-l,  CMDJ (encoding yeast                               dam-sensitive site exists position 3435). <br />  1884         DAVIS ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                       -221                         -191                          -161                       -130 <br />              GTCCCACTCGCCGGCCGCTGCACTTGATATAAAAGCTTATGATTCCACCAATTCCGTTCCCTTGGCGGTTGCGCGCCAmTT7TCATGCGCTCAGA^AAACCACCATTCTGCTATT <br />                                       -100                          -70                           -39                        -10 <br />              ATACCGAAAACCGGCGG                                   TTCAAACATAAATAAAGTACGATCGTATTCCAATATTAACGAACAAAAGTATAACACACACA <br />                     -1 <br />              GCACCTCAA <br />                                             30                            60                             90                             120 <br />              ATGACGTcAcGAGTTT      ;AGGCTGGAcCGG <br />                                                CAGGTATT   AAGAGACGGTTCCACCCAAGTATCGGAACTGTGCCG&amp;CATGAATGGCCA AACAGCACCCCGAA <br />                                                           E T G S T 0 V V R N M D Q M G D L K O H P E <br />         1    M T S R V S         R   L D R    V L <br />                                                  0 <br />                                            150                           180                            210                             240 <br />              GACATTCTCGTTGCTGTCCAGAGTT1ACCC TTACTAGTCAGAGAATGGGAACGCGTGTTACTGCAGCAAGAGTTCi-CCATGCTCCixTTGGGATCCG <br />        41    D L S L L S R V Y P F L L V K K W E T R V T R V G G V H P S W D P <br />                                            270                           300                            330                             360 <br />              AATGAATCTATGGTAGGTGG                                                                              TATAACATATGA <br />        81    N E S D L V G G T N E G S P L D N                    Q    V K L E H E M K            K L E E T O N N Q L N <br />                                            390                           420                            450                             480 <br />              CTCTTCGAGACTACTCAGlTTGTCTGGA5       ATTA AGTCAGGTAAAGCTCCTGGCATTATATCAG <br />                                                 0A                                     TGGTAAGGCT <br />    121       L L Q E D H H L S S L S D W K L N E L K S G K V L L S S M N D Y N V L G K                                            510                           540                            570                             600 <br />              GACGATAATATAAGAAAAAAGCTA_C                                        CGTTGATAAGG   CGAATGaAAAATAAGTAATGCACAAGAGCGAGGATG <br />    161       D   D N     R K   Q <br />                                  K T D <br />                                 D K  E            Q     T S M L N S D K N E N K S N N|  KS R M <br />                                       630                         660                       690        700 <br />              CTTGCTATGGCAAGGCG^AAAAAGAAAATGAGTGCTAAGAATACTCCCaAAATCCTGAATTACM u _AAAGATTCAG <br />    201       L M R K K K1M S K N T P K H P V D T E S S V S K T L L N G K N M T N S <br />                                        730                        760                       790        820 <br />              GCTGCATCGCTAGCCACTCTCCCACCAGTAACCAGCTAAATCCCAAACTGGAGATTACTGAACAAGCCGTGAAGTAAACTAATAAATCACAGTAAGGCCATTGCTAGAACAA <br />    241       S L T S P T S N O L N P K L E T E                  Q D E S K L M E S T V R P L L E Q <br />                                850                 880                                         910                         940 <br />              CACGAAATTGTTGCTGGCCTAGTTTGGCAA CCAAGGCATTTATGAACTACTTTAAAACTAGAGAAAATTGGGAAATAAGGCATGGAGCTGCGTTGGGTCTTCGAGAATTG <br />    281       H E V G L V W Q F Q G Y E L L L D N L M S E N W E R H G L G L R E L <br />                                       970                       1000                       1030                       1060 <br />              GTAAAAAAACTGCTTATGGAGTCAGCAGGGTTAAGGGGAATACCCGAGGACAACCTGAGGAATTCTAGAAGCTTAGAAGACTTGGCGTCCCGTCTACTTCGTTTGAT <br />     321      V K K H Y G V S R V K G N T R E E N N L R N S R S L E D L S R L L T V F L <br />                                      1090                       1120                       1150                       1180 <br />     361 <br />              GTAGT     GTATAGIAGTCAGTTGTGGCCCCTGTACGAGAGTCAG TTGCGCAAACTCTAGCT GCGTTATTAA TTCACCTAGACAGTACCTTATCAATTAAAATATTTAAT <br />              D R F G D Y V Y D T V V P V R E S V Q T L L L H L D S T L S K F N <br />                                1210                1240                1270                1300 <br />                                                                   LCTTTTAGGGA RVKGTTACTTCGTCAGTAGACTC <br />              TGLTTCANTGGTTCTACEAGATCCACTACAAACAGGTCTACCGATTCAT GGAG <br />     401      C L E Q L V L Q D P L Q T G L P N K W E T H G G L L G R Y F V S K T N F <br />                                        1330                          1360                       1390                      1410 <br />              CTATTTGCCCATGGCCTTTAGAAAACGTTGTCCGTATAGTGCTATATGGTTAACAATATAGC=IGTTQAAAGCGTAGCAGCCTCAA=AACCCAATAACCAGCAATT <br />     441      L F H G L L E N V V R V L Y G L N Q S D D D V Q S V S L T P T S E F <br />                                        1440                          1470                       1500                      1530 <br />                  ;CGBTACgkTGcATGTAT                                GCTIG     GAIGATGAT AIC GCGTAGGTTCT   TAATG  ATIGGCAAAG <br />     481      Q)K L N(N)ST  E L 9 TT)W S LUA R(9D D DGS                                       AKVG0  I0  D               L <br />                                        1560                          191620                                                10 <br />              TGCGACCCAG1SATA _CTAGAGTGGTCATTCAAGTCACTG;TCGA TTP-GPTATSTTCCTTTTTCGCCATTCGTC <br />     521      C D H Q E V L D L K N K L E H P S E W S F K S L V P K L Y P F L R H S S S <br />                                        1680                          1710                       1740                       1770 <br />              GTGAGGAGGGC          T        CATTTATC   AATAAATGGATGATCTAAAA1ACTGGTTAAACGGCAAGGTATTCCG            _             561   V R R V L N L L F L S K D D S T K N W L N G K V F R L V F Q N L L E Q <br />                                        1800                          1830                       1860                       1890 <br />              AACCCGGAACTATTGCAACTGTCTu  CGATGTTATGTTGCGTTATTGGAGCATTATAAAGTTAAGCATACCGAAAA_C TAGATCATGTTTCGACTTGCAGCCAATACTA <br />        601   N P E L L Q L S F D V Y V L L E H Y K V K H T E K T L D H V F S K H L Q P L <br />                                        1920                          1950                       1980                       2010 <br />        641 <br />              CATCTTCTAAATACGCCAGTAGGGGAA.AAGGGAAAGAATTATGCCATGAT=CACATTCTAAAGCCTTCCCACCATAAT <br />              H L L N T P V G E K G K N Y M E S Q Y L K P S Q H Y Q L H P E K K R                                         S <br />                                                                                                                               TATATCTGAA <br />                                                                                                                                     S E <br />                                            2040                          2070                          2100                             2130 <br />              ACCACAACAGATTCTGACATTCCACCM                     TCAATATCG     ACCTATIGCTGGAGA TTA           AGGATIGGAC TAATA        CACTAGAATA <br />        681   T     T T D S D P            P6KIN <br />                                               (                                              NNE  D P M( GD()T L(L)G                                        L D    V )T          R                                            2160                          2T0                    ~2250 <br />              &amp;TG Cm        ACATTGTCAATG TTCAAGATTCCACACTTCAATCCTT TCA <br />                                                                      T   AAACGTTT GTTGlGTTTAGAGCTGCCATTCTCAACTCCACGTATGTTG <br />        721      G K F L T L S M F Q D S T L Q S F F T N V L V R C L E L P F S T P R M L <br />                                      2280                           2310                     2340                        2370 <br />              GCAGGCATTATTGTTCACAGTTTTGCTCCTCGTGGTTACAAAAGCATCCAGAAGGGGAAATC.L%-TTACTGCTTATCA AGAmTTTTCATC TGTGATGAATAACGTCGA <br />        761   G I V S O F C S S W L O K H P E G E K L P S F V S E F S P V M N KO                               L L N <br />                                      2400                           2430                     2460                        2490 <br />              CGCGATGAATTCCTGTATTAGGGAGCTAGTACCTAGCTTAAAAGCTCTGAGGACTATTAACCTACTGGTACCTTTGTGGATGTAGGTATGTTACCTATCACAC    C <br />        801   R D E F P V F R E L V P S L K L R T Q C Q S L L T F V D V G M L P Q Y K L P <br />                                      2520                           2550                     2580                        2610 <br />        841 <br />              AACGTAGCAATTGTGGTTCAAGGCGAAACAGAAGCCGGTCCACATGCGTTTGGTTTAAACAGAGAATTACGGCGAATACThG <br />              N V V V 0 G E T E G P H F G V E T E K V Y G E Y Y D K M F K S M N N S <br />                                                                                                                _AAGTCAACCTGAACAATCT <br />                                2640                2670                2700               2730 <br />               YTACAKACLATKKPEAAGA           GTATG <br />                              AGKHATTAGRVAGMATCTGC   INSATC AAAGAAT C AA K RCTTCGTGGSCITTCTCGCTAATTACGSTC <br />                                                                            L <br />        881    Y K L L K K P L E D S K H R V L M N S K E S K L R T G S L N Y S <br />   FIG. 4. Nucleotide sequence predicted <span id='am-61' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-67' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> sequence MOTI. Nucleotide positions shown line; amino acid <br /> positions shown left line. Amino acid sequence letter code nucleotide sequence. potential nuclear <br /> targeting signal boxed. 568-residue domain (amino acids 1257 1825) homology helicases SNF2 RAD54 underlined. number segments amino-terminal domain, example, residues 481 520 689 721 (circled), fit 4-3 hydrophobic repeat diagnostic amphipathic helices capable forming coiled coils (11). <br />   921 <br />  <br />  961 <br />  <br /> 1001 <br />         TCCATrC <br />         S L L <br />  <br />  <br />         H <br />  <br />         GMT1ATACAGAAAMTATAGAAT <br />         E <br />                 L <br />  <br />  <br />                 Y <br />                       <br />  <br />                      K E <br />                          = _~ ~ T TG TC G TA GAT GTCA TC G  <br />                          Q <br />                               F <br />  <br />  <br />                               Q <br />  <br />  <br />                               K <br />                            3120 <br />                                 3000 <br />                                       D <br />  <br />  <br />                                       L <br />  <br />  <br />                                                                                 AATACCAAACACCCATTATCAGGTCGCTAATGGACAGTGTTAAAGAGAGGAATGAAAGTACAAACGATGGCCGGAGAACGGTCGTA <br />                                           G <br />  <br />  <br />                                           L <br />  <br />  <br />                                           L <br />                                            TCTTA <br />                                                L <br />  <br />  <br />                                                E <br />  <br />                                                T <br />                                                 3150 <br />         ACGAAATACTATGAGTCTTGCGCTTAAT TAAGAAATCTCAGAT <br />                                                     P <br />  <br />  <br />                                                     N N <br />  <br />  <br />                                                     L <br />                                                          2760 <br />                                                           L K <br />                                                          2880 <br />  <br />                                                           3030 <br />                                                               K E <br />                                                                      L N <br />  <br />  <br />                                                                 K V N V S <br />  <br />  <br />  <br />  <br />                                                                      T <br />                                                                              P <br />  <br />  <br />  <br />  <br />                                                                          S N S <br />                                                                            F <br />                                                                                   <br />  <br />  <br />                                                                                  3060 <br />                                                                                           <br />  <br />                                                                                          G <br />  <br />  <br />                                                                                                                                                                                           R <br />  <br />  <br />                                                                                                  K <br />                                                                                                      S <br />  <br />  <br />                                                                                                                                                                                                                 2790 <br />                                                                                                           L M D <br />                                                                                                            2910 <br />                                                                                                           V K N <br />                                                                                                          3090 <br />                           AAAGAATCTAATTCGATTGCCGCACAAGATGACATTAATC GCGCAGATGTCGt;AAGAGGCTCAACTAAAAGAGGGGTGT <br />                                                                                           E Q L K R K G G L                                                                                          3180 <br />                                                                                                  Q D          D <br />                                                                                                      GAAAGCAAG3AAGTT <br />                                                                                                                       S <br />  <br />  <br />                                                                                                                       L <br />  <br />  <br />                                                                                                                                                                                                                                                 V K <br />  <br />  <br />                                                                                                                           C <br />  <br />  <br />                                                                                                                           N <br />                                                                                                                               G <br />  <br />  <br />                                                                                                                               L K M <br />                                                                                                                              3210 <br />                                                                                                                                   E E <br />  <br />  <br />                                                                                                                                   F   L <br />                                                                                                                                            R N <br />  <br />  <br />                                                                                                                                            C    V D <br />  <br />  <br />                                                                                                                                                 S <br />                                                                                                                                                     E <br />  <br />                                                                                                                                            ATCTGAATCAGAGTCATrCGTAATC <br />  <br />  <br />                                                                                                                                                     E <br />                                                                                                                                                         2820 <br />                                                                                                                                                          K L Q T M G <br />                                                                                                                                                         2940 <br />  <br />                                                                                                                                                          T S       E    V P     D   F <br />                                                                                                                                                                                         E <br />  <br />  <br />                                                                                                                                                                                         S <br />                                                                                                                                                                                             S V V <br />  <br />  <br />                                                                                                                                                                                             V N                                                                                                                                                                                                  2850 <br />  <br />  <br />                                                                                                                                                                                                  2970 <br />  <br />  <br />  <br />  <br /> 1041    T L K L F E V L G P S L Q K L P Q L R S L F D S L S D H E N E E S K V D <br />                                  3240                        3270                        3300                                3330 <br />         AACGAGCAAGGTC:AAAAATAGTCGACTCATTGGGACTAAGAGCTCTATTCCCATTTATGTCTGATCCTACGTrCTAGTGAGGT lACCCGlCCCXToTMGT <br /> 1081    N E Q G Q K V D S F G V L R L F                P  F  M S  D  S  L R   S S  E  V   F   T   R    F   P   V L  L  T  F <br />                                  3360                        3390                        3420                                3450 <br />                            R=GTTCARTTAAVCTCCGTGGAGGTAATCACCATACG                                                          ATG <br /> 1121    L R S N L S V F R Y S R T F D L K S S V E V M Y T R E L P L M N <br />                                  3480                        3510                        3540                                3570 <br />         AGCGCTGGTT&apos;CTCTATCCGATCG             _                 GAAACCGATGTTCTACCTTACGC           ATGTCAl                    GC <br /> 1161    S G S L S D R Q G S T E L Y H L S L S M E T D V L P Y V F L V P L L G R <br />                                  3600                        3630                        3690                                3720 <br />         ATGAGTGATTCTLCG^AAGTTAGATTTOAGT           ATMGTCT               TAG     AAGTGGATCGCCGATCCAAAGACTCCCAGAAGAACAGTG <br /> 1201    M S D S N E D V R N L T T T F S I K L V P L E G D P K G L P E E L V <br />                                  3750                        3780                        3810                                3840 <br />         GCAGAAAGAGGGAGCGTGATTTTwATCCAACQAATGTGATCCTTCTAAAGCTAAGCCATTTAAATTACCCATTGCCA           lcA^GGCGACTTAAGAAAATATCGAGAGTT <br /> 1241    S R E R E R D F Q0            M M D P S K K P F K L P I K T L R K Y Q 0 D G V <br />                                  3870                        39ou                        3930                                3960 <br />         AACTGGTTGGCATTCCTGAAATCCAAGATATTGTTGACGATATGGGTTGGTAACATTACAACCATGCTATCGCACACAATGGAGA <br /> 1281    N        W   L F            L   N    K     Y    HL G K T L Q T C I S D Q Y L R K E <br />                                                                 L    H   G     L C             D   D    M G <br />                                                       4020 <br />                                                          3990                     4050                          4080 <br />         GATTATGAAAAACGC     CGGAGGCTAQ               CCCCCATCTTTAATGG CCCATGG      GATTGACAGTATGCTCCT AAAGTT <br /> 1321    D Y E K T R S V E S R L P S L I C P P S L T G H W E N E F D Q Y P F L K V <br />                             4110                      4140                        41-10                         4200 <br />         GTTGTGTATGCCGGTGGCCCAACT                     GGTAA        T G T          ACC           CAGTCCTAACAAAC <br /> 1361    V V Y G G P T V R L T L R P Q L        SD      DII   VT   S  Y    DV      RN   DL    AV   L N   K  T <br />                             4230                      4260                        4290                          4320 <br />              TATAATTATTGNHRL <br />                     GAA                                                                 CTGTTAALTTTAACCGGACT CCCAT C <br /> 1401    E Y N Y C V L D E G H I K N S Q S K L K V K E T  N H R L L T G T P                             4350                      4380                        4410                          4440 <br />         CAAAACAACGTTTTAGAATTGTGGTCTGTA CTTCTTAATGCCGGGATTTGGAGGAAA   CTCCAGGAAGGTTCGCAA GCCATGGCCAAAATAGTAAA <br /> 1441    QN N V L E L W S L F D F L M P G F L G T E K M F Q               ER     F  AK      PI   AA   S  RN    SK <br />                              4470.                    4500                        4530                          4560 <br />         ACACATCTAAGGAGC=GGAAGCGTGTACTGGCTTlTGGAGGCACTACATAAG;CAAGTTCTACCTTTCA7CTGAlAAGATGAGAGACGTTCTATCTGATTACCACCTAAAATT <br /> 1481 <br />                                                          4590                                               4620                                        4650                                     4680 <br />         ATTCAGGATTACTACTGTGAATTAGGTGATTTACAAAAACACTATACTGGAAG <br /> 1521           Q D     Y     Y      CL Q K Q L Y M D F T K K 0 K N V V E K D E N S E D G K <br />                                           E    L G D <br />                                   4710                        4740                       4770                      4800 <br />         CAACATATTTTCCAAG TACAATACATGAGAAAATTGTGTAATCATCCAGCTTTGGT&apos;CCTTTCACCAAT ACCCGCAATTAGCGCAAGTAAGCTATTAAGCAACTGTCTA <br /> 1561    Q H F Q L Q Y M R KLCNHPALVLSPNHPLAVDYLKTGL <br />                                   4830                        4860                       4890                       4920 <br />         GATTTACACGATATTATCAACGCTCCAAAACTAGCGCATTGAGAACATTACTC=GAGGGTATAGGTGAAGAAGATATCGACA AAAA MGCAAGCCAAGTAA7CTATT <br /> 1601    D        L   H D              N    P        K L        S   L R T                 LC G G E E D D K K S Q D Q N F P                                                                                                      L    F E <br />                                                          4950                                4980                     5010                    5040 <br />         CAATTAACCACA&gt;CAG AGCCT          t                                                AGACAGGTAAGC <br />                                                                              CATAAGATGCTACOT            TAAAAAGlTATATGCCCTCCGTAACCTATATGAGGCTAGAT <br /> 1641    Q N V S QHRALIFCLKDMLDMVENDLFKKYMPSVTYMRLD <br />                                  5070                        5100                          5130                           5160 <br />         GGAAGCATGCCCAAAGACAGAACAAGTTGTTCGG <br />                                          AAATTTAACGAAGATC CTCTA   TTGATTGCCTACTGTTGACCACCAAGGTCGGAGGGCTGGGTCTG;AATTAACTGGTGCA <br /> 1681    G S D P R D R Q K V V R K F N E D P S D C L L L T T K V G G L G L N L T G                                                           5190                          5220                   5250                  5280 <br />         GACACC                TAT=&gt;GA                                                                             TTTACAGAATATTACG1AAAG <br />                                                                      CAATGAATGATCTACAGGAAGACAGAGCCATAATGTAAAAGTTAATG <br /> 1721    D        T V       W N P M N D L <br />                              F      V   E     H    D                           Q M D R H R IG O K K V V N V Y R         K G <br />                           5310                                                                              5340           5370                5400 <br />           ACV7TTGAGAAAAA7CTGG                                                                                             CATCCATGGATAAACGTCTACT_ <br />  1761   T L E E K M G L Q K F K M N                                                          S T V V N Q Q N S G L S M D T H Q L L D L F <br />                                  5430                                                                  5460                5490                5520 <br />         GACCCCGATAACGTTACCTCACGGACAATGAGGAAAAGAATAATGGCGATTCCCAAGCAG;CCAAGGATGGAAGATATTGCCAACGAGACTGGTTAACTGGCAAAGCCAAAAGGC <br />  1801   D        P   D N V T                  S Q D N            E   E   K N N               G   D       S Q K G M E D                    N      E     T   G    L   T   G K K E                                                           5550                                              5580 <br />         CTCGGTGAATGAAAGGTATGGGACCCATCTCAATACGAGGAGGAGTATAATTTAGACACCTTCATCAAAACTTTACGA <br />  1841       L    G E     L        K   E   L W D           P      S   Q   Y E         E   E       Y N      L D T F           K T      L    R <br />                                                                                                                                                                                                   5721 <br />         TAAACAAATTCTTGCATTGGAAGAlGCGTATACAAGG                                                    GCTTGTTATTTAACCmTGTATTATCAACTTG                                       TATTAAGTTT <br />                                                                                                                                                                                                   5841 <br />             C                                       TTGTTAAAATTCAG;GTTATTAGCTGTCTATTGATTATGGCTTGG                                                                        TTT             GT       <br />                                                                                                          5961 <br />             GTTGCGCAAAGGGAAACTGACGTTTAGTCGAACTCTGCA GAGCTTCATCTAAGAATTTCTrGCTATCGAAACGTTTGGAA AAAATCAAACCTCG <br />                                                                                                          6081 <br />             TTCTTC GCAATMGCATTCGCTGGAAACGZATAGACACATCTAACTVTCCGC <br />                                                                                                                                                                                                   6201 <br />                                C__C <br />             CTGACCTCGTTGAAATGCTGCT                                                                                    CAAAG           GCTAI 7TTCAAGTC                         l <br />                                                                                                                                                                             TCATA ICATAGCTG <br />                                                                                                                                                                                                   6321 <br />             TTTGATACCATACGTAATAAGAGGCACAACAGTAAAATC7GAGT=AAGAGTCCTATAGTTCAT ATCAACGTACTTAAACCC                                                                              AAAGGCAGC=ATGCAAACGA <br />                                                                                                                                                                                                   6441 <br />             TGGGAAGGTGTID AAGCGAA                        CTCTGGCllTTGATGAGTmAATTATAGATTA <br />                                                                                       TTA GAAGCAGGCTCTAA                                                                                      MAAGTA <br />                                                                                                                                                                                                   6561 <br />             AGCACGAAACAAGATTATACTCAGTATGGAGGATTTGTATTCGlT <br />                                         6596 <br />                 CAGCA ICAAGAAGCAGCAGTCAAGACTTCTTT <br />                                                                                              FIG. 4-Continued. <br />                                                                                                           1885 <br />  1886     DAVIS ET AL.                                                                                           MOL. CELL. BIOL. <br />  growth phenotypes associated original motl-l         simply indication MOTI transcript isolate segregated 2:2 cosegregated transcrip-       unstable G1-arrested cells growing cultures. <br /> tional phenotypes associated original motl-l muta-          MOT] essential cell growth. MOTI gene disruption <br /> tion;  motl-l allele represented alteration   constructed recently devised y transformation single nuclear gene. MOTJ gene cloned            procedure (65). motlA1 allele prepared subclon- <br /> YCpSO yeast genomic library (59) complementation            ing 474-bp SacI-BamHI fragment (nucleotide positions slow-growth/temperature-sensitive phenotypes.          3539 4013) 1,058-bp HindIII-ClaI fragment (nu- <br /> complementing plasmids isolated fell        cleotide positions 1286 2344) (Fig. 3 4) tandem distinct overlapping classes based restriction map-        reverse order integrating plasmid pRS306 (URA3) <br /> ping. complementing region localized approx-        (65). construct linearized unique restriction site <br /> imately 8-kb BglII fragment (Fig. 3). demonstrate     fragments (HindIII) introduced cloned DNA corresponded MOTI locus, following          unrelated diploid strains (YPH501 1788) DNA- <br /> experiment performed. SacI fragment (Fig. 3)            mediated transformation. resulting disrupted gene taining complementing region plus flanking se-        predicted 1,194 bp (398 amino acids) quences subcloned integrating plasmid pRS306         ClaI Sacl sites (Fig. 3 4) deleted replaced (URA3) (65). plasmid linearized unique     plasmid sequences. BamHI fragment spans site <br /> HpaI site unique BglII site subcloned           deletion/insertion mutation (Fig. 3) used fragment (Fig. 3), target integration homologous        probe detect wild-type disrupted MOT1 <br /> genomic locus, introduced AL4 Ta strain (YPH499)       alleles Southern blot analysis (67) genomic DNA <br /> (65) DNA-mediated transformation. Ura+ transformants           isolated recipients Ura+ transformants selected. transformants exhibited phenotype,        digested BamHI (Fig. 6). wild-type allele representative crossed AL4Ta motl-l strain       migrated 3.5 kb larger deletion/insertion allele <br /> (JD102C). Tetrad analysis resulting diploids showed        migrated 6.5 kb, confirming diploid strains site integration allelic MOT] locus;    heterozygous motlAl mutation. heterozy- 22 tetrads examined, slowly growing, Ura+       gous diploids subjected tetrad analysis. 11 12 normal-growing, Ura- recombinants. Hybridization          10 11 tetrads strain, respectively, 2:2 <br /> BamHI fragment MOTI coding region (Fig. 3)            segregation live spores:dead spores observed (Fig. 6). <br /> chromosome blot  Fig. SC), ordered set           live spores Ura-. spores did cloned yeast DNA fragments bacteriophage X vector            form visible colonies examined microscopically.  Materials Methods), revealed MOT] resides        cases, spores germinated yielded microcol- <br /> chromosome XVI. genes affect transcription,        onies consisting 10 20 normal-looking cells.  <br /> SIT3 (2) SPT14 (21), reside chromosome XVI.           MOTI essential gene required cell growth. motl-l mutation allelic loci,   result supports conclusion Motl affect indicated following observations.  plasmid-     expression genes outside mating response <br /> borne copy SIT3 (provided K. Arndt) does com-           pathway pheromone-responsive genes <br /> plement motl-l, cloned SIT3 MOTI DNAs                  described <span id='am-14' about='oboInOwl:date' typeof='owl:Thing'>date</span> nonessential. course, Motl cross-hybridize (data shown). Second, SPT14 encodes         involved additional cellular processes critical <br /> small transcript tightly linked MFWl (21),      cell viability. <br /> MOT1 encodes large transcript 6.2 kb  Fig. 5A)          Analysis MOTI coding sequence. putative pro- unlinked MFoxl (data shown).                            tein sequence Motl compared sequences    Nearly 7 kb complementing surrounding             GenBank data base, using TFASTA algorithm (56). regions subjected DNA sequence analysis.               search revealed (Fig. 7) Motl 40% identical nucleotide predicted amino acid sequences shown        stretch 530 amino acids S. cerevisiae Snf2 <br /> Fig. 4. large open reading frame 5,601 nucleotides      protein (47), involved transcriptional regula- <br /> 1,867 amino acids (predicted molecular weight 209,989)         tion glucose-repressible invertase gene, SUC2. Cells identified. significant open reading frames      carrying snf2 mutation unable derepress SUC2 opposite strand. analysis deduced primary        transcriptional level grown carbon source sequence presented later.                                      sucrose raffinose. addition, like motl-1, snf2 mutations <br />    MOT] expressed cell types. examine expression     pleiotropic, indicated defects galactose MOTI cell type, internal 3.5-kb BamHI               glycerol utilization effects expression acid phos- <br /> fragment (Fig. 3) used hybridization probe blot     phatase, non-glucose-repressible gene. Furthermore, containing poly + RNA isogenic MATa, MA             SNF2 message present cells grown <br /> MATa strains MOT1+ motl-l.             repressing nonrepressing conditions.  <br /> probe detected single transcript 6.2 kb       unlike MOT1, snJ2 null mutation lethal present cell types level (Fig.     identical phenotype point mutations. Recent evi- <br /> 5A). size transcript good agreement    dence suggests Snf2 nuclear protein (47), size predicted open reading frame.           consistent proposed role transcription. Motl <br /> MOT1 message motl-l cells higher        protein sequence motif (residues 195 209) corresponding isogenic wild-type strains (Fig. SA), sug-      serve nuclear targeting signal (57). <br /> gesting autoregulatory mechanism involved             Motl 33% identical 556-amino-acid domain MOTI expression.                                               S. cerevisiae RAD54 gene product (18), exactly <br />    Expression MOTI transcript response ao-factor     region Motl aligns Snf2 (Fig. 7A). examined. decrease MOTI mRNA                 Examination alignment shown Fig. 7A revealed factor treatment MOTI+ cells motl-l           regions amino acid identity clustered, cells (Fig. SB). clear experiment     significantly identity intervening sequences. decrease observed direct effect ax-factor action     clusters matches exactly site  VOL. 12, 1992 <br />  <br />              rT, <br />                     .            Ii <br />                                                PUTATIVE HELICASE AFFECTING GENE EXPRESSION GROWTH <br />  <br />                                                                                  kb <br />                                                                                    6. 6. <br />                                                                                              1    2 <br />                                                                                                                                          1887 <br />  <br />  <br />  <br />  <br />                                                      _m                            4 4. <br />                                                                                      . <br />  <br />  <br />  <br />  <br />                                                                                    2 3 <br />                                                                                      . <br />  <br />  <br />  <br />  <br />                                                                        B.          kb <br />                                                                                    6. 6. <br />                                                                                    4 4D <br />                                                                                      . <br />  <br />  <br />  <br />  <br />                                                                                    2 3- <br />                                                                                      . <br />  <br />  <br />  <br />  <br />                                                                           FIG. 6. Tetrad analysis MOTJ/motlAl strains.   Hybridiza- <br />                    ,,,   -7 <br />  <br />                               ---7 <br />                                      -   7 <br />                                                                        tion analysis (left) tetrad analysis (right) heterozygous <br />          -&apos;-17                                                       MOTI/motlAl derivative strain YPH501. (B) Hybridization anal- <br />                                                                        ysis (left) tetrad analysis (right) heterozygous MOTI! <br />              Al-                         4m <br />                                                                        motlWl derivative strain 1788. Southern blots, genomic DNA <br />                                                                        isolated wild-type (MOTI/MOTI) parent (lane 1)                                                                        heterozygous disruption strain (MOTJ/motlAl) (lane 2), <br />         *:.::.-,- <br />             _E                                                         digested BamHI, probed MOTI BamHI fragment <br />                                                                        (Fig. 3). wild-type allele migrates 3.5 kb; motiAl allele <br />    FIG. 5. Expression MOTI mRNA. internal 3.5-kb BamHI           migrates 6.5 kb. Positions migration HindIll markers fragment MOTJ (Fig. 3) used probe.   Poly &apos;          shown left blot. tetrad dissections, spores <br /> RNA MA Ta, MA4Ta, MA Ta cells. relevant genotype        tetrad placed rows left right indicated lane; strains STE12&apos; isogenic   center streak growth. tetrad dissection yielded ALT MOTI loci. Migration molecular weight markers        1:3 segregation live dead spores, presumably result shown left. PKCI probe  Table 2) used       gene conversion event. <br /> control uniformity RNA loading. (B) Effect pheromone <br /> treatment. Left lanes, RNA AL4 Ta MOTI + cells; right lanes, RNA MATa motl-l cells. CMDI probe  Table <br /> 2) used control RNA loading. (C) Chromosome blot          vitro (71). sequence signatures seven <br /> showing hybridization MOTI chromosome XVI.                       domains distinct SFI SFII, clearly <br />                                                                        related (28). composite consensus domain (28) <br />                                                                        shown Fig. 7B. known putative heli- <br />  nucleotide-binding sequence diverse enzymes             cases described date, domains  II, VI  utilize ATP <span id='am-6' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span> (76) (domain  Fig. 7B).             highly conserved, domains Ia, III, IV, V   B site nucleotide-binding domain (14)                conserved. cases, degenerate forms  (domain II; Fig. 7B) present. Domain II thought         domains identified (28). Mg2e-binding domain interact Mg2+-                       homologies MOT], SNF2, RAD54 <br /> <span id='am-56' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-57' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleoside</span> <span id='am-43' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-44' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>triphosphate</span> conserved Asp residue              coding sequences cluster regions perfect,  (14).  likely Motl, Snf2, Rad54        near-perfect, matches seven diagnostic helicase <br /> nucleotide-binding proteins.                                           domains, correct order     Motl, Snf2, Rad54 putative helicases.              appropriate spacing. cases  example, domain <br /> examination Motl, Snf2, Rad54 alignments                 III), sequences Motl, Snf2, Rad54 match vealed nucleotide-binding site matched best             consensus better does Rad3 sequence. Domain <br />  sensus kind site helicases (27). fact,    IV Motl, Snf2, Rad54 interesting stand- <br />  Motl, Snf2, Rad54 sequences possess seven              point intermediate consecutively placed signature domains diagnostic             previously recognized superfamilies. sequence RNA DNA helicases (27, 28, 37, 49). primary                 proteins begins F residue defines <br /> sequence proteins begins B              domain IV SFII, rest sequence sites nucleotide-binding domain, followed               similar proteins belonging SFI. particular, domain IV <br /> additional motifs. seven helicase domains              Motl Snf2 quite similar domain IV delineated comparing sequences 25 known                 helicases Semliki Forest virus, Sindbis virus, putative RNA DNA helicases variety                   brome mosaic virus  members SFI) different organisms, including Escherichia coli, various               sequence R-R-X-X-X-D-V (28, 37). <br /> RNA DNA viruses, S. cerevisiae, Drosophila melano-                    align domain V Motl, Snf2, Rad54 domain <br /> gaster, humans (27, 28, 37, 49). helicases              V Rad3, gaps need introduced (Fig. 7B). split superfamilies, SFI SFII, based          Gorbalenya et al. (28) chose delete corresponding <br /> amino acid sequences seven domains (28).                residues Rad3 means maximize alignment Proteins belonging SFI include viral proteins           Rad3 putative helicases. Apparently, spacing yeast mitochondrial protein Pifl (37),              conserved residues domain V somewhat <br /> shown DNA helicase activity vitro (46). Proteins            variable. <br /> belonging SFII exemplified Rad3 (28),                 Finally, domain VI, Motl, Snf2, Rad54 demonstrated DNA helicase activity                lack G residue normally proteins belonging  1888   DAVIS ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />                                 YSCSNF2 (751) <br />                 YSCMOT1 (1257) <br />                                              IMDIKIJQ   LVG6G6KDfIDIUL5VSEFNNUI <br />                                              SIAKP FEL-AI KA   3RKUQDUVNELAFKFHKYI <br />                                                                                                                          M5liuWS N----- <br />                                                                                                                        CEDEUEC ----- <br />                 YSCRAD54 (294)               YLEAEA FNTUSEDPLKSDEKALTESKTEQNAJRGAYG                                    DNEf3CjOA3&quot;WTLL <br />                 YSCNSF2 (806)                ------------------LT           IYENKhIRGPYEVIVPUSTuSNWSSEFAKW.-------- <br />                 YSCMOT1 (1311)               ---------------.IASDQRKEDYKTRSVESRAEPSUIICPPSLTGHWEIEFDQYI <br />                 YSCRAD54 (354)               RQGPQ6KRLIDKCIIVCPSSLVNNWA                LIKVUGPNTETPUAVDGKKSSNGIJTTVSQU <br />  <br />                 YSCSNF2 (840)                                             VFT FVEY IKFRALELSffVKR VHI                                              OTERT FKSIINEIKAKAKI RAGEFfI <br />                 YSCMOT1 (1356)               *FEKVVVYAUGUTV1LTLRPQLSDADIIETSYDVARNDL--AVLNETEYNYCVLEIUfII <br />                 YSCRAD54 (414)               IHAWAQAQGRNIVKPVLIISYETLRRNVD                   ------------. KNCNVGLULA                <br />                 YSCSNF2 (900)                  DAEaSLTINTHYYDHELEEINMVLEKI FNEVMSEDEWUNTE <br />                 YSCMOT1 (1414)               MS~ IDV K- E TON H        Vl ISFDILNMFTETEMQERUAKI <br />                 YSCRAD54 (461)               QIGDELT FTMUDS SCPR-IIVMS <br />                                                                       JDlDSUY FMSUSNMLRAEORKNDENE <br />                 YSCSNF2 (960)                FUNTGGQKI ELSEIETULVIRRIERVLRLEEIIKEMENDUVEUVEKUKNSAU <br />                 YSCNOT1 (1473)               uASuNSKTSSIQAUVEA!E Q        LM     fErLSDMPFI IQDYYBEUGDU <br />                 YSCRAD54 (520)               DLRGMDAUATDIITIQEAQUQIJSTIUSKI IIETNDILAUYCEYEHUI FVNJKPU <br />                 YSCSNF2 (1020)               QImJQQMYMRL FIG LN6NVNLRGFNN PIGVF-1IE   DFV FEEEEDQINITRE <br />                 YSCMOT1 (1533)               KQIUDFTICQKNVVEIIEUSEIADGKQHIFEALIYNREEAIAj---ILSPNHIQLA <br />                 YSCRAD54 (580)               NEINKLIESIEV---J-KVVEGVGUSQPL-RAUGI- ENNIjLNFEDEFDDEDDL <br />                 YSCSNF2 (1078)               TNDJIWRVAUKFELLDRILPKLUATGHVIUFQTIQIENMEDFLRUIMIKY .------ <br />                 YSCMOT1 (1590)               QVQUYLKQTULLHDI --APjLEAEDTC6ECGIGEEMIDKKAEQDEDFPIQNVISQ <br />                 YSCRAD54 (634)               ELPFDYNNPUSKARDVQTKYSAUFEIIERFHKIKfESDEKIVLINUDQTLD---UEK <br />                 YSCSNF2 (1132)                                                 RC UCobJLULhAh3 L1l <br />                                                                           vvcj3 <br />                                              ------------------------------- <br />                 YSCMOT1 (1647)               HRALIFCQLKDNLDNVENDLFKKYNPSVTYIEflSIDPRDWVEIcK-EIPSIDILE <br />                 YSCRAD54 (691)               MCRYKHYSAV     ---------------U!lTSINKI133DREDLEGQEFIM <br />                 YSCSNF2 (1161) <br />                 YSCMOT1 (1706) <br />                 YSCRAD54 (730)               M5CCMIMIMN R LULIPlPAAsL AllVWMDlD C FIFHSEII <br />                 YSCSNF2 (1221)               *LEUAYKIQLDEDGaIQUGKFDNKSTMEEEEALERSLLDAEEERRKKRESfEEEEELKD <br />                 YSCMOT1 (1766)               PMGLDKFENDAUTUNQQNSGLASNDTHDLDDDPDNVOSQDNEIKNNEDSQAUKqE <br />                 YSCRAD54 (790)               BFQIS?jSLSECCDIKEDVERLFESDNERQKQKNENI CETHETYHCKRCNIQEKQ <br />  <br />                B                                                                                     Ia <br />                YSCRAD3     (39)      MEMPSETVSLL                              (71)     fCSRTNSIIEEAM <br />                YSCRAD54 (332)                                                 (374)     LVNNWAMLIIWEG <br />                YSCSNF2     (789)                                              (808)    *YENJI R6PYM <br />                YSCMOT1     (1294)    mcc                                      (1316)    MRKEDYIKTRSVE <br />                consensus             O                G GKT             0               00              0       ZO <br />                                                           S                                   T <br />  <br />                                                        II                                                 III <br />                YSCRAD3     (226)     VSUDSIVUFMAWNE                           (457)     I1TISPLNYPRNLNU <br />                YSCRAD54 (446)        NCNVGLMLAINWCL                           (480)          VHNESEY FM <br />                YSCSNF2     (885)     EVUWVHMIEIIIN                            (920) <br />                YSCMOT1     (1399)    ETEYNYCVLEIH                             (1433) <br />                consensus                           OOOODE H                           0 OTAT                       OZ <br />                                                             D                              S6S <br />                                                        IV                                                               V <br />                YSCRAD3     (539)     VDFPSYEYNES IV                           (590)     CSNGRGAIVARDCVS EDFDHUYGRL <br />                YSCRAD54 (549)        KUII*TNDILAE                             (721)     EEGQE F -UCID--IENRLI <br />                YSCSNF2     (989)     MLINKMIEE                                (1152)    MSEYLMIR--ULEl--U ED <br />                YSCMOT1      (1502)       MEEIIEL S                            (1697)    EUPSIDELUTENVE--ELUEII-T-T <br />                consensus                 F           S                                        OOO SZO               G ZO    OO <br />                                          Y           T                                                    T                 S <br />  <br />                                                            VI <br />                YSCRAD3      (658)    ANRHAAUCEGULUGKD <br />                YSCRAD54 (755)        MAAMQMVMD <br />                YSCSNF2      (1186)   MHR_ <br />                 YSCMOT1     (1731) <br />                 consensus                         Q     GR      R <br />                                                       H <br />  VOL. 12, 1992                                 PUTATIVE HELICASE AFFECTING GENE EXPRESSION GROWTH                                      1889 <br />  <br />   FIG. 7. Homology MOTI SNF2 RAD54.   Alignment C terminus Motl Snf2 Rad54. alignment generated using FASTA algorithm (56) optimized visual inspection. Gene designation far left, followed amino acid position parentheses. Amino acids depicted letter code; identities proteins presented white black letters. (B) Comparison helicase domains Rad3, Rad54, Snf2, Motl seven consensus <br /> helicase domains. Gene designation far left, amino acid positions parentheses. Amino acids depicted letter <br /> code; identities proteins given white black letters. consensus sequence domain described Gorbalenya et al. (28). 0, hydrophobic residues   L, V, M, F, Y, W);  charged/polar residues (S, T, D, E, N, Q, K, R). <br />  <br />  <br />  <br /> SFII, rest domain perfect match                 consistent data normal Motl operates consensus. Interestingly, members SFI lack G          conjunction general transcription apparatus. Fur- <br /> residue. Motl, Snf2, Rad54, domain VI, like                thermore, motl-l recessive allele likely, <br /> domain IV, appears hybrid SFI SFII                   represent loss function mutation,  <br /> superfamilies.                                                         mal role Motl remove basal transcription <br />                                                                        factors DNA. persistent occupancy                           DISCUSSION                                   promoter region gene factors motl-l cell <br />                                                                        explain enhancement gene expression observed <br />    genetic screen described designed identify          Stel2 absent. higher level expression  factors operate absence Stel2 protein,        genes supported presence specific transcriptional <br />  DNA-binding protein required transcriptional             activator, like Stel2, obscure modest effect  activity pheromone-responsive genes (16, 17, 19, 23, 24).          motl-l mutation. <br />  screen uncover mutations specific factors               MOTJ gene contains open reading frame  directly downstream Stel2, possibly belonging               capable encoding protein molecular weight newly proposed class coactivator proteins (3, 42),          210,000. C-terminal 568 amino acids Motl homol- factors operating independently Ste12              ogous corresponding amino acids S. cerevisiae <br />  affecting general transcriptional machinery, chromatin            Snf2 Rad54 proteins. proteins  structure, nontranscriptional aspects gene                seven signature sequences RNA, RNA/DNA, DNA <br /> expression mRNA transport stability. Cells car-             helicases (27, 28, 37, 49). basis following <br /> rying motl-l allele increased expression              observations, propose Motl, Snf2, Rad54 define pheromone-responsive genes exhibit low basal                new subfamily helicases.  degree transcription absence Stel2. Alteration MOTJ              sequence identity helicase domains greater discernible effect expression phero-            Motl, Snf2, Rad54 proteins <br /> mone-responsive genes completely Stel2-                   previously described member helicase <br /> dependent mode transcription. motl-l does               family. Second, proteins share additional <br /> affect pheromone-responsive genes equivalently, probable effect transcription indirect specific pheromone-regulated genes. <br />    Additional evidence presented supports view.                                    Doiain                  Domain B  cycl-lacZ plasmid lacking UAS                  TPR Consensus   AEAIFGEHIIEKLGDLEK|LDAEQK|LLLDINN <br /> exhibits higher level expression motl-l strains                             U,.                            m      Ia <br /> isogenic wild-type controls. mutations appear               MOTI     75-    APSEDPRESDEVGGTNEGSPLD   VKLEHEMKI 4/8 alter chromatin structure vivo shown                    131-    LSDEKLNEILKSGKVLLASNMNDINVIGKA D&#194;&#174;NI 6/8 <br /> suppress UAS promoter; example, cells                       286-    GLVEQF(SIY&#194;&#174;LLLDNLMSINWR HGAAL     E 7/8 <br />                                                                                 417-    NKIEEATHGGELGIRYFVSIKTN LFMHGLLUNV 5/8 <br /> depleted histone H2B H4, elevated transcription                     493-    TTIESL RLDDDISSSVGEIMDELAKLCD     V 6/8 PH05 promoter lacking UAS observed                            580-    TKNELN   M <br />                                                                                                  FRLVFQNILLEQNPELLQLSF DYV 4/8 <br /> pressing derepressing conditions (32). Second,             FIG. 8. Potential TPR motifs amino-terminal domain identified gene (DED1) (70) previously implicated           MOTJ gene product. consensus sequence derived com- way mating response expression                  parison 47 34-amino-acid TPR repeats different yeast <br /> increased motl-l background, raising possibility            nuclear proteins, defined Sikorski et al. (64) Hirano et al. numerous cellular genes affected alteration           (36), indicated line. amino acids indicated MOT1. interesting note DNA sequence                   line white black letters frequently  analysis DEDI gene product reveals            40% repeats) positions; amino acids <br /> presumptive helicase, like Motl, furthermore, appears           indicated line frequently observed <br />                                                                        substitutions positions, amino acids indicated role mRNA splicing (40). Finally, demon-               line frequently observed replacements strated MOTJ gene essential cell growth            positions. segments predicted primary sequence viability, consistent conclusion function         MOTI gene product, beginning residue indicated Motl includes vital cellular processes exclusively          number left, shown. Amino acids indicated white  <br /> confined expression genes mating pathway.                 black letters perfect matches highly conserved <br />    Stel2-dependent genes expression                residues TPR consensus; amino acids circled match <br /> elevated motl-l mutation, relative effect            residues identical position motl-l allele pronounced stel2 background                TPR repeats used compile consensus. number matches STE12 background. sense, presence             putative TPR repeats Motl highly <br />                                                                        conserved positions TPR consensus given right. Stel2 appears counteract influence               noted 47 TPR repeats used derive motl-l defect, absolute level expression            consensus sequence, distribution, terms degree match Stel2-dependent genes  course, higher STE12               conserved positions, 1/8, 2; 2/8, 2; 3/8, 6; 4/8, 6; <br /> cells. Motl DNA helicase, hypothesis               5/8, 6; 6/8, 9; 7/8, 10; 8/8, 6 (36, 64). <br />  1890    DAVIS ET AL.                                                                                                MOL. CELL. BIOL. <br />  <br /> blocks amino acid identity (1565 1579 1678 1696     4-3 hydrophobic repeat characteristic amphipathic (x Motl) helicases.            helices potential form intramolecular helicase domains Motl, Snf2,         intermolecular coiled coils (11) (Fig. 4).  Rad54 match SFII subfamily,              tempting speculate Motl nucleoskeletal protein <br /> domains, IV VI, somewhat unique       capable interacting DNA C-ter- <br /> hybrid motifs present SFI SFII subclasses.     minal helicase domain associating Based sequence comparisons, additional potential          nuclear matrix TPR units members newly recognized subfamily             formation helical bundles proteins. identified S. cerevisiae, RAD16 (61a) STH1, ho-         structural features protein serve correctly <br /> molog SNF2 (47a). fact proteins         position C-terminal helicase domain carries similar respective helicase domains       function properly. <br /> suggests involved similar areas nucleic <br /> acid metabolism, interact similar substrates                                ACKNOWLEDGMENTS <br /> accessory molecules, regulated similar <br /> manner.                                                            thank David Schild Marian Carlson communication    Helicases implicated numerous cellular       results prior publication helpful discussions; Stan Fields, <br />                                                                  Gustav Ammerer, Tony Brake, Bob Sikorski, Phil Hieter processes require DNA RNA unwinding, including          providing strains, plasmids,  Gary Anderson yeast <br /> DNA recombination repair (43, 71).  cells            chromosome blot; L. Riles M. Olson ordered yeast <br /> carrying motl-l mutation sensitive UV      contig set. <br /> X-ray irradiation isogenic normal cells (data             work supported special grant research (PF-3308) <br /> shown), making unlikely Motl involved major      American Cancer Society, California Division  J.L.D.), <br /> DNA repair pathway. cellular processes            NIH research grant GM21841, funds Lucille P. <br /> helicases implicated initiation progres-   Markey Charitable Trust, facilities provided Cancer <br /> sion DNA replication (7),. cell proliferation (26, 34), RNA   Research Laboratory University California, Berkeley  splicing (30), ribosome assembly (53), initiation transla-    J.T.). <br /> tion (1), transcription (66). mutation MOTI                               REFERENCES shown elevate transcription certain genes,     1. Abramson, R. D., T. E. Dever, T. G. Lawson, B. K. Ray, R. E. tempting speculate helicase involved        Tach, W. C. Merriclk 1987. ATP-dependent interaction <br /> transcription. motl-l allele recessive increases          eukaryotic initiation factors mRNA. J. Biol. Chem. basal level expression classes genes.           262:3826-3832. <br /> Taken  observations suggest wild-         2. Arndt, K. T., C.  Styles, G. R. Fink. 1989. suppressor <br /> type Motl protein functions, formally, negative regula-         HIS4 transcriptional defect encodes protein homol- <br /> tor transcription. function Motl normally       ogy catalytic subunit protein phosphatases. Cell strip expel resident DNA-binding proteins           56:527-537. <br /> cognate sites melting duplex DNA regions            3. Berger, S. L., W. D. Cress,  Cress, S. J. Triezenberg, L. <br />                                                                      Guarente. 1990. Selective inhibition activated basal transcriptional activators initiation factors         transcription acidic activation domain VP16: evidence bind. hand, putative helicase         transcriptional adaptors. Cell 61:1199-1208. <br /> pivotally involved different nuclear process,        4. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. positive <br /> transcriptional effects observed motl-l cells           selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br /> fore, indirect. situation account            ylase activity yeast: 5-fluoro-<span id='am-27' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span>orotic acid</span> resistance. Mol. Gen. <br /> relatively modest transcriptional effects (threefold) com-           Genet. 197:345-346. <br /> pared dramatic phenotypic effects slow-growth         5. Boyer, H., D. Roilland-Dussoix. 1969. complementation temperature-sensitive lethality conferred motl-l          analysis restriction modification DNA Esche- <br /> mutation. important determine              richia coli. J. Mol. Biol. 41:459-472. <br />                                                                   6. Brake,  J., C. Brenner, R. Najarian, P. Laybourn, J. <br /> primary role Motl transcription lies       Merryweather. 1985. Structure genes encoding precursors aspect cellular physiology.                            yeast <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> mating pheromone factor, p. 103-108.    additional features MOTI coding          M.-J. Gething (ed.), Protein transport secretion. Cold <br /> sequence worth noting.  numerous           Spring Harbor Laboratory Press, Cold Spring Harbor, N.Y. <br /> potential phosphorylation sites different types    7. Brill, S. J., B. Stillman. 1989. Yeast replication factor protein kinases, revealed search protein sequence         functions unwinding SV40 origin DNA replica- <br /> motifs Keybank7 data base (8).  Motl             tion. Nature (London) 342:92-95. phosphoprotein activity regulated phos-        8. Brutlag, D. L., J. P. Dautricourt, S. Maulik, J. Relph. 1990. <br /> phorylation.                                                         Improved sensitivity biological sequence database searches. <br />                                                                      Comp. Appl. Biol. Sci. 6:237-245. <br />    Second, entire protein lacks contiguous blocks      9. Burkholder,  C., L. H. Hartwell. 1985. yeast factor <br /> strikingly hydrophobic segments significant length,           receptor: structural properties deduced sequence making unlikely Motl transmembrane protein.             STE2 gene. Nucleic Acids Res. 13:8463-8475. <br />    Finally, N-terminal region precedes       10. Chung, C. T., R. H. Miller. 1988. rapid convenient <br /> helicase domain, motifs similar            method preparation storage competent bacterial called tetratricopeptide (TPR) units proteins        cells. Nucleic Acids Res. 16:3580. modulate RNA synthesis, encoded SSN6                11. Cohen, C., D.  D. Parry. 1986. Helical coiled coils SKI3, proteins important mitosis,          widespread motif proteins. Trends Biochem. Sci. 11:245-248. <br /> encoded CDC23, nuc2+, CDC16, bimA (36, 64) (Fig.          12. Courschesne, W. E., J. Thorner. 1986. Control phero- <br />                                                                      mone response Saccharomyces cerevisiae: isolation 8). case nuc2+ gene product, nuclear scaffold        SS72 gene. Yeast 2:S74. <br /> protein, Ssn6, nuclear phosphoprotein, repeats       13. Davis, T. N., M. S. Urdea, F. R. Masiarz, J. Thorner. 1986. shown essential function (36, 62).            Isolation yeast calmodulin gene: calmodulin essen- <br /> addition, long segments Motl          tial protein. Cell 47:423-431. <br />  VOL. 12, 1992                                    PUTATIVE HELICASE AFFECTING GENE EXPRESSION GROWTH                                  1891 <br />  <br />  14. Vos,  M., L. M. Tong, M. V. Milburn, P. M. Natias, J.               nuclear protein nuc2+. Cell 60:319-328. <br />      Jancarik, S. Noguchi, S. Nishimura, K. Miura, E. Ohtsuka,        37. Hodgman, T. C. 1988. new superfamily replicative pro- <br />      S. H. Kim. 1988. dimensional structure oncogene              teins. Nature (London) 333:23. (Erratum, 333:578.) <br />      protein: catalytic domain human c-H-ras p21. Science 239:         38. Hoffman, C. S., F. Winston. 1987. minute DNA <br />      888-893.                                                                 preparation yeast efficiently releases autonomous plasmids <br /> 15. Dietzel, C., J. Kurjan. 1987. Pheromonal regulation               transformation Escherichia coli. Gene 57:267-272. <br />      sequence Saccharomyces cerevisiae SST2 gene: model          39. Ito, H., Y. Fukuda, K. Murata,  Kimura. 1983. Transfor- <br />      desensitization pheromone. Mol. Cell. Biol. 7:4169-4177.          mation intact yeast cells treated alkali cations. J. <br /> 16. Dolan, J. W., S. Fields. 1990. Overproduction yeast            Bacteriol. 153:163-168. <br />      Stel2 protein leads constitutive transcriptional induction.       40. Jamieson, D. J., B. Rahe, J. Pringle, J. D. Beggs. 1991.      Genes Dev. 4:492-502.                                                    suppressor yeast splicing mutation (prp8-1) encodes 17. Dolan, J. W., C. Kirkman, S. Fields. 1989. yeast Stel2            putative ATP-dependent RNA helicase. Nature (London) 349: <br />      protein binds DNA sequence mediating pheromone                    715-717. <br />      induction. Proc. Natl. Acad. Sci. USA 86:5703-5707.                  41. Jones, E. W. 1977. Proteinase mutants Saccharomyces cere- <br />  18. Emery, H. S., D. Schild, D. E. Kellogg, R. K. Mortimer.              visiae. Genetics 85:23-33. <br />      1991. Sequence RAD54, Saccharomyces cerevisiae gene             42. Kelleher, R. J., III, P. M. Flanagan, R. D. Kornberg. 1990. <br />      involved recombination repair. Gene 104:103-106.                  novel mediator activator proteins RNA <br />  19. Errede, B., G. Ammerer. 1989. STE12, protein involved           polymerase II transcription apparatus. Cell 61:1209-1215. <br />      cell-type-specific transcription signal transduction yeast,   43. Kodahek, T., B. Alberts. 1987. Stimulation protein strand <br />      protein-DNA complexes. Genes Dev. 3:1349-1361.                exchange DNA helicase. Nature (London) 326:312-314. <br /> 20. Estruch, F., M. Carlson. 1990. SNF6 encodes nuclear             44. Kronstad, J. W., J.  Holly, V. L. MacKay. 1987. yeast <br />      protein required expression genes Sac-            operator overlaps upstream activation site. Cell 50:369-377. <br />      charomyces cerevisiae. Mol. Cell. Biol. 10:2544-2553.                45. Kuchler, K., R. E. Sterne, J. Thorner. 1989. Saccharomyces <br /> 21. Fassler, J. S., W. Gray, J. P. Lee, G. Yu, G. Gingerich.              cerevisiae STE6 gene product: novel pathway protein <br />      1991. Saccharomyces cerevisiae SPT14 gene essential           export eukaryotic cells. EMBO J. 8:3973-3984. <br />      normal expression yeast transposon, Ty,        46. Lahaye, , H. Stahl, D. Thines-Sempoux, F. Foury. 1991. <br />      expression HIS4 gene genes mating              PIFl: DNA helicase yeast mitochondria. EMBO J. 10:997- <br />      pathway. Mol. Gen. Genet. 230:310-320.                                   1007. <br /> 22. Feinberg,  P., B. Vogelstein. 1983. technique             47. Laurent, B. C., M.  Treitel, M. Carlson. 1991. Functional <br />      radiolabeling DNA restriction endonuclease fragments high             interdependence yeast SNF2, SNF5 SNF6 proteins      specific activity. Anal. Biochem. 132:6-13.                              transcriptional activation. Proc. Nati. Acad. Sci. USA 88:2687- <br /> 23. Fields, S., D. T. Chaleff, G. F. Sprague, Jr. 1988. Yeast             2691. <br />      STE7, STE11, STE12 genes required expression          47a.Laurent, B. C., X. Yang, M. Carlson. Personal communi- <br />      cell-type-specific genes. Mol. Cell. Biol. 8:551-556.                    cation. <br /> 24. Fields, S.,  Herskowitz. 1985. yeast STE12 product       48. Levin, D. E., F. 0. Fields, R. Kunisawa, J. M. Bishop, J. <br />      required expression sets cell-type-specific genes.         Thorner. 1990. candidate protein kinase C gene, PKC1,      Cell 42:923-930.                                                         required S. cerevisiae cell cycle. Cell 62:213-224. <br /> 25. Fields, S.,  Herskowitz. 1987. Regulation yeast          49. Linder, P., P. F. Lasko, P. J. Nielsen, K. Nishi, P. P. <br />      mating-type locus STE12, gene required cell-type-               Slonimski. 1989. Birth D-E D  <br /> </body></html>