<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Mar. 1990, p. 887-897                                                                      Vol. 10,  3 <br /> 0270-7306/90/030887-11$02.00/0 <br /> Copyright C) 1990, American Society Microbiology <br />  <br />  <br />         Yeast ARS-Binding Protein Activates Transcription <br />  Synergistically Combination Weak Activating Factors <br />                               ANDREW R. BUCHMAN*t ROGER D. KORNBERG <br />  Department Cell Biology, Beckman Laboratories, Stanford University School Medicine, Stanford, California 94305 <br />                                           Received 8 August 1989/Accepted 14 November 1989 <br />  <br />              ABFI (ARS-binding protein  yeast protein binds specific DNA sequences associated           autonomously replicating sequences (ARSs). ABFI binds sequences located promoter regions           yeast genes, including DEDI, essential gene unknown function transcribed constitutively high <br />           level. ABFI purified specific binding DED) upstream activating sequence (UAS)           recognize related sequences promoters, ARS (ARSJ), transcriptional silencer <br />           (HMR E). ABFI-binding sites, regardless origin, provided weak UAS function vivo examined           test plasmids. UAS function abolished point mutations reduced ABFI binding vitro. Analysis           DEDI promoter showed ABFI-binding sites combine synergistically adjacent T-rich <br />           sequence form strong constitutive activator. DEDI T-rich element acted synergistically           ABFI-binding sites binding sites multifunctional yeast activators. examination           properties sequences surrounding ARSI left open possibility ABFI enhances initiation DNA <br />           replication ARS) transcriptional activation. <br />  <br />    Promoters higher eucaryotes complex structures                  diverse genetic elements. members class separable components  reviews,                     identified far: centromere-binding protein 1 (CP1), <br /> references 28 37). Deletion studies revealed                  factor Y, general regulatory factor (GRFI), ARS- <br /> groups regulatory sequences needed activate transcrip-               binding factor (ABFI). CP1 binds sequences conserved <br /> tion-enhancers, located kilobase pairs                 domain yeast centromeres, CDE  intergenic <br /> promoter, proximal elements positioned just upstream                   regions (5). Factor Y-binding sites UASs TATA sequence. Typically, enhancers proximal                    contained telomere-associated DNA <br /> promoter elements contain binding sites                   elements X Y&apos; (18;  R. Buchman N. F. Lue, <br /> different transcription factors. Binding sites factors           unpublished data). Binding sites GRFI, known greatly stimulate transcription various               RAP1 (53) TUF (24, 25), serve UASs wide variety <br /> combinations little effect (14, 19, 22, 40,          genes involved transcription, translation, RNA proc- <br /> 49, 50). contrast, studies promoters Saccharomyces                essing, glycolysis, cell growth, functions (8, 9). <br /> cerevisiae suggested streamlined organization                  Interestingly, GRFI-binding sites important  review, reference 61). Upstream activating se-                  transcriptional silencers inactive mating type loci, <br /> quences (UASs) S. cerevisiae usually clustered               HML HMR (6, 8, 9, 30, 53).  C1_3A repeats base pairs start transcription.                  yeast telomeres contain binding sites GRFI (8, 9). <br /> Furthermore, single binding site regulatory protein,               ABFI binds sequences associated ARSI ARS2, GAL4 (20, 34) GCN4 (23), suffices                      autonomously replicating sequences (ARSs) activate transcription. investigators (14, 59)                   regulatory elements silent mating-type loci-HMR E, <br /> proposed rudimentary mechanism explain                     HMR  HML (8).  fully functional constitutive transcription yeast genes contain                 ARSs lack binding sites ABFI, ABFI-binding <br /> <span id='am-19' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-20' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>thymidine</span>-rich UASs. notion based binding                sites far removed ARS. suggestive <br /> sequence-specific transcription factors                  situation discovery ABFI-binding sites inability poly(dT)-poly(dA) form nucleosomes vitro                promoter region DED1 gene, essential gene (31, 42). suggested T-rich UASs simply prevent                 unknown function transcribed constitutively high assembly nucleosomes promoter, making                    level. Previous investigators determined transcription region accessible transcription apparatus.                     DEDI depends 39-base-pair (bp) T-rich sequence lo- <br />   gain deeper understanding molecular mecha-                   cated ABFI-binding sites TATA ele- <br /> nisms governing gene expression chromatin structure                 ment, keeping nucleosome exclusion idea (59). <br /> S. cerevisiae, taken biochemical approach                  preliminary experiments identifying transcription factors assaying functions             ABFI-binding region T-rich sequence play impor- vitro. investigations uncovered class                     tant activating transcription (35). observations <br /> multifunctional DNA-binding proteins distinguished                motivated careful study properties ABFI abundance 103 molecules                     possible function activator DEDI haploid cell ability bind specific sequences               genes. results lead conclusion ABFI func- <br />                                                                            tions relatively weak activator compared   * <br />     Corresponding author.                                                  characterized yeast transcription factors. high <br />   t Present address: Department Molecular Cell Biology, 310         level constitutive transcription achieved DEDI <br /> South Frear Laboratory, Pennsylvania State University, Uni-            synergistic actions multiple weak activating ele- <br /> versity Park, PA 16802.                                                    ments, contain binding sites ABFI factors                                                                      887 <br />  888    BUCHMAN KORNBERG                                                                                  MOL. CELL. BIOL. <br />  <br /> bind T-rich region. Comparison properties          BamHI fragment pCZ-TEL, generating pCZ-EI. pCZ- <br /> ABFI GRFI factor Y suggests weak          SPT2, -SPT2mtA, -SPT2mtB, -mtIA, -mtIB, -TCM1, -PAB, <br /> multifunctional activators various combinations fre-          -SYM, -ARS1, -TGS2 constructed ligating quently play role yeast gene expression.                    oligonucleotide 1-kbp HindIII-ClaI 7.9-kbp ClaI- <br />                                                                  BamHI fragments pCZ-EI. construction pCZ- <br />                MATERIALS METHODS                             DEDSPT, oligonucleotide ligated 9-kbp <br />                                                                  XhoI-XbaI fragment pCZA. pCZ-DED48  <br />    assisted searches DNA sequences. Yeast DNA        scribed (35) contains oligonucleotide inserted <br /> sequences GenBank EMBL data bases                EcoRI XhoI sites polylinker pCZA. <br /> searched matches ABFI consensus sequence           Plasmids containing DED48 sequence using programs obtained University          synthetic oligonucleotide (pCZ-oligonucleotide/DED48 plas- <br /> Wisconsin Group (15). Matches PAB,               mids) constructed ligation 7.9-kbp EcoRI-ClaI <br /> SP12, TCMJ, tRNAser genes span nucleotides 460            fragment corresponding pCZ derivative, having 477 (46), 214 228 (44), -232 -218 (51), 244 258     oligonucleotide  1-kbp EcoRI-ClaI fragment (1), respectively, published sequences.                   pCZ-DED48. pCZ-(DED48)2 generated creating    Synthetic oligonucleotides oligonucleotide-Sepharose      tandem duplication DED48 sequence. pCZ-DED48 <br /> resin. Oligonucleotides produced Applied            DNA digested separately XhoI EcoRI. Biosystems DNA synthesizer purified              linearized DNAs treated Klenow fragment nealed described previously (4). oligonucleotides         DNA polymerase deoxynucleoside triphosphates contain sequences derived yeast DNA,             digested ClaI endonuclease. 1-kbp EcoRI (blunt)- <br /> sequences end allow ligation restriction   ClaI fragment 7.9-kbp ClaI-XhoI (blunt) fragment fragments.  sequences oligonucleotides,       ligated form pCZ-(DED48)2. pCZ-dAT2  Table 2.) sequence TEL oligonucleotide        scribed previously  pdAT-2; [35]) contains  preliminary descriptions       116-bp AccI-XhoI fragment DEDI DNA (nucleotides 756 binding properties El, mtIA, mtIB, SYM,           880 numbering scheme Struhl [60]) inserted <br /> DED1 oligonucleotides (8). strand DOMA oligo-         SalI XhoI sites polylinker pCZA. <br /> <span id='am-6' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>nucleotide</span>, contains ARS consensus match          insertion ARSI domain B fragments pCZ <br /> ARS1, sequence 5&apos;-TCGAGTTGTTATTTTACAGA                   plasmids, ARSI DNA modified TF&apos;TTTATG1TTAGATCTTTTATGCTITGC1TIT&apos;CAAAAGG                       placed directly upstream CYCI promoter. YRP14/ <br /> CCTGCA-3&apos;. complementary strand annealed                CEN4/ARSJA108 A149 DNAs (54) digested <br /> bases 5 56. DNA affinity chromatography, double-          BglII, treated Klenow fragment DNA <br /> stranded DED1 oligonucleotide coupled Sepharose           polymerase, ligated XhoI octanucleotide linkers (5&apos;- <br /> CL-2B modification previously described proce-         CCTCGAGG-3&apos;), digested XhoI BamHI. dure (29). oligonucleotide (500 ,ug) allowed react    respective 140- 99-bp XhoI-BamHI fragments 4 ml CNBr-activated Sepharose beads. coupling        isolated gel electrophoresis, ligated 1-kbp <br /> efficiency, measured spectrophotometrically 260 nm,       XhoI-ClaI 7.9-kbp ClaI-BamHI fragments pCZ-TEL <br /> greater 60%.                                                produce pCZ-A108 pCZ-A149. 117-bp PstI-XbaI <br />    Plasmid probe DNAs. pCZ family plasmids (9         fragment pCZ-A108, containing ARSI domain <br /> kbp) shuttle vectors contain pUC18 replication        B, ligated 9-kbp XhoI-XbaI fragment pCZA <br /> origin P-lactamase gene propagation Escherichia       DOMA oligonucleotide produce pCZ- <br /> coli, URA3, SUP]], ARSI, CEN4 sequences           A108/DOMA. Probe DNAs restriction fragments  maintenance S. cerevisiae (plasmid maps      rived pCZ plasmids labeled  32P]<span id='am-1' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="dATP" datatype="xsd:string"></span>dATP</span> references 9 35). test UAS functions inserted   (Amersham Corp.) DNA polymerase Klenow frag- <br /> DNAs, pCZ plasmids contained yeast CYCI pro-             ment replacement synthesis (36). specific radioactiv- <br /> moter (nucleotides -138 +5 published numbering         ity probes varied 5 x 103 2 x 104 cpm/fmol. <br /> scheme [38]) fused lacZ protein-coding sequences.            Protein-DNA binding assays. Generally, nitrocellulose fil- <br /> CYCI UAS replaced polylinker sequence, having            ter-binding assays gel electrophoresis mobility shift following configuration restriction sites, plasmid     assays performed described previously (8). Binding <br /> pCZ-TEL (9): HindIII-SphI-PstI-SalI-XbaI-BamHI-TEL               reactions carried 20 ,ul buffer (25 mM <br /> oligonucleotide-EcoRI-SacI-KpnI-SmaI-SalI-XhoI-CYCl-             HEPES [N-2-hydroxyethylpiperazine-N&apos;-2-ethanesulfonic <br /> lacZ. Plasmid pCZA (35) contains inserted oligonucleo-        acid] [pH 7.5], 5 mM MgCl2, 0.1 mM EDTA, 0.5 mM <br /> tide lacks BamHI site listed  Synthetic oligo-     dithiothreitol, 10% glycerol, 1 mM phenylmethylsulfonyl <br /> nucleotides inserted pCZ DNA different ways         fluoride, 2 puM pepstatin  0.6 p.M leupeptin, 5 ,ug depending kinds ends possessed. pCZ-DED1,         antipain ml) containing 50 mM KCI, 0.01% Nonidet P-40 <br /> -HML35, -RAP1, -GAL2 produced ligating           (NP-40), 1 5 fmol 32P-labeled probe DNA, 2 ,ug indicated oligonucleotide 1-kbp EcoRI-ClaI 7.9-kbp      bovine serum albumin, 1 ,ug poly(dI-dC), various <br /> ClaI-BamHI fragments pCZ-TEL. construction         amounts yeast protein competitor DNA. incu- <br /> pCZ-EI, HindIII site introduced EcoRI    bation room temperature, reaction mixtures site polylinker. pCZ-TEL digested EcoRI,         filtered nitrocellulose electrophoresed 3% ends blunt DNA polymerase             NuSieve agarose (FMC Corp.) gels. relative binding <br /> Klenow fragment (Pharmacia LKB Biotechnology, ).             affinities different sequences determined gel <br /> Synthetic HindIII linkers (5&apos;-CAAGCTTG-3&apos;; New England           electrophoresis mobility shift assays performed <br /> BioLabs, ) ligated linearized DNA,         0.7 0.35 ng purified ABFI labeled probe. subsequently digested HindlIl ClaI. 1-kbp         probe reaction mixture free <br /> HindIII-ClaI fragment purified gel electrophoresis        bound ABFI measured quantitating joined oligonucleotide El 7.9-kbp ClaI-           radioactivity bands dried gel AMBIS <br />  VOL. 10, 1990                                    TRANSCRIPTION ACTIVATION YEAST ARS-BINDING PROTEIN                      889 <br />  <br /> Radioanalytic Imaging (Ambis Systems , San                                      RESULTS <br /> Diego, Calif.). Equations used calculate relative binding <br /> affinities data discussed          Homology search ABFI-binding sites. gain insight <br /> (33). purpose estimates assumed      possible function ABFI gene activator, <br /> ABFI pure, 100% active, bound monomer (135           looked additional binding sites yeast DNA  <br /> kilodaltons [kDa]). specific radioactivities probes        ing homology search program. Data bases determined scintillation counting measuring           yeast sequences (GenBank EMBL) searched DNA concentrations Hoechst 33258 dye fluorom-          matches ABFI consensus sequence, 5&apos;-RDCNYN <br /> eter (TKO 100; Hoefer Scientific Instruments).                    NNNNACGAD-3&apos; (R = G; Y = C T; D =  G,    Purification ABFI. Cells yeast strain BJ926 (ala     T; N =  G, C, T), derived regions known bind <br /> trpll+ prcl-1261prcl-126 pep4-31pep4-3 prbl-I1221prblI 122        factor high affinity (8). large number matches <br /> canllcanl) grown YPD medium (2% Bacto-Peptone              intergenic regions. matches <br /> [Difco Laboratories], 1% yeast extract, 2% glucose)     chosen study based location  <br /> culture reached optical density 600 nm 6.0           stream genes  match upstream <br /> harvested centrifugation. pellet washed          region gene, PAB, encoding poly -binding protein <br /> equal volume buffer resuspended              (46). located SPT2 gene (44), volume buffer  subsequently adjusted 0.4        encodes protein regulates transcription trans- <br /> M KCl. Cells broken 4&#194;&#176;C glass bead disruption,         poson TY]. match, located upstream ribo- <br /> using 10 30-s cycles bead beater (Biospec Products).     somal protein gene TCMI (51), coincides sequence <br /> cell lysate centrifuged Sorval SS-34 rotor 12,000     previously shown function UAS (21). fourth <br /> x g 10 min; supernatant decanted centri-          match adjacent sequence encoding 5&apos; end fuged Beckman Ti6O rotor 250,000 x g 3 h 4&#194;&#176;C.      tRNA9er (1). compare properties new se- <br />                                                                   quences characterized previously, clarified extract adjusted 100 mM KCl dilution      synthetic oligonucleotides produced contained buffer (total volume, 560 ml) loaded       matches  Table 2). Initially, exam- <br /> column DEAE-cellulose (5 12 cm; Whatman DE52)            ined ability new oligonucleotides bind ABFI rate 250 ml/h. column washed 500 ml          testing competitors nitrocellulose filter-binding <br /> buffer 0.05 M KCl, protein eluted       assays (8), carried cell extract <br /> 1,150-ml gradient buffer 0.05 0.5 M KCl.             32P-labeled probe containing ABFI-binding site Fractions containing ABFI (0.22 M KCI peak) com-             HMR E (El oligonucleotide). SPT2, PAB, TCM1, bined, diluted 0.05 M KCI, loaded phospho-          TGS2 (tRNAs&#194;&#182;) oligonucleotides blocked specific protein <br /> cellulose column (2.5 18 cm; Whatman P11) 100 ml/h.         binding probe concentrations indicated column washed 180 ml buffer 0.05 M         bound ABFI high affinity (data shown). <br /> KCI, protein eluted 450-ml linear gradient        Purification ABFI. Considering variety DNA buffer 0.05 0.5 M KCl. Peak fractions containing     elements bound crude ABFI vitro-ARSs, <br /> ABFI (0.24 M KCI) combined diluted 0.1 M              silencers, UASs possible factor <br /> KCl, NP-40 added 0.01%. protein mixed          actually composed multiple proteins related 1.7 mg salmon sperm DNA applied 2-ml             binding specificities. sought clarify point column DED1 oligonucleotide-Sepharose 13 ml/h.          purification ABFI. Fractionation monitored column washed sequentially 5 ml buffer         using nitrocellulose filter-binding assays 32P-labeled <br /> 0.05 M KCI 0.01% NP-40, 7 ml buffer 0.3 M           probes having different ABFI-binding sites, joined <br /> KCl 0.01% NP-40, 10 ml buffer 1.0 M KCl         184 bp CYCI DNA. Probes containing ARS1, EI, 0.01% NP-40. Peak fractions ABFI 1.0 M            DED1, TCM1 oligonucleotides  Table 2) se- <br /> KCI step combined, adjusted 0.1 M KCl,                quences derived ARS, silencer (HMR E), reapplied oligonucleotide-Sepharose column,          UASs, respectively. yeast cell extract fraction- washed procedure. Fractions containing            ated chromatography DEAE-cellulose, phosphocellu- <br /> ABFI stored freezing liquid nitrogen. Protein          lose, DED1 oligonucleotide-Sepharose  Materials <br /> concentrations determined method Bradford          Methods). single peak binding activity (3), using bovine serum albumin standard.                  chromatography resin, peak profile    Assays UAS function. Plasmid DNAs introduced           DNA probes. Overall, yeast strain 5C  his3 A200 ura3-52) using         binding activity purified 5,160-fold 15% yield <br /> lithium acetate transformation procedure (26), selecting      (Table 1). purified ABFI oligonucleotide- <br /> <span id='am-14' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> prototrophy minimal medium (0.67% yeast nitrogen        Sepharose column analyzed sodium dodecyl sulfate- <br /> base, 0.5% ammonium sulfate, 2% glucose) supplemented             polyacrylamide gel electrophoresis, proteins histidine. Transformants picked grown            detected silver stain (Fig. 1A). major polypep- <br /> liquid minimal medium supplemented histidine       tides resolved masses 135 145 kDa. <br /> culture reached optical density 600 nm 1.0. Cells           integrity purity ABFI judged using gel harvested centrifugation, washed, suspended        electrophoresis mobility shift assays 32P-labeled probes buffer (200 mM Tris hydrochloride [pH 7.5],          containing various oligonucleotides joined 184 bp CYCI <br /> 200 mM NaCl, 10 mM magnesium acetate, 5% glycerol, 10             DNA (Fig. 1B). single specific complex formed mM P-mercaptoethanol, 1 mM phenylmethylsulfonyl fluo-             purified ABFI EI (HMR E) probe ride) equal volume cell pellet. Cells broken   mobility complex produced cell extract. glass bead disruption, extract clarified        Probes containing sequences recognized abundant <br /> centrifugation 10 min 14,000 x g microcentrifuge.     yeast DNA-binding factors tested. probe P-Galactosidase enzyme assays performed o- <br /> nitrophenyl-3-D-galactoside described previously (39). <br />                                                                   binding sites GRFI sequence (TEL oligonucleotide) <br />                                                                   derived C1_3A repeat yeast telomeres (8). Probes <br />  890     BUCHMAN KORNBERG                                                                                                                                         MOL. CELL. BIOL. <br />  <br />                                                                 TABLE 1. Purification ABFI <br />                                                                                                                                                                           Purification <br />           Fraction                        Protein (mg)                Vol (ml)                    Activity (U)                       Sp act (U/mg) <br />                                                                                                                                                                Yield (%)                 Fold cell extract                        1,980                        560                            1,160,000                             585                     100                  1 <br /> DEAE-cellulose                              205                        132                              654,000                           3,200                      56                  5.5 <br /> Phosphocellulose                             13                         80                              254,000                          19,500                      22                 33.4 <br /> Oligonucleotide-Sepharose                     0.058                      2.9                            175,200                       3,020,000                      15              5,160 <br />  unit DNA-binding activity required retain 1 fmol labeled probe nitrocellulose filter. <br />  <br />  <br /> capable binding CP1 factor Y contained sequences                                       ABFI.  purified ABFI appeared largely intact upstream GAL2 gene (5) RAPJ gene (53),                                    free DNA-binding activities. <br /> respectively. (Note RAP] gene encodes DNA-                                          Binding properties ABFI vitro. correlate binding protein GRFI, factor Y GRFI                                    binding purified ABFI vitro function vivo, binds upstream region gene   R. Buchman,                                        characterized binding properties panel synthetic <br /> unpublished data].) Assays showed cell ex-                                     oligonucleotides containing matches ABFI consensus <br /> tract contained comparable amounts ABFI, GRFI, factor                                      sequence. addition matches discussed  Y, CP1, producing complexes varied mobility                                   oligonucleotide tested contained synthetic sym- factor (Fig. 1B);  GRFI, CP1, factor Y                                       metrized version ABFI consensus sequence, SYM (8), purified ABFI. previous study (35)                                    derived hypothetical half site. oligonucleotide, <br /> suggested ABFI-binding region DED1 gene                                       joined 184 bp CYCI DNA, used probe gel <br /> functions synergistically adjacent T-rich sequence                                 electrophoresis mobility shift assays. single defined <br /> activate transcription vivo. Factors bind T-rich                                 complex formed probe, relative <br /> element stimulated transcription 30-fold                                      affinities cell extract yeast vitro (35). probe containing                                       purified ABFI: SPT2 &gt; El &gt; TCM1 &gt; DED1, PAB &gt; SYM <br /> DEDI T-rich sequence (DED48) mixed cell                                        &gt; ARS1 &gt; TGS2 (Fig. 2B C). Quantitative measure- <br /> extract, specific complexes elec-                                   ments relative binding affinities sequence  trophoretic mobility shift assay (Fig. 1B C). T-                                      Materials Methods) revealed sixfold difference  <br /> rich-sequence-binding factors detected assay ap-                                      tween highest lowest affinities (Table 2). <br /> peared abundant bound lower affinity                                  examined binding properties oligonucle- ABFI, GRFI, factor Y, CP1. T-rich-sequence-                                        otides contained single base changes ABFI <br /> binding factors detected highly fractionated                                      consensus region (Table 2). Oligonucleotides SPT2mtA  <br />                                                          ABF!          GRF!        Factor Y                OPI.        T-rich             BEnoi        T- rich <br />                                                                                                                                           Facto&apos;r <br />                                                            El          TEL             RAP1            GAL2            DED48              Probe           DED48 <br />                                                               -_i      r         -              1                                                            -I1 <br />  <br />                                                                 m <br />                                                                 c:            m <br />                                                                               M. <br />                                                                               c:             m <br />                                                                                              &lt;:                   m <br />                                                                                                                   &lt;r <br />                                                                                                                        ;,       c&lt;        Prote                                                                 &lt;             &lt; <br />  <br />  <br />  <br />                        200 - <br />  <br />                                       =   ABFI <br />                                                                                          116   - <br />                                                                                                                                        Complexes <br />                         97 -                                                                                                                       .. <br />                         66 - <br />                                                       d_              uIqI                                                f        }-    Probes <br />  <br />                         43 - <br />                                                                                        B                                                                    C <br />   FIG. 1. Purification ABFI.   Purified ABFI (0.2 ,ug) 1.0 M KCl step DED1 oligonucleotide-Sepharose column <br /> procedure analyzed sodium dodecyl sulfate-7.5% polyacrylamide gel electrophoresis, proteins reveal silver stain (643). positions sizes kilodaltons protein standards shown left: myosin (200 kDa), 3-galactosidase (116 kDa), phosphorylase <br /> b (97 kDa), bovine serum albumin (66 kDa), ovalbumin (43 kDa). (B) Gel electrophoresis mobility shift assays performed 32P-labeled BamHI fragments (0.22 kbp) pCZ-TEL, -El, -RAP1, -GAL2 EcoRI-BamHI fragment (0.20 kbp) pCZ-DED48. Reaction mixtures contained 2.7 ,ug cell extract (WCE), 1 ng purified ABFI (El reaction), 10 ng ABFl <br /> (TEL, RAP1, GAL2, DED48 reactions). dried gel analyzed autoradiography exposed 1 h. (C) right lanes <br /> shown panel B exposed 17 h. <br />  VOL. 10, 1990                                                                                     TRANSCRIPTION ACTIVATION YEAST ARS-BINDING PROTEIN                      891 <br />  <br />                             ABFI                        WKE                                                   consensus sequence increased insertion                                                                                                                 residue center. alteration mtIB reduced specific <br />                                                                                   Protein <br />  <br />                                                               m <br />                                                                                                                 binding 36-fold crude purified ABFI (Fig.1A). <br />                                    .,:                     &lt; <br />  <br />                     cu-             E                                             Probe <br />                                                                                                                 binding detected, using cell extract                                                                                                                 purified ABFI, probes containing mtIA, SPT2mtA, <br />                                                                                                                 SPT2mtB oligonucleotide (Fig. 1).                                                                                                                  base changes reduced binding affinity 500-fold  <br />                                                                                                                    UAS function ABFI-binding sites. studies                                                                          Complex                           locations binding sites ABFI raised possibility                                                                                                                 ABFI act transcription factor situations <br />                                                                                                                 (8). Binding sites promoter regions                                                                                                                 DEDI    gene 1,950-base RNA 2p.m plasmid. <br />                                                                                                                 Recent results lend support notion ABFI <br />                                                                               Probe <br />                                                                                                                 acts transcriptional activator. experiments described <br />                                                                           - <br />  <br />  <br />  <br />  <br />                                                                                                                  demonstrated binding site ABFI                                                                                                                 TCMI gene coincides 19-bp region shown                                                                                                                 (21) UAS.  analysis regulatory region         B                                              ABFI                                                     DEDI     promoter revealed composed                                                                                                                 segments activate transcription synergistically (35).                                                                                    &lt;      m <br />                                                                                                                 segment, just upstream TATA element, contains                 N <br />  <br />                 U  LUI <br />                           X <br />                               _ <br />  <br />                               C. <br />                                                                                   LU <br />                                                   mm              V)u <br />                                                                          N <br />  <br />                                                                         c)) <br />                                                                           0 <br />                                                                                                       Probe     39-bp region rich T residues coding strand. Imme- <br />                 (                        0 <br />                                               | <br />                                                   CL <br />                                                                                  W  n <br />                                                                                                                 diately adjacent T-rich segment 68-bp region <br />                                                                                                                 containing binding sites ABFI  .                                                                                                                 region weak activator transcription,                                                                                                                 combined produced strong activating effect. <br />                                                                                                   - <br />                                                                                                       Complex    remained determined UAS <br />                             - - -                        - - - <br />                                                                                                                 functions sequences containing ABFI-binding sites <br />                                                                                                                 fact action ABFI factors                                                                                                                 bind regions. reasoned ABFI                                                                                                                 function activator, oligonucleotides bind <br />                               _m                  si <br />                                                    m n - Probe                                                  ABFI vitro provide weak UAS function                                                                                                                 vivo examined test plasmids. Furthermore,                                                                                                                 ABFI-binding oligonucleotides produce strong ac- <br />                                                                                                                 tivating effect combined T-rich element         c <br />                                                                                                                 DEDI   gene. <br />                                                   VCE                                                             predictions tested use yeast centro- <br />                                                                                   E       E <br />                                                                                                                 meric plasmids carrying CYCJ-lacZ (promoter-coding  <br />             CM <br />                             &gt;,- n <br />                           - C,,               s u:cn tN <br />                                               m-            N <br />                                                                               E <br />                                                                                                       Probe     gion fusion) reporter gene URA3 selectable marker. <br />              <br />                                             LL               CC&lt; n CL :                                                     normal UAS       CYCI   promoter replaced                                                                                                                 plasmids polylinker region, receive <br />                                                                                                                 sequences tested UAS function. set                                                                                                                 constructions,    ABFI    oligonucleotide inserted <br />                                                                                                                 position 56 bp upstream CYCJ TATA <br />                                                               __                              -   Complex <br />                            _4EI4                                                                                element (pCZ-oligonucleotide plasmids). set                                                                                                                 constructions, 48-bp sequence containing DEDI T-rich <br />                                                                                                                 activator (DED48 oligonucleotide) inserted                                                                                                                 ABFI oligonucleotide    CYCI       TATA element <br />                                                                                                                 (pCZ-oligonucleotide/DED48 derivatives). Plasmids  <br />                                                                                                       Probe     troduced yeast strain     5C his3-A200 ura3-52), <br />                                                                                                                                                                                                                                                                         selecting uracil prototrophy, UAS function   FIG. 2. Gel electrophoresis mobility shift assays carried <br />                                                                                                                 assayed measuring   ,-galactosidase    activity extracts                                                                                                                 transformants (Table 2). constructs containing oligonu- 32P-labeled BamHI fragments (0.22 kbp) pCZ-SPT2, <br /> -SPT2mtA, -SPT2mtB, -TCM1, -DED1, -PAB, -SYM, -ARS1, <br />                                                                                                                 cleotides bound  ABFI   vitro showed significant UAS <br />                                                                                                                 function vivo. Plasmids containing just ABFI oligonu- <br /> -TGS2, -El, -mtIA, -mtIB. Reaction mixtures contained                                                cleotides exhibited levels expression 3- 26-fold <br /> 2.7 ,ug cell extract (WCE) 0.7 ng purified ABFI. <br /> Reaction mixtures cell extract contained 20 ng RAP1                                               grater control plasmid insert (pCZA, 0.9 U). <br /> oligonucleotide order block formation complexes                                                 constructions having ABFI-binding site probes factor Y, binds low affinity sequence                                            T-rich element (DED48) exhibited higher levels expres- SmaI-KpnI region polylinker (9).                                                                     sion, varying 6- 20-fold greater plasmid                                                                                                                 T-rich element (pCZ-DED48, 9 U). combina- <br /> SPTF2mtB derivatives SPT2 contain alterations                                                                                                                 tion   DED1 DED48 oligonucleotides (pCZ-DED1/ <br />                                                                                                                 DED48, 180 U) appeared fully reconstitute activating <br /> positions 12 13, respectively, consensus se-                                                         effect normal upstream regulatory region DED) <br /> quence. mtIA mtIB mutant forms oligonucle-                                                       gene, 116-bp AccI-XhoI fragment (pCZ-dAT2, 170 U). otide El. mtIB, position 3 consensus sequence                                                     cases, ABFI T-rich elements acted altered. mtIA, spacing halves                                                          synergistic fashion (3.9 9.6 times sum individual <br />  892      BUCHMAN KORNBERG                                                                                                                MOL. CELL. BIOL. <br />  <br />                                       TABLE 2. Binding properties UAS function different oligonucleotides <br />                                                     UAS function <br /> Oligonucleotide           Relative <br />    inserta             affinityb           CZC <br />                                              PCZ&apos;            pCZ plus                                              Sequencee <br />                                                              DED48d <br />  Consensus                                                                                                                      RDCNYNNNNNACGAD <br />  SPT2                       1.0               4.5             130                                              GATCCAAATAATCATGTTAAACGAAAAAGTA <br />  SPT2mtA                  &lt;0.002              0.30                 3.0                                         GATCCAAATAATCATGTTAAAgGAAAAAGTA <br />  SPT2mtB                  &lt;0.002              0.30                 4.6                             GATCCAAATAATCATGTTAAACcAAAAAGTA <br />  DEDSPT                     1.0              50                                 CTAGACGTGTCATTCTGAACGAGGCGATAATCATGTTAAACGAAAAAC <br />  El                        0.73               7.0              63                                  GATCCCATACATCATAAAATACGAACGA <br />  mtIA                     &lt;0.002              0.33              3.4                                GATCCAATACATCATAAaAATACGAACGA <br />  mtIB                       0.02              0.54              6.0                                GATCCAATACATaATAAAATACGAACGA <br />  TCM1                       0.64              7.1              74                                  GATCCTGAAAAACGTACAAAACGATTACTAA <br />  DED1                       0.45             24               180                    GATCCAATGCATCATTCTATACGTGTCATTCTGAACGAGGCG <br />  PAB                        0.44              6.4              74                                  GATCCGATCAATCATGTTTAACGAAAACTGA <br />  SYM                        0.30              5.0              84                                  GATCCAATATATCGTTATTAACGATATATTA <br />  ARS1                       0.26              8.0              89                                  GATCCTTCTTAGCATTTTTGACGAAATTTGA <br />  TGS2                       0.17              2.8              57                                  GATCCATGATATCATGTAAAACGAGGCAACA <br />                                             0.90 <br />  dAT2                                       170 <br />  DED48                                        9.0                               AATTCTTTCCTTTTTTCTTTTTGCTTTTTCTTTTTTTTTCTCTTGAAC <br />  (DED48)2                                    70 <br />  HML35                                       11               110               GATCCAAAAATCAAAAACCCATTCATAAGATACTG <br />  RAP1                                         2.6              55               GATCCCAATGGCCGCTACCCGCCTCTTG <br />  GAL2                                         0.52              9.0             GATCCAAAATGGGTCACGTGATCTATATTG <br />   Oligonucleotides DNAs inserted polylinker region pCZ plasmids. <br />   b relative binding affinity sequences purified ABFI measured gel electrophoresis mobility shift assays  Materials Methods) normalized SPT2. Estimates indicated apparent association constant ABFI SPT2 sequence 5 x 1010 M conditions (data shown). <br />   &apos; UAS function oligonucleotide assessed based expression CYCI-IacZ fusions pCZ derivatives, determined measuring specific activity ,B-galactosidase  nanomoles minute milligram protein) extracts yeast cells containing plasmid. <br />   d UAS function oligonucleotide DED48 T-rich sequence determined based p-galactosidase activity extracts yeast cells <br /> containing pCZ-oligonucleotide/DED48 plasmid. <br />   e sequence strand oligonucleotide listed 5&apos; 3&apos; left right. complementary strand anneals fifth base left base shown 4-base 5&apos; protruding end 5&apos;-AATT-3&apos; DED1, HML35, RAP1, GAL2; 5&apos;-TCGA-3&apos; DEDSPT DED48; 5&apos;-AGCT-3&apos;  ABFI consensus region oligonucleotide boldface. Second occurrences ABFI consensus sequence underlined DED1 DEDSPT. Sequences thought recognized DNA-binding proteins underlined DED48 (T rich), HML35 (GRFI), RAP1 (factor <br /> Y), GAL2 (CP1). <br />  <br />  <br />  <br /> elements). Activation ABFI oligonucleotides vivo                              ABFI added bind half probe depended ability bind ABFI vitro.                                reaction mixture, major complex formed DED1 <br /> synthetic symmetrized sequence SYM activat-                            probe (&gt;95%) mobility complexes formed <br /> ing effect oligonucleotides natural yeast DNA                         ABFI-binding sites (Fig. 2B), indicating sequences. Importantly, mutant oligonucleotides                            probe associated single molecule ABFI. diminished binding vitro (mtIA, mtIB, SPT2mtA,                          different result observed ABFI used (7- <br /> SPT2mtB) failed increase expression vivo,                        14-fold  Fig. 3A). single complex produced combination T-rich element. control                              El probe (HMR E), complexes formed <br /> plasmids pCZA pCZ-DED48 expressed slightly                              DED1 probe. major DED1 complex higher levels constructions containing mutant                          reduced mobility expected binding oligonucleotides. explained                          molecules ABFI. doubly bound DED1 complex <br /> weak activator sequence control plasmids                             appeared relatively unstable, evidenced disrupted insertion oligonucleotides.                                     smear radioactivity singly bound <br />    Interactions compound UASs. original analysis                         complex, presumably caused dissociation lower- DEDJ promoter suggested tandem                           mobility complex electrophoresis. instability <br /> binding sites ABFI (8). DNase protection assays,                           arise partial steric interference prox- <br /> crude ABFI nuclear extracts protected larger region                          imity binding sites DED1 sequence  DEDI (35 bases) ABFI-binding sites (20 25                       consensus sequences overlap 1 bp, Table 2). bases). region DEDI protected assays                                mind examined properties oligonucleotide, <br /> contained adjacent sequences homology                             DEDSPT, containing ABFI-binding sites (DEDJ ABFI-binding sites (marked sequence Table 2).                         SPT2) separated greater distance (6 bp). DEDSPT <br /> UAS assays described indicated occurrence                            oligonucleotide,  behaved DED1 ABFI-binding sites functional significance                         sequence gel electrophoresis mobility shift assays. Large <br /> vivo. plasmids DED1 oligonucleotide, pCZ-                              amounts ABFI required produce doubly <br /> DED1 pCZ-DED1/DED48, expressed higher                                 bound complex (Fig. 3A; data shown), level constructs ABFI-binding oligonucleo-                          appeared unstable. examined test plasmid tides. physical basis enhanced activation                          vivo, DEDSPT oligonucleotide, like DED1, produced <br /> explored gel electrophoresis mobility shift assays.                         elevated UAS function (pCZ-DEDSPT, 50 U, Table 2). <br />  VOL. 10, 1990                                                                           TRANSCRIPTION ACTIVATION YEAST ARS-BINDING PROTEIN                          893 <br />  <br />                                                                                                        mobility shift assays (Fig. 3B) probes containing just                                       DED1                                      Probe <br />                                                                                                          ABFI-binding site (El) ABFI-binding site                                                                                                          T-rich element (EI/DED48). El probe, addition                                                     DEDSPT <br />  <br />  <br />  <br />  <br />                              +   ++   +    ++       +           ++              ABFI                     cell extract produced major complex ABFI <br />                                                                                                          minor complexes (&lt;1%), presumably derived <br />                                                                                                          degraded ABFI (lane 2). Complex formation                                                                                                          El probe completely blocked excess EI oligonucle- <br />                                                                                                          otide (20 ng, lane 6) DED48 oligonucleotide <br />                                                                                                          (20 ng, lane 4). EI/DED48 probe additional <br />                                       -0                                                                 complexes resolved having lower mobilities                                            . <br />                                                                                                          ABFI-derived complexes produced EI probe (lane <br />                                                                                - Complexes               3). lower-mobility complexes EI/DED48                                                                                                          abolished addition El (lane 7) DED48 <br />                                                                                                          (lane 5) oligonucleotide, indicating complexes <br />                                                                                                          contained ABFI- T-rich-sequence-binding factors. <br />                                                                                                          Notably, binding ABFI did appear increase                                                                                                          association factors T-rich sequence.                                                                                                          increase complexes blocked DED48 <br />                                                                          -   Probe                     oligonucleotide ABFI bound complexes <br />                                                                                                          formed DED48 sequence absence ABFI <br />                                                                                                          (compare Fig. 3B, lanes 3 7; bands Fig. 3B, lane 7; <br />                                                                                                          represent complexes formed gel <br />             E3                        DED48             E =l                    Competitor               shown Fig. 1C DED48 probe). <br />                                                 r <br />                                                                        l                                    results experiments suggest combi- <br />                                  0 <br />                                  0 <br />                                            0 <br />                                            0                   0                                         nation ABFI T-rich elements produces strong UAS <br />                                  wLw                           ul <br />                                                                0                Probe                    activity mechanisms involving highly coopera- <br />                                                                                                          tive binding factors elements. raises                          +       +    +    +        +           +               WCE                      issue various weak activating elements                  1 <br />                  l <br />                          2       3    4    5        6            7                                       generally combine create strong activators. pursued <br />                                                                                                          question comparing activating properties                                                                                                          members class abundant yeast DNA-binding fac- <br />                                                                                                          tors. UAS test plasmids constructed contained <br />                                                                                                          binding sites factor Y (pCZ-RAP1), CP1 (pCZ-GAL2), <br />                                                                                                          GRFI (pCZ-HML35)                                                                                                           binding sites joined T-rich element DEDI <br />                                                                                                          gene (DED48 derivatives). showed previously GRFI <br />                                                                                                          potent transcriptional activator vivo vitro (9).                                                                Jk. ,           Complexes                 present study, GRFI-binding site used HML <br />                                                                                                          E DNA (HML35 oligonucleotide) relatively low <br />                                                                                                          binding affinity correspondingly low UAS activity (36- <br />                                                                                                          fold lower activity strongest GRFI-binding sites). <br />                                                                                                          ,-Galactosidase assays performed ex- <br />                                                                                }Probes <br />                                                                                                          tracts yeast strains containing different plasmid  <br />                                                                                                          structions (Table 2). weak GRFI-binding site (pCZ- <br />                                                                                                          HML35, 11 U) gave level expression comparable   FIG. 3.   Gel electrophoresis mobility shift assays car-                                        active ABFI-binding sites, expression greatly <br /> ried 32P-labeled XbaI-BamHI fragment pCZ-                                                enhanced combined T-rich element <br /> DEDSPT (198 bp) BamHI fragments pCZ-DED1 (225 bp)                                               (pCZ-HML35/DED48, 106 U). ABFI-binding pCZ-EI (218 bp). Reaction mixtures contained 5 ng (+)                                      sites, factor Y-binding site relatively weak UAS <br /> 10   ng   (+ +) purified ABFI. (B) Gel electrophoresis mobility shift                                 (pCZ-RAP1, 2.6 U), activity increased syner- <br /> assays       performed 32P-labeled BamHI fragments                                        gistically conjunction T-rich element (pCZ- <br /> pCZ-EI (218 bp) pCZ-EI/DED48 (233 bp). Reaction mixtures                                              RAP1/DED48, 55 U). plasmid carrying <br /> contained 4 pLg cell extract (WCE), contained 20                                       tandem duplication T-rich element [pCZ-(DED48)2] <br /> ng   indicated competitor oligonucleotide. <br />                                                                                                          expressed seven times (70 U) level                                                                                                          plasmid single T-rich sequence (pCZ-DED48, 9  highly cooperative binding ABFI                                                U). contrast, binding site centromere-binding vitro, addition weak secondary ABFI-                                                   protein CP1 did significantly increase expression, binding site significantly increased activation vivo.                                                 (pCZ-GAL2, 0.5 U) T-rich sequence <br />   similar lines, explanation observed                                                  (pCZ-GAL2/DED48, 9 U). <br /> synergism vivo ABFI-binding sites T-rich                                                 Transcriptional activation ARSI DNA. observation <br /> element ABFI recruits transcription                                                  ABFI participates activating transcription factors T-rich region direct cooperative inter-                                           genes leads question related actions. mechanism plausible considering                                                 function ARSI, origin cellular DNA replication (7). apparently low abundance affinity T-rich-                                                  ABFI-binding site ARS1 lies  sequence-binding factors (Fig. 1B C). looked                                                  functional domains characterized deletion analysis (13, <br /> evidence factor recruitment using electrophoretic                                                  54, 56, 58). Domain  required autonomous <br />  894           BUCHMAN KORNBERG                                                                                 MOL. CELL. BIOL. <br />  <br />                               B                   c                  just ABFI site ARSJ (pCZ-ARS1, Table 2). <br />                                                                        reduced level activation simply resulted <br />                TRP1                                                    increased distance ABFI site CYCI <br />                        ABFI              ARS <br />                                                                        TATA element (35). Removal ABFI-binding site                                                                        domain B reduced expression sixfold (pCZ-A149, 0.5 U). <br />        A108        A149           pstI   BgI11                         Adding ARS consensus sequence domain domain <br />                                                                        B reduced did eliminate expression (pCZ-A108/ <br />                pCZ-A1 49 <br />                            -Y&quot;,c                                       DOMA, 0.6 U). comparison, silencer fragment                               &quot;,, CYC1/lacZ                            HMR E completely blocks expression ,-galactosidase                                                                        similar constructions (9; &lt;0.02 U). <br /> pCZ-A108 <br />                                                  Xhol TATA <br />                                                                                                 DISCUSSION <br /> pCZ- <br /> AB10MA                                                                    Properties ABFI. purified ABFI specific binding <br />   FIG. 4. Transcriptional activation ARSI sequences. map       DEDI UAS DNA. purified protein bound specific <br /> ARS1 shown, ABFI-binding site ARS consensus            sequences variety genetic elements-UASs, silenc- <br /> sequence depicted closed open boxes, respectively.          ers, ARSs appeared composed ARS1 domains  B, C shown  arrow                polypeptides 135 145 kDa. results indicates end TRPI coding region. endpoints          excellent agreement reported groups deletions A108 A149 (54) marked map.           purified protein specifically binds ARSl DNA <br /> deletions remove sequences left endpoint. A149, 11 <br /> bp ABFI consensus region removed, A149 DNA does         (16, 62). Sweder et al. (62) described doublet 135 kDa bind ABFI vitro (data shown). various ARSI              following sodium dodecyl sulfate-polyacrylamide gel elec- <br /> fragments inserted pCZ plasmids illustrated  test    trophoresis purified preparation, UAS function, fragments fused CYCI        renaturation bands doublet promoter, depicted right. Restriction sites used construc-   specifically bind ARS1 DNA.  tions marked maps ARSI CYCI.                         bands slightly different forms polypeptide. <br />                                                                        Hamil et al. (21) described partially fractionated <br />                                                                        activity, designated TAF (TCMJ activation factor), replication, contains match consensus sequence            binds specifically UAS ribosomal protein gene ARS elements  review, reference 10).                TCMI. Photochemical cross-linking experiments indicated <br /> Domains B C, flank domain  absolutely              TAF contains 147-kDa polypeptide, reasonable <br /> required significantly affect stability minichro-        agreement reported ABFI. <br /> mosomes. example, removal 41 bp domain B,                       Previous suppositions features recognized including ABFI-binding site, reduces minichromosome                ABFI derived comparing binding sites revealed stability fourfold certain growth conditions (54;            DNase protection assays (8). numerous <br /> difference A108 A149, Fig. 4). prece-            matches consensus sequence 5&apos;-RDCNYNNNN <br /> dents transcription factors process transcrip-          NACGAD-3&apos;, varied considerably positions, <br /> tion playing role DNA replication. E. coli, transcrip-         bound protein high affinity vitro. Binding tion RNA polymerase apparently activates initiation              greatly reduced single base changes modified  <br /> replication unwinding DNA duplex (2). adenovirus             served positions consensus sequence changed <br /> genomes, binding sites cellular transcription factors          spacing halves binding site. important origin DNA replication (27, 41,              matches consensus <br /> 45). showed   oligonucleotide contain-            sequence fail bind ABFI high affinity vitro   R. <br /> ing ABFI-binding site domain B ARS1                    Buchman, unpublished results). consensus sequence activate CYCI promoter vivo.  domain B                    used optimally reflect residues recognized <br /> appears complex structure. Sequences             ABFI, regional features DNA sequence <br /> ABFI-binding site ARS consensus sequence               significantly affect binding. example, ABFI- <br /> impact ARS function (16, 54, 56).                 binding site ARSI region DNA additional DNA-binding factors, ABF-II (16) ABF-III                intrinsically bent structure (54). <br /> (62), recognize sequences domain B.                    Transcriptional activation ABFI. Previous studies gen- <br /> DEDI promoter, elements ARSI com-                 erated confusing picture function ABFI bine ABFI site greatly increase activating             diversity elements bound factor vitro. <br /> effect. hand, possible activating          Results present experiments demonstrate convinc- <br /> ability ABFI counteracted ARSI. instance,             ingly ABFI functions situations gene <br /> silencer HMR E contains binding sites activator             activator vivo. Binding sites associated proteins, ABFI GRFl. key sequence, ARS,                 promoter regions numerous genes supplying important <br /> converts silencer negative regulatory element (6).            functions.  reference 17 description ABFI-  ARS1 itself appears weak silencing effect            binding sites located upstream genes involved mito- inserted place HMR E (30). investigated              chondrial functions.) binding sites correspond <br /> activating properties sequences surrounding ARSI                 UASs defined deletion analysis (21, 35). showed <br /> inserting portions domains B plasmid pCZA             ABFI-binding sites act weak UASs strong position 22 bp upstream CYCI TATA element (Fig.               UASs conjunction T-rich element, regardless 4). UAS function measured assaying ,B-galactosidase             sites normally located ARS, silencer, <br /> activity extracts yeast strains carrying plasmid.           promoter region. completely synthetic symmetrical <br /> Domain B showed significant UAS activity (pCZ-                   binding site activating properties natural <br /> A108, 3.2 U), 2.5-fold construction            ABFI-binding sequences, point mutations abolished <br />  VOL. 10, 1990                                     TRANSCRIPTION ACTIVATION YEAST ARS-BINDING PROTEIN                               895 <br />  <br /> binding resulted loss UAS activity. single binding site    ABFI-binding site binding site GRFI ABFI large effect gene expression.           eliminated. GRFI, like ABFI, functions transcrip- <br /> 10-fold effect expression promoter             tional activator contexts. known containing ABFI-binding site (pCZ-SPT2 versus pCZ-             activator proteins substitute GRFI ABFI SPT2mtA -SPT2mtB pCZ-EI versus pCZ-mtIA                silencer special features ABFI -mtIB; Table 2), 40-fold effect               GRFI possessed activators required observed conjunction T-rich element (pCZ-SPT2/           proper silencer function. <br /> DED48 versus pCZ-SPT2mtA/DED48, Table 2). appears,                 Combinatorial effects activator proteins. higher eu-  ABFI intrinsically weaker         caryotes, promoters generally composed multiple <br /> characterized yeast regulatory factors. examined           binding sites activator proteins combine synergisti- <br /> similar plasmid constructions, high-affinity binding sites     cally affect transcription. view emerged GAL4 GRFI activator proteins 5- 30-fold            analysis DED1 regulatory region similar stronger UASs single ABFI-binding sites (9, 35).           situation higher eucaryotes contrasts directly unlikely limited activity ABFI-binding sites     model proposed earlier, based solely nucle- partial occupancy protein vivo,      osome exclusion T-rich sequences (59). ABFI-binding correlation relative binding affinity        sites combined T-rich element vitro UAS activity (Table 2). examined        synergistic manner create strong constitutive activator <br /> capacity ABFI stimulate transcription vitro, using      DEDI. T-rich sequence itself compound <br /> yeast nuclear extract templates containing mul-        element, appeared recognized specifically tiple ABFI-binding sites (35;  R. Buchman, unpublished           factors vitro. Unfortunately, little said <br /> data). T-rich sequence DEDI gene               mechanisms responsible high level caused marked stimulation vitro  30-fold),      activation. Synergistic effects generated factors <br /> significant stimulation observed ABFI-binding sites       act entirely different mechanisms. stimulation seen ABFI-               results suggest highly specific interactions binding sites combined T-rich sequence.              activators necessary synergism, combina- <br /> Supplementing nuclear extract purified ABFI          tions weak activators tested effect. Furthermore, <br /> failed increase transcription templates.       cooperative binding ABFI- T-rich-sequence-binding possible explanations lack activation    factors vitro. remains possible,  <br /> ABFI vitro. ABFI act mechanism operating         detect interactions assays vitro reaction, ABFI damaged          used. Studies higher eucaryotes implied preparation nuclear cell extracts. Alterna-           separate classes activator proteins based tively, intrinsic activating ability ABFI        different abilities combine synergistically weak detected conditions used.                     activators (19). tested combi- <br />    functions ABFI. functions ABFI provides         nations yeast activators described  Conse- <br /> elements associated RNA polymerase II promoters           quently, clear combinations weak acti- <br /> remain mystery. conceivable          vators invariably act synergistically S. cerevisiae (43) activating capacity ABFI applies situations.      yeast activators grouped different <br /> Generally, RNA polymerase III promoters tRNA genes              mechanistic categories.  gained infor- located transcribed region, 5&apos;-flanking    mation basic characteristics members sequences, ABFI-binding site tRNASer            highly abundant class yeast DNA-binding proteins. gene (TGS2), positive negative effects               factor clearly analogous binding specificity <br /> expression (32, 52).  DNA-binding proteins ap-           transcription factor higher eucaryotes CP1, pear act promoters transcribed RNA poly-           recognizes sequences mammalian activator <br /> meraseII III (11). yeast tRNA genes           USF (48) MLTF (12). Curiously, contain matches ABFI consensus sequence             evidence CP1 functions activator S. <br /> position, yeast sigma elements contain        cerevisiae. GRFI fairly strong transcriptional activator <br /> weak ABFI-binding site  half fourth affinity     key component UASs genes. TGS2;  R. Buchman, unpublished data). Transposable               present studies ABFI detailed studies sigma elements closely associated 5&apos; ends          factor Y (D  Chasman, N. F. Lue, Y. Lorch,  R. tRNA genes  20 bp [47]).                               Buchman, J. Lapointe, R. D. Kornberg, manuscript    remains possibility ABFI enhances initiation      preparation) indicate proteins intrinsi- DNA replication ARSI transcriptional activation.          cally weak activators combine various ways Domain B ARS1, contains ABFI-binding site,            activators yeast genes. Considering abundance weak UAS activity. Addition ARS consensus               proteins, likely general <br /> sequence reduced did eliminate UAS function.         organizational features DEDI promoter repeated <br /> investigators,  detected stable transcripts       times yeast genome. emanate normal ARSI region (57). reasons carefully regulating transcription ARSs.                             ACKNOWLEDGMENTS <br /> High levels transcription ARS1 interfere         thank Pam Patek producing synthetic oligonucleotides. <br /> function (55). Future studies special plasmid construc-           work supported Public Health Service         grant   GM- <br /> tions  derivatives pCZ-A108/DOMA, Fig. 4)               36659 National Institutes Health   R.D.K. permit simultaneous analysis transcriptional                              LITERATURE CITED <br /> activation ARS function.                                        1. Baker, R. E., Eigel, D. Vogel, H. Feldman. 1982. <br />    Earlier studies demonstrated binding site ABFI           Nucleotide sequences yeast genes tRNAser, <br />                                                                                                                      tRN2rg significant constituent transcriptional silencer       tRNA Val: homology blocks occur vicinity different <br /> HMR E (6, 30). Silencer activity reduced mutation            tRNA genes. EMBO J. 1:291-295. <br />  896      BUCHMAN KORNBERG                                                                                              MOL. CELL. BIOL. <br />  <br />  2. Baker, T. ,  Kornberg. 1988. Transcriptional activation      23. Hill, D. E.,   Hope, J. P. Macke, K. Struhl. 1986. <br />     initiation replication E. coli chromosomal origin:         Saturation mutagenesis yeast his3 regulatory site: require- <br />     RNA-DNA hybrid near oriC. Cell 55:113-123.                             ments transcriptional induction binding GCN4 <br />  3. Bradford, M. M. 1976. rapid sensitive method                activator protein. Science 234:451-457. <br />     quantitation microgram quantities protein utilizing        24. Huet, J., P. Cottrell, M. Cool, M. L. Vignais, D. Thiele, C. <br />     principle protein-dye binding. Anal. Biochem. 72: 248-254.             Marck, J. M. Buhler,  Sentenac, P. Fromageot. 1985.  4. Bram, R., R. Kornberg. 1985. Specific protein binding far          general upstream-binding factor genes yeast transla- <br />     upstream activating sequences polymerase II promoters.                 tion apparatus. EMBO J. 4:3539-3547. <br />     Proc. Natl. Acad. Sci. USA 82:43-47.                                 25. Huet, J.,  Sentenac. 1987. TUF, DNA-binding factor <br />  5. Bram, R. J., R. D. Kornberg. 1987. Isolation Saccha-             specific UAS,pg upstream activating sequences: identifica- <br />     romyces cerevisiae centromere DNA-binding protein, human              tion protein DNA-binding domain. Proc. Natl. <br />     homolog, possible role transcription factor. Mol.            Acad. Sci. USA 84:3648-3652. <br />     Cell. Biol. 7:403-409.                                               26. Ito, H., Y. Fukuda, K. Murata,  Kinura. 1983. Transfor- <br />  6. Brand,  H., G. Micklem, K. Nasmyth. 1987. yeast                   mation intact yeast cells treated alkali cations. J. <br />     silencer contains sequences promote autonomous plas-             Bacteriol. 153:163-168. <br />     mid replication transcriptional activation. Cell 51:709-719.     27. Jones, K. , J. T. Kadonaga, P. J. Rosenfeld, T. J. Kelly, R. <br />  7. Brewer, B. J., W. L. Fangman. 1987. localization               Tjian. 1987. cellular DNA-binding protein activates <br />     replication origins ARS plasmids S. cerevisiae. Cell                eukaryotic transcription DNA replication. Cell 48:79-89. <br />     51:463-471.                                                          28. Jones, N. C., P. W. J. Rigby, E. B. Ziff. 1988. Trans-acting <br />  8. Buchman,  R., W. J. Kimmerly, J. Rine, R. D. Kornberg.              protein factors regulation eukaryotic transcription: <br />     1988. DNA-binding factors recognize specific sequences             lessons studies DNA tumor viruses. Genes Dev. <br />     silencers, upstream activating sequences, autonomously repli-             2:267-281. <br />     cating sequences, telomeres Saccharomyces cerevisiae.         29. Kadonaga, J. T., R. Tjian. 1986. Affinity purification     Mol. Cell. Biol. 8:210-225.                                               sequence-specific DNA-binding proteins. Proc. Natl. Acad. Sci. <br />  9. Buchman,  R., N. F. Lue, R. D. Kornberg. 1988. Connec-              USA 83:5889-5893. <br />     tions transcriptional activators, silencers, telo-       30. Kimmerly, W. J.,  Buchman, R. Kornberg, J. Rine. 1988. <br />     meres revealed functional analysis yeast DNA-                  Roles DNA-binding proteins replication, segregation, <br />     binding protein. Mol. Cell. Biol. 8:5086-5099.                            transcriptional repression mediated yeast silencer. <br /> 10. Campbell, J. L. 1983. Yeast DNA replication vitro               EMBO J. 77:2241-2253. <br />     vivo, p. 109-155. J. Setlow M. Hollander (ed.), Genetic       31. Kunkel, G. R., H. G. Martinson. 1981. Nucleosomes     engineering, vol. 5. Plenum Publishing Corp., New York.                   form double-stranded RNA poly(dA):poly(dT) 11. Carbon, P., S. Murgo, J. P. Ebel,  Krol, T. Graham,  W.           recombinant DNA. Nucleic Acids Res. 12:7057-7070. <br />     Mattaj. 1987. common octamer motif binding protein               32. Larson, D., J. Bradford-Wilcox, L. S. Young, K. Sprague. <br />     involved transcription U6 snRNA RNA polymerase                1983. short 5&apos;-flanking region containing conserved sequences <br />     II U2 snRNA RNA polymerase II. Cell 51:71-79.                      required silkworm alanine tRNA gene activity. Proc. Natl. <br /> 12. Carthew, R. W., L.  Chodosh, P.  Sharp. 1985. RNA              Acad. Sci. USA 80:3416-3420. <br />     polymerase II transcription factor binds upstream element       33. Letovsky, J., W. S. Dynan. 1989. Measurement     adenovirus major late promoter. Cell 43:439-448.                   binding transcription factor SP1 single GC box recogni- <br /> 13. Celniker, S. E., K. Sweder, F. Srienc, J. E. Bailey, J. L.            tion sequence. Nucleic Acids Res. 17:2639-2653. <br />     Campbell. 1984. Deletion mutations affecting autonomously             34. Lorch, Y., R. D. Kornberg. 1985. region flanking     replicating sequence ARSI Saccharomyces cerevisiae. Mol.                GAL7 gene binding site GAL4 protein upstream <br />     Cell. Biol. 4:2455-2466.                                                   activating sequences yeast. J. Mol. Biol. 186:821-824. <br /> 14. Chen, W., S. Tabor, K. Struhl. 1987. Distinguishing            35. Lue, N. F.,  R. Buchman, R. D. Kornberg. 1989. Activa- <br />     tween mechanisms eukaryotic transcriptional activation             tion yeast RNA polymerase II transcription thymidine- <br />     bacteriophage T7 RNA polymerase. Cell 50:1047-1055.                        rich upstream element vitro. Proc. Natl. Acad. Sci. USA <br /> 15. Devereux, J., P. Haeberli, 0. Smithies. 1984. comprehen-             86:486-490. <br />      sive set sequence analysis programs VAX. Nucleic          36. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br />     Acids Res. 12:387-395.                                                     cloning: laboratory manual. Cold Spring Harbor Laboratory, <br /> 16. Diffley, J. F. X., B. Stillman. 1988. Purification yeast          Cold Spring Harbor, N.Y. <br />      protein binds origins DNA replication transcrip-    37. Maniatis, T., S. Goodbourn, J.  Fischer. 1987. Regulation <br />      tional silencer. Proc. Natl. Acad. Sci. USA 85:2120-2124.                 inducible tissue-specific gene expression. Science 236: <br /> 17. Dorsman, J. C., W. C. van HeeswiJk, L.  Grivell. 1988.               1237-1245. <br />      Identification factors bind upstream se-         38. McNeil, J. B., M. Smith. 1985. Transcription initiation      quences number nuclear genes coding mitochondrial             Saccharomyces cerevisiae Iso-1-cytochrome c gene: multiple, <br />      proteins genetic elements important cell division           independent T T sequences. J. Mol. Biol. 187:363-378. <br />      yeast. Nucleic Acids Res. 16:7287-7301.                              39. Miller, J. H. 1972. Experiments molecular genetics. Cold <br /> 18. Fedor, M. J., N. F. Lue, R. D. Kornberg. 1988. Statistical             Spring Harbor Laboratory, Cold Spring Harbor, N.Y. <br />      positioning nucleosomes specific protein-binding         40. Ondek, B., L. Gloss, W. Herr. 1988. SV40 enhancer <br />      upstream activating sequence yeast. J. Mol. Biol. 204:                 contains distinct levels organization. Nature (London) <br />      109-127.                                                                333:40-45. <br /> 19. Fromental, C., M. Kanno, H. Nomiyama, P. Chambon.                41. Pruijn, G. J. M., W. van Driel, P. C. van der Viiet. 1986. <br />      1988. Cooperativity hierarchical levels functional orga-         Nuclear factor III, novel sequence-specific DNA-binding <br />      nization SV40 enhancer. Cell 54:943-953.                         protein HeLa cells stimulating adenovirus DNA replica- <br /> 20. Giniger, E., S. Varnum, M. Ptashne. 1985. Specific DNA               tion. Nature (London) 322:656-659. <br />      binding GAL4, positive regulatory protein yeast. Cell       42. Prunell,  1982. Nucleosome reconstitution plasmid  <br />      40:767-774.                                                             serted poly(dA):(dT). EMBO J. 1:173-179. <br /> 21. Hamil, K. G., H. G. Nam, H. M. Fried. 1988. Constitutive         43. Ptashne, M. 1988. eukaryotic transcriptional activators <br />      transcription yeast ribosomal protein gene TCMJ promoted          work. Nature (London) 335:683-689. <br />      uncommon cis- trans-acting elements. Mol. Cell. Biol.        44. Roeder, G. S., C. Beard, M. Smith, S. Keranen. 1985. <br />       8:4328-4341.                                                           Isolation characterization SPT2 gene, negative <br /> 22. Herr, W., J. Clarke. 1986. SV40 enhancer composed             regulator TY-controlled yeast gene expression. Mol. Cell. <br />     multiple functional elements compensate              Biol. 5:1543-1553. <br />      Cell 45:461-470.                                            45. Rosenfeld, P. J., E.  O&apos;Neil, R. J. Wides, T. J. Kelly. 1987. <br />  VOL. 10, 1990                                         TRANSCRIPTION ACTIVATION YEAST ARS-BINDING PROTEIN                              897 <br />  <br />       Sequence specific interactions cellular DNA-binding       54. Snyder, M.,  R. Buchman, R. W. Davis. 1986. Bent DNA <br />       proteins adenovirus origin replication. Mol. Cell.         yeast autonomously replicating sequence. Nature (London) <br />       Biol. 7:875-886.                                                      324:87-89. <br /> 46.   Sachs,  B., M. W. Bond, R. D. Kornberg. 1986. single      55. Snyder,  <br /> </body></html>