<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Ctf7p essential sister chromatid <br /> cohesion links mitotic chromosome <br /> structure DNA replication <br /> machinery <br />  <br /> CTF7 (chromosome transmission fidelity) gene budding yeast encodes essential protein required high-fidelity chromosome transmission contains regions identity conserved yeast man. ctf7 <br /> mutant cells arrested prior anaphase onset contain separated sister chromatids.  Ctf7p essential cohesion. Cohesion established S phase maintained mitosis.  Ctf7p activity required S phase, suggesting Ctf7p functions establishment cohesion. addition, <br /> ctf7 genetically interacts DNA metabolism mutations pol30 (PCNA) ctf18  RF-C like protein) ctf7 temperature sensitivity chromosome loss rescued high levels POL30. findings provide evidence links establishment sister chromatid cohesion DNA replication machinery suggest assembly cohesion  possibly condensation) complexes coupled PCNA-dependent DNA replication. analysis Ctf7p reveals important connection sister <br /> chromatid cohesion, spindle integrity spindle assemby checkpoint. <br /> [Key Words: CTF7; chromosome loss; BUB3; MAD2; POL30; chromatid cohesion; condensation] <br /> Received October 9, 1998; revised version accepted November 30, 1998. <br />  <br />  <br />  <br />  <br /> Cohesion replicated sister chromatids funda-                  cohesion complexes inactivated sister chro- <br /> mental cell viability. Cohesion tethers replicated sis-               matids segregate ter chromatids entire lengths S                spindle poles. <br /> phase anaphase. Chromatid cohesion sterically                    Despite essential role chromatid cohesion constrains sister kinetochores face away                    mitosis cell division, recently progress  promoting proper chromosome attach-                     elucidating molecular basis form ment microtubules nucleated opposing spindle                     chromosome metabolism. budding yeast, genetic poles (Rieder Salmon 1994; Hyman Sorger 1995).                   biochemical analyses identified complex attachment kinetochores spindle microtubules,                   contains regulators sister chromatid cohesion, sister chromatid experiences microtubule-depen-                     Pds1p Esp1p (Cohen-Fix et al. 1996; Yamamoto et al. <br /> dent poleward forces. poleward forces resisted                 1996 b; Ciosk et al. 1998). Pds1p inhibitor ana- cohesion sister chromatids separate                phase onset degraded metaphase ana- <br /> instead come tension. Observations mei-                    phase transition ubiquitin-dependent process  <br /> otic cells led proposal tension moni-                volving anaphase promoting complex (APC). Pds1p <br /> tored cell ensure sister chromatid                proteolysis thought free Esp1p, turn pro- <br /> properly associated microtubules.                 motes inactivation chromatid cohesion. chromosomes placed tension does cell                   Pds1p&#226;&#8364;&#8220;Esp1p complex functionally related fis- <br /> signal onset anaphase (Nicklas 1997).                    sion yeast Cut1p&#226;&#8364;&#8220;Cut2p complex potential mamma- <br /> chromatid cohesion critical link                    lian homologs Esp1p identified (Funabiki et <br /> mechanism regulates anaphase onset. anaphase,                    al. 1996). addition regulatory proteins,                                                                          candidate cohesion proteins identified: <br /> 4 <br />  Present address: Centre Molecular Medicine Therapeutics, Uni-   Mcd1p/Scc1p, Scc2p, Smc1p, Smc3p budding <br /> versity British Columbia, Vancouver, British Columbia V5Z 4H4,        yeast (Guacci et al. 1997; Michaelis et al. 1997) Canada <br /> 5 <br />  Corresponding author.                                                   Mis4p fission yeast (Furuya et al. 1998). Smc1p E-MAIL skibbens@mail1.ciwemb.edu; FAX (410) 243-6311                     Smc3p members recently discovered gene fam- <br />  <br />  <br />       GENES <span id='am-2' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; DEVELOPMENT 13:307&#226;&#8364;&#8220;319 &#194;&#169; 1999 Cold Spring Harbor Laboratory Press ISSN 0890-9369/99 $5.00; www.genesdev.org      307 <br />  Skibbens et al. <br />  <br />  <br />  <br /> ily required structural maintenance chromo-       fective cohesion centromere arms <br /> somes (SMC) related protein motifs iden-    (Guacci et al. 1997; Michaelis et al. 1997). results <br /> tified Smc1p (Strunnikov et al. 1993). Inactivation    suggest common cohesion mechanism <br /> Mcd1p, Scc2p, Smc1p, Smc3p, Mis4p causes preco-           exists entire chromosome, cohesion cious sister separation M phase. Consistent          chromosome differentially regulated. Finally, <br /> Mcd1p&#226;&#8364;&#8482;s role maintenance cohesion, Mcd1p           Losada et al. (1998) recently reported Mcd1&#226;&#8364;&#8220; <br /> associates chromosomes S phase M           Smc1&#226;&#8364;&#8220;Smc3 protein complex Xenopus dissociates <br /> phase. Mcd1p exists complex Smc1p         chromatin early mitosis, suggesting pro- <br /> requires Smc1p associate chromatin (Guacci et        cess cohesion mechanistically compli- <br /> al. 1997; Michaelis et al. 1997). Xenopus homologs        cated anticipated factors involved Mcd1p, Smc1p, Smc3p, pro-           cohesion identified. <br /> teins, form cohesin complex required inter-        Previously, collection 136 chromosome transmis- <br /> phase maintain cohesion subsequent mitosis         sion fidelity (ctf) mutants identified Saccharomy- <br /> (Losada et al. 1998). similarities yeast   ces cerevisiae basis increased rate loss vertebrate proteins suggest mechanisms chro-     nonessential chromosome fragment (Spencer et al. <br /> matid cohesion likely conserved eukayotes.   1990). ctf mutants allowed transcription    budding yeast, cohesion factor Mcd1p       proceed centromere stable propagation <br /> required chromosome condensation (Guacci et al.          dicentric chromosome  vivo assays kineto- <br /> 1997), revealing important, poorly understood,        chore integrity) proved contain mutations genes <br /> link cohesion condensation. Chromosome           subsequently shown encode kinetochore components <br /> condensation ensures disentanglement sisters         bind yeast centromere (Lechner Carbon compaction chromosome arms away          1991; Doheny et al. 1993; Goh Kilmartin 1993; Jiang <br /> cytokinetic furrow (Koshland Strunnikov 1996).        et al. 1993).  vast majority ctf mutants terestingly, SMC family members, Smc2p         appear correspond integral kinetochore compo- <br /> Smc4p, required condensation appear          nents constitute powerful resource served evolution (Chuang et al. 1994; Hirano      identify proteins involved novel processes crucial Mitchison 1994; Saitoh et al. 1994; Saka et al. 1994;        proper chromosome transmission. paper,  <br /> Strunnikov et al. 1995; Jessberger et al. 1996; Sutani   port CTF7 encodes essential nuclear protein Yanagida 1997). Xenopus, Smc2p (XCAP-E)               required cohesion. analysis CTF7 es- <br /> Smc4p (XCAP-C) homologs combine addi-             tablishes novel connection cohesion tional proteins form condensin complex asso-       DNA replication. <br /> ciates chromosomes (Hirano et al. 1997). Biochemi- <br /> cal analysis Xenopus extracts revealed  <br /> hesin condensin complexes function independent        Results  indicating processes cohesion <br />                                                              Ctf7p essential nuclear protein contains condensation uncoupled cell                                                              domains highly conserved <br /> (Losada et al. 1998). <br />    recent advances iden-      study, characterized ctf7 mutants, tification proteins required cohesion conden-     complementation group ctf collection exhibit <br /> sation, little known cohesion proteins      elevated loss nonessential chromosome fragment <br /> assembled chromatin  assembled,            (Spencer et al. 1990). CTF7 gene cloned complexes regulated. instance, temporal        basis complementation chromosome loss phe- <br /> analysis cohesion human cells genetic analysis     notype confirmed integrative mapping (Materials cohesion yeast shown cohesion estab-      Methods). CTF7 shown encode essential <br /> lished S phase (Selig et al. 1992; Guacci et al.      gene tetrad analyses heterozygous CTF7/ <br /> 1997; Michaelis et al. 1997; Furuya et al. 1998; Uhlmann     ctf7&#226;&#338;&#172; strain plasmid shuffle methods  Materials Nasmyth 1998).  specialized factors          Methods). CTF7 identical unannotated ORF <br /> semble cohesion complexes load complexes            YFR027W chromosome VI (Murakami 1995)  chromatin remain discovered. funda-       codes 281 <span id='am-22' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> protein predicted molecu- <br /> mental issue pertains regulation chromatid      lar mass 31.8 kD (Fig. 1A). search expressed <br /> hesion. early mitotic vertebrate cells, sister chroma-    sequence tag database (dBEST) revealed portions tids paired entire length. metaphase-     Ctf7p contained strong homologies human <br /> arrested cells,  chromosome arms          mouse EST sequences (Fig. 1B). Similar homology <br /> longer paired, centromeres remain            ORF unknown function fission <br /> paired extended period time (Sumner 1991).       yeast. results Ctf7p, essential yeast <br /> similar spatial temporal regulation observed       protein, contains motif conserved king- <br /> meiotic cells: Cohesion lost chromosome          doms suggests Ctf7p-related proteins play <br /> arms meiosis resolve recombinant interme-        role important chromosome transmission eu- <br /> diates, cohesion retained centromeres    karyotic cells. Overexpressed HA-tagged Ctf7p localized <br /> meiosis II. budding yeast, mcd1 mutant cells       nucleus (HA-tagged Ctf7p control  <br />  <br /> 308       GENES &amp; DEVELOPMENT <br />                                                                                                    Chromatid cohesion replication <br />  <br />  <br />  <br />  <br />                                                      Figure 1.   <span id='am-14' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>Nucleotide</span> amino acid sequences CTF7. MluI cell <br />                                                      cycle box 5&#226;&#172;&#732; untranslated region, directs expression                                                      genes required S phase, boxed (Johnston Lowndes 1992; <br />                                                      Toyn et al. 1995). (B) Ctf7p homology fission yeast, mouse human <br />                                                      ESTs shown. Identical residues indicated placement amino <br />                                                      acid sequence, conserved residues indicated plus.                                                      carboxy-terminal region Ctf7p (211&#226;&#8364;&#8220;267 amino acids) 49%, 40%,                                                      41% identical 70%, 63%, 60% similar fission yeast, mouse,                                                      human ESTs, respectively. <br />  <br />  <br />  <br /> endogenous CTF7 promoter undetectable; Materials Methods), consistent Ctf7p&#226;&#8364;&#8482;s essential role chromosome transmission. <br />  <br />  <br /> ctf7 mutants delay mitosis 2C DNA content, <br /> separated DNA masses, partially <br /> elongated spindles investigate function Ctf7p vivo, tem- <br /> perature-sensitive alleles CTF7 obtained PCR-based mutagenesis strategy integrated genome  Materials Methods). permissive <br /> temperature, temperature-sensitive alleles, <br /> ctf7-201, ctf7-202, ctf7-203, exhibited increased <br /> chromosome fragment loss (Fig. 2A), recapitulating original Ctf phenotype. restrictive temperature, <br /> ctf7 mutant strains rapidly lost viability compared wild type, indicating alleles caused  <br /> reversible defect (Fig. 2B). <br />   ascertain terminal phenotype ctf7 mu- <br /> tant strains, log phase cultures mutant shifted 25&#194;&#176;C 37&#194;&#176;C processed assess DNA <br /> content cell morphology. 25&#194;&#176;C, wild-type mutant strains exhibited similar DNA contents,           Figure 2. Phenotypes ctf7 conditional mutants.   ctf7 cells <br /> expected cycling cells.  3 hr growth       (ctf7-201 shown) exhibit elevated chromosome loss wild <br /> 37&#194;&#176;C, significant increase number cells      type. Faithful maintenance reporter chromosome fragment <br /> 2C DNA content observed ctf7 mutant strains              yields white colonies. Loss reporter chromosome results <br />                                                                 red sectored colonies. (B) ctf7 mutants temperature-sen- <br /> compared CTF7 control cells (Fig. 2C). Analyses                                                                 sitive lethal. Log-phase cultures shifted 25&#194;&#176;C 37&#194;&#176;C <br /> cell morphologies time point showed              assayed cell viability. (&#228;&#352;&#8240;) ctf7-201; (&#228;&#352;&#65533;) ctf7-202; (&#228;&#8240;&#177;) ctf7- <br /> &#226;&#710;&#188;51% ctf7 mutant cells large budded              203; (*) wild type. (C) ctf7 mutants delay mitosis. DNA  63% contained separating bi-lobed DNA                 content log phase cultures grown 25&#194;&#176;C 37&#194;&#176;C 3 hr <br /> masses (Fig. 2D; Table 1A). bilobed DNA masses              assessed flow cytometry (ctf7-203 shown). (D) Micro- remained entirely mother cell oc-          graphs ctf7 mutant cell (Phase), DNA (DAPI), microtu- <br /> casionally contained unequal amounts DNA. Spindles           bule (Tubulin) morphologies 3 hr growth 37&#194;&#176;C. <br />  <br />  <br />                                                                                                 GENES &amp; DEVELOPMENT              309 <br />  Skibbens et al. <br />  <br />  <br />  <br /> Table 1.    ctf7 morphologies 37&#194;&#176;C                             mad2-1 ctf7-203 bub3&#226;&#338;&#172; strains exhibited complete <br />                                                                   loss 2C DNA accumulation restrictive tem- <br />                                                                   perature (Fig. 3; data shown). Consistent                                                                   result, ctf7-203 mad2-1 double mutant strain                                                                   failed accumulate partially elongated spindles bi-                                                                   lobed DNA masses, instead proceeded mi- <br /> CTF7          63.2   13.5    6.2        11.9    3.1   0     2.1 <br /> ctf7-203      25.1    9.1    9.7         8.6   31.9   5.7   9.9   tosis evidenced reappearance G1-like (unbud- <br /> B                                                                 ded) cells (Table 1B). results mitotic <br /> CTF7          69.4   12.2    5.2         6.1    4.5   0     2.6   delay ctf7 cells requires functional kinetochore/ <br /> ctf7-203      19.1   15.3    7.4         0     56.3   1.9   0     spindle assembly checkpoint functional DNA <br /> ctf7-203                                                          damage checkpoint. <br />   mad2-1      59.2    8.2    3.1         9.3   15     3.1   2.1 <br />  <br />   ctf7 mutant cells accumulate large budded cells contain- <br /> ing bilobed/elongated nucleus. Cell nuclear morphologies    ctf7 cells delay metaphase prior anaphase onset log-phase CTF7 cft7-203 strains assayed 150 <br />                                                                   Typically, activated kinetochore/spindle assembly <br /> min growth 37&#194;&#176;C. (B) bilobed nuclear morphology ctf7 <br /> mutants depends MAD2 checkpoint. Log-phase cultures <br />                                                                   checkpoint causes cells arrest prior anaphase onset CTF7, ctf7-203, ctf7-203/mad2-1 synchronized       cells contain short spindles single DNA <br /> G1 used &#226;&#65533;&#163;-factor, washed, grown 37&#194;&#176;C 3 hr       mass.  MAD/BUB-dependent mitotic delay <br /> analysis.                                                         ctf7 mutant cells lead accumulation cells                                                                   contained partially elongated spindles separated <br />                                                                   DNA masses. unusual phenotype represent cells atypical             aberrant metaphase. alternative possibility entirely contained mother cell (Fig. 2D)           ctf7 cells delayed midanaphase, suggesting appeared longer preanaphase spindles. quantify            novel postmetaphase role Mad/Bub checkpoint pro- observation, log phase cultures CTF7, ctf7-          teins. determine mitotic delay occurred <br /> 203, ctf13-30  kinetochore mutant arrests              prior anaphase onset, assessed levels Pds1p <br /> prior anaphase onset) shifted 35&#194;&#176;C 3 hr,           wild-type ctf7 mutant cells. Pds1p inhibitor <br /> fixed, immunostained tubulin visualize             anaphase onset degraded metaphase  <br /> spindle. Measurements spindle lengths revealed            anaphase transition (Cohen-Fix et al. 1996; Yamamoto et <br /> ctf7 mutant cells bilobed DNA masses contained               al. 1996a,b). <br /> partially elongated spindles 3.7 &#194;&#181;m (&#194;&#177;0.4, n = 7)              Wild-type ctf7 mutant cells transformed <br /> length; &#226;&#710;&#188;2.5 times longer 1.4 &#194;&#181;m (&#194;&#177;0.3, n = 8)           plasmid directing expression HA&#226;&#8364;&#8220;Pds1p <br /> preanaphase spindles ctf13-30 mutants roughly              (Cohen-Fix et al. 1996). level HA&#226;&#8364;&#8220;Pds1p ana- <br /> half long 6.9 &#194;&#181;m (&#194;&#177;0.9, n = 11) elongated               lyzed synchronized G1 cells released rich <br /> spindles anaphase CTF7 wild-type cells. increased          medium grown 3 hr 37&#194;&#176;C. Western blot analy- <br /> chromosome loss, large-budded cell morphology, 2C                 sis revealed wild-type cells low levels HA&#226;&#8364;&#8220; <br /> DNA content, abnormal spindle morphology,                  Pds1p cells completed mitosis <br /> equal chromatin distribution observed ctf7 mutation <br /> strains indicate loss Ctf7p function leads defect mitosis mitotic delay. <br />  <br />  mitotic delay ctf7 cells requires kinetochore/spindle assembly checkpoint <br /> Mitotic delays typically arise activation DNA damage kinetochore/spindle assembly check- <br /> point pathways. determine ctf7 cells  <br /> quired checkpoint mechanisms delay mitosis, constructed strains containing ctf7 combination mutations DNA damage <br /> checkpoint pathway (rad9&#226;&#338;&#172; mec1-1) (Weinert Hartwell 1988, 1990; Weinert et al. 1994) mutations kinetochore/spindle assembly checkpoint pathway <br /> (mad2-1 bub3&#226;&#338;&#172;) (Hoyt et al. 1991; Li Murray <br /> 1991). Log phase cultures resulting strains shifted 37&#194;&#176;C 3 hr DNA content measured           Figure 3. mitotic delay ctf7 cells requires kineto- flow cytometry. rad9&#226;&#338;&#172; mec1-1 mutations             chore/spindle checkpoint pathway. Flow cytometry analysis abrogated ctf7-203 mediated 2C DNA accumulation               DNA content log phase ctf7, checkpoint mutant, double- <br /> (Fig. 3, data shown). contrast, ctf7-203              mutant strains grown 3 hr 25&#194;&#176;C (left) 37&#194;&#176;C (right). <br />  <br />  <br /> 310        GENES &amp; DEVELOPMENT <br />                                                                                                  Chromatid cohesion replication <br />  <br />  <br />  <br />                                                                  FISH ask sister chromatid cohesion normal                                                                  ctf7 mutant cells. wild-type cells arrested mitosis, <br />                                                                  &#226;&#710;&#188;10% DNA masses contained FISH sig- <br />                                                                  nals using CEN16 proximal DNA probe (Fig. <br />                                                                  5A,B). contrast, &#226;&#710;&#188;40% DNA masses ctf7 <br />                                                                  cells contained FISH signals, sister cen- <br />                                                                  tromeres residing opposite sides nuclear masses <br />                                                                  (Fig. 5A,B). ctf7 defect affected chromosomes, <br />                                                                  &#226;&#710;&#188;40% DNA masses ctf7 cells contained FISH <br />                                                                  signals CEN1 proximal probe (Fig. 5B). Finally,                                                                  similar loss cohesion obtained use centro- <br />                                                                  mere-distal (arm) probes chromosome XVI, indicating <br />                                                                  Ctf7p required cohesion entire <br />                                                                  length chromosome (Fig. 5B). findings  <br />                                                                  veal Ctf7p essential sister chromosome cohe- <br />                                                                  sion supports previous findings cohesion  <br /> Figure 4. ctf7 cells delayed prior anaphase onset.  . Western blot shows HA&#226;&#8364;&#8220;Pds1p levels wild-type ctf7     diated mechanism common chro- <br /> cells synchronized G1 released     mosome (Guacci et al. 1997). <br /> dium grown 37&#194;&#176;C 3 hr. Tubulin levels verified equal protein loads present lane. (B) Micrographs localization DNA (DAPI) HA&#226;&#8364;&#8220;Pds1p (HA&#226;&#8364;&#8220; <br /> Pds1p) ctf7 mutant wild-type cells treated  <br />  <br />  <br />  <br /> (Fig. 4A). contrast, ctf7 cells contained high HA&#226;&#8364;&#8220;Pds1p <br /> levels, indicating ctf7 cells delayed prior anaphase onset. confirm ctf7 cells delayed metaphase, HA&#226;&#8364;&#8220;Pds1p visualized immunoflu- <br /> orescence wild-type ctf7 cells treated  expected, wild-type cells separating DNA masses <br /> did contain HA&#226;&#8364;&#8220;Pds1p (Cohen-Fix et al. 1996). contrast wild-type cells, significant population ctf7 cells separating DNA masses contained high <br /> levels HA&#226;&#8364;&#8220;Pds1p (Fig. 4B).  ctf7 cells delayed prior anaphase onset. surprising ctf7 cells separating DNA masses did contain detectable HA&#226;&#8364;&#8220;Pds1p levels, ctf7 cells delayed cell cycle eventually exit <br /> mitosis. results reveal partial spindle elonga- <br /> tion DNA separation occurs prior anaphase onset ctf7 mutant cells. <br />  <br />  <br /> Ctf7p required sister chromatid cohesion cells delayed prior anaphase onset sepa- <br /> rated DNA masses? possibility defects sister chromosome cohesion allowed sister chromatids separate prematurely, resulting movement            Figure 5. Ctf7p required sister chromatid cohesion.   <br /> disjoined sisters spindle poles. test         Micrographs wild-type ctf7 mutant cells grown 3 hr model, wild-type ctf7-203 cells synchronized         37&#194;&#176;C medium containing nocodazole processed FISH. <br /> G1 &#226;&#65533;&#163;-factor, released medium contained            single FISH signal represents paired sister chromatids, nocodazole, grown 3 hr 37&#194;&#176;C. Nocodazole               FISH signals represents disjoined sister chromatids. (red) Prop- <br /> blocks microtubule assembly anaphase onset              idium iodide staining DNA mass; (green) hybridization                                                                  CEN16 probe. (B) Quantification single (nondisjoined chro- sister chromatids typically remain closely ap- <br />                                                                  mosomes) vs. double FISH signals (disjoined chromosomes) <br /> posed extended periods time. FISH used           propidium iodide staining DNA masses. Synchronized <br /> previously demonstrate pairing sister chroma-          G1 wild-type ctf7 cells contained equal percentage tids M-phase yeast cells assess           single FISH signals (data shown), indicating  <br /> pairing different loci centromere chro-       crease FISH signals observed ctf7 mutant cells mosome arms (Guacci et al. 1994).  used             pre-existing aneuploid. <br />  <br />  <br />                                                                                                GENES &amp; DEVELOPMENT            311 <br />  Skibbens et al. <br />  <br />  <br />  <br /> Ctf7p required S phase                                     &#226;&#65533;&#163;-factor release prior DNA replication) (Fig. 6A,B). <br />                                                                      Cultures exhibited high viability shifted 37&#194;&#176;C <br /> Sister chromatid cohesion apparently established dur-             80 min growth 23&#194;&#176;C (Fig. 6A). increase ing S phase maintained anaphase onset.                  viability correlated progression DNA replication: <br /> assess role Ctf7p establishment main-              Maximum viability realized bulk tenance cohesion, used reciprocal temperature-                 DNA replication completed (80 min) (Fig. 6B). Fur- <br /> shift experiments ctf7 cells map portion           thermore, maximal cell viability preceded anaphase  <br /> cell cycle requires Ctf7p function (Fig. 6). shift         set (spindle elongation) &gt;20 min (Fig. 6A).  <br /> experiment marks earliest point cell cycle                sults suggest Ctf7p required S phase Ctf7p inactivated reducing cell vi-              mitosis. <br /> ability  e., Ctf7p function longer essential).            shift experiment marks latest point ctf7-203 cells synchronized G1 &#226;&#65533;&#163;-factor             cell cycle Ctf7p active produce viable released medium 23&#194;&#176;C allow cells              cells. ctf7-203 cultures synchronized G1 proceed cell cycle. Starting release              &#226;&#65533;&#163;-factor 23&#194;&#176;C, released 37&#194;&#176;C rich medium &#226;&#65533;&#163;-factor (t = 0), aliquots shifted 37&#194;&#176;C 20           absence &#226;&#65533;&#163;-factor allows cells proceed min intervals. time shift, aliquots          cell cycle. Aliquots shifted 23&#194;&#176;C <br /> harvested flow cytometry immunofluorescence                  20 min intervals harvested flow cytometry <br /> analyses position cells cell cycle  Ma-            immunofluoresence analyses described  terials Methods). end experiment,              release &#226;&#65533;&#163;-factor (t = 0), cells retained high viability <br /> samples plated 23&#194;&#176;C assess viability (Materials           80 min growth 37&#194;&#176;C prior shifting Methods). Cultures exhibited low viability                  23&#194;&#176;C (Fig. 6C). Decreased cell viability (starting t = 80) <br /> shifted 37&#194;&#176;C time 60 min               correlated entry DNA replication (Fig. 6C,D) <br />  <br />  <br />  <br />  <br /> Figure 6. Ctf7p required S phase.  B) Shift experiment: G1 ctf7-203 cells released &#226;&#65533;&#163;-factor shifted 23&#194;&#176;C 37&#194;&#176;C 20&#226;&#8364;&#8220;30 min increments. (C,D) Shift experiment: G1 ctf7-203 cells released &#226;&#65533;&#163;-factor 37&#194;&#176;C <br /> medium shifted 25&#194;&#176;C 20&#226;&#8364;&#8220;30 min increments. Wild-type strains retain high viability procedures (data shown). time point, aliquot assayed spindle elongation/DNA mass separation (&#226;&#189;&#167;), viability (&#228;&#352;&#65533;), DNA <br /> content (flow cytometry profile). (E,  Viability G1 synchronized wild-type, ctf7-203, mcd1-1 cells released medium <br /> containing nocodazole permitted arrest mitosis 23&#194;&#176;C prior shifting 37&#194;&#176;C. (E,  Viability G1 synchronized <br /> wild-type, ctf7-203, mcd1-1 cells released medium containing nocodazole 37&#194;&#176;C. 10-fold dilution culture shown. <br />  <br />  <br /> 312       GENES &amp; DEVELOPMENT <br />                                                                                                  Chromatid cohesion replication <br />  <br />  <br />  decline cell viability coincided pro-    sister centromeres, Ctf7p required mainte- <br /> gression S phase.  Ctf7p active        nance sister chromatid cohesion. <br /> cells enter S phase obtain viable cells. Activation later cell cycle resulted cell death. finding experiment ctf7 mutants progressed          Genetic interactions CTF7 DNA S phase kinetics similar wild-type cells       replication machinery <br /> (data shown).  lesion incurred S <br />                                                                Typically, high copy suppressors proteins phase ctf7 mutant cells manifested defect                                                                interact target protein function parallel <br /> DNA replication. combination results                                                                pathway. gain insight role Ctf7p shift shift experiments ctf7 cells suggest <br />                                                                cohesion, high-copy suppressors temperature sen- Ctf7p function essential S phase  <br />                                                                sitive ctf7 phenotype identified. plasmids <br /> hesion established mitosis cohesion                                                                obtained enabled ctf7 mutant cells grow maintained. <br />                                                                34&#194;&#176;C 37&#194;&#176;C. plasmids contained <br />    decided use second method test                                                                overlapping DNA sequences, suggesting suppression Ctf7p required M phase S phase. Mcd1p <br />                                                                single gene. ctf7 temperature sensitivity appears required M phase S phase <br />                                                                suppressed minimal DNA sequence defined BglII included indicator loss cohesion <br />                                                                StuI restriction sites (Fig. 7A). Sequence analysis  <br /> (Guacci et al. 1997). test Ctf7p required <br />                                                                vealed 1.7-kb fragment contained POL30, mitosis, wild-type, ctf7-203, mcd1-1 mutant <br />                                                                encodes DNA replication processivity factor <br /> cells synchronized G1 released medium  <br />                                                                proliferating cell nuclear antigen (PCNA) (Bauer taining nocodazole allowed proceed G1                                                                Burgers 1990; Burgers 1991). frame-shift mutation mitosis permissive temperature 23&#194;&#176;C. Aliquots <br />                                                                POL30 ORF rendered resulting plasmid inca- cultures shifted 37&#194;&#176;C 1 hr inter- <br />                                                                pable rescuing ctf7-203 temperature sensitivity, indi- <br /> vals. incubation 37&#194;&#176;C, cultures plated <br />                                                                cating POL30 high-copy suppressor ctf7-203. allowed grow 2 days 23&#194;&#176;C assess viability <br />                                                                Overexpression POL30 rescued temperature <br /> (Fig. 6E). Wild-type cells adversely affected procedure maintained high viability. ex- <br /> pected, mcd1-1 cells exhibited dramatic decrease viability hour incubation 37&#194;&#176;C. contrast, ctf7-203 cells retained high viability 3 hr incubation 37&#194;&#176;C.  Ctf7p required dur- <br /> ing mitosis sister chromatid cohesion main- <br /> tained.  Ctf7p required S phase cohesion established. ctf7-203 mcd1-1 mutant <br /> cells exhibited decreased viability synchro- <br /> nized cells allowed proceed G1 mitosis 37&#194;&#176;C. combination results suggest Ctf7p plays role cohesion distinct Mcd1p. <br />    high viability ctf7-203 cells allowed proceed mitosis shifting restrictive tem- <br /> perature implied Ctf7p required maintenance cohesion. demonstrate unam- <br /> biguously, performed FISH analysis wild-type ctf7-203 cells.  G1 synchronized cells  <br /> leased medium containing nocodazole allowed proceed mitosis permissive temperature. 3 hr 23&#194;&#176;C, cultures shifted 37&#194;&#176;C held restrictive temperature 2 hr. Aliquots     Figure 7. POL30 overexpression suppressor ctf7 mu- cultures processed FISH analysis use      tant phenotypes.   7.1-kb insert identified sup- CEN1 CEN16 proximal probes. Wild-type              pressed ctf7 temperature sensitivity 34&#194;&#176;C. vector <br /> cells retained sister chromatid cohesion following        (Mock) vector containing insert DNA sequences procedure, 22% (CEN1) 23% (CEN16)           left right Sall digestion site rescued ctf7 <br /> DNA masses contained FISH signals. ctf7-203 cells          mutant 34&#194;&#176;C. 1.7-kb fragment flanked Bglll Stul retained sister chromatid cohesion, 26% (CEN1)        restriction sites rescued ctf7 mutant temperature sensitivity <br />                                                                level growth CTF7 (Vector + CTF7). <br /> 27% (CEN16) DNA masses contained FISH <br />                                                                hatched box starting clone denotes POL30 OFR. (B) ctf7 <br /> signals (data shown). slight increase num-      mutations containing reporter chromosomes transformed <br /> ber DNA masses contained FISH signals ob-          2 &#194;&#181;m vector &#194;&#177; POL30 grown medium allowed <br /> served strains likely pro-        chromosome fragment loss retention 2 &#194;&#181;m plas- <br /> longed mitotic arrest exposure nocodazole.         mid (ctf7-108 non-temperature-sensitive strain obtained use criteria viability localization       original ctf collection). <br />  <br />  <br />                                                                                               GENES &amp; DEVELOPMENT              313 <br />  Skibbens et al. <br />  <br />  <br />  <br /> sensitivity ctf7-201 failed rescue ctf7&#226;&#338;&#172;1 dele-           high-copy suppressor ctf7 phenotypes. genetic <br /> tion strain  shown).                                              interactions link Ctf7p proteins required <br />    basis finding POL30 high-copy              S phase, consistent mapping experi- <br /> suppressor ctf7 temperature sensitivity, asked                  ments, suggest establishment POL30 rescue ctf7 chromosome loss.                 sister chromatid cohesion facilitated components test possibility, POL30 overexpressed                  DNA replication machinery. <br /> strains containing ctf7 temperature-sensitive alle- <br /> les (ctf7-201 ctf7-203) non-temperature-sensi- <br /> tive allele (ctf7-108) obtained original ctf          Discussion <br /> collection (Spencer et al. 1990). Identical results                                                                       Ctf7p, novel link sister chromatid cohesion, <br /> obtained case: Overexpression POL30  <br />                                                                       S phase, DNA replication machinery <br /> creased chromosome loss associated ctf7 <br /> mutations tested (Fig. 7B).                                           study, identified Ctf7p protein  <br />    test genetic interaction               quired sister chromatid cohesion. HA&#226;&#8364;&#8220;Ctf7p CTF7 POL30 reflected important link                    nuclear protein mutations CTF7 result elevated establishment sister chromatid cohesion                    chromosome loss, consistent Ctf7p&#226;&#8364;&#8482;s role chro- <br /> DNA replication machinery, identify gene               mosome segregation  paper; Spencer et al. 1990). products require CTF7 function, assayed                 discovered Ctf7p required cohesion synthetic lethal interactions. strain harboring             centromeres chromosome arms, ctf7-203 allele mated strains containing                cohesion factor Mcd1p  paper; Guacci et al. <br /> checkpoint, kinetochore, DNA metabolism,                     1997; Michaelis et al. 1997). Independent analysis chromosome loss mutants. inability recover                     shown Ctf7p (called Eco1p) required  <br /> double-mutant spores scored synthetic lethal                 hesion   Toth, R. Ciosk, F. Uhlmann, M. Galova,  <br /> interaction (Table 2). Synthetic lethal interactions             Schleiffer, K. Nasmyth, pers. comm.).  al- <br /> observed ctf7-203 combination ctf18&#226;&#338;&#172;1              evidence vertebrate cell chromo- pol30-104. CTF18 encodes RF-C-like protein                  somes cohesion differentially regulated RF-C complexes known load PCNA double-                    length chromosome (Sumner 1991), Ctf7p stranded DNA (Burgers 1988; Kouprina et al. 1994; Ay-                 Mcd1p appear function aspects cohesion yagari et al. 1995; <span id='am-29' about='obo:IMR_0001693' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-30' property="rdfs:label" content="amin" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0001693" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="CHEBI:32952" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="MeSH:D000588" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="A group of compounds derived from ammonia by substituting organic radicals for the hydrogens. (From Grant &amp; Hackh&apos;s Chemical Dictionary, 5th ed)" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Amin</span> Holm 1996). contrast               common regions chromosome. findings, synthetic lethal interactions ob-                Cell viability sister chromatid cohesion abol- <br /> served ctf7-203 combined checkpoint                     ished ctf7 cells placed restrictive tem- <br /> mutants (rad9&#226;&#338;&#172;, mec1-1, mad2-1, bub3&#226;&#338;&#172;), kineto-                    perature S phase mitosis. chore mutants (ctf13-30, ctf14-42, skp1-4, skp1-3),             findings demonstrate Ctf7p, unlike Mcd1p, func- mutants involved DNA replication (ctf4-25,                tions specifically establishment cohesion  allele Pol&#226;&#65533;&#163;-binding protein, cdc6/ctf10-103,               paper; Guacci et al. 1997; Michaelis et al. 1997).  allele ORC interacting protein). 12               dependent allele ctf7 (eco1-1) defective mutant strains tested (Table 3), synthetic lethal-            establishment maintenance cohesion   <br /> ity observed ctf7 combination pol30              Toth, R. Ciosk, F. Uhlmann, M. Galova,  Schleiffer, K. ctf18 mutations. specificity genetic               Nasmyth, pers. comm.), indicating inability teractions substantiated related functions               reveal function Ctf7p maintenance cohe- <br /> PCNA Ctf18p finding POl30                    sion allele-specific effect. model  <br />  <br /> Table 2.    ctf7-203 synthetically lethal cft18&#226;&#338;&#172;1 pol30-104 <br />  <br />                                                                                           Observed          Expected <br />  <br /> pol30&#226;&#8364;&#8220;104   mms <br />                   &#226;&#8364;&#8220;LEU2 &#195;&#8212; ctf7&#226;&#338;&#172;&#226;&#172;&#8249;HIS3 ctf7-203 :LEU2 <br />                                              ts <br />                                                              61% viable <br />                                                                (68 0f 112 spores) <br />                                                                wild type                     32+               28 <br />                                                                pol30                         28                28 <br />                                                                ctf7                           8                14 <br />                                                                pol30 ctf7                     0                14 <br />   ctf18&#226;&#338;&#172;&#226;&#172;&#8249;URA3 &#195;&#8212; ctf7&#226;&#338;&#172;&#226;&#172;&#8249;HIS3 ctf7-203ts:LEU2                   55% viable <br />                                                                (57 104 spores) <br />                                                                wild type                     30                26 <br />                                                                ctf18&#226;&#338;&#172;                        13                26 <br />                                                                ctf7                          14                13 <br />                                                                ctf18&#226;&#338;&#172; ctf7                    0                13 <br />  <br /> ctf7-203 mated pol30-104 ctf18&#226;&#338;&#172;1, resulting diploid dissected identify spores contained mutations. expected observed number wild-type, single double mutant spores shown cross. Total spore viability shown. <br />  <br />  <br /> 314        GENES &amp; DEVELOPMENT <br />                                                                                            Chromatid cohesion replication <br />  <br />  <br /> Table 3.   Strains used study <br />  <br /> Strains                                          Genotypes                                             References <br />  <br /> YBS2     MATa/MAT&#226;&#65533;&#163; ura3-52/ura3-52 lys2-801/lys2-801 ade2-101/ade2-101 trpl-&#226;&#338;&#172;63/trp1-&#226;&#338;&#172;63      study <br />           his3-&#226;&#338;&#172;200/his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;/leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3/CTF7 CFIII (CEN3.L) TRP1 SUP11 <br /> YBS3     MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 CTF7:HIS3 <br /> YBS5     MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 CTF7&#226;&#172;&#8249;HIS3 CFIII <br />           (CEN3.L) TRP1 SUP11 pRS316CTF7 <br /> YBS204   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7:LEU2 <br />           rad9&#226;&#338;&#172;1&#226;&#172;&#8249;TRP1 <br /> YBS205   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 <br />           ctf7-203:LEU2 rad9&#226;&#338;&#172;1&#226;&#172;&#8249;TRP1 <br /> YBS213   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1,63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7:LEU2 <br />           mad2-1 <br /> YBS214   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1,63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 <br />           ctf7-203:LEU2 mad2-1 <br /> YBS504   MATa ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7:LEU2 <br />           bub3&#226;&#338;&#172;1&#226;&#172;&#8249;LEU2 <br /> YBS568   MATa ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 ctf7-203:LEU2 <br />           bub3&#226;&#338;&#172;1&#226;&#172;&#8249;LEU2 <br /> YBS255   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7:LEU2 <br /> YBS514   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 <br />           ctf7-203:LEU2 <br /> YBS268   MAT&#226;&#65533;&#163; ura3 trp1 his3 leu2 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 ctf7-203:LEU2 mec1-1:HIS3 <br /> YBS271   MAT&#226;&#65533;&#163; ura3 trp1 his3 leu2 ctf7&#226;&#338;&#172;1&#226;&#172;&#8249;HIS3 CTF7&#226;&#8364;&#8220;LEU2 mec1-1:HIS3 <br /> YCTF108 MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1 leu2-&#226;&#338;&#172;1 ctf7-108 CFIII (CEN3.L) URA3 SUP11     Spencer et al. (1990) <br /> YCTF103 MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1 leu2-&#226;&#338;&#172;1 ctf10-103 CFIII (CEN3.L) URA3 <br />           SUP11 <br /> YCTF42   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf14-42 CFIII (CEN3.L) URA3 <br />           SUP11 <br /> YCTF30   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf13-30 CFIII (CEN3.L) URA3 <br />           SUP11 <br /> YCTF25   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 cf4-25 CFIII (CEN3.L) URA3 SUP11 <br /> YCTF10   MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf7-10 CFIII (CEN3.L) URA3 <br />           SUP11 <br /> YPH1182  MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 skp1-4:LEU2                Connelly Heiter (1996) <br /> YPH1188  MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 skp1-3:LEU2 <br /> YCH2165 MATa leu2-3,112 ura3-52 POL30:LEU2                                                    Amin Holm (1996) <br /> YCH2161 MATa leu2-3,112 ura3-52 pol30&#226;&#8364;&#8220;104:LEU2 <br /> YJE106   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 ctf18&#226;&#338;&#172;1:LEU2               Kouprina et al. (1994) <br /> YJE188   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1 ctf18&#226;&#338;&#172;1&#226;&#172;&#8249;URA3 <br /> DLY285   MATa ura3 trp1 his3 leu2 trp1 mec1-1&#226;&#8364;&#8220;HIS3                                            Paulovich (1995) <br /> YCB624   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 rad9&#226;&#338;&#172;1&#226;&#172;&#8249;TRP1                Brachmann (1995) <br /> YFS451   MATa ura3-52 lys2-801 ade2-101 trp1-&#226;&#338;&#172;1,63 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 mad2-1 CFIII (CEN3.L)    Pangilinan (1996) <br />           URA3 SUP11 <br /> YFP74    MAT&#226;&#65533;&#163; ura3-52 lys2-801 ade2-101 his3-&#226;&#338;&#172;200 leu2-&#226;&#338;&#172;1 bub3&#226;&#338;&#172;1&#226;&#172;&#8249;LEU2 <br /> VG955-7D MATa mcd1-1 trp1 leu2 bar1 gal1                                                      Guacci et al. (1997) Materials Methods details. <br />  <br />  <br /> Ctf7p functions establishment cohesion       CTF7, POL30, CTF18 appear  <br /> S phase supported genetic interactions      flect general response ctf7 mutant strains alter- <br /> observed mutations CTF7 DNA             ations S-phase progression. instance, ctf18&#226;&#338;&#172;1 <br /> tabolism genes POL30 CTF18 (Kouprina et al. 1994;         pol30-104 cold-sensitive mutations  el- <br /> Amin et al. 1996). POL30 gene         evated temperatures, exhibit normal S-phase kinetics <br /> genome-wide screen  high copy, suppressed ctf7        (Kouprina et al. 1994; Amin Holm 1996). addition, <br /> temperature sensitivity chromosome loss. addi-         ctf7 mutant strain temperature sensitivity sup- <br /> tion, pol30-104 ctf18&#226;&#338;&#172;1 synthetically lethal     pressed levels hydroxyurea slow DNA elonga- <br /> ctf7-203. significance genetic interactions   tion (R.V. Skibbens D. Koshland, unpubl.). Taken <br /> augmented specificity; synthetic lethal inter-     observations suggest Ctf7p functions <br /> actions observed ctf7 combination mu-        S phase establishment cohesion tations affect initiation DNA replica-     reveals important link DNA replication tion, kinetochore function, DNA damage spindle          sister chromatid cohesion. <br /> assembly checkpoint pathways. genetic interactions           connection CTF7 POL30 particu- <br />  <br />  <br />                                                                                         GENES &amp; DEVELOPMENT             315 <br />  Skibbens et al. <br />  <br />  <br />  <br /> larly intriguing. PCNA forms doughnut-like sliding            structure? preanaphase budding yeast cells, spindle <br /> clamps continuously loaded DNA poly-          pole bodies mechanically linked interdigi- <br /> merases replication fork (Krishna et al. 1994). Oth-   tating polar microtubules kinetochore mi- <br /> ers noted PCNA appears landing           crotubules indirectly connected virtue pads variety proteins directly related DNA    attachment paired sister chromatids (O&#226;&#8364;&#8482;Toole et <br /> replication play role chromatin remodeling       al. 1997). propose dissolution sister chro- <br /> (Kelman 1997). combination results,         matid cohesion complexes weakens linkage pro- <br /> data suggest loading cohesin complexes       foundly affects spindle integrity. Abnormal spindles <br /> chromatin coupled PCNA-dependent DNA rep-             reported mutants defective cohe- <br /> lication. model attractive places cohe-    sion smc1 mcd1 (Strunnikov et al. 1993; <br /> sin complexes proximal newly replicated sister chro-       Guacci et al. 1997; Michaelis et al. 1997).  sister <br /> matids ensure pairing sister chromatids         chromatid cohesion appears function preclude pairing nonsister chromatids. Loading      mechanical linkage required maintain spindle <br /> cohesin complexes chromatin uncoupled             integrity. replication, observed Xenopus ex-            second question raised observations tracts budding yeast cells, yeast    ctf7 mutants activate kinetochore/spindle assem- <br /> association fails generate functional sister chromatid     bly checkpoint delay mitosis? <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, kineto- <br /> cohesion (Losada et al. 1998; Uhlmann Nasmyth             chore/spindle damage checkpoint pathway im- <br /> 1998).                                                        plicated monitoring defects centromere, kineto- <br />    budding yeast, Mcd1p required cohesion      chore, spindle function responds delaying cells condensation chromosome arms, providing             mitosis (Skibbens Hieter 1998). Currently, link processes (Guacci et al.     rule model ctf7 cells contain 1997). Given precedent, Ctf7p          defective kinetochores sister chromatid cohesion required condensation rDNA locus, al-         complexes.  necessary invoke ki- defect ctf7 mutants pronounced       netochore defect explain checkpoint-mediated  observed mcd1 mutants (R.V. Skibbens D.        lay ctf7 cells. Interestingly, cohesion mutants <br /> Koshland, unpubl.). Concomitant study, Trf4          exhibit mitotic delay, unknown <br /> protein shown required        delay dependent spindle assembly <br /> hesion condensation (Castano et al. 1996;  Castano      checkpoint (Strunnikov et al. 1993; Guacci et al. 1997; M. Christman, pers. comm.).  study          Michaelis et al. 1997). basis evidence, Trf4p study ctf7p establishes         envision cohesion defects activate checkpoint yeast cohesin condensin complexes share            altering state tension exerted chromosomes. <br /> common components, regulators, assembly factors.        Chromosomes attached microtubules experience <br /> contrast, Losada et al. (1998) reported depleting     poleward forces place chromosome  <br /> hesin components Xenopus extracts did ad-            sion. Evidence meiotic cell systems revealed versely affect chromosome condensation. addition,          tension exerted chromosomes critical signaling <br /> Xenopus 13S condensin 14S cohesin complexes          anaphase onset (Nicklas 1997): decrease loss  associate chromatin independent          sion generates mitotic delay. predicted effect  indicating separation function (Losada et al.   dissolution chromatid cohesion decrease 1998). important pursue cou-        level tension exerted chromosomes spindle mi- <br /> pling/uncoupling cohesion condensation reflect         crotubules activate spindle assembly checkpoint <br /> differences organisms, cell systems (somatic versus        (Wells Murray 1996). model suggests sister <br /> embryonic) segregation cycles (mitotic versus mei-         chromatid cohesion fundamental component otic). differences documented         mechanism maintains tension chromo- appeared conflicting data historically       somes. <br /> proven tremendous scientific value.                     Typically, checkpoint mechanisms maintain cell vi- <br />                                                               ability delaying cell cycle allowing cells time <br />                                                               repair damage correct mistakes. mutations Cohesion, checkpoints, spindle integrity <br />                                                               induce damage, additional loss checkpoint results showed ctf7 mutant cells exhibited         function results synthetic lethality.  mitotic delay required kinetochore/spindle        question ctf7 synthetically lethal <br /> sembly checkpoint. Consistent preanaphase         mutations kinetochore/spindle assembly <br /> lay typically imposed checkpoint, ctf7 cells      checkpoint? loss checkpoint little impact <br /> tained high levels anaphase inhibitor Pds1p.        ctf7 cells reasons.  component hand, ctf7 cells contained elongated spindles       checkpoint mechanism block sister separation. separated DNA masses, uncharacteristic preana-          ctf7 cells defective sister chromatid  <br /> phase cells. aberrant metaphase delay raises       hesion subvert component checkpoint func- <br /> issues reveal important functions sister      tion. Second, sister separation irreversible event. <br /> chromatid cohesion.                                            cell delays response     does ctf7 mutation disrupt spindle          checkpoint function, cells precociously separated <br />  <br />  <br /> 316       GENES &amp; DEVELOPMENT <br />                                                                                                       Chromatid cohesion replication <br />  <br />  <br />  <br /> sisters rescue. results analogous              Generation, integration suppression conditional reported decade ago, double-strand breaks               ctf7 alleles <br /> generated G1 haploid cells resulted RAD9-medi-               PCR-based mutagenesis pRS316&#226;&#8364;&#8220;CTF7 performed  <br /> ated cell cycle delay G2 /M. Despite delay,              scribed (Kassenbrock et al. 1993) minor modifications. <br /> cells inviable (Weinert Hartwell 1988).                     YBS5 (ctf7&#226;&#338;&#172; covered pRS316&#226;&#8364;&#8220;CTF7) cotransformed                                                                      pooled PCR product MluI&#226;&#8364;&#8220;NdeI-gapped pRS315&#226;&#8364;&#8220;CTF7                                                                       digestion removes entire CTF7 ORF leaves 5&#226;&#172;&#732; 3&#226;&#172;&#732; <br />                                                                      ends homologous CTF7 genomic locus). 12,500 Leu+ <br /> Materials methods                                                transformants screened, 11 scored positive temperature-sen- <br />                                                                      sitive nonpetite strains virtue inability grow Yeast strains medium                                             5-FOA plates 37&#194;&#176;C growth glycerol plates 25&#194;&#176;C. <br /> Medium used growth sporulation described previ-          Plasmids containing putative mutated CTF7 sequences ously (Rose et al. 1990). Yeast transformations performed       recovered strains. YBS5 transformed described (Ito et al. 1983; Schiestl Gietz 1989). S288C-      rescued plasmids confirm plasmid-dependent tempera- <br /> derived Saccharomyces cerevisiae yeast strains (YPH) plas-       ture sensitivity. plasmids exhibited mids (pRS) described previously (Sikorski Hieter 1989;      lowest reversion rate conferred tightest temperature <br /> Doheny et al. 1993; Connelly Hieter 1996). strains     sensitivity chosen integration genome. listed Table 3. establish genetic interactions, inde-      2.1-kb insert cloned pRS304 resulting plasmid <br /> pendent double heterozygous diploids obtained, sporu-           linearized HpaI target integration LEU2 locus lated, haploid spores containing mutations identified.      YBS5. passage 5-FOA,  Leu+ Ura&#226;&#710;&#8217; transformants nonessential reporter chromosome &#226;&#710;&#188;150 kb contains         temperature sensitive growth. temperature-sensi- <br /> short arm centromere chromosome III stable propa-         tive phenotype rescued retransformation pRS316&#226;&#8364;&#8220; <br /> gation SUP11 suppress formation red byprod-        CTF7.  mutations recessive. <br /> uct <span id='am-9' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span>adenine</span> biosynthetic pathway ade2 mutations               overexpression suppression, ctf7-203 transformed <br /> (Spencer et al. 1990).                                               2 &#194;&#181;m URA3 library (Connelly et al. 1996). 24,900 <br />                                                                      transformants, 26 plasmids obtained  retransfor- <br />                                                                      mation, rescued growth 34&#194;&#176;C. twenty- <br />                                                                      plasmids contained CTF7 ORF. remaining Cloning molecular characterization CTF7                       contain overlapping sequences plasmid <br />                                                                      subcloned identify suppressor ORF. region DNA rescued chromosome fragment-loss <br /> phenotype ctf7 mutant strain (ctf7-108) original ctf <br /> collection (Spencer et al. 1990) identified CEN/ <br />                                                                      Morphological growth analyses <br /> LEU2 library subcloned following procedures reagents <br /> described previously (Gerring et al. 1990). 2.1-kb region       Flow cytometry, Western blot analysis, indirect immuno- shown encode chromosome transmission fidelity             fluorescence (IIF) performed described previously (Ger- <br /> locus CTF7 following manner. 2.1-kb fragment           ring et al. 1990; Cohen-Fix et al. 1996) minor modifica- cloned pRS303, linearized CTF7 ORF          tions. Viability determined described previously (Mor- <br /> XbaI integrated genome YPH250.  transfor-        row et al. 1995) minor modifications. Briefly, cells mants shown contain HIS3-marked CTF7 integrated           plated experimental procedures rich YPD medium proper locus Southern analysis basis known      plates grown permissive temperature 18 hr. restriction map region. resulting strain (YBS3)       plates assessed microscopically formation mated strain original collection contains       microcolonies. Typically, microcolonies rescored ctf7-10 (Spencer et al. 1990) sporulated.  spores (22     additional 24 hr. execution experiments, incidence 22) exhibited wild-type levels chromosome loss,       anaphase onset based spindle elongation sepa- <br /> 14  spores tested sectored dramatically grown non-       ration DNA masses. shift experiments, percent <br /> selective medium. Database searches yeast genome indi-        anaphase cells rose dramatically cell cycle <br /> cated CTF7 contained Chromosome VI. chro-            delay observed ctf7 mutants, consistent accumula- <br /> mosome assignment confirmed hybridization yeast            tion 2C DNA content. localization Ctf7p, entire <br /> genomic clone contigs (L. Riles M. Olson, pers. comm.).          CTF7 ORF placed frame encoded HA epitope <br />    CTF7 ORF replaced selectable marker HIS3         tag contained pAD5 (Neiman et al. 1993). YBS5 trans- PCR strategy described previously (Baudin et al. 1993).     formed plasmid pRS316-CTF7 shuffled deletion DNA fragments generated PCR primers          resulting strain use medium plates containing 5-Fluo- <br /> 5&#226;&#172;&#732;-AGTAATAATACATAAGGAGATGCCAGCTACAACGG-                              rorotic acid, FOA (Boeke et al. 1987) prior immunolocaliza- <br /> ACATCAACATATATCTAGATTGTACTGAGAGTGCACC-3&#226;&#172;&#732;                             tion. Immunostaining performed use anti-HA  5&#226;&#172;&#732;-ACCTTGAAAAGTTTTATGCCATTTTTCCAGTGT-                            tibody 12CA5 (BabCo) combination fluorescein- Cy3- <br /> CCCTTCTCGCTGTCTTTCTGTGCGGTATTTCACACCG-3&#226;&#172;&#732;.                            conjugated goat anti-mouse antibodies (Jackson Immuno- wild-type diploid strain YPH987 transformed             Research) visualize epitope-tagged Ctf7p Pds1p (Cohen- <br /> PCR product His3+ colonies identified obtain hetero-        Fix et al. 1996). nuclei stained 12CA5, indicating <br /> zygous CTF7/ctf7&#226;&#338;&#172; deletion strain (YBS2). CTF7 deletion          protein levels varied considerably constructs car- <br /> confirmed Southern blot analysis. Tetrad analysis yielded         ried 2 &#194;&#181;m plasmid. YBS5 similarly transformed 2+:2&#226;&#710;&#8217; spore death recovery  spores. confirm        pAD5 vector (containing CTF7 insert) exhibited CTF7 encodes essential protein, diploid YBS2             nuclear staining, nuclear signal observed sec- <br /> (ctf7&#226;&#338;&#172;::HIS3/CTF7) transformed pRS316&#226;&#8364;&#8220;CTF7              ondary antibody  YOL1/34 (Serotec) fluorescein- sporulated.  Ura+ spores tested inviable          rhodamine-conjugated goat anti-rat antibodies (Cappel Boe- <br /> dium plates containing 5-FOA (Boeke et al. 1987).                    hringer Mannhein) used visualize microtubules. DNA <br />  <br />  <br /> masses visualized DAPI (Sigma). Western blot           <br /> analysis HA&#226;&#8364;&#8220;Pds1, cell densities normalized optical       <br /> density. Equal loads cell extracts confirmed Western         <br /> blot analysis tubulin levels.                                        <br />   Reagents used perform FISH analysis detailed proce-        <br /> dures described previously (Guacci et al. 1994, 1997).               <br />                                                                          <br /> </body></html>