<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Nat Biotechnol. 2014 November ; 32(11): 1141&#226;&#8364;&#8220;1145. doi:10.1038/nbt.3011. <br />  <br />  <br />  <br />                            Sequence-specific antimicrobials using efficiently delivered <br />                            RNA-guided nucleases <br />                            Robert J. Citorik1,2,7, Mark Mimee1,2,7, Timothy K. Lu1,2,3,4,5,6 <br />                            1MIT   Microbiology Program, Massachusetts Institute Technology, Cambridge, Massachusetts, <br />                            USA <br />                            2MITSynthetic Biology Center, Massachusetts Institute Technology, Cambridge, <br />                            Massachusetts, USA <br />                            3Departmentof Biological Engineering, Massachusetts Institute Technology, Cambridge, <br />                            Massachusetts, USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            4Department Electrical Engineering Science, Massachusetts Institute                            Technology, Cambridge, Massachusetts, USA <br />                            5Harvard     Biophysics Program, Harvard University, Boston, Massachusetts, USA <br />                            6Broad    Institute MIT Harvard, Cambridge, Massachusetts, USA <br />  <br />                            Abstract <br />                                 Current antibiotics tend broad spectrum, leading indiscriminate killing commensal <br />                                 bacteria accelerated evolution drug resistance.  use CRISPR-Cas technology                                 create antimicrobials spectrum activity chosen design. RNA-guided nucleases <br />                                 (RGNs) targeting specific DNA sequences delivered efficiently microbial populations using <br />                                 bacteriophage bacteria carrying plasmids transmissible conjugation. DNA targets                                 RGNs undesirable genes polymorphisms, including antibiotic resistance virulence <br />                                 determinants carbapenem-resistant Enterobacteriaceae enterohemorrhagic Escherichia coli. <br />                                 Delivery RGNs significantly improves survival Galleria mellonella infection model. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 RGNs enable modulation complex bacterial populations selective knockdown <br />                                 targeted strains based genetic signatures. RGNs constitute class highly discriminatory, <br />                                 customizable antimicrobials enact selective pressure DNA level reduce                                 prevalence undesired genes, minimize target effects enable programmable remodeling <br />                                 microbiota. <br />  <br />  <br />  <br />                            &#194;&#169; 2014 Nature America,  rights reserved. <br />                            Correspondence addressed T.K.L. (timlu@mit.edu). <br />                            7These authors contributed equally work. <br />                            Note: Supplementary Information Source Data files available online version paper. <br />                            AUTHOR CONTRIBUTIONS <br />                            R.J.C. M.M. designed performed experiments. R.J.C., M.M. T.K.L. conceived study, analyzed data, discussed <br />                            results wrote manuscript. <br />                            COMPETING FINANCIAL INTERESTS <br />                            authors declare competing financial interests: details available online version paper. <br />                            Reprints permissions information available online http://www.nature.com/reprints/index.html. <br />                             Citorik et al.                                                                                         Page 2 <br />  <br />  <br />                                             mounting concern emergence proliferation multidrug-resistant <br />                                             bacterial pathogens dwindling treatment options organisms1. Recently, <br />                                             carbapenem-resistant Enterobacteriaceae, group intestinal Gram-negative bacteria <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             known cause life-threatening opportunistic infections, highlighted Centers <br />                                             Disease Control Prevention urgent threats antibiotic- <br />                                             resistant bacteria1. Carbapenems traditionally reserved resort treatment <br />                                             Gram-negative infections, spread extended-spectrum &#206;&#178;-lactamases                                             necessitated increased usage carbapenems favored emergence carbapenem- <br />                                             resistant strains refractory current treatment options. responsible <br />                                             enzymes, including New-Delhi metallo-&#206;&#178;-lactamase 1 (NDM-1), confer pan-resistance <br />                                             &#206;&#178;-lactam antibiotics frequently harbored additional resistance <br />                                             determinants mobile plasmids facilitate rapid dissemination                                             Enterobacteriaceae2. diversity multidrug-resistant bacteria compounds difficulty <br />                                             developing treatments target pathogens commensal reservoirs avoid <br />                                             nonspecific broad-spectrum activity. <br />  <br />                                              introduce alternative antimicrobial approach imposes direct evolutionary <br />                                             pressure gene level using efficiently delivered, programmable RGNs. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             engineered clustered, regularly interspaced, short palindromic repeats (CRISPR)&#226;&#8364;&#8220; <br />                                             CRISPR-associated (Cas)  naturally employed bacteria defense strategy <br />                                             mobile elements3,4, effect cell death plasmid loss detection genetic <br />                                             signatures associated virulence antibiotic resistance. Type II CRISPR-Cas <br />                                             Streptococcus pyogenes effective, programmable tool genome editing <br />                                             gene expression wide variety organisms5. specificity CRISPR-Cas                                             dictated short, spacer sequences flanked direct repeats encoded CRISPR locus, <br />                                             transcribed processed CRISPR RNAs (crRNA)6. aid trans- <br />                                             activating small RNA (tracrRNA), crRNAs enable Cas9 endonuclease introduce <br />                                             double-stranded breaks target DNA sequences6,7. simple modifications                                             spacers CRISPR locus, RGN direct cleavage DNA sequence, <br />                                             design restriction requisite NGG motif immediately 3&#226;&#8364;&#178; target <br />                                             sequence7. packaging RGNs bacteriophage particles harnessing mobilizable <br />                                             plasmids, implemented conditional-lethality devices high specificity, modularity <br />                                             multiplexability undesired DNA sequences (Fig. 1a Supplementary Fig. 1). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             establish RGN functionality mediating sequence-specific cytotoxicity, designed <br />                                             RGNs induce double-stranded breaks blaSHV-18 blaNDM-1, encode extended- <br />                                             spectrum pan-resistance &#206;&#178;-lactam antibiotics, respectively (Supplementary Table <br />                                             1)8,9. Transformation E. coli plasmid-borne RGNs (pRGNs) containing                                             chromosomal copy target genes resulted nearly 1,000-fold decrease                                             transformation efficiency compared wild-type cells lacking target (Supplementary <br />                                             Fig. 2a). results corroborate mutual exclusivity functional crRNA                                             cognate target locus10,11. Sequence analysis 30 escape mutants (cells receive                                             maintain RGN plasmid despite presence target sequence) revealed tolerance <br />                                             exclusively defective construct frequently arose spacer deletion <br />                                             CRISPR locus (Supplementary Fig. 3). Furthermore, deletion tracrRNA  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                             Page 3 <br />  <br />  <br />                                             inactivation RuvC-like nuclease domain Cas9 (D10A)7 abrogated loss <br />                                             transformation efficiency cells harbored target sequence.  catalytically <br />                                             active endonuclease, tracrRNA crRNA necessary sufficient mediate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             sequence-specific cytotoxicity E. coli (Fig. 1a). <br />  <br />                                             Antibiotic resistance genes present large, multicopy plasmids capable                                             autonomous transfer microbial populations, resulting horizontal dissemination drug <br />                                             resistance2. RGN activity high-copy plasmids verified using GFP-expressing, <br />                                             ColE1-derived vector containing standard &#206;&#178;-lactamase selectable marker (pZE-blaz-gfp)12 <br />                                             blaNDM-1 (pZE-blaNDM-1-gfp). Vectors bearing ColE1 origin reported                                             present copy numbers 50&#226;&#8364;&#8220;70 cell12. Transformation cells containing pZE- <br />                                             blaNDM-1- gfp pRGNndm-1, plasmid-borne RGN targeting blaNDM-1, led  <br />                                             log10 reduction transformants retaining resistance &#206;&#178;-lactam antibiotic carbenicillin, <br />                                             transformation cells containing target-free pZE-blaz-gfp pRGNndm-1 did <br />                                             lead reduction resistant transformants (Supplementary Fig. 2b). activity                                             RGNs sufficient exclude high-copy antibiotic resistance plasmids                                             cells resensitize resistant population antibiotics. Similarly, transformation                                             cells possessing pZE-blaNDM-1-gfp pRGNndm-1 led ~1,000-fold decrease NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             GFP-expressing cells, measured flow cytometry, decrease                                             transformation cells possessing pZE-blaz-gfp pRGNndm-1 (Supplementary Fig. 4). <br />  <br />                                             usefulness RGNs antimicrobial therapy hinges high-efficiency delivery                                             genetic constructs bacterial cells. explored mechanisms horizontal gene <br />                                             transfer used bacteria acquire foreign genetic elements: plasmid conjugation <br />                                             viral transduction. constrained requirements cell-cell contact, <br />                                             conjugative plasmids wide host ranges recipient factors necessary                                             DNA uptake identified13. Efficient transfer RGNs achieved using                                             broad-host-range plasmid R1162 mobilized E. coli S17-1, contains conjugative <br />                                             machinery plasmid RP4 integrated chromosome. filter mating experiments, <br />                                             conjugative transfer RGNs resulted 40- 60-fold reduction target carbenicillin- <br />                                             resistant recipient cells (Supplementary Fig. 1b). selection transconjugants, <br />                                             transfer RGNs recipients yielded 2- 3-log10 reduction target cells compared <br />                                             controls, suggesting conjugation efficiency, opposed RGN activity, limited <br />                                             RGN efficacy context (Supplementary Fig. 1c). Future work necessary NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             optimize efficiency conjugation-based delivery vehicles antimicrobials <br />                                             based mobilizable RGNs. <br />  <br />                                             Bacteriophages natural predators bacteria highly adept injecting DNA                                             host cells. adapt phage RGN delivery, engineered phagemid vectors pairing <br />                                             RGN constructs targeting blaNDM-1 blaSHV-18 f1 origin facilitate packaging <br />                                             M13 particles. Phage-packaged RGNndm-1 (&#206;&#166;RGNndm-1) capable                                             comprehensively transducing population E. coli EMG2 (Supplementary Fig. 5). test <br />                                             &#206;&#166;RGNs, conjugated native plasmids containing blaNDM-1 (pNDM-1) blaSHV-18 <br />                                             (pSHV-18) clinical isolates EMG2. Treatment EMG2 pNDM-1 EMG2 <br />                                             pSHV-18 strains cognate &#206;&#166;RGNs resulted 2- 3-log10 reductions viable cells <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                              Page 4 <br />  <br />  <br />                                             absence selection (Fig. 1b). Furthermore, &#206;&#166;RGNs engendered toxicity <br />                                             wild-type EMG2 EMG2 containing noncognate plasmids (Fig. 1b). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             naturally occurring Type II CRISPR-Cas systems, CRISPR locus contain <br />                                             multiple spacers, processed independent crRNA molecule                                              tracrRNA, enables Cas9 cleave cognate DNA sequences6. explore utility                                             single &#206;&#166;RGN exhibiting activity genetic signature, engineered                                             construct containing spacers encoding different crRNAs targeting blaNDM-1 <br />                                             blaSHV-18 resistance genes (&#206;&#166;RGNndm-1/shv-18). &#206;&#166;RGNndm-1/shv-18 generated 2-                                             3-log10 reductions viable cells EMG2 pNDM-1 EMG2 pSHV-18, wild- <br />                                             type EMG2 (Fig. 1b).  RGNs multiplexed different genetic signatures, <br />                                             enabling simultaneous targeting variety virulence factors resistance genes                                             exist microbial populations. <br />  <br />                                             addition antibiotic-modifying enzymes, &#206;&#178;-lactamases, alterations host <br />                                             proteins constitute major antibiotic resistance mechanism bacteria14. Owing                                             specificity CRISPR-Cas  suspected RGNs discriminate                                             susceptible resistant strains differ single-<span id='am-1' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>nucleotide</span> mutation DNA gyrase <br />                                             (gyrA), confers resistance quinolone antibiotics (Supplementary Table 1)14.  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             &#206;&#166;RGNgyrAD87G specifically cytotoxic quinolone-resistant E. coli harboring                                             chromosomal gyrAD87G mutation isogenic strains wild-type <br />                                             gyrA gene (Fig. 1c). <br />  <br />                                             Killing curves revealed &#206;&#166;RGNs mediated rapid killing target cells, viable cell <br />                                             counts decayed exponentially (t1/2 ~ 13 min) maximal bactericidal effect                                             achieved 2&#226;&#8364;&#8220;4 h (Fig. 2a).  &#206;&#166;RGN antimicrobial activity increased                                             phagemid particle concentration (Fig. 2b Supplementary Fig. 6). characterize <br />                                             cellular response RGN-mediated targeting, assessed treatment cells                                             harbored GFP reporter SOS regulation. E. coli bacteria respond                                             chromosomal double-stranded breaks, including artificially generated                                             meganuclease SceI, inducing DNA repair activation SOS response15. <br />                                             observed 2.6- 4.0-fold increase fluorescence cells containing reporter <br />                                             plasmid plasmid-borne (blaNDM-1) chromosomal (gyrAD87G) target site, <br />                                             respectively, treated cognate versus noncognate &#206;&#166;RGNs (Supplementary Fig. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             7). results confirm RGNs induce DNA damage target cells                                             demonstrate coupled SOS-based reporters detect specific genes                                             sequences, single-nucleotide level. <br />  <br />                                             intrigued observe targeted cleavage blaNDM-1 &#206;&#166;RGNndm-1                                             context native plasmid lethal host cells, targeted cleavage                                             gene standard cloning vector (&#226;&#8364;&#732;pNDM-1&#226;&#8364;&#8482; versus &#226;&#8364;&#732;pZA-ndm1-gfp&#226;&#8364;&#8482;, respectively, <br />                                             Fig. 2c).  hypothesized &#206;&#166;RGN-induced plasmid loss itself does                                             elicit lethality, results cytotoxicity means harbored plasmid- <br />                                             borne functions. Toxin-antitoxin systems components natural plasmids ensure <br />                                             persistence bacterial populations inhibiting growth daughter cells                                             inherit plasmid. Toxin-antitoxin systems traditionally consist labile antitoxin  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                            Page 5 <br />  <br />  <br />                                             quenches activity stable toxin. Owing differential stability                                             components, cessation gene expression plasmid loss leads depletion                                             antitoxin pool faster toxin pool, resulting repression toxin activity  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             ultimately, stasis programmed cell death16. Analysis sequenced pSHV-18 plasmid <br />                                             revealed presence single toxin-antitoxin module, pemIK, commonly                                             isolates harboring extended-spectrum &#206;&#178;-lactamases17. complemented                                             PemI antitoxin expressed constitutively trans (pZA31-pemI), &#206;&#166;RGNshv-18 treatment                                             EMG2 pSHV-18 abrogated cytotoxicity instead resulted resensitization                                             multidrug-resistant strain carbenicillin (Fig. 2d). attributed effect PemI <br />                                             inactivation PemK toxin, enabling loss pSHV-18 plasmid                                             concomitant bacterial killing. Toxin-antitoxin systems dictate outcome                                             &#206;&#166;RGN activity episomal targets, presence leads cytotoxicity absence <br />                                             neutralization plasmid loss. <br />  <br />                                             demonstrate versatility RGNs specifically combating pathogens,                                             designed &#206;&#166;RGN target intimin, chromosomally encoded virulence factor                                             enterohemorrhagic E. coli O157:H7 (EHEC) necessary intestinal colonization                                             pathology. Encoded eae gene, intimin cell-surface adhesin mediates intimate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             attachment host epithelium, permitting subsequent disruption intestinal tight <br />                                             junctions effacement microvilli18. Treatment EHEC &#206;&#166;RGNeae resulted                                             20-fold reduction viable cell counts; cytotoxicity increased additional 100- <br />                                             fold kanamycin selection &#206;&#166;RGNeae transductants (Fig. 3a). increase                                             cytotoxicity selection cells receiving construct suggests efficacy                                             &#206;&#166;RGN treatment limited delivery strain. Furthermore, &#206;&#166;RGN treatment                                             assessed G. mellonella larvae, infection model yields virulence data                                             predictive higher-order mammals19. model used evaluate                                             efficacy antimicrobials phage therapy various Gram-negative, Gram-positive <br />                                             fungal pathogens19. Administration &#206;&#166;RGNeae EHEC-infected G. mellonella <br />                                             larvae significantly improved survival treatment target &#206;&#166;RGN control (log- <br />                                             rank test, P &lt; 0.001) (Fig. 3b).  &#206;&#166;RGNeae significantly effective                                             chloramphenicol treatment, EHEC strain resistant (log-rank test, P &lt; 0.05) <br />                                             (Supplementary Fig. 8 Supplementary Table 1). &#206;&#166;RGNeae treatment                                             inferior carbenicillin, bacteria susceptible (Supplementary Fig. 8 NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Supplementary Table 1), data support RGNs viable alternatives cases                                             bacteria highly resistant existing antibiotics. Improvements delivery efficiency                                             &#206;&#166;RGNeae expected improve treatment efficacy outcome. <br />  <br />                                             addition implementing targeted antimicrobial therapies, RGNs used modulate <br />                                             composition complex bacterial populations (Fig. 4). Current therapies use                                             prebiotic, probiotic drug modify human microbiota demonstrated potential                                             alleviating various disease states, therapies remain poorly characterized terms                                             target effects specific mechanisms act20. concert host <br />                                             range delivery vehicle, RGN activity selectively remove bacteria specific <br />                                             genomic content. reduce prevalence unwanted genes, including antibiotic <br />  <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                            Page 6 <br />  <br />  <br />                                             resistance virulence loci, metabolic pathways bacterial communities                                             affecting bystanders. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             proof principle experiment &#226;&#8364;&#732;bacterial knockdowns&#226;&#8364;&#8482; using RGNs, constructed <br />                                             synthetic consortium comprising phage-susceptible E. coli strains differential <br />                                             antibiotic resistance profiles. used &#206;&#178;-lactam&#226;&#8364;&#8220;resistant E. coli EMG2 pNDM-1, <br />                                             quinolone-resistant RFS289 (gyrAD87G) chloramphenicol- resistant CJ236. Application <br />                                             &#206;&#166;RGNndm-1 elicited &gt;400-fold greater killing EMG2 pNDM-1, compared control, <br />                                             leaving RFS289 CJ236 cell populations intact. Treatment &#206;&#166;RGNgyrAD87G <br />                                             resulted &gt;20,000-fold greater killing RFS289, compared control,                                             concomitant reduction EMG2 pNDM-1 CJ236 (Fig. 4). results demonstrate                                             RGNs selectively knockdown bacteria contain target DNA sequences,                                             allowing remaining nontarget bacteria dominate population. Adapting                                             approach tuning endogenous microbiota accomplished delivering RGNs                                             vivo means broad-host-range phages phage cocktails, conjugative plasmids. <br />                                             appropriately targeted bacterial knockdown approach used functional studies <br />                                             complex microbiota complement additive therapies, probiotics,                                             microbiome-associated diseases clearing specific niches removing defined genes NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             bacterial populations. <br />  <br />                                             light rising tide antibiotic resistance, engineered cellular viral <br />                                             therapeutics potential biological solutions infectious disease resurged.                                             repurposing parts developed nature, synthetic biologists designed artificial gene <br />                                             circuits antimalarial production21 engineered probiotics22 phage therapeutics                                             eradicate biofilms23 potentiate antibiotic activity24,25.  demonstrate                                             transmissible CRISPR-Cas systems act platform programmable antimicrobials <br />                                             harness site-specific cleavage induce cytotoxicity, activate toxin-antitoxin systems, <br />                                             resensitize bacterial populations antibiotics modulate bacterial consortia. work <br />                                             complements recent finding Vibrio cholerae phage, ICP1, encodes                                             CRISPR-Cas counteract host-encoded phage defense locus26, CRISPR- <br />                                             Cas constructs transformed electrocompetent cell populations incompatible                                             cells contain cognate target sequences11,27,28. contrast studies,                                             CRISPR-Cas technology applied situ removal undesired <br />                                             genes microbial populations vivo treat infection absence artificial <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             selection.  demonstrate RGNs used activate plasmid-borne <br />                                             toxin-antitoxin systems recently attractive antimicrobial strategy29.                                             addition validating antimicrobial activity, demonstrate potential applications                                             RGNs deletion plasmids cells detection DNA elements                                             single-nucleotide resolution using DNA-damage-responsive reporter. <br />  <br />                                             CRISPR-Cas systems widely conserved bacteria archaea, isolation, <br />                                             development optimization delivery vehicles required creation RGNs <br />                                             capable targeting additional strains, including multidrug-resistant pathogens                                             key members endogenous microbiota. generation phage systems                                             designed goal avoiding anticipated shortcomings, including host range <br />                                             limitations, phage resistance concerns immune reactions (Supplementary <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                          Page 7 <br />  <br />  <br />                                             Discussion). Delivery systems function higher organisms enable RGNs                                             modulate prevalence specific genes wild-type populations30. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Owing modularity simplicity CRISPR-Cas engineering, libraries                                             multiplexed &#206;&#166;RGNs rapidly constructed simultaneously target plethora                                             antibiotic resistance virulence determinants modulate composition complex <br />                                             microbial communities. addition facile, sequence-informed, rational design field <br />                                             dominated time- cost-intensive screening broad-spectrum, small- <br />                                             molecule antibiotics potential reinvigorate pipeline new antimicrobials. <br />  <br />                                ONLINE METHODS <br />                                Strains culture conditions <br />                                             Unless noted, bacterial cultures grown 37 &#194;&#176;C Luria-Bertani (LB) <br />                                             medium (BD Difco). Bacterial strains used study listed Supplementary Table 2. <br />                                             indicated, antibiotics added growth medium following final <br />                                             concentrations: 100 &#206;&#188;g/ml carbenicillin (Cb), 30 &#206;&#188;g/ml kanamycin (Km), 25 &#206;&#188;g/ml <br />                                             chloramphenicol (Cm), 100 &#206;&#188;g/ml streptomycin (Sm) 150 ng/ml ofloxacin (Ofx). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Strain construction <br />                                             E. coli EMG2 SmR generated plating overnight culture E. coli EMG2 LB <br />                                             +Sm. Spontaneous resistant mutants streaked LB+Sm isolated colony <br />                                             picked used recipient conjugation multidrug resistance plasmids. <br />                                             Overnight cultures EMG2 SmR (recipient), E. coli CDC1001728 (donor pNDM-1) <br />                                             K. pneumoniae K6 (donor pSHV-18) washed sterile PBS 100 &#206;&#188;l                                             donor recipient spotted LB agar plates incubated 37 &#194;&#176;C overnight. <br />                                             Transconjugants harvested scraping cells 1 ml sterile PBS plating                                             LB+Sm+Cb. <br />  <br />                                             chromosomal integrations blaNDM-1 blaSHV-18 &#206;&#178;-lactamase genes                                             generation EMG2 gyrAD87G performed &#206;&#187;-Red recombineering using pSIM9 <br />                                             system31. Templates integration nonessential lacZYA locus generated                                             amplifying blaNDM-1 blaSHV-18 genes lysates CDC1001728 K6 using <br />                                             primers rcD77/78 rcD73/74, respectively. Templates construction EMG2 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             gyrAD87G obtained amplifying gyrA RFS289 using primers mmD155/161. <br />  <br />                                Plasmid construction <br />                                             generate RGN plasmids (Supplementary Fig. 2), created intermediate vector <br />                                             pZA-RGN&#195;&#732;, lacks CRISPR locus. tracrRNA PL(tetO-1) promoter                                             synthesized (Genewiz) amplified using primers mmD98/99, cas9 amplified                                             pMJ806 (ref. 7) using primers mmD74/75, vector backbone amplified                                             pZA11G using primers mmD82/83. PCR product purified ligated Gibson <br />                                             assembly32. create final backbone vector RGN plasmids, pBBR1 origin, <br />                                             chloramphenicol resistance marker, tL17 terminator, CRISPR locus cloning site                                             amplified intermediate vector pBBR1-MCS1-tL17 using mmD151/154, digested <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                            Page 8 <br />  <br />  <br />                                             NheI SacI-HF, ligated pZA-RGN&#195;&#732; digested SacI-HF AvrII                                             create pZB-RGN&#195;&#732;. Digestion vector PstI-HF XbaI allowed insertion <br />                                             assembled CRISPR loci. tracrRNA pRGNndm-1 plasmid created NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             amplification pRGNndm-1 mmD162/163, ClaI digestion, self-ligation.                                             Cas9D10A mutant plasmid constructed site-directed mutagenesis                                             pRGNndm-1 primers mmD108/109 KAPA HiFi PCR kit (KAPA Biosystems). <br />  <br />                                             CRISPR loci constructed isothermal annealing ligation short, <br />                                             single-stranded oligonucleotides (Integrated DNA Technologies). spacer repeat <br />                                             piece built corresponding oligo duplex connected adjacent pieces 6-bp <br />                                             overhangs. addition, terminal repeats designed contain 17-bp extension <br />                                             comprised BsaI restriction site generate overhang allowed insertion                                             pUC57-Km-crRNA&#195;&#732; backbone vector synthesized Genewiz. oligos used build <br />                                             RGN listed Supplementary Tables 3 4. <br />  <br />                                             assemble CRISPR loci, 500 pmol sense antisense oligos given duplex <br />                                             annealed boiling 10 min 99 &#194;&#176;C cooled room temperature. 300 pmol                                             annealed duplex combined 15 U T4 polynucleotide kinase (Affymetrix), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             400 U T4 DNA ligase (NEB), T4 ligase buffer (NEB) ddH2O volume 20 &#206;&#188;l. <br />                                             Following incubation 25 &#194;&#176;C 1 h, reaction products purified using Qiagen <br />                                             QIAquick PCR Purification Kit. Purified products digested 3 h BsaI-HF                                             purified using QIAquick. prepare crRNA backbone vector, pUC57-Km-crRNA&#195;&#732; <br />                                             amplified using primers mmD104/105, subsequently digested BbsI generate <br />                                             compatible overhangs, ligated assembled CRISPR loci. Positive clones                                             CRISPR loci digested entry vector using PstI-HF XbaI ligated                                             pZB-RGN&#195;&#732; digested enzymes create final RGN plasmids. <br />  <br />                                             Phagemid vector pZEf-gfp created previously adding f1 origin amplified                                             yeast shuttle pRS series33 pZE22-gfp12. RGN constructs consisting genes <br />                                             encoding tracrRNA, Cas9 sequence-targeting crRNA amplified single <br />                                             product respective pRGN vectors using KAPA HiFi polymerase (Kapa <br />                                             Biosystems) primers rcD169/183 digested AvrII XmaI (New England <br />                                             Biolabs). inserts ligated backbone derived amplifying                                             kanamycin resistance cassette, ColE1 replication origin f1 origin required NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             packaging M13 particles pZEf-gfp primers rcD184/185 digesting                                             enzymes. E. coli DH5&#206;&#177;Pro transformed ligated plasmids sequence <br />                                             verification plasmid purification. <br />  <br />                                             pZE-blaNDM-1-gfp pZA-blaNDM-1-gfp vectors constructed swapping                                             antibiotic resistance cassette Lutz-Bujard vectors pZE12G pZA12G12.                                             blaNDM-1 gene amplified lysate CDC1001728 using primers mmD8/9,                                             PCR product digested SacI-HF XhoI. digested product ligated                                             Lutz-Bujard vectors digested enzymes. <br />  <br />                                             PemI antitoxin complementation plasmid pZA31-pemI created amplifying <br />                                             pemI coding sequence pSHV-18 mmD253/254. PCR product  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                            Page 9 <br />  <br />  <br />                                             digested BamHI KpnI ligated large fragment pZA31G digest                                             enzymes. SOS-responsive pZA3LG reporter plasmid derived                                             pZE1LG34 swapping origin replication antibiotic resistance marker NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             pZA31G using AatII AvrII restriction enzymes. <br />  <br />                                             Mobilizable RGNs created amplifying R1162 replication origin oriT <br />                                             using mmD266/267. chloramphenicol selection marker RGN locus amplified <br />                                             pRGNndm-1 pRGNshv-18 mmD247/248. PCR products digested                                             SpeI XmaI, ligated used transform E. coli S17-1 &#206;&#187;pir create donor cells <br />                                             used matings. <br />  <br />                                Minimum inhibitory concentration (MIC) determination <br />                                             MICs determined broth microdilution using LB broth according CLSI <br />                                             guidelines35. <br />  <br />                                Transformation assays <br />                                             Overnight cultures diluted 1:100 fresh LB grown optical density (OD600) <br />                                             ~0.3&#226;&#8364;&#8220;0.5. Following 15 min incubation ice, cultured cells centrifuged NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             3,200g, pellets resuspended tenth volume TSS buffer (LB, 10% <br />                                             polyethylene glycol, 5% dimethyl sulfoxide, 50 mM Mg2+ pH 6.5)36. 100 &#206;&#188;l aliquot                                             cells incubated 10 ng RGN plasmid DNA. Plasmids purified                                             DH5&#206;&#177;Pro cloning host using Qiagen QIAprep Spin Miniprep Kit concentration <br />                                             determined using Quant PicoGreen dsDNA Assay Kit (Invitrogen). Following 30 <br />                                             min incubation ice, cells heat shocked 42 &#194;&#176;C 30 s, returned ice 2 min <br />                                             recovered 1.5 h 37 &#194;&#176;C 1 ml SOC broth (HiMedia). chromosomal <br />                                             target assay, serial dilutions cells plated LB+Cm select transformants. <br />                                             Plates incubated overnight 37 &#194;&#176;C, number colony-forming units (CFU) <br />                                             enumerated following day. Transformation efficiency used assess                                             given RGN plasmid toxic cells calculated CFU/ml &#206;&#188;g DNA <br />                                             transformed (Supplementary Fig. 2a). <br />  <br />                                             episomal target assay (Supplementary Fig. 2b), following recovery, cultures                                             washed fresh LB, diluted 1:100 LB supplemented chloramphenicol select NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             transformants incubated 16 h 37 &#194;&#176;C. Samples washed sterile PBS, serially <br />                                             diluted plated LB+Cm LB+Cm+Cb analyzed flow cytometry <br />                                             (Supplementary Fig. 4). Colonies enumerated following day plasmid loss                                             inferred calculating ratio CbR+CmR CFUs CmR CFUs. <br />  <br />                                             Overnight cultures RGN transformants diluted 1:100 sterile PBS, aliquoted <br />                                             duplicates 96 plate immediately assayed using BD LSRFortessa cell <br />                                             analyzer. Cells consistently gated scatter forward scatter                                             independent biological replicates. Fluorescence measurements performed using 488- <br />                                             nm argon excitation laser. GFP gate laser voltages initially determined using <br />                                             untreated pZE-blaZ-gfp EMG2 cells positive negative controls, respectively,  <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                         Page 10 <br />  <br />  <br />                                             implemented biological replicates. BD FACSDIVA software used data <br />                                             acquisition analysis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Sequence analysis <br />                                             Escape mutants transformation assays isolated passaging surviving colonies <br />                                             LB+Cm+Cb. DNA isolation escape sequencing analysis                                             extracting plasmid DNA isolated escape mutants using Qiagen QIAprep Spin <br />                                             Miniprep Kit amplifying integrated target locus using primers mmD9/234                                             mmD3/4 blaNDM-1 blaSHV-18, respectively. Sequencing performed Genewiz <br />                                             using primers mmD112&#226;&#8364;&#8220;115/153 rcD11 analysis RGN plasmids                                             mmD3 mmD234 examination integrated resistance genes. <br />  <br />                                Phagemid purification <br />                                             Phagemids encoding RGNs purified using Qiagen QIAprep Spin Miniprep Kit <br />                                             (Qiagen) used transform E. coli DH5&#206;&#177;Pro m13cp helper plasmid                                             generation phagemid-loaded M13 particles37. Strains inoculated grown <br />                                             overnight 250 ml LB+Cm+Km maintain M13cp phagemid, respectively. Cells <br />                                             pelleted supernatant fluid containing phagemid particles passed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             0.2-&#206;&#188;m filter. purifications &#206;&#166;RGNgyrAD87G purification                                             dose-response curve, M13 phagemid particles precipitated addition 5% <br />                                             polyethylene glycol (PEG-6000) 0.5M NaCl incubation overnight 4 &#194;&#176;C (ref. 38) <br />                                             pelleted 12,000g 1 h. Purified phagemid pellets resuspended gently                                             1/100th volume SM buffer (50 mM Tris-HCl [pH 7.5]), 100 mM NaCl, 10 mM MgSO4) <br />                                             stored 4 &#194;&#176;C. &#206;&#166;RGNgyrAD87G purification dose-response curve (Fig. <br />                                             2b), M13 phagemid particles precipitated39 addition concentrated HCl <br />                                             pH 4.2 subsequently pelleted 13,000g 15 min. phagemid pellet                                             resuspended 1/100th volume water concentrated NaOH added pH 7.0                                             solubilize phagemid particles. Tris-HCl [pH 7.5], NaCl MgSO4 added                                             reconstitute composition SM buffer. <br />  <br />                                             Titers measured incubating sample dilutions E. coli EMG2 30 min                                             enumerating transductants plating LB LB+Km. Titers defined TFU100/ml, <br />                                             concentration phagemid ~100% recipient population NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             equivalent cell concentration transduced. <br />  <br />                                Phagemid kill assays <br />                                             Cultures inoculated grown overnight LB appropriate antibiotics 37 &#194;&#176;C <br />                                             shaking. following day, overnights subcultured 1:100 3 ml LB                                              antibiotics) grown 37 &#194;&#176;C shaking OD600 reached ~0.8. Cultures                                             diluted LB ~108 CFU/ml pNDM-1 pSHV-18 assays (Fig. 1b, 2a) ~106 <br />                                             CFU/ml gyrAD87G (Fig. 1c) EHEC assays (Fig. 3a) 245 &#206;&#188;l suspension                                             added 5 &#206;&#188;l purified phagemid stock 96 plate incubated static 37 &#194;&#176;C. <br />                                             number viable cells samples interval time-course 2 h                                             endpoint assays determined serial dilution spot plating LB, LB+Cb, LB <br />                                             +Km analyze cytotoxicity, plasmid loss phagemid delivery, respectively. Initial <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                          Page 11 <br />  <br />  <br />                                             suspensions diluted plated LB quantify initial bacterial inocula. <br />                                             Colonies enumerated 8&#226;&#8364;&#8220;9 h incubation 37 &#194;&#176;C calculate cell viability <br />                                             (CFU/ml) averaged independent experiments. Nonlinear curve fitting NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             time-course exponential decay curve performed using GraphPad Prism. <br />  <br />                                G. mellonella model <br />                                             Larvae model organism G. mellonella19 purchased Vanderhorst Wholesale, <br />                                              (St. Marys, OH, USA) received final larval instar survival assays. Larvae <br />                                             removed food source, allowed acclimate 24 h room temperature <br />                                             dark, used 4 d receipt. injections, KDS100 (KD Scientific)                                             Pump 11 Elite (Harvard Apparatus) automated syringe pump set dispense 10 &#206;&#188;l <br />                                             volume flow rate ~1 &#206;&#188;l/s 1-ml syringe (BD) 26 G needle (BD).                                             prepare bacteria injection, overnight culture E. coli O157:H7 43888 F&#226;&#8364;&#178;                                             subcultured Dulbecco&#226;&#8364;&#8482;s Modified Eagle Medium (Gibco) 4 h 37 &#194;&#176;C shaking <br />                                             OD600 ~0.6. Cultures washed twice PBS diluted concentration ~4 &#195;&#8212; <br />                                             105 CFU/ml. accordance studies40, 20 larvae treatment group                                             randomly selected based size (150&#226;&#8364;&#8220;250 mg) excluded based poor health                                             evidenced limited activity, dark coloration reduced turgor experiments. Larvae <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             delivered injections blinding PBS bacteria final left <br />                                             proleg. Approximately hour injection, SM buffer, antibiotic &#206;&#166;RGN <br />                                             treatment administered final right proleg (Fig. 3c, Supplementary Fig. 8). <br />                                             Larvae incubated 37 &#194;&#176;C survival monitored 12 h intervals 72 h,                                             death indicated lack movement unresponsiveness touch19. Kaplan-Meier <br />                                             survival curves generated analyzed log-rank test using GraphPad Prism. <br />  <br />                                LexA reporter assay <br />                                             Overnight cultures EMG2 WT, EMG2 pNDM-1 EMG2 gyrAD87G containing                                             SOS-responsive reporter plasmid pZA3LG diluted 1:50 LB incubated                                             SM buffer, &#206;&#166;RGNndm-1 &#206;&#166;RGNgyrAD87G MOI ~5 2 h 37 &#194;&#176;C <br />                                             (Supplementary Fig. 7). Cultures diluted 1:5 250 &#206;&#188;l sterile PBS analyzed <br />                                             using BD LSR Fortessa cell analyzer,  BD FACSDIVA software used                                             data acquisition analysis performed using FlowJo software. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Bacterial conjugation <br />                                             Donor recipient strains grown overnight LB appropriate antibiotics diluted <br />                                             1:100 fresh media grown OD600 ~1. Cells pelleted resuspended                                             sterile PBS, mating pairs mixed donor recipient ratio 340 &#194;&#177; 66:1. Mating <br />                                             mixtures pelleted, resus-pended 20 &#206;&#188;l PBS spotted nitrocellulose filters <br />                                             placed LB agar plates. Initial bacterial suspensions serially diluted plated LB <br />                                             agar plates quantify initial inocula. Matings proceeded 37 &#194;&#176;C 3 h single <br />                                             mixing step. 90 min, mating mixtures collected vigorously vortexing filters <br />                                             1 ml sterile PBS. Cells pelleted, resuspended 20 &#206;&#188;l PBS seeded filters <br />                                             incubated remaining 90 min. end 3 h mating, cells                                             recovered vigorously vortexing filters 1 ml sterile PBS. Mating mixtures <br />  <br />  <br />  <br />                                                 Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                                            Page 12 <br />  <br />  <br />                                             serially diluted PBS plated LB+Cb select total number Cb- <br />                                             resistant recipient cells LB+Cb+Cm select transconjugants. Colonies                                             enumerated following overnight incubation 37 &#194;&#176;C determine viable cell counts NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             averaged independent biological replicates (Supplementary Fig. 1). <br />  <br />                                Synthetic consortia remodeling <br />                                             E. coli CJ236, EMG2 pNDM-1 RFS289 strains grown overnight LB appropriate <br />                                             antibiotics diluted 1:100 fresh LB  antibiotics) grown OD600 ~0.8. <br />                                             Cultures seeded fresh LB initial mixture contained ~1 &#195;&#8212; 106 CFU/ml <br />                                             strain 245 &#206;&#188;l suspension added 5 &#206;&#188;l SM buffer purified <br />                                             &#206;&#166;RGNndm-1 &#206;&#166;RGNgyrAD87G triplicate 96 plate spotted LB +Cm, <br />                                             +Sm +Ofx quantify initial concentration CJ236, EMG2 pNDM-1 RFS289, <br />                                             respectively. Samples incubated, plated enumerated phagemid kill <br />                                             assays. composition synthetic ecosystem treatment condition                                             determined counting viable colonies plates selective strain data <br />                                             calculated viable cell concentration (CFU/ml) averaged biological <br />                                             replicates (Fig. 4). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Data analysis statistics <br />                                             data analyzed using GraphPad Prism version 6.0 (GraphPad Software, San Diego, <br />                                             CA, USA, http://www.graphpad.com/). <br />  <br />  <br />                                Supplementary Material <br />                                             Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                Acknowledgments <br />                                             like thank R. Meyer (Institute Cell Molecular Biology, University Texas Austin) R1162, <br />                                             D.L. Court (Center Cancer Research, National Cancer Institute Frederick) pSIM9, R.M. Bradbury <br />                                             (Los Alamos National Laboratory) M13cp. authors thank J. Rubens assistance flow cytometry <br />                                             experiments H. Gancz D. Zurawski assistance Galleria infection model. T.K.L. acknowledges <br />                                             support National Institutes Health (NIH) New Innovator Award (1DP2OD008435), NIH <br />                                             National Centers Systems Biology grant (1P50GM098792), Defense Threat Reduction Agency <br />                                             (HDTRA1-14-1-0007), Army Research Laboratory Army Research Office Institute <br />                                             Soldier Nanotechnologies (W911NF13D0001) Henry L. Grace Doherty Professorship Ocean <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Utilization. R.J.C. supported funding NIH/National Institute General Medical Sciences <br />                                             Interdepartmental Biotechnology Training Program (5T32 GM008334), M.M. Howard Hughes Medical <br />                                             Institute International Student Research fellow recipient Fonds recherche Sant&#195;&#169; Qu&#195;&#169;bec Master&#226;&#8364;&#8482;s <br />                                             Training Award. <br />  <br />  <br />                                References <br />                                             1. Centers Disease Control Prevention. Antibiotic Resistance Threats United States. <br />                                                2013 <br />                                             2. Nordmann P, Dortet L, Poirel L. Carbapenem resistance Enterobacteriaceae: storm! <br />                                                Trends Mol Med. 2012; 18:263&#226;&#8364;&#8220;272. [PubMed: 22480775] <br />                                             3. Barrangou R, et al. CRISPR provides acquired resistance viruses prokaryotes. Science. <br />                                                2007; 315:1709&#226;&#8364;&#8220;1712. [PubMed: 17379808] <br />                                             4. Garneau JE, et al. CRISPR/Cas bacterial immune cleaves bacteriophage plasmid <br />                                                DNA. Nature. 2010; 468:67&#226;&#8364;&#8220;71. [PubMed: 21048762] <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                                  Page 13 <br />  <br />  <br />                                             5. Mali P, Esvelt KM, Church GM. Cas9 versatile tool engineering biology. Nat Methods. <br />                                                2013; 10:957&#226;&#8364;&#8220;963. [PubMed: 24076990] <br />                                             6. Deltcheva E, et al. CRISPR RNA maturation trans-encoded small RNA host factor RNase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                III. Nature. 2011; 471:602&#226;&#8364;&#8220;607. [PubMed: 21455174] <br />                                             7. Jinek M, et al. programmable dual-RNA-guided DNA endonuclease adaptive bacterial <br />                                                immunity. Science. 2012; 337:816&#226;&#8364;&#8220;821. [PubMed: 22745249] <br />                                             8. Rasheed JK, et al. Characterization extended-spectrum beta-lactamase reference strain, <br />                                                Klebsiella pneumoniae K6 (ATCC 700603), produces novel enzyme SHV-18. <br />                                                Antimicrob Agents Chemother. 2000; 44:2382&#226;&#8364;&#8220;2388. [PubMed: 10952583] <br />                                             9. Rasheed JK, et al. New Delhi metallo-&#206;&#178;-lactamase-producing Enterobacteriaceae, United States. <br />                                                Emerg Infect Dis. 2013; 19:870&#226;&#8364;&#8220;878. [PubMed: 23731823] <br />                                             10. Bikard D, Hatoum-Aslan  Mucida D, Marraffini LA. CRISPR interference prevent natural <br />                                                  transformation virulence acquisition vivo bacterial infection. Cell Host Microbe. <br />                                                  2012; 12:177&#226;&#8364;&#8220;186. [PubMed: 22901538] <br />                                             11. Gomaa AA, et al. Programmable removal bacterial strains use genome-targeting CRISPR- <br />                                                  Cas systems. MBio. 2014; 5:e00928&#226;&#8364;&#8220;13. [PubMed: 24473129] <br />                                             12. Lutz R, Bujard H. Independent tight regulation transcriptional units Escherichia coli                                                  LacR/O, TetR/O AraC/I1&#226;&#8364;&#8220;I2 regulatory elements. Nucleic Acids Res. 1997; 25:1203&#226;&#8364;&#8220; <br />                                                  1210. [PubMed: 9092630] <br />                                             13. P&#195;&#169;rez-Mendoza D, la Cruz F. Escherichia coli genes affecting recipient ability plasmid <br />                                                  conjugation:  BMC Genomics. 2009; 10:71. [PubMed: 19203375] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             14. Jacoby GA. Mechanisms resistance quinolones. Clin Infect Dis. 2005; 41(suppl 2):S120&#226;&#8364;&#8220; <br />                                                  S126. [PubMed: 15942878] <br />                                             15. Pennington JM, Rosenberg SM. Spontaneous DNA breakage single living Escherichia coli cells. <br />                                                  Nat Genet. 2007; 39:797&#226;&#8364;&#8220;802. [PubMed: 17529976] <br />                                             16. Hayes F. Toxins-antitoxins: plasmid maintenance, programmed cell death, cell cycle arrest. <br />                                                  Science. 2003; 301:1496&#226;&#8364;&#8220;1499. [PubMed: 12970556] <br />                                             17. Mnif B, et al. Molecular characterization addiction systems plasmids encoding extended- <br />                                                  spectrum beta-lactamases Escherichia coli. J Antimicrob Chemother. 2010; 65:1599&#226;&#8364;&#8220;1603. <br />                                                  [PubMed: 20507859] <br />                                             18. Kaper JB, Nataro JP, Mobley HL. Pathogenic Escherichia coli. Nat Rev Microbiol. 2004; 2:123&#226;&#8364;&#8220; <br />                                                  140. [PubMed: 15040260] <br />                                             19. Desbois AP, Coote PJ. Utility greater wax moth larva (Galleria mellonella) evaluating                                                  toxicity efficacy new antimicrobial agents. Adv Appl Microbiol. 2012; 78:25&#226;&#8364;&#8220;53. [PubMed: <br />                                                  22305092] <br />                                             20. Sonnenburg JL, Fischbach MA. Community health care: therapeutic opportunities human <br />                                                  microbiome. Sci Transl Med. 2011; 3:78ps12. <br />                                             21. Paddon CJ, et al. High-level semi-synthetic production potent antimalarial artemisinin. <br />                                                  Nature. 2013; 496:528&#226;&#8364;&#8220;532. [PubMed: 23575629] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             22. Duan F, March JC. Engineered bacterial communication prevents Vibrio cholerae virulence                                                  infant mouse model. Proc Natl Acad Sci USA. 2010; 107:11260&#226;&#8364;&#8220;11264. [PubMed: 20534565] <br />                                             23. Lu TK, Collins JJ. Dispersing biofilms engineered enzymatic bacteriophage. Proc Natl Acad <br />                                                  Sci USA. 2007; 104:11197&#226;&#8364;&#8220;11202. [PubMed: 17592147] <br />                                             24. Lu TK, Collins JJ. Engineered bacteriophage targeting gene networks adjuvants antibiotic <br />                                                  therapy. Proc Natl Acad Sci USA. 2009; 106:4629&#226;&#8364;&#8220;4634. [PubMed: 19255432] <br />                                             25. Edgar R, Friedman N, Molshanski-Mor S, Qimron U. Reversing bacterial resistance antibiotics <br />                                                  phage-mediated delivery dominant sensitive genes. Appl Environ Microbiol. 2012; 78:744&#226;&#8364;&#8220; <br />                                                  751. [PubMed: 22113912] <br />                                             26. Seed KD, Lazinski DW, Calderwood SB, Camilli  bacteriophage encodes                                                  CRISPR/Cas adaptive response evade host innate immunity. Nature. 2013; 494:489&#226;&#8364;&#8220;491. <br />                                                  [PubMed: 23446421] <br />                                             27. Jiang W, Bikard D, Cox D, Zhang F, Marraffini LA. RNA-guided editing bacterial genomes <br />                                                  using CRISPR-Cas systems. Nat Biotechnol. 2013; 31:233&#226;&#8364;&#8220;239. [PubMed: 23360965] <br />  <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                                    Page 14 <br />  <br />  <br />                                             28. Vercoe RB, et al. Cytotoxic chromosomal targeting CRISPR/Cas systems reshape bacterial <br />                                                 genomes expel remodel pathogenicity islands. PLoS Genet. 2013; 9:e1003454. [PubMed: <br />                                                 23637624] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             29. Williams JJ, Hergenrother PJ. Artificial activation toxin-antitoxin systems antibacterial <br />                                                 strategy. Trends Microbiol. 2012; 20:291&#226;&#8364;&#8220;298. [PubMed: 22445361] <br />                                             30. Esvelt KM, Smidler AL, Catteruccia F, Church GM. Concerning RNA-guided gene drives                                                 alteration wild populations. Elife. 2014:e03401. [PubMed: 25035423] <br />                                             31. Datta S, Costantino N, Court DL. set recombineering plasmids Gram-negative bacteria. <br />                                                 Gene. 2006; 379:109&#226;&#8364;&#8220;115. [PubMed: 16750601] <br />                                             32. Gibson DG, et al. Enzymatic assembly DNA molecules kilobases. Nat <br />                                                 Methods. 2009; 6:343&#226;&#8364;&#8220;345. [PubMed: 19363495] <br />                                             33. Sikorski RS, Hieter P. shuttle vectors yeast host strains designed efficient <br />                                                 manipulation DNA Saccharomyces cerevisiae. Genetics. 1989; 122:19&#226;&#8364;&#8220;27. [PubMed: <br />                                                 2659436] <br />                                             34. Dwyer DJ, Kohanski MA, Hayete B, Collins JJ. Gyrase inhibitors induce oxidative damage <br />                                                 cellular death pathway Escherichia coli. Mol Syst Biol. 2007; 3:91. [PubMed: 17353933] <br />                                             35. Clinical Laboratory Standards Institute. Methods Dilution Antimicrobial Susceptibility <br />                                                 Tests Bacteria Grow Aerobically; Approved Standard. 7. Clinical Laboratory <br />                                                 Standards Institute; Wayne, Pennsylvania, USA: 2006. <br />                                             36. Chung CT, Niemela SL, Miller RH. step preparation competent Escherichia coli: <br />                                                 transformation storage bacterial cells solution. Proc Natl Acad Sci USA. 1989; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 86:2172&#226;&#8364;&#8220;2175. [PubMed: 2648393] <br />                                             37. Chasteen L, Ayriss J, Pavlik P, Bradbury ARM. Eliminating helper phage phage display. <br />                                                 Nucleic Acids Res. 2006; 34:e145. [PubMed: 17088290] <br />                                             38. Westwater C, et al. Use genetically engineered phage deliver antimicrobial agents bacteria: <br />                                                 alternative therapy treatment bacterial infections. Antimicrob Agents Chemother. 2003; <br />                                                 47:1301&#226;&#8364;&#8220;1307. [PubMed: 12654662] <br />                                             39. Dong D, Sutaria S, Hwangbo JY, Chen P. simple rapid method isolate purer M13 phage <br />                                                 isoelectric precipitation. Appl Microbiol Biotechnol. 2013; 97:8023&#226;&#8364;&#8220;8029. [PubMed: <br />                                                 23807666] <br />                                             40. Ramarao N, Nielsen-Leroux C, Lereclus D. insect Galleria mellonella powerful infection <br />                                                 model investigate bacterial pathogenesis. J Vis Exp. 2012; 439210.3791/4392 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                         Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 1. <br />                                             RGN constructs delivered bacteriophage particles (&#206;&#166;RGN) exhibit efficient specific <br />                                             antimicrobial effects strains harboring plasmid chromosomal target sequences.   <br />                                             Bacteriophage-delivered RGN constructs differentially affect host cell physiology                                             sequence-dependent manner. target sequence    absent, RGN exerts effect; <br />                                             (ii) chromosomal, RGN activity cytotoxic; (iii) episomal, RGN leads (iiia) <br />                                             cell death (iiib) plasmid loss, depending presence absence toxin-antitoxin <br />                                             (TA) systems, respectively. (b) Treatment EMG2 wild-type (WT) EMG2 containing <br />                                             native resistance plasmids, pNDM-1 (encoding blaNDM-1) pSHV-18 (encoding <br />                                             blaSHV-18), SM buffer, &#206;&#166;RGNndm-1, &#206;&#166;RGNshv-18, multiplexed &#206;&#166;RGNndm-1/ <br />                                             shv-18 multiplicity infection (MOI) ~20 showed sequence-dependent cytotoxicity                                             evidenced strain-specific reduction viable cell counts (n = 3). CFU, colony-forming <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             units. (c) E. coli EMG2 WT EMG2 gyrAD87G populations treated SM buffer, <br />                                             &#206;&#166;RGNndm-1 &#206;&#166;RGNgyrAD87G MOI ~20, viable cells determined plating <br />                                             Luria-Bertani agar (n = 3). <br />  <br />  <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                          Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 2. <br />                                             Characterization &#206;&#166;RGN-mediated killing antibiotic-resistant bacteria.   Time-course <br />                                             treatment EMG2 WT EMG2 pNDM-1 SM buffer, &#206;&#166;RGNndm-1 &#206;&#166;RGNshv-18 <br />                                             multiplicity infection (MOI) ~20. Data represent fold change viable colonies                                             indicated time points relative time 0 h. (b) Dose-response curve EMG2 WT EMG2 <br />                                             gyrAD87G treated various concentrations &#206;&#166;RGNgyrAD87G 2 h. Data represent <br />                                             fold change viable colonies relative samples treated SM buffer. Error bars   b), <br />                                             s.e.m. independent biological replicates (n = 3). (c) EMG2 E. coli containing                                             natural pNDM-1 plasmid blaNDM-1 gene synthetic expression vector (pZA-ndm1- <br />                                             gfp) treated &#206;&#166;RGNndm-1 &#206;&#166;RGNshv-18 MOI ~ 20 plated                                             nonselective LB LB + carbenicillin (Cb) select blaNDM-1-containing cells. <br />                                             &#206;&#166;RGNndm-1 treatment cells harboring pNDM-1 resulted reduction viability                                             absence selection, &#206;&#166;RGNndm-1 treatment cells pZA-ndm1-gfp <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             demonstrated similar cytotoxicity selective pressure maintenance pZA- <br />                                             ndm1-gfp plasmid. (d) EMG2 pSHV-18 complemented cognate antitoxin (pZA31- <br />                                             pemI) PemK toxin control vector (pZA31-gfp) treated SM buffer, <br />                                             &#206;&#166;RGNndm-1 &#206;&#166;RGNshv-18. Cultures plated LB LB + Cb colonies                                             enumerated assess cytotoxicity plasmid loss. <br />  <br />  <br />  <br />  <br />                                                  Nat Biotechnol. Author manuscript; available PMC 2015 01. <br />                             Citorik et al.                                                                                       Page 17 <br /> NIH-PA Author  <br /> </body></html>