<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br /> Dihydroxyacid dehydratase important gametophyte <br /> development disruption causes increased susceptibility salinity stress Arabidopsis <br />  <br />  <br />  <br />  <br /> Abstract <br /> Dihydroxyacid dehydratase (DHAD) catalyses key step branched-chain <span id='am-9' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-12' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> (BCAA) biosynthetic pathway exists numerous organisms, including bacteria, fungi, plants, humans. Arabidopsis thaliana, DHAD encoded single gene (AT3G23940), biological function controlling plant development remains uncharacter- <br /> ized. study, showed DHAD highly expressed vegetative reproductive tissues. essential <br /> gene, complete disruption caused partial sterility male female gametophyte phases. addition, reduced <br /> expression DHAD knockdown mutants resulted reduction accumulation BCAAs roots  consequence, led shorter root phenotype, restored exogenous supplement free BCAAs. <br /> Interestingly, knockdown mutants hypersensitive salt stress, heavy metal stress, implying BCAAs act osmolytes salt tolerance. second amino acid shown confer function addition documented proline. results provide evidence BCAA biosynthesis plays important roles gametophyte root development, BCAA homeostasis contributes adaptation Arabidopsis salinity stress. <br />  <br /> Key words:&#226;&#8364;&#8218; Arabidopsis, branched-chain amino acid (BCAA), dihydroxyacid dehydratase (DHAD), gametogenesis, root <br /> development, salt stress. <br />  <br />  <br />  <br />  <br /> Introduction <br /> Isoleucine (Ile), valine (Val), leucine (Leu) grouped                                diet sources (Jander Joshi, 2009, 2010). Second, branched-chain amino acids (BCAAs), contain                                   enzymes involved BCAA biosynthesis targets short branched hydrocarbon residues. Research                                herbicide applications (Tan et&#194;&#160;al., 2006). <br /> biosynthetic pathway BCAAs plants motivated                                       enzymes participating BCAA biosynthesis main reasons.  like animals, human unable                                   widely conserved bacteria, fungi, plants. Threonine <br /> synthesize BCAAs novo, obtain                                    deaminase (TD) catalyses committed step <br />  <br /> Abbreviations: AHAS, acetohydroxyacid synthase; BCAA, branched-chain amino acid; BCAT, branched-chain aminotransferase; DHAD, dihydroxyacid dehy- <br /> dratase; EDTA, ethylene diamine tetraacetic acid; HPLC, high performance liquid chromatography; KARI, ketolacid reductoisomerase; Ile, isoleucine; IPMDH, <br /> isopropylmalate dehydrogenase; Leu, leucine; MS, Murashige Skoog; PCR, polymerase chain reaction; T-DNA, transferred DNA; TD, threonine deaminase; TE, <br /> transmission efficiency; Val, valine; WT, wild-type. <br /> &#194;&#169; Author 2014. Published Oxford University Press behalf Society Experimental Biology. Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 3.0/), permits unrestricted reuse, distribution, reproduction medium, provided original work properly cited. <br />  880&#226;&#8364;&#8218;|&#226;&#8364;&#8218;Zhang et&#194;&#160;al. <br />  Ile biosynthesis converting threonine 2-oxob-         dhad-4 (WiscDsLoxHs184_11A) obtained utanoate (Mourad King, 1995; Joshi et&#194;&#160;al., 2006).            Arabidopsis Biological Resource Center (ABRC). Seeds lib <br />                                                                       mutant kindly offered Dong Liu (Yu et&#194;&#160;al., 2013). seeds <br /> condensed molecule pyruvate following                                                                       surface sterilized using 70% ethanol 1&#226;&#8364;&#8240;min, 50% bleach enzymatic reactions, 2-oxobutanoate transformed               10&#226;&#8364;&#8240;min, washing times sterilized water 1&#226;&#8364;&#8240;min Ile (Yu et&#194;&#160;al., 2013). unique feature BCAA biosyn-          time. Seeds germinated half-strength Murashige <span id='am-2' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; <br /> thesis single set enzymes catalysing Ile pro-       Skoog (half MS) agar medium containing 1% sucrose, cul- <br /> duction utilized Val biosynthesis. enzymes          tivated growth chamber photoperiod 16&#226;&#8364;&#8240;h light                                                                       22&#194;&#176;C 8&#226;&#8364;&#8240;h dark 20&#194;&#176;C, 70% humidity, 7&#194;&#160; days. acetohydroxyacid synthase (AHAS; EC 4.1.3.18), ketol- <br />                                                                       seedlings transferred soil grown condi- <br /> acid reductoisomerase (KARI; EC 1.1.1.86), dihydroxyacid              tions. salt nickel stress experiments, Arabidopsis seedlings <br /> dehydratase (DHAD: EC 4.2.1.9) branched-chain ami-                germinated grown half MS medium  <br /> notransferase (BCAT; EC 2.6.1.42). intermediate              taining different concentrations Na+ Ni2+ adding NaCl Val biosynthesis, 3-methy-2-oxobutanoate, serves           NiSO4, respectively. root length measured ruler <br />                                                                       number lateral roots recorded using dissecting light initial substrate pathway Leu biosynthesis. <br />                                                                       microscopy. undergoing enzymatic steps, including condensa- <br /> tion, isomerization, decarboxylation transamination, intermediate ultimately converted Leu  Kraker            DNA extraction genotyping <br /> et&#194;&#160;al., 2007; Knill et&#194;&#160;al., 2008; et&#194;&#160;al., 2009; et&#194;&#160;al., 2010;   rapid genomic DNA extraction protocol utilized study. et&#194;&#160; al., 2011). <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, virtually enzymatic activities        Briefly, pieces leaf tissue collected 3-week-old plants                                                                       grounded extraction buffer (200&#226;&#8364;&#8240;mM Tris/HCl pH 8.0, 250&#226;&#8364;&#8240;mM <br /> required BCAA biosynthesis detected plants,          NaCl, 25&#226;&#8364;&#8240;mM EDTA, 0.5% SDS) using small blue pestle. <br /> including model plant Arabidopsis.  genes         centrifugation 14,000&#226;&#8364;&#8240;rpm 5&#226;&#8364;&#8240;min 4&#194;&#176;C, superna- <br /> encoding enzymes characterized              tant transferred new tube precipitated adding (Binder, 2010). example, little known KARI          equal isopropanol. centrifugation 14,000&#226;&#8364;&#8240;rpm DHAD, corresponding genes Arabidopsis             5&#226;&#8364;&#8240;min 4&#194;&#176;C, DNA pellets rinsed 70% ethanol, air dried <br />                                                                       10&#226;&#8364;&#8240;min, finally resuspended 1XTE buffer (10mM Tris/HCl annotated years based sequence similarity         pH 8.0 0.1mM&#194;&#160;EDTA). <br /> microbial homologues (Binder, 2010).                                     PCR reactions performed total volume 10&#194;&#160; &#206;&#188;l    Proline accumulation long recognized play              5&#194;&#160;&#206;&#188;l 2X Green GoTag Master mix (Promega, http://www.promega. important role osmotic regulation wide range            com/), 1&#194;&#160; &#206;&#188;l DNA template, 0.2&#194;&#160; &#206;&#188;l 10ng &#206;&#188;l&#226;&#8364;&#8220;1 forward reverse abiotic stresses (Liu Zhu, 1997; Verbruggen                primers, 3.6&#194;&#160;&#206;&#188;l ddH2O. PCR programme follows: 30 <br />                                                                       s 94&#194;&#176;C, 30 s 50&#226;&#8364;&#8220;55&#194;&#176;C (annealing temperature dependent Hermans, 2008; Mattioli et&#194;&#160; al., 2009; Hayat et&#194;&#160; al., 2012).          specific primer pairs; Supplementary Table S1), 1&#226;&#8364;&#8240;min kb&#226;&#8364;&#8220;1 Partial deficiency proline mutants impaired proline          72&#194;&#176;C total 37 cycles. T-DNA-specific primers SALK biosynthesis, p5cs1, caused plants hyper-           WiscDsLoxHs lines LBa1 LB4, respectively. dhad-1 sensitive salt stress (Sz&#195;&#169;kely et&#194;&#160;al., 2008).       dhad-2 genotyped using LBa1/DHAD-1RP DHAD-1LP/ recognized proline overproducing           DHAD-1RP, dhad-3 genotyped using LB4/DHAD-2RP                                                                       DHAD-2LP/DHAD-2RP, dhad-4 genotyped using LB4/ <br /> amino acid osmotic stresses studies. fact,       DHAD-3RP DHAD-3LP/DHAD-3RP. <br /> levels amino acids, particular types BCAAs, elevated levels comparable proline greater (Jander Joshi, 2009; Joshi et&#194;&#160;al., 2010). sug-       RNA extraction quantitative real-time&#194;&#160;PCR <br /> gests BCAA biosynthesis involved plant          RNA extracted leaves roots using RNeasy plant <br />                                                                       mini kit (Qiagen, http://www.qiagen.com/). cDNA synthesized <br /> stress tolerance, direct evidence obtained                                                                       5&#194;&#160;&#206;&#188;g total RNA using ProtoScript&#194;&#174; Strand cDNA <br /> support hypothesis.                                              Synthesis Kit (New England Biolabs) following manufacturer&#226;&#8364;&#8482;s <br />    investigate important roles DHAD regulating           instructions. Quantitative real-time PCR conducted ABI <br /> plant development stress tolerance, characterized          7500 real-time PCR (Applied Biosystems) according physiological alterations Arabidopsis DHAD knockout            manufacturer&#226;&#8364;&#8482;s instructions. PCR reactions performed                                                                       total volume 20&#194;&#160;&#206;&#188;l containing 10&#194;&#160;&#206;&#188;l iTaq Universal SYBR Green <br /> knockdown mutants. lethality knockout mutants <br />                                                                       Supermix (Bio-Rad), 0.5&#194;&#160;&#206;&#188;l 10ng &#206;&#188;l&#226;&#8364;&#8220;1 forward reverse primers identified partially caused impairment male          (DHAD-F DHAD-R), 2&#194;&#160; &#206;&#188;l diluted cDNA. PCR female gamete development. knockdown mutants,              programme follows: 95&#194;&#176;C 30 s, followed 40 cycles levels types BCAA reduced roots,          95&#194;&#176;C 5 s 60&#194;&#176;C 34s. levels gene expression leading shorter root phenotype. interestingly,          calculated using comparative CT method expression                                                                       ACTIN2 used internal control data normalization. <br /> knockdown mutants exhibited higher sensitivity salt stress, <br />                                                                       Data shown averages independent experiments providing evidence time BCAA homeostasis           primer sequences listed Supplementary Table&#194;&#160;1. <br /> plays role plant salt tolerance. <br />                                                                       Promoter-GUS fusion, Arabidopsis transformation, Materials methods                                                 GUS&#194;&#160;assay <br />                                                                       Arabidopsis DHAD promoter  &#226;&#710;&#8217;965 &#226;&#710;&#8217;1&#226;&#8364;&#8240;         bp) Plant materials                                                       amplified genomic DNA using primers DHAD-1L Seeds Arabidopsis thaliana ecotype Columbia (Col-0, CS3879)        DHAD-1R, PCR product cloned pGEM-T vec- SALK mutant lines dhad-1 (SALK_062347), dhad-2                    tor (Promega). accuracy verified sequencing, pro- <br /> (SALK_075098/SALK_130404), dhad-3 (WiscDsLoxHs135_03D),               moter sequence subcloned pCAMBIA1305 using SacI                                                                            Molecular characterization DHAD Arabidopsis&#226;&#8364;&#8218;|&#226;&#8364;&#8218;881 <br />  <br /> NcoI restriction sites produce PromoterDHAD::GUS construct             worth noting DHAD occur single-copy <br /> (GUS, &#206;&#178;-glucuronidase).                                                       gene plant species investigated, implying    construct transformed Agrobacterium tumefaciens <br />                                                                               selection pressure preserve DHAD genes singletons fol- <br /> strain GV3101 subsequently transformed plants floral dipping method (Clough Bent, 1998). Transgenic                 lowing genome-wide small-scale duplication events plants selected using 50&#226;&#8364;&#8240;ng &#194;&#181;l&#226;&#8364;&#8220;1 hygromycin.                             evolution. <br />    Histochemical GUS assays performed homozygous T3                      investigate tissue-specific expression DHAD, <br /> transgenic plants described previously (Jefferson et&#194;&#160;al., 1987).           histochemical assay GUS activity performed using <br />                                                                               transgenic Arabidopsis containing DHAD promoter-GUS <br /> Amino acid analysis                                                           construct. 20 independent transgenic lines investigated, <br /> Free amino acids extracted using methanol chloroform               19 displayed similar patterns GUS expression. Strong <br /> method leaves (100&#226;&#8364;&#8240;mg) roots (50&#226;&#8364;&#8240;mg) (Colebatch et&#194;&#160; al.,             GUS staining observed cotyledons roots <br /> 2004), profiled using HPLC-based pre-column derivatization             3&#194;&#160; days germination (Fig.&#194;&#160; 1B). 10&#194;&#160; days, intense <br /> protocol (Schuster et&#194;&#160;al., 2006).                                             staining developing leaves roots (Fig.&#194;&#160;1C). <br />                                                                               flower, GUS staining observed nearly                                                                               organ parts, including sepals, petals, anthers, pistils <br /> Results                                                                       (Fig.&#194;&#160;1D, E). <br /> AtDHAD encoded single gene constitutively expressed                                                      Identification T-DNA mutants AtDHAD <br /> Based high sequence similarity orthologues                     AtDHAD gene contains 14 exons 13 introns, organisms, gene encoding DHAD Arabidopsis                          encodes putative 608 amino acid protein (Fig.&#194;&#160; 2A). assigned AT3G23940 Plant Metabolic Pathway                     investigate biological importance AtDHAD plant <br /> Database (www.PlantCyc.org) reviews (Jander                      development stress tolerance, obtained T-DNA Joshi, 2009; Binder, 2010). evaluate evolution-                    insertion lines SALK WiscDsLox collections <br /> ary relationships DHAD different plant organ-                       (Alonso et&#194;&#160; al., 2003; Woody et&#194;&#160; al., 2007), named  <br /> isms, phylogenetic tree based amino acid sequences                       dhad-1 (SALK_062347), dhad-2 (SALK_075098/ constructed. result showed DHAD genes                       SALK_130404), dhad-3 (WiscDsLoxHs135_03D), dhad- <br /> monocots dicots formed distinct clades, indicating                    4 (WiscDsLoxHs184_11A). insertions sites sequences fundamentally conserved,                     mutant lines verified sequencing junctions DHAD genes underwent subsequent divergence accompa-                           gene/T-DNA. noted sequencing results <br /> nied evolution groupss (Fig.&#194;&#160; 1A).                 showed SALK_075098 SALK_130404 harbour <br />  <br />  <br />  <br />  <br /> Fig.&#194;&#160;1.&#226;&#8364;&#8218; Phylogenetic analysis tissue-specific expression Arabidopsis DHAD.   phylogenetic tree constructed neighbour-joining <br /> method using 1000 bootstraps. DHAD phylogeny rooted Escherichia coli DHAD orthologue. (B&#226;&#8364;&#8220;E) Histochemical assay GUS activity <br /> using AtDHAD promoter-GUS plants: (B) 3-day-old seedling; (C) 10-day-old seedling; (D) (E) flowers 3-week-old plants. figure available colour JXB online. <br />  882&#226;&#8364;&#8218;|&#226;&#8364;&#8218;Zhang et&#194;&#160;al. <br />  <br />  <br />  <br />  <br /> Fig.&#194;&#160;2.&#226;&#8364;&#8218; AtDHAD gene structure, mutation sites, phenotype loss function mutant alleles.   Exon-intron structure T-DNA insertion sites. arrow indicates T-DNA insertion site different mutant alleles: 1, dhad-1; 2, dhad-2; 3, dhad-3; 4, dhad-4. (B&#226;&#8364;&#8220;D) Alexander staining pollen <br /> grains WT (B), DHAD-3/dhad-3 (C), DHAD-4/dhad-4 plants (D). Arrows highlight smaller pollen grains. Bar, 100&#194;&#160;&#206;&#188;m. (E) Statistical <br /> counting aborted pollen grains. Error bars indicate standard error. (F&#226;&#8364;&#8220;H) Examination open siliques WT (F), DHAD-3/dhad-3 (G), DHAD-4/dhad-4 plants (H). Arrows highlight aborted ovules.   Statistical counting aborted ovules. Error bars indicate standard error. figure available colour JXB online. <br />  <br />  <br /> T-DNA location, SALK_130404                            donors, TEs dhad 46.8% 44.3% dhad-3 <br /> chosen study. insertions dhad-1 dhad-2                  dhad-4, respectively, indicating proportion dhad <br /> reside putative promoter 5&#226;&#8364;&#8482;UTR regions 273&#226;&#8364;&#8240;bp                   male gametes aborted. Likewise, female TEs 88&#226;&#8364;&#8240;bp upstream start codon, respectively.                       43.9% 41.3% dhad-3 dhad-4, respectively, indicat- <br /> insertions dhad-3 dhad-4 locate exon 1 intron 3,                 ing female gametophyte affected similar <br /> respectively (Fig.&#194;&#160;2A).                                                        extent male gamete (Table&#194;&#160;1). <br />                                                                                   clarify effects disruption DHAD male gam- <br />                                                                                ete development, Alexander staining used observe <br /> AtDHAD essential gene Arabidopsis <br />                                                                                pollen viability morphology (Alexander, 1969). heterozygous plants dhad-3 dhad-4 did                      WT, pollens grains uniformly shaped sized apparent phenotypes normal growth conditions,                        (Fig.&#194;&#160;2B). comparison, largely red stained, ~40% failed homozygous plants large                   pollen grains DHAD-3/dhad-3 (40.6%; n&#194;&#160; =&#194;&#160; 1225) <br /> population T2 progeny (Table&#194;&#160;1), suggesting loss                    DHAD-4/dhad-4 (38.3%; n&#194;&#160;=&#194;&#160;1335) smaller size <br /> function DHAD caused lethality.  segregation                      (Fig.&#194;&#160;2C&#226;&#8364;&#8220;E). Given percentage pollen grains dis- <br /> distortions observed dhad-3 dhad-4 alleles,                   playing morphological alterations correlated ratios wild-type (WT) heterozygous plants far                   observed frequency male gametophytes mutant different expected 1:2 (indicative embryo                     genotype (Table&#194;&#160;1), inferred genotype small <br /> lethality) 1:1 (indicative single gamete lethality), sug-                pollen grains dhad, abnormality cause <br /> gesting abortion occur development                   pollen grains  non-viable compared WT <br /> male female gametophytes (Table&#194;&#160;1).                                        pollen (Fig.&#194;&#160;2C, D, E). <br />    ascertain origin lethality male, female,                     According TE data, female gametophyte develop- gametes, reciprocal cross-pollination study                   ment impaired dhad mutants (Table&#194;&#160;1). formed transmission efficiency (TE)                       supported examination showing small, <br /> male female calculated. shown Table&#194;&#160;1,                      white, unfertilized ovules substantially observed heterozygous dhad-3 dhad-4 plants used pollen                       developing siliques DHAD-3/dhad-3 DHAD-4/ <br />                                                                        Molecular characterization DHAD Arabidopsis&#226;&#8364;&#8218;|&#226;&#8364;&#8218;883 <br />  <br /> Table&#194;&#160;1.&#226;&#8364;&#8218; Genetic segregation transmission efficiency dhad- <br /> 3 dhad-4 alleles reciprocal crossesa <br />  <br />  <br /> Parental genotypes                    Progeny genotypes         TE <br />                                                                (%)b <br /> Female             Male               WT      HT       HH <br /> DHAD-3/dhad-3      DHAD-3/dhad-3      234     165      0       &#226;&#8364;&#8220; <br /> WT                 DHAD-3/dhad-3      312     146      &#226;&#8364;&#8220;       46.8c <br /> DHAD-3/dhad-3      WT                 246     108      &#226;&#8364;&#8220;       43.9d <br /> DHAD-4/dhad-4      DHAD-4/dhad-4      232     158      0       &#226;&#8364;&#8220; <br /> WT                 DHAD-4/dhad-4      273     121      &#226;&#8364;&#8220;       44.3c <br /> DHAD-4/dhad-4      WT                 208      86      &#226;&#8364;&#8220;       41.3d <br />  <br />        TE, transmission efficiency; HT, heterozgyous; HH, homozygous. <br />   b <br />      TE (%)&#194;&#160;=&#194;&#160;(observed number HT alleles/observed number WT <br /> alleles) &#195;&#8212; 100 (Ebel et&#194;&#160;al., 2004). <br />    c <br />      Male TE. <br />    d <br />      Female TE. <br />  <br /> dhad-4 (Fig.&#194;&#160;2F&#226;&#8364;&#8220;H). frequencies aborted ovules 45.6% (n&#194;&#160; =&#194;&#160; 542) 42.1% (n&#194;&#160; =&#194;&#160; 503) DHAD-3/dhad-3 DHAD-4/dhad-4, respectively (Fig.&#194;&#160; 2I). consist- <br /> ent values calculated TE analysis (Table&#194;&#160; 1). addition, noted male female gamete development resulted sterile gam- <br /> etes, small proportion viable; opposed genetic analysis homozygous mutant <br /> plants recovered F2 progeny, implying loss function DHAD cause embryo lethality. <br />  <br /> Decreased expression AtDHAD led retarded <br /> root&#194;&#160;growth contrast lethality dhad-3 dhadh-4 knock- <br /> outs, homozygous plants successfully obtained self-pollinating progenies heterozygous dhad-1 dhad- <br />                                                                         Fig.&#194;&#160;3.&#226;&#8364;&#8218; Rescue shorter root phenotype dhad-1 dhad-2 <br /> 2, alleles containing insertions promoter             mutants exogenous application Ile.  H) 10-day-old WT, dhad-1, <br /> 5&#226;&#8364;&#8482;UTR regions DHAD (Fig.&#194;&#160;1A). Real-time quantitative                 dhad-2 seedlings grown half MS medium supplemented PCR analysis showed expression levels DHAD               0.1mM Ile (B), 0.1mM Leu (C), 0.1mM Val (D), 0.1mM Ile/Leu (E), 0.1mM <br /> homozygous dhad-1 dhad-2 decreased 36%                  Ile/Val (F), 0.1mM Leu/Val (G), 0.1mM Ile/Leu/Val (H). Bar, 10&#226;&#8364;&#8240;mm.   <br /> 25% compared WT sibling plants (Supplementary Figure                 Root length mutants measured shown percentage <br />                                                                         relative WT grown conditions. Error bars indicate <br /> S1A), indicating T-DNA insertions promoter               standard error. figure available colour JXB online. <br /> 5&#226;&#8364;&#8482;UTR regions downregulated expression DHAD. <br />    grown soil, homozygous plants did obvious phenotypes, morphological appear-                   mutants. shown combinations contain- <br /> ance, growth rate, bolting time compared WT                      ing Ile rescue root-growth phenotype (Fig.&#194;&#160;3E, F, <br /> (Supplementary Figure S1B).  seeds ger-               H). worth mentioning exogenous application minated half MS medium grown vertically 10&#194;&#160;days,             Leu caused cellular toxicity (Fig.&#194;&#160;3C), toxicity average root length dhad-1 dhad-2 signifi-          exaggerated Leu Val added cantly shorter WT (Fig.&#194;&#160;3A), lateral root density          (Fig.&#194;&#160; 3G). described root-growth assays decreased (Supplementary Figure S2), suggesting                independently repeated times, root growth rates primary lateral root development affected.           summarized quantified Fig.&#194;&#160;3I. clarify shorter root phenotype caused                range concentrations 2-oxobutanote exog- deficiency free BCAAs, plants grown medium               enously applied growth medium. test exogenously supplied individual type variety             possibility potential buildup 2-oxobutanote, combinations BCAAs. Interestingly,            Ile biosynthetic intermediate, harm root growth dhad observed supplied Ile (Fig.&#194;&#160;3B), Leu             knockdown mutants. shown Supplementary Figure <br /> (Fig.&#194;&#160; 3C) Val (Fig.&#194;&#160; 3D), root length mutants            S3, 0.5&#226;&#8364;&#8240;mM 2-oxobutanoate did yield distinct <br /> restored level similar WT, suggesting decreased        responses dhad mutants compared WT, indicat- <br /> root growth caused reduced level Ile              ing stunted root growth related  884&#226;&#8364;&#8218;|&#226;&#8364;&#8218;Zhang et&#194;&#160;al. <br />  cellular concentration 2-oxobutanoate, alterna-                  contribute multifaceted stress tolerance systems tively toxicity 2-oxobutanoate plant growth              plants. test possibility reduction BCAA abun- <br /> extremely minimal.                                                         dance lead hypersensitivity abiotic stress, effects <br />    corroborate shorter root phenotype correlated               salt stress root growth investigated. shown reduced level BCAAs, free amino acids                     Fig.&#194;&#160;4A, low concentration Na+ (50&#226;&#8364;&#8240;mM), root <br /> quantified root leaf tissues. noted           length dhad knockdown mutants 62&#226;&#8364;&#8220;64% alleles, dhad-1 dhad-2, exhibited identical phenotypes                 WT (Fig.&#194;&#160;4G), lower 68&#226;&#8364;&#8220;71% root-length assays, dhad-2 used amino acid                  salt treatment (Fig.&#194;&#160; 3A 3I). Na+ concentration <br /> analysis. shown Table&#194;&#160; 2, roots, abundances                increased 75&#226;&#8364;&#8240;mM, ratio root length BCAAs dhad-2 showed statistically significant               mutant WT 53&#226;&#8364;&#8220;55% (Fig.&#194;&#160;4B 4G), reduction compared WT, suggesting decreased                decreased 39&#226;&#8364;&#8220;41% treatment 100&#226;&#8364;&#8240;mM <br /> gene dosage DHAD limit rate metabolic flux                 Na+. indicates dhad knockdown mutants exhib- form end products, BCAAs. addition, contents                 ited greater sensitivity compared WT increas- precursor amino acids, aspartate threonine,                ing concentration Na+. importantly, sensitivity <br /> moderately increased, suggesting partial blockage              Na+ treatment different concentrations BCAA biosynthesis lead ectopic buildup                 restored similar levels WT 0.1&#226;&#8364;&#8240;mM Ile precursor intermediates. comparison, leaves, amino                supplied exogenously. results suggest high sen- <br /> acids showed significant changes (Table&#194;&#160; 2), consistent               sitivity dhad Na+ stress attributed reduced indistinguishable phenotypes mutants WT                     level Ile (Fig.&#194;&#160;4D&#226;&#8364;&#8220;G). <br /> (Supplementary Figure S1B). Taken  results                     hypersensitive response salinity stress indicate decreased expression DHAD resulted                    directly derived salt ionic toxicity, alternatively partial deficiency BCAA biosynthesis, subsequent                 represent universal reaction osmotic stress. ascer- <br /> impairment root development.                                            tain possibilities, dhad knockdown mutants  <br /> Decreased expression DHAD resulted increased <br /> sensitivity salt&#194;&#160;stress <br /> Previous studies shown accumulation BCAAs greatly elevated abiotic stresses (Joshi Jander, <br /> 2009; Joshi et&#194;&#160;al., 2010), suggesting BCAA homeostasis <br />  <br />  <br /> Table&#194;&#160;2.&#226;&#8364;&#8218; Amino acid contents roots leaves WT dhad-2a,b,c <br />  <br />  <br /> Amino acid Roots                            Leaves <br />               WT            dhad-2          WT             dhad-2 <br /> Alanine        13.76&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.49 12.43&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.15    14.85&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.32   15.31&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.32 <br /> Arginine        0.54&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02   0.45&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    2.11&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.54    2.61&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.18 <br /> Asparagine     24.31&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;2.25 33.74&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.19 31.16&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.68    27.27&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.51 <br /> Aspartate       9.38&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.15 11.96&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.26 24.77&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.73    26.32&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.52 <br /> Glutamine      38.32&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.05 44.99&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.26 96.08&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.14    91.23&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.71 <br /> Glutamate     230.65&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.42 272.93&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;11.68 631.25&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;38.44 562.25&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;14.34 <br /> <span id='am-16' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>Glycine</span>         3.65&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.15   3.14&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02   20.61&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;4.48    16.3&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.71 <br /> Histidine       1.33&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03   1.42&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.06    1.42&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.12    1.11&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.11 <br /> Isoleucine      0.93&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01   0.66&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03 b  0.49&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01    0.46&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01 <br /> Leucine         1.35&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.04   0.96&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.04 b  0.66&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01    0.55&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.05 <br /> Lysine           1.1&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01   0.91&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01 b  0.53&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    0.61&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.10 <br /> Methionine      3.19&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.07   2.78&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.07    1.01&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.07    0.79&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02 <br /> Phenylalanine   0.51&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03   0.47&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01    0.64&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    0.73&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02 <br /> Proline        16.22&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;2.00 18.55&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.5     21.03&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.79   25.08&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;3.00 <br /> Serine         15.74&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;1.38   19.2&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.4    37.95&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.57   42.81&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;2.20 <br /> Threonine       6.71&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.17    9.8&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.32  5.48&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    6.54&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.59 <br /> Tyrosine        0.46&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02   0.39&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.03    0.31&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.02    0.31&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01 <br /> Valine          2.64&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.01   2.21&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.05 b  2.27&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.04    2.13&#226;&#8364;&#8240;&#194;&#177;&#226;&#8364;&#8240;0.15    Fig.&#194;&#160;4.&#226;&#8364;&#8218; dhad-1 dhad-2 hypersensitive salt stress.  F) <br />                                                                            10-day-old WT, dhad-1, dhad-2 seedlings grown half MS medium <br />      Values significantly higher WT (P&#194;&#160;&lt;&#194;&#160;0.05), Student&#226;&#8364;&#8482;s t-test.   supplemented different levels NaCl  C) 0.1mM Ile  F). Bar, <br />   b <br />    Values significantly lower WT (P&#194;&#160;&lt;&#194;&#160;0.05), Student&#226;&#8364;&#8482;s t-test.    10&#226;&#8364;&#8240;mm. (G) Root length mutants measured shown  c <br />    Amino acid contents given pmol mg FW&#226;&#8364;&#8220;1 (FW, fresh                percentage relative WT grown conditions. Error <br /> weight). Data mean &#194;&#177; SD replicates.                       bars indicate standard error. figure available colour JXB online. <br />                                                                          Molecular characterization DHAD Arabidopsis&#226;&#8364;&#8218;|&#226;&#8364;&#8218;885 <br />  <br /> treated increased concentrations mannitol. shown Fig.&#194;&#160; 5, low concentration mannitol, root <br /> length dhad mutants 72&#226;&#8364;&#8220;77% WT, comparable 68&#226;&#8364;&#8220;71% treatment (Fig.&#194;&#160;3A 3I).  increased concentrations manni- <br /> tol, ratio root length mutant WT increased 84&#226;&#8364;&#8220;89%, clearly different salt <br /> response, indicating exaggerated effect osmotic stress.  conclude hypersen- <br /> sitivity dhad salinity stress attributed salt <br /> ionic toxicity. <br />    inhibition root growth dhad, salt stress, reflect common response various abiotic stresses. test possibility, root-growth assays conducted different concentrations Ni2+ treatment, represents type heavy metal stress. shown Fig.&#194;&#160;6, mutants did exhibit signs high sensitivity increased concentrations Ni2+. contrast, ratios root length mutants WT slightly <br /> increased elevated concentrations Ni2+. sug- <br /> gests high sensitivity salt stress dhad quite <br /> specific. <br />  lib mutant shows pattern increased sensitivity salt&#194;&#160;stress establish functional role Ile biosynthesis tolerance salinity stress, root-growth assays  <br /> ducted lib (Yu et&#194;&#160;al., 2013). shown Fig.&#194;&#160;7, similar dhad alleles, lib shows higher sensitivity increased              Fig.&#194;&#160;6.&#226;&#8364;&#8218; dhad-1 dhad-2 hypersensitivity Ni2+ stress.  F) <br />                                                                             10-day-old WT, dhad-1, dhad-2 seedlings grown half MS medium <br /> salt concentration, hypersensitivity rescued              supplemented different levels NiSO4  C) 0.1mM Ile  F). exogenous application Ile, unambiguously indicating                   Bar, 10mm. (G) Root length mutants measured shown Ile biosynthetic pathway plays critical role salt             percentage relative WT grown conditions. Error <br /> tolerance.                                                                  bars indicate standard error. figure available colour JXB online. <br />  <br />  <br />  <br />  <br /> Fig.&#194;&#160;5.&#226;&#8364;&#8218; dhad-1 dhad-2 hypersensitivity mannitol stress.  C) 10-day-old WT, dhad-1, dhad-2 seedlings grown half MS <br /> medium supplemented different levels mannitol. Bar, 10&#226;&#8364;&#8240;mm. (D) Root length mutants measured shown percentage relative WT grown conditions. Error bars indicate standard error. figure available colour JXB online. <br />  886&#226;&#8364;&#8218;|&#226;&#8364;&#8218;Zhang et&#194;&#160;al. <br />  <br />                                                                                   requirements transported sporophytic                                                                                   reproductive tissues; (iii) potential cellular toxicities <br />                                                                                   resulting abnormal accumulation biosynthetic <br />                                                                                   intermediates having different effects plant development. <br />                                                                                      decreased expression DHAD resulted simul- <br />                                                                                   taneous reduction levels BCAAs roots, <br />                                                                                   leaves (Table&#194;&#160;2), indicating DHAD exert <br />                                                                                   rate-limiting control regulating BCAA biosynthesis                                                                                   tissue-specific manner.  possibility exist- <br />                                                                                   ence alternative pathway BCAA biosynthesis                                                                                   leaves completely ruled study. addi- <br />                                                                                   tion, retarded root growth rescued                                                                                   exogenous application Ile, Leu Val, suggesting                                                                                   impaired root growth probably caused                                                                                   deficiency Ile accumulation. result consistent                                                                                   results recent study mutation threonine <br />                                                                                   deaminase/dehydratase catalysing commit- <br />                                                                                   ted step Ile biosynthesis resulted partial Ile deficiency, <br />                                                                                   consequently retarded root development (Yu et&#194;&#160;al., 2013). <br />                                                                                    clear Ile homeostasis plays important <br />                                                                                   role regulating root development Arabidopsis.  <br />                                                                                   perturbation DHAD lead buildup meta- <br />                                                                                   bolic intermediates. instance, previously shown <br />                                                                                   inhibition AHAS cause toxic accumulation <br />                                                                                   ketobutyrate Salmonella (LaRossa, 1987).                                                                                    study, hint cellular toxicity caused 2-oxobu- <br />                                                                                   tanoate suggested, exogenous application                                                                                   substantial amounts 2-ketoburate didn&#226;&#8364;&#8482;t produce dis- <br />                                                                                   tinct response dhad relative WT. Furthermore,                                                                                   observed concentrations amino acids <br />                                                                                   affected, including increased levels asparagine, aspar- <br /> Fig.&#194;&#160;7.&#226;&#8364;&#8218; low isoleucine biosynthesis (lib) mutant hypersensitive salt   tate, glutamate, threonine, reduced level stress.  F) 10-day-old WT lib seedlings grown half MS medium              lysine. precursor amino acid Ile, accumulation <br /> supplemented different levels NaCl  C) 0.1mM Ile  F). Bar,        threonine simply explained potential build 10&#226;&#8364;&#8240;mm. (G) Root length mutants measured shown percentage relative WT grown conditions. Error <br />                                                                                   blockage downstream metabolic flux. Similarly, <br /> bars indicate standard error. figure available colour JXB online.   BCAAs aspartate-derived amino acids,                                                                                   reduction BCAAs lead build aspar- <br />                                                                                   tate, subsequently recruited asparagine.  <br /> Discussion <br />                                                                                   way glutamine increased lysine decreased essential components protein synthesis                      unclear, suggesting biosynthesis different amino <br /> important signalling molecules, BCAAs thought                        acids regulation complex crosstalk, regulate numerous aspects plant growth required plant                      precise change leading imbalance different amino <br /> survival.  little information available spe-                     acids deserves investigation. lib mutant, addi- <br /> cific roles BCAAs plant growth development                      tion reduced level Ile, amino acids interaction environment. recent years, extensive                   showed statistically significant changes abundance; studies Arabidopsis led identification                  increase phenylalanine decrease lysine. genes involved BCAA biosynthesis. date,                        consistent reduction Ile content lib dhad <br /> genes, KARI DHAD, remain functionally uncharacter-                            indicates genes involved Ile synthesis, ized. study, identified null mutations DHAD                    distinct response amino acids impair- <br /> caused partial abortion male female gametogen-                        ment different steps biosynthetic pathway <br /> esis. reminiscent loss function LeuD3                    suggests metabolic network associated step <br /> IPMDH2/IPMDH3, involved Leu biosynthesis,                            slightly dissimilar. Taken  possibil- <br /> lethality detected female male gamete development,                      ity stunted root growth consequence inter- <br /> respectively  et&#194;&#160; al., 2009; et&#194;&#160; al., 2010). underly-                   mediate accumulation completely ruled  ing causes lethality distinct organs different mutants                   speculate root phenotype likely caused include   different organs having exceptional requirements                      Ile deficiency. novo biosynthesis certain types BCAAs; (ii)                         abiotic stresses, levels amino acids <br /> different biosynthetic intermediates having dissimilar abilities                  significantly increased. Proline biosynthesis                                                                Molecular characterization DHAD Arabidopsis&#226;&#8364;&#8218;|&#226;&#8364;&#8218;887 <br />  studied shown play important roles plant           Acknowledgements <br /> tolerance various stresses Liu Zhu, 1997;              thank members laboratories helpful assistance Verbruggen Hermans, 2008; Mattioli et&#194;&#160;al., 2009; Hayat        research critical comments manuscript. Special thanks Dr <br /> et&#194;&#160; al., 2012). Disruption proline biosynthesis renders        Dong Liu TsingHua University, China, kind offer lib mutant seeds. <br /> plants hypersensitive salinity stress (Liu Zhu, 1997; <br /> Mattioli et&#194;&#160; al., 2009). Interestingly, long rec- <br /> ognized accumulation BCAAs exhib-           References <br /> ited significant induction proline cases <br />                                                                   Alexander M. 1969. Differential staining aborted nonaborted osmotic stresses, implying BCAAs play          pollen. Biotechnic Histochemistry 44, 117&#226;&#8364;&#8220;122. <br /> role plant stress responses (Joshi Jander, 2009; Joshi     Alonso JM, Stepanova  Leisse TJ, Kim CJ, Chen H, Shinn P, <br /> et&#194;&#160;al., 2010).  functional evidence obtained       Stevenson DK, Zimmerman J, Barajas P, Cheuk R. 2003. Genome- draw conclusion. study,       wide insertional mutagenesis Arabidopsis thaliana. Science 301, <br />                                                                   653&#226;&#8364;&#8220;657. <br /> decreased expression DHAD led reduction BCAA <br />                                                                   Binder S. 2010. Branched-chain amino acid metabolism Arabidopsis <br /> levels rendered plants hypersensitive salinity stress      thaliana. Arabidopsis Book 8, e0137. <br /> (Table&#194;&#160;2 Fig.&#194;&#160;4), providing time strong evi-    Clough SJ, Bent AF. 1998. Floral dip: simplified metho <br /> </body></html>