<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.12 1 pp.285-294, 1993 <br />  <br />  <br /> Isolation characterization S.cerevisiae mutants <br /> defective somatostatin expression: cloning functional role yeast gene encoding aspartyl <br /> protease precursor processing monobasic cleavage <br /> sites <br /> Yves Bourbonnais&apos; 3, Josee Ash1,                                      terminally 14-<span id='am-18' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-19' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> extended form SRIF-14 (somato- <br /> Mireille Daigle&apos; David Y.Thomas&apos; 2                                tropin releasing inhibitory factor). peptides display <br />                                                                       widespread effects inhibiting secretion numerous <br /> &apos;Eukaryotic Genetics Group, National Research Council Canada,      hormones including growth hormone, glucagon insulin. <br /> Biotechnology Research Institute, 6100 avenue Royalmount, Montreal, <br /> Quebec, H4P 2R2 2Department Biology, McGill University,        overlap activity SRIF-14 Montreal, Quebec, H3A IBI, Canada                                     -28, distinctive biological effects. mammals <br /> 3Corresponding author                                                 bioactive peptides synthesized common <br />                                                                       precursor released endoproteolytic activation; single <br /> Communicated Hugh R.B.Pelham                                       arginine residue (Arg - 15) constitutes cleavage site                                                                       release SRIF-28, proteolytic cleavage <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> somatostatin exists different molecular            internal dibasic sequence (Arg -2 Lys -1) generates <br /> species. addition common form, somato-                 SRIF- 14. Differential processing cleavage sites statin-14, fourteen amino acid N-terminally           shown tissue-specific (Patel et al., 1981; Noe <br /> extended form tetradecapeptide, somatostatin-28.               Spiess, 1983).  species fish, peptides synthesized larger precursors                    SRIF-14 -28 encoded distinct related <br /> containing paired basic monobasic amino acids                  precursors, preproSRIF -II, expressed processing sites, cleavage generate                  different subpopulations pancreatic islet cells (McDonald somatostatin-14 -28, respectively.                  et al., 1987). Despite possessing potential cleavage <br /> species fish distinct, homologous, precursors              sites  e. Arg -5 Arg -2 Lys -1), proSRIF cleaved <br /> (prosomatostatin -II) rise somatostatin-14              Arg -Lys pair yield SRIF-14, SRIF-28 -28, respectively. anglerfish prosomato-                  unique end product maturation proSRIF-II. <br /> statin-TI previously shown release exclusively                    attempt understand molecular basis somatostatin-28, yeast Saccharomyces cerevisiae                   differential processing anglerfish prosomatostatins, proteolytically matures homologous prosomatostatin              initiated studies biosynthesis precursors precursor somatostatin-28 -14 lysine-          yeast Saccharomyces cerevisiae. choice yeast extended form somatostatin-14. Kex2 endo-                      model predicated observation time <br /> protease appears essential formation lysine          yeast unique eukaryote prohormone- <br /> somatostatin-14 involved directly                    converting enzyme unambiguously identified (Julius <br /> indirectly release mature somatostatin-14.              et al., 1984). Furthermore, recent studies indicated isolation yeast mutants defective somatostatin-28               yeast KEX2 gene product, member subtilisin <br /> expression (sex mutant) allowed cloning non-                 family serine proteinases, functionally essential gene, encodes aspartyl protease,             structurally related mammalian counterparts (Thomas <br /> disruption severely affects cleavage mature                    et al., 1988; Seidah et al., 1990; Smeekens Steiner, <br /> somatostatin-28 somatostatin precursors.                 1990; Smeekens et al., 1991). previous study revealed <br /> conclude distinct endoproteases, demon-                native somatostatin precursors faithfully <br /> strate cross specificity vivo, involved            transported yeast secretory pathway unless signal <br /> proteolytic maturation prosomatostatin mono-                peptides replaced leader sequence yeast <br /> dibasic processing sites yeast.                                    ca-factor precursor (Bourbonnais et al., 1991). Expression <br /> Key words: heterologous gene expression/prohormone                    ct-factor - proSRIF-IH chimera promoted efficient <br /> processing/somatostatin/yeast aspartyl protease/yeast mutants         transport led secretion mature SRIF-28,                                                                       demonstrating accurate proteolytic cleavage monobasic <br />                                                                       processing site (Arg - 15). importantly, provided <br />                                                                       genetic evidence convertase involved Introduction                                                          maturation proSRIF-II Arg -15 distinct small peptide hormones neuropeptides                     paired basic-specific endoprotease encoded KEX2 gene <br /> synthesized larger polyproteins undergo series           (Julius et al., 1984). present study investigated <br /> post-translational modifications generate bioactive            biosynthesis similar chimeric protein proSRIF molecule (Douglass et al., 1984). common feature                 developed screening procedure isolation yeast precursors pairs basic amino acids flank          mutants defective somatostatin-28 expression (sex <br /> active peptide sequences constitute endoproteolytic           mutants). Genetic complementation sex]-] mutant <br /> cleavage site.  examples                 strain led cloning gene, YAP3, recently isolated <br /> prohormones bioactive peptide cleaved              group based ability partially suppress <br /> monobasic processing sites paired basic                phenotype kex2 null mutant (Egel-Mitani et al., <br /> residues (Schwartz, 1986; Devi, 1991). cleavage usually           1990). gene, encodes occurs single arginine residues. Somatostatin-28 N-          aspartyl protease, required release SRIF-28                                                                                                                                  285 <br />  Y.Bourbonnais et al. <br />  <br />  SRIF precursors cleavage single arginine <br />  residue (Arg -15). Consistent role pro factor <br />  maturation absence functional KEX2 gene                                                     %pSI D                       14S aa <br />  (Egel-Mitani et al., 1990), data endo- <br />  proteases encoded KEX2 YAP3 demonstrate               Fig. 1. Schematic representation chimeric protein encoded                                                                    plasmid pSiD. fusion protein encoded plasmid pSID 145 <br />  cross specificity vivo.                                        amino acids long composed 49 N-terminal residues <br />                                                                    prepro factor ( 1 ), including N-linked glycosylation site <br />                                                                    (?), fused frame anglerfish proSRIF  SRIF precursor  Results                                                           composed 82 amino acid proregion ( g ) containing                                                                    mono- dibasic processing site (0). basic residues   Mature SRIF- 14, mature SRIF-28 Lys-SRIF- 14             cleavage sites indicated single letter code. mature <br />   end products maturation proSRIF                  SRIF-14 peptide (U) located carboxyl terminal end   chimera wild type yeast                                       precursor. <br />   strategy used construction  proSRIF-IH (Bourbonnais et al., 1991) proSRIF  chimeric genes. plasmid carrying proSRIF fusion                   50   0)                                                   2000 <br />  gene called pSID gene expression  control ca-factor promoter. hybrid protein  composed 49 amino terminal residues prepro                  &apos;1200 <br />  factor fused frame length proSRIF (preproSRIF- <br />  26-121) (Figure 1). peptide SRIF-14 located              CL <br />                                                                              3000 <br />  carboxyl-terminal end precursor flanked                                                                                                                                                   1 000 <br />  pair basic amino acid residues (Arg -2 Lys -1).             ulI <br />                                                                              2000 <br />  single arginine residue, cleavage generates SRIF-28, <br />  located 15 residues upstream SRIF-14 (Arg- 15). <br />  hybrid precursor possess cysteine residues,                    &apos; 000 <br />  reside SRIF-14 moiety positions 3   14 respectively. Consequently, [35S]cysteine used  metabolic labelling experiments. labelled proteins  cell lysates media immunoprecipitated <br />  anti-SRIF-14 polyclonal antibody (RSSI B23;                                       FR ACT <span id='am-12' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-13' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ION</span>     NUMBEFR <br />  Materials methods). Analysis SRIF-14 immuno- <br />  reactive material secreted wild type yeast transformed       Fig. 2. Biosynthesis  factor-proSRIF fusion protein wild <br />  pSID (strain 20B-12) high performance liquid             type yeast. Strain 20B-12 transformed plasmid pSID labelled <br />                                                                   [35S]cysteine culture medium (U-U)  chromatography (HPLC) revealed presence major             intracellular labelled material (0-0) submitted  SRIF-14-like species eluting C18 column              immunoprecipitation anti-SRIF-14 antibodies  Materials  fractions 15, 19 23, respectively (Figure 2). labelled   methods). SRIF-14 immunoreactive material loaded  immunoreactive material fraction 23 eluted          C18 column, eluted gradient acetonitrile 1 min <br />  synthetic SRIF-14 used standard. major            fractions collected radioactivity determined                                                                   scintillation counting described previously Bourbonnais et al. <br />  secreted peptides resolved high resolution    (1988). arrow indicates position elution synthetic <br />  SDS -Tricine gel (Figure 2, inset). calibration experiment     SRIF-14. Note ordinates secreted intracellular <br />  performed HPLC-purified peptides indicated         material left- right figure respectively. Inset, <br />  3, 2.2 1.8 kDa species correspond immuno-         analysis secreted SRIF-14 immunoreactive material 20B-12 <br />                                                                   transformed plasmid pSID high-resolution SDS-Tricine gel <br />  reactive material eluting fraction 15, 19 23, respect-     Materials methods details). Note lower <br />  ively  shown). 3 kDa peptide migrated            portion corresponding autoradiogram shown. hSRIF-28  synthetic human SRIF-28, 1.8 kDa species             SRIF-14 indicate positions migration human synthetic migrated synthetic SRIF-14. identity         SRIF-28 synthetic SRIF-14, respectively. <br /> peptides confirmed partial amino acid sequencing HPLC-purified radiolabelled peptides (data              44 54, respectively. Based elution position shown). expected amino acid sequence mature         molecular masses, estimated SDS -Tricine gels <br /> SRIF-14, radioactivity 1.8 kDa peptide (fraction        (17 19 kDa), correspond pro prepro <br /> 23 HPLC) came cycles 3 14. Amino acid              forms hybrid protein. <br /> sequence determination 2.2 kDa species (fraction 19           Quantification radioactivity recovered HPLC) revealed presence cysteine residues          SRIF species intracellularly indicates positions 4 15 respectively, identifying       processed forms accounted &gt;40% total SRIF- <br /> peptide lysine-extended form SRIF-14 (Lys-SRIF-14).       immunoreactive material. Overall (cell lysate plus medium) material eluting fraction 15, possible       - 80% radioactivity recovered mature sequence. probably              peptides  86% culture medium <br /> relatively low radioactive material available       end labelling period (30 min). precursor <br /> radiosequencing remote locations  positions 17         forms secreted significant amounts (&lt;5 %). 28) cysteine residues SRIF-28. addition       observations strongly argue chimeric protein secreted peptides, HPLC analysis intracellular       transported efficiently secretory pathway material previously treated endoglyco-         processing precursor did place culture <br /> sidase H identified SRIF species eluting fractions   medium. Maturation hybrid precursor SRIF-14 286 <br />                                                                                S.cerevisiae mutants defective somatostatin expression <br />  <br />                                                                          arginine residue (Arg -15), releases SRIF-28                                 12                                       SRIF precursors, effected endoprotease distinct <br />                                                                          Kex2. <br />                                                                          Isolation partial characterization                                                                          somatostatin-28 expression mutants <br />                                                                          Problems inherent purification non-abundant <br />                                                                          proteinases prompted clone gene encoding novel <br />                                                                          activity direct genetic complementation mutant <br />                                                                          yeast unable recognize cleave                                                                          monobasic site present proSRIFs. facilitate                                                                          screening procedure involved isolation                                                                          mutant plasmid pS2B, encodes proSRIF-I1 <br />                                                                          fusion protein (Figure 4A), modified site-directed <br />                                                                          mutagenesis order replace SRIF-28 moiety                                                                          single copy factor peptide. resulting gene                                                                          named pS3C encodes fusion protein 132 amino acids <br /> Fig. 3. High-resolution SDS-Tricine gel secreted SRIF-14           Figure 4A structure). Expression gene immunoreactive material wild type kex2 mutants transformed      control factor promoter. Plasmid pS3C pSID. Wild type (strain 20B-12, lane 1) kex2 yeast (strain      introduced ca cells carry disrupted copy <br /> DS20, lane 2) transformed plasmid pSID labelled           MFal MFa2 genes (strain DS7) secretion <br /> [35S]cysteine culture media immunoprecipitated anti-   bioactive ca-factor strain requires proper <br /> SRIF-14 antibodies described earlier (Figure 2). labelled <br /> proteins resolved high resolution SDS-Tricine gel.        maturation hybrid precursor monobasic site. <br /> Arrowheads left indicate position migration         rationale use assays based secretion prestained low molecular weight standards (Gibco/BRL): ovalbumin         bioactive ca-factor  e. &apos;halo&apos; mating assays, Julius <br /> (43.0 kDa); carbonic anhydrase (29.0 kDa); f-lactoglobulin (18.4         et al., 1984). distinguish mutants specific- <br /> kDa); lysozyme (14.3 kDa); bovine trypsin inhibitor (6.2 kDa)  <br /> insulinf-  chains (3.0 kDa).                                       ally affect maturation hybrid precursor                                                                          perturb transcriptional regulation factor <br /> Lys-SRIF- 14, accounted approximately                 promoter, mutant primary defect does thirds processed peptides,   sharp              directly involve monobasic-specific endoproteolytic <br /> contrast processing proSRIF-I1 chimera.             activity, plasmids constructed. Plasmid matured exclusively SRIF-28                 pGS3C derivative pS3C chimeric gene evidence futher processing Arg -2 Lys            -      control GAL] promoter paFF cleavage site (Bourbonnais et al., 1991).                                truncated version MFal gene encodes poly- <br />                                                                          peptide 98 amino acids retained single copy <br /> Proteolytic processing proSRlF chimera                       mature factor peptide immediately preceded Lys-SRIF- 14 requires functional KEX2 gene                             Lys Arg dibasic processing site. reasoned mutant release SRIF-28 proSRIF chimera               strain capable secreting bioactive factor <br /> cleavage monobasic processing site (Arg - 15)                   carrying paFF, pS3C, potential <br /> previously shown occur absence functional                 candidate defect gene encoding monobasic- <br /> KEX2 gene (Bourbonnais et al., 1991). consistent                 specific endoprotease. known vitro specificity Kex2 endoprotease                 Control &apos;halo&apos; mating assays showed DS7 paired basic amino acids (Julius et al., 1984). test              transformants expressing pS3C paFF, SRIF-14-like peptides secreted yeast             expressing pS2B, elicit zone inhibition (&apos;halo&apos;) <br /> carrying pSID released action Kex2,                  lawn supersensitive cells (Figure 4B) kex2 mutant yeast (strain DS20) transformed                   competent mating (Figure 4C). Strain DS7, carrying <br /> plasmid pSl D. comparison labelled SRIF-immuno-                 pS3C, subjected UV irradiation  Materials <br /> reactive material secreted wild type kex2 yeast                 methods). independent experiments total <br /> strains SDS - Tricine gels (Figure 3, compare lanes 1             &gt; 9000 colonies survived UV exposure 2) showed release SRIF- 14 SRIF-28              screened mating competence MATa cells. affected absence functional Kex2 endo-                   mutated colonies, 39 mating <br /> protease.  peptide Lys-SRIF-14 completely                defective (Table . Halo assays performed absent culture medium kex2 transformant.                 mutants identified class mutant (16 colonies, class  results obtained pSiD expressed                   mating defect associated reduced strain carrying disrupted KEX2 gene (strain                       production secretion bioactive factor (Table . <br /> M213-1OC)  shown). indicates                  remaining 23 mutants, correlation differences kex2-1 kex2 null alleles             mating secretion biologically active factor release SRIF-14 SRIF-28. likely                      observed, distinguish classes. class <br /> explanation results Kex2                     II mutants (15 colonies) unable produce zone endoprotease involved cleavage proSRIF                 inhibition lawn supersensitive strain <br /> chimera Lys-SRIF- 14. extends previous                     M200-6C size &apos;halo&apos; produced observations proSRIF-II hybrid (Bourbonnais et al.,             class III mutants  colonies) significantly reduced. <br />  1991) demonstrates cleavage single                   Expression plasmid pGS3C representative members <br />                                                                                                                                   287 <br />   Y.Bourbonnais et al. <br />  <br />  <br />  <br />              ff <br />           l <br />  <br />  <br />  <br />  <br />   C4--..=_= <br />  <br />   rffl-- - d <br />                1-_ . {;j &apos; <br />                f <br />  <br />  <br />  <br />  <br />                _.d <br />                       &lt;X ;- <br />  <br />  <br />  <br />  <br /> Fig. 4. Strategy used isolation somatostatin expression mutants.   Schematic representation plasmids pS2B, pS3C paFF: ( <br /> ca-factor preproregion; (?) N-linked glycosylation site; ( g ) length proSRIF-II; (0) cleavage site; (-) mature SRIF-28; (O) mature <br />  ax-factor. Note basic residues cleavage sites indicated single letter code. (B) Halo assay  Materials methods) <br /> performed indicated DS7 transformants lawn indicator MATa strain M200-6C (sstl sst2). (C) Prototroph selection diploid <br /> yeast resulting cross indicated DS7 transformants MATa  tester strain. <br />  classes showed contrast class III mutants, <br /> mutants   pertaining class II elicit wild type &apos;halo&apos; hybrid gene expressed control GAL] promoter. strongly suggests class <br /> II mutants defective expression hybrid gene <br />                                                                                 mutants) <br />                                                                                  Phenotypic <br />                                                                                 class <br />                                                                                                       ...F&lt;. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                    mutants <br />                                                                                                                Mating <br />                                                                                                                            pS3C     pGS3C <br />                                                                                                                                                      ) <br />  <br />  <br />  <br />  <br />                                                                                 Table  Isolation mutants defective somatostatin expression (sex <br />  <br />                                                                                                                            Production factora                                                                                                                                                    pcaFF control ct-factor promoter. test class .111 mutants retained ability                       Wild type       NA            +           +++      +++            +++ <br />                                                                                  Class         16            -           +++      ND             ND <br /> secrete bioactive ct-factor flanked normal                    Class II        15            -           _        +++ (4)b       ND <br /> processing site (Lys Arg) precursor, plasmid pceFF                        Class III        8            -           +        + (2)          +++ (2) introduced members class III. trans-                                                                            + (1)          + (1) <br /> formant (B51) secrete low levels pheromone regardless nature cleavage site (Lys                    aProduction mature factor estimated &apos;halo&apos; assay       Arg pcxFF; single Arg pS3C), suggesting                         lawn supersensitive strain (M200-6C; Table III). <br /> mutation disrupted cellular function directly                        &quot;Number mutants phenotypic class transformed <br />                                                                                indicated plasmid assayed production halo involved maturation monobasic processing site.                   M200-6C. strains (B34 B22),  secreted wild type                      Absence mating halo indicated (-). size halo <br /> near wild type levels bioactive o-factor carrying                      produced wild type strain indicated (+++ +); reduction <br /> paFF estimated size &apos;halo&apos; lawn                       25-50% diameter halo indicated (+ +); supersensitive cells (strain M200-6C).                          reduction 50-75% diameter halo indicated (+). <br />                                                                                 NA,     applicable; ND,      determined. <br /> conclude strains B34 B22 carry mutations affecting directly  gene encoding endoprotease itself)                    copies MFct genes. initial 7 positive clones, indirectly (e.g. gene involved proper localiza-                  5 shown mate efficiently secrete wild type levels <br /> tion function endoprotease) putative monobasic                   ca-factor absence plasmid pS3C. Hybridiza- <br /> endoprotease activity. characterization, strain                tion plasmid DNA carried clones B34 (sex]-]) used recipient cell cloning                   radiolabelled probe derived patFF confirmed putative monobasic-specific endoprotease gene.                             functional copy MFct genes introduced                                                                                strains (data shown). positive clones, <br /> Cloning characterization multicopy                                    expression plasmids required restoration <br /> suppressor sex 1-1     mutation                                         wild type pheromone production mating yeast genomic library constructed multicopy plasmid                   competence. YEP24-based plasmids carried YEP24 used transform strain B34 carrying pS3C.                          strains rescued transformation Escherichia coli total &gt; 10 000 Ura+ Trp+ transformants,                           designated p9A242 p8B234. Partial restriction maps <br /> isolated seven clones mating defect                    p9A242 (9.8 kb insert) p8B234 (14 kb insert) ability produce &apos;halo&apos; lawn sensitive cells                 suggested share common SalI -XbaI DNA <br /> (M200-6C) corrected  shown). Given                            fragment -2.7 kb (Figure 5). confirmed phenotype parental strain (MFcel :: LEU2                                hybridizing plasmids probe derived MFoc2:: LEU2) sex]-] mutation did                            Sall -XbaI fragment p9A242  shown). significantly affect proteolytic processing native                      subcloning common fragment pTZ 1 8R ct-factor precursor, anticipated cloning functional                    generation precise restriction map, 288 <br />                                                                                        S.cerevisiae mutants defective somatostatin expression <br />  <br />                                              M              G <br />                  G                                              M                                                                                  C <br />            3&apos;                                                            5&apos; <br />                                                                                                  n                                     ,::-            DU <br />                                                                                                                                                        *      sm&quot;d repaired VT <br />                                                                                                  * aso <br />                                                                                                       e <br />                                                                                                                                               .C      U     lasmid repaired sexl <br />  <br /> p9a242 <br /> (9.8 Kb)                                         (B/Sa) <br />                 (B/Sa) <br />  <br />  <br />                                                                                       -[ <br />                       XNK NG S      XC   X       G NCK BB <br /> p8b234                  Ia        V             (14 Kb)                                                         (B/Sa) <br />            (B/Sa) <br />  <br />  <br />                      - 2 Kb <br />  <br /> Fig. 5. Partial restriction map plasmids p9A242 p8B234. partial restriction map inserts carried plasmids p9A242         Fig. 6. gene carried plasmids p9A242 p8B234 acts p8B234 established following restriction enzymes: BamHI          multicopy suppressor sexl-1 mutation. Quantitative mating <br /> (B); BgllI (G); KpnI (K); MluI (M); Ncol (N); Sacl (C); Sall (S);         assays  Materials methods) performed indicated <br />  <br /> XbaI (X). solid bar restriction maps indicates              MATo! transformants MATa  tester strain. efficiency  <br /> position common XbaI-SalI fragment inserts.           mating expressed percentage diploid cells formed enlarged portion restriction map shows location                indicated transformant relative mating efficiency parental <br />  <br /> orientation coding region cloned gene (U).                      strain DS7 (set 100%). assays                        performed       duplicate    <br />                                                                               average        mating efficiency           reported   standard deviation. <br />                                                                                   averages             obtained            independent <br /> obvious gene cloned genetic complementa-                        determinations.   Wild type (WT; strain DS7),                            sexl-]   (sexi; strain <br /> tion sexl mutant remarkably similar yeast                 B34)         strain    carrying      disrupted    copy cloned gene <br />  <br /> gene recently identified Egel-Mitani et al. (1990)                     (YAP3:: HIS3, strain YBADl)                        transformed indicated <br />  <br /> called YAP3. YAP3 gene (Yeast Aspartyl Protease 3)                        plasmids       challenged mate. (B) Wild type (WT; strain DS7)                                                                                                                                                                            sexl-J  (sexi; strain B34) yeast transformed pS3C cloned ability partially suppress                   indicated multicopy centromere-based plasmids carrying cloned <br /> kex2 mutation present multicopy plasmid.                            gene (p9A242 pBA09, respectively) (YEP24 YCp5O, <br />   normal function cell                         respectively) challenged mate. (C) Yeast carrying disrupted <br />  <br /> investigated. Partial DNA sequencing confirmed                        copy cloned gene                (YAP3::     HIS3;    strain    YBADI)         <br />  <br />                                                                               transformed             pS3C     indicated           gap-repaired plasmids. <br /> genes identical (data shown).                                         Plasmids       pBA12-13 pBA14-15  Materials methods) <br />    Quantitative mating assays quite sensitive provide                 contain      Bglll Mlul deletion YAP3 gene respectively. <br />                                                                                                                <br />  good estimate relative secreted factor.                               assayed    relative      mating efficiency. <br />  <br /> Quantitative mating assays performed strain YBAD1 <br /> carrying disrupted copy cloned gene indicated                   6C).     expected                   gene allelic SEX], <br />  null mutant slightly competent mating                           mating efficiency                                regardless      host <br />  <br /> (167%) wild type (set 100%) sex]-]                   e. wild type                   sex]-] mutant)      plasmids <br /> mutant (67%) strain expressing plasmid paFF (Figure                      containing             deletions          repaired. conclude <br /> 6A). contrast, mating efficiency null mutant                    YAP3 gene acts                      multicopy suppressor sex]-] <br /> expressing plasmid pS3C greater 10-fold lower                        mutation. <br />  <br /> (7.3%) wild type strain similar obtained strain carrying sex]-] mutation (12.4%).                 Functional role YAP3 gene                                     prosomatostatin  cloned gene shown restore near wild                       processing <br /> type mating competence strain B34 present                        Deletion    YAP3 gene, encodes aspartyl <br />  <br /> multicopy plasmid (YEP24), unable                       protease,     profound effect mating competence <br />                                                                                                            <br />  <br /> expressed centromere-based plasmid (YCp5O) (Figure                     strain expressing pS3C. role proSRIF proteolytic <br />  <br /> 6B). provided indication gene cloned                  maturation investigated metabolic labelling <br />  allelic SEX], acts multicopy                        appropriate yeast strain transformed  <br /> suppressor sex]-] mutation. phenotype                      pS2B pSlID, followed specific immunoprecipita- <br />  sex]-] mutation observed cells,                     tion secreted SRIF species. Identical amounts  <br /> complementation tests strain carrying disrupted copy                 radiolabelled material secreted wild type mutant <br />                                                                                                                Tricine gels.- Strain YBAD1I YAP3 gene performed. circumvent                        yeastwere loaded SDS <br />  <br />  problem decided clone YAP3 locus                      (AYAP3) transformed pS2B secrete <br /> sex]-] mutant using &apos;gap repair&apos; technique (Orr-Weaver                    considerably mature SRIF-28 ( &lt;24 % estimated et al., 1981) test ability correct mating                      densitometric analysis) wild type strain (Figure 7A, <br />  <br /> deficiency strain YBAD1 (AYAP3) expressing pS3C.                           compare lanes                       2). supports           hypothesis  Centromere-based plasmids (YCp5O) containing                            YAP3 gene                  required           proteolytic processing BglH MluI deletion, covered                    monobasic              cleavage sites.            absence,                   clearly <br />  coding region YAP3 gene  Figure 5), used                     alternativeendoproteolytic activities generate  transform parental original sex]-] mutant               mature peptide. Km, reaction  <br />  <br />  <br />  yeast allow repair homologous recombination.                       siderably higher (&gt; 10-fold) single arginine residues <br />  resulting plasmids rescued transformation                  Lys Arg- Arg Arg-containing substrates, <br />  <br />  E. coli introduced strain YBAD 1 (AYAP3) (Figure                    previous studies specificity Kex.2 endoprotease <br />  <br />                                                                                                                                                                               289 <br />   Y.Bourbonnais et al. <br />  <br />  <br />                                  B <br />                                                                             RHRDLE]                                       ,C     proSRIF  <br />                    1 2                    1 2 34 <br />  <br />                                                                            GRMNLE          SUDSTNNLPPRE               RGC        proSR I <br />                                                      AA <br />  <br />  <br />  <br />  <br />                  OWb <br />                 Eo                                                         Fig. 8. Model proteolytic processing anglerfish <br />                                                                            somatostatin precursors yeast S.cerevisiae. amino acid <br />                                                                            sequence proSRIF -II, comprising processing sites <br />                                                                            (grey boxes), represented using single letter code. Filled                                                                            open arrows represent KEX2- YAP3-dependent cleavage, <br />                         C                                                  respectively. Larger arrows emphasize prevalence indicated <br />                                                                            gene proteolytic cleavage. <br />  <br />  <br />                               1 234                                         expected preproSRIF-II 63-125, previously <br />                                                                            described SRIF-28 processing intermediate release <br />                                                                             requires cleavage Lys Arg doublet (Noe et al., 1986). <br />                                                                            decrease production SRIF-28 yield                                                                            concomitant increase precursor forms. Accumulation <br />                                                                            secreted, hyperglycosylated proSRIF-II hybrid <br />                                                                            precursor easily detected SDS -Tricine gels. <br />                                                                            absence Kex2 Yap3 leads                                                                            secretion unprocessed precursor forms best demon- <br />                                                                            strated subjecting immunoreactive material  Fig. 7. Biosynthesis prepro ca-factor-proSRIF -II            endoglycosidase H treatment (Figure 7B, lanes 3 4).  chimeras wild type, single (AYAP3) double (AYAP3 AKEX2) <br />  mutant yeast. Yeast labelled [35S]cysteine                                                                            deglycosylated precursor (16 kDa band) detected <br />  radioactive material specifically immunoprecipitated anti-       culture media wild type double <br />  SRIF-28  B) anti-SRIF-14 (C) antisera resolved          mutant yeast.  relative abundance respect <br />  high resolution SDS-Tricine gel described Figure 3.               mature SRIF-28 indicated minor species <br />  Secreted immunoreactive material wild type (strain DS7; lane 1)      secreted wild type strain predominant secreted <br />  LYAP3 (strain YBADI; lane 2) yeast transformed plasmid <br />  pS2B. (B) Secreted immunoreactive material wild type (strain         peptide AYAP3 AKEX2 mutant. <br />  DS7; lanes 3) AYAP3AKEX2 (strain YBAD2; lanes 2                 shown  expression proSRIF yeast leads <br />  4) yeast transformed plasmid pS2B. Samples lanes 3 4          secretion major peptides (SRIF-28, Lys- <br />  incubated endoglycosidase H prior electrophoresis.           SRIF-14 SRIF-14) genetic evidence suggested Kex2 <br />  (C) Secreted immunoreactive material wild type (strains DS7      directly involved release Lys-SRIF-14  W303-lb; lanes 1 3, respectively), AYAP3 (strain YBADI; lane 2) <br />  AYAP3 AKEX2 (strain YBAD2; lane 4) yeast transformed             cleavage middle pair basic residues (Arg Lys). <br />  plasmid pSID. Note identical radioactivity          Consistent finding, expression pSlD  loaded pair lanes. Asterisks indicate migration position   absence functional YAP3 gene did affect conversion <br />  major glycosylated precursor form chimeras. Upward <br />  pointing arrowheads indicate position deglycosylated <br />                                                                            precursor Lys-SRIF-14 (Figure 7C, lanes 1  proSRIF-II fusion protein (16 kDa). Arrowheads left indicate       2).  nearly completely prevented accumulation <br />  position prestained protein standards described Figure   SRIF-28 significantly reduced secreted <br />  3.                                                                        SRIF-14. accompanied appearance                                                                            diffuse band molecular mass (45 kDa) consistent <br /> indicated enzyme recognizes cleaves,                hyperglycosylation, contributes 30-50 kDa vitro, single arginine residue (Brenner Fuller,               chain (Julius et al., 1984) proSRIF hybrid precursor <br /> 1992). test idea Kex2  absence                 (17 kDa). mature SRIF-14 secreted kex2 <br /> YAP3, excise mature SRIF-28 hybrid precursor                  (Figure 3, lane 2) AYAP3 mutant yeast <br /> double mutant strain YBAD2 (AYAP3 AKEX2)                               (Figure 7C, lane 2) suggested gene products constructed. Expression plasmid pS2B strain                     involved release peptide. Expression pS <span id='am-25' about='oboInOwl:id' typeof='owl:Thing'>ID</span> <br /> resulted decrease (&lt;105% wild type                     double mutant strain YBAD2 confirmed production SRIF-28) proSRIF-I1 conversion                         hypothesis (Figure 7C, lanes 3 4). mutant yeast <br /> SRIF-28. totally prevented appearance 6 kDa               proteolytic maturation fusion protein peptide, consistently observed culture                    completely abolished intact hyperglycosylated <br /> medium wild type YAP3 yeast. nature                  precursor forms (45 kDa peptide) recovered remains confirmed, mobility SDS -Tricine gels                 culture medium. <br />  <br />  290 <br />                                                                            S.cerevisiae mutants defective somatostatin expression <br />  <br />  Discussion                                                          corresponding SEX genes undoubtedly provide <br />  main objective define molecular basis                                                                     new    insights yeast cellular machinery involved                                                                      precursor processing. work currently progress. <br />  differential processing anglerfish SRIF -II            Similarly, rapid sensitive biological assay used  precursors using yeast S. cerevisiae model          isolation sex mutants, combined &apos;cassette&apos;- <br />  step goal recently expressed              directed mutagenesis, provide powerful tool proSRIF-II yeast, fusion protein, identified           investigating role amino acid residues upstream <br /> novel endoproteolytic activity capable generating vivo         processing sites differential maturation mature SRIF-28 (Bourbonnais et al., 1991). present           anglerfish SRIF precursors. particular study wanted investigate biosynthesis              despite possessing mono- dibasic cleavage <br /> analogous anglerfish somatostatin precursor (SRIF              sites appear differentially processed identical <br /> characterize gene encoding novel yeast endoprotease.        cellular environment  . recent cloning mammalian dibasic processing                  Expression proSRIF -11 fusion proteins single <br /> enzymes PC2, PC1/PC3 PC4 (Seidah et al., 1990;                  double mutant yeast enabled propose model Smeekens Steiner, 1990; Smeekens et al., 1991;                  role KEX2 YAP3 genes proteolytic <br /> Nakayama et al., 1992) based sequence similarity           maturation somatostatin precursors yeast (Figure 8). yeast Kex2 enzyme. key finding cloning            hybrid precursors efficient cleavage single mammalian Kex2 homologues observation              arginine residue (Arg-15) requires functional YAP3gene. Kex2 endoprotease, required               absence dramatic production <br /> maturation yeast factor (Julius et al., 1984),       SRIF-28 proSRIF  clearly enzymatic <br /> faithfully recognize cleave analogous dibasic processing        activities  partially, process SRIF precursors <br /> sites  e. Lys Arg Arg Arg) heterologous precursors         generate peptide. Consistent previous vitro <br /> (Thomas et al., 1988; Germain et al., 1990; Zollinger et al.,       itt vivo studies substrate specificity Kex2 <br /> 1990). cloning mammalian monobasic processing            endoprotease (Brenner Fuller, 1992; Zhu et al., 1992), <br /> enzyme benefit isolation gene       results implicate KEX2 gene alternative,  cognate enzyme yeast.                                 efficient processing. reported pancreatic islets <br />     shown expression anglerfish proSRIF           (Noe Spiess, 1983), proteolytic maturation proSRIF-IH <br />  yeast leads secretion mature SRIF-28, mature          yeast restricted single arginine residue  study; <br />  SRIF-14 extended form SRIF-14 (Lys-SRIF-14).             Bourbonnais et al., 1991). contrast, proSRIF  Accumulation SRIF-28 culture medium indicates            processed, presumably Arg -2 Lys -1 site, generate <br />  processing yeast internal Arg -2 Lys -1 basic pair       Lys-SRIF-14 mature SRIF-14. basic pair  does proceed completion. contrast             present precursors supports hypothesis <br />  synthesis prohormone pancreatic islets,        (Bourbonnais et al., 1991) lack proSRIF-H cleavage <br />  exclusively yield mature SRIF-14. higher eukaryotes,            Arg -2 Lys -1 results specific conformation <br />   secretion peptide hormones neuro-              rendering site inaccessible endoproteolytic processing. <br />  peptides regulated, yeast rapidly constitu-         dibasic-specific Kex2 endoprotease essential  tively secretes proteins destined extracellular        release Lys-SRIF-14, suggesting endoproteolysis <br />  medium. limited contact time pro-             occurs basic doublet. favor idea, <br />  hormone convertase substrate yeast        previous studies specificity purified Kex2 <br /> account apparent discrepancy observed yeast         endoprotease peptidyl-MCA substrates revealed pancreatic islets. Expression proSRIF               P1 binding site enzyme highly specific Arg kex2 yeast mutant extended previous observation               Arg P1 position, substitution P2 <br /> (Bourbonnais et al., 1991) provided genetic evidence            little effect overall catalytic rate (Brenner SRIF precursors processed mature SRIF-28           Fuller, 1992).  additional experiments required Kex2-independent cleavage. Genetic complementation             confirm hypothesis  . Kex2 enzyme yeast mutant (sexl-1), specifically affects cleavage     involved directly  e. cleavage Arg monobasic processing site (Arg -15), resulted         Lys doublet)     indirectly  e.     limiting aminopeptidase <br /> cloning gene encoding aspartyl protease. gene,         B-like activity remove extra lysine residue YAP3, recently cloned group               Lys-SRIF-14 processing intermediate) release (Egel-Mitani et al., 1990) based ability partially        mature SRIF-14. support hypothesis, amino- <br /> suppress phenotype kex2 null mutant, shown             peptidase B-like enzyme purified rat brain cortex act multicopy suppressor sex]-] mutation.         recently proposed required conversion unclear point primary defect sex]        processing intermediates mature SRIF-14 (Gluschankof <br /> mutant  null phenotype YAP3 gene           et al., 1987; Gomez et al., 1988). Finally data showed <br />     unlikely Sexl directly          YAP3 gene, predominantly involved involved proteolytic maturation proSRIFs.            processing monobasic cleavage site (Arg - 15), probably, involved function  e. proteolytic   contributes extent release mature SRIF- 14. <br /> activation zymogen) intracellular localization       consistent previous study proteolytic YAP3 gene product.  screening                maturation overexpressed pro factor-insulin <br /> procedure primarily designed facilitate cloning       chimera absence functional KEX2 gene <br /> gene encoding putative monobasic-specific endoprotease,         (Egel-Mitani et al., 1990). conditions 20% characterization class III mutants cloning       proteolytic processing, ascribed action Yap3, <br />  <br />                                                                                                                                  291 <br />  Y.Bourbonnais et al. <br />  <br /> occurred pro factor Lys Arg cleavage site.                    argued proteolytic maturation occurs intracellularly. <br /> proteolytic maturation anglerfish SRIF precursors yeast                presence mature SRIF peptides cell extracts <br /> implicates endoproteases distinct specificities                 prepared spheroplasts kex2-transformed yeast mono- dibasic processing sites. present model                    confirmed Yap3-dependent processing occurs proSRIF processing yeast based largely genetic                   secretory pathway periplasmic space  arguments. actual sites endoproteolytic cleavage need                 shown). addition, previous observation confirmed vitro incubation SRIF                 processing proSRIF-II completely blocked precursors purified endoproteases. instance,                expressed sec,18 mutant grown restrictive <br /> data exclude possibility Lys-SRIF-14                     temperature, indicated proteolytic maturation <br /> peptide generated Kex2 endoproteolytic                         post-ER event (Bourbonnais et al., 1991). results <br /> cleavage C-terminal Arg-4  e. Pro-5 Arg-4                     good keeping factor maturation putative monobasic site), instead Arg -2, followed                  Kex2 endoprotease  probably, takes place exoproteolytic cleavage aminopeptidase.                  late-Golgi compartment (Julius et al., 1984; Redding supported present finding implicating Kex2                  et al., 1991). Consistent physiological relevance <br /> release mature SRIF-28, recent                     Kex2 Yap3 proSRIF maturation, groups report Kex2-dependent processing yeast K1                      documented action distinct somatostatin convertases <br /> killer toxin Pro43 Arg44 site (Zhu et al., 1992).                     production SRIF-28 SRIF-14, respectively,    precise site action YAP3 gene                      higher eukaryotes (Gluschankof et al., 1987; Mackin product determined, preliminary studies                 Noe, 1987; Beinfeld et al., 1989). recently Mackin <br /> localization Yap3-dependent cleavage proSRIFs                      et al. (199 la, 1991b) purified apparent homogeneity support role precursor processing.                         candidate SRIF convertases anglerfish secretory <br /> accumulation significant various SRIF                       granules capable generating SRIF-28 -14 peptides <br /> peptides cell extracts pSlD-  study) pS2B-                    respectively, vitro physiological precursor <br /> transformants (Bourbonnais et al., 1991) virtual                      e. proSRIF-II  . Partial N-terminal sequencing absence precursor forms culture media strongly                     purified proteins revealed candidate SRIF-14  <br />  <br /> Table II. Plasmids used work <br />              Description                                                                                        Reference <br />  pS2B             2 itm based vector derived p82-5 (Green et al., 1986) expressing prepro                 Bourbonnais   et   al. (1991) <br />                   factor-proSRIF-II chimeric gene control factor promoter <br /> pSID              pS2B, expressing prepro factor-proSRIF chimeric gene                           study <br /> pS3C              pS2B, expressing prepro factor-proSRIF-II-mature factor chimeric gene          study <br /> pGS3C             centromere-based vector derived pJK6 (kindly provided J.Kurjan) prepro         study <br />                   factor-proSRIF-II-mature factor chimeric gene control GALI promoter <br /> paFF              pS2B, expressing truncated version prepro factor single copy    study <br />                   mature peptide retained <br /> YEP24G- <br /> B                 yeast genomic library constructed plasmid YEP24                                               Rose Broach (1991) <br /> pBA09             Plasmid YCp5O 5.8 kb XbaI (filled  fragment p9A242 cloned       study <br />                   Sall (filled  site <br /> pBA12             Derived YCp5O carrying 6.0 kb Sall p9A242, plasmid contained BglIl deletion     study <br />                   (3.8 kb) removed seven codons Yap3 protease -2.1 kb 3&apos; <br />                   flanking nucleotides repaired vivo homologous recombination parental <br />                   strain DS7 <br /> pBA 13            pBA 12, repair plasmid mutant strain B34                     study <br /> pBA14             Derived pBA09, plasmid contained MluI deletion (1.8 kb) removed 0.4 kb 5&apos;      study <br />                   flanking sequence 459 codons  569) YAP3 gene, repaired <br />                   vivo homologous recombination parental strain DS7 <br /> pBA15             pBA14, repair plasmid mutant strain B34                      study <br />  <br />  <br /> Table III. Yeast strains                               Genotype                                                                          Source reference <br /> 20-B12                             MATa pep4-3 trpl                                                                  Green et al. (1986) <br /> DS20                               MATa kex2-1 trpl ural                                                             Bourbonnais et al. (1991) <br /> DS7                                MATa MFaJ/a2 ::LEU2 ade2 his3 leu2 trpl ura3                                      J.Kurjan <br /> M200-6C                            MATa sstl sst2 adel ilv3 ura3                                                     Whiteway et al. (1988) <br /> B34                                MATa sexl-l MFaJ/a2 ::LEU2 ade2 his3 leu2 trpl ura3                               study <br /> YBAD1                              MATa YAP3::HIS3 MFaJ/a2 ::LEU2 ade2 his3 leu2 trpl ura3                           study <br /> M213-IOC                           MATa KEX2::HIS3 his3 leu2 trpl ura3                                               Dignard et al. (1991) <br /> W303-lb                            MATa ade2 his3 leu2 trpl ura3                                                     S.Henry <br /> YBAD2                              MATa YAP3::LEU2 KEX2::HIS3 his3 leu2 trpl ura3                                    study <br />  <br /> 292 <br />                                                                                                S.cerevisiae mutants defective somatostatin expression <br />  <br /> vertase homologous human mouse PC2,                                        Gene disruption <br /> recently identified member Kex2-like mammalian                                   disrupt yeast YAP3 locus YAP3 SphI-XbaI fragment homologues (Seidah et al., 1990; Smeekens Steiner,                                  subcloned plasmid pTZ18R. YAP3 coding region                                                                                         deleted (544/569 467/569 codons, respectively) replaced 1990). inhibition profile partial N-terminal                           HIS3 gene (1.8 kb BarnHI fragment plasmid pJJ215; Jones Prakash, <br /> sequencing protein indicated putative SRIF-28                                1990) LEU2 gene (2.0 kb BamHI-HindHI fragment pJJ283; <br /> convertase belongs aspartyl protease family.                               Jones Prakash, 1990). step disruptions (Rothstein, 1983) tempting speculate YAP3 gene                               HIS3 marker obtained use SphI-BamHI fragment <br />                                                                                         transformation yeast strains. disruption LEU2 marker <br /> yeast functional homologue endoprotease.                                obtained using HindIII-BamHI fragment. disruption possible functionally related higher                           confirmed Southern analysis. <br /> cell convertase involved cleavage monobasic sites, Yap3 endoprotease closely structurally related.                             Metabolic labelling yeast, immunoprecipitation protein analogous carboxypeptidase B-like activity                              analysis <br />                                                                                         Exponentially growing yeast labelled 250 ItCi [35S]cysteine <br /> required processing transient intermediates generated                            (Amersham) 30 min incubation 30&#194;&#176;C described previously endoproteolysis pair basic residues.                              Bourbonnais et al. (1988, 1991). Cell extracts media prepared carboxypeptidases thought fulfil                             described Bourbonnais et al. (1988, 1991) subjected function cell, sequence yeast KEX1 gene                               immunoprecipitation anti-SRIF-28 (R33 B5) anti-SRIF-14 (RSSI <br />                                                                                         B23) antibodies indicated. antisera generous gifts Drs Shields <br /> product (Dmochowska et al., 1987) does resemble                                Danoff (Albert Einstein College Medicine, Bronx, NY). Produc- mammalian carboxypeptidase E (Fricker, 1988). Future                                 tion specificity antibodies described previously experiments addressing physiological function                              Bourbonnais et al. (1991) Danoff Shields (1991). Anti-SRIF-28 <br /> endoproteases, information complete amJno acid                               specifically recognizes SRIF-28 precursor form, does cross- <br /> sequence anglerfish aspartyl protease, need-                            react SRIF-14 proSRIF  Anti-SRIF-14 efficiently <br />                                                                                         immunoprecipitates SRIF-14 precursor; recognizes SRIF-28 <br /> ed clarify exact relationship enzymes.                           proSRIF-II, albeit lower affinity. Radiolabelled proteins step goal currently investigating                         analyzed high resolution Tricine-SDS gels reversed-phase HPLC nature Yap3 substrate(s) yeast.                                           using C18 column (Vydac) described previously Bourbonnais et al. <br />                                                                                         (1991). Samples eluted column 0.1% trifluoroacetic <br />                                                                                         acid containing acetonitrile flow rate 1.5 ril/min according Materials methods                                                                   following gradient: 5% acetonitrile 5 min; 5-20% 1 min; <br />                                                                                         20-35% 30 min; 35-50% 15 min; 50-80% 1 min; isocratic- <br /> Plasmids strains                                                                    ally 80% 13 min. <br /> Plasmids used study brief description genes carry listed Table II. DNA modifications subcloning performed             Partial amino acid sequencing <br /> according standard procedures. Site-directed mutagenesis performed               Peptides sequenced purified HPLC submitted directly single-stranded DNA isolated dut-ung- bacteria using single                 Edman degradation <span id='am-6' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>gas</span> phase sequenator (Applied Biosystem model <br /> primer annealing technique according procedure developed Kunkel               470A) described earlier Bourbonnais et al. (1991). <br /> (1985). Restriction endonucleases New England Biolabs T4 <br /> DNA polymerase Pharmacia LKB Biotechnology  Yeast strains                  Endoglycosidase H treatment <br /> used study listed Table Ill. Yeast transformation carried            Treatment immunoprecipitates endoglycosidase H carried using lithium acetate procedure described Ito et al.(1983).                  described previously Bourbonnais et al. (1991). <br /> Yeast mutagenesis <br /> Strain DS7, expresses plasmid pS3C, grown overnight plated                Acknowledgements selective medium. dishes immediately exposed short- <br /> wave UV light 35 s, wrapped aluminum foil placed 22&#194;&#176;C                 grateful Susan Henry, Janet Kurjan Malcolm Whiteway 3 days allow formation colonies. Control plates indicated treatment          kindly provided yeast strains, Mark Rose generous gift yeast <br /> resulted 90% killing. Colonies survived UV exposure               genomic library Dennis Shields Ann Danoff supplied screened production bioactive factor described                        SRIF antibodies; France Dumas amino acid sequencing                                                                                         labelled peptides Caroline Kunze precious technical assistance <br /> Halo quantitative mating assays                                                     early work; Malcolm Whiteway Thierry Vernet Halo assays performed strain M200-6C, carries mutant                critical reading manuscript. NRCC publication  33655. <br /> alleles SSTJ SS72 genes supersensitive factor. <br /> Following overnight incubation 30&#194;&#176;C YPD medium, 5 ll M200-6C mixed 10 ml YPD medium containing 0.7 % agar,                      References prewarmed 42&apos;C, overlaid YPD plates. Colonies tested spotted directly solidified agar            Beinfeld,M.C., Bourdais,J., Kuks,P., Morel  Cohen,P. (1989) J. <br /> plates incubated overnight 30&apos;C. mating assay, haploid                    Biol. Chem., 264, 4460-4465. <br /> yeast grown overnight YPD medium (MATa tester strain)                  Bourbonnais,Y., Bolin,D. Shields,D. (1988) J. Biol. Chem., 263, appropriate SC medium (AATa strains). day cells diluted                15342-15347. <br /> 1/10 SC medium incubation resumed additional 2 h                  Bourbonnais,Y., Danoff , Thomas,D.Y. Shields,D. (1991) J. Biol. 30&apos;C. MATa strains mixed tester strain (MATa)                   Chem., 266, 13203 - 13209. following ratio: 100 Al diluted MATa cells, 50 ,l MATa             Brenner,C. Fuller,R.S. (1992) Proc. Natl. Acad. Sci. USA, 89, <br /> cells (undiluted overnight culture) 100 A1 YPD medium. mixture                 922-926. gently filtered sterile nitrocellulose filters (Sartorius), filters   Danoff  Shields,D. (1991) J. Biol. Chem., 266, 10004-10010.  colonies  YPD plates incubated                 Devi,L. (1991) FEBS Lett., 280, 189-194. <br /> 30&apos;C 4-6 h. Following incubation cells gently washed                   Dignard,D., Whiteway,M., Germain,D., Tessier,D. Thomas,D.Y. filters 300 Al selective medium appropriate dilutions                (1991) Mol. Gen. Genet., 227, 127-136. <br /> plated SC plates allowing diploid cells grow prototroph selection.           Dmochowska , Dignard,D., Henning,D., Thomas,D.Y. Bussey,H. <br /> Parallel nitrocellulose filters containing YPD medium cells                 (1987) Cell, 50, 573-584. plated YPD plates following appropriate dilutions. percentage             Douglass,J., Civelli,O. Herbert,E. (1984) Annu. Rev. Biochem., 263, mating determined according following equation:                        665-715. <br /> colonies SC plate  colonies YPD plate) x 100. Alternatively,                 Egel-Mitani,M., Flygenring,H.P. Hansen,M.T. (1990) Yeast, 6, <br /> semi-quantitative mating assay performed directly replicating                   127-137. <br /> nitrocellulose filters SC plates.                                                  Fricker,L.D. (1988) Annu. Rev. Physiol., 50, 309-321. <br />                                                                                                                                                                293 <br />  Y.Bourbonnais et al. <br />  <br /> Germain,D. Zollinger,L., Racine,C., Gossard,F., Dignard,D., <br />    Thomas,D.Y., Crine,P. Boileau,G. (1990) Mol. Endocrinol., 4, <br />     1572-1579. <br /> Gluschankof,P., Gomez,S., Morel  Cohen,P. (1987) J. Biol. Chem., <br />    262, 9615-9620. <br /> Gomez,S., Gluschankof,P., Lepage  Cohen,P. (1988) Proc. Natl. <br />    Acad. Sci. USA, 85, 5468-5472. <br /> Green,R., Schaber,M.D., Shields,D. Kramer,R. (1986) J. Biol. Chem., <br />    261, 7558-7565. <br /> Ito,H., Fukada,Y., Murata,K. Kimura  (1983) J. Bacteriol., 153, <br />     163-168. <br /> Jones,J.S. Prakash,L. (1990) Yeast, 6, 363-366. <br /> Julius,D., Brake , Blair,L., Kunisawa,R. Thomer,J. (1984) Cell, <br />    37, 1075-1089. <br /> Kunkel,T  (1985) Proc. Natl. Acad. Sci. USA, 82, 488-492. <br /> Mackin,R.B. Noe,B.D. (1987) J. Biol. Chem., 262, 6453-6456. <br /> Mackin,R.B., Noe,B.D. Spiess,J. (1991a) Endocrinology, 129, <br />    1951-1957. <br /> Mackin,R.B., Noe,B.D. Spiess,J. (1991b) Endocrinology, 129, <br />    2263-2265. <br /> McDonald,J.K., Greiner,F., Bauer,G.E., Elde,R.P. Noe,B.D. (1987) <br />    J. Histochem. Cytochem., 35, 155-162. <br /> Nakayama,K., Kim,W.-S., Torii,S., Hosaka,M., Nakagawa,T., Ikemizu,J., <br />    Baba,T. Murakami,K. (1992) J. Biol. Chem., 267, 5897-5900. <br /> Noe,B.D. Spiess,J. (1983) J. Biol. Chem., 258, 1121-1128. <br /> Noe,B.D., Andrews,P.C., Dixon,J.E. Spiess,J. (1986) J. Cell Biol., <br />    103, 1205-1211. <br /> Orr-Weaver,T.C., Szostak,J. Rothstein,R.J. (1981) Proc. Natl. Acad. <br />    Sci. USA, 78, 6354-6358. <br /> Patel,Y.C., Wheatley,T. Ning,C. (1981) Endocrinology, 109, <br />    1943-1949. <br /> Redding,K., Holcomb,C. Fuller,R.S. (1991) J. Cell Biol., 113, <br />    527-538. <br /> Rose,M.D. Broach,J.R. (1991) Methods Enzymol., 194, 195-230. <br /> Rothstein,R.J. (1983) Methods Enzymol., 101, 202-211. <br /> Schwartz,T.W. (1986) FEBS Lett., 200, 1-10. <br /> Seidah,N.G., Gaspar,L., Mion,P., Marcinkiewicz,M., Mbikay,M.    Chretien,M. (1990) DNA Cell Biol., 9, 415-424. <br /> Smeekens,S.P. Steiner,D.F. (1990) J. Biol. Chem., 265, 2997-3000. <br /> Smeeke <br /> </body></html>