<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Plant Cell, Vol. 16, 1091&#226;&#8364;&#8220;1104, 2004, www.plantcell.org &#194;&#170; 2004 American Society Plant Biologists <br />  <br />  <br />  <br />  <br /> ATR Regulates G2-Phase Cell-Cycle Checkpoint Arabidopsis thaliana <br />  <br /> Kevin Culligan 1 Alain Tissier,b,2 Anne Britta Sectionof Plant Biology, University California, Davis, California 95616 <br /> b Commissariat   l&#226;&#8364;&#8482;Energie Atomique Cadarache, Direction des Sciences du Vivant partement d&#226;&#8364;&#8482;Ecophysiologie Ve&#194;&#180;ge&#194;&#180;tale et Microbiologie, Laboratoire Radiobiologie Ve&#194;&#180;ge&#194;&#180;tale, 13108 St. Paul-lez-Durance Cedex, France <br />  <br />  <br />  <br /> Ataxia telangiectasia-mutated Rad3-related (ATR) plays central role cell-cycle regulation, transmitting DNA damage <br /> signals downstream effectors cell-cycle progression. animals, ATR essential gene.  Arabidopsis (Arabidopsis thaliana) atr&#195;&#191;/&#195;&#191; mutants viable, fertile, phenotypically wild-type absence exogenous DNA damaging agents exhibit altered expression AtRNR1 (ribonucleotide reductase large subunit) alteration damage-induced cell-cycle checkpoints. atr mutants hypersensitive hydroxyurea (HU), <br /> aphidicolin, UV-B light mildly sensitive g-radiation. G2 arrest observed response g-irradiation wild-type atr plants, albeit slightly different kinetics, suggesting ATR plays secondary role response double-strand breaks. G2 arrest observed wild-type plants response aphidicolin defective atr <br /> mutants, resulting compaction nuclei subsequent cell death. contrast, HU-treated wild-type atr plants <br /> arrested G1 showed obvious signs cell death. propose  plants, HU invokes novel checkpoint <br /> responsive low levels deoxynucleotide triphosphates. results demonstrate important role cell-cycle <br /> checkpoints ability plant cells sense cope problems associated DNA replication. <br />  <br />  <br />  <br /> INTRODUCTION                                                                   (RNR), catalyzes final reduction step production <br />                                                                                dNTPs. Genetic studies Saccharomyces cerevisiae suggest detection DNA damage important aspect damage                   elevation dNTP levels  transcriptional resistance. Cells routinely express DNA repair                      posttranslational regulation RNR) response DNA damage <br /> mechanisms, instead upregulate functions required,                result efficient translesion DNA synthesis, enabling <br /> concomitantly slowing cell cycle permit time repair                 cell complete replication spite presence S-phase  replication past damaged sites                       persisting lesions (Tanaka et al., 2000; Chabes et al., 2003; Yao <br /> mutagenic) M-phase  point double-strand                  et al., 2003). <br /> breaks lesions daughter strand gaps far                         proteins sense damaged single-stranded DNAs problematic repair). Damage generally detected                   trigger elaborate responses extensively <br /> protein complexes form single-stranded DNA (Crowley                  characterized organisms ranging bacteria human Courcelle, 2002; Zou Elledge, 2003), created                  (Crowley Courcelle, 2002; Melo Toczyski, 2002). replication fork physically blocked DNA lesions                plants,  regulation DNA replication repair process DNA repair (e.g., homologous recombination,                     cell cycle response DNA damage beginning <span id='am-31' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-32' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> excision repair, mismatch repair). damage                 understood. <br /> sensors activate downstream checkpoint mechanisms                         Schizosaccharomyces pombe Rad3 homologs (ataxia <br /> prevent progression phase cell cycle              telangiectasia-mutated Rad3-related [ATR] ataxia upregulate DNA repair (Nyberg et al., 2002).                                telangiectasia-mutated [ATM] mammals, Mec1 S. cerevi- <br />    damage sensors involved regulation                  siae) essential regulators cell-cycle checkpoints, sensing <br /> deoxynucleotide <span id='am-26' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span>triphosphate</span> (dNTP) pools, turn                   DNA damage single-stranded DNA activating <br /> influence mode DNA replication. primary targets              downstream effectors cell-cycle progression DNA repair. dNTP pool regulation enzyme ribonucleotide reductase                ATM activated primarily DNA double-strand breaks. ATM- <br />                                                                                deficient plants animals hypersensitive g-irradiation <br />                                                                                replication-blocking agents UV-B light, <br /> 1 correspondence addressed. E-mail kmculligan@ <br />                                                                                hydroxyurea (HU), aphidicolin (Abraham, 2001; Garcia et al., <br /> ucdavis.edu; fax 530-752-5410. <br /> 2 Current address: Librophyt SAS, Centre Cadarache, Ba      &#203;&#8224; timent 185,   2003; K. Culligan  Britt, unpublished data). Furthermore, <br /> 13108 St. Paul-lez-Durance Cedex, France.                                      Arabidopsis (Arabidopsis thaliana) ATM homolog required author responsible distribution materials integral           transcriptional induction repair genes response findings presented article accordance policy described     g-irradiation efficient progression meiosis Instructions Authors (www.plantcell.org)  Kevin M. Culligan <br /> (kmculligan@ucdavis.edu). <br />                                                                                (Garcia et al., 2003), role plant cell-cycle responses <br /> Article, publication <span id='am-6' about='oboInOwl:date' typeof='owl:Thing'>date</span>, citation information            DNA damage determined. animals, <br /> www.plantcell.org/cgi/doi/10.1105/tpc.018903.                                  activated ATM phoshorylates downstream components  1092      Plant Cell <br />  <br />  <br />  <br /> checkpoint pathways include p53, BRCA1, NBS1,                     Arabidopsis genome homologs ATR, using S. pombe <br /> CHK2, initiating G1-, S-, G2-phase arrest apoptosis.             Rad3 C-terminal domain query sequence,    contrast, ATR activated primarily agents block            candidate genes. Phylogenetic analysis corre- progression replication forks. ATR dominant-negative                sponding predicted length protein sequences indicated conditional knockout mammalian cell lines display hyper-               ortholog Target <span id='am-1' about='obo:IMR_0002818' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C07909" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="rapamycin" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0002818" datatype="xsd:string"></span>Rapamycin</span> (TOR), sensitivity UV-B light, HU, aphidicolin sen-           ortholog ATM, ortholog ATR (Figure 1A). <br /> sitive g-radiation.  animal cells, ATR thought            candidate genes fell broad categories. play generalized role response DNA damage                ATR-orthologous sequence (previously reported GenBank ATM. Activated ATR phosphorylates CHK1 initiate G2-phase                mRNA AtRad3 [AB040133], designate AtATR) <br /> arrest act points cell cycle,              revealed high similarity entire length ATR <br /> evidenced ATR-dependent phosphorylation BRCA1                        proteins, similarity ATM TOR sequences (Tibbetts et al., 2000; Melo Toczyski, 2002).              limited highly conserved middle C-terminal <br /> determination extent ATR&#226;&#8364;&#8482;s role genome main-             regions. confirmed GenBank sequence intron/exon <br /> tenance complicated fact ATR homozygous               boundaries isolating sequencing cDNAs AtATR <br /> knockouts die early embryogenesis (Brown Baltimore,                 ( 8.1 kb, ecotype Wassilewskija [Ws]) using RT-PCR, 2000; Klein et al., 2000); cre/lox-induced nulls survive           resulting sequence predicted 17 exons encoding protein rounds cell division, dominant-negative                2702 amino acids (Figure 1B). <br /> duction function cell lines yielded conflicting results <br /> (Cliby et al., 1998; Wright et al., 1998).  tractable genetic null allele ATR tolerated         Identification T-DNA Insertion-Mutation Alleles development higher organism particularly                 AtATR <br /> desirable. <br />                                                                            study function ATR Arabidopsis, searched     investigate effects ATR T-DNA insertion <br />                                                                            lines T-DNA insertions ATR. independent lines <br /> allele plant Arabidopsis. allele 6-kb insertion <br />                                                                            identified different collections termed atr-1 kinase domain presumably null.                                                                            (Arabidopsis Knockout Facility, ecotype Ws), atr-2 (SIGnAL data- mutant plant (atr) viable developmentally normal                                                                            base, ecotype Columbia), atr-3 (FLAG database, ecotype absence exogenous DNA-damaging treatments                                                                            Ws). atr-1 atr-2 contain insertions middle ATR, hypersensitive replication-blocking agents UV-B light, <br />                                                                            exons 7 10, respectively (Figure 1B). atr-3 contains single, <br /> HU, aphidicolin.  AtATR does appear act                                                                             6-kb T-DNA insertion highly conserved C-terminal <br /> functional homolog mammalian counterpart.                                                                             kinase (PI3Kc) domain (Figures 1B 1D), causes 30-bp <br /> Arabidopsis atr mutants differ mammalian mutants                                                                            deletion adjacent catalytic Asp required kinase function. terms viability degree sensitivity <br />                                                                            RT-PCR analysis C-terminal region ATR confirms g-radiation. <br />                                                                            transcript absent atr-3 (Figure 7).  believe <br />    course experiments, observed <br />                                                                            atr-3 likely null mutation focused major- differential response HU versus aphidicolin wild-type <br />                                                                            ity characterization line (termed atr . <br /> cells, HU primarily inducing G1 arrest, aphidicolin <br />                                                                            Nonetheless, lines exhibit similar recessive phenotype <br /> induced arrest G2. stands contrast G2 arrest <br />                                                                              cosegregates respective T-DNA <br /> induced agents yeast mammals  concen- <br />                                                                            insertions ATR. <br /> trations allow S-phase progression) suggests plants possess novel G1 checkpoint response low dNTP <br /> levels. G2 arrest response aphidicolin abrogated          AtATR AtATM Required Normal <br /> atr mutants, suggesting ATR regulates G2 checkpoint               Somatic Development <br /> response replication blocks, does mammals.  <br /> programmed cell death observed atr mutants response              atr homozygous alleles phenotypically wild-type aphidicolin, HU, experimental conditions.             growth  root shoot) development (leaf flower propose model suggests HU aphidicolin               development, seed set viability) standard conditions. <br /> hypersensitivities atr mutants result defect       ATR essential normal somatic development G2 checkpoint response blocked replication forks.                       Arabidopsis. contrast, ATR essential gene animals; <br />                                                                            ATR knockout mice die early embryogenesis (Brown RESULTS                                                                    Baltimore, 2000; Klein et al., 2000), conditional <br />                                                                            knockout human cell lines divide times dying <br /> Identification ATR Ortholog Arabidopsis                          (Cortez et al., 2001; Brown Baltimore, 2003). ATR <br />                                                                            ATM paralogs known play partially overlapping roles <br /> ATR orthologs, including Mec1 S. cerevisiae Rad3 S.              animal cells (Abraham, 2001), possible ATM pombe, members large gene family eukaryotes               compensate ATR deficiency Arabidopsis. test encode Ser-Thr kinases, C-terminal catalytic domains                 hypothesis, established atr atm double mutants share similarity yeast mammalian <span id='am-7' about='obo:IMR_0001650' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-8' property="rdfs:label" content="inositol phospholipid" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001650" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span>phosphoinositide</span>                   obvious phenotypic differences  vegetative growth 3-kinases (Tibbetts Abraham, 2000). searched                    development) compared wild-type single-mutant <br />                                                                                                           AtATR Regulates G2 Checkpoint            1093 <br />  <br />  <br />  <br />  <br /> Figure 1. Phylogenetic Analysis, Structure, T-DNA Insertions AtATR. <br />  <br />   neighbor-joining tree Rad3-like protein sequences performed described (Culligan et al., 2000). Numbers branch represent number times branch 1000 bootstrap replicas. Arabidopsis sequences bold. DNA-PKcs (DNA&#226;&#8364;&#8220;protein kinase catalytic <br /> subunit)-like sequences identified Arabidopsis genome. <br /> (B) Overview AtATR gene, positions atr-1, atr-2, atr-3 T-DNA insertions, corresponding cDNA structure. Numbered gray boxes <br /> indicate exons AtATR; positions T-DNA insertions denoted triangles. cDNA structure   shows corresponding highly <br /> conserved domains ATM, ATR, TOR, FKBP12-rapamycin-associated protein sequences, including kinase domain denoted black <br /> rectangle (PI3Kc). cDNA sequence confirmed isolating sequencing independent cDNAs AtATR (ecotype Ws) using RT-PCR. resulting sequence predicts 17 exons encode protein 2702 amino acids. <br /> (C) Structure atr-3 T-DNA insertion. Capital letters denote T-DNA insertion sequence. Corresponding amino acids shown DNA <br /> sequence. left-border sequence encodes 10 new amino acids AtATR reading frame premature stop (boxed TGA). approximate size entire insertion 6 kb. <br /> (D) Partial alignment kinase domain atr-3 inserted: 1 denotes 30-bp deletion (equivalent 10 amino acids) caused insertion, 2 denotes position catalytic Asp required kinase function. <br />  1094     Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 2. Siliques Harvested Mature Wild-Type, atr, atm-1 (Ecotype Ws) Mutants. <br /> Wild-type, atr&#195;&#191;/&#195;&#191;, atr&#195;&#191;/&#195;&#191; atm1/&#195;&#191; average produce  50 seeds silique. atm plants, partially sterile, produce range silique <br /> sizes fewer (&lt;10 average) seeds. siliques double atr atm line produced seeds unable outcross males females, suggesting complete sterility. <br />  <br />  <br /> plants, indicating ATR ATM play essential role            wanted determine RNR transcription normal vegetative (somatic) growth.                                 regulated AtATR-dependent manner challenged    double mutants,  completely sterile (Figure            unchallenged plants. searched GenBank identified <br /> 2). atr mutant plants fertile produce normal gametes;              Arabidopsis homolog RNR1, large subunit RNR <br /> pollen staining viable spores revealed differences versus           (accession number AF092841). Transcription RNR1 wild type, siliques (normal seed set). atm           shown regulated cell-cycle dependent <br /> mutant plants,  partially sterile abundant          manner tobacco (Nicotiana tabacum) (Chaboute&#194;&#180; et al., 1998, <br /> chromosomal fragmentation meiosis, resulting                2002), transcription induced DNA damage plant <br /> duction viable pollen reduced seed set (Garcia et al.,              (M.E. Chaboute&#194;&#180;, personal communication). probed RNA gel <br /> 2003). suggest AtATM AtATR play partially redun-               blots AtRNR1 AtRNR1 transcript reduced <br /> dant roles meiosis Arabidopsis.                                  atr mutants,  40% versus wild type                                                                            standard conditions (Figure 3), suggesting ATR required <br /> AtATR Required Transcriptional Induction                support normal expression RNR1. Plants treated Large Subunit RNR, AtRNR1                                               chronic doses UV-B light, sufficient inhibit eliminate <br />                                                                            growth, display level RNR1 transcript similar budding yeast S. cerevisiae, Mec1  ATR ortholog)                observed standard (UV-free) conditions, atr <br /> regulates expression RNR transcriptional                plants exhibit reduced levels RNR1 transcript. Interestingly, <br /> posttranscriptional levels (Zhao et al., 2001). Null alleles mec1       plants treated acute doses UV-B light displayed lethal, lethality suppressed mutations genes         transient inhibition AtRNR1 transcript  2 h postirradia- negatively regulate RNR (Huang et al., 1998; Zhao et al.,             tion), followed return standard condition levels 6 h <br /> 1998), transcriptional repressor Crt1 (Huang et al.,           postirradiation. UV-induced suppression ATR  <br /> 1998) inhibitor RNR activity Sml1 (Zhao et al., 1998).           pendent,  overall level RNR1 transcript Derepression RNR transcription response DNA damage                reduced (Figure 3). dependent Mec1 downstream kinase Dun1,                    used HU  inhibitor RNR activity) DNA <br /> indirectly induces inhibitory phosphorylation Crt1 (Elledge          replication-blocking agent   characterize pheno- <br /> et al., 1992; Zhao Rothstein, 2002). Mec1 ATR               typic effects atr mutant, wanted determine play role normal cell-cycle specific regulation RNR       effects HU treatment RNR expression plants. <br /> transcription unchallenged cells remains unclear.                       Surprisingly, presence HU, RNR1 expression                                                                                                        AtATR Regulates G2 Checkpoint         1095 <br />  <br />  <br />  <br />                                                                                   determine role Arabidopsis ATR cell-cycle <br />                                                                                checkpoint regulator response g-irradiation, established <br />                                                                                homozygous wild-type atr mutant lines carrying cyclinB1 <br />                                                                                promoter coupled b-glucuronidase (PcyclinB1:GUS) reporter <br />                                                                                construct. construct harbors GUS gene fused mitotic <br />                                                                                destruction sequence cyclinB1 promoter (Colon-Carmona <br />                                                                                et al., 1999). GUS fusion gene expressed entry                                                                                G2  cyclinB1 promoter), fusion protein                                                                                degraded just anaphase mitosis. GUS-positive <br />                                                                                cells stain blue G2/early M, allowing visually monitor                                                                                progression cell cycle root meristems (Preuss Britt, <br />                                                                                2003). determine G2-specific arrest (versus post-G2 mitotic <br />                                                                                arrest), stained root tips 49,6-diamidino-2- <br />                                                                                phenylindole (DAPI) visualize mitotic nuclei. response                                                                                400 Gy g-radiation, atr mutants showed slightly altered <br />                                                                                cell-cycle response compared wild type; Figure 3. RNA Gel Blot Analysis Wild Type atr.                      wild-type atr meristematic cells arrest G2 (DAPI staining <br /> RNA samples prepared 7-d-old wild-type atr plants                revealed accumulation mitotic figures) differ slightly  seedlings) sown MS agar plates plus 1 mM HU, presence         timing response. instance, 8 h postirradiation, atr UV-B (chronic dose), left untreated (n.t.). acute doses UV-B      root tips accumulated G2 cells versus wild type, light, wild type atr grown 7 d MS agar plates          1 d postirradiation root tips accumulated G2 cells ver- <br /> irradiated UV-B light (10 kJ/m2) dark harvested 2 h    sus wild type (Figure 4B). suggests Arabidopsis <br /> 6 h UV irradiation. Approximately 30 mg (lanes 1 6) 25 mg (lanes <br />                                                                                ATR absolutely required G2 checkpoint response 7 10) total RNA sample blotted probed                                                                                acute doses g-radiation does influence rate induction putative AtRNR1 (GenBank AF092841) mRNA sequence. AtRNR1 <br /> mRNA sequence 2.65 kb. bands corresponding 4.4 2.4 kb <br />                                                                                persistence. RNA ladder (Ambion) shown left denoted arrows. ethidium bromide&#226;&#8364;&#8220;stained gel shown  , relative           Arabidopsis atr Mutants Hypersensitive rates expression (Rel. exp.) determined respect wild-type      Replication-Blocking Agents treatment control. experiment repeated independently <br /> isolated RNA samples similar results.                                hsATR, scMec1, spRad3 deficiencies associated <br />                                                                                hypersensitivity replication blocking agents. exam- wild type atr (Figure 3) levels similar            ple, yeast mec1 mutants fail undergo G2 arrest normally <br /> wild type standard conditions. suggests                  induced stalled replication instead proceed directly additional ATR-independent levels RNR1 regulation                    mitosis. Consequently, mutant cells accumulate single-strand <br /> Arabidopsis consistent previous studies showing               gaps double-strand breaks, leading cell death (Weinert <br /> HU induces RNR expression plants (Chaboute&#194;&#180; et al., 1998).                  et al., 1994). animals, failure arrest induces apoptosis <br />                                                                                (Brown Baltimore, 2000; Cortez et al., 2001). determine <br />                                                                                ATR required plant cell-cycle responses Arabidopsis atr Mutants Mildly Sensitive                                replication blocks, challenged wild-type homozygous atr <br /> Acute Doses g-Radiation Proficient                                      mutant plants replication-blocking agents HU (1 mM), g-Induced G2 Arrest                                                         aphidicolin (12 mg/mL), UV-B light. agents <br />                                                                                inhibit replication fundamentally different ways. HU inhibits <br /> Conditional mutants ATR/Mec1 mammalian cells S.                      RNR, depleting available dNTPs DNA polymerases. <br /> cerevisiae, Rad3 mutants S. pombe,                           Aphidicolin inhibitor replicative DNA polymerases d <br /> hypersensitive g-radiation (Jimenez et al., 1992; Cliby et al.,             e, UV-B light creates physical replication blocks (e.g., <br /> 1998; Wright et al., 1998). Mutants ATM/scTel1, including                   cyclobutane pyrimidine dimers) DNA. chose focus Arabidopsis, hypersensitive g-radiation                   analysis root meristems (Wiedemeier <br /> (Sedgwick Boder, 1991; Garcia et al., 2003). sensitivity              et al., 2002) HU aphidicolin inhibit root generally interpreted failure checkpoint responses               growth wild-type plants, suggesting agents double-strand breaks (Abraham, 2001) involve                      absorbed root. germinated wild-type homozy- failure induce DNA repair genes (Garcia et al., 2003).                 gous atr seeds control MS agar plates plates <br /> determine ATR plays role g-resistance Arabi-                   containing HU aphidicolin, selecting dose dopsis, challenged wild-type, atr, atm lines sublethal             agent mild perceptible effects growth doses g-radiation. contrast atm mutants, atr lines                  wild-type root. wild type atr grew similar <br /> exhibited mild sensitivity g-radiation. root length atr         rates control plates. plates containing 1 mM HU, wild- reduced  30% versus wild type 7 d seeds                  type root growth reduced  40%, atr root <br /> g-irradiated 200 Gray (Gy), atm root length                     growth reduced &gt;95% (Figures 5A 5D) produced <br /> reduced &gt;95% (Figure 4A).                                                   abnormally long root hairs densely clustered  1096      Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 4. Root Growth Cell-Cycle Arrest Phenotypes g-Irradiated Wild-Type atr Plants. <br />   Wild-type, atm, atr seeds g-irradiated dose 200 Gy. seeds immediately germinated MS agar plates, root growth measured 2 weeks. irradation controls grew described (Garcia et al., 2003) Figure 5D. <br /> (B) day-old wild-type atr seedlings containing PcyclinB1:GUS fusion construct  described text Figure 6) g-irradiated 400 Gy, individual wild-type atr plants harvested 8 h, 1 d, 2 d time points GUS staining (n &gt; 12). representative examples treatment shown. <br />  <br />  <br />  <br />  <br /> root tip.  lateral root growth initiated earlier              AtATR Regulates G2 Checkpoint Response atr HU plates compared control plates. shoot                   Replication Blocks <br /> reduced size produced anthocyanins (stress-induced <br /> pigments), indirect effect lack root                   Mec1 ATR required G2 checkpoint response growth. differences wild type atr                     replication blocks fungi mammals, respectively (Melo (40% versus &gt;95% growth inhibition, overproduction root                     Toczyski, 2002). determine ATR performs similar <br /> hairs lateral roots) observed germinated                      function plants, GUS DAPI stained homozygous wild- <br /> plates containing 12 mg/mL aphidicolin 5-d-old seed-                type atr mutant lines carrying PcyclinB1:GUS reporter <br /> lings transferred plates containing HU aphidicolin,                 construct (described  growth HU aphidicolin. <br /> starting  2 d transfer (Figures 5C 5D). Furthermore,                 shown Figure 6, wild type atr displayed typical <br /> chronic doses UV-B light similar phenotypic effect                    patterns PcyclinB1:GUS expression control plates ( 10 20 atr roots compared wild type (Figure 5B);                      blue cells root tip), representing fraction G2/early <br /> mutant retarded  lesser extent dose)                  M cells present normal growth. contrast, meri- root growth formed root hairs tip root.                  stematic cells aphidicolin treated wild-type plants displayed  treatments block replication differ                substantial increase number PcyclinB1:GUS-expressing <br /> mechanism inhibition produced similar gross phenotypic                      cells density staining (Figure 6). Using <br /> effects atr roots.                                                          DAPI staining visualize mitotic nuclei, observed                                                                                                           AtATR Regulates G2 Checkpoint             1097 <br />  <br />  <br />  <br />  <br /> Figure 5. Phenotypes atr Challenged HU, Aphidicolin, UV-B Light. <br />   Wild-type atr seeds germinated MS plates MS plus 1 mM HU grown 3 weeks. <br /> (B) Wild-type atr seeds germinated MS plates grown 3 weeks presence absence UV-B light  Methods). <br /> (C) Wild-type atr seeds germinated MS plates grown 5 d transferred plates containing 12 mg/mL aphidicolin (Aph) 5 d. <br /> (D) day-old wild-type atr seedlings grown presence absence (n.t., treatment) HU aphidicolin (Aph).  make direct comparison effects HU versus aphidicolin, employed concentrations HU aphidicolin inhibited wild-type root <br /> growth  40% treatments, case, 1 mM HU 12 mg/mL aphidicolin. measured root growth (n &gt; 30) treatment 8 d transfer. Standard deviations shown error bars. <br />  <br />  <br />  <br /> significant aphidicolin-induced accumulation mitotic figures              occurred atr root meristems appeared slightly wild-type atr meristematic cells, ruling post-G2             accelerated. <br /> mitotic arrest likely cause PcyclinB1:GUS accumula-                   confirm PcyclinB1:GUS analysis, employed semi- <br /> tion. indicates dividing cells wild-type plants                quantitative RT-PCR, primers specific cyclinB1 gene <br /> experiencing aphidicolin-induced replicational stress spend                      does amplify PcyclinB1:GUS transcript),  greater fraction cell cycle G2 atr plants            treated, HU-, aphidicolin-treated wild-type atr root tips. defective G2 arrest (Figure 6).                                         shown Figure 7, cyclinB1 gene expression reduced  20% <br />    contrast aphidicolin response, HU did induce                 (relative untreated control root tips) wild type G2-phase arrest wild-type roots. fact, decline                   grown presence HU elevated nearly threefold fraction G2 cells observed day 4 6  measured                  presence aphidicolin, consistent PcyclinB1:GUS lack PcyclinB1:GUS staining; Figure 6). cells                 experimental conclusion HU induces G1- S-phase arrest <br /> presumably arresting G1- S-phase increase                   aphidicolin induces G2 arrest. Furthermore, aphidicolin- <br /> mitotic figures observed DAPI staining. response                   treated atr root tips display enhancement cyclinB1 <br />  1098      Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 6. GUS Staining Root Tips. day-old wild-type atr-3 seedlings (carrying PcyclinB1:GUS) transferred aphidicolin (Aph), HU, control (n.t.) plates, root growth measured day transfer 8 d (Figure 5D). Individual wild-type atr plants harvested time point GUS staining root tips. Wild-type atr grown 5 d MS plates transferred control plates plates containing 1 mM HU 12 mg/mL aphidicolin. experiments repeated times similar results;  representative pictures shown. GUS stained root tips 2, 4, 6 d transfer shown. 12 individual plants harvested stained treatment time point. representative <br /> examples shown treatment. <br />  <br />  <br /> transcript response aphidicolin, suggesting lack             organized cells enlarged, nuclei similar G2 arrest atr.                                                               appearance wild type (round slightly diffuse). cells <br />    distinguish G1- S-phase arrest wild-type HU-                root cap defined nucleus (Figure 8A). treated roots, employed RT-PCR analysis S-phase                       meristems atr grown aphidicolin disorganized, <br /> specific gene Histone H4. plants, expression Histone H4               appearance nuclei differed observed shown strongly induced entry S-phase                   wild type  conditions) atr grown HU. Starting maintained G2 (Reichheld et al., 1995; Menges                   day 4, meristematic nuclei near tip condensed, <br /> Murray, 2002). HU induces S-phase arrest (e.g., wild-type                 DAPI staining reduced relative atr grown HU root tips), predict concomitant increase S-phase&#226;&#8364;&#8220;                 wild type  conditions). day 6, cells meristematic <br /> specific gene expression. Likewise, G1-phase arrest                       zone aphidicolin-treated roots completely lost nuclei, <br /> predict concomitant decrease S-phase&#226;&#8364;&#8220;specific gene ex-                     leaving blue haze (Figures 8A 8B). indicates pression. shown Figure 7, Histone H4 expression reduced                atr mutant cells undergo nuclear degradation (cell death) HU-treated wild-type root tips elevated aphidicolin-                  treated aphidicolin, HU, dose levels agents <br /> treated wild-type root tips, suggesting G1- G2-phase arrest,                inhibit wild-type growth  40%. cell death <br /> respectively. atr mutant, Histone H4 expression elevated              seen higher doses HU (data shown), untreated root tips drastically reduced HU-            higher doses inhibited wild-type growth &gt;95%. suggests aphidicolin-treated root tips. suggests atr mutant,               (1) aphidicolin-induced AtATR-dependent G2 arrest plays phenotypically normal root growth, induces S-                  critical role protecting genome, (2) additional <br /> phase arrest (cyclinB1 induced) standard growth                    factors, G1 arrest induced HU, rescue atr <br /> conditions presence replication blocking agents,                  meristems massive genomic instability induced pro- <br /> arrests exits cell cycle G1.                                      gression M-phase absence complete replication. <br />    Altogether, data suggest independent check- <br /> points responsible arrest phenotypes observed                                                                                 DISCUSSION <br /> wild type versus atr: aphidicolin-induced G2 checkpoint AtATR dependent HU-induced G1 checkpoint.                              ATR plays central role cell&#226;&#8364;&#8482;s response DNA damage                                                                                 activating cell-cycle checkpoints allow time DNA repair. <br /> Aphidicolin, HU, Induces Cell Death atr                              shown Arabidopsis atr mutants viable                                                                                 normal standard growth conditions hypersensi- characterize different cellular responses HU              tive replication blocks mildly sensitive g-radiation. <br /> aphidicolin, DAPI stained wild-type atr root tips (time                  Furthermore, shown AtATR required points treatments  observe integrity                 G2-phase checkpoint response replication blocks nuclei. seen Figure 8A, wild-type roots organized                absolutely required G2-arrest response double- <br /> meristems round slightly diffuse nuclei,                      strand breaks (using acute doses g-radiation). differences <br /> retained morphology HU aphidicolin.  atr                    seen response HU, aphidicolin, g-irradiation suggest <br /> exhibited different cellular morphologies HU aphidi-                additional checkpoints play important roles cell- <br /> colin. cell files atr meristems grown HU dis-                    cycle cell-death phenotypes observed. <br />                                                                                                       AtATR Regulates G2 Checkpoint          1099 <br />  <br />  <br />  <br />                                                                                 cell-cycle arrest phenotypes atr comparison                                                                              wild type suggest mild g-sensitivity atr probably                                                                              absolute effect abrogated checkpoint G2; wild <br />                                                                              type atr display accumulation GUS meristem <br />                                                                              consistent G2 arrest, slightly different kinet- <br />                                                                              ics. know AtATM contributes                                                                              g-induced G2 checkpoint, possible AtATR AtATM <br />                                                                              play partially redundant roles response. case, AtATM <br />                                                                              partially compensate AtATR deficiency                                                                              cell cycle. mammalian cells, experiments employing <br />                                                                              conditional knockout (null) alleles ATR demonstrated                                                                              G2 arrest abrogated response g-radiation, indicating                                                                              ATR necessary G2 checkpoint response double- <br />                                                                              strand breaks (Brown Baltimore, 2003).                                                                               experiments used synchronized cells, irradiating cells S/ <br />                                                                              G2-phase assaying progression M-phase. <br />                                                                              acute doses g-radiation used create double- <br />                                                                              strand breaks phases cell cycle, possible ATR <br />                                                                              primarily activated specific phase cell cycle (e.g., <br />                                                                              S-phase) response double-strand breaks  review,                                                                              Abraham, 2001). scenario, fraction                                                                              meristematic cells  G1) DNA damage                                                                              invoke ATR-dependent G2 arrest, cells progressing <br />                                                                              S-phase. G1 cells acquire damage                                                                              activated G2 arrest ATR-independent mechanism, <br />                                                                              progress S-phase, arrest G2-phase.                                                                              interesting determine cell-cycle responses acute                                                                              chronic doses g-radiation atm atr atm double mutant <br /> Figure 7. Semiquantitative RT-PCR Analysis G2-Phase S-Phase&#226;&#8364;&#8220;          plants comparison atr. Nonetheless, preliminary results <br /> Specific Genes.                                                              suggest plants regulate G2-phase checkpoint Wild-type atr plants grown described Figure 6. Root tips      response g-radiation differently animals. <br /> (n &gt; 200) excised (1 2 mm root cap) wild-type atr plants 4-d transfer RNA extraction, cDNA preparation, RT-PCR analysis. microliters RT-PCR sample                                                                              AtATR Responses Replication Blocks <br /> loaded ethidium bromide&#226;&#8364;&#8220;stained agarose gel, representative <br /> gels shown. RT-PCR products 400 bp 850 bp           clear yeast animal studies ATR plays size. RT-PCR product eIF4A (eukaryotic initiation factor)      critical role G2 checkpoint response replication blocks. <br /> employed standard RT-PCR amplification. Numbers                                                                              studies employed HU aphidicolin gel band denote relative expression  normalized eIF4A)                                                                              model agents replication blocks. HU aphidicolin treatment wild-type untreated (n.t.) sample. Standard deviations replicates shown parentheses. Aph, aphidicolin.                                  wild-type yeast animals cells  concentrations allow S- <br />                                                                              phase progression) results G2 arrest. ATR-deficient mam- <br />                                                                              malian cells, progression mitosis  G2 delay) leads                                                                              caspase-independent chromosomal fragmentation, turn <br /> AtATR Plays Relatively Minor Role g-Resistance                          induces apoptotic response (Brown Baltimore, 2000).                                                                              shown atr plants hypersensitive growth- 400 Gy g-radiation, atr mutant similar wild type      inhibitory effects HU aphidicolin. G2 arrest  mea- <br /> root growth ability recover g-radiation treatments.          sured PcyclinB1:GUS) wild-type root meristems aphidicolin contrast, atm able recover g-radiation doses            concomitant lack G2 arrest atr root meristems indi- <br /> low 80 Gy (Garcia et al., 2003). suggests AtATR plays           cates ATR required G2-phase checkpoint response relatively minor role g-resistance versus AtATM plants.              replication blocks. lack wild-type G2 arrest response Human cell lines expressing dominant-negative kinase mutant                HU,  surprising. fundamental difference ATR used characterize relative contributions             HU aphidicolin blocking S-phase replication ATR ATM g-resistance (Cliby et al., 1998; Wright et al.,           HU-treated cells reduced dNTPs, implies G1 <br /> 1998). experiments clearly demonstrated ATR               arrest activated novel checkpoint responsive low dNTPs. <br /> involved g-resistance, independent studies employing                  Previous studies human cells demonstrated p53-dependent approaches yielded inconclusive results relative           G1 arrest response low ribonucleotide levels, contributions ATR ATM g-resistance (Jimenez et al.,                deoxynucleotide levels (Linke et al., 1996). response low <br /> 1992; Cliby et al., 1998; Wright et al., 1998).                              dNTPs help prevent plant cells initiating S-phase DNA <br />  1100      Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 8. DAPI-Stained Root Tips Wild-Type atr Grown Figure 6. <br />  <br />   Shown root tips day 4 day 6 transfer. <br /> (B) Close 6 d transfer aphidicolin wild-type (left) atr (right) DAPI-stained root tips. <br />                                                                                                    AtATR Regulates G2 Checkpoint        1101 <br />  <br />  <br />  <br /> replication absence sufficient dNTP pools.               AtATR-Dependent AtATR-Independent Regulation <br /> phosphate relatively immobile limiting nutrient soil (Abel     AtRNR1 Transcription <br /> et al., 2002), checkpoint represent plant-specific <br /> response helps direct root growth away phosphate-                regulation RNR yeast defined involves <br /> depleted regions soil, eliminating growth root meristems          transcriptional posttranslational control  e., inhibitory <br /> phosphate-depleted regions, promoting formation               protein&#226;&#8364;&#8220;protein interactions, modification protein localization, lateral root meristems root.                     allosteric regulation). instance, yeast RNR1 (large <br />     cell death observed atr response aphidicolin,         subunit) gene expressed vegetative growth, upregu- HU, suggests G1 (dNTP-dependent) checkpoint               lated late G1 early S, upregulated response <br /> rescue atr cell death.  atr mutants hypersen-             DNA damage (Elledge et al., 1992). expression patterns sitive effects HU growth root, fact         RNR genes characterized tobacco, appear paradoxical given hypothesis line              including RNR1-like gene (Chaboute&#194;&#180; et al., 1998, 2002). proficient G1 arrest induced agent. reason,        reduced level RNR1 transcript (Figure 3) observed atr remind reader HU-treated wild-type roots               versus wild type, absence DNA damaging agents growing, slightly reduced rate. words,            chronic UV-B light, suggests RNR1 transcriptionally <br /> cells progressing entire cell cycle,         maintained Arabidopsis AtATR-dependent spending relatively longer fraction cycle G1.          AtATR-independent mechanisms. analysis employing HU propose following model Arabidopsis cell-cycle                seedlings (Figure 3) indicate RNR1 checkpoint regulation response HU. HU treatment induces              regulated transcriptionally, independently AtATR, independent checkpoints Arabidopsis: G1                   feedback-type mechanism boosts RNR1 transcripts response low dNTPs (AtATR independent) G2                 dNTPs low. <br /> response replication blocks (AtATR dependent). HU-treated                  rapid reduction AtRNR1 transcript response acute <br /> wild-type atr meristematic cells arrest G1 response           doses UV-B wild-type atr plants suggests additional <br /> low dNTP pools, retarding entry S-phase. G1            mode transcriptional regulation AtRNR1 response DNA <br /> checkpoint eventually relieved dNTPs accumu-               damage independent AtATR. yeast, RNR upregulation <br /> late permit entry S-phase, fraction cells leak         response DNA damage rapid, occurring 30 min S-phase. resulting S-phase&#226;&#8364;&#8220;induced depletion              treatment (Elledge Davis, 1990; Elledge, 1993). dNTPs presence HU results requirement           surprising experimental conditions, acute dose <br /> AtATR-dependent G2 checkpoint activation. inability                 UV-B light caused rapid reduction AtRNR1 transcript <br /> atr activate G2 checkpoint results defective DNA                2 h UV-B treatment, followed return  <br /> replication meristematic failure.                                      treatment control levels 6 h UV-B treatment.     contrast, aphidicolin-treated atr cells invoke G1        response represent process inhibition AtRNR <br /> checkpoint normal levels dNTPs enter              expression leads low dNTP levels concomitant arrest S-phase delay. atr cells fail arrest G2             cell cycle DNA damage, possibly G1 <br /> proceed mitosis incompletely replicated genomes,                 checkpoint described  decrease RNR expression larger degree HU-treated atr cells.            expected inhibit potentially mutagenic <br /> results majority daughter G1 cells having intolerable       actions bypass polymerases, require high levels single-strand gaps double-strand breaks, leading             dNTPs activity (Johnson et al., 2001; Chabes et al., 2003). meristematic failure cell death.                                    plausible model mode regulation     cell death seen atr unexpected. Mutants DNA              immediately experiencing acute DNA damage, cells initially <br /> repair/maintenance genes Arabidopsis, DNA ligase IV,            try eliminate damage possible error-free <br /> ERCC1, telomerase, exhibit obvious cell death,             pathways. Initial downregulation dNTPs inhibit response treatment DNA damaging agents (Riha et al.,            inherently mutagenic bypass polymerases contribute 2001; Friesner Britt, 2003; Hefner et al., 2003). prevailing         arrest cell cycle. Return normal levels dNTPs hypothesis plants, p53 homolog, lack DNA             allows cell cycle progress enables cell utilize <br /> damage&#226;&#8364;&#8220;induced cell death (apoptosis). believe atr                  bypass polymerases needed complete replication. response aphidicolin example programmed cell             model suggestive SOS response Escherichia coli, <br /> death, indicated fact cells root tip die,        error-free repair mechanisms induced early, including normally nondividing cells (e.g., epidermal cells), sug-         error-prone translesion synthesis induced later (Goodman gesting signal cell death diffusible. Alternatively,   Woodgate, 2000; Rangarajan et al., 2002) response acute possible cells surrounding dying meristem           induction DNA damage. <br /> recruited reconstitute missing meristem, induction cell cycle induces death. case, death           AtATR Required Completion Meiosis simply direct result chromosomal fragmentation               Absence AtATM <br /> induced progression M-phase incompletely <br /> replicated genome. event, clear replication-             Given partial sterility atm mutant plants nonsterility induced checkpoints play important role protecting cells             atr, observation complete sterility atr atm double consequences replicational stress.                             mutant suggests ATR partially complement ATM <br />  1102      Plant Cell <br />  <br />  <br />  <br /> function meiosis. clear ATR            using aphidicolin, control HU plates contained equivalent distinct meiotic function ATR simply filling lack   (0.05%) DMSO allow direct comparison. <br /> ATM  vice versa), experiments suggest ATR present functional meiosis Arabidopsis.            UV-B g-Irradiation Arabidopsis mammalian cells, situ localization studies shown ATM ATR localized meiotic chromosomes                        chronic doses UV-B light, plants sown MS plates                                                                              grown 24-h light cycle cool-white lamps, supplemented (Keegan et al., 1996; Moens et al., 1999). studies using atr <br />                                                                              UV-B light using medium wave UV-B lamp (Spectronics, Westbury, NY; atm single double mutant plants allow better <br />                                                                              peak intensity 312 nm) filtered sheets cellulose acetate <br /> understand role ATR plant meiosis, possibly leading             (Golden State Plastics, Sacramento, CA; thickness 0.12 mm) eliminate <br /> additional insights ATR function animal meiosis.                 UV-C (&lt;280 nm) fraction emitted light. cellulose acetate                                                                              replaced 3 d. dose rate UV-B 2.5 J/s/m2. acute <br /> Conclusions                                                                  doses UV-B, 5-d-old seedlings irradiated UV trans- <br />                                                                              illuminator (Fisher, Houston, TX), peak output 305 nm, filtered shown plants encode ortholog ATR                 single cellulose acetate sheet. UV-B intensities measured role G2 checkpoint regulation similar respects          UV-B radiometer peak response 310 nm 50% response yeast mammalian counterparts. presence ATM                    280 340 nm (UV Products, San Gabriel, CA). g-radiation downstream effectors cell cycle DNA repair,             experiments, Arabidopsis seeds plants irradiated using Cs137 <br /> including BRCA1 (Lafarge Montane, 2003) putative                   source (Institute Toxicology Environmental Health, University CHK1 (S.B. Preuss, K. Culligan,  Britt, unpublished data)              California, Davis). Seeds imbibed water 48C 3 d, irradiated, <br />                                                                              immediately placed MS agar plates germination example, suggest G2 cell-cycle checkpoint <br />                                                                              placed growth chamber; plants grown MS plates  5 d), <br /> responses DNA damage replication blocks regulated <br />                                                                              irradiated, immediately returned growth chamber. similar fashion plants animals. say plant responses DNA damage replication blocks identical animals; significant                   Preparation RNA RNA Gel Blotting <br /> difference plants animals lack p53-                  Total RNA prepared 7-d-old seedlings using RNeasy plant <br /> dependent programmed cell death response persisting DNA                   mini kit (Qiagen, Valencia, CA). RNA blotted positively charged <br /> damage. Plants encode analogous pathway                       membranes (BriteStar-Plus; Ambion, Austin, TX) probed degradation hopelessly compromised cells, evidenced                 32P radioactively labeled AtRNR1 (GenBank AF092841) cDNA fragment. <br />  cell death observed atr, probably response           761-bp fragment generated PCR primers RNR1 (59-AAG- <br /> massive disruption DNA replication coupled uninhibited                 AATGGAGTGAGGAACTCTC-39) RNR3 (59-CTCAAGCAACATTAA- <br /> progression cell cycle. plants                GCATTAG-39). resulting bands visualized Storm 860 <br />                                                                              PhosphorImager quantified using ImageQuant software (Molecular die cancer, cell death response minor levels DNA <br />                                                                              Dynamics, Sunnyvale, CA). <br /> damage  occur spontaneously persist repair defective mutants) counteradaptive. insensitivity (versus animals) persisting DNA damage,                Semiquantitative RT-PCR <br /> plants ultimately prove robust model                                                                              Wild-type atr Arabidopsis plants sown MS agar plates, <br /> study fundamental processes cell-cycle regulation                 grown 5 d, transferred MS plates (n.t.), MS plates plus 1 mM DNA repair pathways eukaryotes  review, Hays,                 HU (1HU), MS plates plus 12 mg/mL aphidicolin (1Aph) 4 d 2002).                                                                       additional growth. plants verified respective pheno- <br />                                                                              types described  Approximately 1 2 mm root tips, &gt;200 <br /> METHODS                                                                      individual plants treatment, excised pooled                                                                              RNA extraction (RNeasy mini-prep; Qiagen). RNA samples                                                                              DNase treated (DNase set; Qiagen) quantified. produce cDNA <br /> Isolation Mutants <br />                                                                              RT-PCR, 2 mg total RNA condition described atr-1 allele (ecotype Ws) isolated Arabidopsis Knockout     reverse transcribed, employing random hexamer primers, Facility described (Krysan et al., 1999). atr-2 allele (ecotype       Superscript strand synthesis kit (Stratagene, La Jolla, CA) 40- <br /> Columbia; SALK_032841) identified using Salk SIGnAL Web site         mL reaction according manufacturers protocol. RT-PCR (http://signal.salk.edu), seeds obtained ABRC. atr-3    performed 50 mL reactions using Ex-Taq polymerase (Takara Bio, <br /> allele (ecotype Ws) identified using FLAG database (Genoplante;      Otsu, Shiga, Japan), 2 mL cDNA, following gene-specific <br /> http://genoplante-info.infobiogen.fr/ ).                                     primers: eIF4A-1 (59-CTCTCGCAATCTTCGCTCTTCTCTTT-39), eIF4A-5 <br />                                                                              (59-TCATAGATCTGGTCCTTGAAAC-39), CycB1-2 (59-GTCGCTTTCT- <br />                                                                              TCTTAGTAGCCTTCT-39), CycB1-6 (59-GGCCTCCATTCACTCTCAA- <br /> Growth Arabidopsis <br />                                                                              CAG-39), H4-5 (59-ATGTCTGGTCGTGGAAAGGGAG-39), H4-8 (59-AC- standard growth conditions, seeds sown 13 MS (GIBCO,             CAAATTGCGTGTTTCCATTG-39), cATR-1 (59-CAGCGCCCAAAGAAGA- <br /> Cleveland, OH) phytagel (Sigma, St. Louis, MO) agar, pH 5.8, grown       TCATTC-39), cATR-3 (59-GGCCCGCTGAGCATGTGGGTTC-39). cool-white lamps filtered Mylar intensity 100      primers  Histone H4) spanned intron-exon boundaries 150 mmol/m2/s 24-h day light cycle 218C. HU (Sigma; 500 mM         eliminate possibility amplifying contaminating genomic DNA. <br /> stock water) aphidicolin (ICN Biomedicals, Basingstoke, UK;           PCR-cycle parameters primer pairs 20 s 948C, 20 s 20 mg/mL stock DMSO) added MS plates              628C, 45 s 728C. Primer pairs tested log-linear <br /> centrations described text figure legends. experiments        amplification using 18, 21, 24, 27, 30 cycles PCR. optimized <br />                                                                                                          AtATR Regulates G2 Checkpoint              1103 <br />  <br />  <br />  <br /> number cycles primer pair  eIF4A, 23 cycles; cycB1, 27           (1999). Spatio-temporal analysis mitotic activity labile cyclin- <br /> cycles; Histone H4, 29 cycles; ATR, 27 cycles. PCR products               GUS fusion protein. Plant J. 20, 503&#226;&#8364;&#8220;508. <br /> ethidium bromide&#226;&#8364;&#8220;stained gels quantified using ImageQuant                 Cortez, D., Guntuku, S., Qin, J., Elledge, S.J. (2001). ATR software                                                                ATRIP: Partners checkpoint signaling. Science 294, 1713&#226;&#8364;&#8220;1716. <br />                                                                                Crowley, D.J., Courcelle, J. (2002). Answering  Coping <br /> GUS DAPI Staining                                                            DNA damage inopportune time. J. Biomed. <br />                                                                                  Biotechnol. 2, 66&#226;&#8364;&#8220;74. <br /> GUS staining performed described (Jefferson et al., 1987; Preuss <br />                                                                                Culligan, K.M., Meyer-Gauen, G., Lyons-Weiler, J., Hays, J.B. Britt, 2003). DAPI staining performed fixing roots solution <br />                                                                                  (2000). Evolutionary origin, diversification specialization of ethanol <span id='am-39' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span>acetic acid</span> (3:1). roots washed times                                                                                  eukaryotic MutS homolog mismatch repair proteins. Nucleic Acids <br /> PBS, pH 7.4, stained PBS containing 2.5 mg/mL DAPI (Molecular <br />                                                                                  Res. 28, 463&#226;&#8364;&#8220;471. <br /> Probes, Eugene,  20 min. roots washed PBS                                                                                Klein, , Muijtjens, M., van Os, R., Verhoeven, Y., Smit, B., Carr, <br /> times mounted slides 50% glycerol. Slides visualized                                                                                  , Lehmann, , Hoeijmakers, J. (2000). Targeted disruption Zeiss Axiophot microscope, photographs taken Zeiss <br />                                                                                  cell-cycle checkpoint gene ATR leads early embryonic lethality <br /> MC100 camera. DAPI staining visualized standard UV <br /> fluorescence filter set.                                                         mice. Curr. Biol. 10, 479&#226;&#8364;&#8220;482. <br />                                                                                Elledge, S., Zhou, , Allen, J.B. (1992). Ribonucleotide  <br />   Sequence data article deposited EMBL/             ductase: Regulation, regulation, regulation. Trends Biochem. Sci. 17, <br /> GenBank data libraries accession numbers AB040133                      119&#226;&#8364;&#8220;123. <br /> AF092841.                                                                      Elledge, S.J. (1993). DNA damage cell cycle regulation                                                                                  ribonucleotide reductase. Bioessays 15, 333&#226;&#8364;&#8220;339. <br />                                                                                Elledge, S.J., Davis, R.W. (1990). genes differentially <br />                                                                                  regulated cell cycle DNA-damaging agents encode <br /> ACKNOWLEDGMENTS                                                                  alternative regulatory subunits ribonucleotide reductase. Genes <br />                                                                                  Dev. 4, 740&#226;&#8364;&#8220;751. thank Ying Peng assistance isolating atr atm double mutant <br />                                                                                Friesner, J.D., Britt, B. (2003). Ku80- DNA ligase IV-deficient Cheryl Whistler, Joanna Friesner, Eli Hefner critical reading                                                                                  plants sensitive ionizing radiation defective T-DNA manuscript. thank Arabidopsis Knockout Facility <br />                                                                                  integration. Plant J. 34, 427&#226;&#8364;&#8220;440. <br /> (University Wisconsin, Madison) SIGnAL (Salk Institute)                                                                                Garcia, V., Bruchet, H., Camescasse, D., Fabienne, G., Bouchez, <br /> FLAG databases providing T-DNA knockout resources. work                                                                                  D.L., Tissier,  (2003). AtATM essential meiosis <br /> supported USDA postdoctoral fellowship (2001-01860) K.M.C. <br />                                                                                  somatic response DNA damage plants. Plant Cell 15, National Science Foundation Grant (MCB-9983142) B.B. <br />                                                                                  119&#226;&#8364;&#8220;132. <br />                                                                                Goodman, M.F., Woodgate, R. (2000). biochemical basis                                                                                  vivo regulation SOS-induced mutagenesis promoted Received October 30, 2003; accepted February 13, 2004. <br />                                                                                  Escherichia coli DNA polymerase V (UmuD&#226;&#8364;&#8482;2C). Cold Spring Harb. <br />                                                                                  Symp. Quant. Biol. 65, 31&#226;&#8364;&#8220;40. <br />                                                                                Hays, J. (2002). Arabidopsis thaliana, versatile model study <br /> REFERENCES <br />                                                                                  eukaryotic genome-maintenance functions. DNA Repair 1, <br /> Abel, S., Ticconi, C., Delatorre, C. (2002). Phosphate sensing            579&#226;&#8364;&#8220;600. <br />   higher plants. Physiol. Plant 115, 1&#226;&#8364;&#8220;8.                                      Hefner, E , Preuss, S.B., Britt, B. (2003). Arabidopsis mutants <br /> Abraham, R. (2001). Cell cycle checkpoint signaling ATM              sensitive gamma radiation include homolog human repair <br />   ATR kinases. Genes Dev. 15, 2177&#226;&#8364;&#8220;2196.                                     gene ERCC1. J. Exp. Bot. 54, 669&#226;&#8364;&#8220;680. <br /> Brown, E., Baltimore, D. (2003). Essential dispensable roles        Huang, M., Zhou, , Elledge, S. (1998). DNA replication   ATR cell cycle arrest genome maintenance. Genes Dev. 17,                damage checkpoint pathways induce transcription inhibition   615&#226;&#8364;&#8220;628.                                                                       Crt1 repressor. Cell 94, 595&#226;&#8364;&#8220;605. <br /> Brown, E.J., Baltimore, D. (2000). ATR disruption leads                 Jefferson, R., Kavanagh, T , Bevan, M.W. (1987). GUS fusions: <br />   chromosome fragmentation early embryonic lethality. Genes                  Beta-glucuronidase sensitive versatile gene fusion marker   Dev. 15, 397&#226;&#8364;&#8220;402.                                                              higher plants. EMBO J. 6, 3901&#226;&#8364;&#8220;3907. <br /> Chabes, , Clement, B., Domkin, V., Zhao, X., Rothstein, R.,              Jimenez, G., Yucel, J., Rowley, R., Subramani, S. (1992).   Thelander, L. (2003). Survival DNA damage yeast directly                 rad31 gene Schizosaccharomyces pombe involved multiple <br />   depends increased dNTP levels allowed relaxed feedback                   checkpoint functions DNA repair. Proc. Natl. Acad. Sci. USA <br />   inhibition ribonucleotide reductase. Cell 112, 391&#226;&#8364;&#8220;401.                     89, 4952&#226;&#8364;&#8220;4956. <br /> Chaboute&#194;&#180;, M., Combettes, B., Clement, B., Gigot, C., Philipps,            Johnson, R., Haracska, L., Prakash, S., Prakash, L. (2001). Role <br />   G. (1998). Molecular characterization tobacco ribonucleotide                DNA polymerase zeta bypass (6-4) TT photoproduct. <br />   reductase RNR1 RNR2. Plant Mol. Biol. 38, 797&#226;&#8364;&#8220;806.                         Mol. Cell. Biol. 21, 3558&#226;&#8364;&#8220;3563. <br /> Chaboute&#194;&#180;, M.E., Clement, B., Philipps, G. (2002). S phase             Keegan, K.S., et al. (1996). Atr Atm protein kinases associate <br />   meristem-specific expression tobacco RNR1b gene medi-                different sites meiotically pairing chromosomes. Genes <br />   ated E2F element located 59 leader sequence. J. Biol.             Dev. 10, 2423&#226;&#8364;&#8220;2437. <br />   Chem. 277, 17845&#226;&#8364;&#8220;17851.                                                      Krys <br /> </body></html>