<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 92, pp. 3127-3131, April 1995 <br /> Genetics <br />  <br />  <br /> Identification gene (SSU71/TFGJ) encoding largest <br /> subunit transcription factor TFIIF suppressor TFIIB mutation Saccharomyces cerevisiae <br />      (SSU71/transcription start-site selection) <br /> ZU-WEN SUN MICHAEL HAMPSEY <br /> Department Biochemistry Molecular Biology, Louisiana State University Medical Center, Shreveport, LA 71130-3932 <br /> Communicated Phillip  Sharp, Massachusetts Institute Technology, Cambridge, MA, December 15, 1994 <br />  <br /> ABSTRACT          Mutations Saccharomyces cerevisiae                         potential amphipathic helix flanked repeats (10, 13). <br /> SSU71 gene isolated suppressors transcription                          elements organized structurally distinct  <br /> factor TFIIB defect confers cold-sensitive growth                       mains defined compact, protease-resistant C-terminal <br /> defect downstream shift transcription start-site                           core, protease-susceptible N-terminal region (14, 15). <br /> selection cycl locus. ssu71-1 suppressor                        Mutational analysis demonstrated C-terminal core suppresses conditional phenotype restores                          necessary sufficient interaction TBP- <br /> normal pattern transcription initiation cycl. addition,                    promoter complex N-terminal region interacts ssu71-1 suppressor confers heat-sensitive phenotype                      pol II-TFIIF essential basal transcription dependent presence defective form TFIIB.                      initiation (14-18). N-terminal region includes Molecular genetic analysis cloned SSU71 gene                             phylogenetically conserved region TFIIB. se- <br /> demonstrated SSU71 single-copy essential gene                             quence lies immediately distal zinc finger required <br /> encoding highly charged protein molecular mass                          accurate start-site selection vivo (19). <br /> 82,194 daltons. Comparison deduced Ssu71 <span id='am-8' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span>                             TFIIF identified human [RAP30/RAP74 <br /> sequence protein data banks revealed significant                           (20)], rat [3/,y (21)], Drosophila [factor 5 (22)], S. cerevisiae <br /> similarity RAP74, larger subunit human general                        [factor g (23)]. Human TFIIF composed subunits transcription factor TFIIF.  Ssu71 identical                         apparent molecular masses 30 kDa 74 kDa (24), <br /> p15, component yeast TFIIF. Taken  data                         factor g composed subunits 105 kDa, 54 <br /> demonstrate functional interaction TFIIB                          kDa, 30 kDa (23). number biochemical properties <br />                                                                                     assigned TFIIF, including high affinity pol II <br /> large subunit TFIIF interaction affect                         (20, 22, 23, 25), stabilization preinitiation complex <br /> start-site selection vivo.                                                       (26-29), accurate start site recognition (24), ability                                                                                     stimulate rate elongation (22). individual contri- <br /> Accurate transcription initiation RNA polymerase II (pol                         butions RAP30 RAP74 subunits TFIIF functions <br /> II) requires auxiliary factors include TATA-binding                        understood. RAP30 implicated <br /> protein (TBP) transcription factors TFIIA, TFIIB, TFIIE,                        functions attributed TFIIF (5, 27,30-32), TFIIF, TFIIH (1-3). factors assemble defined                         role RAP74 remained obscure.  recent evi- <br /> order DNA template generate functional transcrip-                         dence indicates subunits required tion preinitiation complex vitro (4). minimal transcription                    functions attributed TFIIF (29, 33). composed pol II, TBP, TFIIB, TFIIF neces-                             using genetic approach identify components sary sufficient accurate basal transcription (5, 6),                        transcription preinitiation complex, emphasizing identifi- TFIIF dispensable transcription super-                       cation factors functionally interact TFIIB. coiled template (7). yeast Saccharomyces cerevisiae,                         recently reported replacement phylogenetically factors exist holoenzyme complex pol                     invariant Glu-62 lysine TFIIB dramatic effect <br /> II, consisting 12 pol II subunits Srb proteins,                      transcription start-site selection (19). addition, TFIIB, TFIIF, TFIIH (8) TFIIF, Sugl,                       Glu-62 -&gt; Lys replacement confers pronounced cold- Galil (9).                                                                      sensitive (Csm-) growth defect, phenotype associated <br />    functional role basal transcription factors                   defects assembly multisubunit complexes (34). mechanism transcription initiation subjects                           taken advantage Csm- phenotype isolate <br /> intense investigation. Mutations yeast genes encoding                        extragenic suppressors TFIIB defect. report <br /> TFIIB (SUA7) largest subunit pol II (RPB1/SUA8)                          mutations gene designated SSU71 * compensate dramatically affect transcription start-site selection,                         Csm- phenotype altered initiation pattern establishing role components                             conferred Glu-62 -- Lys replacement. SSU71 defining start sites vivo (10, 11).  TFIIB pol                      identified gene encoding p105 subunit factor g. <br /> II appear sole determinants accurate start-site                        information shows functional interaction                                                                                     TFIIB large subunit TFIIF shows selection vitro, replacement TFIIB pol II                      interaction affect start site selection vivo. S. cerevisiae counterparts Schizosaccharo- <br /> myces pombe sufficient switch start-site selection pattern characteristic S. pombe (12).                                                          MATERIALS METHODS <br />    Sequence analysis TFIIB revealed structural fea-                          Yeast Strains, Genetic Methods, Nomenclature. tures: zinc-finger motif near N terminus, imperfect                       yeast strains used study listed Table 1. Standard <br /> repeats C-terminal thirds protein,                                                                                     Abbreviations: pol II, RNA polymerase II; TBP, TATA-binding protein; publication costs article defrayed page charge          Csm-, cold-sensitive; Tsm-, heat-sensitive. <br /> payment. articie marked &quot;advertisement&quot;             sequence SSU71 reported paper deposited accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                        GenBank data base (accession  U14527). <br />                                                                              3127 <br />  3128        Genetics: Sun Hampsey                                                                   Proc. Natl. Acad Sci. USA 92      (1995) <br />                        Table 1. Yeast strains, relevant genotypes, phenotypes <br />                                                                                                          Phenotypet <br />                              Strain*                      Relevant genotype                   Cyt c, %          Csm         Tsm <br />                        T16                         cycl-5000                                     &lt;2              +           + <br />                        YDW546                      cycl-5000 sua7-1                               30             -           + <br />                        YMH71-9C                    cycl-5000 sua7-1                               30             -           + <br />                        YZS14                       cycl-5000 sua7-1 ssu7l-1                      &lt;2              + <br />                        YZS45                       cycl-5000 sua7-1 ssu7l-2                      &lt;2              + <br />                        YZS14-2B                    cycl-5000 sua7-1 ssu71-1                      &lt;2              + <br />                        YZS14-3B                    cycl-5000 sua7-1 ssu71-1                      &lt;2              + <br />                        YZS14 x YDW546              cycl-5000/cycl-5000                            30             -           + <br />                                                      sua7-1/sua7-J SSU71/ssu71-1 <br />                        *Strains T16, YDW546, YMH71-9C described previously (10, 11); described                         study. <br />                        tPhenotypes defined follows: Cyt c corresponds iso-1-cytochrome c content determined <br />                         low-temperature, cell spectroscopy (35) reported percentage iso-1-cytochrome <br />                         c content CYC1+ strain; Csm Tsm refer growth rich (yeast extract/peptone/dextrose) <br />                         medium 16&#194;&#176;C 37&#194;&#176;C, respectively. <br /> procedures used crossing strains, selecting diploids,                         restriction analysis 3.2-kb Kpn <span id='am-29' about='obo:IMR_0100178' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-30' property="oboInOwl:hasExactSynonym" content="sphingenine" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="sph" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0100178" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="sphingenine" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="Sphingoid" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00319" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Sphingenine" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Sphingenine" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="sphingosine" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Sphingoid" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="sph" datatype="xsd:string"></span>Sph</span> DNA fragment (Fig. <br /> inducing sporulation, dissecting tetrads (36). Standard                            1). Southern blot analysis confirmed 3.2-kb Kpn Sph <br /> yeast nomenclature used designate                                DNA fragment isolated intact yeast genome genotypes phenotypes. example, ssu71-1 denotes                               SSU71 single-copy gene (data shown). <br /> recessive allele wild-type SSU71 gene. symbols                                 DNA Sequence Analysis. sequences strands Csm- (cold-sensitive) Tsm- (heat-sensitive) refer dis-                          SSU71 determined dideoxy-terminator method <br /> tinctly impaired growth  growth) rich medium (yeast                           using M13 universal primer SSU71-specific primers extract/peptone/dextrose) 16&#194;&#176;C 37&#194;&#176;C, respectively. Cyt                          single-stranded DNA derived exodeoxyribonuclease III <br /> c Table 1 corresponds iso-1-cytochrome c content                          deletions SSU71. <br /> determined low-temperature, cell spectroscopy (35)                               Sequence Comparisons Homologies. DNA protein reported percentage iso-1-cytochrome c                              sequence comparisons performed using BLAST <br /> content CYCl+ strain.                                                             algorithm (40), accessed National Center Biotech- <br />    Isolation SSU71 Gene. SSU71 gene isolated                            nology Information E-mail service. YCp5O genomic library (37) complementation                               Determination Transcript Start Sites. Primer extension <br /> Tsm- phenotype conferred ssu71-1 allele using strain                            performed described previously, using total RNA YZS14 host. relationship complement-                            cycl-specific primer oIP-13 (10). Primer extension prod- <br /> ing DNA genetically defined ssu7l locus estab-                             ucts resolved 8% polyacrylamide DNA sequencing <br /> lished integration cloned DNA genome                             gel visualized autoradiography. <br /> strain YDW546 (sua7 SSU71 + leu2) homologous recombi- <br /> nation (38). tags chromosomal locus cloned                                                        RESULTS <br /> DNA LEU2 marker. resulting strain crossed strain YZS14-2B (sua7 ssu7l leu2), sporulated,                                  Isolation SSU71 Gene. sua7-1 allele strain <br /> dissected. LEU2 marker segregated opposite ssu7l                                YMH71-9C encodes TFIIB Glu-62 -&gt; Lys replacement (Tsm-) progeny. SSU71 gene delimited                              confers altered start-site selection severe Csm- <br />  <br />                                                                                                      1                    2                     3                    4                    5 <br />        E <br />            ~~~~~~~~~~~~~~~~~~~~~c <br />  <br />  n. cT <br />     0  0,c &apos;2CO                                                                , <br /> C2                                                             CIOU71 <br />  <br />                    TyrRSSS7 <br />  <br />  <br />                                                                                                               FIG. 1. Restriction map SSU71  <br />                                                                                                             gion. partial restriction map SSU71 <br />                                                                                                             locus shown. DNA fragments comple- <br />                                                                                                         ment Tsm- Csm+ phenotypes <br />                                                                                                             associated ssu71 suppressor sua7-1 <br />                                                                                                             background indicated black bars, <br />                                                                                                             open bars denote failure comple- <br />                                                                                                             ment. complementing 3.2-kb Kpn Sph                                                                                                             fragment vector YCplac33 designated <br />                                                                                                             pM435. TyrRS gene, encodes ty- <br />                                                                                                             rosine tRNA synthetase (39), lies immediately <br />                                                                                                         adjacent SSU71. <br />           Genetics: Sun   Hampsey                                                Proc. Natl. Acad Sci. USA 92 (1995)          3129 <br /> phenotype (19). Spontaneous Csm+ revertants strain                           MSRRNPPGSRNGGGPTNASPFIKRDRMRRN       30 <br />                                                                                 FLRMRMGQNGSNSSSPGVPNGDNSRGSLVK       60 <br /> YMH71-9C selected rich medium 16&#194;&#176;C.                             KDDPEYAEEREKMLIQIGVEADAGRSNVKV       90 <br /> scored potential pleiotropic phenotypes, 3 120 Csm+                      KDEDPNEYNEFPLRAIPKEDLENMRTHLLK      120 <br />                                                                                 FQSKKKINPVTDFHLPVRLHRKDTRNLQFQ      150 <br /> revertants Tsm-, failing grow 37&#194;&#176;C (Fig.                 LTRAEIVQRQKEISEYKKKAEQERSTPNSG      180 <br /> 2). strains reacquired Csm- Tsm+ pheno-                       GMNKSGTVSLNNTVKDGSQTPTVDSVTKDN <br />                                                                                 TANGVNSSIPTVTGSSVPPASPTTVSAVES <br />                                                                                                                     210 <br />                                                                                                                     240 <br /> types crossed sua7-1 mutant, indicating                   NGLSNGSTSAANGLDGNASTANLANGRPLV      270 <br /> phenotypes result recessive mutations (Fig. 2).                    TKLEDAGPAEDPTKVGMVKYDGKEVTNEPE      300 <br />                                                                                 FEEGTMDPLADVAPDGGGRAKRGNLRRXTR      330 <br /> diploid strain derived revertants (YZS14;                     QLKVLDENAKKLRFEEFYPWVMEDFDGYNT      360 <br /> Table 1) sporulated dissected. Csm+ Tsm-                        WVGSYEAGNSDSYVLLSVEDDGSFTMIPAD      390 <br />                                                                                 KVYKFTARNKYATLTIDEAEKRMDKKSGEV      420 <br /> phenotypes cosegregated, defining Tsm- marker                      PRWLMKHLDNIGTTTTRYDRTRRKLKAVAD      450 suppressor sua7-1.                                                   QQAMDEDDRDDNSEVELDYDEEFADDEEAP      480 <br />                                                                                 IIDGNEQENKESEQRIKKEMLQANAMGLRD      510 <br />    Tsm- phenotype exploited clone suppressor                     EEAPSENEEDELFGEKKIDEDGERIKKALQ      540 <br /> gene. yeast genomic library introduced strain                        KTELAALYSSDENEINPYLSESDIENKENE      570 <br />                                                                                 SPVKKEEDSDTLSKSKRSSPKKQQKKATNA      600 <br /> YZS14, transformants selected 37&#194;&#176;C. classes                     HVHKEPTLRVKSIKNCVIILKGDKKILKSF      630 transformants identified. Consistent antici-                   PEGEWNPQTTKAVDSSNNASNTVPSPIKQE      660 <br />                                                                                 EGLNSTVAEREETPAPTITEKDIIEAIGDG      690 <br /> pated phenotypes wild-type suppressor gene, class                   KVNIKEFGKFIRRKYPGAENKKLMFAIVKK <br />                                                                                 LCRKVGNDHMELKKE* <br />                                                                                                                     720 <br />                                                                                                                     735 Tsm+ transformants reacquired Csm- phenotype <br /> associated sua7-1. cured plasmid DNA,             FIG. 3. Ssu71 deduced amino acid sequence (single letter <br /> strains reacquired Tsm- Csm+ phenotypes, indicating        code). asterisk signifies C terminus. conditional phenotypes transformants consequence plasmid DNA, strain reversion.          structural characteristics. strikingly, Ssu71 <br /> Plasmid DNA recovered transformants,            RAP74 highly charged, consisting 34% 38% D, <br /> subsequent genetic analysis established wild-type         E, R, K residues, respectively. protein contains <br /> suppressor gene cloned  Materials          structural features commonly associated transcrip- <br /> Methods). Plasmid pM435 restored Tsm+ Csm- phe-            tion factors, including leucine-zipper zinc-finger motifs  notypes YZS14 YZS45, indicating         proteins include (RAP74) (Ssu7l) cys- <br /> strains compensate sua7-1 defect consequence       teine residues]. proteins contain multiple potential phos- <br /> mutations gene. gene designated SSU71         phorylation sites <span id='am-15' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-16' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span>cAMP</span>-dependent protein kinases (suppressor sua7, gene 1), suppressor alleles      protein kinase C, sites concentrated designated ssu71-1 (YZS14) ssu71-2 (YZS45) (Table 1).          C-terminal half protein (41). information suggests <br />    second class Tsm+ transformants remained Csm+.           functional relationship Ssu71 RAP74. <br /> Analysis plasmid DNA recovered transformants            SSU71 Encodes p105 Subunit Yeast TFIIF. Factor g <br /> revealed restriction pattern characteristic SUA 7 gene.   reported yeast counterpart human TFIIF suggested Tsm- phenotype conferred ssu71       composed polypeptides apparent molecular <br /> dependent sua7-1 mutation. confirmed             masses 30, 54, 105 kDa (23). genes encoding genetically. Tsm-:Tsm+ segregated 2:2            subunits recently isolated sequenced. meiotic progeny cross YZS14 sua7-1 mutant <br /> (sua7/sua7 SSU71 /ssu 71), Tsm -:Tsm+ segregated        deduced N termini (14 residues) TFG1-encoded p105 ratio 1:3 cross YZS14 wild-type strain       subunit Ssu71 protein compared; se- <br /> (SUA7/sua7 SSU71/ssu7l). dependency Tsm-                quences identical (42). Subsequent compar- <br /> phenotype associated ssu71 sua7-1 allele implies     ison entire sequences confirmed Ssu71 p105 functional relationship TFIIB product           identical (42).  SSU71 identical TFG1 SSU71.                                                             encodes largest subunit yeast TFIIF. discrepancy <br />    SSU71 Sequence. DNA sequence SSU71 iden-             deduced (82 kDa) apparent (105 kDa) <br /> tified open reading frame encoding 735 amino acids,             molecular masses Ssu71/p105 protein presumably corresponding protein molecular mass 82,194         anomalous migration SDS/PAGE. Highly charged pro- <br /> daltons (Fig. 3). Consistent size coding region,   teins known exhibit anomalously slow migration RNA (Northern) blot analysis detected single SSU71 tran-         SDS/PAGE (43), RAP74 protein, script 2.3 kb (data shown). Ssu71 amino acid            calculated molecular mass 58 kDa apparent molec- <br /> sequence compared protein sequence data banks.          ular mass 74 kDa (41, 44). slower mobility significant similarities RAP74 subunit     RAP74 consequence posttranslational modifi- <br /> human, Drosophila, Xenopus TFIIF. similarity            cation, recombinant RAP74 synthesized Escherichia coli <br /> Ssu71 RAP74 evident considering              migrates 74 kDa (44). <br />  <br />                                                                                           FIG. 2. Phenotypes associated sua7 <br />                                                                                        ssu7l mutants. Growth strains <br />                                                                                        YMH71-9C (sua7-1 SSU71), YZS14 (sua7-1 <br />                                                                                        ssu71-1), diploid strain (2N) derived                                                                                        cross YZS14 YDW546 (sua 7-1 SSU71) <br />                                                                                        shown yeast extract/peptone/dextrose <br />                                                                                        (YPD) reduced (16&#194;&#176;C), normal (30&#194;&#176;C),                                                                                        elevated (37&#194;&#176;C) temperatures. YZS14 se- <br />                                                                                        lected spontaneous revertant                                                                                        YMH71-9C YPD medium 16&#194;&#176;C.                                                                                        YMH71-9C phenotypically Csm- Tsm+, <br />                                                                                        YZS14 Csm+ Tsm-. Csm+                                                                                        Tsm- result recessive mutation(s), <br />                                                                                        Csm- Tsm+ phenotypes                                                                              37 0      restored YZS14 backcrossed strain <br />                                                                                        YDW546. <br />  3130           Genetics: Sun Hampsey                                                                    Proc. Natl. Acad ScL USA 92           (1995) <br />    SSU71 Essential Cell Viability. SSU71 open                                                            DISCUSSION <br /> reading frame disrupted insertion LEU2 gene BamHI site (Fig. 1), resulting construct                             data presented paper establish functional inter- <br /> used disrupt allele SSU71 homozygous leu2-                                 action TFIIB largest subunit TFIIF diploid strain. sporulation dissection, viable                                yeast.  SSU71 gene, encodes largest subunit <br /> inviable spores segregated ratio 2:2 (14 tetrads),                        TFIIF, uncovered suppressor Glu-62 -&gt; Lys <br /> viable spores phenotypically Leu- (data shown).                                  replacement (sua7-1) TFIIB. Second, sua7 ssu7l double spore viability restored diploid strain                             mutants exhibit synthetic phenotype: double sua7 ssu7l <br /> transformed plasmid pM435 (SSU71) prior dissection.                               mutants heat-sensitive, sua7 SSU71+ experiments define SSU71 essential gene.                                      SUA7+ ssu7l single mutants exhibit phenotype 37&#194;&#176;C. <br />    ssu71 Suppressor Restores Normal Start-Site Selection                              Finally, distinct downstream shift transcription start-site sua7 Background. sua7-1 mutation confers marked                               selection cycl locus associated sua7-1 allele effect transcription start-site selection. Csm-                              fully suppressed ssu71-1 allele, resulting normal <br /> phenotype start site effects consequence                            pattern initiation cycl. <br /> mutation (10, 19), asked ssu71-1 suppressor                                   Glu-62 invariant residue phylogeneti- compensates start-site shift. Restoration                             cally conserved region TFIIB. region adjacent <br /> normal initiation pattern ssu7l strains anticipated                            zinc-finger motif included proteolytically sus- <br /> based iso-1-cytochrome c phenotypes. strains                               ceptible domain TFIIB required recruitment carry cycl-5000 allele, result aberrant                            pol II-TFIIF promoter-TBP-TFIIB complex ATG translation start codon [upstream ATG (uATG), posi-                                   transcription initiation (14-18). TFIIF thought mediate <br /> tion -20] transcribed leader region cycl-SOOC (10).                             interaction pol II promoter-TBP-TFIIB mutation diminishes iso-1-cytochrome c synthesis &lt;2%                              complex function attributed specifically normal. sua7-1 mutation partially suppresses defect                           RAP30 subunit (27). Consistent premise, TFIIB shifting transcription initiation downstream uATG,                              binds RAP30, interaction mapped enhancing iso-1-cytochrome c level &quot;30% normal                                  N-terminal domain TFIIB (18).  suppression (10).  iso-1-cytochrome c level restored                               Glu-62 -&gt; Lys replacement yeast TFIIB mutation &lt;2% normal ssu7l mutants (Table 1), suggesting                             gene encoding homolog RAP74 suggests interaction <br /> initiation shifted upstream uATG.                                         TFIIB TFIIF involves subunits TFIIF. <br />    effect ssu7l start-site selection confirmed                             currently evidence direct interaction <br /> mapping transcription start sites cycl gene primer                              TFIIB RAP74 Ssu71/p105; efforts extension analysis. Consistent previous results, sua7-1                              characterize interaction TFIIB Ssu71/p105 <br /> enhanced..initiation downstream sites, including discrete                              hampered toxicity associated expression <br /> sites positions -28 -14 (Fig. 4, lanes 1 vs. 2).                          SSU71 derivatives E. coli (unpublished observation). <br /> normal initiation pattern restored ssu71-1 mutant                                 Activities attributed TFIIF reminiscent (Fig. 4, lane 3), effect cosegregated ssu71-1 (Fig.                         assigned ur70 family bacterial transcription factors, <br /> 4, lane 4).  ssu71-1 does simply compensate Csm-                             including tight binding pol II, suppression nonspecific influences start-site selection, presumably interaction, direct indirect, TFIIB.                                       binding pol II free DNA, stabilization                                                                                           preinitiation complex (45). Furthermore, limited sequence <br />                                       1 2 3 4 C G T                                     similarities 70 noted RAP30                                                                                           RAP74 (46-48). Comparison Ssu71/p105 RAP74 <br />                                                                                           sequences E. coli a70 revealed regions sequence <br />                                                                                           similarity corresponding o,70 RpoD box, straddles <br />                              -464o <br />                                                                                           homology regions 2.3 2.4, homology region 4.2 (Fig. <br />                                                                                           5). observation particularly noteworthy regions <br />                                                                                           2.4 4.2 involved recognition -10 -35 <br />                                                                                           regions, respectively, bacterial promoter (49).                                    0. <br />                                  -28 <br />                                                                                           question functional significance                                                                                           sequence similarities readily investigated mu- <br />                                                                                           tational analysis cloned SSU71 gene. <br />                                                                                                                   RpoD box <br />                                                                                                 421     EYRRGYKFST---YATWWIRQAITR           a70 <br />                                                                                                           :: III::   III                               -14      40      - <br />                                                                                                 388     PADKVYKFTARNKYATLTIDEAEKR           Ssu7l/p105 <br />                                                                                                         1::: 1:11: :P111::111:: <br />                                                                                                  13 9   PVHNWYNFTPLARHRTLTAEEAEEE           RAP7 4 <br />  <br />                                                                                                                region 4. 2 <br />                                                                                                 572     TLEEVGKQFDVTRERIRQIEAKALRK a70 <br />                                                                                                          : <br />                                                                                                         11:              :   1: @ <br />   FiG.   4.    Primer extensionanalysis cycl transcription start sites.                       607    TLRVKSIKNCVIILKGDKKILKSFPE Ssu7l/p105 <br /> Lanes: 1, T16  (SUA7SU7J); 2, YMH71-9C (sua7-l SSU71); 3, YZS14                                         III:          1:: <br /> (sua7-J ssu7l-1); 4, YZS14-3B (sua7-l ssu7l-1). Strain YZS14-3B                            400     TLRAAASK-----LEQGKRVSEM-PA RAP74 <br /> segregant backcross YZS14 YDW546 (Table 1). Tran- <br /> scription start sites numbered relative ATG translation start                     FIG. 5. Alignment E. coli oJ70, yeast Ssu71/p1O5, human <br /> codon. sequence ladder  C,G,T) corresponds SUA7 DNA                              RAP74. Sequence identity similarity indicated vertical lines       used          molecular   size   marker.     initiation     colons, respectively. Similarity rules PrA==S, G S, <br /> enhanced downstream                sites sua7 mutant relative            T S, D-E==N=Q=K=R=H, -V L, F=Y=W. <br /> wild-type       (compare lanes 1 2), normal initiation pattern <br />               strain                                                                      E. coli RpoD box underlined encompasses portions Co70 restored     isogenie sua7 ssu7l strain (lane 3); effect <br />                                                                                       regions 2.3 2.4. entire region 4.2 a70 underlined. cosegregates ssu7l marker (lane 4).                                              similarity co70 RAP74 noted previously (48). <br />            Genetics: Sun    Hampsey                                                   Proc. Natl. Acad Sci. USA 92 (1995)              3131 <br />  <br />    mechanism transcription start-site selection           17. Hisatake, K., Roeder, R. G. <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Horikoshi, M. (199 <br /> </body></html>