<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>   <br />                                                           Polarity-Dependent Asymmetric Distribution <br />                                                           MEX-5/6&#226;&#8364;&#8220;Mediated Translational <br />                                                           Activation Era-1 mRNA C. elegans <br />                                                           Embryos <br />   <br />                                                           Abstract <br />                                                           early C. elegans embryo attractive model investigate fundamental  <br />                                                           velopmental processes. exception mex-3 mRNA, maternally contributed <br />     OPEN ACCESS                                           mRNAs thought distributed uniformly cell embryo.  report                                                           characterize striking distribution mRNA encoding novel protein ERA-1. Citation: Spir&#195;&#179;  G&#195;&#182;nczy P (2015) Polarity- <br /> Dependent Asymmetric Distribution MEX-5/6&#226;&#8364;&#8220;            era-1 mRNA enriched anterior cell embryo present solely <br /> Mediated Translational Activation Era-1 mRNA       anterior blastomeres  era-1 essential gene, uncovered C. elegans Embryos. PLoS 10(3): e0120984.          era-1 null mutant embryos sensitive slight impairment embryonic polarity. doi:10.1371/journal.pone.0120984 <br />                                                           asymmetric distribution era-1 mRNA depends anterior-posterior polarity <br /> Academic Editor: Myon-Hee Lee, East Carolina              cues era-1 3&#226;&#8364;&#8482;UTR. Similarly era-1 mRNA, YFP-ERA-1 protein  <br /> University, UNITED STATES <br />                                                           riched anterior blastomeres. Interestingly, RNA-binding protein MEX-5 <br /> Received: October 9, 2014                                 required era-1 mRNA asymmetry. Furthermore, MEX-5, Accepted: February 9, 2015                                partially redundant partner MEX-6, needed activate era-1 mRNA translation anteri- <br /> Published: March 30, 2015                                 blastomeres. findings lead propose MEX-5/6&#226;&#8364;&#8220;mediated regulation                                                           era-1 mRNA contributes robust embryonic development. <br /> Copyright: &#194;&#169; 2015 Spir&#195;&#179;, G&#195;&#182;nczy. open <br /> access article distributed terms Creative Commons Attribution License, permits <br /> unrestricted use, distribution, reproduction   Introduction <br /> medium, provided original author source credited.                                                 zygotic transcription begins, restricted distribution maternally provided mRNAs                                                           translational regulation widely used mechanisms orchestrate early developmental <br /> Data Availability Statement: relevant data paper Supporting Information files.    processes, including cell embryo  referred zygote) (reviewed                                                           [1&#226;&#8364;&#8220;3]). Compared metazoan organisms issue investigated, C. <br /> Funding: work supported grant                                                           elegans stands apart mex-3 [4], mRNA restricted distribution Swiss National Science Foundation (grant <br /> number: 3100A0-122500/1, website: http://www.snf. <br />                                                           zygote reported <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>. <br /> ch Pages/default.aspx). funders role        C. elegans zygote polarized shortly fertilization anterior-poste- <br /> study design, data collection analysis, decision   rior  P) embryonic axis action PAR proteins associated components, <br /> publish, preparation manuscript.                localize anterior (PAR-6, PAR-3, PKC-3) posterior (PAR-2, PAR-1) cell cortex <br /> Competing Interests: authors declared            (reviewed [5]). P asymmetries generated result, including anterior  competing interests exist.                        richment partially redundant CCCH RNA-binding proteins MEX-5 MEX-6, <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                                      1 / 14 <br />                                                                                           Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          function reinforce polarity [6,7]. anterior enrichment requires phosphorylation <br />                                          MEX-5 PAR-1 kinase, results increased MEX-5 diffusion posterior <br />                                          P MEX-5 gradient [8]. result proper P polarity, zygote divides <br />                                          unequally, yielding larger anterior cell AB smaller posterior cell P1.  AB <br />                                          divides symmetrically, generating ABa ABp, P1 divides asymmetrically, resulting <br />                                          EMS P2. <br />                                              Translational regulation plays important role C. elegans development past                                          cell stage.  MEX-5/6, MEX-3 SPN-4, regulate translation                                          mRNAs anterior blastomeres, POS-1 regulates mRNAs posterior <br />                                          blastomeres [9&#226;&#8364;&#8220;11]. general, translational repressors activators act binding                                          3&#226;&#8364;&#8482;UTR target mRNAs. example, translation uniformly localized mRNA encoding <br />                                          Notch receptor GLP-1 activated ABa ABp binding MEX-5, SPN-4                                          MEX-3 glp-1 3&#226;&#8364;&#8482;UTR, repressed, 3&#226;&#8364;&#8482;UTR, POS-1 EMS <br />                                          P2 [11,12]. Similarly, homogenously distributed zif-1mRNA, encodes E3 ubi- <br />                                          quitin protein ligase complex component, dual regulation, MEX-5  <br />                                          diated activation winning POS-1-directed repression anterior blastomeres,                                          giving rise asymmetric ZIF-1 protein distribution [13]. <br />                                              majority maternally provided mRNAs exhibits uniform distribution                                          early C. elegans embryos  called class mRNAs [14]), identified po- <br />                                          larized embryos past 2-cell stage.  class II mRNAs cey-2, skn-1                                          pos-1 distributed uniform manner zygote, enriched P-lineage <br />                                          following preferential degradation cells [10,14]. Single cell mRNA <br />                                          deep sequencing experiments revealed 4 mRNAs abundant P1 AB, <br />                                          converse true 13 mRNAs [15]. mRNAs exhibits <br />                                          asymmetric distribution zygote addressed.  mex-3 mRNA, <br />                                          enriched anterior zygote present solely anterior blastomeres <br />                                           mRNA known exhibit restricted distribution C. elegans zygote, <br />                                          mechanisms governing distribution understood [4,9] <br />  <br />  <br />                                          Materials Methods <br />                                          Worm strains molecular biology <br />                                          Wild type (N2) animals maintained 20&#226;&#8364;&#8220;24&#194;&#176;C, transgenic worms expressing YFP-ERA- <br />                                          1[3&#226;&#8364;&#8482;era-1] UTR, YFP[3&#226;&#8364;&#8482;era-1] YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] 24&#194;&#176;C. constructs subcloned <br />                                          pCFJ178 [15], expresses C. briggsae unc-119 genomic sequence. yfp gene <br />                                          3 introns [16] fused era-1 genomic sequence, start codon end <br />                                          174 nt-long 3&#226;&#8364;&#8482;UTR (yfp-era-1[3&#226;&#8364;&#8482;era-1) stop codon open reading frame <br />                                          (yfp-era-1[3&#226;&#8364;&#8482;pie-1]). 120 nt-long pie-1 3&#226;&#8364;&#8482;UTR used yfp-era-1[3&#226;&#8364;&#8482;pie-1]. Transgenic <br />                                          lines generated ballistic bombardment unc-119(ed3) animals [17]; designations <br />                                          resulting strains follows: GZ1146&#226;&#8364;&#8221;isIs101[pie-1::yfp-era-1[3&apos;era-1] unc-119(+); <br />                                          GZ1280 -isIs102[pie-1::yfp[3&apos;era-1] unc-119(+) GZ1169&#226;&#8364;&#8221;isIs103[pie-1::yfp-era-1[3&apos;pie-1] <br />                                          unc-119(+). Note strain (GZ1169), phenotypically wild-type animals segre- <br />                                          gate *50% wild-type, *25% Uncs *25% sterile animals protruding vulva (Ste/Pvl). <br />                                          Ste/Pvl phenotype suppressed gfp(RNAi), surmise does                                          stem presence YFP-ERA-1 protein, instead disruption, transgene <br />                                          insertion, gene important vulva development.  exclude possibil- <br />                                          ity GZ1169 carries extrachromosomal array. era-1(tm5852) era-1(tm6426) <br />                                          strains provided National Bioresource Project (NBRP) maintained 20 <br />                                          24&#194;&#176;C. Bacterial feeding strains era-1, mex-5, par-3 Vidal library [18],  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                           2 / 14 <br />                                                                                           Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          par-2 Ahringer library [19]. feeding clones mex-6 par-1 gen- <br />                                          erated amplifying region mex-6  position 241 1014) par-1                                           positions 1288 2395) cDNAs, cloned L4440 vector [20]. Feeding <br />                                          RNAi performed 24 hours 24&#194;&#176;C, par-1(RNAi), 48 hours incubation <br />                                          24&#194;&#176;C needed. Double depletion achieved mixing IPTG-induced individual <br />                                          bacterial cultures 1:1 ratio. RNAi injection soaking, dsRNA targeting  <br />                                          tire era-1 mRNA synthesized SP6 T7 vitro synthesis kits (Promega). Soak- <br />                                          ing performed L3 animals 24 hours 20&#194;&#176;C resulting embryos <br />                                          investigated 24 hours recovery. similar recovery period followed injection L4 animals <br />                                          era-1 dsRNA. era-1 RNAi feeding performed 20&#226;&#8364;&#8220;24 hours 24&#194;&#176;C. partially <br />                                          deplete par-2 par-3, L4 animals fed bacteria expressing relevant dsRNAs 4 <br />                                          hours 24&#194;&#176;C transferred plates containing OP50 20&#194;&#176;C, laid eggs <br />                                          scored embryonic viability. <br />  <br />  <br />                                          Live imaging <br />                                          Dual time-lapse DIC fluorescence imaging performed Zeiss Axioplan 2  <br />                                          scribed [21]. motorized filter wheel, external shutters, 1,392 x 1,040 pixels <br />                                          12-bit Photometrics CoolSNAP ES2 controlled &#206;&#188;Manager [22]. Images acquired <br />                                          exposure time 10&#226;&#8364;&#8220;100ms DIC 300 ms fluorescence channel using <br />                                          Zeiss Filter Set 10. Embryos allowed develop coverslip imaging <br />                                          snapshots taken indicating times. <br />  <br />  <br />                                          Immunostaining <br />                                          immunostaining, embryos fixed methanol &#226;&#710;&#8217;20&#194;&#176;C 30 minutes followed  <br />                                          cubating overnight 4&#194;&#176;C primary antibodies. Primary mouse antibodies &#206;&#177;-tubu- <br />                                          lin (1/300; DM1A, Sigma) GFP (1/100; MAB3580, Millipore) used rabbit <br />                                          antibodies GPA-16 (1/200, [23] PAR-2 (1:200, [24]. Secondary antibodies Alex- <br />                                          a488-conjugated goat anti-mouse (1/500; Life Technologies, A11001) Cy3-conjugated goat <br />                                          anti-rabbit (1/1000; Jackson ImmunoResearch, 711-165-152). Slides counterstained                                          1mg/ml Hoechst 33258 (Sigma) visualize DNA. Images acquired LSM700 confo- <br />                                          cal microscope (Zeiss) processed ImageJ. <br />                                              quantification cell membrane enrichment, fluorescence intensities YFP                                          GPA-16 measured membrane middle plane embryo values ex- <br />                                          pressed ratio YFP GPA-16 intensities. <br />  <br />  <br />                                          RT-qPCR <br />                                          Embryos bleached total mRNA isolated using trizol-chloroform extraction [25]. <br />                                          200 &#206;&#188;g total RNA reverse transcribed using poly-dT random hexamer primers. <br />                                          Specific primer pairs used amplify cDNA Power SYBR Green PCR mix <br />                                          (Applied Biosystems). PCR reaction performed 7900HT Fast Real-Time PCR <br />                                          machine (Applied Biosystems) SDS2.4 software using 60&#194;&#176;C annealing                                          72&#194;&#176;C elongation. &#206;&#8221;&#206;&#8221;CT method used quantify mRNA levels relative refer- <br />                                          ence gene, act-1. primers follow: <br />                                           act-1 F &#226;&#8364;&#8220; AGTCCGCCGGAATCCACGAG <br />                                           act-1 R &#226;&#8364;&#8220; CTTGATCTTCATGGTTGATG <br />                                           yfp F &#226;&#8364;&#8220; GGGAACTACAAGACACGTGC <br />                                           yfp R &#226;&#8364;&#8220; GTGTCCAAGAATGTTTCC <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                           3 / 14 <br />                                                                                           Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          Digoxigenin situ hybridization <br />                                          Digoxigenin-labeled single stranded DNA probes prepared using PCR DIG Probe <br />                                          Synthesis Kit (Roche) following manufacturer&#226;&#8364;&#8482;s instructions. Briefly, plasmid carrying <br />                                          gene served template PCR-based reaction, reverse primer binding <br />                                          3&#226;&#8364;&#8482;UTR added DIG-11-<span id='am-2' about='obo:IMR_0200056' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-3' property="rdfs:label" content="dUTP" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0200056" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="KEGG:C00460" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>dUTP</span> containing dNTP mix. Note did <br />                                          linearize plasmid preparing probes used circular form asym- <br />                                          metric PCR instead, reasoning non-specific sequences probe mixture help <br />                                          block non-specific binding sites. resulting PCR product ethanol-precipitated  <br />                                          suspended hybridization buffer (50% deionized formamide, 5x SSC, 100 &#206;&#188;g/ml sonicated <br />                                          salmon testis DNA, 100 &#206;&#188;g/ml yeast tRNA, 100 &#206;&#188;g/ml heparin, 0.1% Tween-20) concentra- <br />                                          tion 50 ng/&#206;&#188;l, boiled 95&#194;&#176;C 75 minutes reduce size (Seydoux  1995)                                          stored &#226;&#710;&#8217;20&#194;&#176;C. <br />                                              situ hybridization protocol developed combination previously available <br />                                          procedures [26,27]. Gravid hermaphrodites bleached resulting embryos placed <br />                                          poly-D-lysine coated slide total volume 7&#206;&#188;l diluted water. coverslip                                          gently deposited embryos, freeze-crack method used remove eggshell. <br />                                          Subsequent steps performed plastic slide holder containing 150 ml indicated <br />                                          solutions, unless stated.  slides submerged &#226;&#710;&#8217;20&#194;&#176;C methanol <br />                                          5 minutes sequential mixtures methanol Hepes-PBS-formalde- <br />                                          hyde (HPF) solutions 4&#194;&#176;C (1x PBS, 75mM Hepes pH = 6.9, 0.03% EGTA, 1.5 mM MgSO4, <br />                                          3.6% <span id='am-31' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>formaldehyde</span>); ratios water:methanol:Hepes-PBS-formaldehyde 50:15:35, <br />                                          50:25:25 finally 50:35:15), 2 minutes 4&#194;&#176;C, fixation 3.6% formalde- <br />                                          hyde HPF solution 20 minutes 4&#194;&#176;C [26]. Following wash PBS-Tween (0.1%, <br />                                          PBS-Tw) 5 minutes room temperature, Proteinase K (5.6 &#206;&#188;g/ml) digestion performed <br />                                          10 minutes 24&#194;&#176;C PBS-Tw stopped submerging slides 2 g/l <span id='am-7' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-8' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="glycine" datatype="xsd:string"></span>glycine</span> <br />                                          dissolved PBS 2 min followed 2 washes PBS-Tw 2 minutes  slides                                          fixed HPF solution 50 minutes room temperature, washed twice PBS-Tw <br />                                          5 minutes  treated 2 mg/ml <span id='am-19' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-20' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="glycine" datatype="xsd:string"></span>glycine</span> 5 minutes washed                                          PBS-T. Slides submerged Formamide-SSC solution (50% formamide-5xSSC <br />                                          (pH = 7) -100 &#206;&#188;g/ml heparin, 0.1% Tween-20), mixed PBS-Tw 1:1 ratio 10 min- <br />                                          utes Formamide-SSC 10 minutes. Pre-hybridization carried <br />                                          50 &#206;&#188;l hybridization buffer (50% deionized formamide 50%, 5xSSC, 100 &#206;&#188;g/ml sonicated <br />                                          salmon testis DNA, 100 &#206;&#188;g/ml yeast tRNA, 100 &#206;&#188;g/ml, heparin 0.1% Tween 20) 1 hour                                          48&#194;&#176;C, probe applied concentration 40 ng/&#206;&#188;l 50 &#206;&#188;l overnight,                                          48&#194;&#176;C. prevent evaporation, parafilm squares glued rubber cement mounted                                          slides. probe washed twice Formamide-SSC:PBS-Tw (1:1) 10 minutes <br />                                          4 times using 0.1% Chaps 0.8x PBS 48&#194;&#176;C 20 minutes [26],                                          blocking step included using 0.1%BSA&#226;&#8364;&#8221;0.1%Triton-X 1xPBS (PBS-BSA-Tx) 1.5 <br />                                          hours room temperature. alkaline-phosphatase (AP)-conjugated anti-DIG antibody <br />                                          (Roche) diluted 1:2000 1x PBS incubated 4&#194;&#176;C overnight. slides washed <br />                                          5 times PBS-BSA-Tx twice staining buffer (100 mM Tris-HCl (pH = 9.5), <br />                                          100mM NaCl, 5mM MgCl2, 1mM Levamisole, 0.1% Tween-20); signal developed <br />                                          using mixture 333 &#206;&#188;g/ml NBT (Roche) 70 &#206;&#188;g/ml BCIP (AppliChem) 20&#226;&#8364;&#8220;60 min- <br />                                          utes room temperature. slides washed PBS-Tw 5 minutes                                          room temperature, stained 1mg/ml Hoechst 33258 (Sigma) PBS-Tw, washed                                          PBS, mounted stored 4&#194;&#176;C. situ hybridization experiments performed                                          twice representative embryos shown figure panels. Control embryos                                          included slide instance. <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                           4 / 14 <br />                                                                                            Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                             Quantification mRNA levels performed measuring mean intensities cyto- <br />                                          plasm AB P1   P), background outside embryos (B), later stage <br />                                          embryos, hybridization did signal (C). value B subtracted                                           P C, resulting corrected values used determine ratios C P/C,                                          reported figures arbitrary units  U.). instances, cell embryo <br />                                          shown figures surrounded multicellular embryos possessing high Hoechst signal <br />                                          intensities, leading overflow blue signal embryo  cases, <br />                                          background subtracted Hoechst image divided Gaussian blurred value, <br />                                          resulting clearing non-pertinent signal. brightness enhanced clearer <br />                                          visualization relevant DNA signal. <br />  <br />  <br />                                          Results Discussion <br />                                          era-1 mRNA enriched anterior C. elegans zygote                                          anterior blastomeres                                          determine localized mRNAs zygote apart mex-3, analyzed <br />                                          distribution 11&#226;&#8364;&#8482;237 mRNAs available situ hybridization database (Nematode <br />                                          Expression Database (NEXTDB), http://nematode.lab.nig.ac.jp) database contains typical- <br />                                          ly 4&#226;&#8364;&#8220;6 images, 7&#226;&#8364;&#8220;8 embryos different stages development. analyzed <br />                                          images low magnification; non-uniform distribution spotted                                          early stage embryos manner, images magnified assess mRNA distri- <br />                                          bution. analysis led uncover transcript uncharacterized open reading <br />                                          frame W02F12.3 displays suggestive distribution, bias anterior zy- <br />                                          gote embryos subsequent stages embryogenesis. dubbed gene era-1                                           embryonic mRNA anterior) investigated  <br />                                              ascertained asymmetric distribution era-1 mRNA digoxygenin-alkaline phos- <br />                                          phatase situ hybridization. asymmetric enrichment era-1 mRNA                                          apparent prophase cell cycle (Fig. 1A-B).  era-1 mRNA enriched <br />                                          anterior blastomere AB compared posterior blastomere P1 cell embryos <br />                                          (Fig. 1C, 1I). difference persists comparing AB daughters ABa ABp                                          P1 daughters EMS P2 (Fig. 1D). signal detected era-1(RNAi) embryos, indicating <br />                                          probe specificity (Fig. 1F, 1I). <br />                                              Intriguingly, era-1 genes identified single cell mRNA sequencing                                          enriched AB versus P1 [28]. Conversely, 13 mRNAs identified study                                          enriched AB versus P1, mex-3 mRNA shows clear asymmetry Nematode <br />                                          Expression Database. 6 mRNAs appear uniformly distributed, 5                                          tested 1 analyzable oversaturated signal.  mRNA sequencing <br />                                          large-scale situ hybridization methods provide complementary means identify <br />                                          mRNAs asymmetric distribution C. elegans embryos. <br />  <br />  <br />                                          Structure function era-1 <br />                                          era-1 gene predicted encode 388 amino acids long polypeptide, clear protein <br />                                          domains (S1A Fig.). distant relatives ERA-1 nematodes (S1B Fig.), al- <br />                                          bona fide orthologues, performing BLAST analysis                                          C. elegans proteome did identify ERA-1 hit, indicating fast divergence                                          common ancestor. <br />                                             set address function era-1 C. elegans. comprehensive RNAi-based <br />                                          functional genomic screen did reveal requirement era-1 [29]. Accordingly,  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                             5 / 14 <br />                                                                                                        Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />  <br /> Fig 1. era-1 mRNA enriched anterior early C. elegans embryos. D era-1 mRNA distribution wild-type zygote (P0), prophase   prometaphase (B), 2-cell (C) 4-cell (D) embryos. figures, mRNA appears dark grey, DNA light blue scale bars <br /> represent 10 microns. E-H era-1 mRNA distribution control (E), era-1(RNAi) (F), par-3(RNAi) (G) mex-5(RNAi) (H) 2-cell embryos. Corresponding <br /> quantifications mRNA levels AB P1 (Materials Methods). Number embryos quantified: wild type, n = 9; era-1(RNAi), n = 12; par-3(RNAi), <br /> n = 10; mex-5(RNAi), n = 7. figures, experiments performed twice, mean values &#194;&#177; standard deviations (SD) shown, exact values given S1 Table. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare mRNA levels AB versus P1, yielding following p-values: wild type&#226;&#8364;&#8221;p = 5.67&#195;&#8212;10&#226;&#710;&#8217;10, era-1(RNAi), p = 0.326; par-3(RNAi), p = 0.907; mex-5(RNAi), p = 0.287. <br /> doi:10.1371/journal.pone.0120984.g001 <br />  <br />                                                performing era-1(RNAi) injection, soaking feeding did result lethality                                                obvious phenotype resulting progeny. Furthermore, deletion alleles <br />                                                era-1(tm5852) era-1(tm6426) viable fertile exhibit apparent pheno- <br />                                                type embryo (S1A, S1C Fig.). conclude era-1 essential gene stan- <br />                                                dard laboratory conditions. <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                            6 / 14 <br />                                                                                           Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                             intriguing polarized distribution era-1 mRNA, rapid <br />                                          sequence divergence nematodes, explored era-1 exhibit synthetic le- <br />                                          thality genes required embryonic polarity C. elegans. Intriguingly,                                          era-1(tm6426) embryos sensitive wild-type partial RNAi-mediated <br />                                          depletion PAR-3 PAR-2 (S1C Fig.). future work needed determine                                          mechanism underlying synthetic lethality, conclude ERA-1 plays non-essential <br />                                          role ensuring robust embryonic development. <br />  <br />                                          era-1 mRNA distribution requires MEX-5 era-1 3&#226;&#8364;&#8482;UTR <br />                                          sought address regulates remarkable distribution era-1 mRNA.                                           asked anterior enrichment polarity-dependent. shown Fig. 1G 1I,                                          case, era-1 mRNA distribution uniform par-3(RNAi) embryos.                                          trans-acting factor govern anterior enrichment? anteriorly localized <br />                                          CCCH finger protein MEX-5 plausible candidate likewise enriched                                          anterior cell stage embryos [9]. Accordingly, era-1 mRNA localized                                          uniform manner mex-5(RNAi) embryos (Fig. 1H, 1I). merely reflect                                          role MEX-5 polarity establishment, evidence shown suggests MEX-5 plays                                          direct role regulating era-1 mRNA. <br />                                              identify cis-acting element regulating era-1 mRNA distribution, fused sequence <br />                                          coding yellow fluorescent protein (YFP) era-1 genomic sequence (exons, introns <br />                                          3&#226;&#8364;&#8482;UTR) expression vector driven pie-1 promoter (yielding yfp-era-1[3&#226;&#8364;&#8482;era-1], <br />                                          Fig. 2A). generated corresponding transgenic line performed situ hybridization <br />                                          yfp mRNA. yfp-era-1[3&#226;&#8364;&#8482;era-1] mRNA enriched anterior <br />                                          zygote inherited anterior blastomeres (Fig. 2B, 2G). <br />                                              Sequences 3&#226;&#8364;&#8482;UTR responsible localizing mRNAs [30,31]. test                                          contribution era-1 3&#226;&#8364;&#8482;UTR era-1 mRNA distribution, generated transgenic animals <br />                                          expressing yfp fused directly era-1 3&#226;&#8364;&#8482;UTR (yielding yfp[3&#226;&#8364;&#8482;era-1], Fig. 2C, 2D). shown <br />                                          Fig. 2G, analysis uncovered significant anterior bias yfp[3&#226;&#8364;&#8482;era-1] mRNA,                                          extent enrichment AB versus P1 pronounced yfp-era-1[3&#226;&#8364;&#8482;era-1] <br />                                          (compare Fig. 2D 2B). conclude era-1 3&#226;&#8364;&#8482;UTR, sufficient  <br />                                          contributes mRNA anterior distribution, help elements located coding <br />                                          sequence. <br />                                              test era-1 3&#226;&#8364;&#8482;UTR necessary anterior mRNA distribution, modified yfp- <br />                                          era-1[3&#226;&#8364;&#8482;era-1] replacing era-1 3&#226;&#8364;&#8482;UTR pie-1 3&#226;&#8364;&#8482;UTR (yielding yfp-era-1[3&#226;&#8364;&#8482;pie-1], <br />                                          Fig. 2E), generated corresponding transgenic line. Strikingly, yfp-era-1 <br />                                          [3&#226;&#8364;&#8482;pie-1] mRNA localizes homogenous manner, difference AB P1 <br />                                          (Fig. 2F, 2G).  era-1 3&#226;&#8364;&#8482;UTR necessary anterior mRNA distribution. <br />  <br />                                          YFP-ERA-1 protein enriched anterior blastomeres <br />                                          address anterior enrichment era-1 mRNA accompanied likewise pro- <br />                                          tein distribution, analyzed embryos expressing yfp-era-1[3&#226;&#8364;&#8482;era-1] using dual differential  <br />                                          terference contrast (DIC) fluorescent time-lapse microscopy. revealed YFP-ERA- <br />                                          1[3&#226;&#8364;&#8482;era-1] protein detected starting *10 cell stage solely descendants                                          anterior blastomere AB (Fig. 3A). performed immunofluorescence analysis using anti- <br />                                          bodies YFP offer sensitive detection means. YFP-ERA- <br />                                          1[3&#226;&#8364;&#8482;era-1] distributed uniformly zygote enriched AB cell (S1D, S1E <br />                                          Fig.), anterior bias pronounced 4-cell stage embryos (Fig. 4A, 4E).  <br />                                          triguingly, noted YFP-ERA-1[3&#226;&#8364;&#8482;era-1] slightly enriched cell cortex, <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                           7 / 14 <br />                                                                                                          Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />  <br /> Fig 2. yfp-tagged era-1 mRNA recapitulates endogenous era-1 mRNA distribution 3&#226;&#8364;&#8482;UTR-dependent manner.  C, E Schematic representation yfp-era-1[3&#226;&#8364;&#8482; era-1]  , yfp[3&#226;&#8364;&#8482;era-1] (C) yfp-era-1[3&#226;&#8364;&#8482;pie-1] (E) constructs. B, D, F Localization constructs 2-cell embryos. G Corresponding <br /> quantifications mRNA levels AB P1 (Materials Methods). Number embryos quantified: yfp-era-1[3&#226;&#8364;&#8482;era-1], n = 9;. yfp[3&#226;&#8364;&#8482;era-1], n = 8; yfp-era- <br /> 1[3&#226;&#8364;&#8482;pie-1], n = 10. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare mRNA levels AB versus P1, yielding following p- <br /> values: yfp-era-1[3&#226;&#8364;&#8482; era-1], p = 1.03&#195;&#8212;10&#226;&#710;&#8217;7; yfp[3&#226;&#8364;&#8482;era-1], p = 2&#195;&#8212;10&#226;&#710;&#8217;4; yfp-era-1[3&#226;&#8364;&#8482;pie-1], p = 0.362. Note anterior enrichment yfp-era-1[3&#226;&#8364;&#8482;era-1] somewhat pronounced endogenous era-1 mRNA, era-1 5&#226;&#8364;&#8482;UTR contributes mRNA distribution presence yfp perturbs localization. <br /> doi:10.1371/journal.pone.0120984.g002 <br />  <br />                                                 apparent boundary anterior blastomeres (Fig. 4A), rais- <br />                                                 ing possibility ERA-1 cortical protein. address anterior enrichment                                                 YFP-ERA-1[3&#226;&#8364;&#8482;era-1] protein dependent P polarity, analyzed embryos depleted                                                 PAR-3. anticipated uniform distribution era-1 mRNA embryos                                                  Fig. 1I), YFP-ERA-1[3&#226;&#8364;&#8482;era-1] present comparable levels blastomeres                                                 par-3(RNAi) embryos (Fig. 4C, 4E). <br />                                                    test era-1 3&#226;&#8364;&#8482;UTR important restricting fusion protein anterior blasto- <br />                                                 meres, examined embryos expressing yfp-era-1[3&#226;&#8364;&#8482;pie-1]. Interestingly,                                                 YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] protein present uniform manner zygote (Figs. <br />                                                 3B, 4F, 4H). conclude era-1 3&#226;&#8364;&#8482;UTR important restricting presence                                                 fusion protein anterior blastomeres embryogenesis.  noted                                                 YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] protein levels higher YFP-ERA-1[3&#226;&#8364;&#8482;era-1],                                                 anterior blastomeres (compare Figs. 3A 3B 4A 4F). despite yfp-era-1 <br />                                                 [3&#226;&#8364;&#8482;pie-1] mRNA levels lower yfp-era[3&#226;&#8364;&#8482;era-1] (Fig. 2G). pro- <br />                                                 tein present newly fertilized zygote comparing YFP-ERA-1[3&#226;&#8364;&#8482;pie-1]                                                 YFP-ERA-1[3&#226;&#8364;&#8482;era-1] (compare Fig. 3B 3A, row), findings indicate                                                 era-1 3&#226;&#8364;&#8482;UTR prevents translation oogenesis. <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                               8 / 14 <br />                                                                                                       Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />  <br /> Fig 3. YFP-ERA-1 enriched anterior blastomeres 3&#226;&#8364;&#8482;UTR- MEX-5/6-dependent manner. B Snapshots dual DIC fluorescent <br /> time-lapse microscopy recordings embryos expressing YFP-ERA-1[3&#226;&#8364;&#8482;era-1]   YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] (B). Time indicated min:sec, t = 0 <br /> corresponding cytokinesis onset P0. Arrowheads points YFP signal cell cortex. Brightness contrast lowered panel B avoid <br /> saturating signal. C-D Snapshots late stage mex-5/6(RNAi) embryos expressing YFP-ERA-1[3&#226;&#8364;&#8482;era-1] (C) YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] (D). <br /> doi:10.1371/journal.pone.0120984.g003 <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                               9 / 14 <br />                                                               Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                              10 / 14 <br />                                                                                                          Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br /> Fig 4. MEX-5/6-mediated translational activation era-1. D 4-cell embryos expressing YFP-ERA-1[3&#226;&#8364;&#8482; era-1] control  , mex-5/6(RNAi) (B), par-3 <br /> (RNAi) (C) par-1(RNAi) (D) conditions, stained antibodies GFP (green) GPA-16 (red); DNA viewed blue. E Signal intensities cell-cell boundaries ABa/ABp EMS/P2 relative GPA-16 (Materials Methods). Number embryos quantified: control, n = 10; mex-5/6(RNAi), <br /> n = 10; par-3(RNAi), n = 9; par-1(RNAi), n = 10. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare control RNAi conditions anterior posterior blastomeres, yielding following p-values: mex-5/6(RNAi), anterior: p = 4.42&#195;&#8212;10&#226;&#8364;&#8220;7; posterior: p = 1.46&#195;&#8212;10&#226;&#8364;&#8220;3; par-3(RNAi), <br /> anterior: p = 9.32&#195;&#8212;10&#226;&#8364;&#8220;3; posterior: p = 0.04; par-1(RNAi), anterior: p = 0.741; posterior: p = 1.81&#195;&#8212;10&#226;&#8364;&#8220;5. F-G 4-cell embryos expressing YFP-ERA-1[3&#226;&#8364;&#8482; pie-1] control (F) mex-5/6(RNAi) (G). Red arrowheads F indicate cytoplasmic foci resembling endomembranes, suggesting ERA-1 traffics endosomal compartments. H Signal intensities cell-cell boundaries ABa/ABp EMS/P2 relative GPA-16 (Materials Methods). Number embryos quantified: control, n = 10; mex-5/6(RNAi), n = 8. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare control mex-5/6 <br /> (RNAi) anterior posterior blastomeres, yielding following p-values: mex-5/6 (RNAi), anterior: p = 0.113; posterior: p = 0.205. Working model MEX-5/6-mediated control era-1 mRNA asymmetric distribution P0 translational activation 2-cell embryos; main text. <br /> doi:10.1371/journal.pone.0120984.g004 <br />  <br />  <br />                                                 MEX-5/6 positively regulates era-1 translation era-1 3&#226;&#8364;&#8482;UTR <br />                                                 era-1 mRNA YFP-ERA-1 protein enriched anterior blastomeres,                                                 distribution protein dictated merely mRNA. Alternatively                                                 addition, translational activation occur anterior blastomeres. factors                                                 responsible putative activation? MEX-5 MEX-6 activate translation mRNAs <br />                                                 mex-3 glp-1 anterior blastomeres [9]. Compatible possibility,                                                 MEX-5/6 target consensus sequences  e.  6 U 13 nucleotides [12] era-1 <br />                                                 3&#226;&#8364;&#8482;UTR, additional ones era-1 coding sequence (S2A Fig.). <br />                                                    depleted MEX-5/6 explore regulate era-1 mRNA translation. Strik- <br />                                                 ingly, YFP-ERA-1[3&#226;&#8364;&#8482;era-1] protein levels greatly reduced mex-5/6(RNAi) <br />                                                 embryos (Figs. 3C, 4B, 4E). RT-qPCR analysis revealed slight statistically non-significant <br />                                                 decrease mRNA levels (S2B Fig.), suggesting reduced YFP signal                                                 mRNA degradation. Given MEX-5/6 play role downstream PAR proteins                                                 establishing P polarity [9], given era-1 mRNA distribution homogenous mex-5 <br />                                                 (RNAi) embryos, principle uniform distribution merely reflect polarity defects. <br />                                                  view unlikely, YFP-ERA-1[3&#226;&#8364;&#8482;era-1] levels lower mex-5/6 <br />                                                 (RNAi) embryos par-3(RNAi) embryos, polarity likewise affected (Fig. 4B, <br />                                                 4C, 4E). note, depletion translational regulator MEX-3 did affect YFP-ERA-1 <br />                                                 [3&#226;&#8364;&#8482;era-1] protein levels (S2C Fig.), indicating specificity effect incurred following MEX- <br />                                                 5/6 depletion. Furthermore, levels YFP-ERA-1[3&#226;&#8364;&#8482;pie-1] protein unaf- <br />                                                 fected mex-5/6(RNAi) embryos (Figs. 3D, 4G), indicating translational activation occurs <br />                                                 era-1 3&#226;&#8364;&#8482;UTR. <br />                                                    reasoned MEX-5/6 promote translational activation era-1 mRNA suggested <br />                                                 experiments, posterior YFP-ERA-1[3&#226;&#8364;&#8482;era-1] protein levels higher em- <br />                                                 bryos depleted PAR-1, MEX-5/6 activity increased posterior [8]. shown <br />                                                 Fig. 4D 4E, case  Overall, observations lead                                                 conclude MEX-5/6 needed activate translation era-1 mRNA                                                 3&#226;&#8364;&#8482;UTR embryogenesis. <br />  <br />                                                 Conclusions <br />                                                 findings suggest following working model regulation era-1 mRNA pro- <br />                                                 tein asymmetric distribution (Fig. 4I). model, era-1 mRNA anterior enrichment regu- <br />                                                 lated P polarity cues, MEX-5/6 era-1 3&#226;&#8364;&#8482;UTR. MEX-5/6 potentially <br />                                                 transport era-1 mRNA anterior, giving rise asymmetric distribution (Fig. 4I). Fur- <br />                                                 thermore, MEX-5/6 act 3&#226;&#8364;&#8482;UTR, directly indirectly, activate translation                                                 era-1 mRNA embryo. findings provide novel insights mechanisms                                                 mRNA distribution coupled translational regulation contribute robust develop- <br />                                                 ment C. elegans embryos. <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                                              11 / 14 <br />                                                                                             Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          Supporting Information <br />                                          S1 Fig. C. elegans specific era-1 gene contributes robust embryonic development.  <br />                                          Schematic wild-type era-1 gene era-1(tm5852) era-1(tm6426) mutant alleles, <br />                                          indication premature STOP codons (lack translation depicted grey). Analysis <br />                                          Scanprosit [1] failed identify clear protein domains ERA-1. B. Phylogenetic relation- <br />                                          ships ERA-1-related proteins select nematodes identified Blast search <br />                                          (Ensembl). tree built Clustal W2 software (http://www.ebi.ac.uk/Tools/ <br />                                          phylogeny/clustalw2_phylogeny/) sequence alignment performed MAFFT server <br />                                          (http://mafft.cbrc.jp/alignment/server/). values indicate length branch leading                                          previous node number substitutions proportion alignment <br />                                          length. C. Embryonic lethality following partial depletion PAR-3 PAR-2 wild type                                          era-1(tm6426) worms. &gt;800 embryos scored 8 independent experiments par-3 <br />                                          (RNAi), &gt;200 embryos 3 independent experiments par-2(RNAi). Note enhance- <br />                                          ment observed glp-1(RNAi) apx-1(RNAi) compared wild type, indicating <br />                                          enhanced lethality observed par-3(RNAi) par-2(RNAi) generic effect <br />                                          response RNAi. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test com- <br />                                          pare embryonic lethality wild type versus era-1(tm6426) indicated conditions, yielding <br />                                          following p-values: partial par-3(RNAi), p = 4.6&#195;&#8212;10&#226;&#710;&#8217;3; partial par-2(RNAi), p = 0.048. D-E. <br />                                          YFP-ERA-1[3&#226;&#8364;&#8482;era-1] protein telophase cell division (D) 2-cell stage (E). <br />                                          upper images YFP signal  lower ones merge. Scale bar represents <br />                                          10 microns. <br />                                          1. Castro E, Sigrist CJ, Gattiker  Bulliard V, Langendijk-Genevaux PS, Gasteiger E, et al. <br />                                          (2006) ScanProsite: detection PROSITE signature matches ProRule-associated function- <br />                                          al structural residues proteins. Nucleic acids research 34: W362&#226;&#8364;&#8220;365. <br />                                          (PDF) <br />                                          S2 Fig. MEX-5-dependent translational regulation era-1 mRNA potential MEX-5 <br />                                          binding sites.  Putative MEX-5 binding sites (red) era-1 exons (blue)                                          3&#226;&#8364;&#8482;UTR (green). sequences regions numbered 1&#226;&#8364;&#8220;7 shown red. B. mRNA <br />                                          levels yfp-era-1[3&#226;&#8364;&#8482;era-1], yfp[3&#226;&#8364;&#8482;era-1] yfp-era-1[3&#226;&#8364;&#8482;pie-1] measured RT-qPCR                                          MEX-5 depletion (Materials Methods). Values shown relative act-1 mRNA. <br />                                          experiment performed 3 times. Average values indicated error bars represent- <br />                                          ing standard error mean. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s <br />                                          t-test compare mRNA levels control mex-5(RNAi) conditions, yielding following <br />                                          p-values: yfp-era-1[3&#226;&#8364;&#8482;era-1], p = 0.429; yfp[3&#226;&#8364;&#8482;era-1], p = 0.234, yfp-era-1[3&#226;&#8364;&#8482;pie-1], p = 0.284. <br />                                          C. Quantification membrane enrichment relative GPA-16 4-cell stage embryos express- <br />                                          ing YFP-ERA-1[3&#226;&#8364;&#8482;era-1] mex-5 mex-3 depletion, control embryos (Materials <br />                                          Methods). Number embryos quantified: control, n = 11; mex-5(RNAi), n = 7; mex-3 <br />                                          (RNAi), n = 7. Statistical analysis performed using unpaired Student&#226;&#8364;&#8482;s t-test compare <br />                                          control RNAi conditions anterior posterior blastomeres, yielding follow- <br />                                          ing p-values: mex-5(RNAi), anterior: p = 0.021; posterior: p = 0.012; mex-3(RNAi), anterior: <br />                                          p = 0.465; posterior: p = 0.124 Note relative levels  e. levels YFP-ERA-1 <br />                                          [3&#226;&#8364;&#8482;era-1] versus GPA-16) lower Fig. 4E variation signal intensities <br />                                          experiment experiment, ratio anterior posterior YFP-ERA-1[3&#226;&#8364;&#8482;era-1] <br />                                          signals comparable experiments. <br />                                          (PDF) <br />                                          S1 Table. Exact values statistical analysis. designation experiment  col- <br />                                          umn), nature measurement (second column), genotype/RNAi condition   <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0120984 March 30, 2015                                                                            12 / 14 <br />                                                                                                  Asymmetric mRNA Distribution C. elegans Embryos <br />  <br />  <br />  <br />                                          anterior  P: posterior   column), actual value (fourth column), number <br />                                          embryos experiments analyzed (fifth column), p-value using unpaired Student&#226;&#8364;&#8482;s t-test <br />                                          (sixth column) corresponding Figure  column) reported. <br />                                          (PDF) <br /> </body></html>