<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Bacteriophage T7 gene 2.5 protein: essential protein DNA replication <br />      (DNA binding protein/recombination/T7 DNA polymerase) <br />  <br /> ABSTRACT           product gene 2.5 bacteriophage T7,                      15). direct indirect evidence support interaction single-stranded DNA binding protein, physically interacts                     essential replication proteins T7 gene phage-encoded gene S protein (DNA polymerase) gene                           2.5 protein. <br /> 4 proteins (helicase primase) stimulates activities.                      T7 gene 2.5 protein stimulates DNA synthesis catalyzed Genetic analysis T7 phage defective gene 2.5 shows                    T7 DNA polymerase/thioredoxin complex single- <br /> gene 2.5 protein essential T7 DNA replication growth.                     stranded DNA templates (4, 5, 11, 14) increases T7 phages contain null mutants gene 2.5 constructed                     processivity reaction (11). shown affinity homologous recombination. gene 2.5 null mutants                             chromatography fluorescence emission anisotropy contain deletion gene 2.5 (T7A2.5) insertion                  T7 DNA polymerase gene 2.5 protein physically interact <br /> gene 2.5 grow Escherichia coli (efficiency                          dissociation constant 1.1 ,uM (11). Similarly, inter- <br /> plating, &lt;10-8). infection E. coli T7A2.5, host                        action T7 helicase/primase gene 2.5 protein DNA synthesis shut  phage DNA synthesis reduced                        inferred ability gene 2.5 protein stimulate &lt;1% phage DNA synthesis wild-type T7-infected E. coli                       synthesis primers (10, 16). T7 helicase/primase-gene <br /> cells measured incorporation [3H]<span id='am-33' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>thymidine</span>.                          2.5 protein interaction confirmed affinity <br /> trast, RNA synthesis essentially normal T7A2.5-infected                        chromatography (11). recently (unpublished <br /> cells. defects growth DNA replication overcome                        data) C-terminal acidic domain gene 2.5 protein wild-type gene 2.5 protein expressed plasmid har-                          required T7 growth vivo participates gene 2.5 <br /> boring T7 gene 2.5.                                                              dimerization protein-protein interactions. <br />                                                                                         role gene 2.5 protein recombination Single-stranded DNA binding proteins (SSBs), Esch-                           understood. Sadowski et al. (17) demonstrated extracts <br /> erichia coli SSB T4 gene 32 protein, essential                               T7 phage-infected cells contain activity promotes <br /> components DNA metabolism prokaryotic cells (1-3).                             renaturation complementary single strands suggested gene 2.5 protein bacteriophage T7, originally isolated                        activity resided T7 SSB. Recent studies (S. <br /> based strong affinity single-stranded DNA                         Tabor C.C.R., unpublished results)  fact, dem- <br /> ability stimulate DNA synthesis T7 DNA polymerase                              onstrated gene 2.5 protein facilitates renaturation (4, 5), thought analogous characterized                       homologous single-stranded DNA efficiently SSBs. Like E. coli SSB T4 gene 32 protein, gene 2.5                              does E. coli recA protein, E. coli SSB, T4 gene 32 protein. <br /> protein implicated T7 DNA replication, recom-                               determination definitive role T7 gene 2.5 <br /> bination, repair (6-11). purified gene 2.5 protein                       protein DNA replication recombination vivo cells overexpressing gene characterized physical                         dependent genetic analysis gene 2.5 mutants. Previ- <br /> properties interactions DNA (6). Gene 2.5 protein                           ously, T7 phage containing mutations gene 2.5                                                                                      isolated based inability grow E. coli strains exists dimer identical subunits Mr 25,562.                         defective SSB (8). T7 mutant phages contain binds specifically single-stranded DNA stoichiom-                          amber mutation gene 2.5 leads synthesis etry -7 nt bound monomer gene 2.5 protein                              shortened polypeptide -90% length wild-type <br /> extends length DNA molecules measured                               protein (9). gene 2.5 mutant phages grow electron microscopy. binding constant gene 2.5 pro-                           wild-type E. coli strains strains expressing tein single-stranded DNA -2.5 x 106 M-1, deter-                            temperature-sensitive SSB nonpermissive tempera- <br /> mined fluorescence quenching nitrocellulose filter                            ture. Furthermore, T7 phages defective recom- <br /> binding assays (6). Fluorescence studies suggest tyrosine                       bination (8). Recently,  mutational analysis residue(s) gene 2.5 protein interacts single-stranded                        identified T7 gene 2.5 mutants grow E. <br /> DNA, tryptophan residues (6).                                         coli strains producing wild-type SSB (F. W. Studier,  <br />    T7 DNA replication, gene 2.5 protein                               sonal communication). T7 phages potential play essential roles. T7                          deletion gene 2.5 (T7A2.5) grow wild-type E. coli <br /> replication fork, proteins, products T7 genes 4                         detectable T7 DNA replication; T7A2.5 5 host trxA gene, account fundamental                            phages grow normally E. coli strains expressing wild-type <br /> reactions (12, 13). Gene 5 protein DNA polymerase,                              gene 2.5 plasmid. <br /> catalyzing polymerization nucleotides low pro- <br /> cessivity (14, 15). E. coli thioredoxin, product trxA                                  MATERIALS METHODS <br /> gene, binds gene S protein 1:1 stoichiometry                               Bacterial Strains. E. coli HMS157 (F- recB21 recC22 <br /> confers processivity polymerization reaction                            sbcA5 endA gal thi sup) (laboratory collection), E. coli <br /> creasing affinity enzyme primer/template (14, <br />                                                                                      Abbreviations: SSB, single-stranded DNA binding protein; moi, publication costs article defrayed page charge           multiplicity infection. <br /> payment. article marked &quot;advertisement&quot;                *Present address: Department Microbiology, National Fisheries accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                      University Pusan, Pusan, 608-737, Korea. <br />                                                                              10173 <br />  10174    Biochemistry: Kim Richardson                                       Proc. Natl. Acad. Sci. USA 90 (1993) <br /> HMS174 (F- hsdR rK,2- mKj2+ recAl) (18), E. coli HB101            using primers EcoRI site primers <br /> [F- mcrCmrr) leu supE44 aral4 galK2 lacYl proA2                  (5&apos;-CGACGAATTCTAAGTGGAACTGCGGG-3&apos; 5&apos;- <br /> rpsL20(Strr) xyl-5 mtl-l recA13] (19), E. coli HMS262 (F-         CGACGAATTCCCTTTAGCGCCGTAAC-3&apos;). PCR- <br /> hsdR pro leu-lac-thi-supE tonA-trxA) (laboratory collec-          generated fragment T7 gene 2 cloned BamHI- <br /> tion), E. coli JH21(F- pcnB80, AtrxA307) (20), E. coli        linearized pBR322 generate pGP2. pGP2 digested AN1(F- AtrxA307, metE::TnlO), derivative E. coli C600        EcoRI, PCR-generated gene 2.8 fragment cloned <br /> (21) P1 transduction (22) described. Growth        EcoRI site pGP2. resulting plasmid pGP2- manipulation bacteriophage T7 E. coli             2.8. Finally, PCR-produced trxA DNA cloned performed described (23, 24).                                  pGP2-2.8 digested Sma junction <br />    Construction Gene 2.5 Plasmids. DNA fragments          genes 2 2.8. isolation plasmid containing gene <br /> prepared cloned standard procedures (25). E. coli          2-trxA-gene 2.8 determined PCR suitable prim- <br /> HB101 transformed designated plasmids (26). different plasmids containing wild-type gene 2.5     ers. resulting plasmid, pGP2T2.8, expresses functional <br /> constructed standard PCR protocol (27). construct         thioredoxin. pGP2.5::trxA pGP2T2.8 transformed plasmid, primers, 5&apos;-end primer Nde       toE. coli HMS157, generating E. coli HMS157/pGP2.5::trxA site (5&apos;-CGTAGGATCCATATGGCTAAGAAGATTT-                          E. coli HMS157/pGP2T2.8, respectively. E. coli <br /> TCACCTC-3&apos;) 3&apos;-end primer BamHI site (5&apos;-            HMS157/pGP2T2.8 grown LB medium 37&#194;&#176;C CGTAGGATCCACTTAGAAGTCTCCGTC-3&apos;), used                        vigorous shaking A600 = 0.5 infected [multiplicity amplify T7 DNA sequences 9157-9862 containing gene 2.5         infection (moi) = 0.1] wild-type T7 phage. cell <br /> coding sequence (28). PCR-generated DNA fragments            lysis, phages harvested described (31). select incubated large fragment E. coli DNA polymerase       amplify recombinant (T7A2.5::trxA), HMS262 cells  presence <span id='am-16' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span>nucleoside</span> 5&apos;-triphosphates,    taining pGP2.5-1 supplements wild-type gene 2.5 protein resulting fragments containing blunt ends           grown midexponential phase infected ligated EcoRV site plasmid pBR322 (29).           lysate produced wild-type T7 infection E. coli <br /> resulting plasmid pGP2.5-1 contains gene 2.5 protein          HMS157/pGP2T2.8. cell lysis, amplified T7A2.5::trxA <br /> coding sequence control pBR322 tetracycline          plated E. coli HMS262/pGP2.5-1, presence gene promoter.                                                    trxA gene absence gene 2.5 T7 genome <br />    second plasmid contains gene 2.5 control      confirmed PCR restriction analysis individual T7 RNA polymerase promoter. 700-bp Nde BamHI              T7 plaques. resulting T7 phage (T7A2.5::trxA) substi- <br /> fragment containing gene 2.5 excised pGP2.5-1        tutes trxA gene 2.5. Isolation T72.5::trxA phage cloned Nde BamHI sites plasmid pT7-7,      carried analogous procedure plasmid <br /> generating plasmid pGP2.5-2. Plasmid pT7-7, constructed        pGP2.5::trxA instead pGP2T2.8. <br /> S. Tabor (Harvard Medical School, Boston), contains T7           Measurements DNA RNA Synthesis. DNA RNA <br /> RNA polymerase promoter 410 strong translation       synthesis measured essentially described (32). E. coli <br /> initiation region prior polylinker (30).                   HMS174 cells grown shaking 30&#194;&#176;C M9 CAA <br />    plasmid, pGP2.5-3, contains T7 sequence       medium. 3 x 108 cells ml, bacteria infected C-terminal half gene 2, entire sequence gene 2.5,   indicated T7 phage moi = 7. indicated gene 2.8 (T7 sequence nt 8961-10279). Plasmid       times, aliquots (0.2 ml) phage-infected cells removed <br /> pGP2.5-3 constructed follows. Using primers        placed tubes containing 10 ,ul [3H]thymidine (50 <br /> 5&apos;-CGTAGGATTCTCGGAGCATTCC-3&apos; 5&apos;-CGTAG-                        ,uCi/ml; 1 Ci = 37 GBq) [3H]<span id='am-28' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Uridine" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span>uridine</span> (50 ,uCi/ml) GATCCACTTAGAAGTCTCCGTC-3&apos;, PCR-generated                        measure DNA RNA synthesis, respectively. 90 sec <br /> fragment created inserted blunt-end ligation      incubation 30&#194;&#176;C, growth terminated addition plasmid pBR322 linearized BamHI digestion        3 ml cold 5% trichloroacetic acid. acid-insoluble followed filing using large fragment E. coli     material collected GF/C filters (Whatman) DNA polymerase  resulting pGP2.5-3 clone se-            washed times 3 ml ice-cold 1 M HCl lected shown contain insert correct size     times 3 ml 95% ethanol. acid-insoluble radioac- <br /> orientation. clones generated PCR-amplified          tivity measured toluene-based solvent liquid <br /> DNA sequenced free mutations.             scintillation counter. <br />    Construction Gene 2.5 Mutants Bacteriophage T7.             Vivo Labeling T7 DNA. E. coli HMS174 cells Gene 2.5 mutant phages constructed homologous <br /> recombination (18). construct gene 2.5 null mutant,       grown 3 x 108 cells ml M9 CAA medium infected <br /> used different plasmids vitro recombination sub-        various T7 phage moi = 3-5. desired times strates. plasmid, pGP2.5::trxA, inserts E. coli     infection, aliquots (2 ml) removed [3H]thymidine <br /> thioredoxin gene (trxA) middle gene 2.5.          (80 Ci/mmol) added infected cells (final concen- <br /> second plasmid, pGP2.5T2.8, replaces gene 2.5 T7      tration, 40 Ci/ml). Labeling stopped addition E. coli thioredoxin gene (trxA). Plasmid pGP2.5::trxA         equal volume freshly stop solution (75% ethanol/2% <br /> constructed follows. E. coli trxA amplified PCR         phenol/21 mM CH3COONa, pH 5/2 mM EDTA). ra- primers, containing Mlu site        dioactively labeled T7 DNA isolated described (33), <br /> sequence (5&apos;-CAACACGCGTGGCTTATTCCTGTG-3&apos;                      cleaved Xmn  electrophoresed 0.8% <br /> 5&apos;-CGAACGCGTTTAAGCCAGGTTAG-3&apos;). PCR-ampli-                        agarose gel TBE (12.1 g Tris base liter/4.1 g boric <br /> fled DNA fragments trxA cloned pGP2.5,         acid liter/0.74 g Na2EDTA liter) =0.5 V/cm. linear cleavage Mlu  generating           gels treated EN3HANCE (NEN), dried, pGP2.5::trxA. resulting plasmid expresses truncated         exposed XAR Kodak film intensifying screen gene 2.5 protein active thioredoxin. Plasmid pGP2T2.8      -70&#194;&#176;C; bands identified autoradiography. constructed follows. gene 2, 8961-9096         Methods. Plating efficiencies T7 wild-type <span id='am-8' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-9' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> sequence T7 DNA, amplified using         T7A2.5::trxA phage measured follows. Bacterial <br /> primers containing BamHI site primers (5&apos;-          strains grown 2 x 108 cells ml. Various T7 phages <br /> CGTAGGATCCTCGGAGCATTCC-3&apos; 5&apos;-GAAT-                            LB diluted (0.1 ml) mixed 0.2 ml bacterial <br /> TCGTCGACGGATCCCGGGCGTATTACTTCGGTGCT-                              culture 3 ml agar, plated LB LB/ampicillin <br /> 3&apos;). Similarly, gene 2.8 (nt 9854-10279) amplified         plates, allowed incubate 30&#194;&#176;C. <br />            Biochemistry: Kim Richardson                                                         Proc. Natl. Acad. Sci. USA 90 (1993)           10175 <br />                            RESULTS                                             Table 1. Plating efficiencies T7 phages various E. <br />                                                                                coli strains <br />    Construction Bacteriophage T7 Gene 2.5 Mutants Gene 2.5 Expression Vectors. T7 phage defective gene 2.5                                                       Efficiency plating constructed homologous recombination. shown                           Strain/plasmid        17 (WT)      T7A2.5::trxA       T72.5::trxA Fig. 1, plasmid pGP2T2.8 provides flanking se-                      HMS262 trxA-                0            &lt;10-9             &lt;10-9 <br /> quences (gene 2 gene 2.8) gene 2.5 genetic                          HMS262/pGP2.5-1          &lt;10-9            0.95              0.92 <br /> crossover replacement gene 2.5 E. coli                  JH21/pGP2.5-1            &lt;10-9            0.93              0.95 <br /> thioredoxin gene trxA. Recombinant T7 phage having gene                        AN1/pGP2.5-1             &lt;10-9            0.94              0.92 <br /> 2.5 replaced trxA (T7A2.5::trxA) resulting cross-                    AN1/pGP2.5-2*            &lt;10-9            0.91              0.93 selected growth E. coli trxA- strain (E.                   AN1/pGP2.5-3*            &lt;10-9            0.93              0.95 <br /> coli HMS262) wild-type gene 2.5 protein provided                   HMS157                      1           &lt;10-8t              10-2 ofthe plasmids containing gene 2.5 (pGP2.5-1, <br /> pGP2.5-2, pGP2.5-3). pGP2.5-1 pGP2.5-2 contain gene                          Efficiency plating calculated dividing number 2.5 control tetracycline gene promoter                     plaque-forming units given strain number wild-type <br /> pBR322 T7 RNA polymerase promoter pT7-7,                            (WT) 17 plaque-forming units E. coli HMS157. 17A2.5::trxA respectively. pGP2.5-3 contains gene 2.5 flanked                        172.5::trxA phages selected propagated E. coli <br />                                                                                HMS262/pGP2.5-1 cells. <br /> proximal half gene 2 gene 2.8                  *E. coli HMS262/pGP2.5-2, JH21/pGP2.5-2, HMS262/pGP2.5-3,  expression gene 2.5 control                   JH21/pGP2.5-3 cells similar values efficiency plating promoter. T7A2.5::trxA lacks DNA sequence nt                         17A2.5::trxA 172.5::trxA phages. <br /> 9097-9853 T7 DNA molecule (28). region contains                    tWhen T7A2.5::trxA selected propagated E. coli entire gene 2.5 sequence promoter gene                   HMS262/pGP2.5-2 HMS262/pGP2.5-3 cells, wild-type 17 <br /> 2.5. second T7 gene 2.5 mutant, T72.5::trxA, prepared                         phages arose frequencies 10-5 10-2, respectively. <br /> similarly T7A2.5::trxA, E. coli trxA gene inserted coding sequence gene 2.5.                                          E. coli HMS262/pGP2.5-1 grow E. coli HMS262 <br />    T7A2.5::trxA phages grown E. coli HMS262                        trxA- E. coli HMS157; plating efficiencies &lt;10-9 <br /> containing plasmid pGP2.5-2 pGP2.5-3, wild-type                      &lt;10-8, respectively. Similar results obtained T7 phage appears resulting lysate frequency 10o-                T7A2.5: :trxA phage plated trxA mutant 10-2, respectively (Table 1). Presumably, wild-type                   strains-E. coli JH21 AN1 (data shown). T7 phage <br /> T7 phages produced recombination                                insertion E. coli thioredoxin gene gene 2.5 <br /> T7A2.5::trxA complementing plasmid (pGP2.5-2                        (T72.5::trxA) grow strains, pGP2.5-3).  complementation provided                        wild-type T7 revertants arise frequency -2% (titered <br /> pGP2.5-1, wild-type T7 revertants (&lt;10-8) detected                      E. coli HMS157) result homologous recombination sequence homology exists flanking                         propagation (Table 1). control experiments, regions gene 2.5 promoter regions T7. case                 gene 2.5 mutant T7 phages grow normally E. coli strains T72.5::trxA, wild-type revertants arise frequency                   expressing gene 2.5 plasmid harboring cloned gene <br /> %2% homologous recombination propagation                         (Table 1). results gene 2.5 essential E. coli strains used complementation.                                  growth T7. <br />    Gene 2.5 Essential T7 Growth. Biochemical evidence                      DNA RNA Synthesis T7A2.5-Infected Cells. suggests gene 2.5 protein plays crucial roles                      attempt define biochemical lesions responsible aspects T7 DNA metabolism. Consequently,                         inability T7 gene 2.5 mutants grow E. coli cells, used T7 gene 2.5 mutants T7A2.5::trxA                             examined DNA RNA synthesis E. coli cells infected <br /> T72.5: :trxA examine role gene 2.5 protein vivo.               mutant phage. kinetics DNA synthesis shown Table 1, T7A2.5::trxA phage selected amplified                    infection E. coli HMS174 T7A2.5::trxA phage                                                                                compared (Fig. 2) obtained wild-type phage <br />                                                                                identical conditions. wild-type T7-infected cells                                                                                rate DNA synthesis increased rapidly %7 min infec- <br />                                                                                tion reached maximum rate 20 min infection,                                             pGP2T2.8                     ori   cells begin lyse. striking contrast,                                                                                decrease DNA synthesis infection 17A2.5::trxA, <br />                                 BamHI                         EcoRI            presumably shut host DNA synthesis. 20 <br />                                                                                min infection, time wild-type T7 DNA replica- <br />                                                                                tion maximal, detectable DNA synthesis                T7WT           H ~~~Gene 2       Gene 2.     8                  mutant-infected cells (Fig. 2). defect DNA synthesis <br />          T7WT             [      Gene         Gee.         ee. <br />                                                                                       60 <br />                                                    Select A2.5::tx,4                     T7 WT <br />  <br />                                                                                                           T7A2.5::trxA / <br />          T 7A2.5:: trxA       [::1              1         }                           40                   pGP2.5 (WTl <br />                                                                                 O                   \      X            FIG. 2. Time course DNA <br />   FIG. 1. Construction T7A<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>;2.5::trxA. Plasmid pGP2T2.8                                                         synthesis wild-type (WT) 17- <br /> derivative pBR322 containing 17 gene 2-E. coli thioredoxin gene              c&amp;.           ]        2\              17A&amp;2.5-infected cells. Rates <br /> (trxA)-T7 gene 2.8 sequences BamHI EcoRI sites.                       20 20                            DNA    synthesis measured <br /> DNA fragments gene 2, gene 2.8, E. coli trxA gene                                         l / /       bas described various intervals <br /> amplified PCR, pGP2T2.8 constructed described.                                                           infection E. coli HMS174 wild-type T7 infection HMS157/pGP2T2.8, pGP2T2.8                                                             cells T7 wild-type (o) recombined homologously 17 wild-type (WT) phage generate                                  1T7A2.5::trx4 1)&gt;&gt;T7A2.5::trxA (o) phages T7A2.5::trxA phage. presence trxA gene                                                          infection E. coli HMS174/ <br /> absence gene 2.5 17A2.5::trxA phage confirmed PCR                         0             20          40    pGP2.5-lwithT7A2.5::trxAphage restriction analysis.                                                                   TimeAfterInfection (min.)  (v) moi 7 30&#194;&#176;C. <br />  10176              Biochemistry: Kim Richardson                                                                Proc. Natl. Acad Sci. USA 90 (1993) <br />                                                                          T7A2.5 /                    infection T7 wild-type phage identical  <br />                               T7 WT  T7A2.5 pGP2.5WT                                                 ditions (Fig. 4). <br />                       Min.: 5 10 15 5 10 15 5 10 15 <br />                                                                                                  1                            DISCUSSION <br />                                                                                         J:~2 <br />                                                                                         - <br />                                                                                                      used genetic approach examine role                                                                              .            -J       gene 2.5 protein bacteriophage T7. T7 lacking gene 2.5 <br />                                                                                         -5 <br />                                                                                         -6           does grow E. coli defective DNA synthesis. <br />                                                                                         -7           defects overcome gene 2.5 protein produced <br />                                                                                                      plasmid containing cloned gene 2.5. Earlier studies <br />                                                                                         _8           suggested E. coli SSB substitute T7 <br />                                                                                         -9 <br />                                                                                                      gene 2.5 protein vivo (8).  gene 2.5 mutant <br />                                                                                         -10 <br />                                                                                                      used studies produced shortened polypeptide                                                                                         -   12       contained significant portion intact gene 2.5 protein <br />                                                                                                      (9). results clearly E. coli SSB substi- <br />                                      1     2       3    4       5    6   7   8      9                tute gene 2.5 protein vivo. <br />                                                                                                         multiple roles SSBs DNA metabolism    FIG. 3. Replication wild-type 17 T7A2.5 phage DNA. E.                                      difficult define precisely essential nature vivo. coli HMS174 cells infected wild-type T7 phage (T7 WT)                                   E. coli, example, SSB plays important roles DNA <br /> T7A2.5::trxA (M7A2.5). E. coli HMS174/pGP2.5-1 cells                                         replication DNA recombination repair (1-3, <br /> fected T7A2.5::trxA phage [T7A2.5/pGP2.5(WT)]. 5 (lanes 1,                                   34). Strains carrying mutations structural gene SSB, <br /> 4, 7), 10 (lanes 2, 5, 8), 15 (lanes 3, 6, 9) min                                    ssb, temperature sensitive growth DNA synthesis <br /> postinfection, cells labeled [3H]thymidine 35 sec. labeled 17 DNA isolated, digested Xmn  treated                                     (35) defective DNA repair recombination (36). <br /> described. Restriction fragments numbered 1-12                                       T7 gene 2.5 protein E. coli SSB extensive sequence <br /> catameric joint fragment labeled J.                                                               homology; 24% amino acids identical                                                                                                      47% represent conserved changes (37). bacteriophage T4, T7 gene 2.5 mutant-infected cells overcome                                        roles gene 32 protein characterized <br /> complementation gene 2.5 protein provided                                                vitro vivo. Genetic studies demonstrated <br /> plasmid (Fig. 2).  pattern DNA synthesis                                           product gene 32 essential DNA replication, <br /> E. coli HMS174/pGP2.5-1 infected T7A2.5: :trxA                                                  recombination, repair (38-42). binds cooperatively showed pattern different wild-type T7 phage                                           single-stranded DNA, affects denaturation renaturation <br /> infection onset DNA synthesis delayed                                         DNA, interacts specifically T4 DNA polymerase <br /> reached maximal rate =40 min postinfection.                                                     recombination proteins uvsX uvsY (43-46). <br />    examine T7 DNA synthesis specifically                                                 Gene 2.5 protein shares structural features  radioactively labeled newly synthesized DNA                                           SSBs (1). proteins possess DNA binding  pulse-labeling [3H]thymidine phage infection,                                          mains acidic C-terminal domains presumed cleaved DNA restriction enzyme Xmn                                                involved protein-protein interactions (47). Studies examined pattern labeling DNA fragments                                             structural features binding properties gene 2.5 protein <br /> electrophoreses agarose gel (Fig. 3). E. coli                                             indicate gene 2.5 protein contains distinctive func- <br /> HMS174 HMS174/pGP2.5-1 cells infected T7                                                tional domains: N-terminal single-stranded DNA binding <br /> wild-type T7A&amp;2.5::trxA phages logarithmic-phase                                           domain C-terminal acidic domain (11). Tyrosine resi- <br /> growth moi        5. 10 15 min postinfection,                                                                                                                            dues DNA binding domain required gene 2.5 <br /> fragments T7A2.5::trxA DNA derived T7A2.5-                                                   protein binding single-stranded DNA (11). recent <br /> infected E. coli HMS174/pGP2.5-1 cells (lanes 8 9)                                           studies (unpublished data) C-terminal acidic <br /> labeled obtained E. coli HMS174 infected                                           domain gene 2.5 protein essential gene 2.5 protein wild-type T7 phage (lanes 2 3).                                                  function vivo. truncated form gene 2.5 protein <br /> radioactively labeled T7 DNA fragments detected E.                                           lacking C-terminal 21-<span id='am-21' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-22' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> residues support <br /> coli HMS174 cells infected T7A2.5 phage (lanes 5 6).                                        growth T7 phage lacking gene 2.5. addition, addition, shut host DNA synthesis T7A2.5-                                           purified truncated gene 2.5 protein longer form dimers <br />                                                                                                      physically interact T7 DNA polymerase; <br /> infected cells visualized experiment accounts                                         ability bind single-stranded DNA decrease DNA synthesis infection. ex-                                            affected. <br /> periments, conclude gene 2.5 essential T7 DNA                                            gene 2.5 protein essential T7 growth replication.                                                                                         replication? essential nature protein does arise <br />    DNA synthesis observed T7A2.5-infected                                          solely ability bind single-stranded DNA E. <br /> cells, RNA synthesis grossly normal (Fig. 4). time                                            coli SSB, protein higher affinity course ofthe rate RNA synthesis T7A2.5 phage-infected <br /> cells did differ significantly observed cells <br />                                                                                                      single-stranded DNA, substitute gene 2.5 protein <br />                                                                                                      vivo.  postulate gene 2.5 protein essential <br />                                                                                                      reasons:   specific interaction phage- <br />                                                                                                      encoded replication proteins, (ii) role recombina- <br />      60 <br />                                                                                                      tion. Biochemical studies purified gene 2.5 protein <br />            c              T7   Wr <br />                                                                                                      demonstrated physical interaction T7 DNA <br />                                                                                                      polymerase T7 helicase/primase (11) stimulation X~   40                                                    FIG. 4. Time course RNA                activities proteins (4, 5, 10, 11, 14, 16). Precisely <br />                                                         synthesis. rate RNA synthe-           specific interactions occur 17,                                                          sis measured pulse labeling <br />      20X                                                                                             known SSBs provide similar degrees       620      T7 A2.5::tixA <br />                            &#194;&#176;                            [3H]uridine 90 sec 4-min <br />                                                         intervals infection E. coli         stimulation. exception specific stimulation <br />                                                         HMS174 cells 30&#194;&#176;C T7 wild-           primer synthesis gene 2.5 protein (10, 16). Presum- <br />        0         10       20        30    40       50   type (o) T7A2.5 phage (e) moi        ably, specific interaction proteins reflects                TimeAfter Infection       (min. )            =   7.                                   requirement highly ordered structure replication <br />  <br />  <br /> fork coordinate leading- lagging-strand DNA synthe-               <br /> sis.                                                                   <br />    Gene 2.5 required DNA synthesis              <br />                                                                       T7 recombination (8, 9). process, undoubtedly           interact recombination replication             <br /> proteins, T7 DNA polymerase primase/helicase.             biochemical studies support direct role           <br /> 17 gene 2.5 protein recombination. infection E.          <br /> coli 17 phage, single-stranded DNA renaturation                      <br /> activity induced, suggesting 17 gene 2.5 protein                 <br /> involved activity (17).  fact, recently      <br /> shown gene 2.5 protein facilitates renaturation                 <br /> single-stranded DNA efficiently does E. coli              <br /> SSB, recA, T4 gene 32 protein (S. Tabor C.C.R.,                        <br /> unpublished results). activity likely impor-     <br /> tant catalysis homologous pairing provide                   <br /> explanation essential role protein vivo               <br /> recombination (8, 9).                                                      <br />    mechanism defect recombination lead                     severe defect replication observed cells infected          gene 2.S 17 mutant? possibility            <br /> absence gene 2.5 protein catalysis homologous               <br /> pairing, recombination intermediates accumulate                   <br /> subsequently degraded. hand, quite likely                <br />                                                                        recombination plays essential role T7 DNA repli-               <br /> cation. genome bacteriophage T7 replicated       <br /> exclusively linear intermediates (concatemers) (48, 49).        <br /> mation concatemers provides intermediate                   <br /> 3&apos; ends 17 chromosome replicated. Concate-                  <br /> mers processed yield fully replicated duplex              <br /> chromosomes (50, 51). evidence indicates concate-                   <br /> mers formed vivo annealing single-stranded termi-                <br /> nally redundant ends replication intermediates (28).                <br /> propose gene 2.5 protein required rapid           <br /> efficient homologous pairing exposed terminal redun-                   <br /> dancy phage-infected cells. absence concatemer           <br /> formation, subsequent DNA synthesis defective,                <br />                                                                           <br /> terminating replication parental DNA.               <br /> round DNA synthesis represent 1%                total DNA synthesis usually observed phage-infected                    <br /> cells. regard, concatemer formation phage T7-                  <br /> infected cells requires recombination (52, 53).                                <br /> </body></html>