<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Host adaptation viruses relies genes different cross-resistance properties <br /> Nelson E. Martinsa,1, V&#195;&#173;tor G. Fariaa,1, Viola Nolteb, Christian Schl&#195;&#182;ttererb, Luis Teixeiraa, &#195;&#8240;lio Sucenaa,c,2,3, Sara Magalh&#195;&#163;esc,d,2,3  Instituto Gulbenkian Ci&#195;&#170;ncia, 2780-156 Oeiras, Portugal; bInstitut f&#195;&#188;r Populationsgenetik, University Veterinary Medicine Vienna, 1210 Vienna, Austria; <br /> c <br />  Departamento Biologia Animal, Faculdade Ci&#195;&#170;ncias, Universidade Lisboa, 1749-016 Lisbon, Portugal; dCentro Biologia Ambiental, Faculdade Ci&#195;&#170;ncias, Universidade Lisboa, 1749-016 Lisbon, Portugal <br />  <br /> Edited Trudy F. C. Mackay, North Carolina State University, Raleigh, NC, approved March 14, 2014 (received review January 9, 2014) <br />  <br /> Host adaptation parasite affect response                knowledge, study host&#226;&#8364;&#8220;parasite interactions  genetics direct correlated responses                   combined experimental evolution genomics. <br /> remains poorly studied. overlap responses                      important aspect experimental evolution instrumental understanding host evolution multi-                   allows measurement consequences evolving parasite environments. determined genetic phenotypic                  environment performance environments (3). <br /> changes underlying adaptation Drosophila melanogaster                       adaptation environment entail fitness  <br /> Drosophila C virus (DCV). 20 generations, flies selected                  crease environments, possibly hampering future evolu- DCV showed increased survival DCV infection,                 tion settings (17, 18). Despite common, costs cricket paralysis virus (CrPV) flock house virus (FHV)              universal (4) experiments (17).  <br /> fection. genome sequencing identified regions sig-                  adapting environment lead increased  <br /> nificant differentiation treatments, candidate                  formance environments (e.g., 5, 19). host&#226;&#8364;&#8220;parasite <br /> genes functionally tested RNAi. genes vali-                 interactions, question particularly important dated&#226;&#8364;&#8221;pastrel, known DCV-response gene, loci,                    epidemiological consequences infecting resisting mul- <br /> Ubc-E2H CG8492. Knockdown Ubc-E2H pastrel                        tiple hosts parasites, respectively. <br /> led increased sensitivity CrPV, knockdown                          Despite ample knowledge genes triggered parasite <br /> CG8492 increased susceptibility FHV infection.  Dro-                attacks Drosophila, key studies analyzed <br /> sophila adaptation DCV relies major genes,                   outbred fly population adapt given parasite (11&#226;&#8364;&#8220; <br /> different cross-resistance properties, conferring host resistance             13, 15).  genetic basis consequences parasites.                                                               adaptation host susceptibility parasites                                                                                  determined. <br />                           | <br /> host&#226;&#8364;&#8220;parasite interactions experimental evolution   | <br />                       | <br /> Drosophila immunity evolve resequence <br />                                                                                       Significance <br />  <br />  <br /> P   arasites impose strong fitness cost hosts     develop reproduce expenses host resources.  expected host strategies selected                                                                                       Despite ample knowledge genetics physiology                                                                                       host responses parasites, little known genetic <br />                                                                                       basis host adaptation parasites.  adaptation <br /> cope parasite burden. ample variety                          parasite likely impact outcome different <br /> strategies, behavioral intracellular responses (1).                        infections. correlated responses, seminal  <br /> cause range possibilities broad, difficult                    derstanding host evolution multiparasite environments, <br /> predict strategy,  evolve host populations                       remain poorly studied. determined genetic phe- parasite attack.  natural populations, hosts                     notypic changes underlying adaptation experimental <br /> exposed simultaneously parasite species                           evolution Drosophila melanogaster population viral selection pressures. selection pressures vary                   infection [Drosophila C virus (DCV)]. 20 generations, se- <br /> independently  clear establishment causality                       lected flies showed increased survival infection DCV changes host traits selection pressure posed                       viruses. Using genome sequencing given parasite species hampered.                                          RNAi, identified functionally validated    Experimental evolution enables establishment direct                       genes underlying adaptive process revealed dif- <br /> link selection imposed given environment                         ferential roles correlated responses observed. genetic phenotypic changes observed population. explanatory power methodology relies major                  Author contributions: N.E.M., V.G.F., L.T., &#195;&#8240;.S., S.M. designed research; N.E.M., V.G.F., <br /> characteristics:   knowledge ancestral state; (ii) control              V.N. performed research; C.S. &#195;&#8240;.S. contributed new reagents/analytic tools; selection forces driving different sets replicated pop-                N.E.M., V.G.F., C.S., L.T., &#195;&#8240;.S., S.M. analyzed data; N.E.M., &#195;&#8240;.S., S.M. wrote <br />                                                                                  paper. <br /> ulations; (iii) ability follow dynamics process, <br /> instead measuring end-product (2). addition,                 authors declare conflict  <br />  <br /> methodology allows addressing consequences adap-                      article PNAS Direct Submission. <br /> tation process performance environments (3&#226;&#8364;&#8220;5).                  Freely available online PNAS open access option. <br />    Experimental evolution coupled genome approaches                   Data deposition: sequences reported paper deposited Se- provide nearly unbiased view actual targets selec-               quence Read Archive European <span id='am-8' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-9' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>Nucleotide</span> Archive, www.ebi.ac.uk/ena  <br /> tion, long-standing aim evolutionary biology (2).                   search_and_browse (accession nos. ERS409776&#226;&#8364;&#8220;ERS409787). <br /> day examples exist combined methodologies                     1 <br />                                                                                  N.E.M. V.G.F. contributed equally work. used multicellular sexual organisms                   2 <br />                                                                                  &#195;&#8240;.S. S.M. contributed equally work. <br /> adaptation comes standing genetic variation (SGV) instead                   3 <br />                                                                                      correspondence addressed. E-mail: esucena@igc.gulbenkian.pt novel mutations (6&#226;&#8364;&#8220;10).  despite centrality                        snmagalhaes@fc.ul.pt. <br /> host&#226;&#8364;&#8220;parasite interactions evolutionary biology                   article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> experimental evolution studies host&#226;&#8364;&#8220;parasite systems (11&#226;&#8364;&#8220;16),                 1073/pnas.1400378111/-/DCSupplemental. <br />  <br />  <br />  <br /> 5938&#226;&#8364;&#8220;5943 | PNAS | April 22, 2014 | vol. 111 |  16                                                                  www.pnas.org/cgi/doi/10.1073/pnas.1400378111 <br />                                                                                    linear mixed model (GLMM), &#207;&#8225;21 = 154.98, P &lt; 0.0001]. <br />                                                                                   Changes survival VirSys selection regime consis- <br />                                                                                   tent replicate populations (Fig. S1A). difference                                                                                   survival absent early generations increased                                                                                   time, leading significant interaction generation                                                                                   selection regime (Fig. 1A, Dataset S1, Fig. S1A; GLMM, <br />                                                                                   &#207;&#8225;230 = 163.54, P &lt; 0.0001). tested independently                                                                                   sexes, effects selection regime (GLMM, &#207;&#8225;21 = 20.489                                                                                   24.288, P &lt; 0.0001 males females, respectively)                                                                                   interaction generation (GLMM, &#207;&#8225;230 = 236.95 &#207;&#8225;226 = <br />                                                                                   145.89, P &lt; 0.0001 males females, respectively)                                                                                   significant. Given comparing control VirSys <br />                                                                                   individuals, ContSys populations used sub- <br />                                                                                   sequent tests, survival rates ContSys control populations <br />                                                                                   directly compared generations 15 25. significant <br />                                                                                   differences observed sets control lines <br />                                                                                   (Table S1). <br />                                                                                      VirSys lines showed strong reduction virus numbers <br />                                                                                   compared ContSys lines (Fig. 1B; ANOVA, F1,6 = 39.55, <br />                                                                                   P = 0.0008) indicating selection relied  partially) <br />                                                                                   evolution resistance. <br />                                                                                       tested contribution Wolbachia evolution <br />                                                                                   resistance populations endosymbiont                                                                                   shown protect Drosophila viral infections (27).                                                                                   end, removed Wolbachia replicates VirSys ContSys <br />  <br />  <br />  <br />  <br />                                                                                                                                                            EVOLUTION <br />                                                                                   populations, 25 generations selection measured sur- <br /> Fig. 1. Evolution increased resistance DCV.   Experimental evolution      vival DCV infection (Fig. 1C). significant interaction <br /> trajectories control (control) virus-exposed (VirSys) populations     sex Wolbachia selection  <br /> 34 generations experimental evolution. Circles represent populations           gime (Cox model, &#207;&#8225;21 = 56.705 17.150, respectively P &lt; <br /> exposed virus. Squares represent control lines. Vertical bars corre- <br />                                                                                   0.0001 comparisons).  tested effects spond SEM survival selected populations (VirSys)                                                                                   Wolbachia selection regime independently sexes pool control individuals. straight dotted line corresponds original mortality rate imposed populations (66%). (B) Relative DCV <br />                                                                                   (Fig. S1B). cases significant Wolbachia loads (DCV/RpL32 copies) females, 5 d postinfection, ContSys VirSys     selection-regime effect (Cox model, &#207;&#8225;21 = 29.110 34.94, populations. Points represent individual measurements; horizontal lines       Wolbachia selection-regime effect males; &#207;&#8225;21 = 24.865 mean 95% confidence intervals). (C) Survival DCV infection           22.824 Wolbachia selection-regime effects females, <br /> control virus selected lines, Wolbachia (solid lines/filled   respectively; P &lt; 0.0001 comparisons).  pro- <br /> symbols, Wol + dotted lines/open symbols, Wol&#226;&#710;&#8217; , respectively).                tective role Wolbachia viral infections (27) confirmed <br />                                                                                   study experimental control lines.                                                                                    significant effect interaction Wolbachia X selection regime <br />                                                                                   sex (Cox model, &#207;&#8225;21 = 0.255, P = 0.613    shown natural Drosophila melanogaster pop-                    &#207;&#8225;21 = 1.007, P = 0.316 males females, respectively). ulations contain SGV resistance natural viruses.                      indicates significant contribution host genome studies variation                      evolution resistance, statistically independent attributed limited number genes major effect (20&#226;&#8364;&#8220;                    effect Wolbachia infection status. <br /> 23), indicate significant fraction genetic <br /> variation resistance polygenic (24, 25). Interestingly,                Cross-Resistance Parasites. shown Fig. 2, VirSys <br /> alleles contribute variation resistance given                 populations average higher survival, relative virus genes unrelated canonical insect antiviral                    ContSys, infection parasites cricket paralysis virus <br /> defense pathways (26).  variation                     (CrPV) flock house virus (FHV) (Cox model, jzj = 19.857, <br /> specific mediating responses distinct natural pathogens (21).               11.329, 5.226 infection DCV, CrPV, FHV,  <br />     addressed genetics host adaptation para-                    spectively; P &lt; 0.0001 comparisons). signifi- <br /> sites effects cross-resistance D. melanogaster&#226;&#8364;&#8220;virus              interaction effect generation test  aim, performed experimental evolution                    conducted different parasites (Cox model, &#207;&#8225;23 = 31.276, <br /> outbred D. melanogaster population exposed natural viral                     P &lt; 0.001 DCV; &#207;&#8225;21 = 4.192, P &lt; 0.05 CrPV; &#207;&#8225;22 = <br /> parasite (Drosophila C virus DCV), analyzed basis                  6.819, P &lt; 0.05 FHV).  difference response using genome-wide approach, functionally tested                    VirSys ContSys regimes significant separate tests <br /> candidate genes role response DCV                    performed different generations different viruses parasites.                                                                  (Cox model, jzj = 14.480, 10.790, 13.454, 7.337 DCV <br />                                                                                   infections performed generations 15, 20, 25, 30; jzj = <br /> Results                                                                           1.122 1.438 CrPV infections generations 15 30; <br /> Adaptation DCV Infection. performed experimental                       jzj = 0.514, 0.327, 0.804 FHV infections gen- <br /> evolution outbred D. melanogaster population exposed                     erations 15, 20, 30. P &lt; 0.001 comparisons, recurrent systemic DCV infection (VirSys). DCV infection                      FHV infection generation 20, P &lt; 0.05).  <br /> imposed generation using  coevolved)                        hazard ratios ContSys VirSys exposed FHV <br /> ancestral virus strain. parallel, control conditions                  infection significantly lower observed ex- <br /> established, individuals subjected pro-                    posure DCV (used selection) CrPV, close <br /> cedure virus-selected population pricked buffer                 DCV relative (Fig. 2). <br /> solution (ContSys) pricked (Control). ex-                     significant difference survival selection regimes <br /> periment performed replicates condition.                   flies infected bacteria Pseudomonas <br />   exposed DCV, VirSys populations showed higher                           entomophila Enterococcus faecalis (Cox model, jzj &lt; 0.446, P &gt; <br /> survival individuals control lines [Fig. 1A; general                    0.66 comparisons infection P. entomophila  <br /> Martins et al.                                                                                          PNAS | April 22, 2014 | vol. 111 |  16 | 5939 <br />                                                                                negative control respective genetic background <br />                                                                               (jzj = 0.71, 0.93, 0.19 DCV, CrPV FHV respectively; <br />                                                                               P &gt; 0.97). <br />                                                                                  RNAi knockdown 12 genes 3L region <br />                                                                               revealed case, gene CG8492 (stock P{KK100300} <br />                                                                               VIE-260B), reduced survival exposure DCV                                                                               FHV (Fig. 4B, jzj = 4.23 3.23, P &lt; 0.001 P &lt; 0.05                                                                               DCV FHV, respectively), CrPV (jzj = 0.24, P = 1). <br />                                                                               P values Bonferroni corrected number                                                                               performed comparisons. <br /> Fig. 2. Specificity evolved response. Hazard ratios ContSys <br />                                                                               Discussion VirSys populations, exposed DCV, CrPV, FHV, P. entomophila <br /> (P.ent), E. faecalis (E.fae). Shown average hazard ratios   study, resistance DCV evolved rapidly independent experiments, different generations. Vertical    experimental Drosophila populations. Cross-resistance bars correspond 95% confidence intervals estimated hazard       detected infection viruses (CrPV FHV) ratios. ***P &lt; 0.001.                                                         bacteria. Using genome sequencing, identified                                                                               regions genetic changes occurred populations evolving <br />                                                                               DCV challenge, 3L chromosome arm generations 15 25 E. faecalis generations 34               smaller region X chromosome. RNAi assays <br /> 35). conclude evolution resistance DCV <br />                                                                               candidate genes regions, confirmed role leads partial protection positive strand RNA <br />                                                                               pst, gene variants previously associated differential <br /> viruses, bacterial pathogens. <br />                                                                               response DCV infection Drosophila (21), loci <br /> Genetic Basis Host Adaptation. identify changes allele <br />                                                                               linked previously antiviral responses: Ubc- <br /> frequencies underlying observed increased resistance                   E2H X chromosome CG8492 3L chromosome <br /> Drosophila populations evolving presence DCV,                    arm. Knockdown pst Ubc-E2H led increased sensi- <br /> formed genome-wide sequencing DNA pools (Pool-Seq)                  tivity CrPV FHV, opposite pattern <br /> populations (Fig. 3) (28). Patterns overall genetic diversity          CG8492.  flies adapted resist <br /> presented Fig. S2.                                                         DCV better surviving infection viruses, <br />    Using chromosome-wide cutoff, observed consistent                     correlated responses rely different sets genes. <br /> significant changes allele frequencies 853 SNPs  <br /> gion spans &#226;&#710;&#188;4 Mb left arm chromosome                 Genetic Basis Resistance. Using combination genomics (3L)  5&#226;&#8364;&#178; SNP, 3L:5127093 3&#226;&#8364;&#178; SNP, 3L:9149494)                    experimental evolution, identified genetic changes  5 SNPs X chromosome 300-kb region                         derlying evolution host population (D. melanogaster) <br /> (X:7638809&#226;&#8364;&#8220;7984449). result did change qualitatively                 adapting natural parasite (DCV). regions using genome-wide cutoff, region significance                differentiation populations evolving presence reduced positions 3L:5221901&#226;&#8364;&#8220;8901948  e., 384 SNPs),                 virus control populations. changes parallel 2 SNPs X chromosome. significantly differ-                 replicates (Fig. S2 Dataset S2) correlate entiated SNP 3L region corresponds position 3L:7350895              observed parallel changes survival (Fig. S1A). indicates maps gene pastrel (pst). significantly differ-             selection, drift, shaped adaptive response. entiated SNPs X chromosome (X:7984325 X:7984449)                   region, peak differentiation matched pst, gene located introns gene Ubc-E2H. Initial final             previously shown involved Drosophila response frequencies significantly differentiated SNPs                DCV association study (21). high number 0.167 0.7 3L:7350895 (pst) 0.267 0.6 X:7984325 (Ubc-E2H), respectively. Considering changes frequency, assuming additive effects  estimated <br /> selection coefficients 0.24 0.14 SNP pst Ubc-E2H, respectively. Changes significantly differen- <br /> tiated SNPs described Dataset S2. <br />  <br /> Functional Validation Candidate Genes. used RNAi functionally validate genes associated significant SNPs identified genome-wide analysis. tested 12 genes 3L region, contained non- <br /> synonymous mutations (Fig. 4). <br />    Knockdown pastrel Ubc-E2H  stock w1118; <br /> P{GD9765}v33510; Table S2 details) led reduced <br /> survival flies exposed DCV CrPV infection (Fig. <br /> 4A, Ubc-E2H: jzj = 3.98 3.09, P &lt; 0.01 P &lt; 0.05, DCV CrPV infection, respectively; Fig. 4B, pst jzj = 5.94 5.93, P &lt; 0.001 DCV CrPV infection), exposed FHV infection (Ubc-E2H: jzj = 1.35, P &gt; 0.9 pst: jzj = 0.08 knockdown genes). Using RNAi line targeting Ubc-E2H  stock P{KK108626}VIE-                       Fig. 3. Differentiation selection regimes. &#226;&#710;&#8217;log10 values 260B; Table S2 details) did differences sur-              CMH statistic polymorphic SNP, major chromosomal <br /> vival viruses (jzj = 2.25, 0.11, 0.12              arms pairwise comparison allele frequencies ancestral <br /> DCV, CrPV, FHV respectively, P &gt; 0.3) (Fig. 4A).                    ContSys populations generation 20  , ancestral VirSys pop- <br /> tribute survival difference using different RNAi lines            ulations generation 20 (Middle), ContSys VirSys lower knockdown efficiency construct, revealed                generation 20  . solid dotted lines represent 99.99% <br /> semiquantitative gene expression analysis (Fig. S3). differ-               quantile P values ancestral vs. ContSys comparison genome- <br /> ences susceptibility viruses observed comparing               wide chromosome-wide levels, respectively. <br />  <br />  <br /> 5940 | www.pnas.org/cgi/doi/10.1073/pnas.1400378111                                                                                         Martins et al. <br />                                                                                                  Fig. 4. RNAi knockdown candidate genes. Nat- <br />                                                                                                 ural logarithm hazard ratios survival <br />                                                                                                 flies knocked candidate genes                                                                                                 controls infection DCV  , CrPV <br />                                                                                                 (Middle), FHV  , using genetic <br />                                                                                                 background KK (gray bars), GD (black bars)                                                                                                  construct available                                                                                                 backgrounds.   RNAi interference                                                                                                 candidate genes identified peaks Fig. 3, <br />                                                                                                 pst Ubc-E2H. (B) Tests genes                                                                                                 large 3L peak. Vertical bars correspond 95% <br />                                                                                                 confidence intervals estimated hazard ra- <br />                                                                                                 tios. *P &lt; 0.05; **P &lt; 0.01; ***P &lt; 0.001. <br />  <br />  <br />  <br />  <br />                                                                                                                                                      EVOLUTION <br /> differentiated SNPs locus, extending region          Cross-Resistance. strong positively correlated  <br /> &#226;&#710;&#188;4 Mb, observed pattern local decrease hetero-             sponse CrPV, moderate response FHV, zygosity suggest occurrence incomplete soft sweep              response bacteria.  correlated response positive pst (29).                                                         diminishes decreasing similarity DCV.     influence genes region            findings match recent theoretical predictions sided host excluded, shown increased susceptibility flies           evolution (14).  studies host evolution expressing RNAi CG8492, gene located near cen-             tradeoffs (16, 35) significant correlated response <br /> tromeric end peak. determination haplotype             (36, 37) resistance different parasites, gen- <br /> structure region, effect virus resistance   erality finding remains shown. variants CG8492 possible interactions pst,            analyzed correlated responses genes involved deserve examination.                                             DCV resistance flies infected viruses.    result particularly interesting departs   knowledge, constitutes direct test genetic <br /> inconsistency observed comparing genome-wide associa-               basis correlated responses selection driven SGV. Anal- <br /> tion studies (GWASs) using inbred lines vs. outbred populations          ysis effects novo mutations arise Escherichia <br /> (30). far, weak significant correlation         coli populations adapting glucose-limited environment SNPs associated polygenic traits GWAS              placed environments, shown set evolve resequence (E<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>;R) approaches (31).                 mutations conferring fitness increases varies environ- <br /> confirm pst, gene GWAS approach (21),               ments (38). Similarly study, distinct genes central player adaptation outbred population           allelic frequencies changed response DCV  <br />                                                                          fection affect correlated responses differently.  knock- <br /> Drosophila DCV infection. <br />    Furthermore, using RNAi confirmed role pst,             pst does affect susceptibility FHV, confirming <br /> unraveled effect Ubc-E2H CG8492 antiviral defense.          earlier results (21); knockdown pst Ubc-E2H <br />                                                                          affects cross-resistance CrPV. contrast, knockdown results confirm power E&amp;R approach                                                                          CG8492 does affect response CrPV leads higher <br /> identification targets selection (32). methodology                                                                          susceptibility FHV.  populations, evo- used identify changes allele frequencies following <br />                                                                          lution generalized response viral parasites specifically <br /> selection complex traits developmental time (7), body         partitioned different loci. <br /> size (8), hypoxia tolerance (6), increased life span (33), adapta-           genetic analysis correlated responses tion high/low temperatures (9, 34), courtship behavior            relied measuring genetic correlation traits dif- <br /> (10, 31). studies identified polygenic basis        ferent environments using quantitative genetics designs (3). studied traits, hampering identification candidate genes          methodology used study host&#226;&#8364;&#8220;parasite subsequent functional analysis. exception study         interactions (39, 40).  shown genetic Zhou et al. (6), differentiated genes            correlations poor predictors evolution correlated <br /> belonged Notch signaling pathway, permitting                 responses selection, mainly hinges functional validation pathway hypoxia tolerance             genetic architecture traits environment (41). evolution.  relatively high number genes involved         study, measure genetic architecture responses permit assessment role              traits selection, primarily miss genes  <br /> played gene phenotypic effect                 volved resistance fixed changes oc- <br /> partitioned. case, genes underlying evolution         curring threshold value.  detect genes resistance DCV work (partially) additive             allele frequencies change generations, fashion, gene tested independently confers resistance.           contribute evolutionary response. describing  studies needed establish relative role       genes different cross-resistance properties different additivity genetic interactions response.                 parasites, genetics correlated responses  <br /> Martins et al.                                                                                PNAS | April 22, 2014 | vol. 111 |  16 | 5941 <br />  complex, cases genetic basis adaptation                    surviving individuals mated randomly; reproduction took place days 5&#226;&#8364;&#8220;7 <br /> relatively simple.                                                                    infection providing fresh oviposition substrate. number    findings raise important issue: forces main-                          individuals control populations reduced initial <br /> tain SGV based host adaptation viral                             number infected individuals  e., 600). <br /> infection? costs susceptibility                            Egg density limited 400 cup, density determined experi- <br /> parasites associated evolution resistance DCV.                           mentally enable optimal larval development. generation cycle lasted  results support maintenance di-                              3 wk. beginning experiment, absence vertical trans- <br /> versity antagonistic pleiotropy (3). does rule                       mission parasite progeny verified (Fig. S5). tradeoffs susceptibility parasites exist,                       monitor survival generations, infected gen- included tests.  parasites                           eration additional sample males female flies VirSys lines <br /> tested, evolution positively correlated responses,                         control lines monitored survival vials 10 d depend different genetic architectures parasite-                        (Dataset S1). <br /> specific manner. raises possibility  cases generalized response evolves, specificities ge-                        Parasite Loads. Virus quantifications performed described netic level lead different genetic responses envi-                          Teixeira et al. (27) minor modifications. assay, 75&#226;&#8364;&#8220;125 <br /> ronments qualitatively different parasite challenges.                       females population ContSys VirSys generation 33 <br /> extends possibility maintaining genetic diversity                       infected survival assays. Surviving flies collected                                                                                       day 5 infection, pooled 5 replicates 10 individuals pop- <br /> host populations (42), phenotypic responses sug- <br />                                                                                       ulation, snap-frozen liquid N2. RNA extracted using TRIZOL. gest generalized response parasites. formal test <br />                                                                                       avoid possible artifacts different maternal effects, flies used hypothesis require evolving resequencing <br />                                                                                       tests progeny flies spent generation outbred populations environments different combi- <br />                                                                                       common environment virus. <br /> nations viruses. <br />    generally believed occurrence specific host <br />                                                                                       Wolbachia. Wolbachia-free replicates ContSys VirSys pop- <br /> genotype x parasite genotype interactions (Gh &#195;&#8212; Gp) relies                                                                                       ulations derived generation 25, raising progeny simple genetic bases (43&#226;&#8364;&#8220;45).                          generations food tetracycline (0.05 mg/mL). generations <br /> genetic basis host adaptation parasite simple, generalist                tetracycline treatment, 100 individuals (males females) response evolved.  simple genetic basis neces-                   replicate population VirSys ContSys selection regimes sary sufficient condition evolution specific                       Wolbachia-free counterparts, systemically infected DCV <br /> interactions.  noted findings concern                   survival followed 16 d. outcome evolutionary process coevolution occurred.  studies identifying genetic basis                   Cross-Resistance Parasites. test adaptation specific coevolution required (44, 46). particular, highly                parasite affected host responses parasites, 100 individuals (males informative compare genetic architecture cross-correlations                 females) replicate population VirSys ContSys selection coevolved systems present study.                                  regimes, spent generation common environment,                                                                                       systemically infected following parasites: CrPV (undetermined <br /> Materials Methods                                                                 TCID50), FHV (TCID50 = 5 &#195;&#8212; 106), P. entomophila (OD600 = 0.01), E. faecalis <br /> Fly Populations. used outbred population D. melanogaster                     (OD600 = 3). tests performed generations 15, 20, 25, 30 <br /> founded maintained described Martins et al. (15) kept             (DCV); 15, 20, 30 (FHV); 15 25 (P. entomophila); 15 35 (CrPV); high effective populations size (SI Materials Methods).              34 35 (E. faecalis). <br /> initiation experimental evolution, population serially expanded generations allow establishment 36 new populations               Genome Sequencing. Genomic DNA preparation sequencing 12 used work. Unless noted, flies                   Orozco-terWengel et al. (9). Briefly, pool 200 indi- <br /> maintained constant temperature (25 &#194;&#176;C), humidity (60&#226;&#8364;&#8220;70%),                 viduals selection line homogenized Ultraturrax T10 <br /> light&#226;&#8364;&#8220;darkness cycle (12:12); fed standard cornmeal&#226;&#8364;&#8220;agar  <br />                                                                                       (IKA-Werke), DNA extracted homogenate using high- <br /> dium. populations fully infected Wolbachia onset                                                                                       salt extraction protocol. Genomic DNA sheared using Covaris S2 <br /> experiment, infection status populations monitored <br />                                                                                       device (Covaris, ) paired-end libraries prepared using experiment. <br />                                                                                       TruSeq v2 DNA Sample Prep Kit (Illumina). Libraries size-selected <br />                                                                                       mean insert size 300 bp agarose gels amplified 10 <br /> Parasite Stocks Cultures. DCV, CrPV  gift Peter Christian, National <br />                                                                                       PCR cycles, 2&#195;&#8212; 100-bp paired-end reads sequenced HiSeq <br /> Institute Biological Standards Control, Potters Bar, United King- <br />                                                                                       2000 (Illumina). groups populations sequenced: rep- <br /> dom), FHV, grown titrated described (27). Virus <br />                                                                                       licates base population (&#226;&#8364;&#339;ancestral&#226;&#8364;?) replicates aliquots kept &#226;&#710;&#8217;80 &#194;&#176;C thawed infection. P. entomophila <br />                                                                                       ContSys VirSys selection regimes generation 20. E. faecalis generous gifts B. Lemaitre (&#195;&#8240;cole Polytechnique <br /> F&#195;&#169;d&#195;&#169;rale Lausanne, Lausanne, Switzerland) T. Rival (Aix-Marseille <br /> Universit&#195;&#169;, Centre National la Recherche Scientifique, Institut Biologie        Read Quality Control Mapping. Reads mapped following du D&#195;&#169;veloppement Marseille-Luminy, Marseille, France), respectively.               previously described pipeline Pool-Seq analysis. Briefly, 100-bp paired- <br /> Bacteria stocks kept glycerol &#226;&#710;&#8217;80 &#194;&#176;C. use,                end reads filtered minimum average base quality score 18 <br /> streaked fresh Petri dishes, single colony picked let grow       trimmed using PoPoolation (28). Reads minimum length &#226;&#8240;&#165;50 LB 30 &#194;&#176;C (P. entomophila) 37 &#194;&#176;C (E. faecalis). culture          bp mapped reference containing FlyBase D. <br /> centrifuged adjusted desired OD.                                           melanogaster genome r5.38 (http://flybase.org). details filtering <br />                                                                                       parameters coverage, SI Materials Methods. <br /> Experimental Evolution. Starting base population, derived 12 lines <br /> evolving 3 different regimes (4 replicates treatment). VirSys        SNP Calling. SNPs met following quality criteria  <br /> treatment, adult flies pricked thoracic region DCV [2 &#195;&#8212; 107 tissue   sidered:  ) occurrence 2 replicate populations, (ii ) minor <br /> culture ID50 (TCID50)] generation. second treatment consisted            allele covered 10 reads populations analyzed, control pricking, needle dipped sterile medium              (iii) maximum coverage did exceed 500. <br /> (ContSys). Finally, second group control lines consisted flies kept standard food pricked (control). differences ContSys         Genetic Diversity. characterize genome-wide patterns genetic diversity, control lines test sets lines. dose      estimated site heterozygosity (&#207;&#8364;), following PoPoolation analysis DCV used caused average mortality 66% initial pop-           pipeline (28). considered polymorphic sites minor <br /> ulation 10 d infection (Fig. S4).                                               allele supported reads standardizing coverage <br />    treatments administrated 310 males 310 females (4&#226;&#8364;&#8220;6 d            30 reads site, used unbiased estimators pooled data eclosion). Selection lines kept large population cages              correct pool size coverage (28, 47). graphical representation,  <br />  <br /> 5942 | www.pnas.org/cgi/doi/10.1073/pnas.1400378111                                                                                                     Martins et al. <br />  calculated average values sliding 500-kb windows, step size 100                   developmental lethality. tested constructs shown Table S2. kb entire genome (Fig. S1A).                                                        details available SI Materials Methods. <br />  <br /> Identification Candidate SNPs. used Cochran&#226;&#8364;&#8220;Mantel&#226;&#8364;&#8220;Haenszel                          Statistical Analysis. statistical analyses using R (Version <br /> (CMH) test, implemented PoPoolation2 (48) identify SNPs                          2.15; www.r-project.org). details provided SI Materials changes allele frequencies different regimes                          Methods. <br /> consistent replicates described Orozco-terWengel et al. (9) (SI <br /> Materials Methods).                                                                        ACKNOWLEDGMENTS. authors thank Dieter Ebert Britt Koskella <br />                                                                                                critical comments significantly improved manuscript; RNAi. performed vivo RNAi knockdown assays candidate genes                       laboratories Patr&#195;&#173;cia Beldade, Christen Mirth, &#195;&#8240;.S.; Margarida 3L X (pst Ubc-E2H) set genes 3L peak                     Matos inspiring discussions project. N.E.M. (SFRH/BPD/62964/ <br />                                                                                                2009) V.G.F. (SFRH/BD/82299/2011) funded Funda&#195;&#167;&#195;&#163;o para <br /> differentiation, selected according   significantly <br />                                                                                                Ci&#195;&#170;ncia e Tecnologia. C.S. funded Austrian Science Fund <br /> differentiated nonsynonymous SNPs (ii) gene ontology previous                            (FWF) (P22725 P19467) European Research Council <br /> functional assays suggested role antiviral immunity. took advantage                    (ArchAdapt). work supported Funda&#195;&#167;&#195;&#163;o para Ci&#195;&#170;ncia e Tecno- large RNAi collections Vienna Drosophila RNAi Center (49),                   logia (PTDC/SAU-IMU/120673/2010), Instituto Gulbenkian Ci&#195;&#170;ncia/Fun- used Gal80ts/Tub-Gal4 inducible rescue                                da&#195;&#167;&#195;&#163;o Calouste Gulbenkian, Vetmeduni funding. <br />  <br />  <br />  1. Schmid-Hempel P (2011) Evolutionary Parasitology: Integrated Study               25. Carpenter JA, Hadfield JD, Bangham J, Jiggins FM (2012) Specific interactions  <br />     fections, Immunology, Ecology, Genetics (Oxford Biology, Oxford).                          tween host parasite genotypes act constraint evolution  2. Kawecki TJ, et al. (2012) Experimental evolution. Trends Ecol Evol 27(10):547&#226;&#8364;&#8220;560.             antiviral resistance Drosophila. Evolution 66(4):1114&#226;&#8364;&#8220;1125. <br />  3. Kassen R (2002) experimental evolution specialists, generalists,            26. Xu J, Cherry S (2014) Viruses antiviral immunity Drosophila. Dev Comp Im- <br />     maintenance diversity. J Evol Biol 15(2):173&#226;&#8364;&#8220;190.                                           munol 42(1):67&#226;&#8364;&#8220;84. <br />  4. Lopes PC, Sucena E, Santos  Magalh&#195;&#163;es S (2008) Rapid experimental evolution          27. Teixeira L, Ferreira &#195;?, Ashburner M (2008) bacterial symbiont Wolbachia induces <br />     pesticide resistance C. elegans entails costs affects mating  PLoS        resistance RNA viral infections Drosophila melanogaster. PLoS Biol 6(12):e2. <br />     3(11):e3741.                                                                           28. Kofler R, et al. (2011) PoPoolation: toolbox population genetic analysis  5. Magalh&#195;&#163;es S, Blanchet E, Egas M, Olivieri (2009) adaptation costs necessary           generation sequencing data pooled individuals. PLoS 6(1):e15925. <br />     build local adaptation pattern? BMC Evol Biol 9:182.                                  29. Burke MK (2012) does adaptation sweep genome? Insights  <br />  <br />  <br />  <br />                                                                                                                                                                                              EVOLUTION <br />  6. Zhou D, et al. (2011) Experimental selection hypoxia-tolerant Drosophila mela-              long-term selection experiments. Proc Biol Sci 279(1749):5029&#226;&#8364;&#8220;5038. <br />     nogaster. Proc Natl Acad Sci USA 108(6):2349&#226;&#8364;&#8220;2354.                                         30. Huang W, et al. (2012) Epistasis dominates genetic architecture Drosophila <br />  7. Burke MK, et al. (2010) Genome-wide analysis long-term evolution experiment               quantitative traits. Proc Natl Acad Sci USA 109(39):15553&#226;&#8364;&#8220;15559. <br />     Drosophila. Nature 467(7315):587&#226;&#8364;&#8220;590.                                                 31. Turner TL, Miller PM, Cochrane VA (2013) Combining genome-wide methods  <br />  8. Turner TL, Stewart AD, Fields  Rice WR, Tarone (2011) Population-based                vestigate genetic complexity courtship song variation Drosophila mela- <br />     sequencing experimentally evolved populations reveals genetic basis body             nogaster. Mol Biol Evol 30(9):2113&#226;&#8364;&#8220;2120. <br />     size variation Drosophila melanogaster. PLoS Genet 7(3):e1001336.                       32. Kofler R, Schl&#195;&#182;tterer C (2014) guide design evolve resequencing <br />  9. Orozco-terWengel P, et al. (2012) Adaptation Drosophila novel laboratory               studies. Mol Biol Evol 31(2):474&#226;&#8364;&#8220;483. <br />     environment reveals temporally heterogeneous trajectories selected alleles. Mol         33. Remolina SC, Chang PL, Leips J, Nuzhdin SV, Hughes KA (2012) Genomic basis aging <br />     Ecol 21(20):4931&#226;&#8364;&#8220;4941.                                                                         life-history evolution Drosophila melanogaster. Evolution 66(11):3390&#226;&#8364;&#8220;3403. <br /> 10. Turner TL, Miller PM (2012) Investigating natural variation Drosophila courtship        34. Tobler R, et al. (2014) Massive habitat-specific genomic response D. melanogaster <br />     song evolve resequence approach. Genetics 191(2):633&#226;&#8364;&#8220;642.                           populations experimental evolution hot cold environments. Mol Biol <br /> 11. Ye YH, Chenoweth SF, McGraw EA (2009) Effective costly, evolved mechanisms              Evol 31(2):364&#226;&#8364;&#8220;375. <br />     defense virulent opportunistic pathogen Drosophila melanogaster. PLoS         35. Gwynn DM, Callaghan  Gorham J, Walters KF, Fellowes MD (2005) Resistance     Pathog 5(4):e1000385.                                                                          costly: Trade-offs immunity, fecundity survival pea aphid. Proc <br /> 12. Kraaijeveld AR, Godfray HC (1997) Trade parasitoid resistance larval           Biol Sci 272(1574):1803&#226;&#8364;&#8220;1808. <br />     competitive ability Drosophila melanogaster. Nature 389(6648):278&#226;&#8364;&#8220;280.                  36. Lazzaro BP, Sackton TB, Clark AG (2006) Genetic variation Drosophila melanogaster <br /> 13. Vijendravarma RK, Kraaijeveld AR, Godfray HCJ (2009) Experimental evolution shows              resistance infection: comparison bacteria. Genetics 174(3):1539&#226;&#8364;&#8220;1554. <br />     Drosophila melanogaster resistance microsporidian pathogen fitness costs.         37. Kraaijeveld AR, Layen SJ, Futerman PH, Godfray HCJ (2012) Lack phenotypic     Evolution 63(1):104&#226;&#8364;&#8220;114.                                                                       evolutionary cross-resistance parasitoids pathogens Drosophila mela- <br /> 14. Antonovics J, et al. (2013) origin specificity means natural selection:           nogaster. PLoS 7(12):e53002. <br />     Evolved nonhost resistance host-pathogen interactions. Evolution 67(1):1&#226;&#8364;&#8220;9.         38. Ostrowski EA, Rozen  Lenski (2005) Pleiotropic effects beneficial mutations <br /> 15. Martins NE, Faria VG, Teixeira L, Magalh&#195;&#163;es S, Sucena E (2013) Host adaptation              Escherichia coli. Evolution 59(11):2343&#226;&#8364;&#8220;2352. <br />     contingent infection route taken pathogens. PLoS Pathog 9(9):e1003601.         39. Luong LT, Polak M (2007) Costs resistance Drosophila-macrocheles  <br /> 16. Zbinden M, Haag CR, Ebert D (2008) Experimental evolution field populations         <br /> </body></html>