<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>AtVPS34, <span id='am-25' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span>phosphatidylinositol</span> 3-kinase Arabidopsis thaliana, essential protein homology calcium-dependent <span id='am-19' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span>lipid</span> <br /> binding domain <br />      (vacuoe/prIn t            rt/yeast come                       c gene/ant-ene) <br />  <br /> ABSTRACT          cDNA e pho atldylinositol (PI)                                 PI 3-kinase required internalization degradation 3-knim cloned bomArabidpsis t              derived                       platelet-derived growth factor receptor (16). <br /> <span id='am-49' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-50' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-55' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> seuence (AtVPS34) s              antl higher                          Using PCR oligonucleotides based conserved <br /> homology yeast PI 34-dase (VPS34) mammalian                           sequences PI 3-kinases bovine tissues (17), yeast (9), <br /> (plO). protein conserved domains: catlic site                          PI 4-kinase yeast (18), identified  thaliana <span id='am-38' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>-bdng site near C terinns calcium-                             PI 3-kinase cDNA (AtVPS34). considerable amino <br /> depenldent lipid-binding domain near N te s.                                 acid sequence identity AtVPS34 yeast VPS34, <br /> plant cDNA does rescue yeast vps34 deletion mutat,                         mammalian p110, especially C-terminal half chenrV gene coding sequence C-termi-                          catalytic domain. N-terminal half plant <br /> nal VPS34 repl correspondig sequence                              protein homology calcium-dependent <span id='am-14' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span>phospholipid</span>- plant gene does rescue yeast mutant. PI 3-kinase                        binding (CaLB) domain present, proteins activit s d b extracts plants overexpres                                rabphilin-3A, protein involved fusion synaptic plant PI 3-kinase. Expressn antse c                gives                      vesicles plasma membrane (7). <br /> rise second-generation transformed plants severely inhibited growth development.                                                                        MATERIALS METHODS <br /> Sigaling cascades involving phosphorylated phospholipids                               Mdecular Biolgy Techniques. Cloning, transformation, function mammalian yeast cells operate                          blotting, hybridization standard methods, plant cells (1). importance phosphati-                          described (19). PCR conducted denaturation dylinositol (PI) 3-kinases signaling cascades mammalian                        94&#194;&#176;C 3 min, followed 25-40 cycles 1-min denatur- <br /> cells (2) vacuolar targeting yeast (3), studied                    ation 94&#194;&#176;C, 1-min annealing 65&#194;&#176;C, 1-min elongation <br /> role PI 3-kinase Arabidopsis thaliana. enzyme                         72&#194;&#176;C. plant extracts, activity                         Plasnid Consthucti. Oligonucleotides used amplifi- demonstrated metabolic labeling lipids ra-                           cation VPS34 homolog  thaliana 2048-fold <br /> dioactive precursors (4, 5).                                                         degenerated covering following sequences: 1, CAGY- <br />   Targeting vacuolar proteins requires information                           CVA  2, GDDLRQD/E; 3, HI/AD-FGH/Y/F. Se- proteins (vacuolar sorting signals)                        quences 1 3 BamHI restriction site 5&apos; ends <br /> machinery protein targeting: receptors proteins                       amplified complementary strand; sequence 2 outside vesicles (coat regulatory proteins)                        Pst site specific coding strand. started vacuolar membrane. GTPases likely                               GTTT ensure function restriction enzyme. important processes (6), proteins necessary                      resulting PCR fiagment (291 bp) isolated agarose vesicle fusion target membrane, possibly                             gel treatment cloned pCRII (Invit- <br /> binding specific phospholipids membranes (7).                            rogen). pMM9.1 (AtVPS34) isolated cDNA li- proteins interact deliver vacuolar protein                              brary  thaliana seedling mRNA pFL61 (20). remains elucidated, VPS34 VPS15                              antisense sense expression plants, 2.8-kb shown components membrane-associated protein                              fragment AtVPS34 cloned Xba Sst sites complex needed correct targeting ofvacuolar proteins                         pBI121 Klenow fragment reaction (p9.1A Saccharomyces cerevisiae (8). VPS15 encodes protein                             p9.1S, respectively). ScAtI fusion N-terminal yeast <br /> kinase (9) VPS34 encodes PI 3-kinase (10, 11).                                 C-terminal plant clone single Dra restriction site <br />    importance PI vesicle transport underscored                         AtVPS34 conserved ScVPS34 yielding fusion protein crucial role PI transfer protein, product                       amino acids 1-601 yeast 540-815 plant <br /> SEC14 gene yeast, exit proteins late Golgi                          VPS34. ScAtH fusion filled Bgi II ends compartment (12). protein transfers PI lipid                            AtVPS34 Ava ends ScVPS34, resulting frame <br /> bilayers required maintain correct PI level                         fusion amino acids 1-463 ScVPS34 402-815 Golgi membranes. <br />    association PI 4-kinases cytoskeleton (13)                        Abbreviations: CaLB, calcium-dependent lipid-binding; PI, phospha- colocalization vesicles (14) interaction                       tidylinositol. products proteins like profilin (15) point                     tTo reprint requests sent present address: Centre <br /> involvement PI kinases protein transport. Furthermore,                          National la Recherche Scientifique, Unite Recherche Asso- <br />                                                                                       ciee 203, Faculte des Sciences, Universite Rouen, 76821 Mont <br />                                                                                       Saint Aignan Cedex, France. publication costs article defrayed page charge           &#194;&#167;Present address: Department Bioresource Science, Faculty payment. article marked &quot;advertisement&quot;                 Agriculture, Kagawa University, Miki-cho, Kagawa-ken, Kagawa accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                      761-07 Japan. <br />                                                                              11398 <br />            Plant Biology: Welters et                                                                   Proc. Natl. Acad. Sci. USA 91 (1994)   11399 <br /> AtVPS34. ScAtI, ScAtII, ScVPS34 derivatives                      resulting fragments yeast VPS34 probe (data single-copy yeast vector (pRS316).                                            shown). stringent conditions PCR <br />   Yeast Strains. S. cerevisiae strains used SEY6210                      (650C annealing temperature compared calculated melt- <br /> [MATa, leu2-3, 112 ura3-52, his3-4200, trpl-A901, lys2-801,                   ing temperature 62&#194;&#176;() amplification specific suc2-A9 (ref. 21)] vps34 deletion mutant PHY102                       yield fragment hybridizing yeast probe.  <br /> [SEY6210 vps34A1::TRP1 (ref. 10)].                                            inner primer pair (2/1, Fig. 1) amplify   Yeast Transformation. modified version published                     sequence. deduced amino acid sequence 291-bp <br /> method (22) used transformation. highest effi-                    PCR fiagment 61% identical corresponding yeast <br /> ciency achieved applying heat shock 15 min                     VPS34 region. Identity bovine PI 3-kinase (36%6) 420C despite heat sensitivity strains using                    yeast PI 4-kinase (31%) significant.  <br /> selection plates containing 1 M sorbitol.                                     length cDNA clone, PCR fragment used screen   Plant Transformation. Calli induced roots                          thaliana cDNA library pFL61 (20). 40,000 <br /> thaliana ecotype Landsberg transformed cocultiva-                     colonies, 12 clones isolated. (pMM9.1) <br /> tion Agrobacterium tumefaciens LBA4044 bearing plas-                     yielded PCR fragments appropriate size  <br /> mids pBI121, p9.1A, p9.1S. Transgenic plants                         generate primers; 2.8-kb clone sequenced obtained regeneration kanamycin-containing                        entirety. <br /> dium, described (23).                                                        cDNA clone consists 121-bp untranslated leader <br />   PI 3-Kinase Assay. described method (3) used                      sequence, 2442-bp coding sequence, 198-bp 3&apos; region <br /> yeast protein extraction assay conditions. plant pro-                 followed stretch poly  (GenBank accession  <br /> tein extracts, plant tissue pulverized liquid nitrogen.                U10669). deduced amino acid sequence 25% identical resulting powder suspended yeast protein extrac-                   bovine p110 40% identical yeast PI 3-kinase <br /> tion buffer [15 mM Hepes, pH 7.5/100 mM KCI/3 mM                              (ScVPS34 Fig. 1). lower identity mammalian <br /> EGTA/10%o (vol/vol) glycerol/2 mug tissue] pro-                     <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> sequence caused,  variable <br /> teinase inhibitor mixture (leupeptin, pepstatin, aprotenin, phe-              N-terminal sequence p110 polypeptide. Common nylmethylsulfonyl fluoride concentrations 1 10                 plant mammalian PI 3-kinases deletion 60-aa <br /> ng/ml). supernatant centrifugation 700 x g 5                 stretch yeast sequence, 400 aa C terminus <br /> min used PI 3-kinase assays. Briefly, suspension                 (Fig. 1, black box). 60-aa insertion yeast protein 1.5 pg PI 15 mM Hepes, pH 7.5/100 mM KCl                            serine-rich sequence unknown function. <br /> incubated protein extracts yeast plants, 10 pCi                   AVPS34 Contains Lipid-Bin          Domain. BLAST Net- <br /> [y-32P]ATP (1 Ci = 37 GBq), 0.3 mM unlabeled ATP,                      work Service search entire amino acid sequence 10 mM MgCl2 20 mM Hepes, pH 7.5, final 5SO- vol                      AtVPS34 closest matches yeast VPS34, bovine 5 min room temperature. reaction stopped                  murine pl10, PI 4-kinase yeast, TOR2, protein <br /> 80  1 M HCl; lipids extracted 160 A4                    involved cell cycle control, order decreasing <br /> CHCl3/CH30H, 1:1, evaporation solutes                    similarity. highest similarity ATP- <br /> redissolving lipids CHC13 separated TLC silica                binding domains near C terminus ofthese proteins. <br /> gel 60 TLC plates (Merck) borate buffer (24).                   search parts N-terminal sequences (aa 85-150 <br />   Programs. programs used developed                         AtVPS34) revealed strong homology CaLB domain Genetics Group (25), Smith (11), Henikoff                     rabphilin-3A (24% identity 47% similarity; Fig. 2). Henikoff (ref. 26; BLOCKS search), Altschul et al. (ref.              high homology portion AtVPS34 <br /> 27; BLAST Network Service).                                                   core stretches CaLB domain rabphilin-3A (47% <br />                                                                               identity aa 96-119 AtVPS34; box  Fig. 2) p110 <br />                            RESULTS                                            (44% identity aa 122-139 AtVPS34; box II, Fig. 2),                                                                               postulate phospholipid-binding site located   Cloning Plant PI 3-Kinase cDNA. cloning                         region plant protein. spite moderate identity <br /> fragment AtVPS34 achieved using PCR                      value regions yeast plant VPS34 proteins plant  thaliana cDNA library degenerate primers                   (21%), good similarity scores (46%) lead suggest derived conserved amino acid sequence domains                     contrast reports (2) yeast VPS34 mammalian (17) yeast PI 3-kinases (3, 9), yeast              similar functional domain region. <br /> PI 4-kinase (18). Positions primers indicated                Chileric Gene Rescues S. cerevmuja Mutant. Fig. 1. PCR conditions optimized probing                      AtVPS34 gene 2,u plasmid expressed control <br />                                                                                      4       4. <br />                             s cV <br />                                v P3 4                      1\\\\\ <br />                             40%(61 %) <br />                                                                                      L       .w   ... <br />  <br />                             tvPS 34 <br />                             26%(52%) <br />                                                14 8                                      C                             _ <br />                                                              ~~~~.... <br />                                                                  ~ <br />                              c11o <br />  <br />  <br />                                                              G  <br />                                                                 D L  <br />                                                                &apos;D <br />  <br />                                                             GDD LRQ D     -                                    - <br />  <br />                                                       C)    C-D DLRQADD        .1l           -                <br />  <br />  <br />  <br />   FiG. 1. PI 3-kinases  thaliana (open box), S. cerevisiae (hatched), mammal (bovine p110, stippled). amino acid sequence <br /> AtVPS34 40%o identity (61% similarity) ScVPS34 26% (52%) p110. N-terminal half conserved 29%O <br /> identity plant (aa 1-401) fungal (aa 1-418) domains 22% identity plant mammalian (aa 148-646) polypeptides. <br /> Black boxes regions homology gaps &gt;20 aa. Bars c indicate conserved regions used design oligonucleotides; arrowheads indicate orientations positions. <br />  11400      Plant Biology: Welters et al.                                                 Proc. Nad. Acad Sci. USA 91                                       (1994) <br />                                                                                                               7.           --                                                                                           30 C <br />                                                                                                             AtVPS34 ScAt                   \ <br />     p110 <br />     AtVPS34                                                                                        __ <br />     rabphilin-3A          <br />  <br /> B <br />     p110                      369     .DNVNTQRVPCSNPRWNEWLNYDIYI <br />     AtVPS34                      68   RTRLKTTGPPYC ... WNELITLSSKY <br />                                       :1.:1...         :11 : ..  <br />      rabphilin-3A             615     KTQIKKKTLNPE ... FNEEFFYDIKH <br />  <br />     PDLPRAARLCLSICSVKGRKGAKEEHCPLAWGNI LFDYTDTLVSGKMA. LNLW <br />                                                                        B <br />     R DLTAHSQLAITVWDVSCGKT .... EGLIGG ATVL.LFNSKMOMKSGKQK   LgkWj             ScAt                \X-N\E-bCKNENS\XES                                      ~~~~~~~----- <br />      .1..11.1.111 ..11.          :.1     :11.1                                 (58976i         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~9 , 1 <br />  <br />     S bLAKKS.LDISVWDYDIGKS....   NDYIGGCQL ........ GISAKGERLKHW           2 <br />                                                                                                               r -- -- ---         S~4~    40 2             _ <br />                                                                                ArVPS34         F <br />                                                                                                                                                7 <br />  <br />     PVPHGLEDLLNPIGVTGSNPNKETPCLELEFDWFSSV        483 <br />           :.: <br />       ....:          ..1.1.1 ..     1::..                                  3   ScAti                                                                         ____        _- <br />  <br />                                                  170                           (50I69                                                      463 <br />     QGKEADGSF ...... PTSTPGKVPRHERGELERLEKL <br />                                                                            4   Sc VPS34 <br />     YECLKNKDKKI .......... ERWHQLQNE.NHVSSD      704 <br />                                                                        C <br />   FiG. 2. Homology AtVPS34 CaLB-domain plo rab- <br /> philin-3A.   Schematic location ofCaLB domains (hatched boxes);                                                                                                -13*. <br /> black bars, ATP-binding domain p110 AtVPS34. (B) Compar-                                                                                                  -N  <br />                                                                            PI 3-P                 &quot;-&quot; <br />  <br /> ison ofamino acid sequences ofrabphilin-3A pliOCaLB-domain                                       --gl <br />                                                                                                       im <br />  AtVPS34. Dots represent different degrees similarity; dashes                                                         15MM                    -jai <br /> indicate identity. Box emphasizes regions highest identity                                                                                                                                                                                                      .17-. <br /> betweenAtVPS34 p110 (44%) CaLB-domains; box emphasizes               PI 4-P       00-                        - AtVPS34 second CaLB-domain rab- <br /> philin-3A (47%). phosphoglycerate kinase promoter terminator (20)             FiG. 3. Growth different temperatures PI 3-kinase assay <br /> failed rescue vps34 deletion mutant temperature         vps34 deletion mutant S. cerevisiae (PHY102) transformed <br /> sensitive growth 3rC (Fig. 3A, upper left quadrant).          ScVPS34, AtVPS34, chimeric genes    Growth Extracts cells grown 30TC expressed AtVPS34             PHY102 transformed withAtVPS34, ScVPS34, ScAtII, ScAtI shown Northern blots) contained PI 3-kinase                yeast extract/peptone/dextrose 30&#194;&#176; 37C. ScVPS34 activity (Fig. 3C, lane 2).                                          ScAtI fusion complemented temperature-sensitive phenotype <br />   test phosphoglycerate kinase promoter           yeast mutant. PHY102 transformed ScAtI                                                                      AtVPS34 unable grow 37TC. [Fewer colonies upper <br /> responsible lack enzyme activity,               left plate grown 30&#194;&#176; (AtVPS34) caused gene <br /> construct plant coding sequence controlled          presence plates.] (B) Schematic drawing yeast VPS34 promoter. construct, contained             PI 3-kinases yeast  thaliana chimeric fusions tested proper yeast expression signals, failed rescue       PHY102 complementation. Numbers barsdenote <br /> yeast deletion mutant (data shown).                              amino acids point fusion, respectively, Arabidopsis yeast <br />   extent homology N-terminal half ofthe VPS34          proteins; brackets percentage identity/percentage similarity <br /> proteins relatively low (29%, Fig. 1).  plant       ratios ofthe exchangedArabidopsis segment yeast counterpart. <br /> protein associate necessary effector pro-           (C) PI 3-kinase assay protein extracts strains described <br />                                                                      grown liquid medium 30TC. Corresponding results <br /> teins, VPS15. test hypothesis,               growth phenotype, strains able grow 37C, PHY102/ <br /> chimeric yeast-plant cDNAs domains              ScVPS34 (lane 4) PHY102/ScAtI (lane 1), PI 3-kinase proteins swapped. shown Fig.            activity (PI 3-P), PI 3-kinase activity detectable 3B. construct ScAtI terminal plant           PHY102/AtVPS34 (lane 2) PHY102/ScAtl (lane 3). PI 4-P, PI <br /> cDNA fused thirds yeast gene N             4-kinase activity. <br /> terminus. Taking advantage conserved Dra restriction          right quadrant), cells grown 30TC detectable PI <br /> site genes, fusion N-terminal end       3-kinase activity (Fig. 3C, lane 3). conserved domain (58% identity, Fig. 3B). ScAtII           AtPS34 Expresed Transgenc Cels Encdes C-terminal half AtVPS34 (aa 402-815) aa 1-463 <br /> ofScVPS34 (Fig. 3B). coding sequences expressed            Active PI 3-Klnase. presence stretches amino yeast expression signals (ScVPS34 promoter phos-            acid identity (100% match, aa 563-577 665-676), defined <br /> phoglycerate kinase terminator). constructs,            Prosite 11 (keyed SwissProt 27) specific PI shorter plant coding segment rescued           3-kinases AtVPS34 high homology temperature-sensitive growth defect yeast vps34-          proteins, especially catalytic domain, suggest similar <br /> deletion strain (Fig. 3A, upper right quadrant).           enzymatic activity AtVPS34. PI 3-kinase activity complemented strain grown 300C, cell extracts             plant protein extracts. demonstrate <br /> tained PI 3-kinase activity (Fig. 3C, lane 1). substitution      AtVPS34 PI 3-kinase activity plant extracts, gene ScAtI involves conserved domains yeast            overexpressed plants.  thaliana root explants plant proteins (76% similarity; 58% identity),          transformed AtVPS34 construct expression <br /> domain contains protein catalytic activity (3).           driven cauliflower mosaic virus 35S promoter <br /> slightly larger plant protein present         (p9.1S) control vector (pBI121) lacked chimeric construct, complementation failed (Fig. 3A, lower           AtVPS34 gene. resulting transformed cells propa- <br />             Plant Biology: Welters et al.                                              Proc. Nati. Acad. Sci. USA 91 (1994)           11401 <br /> gated callus regenerated plants. plants appeared normal normal seed set; seeds <br /> germinated kanamycin-containing medium. RNA extracted control, pBI121-transformed, AtVPS34- <br /> transformed plants, Northern blot probed AtVPS34 (Fig. 4A). blot probed (-tu- <br /> bulin cDNA control. Transformed plants highly expressed 2.8-kb RNA, dilution series shown lanes <br /> 1-3. Control plants vector-transformed plants lower levels mRNA (lanes 5 6, respectively). mRNA readily detected transformed callus <br /> cells level considerably control callus (data shown). Extracts control transformed callus assayed PI 3-kinase activity (Fig. 4B). AtVPS34- <br /> transformed callus cells contained low, detectable, levels PI 3-kinase (lane 3, radioactivity applied represents <br /> enzyme activity 2.5 times protein plant extract             FiG. 5. Phenotype AtVPS34 antisense plants. Shown yeast extract). enzyme activity cells               3-mo-old plants germinated seeds regenerated transgenic <br /> transformed pBI121 vector (lane 2) wild-type           plants derived transformation p9.1A. (Left) severe <br /> cells, 8 times protein used (data                viable phenotype shoot elongation. (Right) Plant nearly <br /> shown). differences detected radioac-              wild-type phenotype. <br /> tivity-labeled lipids comparing extracts control overexpressing plants. PI 3-kinase activity abun-              inward dipped agar. growth stopped, al- <br /> dant extracts yeast cells (lane 1).                               callus eventually formed small leaves <br />   AtVPS34 Enes Esl Protein. gain insight                     grow (Fig. 5 Left). severe phenotype,  plants) role AtVPS34 plants, plant gene inactivated           leaves rolled, short shoot formed. expressing cDNA  thaliana antisense                  types, leaves appeared shiny, water covered leaf <br /> orientation (28) using cauliflower mosaic virus 35S pro-            surface. plant containing antisense gene exhibited moter kanamycin selectable marker. Plants ex-                nearly wild-type phenotype (Fig. 5 Right). pressing antisense gene used determine phe-              plants wild-type phenotypes tested PCR. <br /> notype resulting lower level absence PI <br /> 3-kinase.                                                                                        DISCUSSION <br />    Characteristic initial transformants (T.) antisense construct (p9.1A) slower regeneration              plant, mammalian, yeast cells, proteins synthesized <br /> shoots (6 weeks instead 4 weeks) fewer shoots arising            endoplasmic reticulum, sorted Golgi body, calli. 20 independent shoots obtained.              delivered extracellular space vacuoles (lysos- seven  callus lines generated main-              omes), depending specific targeting information. tar- <br /> tained. regenerated plantlets flowered selling            geting information differs organisms, com- <br /> produced seeds; seed set poor, gemination             ponents ofthe secretory machinery appear conserved.  <br /> seeds; plants fertile seeds obtained.              proteins ofthe secretory ofplant cells kanamycin (50 pg/ml) seedlings died 2 weeks.                identified complementing yeast secretion mutants (29,30). kanamycin, recovered 11 plants, trans-               enzyme PI 3-kinase, catalyzes phosphorylation gene plants shown PCR.                                  PI, involved protein sorting mammalian yeast <br />    severely affected T, plants leaves           cells (3, 16). yeast, vps34 mutants secrete vacuolar  <br /> grew normally  petioles failed         zymes, protein encoded VPS34, PI 3-kinase, binds <br /> elongate, stem formed. leaf edges curled                 directly membrane-associated protein kinase encoded                                                                         VPS1S. protein complex postulated regulate                       p9lS       pB121 wt    B                        sorting/packaging vacuolar hydrolases late Golgi com- <br />                                                                         partment (8). mammals, PI 3-kinase plays role        AtVPS34                                 PI   3-P -               internalization human platelet-derived growth factor recep- <br />        RNA   (L4g) 20 2 0.2 50 20 20                        9           tors receptor tyrosine kinases (16). <br />        -tubulin   _W#M                       PI   4-P *       9 <br />                                                                 |          Structure Functon AtVPS34,  tan PI <br />                                                                         3-Kinase. addition size similarity, homology  <br />                     1    2     3   4   5   6                    2   3 <br />                                                                         thaliana PI 3-kinase yeast VPS34 (40% identity) signif- <br />    FIG. 4.   Detection AtVPS34 Northern blotting. Total <br />                                                                         icantly higher mammalian p110 (26% identity). RNA shoots wild-type transgenic plants probed       result favors conclusion AtVPS34 similar prop- 1.6-kb C-terminal Sac frgent AtVPS34, detecting 2.8-kb         erties ScVPS34. C-terminal domains proteins <br /> band. dilution series evaluate extent overexpression (lanes   contain ATP-binding kinase motifs define pre- <br /> 1-3) suggest a2- 5O-fold higherAtVPS34 expressioninp9.1S (lane       sumed catalytic domain. Point mutations yeast <br /> 1) compared RNA transgenic control           VPS34 abolish detectable PI 3-kinase activity (lane 5) wild-type (ane 6) plants. blot reprobed        complement yeast vps34 deletion mutant (3). Exchange P-tubulin internal control. callus tissue p9.1S showed      255 aa yeast protein plant sequence <br /> similar increase AtVPS34 expression regenerated p9.1S plants.     created fusion protein able restore temperature resis- <br /> (B) PI 3-kinase activity (PI 3-P) callus tissue overexpressing       tance PI 3-kinase activity yeast vps34 deletion <br /> AtVPS34. Protein extracts kanamycin-resistant calli trans- <br /> formation p9.1S control plasmid assayed PI        mutant (Fig. 3). <br /> 3-kise activity, labeled lipids separated TLC.           N-terminal protein thought expressing Atp9.1S calli (lane 3) times PI 3-kinase      ligand binding regulatory role interacting activity yeast cells (lane 1) noalizd equal PI 4-kinase         proteins, VPS15 p85. low sequence <br /> activity (PI 4-F). PI 3-kinase activity detection limit     conservation domain account failure extracts control calli (lane 2).                                     complement yeast mutant, plant VPS34  11402    Plant Biology: Welters et al.                                               Proc. Nad. Acad. Sci. USA 91 (1994) <br /> efficiently recognize yeast VPS15. interaction p110           endocytosis receptors (16). yeast, PI 3-kinase regulatory subunit p85 maps           essential vesicle-mediated delivery proteins served N-terminal region (2).                                        vacuole (3).  AtVPS34 serve essential role <br />    finding AtVPS34 contains putative lipid-binding        similar targeting events  consequence domain N-terminal end (aa 39-170) similar       defect, prevent normal plant development growth p110, rabphilin-3A, synaptotagmin (p65),        antisense plants. <br /> proteins (2) strengthens hypothesis AtVPS34                 <br /> involved vesicle-transport events. CaLB domain         <br /> help bind substrate (PI) permit PI 3-kinase           <br /> association membrane trigger specific transport        <br /> events. resulting phosphorylation PI recruit            <br /> additional proteins membrane, ionic inter-          <br /> action CaLB-containing GTPase activating protein          <br />                                                                     <br /> (GAP) protein (2) covalent binding shown           <br /> elongation factor (EF) la. membrane-bound portion pool <span id='am-6' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span>-binding protein PI anchor cova-            <br /> lently attached internal amino acid (31).              <br />                                                                           <br /> function protein binding bundling actin.      recent finding EF-la plant homolog                   <br /> membrane-bound activator PI 4-kinase (32),           <br /> reports PI 4-kinases surface vesicles (14)             isolated F-actin (33), indicate        <br /> kinase activities crucial vesicle budding processes,             <br /> shown internalization tyrosine receptor kinases        <br /> endocytotic events mammalian cells (2).                                 <br />    PI 3-Klnase Actvity Prnts. PI 3-kinase activity far           plant protein extracts (13),                 <br /> failed detect activity control plant extracts (Fig. 4B,    <br /> lane 2). existence PI 3-kinase inferred              <br /> labeling patterns lipids extracted duckweed (5)            <br /> guard cells Commelina communes (34) labeled appro-           <br /> priate radioactive precursors. detected low levels        <br /> PI 3-kinase activity (compared PI 4-kinase activity)         <br /> transgenic callus cells overexpressing AtVPS34 (Fig. 4B). PI                <br /> 3-phosphate PI 4-phosphate completely sepa-             <br /> rated unique spots chromatographic  Nev-                 <br /> ertheless, believe spot identified PI 3       <br /> caused &quot;fronting&quot; PI 4 spot increased                   <br /> loading (5-fold) did produce spot extract                  untransformed plant used. Headgroup analysis,            yeast (3), necessary confirm unequiv-               <br /> ocally spot represents PI 3-kinase activity. mRNA                enzyme readily detected enzyme                        <br /> activity (Fig. 4A). low level active enzyme                  <br /> caused rapid inactivation cell extracts absence     required activation step, leaving enzyme                     <br /> inactive, occurs ScVPS34-overexpressing yeast cells <br /> </body></html>