<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 92, pp. 9786-9790, October 1995 <br /> Biochemistry <br />  <br />  <br /> Cloning, expression, function TFC5, gene encoding B&quot; component Saccharomyces cerevisiae RNA polymerase <br /> III transcription factor TFIIIB <br />      (promoter complexes/initiation factors/protein-DNA interactions/multiprotein complex assembly/DNA footprinting) <br />  <br /> GEORGE  KASSAVETIS*, STEPHANIE T. NGUYEN*, RYuJi KOBAYASHIt, ASHOK KuMAR*, E. PETER GEIDUSCHEK*, MARINA PISANO*t <br /> *Department Biology Center Molecular Genetics, University California San Diego, La Jolla, CA 92093-0634;   tCold Spring Harbor <br /> Laboratory, P.O. Box 100, Cold Spring Harbor, NY 11724 <br />  <br /> Contributed E. Peter Geiduschek, July 19, 1995 <br />  <br /> ABSTRACT         TFC5, unique essential gene encod-                       pCS6 (containing SNR6 gene coding U6 snRNA) ing B&quot; component Saccharomyces cerevisiae RNA                          described (13, 14). p539H6-derived PU6R  <br /> polymerase III transcription factor (TF)IIIB cloned.                     scribed (ref. 14; S. K. Whitehall, G.  K., encodes 594-<span id='am-6' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> protein (67,688 Da). Escherichia                      E.P.G., unpublished data). TFIIIC affinity-purified box <br /> coli-produced B&quot; used reconstitute entirely recom-                    B-DNA, Mono Q-purified pol III, recombinant yeast TBP <br /> binant TFIIIB fully functional TFIIIC-directed,                    prepared described (1, 7). His6-tagged Brf (10) TATA box-dependent, DNA binding transcription.                        purified Ni2+-chelating agarose cited purification DNase footprints entirely recombinant TFIIIB,                            recombinant B&quot;  Greater purification achieved composed B&quot;, 67-kDa Brf, TATA box-binding                              eliminating previously used step Mono S chromatography <br /> protein, TEIIIB reconstituted natural B&quot; indis-                      (7)  instead, allowing Brf precipitate direct dialysis <br /> tinguishable. truncated form B&quot; lacking 39 N-terminal                        Ni2+-chelating agarose Brf fraction 7 M <span id='am-30' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span>urea</span> 107 C-terminal amino acids functional tran-                       40 mM Tris-HCl, pH 8/100 mM NaCl/10% (vol/vol) glycerol <br /> scription.                                                                        MPLP (10 mM 2-mercaptoethanol/0.5 mM phenylmeth- <br />                                                                                   ylsulfonyl fluoride/1 ,ug leupeptin ml/1 ,g pepstatin central transcription factor (TF) RNA polymerase (pol)                     ml). Brf precipitate resuspended 6 M guanidine <br /> III TFIIIB: TFIIIB suffices recruit pol III                       hydrochloride renatured (15), using bovine serum albumin <br /> multiple rounds accurately initiated transcription (1). Yeast                  (BSA) diluent (16) 500 mM NaCl dilution buffer. <br /> TFIIIB binds avidly greatly varying DNA sequence                            preparation Brf (Fig. 1, lane d), contains stream transcriptional start sites pol 111-transcribed                  67-kDa length protein 49-kDa fragment (resulting <br /> genes requires assembly factor, TFIIIC, proper                         proteolysis internal translational initiation) ex- <br /> positioning (reviewed refs. 2-4). exception                         tends amino acid V165 C-terminal His6 tag, general rule provided yeast U6 snRNA&apos; gene (SNR6),                      15% active formation heparin-resistant TFIIIB-DNA TATA box TFIIIB bind  fact,                            complexes (12). <br /> accurately initiating transcription SNR6 gene                          Conditions analysis transcription, DNase protection, <br /> executed vitro TFIIIB pol III (5-7).                              gel mobility shift SUP4 SNR6 genes    Yeast TFIIIB comprises components (8): TATA                          described (7, 16). Quantities protein specified fmol box-binding protein, TBP; TBP-interacting 67-kDa TFIIB-                       DNA-binding activity TFIIIC, fmol protein capable related protein, Brf (gene BRFI/PCF4/TDS4) (9-11);                          forming heparin-resistant TFIIIB-DNA complexes Brf, <br /> chromatographically separable component named B&quot; (12).                            TBP, B&quot;, fmol pol III active specific transcription <br /> Active B&quot; previously isolated 90-kDa &quot;band&quot;                       (12, 13). <br /> denaturing (SDS) gels, renatured, shown reconstitute                          Purification B&quot; Component TFIIIB Micro- <br /> active TFIIIB. experiments described                     Sequence Analysis. B&quot; extracted 6 M urea S-100 <br /> complete proof constitution TFIIIB cloning                       pellets derived 1600 g Saccharomyces cerevisiae BJ926 <br /> gene encoding B&quot; protein, expressing Escherichia coli,                  subjected chromatography Bio-Rex 70 described using resulting protein reconstitute transcriptionally                 (8). Ninety percent material (70 mg protein) active TFIIIB entirely recombinant constituents.                   loaded 60-ml hydroxylapatite column (Bio-Gel HTP; <br /> Wholly recombinant TFIIIB suffices precise binding                     Bio-Rad) equilibrated 100 mM sodium phosphate, pH <br /> upstream promoter yeast SNR6 gene, accurate                            7.8/20% (vol/vol) glycerol/0.01% (vol/vol) Tween 20 recruitment highly purified TFIIIC yeast tRNA gene                        MPLP, washed 60 ml buffer followed promoter, accurate initiation U6 snRNA tRNA                        sequential 120-ml washes buffer 135, 160, 185, <br /> synthesis yeast pol III.&#194;&#167;                                                      215, 500 mM place 100 mM sodium phosphate (pH <br />                                                                                   7.8). B&quot; TP activity eluted end 160 mM                                                                                   185 mM sodium phosphate step. Eighty  <br />              MATERIAL METHODS                                                 cent material (3.6 mg protein) concentrated <br />   DNA Proteins. Plasmids pTZ2 (containing SUP4                            5-fold (Centriprep 30; Amicon), diluted buffer D [40 mM <br /> tRNATyr G62 -&gt; C promoter mutation), pLNG56  gene C56 -&gt; G promoter mutation), pLNWT                             Abbreviations: TF, transcription factor; pol, RNA polymerase; BSA, <br /> (containing wild-type SUP4 gene; similar pTZ1),                        bovine serum albumin; RACE, rapid amplification cDNA ends; <br />                                                                                   TBP, TATA box-binding protein. <br />                                                                                   tPresent address: Consiglio Nazionale delle Ricerche, Istituto Di publication costs article defrayed page charge         Genetica Molecolare, Alghero (Sassari), Italy. <br /> payment. article marked &quot;advertisement&quot; inL          sequence reported paper deposited accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                      GenBank data base (accession  U31819). <br />                                                                            9786 <br />            Biochemistry: Kassavetis et al.                                                     Proc.   Natli. Acad.   Sci. USA 92   (1995)   9787 <br />  <br />  <br />  <br />                               b   c   d                                    bands estimated Coomassie stain <br />                                                                              density, using dilution series BSA gel                                                                              standard curve. 31 Zg 90-kDa polypeptide (340 pmol) <br />                                              -109                            yielded 170 pmol active B&quot;, concluded                                                                              material desired B&quot; polypeptide(s). <br />                                             &apos;&apos; Brf                              protein major &apos;90-kDa band weaker <br />                                              -70 <br />                   AB&quot;-&apos;                                                      (shadow) band separately isolated microse- <br />                                                (ABtf)                        quencing: 406 Zg protein Mono S B&quot; fraction                                                                              precipitated acetone, resuspended lx sample buffer <br />                                              -44                             (18), separated SDS/PAGE, stained 0.05% Coo- <br />                                                                              massie brilliant blue G. Bands corresponding major                                                                              minor 90-kDa proteins precisely excised, soaked                                                                              HPLC-grade water, stored -70&#194;&#176;C. gel digestion <br />                                                                              method (19) used modification obtain internal <br />                                              -28                             amino acid sequence B&quot;. Individual gel slices digested <br />    FIG. 1. Polypeptide composition B&quot; fractions Brf.        Achromobacter protease (lysylendopeptidase), micrograms Mono S-purified B&quot; (lane  dot right        resulting peptides separated reverse-phase HPLC <br /> lane identifies material subjected microsequencing; slower        (Vydac C18 column: 1.0 x 250 mm, 10 p-im, 300  analysis migrating minor shadow band resolved major B&quot;                amino acid sequence (Applied Biosystems sequencer, models <br /> polypeptide gel), 0.9 ,jg His6-tagged recombinant B&quot; (lane        470 477, line HPLC analysis phenylthiohydan- <br /> b), 1.2 Atg His6-tagged truncated B&quot; (AB&apos;; lane c), -0.5 ,ug       toin-amino acids). reverse-phase chromatographic pro- <br /> purified His6-tagged Brf (lane d; length Brf fragment           files peptides generated digestion major minor <br /> referred text identified right) analyzed      90-kDa protein lysylendopeptidase nearly identical <br /> 10% polyacrylamide/SDS gel stained Coomassie brilliant <br /> blue R.                                                                       additional peptides possibly present                                                                              minor 90-kDa protein). <br /> sodium Hepes, pH 7.8/7 mM MgCl2/20% (vol/vol) glycerol/                         Identification Cloning TFC5 Gene. Degenerate <br /> 0.01% (vol/vol) Tween 20 MPLP] conductivity                    <span id='am-25' about='obo:IMR_0200194' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00294" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200194" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Inosine" datatype="xsd:string"></span>inosine</span>-containing oligodeoxynudt6otides based <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> <br /> buffer D plus 150 mM NaCl, loaded Mono S (Pharmacia,                    sequence synthesized (Operon Technologies, Alameda, <br /> HR 5/5 column) equilibrated buffer D plus 150 mM NaCl,                    CA) sense antisense orientation amplification eluted 25-ml 150-600 mM NaCl gradient buffer                   genomic DNA fragments coding portions B&quot; gene D. B&quot; activity eluted 290 350 mM NaCl; 2 mg                   PCR (Table 1). Sense antisense primer pairs (100 pmol <br /> total protein contained &quot;170 pmol B&quot; measured forma-                    combined 10 ng yeast genomic DNA tion heparin-resistant TFIIIB-DNA complexes excess                   20 dNTPs touchdown PCR amplification (20). Single- <br /> recombinant Brf TBP gel retardation assays (8).                       primer control reactions (21) proved useful eliminating <br />    Substantial purification Mono S-purified B&quot; (Fig. 1, lane              incorrect products. sense antisense primer pair  achieved chromatography poly(dA-dT)-poly(dA-                    capable correctly amplifying portion B&quot; gene did  <br /> dT) coupled CNBr-activated Sepharose (17)                   resulting map PCR products confirmed gener- <br /> polypeptide band 67- 100-kDa range (data shown                 ating predicted internal PCR products fromh larger, gel-isolated marked Fig. 1, lane  coeluting B&quot; activity.                PCR products (22) identifying additional  <br /> identity &quot;90-kDa polypeptide B&quot; confirmed                  determined peptide sequence DNA sequence. <br /> subjecting 7 ,ug protein Mono S-derived B&quot; fraction                 Confirmed PCR products derived primer combina- SDS/PAGE. material stained Coomassie brilliant                   tions 2 + 7, 2 + 8, 4 + 8 wete labeled nick-translation <br /> blue G precisely excised, eluted, renatured                    (23) (4-6 x 108 cpm/yg DNA) sequentially used scribed (8) slices similar size taken                probe mapping membranes- S. cerevisiae (prime clone grid presumptive band B&quot; protein. B&quot; activity                    filters; ATCC 77284) colony blots S. cerevisiae <br /> observed gel slices: presumptive B&quot; polypeptide               genomic plasmid library [ATCC 77162, kindly provided weak (shadow) band directly  quantity protein              J. L. Cereghino S. Emr, University California San <br />                      Table 1. Oligonucleotides <br />                                               Peptide <br />                       Location*           sequence         Dir.t                      5&apos; -&gt;3&apos; sequence* <br />                        1       33-39        TPQLFIP             S      ACICCICARYT-ITTYATHCC <br />                        2     218-224       PPTAMTD              S      CCICCIACIGCNATGACNGA <br />                        3     218-224       PPTAMTD                   TCIGTCATIGCIGTNGGNGG <br />                        4     396-403 KVDENPFA                   S      AAIGTIGAIGARAAYCCNTTYGC <br />                        5     396-402        KVDENPF                  AAIGGITTYTCRTCNACYTT <br />                        6     406-412       YNYGSYG              S      TAIAAYTAYGGITCNTAYGG <br />                        7     406-413 YNYGSYGR                        CTICCRTAIGAICCRTARTTRTA <br />                        8     490-496        KNIGTVA             S      GCIACIGTDCCDATRAAYTT&#194;&#167; <br />                        9         N               -S                    GGTATCCATGGGTAGTATTGTTAATAAAAGTG <br />                       10         C-                                  AATGTCT7CGAGTTGATCAATCTCAGG <br />                       11      N (40)             -S                    TTGTCCATGGCCGAAAGTAAAGAAATAGA <br />                       12      C (487)                                TCTTCITCGAGTTCGCAGCAATACTCAT <br />                      *Amino~ acid positions Fig. 2; oligonucleotides 9-12 used generate expression plasmids                       complete truncated B&quot; proteins. <br />                      t(S)ense  ntisense orientation oligonucleotide. <br />                      tI: inosine; R: purine; Y: pyrimidine; H:  C, T; D:  G, T; N:  G, C, T. <br />                        underlined residues T residues, error specification peptide <br />                       microsequencing. <br />  9788      Biochemistry: Kassavetis et al.                                              Proc. Natl. Acad. Sci. USA 92    (1995) <br />                                                                         rose (Qiagen) 25 g wet weight cells denaturing <br /> MSSIVNKSGTRFAPKVRQRRAATGGTPTPKPRTPQLFIPESKEIEEDNSD-50                   conditions, yielding -2.6 mg -6 mg length NDKGVDENETAIVEKPSLVGERSLEGFTLTGTNGHDNEIGDEGPIDASTQ-100                  truncated B&quot;, respectively (Fig. 1, lanes b c). B&quot;                                                                         renatured Ni2+-chelating agarose elution buffer <br /> NPKADVIEDNVTLKPAPLQTHRDQKVPRSSRLASLSKDNESRPSFKPSFL-150                  containing 7 M urea (adjusted pH 8 Tris base soon                                                                         elution) rapid addition 100 volumes BSA diluent (16). <br /> DSSSNSNGTARRLSTISNKLPKKIRLGSITSNDMNLKTFiRHRVLGKPSS-200                  activity recombinant, length B&quot; determined                                                                         double-reciprocal plots formation heparin-resistant <br /> AKKPAGAHRISIVSKISPPTArMTDSLDRNEFSSETSTSREADENENYVIS-250                 TFIIIB-tDNA complexes fixed quantity B&quot; (estimat- <br />                                                                         ed relative BSA standard) increasing concentrations <br /> KVKDIPKKVRDGESAKYFIDEENFTMAELCKPNFPIGQISENFEKSKMAK-3 00                 TFIIIC-Brf-TBP-tDNA complexes. Extrapolation KAKLEKRHLRELRMRARQEFKPLHSLTKEEQEEEEEKRKEERDKLLtAD-350                   squares best-fit linear plot (r2 = 0.9996) infinite DNA <br />            *                         II <br />                                                                         substrate indicated =100% active molecules. specific <br /> IPESDRKAHTAIQLKLNPDGTMAIDEETMVVDRHKNASIENEYKEKVDEN-400                  activity truncated version B&quot;, quantified, <br />                                                                         appears comparable. <br /> PFANLYNYGSYGRGSYTDPWTVEEMIKFYKALSMWGTDFNLISQLYPYRS-450 <br />                                              RKQVKAKFVNEEKKRPILIELALRSKLPPNFDEYCCEIKKNIGTVADFNE-500                                             RESULTS <br />            .1 <br /> KLIELQNEHKHHMKEIEEAKNTAKEEDQTAQRLNDANLNKKGSGGIMTND-550 <br />                                                                            Identification Isolation TFC5 Gene. B&quot; <br />                                                                         component TFIIIB purified 6 M urea-extracted <br /> LKVYlTEVVIGTIDDLKRKKLKERNNDDNEDNEGSEEEPEIDQl )-594                      S-100 cellular debris basis ability reconstitute <br />                                                                         active TFIIIB combination recombinant TBP Brf. <br />    FIG. 2. Amino acid sequence B&quot; component TRIIIB.           DNA-protein photocrosslinking (28) renaturation B&quot; <br /> Peptide sequences obtained microsequencing Mono S-purified B&quot;        TF activity denaturing gels (8) identified B&quot; underlined (dots placed unassigned uncertain    consisting =90-kDa polypeptide(s). pu- turned  incorrectly assigned residues). N C termini   rification B&quot; precise identification corresponding <br /> B&quot; sequence remaining truncated form B&quot; (residues 40      polypeptide band (Fig. 1, lane  described Material 487) designated line. <br />                                                                                                  Methods. gel digestion lysylendopeptidase micro- <br /> Diego (UCSD)] specified (24). Hybridization             sequencing generated subset peptide sequences   <br /> probes prime clone grid filters identified cosmid c8487          lined Fig. 2) used design degenerate oligonu- <br /> (ATCC 71191) containing gene coding B&quot;. Adjacent             cleotides (Table 1, oligonucleotides 1-8) PCR amplifica- partially overlapping clones 7151 2713 failed              tion genomic DNA fragments encoding B&quot;. hybridize probes, limiting location B&quot;        confirmed amplification products  Material Methods) <br /> gene kbp 534.8 553.1 physical map                                                                         sequentially used probe phage cosmid prime <br /> chromosome XIV. clones B&quot; gene                   clone grid filters covering 95% S. cerevisiae genome. obtained screening colony blots genomic plasmid               probes identified presence gene encoding B&quot;, <br /> library. presence gene cosmid c8487                                                                         TFCS  fifth transcription factor gene class C (pol III) library-derived plasmids confirmed PCR ampli- <br />                                                                         transcription], cosmid c8487, containing centromere- <br /> fication B&quot; gene sequence sources. gene <br />                                                                         proximal sequence chromosome XIV. sequence                                                                         2445-bp region cosmid c8487 contains open reading <br /> coding B&quot; sequenced strands                  frame deduced 594-amino acid sequence (67,688 Da) <br /> upstream (507 bp) downstream (157 bp) flanking se-                  contains 14 peptide sequences derived B&quot;. <br /> quence, continuing strand 220 bp upstream               disparity apparent sequence-deduced 450 bp downstream. Mapping 5&apos; 3&apos; ends                 size B&quot; (90 vs. 68 kDa) led examine possibility mRNA coding B&quot; utilized 5&apos; 3&apos; RACE systems (RACE:               gene coding B&quot; contains intron.  analysis <br /> rapid atnplification cDNA ends; GIBCO/BRL) se-                   3&apos; 5&apos; ends mRNA coding B&quot; RACE quencing amplified products using total yeast RNA gen-           sequencing products placed poly  sites -52 =106 <br /> erously provided G. Cereghino  E. Scheffler (UCSD).            nt downstream ochre stop codon TFCS open <br />    Disruption B&quot; gene followed standard procedures (25)          reading frame 5&apos; end -39 nt upstream initiation HindlIl fragment  pFL1, ref. 26) containing             codon, evidence splice junctions (data shown). <br /> URA3 gene replacing HindIII fragment TFCS encoding                 Comparison amino acid sequence B&quot; amino acids 170-366 (Fig. 2) pET21d expression clone             nonredundant combined data base National Center described introduction Xba Xho fragment              Biotechnology Information (BLASTP, 1995) (29) revealed disrupted B&quot; gene diploid S. cerevisiae strain             significant homologies outside <span id='am-13' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-14' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span>glutamic acid</span>-rich, <br /> MY4298 (ura3, leu2, his3) (27). MY4298 pFL1 kindly             highly charged region residues 329-357, homology <br /> provided M. Nickas M. Yaffe (UCSD).                              nucleolin ORF YKL202W S. cerevisiae <br />    TFCS Gene Expression E. coli Purification               chromosome XIV noted. did discern known <br /> combinant B&quot;. B&quot; coding sequence PCR-amplified                  sequence motifs B&quot; (PROsmrE, 1995) (30) cosmid c8487 high-fidelity Pfu DNA polymerase             exception potential signals glycosylation, phosphoryla- <br /> (Stratagene), using oligonucleotides 9 10 (Table 1)              tion, nuclear localization. TFCS essential cell  <br /> generate Nco site N-terminal methionine (mutating           bility: replacement B&quot; amino acid residues 170-366 adjacent S G) Xho site ochre stop codon.         URA3 gene transformation ura- diploid yeast, fol- amplified product cut Nco Xho                  lowed sporulation, produced asci, 110 dissected, <br /> inserted pET2ld, extending C-terminal amino acid               containing viable spores. <br /> sequence LEHHHHHH. similar strategy, oligo-                    Reconstitution Entirely Recombinant TFIIIB. confirm <br /> nucleotides 11 12 (Table 1), used clone truncated          TFCS gene coding B&quot;, B&quot; consists <br /> version B&quot;, comprising amino acids 40-487, MA            single 68-kDa polypeptide migrates anomalously N terminus LEHHHHHH C terminus, pET2ld                  denaturing electrophoresis (apparent size, =90 kDa), expression E. coli BL21(DE3)pLysS. resulting                 His6-tagged B&quot; E. coli purified Ni2+-chelating tagged B&quot; proteins purified Ni2+-chelating aga-             agarose denaturing conditions  Material Meth- <br />            Biochemistry: Kassavetis et al.                                                         Proc. Natl. Acad. Sci. USA 92 (1995)            9789 <br />                                 b c d e f g h                                relatively impure yeast B&quot;  TFIIIE (32)] affect                                                                                properties. Fig. 4 compares electrophoretic mobilities                                   nI   -    * <br />                                                       * rm                     TFIIIB-DNA complexes formed recombinant B&quot;, recom- <br />                                                                                binant TBP, recombinant Brf TFIIIC-dependent assem- <br />                                                                                bly SUP4 tRNA gene (lanes  TFIIIC-independent <br />                                                                                assembly SNR6 gene (lanes j-n). SUP4 gene, <br />                                                                                addition Brf TBP TFIIIC-DNA complexes (lane b) <br />                  SUP4-o-*                             *U6                      generated slightly slower migrating B&quot;-TFIIIC-DNA com- <br />                                                                                plex (lane c). Inclusion recombinant B&quot; generated ftu-ther <br />                                                                                retardation expected TFIIIB-TFIIIC-DNA complexes (lane <br />                      B&quot;: -y r rA- y r rA                                       d). Treatment heparin stripped TFIIIC generated                           tRNA     U6                                          TFIIIB-DNA complex (lane e) mobility corre- <br />   FIG. 3. length truncated recombinant B&quot; competent               sponding complexes formed Mono S-purified B&quot; (lane g). vitro transcription SUP4 tRNA SNR6 U6 small nuclear          Formation complexes SUP4 gene completely <br /> RNA (snRNA) genes. nanograms pTZ2 encoding                  TFIIIC-dependent (lane f). Truncated B&quot; generated slightly <br /> SUP4 tRNATYr gene (lanes d) pCS6 encoding SNR6 U6                     faster migrating TFIIIB complex stable heparin <br /> snRNA gene (lanes e-h) incubated 100 fmol recombinant              (lane h; appearance discrete heparin-resistant bands <br /> TBP, 27 fmol recombinant Brf, 10 fmol purified pol III, 29 fmol          reproducible resulted adventitious purified TFIIIC (lanes d , 5 finol Mono S-purified            proteolysis). Recombinant natural B&quot; likewise generated <br /> yeast B&quot; (lanes b f; designated &quot;y&quot; figure), 46 fmol recombinant His6-tagged B&quot; (lanes c g; designated &quot;r&quot;), 4 ng         heparin-resistant complexes SNR6 gene identical <br /> truncated B&quot; (lanes d h; designated &quot;rA&quot;) 60 min 20&#194;&#176;C.              mobilities (lanes m k). absence component <br /> Labeled ribonucleotides added 30 min, generating                 derived yeast reaction proves heparin <br /> multiple rounds transcription. SUP4 U6 transcripts              resistance TFIIIB-DNA complex solely property <br /> identified; r.m., labeled DNA recovery marker.                                 constituent polypeptides. presence hep- <br />                                                                                arin routinely diminishes electrophoretic mobility ods). Recombinant B&quot; migrated anomalously &quot;90-kDa                        TFIIIB-DNA complex (compare lanes 1 m), possibly protein SDS/PAGE (Fig. 1, lane b), indistinguishable                     interaction TFIIIB  ref. 7, Fig. 3). <br /> mobility natural B&quot; (Fig. 1, lane . truncated variant B&quot;, lacking 39 N-terminal 107 C-terminal amino acids                                          : h   r d     f   n   h   (AB&quot;; Fig. 2; molecular mass, 52 kDa), purified E. coli C-terminally His6-tagged protein, migrated anomalously -61-kDa protein SDS/PAGE (Fig. 1, lane c). <br />    Recombinant length B&quot; truncated AB&quot; able replace natural yeast B&quot; TFIIIC-dependent tRNA gene <br /> transcription (Fig. 3, compare lanes c d lane b)                               +1   - <br />                                                                                                                                       -40 U6 snRNA gene transcription (Fig. 3, compare lanes g                                -7 <br /> h lane f); nonrecombinant protein U6 assay highly purified pol III (31). <br />    stability structure TFIIIB-DNA complexes formed recombinant natural B&quot; compared deter- missing modification recombinant <br /> protein lack uncharacterized protein                                                                              -10 <br />                                                                                            -39- <br />                   b c d e f g h          j k        m n <br />  B-C-tDNA-, <br /> B&apos;-C-tDNA <br />                     4   .!-                                                                                                        -   +1 <br />    C-tDNA--:r <br />    B-tDN                          *0                       --c-   B-U6 DNA <br />  <br />                                                                                               B&quot;:     - y rrA - - y rrA - <br />                                                                                             gene:       SUP4       SNR6 <br />      tDNA- -                                                        U61DNA        FIG. 5. TFIIIB reconstituted natural recombinant B&quot; <br />                    -c cc c-cc                -                                 generates identical DNase footprints SUP4 tRNAtYr SNR6 <br />                   - - B&apos; B&apos; B&apos;B&apos;BP Bl B&apos;BBB&apos; B&apos;                                (U6 snRNA) genes. femtomoles 298-bp fragment pLNWT <br />            B&quot;:    ---r r r y rA rA - y r r rA                                  containing wild-type SUP4 gene, 5&apos; end-labeled Xba site <br />      heparin:                    +   + +    +         + +                       nontranscribed strand), incubated (lanes                                 SUP4             U6                            e) TFIIIC (57 fmol), TBP (100 fmol), Brf (34 fmol),                                                                                Mono S-purified yeast B&quot; (5 fmol, lane b), recombinant B&quot; (36 fmol, <br />   FIG. 4. Fully recombinant TFIIIB (Bn + Brf + TBP) forms                      lane c), truncated, recombinant AB&quot; (6 ng, lane d) 15 ,ul buffer <br /> heparin-resistant complexes SUP4 SNR6 genes.                  60 min 20&#194;&#176;C. Heparin added  125 ,ug/ml) 8 min, <br /> components incubated 60 min 21&#194;&#176;C 15-,ul volume included 5.5          followed DNase digestion 30 sec, 21&#194;&#176;C, reactions <br /> fmol &quot;300 bp end-labeled fragment pTZ2 containing                 terminated adding EDTA 20 mM. Free DNA heparin- <br /> SUP4 tRNAtyr gene (lanes  5.8 fmol 201-bp end-labeled              resistant TFIIIB-DNA complexes separated 4% native <br /> fragment pU6R containing SNR6 gene (lanes j-n)                  polyacrylamide gel partial DNase digestion patterns following proteins designated: TFIIIC (C; 58 fmol), recombinant             analyzed 10% polyacrylamide/8 M urea gel, shown. yeast Brf TBP (B&apos;; 42 100 fmol, respectively), Mono S-purified         samples displayed lanes f-j identical lanes e, <br /> yeast B&quot; (y; 5 fmol), recombinant His6-tagged B&quot; (r; 46 fmol),             respectively, TFIIIC omitted, 4.4 fmol SNR6 <br /> truncated, recombinant AB&apos; (rA; 4 ng). Heparin added 125                gene probe (3&apos; end-labeled transcribed strand) noted Fig. <br /> ,ug/ml 5 min prior loading 4% native polyacrylamide gel,           4 replaced SUP4 tRNA gene probe. start sites transcription indicated. DNA complexes containing TFIIIC, TFIIIB, B&apos;              (+ 1) extent footprints (determined Phosphorlmager <br /> component TFIIIB  e., Brf + TBP) identified sides.             scan) indicated sides. <br />  9790      Biochemistry: Kassavetis et al.                                                   Proc. Natl. Acad. Sci. USA 92         (1995) <br />   DNase footprints SUP4 (Fig. 5, lanes b-d)                  known (35). combinations candidate sites <br /> SNR6 complexes (lanes g  formed TFIIIB reconstituted                 deviating nucleotides. sequence 5&apos; end natural (Mono S-purified) B&quot; (lanes b g) recom-              5&apos; RACE product shows indication splicing binant length B&quot; (lanes c h) indistinguishable.           N-proximal end gene, 3&apos; RACE indicates poly  <br /> subtle diminution enhancement near start                    addition sites relatively close downstream end site-proximal border noted complex recombi-                translational open reading frame. <br /> nant truncated AB&quot; (lanes d  SUP4 (lane d). absence yeast-derived component TFIIIB-SNR6                   grateful C. Bowman S. Beardsley experimental <br /> complex shown lane h proves extent TFIIIB              assistance, P. Scott generous help tetrad dissections, <br /> footprint solely B&quot;, Brf, TBP.                              S. Emr generous hospitality,&quot; C.  P. Joazeiro, S. K. <br />                                                                           Whitehall, G. Cereghino, J. Kadonaga discussions                                                                           advice, J. Kadonaga S. K. Whitehall critical reading                           DISCUSSION                                      manuscript. M. P. graduate fellow University                                                                           Sassari. research supported National Institute experiments complete pr <br /> </body></html>