<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Accumulation Phosphatidic Acid Increases Vancomycin Resistance Escherichia coli <br /> Holly  Sutterlin Sisi Zhang,b Thomas J. Silhavya <br /> Department Molecular Biology, Princeton University, Princeton, New Jersey, USAa; Lewis-Sigler Institute Integrative Genomics, Princeton University, Princeton, New <br /> Jersey, USAb <br />  Gram-negative bacteria, <span id='am-49' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-50' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-52' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="LPS" datatype="xsd:string"></span>lipopolysaccharide</span> (<span id='am-39' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-47' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span>LPS</span>) contributes robust permeability barrier outer membrane, pre- <br /> venting entry toxic molecules antibiotics. Mutations lptD, beta-barrel component LPS transport  <br /> sembly machinery, compromise LPS assembly result increased antibiotic sensitivity.  report rare vancomycin- <br /> resistant suppressors improve barrier function subset lptD mutations. seven suppressors analyzed <br /> mapped essential gene cdsA, responsible conversion phosphatidic acid <span id='am-65' about='obo:IMR_0200468' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="KEGG:C00112" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0200468" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="CDP" datatype="xsd:string"></span>CDP</span>-<span id='am-104' about='obo:IMR_0100027' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-105' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="diacylglycerol" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:id" content="IMR:0100027" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="KEGG:C00165" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasExactSynonym" content="DAG" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="diacylglyceride" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="diacylglyceride" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasExactSynonym" content="Diglyceride" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Diglyceride" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="DAG" datatype="xsd:string"></span>diacylglycerol</span> phos- <br /> pholipid biosynthesis. cdsA mutations cause partial loss function  expected, accumulate phosphatidic acid. suppression confined mutations cause defects outer membrane biogenesis cdsA mutations confer general increase vancomycin resistance, wild-type cell. use genetics quadrupole time flight (Q-TOF) liquid chromatography-mass spectrometry (LC-MS) accumulation phosphatidic acid means cdsA mutations increases resistance vancomycin. suggest increased levels phosphatidic acid change physical properties outer membrane impede entry vancomycin periplasm, hindering access target, intermediate required synthesis <span id='am-82' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-83' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-91' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="PGN" datatype="xsd:string"></span>peptidoglycan</span> cell wall. <br />  <br />  <br />  <br /> T   outer membrane (OM) Gram-negative bacteria serves     robust permeability barrier (1). Composed lipopolysac- <br /> charide (LPS) molecules outer leaflet glycerophospho- <br />                                                                                        CDP-diacylglycerol step commitment                                                                                        zwitterionic GPL pathway acidic GPL pathway (6, 12). <br />                                                                                            work, focus E. coli mutants defective LPS  <br /> lipids (GPLs) inner leaflet, OM protects cell                      sembly exhibit increased sensitivity antibiotics. Mutations <br /> harmful agents, detergents antibiotics (1&#226;&#8364;&#8220;3). LPS                       compromise function LptD, beta-barrel compo- <br /> synthesized inner membrane (IM), transported                         nent Lpt machinery, sensitive vancomycin. aqueous periplasm, finally assembled outer leaflet                     order act cell wall target, vancomycin traverse OM (4). Lpt proteins responsible transport                        OM. periplasm, vancomycin binds termi- assembly (5). Glycerophospholipids (GPLs) synthesized                          nal D-alanine residues cell wall precursor <span id='am-33' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-34' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-37' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipid</span> II, pre- IM, little known transport                        venting polymerization peptidoglycan cell wall (13, 14). <br /> periplasm assembly OM (6). coordination LPS                        lptD mutants exhibit increased sensitivity vancomycin GPL assembly important maintain effective OM                          allow efficient entry drug periplasm <br /> meability barrier (1). <br />                                                                                        result compromised OM barrier. <br />     high density LPS outer leaflet, OM <br />                                                                                            report isolation characterization vancomy- <br /> carries net negative charge (7). human innate immune sys- <br />                                                                                        cin-resistant suppressors subset lptD mutations. sup- <br /> tem takes advantage property OM producing <br />                                                                                        pressors map essential gene cdsA. Vancomycin-resistant <br /> cationic antimicrobial peptides (CAMPs) (8). Positively charged <br /> CAMPs associate negatively charged cell surface                        suppression specific lptD mutants; cdsA mutations <br /> sert OM, killing bacterial cell (8). Bacteria sys-                   actually increase vancomycin resistance wild- <br /> tems place, PhoPQ virulence Salmonella                        type cell. cdsA mutations cause recessive, partial loss enterica serovar Typhimurium, modify lipid PL                           function, accumulate PA like cdsA mutants previ- <br /> content decrease negative charge OM,                          ously described (15, 16). suggest increased PA alters creasing affinity CAMPs OM (9&#226;&#8364;&#8220;11). increases                       OM causes increased resistance vancomycin. <br /> bacterial resistance CAMPs enables evasion immune  <br />     Like LPS, GPLs contribute electric charge membrane. Escherichia coli membranes composed major GPLs: phosphatidylethanolamine (PE), phosphatidylglyc-                              Received 20 2014 Accepted 16 June 2014 <br /> erol (PG), cardiolipin (CL). relative abundances                          Published ahead print 23 June 2014 GPL species similar IM OM (PE, 75%; PG,                           Address correspondence Thomas J. Silhavy, tsilhavy@exchange.Princeton.EDU. <br /> 20%; CL, 5%) (12). PE zwitterionic GPL neutral charge,                        Supplemental material article http://dx.doi.org/10.1128 PG CL acidic GPLs carry negative charges (6).                          /JB.01876-14. GPLs different <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> properties                             Copyright &#194;&#169; 2014, American Society Microbiology. Rights Reserved. head groups, synthesized precursor                           doi:10.1128/JB.01876-14 <br /> molecule, phosphatidic acid (PA) (6, 12). conversion PA  <br />  <br /> 3214   jb.asm.org                                   Journal Bacteriology   p. 3214 &#226;&#8364;&#8220;3220                                     September 2014 Volume 196 Number 18 <br />                                                                                                            Phosphatidic Acid Increases Vancomycin Resistance <br />  <br />  <br />  <br /> TABLE 1 Strains plasmids used study                               romycin, 30 &#226;?&#174;g moxalactam, 0.1 &#226;?&#174;g norfloxacin, 260 &#226;?&#174;g paraquat, 300 <br /> Strain                                                  Reference          units polymyxin B, 1 mg vancomycin. Isopropyl-&#226;?&#164;-D-1-thiogalactopy- <br /> plasmid            Relevant features                       source      ranoside (IPTG; 0.1 mM) present LB agar indicated. <br />                                                                                   Growth rate determination. Overnight cultures diluted 1:100 Strains                                                                       100 &#226;?&#174;l LB medium 96 plates. microliters min- <br />    MC4100          F&#226;&#171;&#186; araD139 &#226;&#338;&#172;(argF-lac)U169 rpsL150      17                 eral oil overlaid prevent evaporation. Optical density 600 nm <br />                      relA1 flbB5301 deoC1 ptsF25 rbsR                         (OD600) measured 20 min intermittent orbital shaking                      thi                                                      37&#194;&#176;C. 4-hour OD600 growth curves used determine   NR754            MC4100 Ara&#226;&#171;&#185;                             18                 growth rate. OD600 plotted time logarithmic scale,   HC334            NR754 &#226;&#338;&#172;lptD pET2342::lptDI54E           21, 48, 49;   linear regression analysis used determine growth rate                                                               study           culture. strain, technical replicates biological rep- <br />   HC338            NR754 &#226;&#338;&#172;lptD pET2342::lptDV52E-I54E      21, 48, 49;   licates averaged mean averages calculated                                                               study           reported standard error mean. <br />   HC340            NR754 &#226;&#338;&#172;lptD pET2342::lptD               21, 48, 49;       Cell size measurements. Overnight cultures diluted 1:100                                                               study           grown OD600 approximately 0.4. Cells suspended 1% <br />   HC342            NR754 cdsA7 dgt::Tn10                   study         agarose pad imaged using 100&#226;&#171;&#187; 1.4-numerical-aperture (NA) ob- <br />   HC351            HC334 &#226;&#338;&#172;clsA::kan                        34; study     jective Nikon 90i microscope equipped QImaging Rolera XR <br />   HC386            HC334 cdsA7 dgt::Tn10                   study         camera NIS Elements software. Cell dimensions quantified   HC509            NR754 &#226;&#338;&#172;lptD pET2342::lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60       21, 48, 49;   MicrobeTracker version 0.936 (22), using MATLAB version R2013a. Ap- <br />                                                               study           proximately 100 cells examined independent exper- <br />   HC510            HC334 pgsA444 zed-3069::Tn10            39, 50, 51;   iments. P values calculated using Student&#226;&#8364;&#8482;s t test. <br />                                                               study               Bligh Dyer lipid extraction. Lipids extracted using acidic <br />   HC530            HC509 cdsA7 dgt::Tn10                   study         Bligh Dyer method previously described (23, 24). milli- <br />   HC571            NR754 &#226;&#338;&#172;clsA::kan                        34; study     liter cultures grown OD600 1.5, cells   HC575            NR754 pgsA444 zed-3069::Tn10            39, 50, 51;   harvested centrifugation. Cell pellets washed phos- <br />                                                               study           phate-buffered saline (PBS) resuspended 1 ml 0.1 N HCl. <br />   HC630            HC509 pgsA444 zed-3069::Tn10            39, 50, 51;   Methanol (2.5 ml) chloroform (1.25 ml) added form single- <br />                                                               study           phase solution (chloroform-methanol-0.1 N HCl [1:2:0.8 {vol/vol/vol}]) <br />   HC960            HC334 cdsA9 dgt::Tn10                   study         incubated 30 min room temperature sporadic vortexing. <br />   HC961            HC509 cdsA9 dgt::Tn10                   study          0.1 N HCl (1.25 ml) chloroform (1.25 ml) added form <br />   HC962            NR754 cdsA9 dgt::Tn10                   study         phase solution (chloroform-methanol-0.1 N HCl [2:2:1.8 {vol/vol/ <br />   HC1045           NR754 pASKA::plsC                       35; study     vol}]). solutions centrifuged 3,000 &#226;&#171;&#187; g 25 min   HC1198           JAS193 cdsA7 dgt::Tn10                  study         room temperature. lower phase recovered dried nitro- <br />   HC1199           JAS193 cdsA9 dgt::Tn10                  study         gen stream. <br />   HC1240           HC342 pASKA::cdsA                       35; study         Mass spectrometry. Samples reconstituted 500 &#226;?&#174;l 1:1:0.3 <br />   HC1241           HC962 pASKA::cdsA                       35; study     methanol-chloroform-water. Mass spectrometric analyses  <br />   JAS193           MC4100 &#226;&#338;&#172;bamE::kan                       J. Schwalm,        formed 6550 Accurate-Mass quadrupole time flight (Q-TOF) mass <br />                                                               unpublished     spectrometer (Agilent Technologies, , Santa Clara, CA), equipped <br />                                                               data            electrospray ionization (ESI) source operated negative- <br /> Plasmids                                                                      positive-<span id='am-59' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-60' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-61' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> mode. Liquid chromatography (LC) performed  <br />   pET2342::lptD    lptD cloned pET2342                48                 ing 1290 Infinity HPLC (Agilent Technologies, , Santa Clara, <br />   pASKA::cdsA      IPTG inducible, Camr                    35                 CA) coupled mass spectrometer, scans m 200 1,000 <br />   pASKA::plsC      IPTG inducible, Camr                    35                 2-GHz extended dynamic range. Nitrogen used drying <span id='am-5' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>gas</span>, neb- <br />                                                                               ulizer gas, collision gas. Mass spectrometer operation conditions                                                                               follows: capillary voltage, 4,000 V; fragmentor voltage, 225 V; nebulizer <br />                                                                               gas, 35 lb/in2; drying gas, 13 liter/min; gas temperature, 275&#194;&#176;C.  <br /> MATERIALS METHODS                                                         strument control, data acquisition, data analysis performed Bacterial strains, plasmids, media, chemicals. strains used    Agilent Mass Hunter software (Agilent Technologies; version <br /> work isogenic derivatives NR754, araD&#226;&#171;&#185; revertant MC4100          B.05.00), controlled chromatography  <br /> (17, 18). Strains constructed generalized P1 transduction trans-       LC parameters follows: column, Agilent Poroshell-120 <br /> formation (19). Suppressor mutations mapped using pools ran-          EC-C18 (2.7 &#226;?&#174;m, 2.1 150 mm); autosampler temperature, 5&#194;&#176;C; injec- <br /> dom Tn10 insertions described previously (20). necessary, kan cas-      tion volume, 10 &#226;?&#174;l; column temperature, 60&#194;&#176;C; solvent flow rate, 150 <br /> settes excised using Flp recombinase (21). Strains plasmids      &#226;?&#174;l/min. Solvent 1 mM ammonium acetate plus 0.2% <span id='am-129' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-130' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span>acetic acid</span> used work listed Table 1. cultures grown Luria-    90:10 water-methanol, solvent B 1 mM ammonium acetate plus <br /> Bertani (LB) broth agar 37&#194;&#176;C. mass spectrometry (MS), high-         0.2% acetic acid 2:98 methanol&#226;&#8364;&#8220;2-propanol. gradient 0 min, <br /> pressure liquid chromatography (HPLC)-grade solvents (water, metha-           25% B; 2 min, 25% B; 4 min, 65% B; 16 min, 100% B; 20 min, 100% B; 21 <br /> nol, isopropanol), acetic acid, ammonium acetate obtained        min, 25% B; 22 min, 25% B. Postrun time 6 min. Fisher Scientific (Pittsburgh, PA). <br />     Antibiotic sensitivity assays. Efficiency plating assays (EOPs)     RESULTS <br /> disk diffusion assays used determine antibiotic sensitivity.                                                                               Vancomycin-resistant suppressors lptD mutations map EOPs, strains grown overnight, serially diluted, replica <br /> plated LB agar LB agar supplemented vancomycin (70 mg/liter,       cdsA. Escherichia coli lpt mutants exhibit defects transport <br /> 120 mg/liter, 140 mg/liter, 150 mg/liter, 200 mg/liter), 625 mg/liter      assembly LPS exhibit sensitivity detergents bacitracin, 1 mg/liter gentamicin, 10 mg/liter rifampin. disk diffu-   antibiotics (18, 25&#226;&#8364;&#8220;31). lptDI54E, lptDV52E-I54E, lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60 <br /> sion assays, disks contained following amounts antibiotics: 1.25   harbor denoted mutations alter soluble N terminus mg ampicillin, 320 &#226;?&#174;g cetyltrimethylammonium bromide, 125 &#226;?&#174;g eryth-           OM beta-barrel protein LptD confer sensitivity vanco- <br />  <br />  <br /> September 2014 Volume 196 Number 18                                                                                                       jb.asm.org 3215 <br />  Sutterlin et al. <br />  <br />  <br />  <br />                                                                                         cdsA essential gene (16, 34), suppressors  <br />                                                                                     null mutations. used complementation analysis  <br />                                                                                     termine cdsA7 cdsA9 mutations dominant <br />                                                                                     recessive. cdsA clone  Complete Set E. coli <br />                                                                                     K-12 ORF Archive&#226;&#8364;? (ASKA) collection introduced                                                                                     cdsA7 cdsA9 backgrounds (35). Basal-level (uninduced) ex- <br />                                                                                     pression wild-type copy cdsA ASKA plasmid  <br />                                                                                     creased vancomycin resistance wild-type levels  Fig. S1 <br />                                                                                     supplemental material). cdsA7 cdsA9 reces- <br />                                                                                     sive, conclude partial loss function mutations. <br />                                                                                         cdsA mutations alter cell size. Previously,                                                                                     shown mutations confer LPS transport defects                                                                                     suppressed mutations alter fatty acid biosynthesis (36), <br />                                                                                     wanted determine suppressors acted similar <br />                                                                                     manner. fabH accD mutations suppress certain lpt muta- <br />                                                                                     tions reducing cell size growth rate, turn decrease <br />                                                                                     rate growth cell envelope (36). cdsA7 cdsA9 <br />                                                                                     mutants did exhibit decrease cell volume  Fig. S2                                                                                     supplemental material), growth rates did differ sig- <br />                                                                                     nificantly wild type  Table S1). conclude <br />                                                                                     results cdsA mutations suppress lptD <br />                                                                                     mutations decreasing rate growth cell envelope. <br />                                                                                         Partial loss function cdsA mutants generally increase <br />                                                                                     vancomycin resistance E. coli strains. cdsA sup- <br />                                                                                     pressors isolated different lptD mutant backgrounds,                                                                                     asked mode suppression specific lptD <br />                                                                                     mutant. simplicity, chose analyze cdsA mu- <br />                                                                                     tants (cdsA7 cdsA9) lptD mutant backgrounds <br />                                                                                     (lptDI54E lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60). cdsA7 cdsA9 able                                                                                     cross-suppress lptDI54E lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60, regardless lptD <br />                                                                                     background isolated (Fig. 2). demonstrates                                                                                     cdsA suppressors allele specific increase <br />                                                                                     vancomycin resistance generalized manner. <br />                                                                                         Mutants beta-barrel assembly machinery (Bam) increase <br /> FIG 1 Vancomycin-resistant lptD suppressors map cdsA, gene respon- <br /> sible conversion phosphatidic acid CDP-diacylglycerol.          sensitivity vancomycin impaired assembly OM pro- <br /> inner membrane CdsA protein predicted seven transmembrane                teins (37, 38). &#226;&#338;&#172;bamE::kan allele null mutation  <br /> domains (33). Vancomycin-resistant cdsA suppressors mapped mainly            moves BamE, lipoproteins Bam complex. Strains domains. Mutations V16G, L33P, &#226;&#338;&#172;56 &#226;&#8364;&#8220; 62, Y156D, L236Q                carrying &#226;&#338;&#172;bamE::kan allele exhibit mild OM protein assembly <br /> isolated lptDV52E-I54E background. I203S isolated lptDI54E  <br />                                                                                     defects display increased sensitivity vancomycin (37). <br /> ground, R263L isolated lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60 background. muta- <br /> tions enclosed circles, I203S (cdsA7) R263L (cdsA9), characterized       cdsA7 cdsA9 introduced &#226;&#338;&#172;bamE::kan strain, study. (B) Biosynthetic pathway major phospholipids          vancomycin resistance increased (Fig. 3A). suggests Escherichia coli. Protein names boxes, arrow   cdsA7 cdsA9 increase vancomycin resistance general man- <br /> signifies step act.                                               ner strain compromised OM. <br />                                                                                         test cdsA7 cdsA9 increase resistance vanco- <br />                                                                                     mycin wild-type strains, evaluated mutants oth- <br /> mycin differing degrees. Spontaneously arising vancomycin-                       erwise wild-type background. LB plates supplemented resistant suppressors isolated LB medium supplemented                       200 mg/liter vancomycin, cdsA7 cdsA9 mutants able 120 mg/liter vancomycin.                                                       survive better wild type (Fig. 3B). suggests     strains carrying mutations, survivors arose                  cdsA7 cdsA9 strains altered way protects cell frequency 10&#226;&#171;&#186;8, suppressors analyzed mapped                   vancomycin. Previously isolated cdsA mutants showed  <br /> essential gene, cdsA. CdsA inner membrane protein respon-                     creased resistance erythromycin strain background <br /> sible conversion PA CDP-diacylglycerol GPL bio-                    tested, effect mutations sensitivity synthesis (6, 15, 32). Figure 1A shows location different                antibiotics reported (16). Using disk diffusion assays, <br /> cdsA mutations predicted CdsA structure (33). muta-                       checked cdsA mutants battery tions map transmembrane helices, affect                         antibiotics. slight increase resistance eryth- <br /> protein stability membrane. <span id='am-75' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-76' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>Amino acid</span> changes I203S                         romycin drug tested (norfloxacin, para- <br /> (cdsA7) R263L (cdsA9) chosen representative exam-                       quat, cetyltrimethylammonium bromide, bacitracin, poly- <br /> ples study located different parts              myxin B). Strikingly,  cdsA mutations <br /> protein, unlike cdsA suppressors isolated,                    cause clear increase sensitivity ampicillin, moxalactam, genta- exhibit growth defects.                                             micin, rifampin. <br />  <br />  <br /> 3216    jb.asm.org                                                                                                                     Journal Bacteriology <br />                                                                                                                 Phosphatidic Acid Increases Vancomycin Resistance <br />  <br />  <br />  <br />  <br /> FIG 2 Partial loss function cdsA mutants pgsA444 suppress vancomycin sensitivity lptD mutants.   Efficiency plating assay (EOP) LB agar <br /> plates supplemented 70 mg/liter, 140 mg/liter, 150 mg/liter vancomycin grown 37&#194;&#176;C overnight. Suppression shown lptDI54E. Note chromosomal lptD allele &#226;&#338;&#172;lptD, lptD allele listed EOP present pET2342::lptD, leaky T7 promoter. T7 RNA polymerase present strains. (B) EOPs described panel  Suppression shown lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60. <br />  <br />  <br />  <br />    Changes <span id='am-19' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span>phospholipid</span> content confer vancomycin resis-                      shown accumulate PA (15, 16). expect tance. enzymes intermediates involved GPL biosyn-                     concomitant decrease CDP-diacylglycerol levels cdsA mu- <br /> thesis identified lab Eugene Kennedy (Fig. 1B).                   tants, result decrease acidic GPL biosynthesis, <br /> Strains harboring partial loss function mutations cdsA                CDP-diacylglycerol higher affinity PssA                                                                                  PgsA (15, 16). <br />                                                                                      hypothesized change phospholipid content                                                                                  OM result increased resistance vancomycin, <br />                                                                                  wondered accumulation PA                                                                                  decrease acidic GPLs. distinguish                                                                                  possibilities, introduced pgsA444, partially inactivating mu- <br />                                                                                  tation phosphatidylglycerophosphate synthetase (39),                                                                                  &#226;&#338;&#172;clsA::kan Keio collection (34) lptD <br />                                                                                  mutant strains (note unable construct                                                                                  lptD&#226;&#338;&#172;D57&#226;&#338;&#172;K60 &#226;&#338;&#172;clsA::kan double mutant, data                                                                                  shown). ClsA responsible significant portion CL pro- <br />                                                                                  duced cell, cls mutants shown lower <br />                                                                                  levels CL (24, 40). pgsA444 partially suppresses <br />                                                                                  vancomycin sensitivity lptD mutants, &#226;&#338;&#172;clsA::kan <br />                                                                                  does (Fig. 2). suggests increase negatively <br />                                                                                  charged PA species membrane decrease acidic <br />                                                                                  GPLs responsible increase vancomycin resistance.                                                                                  likely pgsA444 partially suppresses vancomycin <br />                                                                                  sensitivity lptD mutants does accumulate                                                                                  PA cdsA mutants. discovered pgsA444 <br />                                                                                  confers increased vancomycin resistance wild- <br />                                                                                  type background (Fig. 3B). &#226;&#338;&#172;clsA::kan allele does confer <br />                                                                                  increased resistance fact renders cells sensitive                                                                                  vancomycin. <br />                                                                                      cdsA mutants accumulate PA. confirm genetic results, <br /> FIG 3 cdsA mutants, pgsA444, plsC overexpression increase resistance      investigated GPL content strains harboring cdsA7, cdsA9, <br /> vancomycin.   Partial loss function cdsA mutants suppress vancomy-      pgsA444 mutations. GPLs extracted cells <br /> cin sensitivity cell envelope mutants. EOPs strains supplemented     using acidic Bligh Dyer method (23, 24). Lipids sep- 120 mg/liter vancomycin. (B) Partial loss function cdsA mutants      arated quantified Agilent quadrupole time flight (Q- <br /> pgsA444 increase vancomycin resistance wild-type cells. EOPs strains <br /> supplemented 200 mg/liter vancomycin. (C) Overproduction PlsC <br />                                                                                  TOF) liquid chromatography-mass spectrometry (LC-MS). Dis- <br /> increases vancomycin resistance wild-type cells. EOPs strains supple-      tinct PA species summed, sum normalized mented 200 mg/liter vancomycin 0.1 mM IPTG.                             total PS content strain. PS levels did change signifi- <br />  <br />  <br /> September 2014 Volume 196 Number 18                                                                                                            jb.asm.org 3217 <br />  Sutterlin et al. <br />  <br />  <br />  <br />  <br /> FIG 4 PA accumulates cdsA mutants pgsA444 plsC overexpressed. PA quantified Q-TOF LC-MS. average fold differences PA species &#226;&#171;&#190; standard errors experiments shown  . changes PA/PS reported &#226;&#171;&#190; standard errors experiments  . <br /> (B) CL levels decrease cdsA mutants pgsA444. CL quantified Q-TOF LC-MS. average fold differences CL species &#226;&#171;&#190; standard errors experiments shown  . changes CL/PS reported &#226;&#171;&#190; standard errors experiments  . <br />  <br />  <br />  <br />  <br /> cantly strains tested. PA accumulates cdsA7,           DISCUSSION <br /> cdsA9, pgsA444 strains (Fig. 4A). hypothesize                  OM permeability defects associated lptD <br /> cdsA9 mutation compromises CdsA function severely                    mutants suppressed partial loss function mutations does cdsA7 PA levels higher cdsA9 mutant.              essential gene, cdsA. results selection increased correlates suppression seen lptD mutants                vancomycin resistance striking: suppression rare (Fig. 2); higher vancomycin concentrations, cdsA9 suppresses                seven suppressors analyzed altered seven different codons single <br /> vancomycin sensitivity better does cdsA7, suggesting                 essential gene. cdsA mutations increase vancomycin resistance <br /> greater PA accumulation results better suppression.                     lptD mutants, bam mutants, wild-type <br /> pgsA444 mutant accumulates PA cdsA7                                                                                strain.  markedly increase resistance cdsA9 mutant does, correlates fact pgsA444 <br />                                                                                antibiotic tested.  suppressors gene path- <br /> does suppress vancomycin sensitivity (Fig. 2).                                                                                way specific;  drug specific. Using classic genetic meth- <br /> cdsA7, cdsA9, pgsA444 mutants decreased CL levels <br />                                                                                ods Q-TOF LC-MS, increasing PA different <br /> compared wild type (Fig. 4B), likely PA accumulating expense CL.                                             ways (using cdsA suppressors, pgsA444 mutation, overex- <br />    Overexpression plsC increases PA increases vancomy-                  pression plsC) increases vancomycin resistance E. coli.  <br /> cin resistance. shown loss function mutations               clude vancomycin resistance increases consequence PA phospholipid biosynthesis pathway cause accumulation                  accumulation. PA increase vancomycin resistance. wondered                          difficult speculate accumulation PA results overproduction protein directly upstream PA synthesis,               increased vancomycin resistance primarily PlsC, cause accumulation PA,                    know vancomycin enters periplasm bind target, <br /> increase vancomycin resistance. used plsC ASKA clone                 lipid II (13, 14). lpt compromised, LPS trans- <br /> overexpress plsC wild-type background (35).                    port assembly slow, allowing PLs flip outer plsC expression induced 0.1 mM IPTG, increase                  leaflet OM (20). PLs self-associate form rafts PL vancomycin resistance observed (Fig. 3C). Q-TOF LC-MS                    bilayer OM allow diffusion hydrophobic mole- <br /> confirms level induction plsC expression results            cules periplasm (41). obvious large, hydro- accumulation PA (Fig. 4A). results confirm accu-              philic molecule like vancomycin able cross mem- <br /> mulation PA increases vancomycin resistance E. coli.                     brane way. Instead, possible vancomycin enters  <br />  <br /> 3218    jb.asm.org                                                                                                                   Journal Bacteriology <br />                                                                                                         Phosphatidic Acid Increases Vancomycin Resistance <br />  <br />  <br />  <br /> cell &#226;&#8364;&#339;cracks&#226;&#8364;? bilayer disordered LPS-PL junctions       vated (9). Accumulation PA way OM <br /> (41). predicts lpt mutant vancomycin           modified increase resistance antibiotics. <br /> sensitive increase LPS-PL junctions, e., &#226;&#8364;&#339;cracks&#226;&#8364;?          emergence multidrug-resistant bacteria  life- membrane.                                                             threatening problem (47). Owing barrier properties     Ganong et al. shown cdsA mutants accumulate PA           OM, problem especially acute Gram-negative patho- IM OM (15). true roughly       gens. antibiotics work infections caused times PA IM OM, PA concentrations           Gram-positive bacteria fail used Gram-negative bac- OM 10-fold higher cdsA mutant          teria effectiveness OM barrier, wild type (15). vancomycin needs traverse          know modify agents reach OM act target periplasm, unlikely      targets. better understanding permeability properties PA accumulation IM contributing vancomycin resis-            OM modified help facilitate tance.  suggest PA accumulation OM im-              design new antibacterials. <br /> pedes efficient entry vancomycin periplasm reduc- <br /> ing size altering properties &#226;&#8364;&#339;cracks.&#226;&#8364;? PA         ACKNOWLEDGMENTS <br /> accumulation increases vancomycin resistance             material based work supported National Science <br /> wise wild-type cell  propose presence small      Foundation Graduate Research Fellowship Program grant  PLs outer leaflet produce &#226;&#8364;&#339;cracks&#226;&#8364;? OM        DGE1148900 (H S.). work supported National <br />                                                                           Institute General Medical Sciences grant GM34821 (T.J.S.) NIH/ wild-type strains. <br />                                                                           NIGMS grant P50 GM071508 (S ). <br />     possible accumulation PA OM reduces              opinions, findings, conclusions recommendations ex- size &#226;&#8364;&#339;cracks&#226;&#8364;? increasing LPS assembly, increasing        pressed material author(s) necessarily <br /> activity Lpt  decreasing PLs         reflect views National Science Foundation. outer leaflet increasing activity PldA Mla trans-        thank Silhavy lab, present past, helpful discussion port (20).  PA really intermediate PL       comments. thank Mark Rose Zemer Gitai helpful discus- <br /> biosynthesis, present normally low levels,          sion critical analysis. grateful Thomas Bartlett help think unlikely.                                              imaging cell size analysis Jaclyn Schwalm strains. <br />     propose accumulation PA increases vancomycin <br />                                                                           REFERENCES <br /> resistance altering biophysical properties OM. PA <br />                                                                            1. Silhavy TJ, Kahne D, Walker S. 2010. bacterial cell envelope. Cold <br /> lacks head group increases membrane curvature, <br />                                                                               Spring Harb. Perspect. Biol. 2:a000414. http://dx.doi.org/10.1101 affect size properties &#226;&#8364;&#339;cracks&#226;&#8364;? (42, 43).        /cshperspect.a000414. addition increased membrane curvature, PA increases              2. Nikaido H. 2003. Molecular basis bacterial outer membrane permea- <br /> negative charge OM, increase vancomycin                   bility revisited. Microbiol. Mol. Biol. Rev. 67:593&#226;&#8364;&#8220; 656. http://dx.doi.org <br /> association OM (43). counterintui-               /10.1128/MMBR.67.4.593-656.2003. <br />                                                                            3. Bos MP, Robert V, Tommassen J. 2007. Biogenesis gram-negative <br /> tive, studies shown CL microdomains interact                  bacterial outer membrane. Annu. Rev. Microbiol. 61:191&#226;&#8364;&#8220;214. http://dx CAMPs prevent interacting targets,              .doi.org/10.1146/annurev.micro.61.080706.093245. increasing resistance (44, 45).                                    4. Raetz CR, Whitfield C. 2002. Lipopolysaccharide endotoxins. Annu. Rev. <br />     Changes GPL composition previously shown                  Biochem. 71:635&#226;&#8364;&#8220;700. http://dx.doi.org/10.1146/annurev.biochem.71 <br />                                                                               .110601.135414. <br /> alter resistance susceptibility CAMPs antibiotics (9,           5. Ruiz N, Kahne D, Silhavy TJ. 2009. Transport lipopolysaccharide <br /> 46). Staphylococcus aureus, fmtC lysC mutations,                 cell envelope: long road discovery. Nat. Rev. Microbiol. <br /> reduce lysyl-PG, increase resistance moenomycin vanco-                 7:677&#226;&#8364;&#8220; 683. http://dx.doi.org/10.1038/nrmicro2184. <br /> mycin (46). increased net negative charge mutant              6. Dowhan W. 2013. retrospective: use Escherichia coli vehicle                                                                               study phospholipid synthesis function. Biochim. Biophys. Acta 1831: <br /> membranes increase interaction drugs prevent               471&#226;&#8364;&#8220; 494. http://dx.doi.org/10.1016/j.bbalip.2012.08.007. interacting targets. Salmonella Typhimu-           7. Zhang YM, Rock  2008. Membrane lipid homeostasis bacteria. <br /> rium, PhoPQ responds acidic pH CAMPs                     Nat. Rev. Microbiol. 6:222&#226;&#8364;&#8220;233. http://dx.doi.org/10.1038/nrmicro1839. <br /> increasing levels cardiolipin palmitoylated acyl-PG (9).            8. Wiesner J, Vilcinskas  2010. Antimicrobial peptides: ancient arm                                                                               human immune  Virulence 1:440 &#226;&#8364;&#8220; 464. http://dx.doi.org/10 increase GPLs correlates increase resis-             .4161/viru.1.5.12983. <br /> tance CAMPs helps evade host immune (9).                9. Dalebroux ZD, Matamouros S, Whittington D, Bishop  Miller SI. speculation resistance increased             2014. PhoPQ regulates acidic <span id='am-15' about='obo:IMR_0001384' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001384" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="glycerophospholipid" datatype="xsd:string"></span>glycerophospholipid</span> content Salmo- <br /> membrane curvature <span id='am-115' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-116' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="cAMP" datatype="xsd:string"></span>CAMP</span> association CL mi-                    nella Typhimurium outer membrane. Proc. Natl. Acad. Sci. U. S.  111: <br />                                                                               1963&#226;&#8364;&#8220;1968. http://dx.doi.org/10.1073/pnas.1316901111. <br /> crodomains, occluding CAMP interactions tar-           10. Needham BD, Trent MS. 2013. Fortifying barrier: impact lipid <br /> gets (9, 44, 45).                                                             remodelling bacterial pathogenesis. Nat. Rev. Microbiol. 11:467&#226;&#8364;&#8220; <br />     mode resistance shown inherently different            481. http://dx.doi.org/10.1038/nrmicro3047. regulated PhoPQ  turned         11. Murata T, Tseng W, Guina T, Miller SI, Nikaido H. 2007. PhoPQ- <br />                                                                               mediated regulation produces robust permeability barrier palmitoylation PG increases, results reduction polar-        outer membrane Salmonella enterica serovar Typhimurium. J. Bacte- <br /> ity increase hydrophobicity saturation mem-          riol. 189:7213&#226;&#8364;&#8220;7222. http://dx.doi.org/10.1128/JB.00973-07. <br /> brane (9, 10). work, overall increase PA, regard-   12. Cronan JE. 2003. Bacterial membrane lipids: stand? Annu. chain length saturation state  Fig. S3              Rev. Microbiol. 57:203&#226;&#8364;&#8220;224. http://dx.doi.org/10.1146/annurev.micro.57 <br />                                                                               .030502.090851. <br /> supplemental material). result accumulation PA,    13. Pootoolal J, Neu J, Wright GD. 2002. Glycopeptide antibiotic resistance. reduction CL levels. contrasts increase       Annu. Rev. Pharmacol. Toxicol. 42:381&#226;&#8364;&#8220; 408. http://dx.doi.org/10.1146 <br /> CL levels observed Salmonella PhoPQ acti-               /annurev.pharmtox.42.091601.142813. <br />  <br />  <br />  <br /> September 2014 Volume 196 Number 18                                                                                                     jb.asm.org 3219 <br />  Sutterlin et al. <br />  <br />  <br />  <br /> 14. Barna JC, Williams DH. 1984. structure mode action                    coli K-12 frame, single-gene knockout mutants: Keio collection. <br />     glycopeptide antibiotics vancomycin group. Annu. Rev. Microbiol.             Mol. Syst. Biol. 2:2006.0008. http://dx.doi.org/10.1038/msb4100050. <br />     38:339 &#226;&#8364;&#8220;357. http://dx.doi.org/10.1146/annurev.mi.38.100184.002011.           35.   Kitagawa M, Ara T, Arifuzzaman M, Ioka-Nakamichi T, Inamoto E, <br /> 15. Ganong BR, Leonard JM, Raetz CR. 1980. Phosphatidic acid accumula-                  Toyonaga H, Mori H. 2005. Complete set ORF clones Escherichia <br />     tion membranes Escherichia coli mutants defective CDP-                 coli ASKA library  complete set E. coli K-12 ORF archive): unique <br />     <span id='am-93' about='obo:IMR_0100027' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-94' property="rdfs:label" content="diacylglycerol" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="diacylglyceride" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="KEGG:C00165" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="DAG" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasExactSynonym" content="DAG" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="diacylglyceride" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Diglyceride" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="Diglyceride" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:id" content="IMR:0100027" datatype="xsd:string"></span>diglyceride</span> synthetase. J. Biol. Chem. 255:1623&#226;&#8364;&#8220;1629.                               resources biological research. DNA Res. 12:291&#226;&#8364;&#8220;299. http://dx.doi.org <br /> 16. Ganong BR, Raetz CR. 1982. Massive accumulation phosphatidic acid                /10.1093/dnares/dsi012. <br />     conditionally lethal CDP-diglyceride synthetase mutants <span id='am-70' about='obo:IMR_0200161' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-71' property="oboInOwl:id" content="IMR:0200161" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00475" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Cytidine" datatype="xsd:string"></span>cytidine</span>       36.   Yao  Davis RM, Kishony R, Kahne D, Ruiz N. 2012. Regulation cell <br />     auxotrophs Escherichia coli. J. Biol. Chem. 257:389 &#226;&#8364;&#8220;394.                        size response nutrient availability fatty acid biosynthesis Esch- <br /> 17. Casadaban MJ. 1976. Transposition fusion lac genes selected           erichia coli. Proc. Natl. Acad. Sci. U. S.  109:E2561&#226;&#8364;&#8220;E2568. http://dx.doi <br />     promoters Escherichia coli using bacteriophage lambda Mu. J. Mol.            .org/10.1073/pnas.1209742109. <br />     Biol. 104:541&#226;&#8364;&#8220;555. http://dx.doi.org/10.1016/0022-2836(76)90119-4.            37.   Rigel NW, Schwalm J, Ricci DP, Silhavy TJ. 2012. BamE modulates 18. Ruiz N, Gronenberg LS, Kahne D, Silhavy TJ. 2008. Identification             Escherichia coli beta-barrel assembly machine component BamA. J. Bacte- <br />     inner-membrane proteins required transport lipopolysaccha-               riol. 194:1002&#226;&#8364;&#8220;1008. http://dx.doi.org/10.1128/JB.06426-11. <br />     ride outer membrane Escherichia coli. Proc. Natl. Acad. Sci.        38.   Ricci DP, Hagan CL, Kahne D, Silhavy TJ. 2012. Activation     U. S.  105:5537&#226;&#8364;&#8220;5542. http://dx.doi.org/10.1073/pnas.0801196105.                  Escherichia coli beta-barrel assembly machine (Bam) required essen- <br /> 19. Silhavy TJ, Berman ML, Enquist LW. 1984. Experiments gene                      tial components interact properly substrate. Proc. Natl. Acad. Sci. <br />     fusions. Cold Spring Harbor Laboratory Press, Plainview, NY.                        U. S.  109:3487&#226;&#8364;&#8220;3491. http://dx.doi.org/10.1073/pnas.1201362109. <br /> 20. Malinverni JC, Silhavy TJ. 2009. ABC transport maintains       39.   Nishijima M, Raetz CR. 1979. Membrane lipid biogenesis Escherichia <br />     lipid asymmetry gram-negative outer membrane. Proc. Natl. Acad.              coli: identification genetic loci phosphatidylglycerophosphate syn- <br />     Sci. U. S.  106:8009 &#226;&#8364;&#8220; 8014. http://dx.doi.org/10.1073/pnas.0903229106.           thetase construction mutants lacking phosphatidylglycerol. J. Biol. <br /> 21. Cherepanov PP, Wackernagel W. 1995. Gene disruption Escherichia                  Chem. 254:7837&#226;&#8364;&#8220;7844. <br />     coli: TcR KmR cassettes option Flp-catalyzed excision      40.   Shibuya  Miyazaki C, Ohta  1985. Alteration phospholipid com- <br />     antibiotic-resistance determinant. Gene 158:9 &#226;&#8364;&#8220;14. http://dx.doi.org            position combined defects <span id='am-24' about='obo:IMR_0001358' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C02737" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="phosphatidylserine" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="O-Phosphatidyl-L-serine" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="IMR:0200250" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:18303" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0001358" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="O-Phosphatidyl-L-serine" datatype="xsd:string"></span>phosphatidylserine</span> cardiolipin syn- <br />     /10.1016/0378-1119(95)00193                                                       thases physiological consequences Escherichia coli. J. Bacteriol. 161: <br /> 22. Sliusarenko O, Heinritz J, Emonet T, Jacobs-Wagner C. 2011. High-                   1086 &#226;&#8364;&#8220;1092. <br />     throughput, subpixel precision analysis bacterial morphogenesis        41.   Nikaido H. 2005. Restoring permeability barrier function outer mem- <br />     intracellular spatio-temporal dynamics. Mol. Microbiol. 80:612&#226;&#8364;&#8220; 627.                brane. Chem. Biol. 12:507&#226;&#8364;&#8220;509. http://dx.doi.org/10.1016/j.chembiol <br />     http://dx.doi.org/10.1111/j.1365-2958.2011.07579.x.                                 .2005.05.001. <br /> 23. Bligh  Dyer WJ. 1959. rapid method total lipid extraction         42.   McMahon HT, Gallop JL. 2005. Membrane curvature mechanisms <br />                                                                                         dynamic cell membrane remodelling. Nature 438:590 &#226;&#8364;&#8220;596. http://dx <br />     purification.  J. Physiol. Pharmacol. 37:911&#226;&#8364;&#8220;917. http://dx.doi.org/10 <br />                                                                                         .doi.org/10.1038/nature04396. <br />     .1139/y59-099. <br />                                                                                   43.   Kooijman EE, Burger KN. 2009. Biophysics function phosphatidic <br /> 24. Tan BK, Bogdanov M, Zhao J, Dowhan W, Raetz CR, Guan  2012. <br />                                                                                         acid: molecular perspective. Biochim. Biophys. Acta 1791:881&#226;&#8364;&#8220; 888. http: <br />     Discovery cardiolipin synthase utilizing phosphatidylethanolamine <br />                                                                                         //dx.doi.org/10.1016/j.bbalip.2009.04.001. <br />     phosphatidylglycerol substrates. Proc. Natl. Acad. Sci. U. S.  <br />                                                                                   44.   Tran TT, Panesso D, Mishra NN, Mileykovskaya E, Guan  Munita <br />     109:16504 &#226;&#8364;&#8220;16509. http://dx.doi.org/10.1073/pnas.1212797109. <br />                                                                                         JM, Reyes J, Diaz L, Weinstock GM, Murray  Shamoo Y, Dowhan <br /> 25. Grabowicz M, Yeh J, Silhavy TJ. 2013. Dominant negative lptE mutation <br />                                                                                         W, Bayer  Arias CA. 2013. Daptomycin-resistant Enterococcus faecalis <br />     supports role LptE plug LptD barrel. J. Bacteriol.           diverts antibiotic molecule division septum remodels cell <br />     195:1327&#226;&#8364;&#8220;1334. http://dx.doi.org/10.1128/JB.02142-12.                               membrane phospholipids. mBio 4(4):e00281&#226;&#8364;&#8220;13. http://dx.doi.org/10 <br /> 26. Freinkman E, Chng SS, Kahne D. 2011. complex inserts lipo-                 .1128/mBio.00281-13. <br />     <span id='am-11' about='obo:IMR_0001790' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="polysaccharide" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001790" datatype="xsd:string"></span>polysaccharide</span> bacterial outer membrane forms protein          45.   Hartmann M, Berditsch M, Hawecker J, Ardakani MF, Gerthsen D, <br />     plug barrel. Proc. Natl. Acad. Sci. U. S.  108:2486 &#226;&#8364;&#8220;2491. http://dx          Ulrich  2010. Damage bacterial cell envelope antimicrobial <br />     .doi.org/10.1073/pnas.1015617108.                                                   peptides gramicidin S PGLa revealed transmission scanning <br /> 27. Freinkman E, Okuda S, Ruiz N, Kahne D. 2012. Regulated assembly                  electron microscopy. Antimicrob. Agents Chemother. 54:3132&#226;&#8364;&#8220;3142. <br />     transenvelope protein complex required lipopolysaccharide ex-               http://dx.doi.org/10.1128/AAC.00124-10. <br />     port. Biochemistry 51:4800 &#226;&#8364;&#8220; 4806. http://dx.doi.org/10.1021/bi300592c.       46.   Nishi H, Komatsuzawa H, Fujiwara T, McCallum N, Sugai M. 2004. <br /> 28. Villa R, Martorana  Okuda S, Gourlay LJ, Nardini M, Sperandeo P,                 Reduced content lysyl-phosphatidylglycerol cytoplasmic mem- <br />     Deho G, Bolognesi M, Kahne D, Polissi  2013. Escherichia coli Lpt             brane affects susceptibility moenomycin, vancomycin, genta- <br />     transenvelope protein complex lipopolysaccharide export assembled            micin, antimicrobial peptides, Staphylococcus aureus. Antimicrob. <br />     conserved structurally homologous domains. J. Bacteriol. 195:1100 &#226;&#8364;&#8220;             Agents Chemother. 48:4800 &#226;&#8364;&#8220; 4807. http://dx.doi.org/10.1128/AAC.48.12 <br />     1108. http://dx.doi.org/10.1128/JB.02057-12.                                        .4800-4807.2004. <br /> 29. Sampson BA, Misra R, Benson SA. 1989. Identification character-           47.   Dortet L, Poirel L, Nordmann P. 2014. Worldwide dissemination     ization new gene Escherichia coli K-12 involved outer membrane           NDM-type carbapenemases gram-negative bacteria. Biomed. Res. Int. <br />     permeability. Genetics 122:491&#226;&#8364;&#8220;501.                                                 2014:249856. http://dx.doi.org/10.1155/2014/249856. <br /> 30. Sperandeo P, Pozzi C, Deho G, Polissi  2006. Non-essential KDO              48.   Wu T, McCandlish AC, Gronenberg LS, Chng SS, Silhavy TJ, Kahne D. <br />      <br /> </body></html>