<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                          RESEARCH ARTICLE <br />  <br />  <br />                                                           Probing Metabolic Network                                                           Bloodstream-Form Trypanosoma brucei <br />                                                           Using Untargeted Metabolomics Stable <br />                                                           Isotope Labelled Glucose <br />   <br /> Trypanosoma brucei Using Untargeted Metabolomics <br />                                                            Stable Isotope Labelled Glucose. PLoS Pathog <br />   <br />                                                           Abstract <br /> Copyright: &#194;&#169; 2015 Creek et al. open <br /> access article distributed terms         Metabolomics coupled heavy-atom isotope-labelled glucose used probe <br /> Creative Commons Attribution License, permits       metabolic pathways active cultured bloodstream form trypomastigotes Trypano- <br /> unrestricted use, distribution, reproduction medium, provided original author source                                                           soma brucei, parasite responsible human African trypanosomiasis. Glucose enters <br /> credited.                                                 branches metabolism glycolysis, widely held sole <br /> Data Availability Statement: Relevant mass <br />                                                           route glucose metabolism. Whilst pyruvate major end-product glucose catabo- <br /> spectrometry data available Supporting          lism, transamination product, alanine, produced significant quantities. oxi- <br /> Information files. Raw data files             dative branch pentose phosphate pathway operative, non-oxidative <br /> available contacting corresponding author                                                           branch  Ribose 5-phosphate generated pathway distributes widely michael.barrett@glasgow.ac.uk. processed <br /> files, available Supporting              <span id='am-133' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-134' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-135' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-140' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> synthesis branches metabolism. Acetate, derived glucose, Information, used producing paper.           associated range acetylated amino acids  lesser extent, fatty acids; <br /> Funding: work supported Australian         labelled glycerol glycerophospholipids. Glucose enters inositol <br /> NHMRC fellowship awarded DJC, SysMO                sugar nucleotides serve precursors macromolecule biosynthesis. Al- <br /> SilicoTryp Project (MPB, FA, DHK, RB), ParaMet        Krebs cycle operative, malate, fumarate succinate, primarily labelled PhD programme Marie Curie Initial Training <br />                                                           carbons, present, indicating origin phosphoenolpyruvate Network (MPB FB). EK SULSA PhD <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                              1 / 25 <br />                                                                                                                          Trypanosome Glucose Metabolism <br />  <br />  <br />  <br /> student. JA, DGW, FB RK supported      oxaloacetate. Interestingly, enzyme responsible conversion phosphoenolpyruvate <br /> BBSRC/ANR Systryp Project. MPB          oxaloacetate, phosphoenolpyruvate carboxykinase, shown essential Wellcome Trust Centre Molecular Parasitology <br /> (supported grant 085349). FB&#226;&#8364;&#8482;s group                                                        bloodstream form trypanosomes, demonstrated lethal phenotype induced supported Centre National la Recherche       RNAi-mediated downregulation expression. addition, glucose derivatives enter py- <br /> Scientifique (CNRS), Universit&#195;&#169; Bordeaux,   rimidine biosynthesis oxaloacetate precursor aspartate orotate. <br /> Laboratoire d&#226;&#8364;&#8482;Excellence (LabEx) ParaFrap ANR-11- <br /> LABX-0024, Agence Nationale la <br /> Recherche (ANR) grant ACETOTRYP ANR-BLANC-2010. funders role study <br /> design, data collection analysis, decision        Author Summary <br /> publish, preparation manuscript. <br />                                                          work followed distribution carbon derived glucose blood- <br /> Competing Interests: authors declared <br />                                                          stream form trypanosomes, causative agent African trypanosomiasis, revealing competing interests exist <br />                                                          enter diverse range metabolites. work involved using 13C-labelled glucose fol- <br />                                                          lowing fate labelled carbon LC-MS based metabolomics platform.  <br />                                                          yond glycolysis oxidative branch pentose phosphate pathway <span id='am-235' about='rdfs:label' typeof='owl:Thing'>label</span> <br />                                                          entered <span id='am-30' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-31' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> biosynthesis glycerol 3-phosphate acetate. Glucose  <br />                                                          rived carbon entered nucleotide synthesis ribose pyrimidine synthesis <br />                                                          oxaloacetate-derived aspartate. Appreciable quantities carboxylic acids suc- <br />                                                          cinate malate identified, labelling patterns indicate TCA <br />                                                          cycle derived. Amino sugars sugar nucleotides labelled inositol used <br />                                                          protein modification inositol <span id='am-13' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span>phospholipid</span> headgroup production.  <br />                                                          firm active essential oxaloacetate production bloodstream form trypanosomes                                                          phosphoenolpyruvate carboxykinase essential parasites using RNA <br />                                                          interference. glucose entering metabolites minor compared                                                          quantity enters pyruvate excreted cell, observation enzymes  <br />                                                          tributing metabolism glucose glycolysis essential offers potential <br />                                                          new targets chemotherapy trypanosomiasis. <br />  <br />  <br />  <br />                                                        Introduction <br />                                                        Trypanosoma brucei protozoan parasite, sub-species responsible human <br />                                                        African trypanosomiasis (HAT) animal African trypanosomiasis (Nagana) [1]. Current <br />                                                        treatments HAT inadequate, new preventative therapeutic options urgently <br />                                                        required [2]. parasite transmitted mammalian hosts tsetse fly vector.                                                        physiological environments mammalian blood tsetse midgut diverge significantly, <br />                                                        bloodstream form insect form parasites reveal adaptive differences biochemistry                                                        allow optimised survival environments [3,4]. <br />                                                            T. brucei possess glycosomes, peroxisome-derived organelles contain enzymes required <br />                                                        glycolysis [5]. tsetse fly, glucose generally scarce proline key energy <br />                                                        source [6]. Procyclic form (PCF) trypanosomes preferentially use proline source carbon <br />                                                        energy [6,7,8], glucose-rich culture medium preferentially utilize glucose <br />                                                        glycolysis [7,8]. years widely accepted bloodstream form <br />                                                        (BSF) trypanosomes exhibit greatly reduced metabolic potential, glycosomal glucose uti- <br />                                                        lization glycolysis sole energy source [4,5,9,10]. aerobic conditions pyru- <br />                                                        vate considered sole end-product, redox balance maintained                                                        mitochondrial alternative oxidase shunt acting regenerate dihydroxyacetone phosphate <br />                                                        (DHAP) glycerol 3-phosphate [11]. anaerobiosis, equal quantities pyruvate                                                        glycerol produced, reverse reaction glycerol kinase (GK) generates glycerol <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                  2 / 25 <br />                                                                                                               Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           glycerol 3-phosphate [9]. Early studies BSF trypanosomes isolated rodents iden- <br />                                           tified secreted metabolites, including succinate, aspartate alanine [12&#226;&#8364;&#8220;14].  <br />                                           fact products present small amounts stumpy form trypano- <br />                                           somes, non-replicative form parasite pre-adapted life tsetse fly [15,16],                                           contaminate slender BSF preparations, led observations considered limit- <br />                                           ed relevance formulating models simplified pathway glucose catabolism slen- <br />                                           der replicative form T. brucei [9]. <br />                                               Evidence diverse fate glucose BSF trypanosome metabolism   <br />                                           emerged. example, NMR analysis glucose metabolism [17,18] confirmed glycerol py- <br />                                           ruvate major end products glycolysis, detecting significant amounts ala- <br />                                            conversion pyruvate alanine essential BSF trypanosomes,                                           possible knockout alanine aminotransferase (AAT) gene [19], signifi- <br />                                           reduction transcript RNAi possible [19]. oxidative branch pentose <br />                                           phosphate pathway (PPP) operative [20], 6-phosphogluconate dehydrogenase essential <br />                                           forms [21,22]. Glucose-derived ribose shown incorporated cellular <br />                                           nucleotides BSF trypanosomes [23,24], inositol derived glucose incorporated <br />                                           glycosylphosphatidyl inositol anchors attached abundant variant surface glyco- <br />                                           protein coat cells [25,26]. Recently shown pyruvate dehydrogenase (PDH) <br />                                           essential slender bloodstream forms deprived threonine source acetate <br />                                           [27]; situation PDH needed generate acetate glucose metabolism. <br />                                               systematic application reverse genetics PCF trypanosomes combined NMR <br />                                           analyses end products metabolism demonstrated importance succinate fermenta- <br />                                           tion pathways glycosome mitochondrion, route independent                                           classical TCA cycle [28&#226;&#8364;&#8220;30]. glycosomal succinate shunt converts phosphoenolpyruvate (PEP) <br />                                           produced cytosol oxaloacetate using PEP carboxykinase (PEPCK) following glycosomal <br />                                           reuptake. PEP converted pyruvate, enters mitochondrion feed                                           PDH complex production acetyl-<span id='am-123' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-124' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span>, converted mitochondrial acetate:suc- <br />                                           cinate CoA-transferase (ASCT) <span id='am-246' about='obo:IMR_0200414' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-247' property="rdfs:label" content="Acetyl-CoA" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:id" content="IMR:0200414" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasDbXref" content="KEGG:C00024" datatype="xsd:string"></span>acetyl-CoA</span> thioesterase (ACH) produce acetate [30,31]. <br />                                               advent metabolomics technology [32&#226;&#8364;&#8220;34] allows unbiased analysis metabolism <br />                                           diverse pathways T. brucei. Stable-isotope labelling permits direct detection active <br />                                           metabolic pathways live cell [35&#226;&#8364;&#8220;39] offers direct route follow dis- <br />                                           tribution atoms precursor substrate metabolic network. previ- <br />                                           ously applied approach PCF trypanosomes [38], extend methodology                                           make comprehensive assessment distribution glucose-derived carbon                                           BSF trypanosome. <br />  <br />                                           Results <br />                                           Extensive glucose utilization anabolic processes bloodstream form <br />                                           T. brucei <br />                                           BSF T. brucei fed 50:50 mixture U-12C U-13C-glucose major carbon source <br />                                           hour 24 hours. Glycolytic intermediates rapidly labelled, expected  <br />                                           sult high flux glycolytic pathway [10], complete (50%) labelling                                           major end-product, pyruvate. Significant labelling observed metabolites                                           pathways, indicating extensive anabolic utilization glucose. total, 150 metabolites <br />                                           detected glucose-derived carbon labelling, including sugar phosphates, sugar nucleo- <br />                                           tides, lipids secondary metabolites derived glycolytic intermediates end-products <br />                                           (S1 Table). Whilst excreted pyruvate accounts majority catabolised glucose [10], glu- <br />                                           cose-derived pyruvate serves major source alanine acetyl-CoA cell. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                        3 / 25 <br />                                                                                                                          Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                                Furthermore, metabolism glucose oxidative branch PPP provides ribose <br />                                                5-phosphate, evidenced 50% U-13C-labelled, turn utilized nucleo- <br />                                                tide biosynthesis. significant labelling numerous carboxylic acids includ- <br />                                                ing malate, fumarate succinate, extensive labelling aspartate various <br />                                                pyrimidines.  results demonstrate extensive utilization glucose anabolic pur- <br />                                                poses bloodstream form T. brucei (Fig 1). <br />  <br />  <br />                                                Revisiting glycolysis bloodstream form T. brucei <br />                                                glycolytic pathway highly active T. brucei, explains 6-carbon labelling                                                glucose 6-phosphate (G6P), fructose 6-phosphate (F6P) fructose-1,6-bisphosphate (FBP), <br />                                                3-carbon labelling triose phosphate intermediates lower glycolysis (Fig 2). <br />                                                    Interestingly, additional 3-carbon labelled isotopologues hexose phosphates observed. <br />                                                isotopologues represent 10% intracellular G6P F6P, 40% FBP.                                                metabolites identified exact mass retention time using authentic standards,                                                absolutely rule metabolites mass retention time  e. unan- <br />                                                ticipated hexose phosphate isomers) eluting metabolites. findings incon- <br />                                                sistent unidirectional glucose flow Embden-Meyerhof-Parnas glycolytic <br />  <br />  <br />  <br />  <br /> Fig 1. Overview metabolites labelled <span id='am-193' about='obo:IMR_0200042' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-194' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-195' property="rdfs:label" content="D-Glucose" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:id" content="IMR:0200042" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasDbXref" content="KEGG:C00031" datatype="xsd:string"></span>D-glucose</span> bloodstream form T. brucei. Metabolic network derived detected metabolites annotated enzymes T. brucei. Red nodes indicate labelled metabolites 50% U-13C-glucose, darker red indicating higher percentage labelling. <br /> Grey nodes indicate labelling detected. Nodes spots indicate metabolite detected, presence suggested adjacent <br /> metabolites network. <br /> doi:10.1371/journal.ppat.1004689.g001 <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                     4 / 25 <br />                                                                                                                          Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br /> Fig 2. Labelling glycolytic intermediates 50% U-13C-glucose BSF T. brucei.   Schematic glycolytic pathway BSF T. brucei. Black <br /> font = detected LCMS, grey font = inferred intermediates detected assay. (B) Heatmap relative isotopologue abundances glycolytic <br /> intermediates labelling 24 hours 50% U-13C-glucose HMI11. 0 = carbons unlabelled (e.g. U-12C-glucose), 6 = carbons labelled (e.g. <br /> U-13C-glucose). (C) Isotopologue abundances hexose phosphates labelling 24 hours 50% U-13C-glucose (n = 3, mean &#194;&#177; SD). (D) <br /> Isotopologue abundances glucose 6-phosphate labelling 24 hours 100% U-12C-glucose (white columns), 50% U-13C-glucose (hatched <br /> columns) 100% U-13C-glucose (black columns) (n = 3, mean &#194;&#177; SD). Abbreviations: G6P, glucose 6-phosphate; F6P, fructose 6-phosphate; FBP, fructose- <br /> 1,6-bisphosphate; DHAP, dihydroxyacetone phosphate; GA3P, glyceraldehyde 3-phosphate; 1,3BPG, 1,3-bisphosphoglycerate; 3PG, 3-phosphoglycerate; <br /> 2PG, 2-phosphoglycerate; PEP, phosphoenolpyruvate; G3P, glycerol 3-phosphate. Enzymes: 1, hexokinase; 2, glucose-6-phosphate isomerase; 3, <br /> phosphofructokinase; 4, fructose-1,6-bisphosphatase; 5, aldolase; 6, triosephosphate isomerase; 7, glyceraldehyde 3-phosphate dehydrogenase; 8, <br /> phosphoglycerate kinase; 9, phosphoglycerate mutase; 10, enolase; 11, pyruvate kinase; 12, alanine aminotransferase; 13, glycerol-3-phosphate <br /> dehydrogenase; 14, glycerol kinase; 15, mitochondrial FAD-dependent glycerol-3-phosphate dehydrogenase; 16, methylglyoxal detoxification pathway. <br /> doi:10.1371/journal.ppat.1004689.g002 <br />  <br />  <br />                                                pathway. Labelling studies 100% U-13C-glucose produced little 3-labelled hexose phos- <br />                                                phate (~2% hexose phosphate 3-labelled ~97% glucose labelled <br />                                                inside cells), indicating carbons hexose phosphates originally derived                                                glucose (Fig 2d). Considering lack source absence reactions  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                      5 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           produce 3-labelled hexose-phosphates, 3-labelled hexose-phosphates likely  <br />                                           rive 3-labelled fructose bisphosphate (possibly reverse phosphofructokinase  <br />                                           action contribution fructose bisphosphatase ruled . <br />                                           Gluconeogenesis amino acids carbon sources identified, consistent pre- <br />                                           vious observations [20]. <br />                                               high percentage 3-carbon labelled FBP using 50:50 mix U-13C-glucose <br />                                           U-12C-glucose counter-intuitive, given strong net forward flux <span id='am-203' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-204' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-208' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-209' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-211' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span>gly</span>- <br />                                           colysis high concentration FBP.  computational analysis shows                                           impossible. aldolase reaction reversible  net aldolase flux = forward flux&#226;&#8364;&#8221; <br />                                           reverse flux). observed labelling require FBP pool created                                           &#226;&#8364;&#339;reverse&#226;&#8364;? aldolase reaction, working direction FBP production, represent 85% <br />                                            HMI medium) 97%  CMM medium) total FBP pool  S1 Text detailed ex- <br />                                           planations). large contributions reverse reaction possible certain condi- <br />                                           tions, spite net flux strongly favouring forward direction. Simulations using                                           previously published collection models described [40]                                           classical assumption glycosomes impermeable, reverse aldolase fluxes vary <br />                                           widely plausible kinetic parameter values, frequently 0 60%                                           FBP pool coming &#226;&#8364;&#339;reverse&#226;&#8364;? aldolase reaction  e. 0 29% potentially <br />                                           3-labelled).  glycosome semi-permeable metabolites smaller FBP itself <br />                                           (model 3 ref 40) non-selective pore-forming channels glycosomal membrane <br />                                           [41], high reverse aldolase flux, accounting 60&#226;&#8364;&#8220;99% FBP pool favoured (S1 Text). <br />                                           excellent agreement observed labelling pattern. <br />                                               addition pyruvate, transamination product alanine produced, probably                                           AAT, enzyme previously shown essential BSF trypanosomes [19]. low levels                                           labelled lactate detected, carbons labelled. lactate dehydrogenase <br />                                           identified T. brucei, L-lactate production shown occur <br />                                           unusual methylglyoxal detoxification pathway trypanosomes [42],                                           correspond metabolite corresponding mass. <br />                                               number potential pyruvate-derived conjugated metabolites observed,                                           significance unknown. example, abundant metabolite 3C 6C label- <br />                                           ling corresponding molecular formula C6H8O6, putatively identified parapyruvate <br />                                           [43]. Pyruvate appeared form adducts basic amino acids produce metabolites <br />                                           mass, labelling predicted retention times consistent carboxyethyl-<span id='am-188' about='obo:IMR_0200428' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-189' property="oboInOwl:id" content="IMR:0200428" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="KEGG:C00062" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-192' property="rdfs:label" content="L-Arginine" datatype="xsd:string"></span>L-arginine</span>                                           carboxyethyl-<span id='am-92' about='obo:IMR_0200342' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-93' property="oboInOwl:id" content="IMR:0200342" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="KEGG:C00077" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="L-Ornithine" datatype="xsd:string"></span>L-ornithine</span> (S1 Table). abundant novel 3-carbon labelled metabolite <br />                                           detected molecular formula C6H9NO4S, likely condensation product pyru- <br />                                           vate cysteine, C7H11NO5 correspond formation pyruvate-threonine <br />                                           adduct. metabolites abundant cells grown HMI11 compared CMM, <br />                                           consistent greater abundance <span id='am-82' about='obo:IMR_0200467' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-83' property="oboInOwl:id" content="IMR:0200467" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00097" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="L-Cysteine" datatype="xsd:string"></span>L-cysteine</span> <span id='am-107' about='obo:IMR_0200154' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-108' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="L-Threonine" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="KEGG:C00188" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:id" content="IMR:0200154" datatype="xsd:string"></span>L-threonine</span> richer medium.                                           absence authentic standards,  analysis required confirm                                           structure physiological relevance compounds. <br />  <br />  <br />                                           Pentose phosphate pathway <br />                                           pentose phosphate pathway intermediates (6-phosphogluconate ribose 5-phosphate) <br />                                           fully labelled, indicating linear oxidative PPP derived glucose 6-phosphate (Fig 3). <br />                                           complete (5-carbon) labelling ribose purine nucleotides confirms pathway                                           major source ribose 5-phosphate nucleotide synthesis (Fig 3e). contrasts                                           situation PCF T. brucei, possess active non-oxidative branch PPP <br />                                           [20] characteristic 2-, 3- 5-carbon ribose 5-phosphate labelling  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     6 / 25 <br />                                                                                                                       Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br />                                           Fig 3. Labelling pentose phosphate pathway purine nucleotides 50% U-13C-glucose                                           BSF T. brucei.   PPP BSF T. brucei. Black font = detected LCMS, grey font = inferred intermediates <br />                                           detected assay. (B) Heatmap relative isotopologue abundances pentose phosphate pathway <br />                                           intermediates labelling 24 hours 50% U-13C-glucose HMI11. (C) Representative mass spectra <br />                                           octulose 8-phosphate T. brucei incubated unlabelled (U-12C) glucose   labelled (U-13C) <br />                                           glucose  . +3, +5 +8 isotopologues predominant labelled sample. (D) <br />                                           Chromatograms octulose 8-phosphate isotopologues T. brucei incubated unlabelled (U-12C) <br />                                           glucose, labelled (U-13C) glucose, unlabelled (U-12C) ribose labelled (U-13C) ribose. (n = 3). (E) Heatmap <br />                                           relative isotopologue abundances purine nucleotides cofactors labelling 24 hours 50% <br />                                           U-13C-glucose HMI11. Abbreviations: G6P, glucose 6-phosphate; F6P, fructose 6-phosphate; 6PG, 6- <br />                                           phosphogluconate; Ru5P, ribulose 5-phosphate; R5P, ribose 5-phosphate; O8P, octulose 8-phosphate; <br />                                           N9P, nonulose 9-phosphate; GA3P, glyceraldehyde 3-phosphate. (F) T. brucei transaldolase gene                                           heterologously expressed E. coli protein purified (left hand panel, crude E. coli extract                                           left purified protein right). Purified transaldolase shown produce octulose 8-phosphate <br />                                           provided ribose 5-phosphate fructose 6-phosphate <span id='am-261' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-262' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-263' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-264' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span>acceptor</span> donor substrates <br />                                           respectively (right hand panel). Enzymes: 1, Hexokinase; 2, glucose 6-phosphate isomerase; 17, glucose 6- <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                    7 / 25 <br />                                                                                                               Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           phosphate dehydrogenase; 18, 6-phosphogluconate dehydrogenase; 19, ribose 5-phosphate isomerase; 20, <br />                                           ribokinase; 21, transaldolase. <br />                                           doi:10.1371/journal.ppat.1004689.g003 <br />  <br />                                           consequence transaldolase transketolase (TKT) reactions combine fully labelled <br />                                           fully unlabelled precursors [38]. lack isotopologues confirms lack                                           TKT BSFs [20,44]. <br />                                               high molecular weight sugar phosphates, octulose 8-phosphate (O8P) nonulose 9- <br />                                           phosphate (N9P) detected putatively identified accurate mass predicted reten- <br />                                           tion time  authentic standards available). labelling patterns (Fig 3c-d) indicate <br />                                           production transaldolase activity carbon unit donated aldose donor                                           pentose phosphate  produce O8P) hexose phosphate  produce N9P).  <br />                                           ingly, BSF T. brucei previously shown possess transaldolase activity [20], spite <br />                                           absence TKT. absence 2C- 6C- labelled O8P BSFs, present <br />                                           PCFs [38], indicates PPP intermediate precursor metabolite. ex- <br />                                           pressed purified gene encoding T. brucei transaldolase (Tb927.8.5600) Escherichia <br />                                           coli. enzyme mixed ribose 5-phosphate F6P substrates, mass spec- <br />                                           trometry revealed O8P produced, confirming ability enzyme carry                                           reaction (Fig 3f).  cells labelled U-13C-ribose resulted exten- <br />                                           sive labelling ribose 5-phosphate (produced ribokinase), significant 5-carbon label- <br />                                           ling O8P (Fig 3d). <br />                                               detection metabolites formulas consistent sedoheptulose 7-phosphate <br />                                           (C7H15O10P) erythrose 4-phosphate (C4H9O7P) surprising cell TKT activ- <br />                                           ity.  50% U-13C-glucose labelling reveals 3-carbon labelled isotopologue <br />                                            confirming products TKT. investigation                                           abundance C7H15O10P peak low samples, retention time                                           C4H9O7P inconsistent authentic standard erythrose 4-phosphate. work <br />                                           required elucidate identity source metabolites, confirm <br />                                           non-canonical pathway involved sugar phosphate remodelling exists trypanosomes. <br />  <br />                                           Nucleotide salvage <br />                                           lack uniform labelling purine bases, nucleosides nucleotides confirms  <br />                                           quirement exogenous purine bases trypanosomatids lack purine biosynthetic <br />                                           pathways (Fig 3e).  glucose-derived ribose produces 5-carbon labelled purine nucleo- <br />                                           tides, purine salvage pathways highly active [45]. Transporters efficient uptake                                           purine nucleosides characterized, <span id='am-102' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-103' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="guanosine" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>guanosine</span> <span id='am-230' about='obo:IMR_0200194' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-231' property="oboInOwl:hasDbXref" content="KEGG:C00294" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:id" content="IMR:0200194" datatype="xsd:string"></span><span id='am-233' property="rdfs:label" content="Inosine" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>inosine</span> appear major <br />                                           purine sources vitro culture, presence excess <span id='am-215' about='obo:IMR_0200173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-216' property="oboInOwl:id" content="IMR:0200173" datatype="xsd:string"></span><span id='am-217' property="rdfs:label" content="Hypoxanthine" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasDbXref" content="KEGG:C00262" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>hypoxanthine</span> [46]. la- <br />                                           belling data confirm <span id='am-141' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-142' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-143' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span>nucleoside</span> hydrolases [47] form purine bases prima- <br />                                           ry route nucleotide salvage imported nucleosides glucose-derived ribose <br />                                           5-phosphate added purine phosphoribosyltransferase [48]. complete la- <br />                                           belling ribose moiety <span id='am-158' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-159' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-161' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-162' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-163' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-165' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-171' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-172' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span>adenosine</span> nucleotides indicates minimal purine salvage ex- <br />                                           ogenous <span id='am-173' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-174' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-175' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-176' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-177' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-178' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-182' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-183' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span>adenosine</span> adenosine kinase vitro growth conditions, spite                                           high activity cells [49]. <br />                                              Intracellular synthesis various cofactors, including NAD+ S-adenosyl methionine <br />                                           (AdoMet), confirmed incorporation labelled ribose nucleotide subunits. <br />                                           AdoMet important precursor polyamine synthesis, resulting production                                           5&#226;&#8364;&#8482;methylthioadenosine (MTA). 5-carbon labelling metabolites confirms                                           labelling ribose moiety.  absence labelling methionine precludes <br />                                           activity methionine salvage pathway previously proposed [50]. Methionine uptake  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                         8 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           serum [51] apparently fulfils methionine requirements BSF parasites. absence la- <br />                                           belling &#206;&#177;-ketomethylthiobutyrate (KMTB), intermediate methionine salvage <br />                                           pathway, confirms lack function proposed pathway. KMTB probably <br />                                           arises instead transamination methionine, previously shown excreted <br />                                           parasites <span id='am-146' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-147' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-149' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-150' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span>-derived keto acids [46]. <br />  <br />                                           Carboxylic acid derivatives catabolised glucose <br />                                           active TCA cycle reportedly operative BSF T. brucei, observation &lt;1% <br />                                           detected 2-oxoglutarate contains label glucose corroborates (S1 Table). small <br />                                           acetyl-coA oxaloacetate   directed citrate, seen/ob- <br />                                           served 2-, 3- 5-carbon labelling excess natural abundance 13C total <br />                                           approximately 1% cells grown HMI11, 5% CMM-grown cells (S2 Table).                                           relevance low level citrate production known. PCF T. brucei use classi- <br />                                           cal citrate&#226;&#8364;&#8221;malate shuttle, instead utilizing unique acetate shuttle provide acetyl-CoA                                           cytosol [31]. <br />                                              Isotope analysis intracellular metabolites cells incubated 50% U-13C-labelled <br />                                           glucose reveals carbon labelled isotopologues succinate malate, representing 35% <br />                                           (70% corrected 50% labelled glucose precursor) intracellular malate (Fig 4)                                           26% succinate (52% corrected). labelling pattern consistent production                                           succinate succinate fermentation pathway. enzyme pathway, PEPCK,                                           BSF trypanosomes, albeit lower level PCF [52,53]. oxaloacetate                                           detectable analytical platform, observation 3-carbon labelled isotopologue <br />                                           aspartate  transamination product oxaloacetate) supports production oxaloace- <br />                                           tate PEP. total labelling aspartate (62% corrected) comparable malate (70% <br />                                           corrected), demonstrating intracellular aspartate primarily synthesized glucose  <br />                                           rived oxaloacetate, taken surrounding medium, contains <br />                                           aspartate 100 &#206;&#188;M [46]. BSF trypanosomes failed demonstrate appreciable aspartate trans- <br />                                           port capability [54]. T. brucei encodes separate isoforms malate dehydrogenase <br />                                           (MDH), localized mitochondrion, glycosomes cytosol, respectively [55]. gly- <br />                                           cosomal isoform reportedly absent BSF, cytosolic isoform present                                           higher levels BSF procyclics [55]. Additionally small mitochondrial iso- <br />                                           form reportedly present [55&#226;&#8364;&#8220;57]. succinate fermentation pathway  <br />                                           volve oxaloacetate produced glycosome PEPCK enter cytosol                                           possibly mitochondrion metabolism. work, systematically removing                                           knocking expression different isoforms, followed metabolic profiling,  <br />                                           quired deconvolute sources partially reduced products glucose catabolism                                           BSF trypanosome. <br />  <br />                                           Aspartate use nucleotide biosynthesis <br />                                           Oxaloacetate-derived aspartate key precursor pyrimidine synthesis, converted                                           orotate dihydroorotate. Dihydroorotate orotate detected isotope enrichment <br />                                           equivalent aspartate, ~35% 3-labelled carbons (70% corrected), confirming novo synthesis <br />                                           pyrimidines way. <span id='am-251' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-252' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-255' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>Uracil</span> labelling slightly lower (44% corrected), suggesting                                           uptake uracil medium.  higher levels 2-labelling <span id='am-225' about='obo:IMR_0200409' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-226' property="oboInOwl:hasDbXref" content="KEGG:C00105" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:id" content="IMR:0200409" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-229' property="rdfs:label" content="UMP" datatype="xsd:string"></span>UMP</span> <span id='am-281' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-282' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-284' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-285' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span>UDP</span> sug- <br />                                           gest novo synthesized orotate major source pyrimidines conditions <br />                                           (Fig 4). Labelling <span id='am-220' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-221' property="rdfs:label" content="Uridine" datatype="xsd:string"></span><span id='am-222' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span>uridine</span> <span id='am-97' about='obo:IMR_0200161' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-98' property="rdfs:label" content="Cytidine" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:id" content="IMR:0200161" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="KEGG:C00475" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cytidine</span> nucleotides significantly different                                           different culture media used study, HMI11 CMM.  significant dif- <br />                                           ferences observed <span id='am-271' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-272' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-273' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-274' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-275' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>thymidine</span> nucleotides. <span id='am-72' about='obo:IMR_0200084' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00459" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="dTTP" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:id" content="IMR:0200084" datatype="xsd:string"></span>dTTP</span> <span id='am-153' about='obo:IMR_0200187' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-154' property="rdfs:label" content="dTMP" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasDbXref" content="KEGG:C00364" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:id" content="IMR:0200187" datatype="xsd:string"></span>dTMP</span> abundance CMM-grown <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                      9 / 25 <br />                                                                                                                     Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br />                                           Fig 4. Glucose-derived dicarboxylic acid pyrimidine metabolic pathways BSF T. brucei indicate <br />                                           activity PEPCK.   Metabolic pathways dicarboxylic acid pyrimidine synthesis BSF T. brucei. <br />                                           Red carbons structures indicate predominant 13C labelling. (B) Heatmap relative isotopologue <br />                                           abundances dicarboxylic acid pyrimidine intermediates labelling 24 hours 50% U-13C- <br />                                           glucose HMI11. (C) Isotopologue abundances dTTP labelling 0 (U), 1 (L1) 24 (L24) hours <br />                                           50% U-13C-glucose HMI11 (H) CMM (M) (n = 2&#226;&#8364;&#8220;3, mean &#194;&#177; SD). Extensive novo dTTP synthesis <br />                                           observed CMM  e. absence exogenous thymidine). Abbreviation: PEP, <br />                                           phosphoenolpyruvate; Pyr, pyruvate; OxAc, oxaloacetate; Mal, malate; Fum, fumarate; Succ, succinate; Asp, <br />                                           Aspartate; AdSucc, adenylosuccinate; DHO, dihydroorotate; Oro, orotate; Ura, uracil; Urd, uridine; dUrd, <br />                                           deoxyuridine; Tmd, thymidine. Enzymes: 11, pyruvate kinase; 22, malate dehydrogenase; 23, fumarase; 24, <br />                                           <span id='am-60' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span>NADH</span>-dependent fumarate reductase); 25, aspartate aminotransferase; 26, adenylosuccinate synthase; 27, <br />                                           adenylosuccinate lyase; 28, aspartate carbamoyltransferase dihydroorotase; 29, dihydroorotate <br />                                           dehydrogenase; 30, orotate phosphoribosyltransferase orotidine 5-phosphate decarboxylase; 31, <br />                                           nucleoside diphosphatase; 32, ribonucleoside-diphosphate reductase; 33, thymidylate kinase; 34, <br />                                           thymidylate synthase; 35, thymidine kinase; 36, uracil phosphoribosyltransferase; 37, uridine phosphorylase; <br />                                           38, <span id='am-236' about='obo:IMR_0200198' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-237' property="rdfs:label" content="Nucleoside diphosphate" datatype="xsd:string"></span><span id='am-238' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasDbXref" content="KEGG:C11479" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:id" content="IMR:0200198" datatype="xsd:string"></span>nucleoside diphosphate</span> kinase; 39, cytidine <span id='am-128' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-129' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span>triphosphate</span> synthase. <br />                                           doi:10.1371/journal.ppat.1004689.g004 <br />  <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                               10 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           cells half observed HMI11, contains 20 &#206;&#188;M added thymidine. iso- <br />                                           tope labelling patterns dTTP dTMP CMM matched UMP py- <br />                                           rimidine nucleotides, indicating complete synthesis novo synthesized pyrimidines                                           thymidylate synthetase thymidine-poor media [58,59]. dTMP dTTP HMI11-grown <br />                                           cells 10% labelled, confirming preference thymidine salvage action <br />                                           thymidine kinase exogenous thymidine available (Fig 4c). <br />                                               Aspartate important intermediate purine salvage pathway,  <br />                                           mation adenylosuccinate release fumarate results nitrogen transfer aspartate <br />                                           <span id='am-276' about='obo:IMR_0200221' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-277' property="oboInOwl:hasDbXref" content="KEGG:C00130" datatype="xsd:string"></span><span id='am-278' property="rdfs:label" content="IMP" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:id" content="IMR:0200221" datatype="xsd:string"></span><span id='am-280' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>IMP</span>, producing <span id='am-65' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-66' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>AMP</span>. Adenylosuccinate detected 3-carbon labelling,  <br />                                           firms role glucose-derived aspartate purine salvage pathway (Fig 3e). <br />  <br />                                           PEPCK essential BSF T. brucei <br />                                           discovery glucose enters amino acid nucleotide pathways oxaloacetate indi- <br />                                           cates PEPCK play unexpectedly important role BSF trypanosomes.                                            genome-wide RNAi screen indicated gene essential [60]. test  west- <br />                                           ern blot analysis performed anti-PEPCK immune serum, showing PEPCK ex- <br />                                           pressed BSF, albeit lower abundance PCF (Fig 5). cell lines <br />                                           (RNAiPEPCK-B3 RNAiPEPCK-D6) generated investigate role PEPCK                                           BSF T. brucei RNAi regulation pepck gene expression. cell lines showed                                           growth arrest days tetracycline induction. phenotypic reversion 5 days post-induc- <br />                                           tion correlated expression PEPCK (Fig 5). <br />  <br />                                           RNAi <br />                                                PEPCK labelled metabolomics <br />                                           order measure metabolic role PEPCK, production succinate glucose                                           BSF trypanosomes measured using 1H-NMR, previously described PCF T. brucei <br />                                           [61]. parental BSF 427 strain incubated 5 hours PBS/NaHCO3 medium contain- <br />                                           ing 4 mM D-glucose carbon source incubation medium analyzed                                           1 <br />                                            H-NMR spectroscopy. addition pyruvate, detectable amounts succinate quanti- <br />                                           tatively measured, levels ~2.7% excreted pyruvate (Fig 6a Table 1).                                           excreted succinate decreased 4.3 fold RNAiPEPCK-B3 mutant days  <br />                                           duction compared wild-type cells (47 &#194;&#177;17 versus 201 &#194;&#177;89 nmol/h/mg protein). Succi- <br />                                           nate secretion increased days induction (79 &#194;&#177;18 nmol/h/mg protein) (Fig 6a                                           Table 1), consequence pepck expression (Fig 5). <br />                                               confirm reduction succinate production RNAiPEPCK-B3 tetracycline-induced <br />                                           cell line  2 days), determined incorporation 13C-labelling U-13C-glucose  <br />                                           tracellular metabolites LC-MS. reduction 13C3 incorporation malate (24 &#194;&#177; 10%) <br />                                           succinate (29 &#194;&#177; 2%) observed induced RNAiPEPCK-B3 cell line compared                                           non-induced cells (Fig 6b). Labelling products PEPCK-derived oxaloacetate                                           significantly reduced, including aspartate (15-fold), uracil (3-fold) <span id='am-241' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-242' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-243' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-245' property="rdfs:label" content="UTP" datatype="xsd:string"></span>UTP</span> (2-fold).                                           contrast, 13C incorporation metabolites produced upstream PEPCK step (G6P, 3PG <br />                                           pyruvate), pathways derived glycolytic intermediates (alanine, ace- <br />                                           tyl-CoA, ribose 5-phosphate) significantly altered (Fig 6b). <br />  <br />                                           Acetate production BSF T. brucei <br />                                           addition secretion conversion alanine, pyruvate major source acetyl-CoA <br />                                           BSF trypanosomes. intracellular concentration acetyl-CoA                                           limit detection analytical method, evidence production provided 2-hydro- <br />                                           xyethyl thiamine <span id='am-49' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-50' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>pyrophosphate</span> (Fig 7a), intermediate pyruvate&#226;&#8364;&#8482;s conversion  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                    11 / 25 <br />                                                                                                                          Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br /> Fig 5. PEPCK essential bloodstream form T. brucei.   Growth curve different clones showing regulation pepck RNAi <br /> (RNAiPEPCK-D6 RNAiPEPCK-B3) parental 427 BSF strain (WT) incubated presence (  absence (.ni) 10 &#206;&#188;g/ml tetracycline. <br /> Cells maintained exponential growth phase  105 2x106 cells/ml) cumulative cell numbers normalized dilution cultivation. (B) Western blot analyses parental (WT) mutant cell lines anti-PEPCK anti-enolase sera containing antibodies indicated left margin. (C) Western blot demonstrates significantly higher levels PEPCK procyclic form (PF) compared bloodstream form (BF) WT <br /> T. brucei. *Analyses performed 5 x 106 cells sample HSP-60 shown loading control. <br /> doi:10.1371/journal.ppat.1004689.g005 <br />  <br />  <br />  <br />                                                acetyl-CoA PDH. presence labelled carbons indicates formation directly                                                glucose-derived pyruvate. Labelling minimal hour, complete labelling ob- <br />                                                served 24 hours, suggesting flux pathway significantly slower                                                pathways (Fig 7a). recently shown [27] PDH knockdown <br />                                                conditionally lethal BSF T. brucei. threonine, threonine dehydrogenase (TDH) <br />                                                pathway, glucose, PDH, provide acetate route essential <br />                                                threonine absent, redundant loss TDH [27]. Furthermore, acetylated  <br />                                                tabolites including acetylcarnitine, acetyllysine, acetylornithine acetylglutamine                                                detected 2 labelled carbons glucose (S1 Table). unlabelled proportion                                                metabolites probably relates threonine&#226;&#8364;&#8482;s key role acetate provision [27]. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                     12 / 25 <br />                                                                                                                      Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br />                                           Fig 6. Metabolic profile RNAiPEPCK-B3 BSF T. brucei confirms depletion dicarboxylic acids                                           pyrimidines.   Proton (1H) NMR analysis excreted pyruvate succinate glucose. Pyruvate (Pyr) <br />                                           succinate (Suc) excreted bloodstream form 427 cell line (WT), RNAiPEPCK-B3 mutant <br />                                           non-induced (ni), tetracycline-induced 2 days  2 days), tetracycline-induced 5 days  5 days)                                           determined 1H-NMR. cells incubated PBS containing 4 mM glucose 4 hours.                                           spectrum corresponds representative experiment set 3. spectrum <br />                                           ranging 2.2 ppm 2.4 ppm shown. (B) LC-MS analysis intracellular glycolytic intermediates. <br />                                           Relative isotopologue abundances following incubation PBS containing 4 mM U-13C-glucose 4 hours <br />                                           metabolic intermediates RNAiPEPCK-B3 mutant non-induced (NI) tetracycline-induced 2 days <br />                                            . Significant decreases labelling observed PEPCK-derived metabolites including 3-carbon <br />                                           labelling dicarboxylic acids aspartate, 2-carbon labelling pyrimidine nucleotides  7-carbon <br />                                           labelling ribose-containing metabolites). Labelling significantly decreased glycolytic <br />                                           intermediates glucose-derived metabolites, including alanine, acetate (shown acetyllysine) <br />                                           ribose (shown 5-carbon labelling UTP). <br />                                           doi:10.1371/journal.ppat.1004689.g006 <br />  <br />  <br />  <br />                                           Glucose contributions fatty acid lipid biosynthesis <br />                                           novo fatty acid synthesis detected BSF T. brucei incorporation labelled car- <br />                                           bon units free fatty acids phospholipids (Fig 7b).  low level labelling <br />                                           fatty acids suggests novo fatty acid synthesis minor compared salvage <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                 13 / 25 <br />                                                                                                                    Trypanosome Glucose Metabolism <br />  <br />  <br />  <br /> Table 1. Production succinate, pyruvate, alanine acetate glucose PEPCK RNAi T. brucei measured 1H-NMR. <br />  <br /> Cell line                                   n                                           Excreted molecules glucose <br />  <br />                                                              Pyruvate                       Succinate             Alanine                 Acetate <br />                                                                  nmol / h / mg protein <br /> WTa                                         3                7534&#194;&#177;1535                      201&#194;&#177;89                821&#194;&#177;174                 564&#194;&#177;42 <br /> RNAi <br />       PEPCK-3.nib                           6                7281&#194;&#177;634                       127&#194;&#177;26                754&#194;&#177;49                  328&#194;&#177;51 <br /> RNAi <br />       PEPCK-B3.ic 2days                     7                6913&#194;&#177;760                       47&#194;&#177;17                 731&#194;&#177;8                   361&#194;&#177;18 <br /> RNAi <br />       PEPCK-B3.ic 5days                     3                6880&#194;&#177;797                       79&#194;&#177;18                 722&#194;&#177;64                  281&#194;&#177;44     Strain 427 bloodstream-form T. brucei <br /> b <br />     Non-induced RNAiPEPCK-B3  added tetracycline) <br /> c <br />     Induced RNAiPEPCK-B3  added tetracycline) <br />  <br /> doi:10.1371/journal.ppat.1004689.t001 <br />  <br />  <br />                                                 mechanisms, high level lysophospholipid uptake represent principal source                                                 fatty acids cells [62] elongation accounting modelling fatty acids different <br />                                                 chain length [63]. <br />                                                     additional 3-carbon labelling observed glycerophospholipids demonstrates lipid <br />                                                 salvage utilizes glycolysis-derived glycerol 3-phosphate synthesis phospholipid head- <br />                                                 groups. Despite requirement salvage host  culture medium) lipids provi- <br />                                                 sion choline, fatty acids, half glycerol phosphate lipid head-groups <br />                                                 derived glycolysis (ranging 8 100% corrected labelling detected PC (phos- <br />                                                 phatidylcholine) PE (phosphatidylethanolamine) lipids; Fig 7 S1 Table, variability <br />                                                 relating fact relative contributions salvage novo synthesis pathways <br />                                                 uniform lipid species. <br />  <br />                                                 Nucleotide sugars glycoconjugate precursors <br />                                                 labelled metabolites detected (S1 Table) indicating glucose enters <br />                                                 amino sugars (N-acetyl-<span id='am-87' about='obo:IMR_0200485' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-88' property="oboInOwl:id" content="IMR:0200485" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="KEGG:C00329" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="D-Glucosamine" datatype="xsd:string"></span>D-glucosamine</span> <span id='am-256' about='obo:IMR_0200374' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-257' property="rdfs:label" content="N-Acetyl-D-glucosamine" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasDbXref" content="KEGG:C00140" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:id" content="IMR:0200374" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>N-acetyl-D-glucosamine</span> 6-P), sugar nucleotides <br />                                                 (UDP-Glc, UDP-Gal <span id='am-18' about='obo:IMR_0001357' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="GDP" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001357" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00035" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0200191" datatype="xsd:string"></span>GDP</span>-man UDP-GlcNAC; Fig 7c) low-level labelling                                                 sugars (tentatively identified <span id='am-198' about='obo:IMR_0200177' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-199' property="oboInOwl:id" content="IMR:0200177" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasDbXref" content="KEGG:C00137" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-202' property="rdfs:label" content="myo-Inositol" datatype="xsd:string"></span>myo-inositol</span>, fructose, glucuronate arabinonate,                                                 accurate identification sugars difficult using accurate mass-based metabolomics,                                                 isomers theoretically possible <span id='am-42' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-43' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-45' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span>carbohydrate</span> metabolites;  additional orthog- <br />                                                 onal approaches required confidently identify structures. Nothwithstanding                                                 abundance clearly low compared glucose fructose primary hexoses. Mannose <br />                                                 absent free form, generation occurring nucleotide conjugated                                                 phosphorylated form [24]). Targeted studies sugar nucleotide biosynthesis                                                 demonstrated novo biosynthesis UDP-Glc [64] GDP-Man [24] glucose,                                                 analysis inositol metabolism demonstrated novo synthesis inositol protein glycosyl- <br />                                                 ation, addition inositol salvage lipid biosynthesis [25,26]. Data current study <br />                                                 demonstrated 10% labelling inositol, confirmed inositol moiety                                                 lipid headgroup glycerol-<span id='am-7' about='obo:IMR_0001385' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="phosphoinositol" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001385" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="phosphoinositol" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="inositol phosphate" datatype="xsd:string"></span>phosphoinositol</span> unlabelled  e. formed novo synthesis <br />                                                 glucose, consistent targeted analyses conducted investigate origin                                                 inositol T. brucei [25,26]). <br />  <br />                                                 Discussion <br />                                                 introduction metabolomic technologies, particularly conjunction stable isotope <br />                                                 tracing  recent years, transformed ability analyse metabolism biological <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                            14 / 25 <br />                                                                                                                       Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />  <br />                                           Fig 7. Glucose carbon source acetate, lipids sugar nucleotides WT BSF T. brucei.   <br />                                           Time-dependent incorporation glucose diverse pathways. Relative isotopologue abundances                                           unlabelled cells (0) cells incubated 50% U-13C-glucose HMI11 hour (1) 24 hours (24). <br />                                           Labelling reaches equilibrium 1 hour pyruvate, alanine, malate, aspartate ribose 5-phosphate. <br />                                           Minimal labelling observed 2-hydroxyethyl-<span id='am-266' about='obo:IMR_0200378' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-267' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-268' property="rdfs:label" content="TPP" datatype="xsd:string"></span><span id='am-269' property="oboInOwl:id" content="IMR:0200378" datatype="xsd:string"></span><span id='am-270' property="oboInOwl:hasDbXref" content="KEGG:C00068" datatype="xsd:string"></span>TPP</span> 1 hour, complete (50%) labelling occurs                                           24 hours, indicating relatively slow flux acetate production pathway. (n = 3, mean &#194;&#177; SD). (B) <br />                                           Heatmap relative isotopologue abundances lipids labelling 24 hours 50% U-13C-glucose                                           HMI11 indicating combination salvage mechanisms novo synthesized glycerol phosphate                                           limited incorporation glucose-derived acetyl-CoA. manual curation, isotopologues containing                                           6 labelled carbons confirmed detected lipids. (C) Isotopologue abundances UDP-N- <br />                                           acetyl glucosamine (UDP-GlcNAc) labelling 24 hours 50% U-13C-glucose HMI11 reveal <br />                                           incorporation labelled carbon glucosamine, acetate, ribose uracil. Structures represent                                           likely isotopologue(s) mass 13C atoms bonds shown red. Additional minor isotopologues <br />                                           omitted clarity, abundant isotopologues containing 3-labelled <br />                                           glucosamine 6-phosphate. <br />                                           doi:10.1371/journal.ppat.1004689.g007 <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                                  15 / 25 <br />                                                                                                              Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           systems. example, isotopologue studies Leishmania [65], Plasmodium [36,37]                                           Toxoplasma [35] identified novel pathways resolved long-standing questions  <br />                                           tabolism. African trypanosome, Trypanosoma brucei, causative agent human Afri- <br />                                           trypanosomiasis, neglected tropical disease sub-Saharan Africa, new drugs <br />                                           urgently needed. slender bloodstream form parasites generally  <br />                                           sidered highly streamlined glucose catabolic pathway [3&#226;&#8364;&#8220;5,9,10]. localization                                           seven enzymes glycolytic pathway membrane-bounded peroxisome-like  <br />                                           ganelles known glycosomes [5] proposed instrumental regulation                                           flux glycolysis providing environment <span id='am-112' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-113' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>/<span id='am-24' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span>ADP</span> NAD/NADH <br />                                           balanced [9,10]. arrangement enabled production defined dynam- <br />                                           ic mathematical model glycolysis [9,10] models able predict im- <br />                                           portant phenomena, contribution different enzymes overall control                                           pathway. Recent iterations included addition parameters explicitly ac- <br />                                           count uncertainty related [66] activity pentose phosphate <br />                                           pathway [22] presence permeability pores allow free diffusion glyco- <br />                                           lytic intermediates cytosol glycosome [40]. <br />                                               taken systematic route analyse extent glucose metabolism                                           long slender BSF trypanosomes. <br />                                               Extensive pyruvate production observed, consistent existing models glycolytic <br />                                           flux energy production.  3-carbon labelled isotopologues observed glycolyt- <br />                                           ic hexose phosphates reveals previously unreported complexity pathway. lack                                           transketolase [20,42] indicates canonical non-oxidative pentose phosphate cycle miss- <br />                                           ing cells source 3-labelled hexose phosphates. role                                           transaldolase BSFs uncertain, transaldolase-mediated production 3-labelled fruc- <br />                                           tose 6-phosphate  F6P GA3P) hypothesized, appears                                           redundant reaction. Alternatively, aldolase [67], acting combine U-13C U-12C <br />                                           DHAP U-13C U-12C GA3P produce relative proportions 0-, 3- 6- <br />                                           labelled fructose bisphosphate identified  production 3-labelled F6P                                            G6P) FBP require fructose-1,6-bisphosphatase activity, transcriptomic <br />                                           [68] proteomic [69] data indicate enzyme present BSF T. brucei. family                                           models, simulating flux glycolysis whilst taking account uncertainties                                           distribution glycosome cytosol [40], reveals metabolites <br />                                           smaller FBP able exchange compartments FBP  signifi- <br />                                           proportion total FBP cell arise condensation GA3P <br />                                           DHAP. data,  supports proposed role pores creating semi-permeable <br />                                           glycosome [41]. work required fully investigate intermediate fluxes glycolyt- <br />                                           ic metabolism determine temporal spatial aspects metabolism rise                                           unexpected observations. <br />                                               point rule definitively production compound mass <br />                                           retention time FBP artefact experimental  true                                           exotic, unexpected species identified study.  non-enzymatic aldol  <br />                                           densation triose phosphates highly unlikely, given non-enzymatic condensations                                           additions generally require extreme pH conditions (&lt;1 &gt; 9). confirm  DHAP                                           GA3P mixed CMM medium followed extraction conditions used extract <br />                                           cellular lysates storage seven days -80&#194;&#176;C, FBP potential conden- <br />                                           sation addition products observed LC-MS. consider likely                                           metabolites bona fide products trypanosome metabo- <br />                                           lism, incorporation carbon-13 label confirms endogenous production                                           metabolites (compared unlabelled metabolites primarily acquired serum  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     16 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           growth media), physiological function, relevance remains unknown. <br />                                           Metabolites produced enzymatically chemically cells inevitable prod- <br />                                           ucts metabolism, methylglyoxal example discussed [42]. <br />                                               Apart identification numerous unexpected metabolites pathways operat- <br />                                           ing BSF trypanosomes, experiments rule operation pathways pre- <br />                                           viously assumed functional. example, rapid deamination aromatic amino acids <br />                                           believed contribute methionine cycle involved provision decarboxylated <br />                                           S-adenosylmethionine used polyamine synthesis [50].   demonstrate <br />                                           carbons glucose enter methionine, provided exclusively                                           cellular uptake instead [51]. means pathways methylthioadenosine detoxifi- <br />                                           cation remain elucidated BSF trypanosome. <br />                                               majority glucose flows secreted pyruvate aerobic culture conditions, <br />                                           glucose metabolism pervasive BSF parasites. addition pyruvate                                           glycerol, various metabolites secreted, including alanine, transaminated product <br />                                           pyruvate. reaction appears essential alanine transaminase gene                                           knocked T. brucei genome [19], knockdown RNAi possible. Suc- <br />                                           cinate secreted. succinate produced BSF trypanosomes primarily labelled                                           carbons, indicates derivation glycolytic phosphoenolpyruvate,                                           canonical TCA cycle, produce primarily 2-carbon 4-carbon labelled <br />                                           isotopologues. Analysis phosphoproteome BSFs identified glycosomal NADH-depen- <br />                                           dent fumarate dehydrogenase [70].  possible distinguish glycoso- <br />                                           mal, cytosolic potentially mitochondrial derivation metabolites, systematic <br />                                           metabolite profiling following gene knockout knockdown various enzymes neces- <br />                                           sary deconvolute pathways. <br />                                               suggested glucose makes negligible contribution anabolic pro- <br />                                           cesses BSF trypanosomes [10], numerous isolated studies revealed glucose serves <br />                                           precursor processes including sugar nucleotide glycoconjugate synthesis [23], inosi- <br />                                           tol production [25,26] recently acetate production [27]. Glucose enters pyrimidine <br />                                           biosynthesis oxaloacetate lipid metabolism glycerol 3-phosphate acetate <br />                                           production. RNAi knockdown PEPCK confirmed activity essential BSF try- <br />                                           panosomes. enzyme essential PCFs, contribution major                                           parasitic insect form [71]. recently demonstrated loss bound phosphate                                           glycolytic pathway glycosome PPP requires mechanism retain bound <br />                                           phosphate balance [22]. PEPCK offers obvious means restore glycosomal ATP,                                           link PPP explain unexpectedly essential nature enzyme                                           BSF trypanosome. <br />                                               untargeted mass spectrometry platform use intrinsically unable                                           quantify total flux glucose carbon various pathways discussed  clearly <br />                                           case overwhelming majority glucose catabolised pyruvate (Table 1).                                           total glucose entering pathways order just percent <br />                                           total consumed, agreement measurements Haanstra et al [10].                                           small minority total glucose consumption,  glucose major carbon <br />                                           source production extra-glycolytic metabolites, flux clearly <br />                                           important given fact enzymes, including PEPCK shown  turn es- <br />                                           sential metabolism parasites. <br />                                               identification novel essential enzymes BSF trypanosome important <br />                                           implications chemotherapy. Glycolysis long considered attractive target given                                           absolute dependency glucose energy carbon provision cells [72]; results <br />                                           additional pathways periphery glycolysis equally promising. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     17 / 25 <br />                                                                                                              Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           Materials Methods <br />                                           Parasite culture <br />                                           T. brucei brucei bloodstream form (s427) cultured vitro HMI11 [73] CMM [44] <br />                                           supplemented 10% fetal calf serum (FCS) Gold (PAA, Piscataway, NJ) 37&#194;&#176;C, 5% <span id='am-77' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-78' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span>CO2</span>. <br />                                           Initial 5 ml cultures 25 ml vented flasks (Corning) grown maximum density                                           4 &#195;&#8212; 106 cells/ml subcultured 1 100 1 1,000 dilution 2 3 days, respec- <br />                                           tively. hemocytometer (Neubauer) used cell counts. <br />  <br />  <br />  <br />                                           Labelling experiments sample preparation <br />                                           days prior extraction, 2 &#195;&#8212; 104 cells/ml seeded 30 ml HMI11 CMM. <br />                                           Growth media replaced media containing 50% 13C-labelled glucose (12.5 mM <br />                                           12C-glucose + 12.5 mM 13C-glucose HMI11, 5 mM 12C-glucose 5 mM                                           13 <br />                                              C-glucose CMM culture) 24 h 1 h prior extraction. samples extracted                                           time atequivalent cell densities approximately 2 &#195;&#8212;106 cells/ml. yield 5 &#195;&#8212;107 cells, <br />                                           appropriate volumes cell culture harvested quenched EtOH/dry ice bath pre- <br />                                           viously described [74]. Cells centrifuged 1250 &#195;&#8212;g 10 min spent media removed; <br />                                           cells washed pre-cooled phosphate buffered saline (PBS) 4&#194;&#176;C. centrifugation <br />                                           1250 &#195;&#8212;g 5 min, supernatant removed metabolites extracted adding 100 &#206;&#188;l <br />                                           (ice cold) chloroform/methanol/water (1:3:1) mixed vigorously 1 h 4&#194;&#176;C. ex- <br />                                           traction mixtures centrifuged 13,000 &#195;&#8212;g 10 min supernatants collected                                           stored -80&#194;&#176;C prior analysis. order assess instrument performance, pooled quality <br />                                           control (QC) sample prepared mixing equal volume samples. Independent <br />                                           biological replicates prepared different days. <br />  <br />  <br />  <br />                                           LC-MS methodologies <br />                                           Hydrophilic interaction liquid chromatography (HILIC) carried Dionex UltiMate <br />                                           3000 RSLC (Thermo Fisher Scientific, Hemel Hempstead, UK) using ZIC-pHILIC <br />                                           column (150 mm &#195;&#8212; 4.6 mm, 5 &#206;&#188;m column, Merck Sequant) previously described [74]. Brief- <br />                                           ly, column maintained 30&#194;&#176;C samples eluted linear gradient (20 mM <br />                                           ammonium carbonate water, acetonitrile, B) 46 min flow rate 0.3 ml/min <br />                                           follows: 80% B 20% B 30 min, 5% B 32 min held 39 min washing,                                           80% B 40 min held 46 min equilibration. injection volume 10 &#206;&#188;l                                           samples maintained 4&#194;&#176;C prior injection. MS analysis, Thermo Orbitrap <br />                                           Exactive (Thermo Fisher Scientific) operated polarity switching mode MS set- <br />                                           tings follows: resolution 50,000, AGC 106, m range 70&#226;&#8364;&#8220;1400, sheath <span id='am-1' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-6' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span>gas</span> 40, auxiliary <br />                                           gas 5, sweep gas 1, probe temperature 150&#194;&#176;C, capillary temperature 275&#194;&#176;C. positive <br />                                           mode ionisation: source voltage +4.5 kV, capillary voltage +50 V, tube voltage +70 kV, skim- <br />                                           mer voltage +20 V. negative mode ionisation: source voltage-3.5 kV, capillary voltage-50 <br />                                           V, tube voltage-70 V, skimmer voltage-20 V. Mass calibration performed polarity <br />                                           immediately prior analysis batch. calibration mass range extended cover <br />                                           small metabolites inclusion low-mass contaminants standard Thermo calmix <br />                                           masses  m 1400), C2H6NO2 positive <span id='am-54' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-55' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span> electrospray ionisation (PIESI) mode <br />                                           (m 76.0393) C3H5O3 negative ion electospray ionisation (NIESI) mode (m                                           89.0244). enhance calibration stability, lock-mass correction applied ana- <br />                                           lytical run using ubiquitous low-mass contaminants. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                      18 / 25 <br />                                                                                                              Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           Data processing analysis <br />                                           Raw LC-MS data processed XCMS untargeted peak detection [75],                                           mzMatch.R [76] employed peak matching annotation related peaks. Tentative <br />                                           metabolite identification carried IDEOM using default parameters [77]. Metab- <br />                                           olite identification performed matching accurate masses retention times authen- <br />                                           tic standards (MSI confidence level 1, e. annotated compound matched standard                                           orthogonal approaches), standards available, predicted retention times <br />                                           calculated previously validated model [78] (MSI confidence level 2, e. putatively <br />                                           annotated compound based exact mass determination). Published literature path- <br />                                           way/genome databases considered improve annotation cases isomers                                           differentiated based accurate mass retention time. Metadata support iden- <br />                                           tification metabolite available IDEOM file study (S3 Table). Metabo- <br />                                           lites included manuscript manually annotated using authentic standards                                           available.  note metabolite names given Ideom file generat- <br />                                           ed automatically software provides best match database entries given mass <br />                                           formula. absence additional information considered putatively- <br />                                           annotated hits; confidence score column adjacent hit serves guide  <br />                                           Clearly scope study provide authenticated annotations hun- <br />                                           dreds detected compounds, datasets included spirit open <br />                                           access data. <br />                                               mzmatch-ISO software [39] used extract isotopologue abundances                                           identified putative annotated metabolites. Raw data available associated IDEOM <br />                                           files (S3 Table) chromatograms separate pdf files (S1&#226;&#8364;&#8220;S2 Figs). Labelled metabolites <br />                                           given S4 Table S5 Table. Data analysis interpretation based 24 hour <br />                                           time-point, approximate steady state, unless stated. <br />  <br />                                           NMR experiments <br />                                           T. brucei bloodstream form cells (2.5 x107) collected centrifugation 1,400 g 10 <br />                                           min, washed PBS containing 4 mM glucose incubated 5 h 37&#194;&#176;C 2.5 ml  <br />                                           cubation buffer (PBS supplemented 5 g/l NaHCO3, pH 7.4) containing 4 mM glucose. <br />                                           integrity cells incubation checked microscopic observation.                                           supernatant collected 50 &#206;&#188;l malate solution D2O (20 mM) added internal <br />                                           reference. 1H-NMR spectra performed 125.77 MHz Bruker DPX500 spectrometer <br />                                           equipped 5 mm broadband probe head. Measurements recorded 25&#194;&#176;C                                           ERETIC method. method provides electronically synthesized reference signal [79]. Ac- <br />                                           quisition conditions follows: 90&#194;&#176; flip angle, 5,000 Hz spectral width, 32 K memory size, <br />                                           9.3 sec total recycle time. Measurements performed 256 scans total time <br />                                           close 40 min. experiment, phase ERETIC peak precisely adjusted. Reso- <br />                                           nances obtained spectra integrated results expressed relative ERETIC <br />                                           peak integration. <br />  <br />                                           Inhibition pepck gene expression RNAi <br />                                           Inhibition pepck gene expression RNAi 427 strain performed expression                                           stem-loop &#226;&#8364;&#339;sense/anti-sense&#226;&#8364;? RNA molecules targeted sequences introduced                                           pHD1334 expression vector.  pLew-PEPCK-SAS plasmid constructed target                                           481 bp fragment pepck gene  position 22 bp 503 bp). Briefly, PCR-amplified <br />                                           569 bp fragment, containing antisense pepck sequence restriction sites added                                           primers inserted HindIII BamHI restriction sites pLew100 plasmid. <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     19 / 25 <br />                                                                                                             Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           PCR-amplified fragment containing sense pepck sequence (505 bp) inserted <br />                                           upstream anti-sense sequence, using HindIII XhoI restriction sites (XhoI intro- <br />                                           duced 3&#226;&#8364;&#8482;-extremity antisense PCR fragment). resulting plasmid (pLew- <br />                                           PEPCK-SAS) contains sense antisense version targeted gene fragment, separated <br />                                           52 bp fragment, control PARP promoter linked prokaryotic tetracy- <br />                                           cline (Tet) operator. PEPCK-SAS HindIII-BamHI cassette extracted pLew- <br />                                           PEPCK-SAS plasmid inserted HindIII-BamHI digested pHD1334 vector. RNAi- <br />                                           PEPCK cell lines produced transfecting 427 &#226;&#8364;&#339;double marker&#226;&#8364;? (Hyg-Neo) mono- <br />                                           morphic cell line pHD-PEPCK-SAS plasmid. Transfected cells selected                                           IMDM medium containing, hygromycin (5 &#206;&#188;g/ml), neomycin (2.5 &#206;&#188;g/ml) phleomycin <br />                                           (2.5 &#206;&#188;g/ml). Induction double-stranded RNA expression performed addition 10 <br />                                           &#206;&#188;g/ml tetracycline. <br />  <br />  <br />                                           Cloning expression T. brucei transaldolase gene <br />                                           gene annotated transaldolase T. brucei genome database Tb427.08.5600,  <br />                                           plified following primers 5&#226;&#8364;&#8482;-GCCATATGAATCAACTGGAGAGCCT-3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482;- <br />                                           GCCTCGAGTTACAAAAGTGTAGCGTGA-3&#226;&#8364;&#8482;. PCR product cloned                                           vector pGEM-T easy transferred pET28a(+) expression vector using intro- <br />                                           duced Nde1 Xho1 restriction sites. resultant plasmid sequenced confirm fideli- <br />                                           ty transformed E. coli strain BL21 (DE3) expression 37&#194;&#176;C 4 hours                                           induction IPTG. Expressed protein observed SDS protein gel electrophoresis                                           purified using Ni2+ chelate chromatography (Fig 2f). ability transaldolase produce <br />                                           octulose 8-phosphate determined mixing ribose 5-phosphate (5 mM) fructose <br />                                           6-phosphate (5 mM) Tris-HCl (pH 7.2) adding transaldolase (1.2 mg/mL) incu- <br />                                           bating 20&#194;&#176;C 2 hrs. Alternatively reaction set enzyme. presence <br />                                           octulose 8-phosphate determined direct infusion post-reaction mix                                           Exactive mass spectrometer (Fig 2f). <br />  <br />  <br />                                           Western blot analyses <br />                                           Total protein extracts wild-type mutant BSF T. brucei (5 x106 cells) size-fraction- <br />                                           ated SDS-PAGE (10%) immunoblotted Immobilon-P filters (Millipore) [80]. Immu- <br />                                           nodetection performed described [81,82] using primary antibodies, rat anti-T. brucei <br />                                           PEPCK (diluted 1:1000; gift T. Seebeck, Bern, Switzerland), rabbit anti- T. brucei enolase <br />                                           (diluted 1:1,000, gift P. Michels, Edinburgh, UK) mouse anti-HSP60 (diluted 1:10 <br />                                           000) [83], secondary antibodies, anti-rat, anti-rabbit anti-mouse IgG conjugated                                           horseradish peroxidase (BioRad, 1:5,000 dilution). Revelation performed using Super- <br />                                           Signal West Pico Chemiluminescent Substrate described manufacturer (Thermo Fish- <br />                                           er Scientific). Alternatively, quantitative analyses, revelation performed using                                           Luminata Crescendo Western HRP Substrate (Millipore). Images acquired analyzed <br />                                           KODAK Image Station 4000 MM quantitative analyses performed                                           KODAK MI application. <br />  <br />  <br />                                           Supporting Information <br />                                           S1 Text. PDF outlining use computational models glucose metabolism predict rela- <br />                                           tive contributions fructose 1,6 bis-phosphate pool. <br />                                           (PDF) <br />  <br />  <br />  <br /> PLOS Pathogens | DOI:10.1371/journal.ppat.1004689 March 16, 2015                                                                     20 / 25 <br />                                                                                                                       Trypanosome Glucose Metabolism <br />  <br />  <br />  <br />                                           S1 Table. csv file relative isotopologue abundances BSF T. brucei grown 50%- <br />                                           U-13C-glucose HMI11. <br />                                           (CSV) <br />                                           S2 Table. csv file relative isotopologue abundances BSF T. brucei grown 50%- <br />                                           U-13C-glucose CMM. <br />                                           (CSV) <br />                                           S3 Table. IDEOM macro-enabled Excel file metabolomics data BSF T. brucei <br />                                           grown 50%-U-13C-glucose HMI11 CMM. <br />                                           (ZIP) <br />                                           S4 Table. csv file relative isotopologue abundances BSF T. brucei grown 50%- <br />                                           U-13C-glucose CMM HMI11. <br />                                           (ZIP) <br />                                           S5 Table. Table showing composition HMI11 CMM medium. <br />                                           (DOCX) <br />                                           S1 Fig. PDF chromatograms positive mode ESI isotopologues detected                                           identified putatively annotated metabolites BSF T. brucei grown 50%-U- <br />                                           13C-glucose. <br />                                           (PDF) <br />                                           S2 Fig. PDF chromatograms negative mode ESI isotopologues detected                                           identified putatively annotated metabolites BSF T. brucei grown 50%-U- <br />                                           13C-glucose. <br />                                           (PDF) <br /> </body></html>