<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL VIROLOGY, June 2004, p. 5875&#226;&#8364;&#8220;5882                                                                                    Vol. 78,  11 <br /> 0022-538X/04/$08.00&#226;&#171;&#185;0 DOI: 10.1128/JVI.78.11.5875&#226;&#8364;&#8220;5882.2004 <br /> Copyright &#194;&#169; 2004, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />    NV Gene Snakehead Rhabdovirus (SHRV) Required <br />         Pathogenesis, Heterologous Glycoprotein                Incorporated SHRV Envelope <br />        Marta Alonso,1 Carol H. Kim,2 Marc C. Johnson,3 Meagan Pressley,2 Jo-Ann Leong1,4* <br />      Department Microbiology Center Salmon Disease Research, Oregon State University, Corvallis, Oregon 973301; <br />           Department Biochemistry, Microbiology Molecular Biology, University Maine, Orono, Maine 044692; <br />                  Department Molecular Biology Genetics, Cornell University, Ithaca, New York 148533; <br />                                Hawaii Institute Marine Biology, Kaneohe, Hawaii 967444 <br />                                            Received 13 October 2003/Accepted 4 February 2004 <br />  <br />             Snakehead rhabdovirus (SHRV) affects warm-water fish Southeast Asia belongs genus Novirh- <br />           abdovirus virtue &#226;&#8364;&#339;nonvirion&#226;&#8364;? (NV) gene. examine function NV gene, used recently <br />           developed reverse genetic produce viable recombinant SHRV carrying NV gene deletion.           recombinant virus produced rate final concentrations wild-type virus           cultured fish cells spite NV gene deletion. role NV protein fish pathogenesis           investigated. Zebra fish (Danio rerio) infected NV deletion mutant recombinant virus <br />           containing copy SHRV genome, similar mortality rates final mortalities recorded, <br />           suggesting apparent role NV protein fish pathogenesis. Interestingly, unsuccessful rescue           fully viable recombinants genomes containing deletions G/NV gene junction suggested role           gene junction virus transcription replication. Finally, demonstrated SHRV glycoprotein           replaced glycoprotein infectious hematopoietic necrosis virus (IHNV) hybrid protein <br />           composed SHRV IHNV sequences. <br />  <br />  <br />    Snakehead rhabdovirus (SHRV), rhabdovirus warm-                      rhabdovirus (SHRV) length cDNA clone water fish, isolated diseased snakehead fish (Ophi-               viral genome (11). length cDNA assembled cephalus striatus) epizootic outbreak Thailand                  clones SHRV genes intergenic <br /> (12). disease characterized severe ulcerative                   regions inserted transcription plasmid dermal necrosis seen wide variety freshwater                 T7 promoter hepatitis delta virus ribozyme. gen- estuarine fishes, wild pond culture,               erated plasmid transfected epithelioma papulosum Southeast Asia. disease affected spe-               cyprini (EPC) cells expressed T7 RNA polymerase <br /> cies, cultured snakehead fish, walking catfish (Clarius              recombinant vaccinia virus vTF7-3 (9) sep- <br /> batrachus), sand goby (Oxyeleotris marmoratus)                  arate plasmids encoded viral nucleocapsid protein (N), noticeably affected (10). SHRV nonsegmented                    polymerase protein (L), phosphoprotein (P), respectively, <br /> negative-strand RNA virus Rhabdoviridae family, ge-                 control T7 RNA polymerase promoter. nus Novirhabdovirus. genome SHRV contains open                    expression positive-strand RNA copy genome reading frames (ORF) order 3&#226;&#172;&#732;-N-P-M-G-NV-L-5&#226;&#172;&#732; (13).                   viral proteins necessary vitro viral replication junction genes, stop/polyadenylation                  resulted generation recombinant viruses. examine <br /> signal following start signal separated inter-             function NV gene, constructed SHRV geno- <br /> genic region composed nucleotides                    mic cDNA clone <span id='am-11' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-17' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> changes NV ORF <br /> present mRNAs. major distinguishing feature                resulted stop codon (11). virus produced Novirhabdoviruses presence nonvirion (NV) gene                   template appeared different wild-type viral glycoprotein (G) polymerase (L)                    virus tissue culture infectivity assays.  stop <br /> genes (13). NV gene contains single ORF, 335                  codon introduced NV knockout mutant virus, nucleotides long. NV protein detected infec-                 readthrough single stop codon account tious hematopoietic necrosis virus (IHNV)-infected tissue cul-               apparent lack phenotypic change mutant. <br /> ture cells autoradiography (14) immunofluorescence                       present paper, function NV protein techniques (21) tissues IHNV-infected fish                  examined generating recombinant SHRV carrying <br /> focal microscopy (6). functions IHNV pro-                  complete deletion NV gene. successful recovery teins determined, function NV protein                   viable deletion mutant confirmed NV protein remains unknown.                                                             essential virus replication cell culture. Growth curves <br />    2000, reported recovery recombinant snakehead                revealed similar virus production deletion mutant                                                                              wild-type virus, indicating NV necessary virus <br />    * Corresponding author. Mailing address: Hawaii Institute Ma- <br />                                                                              growth cell culture. NV protein doesn&#226;&#8364;&#8482;t appear play <br /> rine Biology, P.O. Box 1346, Kaneohe, HI 96744. Phone: (808) 236-            significant role virus pathogenesis vivo  7401. Fax: (808) 236-7443. E-mail: joannleo@hawaii.edu.                      zebra fish similarly infected deletion mu- <br />  <br />                                                                       5875 <br />  5876         ALONSO ET AL.                                                                                                                                 J. VIROL. <br />  <br />  <br />                                                               TABLE 1. Primers used study <br />                                                                                                                                                    Position (nt Primer code                                                            Sequencec (5&#226;&#172;&#732;&#226;&#8364;&#8220;3&#226;&#172;&#732;) <br />                                                                                                                                                  underlined sequence <br />  <br />       632b                               GCGGCCGCGAGAAACTAGATGTCCCATCTTCTT                                                                           5265&#226;&#8364;&#8220;5289d <br />       634b                               GCTAGCGGCAGTTTCGTTCTTCTGTTCC                                                                                4872&#226;&#8364;&#8220;4893d <br />       635b                               CTCGTGATTGGGCTTGAGTCTCT                                                                                     4794&#226;&#8364;&#8220;4816d <br />       636b                               CGGTTCTTTGATTGTGCACTATA                                                                                     4834&#226;&#8364;&#8220;4857d <br />       653                                GAGTAGAGATCTGATGCGGTC                                                                                       4729&#226;&#8364;&#8220;4749d <br />       580                                CGTTGTGGGTTGATGGAGGGCCTAGT                                                                                  5424&#226;&#8364;&#8220;5450d <br />       531                                ATATATGAAGACAACGACTCCTGGTAGA                                                                                5239&#226;&#8364;&#8220;5266d <br />       554                                CAAATATCTCGTGGACTCGTG                                                                                       7279&#226;&#8364;&#8220;7290d <br />       719b                               GTCCGGCTAGCTTTGGTTTCATGTTTGGGAG                                                                             3208&#226;&#8364;&#8220;3227d <br />       720b                               ATTGTGCGGCCGCAACATGGGTGTAAATTGGACTC                                                                         4576&#226;&#8364;&#8220;4597d <br />       721b                               ATTGTGCGGCCGCTCAGGCCCCATCACAGTT                                                                             4750&#226;&#8364;&#8220;4567d <br />       722                                GGGAATGGCAATCGGTGTCA                                                                                        2954&#226;&#8364;&#8220;2973d <br />       744                                TCTGTTCCTGGGTGTGCTTG                                                                                        48594877d <br />       749                                GGATTGTCCTGTTTCTGGTGTTC                                                                                     3231&#226;&#8364;&#8220;3253d <br />       750                                TGGGTATCTTCTGGTTTGAGGTC                                                                                     3473&#226;&#8364;&#8220;3495d <br />       738                                GCGCGCGCGCTAGCAACGCAACTCGCAGAGAC                                                                            18&#226;&#8364;&#8220;37e <br />       740                                GCGCGCGCGCGGCCGCGTCTGGTGGGGAGGAAGTGAA                                                                       1596&#226;&#8364;&#8220;1607e <br />       739                                GCGCGCGCGCGGCCGCGTCTTGTACTGGGCGACGTATTT                                                                     1274&#226;&#8364;&#8220;1296e <br />       450                                AACGCAACTCGCAGAGACC                                                                                         17&#226;&#8364;&#8220;36e <br />       451                                GTCTGGTGGGGAGGAAGTGAA                                                                                       1586&#226;&#8364;&#8220;1607e <br />       nt, nucleotides. <br />   b <br />     Mutagenesis primer. <br />   c <br />     Restriction sites boldface. <br />   d <br />     Position SHRV genome (GenBank accession  AF147498). <br />   e <br />     Position IHNV G gene (GenBank accession  X73872). <br />  <br />  <br />  <br /> tant recombinant virus containing entire SHRV                              4834 5265, pMJ-SHRV-(NV-ORF&#226;&#171;&#186; &#226;&#171;&#185;55) missing NV mRNA <br /> genome, significant differences mortality observed.                        region nucleotide 4816 5265. structures engineered mu- <br />                                                                                       tants shown Fig. 1A. Gene manipulations carried sub- examine closely function NV protein,                            clone pMCS-NV, containing HpaI/SacI 1.3-kb fragment pMJ-SHRV <br /> generated additional SHRV cDNAs lacking 37 55 bp                            spanning C-terminal region G gene, G/NV gene junction, NV gene, <br /> G/NV gene junction. failure mutant cDNAs                              NV/L gene border sequence. mutagenesis experi- <br /> rescue infectivity suggested G/NV gene junction                        ments, 0.5 pmol pMCS-NV added 25 &#226;?&#174;l PCR mixture  <br />                                                                                       taining following reagents: 2.5 &#226;?&#174;l 10&#226;&#171;&#187; mutagenesis buffer; 15 pmol essential viral transcription replication. addition, <br />                                                                                       sense antisense primers, sense primer 5&#226;&#172;&#732; phosphorylated; constructed novel recombinant viruses contained                           1 &#226;?&#174;l 25 mM deoxynucleoside <span id='am-6' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span>triphosphate</span> (dNTP) mixture; 5 U IHNV G gene hybrid IHNV G/SHRV G gene place                                ExSite DNA polymerase blend. create pMCS-NV-(ORF&#226;&#171;&#186; &#226;&#171;&#185;55), primers SHRV G gene.                                                                      632 635 added PCR mixture. construct pMCS-NV- <br />                                                                                       (ORF&#226;&#171;&#186;) pMCS-NV-(ORF&#226;&#171;&#186; &#226;&#171;&#185;37), primer pairs 632-634 632-636 <br />                                                                                       added mutagenesis reaction mixture, respectively. sequences <br />                         MATERIALS METHODS                                         primers used mutagenesis experiments shown Table 1.    Cell viruses. EPC cells (8) used virus propagation. cells    PCR amplifications performed PTC-200 Peltier thermal cycler <br /> grown 29&#194;&#176;C minimal essential medium (MEM) supplemented 10%                 cycling parameters: 1 cycle 30 s 94&#194;&#176;C, followed 10 cycles <br /> fetal bovine serum 2 mM <span id='am-1' about='obo:IMR_0200448' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0200448" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="L-Glutamine" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00064" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-glutamine</span>. SHRV obtained John              30 s 94&#194;&#176;C, 30 s 55&#194;&#176;C, 5 min 68&#194;&#176;C. Following completion Fryer, Oregon State University. preparation virus stocks, confluent EPC        PCR, plasmid population subsequently digested 10 U cells infected wild-type SHRV multiplicity infection (MOI)      DpnI restriction enzyme, specific methylated hemimeth- 0.001 MEM 2% fetal bovine serum. 1 h adsorption,              ylated DNA. nonmutated parental plasmids, methylated <br /> inoculum removed, cells incubated 29&#194;&#176;C extensive            growth vivo, digested DpnI restriction enzyme. PCR- <br /> cytopathic effect (CPE) observed.                                                 generated mutagenesis primer-incorporating linear plasmids remain undi- <br />    Oligonucleotide primers. oligonucleotide primers used study        gested. undigested linear DNA blunt ended 1.25 U listed Table 1. location primer SHRV genome              cloned Pfu DNA polymerase remove extended bases placed 3&#226;&#172;&#732; ends <br /> indicated numbers nucleotide positions.                                 PCR products ExSite DNA polymerase blend. Pfu DNA <br />    Plasmid construction mutagenesis. length cDNA clone SHRV             polymerase-treated DNA intramolecularly ligated 37&#194;&#176;C 1 h <br /> RNA (pMJ-SHRV) assembled clones SHRV genes                4 U T4 DNA ligase. ligated DNA used transform Esch- intergenic junctions standard cloning techniques (11). study,     erichia coli XL1-Blue supercompetent cells. Plasmid DNA purified using <br /> SHRV cDNA expression vector engineered contain truncated NV ORF.             Qiaprep Spin Miniprep kit following manufacturer&#226;&#8364;&#8482;s conditions vector, pMJ-SHRV-(NV single stop), created nucleotide              (Qiagen, Valencia, Calif.). Direct sequence analysis using primers 653 alteration NV ORF resulted stop codon 22 nt NV ORF          580 used identify site-specific mutations pMCS-NV-(ORF&#226;&#171;&#186;), <br /> (Fig. 1). construct mutants deletions SHRV genome, region-             pMCS-NV-(ORF&#226;&#171;&#186; &#226;&#171;&#185;37), pMCS-NV-(ORF&#226;&#171;&#186; &#226;&#171;&#185;55). Finally, plasmids <br /> specific ExSite (Stratagene, La Jolla, Calif.) PCR-based site-directed mu-            digested HpaI/SacI, fragments containing altered NV <br /> tagenesis performed. PCR-based mutagenesis strategy uses                  gene cloned pMJ-SHRV create final clones pMJ- <br /> creased template concentration reduced cycling number decrease                 SHRV-(NV ORF&#226;&#171;&#186;), pMJ-SHRV-(NV ORF&#226;&#171;&#186; &#226;&#171;&#185;37), pMJ-SHRV-(NV <br /> potential second-site mutations PCR amplification. following           ORF&#226;&#171;&#186; &#226;&#171;&#185;55), respectively. sequences clones determined pMJ-SHRV cDNA clones mutations NV gene created:                      sequence analysis using primers 653 580. <br /> pMJ-SHRV-(NV-ORF&#226;&#171;&#186;) missing NV ORF nucleotide 4893                         Mutants modifications SHRV glycoprotein gene created. <br /> 5265, pMJ-SHRV-(NV-ORF&#226;&#171;&#186; &#226;&#171;&#185;37) missing NV ORF having                  genomic structures engineered mutants shown Fig. 1B. additional 37-bp deletion G/NV junction region nucleotide              complete SHRV G gene replaced IHNV G gene pMJ-SHRV- <br />  VOL. 78, 2004                                                                            GENETIC MANIPULATION FISH RHABDOVIRUS                                      5877 <br />  <br />  <br />  <br />  <br />   FIG. 1. Diagram recombinant SHRV genomes. orders genes wild-type SHRV recombinant viruses shown. genes transcribed left right negative-strand RNA genome, shown 3&#226;&#172;&#732; 5&#226;&#172;&#732; orientation. Numbers indicate nucleotide positions antigenomic sequence SHRV. Big boxes, coding regions; small boxes, noncoding regions. left identification codes wild-type (wt) recombinant viruses. Open arrowheads vertical boxes, transcriptional start stop/polyade- <br /> nylation signals, respectively. Recovery (&#226;&#171;&#185;) nonrecovery (&#226;&#171;&#186;) recombinant virus corresponding cDNA genomic clone indicated.   organizations length SHRV cDNA NV mutants compared. (B) Comparison structures pMJ- <br /> SHRV-(IHNV G) pMJ-SHRV-(IHNV G/SHRV G hybrid) mutants. Gc, cytoplasmic domain glycoprotein. <br />  <br />  <br /> (IHNV G) cDNA expression vector. Mutant pMJ-SHRV-(IHNV G/SHRV G                            virus 40 promoter confer neomycin resistance. Stably transfected cells hybrid) engineered encode hybrid SHRV-IHNV glycoprotein contain-                  selected culturing presence neomycin G-418 (800 &#226;?&#174;g/ml). ing extracellular transmembrane domains IHNV envelope protein                   period growth, cloned neomycin-resistant population cells isolated <br /> fused cytoplasmic domain SHRV G. construct mutants,                    limiting dilution passage cells 96 plate. Individual colonies <br /> 4.5-kb SplI/SacII fragment pMJ-SHRV containing G gene cloned               grown presence neomycin G-418 subsequently passed pMCS, ExSite PCR-based site-directed mutagenesis performed                    larger culture dishes. NeoR T7pol cells tested T7 polymerase <br /> following manufacturer&#226;&#8364;&#8482;s instructions. Primer pairs 719-721 719-720           activity described Johnson et al. (11). Luciferase expression driven T7 <br /> added mutagenesis reaction mixture generate pMCS-(IHNV G)                    polymerase promoter NeoR T7pol cells comparable better pMCS-(IHNV G/SHRV G hybrid), respectively. PCR conditions                      cells transfected pCMV-T7pol. <br /> follows: 1 cycle 30 s 94&#194;&#176;C, followed 11 cycles 30 s 94&#194;&#176;C, 30 s 58&#194;&#176;C,        Transfection experiments recovery SHRV recombinants cloned 5 min 68&#194;&#176;C. coding sequence IHNV G gene portion                cDNAs. Recoveries carried described previously (11). Approximately G gene encoding amino-terminal domain G protein amplified               106 EPC cells overexpressing T7 RNA polymerase grown overnight PCR pcDNA3-G (1) primer pair 738-740 738-739, respectively.                plates transfected using Lipofectamine Plus reagent   <br /> Primer 738 located ectodomain-coding region, primer 739 overlapped               vitrogen) 0.5 &#226;?&#174;g length cDNA constructs encoding wild-type junction transmembrane cytoplasmic tail coding regions,                mutant SHRV genomes, 0.25, 0.5, 0.2 &#226;?&#174;g support plasmids primer 740 located coding sequence C-terminal end             encoding N, L, P proteins, respectively. Transfected cells incubated IHNV G protein. Primer 738 contained NheI site, primers 740 739               29&#194;&#176;C CPE evident. Cells culture supernatants subjected NotI restriction sites. PCR products cloned PCR 2.1 TA              cycles freezing thawing, clarified centrifugation 1,000 &#226;&#171;&#187; g cloning kit (Invitrogen, Carlsbad, Calif.) verified sequence analysis. Sub-         microcentrifuge (Micromase; Thermo IEC), used inoculate fresh cell <br /> sequently, PCR products digested NheI NotI inserted               monolayers. site G deletions pMCS-(IHNV G) pMCS-(IHNV G/SHRV G                          RNA purification. Total RNA extracted infected EPC cells hybrid), respectively. resulting plasmids digested SplI/SacII,           RNAzol according supplier&#226;&#8364;&#8482;s instructions (Invitrogen). fragments obtained cloned pMJ-SHRV, previously                     Confirmation virus rescue cloned cDNA RT-PCR. RNA digested SplI/SacII. resulting length cDNA plasmids des-                passage presumed recombinant SHRV collected described ignated pMJ-SHRV-(IHNV G) pMJ-SHRV-(IHNV G/SHRV G hybrid).                             analyzed reverse transcriptase PCR (RT-PCR) demonstrate pres- <br /> Primers 722 744, flanking SHRV G gene, used verify                    ence targeted mutations. RT-PCR performed Strand <br /> constructs.                                                                                cDNA synthesis kit following manufacturer&#226;&#8364;&#8482;s recommended protocol <br />    DNA sequence analysis. DNA sequence analyses performed                  (Fermentas, Hanover, Md.). Briefly, 20-&#226;?&#174;l reverse transcription reaction mix- <br /> Oregon State University Center Gene Research Biotechnology Central                 ture containing 5 &#226;?&#174;g total RNA, 0.2 &#226;?&#174;g random hexamers, 2 &#226;?&#174;l 10 mM <br /> Services Facility 3730 DNA sequencer (Dupont Applied Biosystems, Bos-                 dNTP mixture, 20 U RNase inhibitor, 4 &#226;?&#174;l 5&#226;&#171;&#187; Moloney murine leukemia <br /> ton, Mass.).                                                                               virus (M-MuLV) RT buffer (250 mM Tris-HCl [pH 8.3], 250 mM KCl, 20 mM <br />    Generation stably transfected EPC cell line expressing T7 RNA polymer-             MgCl2, 50 mM dithiothreitol), 40 U M-MuLV RT incubated 10 <br /> ase. EPC cells transfected pcDNA3-T7, plasmid capable express-              min 25&#194;&#176;C, 60 min 37&#194;&#176;C, 10 min 70&#194;&#176;C. Following reverse transcription, <br /> ing T7 RNA polymerase control cytomegalovirus immediate               1 &#226;?&#174;l cDNA used template subsequent PCR amplifica- <br /> early promoter, Lipofectamine Plus reagent (Invitrogen). Plasmid pcDNA3-              tions. PCR mixture contained 5 &#226;?&#174;l 10&#226;&#171;&#187; PCR buffer, 50 pmol specific <br /> T7 contains neomycin phosphotransferase gene control simian             sense antisense primers, 10 mM dNTP, 3 &#226;?&#174;l 25 mM MgCl2, 2.5 U  5878        ALONSO ET AL.                                                                                                                          J. VIROL. <br />  <br /> Taq DNA polymerase total volume 50 &#226;?&#174;l. Sequences primers used RT-PCR assays shown Table 1. PCR amplifications performed parameters: 30 cycles PCR consisting 10 s 95&#194;&#176;C, <br /> 1 min 55&#194;&#176;C, 3 min 72&#194;&#176;C. Extension times varied depending PCR <br /> mixture size. PCR products analyzed agarose gels visualized ethidium bromide. <br />    Plaque assays. Plaque assays used determine titers wild-type recombinant viruses accordance method Burke Mulcahy (4). 48 h incubation allow SHRV plaque formation, monolayers fixed, stained crystal violet, photographed. <br />    Virus purification electron microscopy. NV deletion mutant pMJ- <br /> SHRV-(NV ORF&#226;&#171;&#186;) wild-type SHRV grown EPC cells MOI 0.01 complete destruction respective monolayers. low-speed <br /> centrifugation (1,000 &#226;&#171;&#187; g) 4&#194;&#176;C, virus clarified culture fluid purified sucrose gradient centrifugation described Engelking Leong (7). <br /> Purified virus resuspended TNE buffer (0.1 M Tris-HCl [pH 7.6], 0.1 M <br /> NaCl, 0.001 M EDTA) layered electron microscopy grid. grids stained phosphotungstic acid 1 min examined trans-                FIG. 2. RT-PCR analysis pMJ-SHRV, pMJ-SHRV-(NV single <br /> mission electron microscope (Philips).                                                  stop), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) recombinant viruses. Total RNA <br />    Growth curves tissue culture. EPC cells grown 25-cm2-high flasks          EPC cells infected wild-type SHRV (lane 1) infected recombinant virus pMJ-SHRV, pMJ-SHRV-(NV single stop),                recombinant viruses pMJ-SHRV (lane 2), pMJ-SHRV-(NV single pMJ-SHRV-(NV ORF&#226;&#171;&#186;) wild-type SHRV MOI 0.001.                     stop) (lane 3), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) (lane 4) isolated 2 1 h adsorption 30&#194;&#176;C, cells washed remove unadsorbed virus.          days p  used RT-PCR amplifications NV region.  2-h intervals 14 h 24 h, flasks removed titers        plification products separated 1% agarose gel. pairs non-cell-associated total virus assayed. Cell-free virus titer deter-      primers used PCRs: primer pairs 653-580   554-531 <br /> mined supernatant flask following low-speed centrifugation.             (B). Lane M, DNA size markers. Note faint band right <br /> Total virus determined supernatant homogenized cells            lane 4 artifact. <br /> centrifugation 1,000 &#226;&#171;&#187; g microcentrifuge (Micromase; Thermo IEC). cells homogenized cycles freezing thawing. Samples frozen &#226;&#171;&#186;70&#194;&#176;C, end 24-h experiment viral titer                                                                                         pcDNA3-L, pcDNA3-M1. Viable recombinant viruses <br /> determined tissue culture end point dilution assay (50% tissue culture infec- <br /> tive dose [TCID50]/ml) (17).                                                            recovered reproducibly cells transfected    Fish challenge. Adult Ab strain zebra fish (Danio rerio) intraperitoneally      unmodified cDNA copy SHRV genome (pMJ-SHRV), <br /> injected recombinant virus pMJ-SHRV, pMJ-SHRV-(NV single stop),                cDNA copy SHRV genome stop codon pMJ-SHRV-(NV ORF&#226;&#171;&#186;) wild-type SHRV. Triplicate sets 20                 NV gene [pMJ-SHRV-(NV single stop)], cDNA clone <br /> fish anesthetized immersion 50 &#226;?&#174;g 3-aminobenzoic acid ethyl ester <br /> (MS-222; Sigma, St. Louis, Mo.)/ml 1 2 min immediately injected                                                                                         NV gene deleted [pMJ-SHRV-(NV ORF&#226;&#171;&#186;)]. <br /> 10 &#226;?&#174;l virus stock (105 PFU/ml). Control fish treated identically           CPE observed 48 h transfection, non- injected equivalent volume Dulbecco&#226;&#8364;&#8482;s phos-            transfected cells complete monolayers. super- <br /> phate-buffered saline instead infectious virus. fish kept 2.5-liter      natant fluids collected time, diluted 1:100 MEM <br /> tanks flowthrough supplied 28&#194;&#176;C nonchlorinated pathogen- <br />                                                                                         10% fetal bovine serum, placed fresh monolayer. <br /> free water experiment. Mortalities recorded daily 21 <br /> days postchallenge. Contingency tables fish groups replicate tanks          48 h, cells overlaid diluted supernatant estimated chi-square analysis. analyses, differences considered         initial experiment underwent cytopathic changes characteris- <br /> statistically significant correlation coefficient value (P)      tic SHRV lytic infection. CPE observed cells <br /> 0.05. Fig. 5, standard deviation included little   transfected pMJ-SHRV-(NV ORF&#226;&#171;&#186; &#226;&#171;&#185;37) pMJ- <br /> variation replicates. <br />    Nucleotide sequence accession number. length cDNA clone SHRV <br />                                                                                         SHRV-(NV ORF&#226;&#171;&#186; &#226;&#171;&#185;55) clones, contained deletions <br /> RNA (pMJ-SHRV) assigned GenBank accession  AF147498.                        G/NV gene junction NV gene. Trans- <br />                                                                                         fection experiments constructs                                                                                         triplicate,  replicate, recombinant infectious <br />                                    RESULTS <br />                                                                                         viruses recovered. result suggested G/NV <br />    Construction rescue SHRV deletion mutant lacking                            junction region required virus replication. NV gene. previously described recovery recom-                                Identification RT-PCR recombinant virus lacking binant viruses length cDNA copy SHRV                                 NV gene ORF. CPE observed, necessary genome (pMJ-SHRV) (11). study, cells infected                              demonstrate recovered viruses pMJ-SHRV, pMJ- recombinant vaccinia virus (vTF7-3) expressing T7                              SHRV-(NV single stop), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) RNA polymerase transfected T7-driven expres-                              fact recombinants. RNA recombinant viruses <br /> sion plasmids containing positive-sense copy SHRV                              amplified RT-PCR primers flanking intro- <br /> genome genes encoding N, L, P proteins.                           duced mutations, sequence product  present study, used stably transfected helper cell                          firmed direct sequencing. Results RT-PCR assays line constitutively expressing T7 RNA polymerase drive                               shown Fig. 2. primers flanking NV coding se- <br /> expression T7 promoter plasmids. approach                               quence (653 580) used RT-PCR, band 727 <br /> mitted recover recombinant virus contaminating                            bp obtained RNA cells infected wild-type <br /> vaccinia virus, CPE observed transfected cells                           virus pMJ-SHRV-(NV single stop) MJ-SHRV-(NV likely result virus rescue.                                                  ORF&#226;&#171;&#186;) recombinant virus (Fig. 2A). RNA cells <br />    viral rescue, T7 RNA polymerase-expressing EPC cells                             infected pMJ-SHRV-(NV ORF&#226;&#171;&#186;) yielded DNA frag- cotransfected plasmid DNA encoding wild-                               ment 356 bp, size expected deletion mutant. <br /> type mutant SHRV cDNAs plasmids pcDNA3-N,                                   second RT-PCR assay using NV gene ORF primer <br />  VOL. 78, 2004                                                          GENETIC MANIPULATION FISH RHABDOVIRUS                             5879 <br />  <br />  <br />  <br />  <br />    FIG. 3. Growth curves wild-type SHRV (wt) pMJ-SHRV, pMJ-SHRV-(NV single stop), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) recombinant <br /> viruses. EPC cells infected wt SHRV recombinant viruses indicated.   infectivity titer cell-free virus determined supernatant infected cells intervals 0 14 h 24 h infection. (B) indicated times p , infectivity titer total virus determined cells infected wild-type recombinant virus. <br />  <br />  <br /> 531 L gene ORF primer 554 performed.                       size appearance, indicating biological activ- wild-type, pMJ-SHRV, pMJ-SHRV-(NV single stop)                       ity recombinant viruses tissue culture cells sub- <br /> mRNAs RT-PCR produced fragments 2,060 bp.                         stantially altered. assess alterations ge- <br /> contrast, amplification detected NV deletion                 nome SHRV effect viral morphology, wild-type <br /> mutant (Fig. 2B). Sequencing PCR products confirmed                     deletion mutant virus particles purified, negatively original engineered cDNA sequences rescued                 stained, examined electron microscopy. obvious genomes pMJ-SHRV, pMJ-SHRV-(NV single stop),                         morphological differences NV deletion mutant, pMJ-SHRV-(NV ORF&#226;&#171;&#186;) recombinant viruses.                                 pMJ-SHRV-(NV ORF&#226;&#171;&#186;), wild-type virus ob- <br />    Deletion NV gene ORF did affect replication            served (data shown). recombinant virus tissue culture cells. determined                   NV gene ORF deletion role pathogenesis deletion NV gene affect growth rate                SHRV zebra fish. Zebra fish infected recom- SHRV comparing mutant wild-type viruses growth                 binant viruses wild-type SHRV. challenges  <br /> experiments. studies, quantities free              formed triplicate groups 20 fish intraperitoneal <br /> cell-associated virus determined rhabdovi-                injection 105 PFU recombinant virus pMJ-SHRV, <br /> ruses bud intracytoplasmic vacuoles maturation.                 pMJ-SHRV-(NV single stop), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) EPC cells infected wild-type recombinant viruses               wild-type SHRV/ml. Cumulative percent mortalities (CPM) MOI 0.001. unabsorbed virus removed, virus                shown Fig. 5. data represent replicate <br /> growth monitored end point dilution assay. growth                groups combined averages independent ex- <br /> curves shown Fig. 3. wild-type recombinant               periments. Animals injected pMJ-SHRV, pMJ-SHRV- <br /> viruses replicated efficiency, reaching similar               (NV single stop), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) recombinant <br /> infectivity titers 108 TCID50/ml 24 h postinfection             virus showed similar CPM (55%) 21 days postchallenge. <br /> (p ). viruses tested, concentration cell-free          Fish injected wild-type SHRV exhibited CPM virus total virus 14-          66.4%. Statistical analysis indicated differ- <br /> 16-h period reached plateau similar total virus           ence virulence wild-type SHRV concentration 24 h. infectivity titers recombinant            recombinant SHRV strains. <br /> viruses determined plaque assay. recombi-                     Recovery SHRV recombinant viruses expressing nant viruses pMJ-SHRV, pMJ-SHRV-(NV single stop),                       IHNV glycoprotein hybrid protein composed SHRV pMJ-SHRV-(NV ORF&#226;&#171;&#186;) produced infectivity titers 4.2 &#226;&#171;&#187;                     IHNV sequences. analyze heterologous chimeric <br /> 108, 6.8 &#226;&#171;&#187; 108, 3.6 &#226;&#171;&#187; 108 PFU/ml, respectively. wild-               Novirhabdovirus G proteins assemble function type virus yielded similar final infectivity titer (7.2 &#226;&#171;&#187; 108             SHRV-like particles, SHRV cDNA clones  <br /> PFU/ml).                                                                    structed. plasmid pMJ-SHRV-(IHNV G) expresses    Deletion NV gene ORF did affect plaque                complete IHNV G protein instead SHRV glycoprotein. <br /> virus morphology. EPC cells infected recombi-                 recombinant plasmid pMJ-SHRV-(IHNV G/SHRV G hy- <br /> nant virus pMJ-SHRV, pMJ-SHRV-(NV single stop), pMJ-                     brid) engineered encode hybrid IHNV-SHRV glyco- <br /> SHRV-(NV ORF&#226;&#171;&#186;) wild-type SHRV incu-                         protein containing extracellular transmembrane  <br /> bated 48 h. cells stained crystal violet,            mains IHNV envelope protein fused cytoplasmic plaque sizes morphologies compared (Fig.                   domain SHRV G. reverse genetics, recombinant viruses <br /> 4). plaques viruses approximately             rescued cDNA clones. genomic structures <br />  5880     ALONSO ET AL.                                                                                                              J. VIROL. <br />  <br />  <br />  <br />  <br />   FIG. 4. Viral plaques wild-type SHRV   pMJ-SHRV (B), pMJ-SHRV-(NV single stop) (C), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) <br /> (D) recovered viruses. Plaque assays used determine titers wild-type recombinant viruses. EPC cells infected wild-type SHRV   pMJ-SHRV (B), pMJ-SHRV-(NV single stop) (C), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) (D) recovered viruses. 48 h incubation 30&#194;&#176;C allow SHRV plaque formation, monolayers fixed, stained crystal violet, photographed. <br />  <br />  recombinant viruses shown Fig. 1B. IHNV                DNA fragments 1,592 bp obtained RNA cells <br /> glycoprotein able replace SHRV G steps              infected wild-type IHNV recombinant viruses pMJ- <br /> virus life cycle, demonstrated successful rescue            SHRV-(IHNV G) pMJ-SHRV-(IHNV G/SHRV G hy- <br /> infectious viral particles. finding autonomously repli-           brid) (Fig. 6A). contrast, amplification obtained <br /> cating virus pMJ-SHRV-(IHNV G) indicated                  RNA cells infected parental SHRV, demon- cytoplasmic domain SHRV G protein required             strating specificity primer pair 450-742 IHNV expression foreign proteins. identity              glycoprotein. Primer pair 450-451, spanning entire coding <br /> recombinant viruses pMJ-SHRV-(IHNV G) pMJ-SHRV-                      region IHNV G gene, used RT-PCR <br /> (IHNV G/SHRV G hybrid) verified RT-PCR                    amplification. band 1,706 bp just amplified cells <br /> analyses sequencing PCR products. Results               infected wild-type IHNV recombinant virus pMJ- RT-PCR assays shown Fig. 6. primer pair                  SHRV-(IHNV G), confirming presence different cy- <br /> 450-742 spanning extracellular transmembrane                  toplasmic domain glycoprotein pMJ-SHRV-(IHNV <br /> mains IHNV glycoprotein used RT-PCRs,                  G/SHRV G hybrid) (Fig. 6B). Finally, specific primer pair  <br />  <br />  <br />  <br />   FIG. 5. CPM zebra fish injected wild-type SHRV (wt) pMJ-SHRV, pMJ-SHRV-(NV single stop), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) <br /> recovered viruses. Triplicate sets 20 adult zebra fish infected intraperitoneally 105 PFU recombinant virus pMJ-SHRV, pMJ- <br /> SHRV-(NV single stop), pMJ-SHRV-(NV ORF&#226;&#171;&#186;) wild-type SHRV/ml. Mortalities recorded daily 21 days postchallenge. <br />  VOL. 78, 2004                                                            GENETIC MANIPULATION FISH RHABDOVIRUS                       5881 <br />  <br />  <br />                                                                            host virus,  tests conducted,                                                                            conclude NV protein apparent function vitro <br />                                                                            vivo. <br />                                                                               rhabdoviruses relatively small genomes, gen- <br />                                                                            erally 12 kb, little genome coding capacity                                                                            wasted. reason, reasonable assume NV <br />                                                                            gene retained genus critical function.                                                                            analyze G/NV gene junction sig- <br />                                                                            nificant role transcription process, constructed                                                                            SHRV cDNA clones alterations size NV <br />                                                                            mRNA [pMJ-SHRV-(NV ORF&#226;&#171;&#186; &#226;&#171;&#185;37) pMJ-SHRV- <br />                                                                            (ORF&#226;&#171;&#186; &#226;&#171;&#185;55)]. clones presented deletion                                                                            NV ORF 37-bp deletion G/NV gene <br />                                                                            junction including NV transcription initiation signal.                                                                            complete deletion G/NV gene junction                                                                            NV gene engineered pMJ-SHRV-(NV ORF&#226;&#171;&#186; &#226;&#171;&#185;55). <br />                                                                            EPC cells overexpressing T7 RNA polymerase <br />                                                                            transfected pMJ-SHRV-(NV ORF&#226;&#171;&#186; &#226;&#171;&#185;37) pMJ- <br />    FIG. 6. Confirmation pMJ-SHRV-(IHNV G) pMJ-SHRV-                 SHRV-(NV ORF&#226;&#171;&#186; &#226;&#171;&#185;55), recombinants recovered. <br /> (IHNV G/SHRV G hybrid) recombinant viruses rescue cloned              results illustrate importance regulatory sequences <br /> cDNA RT-PCR. RT-PCR performed described Materials <br />                                                                            present nontranslated regions genome Methods primer pair 450-742  , 450-451 (B), 749-750 <br /> (C). Amplification products separated 1% agarose gel.            recognition viral RNA viral polymerases. com- <br /> Lanes: M, DNA molecular weight markers; lane 1, wild-type IHNV;            plete deletion NV gene tolerated, deletions lane 2, wild-type SHRV; lane 3, pMJ-SHRV-(IHNV G) recombinant              G/NV junction change abundance upstream <br /> virus; lane 4, pMJ-SHRV-(IHNV G/SHRV G hybrid) recombinant                 downstream mRNAs render virus <br /> virus; lane 5, plasmid controls pCMV-G  B) pMJ-SHRV <br /> (C). RT-PCR amplifications carried 5 &#226;?&#174;g total infected-     completely defective. speculate NV gene cell RNA.                                                                  spacing function regulating transcription trans- <br />                                                                            lation, additional support Northern blot analyses                                                                            required test hypothesis. SHRV G gene, 749-750, used RT-PCR                          reverse genetics allowed determine wheth- <br /> assay. Amplification primer pair obtained               er viral genes replaced SHRV genome. cells infected wild-type SHRV, indicating                SHRV glycoprotein gene replaced IHNV <br /> presence foreign G gene genomes pMJ-SHRV-                   glycoprotein gene pMJ-SHRV-(IHNV G) (IHNV G) pMJ-SHRV-(IHNV G/SHRV G hybrid)                            IHNV/SHRV hybrid glycoprotein gene pMJ-SHRV-(IHNV <br /> combinant viruses (Fig. 6C). Sequence analysis rescued              G/SHRV G hybrid). hybrid construct encoded ectodo- <br /> virus confirmed genomes contained alterations                 main IHNV glycoprotein fused SHRV cytoplas- <br /> introduced respective cDNA clones. CPE ob-                    mic domain. successful recovery SHRV recombinant <br /> served cells infected pMJ-SHRV-(IHNV G) pMJ-                    viruses plasmids demonstrates SHRV cy- <br /> SHRV-(IHNV G/SHRV G hybrid) recombinant virus 24 h infection shown Fig. 7. Complete destruction cell monolayer achieved 24 h infection wild- <br /> type SHRV, cells infected pMJ-SHRV-(IHNV <br /> G) pMJ-SHRV-(IHNV G/SHRV G hybrid) recombinant <br /> virus developed extensive CPE. <br />  <br />  <br />                           DISCUSSION <br />   recent development systems allow directed ge- <br /> netic manipulation (reverse genetics) fish rhabdovirus ability recover length virus minivirus particles <br /> entirely cloned cDNAs facilitated molecular <br /> genetic analyses viral proteins cis regulatory elements <br /> (2, 3, 11). NV gene present known novirhabdovi- <br /> ruses, function unknown. report continuing efforts determine NV function                 FIG. 7. CPE observed 24 h infection EPC cells pMJ- pathogenesis SHRV fish cells fish. clear            SHRV-(IHNV G)  , pMJ-SHRV-(IHNV G/SHRV G hybrid) (B), recombinant virus containing deletions NV gene                 wild-type SHRV (C) viruses. (D) Uninfected EPC cells. Levels                                                                            growth wild-type SHRV pMJ-SHRV-(IHNV G) pMJ- <br /> coding region behave differently parental virus tis-            SHRV-(IHNV G/SHRV G hybrid) recombinant viruses cultured <br /> sue culture zebra fish. studies require          EPC cells compared infection 30&#194;&#176;C MOI 0.1. CPE <br /> comparing pathogenesis profiles snakehead fish, normal              observed 24 h infection. <br />  5882     ALONSO ET AL.                                                                                                                        J. VIROL. <br />  <br />  <br /> toplasmic domain required incorporation foreign                                           REFERENCES <br /> proteins. generation infectious particles         1. Anderson, E. D., D. V. Mourich, S. C. Fahrenkrug, S. LaPatra, J. Shepherd, <br />                                                                          J.  Leong. 1996. Genetic immunization rainbow trout (Oncorhyn- <br /> taining foreign glycoproteins achieved, recombinant              chus mykiss) infectious hematopoietic necrosis virus. Mol. Mar. Biol. <br /> viruses containing hybrid IHNV glycoprotein genes repli-              Biotechnol. 5:114&#226;&#8364;&#8220;122. <br /> cated cell cultures efficiently wild-type SHRV.      2. Biacchesi, S., M.  Thoulouze, M. Bearzotti, Y. X. Yu, M. Bremont. <br />                                                                          2000. Recovery NV knockout infectious hematopoietic necrosis virus 30&#194;&#176;C, infectivity titer wild-type SHRV 108             expressing foreign genes. J. Virol. 74:11247&#226;&#8364;&#8220;11253. <br /> TCID50/ml 3 days p ,  20&#194;&#176;C, infectivity titer    3. Biacchesi, S., M. Bearzotti, E. Bouguyon, M. Bremont. 2002. Heterol- <br />                                                                          ogous exchanges glycoprotein matrix protein Novirhab- reached wild-type IHNV 15 days p  contrast,           dovirus. J. Virol. 76:2881&#226;&#8364;&#8220;2899. <br /> infectivity titers recombinant viruses 103 TCID50/        4. Burke, J. , D. Mulcahy. 1980. Plaquing procedure infectious  <br /> ml 15 days p  30&#194;&#176;C. difference optimal growth             matopoietic necrosis virus. Appl. Environ. Microbiol. 39:872&#226;&#8364;&#8220;876. <br />                                                                       5. Cain, K. D., S. E. LaPatra, B. Shewmaker, J. Jones, K. M. Byrne, S. S. <br /> temperatures viruses result conforma-             Ristow. 1999. Immunogenicity recombinant infectious hematopoietic <br /> tional instability IHNV glycoprotein 30&#194;&#176;C reported          necrosis virus glycoprotein produced insect cells. Dis. Aquat. Org. 36:67&#226;&#8364;&#8220; Cain et al. (5).                                                      72. <br />                                                                       6. Chiou, P. P., C. H. Kim, P. Ormonde, J.  Leong. 2000. Infectious <br />    requirements successful assembly viral glycopro-           hematopoietic necrosis virus matrix protein inhibits host-directed gene ex- <br /> teins membrane envelope turned different           pression induces morphological changes apoptosis cell cultures. <br />                                                                          J. Virol. 74:7619&#226;&#8364;&#8220;7627. rhabdoviruses. viruses, assembly thought           7. Engelking, H. M., J. C. Leong. 1989. glycoprotein infectious <br /> require specific interactions cytoplasmic tail        hematopoietic necrosis virus elicits neutralizing antibody protective  <br /> viral glycoprotein internal viral components gathered             sponses. Virus Res. 13:213&#226;&#8364;&#8220;230. <br />                                                                       8. Fijan, N. 1983. properties Epithelioma papulosum cyprini (EPC) cytoplasmic plasma membrane. example,                cell line common carp Cyprinus carpio. Ann. Virol. 134E:207&#226;&#8364;&#8220;220. <br /> insertion foreign glycoproteins requires conservation       9. Fuerst, T. R., E. G. Niles, F. W. Studier, B. Moss. 1986. Eukaryotic <br />                                                                          transient-expression based recombinant vaccinia virus syn- <br /> rabies virus glycoprotein cytoplasmic domain (15, 16).            thesizes bacteriophage T7 RNA polymerase. Proc. Natl. Acad. Sci. USA <br /> trast, rescue recombinant vesicular stomatitis virus              83:8122&#226;&#8364;&#8220;8126. <br /> (VSV) mutants lacking genetic information         10. Hedrick, R. P., W. D. Eaton, J. L. Fryer, W. G. Groberg, S. Boonyarat- <br />                                                                          palin. 1986. Characteristics birnavirus isolated cultured sand goby <br /> cytoplasmic domain entire G protein suggested            Oxyeleotris marmoratus. Dis. Aquat. Org. 1:219&#226;&#8364;&#8220;225. <br /> cytoplasmic domain VSV envelope protein             11. Johnson, M. C., B. E. Simon, C. H. Kim, J.  Leong. 2000. Production <br /> quired assembly virus particles (18, 20). Substitutions         recombinant snakehead rhabdovirus: NV protein required                                                                          viral replication. J. Virol. 74:2343&#226;&#8364;&#8220;2350. transmembrane cytoplasmic domains VSV              12. Kasornchandra, J., H. M. Engelking, C. N. Lannan, J. S. Rohovec, J. L. <br /> glycoprotein did affect incorporation CD4            Fryer. 1992. Characteristics rhabdoviruses snakehead fish <br />                                                                          Ophicephalus striatus. Dis. Aquat. Org. 13:89&#226;&#8364;&#8220;94. <br /> VSV envelope (19), CD4 protein efficiently           13. Kurath, G., K. G. Ahern, G. D. Pearson, J. C. Leong. 1985. Molecular <br /> incorporated rabies virus G cytoplasmic domain             cloning mRNA species infectious hematopoietic necrosis virus, present (15). Recently, recombinant IHNV                  fish rhabdovirus, gene order determination R-loop mapping. J. Vi- <br />                                                                          rol. 53:469&#226;&#8364;&#8220;476. <br /> hemorrhagic septicemia virus envelope produced (3).          14. Kurath, G., J. C. Leong. 1985. Characterization infectious hemato- <br /> results confirm results cytoplasmic domain         poietic necrosis virus mRNA species reveals nonvirion rhabdovirus protein. <br /> SHRV glycoprotein required assembly             J. Virol. 53:462&#226;&#8364;&#8220;468. <br />                                                                      15. Mebatsion, T., K. K. Conzelmann. 1996. Specific infection CD4&#226;&#171;&#185; <br /> foreign proteins Novirhabdovirus envelopes.                       target cells recombinant rabies virus pseudotypes carrying HIV-1 <br />    incorporation foreign viral spike glycoproteins           envelope spike protein. Proc. Natl. Acad. Sci. USA 93:11366&#226;&#8364;&#8220;11370. <br />                                                                      16. Mebatsion, T., M. J. Schnell, J. H. Cox, S. Finke, K. K. Conzelmann. envelope recombinant SHRV provides way develop               1996. Highly stable expression foreign gene rabies virus vectors. <br /> new vaccines fish viral diseases.  critical residues        Proc. Natl. Acad. Sci. USA 93:7310&#226;&#8364;&#8220;7314. glycoprotein ectodomain required cell infecti <br /> </body></html>