<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Letter <br />  <br />  <br /> Evolution Topology Yeast Protein <br /> Interaction Network <br /> Stefan Wuchty <br /> Department Physics, University Notre Dame, Notre Dame, Indiana 46556, USA <br />  <br />  <br />        integrity yeast protein&#226;&#8364;&#8220;protein interaction network maintained highly connected proteins,        hubs, hold numerous connected proteins  structural importance increased <br />        essentiality proteins suggest likely conserved evolution, implying strong relationship <br />        number interactions evolutionary distance orthologs organisms.        existence coherence recently reported strongly depend quality protein interaction        orthologs data.  introduce novel method, evolutionary excess retention (ER), allowing uncover        robust strong correlation conservation, essentiality, connectivity yeast protein.        conclude relevance hubs network integrity simultaneously reflected considerable <br />        probability simultaneously evolutionarily conserved essential, observation does        equivalent nonessential proteins. Providing thorough assessment impact noisy incomplete data        findings, conclude results largely insensitive quality utilized data. <br />        [Supplemental material available online www.genome.org.] <br />  recently uncovered scale-free topology protein&#226;&#8364;&#8220;protein            served strongly correlated, trend does interaction networks focused attention important          equivalent nonessential proteins. addition, role small subset highly linked proteins, hubs,          provide thorough assessment impact quality guarantee functional structural integrity network         used data sets ability method determine <br /> (Jeong et al. 2001; Park et al. 2001; Wagner 2001). observed         trends, conclude findings largely  <br /> correlations essentiality connectivity pro-         sensitive incomplete noisy protein interaction tein (Jeong et al. 2001; Wuchty 2002) suggest hubs likely       ortholog data. <br /> maintained evolution, implying emergence correla- <br /> tions number interactions protein            RESULTS DISCUSSION <br /> evolutionary conservation (Hurst Smith 1999). trend           build source ortholog data, browsed protein clusters reported Saccharomyces cerevisiae, concluding          compiled InParanoid database (Remm et al. 2001), proteins higher connectivity smaller evolutionary          provides sequence information orthologous protein pairs  <br /> distance orthologs Caenorhabditis elegans (Fraser et al.     tween S. cerevisiae higher eukaryotes: Homo sapiens, Mus <br /> 2002).  recent reanalyses (Jordan et al. 2002, 2003a,b)          musculus, C. elegans, Drosophila melanogaster, Arabidopsis <br /> alternative approaches (Hahn et al. 2002) rigorously questioned          thaliana. connectivity 3677 yeast proteins inferred strength available evidence, concluding absence        Web 11,249 interactions assembled DIP da- <br /> distinctive correlations connectivity evolution-         tabase (Xenarios et al. 2002). quality interaction <br /> ary conservation proteins. recent reply objections         ortholog data matter debate (Koski Golding <br /> (Fraser et al. 2003) argued utilization incomplete       2001, Goldberg Roth 2003; Wall et al. 2003), did noisy sets protein interactions, inaccurately deter-       perform quality checks, allowing assess ability mined orthologs, reasons asserted absence       method uncover assumed trends putatively flawed <br /> trends. carefully compiling interactions known            data. Using data sets, scatterplot Figure 1A, depicting <br /> high-throughput screens proteins S. cerevisiae (Uetz et al.        evolutionary distance D  Methods) yeast protein 2000; Ito et al. 2001; Gavin et al. 2002; Ho et al. 2002)            function connectivity k, fails indicate convincing corre- <br /> determining evolutionary distances orthologs Schizo-         lations. indicated low average values saccharomyces pombe Candida candidans novel phyloge-            Pearson&#226;&#8364;&#8482;s correlation coefficient, &#194;&#175;r = &#226;&#179;&#174;0.045, P&#194;&#175; = 0.137, netic estimation method (Wall et al. 2003), highly disputed          Spearman&#226;&#8364;&#8482;s rank coefficient &#226;?&#179;&#194;&#175; = &#226;&#179;&#174;0.055, P    &#194;&#175; = 0.092  detailed <br /> correlations ultimately                                       values, Supplemental material). small average slope &#226;?&#163;&#194;&#175; <br />       results statistically significant, sensi-     = &#226;&#179;&#174;0.0072 best linear fit D &#226;&#710;&#188; &#226;?&#163;k supports previous  <br /> tivity data sources remains considerable weakness        clusions dependence evolutionary distance D introduced method.  contribute current debate          protein connectivity k negligible (Jordan et al. 2003a,b; presenting alternative approach, allowing uncover sig-        detailed values, Supplemental material).  ob- <br /> nificant strong correlation highly interacting proteins      served correlations determined enhanced <br /> S. cerevisiae far higher probability evolutionarily         level accounting scale-free nature protein net- <br /> conserved higher eukaryotes. Observing propensity         work.  frequency highly interacting proteins decays <br /> essential, highly connected proteins evolutionarily            power-law (Jeong et al. 2001;Wagner 2001) P(k) &#226;&#710;&#188; k&#226;&#179;&#174;&#226;?&#165;, indi- <br />                                                                          cating sparsely connected proteins significantly outnumber <br />                                                                          highly linked counterparts. uniform sampling used E-MAIL swuchty@nd.edu; FAX (574) 631-5952. <br /> Article publication http://www.genome.org/cgi/doi/10.1101/    Figure 1A does account differences number <br /> gr.2300204.                                                              proteins different connectivity groups. guarantee <br />  <br /> 1310     Genome Research                             14:1310&#226;&#8364;&#8220;1314 &#194;&#169;2004 Cold Spring Harbor Laboratory Press ISSN 1088-9051/04; www.genome.org <br />             www.genome.org <br />                                                                               Evolution Topology Yeast Protein Network <br />  <br />  <br />  <br />  <br /> Figure 1   Scatterplot showing yeast protein evolutionary distance D orthologs higher eukaryote function proteins&#226;&#8364;&#8482; <br /> number interactions k. black line corresponds D &#226;&#710;&#188; &#226;?&#163;k &#226;?&#163;&#194;&#175; = &#226;&#179;&#174;0.0072. (B) applying logarithmic binning horizontal k-axis, obtain improved correlations excess sequence conservation ESCk k (&#226;&#338;&#169;ESCk&#226;&#338;&#170; &#226;&#710;&#188; &#226;?&#164;log k, &#226;?&#164;        &#194;&#175; = &#226;&#179;&#174;0.152). (C) orthologous excess retention <br /> parameter ERk shown function logarithmically binned ks indicates statistically significant monotonic trend conservation connected proteins (ERk &#226;&#710;&#188; &#226;?&#165;log k, &#226;?&#165;&#194;&#175; = 0.813). plot, symbols correspond orthologs S. cerevisiae identified higher organisms <br /> listed legends box C. <br />  <br />  <br />  <br /> balanced sampling k-values, use logarithmic binning              teins groups logarithmically binning connectivity k, k-axis, procedure corrects skewed nature             determine respective fraction number proteins <br /> scale-free distribution  Methods; Albert et al. 2000; Goldstein            having orthologs reference organisms total num- <br /> et al. 2004). Providing comparable numbers proteins                 ber proteins bin. null hypothesis, assume bin, determine mean excess sequence conservation, &#226;&#338;&#169;ESCk&#226;&#338;&#170;                random distribution orthologs quantified frac- <br />  Methods). distributions obtained (Fig. 1B) indicate              tion total number proteins orthologs statistically significant correlations &#226;&#338;&#169;ESCk&#226;&#338;&#170; k (average           reference eukaryote total number proteins present Pearson&#226;&#8364;&#8482;s &#194;&#175;r = &#226;&#179;&#174;0.775, P&#194;&#175; = 2.6 &#226;&#179;&#175; 10 &#226;&#179;&#174;3 ; average Spearman&#226;&#8364;&#8482;s                   interaction network  Methods). Defined orthologous <br /> &#226;?&#179;&#194;&#175; = &#226;&#179;&#174;0.720, P&#194;&#175; = 0.022), best fit logarithmic curve,                  excess retention ERk, ratio k-dependent random <br /> &#226;&#338;&#169;ESCk&#226;&#338;&#170; &#226;&#710;&#188; &#226;?&#164;log k, &#226;?&#164;&#194;&#175; = &#226;&#179;&#174;0.152. fits indicate stronger            fractions orthologous proteins shows visually striking correla- <br /> dependence, result agree earlier conclu-              tions k (Fig. 1C). results supported sions weak interdependence connectivity                  distinct statistically significant average Pearson&#226;&#8364;&#8482;s correlation <br /> protein sequence conservation (Hahn et al. 2002; Jordan                coefficients &#194;&#175;r = 0.892 (P&#194;&#175; = 2 &#226;&#179;&#175; 10&#226;&#179;&#174;3) Spearman&#226;&#8364;&#8482;s rank corre- <br /> et al, 2003a).                                                                 lation coefficients &#226;?&#179;&#194;&#175; = 0.993 (P &#194;&#175; = 4 &#226;&#179;&#175; 10&#226;&#179;&#174;3). linear depen- <br />        apparently weak correlation sequence                 dence log-linear plot indicates ERk &#226;&#710;&#188; &#226;?&#165;log k conservation connectivity does necessarily invalidate                  &#226;?&#165;&#194;&#175; = 0.813, representing clear trend evolutionary original hypothesis, assumed reflect data depen-             conservation connected proteins  detailed values, <br /> dence applied analytical framework (Fraser et al. 2003).                Supplemental material). probe initial assumption connected proteins                         Similarly, observe trend excess retention higher tendency evolutionarily conserved, intro-               essential proteins collected Bioknowledge <br /> duce alternative approach separately identifying frac-               library (Fig. 2A; Costanzo et al. 2001). expected, observe tions orthologous proteins k interactions. Dividing pro-               dilution process nonessential proteins increasing levels  <br />                                                                                                                      Genome Research              1311 <br />                                                                                                                         www.genome.org <br />  Wuchty <br />  <br />  <br />  <br />  <br /> Figure 2   essential nonessential excess retention function logarithmically binned connectivities proteins shows statistically <br /> significant monotonic trends (ERk &#226;&#710;&#188; &#226;?&#165;log k, essential: &#226;?&#165; = 1.386, nonessential: &#226;?&#165; = &#226;&#179;&#174;0.371). (B) trend excess retention essential proteins enhanced focus proteins simultaneously orthologs higher eukaryotes (&#226;?&#165;      &#194;&#175; = 2.128). (C)  proteins orthologs reference eukaryotes dispensable cell&#226;&#8364;&#8482;s survival fail significant excess retention signal. <br />  <br />  <br />  interactions. cases, application logarithmic bin-             comparable correlation proteins nonessential <br />  ning uncovers sharp logarithmic trends, ERk &#226;&#710;&#188; &#226;?&#165;log k  detailed           evolutionarily conserved (Fig. 2C). <br />  values, Supplemental material), supported dis-                   estimate potential influence incomplete noisy <br />  tinctive statistically significant Pearson&#226;&#8364;&#8482;s correlation coeffi-         protein interaction data findings, added (removed)  cients (essential proteins: r = 0.868, P = 4.5 &#226;&#179;&#175; 10&#226;&#179;&#174;4, nonessential          20% interactions randomly selected protein pairs, <br />  proteins: r = &#226;&#179;&#174;0.848, P = 9.5 &#226;&#179;&#175; 10&#226;&#179;&#174;4) Spearman&#226;&#8364;&#8482;s rank coef-              mimicking false positives (negatives). Similarly, simu- <br />  ficients (essential proteins: &#226;?&#179; = 0.976, P = 9.8 &#226;&#179;&#175; 10&#226;&#179;&#174;3, nonessen-           lating presence false-positive (negative) ortholog  tial proteins: &#226;?&#179; = 0.868, P = 4.5 &#226;&#179;&#175; 10&#226;&#179;&#174;3; detailed values,           essential (nonessential) signals, randomly increased   <br />  Supplemental material). Furthermore, excess                 creased) original sets respective proteins 20%.  retention orthologous essential proteins indepen-             case, generated 1000 different realizations removal <br />  dent: Figure 2B shows strong logarithmic, ER k &#226;&#710;&#188; &#226;?&#165;log k,                   (addition) repeated assessment essential/nonessential <br />  &#226;?&#165;&#194;&#175; = 2.128, statistically significant trend (average Pearson&#226;&#8364;&#8482;s           orthologous excess retention. basic <br /> &#194;&#175;r = 0.909, P&#194;&#175; = 7.1 &#226;&#179;&#175; 10 &#226;&#179;&#174;4 , average Spearman&#226;&#8364;&#8482;s &#226;?&#179;&#194;&#175; = 0.971,                 trends remain qualitatively unaltered, albeit slopes  P&#194;&#175; = 8.8 &#226;&#179;&#175; 10&#226;&#179;&#174;3) highly linked proteins simul-              actual curves gradually change, allowing conclude  taneously essential evolutionarily conserved. particular,             uncovered correlations largely unaffected data incom- <br />  proteins interactions excess retention            pleteness  details, Supplemental material). <br />  smaller (ERk &lt; 1), indicating high fraction                summary, results clearly indicate highly  <br />  numerous weakly connected proteins discounted                    nected proteins far likely essential (simulta- <br />  evolution. Compared respective numbers simple                 neously) conserved orthologs higher eukaryotes  thologous excess retention, significantly higher excess reten-           connected counterparts. Focusing D-inde- <br />  tion proteins &gt;10 links (ERk&gt;10 &gt; 1) suggests            pendent measure orthologous excess retention ERk  proteins preferentially conserved higher eukaryotes               correcting scale-free statistics applying logarithmic <br />  essential survival cell.             binning, reduced noise level uncovered significant <br />  <br /> 1312      Genome Research <br />             www.genome.org <br />                                                                         Evolution Topology Yeast Protein Network <br />  <br />  <br /> trend connectivity evolutionary conservation             assumption sequences species underlying data sets. earlier approaches determin-         similar main ortholog sequence ing trends severely impaired used data,          species paralogs belong group  results suggest method widely insensitive        thologs. quality resulting orthologous clusters ex- <br />                                                                         amined increased final bootstrap analysis. Furthermore, <br /> quality data sources, observation holds                                                                         InParanoid provides comprehensive pairwise comparative  <br /> data inconsistencies noise.                                         thologous information S. cerevisiae H. sapiens, D. melanogas- <br />       observations allow reappraisal conclusion       ter, C. elegans, M. musculus  thaliana. study, used dependence evolutionary distance D protein           core pairs cluster provide confidence level connectivity k negligible (Hahn et al. 2002; Jordan et al.           100%.  underlying interaction network, 2003a). logarithmic dependency ex-            1997 proteins orthologs H. sapiens, 1757 D.  <br /> cess retention proteins&#226;&#8364;&#8482; connectivity, ERk &#226;&#710;&#188; log k,       lanogaster, 1754 M. musculus, 1489 C. elegans 1898  <br /> sumption effect evolutionary distance D           thaliana. depend logarithmically k, appears reasonable. Provided effect connectivity k evolutionary reten-        Assignment (non-)Essentiality <br /> tion ERk deemed large, effect evolutionary distance <br />                                                                         Bioknowledge library (Costanzo et al. 2001) compiles scans <br /> D exemplified mean excess sequence conservation &#226;&#338;&#169;ESC&#226;&#338;&#170;         experimental literature provide comprehensive list es- orthologous sequences appears stronger previously          sential nonessential proteins. S. cerevisiae proteins <br /> described.  absence distinctive correlations        appearing interaction network, 810 assigned es- <br /> tween connectivity protein sequence conserva-          sential, 2704 considered nonessential. <br /> tion result incorrect assumption distributions follow linear instead logarithmic trend.              Evolutionary Distance <br />       importantly, uncovered correlations indicate                                                                         Assessing number substitutions site, define evolutionary conservation protein affected                                                                         evolutionary distance (Grishin 1997) D yeast protein protein&#226;&#8364;&#8482;s individual functional role topological    orthologous sequences reference eukaryote contextual placement cellular network. trends          q = [ln(1 + 2D)]/2D, q fraction unchanged sites enhanced adding essential information sets        sequence alignment protein pairs (Fraser et al. 2002, 2003; orthologous proteins. fact proteins essential       Jordan et al. 2003a,b). survivability cell evolutionarily  <br /> served contradiction.  protein core <br />                                                                         Excess Sequence Conservation ensures cells survival strongly privileged evolution.                                                              guarantee balanced sampling k-values, grouped                                                                         proteins bins logarithmically increasing connectivity k.                                                                         bin, determine mean excess sequence conservation, <br /> METHODS                                                                                                  1 <br />                                                                                                               Nk <br />                                                                                                                       &#229;&#8230;&#183;D&#229;&#8230;&#184; &#226;&#710;&#8217; Di <br /> Protein Interactions <br />                                                                                               &#229;&#8230;&#183;ESCk&#229;&#8230;&#184; = <br />                                                                                                          Nk   &#229;&#8230;&#186; <br />                                                                                                                       &#229;&#8230;&#183;D&#229;&#8230;&#184; <br />                                                                                                                               , <br />  <br /> Large-scale hybrid screens allow identification         Nk number proteins respective bin, potential protein&#226;&#8364;&#8220;protein interactions open reading <br /> frames S. cerevisiae genomic sequence (Uetz et al. 2000;                                                 N <br />                                                                                                           1 <br /> Ito et al. 2000, 2001) integral proteomics research.  quality hybrid data significantly affected                                                                                                   &#229;&#8230;&#183;D&#229;&#8230;&#184; = <br />                                                                                                           N   &#229;&#8230;&#186;D                                                                                                                            <br />  <br /> high rates false positives false negatives (von Mering et al.    mean evolutionary distance total number N pro- <br /> 2003), indicated fact results ob-        teins. <br /> tained different groups limited overlap (Hazbun Fields 2001).  identified interactions merely rely positive signals single technique result indi-       Excess Retention <br /> rect observations. study, used database interact-      According degree k underlying yeast protein  <br /> ing proteins (DIP), based extensive literature searches          teraction network, grouped proteins bins logarithmi- <br /> aims, provide curated collection functional link-      cally increasing connectivity k. bin, ratio eA                                                                                                                                        k = nk /Nk <br /> ages proteins obtained experimental methods. majority         represents certain feature  nAk number proteins protein&#226;&#8364;&#8220;protein interaction data relies yeast hybrid          (e.g., essential orthologous reference coimmunoprecipitation experiments. Eighty percent              organism), Nk total number proteins. absence interactions detected single experiment,         correlation position network, eA k 16% confirmed experimental                 general k-independent value e = n/N, n = &#226;&#338;&#186;knA     k method. DIP currently records 3677 proteins involved        total number yeast proteins having feature  N = &#226;&#338;&#186;k Nk 11,249 interactions (Xenarios et al. 2002).                             total number yeast proteins underlying network. <br />                                                                          bin k, define evolutionary excess retention <br /> Assignment Orthology                                                 feature ERA                                                                                                k = ek /e, k-independent determination orthologous pairs sequences em-           value ERk = 1 random assignment  <br /> ploys pairwise BLAST comparisons proteomes. protein represents query entire proteome species. Reciprocal best hits BLAST searches, em-        Logarithmic Binning <br /> phasizing expectation values &lt;10&#226;&#179;&#174;3, considered             guarantee balanced sampling k-values, use logarith- <br /> thologous. choice orthologous protein sequence informa-          mic binning k-axis, procedure curve estimation tion InParanoid database (Remm et al. 2001), runs          corrects skewed nature scale-free distribution. versus BLAST searches sets sequences. Se-           logarithmic scale, define bin size <br />  <br />                                                                                                                       &#229;&#8224;&#8240;&#229;&#8224;&#352; <br /> quence pairs mutual way best hits detected serve central core ortholog pairs                                             1     b <br />                                                                                                   &#226;&#338;&#172;=       log   , <br /> thologs species clustered later steps. initial                                      N      <br />                                                                                                                Genome Research          1313 <br />                                                                                                                        www.genome.org <br />  Wuchty <br />  <br />  N corresponds selected number bins. Values              Hazbun, T. Fields, S. 2001. Networking proteins yeast. Proc. Natl. <br /> b refer minimal maximal value connectivity ki,                  Acad. Sci. 98: 4277&#226;&#8364;&#8220;4278. <br /> b = maxi (ki) = mini (ki).                                         Ho, Y., Gruhler, , Heilbut, , Bader, G.D., Moore, L., Adams, S.L., <br />                                                                                   Millar, , Taylor, P., Bennett, K., Boutilier, K., et al. 2002. <br />  <br />                     ni = log   &#229;&#8224;&#8240;&#229;&#8224;&#352; <br />                                ki <br />                                                                  &#226;&#178;? &#226;&#338;&#172;, ni &#226;&#710;&#710; &#229;&#8230;&#179;0, N &#226;&#710;&#8217; 1&#229;&#8230;&#180; <br />                                                                                   Systematic identification protein complexes Saccharomyces <br />                                                                                   cerevisiae mass spectrometry. Nature 415: 180&#226;&#8364;&#8220;183. <br />                                                                               Hurst, L. Smith, N. 1999. essential genes evolve slowly? Curr. <br />                                                                                   Biol. 9: 747&#226;&#8364;&#8220;750. <br /> reflects number bin assign protein ki inter-             Ito, T., Tashiro, K., Muta, S., Ozawa, R., Chiba, T., Nishizawa, M., <br /> actions. Representing nith bin k-axis, place kni                 Yamamoto, K., Kuhara, S., Sakaki, Y. 2000. end bin using kni = e&#226;&#338;&#172;(ni+1). advantage loga-                protein&#226;&#8364;&#8220;protein interaction map budding yeast: rithmic binning elevated degree noise reduction                  comprehensive examine hybrid interactions dependent bin size (Albert et al. 2000; Goldstein et al.                   possible combinations yeast proteins. Proc. Natl. Acad. <br /> 2004). procedure causes loss accuracy,                Sci. 97: 1143&#226;&#8364;&#8220;1147. <br /> uncover buried trends satisfying extent applying              Ito, T., Chiba, T., Ozawa, R., Yoshida, M., Hattori, M., Sakaki, Y. <br />                                                                                   2001. comprehensive hybrid analysis explore yeast <br /> statistical methods binned data. <br />                                                                                   protein interactome. Proc. Natl. Acad. Sci. 98: 4569&#226;&#8364;&#8220;4574. <br />                                                                               Jeong, H., Mason, S., Barab&#195;&#161;si, -L., Ol <br /> </body></html>