<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization Putative Spindle Assembly <br /> Checkpoint Kinase Mps1, Suggests Involvement Cell Division, Morphogenesis Oxidative Stress <br /> Tolerance Candida albicans <br /> Mohan Kamthan1., Vijaya Kumar Nalla2., Deepa Ruhela1, Ayushi Kamthan1, Protiti Maiti1, Asis Datta1* <br /> 1 National Institute Plant Genome Research, New Delhi, India, 2 Department Microbiology, Bhaskaracharya college, University Delhi, New Delhi, India <br />  <br />  <br />  <br />      Abstract <br />      Saccharomyces cerevisiae MPS1 major protein kinase governs spindle checkpoint pathway. S. <br />      cerevisiae structural homolog opportunistic pathogen Candida albicans CaMPS1, indispensable cell viability.      essentiality Mps1 confirmed Homozygote Trisome test. determine biological function pathogen <br />      conditional mutant generated regulatable MET3 promoter. Examination heterozygous conditional (+ <br />      Met/Cys) mps1 mutants revealed mitosis specific arrest phenotype, mutants showed large buds undivided <br />      nuclei. Flowcytometry analysis revealed abnormal ploidy levels mps1mutant. presence anti-microtubule drug <br />      Nocodazole, mps1 mutant showed dramatic loss viability suggesting role Mps1 Spindle Assembly Checkpoint <br />      (SAC) activation. mutants defective microtubule organization.  heterozygous mutant showed <br />      defective vitro yeast hyphae morphological transition. Growth defect heterozygous mutant suggest <br />      haploinsufficiency gene. qRT PCR analysis showed 3 fold upregulation MPS1 presence serum.      expression MPS1 dependent Efg1and independent hyphal regulators like Ras1 Tpk2. Furthermore, <br />      mps1 mutants sensitive oxidative stress. Heterozygous mps1 mutant did undergo morphological transition <br />      showed 5-Fold reduction colony forming units response macrophage.  vital checkpoint kinase, Mps1 <br />      cell division role morphogenesis oxidative stress tolerance, pathogenic fungus. <br />  <br />   Citation: Kamthan M, Nalla VK, Ruhela D, Kamthan  Maiti P, et al. (2014) Characterization Putative Spindle Assembly Checkpoint Kinase Mps1, Suggests   Involvement Cell Division, Morphogenesis Oxidative Stress Tolerance Candida albicans. PLoS 9(7): e101517. doi:10.1371/journal.pone.0101517 <br />   Editor: Yanchang Wang, Florida State University, United States America <br />   Received February 5, 2014; Accepted June 9, 2014; Published July 15, 2014 <br />   Copyright: &#195;&#376; 2014 Kamthan et al. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported Department Biotechnology, Ministry Science Technology, Government India. funders role <br />   study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * Email: asis_datta@rediffmail.com <br />   . authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                          centrosomes mammals) coordinated fashion response                                                                                       various cell division cues. S. cerevisiae, seven genes    Candida albicans, close phylogenetic relative Saccharomyces                   identified function governing spindle checkpoint <br /> cerevisiae, major fungal pathogen humans systemic                       pathway; MAD1-3 [4], BUB1-3 [5], MPS1 [6]. infections fatal immunocompromised patients [1].                         studies Bub2 revealed prevents unusual multinucle- <br /> Pathogens like C. albicans evolved various counteractive                         ate cells misoriented spindles exiting mitosis.  cellular mechanisms evade host defenses. Despite,                            checkpoint, components strategies damage key cellular components like DNA                         regulate exit mitosis [41]. Cells mutation protein pathogen occurs repaired survival.                   genes fail arrest cell cycle response spindle defects <br /> Cell cycle checkpoints coordinate DNA integrity proper                        undergo aberrant, lethal mitosis. study MAD2 showed <br /> chromosomal segregation cell division essential                   indispensable role gene morphogenesis viability cell viability. Precise control chromosomal segregation carried                 C. albicans murine mouse model [2]. non-essential gene, BUB2 transient cytoskeletal structure termed mitotic                       controls pre-anaphase arrest polarization pseudohy- <br /> spindle. checkpoint senses defects attachment chromo-                       phal-like cells [3]. budding yeast homolog CENP  <br /> somes mitotic spindle preventing chromosomal loss                       CaCSE4, shown play role proper chromosome <br /> stopping chromosome segregation. components checkpoint                         segregation growth C. albicans [7]. <br /> pathways drug targets, especially cancer research.                       master protein kinase check point pathway  C. albicans, Spindle Assembly Checkpoint pathway plays                          MPS1 (Mono Polar Spindle Kinase) [6] (mutants gene form <br /> important role cell division [2,3]. nuclear division,                       monopolar spindles), kinase activity required mitotic spindle forms bipolar structure nucleation                      activating members checkpoint machinery. Homologs spindle microtubules spindle pole body (equivalent                          MPS1 gene identified organisms play <br />  <br />  <br /> PLOS | www.plosone.org                                                        1                                 July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                 Role MPS1 Candida albicans <br />  <br />  <br /> diverse roles checkpoint activation, spindle pole body duplica-         accession number obtained (AY632356.1). allele <br /> tion, chromosome segregation, mitotic arrest, response           disruption MPS1 performed using URA blaster method <br /> hypoxic conditions. budding yeast, MPS1 essential dual            [23] CAF3-1 strain (Methods S1). Ura+ heterozygous <br /> specificity (serine/threonine tyrosine) protein kinase involved        mutant (MFD2) used Ura-curing process, SD+Uri agar normal progression cell cycle [6,8]. humans higher            plates supplemented 59-fluoroorotic acid (5-FOA) (Sigma <br /> expression MPS1 detected human                     Aldrich) 1 mg/ml concentration. Curing confirmed neoplasms, including thyroid, breast lung cancers [9&#226;&#8364;&#8220;12].              southern blot analysis ura2 starin (MFD2-U1) MPS1 considered promising drug target cancer              selected. single transformation based gene function test [17] cells. central role protein kinase cell division,        used confirm essentiality MPS1 (Methods S1). inhibitors MPS1 reported [13&#226;&#8364;&#8220;15]. C.               heterozygous MFD2-U1 strain used construc- <br /> albicans, orf19.7293 MPS1 homolog S. cerevisiae.        tion MET3 regulated conditional mutant, MCM4. create inhibitor CaMps1 known [16] biological function       MET3-MPS1 cassette, 590 bp fragment MPS1 coding region, unknown. report characterized structural           comprising positions +1 +590 relation ATG start site, <br /> homologue budding yeast MPS1 gene proved                        PCR amplified using oligonucleotides, MpBamHI:59TT- <br /> indispensable role survival, cell division, morphogenesis           GACGGATCCATGCCACATGA TTTATTATC39 reverse <br /> oxidative stress tolerance human pathogenic yeast, C. albicans.         primer, MPSphI: 59GCATGCATCTTTCTAAC GACG39, having <br />                                                                            BamHI SphI sites (italicized bases) directional cloning. Materials Methods                                                      fragment cloned pGEMT-easy vector resulted                                                                            plasmid pCaMPF (Table 1). 590 bp fragment digested <br /> Mice                                                                       ligated BamHI SphI digested plasmid, pCaDis [24] <br />    Female Balb/c mouse weighing 18&#226;&#8364;&#8220;20 grams obtained               resulting 5,930 bp plasmid, pDMPS1 (Figure S2). plasmid <br /> pathogen-free mice Animal House Jawaharlal Nehru               linearized BsgI digestion transformed Ura2/2 <br /> University (JNU) New Delhi, INDIA. use Mice duly                MFD2-U1 strain lithium acetate method [25], isolate MET3 <br /> approved Institutional Animal Ethic committee (IAEC)             regulated strain; MCM4, SC agar plates lacking <span id='am-25' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-26' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Uridine" datatype="xsd:string"></span>Uridine</span>, <br /> Jawaharlal Nehru University (JNU), Registration  19/1999                Methionine Cysteine. Correct recombinants containing, <br /> (CPCSEA; Committee purpose control supervision              MET3 promoter confirmed southern blotting digesting experiments animals). Approval code VO/AH/IAEC/                  genomic DNA strains XbaI DraII (Figure S2). <br /> 84/53. housing experimental procedures conducted              Similarly, rescued strain MCM4R; MPS1 coding region guidelines JNU Animal Care.                               expressed promoter, created reintroducing <br />                                                                            gene non-essential RPS1 locus [26] using pCaExP vector <br /> Media Strains                                                          (Generously gifted Peter E. Sudbery) [24], contains    C. albicans strains used study listed Table 1. C.        URA3 marker. construction revertant cassette, MPS1 ORF <br /> albicans routinely grown 30uC YPD (1% yeast extract, 2%          500 bp promoter region (2573 bp) PCR amplified Bacto Peptone, 2% glucose) Sabourad&#226;&#8364;&#8482;s Dextrose (SD) media               genomic DNA SC5314 using oligonucleotides MPSalF: <br /> (0.67% Yeast Nitrogen Base, 2% Dextrose 2% agar  solid             59GTCGACTTAAATTGGTTAAAATTTTC39 MPSalR: <br /> media)). Synthetic Complete (SC) medium used condi-                59GTCGACATACACTAGCTGTAGTTT AAG39 di- <br /> tional mutant (0.67% nitrogen base amino acids, 2%                 gested SalI enzyme.  pCaExP vector digested glucose mixture amino acids) required 80 mg              SalI release 360 bp MET3 promoter fragment. resultant <br /> ml21 Uridine supplemented (Uri+). Transformants HT             5,895 bp pCaEXP-MET3 vector backbone ligated test grown synthetic defined media described Enloe           2,573 bp SalI digested MPS1 fragment creating 8,468 bp <br /> et al [17]. conditional mutant studies, 2.5 mM L-Cystiene              plasmid, pCaExMP. linearizing StuI digestion, (Cys) <span id='am-45' about='obo:IMR_0200488' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00073" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0200488" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="L-Methionine" datatype="xsd:string"></span>L-Methionine</span> (Met) externally added.                    construct used transforming MCM4 strain. Transformants <br /> morphogenetic studies, Spider (adjusted pH 7.3) SLAD                screened SC agar plates supplemented 2.5 mM <br /> media prepared described Lin et al [18] Gimeno et           Met/Cys, inhibited growth untransformed MCM4 <br /> al [19] respectively. Cells induced GlcNAc Biswas         cells. Correct integration pCaExMP MCM4R confirmed <br /> et al [20]. serum induction 10% bovine serum (Sigma)               Southern blot analysis SalI DraII digestion. added pre-warmed SD 37uC. G1 cell synchronization,               ectopic expression URA3 cause phenotypic effects, C.albicans cells cultured Carbon source deficient Emden-           morphological studies targeted reintegration URA3 Meyeroff-Mininmal Media (EMM media) 25uC overnight                      performed heterozygous MFD2-U1 strain using pCaEXP <br /> [21,22]. Chlamydospore formation obtained growing               vector neutral RPS1 locus. <br /> cells coverslips 25uC 7 days cornmeal/Tween agar <br /> (BD) plates.                                                               Southern blotting <br />                                                                               Southern blotting, genomic DNA extracted cells <br /> Cloning, Confirmation Essentiality generation                    grown YPD SD media. micrograms DNA conditional mutants                                                        digested Restriction endonuclease resolved Agarose <br />    entire 2073 base pairs (bp) gene amplified             gel electrophoresis 1% agarose gels, transferring [27] <br /> genomic DNA C.albicans (SC5314) 162 bp upstream              positively charged Nylon membranes (NEN Research 250 bp downstream start stop codon respectively, using          Products ,) capillary transfer. screening MFD2, <br /> oligonucleotides, MPS1F: 59CCTAGTGAGCACACACAT39and                         MFD2-U1,M-UAU M-AU transformants hybridizations MPS1R: 59GTGTTGGCAGG GTGATGC39 cloned                           performed 32P radiolabelled DNA probes, prepared pGEMT-easy vector (Promega) generate plasmid pGEM-                      NotI-digestion pGEM-MPS1 resulted ,2.5 Kb MPS1 <br /> MPS1. Following sequencing confirmation alignment              fragment. 0.597 Kbp fragment pDMPS1, Genome sequence information available Candida genome                digested BamHI <span id='am-6' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; SphI used screening MET3 <br /> database (CGD), length sequence deposited NCBI             transformants. <br />  <br />  <br /> PLOS | www.plosone.org                                             2                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                           Role MPS1 Candida albicans <br />  <br />  <br />  <br />  Table 1. Strains plasmids used study. <br />  <br />  <br />  Strain/Plasmid             Genotype/description                                                                             Source/reference <br />  <br />  Strains <br />  SC5314 (wild-type)         URA3/URA3 <br />  CAF3-1                     Dura3::imm434/Dura3::imm434                                                                      W Fonzi <br />  BWP17                      ura3D::limm434/ura3D::limm434his1::hisG/his1::hisG arg4::hisG/arg4::hisG                         Aaron P. Mitchell <br />  MFD2                       CAF3-1, MPS1/Dmps1::hisG-URA3-hisG                                                        Work <br />  MFD2-U1                    CAF3-1, MPS1/Dmps1::hisG                                                                  Work <br />  M-UAU 1&#226;&#8364;&#8220;30                 BWP17, MPS1/DCamps1::Ura3-ARG4-Ura3                                                       Work <br />  M-AU                       BWP17, Dmps1::URA3/Dmps1::Ura3-ARG4-Ura3/MPS1                                             Work <br />  MCM4,7                     MFD2-U1, Dmps1/pMetMPS1                                                                   Work <br />  MCM4R                      MCM4, pMetMPS1/RP10::MPS1                                                                 Work <br />  CAN52                      Dras1::hisG/Dras1::hph Dura3::ura3::imm434/Dura3::imm434                                         [42] <br />  HLC67                      Defg1::hisG/Defg1::hisG Dura3::imm434/Dura3::imm434                                              [43] <br />  AS1                        Dtpk2::hisG/Dtpk2::hisG Dura3::imm434/Dura3::imm434                                              [43] <br />  11-1-1-4 (Dcph1)         Dacpr1::hisG/Dacpr1::hisG Dura3::imm434/Dura3::imm434                                            Laboratory strain <br />  Plasmids <br />  pGEM-MPS1                  MPS1 ORF cloned pGEM-T Easy vector                                                            work <br />  pMP1                       Carrying Dmps1::hisG-URA3-hisG disruption fragment                                               work <br />  pUC19-CUB                  hisG-URA3-hisG disruption cassette cloned pUC19                                               [20] <br />  pBME101                    Carrying Ura39-ARG4-Ura39 cassette                                                               Aaron P. Mitcehll <br />  pCaDis                     Plasmid carrying MET3 promoter, URA3 marker                                                      P.E. Sudbery <br />  pCaEXP                     Plasmid carrying MET3 promoter URA3 marker RP10 gene                                         P.E. Sudbery <br />  pCaMPF                     0.5 Kb MPS1ORF BamHI SphI sites cloned pGEM-T                                        work <br />  pDMPS1                     pMPF pDis vector                                                               work <br />  pCaExMP                    pEXP 2.5 kb MPS1 ORF promoter                                                          work <br />  pMUAU                      Carrying Dmps1::Ura39-ARG4-Ura39 disruption fragment                                             work <br />  <br />  doi:10.1371/journal.pone.0101517.t001 <br />  <br />  <br />    qRT-PCR. Cells grown OD 0.8 SD media [28]                           Indirect Immunofluorescence induced hrs synthetic media 2% carbon                               Cells grown OD 0.6&#226;&#8364;&#8220;0.8 SC medium source. Total RNA isolated using Tripure reagent (Roche).                        37uC conditional mutants induced 2.5 mM Met/ <br /> cDNA synthesized total RNA (5 mg) using oligo dT                         Cys 4 hrs. Cells fixed 4% <span id='am-20' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span> 30 min <br /> primer (Invitrogen) Superscript reverse transcriptase                          room temperature washed times PBS (150 mM <br /> (Invitrogen). original mRNA template removed                            NaCl: 20 mM Na/K phosphate, pH 7.3). Cells resuspended RNA: DNA heteroduplex RNase H. cDNA samples                             PBS containing 1 M sorbitol (PBSS) 100 ml zymolyase used templates qRT-PCR 7900 Fast Real                          incubated 37uC 1 hr. Cells washed twice Time PCR (Applied Biosystems) using SDS 2.4 program.                          PBSS resuspended 100 ml PBS +1% BSA +0.1 M lysine 10 ml reaction mix included 2X KAPA SYBR FAST (KAPA                              cell added slides allowed stand BIOSYSTEMS), 0.5 ml cDNA 0.2 mM primers.                                  30 min. Cells permeabilized chilled methanol <br /> amplification program included Stage  50uC 2 min, Stage 2-                     (9 min) acetone (6 min), air-dried. rehydration 95uC 10 min, Stage 3- 40 cycles 95uC 15 sec 60uC                      PBS, slides incubated anti-tubulin antibodies 60 sec, Stage 4- 95uC 15 sec, 60uC 15 sec 95                     (Millipore, CBL270) diluted 1:200-1:400. Antibodies immu- <br /> 15 sec. Raw data analyzed using RQ manager 1.2.1 software                        nofluorescence diluted PBS, 1% BSA, antibody determine relative gene expression. ACT1 mRNA level used                      incubations carried 1 h 37uC. slides endogenous control. results represent average                    washed 20 min changes PBS, second <br /> biological replicate experiments performed triplicates.                      antibody FITC conjugated goat anti Rat IgG used 1:200 <br /> Error bars, exist represent standard error                    dilution. Finally, slides washed 20 min 3 separate biological replicates. following primers designed                     changes PBS mounted viewed confocal microscope. Primer 3 software used qPCR analysis. ACT1-F59 <br /> GACAATTTCTCTTTCAGCACTAGTAGTGA -39 ACT1-                                          Nocodazole Sensitivity Assay <br /> R59 GCTGGTAGAGA CTTGACCAACCA -39; MPS1-F59 CC-                                         Strains grown OD 0.6&#226;&#8364;&#8220;0.8 liquid SD media <br /> AGGCAGCAAAACATT 39MPS1-R59 TTTGCCGAACCTTC-                                           induced Nocodazole (50 mM/ml) (Sigma aldrich). <br /> TTT 39.                                                                              Aliquots treated cultures collected 2, 4 hr intervals. <br />                                                                                      Conditional mutants induced Nocodazole                                                                                      presence absence 2.5 mM Met/Cys. viability check, <br />  <br />  <br /> PLOS | www.plosone.org                                                       3                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                              Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 1. Multiple sequence alignment showing conserved regions Mps1p homologs.   Alignment N-terminal region Mps1, <br /> showing conserved Leucine residue. (B) Sequence alignment C-terminal homology CaMps1 (Orf19.7293/AY632356), Mps1 S. cerevisiae <br /> (YDL028C/P54199), Mph1 S. pombe (O94235), hMps1 Human (Q9BW51) mMps1 Mouse (P35761): ORF numbers obtained, <br /> accession numbers provided. subdomains shown Roman numerals  XI). Subdomains kinase domain shown described <br />  <br />  <br />  <br /> PLOS | www.plosone.org                                           4                            July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                         Role MPS1 Candida albicans <br />  <br />  <br />  [30]. conserved amino acids serine/threonine kinases (SER/THR) shown described [31], conserved amino acids shown bold <br /> letters. italicized E sub domain V SER/THR consensus, feature dual specificity kinases [31]. Valine (V) position <br /> 382 site insertional inactivation. conserved aa indicated double dot (:) perfectly conserved aa indicated asterisk (*) <br /> doi:10.1371/journal.pone.0101517.g001 <br />  <br />  <br />  <br />  <br /> cultures serially diluted equal numbers spreaded               hours. Washed PBS stained 50 mg/ml Propidium <br /> SD Agar plates incubated 30uC 2&#226;&#8364;&#8220;3 days. S-phase                  Iodide 4uC 8&#226;&#8364;&#8220;10 hrs. Stained cells diluted folds <br /> arrest, cultures treated 50 mM Hydroxy <span id='am-55' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-56' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Urea" datatype="xsd:string"></span>Urea</span> (USB                      briefly sonicated disperse aggregates formed fixation <br /> biochemicals) 30uC hours addition                      process. Approximately 1&#226;&#8364;&#8220;26104 cells analyzed BD FACS <br /> nocodazole. Cells collected plated described earlier                Calibur machine gated settings linear scale. Forward <br /> [2].                                                                            Scatter analysis (FSC) performed similar way. Histograms <br />                                                                                 processed analyzed Morphid WinMDI softwares, <br /> Cell Synchronization Flowcytometry                                          respectively. <br />    C. albicans cells prepared FACS analysis Nitrogen <br /> starvation based synchronization described [21]. Cells                  Morphogenetic studies mouse peritoneal cavity grown Edinburgh Minimal Medium (EMM) till OD 0.5.                      macrophages <br /> Cells inoculated EMM medium Ammonium                          Freund&#226;&#8364;&#8482;s complete adjuvant (0.4 ml) injected Chloride grown saturation 23uC vigorous aeration.                peritoneal cavity female Balb/c mouse, weighing 18&#226;&#8364;&#8220;20 grams. regular intervals, cells harvested inoculated SD              10 days injection, significant abdomen enlargement SC medium supplemented nocodazole hydroxy urea.                         observed large exudates peritoneal cavity. <br /> Fixation 70% chilled ethanol incubated                   26108 late log phase cells strain 200 ml PBS <br /> hour 25uC. fixation, cells harvested washed                   injected enlarged peritoneal cavity  mice twice 5X Tris-EDTA buffer (pH 8.0).  cells                    strain used). 24 hours injection, 100 ml treated 1 mg/ml RNaseA (Invitrogen) 37uC                      peritoneal exudate retrieved using 23G hypodermic needle <br />  <br />  <br />  <br />  <br /> Figure 2. Construction MET3 promoter regulated conditional mps1 mutant.   Illustration representing conditional mutant preparation. <br /> Plasmid pDMPS1was digested BsgI linearized cassette used transform URA3-cured MFD2 (MPS1/mps1D:hisG) mutants. Genetic <br /> structure MCM4 locus homologous recombination represented. (B) Southern blot analysis confirm promoter replacement. Genomic DNA strains digested DraII XbaI resulted 2.8 Kb 3 Kb bands MCM4 strain, 2.1 Kb band Wild type MCM4R strain. MFD2-U1 URA-cured allele mutant. Complete ORF gene used probe. (C) Spot dilution assay Check growth response conditional mutants absence (2) presence (+) Met/Cys containing Complete Synthetic medium (SC Agar). Mid log phase <br /> grown cultures serially diluted spotted. Plates incubated 30uC 2&#226;&#8364;&#8220;4 days. (D) qRT-PCR analysis compare expression MPS1 wild-type (WT), heterozygous (MFD2) conditional mutant. <br /> doi:10.1371/journal.pone.0101517.g002 <br />  <br />  <br /> PLOS | www.plosone.org                                                  5                               July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                               Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 3. Fluorescence flowcytometry analysis wild type (WT) mps1 mutant strains. studying defects nuclear <br /> division chromatin segregation mps1 mutants, cells stained DAPI 4 hours growth liquid SC medium 37uC. WT  , <br /> heterozygous mutant (MFD2) (B), conditional mutant &#226;&#8364;&#8220;Met/Cys (C) conditional mutant +Met/Cys (D). Arrows indicate defective chromosome <br /> segregation. flowcytometry analysis, chromatin stained Propidium Iodide, indicated time points. X-axis indicates DNA measured Y-axis represents number cells. Approximately, 104 cells strain analyzed FACS-Calibur machine FL2 channel. imaginary lines drawn represent changes position peaks 2 N 4 N content DNA, respectively. Scale bar shown <br /> is10 mm applicable images. <br /> doi:10.1371/journal.pone.0101517.g003 <br />  <br /> attached 1 ml syringe. Peritoneal exudates smeared                     YPD agar incubated 37uC 3 days determine microscopic glass slides stained described [2].                           C.albicans colony forming units (CFU). <br /> Photographs taken Leica digital camera attached microscope.10 ml peritoneal exudates added 190 ml                     <span id='am-50' about='obo:IMR_0200487' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-51' property="rdfs:label" content="H2O2" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:id" content="IMR:0200487" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00027" datatype="xsd:string"></span>H2O2</span> sensitivity <br /> sterile MQ lysing macrophages solution plated                      Cells grown upto OD 0.8 30uC SD media                                                                                     treated different concentrations H2O2 1 hour. Cells <br />  <br />  <br />  Table 2. Comparison wild type Dmps1 strains respect observed phenotypes DAPI staining. <br />  <br />  <br />                                                            Percentage                  Percentage dividing       Percentage mother <br />                                  Percentage non         dividing cells            cells normally          cell bud           cells <br />  Time       Strain               dividing cells            buds nuclei            dividing nucleus             nuclei                     counted <br />  <br />  2Hr        Wild Type            94                        1                              1                            4                          226 <br />             MFD2                 40                        38                             7                            15                         250 <br />             MCM+M/C              25                        55                             8                            12                         278 <br />             MCM-M/C              92                        2                              1                            5                          245 <br />  4Hr        Wild Type            83                        1                              2                            14                         291 <br />             MFD2                 27                        58                             6                            9                          282 <br />             MCM+M/C              21                        67                             8                            4                          274 <br />             MCM-M/C              76                        2                              3                            19                         265 <br />  <br />  Cells stained DAPI counted hemocytometer. Cells grown SD media 30uC indicated time points. Average percentages biological <br />  replicates shown table. <br />  doi:10.1371/journal.pone.0101517.t002 <br />  <br />  <br />  <br /> PLOS | www.plosone.org                                                      6                                 July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                 Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 4. Immunostaining anti  tubulin antibodies. Conditional mutant presence 2.5 mM Met/Cys showed formation mono polar <br /> spindles. Wild type (WT) conditional mutant absence Met/Cys showed proper bipolar spindles. White arrows indicate position spindles <br /> (Green fluorescence) strains. <br /> doi:10.1371/journal.pone.0101517.g004 <br />  spotted H2O2 free SD agar plates incubated             function CaMPS1 gene C. albicans, created 30uC days determine effect H2O2.                      knockout. MPS1 gene essential organisms <br />                                                                            including S.cerevisiae, MPS1 homolog (MPH1) Schizosacchar- <br /> Results                                                                    omyces pombe non-essential [33]. confirm essentiality                                                                            MPS1 gene C. albicans, adopted specially designed single silico analysis putative MPS1 C. albicans                         transformation based gene function test called Homozygote <br />    Genes MPS1 family highly conserved eukaryotes.             Trisome test. Southern blot analysis revealed independent <br /> Orthologs gene present majority organisms,           Arg+ Ura+ segregants mps1::UAU1/mps1::URA3/MPS1 exception Caenorhabditis elegans. cloned 2073 bp        triplication derivatives (Figure S2D).  presence allelic <br /> putative C.albicans MPS1 homolog S. cerevisiae, using gene          triplication confirmed MPS1 essential gene.  <br /> sequence provided Candida Genome Database (CGD) [29].                    created conditional mutants replacing promoter <br /> CaMps1 encodes protein 690 amino acids estimated              region MPS1, heterozygous MFD2 strain (mps1/MPS1), <br /> Mr 77.69 kDa. Clustal W multiple sequence alignment                   methionine (Met)/Cysteine (Cys) suppressible MET3 <br /> (http://www.ebi.ac.uk/) used determine homology              promoter (Figure 2A). positive strains, MCM4 MCM7 <br /> putative CaMps1, characterized Mps1 S. cerevisiae, S.        (Table 1) selected functional analysis promoter <br /> pombe, Mus musculas Human (Figure 1). CaMps1 shared                  replacement confirmed southern blot analysis (Figure 2B). <br /> significant homology 67% protein 54% DNA                      serial dilution spot assay showed MCM4 strain (Dmps1/ <br /> sequence level S. cerevisiae Mps1. Multiple sequence              pMetMPS1) did grow presence 2.5 mM Met/Cys <br /> alignment showed C- terminal region CaMps1             suggesting conditional inhibition growth strain conserved sub domains  amino acids 367 637)                 (Figure 2C). change observed growth MCM4 <br /> characteristic Serine/threonine kinase domain Mps1 [30]              strain absence Met/Cys (Figure 2C). expression levels (Figure S1A). CaMps1also tyrosine kinase specific                  MPS1 determined MET3 promoter <br /> signature, glutamine (E) residue sub domain V  position          (Figure 2D). qRT PCR analysis showed comparison 443) (Figure 1) [31]. kinase function Mps1 family known          native promoter, MPS1 expression little higher associated processes chromosome segregation,            influence MET3 promoter absence Met/Cys. presence <br /> genome stability vital cellular processes [32]. Phylogram        Met/Cys expression MPS1 observed. additional <br /> revealed  CaMps1 clustered D. hanseii&#226;&#8364;&#8482;s predicted Mps1           copy MPS1 promoter introduced MCM4 <br /> sequence closeness ScMps1 Candida glabrata Mps1             strain non essential RPS1 gene (Figure 2E). Correct <br /> (Figure S1B). ScMps2 ScMps3 formed group                    integration confirmed Southern blot analysis (Figure 2B). <br /> phylogram indicating evolutionary divergence compared            Growth rate studies liquid SD media showed conditional <br /> members Mps1 family kinases (Figure S1B). Apart                 mutant absence Met/Cys behaved like wild type strain, CaMps1, Rad53 (cell cycle checkpoint) Orf19.3459                    presence Met/Cys did growth. heterozygous <br /> ortholog S. cerevisiae Mck1p) dual specificity kinases        mutant (MFD2) showed delayed growth compared signatures C. albicans.                                                 wild type strain (Figure 2F). Growth revartant strain (MCM4R) <br />                                                                            having additional copy MPS1 neutral RPS1 locus Disruption CaMPS1 gene                                                  analyzed. significant change observed growth   Mps1 family protein kinase play role spindle checkpoint          MCM4R strain compared wild type. <br /> activation majority organisms studied. determining <br />  <br />  <br /> PLOS | www.plosone.org                                             7                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                   Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 5. Sensitivity MPS1 mutants nocodazole.   Exponential phase yeast cells treated 50 mM Nocodazole 30uC aliquots collected 2, 4 6 hours determine CFU Nocodazole free SD agar plates. (B) Cultures treated hydroxyurea 2 hrs addition Nocodazole; aliquots collected counting CFU. (C) Cells treated Nocodazole (50 mM/ml) 6 hrs plated Nocodazole free plates following washing serial dilution 2 days 30uC. D) Untreated control cells spotted SD agar <br /> plates 2 days 30uC. <br /> doi:10.1371/journal.pone.0101517.g005 <br />  <br />                                                                              Role MPS1 normal ploidy maintenance <br />                                                                                 Mps1 known master protein kinase,                                                                              phosphorylates downstream molecules spindle checkpoint <br />                                                                              pathway, like Mad1/Mad2 Cenp-E [32]. budding yeast, <br />                                                                              mutants genes displayed defects chromosome segrega- <br />                                                                              tion, distribution nuclei ploidy levels [2,34]. Fluorescence <br />                                                                              microscopy 49, 69-diamidino-29-phenylindole (DAPI) stained <br />                                                                              mps1 mutant strains, revealed 58% heterozy- <br />                                                                              gous cells (MFD2) large buds nuclei (Figure 3B) <br />                                                                              (Table 2). MCM4 strain absence Met/Cys behaved wild <br />                                                                              type  presence Met/Cys cells larger, average, <br />                                                                              cell nuclear size compared wild type                                                                              70% cells bud nuclei (Figure 3D) (Table 2)                                                                              suggested CaMps1 required proper chromosomal <br />                                                                              distribution mitosis. Growth defects single allele <br />                                                                              knockout indicates haploinsufficiency gene [35]. MAD2 <br />                                                                              gene C. albicans mammals haploinsufficiency, <br />                                                                              heterozygous mutants displayed premature anaphase                                                                              chromosome instability [2,36]. <br />                                                                                 investigate role Mps1in chromosome segregation,                                                                              quantitatively analyzed DNA content mutants Propidium <br />                                                                              Iodide staining based flowcytometry assay. inducing                                                                              MCM4 strain medium containing Met Cys, observed <br />                                                                              progressive shift DNA content, 2 N 4 N 8 N <br />                                                                              (Figure 3D) indicated accumulation DNA cell <br />                                                                              similarly treated control cells (Figure 3, E). <br />                                                                               MCM4 strain absence Met/Cys, continued                                                                              grow 2 N content DNA cell (Figure 3C),                                                                              prolonged growth (8 hrs) cells 4 N observed.  <br /> Figure 6. Polarized growth study nuclear dynamics                     fluorescence microscopy flowcytometery analysis MET3 <br /> nocodazole treated cells. Exponential phase cells indicated strains       conditional mutants, suggest role Mps1 normal mainte- treated Nocodazole (50 mM/ml) liquid SD media 30uC           nance ploidy genomic stability C. albicans. Analysis harvested indicated time points stained DAPI. <br /> doi:10.1371/journal.pone.0101517.g006                                        microtubule organization performed immuno- <br />  <br />  <br /> PLOS | www.plosone.org                                               8                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                  Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 7. Expression analysis MPS1.  qRT-PCR analysis determine expression MPS1 indicated hyphae inducing media. B) qRT-PCR determine regulation MPS1 expression presence serum. C) Promoter analysis MPS1 gene: 500 bp upstream sequence MPS1 analyzed using Regulatory Sequence Analysis tools (RSAT) server  http://rsat.ulb.ac rsat. Efg1p binding element 59CANNTG39 identified locations. <br /> doi:10.1371/journal.pone.0101517.g007 <br />  <br />  <br />  <br />  <br /> Figure 8. Morphology mps1 mutant strains different hyphae inducing conditions.   Morphogenetic studies solid inducing <br /> media: Cultures study grown mid log phase, serially diluted 25 50 cells plate spreaded indicated hyphae inducing <br /> media. (B) Morphologies respective liquid inducing media; yeast cells overnight cultures inoculated indicated inducing media 26105 cells/ml 3 hours. (C) Chlamydospore formation observed 7 days 25uC cornmeal/Tween agar plates coverslips. <br /> doi:10.1371/journal.pone.0101517.g008 <br />  <br />  <br /> PLOS | www.plosone.org                                              9                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                         Role MPS1 Candida albicans <br />  <br />  <br />                                                                                    passage cell cycle absence SAC <br />                                                                                    function, caused cell death mps1 mutants response                                                                                    Nocodazole. <br />                                                                                       Additionally, serial dilution spot assay indicated loss                                                                                    growth presence Nocodazole (Figure 5C) compared                                                                                    untreated control plate (Figure 5D). heterozygous                                                                                    conditional mutants (+Met/Cys) showed sensitivity Nocodazole <br />                                                                                    treatment (6 hr). observed absence                                                                                    Nocodazole Met/Cys treated conditional mutant showed loss <br />                                                                                    viability (Figure 5D). essential role Mps1 <br />                                                                                    cell division.  rapid fall viability Nocodazole <br />                                                                                    treated conditional mutant presence Met/Cys                                                                                    attributed dual action microtubule disruption                                                                                    suppression MPS1 expression. Conditional mutants absence <br />                                                                                    Met/Cys behaved wild-type. results demonstrate role <br />                                                                                    Mps1 activation SAC C.albicans. <br />                                                                                       morphogenetic response C. albicans checkpoint <br />                                                                                    inducing drugs like Nocodazole quite different                                                                                    budding yeast, S. cerevisiae yeast cells remain largely <br />                                                                                    budded multiple buds [37,38]. C.albicans, conditions <br /> Figure 9. Oxidative stress tolerance mps1 mutants.  Mutants                   arrest cell cycle progression result polarized growth cultured Liquid SD medium till mid log phase resuspended               phenotype [38,39]. determine morphological response, PBS (pH 7.4) containing indicated concentrations H2O2 treated            DAPI staining fluorescence microscopy performed 1 hour 37uC. Cells spotted serial dilution H2O2 free <br /> SC-Agar plates incubated 30uC 2 days. (B) 10 ml peritoneal <br />                                                                                    treating cells 50 mM nocodazole. observed exudates mice injected indicated strain added 190 ml                2 hrs drug treatment nearly wild type cells formed elongated <br /> sterile MQ plated YPD agar plates determine CFU. Error bars              buds, nuclei indicating cell cycle arrest (Figure 6). represents standard error biological replicates. (C)                 4 hr, wild type cells showed pseudo-hyphal like extensions, Morphology wild type (WT) heterozygous mps1 mutant strains                  single nucleus present mother cell elongated bud. injected peritoneal cavity mouse. Peritoneal exudates             heterozygous conditional mutant (MCM4) presence retrieved 24 hrs injection. C. albicans cells indicated strains <br /> engulfed macrophages stained PAS stain. Arrows indicate               Met/Cys showed buds different sizes suggesting absence cell <br /> C. albicans cells engulfed macrophages.                                         cycle arrest (Figure 6). Conditional mutant absence Met/Cys <br /> doi:10.1371/journal.pone.0101517.g009                                              behaved like wild type (Figure 6). 4 hours heterozygous                                                                                    conditional mutant cells showed irregular shape staining tubulin. Wild type conditional mutant (-Met/                    multinucleate (Figure 6).  Mps1 plays important role Cys)) cells displayed normal bipolar spindle organization,                     cell cycle checkpoint, involved regulating spindles appear originating opposite end                  Nocodazole induced polarized growth. cells (Figure 4). contrast, microtubules conditional <br /> mutant (+Met/Cys) present end cells, indicative                 Role Mps1 morphogenesis cells having monopolar spindles (Figure 4).  defective                        C. albicans polymorphic organism exhibits variety microtubule organization reason inaccurate chromo-                      cellular forms. ability organism switch somal segregation mps1 mutants determined DAPI                            distinct modes proliferation contributes virulence. <br /> staining.                                                                          investigated role Mps1 yeast hyphae <br />                                                                                    morphological transition C. albicans. qRT-PCR analysis showed <br /> MPS1 required activation spindle checkpoint                          higher expression MPS1 gene major filamentation inducing <br />                                                                                    conditions like, GlcNAc, spider serum compared pathway                                                                            Glucose containing media (Figure 7A). Maximum expression    Mps1 family protein kinases budding yeast                       MPS1 observed 10% Serum (3 folds) followed GlcNAc (2 <br /> organisms roles activation Spindle Assembly                         folds) compared glucose grown cells. <br /> Checkpoint (SAC) pathway spindle pole body duplication                            identify signaling network required expression [32]. determine loss SAC function mps1 mutants,                  MPS1 gene, qRT PCR analysis performed mutants heterozygous conditional mps1 mutant strains exposed                   major signal transduction pathways control morphogenesis. microtubule toxin, Nocodazole. Cells treated 50 mM                   analysis showed expression MPS1 unchanged <br /> Nocodazole 2, 4 6 hours harvested determine                         Dtpk2, Dcph1 Dras1 mutants, Defg1 mutant colony forming units (CFUs) SC-Met/Cys agar plates (Material                    expression suppressed basal level (Figure 7B).  methods). results showed increase time,                       expression MPS1 dependent <span id='am-31' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-32' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span>cAMP</span> mediated signaling <br /> conditional mutants presence Met/Cys, readily lost viability                 pathway Efg1, independent Tpk2 Ras1. presence Nocodazole compared untreated cells                           Computational analysis MPS1 promoter, showed Efg1 <br /> (Figure 5A).  treatment Nocodazole                            binding elements (CANNTG) 500 base pairs upstream hydroxy-urea treatment given arrest cells S-                    start codon, positions 246, 2181 2417 (Figure 7C). <br /> phase cell cycle inhibiting ribonucleotide reductase.                       Higher expression MPS1 major hyphae inducing condition, cell mortality minimized, ,95%                     conditions suggests role morphogenesis.  analyzed wild type ,85% heterozygous mutant cells retained                    morphological transition mps1 mutants various solid viability (Figure 5B). showed microtubule disruption                  liquid filamentation inducing media like Spider, SLAD, GlcNAc <br /> Nocodazole responsible cell death,                  10% Bovine Serum. heterozygous strain (MFD2) showed <br />  <br />  <br /> PLOS | www.plosone.org                                                    10                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                   Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> Figure 10. Model illustration possible regulatory mechanism activation MPS1 gene C. albicans, Efg1 dependent Ras1 independent manner, leads morphogenesis represented lined arrows. classical cAMP-PKA pathway included comparison. Role Mps1 SAC activation ploidy maintenance shown dotted arrows. model represents Efg1 conjunction <br /> point SAC machinery morphogenesis. <br /> doi:10.1371/journal.pone.0101517.g010 <br />  <br /> defective filamentation solid agar media tested             macrophages [2]. CaMps1is required activation (Figure 8A). conditional mutant (MCM4), absence Met/               spindle checkpoint pathway, analyzed involvement Cys behaved wild type strain (Figure 8A). Like solid               oxidative stress tolerance.  mps1 mutants treated media, liquid inducing media  heterozygous mutant cells              H2O2 (60 min) spotted H2O2 free SC agar plates, reduced response yeast hyphal transition            facilitate growth cells escaped free radical attack. <br /> prolonged incubation (Figure 8B). Morphologically, MFD2 cells                2 days incubation 30uC, mps1 mutant strains showed elongated, swollen formed pseudo hyphal like growth                 sensitivity H2O2 compared wild type strain true hyphae. morphology conditional MCM4                  (Figure 9A). conditional mutant MCM4 incubated H2O2 <br /> strain absence Met/Cys rescued strain (MCM4R)                   presence Met/Cys showed higher sensitivity liquid culture comparable wild type media tested (data         compared heterozygous strain. result demonstrates shown).                                                                  role Mps1 oxidative stress tolerance C. albicans. <br />    distinctive morphological features C. albicans              MPS1 gene required oxidative stress response, formation Chlamydospores oxygen limited environment.                particularly important check response macrophages,  role Mps1in chlamydospore formation analyzed.                oxidative free radical attack hand defence. Strains grown 10 days dark 25uC                         examine heterozygous mutant control wild type strains <br /> microaerophilic conditions. Large numbers chlamydospores                  injected peritoneal cavity mice. Cells were observed wild type strain, situated end                   subsequently retrieved peritoneal exudates 24 hours <br /> branching filaments, heterozygous mutant showed               injection. Survival heterozygous mutant (MFD2) sparse elongated branching filaments significantly               macrophages determined plating exudate YPD <br /> reduced chlamydospores end filaments (Figure 8C).                   agar counting colony forming units (CFU) strains. change chlamydospore formation observed                        5-fold decrease CFU observed MFD2 strain heterozygous strain prolonged incubation 21 days.              comparison wild type (WT) strain (Figure 9B). Conditional mutant presence Met/Cys showed response                  attributed sensitivity MFD2 strain oxidative stress similar wild type (Figure 8C).                                            exposure macrophages.  hyphae formation C. albicans                                                                              required rupturing macrophages facilitate escape Sensitivity mps1 mutants oxidative stress                              hostile environment phagosome. microscopic <br />   important ways macrophages host                     examination peritoneal exudates performed examining <br /> immune defend endocytosed pathogens, release             morphological transition exposure macrophages reactive oxygen species.  pathogens evolved mechanisms               vivo conditions. 24 hrs exposure macrophages wild <br /> required protecting free radical effects.        type strain, engulfed macrophages showed significant filamen- previously demonstrated  checkpoint proteins                    tation (Figure 9C). contrary, MFD2 strain, engulfed required encountering free radical attacks caused                macrophages showed yeast form (Figure 9C).  <br />  <br />  <br /> PLOS | www.plosone.org                                              11                             July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                     Role MPS1 Candida albicans <br />  <br />  <br /> heterozygous mps1 mutant failed undergo morphological                     future challenge nutrient signaling (like <br /> transition response macrophages.                                  GlcNAc) developmental processes C. albicans coordi- <br />                                                                              nated spindle assembly checkpoint machinery, shown <br /> Discussion                                                                   Caenorhabditis elegans [40]. <br />  <br />    microbial pathogens like C. albicans spindle checkpoint            Supporting Information <br /> machinery play important role survival host. organisms grow constant threat host defense               Figure S1    Computational characterization Mps1p.   <br /> mechanisms, damage cellular components like DNA                     Cartoon presentation Kinase domain C-terminal inevitable. checkpoint machinery ensures proper chromo-                  deduced <span id='am-13' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> sequence, identified using motif <br /> somal segregation. Deregulation checkpoint machinery                 finder Expasy server. Computationally predicted Ser/thr kinase <br /> leads aneuploidy chromosomal instability. report,          domain located aa 367 637 Carboxy terminus. characterized S. cerevisiae Mps1 homolog C. albicans.            (B) Phylogenic tree constructed studying evolutionary distances <br /> Essentiality MPS1 C. albicans confirmed Homozygote              known predicted MPS1 family protein kinases. <br /> Trisome test. study function gene, created              phylogram bootstrap analysis performed conditional mutants replacing promoter MPS1                   Megatree server (http://www.megatree.com). Values start <br /> Methionine/Cysteine regulatable MET3 promoter. Analysis                   branch point indicate calculated distances <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span> homology. <br /> mps1 mutants showed normal conditions gene                Organisms respective gene accession numbers <br /> required proper segregation chromosomes. mutant                  provided. Sequences downloaded NCBI genome <br /> cells stained DAPI, displayed single, largely         sequence depository. <br /> stained nuclear region buds nucleus, suggesting            (TIF) <br /> mutant cells failed complete nuclear division. Flowcyto-             Figure S2 Mutant construction essentiality confir- <br /> metry analysis showed increase ploidy levels mps1             mation Mps1.   schematic presentation construction mutants time. particular feature reminiscent                  cassette used disrupting MPS1gene, URA blaster <br /> behavior mps1-1 mutants budding yeast [6].                         technique. Disruption cassette, mps1: hisG:: URA3::hisG:mp s1    eukaryotes spindle assembly checkpoint highly                   employed inactivating allele, homologous <br /> conserved. monitors attachment kinetochores               recombination shown. Restriction enzymes used preparing <br /> microtubules prometaphase. conditions                     cassette DNA fragment used probe screening kinetocores lack proper tension lack microtubule occupancy             transformants southern analysis shown. (B) kinetochore, halts metaphase anaphase transition.               Autoradiogram confirmation allele mutants genes S. cerevisiae C. albicans share evolutionarily          PstI enzyme digestion. indicated 6.1 kb 2 kb bands conserved functions. S. cerevisiae Mps1 play important role         integration hUh cassette MPS1, undisrupted <br /> SAC.  analyzed SAC function Mps1 conserved               second allele respectively. positive transformants (MFD1- C. albicans. adopted Nocodazole sensitivity assay          12) shown support high efficiency recombination. (C) role MPS1 gene spindle checkpoint activation.               Confirmation essentiality Heterozygote Trisome test. <br /> presence Nocodazole, mutants readily lost viability,                Disruption cassette, mps1:Ura3&#226;&#8364;&#8482;:: ARG4::Ura3&#226;&#8364;&#8482;:mps1 used suggesting  gene required normal activation              inactivating allele schematically represented. internal <br /> spindle assembly checkpoint (SAC) pathway.  Nocoda-                 cis-recombination help reconstruct active URA3 gene zole treated mutants cells showed absence cell cycle arrest.         flanking Ura3&#226;&#8364;&#8482; fragments. (D) Southern blot confirmation suggests SAC function conserved C. albicans             allele disruptants triploids (mps1::UAU1/mps1::URA3/MPS1) <br /> Mps1.                                                                        obtained HT test. autoradiogram, recurrent 2.3 kb band <br />     attempt unravel functional link             observed colonies screened; indicates triploids morphogenesis checkpoint signaling genes, studied                 locus MPS1 gene confirmed essentiality gene. <br /> MPS1 expression various filamentation inducing media                  (TIF) <br /> mutants major signal transduction pathway involved                                                                              Methods S1 Supplementary methods. <br /> morphogenesis. qRT-PCR analysis revealed upregulation                                                                              (DOC) <br /> MPS1 presence Serum N-acetyl Glucosamine comparison glucose grown cells. qRT-PCR analysis revealed MPS1 expression regulated Efg1, Ras1,               Acknowledgments <br /> Tpk2 independent pathway.  interpreted                Authors wish thank W Fonzi, Aaron P.Mitchell P.E.Sudbery Efg1 dependent expression checkpoint kinase like MPS1               generously providing C.albicans strains plasmids. grateful suggests additional roles Efg1 regulating morphogenesis                John Carbon, Neil  Gow D.Chattopadyaya suggestions spindle checkpoint signaling pathway (Figure 10). Mor-               cell synchronization flowcytomtery protocols S.S. Thakur                                                                              helping animal experiments. thank Abira Chaudhuri phological studies various vitro hyphae inducing conditions <br />                                                                              critical reading manuscript. VKN thanks ICMR research <br /> showed involvement Mps1 morphogenesis including chla-                  fellowship. MK AK thank CSIR SRA fellowship. AD thank <br /> mydospore formation C. albicans. Mps1 knockouts showed               Department Biotechnology National Institute Plant Genome <br /> sensitivity oxidative stress. mechanisms C.          Research, Ministry Science Technology, Government India, albicans uses evade macrophages morphological                  funding project. <br /> transition. Analysis heterozygous mps1 mutant (MFD2) engulfed Macrophages showed presence yeast forms. Taken                    Author Contributions  experiments MPS1 gene C. albicans             Conceived designed experiments: AD MK VKN AK. Performed <br /> obligatory functions survival, checkpoint activation, morpho-             experiments: MK VKN DR PM. Analyzed data: AD MK VKN <br /> genesis oxidative stress tolerance. step elucidate        AK. Contributed reagents/materials/analysis tools: AD. Wrote paper: interacting proteins assist functions Mps1.           AD MK VKN AK. <br />  <br />  <br /> PLOS | www.plosone.org                                              12                              July 2014 | Volume 9 | Issue 7 | e101517 <br />                                                                                                                                              Role MPS1 Candida albicans <br />  <br />  <br />  <br />  <br /> References <br />  1. Odds FC (1994) Candida albicans, life times pathogenic yeast. J Med         21. Barelle CJ, Bohula EA, Kron SJ, Wessels D, Soll DR, et al. (2003) Asynchronous <br />     Vet Mycol. 32: 1&#226;&#8364;&#8220;8.                                                                          cell cycle asymmetric vacuolar inheritance true hyphae Candida albicans. <br />  2. Bai C, Ramanan N, Wang YM, Wang Y (2002) Spindle assembly checkpoint                         Eukaryot Cell 2: 398&#226;&#8364;&#8220;410. <br />     component CaMad2p indispensable Candida albicans survival virulence           22. Chen CY, Rosamond J (1998) Candida albicans SSD1 suppress multiple <br />     mice. Mol Microbiol 45: 31&#226;&#8364;&#8220;44.                                                            mutations Saccharomyces cerevisiae. Microbiol 144: 2941&#226;&#8364;&#8220;2950. <br />  3. Finley KR, Bouchonville KJ, Quick  Berman J (2008) Dynein-dependent                    23. Fonzi WA, Irwin (1993) Isogenic strain construction gene mapping     nuclear dynamics affect morphogenesis Candida albicans means                    Candida albicans. Genetics 134: 717&#226;&#8364;&#8220;728. <br />     Bub2p spindle checkpoint. J Cell Sci 121: 466&#226;&#8364;&#8220;476.                                       24. Care RS, Trevethick J, Binley KM, Sudbery PE (1999) MET3 promoter:  4. Li R, Murray AW (1991) Feedback control mitosis budding yeast. Cell 66:                new tool Candida albicans molecular genetics. Mol Microbiol 34: 792&#226;&#8364;&#8220;798. <br />     519&#226;&#8364;&#8220;531.                                                                                 25. Gietz D, St Jean  Woods RA, Schiestl RH (1992) Improved method high <br />  5. Hoyt MA, Totis L, Roberts BT (1991) S. cerevisiae genes required cell cycle              efficiency transformation intact yeast cells. Nucleic Acids Res 20: 1425. <br />     arrest response loss microtubule function. Cell 66: 507&#226;&#8364;&#8220;517.                    26. Murad  Lee PR, Broadbent <span id='am-30' about='oboInOwl:id' typeof='owl:Thing'>ID</span>, Barelle CJ, Brown AJ (2000) CIp10,  6. Winey M, Goetsch L, Baum P, Byers B (1991) MPS1 MPS2: novel yeast                        efficient convenient integrating vector Candida albicans. Yeast 16: 325&#226;&#8364;&#8220; <br />     genes defining distinct steps spindle pole body duplication. J Cell Biol 114:             327. <br />     745&#226;&#8364;&#8220;754.                                                                                 27. Sambrook J, Maniatis T, Fritsch EF (1989) Molecular cloning: laboratory manual. <br />  7. Sanyal K, Carbon J (2002) CENP homolog CaCse4p pathogenic                       Cold Spring Harbor, N.Y.: Cold Spring Harbor Laboratory Press. <br />     yeast Candida albicans centromere protein essential chromosome                  28. Sherman F (1991) Methods Enzymol 194: 3&#226;&#8364;&#8220;21 <br />     transmission. Proc Natl Acad Sci USA 99: 12969&#226;&#8364;&#8220;12974.                                    29. Arnaud MB, Costanzo MC, Skrzypek MS, Binkley G, Lane C, et al. (2005)  8. Lauze E, Stoelcker B, Luca FC, Weiss E, Schutz AR, et al. (1995) Yeast spindle               Candida Genome Database (CGD), community resource Candida albicans <br />     pole body duplication gene MPS1 encodes essential dual specificity protein                gene protein information. Nucleic Acids Res 33: D358&#226;&#8364;&#8220;363. <br />     kinase. Embo J 14: 1655&#226;&#8364;&#8220;1663.                                                            30. Hanks SK, Quinn  Hunter T (1988) protein kinase family: conserved <br />  9. Salvatore G, Nappi TC, Salerno P, Jiang Y, Garbi C, et al. (2007) cell                     features deduced phylogeny catalytic domains. Science 241: 42&#226;&#8364;&#8220;52. <br />     proliferation chromosomal instability signature anaplastic thyroid                31. Lindberg RA, Quinn  Hunter T (1992) Dual-specificity protein kinases:     carcinoma. Cancer Res 67: 10148&#226;&#8364;&#8220;10158.                                                       hydroxyl  Trends Biochem Sci 17: 114&#226;&#8364;&#8220;119. <br /> 10. Yuan B, Xu Y, Woo JH, Wang Y, Bae YK, et al. (2006) Increased expression              32. Fisk HA, Mattison CP, Winey M (2004) field guide Mps1 family     mitotic checkpoint genes breast cancer cells chromosomal instability.                protein kinases. Cell Cycle 3: 439&#226;&#8364;&#8220;442. <br />     Clin Cancer Res 12: 405&#226;&#8364;&#8220;410.                                                             33. X, Jones MH, Winey M, Sazer S (1998) Mph1, member Mps1-like <br /> 11. Daniel J, Coulter J, Woo JH, Wilsbach K, Gabrielson E (2011) High levels              family dual specificity protein kinases, required spindle checkpoint     Mps1 checkpoint protein protective aneuploidy breast cancer cells. Proc            S. pombe. J Cell Sci 111: 1635&#226;&#8364;&#8220;1647. <br />     Natl Acad Sci USA 108: 5384&#226;&#8364;&#8220;5389.                                                        34. Weiss E, Winey M (1996) Saccharomyces cerevisiae spindle pole body <br /> 12. Landi MT, Dracheva T, Rotunno M, Figueroa JD, Liu H, Dasgupta  et al.                      duplication gene MPS1 mitotic checkpoint. J Cell Biol 132: 111&#226;&#8364;&#8220;123. <br />     (2008) Gene expression signature cigarette smoking role lung               35. Uhl MA, Biery M, Craig N, Johnson AD (2003) Haploin <br /> </body></html>