<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                          RESEARCH ARTICLE <br />  <br />  <br />                                                           Protein Complexes Bacteria <br />  <br />                                                           Abstract <br />                                                           Large-scale analyses protein complexes recently available Escherichia <br />                                                           coli Mycoplasma pneumoniae, yielding 443 116 heteromultimeric soluble protein <br />                                                           complexes, respectively. coupled results mass spectrometry- <br />                                                           characterized protein complexes 285 &#226;&#8364;&#339;gold standard&#226;&#8364;? protein complexes identified <br />                                                           EcoCyc. comparison databases gene orthology, conservation, essentiality <br />                                                           identified proteins conserved lost complexes species. instance, 285 <br />                                                           &#226;&#8364;&#339;gold standard&#226;&#8364;? protein complexes E. coli, 10% fully conserved set <br />                                                           7 distantly-related bacterial &#226;&#8364;&#339;model&#226;&#8364;? species. Complex conservation follows                                                           models: conserved complexes, complexes conserved core, complexes     OPEN ACCESS                                           partial conservation conserved core. Expanding comparison 894 distinct bac- <br /> Citation: Caufield JH, Abreu M, Wimble C, Uetz P <br />                                                           terial genomes illustrates fractional conservation limits conservation (2015) Protein Complexes Bacteria. PLoS Comput         components protein complexes: just 14 285 model protein complexes perfectly <br /> Biol 11(2): e1004107. doi:10.1371/journal.                conserved 95% genomes used, predict 180 partially <br /> pcbi.1004107 <br />                                                           conserved half genomes. clear relationship gene essenti- <br /> Editor: Christine  Orengo, University College           ality protein complex conservation observed, poorly conserved complexes <br /> London, United Kingdom <br />                                                           contain significant number essential proteins. Finally, identify 183 complexes  <br /> Received: July 31, 2014                                   taining conserved components uncharacterized proteins interesting <br /> Accepted: January 2, 2015                                 targets future experimental studies. <br /> Published: February 27, 2015 <br />  <br /> Copyright: &#194;&#169; 2015 Caufield et al. open <br /> access article distributed terms Creative Commons Attribution License, permits         Author Summary <br /> unrestricted use, distribution, reproduction                                                             20,000 binary protein-protein interactions published medium, provided original author source credited. <br />                                                             studied bacterial species, results rarely capture extent pro- <br />                                                             teins complexes.  use experimentally-observed protein complexes <br /> Data Availability Statement: relevant data                                                             E. coli Mycoplasma pneumoniae, gene orthology, predict protein com- paper Supporting Information files. <br />                                                             plexes species bacteria. Surprisingly, majority protein complexes Funding: project supported               conserved, demonstrating unexpected evolutionary flexibility. observe <br /> National Institutes Health, grant R01GM109895. <br />                                                             broader trends protein complex conservation, especially genome-reduced species funders role study design, data <br /> collection analysis, decision publish,                                                             minimal sets protein complexes. <br /> preparation manuscript. <br />  <br /> Competing Interests: authors declared competing interests exist. <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                                 1 / 23 <br />                                                                                                                  Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            Introduction <br />                                            Abundant genome sequencing revealed astounding diversity bacterial genomes. <br />                                            species inhabit environment share fraction genes.                                            raises question organisms adapted environments using limit- <br />                                            ed number genes.  investigate protein complements bacterial genomes, <br />                                            proteins combined protein complexes species, com- <br />                                            plexes conserved diverse branches prokaryotic tree life. <br />                                                studies compared interaction networks S. cerevisiae, S. pombe E. coli <br />                                            possible systematic screens genetic interactions notable differences <br />                                            structure content [1,2]. Studies comparing baker&#226;&#8364;&#8482;s yeast fission yeast                                            essentiality varies species [3]. explained functional redun- <br />                                            dancy importance mechanism structure. extent differences                                            unexpected make sense seen light evolutionary flexibility [1]. <br />                                                Numerous studies protein-protein interactions revealed organization prote- <br />                                            omes networks interactions protein complexes. Systematic surveys protein <br />                                            complexes exist bacterial species, E. coli [4,5] Mycoplasma pneumo- <br />                                            niae [6]. list binary protein-protein interactomes clearly larger  <br />                                            sidered study. Based limited dataset, investigated complexes <br />                                            model organisms sufficient reconstruct homologous protein complexes <br />                                            species. particular challenge prokaryotes genomes species <br />                                            highly divergent model species used   E. coli Mycoplasma <br />                                            provide important paradigms: E. coli generalist live variety condi- <br />                                            tions Mycoplasma specialized parasite requires host cells grow. ~4,300 <br />                                            ~700 genes, respectively, represent medium-sized minimal genomes                                            medium minimal diversity protein complexes. <br />                                                studies investigated evolution diversity protein complexes wide <br />                                            range taxa. surprising given large-scale experimental data                                            available recent years. combination large number completed genome sequences <br />                                            use data evaluate extent protein complexes likely  <br />                                            served microbial species. Furthermore, evaluate biological role proteins <br />                                            complexes unknown function species. <br />                                                Existing studies comparing sets interactome data, including pure bioinformatics ap- <br />                                            proaches [7] generally limited comparisons characterized protein- <br />                                            protein interaction networks, comparisons S. cerevisiae, S. pombe E. coli [2,7,8]. <br />                                            Methodological frameworks predicting evolution basis gene presence/absence <br />                                            [1,9] employed predict novel interactions species. study,                                            focus distinct bacterial species, seven subject essentiality <br />                                            screens comprehensive protein complex surveys available. ex- <br />                                            pand focus set 894 bacterial genomes. <br />                                                order compare genomes protein complexes species, couple results                                            mass spectrometry-characterized protein complexes [5,6] databases gene orthology <br />                                            [10] essentiality [11] characterize interaction conservation protein complexes. <br />                                            Furthermore, use perspective genome reduction evaluate patterns levels                                            protein conservation. Comparing sets protein complexes divergent bacterial species <br />                                             case, E. coli M. pneumoniae) alleviates bias inherent using single <br />                                            species universal model.  observing protein complexes components <br />                                            present distinct species allows draw conclusions critical <br />                                            components microbial life. <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                        2 / 23 <br />                                                                                                                   Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            Results <br />                                            Conservation proteins bacterial genomes <br />                                            general, small microbial genomes enriched proteins conserved bac- <br />                                            teria (Fig. 1; Table 1). trend noticeable paralogy eliminated,  <br />                                            moving paralogous group (PG) natural effects genome reduction, <br />                                            seen smallest bacterial genomes. cases, average protein conserva- <br />                                            tion decreases genome size increases. Genes larger genomes, Pseudomonas <br />                                            aeruginosa, conserved 20 30 percent bacteria, average.                                            minimal genomes, including Mycoplasma species, share orthologous <br />                                            groups (OGs) 60 80 percent bacterial species, average. results rea- <br />                                            sonable expected: reduced genomes, <span id='am-11' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, lost sequence space                                            lost loci crucial bacterial life itself. Furthermore, genomes  <br />                                            creasing fraction paralogs conserved size increases (Fig. 1),                                            reduced genomes actually greater average conservation potential paralogs  <br />                                            moved. paralogous protein-coding loci reduced genomes enriched accessory <br />                                            genes broadly-conserved core genes. <br />                                                presence multiple members single orthologous group effect aver- <br />                                            age gene conservation.  display effect difference average locus  <br />                                            servation average OG conservation. (Fig. 1, orange vs. blue dots). difference                                            values approximation level paralogy genome; larger genomes ap- <br />                                            pear contain paralogs smaller genomes, especially genome size falls 1 <br />                                            Mb. effect average gene conservation expected, using orthology-based compari- <br />                                            sons compresses paralogs single OGs. data set, 20 genomes  10 unique <br />                                            genera) 3 Mb greater average conservation OGs individually- <br />                                            considered loci. smallest genome set, cicada endosymbiont Hodgkinia <br />                                            cicadicola [12] demonstrates difference average conservation OGs  <br />                                            dividual loci. genomes greater 3 Mb higher average conservation individu- <br />                                            ally-considered loci OGs. <br />  <br />  <br />                                            protein complexomes E. coli Mycoplasma pneumoniae <br />                                            study, used literature-curated set EcoCyc E. coli protein complexes                                            protein complexes isolated Hu et al. [5] set experimentally-determined complexes                                            E. coli (Fig. 2A). set experimentally-determined Mycoplasma pneumoniae complexes <br />                                            identified K&#195;&#188;hner et al. [6] included comparison distantly-related, mini- <br />                                            mal set. datasets differ content approach, E. coli data sets contain <br />                                            300 complexes. complexes EcoCyc set contain 2 4 unique proteins <br />                                            Hu set contains comparatively higher number complexes  30)  <br />                                            taining 5 unique protein components  e, unique proteins mapping different ortho- <br />                                            logous groups). Note Hu et al. complexes appear represent subsets                                            complexes  e., ribosome constitutes single complex EcoCyc represented                                            complexes Hu et al.).  EcoCyc set partially redundant  e., RNA poly- <br />                                            merase holoenzyme represented different protein complex, F1 F0 subre- <br />                                            gions <span id='am-12' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> synthase). <br />                                                size complexes data set produced K&#195;&#188;hner et al. appears differ                                            distribution characterized Hu et al. (Fig. 2A). Specifically, M. pneumoniae <br />                                            complexes unique members contain just unique proteins.                                            cross-species discrepancy result methodology, K&#195;&#188;hner et al. suggest  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                         3 / 23 <br />                                                                                                                                 Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 1. Protein complexes enriched highly conserved components. point indicates single genome average conservation loci orthologous groups (OGs) measured presence 898 bacterial genomes. Representative genomes 8 species focused study indicated vertical lines following labels: M. g., Mycoplasma genitalium; M. p., Mycoplasma pneumoniae; H. p., Helicobacter pylori; S. s., <br /> Streptococcus sanguinis; C. c., Caulobacter crescentus; B. s., Bacillus subtilis; E. c., E. coli; P. , Pseudomonas aeruginosa. Materials Methods specific genome identities. Average gene conservation specified percentage. Average gene conservation values reduced fraction predicted protein-coding genes present eggNOG v.3 account genes predicted orthology. produce OG conservation instead locus <br /> conservation, locus set potential paralogs  case, genes sharing OG) removed prior calculating averages. logarithmic regression fitted sets values. Average OG conservation values shown subsets protein-coding genes present protein complexes E. coli [5] M. pneumoniae [6]. species, values representative members complexomes species predicted complexomes using data sets models. <br /> doi:10.1371/journal.pcbi.1004107.g001 <br />  <br />  <br />  <br />                                                 representative authentic biological differences species. M. pneumoniae  <br />                                                 tains fewer unique proteins E. coli does difference limits number unique <br />                                                 proteins seen single complex. <br />                                                    exact protein complexes defined data set differ. Pairwise comparison pres- <br />                                                 ence absence proteins complex improved mapping components ortholo- <br />                                                 gous groups complexes appear present identical form data <br />                                                 sets. Fig. 2B provides examples types complex matches seen data sets. <br />                                                 instance, DNA polymerase III holoenzyme (EcoCyc: CPLX0&#226;&#8364;&#8220;3803) contains 9 unique <br />                                                 proteins EcoCyc closest match Hu set contains 7, including proteins <br />                                                 EcoCyc complex. &#226;&#8364;&#339;missing&#226;&#8364;? proteins EcoCyc complex                                                 Hu complexes. Hsp70 chaperone complex (EcoCyc: HSP70-CPLX) provides  <br />                                                 example: M. pneumoniae complexes provide better match EcoCyc complex <br />                                                 Hu set does. Topoisomerase IV (EcoCyc: CPLX0&#226;&#8364;&#8220;2424) good match                                                 data sets representative Hu complex contains additional protein. Lastly, RecBCD <br />                                                 serves example good E. coli-specific match components present                                                 M. pneumoniae complexes. <br />                                                    aggregate, EcoCyc complexes reliable matches experi- <br />                                                 mental sets (Fig. 2C). Using 285 EcoCyc complexes guide, best matches                                                 sets classified &#226;&#8364;&#339;good&#226;&#8364;? contain half unique proteins <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                          4 / 23 <br />                                                                                                                             Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Table 1. Protein conservation 8 species. <br />  <br />                      M. pneumoniae      M. genitalium   B. subtilis     S. sanguinis    H. pylori        C. crescentus     P. aeruginosa    E. coli <br /> M. pneumoniae        601                517 (86.02%)    430 (71.55%)    427 (71.05%)    336 (55.91%)     379 (63.06%)      395 (65.72%)     408 (67.89%) <br /> M. genitalium        466 (96.68%)       482             388 (80.50%)    381 (79.05%)    290 (60.17%)     333 (69.09%)      353 (73.23%)     355 (73.65%) <br /> B. subtilis          644 (15.88%)       627 (15.46%)    4056            1974 (48.67%)   1374 (33.88%)    2101 (51.80%)     2465 (60.77%)    2428 (59.86%) <br /> S. sanguinis         538 (26.37%)       519 (25.44%)    1514 (74.22%)   2040            804 (39.41%)     1186 (58.14%)     1341 (65.74%)    1379 (67.60%) <br /> H. pylori            314 (21.43%)       304 (20.75%)    844 (57.61%)    631 (41.21%)    1465             916 (62.53%)      979 (66.83%)     992 (67.71%) <br /> C. crescentus        454 (12.58%)       440 (12.20%)    2064 (57.21%)   1487 (41.21%)   1401 (38.83%)    3608              2618 (72.56%)    2414 (66.91%) <br /> P. aeruginosa        678 (11.74%)       655 (11.34%)    3275 (56.72%)   2353 (40.75%)   1963 (34.00%)    3615 (62.61%)     5774             4007 (69.40%) <br /> E. coli              594 (14.33%)       567 (13.68%)    2418 (58.34%)   1791 (43.21%)   1414 (34.11%)    2409 (58.12%)     3027 (73.03%)    4145 <br />  leftmost column organism basis comparison row organism compared  organism <br /> compared itself shows total number proteins organism dataset used. example, M. pneumoniae shares 86.02% proteins M. genitalium M. genitalium shares 96.68% proteins M. pneumoniae. Data species collected Uniprot proteins mapped using common COGs, NOGs, bactNOGs [10] <br />  <br /> doi:10.1371/journal.pcbi.1004107.t001 <br />  <br />  <br />  <br />  <br />                                                  members orthologous groups) &#226;&#8364;&#339;poor&#226;&#8364;? contain match half                                                 EcoCyc complex&#226;&#8364;&#8482;s components. complex size greater 4 unique proteins                                                 good match Hu et al. K&#195;&#188;hner et al. complex sets. 28 complexes (9.8%)                                                 complexes size 4 good matches sets. majority complexes                                                 size class (153 246) contained matching component Hu E. coli <br />                                                 complexes match K&#195;&#188;hner et al. M. pneumoniae complexes. <br />                                                    set M. pneumoniae complexes serves rough model complexes com- <br />                                                 monly bacterial species  S9 Table S10 Table) predicted conserva- <br />                                                 tion complex). imperfect model: 116 complexes, 28 fully <br />                                                 conserved   components present orthologs) 7 model spe- <br />                                                 cies study. 39 M. pneumoniae complexes appear share 2/3 components <br />                                                 species, 75 complexes share half. Just complex contains <br />                                                 components entirely specific M. pneumoniae (complex 87, composed uncharacterized <br />                                                 proteins Mpn036 Mpn676, respective UniProt entries P75078 P75116). <br />  <br />  <br />  <br />                                                 Using protein complexomes predict complexes conserved                                                 species <br />                                                 variability EcoCyc Hu datasets direct impact usefulness                                                 complexomes models bacterial species. case, EcoCyc Hu com- <br />                                                 plex sets provide comprehensive complex set currently available E. coli. inter- <br />                                                 section sets (Fig. 3A) limited: just 576 unique orthologous groups                                                 shared sets just 132 complexes appear &#226;&#8364;&#339;good&#226;&#8364;? matches sets. <br />                                                 Using 132 complexes model P. aeruginosa shows 120                                                 complexes conserved based orthologous components present P. aeruginosa <br />                                                 genome. uncharacterized P. aeruginosa complexome contains roughly                                                 number complexes E. coli prediction method misses half <br />                                                   150) potential complexes unless use unique complexes                                                 set. used results evidence data sets used independent <br />                                                 models intersecting set: losing half potential model <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                        5 / 23 <br />                                                                                                                            Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 2. Protein complex data sets vary composition.   Count complexes E. coli complex datasets ([5]; EcoCyc [13]) M. pneumoniae <br /> dataset [6], size  number unique protein components). Multimers single proteins  e., homodimers) included. (B) Examples complex <br /> matching data sets. mapped orthologous group (OG), components complex directly comparable complex sets perfect matches rare. instances, OG complex present best matching complex OG present different complex. cases, matched complex contain components (OGs) seen query complex  case topoisomerase IV). (C) Summary matching complex quality data sets. EcoCyc complexes used set query complexes experimental data sets used search space.  poor match requires just matching component, good match requires half components query complex present matching complex. number complexes category shown; complex size  <br /> doi:10.1371/journal.pcbi.1004107.g002 <br />  <br />                                               complexes simply inconsistencies data sets limiting broad cross- <br />                                               species comparision. <br />                                                   Fig. 3B displays distributions protein complex conservation bacterial species <br />                                               E. coli. (M. pneumoniae complexes used comparison.) plots <br />                                               provide median interquartile range protein complex conservation fractions                                               species, using EcoCyc Hu et al. complexes model complex set. compre- <br />                                               hensive set protein complexes identified species  Fol- <br />                                               lowing results shown Fig. 1,  predict bacterial protein <br />                                               complex component sets share half OGs bacterial ge- <br />                                               nomes, average. Basic biology plays role  expect subset crucial protein <br />                                               complexes like polymerases conserved species. set EcoCyc com- <br />                                               plexes, appears highly-conserved P. aeruginosa  entire interquartile range lies  <br />                                               tween 75% complex conservation, showing average EcoCyc complex  <br />                                               represented P. aeruginosa) shows greater range conservation                                               species. Hu complexes lower complex conservation median values EcoCyc                                               H. pylori lower variability P. aeruginosa.  median values                                               useful conservation ranges: distance highest lowest values  <br />                                               cludes possibility 0 100% conservation using model E. coli complexes. <br />                                               species closely related E. coli set&#226;&#8364;&#8221;P. aeruginosa C. cres- <br />                                               centus&#226;&#8364;&#8221;produce different median values interquartile ranges sets  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                   6 / 23 <br />                                                                                                                                  Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 3. Protein complex sets vary conservation bacteria.   Overlap literature-curated (EcoCyc) experimentally-observed (Hu <br /> et al.) E. coli complex sets limited. data set contains unique proteins, mapped orthologous groups (far left). complex E. coli complex sets appear shared complex set (middle; potentially shared complex half components complex sets). Using just set complexes shared E. coli sets model predicting <br /> complexomes species (far right; case, P. aeruginosa used example) limiting. 12 complexes shared set appear conserved P. aeruginosa roughly additional 150 complexes expected based seen E. coli. (B) box plot displays range conservation fractions E. coli protein complexes literature curated (EcoCyc) experimental (Hu et al.) sets respect species E. coli. upper lower edges box correspond quartile conservation fraction values, respectively. upper whisker <br /> corresponds highest value 1.5 times inter-quartile range (IQR) lower whisker corresponds lowest value range. Data points outside 1.5 times IQR represented single data points. <br /> doi:10.1371/journal.pcbi.1004107.g003 <br />  <br />  <br />                                                 protein complexes. Components complexes E. coli sets, used models, clearly <br />                                                 conserved differently bacterial species. higher-resolution comparison necessary                                                 determine complexes highly-conserved. <br />  <br />  <br />                                                 Protein complexes essentiality poorly conserved bacteria <br />                                                 size distribution different E. coli Mycoplasma, hypothesized ho- <br />                                                 mologous complexes similar, size composition.                                                  true: complexes share half components data sets (Fig. 2C). ma- <br />                                                 jority complexes shows 50% overlap EcoCyc Hu, Hu <br />                                                 Mycoplasma. suggests technical (E. coli vs E. coli) biologi- <br />                                                 cal reasons (E. coli vs. Mycoplasma) differences. <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                           7 / 23 <br />                                                                                                                            Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 4. Examples protein complex conservation bacteria. Complexes identified using common EcoCyc <span id='am-23' about='oboInOwl:id' typeof='owl:Thing'>ID</span>. complex <br /> subunit assigned COG ID. Grey areas indicate OG presence, white areas indicate OG absence, blue areas indicate essentiality species- <br /> specific screen  Materials Methods references). Values spaces indicate presence potential paralogs corresponding <br /> species; proteins specified values paralogs. Complex structures available PDB: ATP synthase F1, 3OAA; succinate dehydrogenase, <br /> 1NEN; Outer membrane protein assembly complex, (2KM7, 3TGO, 3TGO, 4K3C, 2YH3). Species arranged taxonomy  Materials Methods details) E. coli Mycoplasma serving distant species. <br /> doi:10.1371/journal.pcbi.1004107.g004 <br />  <br />  <br />                                                  global detailed picture protein complex conservation, com- <br />                                               pared conservation 8 bacterial species, including species protein <br />                                               complex sets exist. EcoCyc complex set used standard species <br />                                               compared. Fig. 4 provides examples ways protein complexes                                               conserved species. Conservation protein complexes roughly grouped                                               categories: conserved complexes, complexes core set proteins conserved, <br />                                               core set appears consistently conserved. conservation es- <br />                                               sentiality related paralogy, compared components complexes                                               presence absence paralogs. <br />                                                  commonly assumed highly conserved proteins important                                               essential cases. Interestingly, true (Fig. 4). example,  <br />                                               conserved succinate dehydrogenase components essential 3 species shown. <br />                                               components complex  defined default structure E. coli) present <br />                                               Pseudomonas aeruginosa Caulobacter crescentus. Helicobacter pylori B. subtilis <br />                                               encode 3 4 components 3 species appear lost entire complex. <br />                                               Similarly, Bam outer membrane protein assembly complex (EcoCyc: CPLX0&#226;&#8364;&#8220;3933) shows <br />                                               partial essentiality complex 4 species components conserved                                               3 species. complex similarly patchy pattern conservation, number <br />                                               zero 5 components conserved. case H. pylori Bam complex, initially <br />                                               like lack conservation result component replacement functionally <br />                                               similar proteins [14]. contrast, F1 ATP synthase conserved species examined.                                               examples complexes conserved important functions <br />                                               indicate  measured presence essential proteins complexes). <br />                                                  Fig. 5A displays EcoCyc E. coli complexes component present M. <br />                                               pneumoniae. case, fraction essentiality  number protein components                                               essential protein components present) shown. Fig. 5B displays conservation <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                    8 / 23 <br />                                                                                                                                  Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 5. Fractional essentiality conservation protein complexes species.   column represents protein complex  defined EcoCyc E. coli) fraction essentiality species shown left. subset complexes component predicted conserved M. pneumoniae. Black stars complex IDs indicate complexes shown Fig. 3. example complexes predicted present M. pneumoniae shown far right complex list. S1 Table key complexes. species E. coli, complexes predicted using orthologous groups (OGs). Colors indicate fraction essentiality: blue&#226;&#8364;&#8221;conserved components essential fraction specified right, grey&#226;&#8364; components conserved conserved components essential. (B) Conservation complexes shown  . Colors indicate fraction conservation ranging dark green  proteins present) red  protein present). General functional group assignments manually assigned based EcoCyc annotations. Columns panels B correspond complexes. (C)  set EcoCyc E. coli complexes; column single complex. extended version heat map provided S1 Fig. (D) B, set EcoCyc E. coli complexes; column single complex. order complexes identical (C). extended version heat map provided S2 Fig. Columns panels C D correspond complexes. <br /> doi:10.1371/journal.pcbi.1004107.g005 <br />  <br />  <br />  <br />  <br />                                                 fractions E. coli complexes protein conserved M. pneumoniae,                                                 necessarily present M. pneumoniae complex. complete survey EcoCyc com- <br />                                                 plexes species terms conservation essentiality provided S1 Fig.                                                 S2 Fig., respectively. Conservation fraction established fraction unique proteins                                                 defined complex present target species. Notably, proteins 21 complete EcoCyc <br />                                                 complexes fully conserved 8 species, just 15 complexes subunits al- <br />                                                 ternate forms  e., RNA polymerase different sigma factors) removed. additional <br />                                                 19 complexes fully conserved species Mycoplasma species.  <br />                                                 maining complexes vary extensively degree extent conservation. number                                                 complexes conserved E. coli, P. aeruginosa, C., crescentus, H. pylori, B. subti- <br />                                                 lis S. sanguinis Mycoplasma (e.g. succinate dehydrogenase, EcoCyc: SUCC- <br />                                                 DEHASE). Overall, 176 EcoCyc complexes 3 unique proteins (S1 Fig.), 128 <br />                                                 appear lost unique protein component species. demon- <br />                                                 strates protein complexes far flexible evolutionary terms                                                 previously assumed. <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                           9 / 23 <br />                                                                                                                   Protein Complexes Bacteria <br />  <br />  <br />  <br />                                                Protein complex function varies similar way conservation (Fig. 5B). expected, <br />                                            highly conserved complexes directly involved DNA replication, tran- <br />                                            scription, translation. protein complexes varying conservation fractions trans- <br />                                            port complexes&#226;&#8364; bacterial membrane structures vary species, degree                                            transporter component evolution expected. distinct complexes involved                                            DNA modification repair demonstrate perfect conservation. <br />                                                E. coli complexes serve &#226;&#8364;&#339;gold standard&#226;&#8364;? protein complexes bacteria                                            cases components complex broadly conserved. property <br />                                            true just small fraction complexes. Fig. 5D displays conservation fractions 285 <br />                                            E. coli complexes EcoCyc set, clustered similarity conservation patterns                                            7 species used study. Just 21 complexes appear fully conserved   <br />                                            orthologs components present) species. broad taxo- <br />                                            nomic range, relaxed cutoff appropriate predict complex conserved; <br />                                             28 complexes contain 2/3 E. coli components species. <br />                                            Lowering cutoff half E. coli components yields 34 complexes.                                            lack broad conservation   matter complex presence absence                                            species.  complex components appear conserved independently                                            members complex. Similarly patchy conservation seen essentiality (Fig. 5C), <br />                                            broadly conserved complexes (far left) generally retain essentiality spe- <br />                                            cies consistently-conserved complexes  retain essentiality <br />                                            appearing lose complex components. <br />  <br />  <br />                                            E. coli protein complexome model species <br />                                            E. coli frequently used model organism bacteria general. Using literature- <br />                                            curated set protein complexes EcoCyc, sought determine protein <br />                                            complexome serves model complexes bacterial species. comparison                                            fractional conservation EcoCyc complex 894 different bacterial genomes                                            result (Fig. 6; S5 Fig. expanded version). genomes comparison ar- <br />                                            ranged NCBI taxonomy definitions, revealing patterns complex conservation closely <br />                                            corresponding numerous taxonomic boundaries. Hierarchical clustering E. coli <br />                                            model complex (specifically, UPGMA) basis fractional conservation                                            species reveals groups complexes similar patterns predicted conservation. <br />                                                species overall conservation E. coli complexes  unsurprisingly, <br />                                            closely related E. coli. Roughly complex set conserved                                            species minimal Rickettsia Mycoplasma genomes,  serving nota- <br />                                            ble exceptions. middle shows difference conservation Pro- <br />                                            teobacteria species. Lactobacillales difference conservation <br />                                            complexes, degree resemble Cyanobacteria closely                                            subset. (far left Fig. 6) complexes demonstrate variable <br />                                            conservation species. complexes missing partially conserved <br />                                            Proteobacteria fully present Firmicutes species extrem- <br />                                            ophiles like Thermus Thermotoga species. Overall, 285 EcoCyc complexes, 12 (~4%) <br />                                            component present 894 bacterial genomes set. perfectly <br />                                            conserved genomes 14 complexes appear conserved 95%                                            genomes. potential complex conservation generously defined conservation                                            half complex components, 3 EcoCyc complexes potentially conserved                                            894 genomes, 25 potentially conserved 95% genomes, 186 potentially <br />                                            conserved half genomes. Variance set complex <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                        10 / 23 <br />                                                                                                                                Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 6. E. coli complex conservation Bacteria corresponds taxonomic boundaries. heat map displays fractional conservation 285 <br /> EcoCyc protein complexes Fig. 5, case 894 different bacterial genomes indicated tree left. Materials Methods taxonomic details. Specific complex names species/strain names S5 Fig. Complexes (columns) clustered basis distance average fractional conservations (average linkage). <br /> doi:10.1371/journal.pcbi.1004107.g006 <br />  <br />                                                conservation fractions 0.189. conservation complexes follows existing <br />                                                taxonomy  generally conserved complexes like RNA polymerases miss- <br />                                                ing entire genera. <br />                                                    experimentally-determined protein complexes identified Hu et al. used                                                model E. coli complexome (S6 Fig.). complexes appear partial conserva- <br />                                                tion nearly species using model. Distinctions seen minimal ge- <br />                                                nomes Rickettsiales Mycoplasma genomes related species.                                                310 Hu et al. complexes, 16 (~5%) component present 894 bacterial <br />                                                genomes set. EcoCyc complexes, perfectly conserved ge- <br />                                                nomes single complex (complex 271) appears conserved 95%                                                genomes. Using 50% cutoff potential complex conservation used  Hu <br />                                                complexes appear conserved 894 genomes, 10 potentially conserved 95% <br />                                                genomes, 182 potentially conserved half genomes.                                                Hu et al. complex values appear similar Ecocyc complexes, variance <br />                                                set Hu complex conservation fractions 0.097, indicating variability <br />                                                values seen EcoCyc complexes. lesser variance                                                seen surprising consistency taxonomic lines (S6 Fig.). <br />                                                    literature-curated EcoCyc model Hu et al.-based experimental model                                                evaluated comparision randomized version respective components. lit- <br />                                                erature-curated model, Pearson correlation 0.185, experimental model, <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                       11 / 23 <br />                                                                                                                                Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 7. Conserved complex components enriched essential proteins. correlation pronounced Mycoplasma (blue). Protein <br /> complexes E. coli [5] compared complexes M. pneumoniae [6] vice versa. Fraction conservation fraction essentiality calculated described Materials Methods. node represents single protein complex relative size corresponding size complex number components. K&#195;&#188;hner complex 50 corresponding Hu complex 77 indicated example complex match. <br /> doi:10.1371/journal.pcbi.1004107.g007 <br />  <br />                                                Pearson correlation 0.293. higher correlation value experimental model indi- <br />                                                cates closer random distribution complex correlation fractions species <br />                                                set. expect complexes conserved random pattern indicate                                                Hu et al. complex set useful EcoCyc complex set prediction wide <br />                                                range genomes. <br />  <br />  <br />                                                Essentiality proteins complexes impact paralogy <br />                                                Mycoplasma species highly reduced genomes generally assumed                                                retained essential proteins. fact, fraction conserved essential proteins                                                higher comparing Mycoplasma pneumoniae E. coli vice-versa (Fig. 7).                                                comparisons, complex components searched genomes essentiality  <br />                                                signed based target species. set Hu et al. E. coli complexes, complexes <br />                                                average conservation fraction 0.198&#194;&#177;0.230 average essentiality fraction                                                0.122&#194;&#177;0.196 M. pneumoniae. High variability conservation complexes expected <br />                                                complex components, like single proteins, subject broad variety evolutionary pres- <br />                                                sures. 53% complexes component present M. pneumoniae, <br />                                                average fractions increase 0.375&#194;&#177;0.184 0.231&#194;&#177;0.218, respectively. set <br />                                                K&#195;&#188;hner et al. M. pneumoniae complexes, complexes average conservation fraction <br />                                                0.716&#194;&#177;0.292 average essentiality fraction 0.32&#194;&#177;0.332 E. coli. 95%                                                complexes component present E. coli, average fractions increase                                                0.755&#194;&#177;0.245 0.337&#194;&#177;0.332, respectively. Overall, Mycoplasma protein complex components <br />                                                likely present essential E. coli E. coli protein complex components <br />                                                Mycoplasma. <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                       12 / 23 <br />                                                                                                                    Protein Complexes Bacteria <br />  <br />  <br />  <br />                                                possible explanation lower fraction conserved essential proteins E. coli                                            presence paralogs renders duplicate genes non-essential, given presence                                            additional copy redundant function. performed comparisons fraction  <br />                                            servation complex sum paralogy   total number copies                                            genes coding complex components target species). number paralogs                                            gene broadly defined using orthologous groups, numbers considered maxi- <br />                                            mum possible values specific counts known paralogous regions. observed                                            inverse trend E. coli complexes vs. M. pneumoniae (S3A Fig.) vice versa (S3B Fig.): <br />                                            paralogs E. coli conserved proteins Mycoplasma <br />                                            vice versa. specifically, E. coli complexes conservation fraction greater 0.6 <br />                                            M. pneumoniae total paralogy sums lower 40 poorly-conserved <br />                                            complexes paralogy sums 2 100. M. pneumoniae complexes  <br />                                            servation fraction greater 0.6 E. coli range sums paralogy 2                                            nearly 80. poorly-conserved complexes paralogy sums 60  Calculat- <br />                                            ed Pearson anti-correlation E. coli complexes vs. M. pneumoniae (S3A Fig.) -0.04                                            Pearson correlation M. pneumoniae complexes model E. coli (S3B Fig.) 0.05,  <br />                                            dicating limited overall correlation comparison. case conser- <br />                                            vation complexes species (Fig. 6), correlation case-specific. <br />                                                fraction essential components protein complexes non-random                                            greater expected, depending complexes compared (Fig. 8). compared                                            random assortment, Hu et al. E. coli complexes essential proteins expected                                            chance (Fig. 8A). Spearman anti-correlation -0.25  E. coli complexes Eco- <br />                                            Cyc (Fig. 8B) demonstrate similar trends, Spearman anti-correlation -0.22. M. pneu- <br />                                            moniae complexes K&#195;&#188;hner et al. (Fig. 8C) trend declining essentiality <br />                                            compared randomized essentiality fractions 0.6&#226;&#8364;&#8220;08. Spearman anti-correlation -0.03 <br />                                            M. pneumoniae complex set. E. coli anti-correlations                                            weak relationship. <br />                                                Paralogy examined function essentiality (S4 Fig.).  average paralogy <br />                                            values determined complex minimize impact complex size, especially                                            components complex essential. E. coli complexes Hu et al. <br />                                            (S4A Fig.) decrease average paralogy fraction essentiality decreases. total, com- <br />                                            pared random assortment, far E. coli complexes expected appear essenti- <br />                                            ality fractions 0.4  E. coli complexes EcoCyc (S4B Fig.) demonstrate similar <br />                                            trends. M. pneumoniae complexes K&#195;&#188;hner et al. (S4C Fig.),  appear  <br />                                            tain relationship essentiality average paralogy. Additionally, M. <br />                                            pneumoniae complexes expected essentiality fractions 0.2                                            fewer expected essentiality fractions greater 0.6. Spearman anti-correlation <br />                                            statistically significant -0.03. Overall, essentiality average paralogy appear                                            related E. coli M. pneumoniae complexes, probably M. pneumoniae  <br />                                            tains relatively paralogs. Example complexes sets OGs shared <br />                                            provided S4D Fig.; example, majority complex com- <br />                                            ponents essential representative genomes contain paralogs coding redun- <br />                                            dant complex components. <br />  <br />  <br />                                            Proteins unknown function <br />                                            Protein complexes attractive targets functional analysis, given proteins embed- <br />                                            ded functional context. especially true proteins unknown function                                            complex (Fig. 9A, B).  conservation defined greater 0.5 conservation <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                         13 / 23 <br />                                                                                                                            Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br />                                            Fig 8. Essentiality proteins complexes. Distribution essential genes E. coli (Hu <br />                                            et al.   EcoCyc (B), respectively) M. pneumoniae (C). fraction essential genes protein <br />                                            complexes determined complex set. E. coli, essential protein complexes enriched                                            essential proteins. contrast, complexes non-essential proteins represented genome- <br />                                            reduced Mycoplasma pneumoniae. distribution expressed binned log2 ratios observed                                            expected frequency. Values indicate observed frequency random results (= 1), respectively. <br />                                            doi:10.1371/journal.pcbi.1004107.g008 <br />  <br />  <br />  <br />  <br />                                            fraction essential complexes essential component target <br />                                            species. highly conserved components, essential 4 8 spe- <br />                                            cies. Using species reduces effect noise inconsistency essential- <br />                                            ity screens. Starting 39 EcoCyc-defined complexes contain unknown proteins,                                            15 appear conserved species study Mycoplasma. Fig. 9C displays <br />                                            example complexes Hu (E. coli) K&#195;&#188;hner (Mycoplasma pneumoniae) complexes, <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                   14 / 23 <br />                                                                                                                                Protein Complexes Bacteria <br />  <br />  <br />  <br />  <br /> Fig 9. Protein complexes rich highly-conserved proteins unknown function.   list EcoCyc E. coli protein complexes compared basis component presence vs. absence seven species study. Conserved complexes, figure, orthologous component present target species. Similarly, essential complexes include component essential E. <br /> coli target species. (B)  , subset EcoCyc complexes containing protein unknown unclear function. instances, complex itself known function roles components remain unclear. (C) Examples experimentally- <br /> observed protein complexes containing proteins unknown function. E. coli complex examples Hu et al. shown left, M. pneumoniae complexes K&#195;&#188;hner et al. shown right. Complexes labeled identifier used corresponding study. <br /> doi:10.1371/journal.pcbi.1004107.g009 <br />  <br />  <br />  <br />                                                respectively. Unlike parts B, complexes shown experimental results                                                literature-defined complexes. complex contains component unknown <br />                                                unclear function, context protein complex broader cellular function. <br />                                                instance, complex 66 Hu et al. (Fig. 9C) consists 6 unique proteins 3  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                                       15 / 23 <br />                                                                                                                   Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            unknown function  remain annotation). 6 proteins, 3 highly conserved <br />                                            1 frequently essential. E. coli protein MraZ, present Hu complex <br />                                            149, shown protein unknown function recently transcrip- <br />                                            tional regulator involved multiple pathways [15]. 149 Hu et al. E. coli complexes <br />                                            34 K&#195;&#188;hner et al. Mycoplasma pneumoniae (183 total) complexes contain                                            component unknown function.  109 Hu et al. E. coli complexes 19 K&#195;&#188;hner et al. <br />                                            M. pneumoniae complexes contain components highly conserved essential proteins.                                            list experimental complexes unknown components available S11 Table. <br />  <br />  <br />                                            Discussion <br />                                            substantial variation protein complexes species supports notion                                            complexes malleable previously thought. possible explanation                                            function complex important content. Complexes serve <br />                                            role contain different proteins function lost,                                            gap. studies functional redundancy lead variation                                            little overlap terms protein interaction species [2,3]. mutational change                                            protein complex catastrophic potential, complexes immutable. fact,  <br />                                            eral complexes essential species varying composition species. <br />                                            instance, 5 9 components E. coli Sec translocation complex (EcoCyc: SEC-SE- <br />                                            CRETION-CPLX) conserved species P. aeruginosa M. genitalium.                                            components, SecA, essential species focused work <br />                                            exception S. sanguinis; orthologs protein present 894 bacterial ge- <br />                                            nomes examined. remaining 4 E. coli components variable conservation                                            species. instance, YajC present 727 894 genomes. Strong selection <br />                                            pressure avoid mutations render entire complex ineffectual. explain <br />                                            observed higher level conservation protein complex components                                            proteins general (Fig. 1). <br />                                                essential &#226;&#8364;&#339;core&#226;&#8364;? components protein complexes conserved taxonomic <br />                                            levels &#226;&#8364;&#339;accessory&#226;&#8364;? components [1]. Given multiple interactions, proteins <br />                                            protein complexes highly conserved &#226;&#8364;&#339;un-complexed&#226;&#8364;? <br />                                            ones, retain essential roles fellow complex members present <br />                                            [16,17]. <br />                                                Components protein complexes  average, likely present bacte- <br />                                            rial species proteins complexes [1]. result high conservation sets <br />                                            large, essential complexes. 128 285 literature-verified E. coli protein complexes                                            fully present B. subtilis, 30 completely present M. genitalium.  <br />                                            stance, components ATP synthase complex (EcoCyc: ATPSYN-CPLX) present                                            species examined, vary essentiality. B. subtilis essentiality screens                                            essential genes ATP synthase, M. pneumoniae compo- <br />                                            nent essential. complexes&#226;&#8364;&#8221;predominantly transmembrane domains <br />                                            transporter functions&#226;&#8364; variable conservation essentiality,                                            provide examples dispensable accessory proteins  <br />                                                protein complexes essential functions E. coli present spe- <br />                                            cies. <span id='am-1' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-2' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="LPS" datatype="xsd:string"></span>lipopolysaccharide</span> transport complex (EcoCyc: CPLX0&#226;&#8364;&#8220;7992) serves excellent <br />                                            example: seven Lpt proteins complex essential E. coli <br />                                            conservation limited Gram-negative species including C. crescentus <br />                                            P. aeruginosa. transmembrane protein complexes follow pattern. <br />                                            Interestingly, species partial complex component conservation vs. E. coli highlight <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                        16 / 23 <br />                                                                                                                     Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            situations core elements complex conserved modified carry <br />                                            functions adapted special physiological circumstances. example, 3 4 <br />                                            succinate dehydrogenase complex (EcoCyc: SUCC-DEHASE) components E. coli                                            present B. subtilis S. sanguinis. especially interesting example <br />                                            components, SdhC SdhD, inner membrane proteins, SdhC                                            present component B. subtilis succinate dehydrogenase. conclude mem- <br />                                            brane proteins complexes particularly malleable, given role signaling                                            transport reflects adaptations specific environments nutrients present <br />                                             <br />                                                Smaller reduced bacterial genomes   relative E. coli) appear code                                            greater fraction highly-conserved protein complexes. conservation evident com- <br />                                            parisons Mycoplasma pneumoniae protein complexes. examination pro- <br />                                            tein complex components 800 bacterial genomes, species                                            M. pneumoniae offers better model protein complexes critical bacterial life. <br />                                            Protein complexes observed M. pneumoniae retained core set func- <br />                                            tions utilized higher degree multifunctionality metabolic enzymes <br />                                            [18,19]. <br />                                                Surprisingly, essential proteins poorly conserved essentiality itself                                            conserved species (Figs. 4 6). suggests functions replaced <br />                                            non-homologous displacement [20] genomes malleable evolutionary <br />                                            terms previously expected. Clearly, evolutionary flexibility contributed                                            success microbes populate possible environments planet. Variability com- <br />                                            plex conservation highlights limitation study: unavoidably limited                                            availability sequenced bacterial genomes. Newly-characterized genomes reveal addi- <br />                                            tional variation consistency protein complexes highly reduced                                            respects. <br />                                                protein components, individual complexes reveal underlying evolutionary <br />                                            processes (Fig. 6 S5 Fig.). highly-conserved complexes functions <br />                                            critical microbial life, including transcription, translation, transcript degradation. <br />                                            different RNA polymerase (RNAP) holoenzymes   RNA polymerases dif- <br />                                            ferent sigma factors) considered distinct complexes study, bacterial species <br />                                            unsurprisingly retained type RNAP. ribosome (EcoCyc: CPLX0&#226;&#8364;&#8220;3964)                                            conserved size high level conservation obscure cross- <br />                                            species differences. <br />                                                Variable conservation complexes visible Escherichia genomes. <br />                                            CPLX0&#226;&#8364;&#8220;7909  RnlA-RnlB toxin-antitoxin complex) appears present K-12 <br />                                            E. coli single species Shewanella Photobacterium. toxin-antitoxin                                            role bacteriophage resistance E. coli [21] unclear function  <br />                                            tained distantly related bacteria. CPLX0&#226;&#8364;&#8220;2001  ferric dicitrate transport  provides <br />                                            example gradual change. complex spans membrane, suggesting conser- <br />                                            vation membrane-dependent. appears case conserved <br />                                            Proteobacteria  Rickettsiales Buchnera species) poorly- <br />                                            conserved species traditionally considered Gram positive. subset com- <br />                                            plexes, including CPLX0&#226;&#8364;&#8220;1163 (HslVU protease) ABC-56-CPLX (aliphatic sulfonate ABC <br />                                            transporter), fit strict conservation model: complexes present                                            form fraction E. coli model complex. complexes ex- <br />                                            ceptions rule. Using E. coli model, complexes conserved perfectly <br />                                            wide range species; fact, complexes fractionally conserved. <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                          17 / 23 <br />                                                                                                                     Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            Materials Methods <br />                                            data management performed using house Python scripts (SPICEDNOG; available                                            http://github.com/caufieldjh/spicednog). Statistical analysis clustering performed <br />                                            using R package vegan [22]. <br />  <br />                                            Data sources <br />                                            set protein complexes Escherichia coli K-12 W3110 defined Hu et al. [5] <br />                                            assigned membership orthologous groups (OGs) version 3 eggNOG database <br />                                            [10] protein complex assigned single OG. remaining loci                                            referred using original locus identifiers  case, b-codes) retained <br />                                            analysis. process repeated protein complexes isolated K&#195;&#188;hner <br />                                            et al. [6] Mycoplasma pneumoniae M129 E. coli protein complexes defined                                            EcoCyc database [13]. representative set species (Bacillus subtilis 168, Caulobacter <br />                                            crescentus, Helicobacter pylori 26695, Mycoplasma genitalium G37, Pseudomonas aeruginosa <br />                                            UCBPP-PA14, Streptococcus sanguinis SK36) genome gene essentiality <br />                                            data selected depth analysis. species set referred focused set. Lists                                            protein-coding loci species obtained using respective proteome sets <br />                                            UniProt  S4 Table taxonomy IDs corresponding genomes used). Essentiality <br />                                            data collected Database Essential Genes [11]. Protein structures obtained <br />                                            Protein Data Bank (www.rcsb.org, [23]) referenced used. <br />                                                set 894 species, referred large set, prepared using bacterial spe- <br />                                            cies present eggNOG v.3 NCBI Taxonomy database [24]. trees shown Figs. <br />                                            4, 5, 6 cladograms intended general relationship species                                            context consensus taxonomy. <br />  <br />                                            Orthologous groups <br />                                            locus genome assigned single orthologous group (OG) eggNOG v.3 <br />                                            [10], loci assigned COG, NOG, bactNOG, depending                                            widely-conserved group assignment available  Powell et al. [10] details regarding <br />                                            OG levels).  presence locus determined entire set bacterial <br />                                            species; locus seen half bacterial species assigned conservation value                                            0.5. presence averaged loci generate value average locus conservation <br />                                            genome. value adjusted based locus coverage eggNOG  e., 70 <br />                                            percent loci genome mapped eggNOG OGs, average value reduced 30 <br />                                            percent.) identical set comparisons performed loci predicted paralogs <br />                                              loci OG assignment) removed prior comparison. Subsets selected <br />                                            species prepared included loci orthologous groups <br />                                            seen Hu et al., EcoCyc, K&#195;&#188;hner et al. protein complex sets. Genome sizes <br />                                            retrieved NCBI GenBank KEGG GENOME [25]. <br />  <br />                                            Comparing complex composition random model observe                                            distribution essentiality <br />                                            observed distributions essential genes coding protein complex compo- <br />                                            nents obtained using protein complex sets [5,6,13], eggNOG (v. 3) [10], Database <br />                                            Essential Genes [11] defined  single protein complex, essentiality fraction <br />                                            defined fraction genes complex essential, set                                            unique protein-coding genes complex. conservation scores used judge <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                          18 / 23 <br />                                                                                                                     Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            participation complex dataset, establishing maximum species data- <br />                                            set combination. Second, essentiality fraction linking essential protein                                            OG. instances multiple proteins shared OGs essentiality, essentiality  <br />                                            sidered primary case OG counted essential. <br />                                                random model created purpose comparing data set background <br />                                            noise. random model, meant represent collection randomly sized complexes,                                            populated proteins randomly assigned essential status. complex sizes <br />                                            randomly assigned value  complex assigned protein val- <br />                                            ues essential non-essential status. probability essential determined <br />                                            overall percent essential genes organism, random model size                                            equal maximum species dataset compared. random model                                            binning process observed data. mean bin ob- <br />                                            tained 10,000 replications. results s bins size longer equal ac- <br />                                            tual data set demonstrates appropriate background noise level comparison purposes. <br />                                            log2(Observed/Expected) values plotted Fig. 8 significant difference  <br />                                            tween observed essentiality expected. <br />  <br />  <br />                                            Comparative proteome complexome analysis <br />                                            general scheme data analysis follows: (1) list orthologous groups (OG) <br />                                            produced 894 bacterial species large set defined  (2) Presence <br />                                            absence OG determined species. (3) Repeated OGs removed                                            list step 2 repeated. (4) list step 1 used map OGs compo- <br />                                            nents sets protein complexes. complexes compared search cross- <br />                                            data set complex matches. Gene essentiality mapped OG species- <br />                                            dependent basis. (5) list 8 taxonomically-divergent species selected used define <br />                                            fractional conservation fractional essentiality protein complex. <br />                                                OGs used basis comparison similarity data sets. Complex size <br />                                            defined number unique proteins isolated complex; e. complex  <br />                                            tain 3 unique OGs 4 distinct protein components, yielding complex size 4.                                            complex, presence OG complex assayed proteome sets                                            seven representative species. resulting binary presence/absence values com- <br />                                            bined produce value percent complex conservation. value intentionally disre- <br />                                            gards gene context similarity   OG present genomes                                            neighboring genes differ genomes) simply expresses fraction complex <br />                                            components specific genome code  target proteome did contain                                            specified complex component, number paralogs component-coding gene  <br />                                            termined number proteins list mapping OG. verifica- <br />                                            tion, necessary define protein-coding genes true paralogs, simply <br />                                            used OGs (including paralogs) determined eggNOG. <br />                                                protein complex components assigned binary essentiality values using pub- <br />                                            lished assays specific species listed  values used define essentiali- <br />                                            ty fraction potentially conserved complex, e. E. coli complex 80%                                            components appear conserved M. pneumoniae 60% components                                            essential species. <br />                                                broader comparison prepared using list 894 species defined  Genome <br />                                            sizes species retrieved KEGG GENOME Database (http://www.genome. <br />                                            jp/kegg/genome.html, [25]). species, total number OG-mapped protein-coding <br />                                            loci divided total number loci produce value percentage mapped. Using <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                          19 / 23 <br />                                                                                                                    Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            list OGs species, OG compared species determine                                            conservation Bacteria. Adjusted average locus conservation particular genome, <br />                                            CAAL(g), calculated  <br />                                                                                               P        <br />                                                                                                     CL &#195;&#176;g&#195;&#382; <br />                                                                                                    L&#195;&#176;g&#195;&#382; <br />                                                                                   CAAL &#195;&#176; g &#195;&#382; &#194;&#188; m <br />                                                                                                    N <br />                                            CL number genomes locus present, L(g) number loci                                            genome, N total number genomes, m percentage loci mapped egg- <br />                                            NOG v.3. Values adjusted using fraction loci actually mapped unmapped loci <br />                                            lower effective conservation. <br />                                               identical list values, repeated OGs reduced single occurrence, aver- <br />                                            aged produce average OG conservation. modification removes effect counting <br />                                            loci share OGs, happen loci paralo- <br />                                            gous. Adjusted average OG conservation particular genome, CAAO(g), calculated  <br />                                                                                                P        <br />                                                                                                     CL &#195;&#176;g&#195;&#382; <br />                                                                                                    O&#195;&#176;g&#195;&#382; <br />                                                                                   CAAO &#195;&#176; g &#195;&#382; &#194;&#188; m <br />                                                                                                    N <br />                                            CL number genomes locus present, O(g) number unique <br />                                            OGs genome, N total number genomes, m percentage loci mapped <br />                                            eggNOG v.3. <br />                                               Species/strains sorted genome size compared average conservation val- <br />                                            ues. set bacterial genomes, N = 943, Fig. 1 presents results remov- <br />                                            ing 45 genomes similar size sequence. subset species, Average Locus <br />                                            Average OG Conservation values calculated using OGs published pro- <br />                                            tein complex data sets. <br />                                               Mapping fractional complex conservation species performed follows                                            focused set (8 species) large set. cladogram species set pre- <br />                                            pared using Interactive Tree Life (iTOL, [26]) project NCBI taxonomy. protein <br />                                            components mapped eggNOG v.3 OGs complex size determined defined <br />                                             Conservation fraction complex species defined number                                            complex component OGs shared model  E. coli complex) target ge- <br />                                            nome size model complex. Heatmaps prepared using R heatmap.2 <br />                                            function gplots package. Randomized models large set heatmaps (Fig. 5, S5 S6 <br />                                            Figs.) retaining species order randomized distribution conservation frac- <br />                                            tions prepared using R function randomizeMatrix  picante package [27])                                            &#226;&#8364;&#732;richness&#226;&#8364;&#8482; null model respect overall conservation levels. <br />  <br />  <br />                                            Supporting Information <br />                                            S1 Fig. EcoCyc complexes fractional conservation selected bacterial species. <br />                                            extended version Fig. 5. Names blue stars indicate example complexes shown                                            Fig. 4. <br />                                            (PDF) <br />                                            S2 Fig. EcoCyc complexes fractional essentiality selected bacterial species. <br />                                            extended version Fig. 5. Names blue stars indicate example complexes shown                                            Fig. 4. <br />                                            (PDF) <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                         20 / 23 <br />                                                                                                                  Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            S3 Fig. Cross-species conservation experimentally-observed protein complexes                                            sums counts potential paralogs components.   Proteins E. coli com- <br />                                            plexes [5] tend paralogs complexes conserved. (B) contrast,                                            M. pneumoniae complexes [6] conserved complexes tend paralogous pro- <br />                                            teins. Fraction conservation sum paralogy calculated described Materials <br />                                            Methods. node represents single protein complex relative size corresponding <br />                                            size complex number components. E. coli complexes defined Hu et al. <br />                                            compared M. pneumoniae proteome M. pneumoniae complexes                                            compared E. coli proteome; cross-species comparison using predicted <br />                                            orthologs described Materials Methods. <br />                                            (PDF) <br />                                            S4 Fig. Essentiality protein complexes average paralogy. essential protein <br />                                            complexes tend fewer components paralogs, E. coli literature-curated <br />                                            Ecocyc   experimentally-observed Hu et al. (B) complexes.  true                                            reduced genomes Mycoplasma pneumoniae (C). node represents single <br />                                            protein complex relative size corresponding size complex number com- <br />                                            ponents. Fraction essentiality average paralogy calculated described Materials <br />                                            Methods. Data Hu et al. [5]  , EcoCyc [13] (B) [6] (C). (D) example com- <br />                                            plex data sets shown. complexes identical composi- <br />                                            tion similar components. complex, defined single data set, offer                                            incomplete set protein components overlook impact paralogy. <br />                                            (PDF) <br />                                            S5 Fig. E. coli literature-curated complex conservation bacteria corresponds taxo- <br />                                            nomic boundaries. figure expands Fig. 6. Using EcoCyc set protein <br />                                            complexes model, column single complex set row distinct <br />                                            bacterial genome. 285 complexes 894 genomes shown total. Genome order corre- <br />                                            sponds cladogram produced using NCBI taxonomy Interactive Tree Life (iTOL, <br />                                            [26]) tree. Color gradients correspond fractional conservation. Complexes (columns)                                            clustered basis distance average fractional conservations (aver- <br />                                            age linkage). <br />                                            (PDF) <br />                                            S6 Fig. E. coli experimentally-observed complex conservation bacteria corresponds <br />                                            taxonomic boundaries. Using Hu et al. set protein complexes model, col- <br />                                            umn single complex set row distinct bacterial genome. 310 com- <br />                                            plexes 894 genomes shown total. Genome order corresponds cladogram <br />                                            produced using NCBI taxonomy Interactive Tree Life (iTOL, [26]) tree. Color gradi- <br />                                            ents correspond fractional conservation. Complexes (columns) clustered                                            basis distance average fractional conservations (average linkage). <br />                                            (PDF) <br />                                            S1 Text. Guide content provided supporting tables. <br />                                            (DOCX) <br />                                            S1 Table. Key protein complex IDs shown Fig. 5. <br />                                            (XLSX) <br />                                            S2 Table. Protein complex conservation bacteria. <br />                                            (XLSX) <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | DOI:10.1371/journal.pcbi.1004107 February 27, 2015                                                       21 / 23 <br />                                                                                                                              Protein Complexes Bacteria <br />  <br />  <br />  <br />                                            S3 Table. Conservation orthologous groups species pairs. <br />                                            (XLSX) <br />                                            S4 Table. Conservation numerous bacterial species. <br />                                            (XLSX) <br />                                            S5 Table. Conservation E. coli complexes Hu et al. (2009). <br />                                            (XLSX) <br />                                            S6 Table. Essentiality E. coli complexes Hu et al. (2009). <br />                                            (XLSX) <br />                                            S7 Table. Conservation E. coli complexes EcoCyc. <br />                                            (XLSX) <br />                                            S8 Table. Essentiality E. coli complexes EcoCyc. <br />                                            (XLSX) <br />                                            S9 Table. Conservation Mycoplasma pneumoniae complexes K&#195;&#188;hner et al. (2009). <br />                                            (XLSX) <br />                                            S10 Table. Essentiality Mycoplasma pneumoniae complexes K&#195;&#188;hner et al. (2009). <br />                                            (XLSX) <br />                                            S11 Table. Experimental protein complexes containing uncharacterized components. <br />                                            (XLSX) <br />  <br /> </body></html>