<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol.10 4 pp.1017- 1023, 1991 <br />  <br />  <br /> Identification sequence analysis gene <br /> mutated conditionally lethal outer membrane <br /> permeability mutant SS-C Salmonella typhimurium <br />  <br /> Laura Hirvas, Pertti Koski Martti Vaara                          <span id='am-5' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span> essential <span id='am-11' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="LPS" datatype="xsd:string"></span>LPS</span> bacterial <br />                                                                      cell, enzymes involved lipid biosynthesis <br /> Department Bacteriology Immunology, University Helsinki,   characterized molecular terms Haartmaninkatu 3, 00290 Helsinki, Finland <br />                                                                      genes identified (Raetz, 1990; Raetz Communicated B D.Stocker                                        Dowhan, 1990).  biosynthesis lipid <br />                                                                      -KDO region LPS novel, attractive target biosynthesis structure-function relationships                antibacterial therapy (Hogenauer Waisetschlager, enterobacterial outer membrane subjects                1981; Allen, 1985; Goldman et al., 1987; Hammond <br /> current intensive research. previously described             et al., 1987; Galloway Raetz, 1990; Raetz, 1990), antibiotic supersensitive SS-C mutant (SH7622)                research lipid biosynthesis Salmonella typhimurium shown outer                      increased considerably years. membrane permeability barrier hydrophobic                    lack detailed knowledge molecular biology antibiotics severely defective. study,            structure -function relationships relative mutant heat-sensitive, conditionally lethal,        impermeability OM hydrophobic compounds <br /> carries missense base-pair substitution novel gene            (Nikaido Vaara, 1987). functions recently reported named ssc                implicated main functions OM, gene. ssc encodes earlier uncharacterized 36 kd                   functions inhibited appropriately tailored <br /> protein  Ssc protein) mutant expresses Ssc               selected antibacterial agents (Vaara Vaara, 1983a,b). valine 291 changed methionine                         Mutants defective OM structure function <br /> methionine-rich region Ssc. plasmid containing              invaluable tools OM research wild-type ssc allele completely reverts antibiotic-              beginning, elucidation biosynthetic pathways heat-sensitive phenotype SS-C mutant. Cor-                OM components studies structure-function <br /> responding plasmids carrying mutant allele,                relationships (reviewed Makela Stocker, 1984; <br /> identical mutant allele prepared loalized mutagenesis,            Lehmann Hansen-Hagge, 1984; Nikaido Vaara, inactive. ssc gene probably analogous               1985; Sukupolvi Vaara, 1989). Novel mutants firA locus Escherichia coli earlier                expected shed light current puzzles OM research. <br /> implicated totally different function, mRNA                     relatively easy select, example, mutants <br /> synthesis. Furthermore, ssc apparently lies close            decreased OM permeability barrier. Ipx genes involved far known steps lipid          hand, spite intensive research, biosynthesis  distance, -560 bp). conclude,                 permeability mutants remained mysterious, basis findings define new essential gene involved             mutation determined DNA level. <br /> generation outer membrane.                                    mutants include abs, acrA, acrB, envC Key words: Escherichia coli/<span id='am-21' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-22' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span>lipopolysaccharide</span>/outer                 mutants E.coli SS-B SS-C mutants membrane permeability/Salmonella typhimuriumIssc                     S.typhimurium (reviewed Nikaido Vaara, 1985; <br /> mutation                                                             Sukupolvi Vaara, 1989). <br />                                                                          described SS-C mutant S.typhimurium 1984. <br />                                                                      supersensitivity number unrelated hydrophobic <br />                                                                      antibiotics  rifampicin, erythromycin, clindamycin, <br /> Introduction                                                          fusidic acid, vancomycin novobiocin) outer membrane (OM) Gram-negative enteric bacteria             hydrophobic drugs detergents indicated extensively studied comprehensively            defective permeability barrier. Unlike hyper- <br /> characterized biological membranes  review,                 permeable OM mutants, SS-C mutant detectable <br /> Nikaido Vaara, 1987). structure elaborate              alterations LPS. obvious changes lipopolysaccharide (LPS) constituent determined              phospholipids OM proteins. Furthermore, growth rate Escherichia coli Salmonella typhimurium              cell division morphology unaltered. model strains (Rietschel et al., 1984; Rick, 1987; Qureshi            molecular basis supersensitivity et al., 1988) primary structures OM            time (Sukupolvi et al., 1984, 1985). <br /> proteins E. coli S. typhimurium known.                     paper, serendipitously <br />     lack complete understanding                 discovered gene mutated SS-C mutant biosynthesis OM. translocation process OM           studying gene, ompH, recently  <br /> proteins synthesis site cytoplasmic mem-            cloned, sequenced (Koski et al., 1989, 1990). brane elucidated (Bolla et al., 1988;      mutation causes Val - Met change previously <br /> Lill et al., 1989; Randall Hardy, 1989). Practically              uncharacterized 36 kd protein, named Ssc protein. known export LPS outer surface         Furthermore, mutation conditionally OM (review Raetz, 1990). Furthermore,                  lethal; mutant does grow 42&#194;&#176;C. corresponding <br />                                                                                                                              101 7 <br />    Oxford University Press <br />  L.Hirvas, P.Koski M.Vaara <br />  <br /> gene, ssc, probably analogous firA gene E. coli           recently  cloned, sequenced (Koski et long linked cellular function quite                al., 1989, 1990). Accordingly, felt important different related OM, e. transcription          investigate possible relationship ompH gene <br /> (Lathe et al., 1980), firA mutants E. coli rpoB,            SS-C mutation. employed plasmid pUCHS16 carry rifampicin-resistant RNA polymerase, rifam-            (Figure 1) carries, ligated vector pUC 19, picin sensitive. Furthermore, ssc gene Ipx             entire ompH gene 2.3 kb genomic BamHI-SalI <br /> genes involved known steps lipid biosynthesis              fragment S.typhimurium SH5014. fragment separated open reading frame.                          contains open reading frame, ORF-36 (Hirvas et al., <br />                                                                        1990b). encodes far uncharacterized 36 kd protein. <br /> Results                                                                complete ORFs present fragment. <br />                                                                           pUCHS 16 introduced SH7622, antibiotic supersensitive thermosensitive                      antibiotic sensitive phenotype SH7622 completely reverted <br /> phenotype SS-C mutant                                           identical parent (Table II). ability reported earlier (Sukupolvi et al., 1984), SH7622             grow 42&#194;&#176;C resumed. Separate controls <br />  SS-C mutant) remarkably sensitive hydrophobic               indicated plasmid vector (pUC 19) did itself <br /> antibiotics. especially evident sensitivity          effect phenotype SH7622 (Table II). measured determining minimum inhibitory                     Furthermore, pUCHS 16 did revert phenotype concentrations (MICs) antibiotics (Table . MICs           unrelated supersensitive S. typhimurium mutant SH7616 hydrophobic drugs tested 16- 256-fold              mutation SS-B. <br /> lower SH7622 parent SH5014. observed                studies, utilizing pUCHS16 derivatives <br /> MICs low comparable corresponding                  pUCHS16A (lacks ORF-36, Figure 1) pUCHS 14-TI <br /> MICs enterobacterial OM permeability mutants             (lacks 5&apos; terminus ompH, Figure 1), indicated heptoseless LPS mutants (Vaara Vaara,                  plasmid carrying ORF-36 required complement 1983b) SS-B mutant (Vaara, 1990).                              phenotype SH7622 presence ompH    Furthermore, temperature             plasmid essential (Table III). Accordingly, susceptibility tests performed greatly affects degree          ORF-36 gene tentatively named ssc gene. sensitivity SH7622.  drugs MICs <br /> significantly lower (4- 16-fold) 39&#194;&#176;C 33&#194;&#176;C               Complementation ssc mutant alleles <br /> (Table ; 36&#194;&#176;C sensitivity intermediate (data          shown  pUCHS14-ll, carrying wild-type ssc <br /> shown). temperature-induced susceptibility increase               gene S.typhimurium SH5014 1.6 kb PstI-PstI common property permeability mutants                   fragment, effective complementation suggested SH7622 possibly regarded                  phenotype SS-C mutant SH7622. cor- <br /> temperature-sensitive mutant. studies indicated           responding 1.6 kb PstI-PstI fragment isolated SH7622 conditionally lethal mutant. does grow              SH7622, ligated pUC19  constitute plasmid 42&#194;&#176;C.  growth rate 370C indiscernible             pUCHS214, Figure 1), introduced SH7622, parent strain.                                        complementation SS-C phenotype observed (Table <br />                                                                        III). Accordingly, ssc gene SH7622 designated <br /> Complementation SS-C phenotype                                  sscl mutant allele. shown earlier SS-C mutation lies map positions 5 7 S.typhimurium chromosome                                               P             Q.                             (Sukupolvi et al., 1984). Alignment studies (Riley                                                                                           c: <br />                                                                                                04 <br />                                                                                                1        cI        0 <br />                                                                                                                        (DO4 <br />                                                                                                                         X        1C0tn) <br />                                                                                                                                    Cl            0 <br /> Krawiec, 1987) indicate 5&apos; region S. typhi-                                                     o-                         <br /> murium genome corresponds 4&apos; region E. coli                                                                     , <br />                                                                                                                                                    0( <br /> genome. E. coli genes 4&apos; include hlpA gene                                      Cl 0lCOl <br />                                                                          pUCHS16 <br /> Bachmann, 1990). shown (Hirvas et al., 1990a), <br />                                                                                                                               cc analogue ompH gene S. typhimurium                                                   g                             Q.        0~ <br />  <br />                                                                         pUCHS16A <br /> Table  antibiotic supersensitive phenotype SS-C mutant                        B             P                E <br /> SH7622 <br /> Antibiotic     MIC (/ig/ml)                           Ratioa            pUCHS1 4   IJ <br />                                                                         pUCHS14 <br />                SH5014            SH7622     SH7622    33&#194;&#176;C 39&#194;&#176;C         pUCHS214                         P                E                 p <br />                33&#194;&#176;CC 39&apos;C    33&apos;C       39&#194;&#176;C <br />                                                                        Fig. 1. Genomic fragments S.typhimurium pUCHS16 related <br /> Fusidic acid    2 512            16          2        -32      -256    plasmids. pUCHS16 carries 2.3 kb BamHI-SalI fragment Rifampicin          8             0.5        0.06     2 16     2 135   S.typhimurium SH5014 genome described previously <br /> Erythromycin       64             4          1          16        64   (Koski et al., 1990; Hirvas et al., 1990a,b). fragment contains <br />                                                                        putative promoter region ompH (-35 region starting nt 142), <br /> Clindamycin        64            32          2           2        32   structural ompH gene (nt 311 -796), structural ssc (= ORF-36) <br /> Vancomycin        256            64         16           4        16   gene (nt 800-1825) 5&apos; terminus open reading frame <br />                                                                        ORF-II  nt 1931). <span id='am-51' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-52' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-58' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> positions relevant restriction values minimum inhibitory concentrations (MICs) various     sites indicated. pUCHS16A pUCHS14-II derivatives hydrophobic antibiotics SH7622 parent strain SH5014     pUCHS16. pUCHS14 contains independently isolated 1.6 kb <br /> 33&#194;&#176;C 39&#194;&#176;C, determined L broth.                               PstI-PstI genomic fragment SH5014 (Koski et al., 1989). <br /> aThe approximate ratio MIC SH5014         pUCHS214 contains corresponding 1.6 kb fragment SS-C <br /> SH7622.                                                                mutant strain SH7622. <br /> 1018 <br />                                                                                           Lethal outer membrane mutant allele novel    ssc gene <br />  <br />  <br />  <br />    surprise, spontaneous ssc mutant                           pUCHS 14, did    produced new <br /> allele (ssc5l). discovered recognized                            protein having apparent molecular mass 14 kd. plasmid pUCHS 14 (Koski et al., 1989; Figure <br /> 1), carrying independently isolated  pUCHS16-                                 Nucleotide sequences mutant ssc genes <br /> derived) 1.6 kb PstI-PstI fragment S.typhimurium                                sequenced SH7622-derived, ompH- sscl- <br /> SH5014, did complement SS-C mutant phenotype                                 containing 1496 bp region pUCHS214 using (Table III). unexpected result                             sequencing strategy primers identical used contradictory obtained pUCHS 16-derived                          sequencing wild-type ompH ssc genes pUCHS14-ll plasmid.  results shown                                     S. typhimurium SH5014. Sequencing revealed mutation, <br /> indicate ssc gene pUCHS14 mutation,                            G - transition, sscl gene (codon 291, position explains discrepancy.                                                       1670 Figure 3). results change amino <br />                                                                                      acid sequence valine methionine. missense <br /> Expression ssc gene mutant alleles                                    mutation unexpected, ssc sscl previously shown ORF-36 gene  ssc                               code proteins mol. wt. changes gene) expressed minicells encodes protein                           observed calculated hydrophobicity secondary apparent molecular mass 36 kd (Hirvas et al., 1990b).                          structure Ssc SscMet291 corresponds molecular mass 35 929                                   Sequencing ssc5l-containing region pUCHS14 <br /> daltons, calculated ssc gene product. Furthermore,                           revealed frameshift mutation, deletion <span id='am-41' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span> codon expected basis deduced <span id='am-59' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-60' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> se-                           107 (position 1119 Figure 3). resulted formation <br /> quence data, detected preprotein forms                            TGA stop codon 17 codons downstream. calculated Ssc protein.                                                                     molecular mass truncated Ssc protein encoded ssc5l <br />    Figure 2, expression ssc mutant alleles                        13 397 daltons (37% wild-type Ssc) compared wild-type ssc. mutant                             corresponds apparent molecular mass determined <br /> allele sscl, present SS-C mutant SH7622, did encode                           using minicells described  36 kd protein, mutant allele ssc5l, present <br />                                                                                      Reconstruction ssc- 1 allele site-directed <br />                                                                                      mutagenesis <br /> Table II. effect plasmid pUCHS16 antibiotic <br />                                                                                      verify single change Val291 Met sensitive phenotype SS-C mutant SH7622                                        Ssc protein responsible phenotype sscl <br />                                                                                      mutant SH7622, reconstructed G        transition Antibiotic             MIC (pug/ml)                                                  mutagenizing wild-type ssc gene pUCHS16                        SH7622                SH7622                  Ratioa          17-mer mutagenic oligonucleotide bears transition <br />                        pUCHS16          pUC19                               Materials methods). Sequencing mutagenized <br />                         -512                   4                      2 128          ssc revealed desired mutation (sscl) achieved. <br /> Fusidic acid <br /> Rifampicin               - 16                  0.06                   2270           vitro-prepared sscl sscl&apos;. Erythromycin               64                  1                           64        sscl&apos;-carrying plasmid pUCHS14II-1 introduced Clindamycin                64                  4                           16        SH7622, complementation sscl phenotype Vancomycin               256                  16                           16        accomplished (data presented connection                                                                                      complementation data, Table III). finding confirms values MICs hydrophobic antibiotics SH7622 carrying                   sequence data. vitro-prepared missense mutations pUCHS16 control plasmid pUC19, determined 39&#194;&#176;C L                     locations ssc pUCHS16 did reduce <br /> broth containing ampicillin (100 yg/ml). <br /> aThe approximate ratio MIC SH7622/pUCHS16                        ability complement sscl mutation (data SH7622/pUC19.                                                               presented . <br />  <br /> Table III. Complementation antibiotic- heat-sensitive phenotype SS-C mutant SH7622 various plasmids containing ompH 3&apos; flanking open reading frame (ORF-36) <br />  Strain      Plasmid        Brief description insert&apos;     Sensitivity antibioticsb                                 Heat sensitivity <br />                             Complete chromosomal Gene origin Fusidic acid Rifampicin Erythromycin Clindamycin Vancomycin Loglo reduction <br />                             genes present                                                                                c.f.u.s 42OCC <br />  <br />  SH7622      pUC19                               -            23           26          19          18           6           &gt;5 <br />  SH7622      pUCHS16        ompH, ORF-36           SH5014   2                   11           6           0           0           &lt;0.3 <br />  SH7622      pUCHS16A       ompH                   pUCHS16 23                   24          20          12          10           &gt;5 <br />  SH7622      pUCHS14-II     ORF-36                 pUCHS16 2                     9           5           0           0           &lt;0.3 <br />  SH7622      pUCHS14        ORF-36                 SH5014  26                   33          21          19          10             4.2 <br />  SH7622      pUCHS214       ORF-36                 SH7622  21                   26          17          14           7           &gt;5 <br />  SH7622      pUCHS14-II-1   ORF-36                 d                19          24          18          11           8           &gt;5 <br />  SH5014                -                       -                4          11           8           0           0           &lt;0.3 <br />  <br />  aSee Figure    1 Materials methods      detailed data. <br />  bMeasured diameter  millimeters) inhibition zone antibiotic disc. diameter disc itself subtracted.  tests performed 39&#194;&#176;C L agar containing ampicillin (100 Ag/ml; SH5014 ampicillin omitted). <br />  cCalculated logl0 (c.f.u.s 36&#194;&#176;C/c.f.u.s 42&#194;&#176;C). c.f.u., colony formingA unit. <br />                                                                                   transition nucleotide codon 291 ORF-36.  dpUCHS14-I1-1   obtained   site-directedmutagenesis carries G      - <br />  <br />  <br />  properties results obtained described later section Results. <br />                                                                                                                                                 1019 <br />   L.Hirvas, P.Koski M.Vaara <br />  <br />  <br />       kDa <br />                                                                          pUCHS16 produce Ssc major protein (P.Koski, <br />                                                                          L.Hirvas M.Vaara, unpublished). Ssc overproducer <br />                                                                          strains obtained placing ssc control <br />                                                                          strong, regulated promoter. constructions  <br />        36                                                                 necessarily stable. shown Figure 3,        32 <br />                                                                          ssc   gene pUCHS 14, plasmids used        30-                                                                present study, base deletion resulted                                                                           formation truncated Ssc protein. reason                                                                           spontaneous appearance inactive plasmid ssc                                                                           studied. <br />                                                                             ssc gene located immediately downstream ompH, <br />                                                                          gene encoding new OM protein recently described                                                                          (Koski et al., 1989, 1990).                                                                          nucleotides stop codon ompH                                                                          initiation codon ssc. results present paper <br />                                                             _ 14         indicate ssc expressed absence ompH <br />                                                                          promoter, e. plasmid pUCHS14-ll lacks 5&apos; <br />                  1           2                       2 <br />                                                                          terminus ompH. Furthermore, orientation <br />                                                                          ssc pUCHS14-II  Materials methods),                                                                          expression promoter plasmid vector <br />                                              B                          results strongly suggest ssc promoter <br />                                                                           putative promoter region  Fig. 2. Proteins synthesized Ecoli minicells harboring pUCHS14-II     159 nucleotides upstream, overlapping protein- <br />  (panel  lane 1), control plasmid pUC19 (panel  lane 2),         encoding region ompH (Hirvas et al., 1990b). <br />  pUCHS214 (panel B, lane 1) pUCHS14 (panel B, lane 2).             gene organization ompH-ssc region S. typhi- <br />  pUCHS14-II pUCHS214 encode 36 kd protein                  murium corresponding region E. coli  pUCHS14 encodes 14 kd truncated protein. <br />                                                                          shown Figure 4. ssc probably analogous E. coli <br /> Discussion                                                               genefirA, sequence published. <br />                                                                          According recent data  Thome et al., 1990; <br />  reviewed Introduction, current OM research              Bachmann, 1990) contrast previous reports <br />  greatly benefit discovery novel mutants                (Aasland et al., 1988; Kroger et al., 1990), firA <br />  characterized mutations affecting function OM.               gene distinct hlpA  named skp) does  shown mysterious SS-C mutant (strain              code 17 kd HlpI (Skp) protein E. coli <br />  SH7622) S. typhimurium severe OM                         analogue OmpH protein S.typhimurium (Hirvas <br />  permeability barrier defect, carries missense base-pair               et al., 1990a). firA known located hipA <br />  substitution mutation novel gene recently            open reading frame encoding 17.4 kd protein <br />  sequenced encode previously uncharacterized              (ORF-17.4)  Bachmann, 1990). distance  36 kd protein. basis, suggest gene            hlpA ORF-17.4 estimated 1.1 kb (Aasland called ssc, product, Ssc protein.                         et al., 1988; Hirvas et al., 1990a)     mutated Ssc protein Val291 changed                        analogous S. typhimurium genes ompH ORF-II methionine. hydropathy indices methionine valine              1125 bp (Hirvas et al., 1990b). Ninety cent reasonably close (Kyte Doolittle, 1982),           (1020 bp) space S. typhimurium occupied helix, fl-sheet coil conformational             ssc structural gene. hlpA ompH 87% <br /> potentials helix-boundary f-sheet-boundary               homologous (Hirvas et al., 1990a) ORF-17.4 <br /> forming potentials (Chou Fasman, 1978). Consequently,               ORF-II 90% homologous (Hirvas et al., 1990b), calculated hydropathicity profiles secondary                    expected firA ssc genes structures Ssc SscMet291 similar. basis              considerably homologous,   sequence data, tempting speculate Ssc            169 bp fragment immediately downstream E. coli hlpA enzymatic function Val291 impor-                 known (Holck Kleppe, 1988). 86% homologous <br /> tant catalytic activity. Interestingly, region span-        corresponding 5&apos; terminus structural ssc gene <br /> ning residues 288-292 unusually rich                     (Hirvas et al., 1990b). <br /> methionine contains methionine Ssc                   firA mutants described Lathe et al. (1980). SscMea9l. currently modifying region             selected rifampicin-sensitive phenotypic revertants regions Ssc localized mutagenesis order               (hencefir versus rif) E. coli rpoB strain carries gain information amino acid residues essential            rifampicin-resistant RNA polymerase. E. coli rpoB function Ssc. Similarly, analysis set            resistant rifampicin target mutation (MIC <br /> spontaneous temperature-resistant revertants SH7622               - <br />                                                                             300 ytg/ml), MIC rifampicin rpoB, firA expected valuable data functionally                strain 3-fold lower (Lathe et al., 1980). Furthermore, <br />                                                                                     - <br />  <br />  <br />  <br /> important domains Ssc, possible bypass                 thefirA, rpoB strain heat-sensitive RNA synthesis <br /> mutations,  analysis started               halted 42&#194;&#176;C. Based preliminary evidence, <br /> laboratory.                                                             firA mutation thought affect RNA transcription <br />    work involves purification Ssc protein.          (Lathe et al., 1980). sensitization firA <br /> JM109 derivatives carrying ssc gene pUCHS 14-I1               mutant rifampicin decreased OM permeability <br /> 1020 <br />                                                                                     Lethal outer membrane mutant allele novel ssc gene <br />  <br />                                        782   AAACAGGTTAAATAAGTAATGCCTTCAATTCGACTGGCTGACTTAGCAGAACAGTTGGAT <br />                                              LysGlnValLys***       MetProSerIleArgLeuAlaAspLeuAlaGluGlnLeuAsp <br />                                        842   GCAGAATTACACGGTGATGGCGATATCGTCATCACCGGCGTTGCGTCCATGCAATCTGCA <br />                                              AlaGluLeuHisGlyAspGlyAspIleValIleThrGlyValAlaSerMetGlnSerAla <br />                                        902   ACAACAGGCCACATTACGTTTATGGTGAATCCTAAGTACCGTGAACACTTAGGTTTATGC <br />                                              ThrThrGlyHisIleThrPheMetValAsnProLysTyrArgGluHisLeuGlyLeuCys <br />                                        962   CAGGCTTCTGCGGTTGTCATGACGCAGGACGATCTTCCTTTTGCTAAGAGTGCGGCGCTG <br />                                              GlnAlaSerAlaValValMetThrGlnAspAspLeuProPheAlaLysSerAlaAlaLeu <br />                                       1022   GTAGTTAAAAATCCCTACCTGACCTACGCGCGCATGGCGCAAATTTTAGATACTACGCCG <br />                                              ValValLysAsnProTyrLeuThrTyrAlaArgMetAlaGlnIleLeuAspThrThrPro <br />                              ssc5l                   .                            - <br />                                       1082   CAGCCCGCGCAGAATATCGCGCCAAGCGCCGTGATTGATGCGACGGCAACGCTGGGTAGC <br />                                              GlnProAlaGlnAsnIleAlaProSerAlaValIleAspAlaThrAlaThrLeuGlySer <br />                              Ssc51                                                        ValArgArgGlnArgTrpValAl <br />                                       1142   AATGTTTCAGTCGGCGCGAATGCGGTGATTGAATCTGGCGTACAACTGGGCGATAACGTG <br />                                              AsnValSerValGlyAlaAsnAlaValIleGluSerGlyValGlnLeuGlyAspAsnVal <br />                                              aMetPheGlnSerAlaArgMetArg*** <br />                                       1202   GTTATCGGCGCAGGCTGTTTCGTCGGAAAAAATAGCAAAATCGGGGCGGGTTCACGCTTG <br />                                              ValIleGlyAlaGlyCysPheValGlyLysAsnSerLysIleGlyAlaGlySerArgLeu <br />                                      1262    TGGGCGAACGTAACGATTTACCACGACATTCAGATCGGTGAGAATTGCCTGATCCAGTCC <br />                                              TrpAlaAsnValThrIleTyrHisAspIleGlnIleGlyGluAsnCysLeuIleGlnSer <br />                                      1322    AGTACGGTGATCGGCGCGGACGGTTTTGGCTACGCTAACGATCGTGGCAACTGGGTGAAG <br />                                              SerThrValIleGlyAlaAspGlyPheGlyTyrAlaAsnAspArgGlyAsnTrpvalLys <br />                                      1382    ATCCCACAACTGGGCCGGGTCATTATTGGCGATCGTGTCGAGATCGGCGCTTGTACCACC <br />                                              IleProGlnLeuGlyArgValIleIleGlyAspArgValGluIleGlyAlaCysThrThr <br />                                      1442    ATTGACCGTGGCGCGTTGGATGATACTGTTATTGGCAATGGCGTGATTATTGATAATCAG. <br />                                              IleAspArgGlyAlaLeuAspAspThrValIleGlyAsnGlyValIleIleAspAsnGln <br />                                      1502    TGCCAGATTGCACATAACGTCGTGATTGGCGACAATACGGCAGTTGCCGGTGGCGTCATT <br />                                              CysGlnIleAlaHisAsnValValIleGlyAspAsnThrAlaValAlaGlyGlyvalIle <br />                                      1562    ATGGCGGGTAGCCTGAAGATTGGCCGTTACTGCATGATTGGCGGCGCCAGCGTGATCAAT <br />                                              MetAlaGlySerLeuLysIleGlyArgTyrCysMetIleGlyGlyAlaSerValIleAsn <br />                             sscl                                                                                                            1622    GGGCATATGGAAATATGCGACAAAGTCACGGTAACTGGCATGGGTATGGTGATGCGTCCC <br />                                              GlyHisMetGluIleCysAspLysValThrValThrGlyMetGlyMetValMetArgPro <br />                            Sscl                                                                        Met <br />  <br />                                      1682    ATCACGGAACCGGGCGTCTACTCCTCAGGCATTCCGCTGCAACCCAACAAAGTATGGCGT <br />                                              IleThrGluProGlyValTyrSerSerGlyIleProLeuGlnProAsnLysValTrpArg <br />                                      1742    AAAACTGCTGCACTGGTGATGAACATTGATGATATGAGCAAGCGTCTCAAAGCGATTGAG <br />                                              LysThrAlaAlaLeuValMetAsnIleAspAspMetSerLysArgLeuLysAlaIleGlu <br />                                      1802    CGCAAGGTTAATCAACAAGACTAACGTTCCGCCTTGTAGTTGCCATTCTTTTCCGGCCTG <br />                                              ArgLysValAsnGlnGlnAsp*** <br />  <br /> Fig. 3. Nucleotide sequences ssc gene (Hirvas et al., 1990b) sscl ssc5l mutant alleles. mutated nucleotides ssc5l sscl indicated wild-type nucleotide sequence. shown deduced amino acid sequences Ssc protein mutant <br /> proteins. putative Shine-Dalgarno sequence initiation codon underlined. numbering consistent gene map <br /> shown Figure 1. sequence data presented available GenBank database accession number M35193. <br />  <br /> barrier function studied. Furthermore,                       Raetz, 1981) lpxA discovered course <br /> sensitivities firA mutant hydrophobic                        sequencing lpxB (Crowell et al., 1986).  antibiotics published. ssc firA                    enzymes involved lipid synthesis genes analogous functions, remains studied.                  purified, genes mapped  probable, phenotypes sscl allele              sequenced. include <span id='am-66' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-67' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UDP</span>-3-monoacyl-N-glucosamine firA, rpoB + alleles need necessarily identical                   acyltransferase (Raetz Dowhan, 1990), lipid 4&apos; kinase mutations differ extent                        (Ray Raetz, 1987) KDO2-lipidIVA lauroyl- <br /> functional lesion produced.                                                  myristoyltransferases (Brozek Raetz, 1990).    Figure 4 shows ssc located close                 lack mutants conditionally defective Ipx cluster. lpxA lpxB genes encode UDP-N-acetyl-                        enzymes greatly hampered research. <br /> glucosamine acetyltransferase lipid disaccharide                          previous studies did reveal detectable <br /> synthase, respectively; enzymes lipid                    alterations LPS sscl mutant (Sukupolvi et al., <br /> biosynthesis (Crowell et al., 1987; Coleman Raetz,                       1985).  time, did know 1988). distance ssc lpxA - 560 bp.                   mutant thermosensitive, studied LPS clustering intimate ssc lipid                 produced 37&#194;&#176;C. OM sscl mutant <br /> synthesis machinery. lipid -KDO                       obviously defective 37&#194;&#176;C evidenced essential LPS bacterial viability growth.                mutant&apos;s sensitivity antibiotics 37&#194;&#176;C 33&#194;&#176;C <br /> biosynthesis systematically extensively studied,                (Table , relative proportion defective LPS molecules <br /> mainly Raetz collaborators, considerable break-                   abnormal precursors  exist) larger <br /> throughs years. lpxA                  42&apos;C, lethal temperature, 37&#194;&#176;C. Accordingly, lpxB genes products                        currently studying lipid -KDO precursors <br /> characterized. lpxB discovered aid E. coli                   synthesized sscl mutant temperature shift <br /> mutants accumulated novel phospholipids (Nishijima                     42&#194;&#176;C. Furthermore, enzymes <br />                                                                                                                                              1021 <br />   L.Hirvas, P.Koski M.Vaara <br />  <br />                                                                                           BamHI-EcoRV ligating ompH-containing 863 bp fragment                                                                                           BamHI -Hincll digested pUC 19. contains entire ompH gene                                                                                         22 codons ssc. <br />                          U). UU) <br />                                                                                              pUCHS14-II lacks ompH  promoter Shine-Dalgarno <br />                        8.           . <br />                                                                  S1 <br />                                                                  0.                lu <br />                                                                                       regions seven codons) contains entire ssc gene. <br />                                       ,    1. <br />  <br />  <br />  <br />                                     - <br />                                                                                           prepared digesting pUCHS16 PstI ligating released <br />                                                                                           1.6 kb PstI fragment PstI-opened pUCHS19. hybrid plasmid                                                                                           contained Salmonella-derived DNA orientation                               topA                       firA    ORF <br />                                                                  17.4 <br />                                                                             IpxA          pUCHS16  e. produced 0.55 kb fragment BamHI-EcoRV diges- <br />                                                                                           tion) selected. <br />                                                                                              pUCHS14 (Koski et al., 1989) carries independently cloned 1.6 kb <br />                                                                                           PstI fragment S.typhimurium SH5014 chromosome, ligated pUC18. <br />                                                                                           fragment corresponds 1.6 kb PstI fragment present pUCHS16 <br />                                itSI           WI <br />                                               ccili             0, <br />    B <br />                      ,         0. <br />  <br />  <br />                                                                                  1. <br />                                                                                                                     IIF <br />                                                                  . <br />                                                                         S <br />                                                                         <br />                                                                                           derivative pUCHS 14-I1 differs having lac <br />                                                                                           promoter proximal ssc gene orientation ssc. <br />                                                                                              Plasmids pUCHS214 pUCHS14-II-l described  <br />                                II <br />                                .npH <br />                               Ci <br />                                Ii <br />                                                                                                         ssc <br />                                                                                                         i3 <br />                                                                 IORF <br />                                                                 ,II911 <br />                                                                                           Cloning ssc gene SH7622 <br />                                                                                           DNA isolated S. typhimurium SH7622 exactly method                                                                               0.5 kb      employed earlier (Koski et al., 1989). digested PstI, run                                                     1 <br />                                1                                                          0.7% agarose gel, analysed Southern hybridization using                                                    <br />                                                                                           ompH specific oligonucleotide probe (probe Koski et al., 1989).                                                                                                                                                                                               1.6 kb fragment, detected probe, ligated pUC19,                                                        C                           87%                1.   ND      90%                      orientation pUCHS16 pUCHS14-II  , named <br />                                                                                           pUCHS214. <br />                                                                                          Site-directed mutagenesis <br />  Fig. 4.   B) Alignment hlpA-firA-ORF-17.4-lpxA region                       produce site-directed mutagenesis ssc allele bears  Ecoli (panel  corresponding region Styphimurium.                      G - transition identical sscl, mutagenic oligonucleotide <br />  sequence firA published. conserved restriction                      5&apos;-GACGCATCATCATACCC-3&apos; prepared BamHI-SalI <br />  sites indicated asterisk (*). (C) Nucleotide identity               fragment pUCHS16 cloned corresponding sites M13mp8. <br />  gene areas shown panels   (B). ND, determined.                         Site-directed mutagenesis performed method Nakamaye                                                                                          Eckstein (1986) using oligonucleotide mutagenesis kit version 2 involved lipid biosynthesis lack characterization,                            Amersham conditions recommended manufacturer. reviewed  enzymatic activity suitably                                  presence desired mutation verified sequencing primer <br /> prepared cell fractions E.coli assayed using                                5&apos;-CAATGTTCATCACCAGT-3&apos;. 1.4 kb PstI fragment mutant <br /> appropriate lipid precursors substrates (Raetz, 1990).                              M13 DNA ligated PstI-opened pUC19, orientation <br />                                                                                          pUC19 recombinant plasmids described  resultant <br /> Consequently, sscl mutant lacks activities,                           plasmid named pUCHS14-II-1. lack detected assays. <br />     mechanism defective Ssc protein                                Antibiotic susceptibility determinations <br /> mediates OM permeability increase growth-permissive                               MICs antibiotics measured L broth containing  litre): <br /> temperature growth inhibition 42&#194;&#176;C, remains                                   tryptone (Difco Laboratories, Detroit, MI), 10 g; yeast extract (Oxoid, <br />                                                                                          Columbia, MD), 5 g; NaCl, 5 g (pH 7.2). inoculum L agar-grown elucidated. elucidation certainly yield                                  cells, inoculum size 104 cells/nil. incubation 18 h <br /> considerable new data function OM                                   37&#194;&#176;C. lowest concentration completely inhibited visual growth form useful basis future development                                  recorded interpreted MIC. following antibiotics pharmacological inhibitors OM biosynthesis.                                           used: rifampicin fusidic acid (Na), (Sigma <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> , St Louis, <br />                                                                                          MO); erythromycin ethylsuccinate, Orion, Helsinki, Finland; clindamycin <br />                                                                                          hydrochloride, Upjohn, Kalamazoo, MI; vancomycin hydrochloride, <br />  Materials methods                                                                   Eli Lilly , Indianapolis.  <br />                                                                                             sensitivity antibiotics measured agar diffusion <br /> Bacterial strains                                                                        method, antibiotic discs (Neo-Sensitabs) S Rosco (Copenhagen, <br /> S.typhimurium LT2 strain SH5014 (ilv.1178, thr914, his6116, metA22,                      Denmark), L agar containing ampicillin (100 fg/ml) stated, <br /> metE551, trpB2, xy1404, Hi-b, H2-e,nx, flaA66, rpsL120, rfaJ4041 causing                 Kirby-Bayer methodology  described disc manufacturer) <br /> LPS chemotype Rb2) typical characterized representative                 used. discs rifampicin (30 pg), fusidic acid (400 Mg), <br /> rough S.typhimurium (Stocker et al., 1979). SH7622 antibiotic                     erythromycin (78 JLg), clindamycin (25 M,g) vancomycin (70 Mg). <br /> supersusceptible class SS-C mutant (Sukupolvi et al., 1984). antibiotic <br /> supersusceptible phenotype SH7622 repeatedly rechecked using                   Testing heat sensitivity <br /> antibiotic discs agar diffusion method  . SH7616                 Serial 10-fold dilutions bacterial suspensions (109 cells/ml) antibiotic supersusceptible class SS-B mutant SH5014 (Sukupolvi et al.,               0.9% NaCl 50 f1 dilution plated L agar plates. 1984; Vaara, 1990). bacterial strains preserved -70&#194;&#176;C 20%                 15 h incubation, colonies (c.f.u.s) developed counted. skimmed milk.                                                                            reduction viability 42&#194;&#176;C expressed logl0 ratio <br />   primary host plasmids Ecoli JM109. plasmids                       colony count 36&#194;&#176;C 42&#194;&#176;C. introduced S.typhimurium, restriction mutant LB5010 S.typhimurium (Bullas Ryu, 1983) used intermediate host                    Expression minicells plasmids.                                                                      plasmid-encoded proteins studied using Ecoli PK251 minicells <br />                                                                                          described earlier (Koski et al., 1990). Briefly, minicells isolated <br />  Plasmids                                                                                method Dougan Kehoe (1984), labeled [35S]methionine <br />  pUCHS16 (Koski et al., 1989; Figure 1) carries cloned 2.3 kb               (Amersham; specific activity, 1 Ci/mol; final concentration, 150 ftCi/ml) <br /> BamHI-SalI fragment S.typhimurium SH5014 chromosome, ligated                       boiled 3.5% SDS, 10% mercaptoethanol. SDS-PAGE (15% cloning vector pUC19 (Pharmacia, Uppsala, Sweden), contains                   polyacrylamide) followed soaking AmplifyTM (Amersham) entire ompH ssc genes, 506 bp downstream region.                    drying autoradiography. complete nucleotide sequence pUCHS16 determined (Koski <br /> et al., 1989, 1990; Hirvas et al., 1990b). plasmid, lac promoter             DNA sequencing vector distal ompH ssc genes opposite orien-             Sequencing plasmid DNA denatured plasmids (Hattori <br /> tation. pUCHS16A (Figure 1) prepared digesting pUCHS16                       Sakaki, 1986) dideoxyribonucleotide chain termination method <br />  <br /> 1022 <br />                                                                                          Lethal outer membrane mutant allele novel ssc gene <br />  <br /> (Sanger et al., 1977) SequenaseTM (United States Biochemical,                 Mizusawa,S., Nishimura,S. Freela,F. (1986) Nucleic Acids Res., 14, <br /> Cleveland, OH)  35S]<span id='am-36' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-37' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span>dATP</span>. Sequencing strategy primers used                   1319-1324. sequencing ompH ssc genes S. typhimurium SH7622              Nakamaye,K. Eckstein,F. (1986) Nucleic Acids Res., 14, 9679-9698. <br /> ssc gene pUCHS 14 identical used earlier sequencing         Nikaido,H. Vaara,M. (1985) Microbiol. Rev., 49, 1-32. wild-type ompH ssc genes S.typhimurium SH5014 (Koski et al.,            Nikaido,H. Vaara,M. (1987) Neidhardt,F.C., Ingraham,J.L., <br /> 1989, 1990; Hirvas et al., 1990b). Occasional problems arising                   Low,K.B., Magasanik,B., Schaechter,M. Umbarger,H.E. (eds), <br /> compressions gel overcome substituting 7-deazaguanosine                Escherichia coli Salmonella typhimurium-Cellular Molecular <br /> 5&apos;-<span id='am-46' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-47' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span>triphosphate</span> (Mizusawa et al., 1986) <span id='am-31' about='obo:IMR_0200286' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00286" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="dGTP" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0200286" datatype="xsd:string"></span>dGTP</span>.                                     Biology. ASM Publications, Washington DC, pp. 7-22. <br />                                                                                    Nishijima, <br /> </body></html>