<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             DNA Repair (Amst). 2008 October 1; 7(10): 1731&#226;&#8364;&#8220;1745. doi:10.1016/j.dnarep.2008.06.019. <br />  <br />  <br />  <br />                            Substrate Binding Pocket Residues Human Alkyladenine-DNA <br />                            Glycosylase Critical Methylating Agent Survival <br />                            Cheng-Yao Chena, Haiwei H. Guoa, Dharini Shahb,  Blanka, Leona D. Samsonb,                            Lawrence  Loeba,* <br />                            Lawrence  Loeb: laloeb@u.washington.edu <br />                            aJosephGottstein Memorial Cancer Research Laboratory, Department Pathology, University                            Washington, Seattle, WA 98195-7705, USA <br />                            bDepartmentof Biological Engineering Center Environmental Health Sciences, <br />                            Massachusetts Institute Technology, Cambridge, Massachusetts 02139, USA <br />  <br />                            Abstract <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  Human alkyladenine-DNA glycosylase (AAG) initiates base excision repair (BER) alkylated <br />                                  deaminated bases DNA.  assessed mutability AAG substrate binding <br />                                  pocket, essentiality individual binding pocket amino acids survival methylation <br />                                  damage. used oligonucleotide-directed mutagenesis randomize 19 amino acids, 8                                  interact substrate bases, created 4.5 million variants. expressed mutant <br />                                  AAG&apos;s repair-deficient E. coli selected protection cytotoxicity                                  methylmethane sulfonate (MMS) methyl-lexitropsin  lex), agent produces 3- <br />                                  methyladenine predominant base lesion. Sequence analysis 116 methylation-resistant <br />                                  mutants revealed substitutions highly conserved Tyr127and His136. contrast,                                  mutation, L180F, greatly enriched MMS- lex-resistant libraries. <br />                                  Expression L180F single mutant conferred 4.4-fold enhanced survival high dose                                  MMS used selection. homogeneous L180F mutant enzyme exhibited 2.2-fold reduced <br />                                  excision 3-methyladenine 7.3-fold reduced excision 7-methylguanine methylated <br />                                  calf thymus DNA. Decreased excision methylated bases mutant glycosylase NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  promote survival high MMS concentrations, capacity downstream enzymes                                  process toxic BER intermediates saturated. mutant displayed 6.6-, 3.0-fold <br />                                  reduced excision 1,N6-ethenoadenine <span id='am-44' about='obo:IMR_0200173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-45' property="oboInOwl:id" content="IMR:0200173" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Hypoxanthine" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00262" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>hypoxanthine</span> oligonucleotide substrates, <br />                                  respectively, 1.7-fold increase binding abasic site-containing DNA. work provides <br />                                  vivo evidence substrate binding mechanism deduced crystal structures, illuminates <br />                                  function Leu180 wild-type human AAG, consistent role balanced <br />                                  expression BER enzymes damage survival. <br />  <br />  <br />  <br />  <br />                            &#194;&#169; 2008 Elsevier B.V. rights reserved. <br />                            * <br />                             Corresponding author. Tel.: 206 543 6015; fax: 206 543 3967. <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service                            customers providing early version manuscript. manuscript undergo copyediting, typesetting, review                            resulting proof published final citable form. note production process errors                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Chen et al.                                                                                                Page 2 <br />  <br />  <br />  <br />                               Keywords <br />                                     Alkylating agents; base excision repair; 3-methyladenine; 7-methylguanine; methyl-lexitropsin; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     random mutagenesis <br />  <br />  <br />                               1. Introduction <br />                                               Alkylating agents widely dispersed environment, occur endogenously,                                               frequently used treatment cancer [1,2]. repaired, alkylated bases DNA                                               mutagenic cytotoxic [3]. example, modification N3 position <span id='am-11' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span>                                               disrupt interaction DNA polymerases template DNA strand [4,5], apparently <br />                                               leading arrest replication machinery cell death [6-8]. Base excision repair <br />                                               (BER) primary mechanism cells remove damaged bases genome, <br />                                               including alkylated bases, conserved evolution. BER pathways initiated <br />                                               DNA glycosylase recognizes adducted base hydrolyzes N-glycosylic <br />                                               bond linking damaged base sugar-phosphate backbone [9]. resulting abasic <br />                                               site processed AP endonuclease Ap lyase, DNA polymerase DNA ligase <br />                                               activities [10]. Importantly, abasic sites downstream interrupted-strand intermediates <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               lead double-strand breaks potentially cytotoxic. DNA glycosylase <br />                                               action commits cells pathway step produces different possibly <br />                                               lethal form damaged DNA, repair complete. consequence, balanced <br />                                               expression [11,12] coordinated action [10] enzymes BER pathways important <br />                                               survival DNA damage. <br />  <br />                                               Alkyladenine DNA glycosylases organisms ranging bacteria plants                                               humans [2,13]. alkyladenine-DNA glycosylase AAG,  known N-methylpurine <br />                                               DNA glycosylase (MPG), ANPG] known human DNA glycosylase                                               removes alkylated bases [1,9]. Human AAG active broad range chemically                                               structurally diverse substrates. addition 3-methyladenine (3-meA), AAG excises <br />                                               alkylated purines 7-methylguanine (7-meG) 3-methylguanine (3-meG) [13];                                               oxidative purine deamination products hypoxanthine, <span id='am-49' about='obo:IMR_0200439' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-50' property="oboInOwl:id" content="IMR:0200439" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Xanthine" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="KEGG:C00385" datatype="xsd:string"></span>xanthine</span>, oxanine [14,15];                                               <span id='am-5' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> peroxidation-derived 1,N6-ethenoadenine (&#206;&#181;A) 1,N2-ethenoguanine (&#206;&#181;G);                                               oxidative lesion 8-hydroxyguanine [16];  inefficiently, normal <span id='am-54' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-55' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span>guanine</span> [13,17-19]. <br />                                               AAG reported excise hypoxanthine efficiently, followed &#206;&#181;A 3-meA, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               leading suggestion AAG primarily evolved hypoxanthine-DNA <br />                                               glycosylase [20,21]. <br />  <br />                                               Crystallographic studies molecular modeling [22,23], site-specific <br />                                               mutagenesis [24], provided substantial information concerning mechanism                                               action human AAG structural basis binding adducted purines. crystal <br />                                               structures AAG complexed lesion-containing DNA reveal AAG, like                                               DNA glycosylases, uses <span id='am-16' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-17' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> flipping mechanism recognize target bases                                               extrahelical conformation [22,23]. AAG binds minor groove DNA, single <br />                                               tyrosine (Tyr165) inserts minor groove, intercalating bases adjacent  <br />                                               occupying space left  flipped nucleotide. flipped nucleotide                                               captured substrate binding pocket,  case &#206;&#181;A, stacks chains <br />  <br />  <br />                                                  DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                              Page 3 <br />  <br />  <br />                                           Tyr127 His136 Tyr159 [23]. Cleavage glycosylic <br />                                           bond catalyzed nucleophilic water molecule activated essential Glu125 <br />                                           [20,23]. evolutionarily conserved amino acids form substrate binding pocket <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           [22,24], site-directed mutagenesis begun illuminate functions. example, <br />                                           Asn169 proposed play important role substrate specificity. normal <br />                                           guanine modeled substrate-binding pocket, exocyclic amino group clashes <br />                                           Asn169 chain [23]; accord, replacements Asn169 enhance excision                                           guanine mispairs [25]. substrate binding pocket residues include Leu180,                                           lies adjacent N3 position &#206;&#181;A, expected pack closely methyl <br />                                           adduct 3-meA [22]. <br />  <br />                                           present understanding roles individual substrate binding pocket residues                                           substrate discrimination AAG, importance damage survival, incomplete. <br />                                            used oligonucleotide-directed random mutagenesis (e.g., 26) selection                                           methylating agent survival examine replaceability amino acids line AAG <br />                                           substrate binding pocket. Analysis methylation-resistant mutants identified mutable                                           immutable amino acids, yielding vivo evidence essential role Tyr127 His136 <br />                                           survival methylation damage. results provide functional validation NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           crystallographically observed interactions, suggest functions Leu180 substrate <br />                                           discrimination wild-type human AAG, provide presumptive example                                           importance damage survival coordinated processing intermediates BER <br />                                           pathways. <br />  <br />                               2. Materials methods <br />                               2.1. Strains materials <br />                                           alkylation repair-deficient strain E. coli MV1932 (alkA1 tag) derivative strain <br />                                           AB1157  [12]]. E. coli BL21(DE3) purchased Agilent technologies (Santa <br />                                           Clara, CA). Enzymes NEB (Beverly, MA), oligonucleotides IDT <br />                                           (Coralville, IA). E. coli endonuclease IV kindly provided Dr. S. Bennett (Oregon <br />                                           State University). Anti-AAG (MPG) antibody obtained Imgenex (San Diego, CA). <br />  <br />                               2.2. Creation pUC-AAG-SK KX mutant libraries <br />                                           starting vector encoded length human AAG cDNA pUC19-derived plasmid. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           schematic diagram construction pUC-AAG-SK pUC-AAG-KX random <br />                                           mutant libraries shown Supplementary Fig. S1. library designations SK KX <br />                                           refer SacII KpnI, KpnI XbaI silent restriction sites, respectively,                                           introduced AAG coding sequence allow substitution oligonucleotide cassettes <br />                                           containing randomized codons. SK library encodes 9 randomized amino acids (Ala126, <br />                                           Tyr127, Leu128, Gly129; Glu133, Ala134, Ala135, His136, Ser137) KX library 10 <br />                                           randomized amino acids (Val158, Tyr159, Ile160; Cys167, Met168, Asn169, Ile170; Val179, <br />                                           Leu180, Leu181). permit library construction, pre-existing SacII XbaI sites                                           pUC19 multiple cloning region eliminated restriction digestion, followed                                           mung bean exonuclease digestion remove non-complementary termini, filling                                           Klenow fragment, religation. generate silent restriction sites AAG reading <br />  <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                           Page 4 <br />  <br />  <br />                                          frame, site-directed mutagenesis using QuickChange kit (Stratagene) performed, <br />                                          introducing silent mutations A354C create SacII site; G462T create KpnI site; <br />                                          A549T, G552A create XbaI site (numbers refer wild type AAG cDNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          sequence.) <br />  <br />                                          hexahistidine tag factor Xa cleavage sequence inserted N-terminus                                          AAG PCR amplification forward primer 5&#226;&#8364;&#178;-GCA CGC GGA TCC ATG GGT <br />                                          GGT TCT CAT CAC CAT CAC CAT CAC GGT GGG ATC GAA GGT CGT GCT GTC <br />                                          ACC CCC GCT TTG CAG ATG AAG AAA CCA AAG CAG-3&#226;&#8364;&#178; reverse primer 5&#226;&#8364;&#178;- <br />                                          GAT GCG GCC GCG GAG TTC TGT GCC ATT AGG AAG TCG CCG-3&#226;&#8364;&#178;. amplified <br />                                          product gel purified, cut BamHI SacII religated correspondingly <br />                                          cut vector create pUC-AAG (Fig. S1A). <br />  <br />                                          avoid contamination libraries circularized vector containing wild-type <br />                                          AAG, inactive constructs (&#226;&#8364;&#339;dummy vectors&#226;&#8364;?) internal deletions AAG cDNA <br />                                          prior library creation (Fig. S1A). generate internal deletion inserts                                          inactive SK KX &#226;&#8364;&#339;dummy&#226;&#8364;? constructs, sense oligonucleotide 5&#226;&#8364;&#178;-CCG CGG CCT <br />                                          GCA CGT GGT CGG TAC CTG TGC TAG CAA GTA TCT AGA-3&#226;&#8364;&#178; annealed NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          antisense oligonucleotide 5&#226;&#8364;&#178;-CTT AGA TAC TTG CTA GCA CAG GTA CCG ACC ACG <br />                                          TGC AGG CCG C-3&#226;&#8364;&#178; generate DNA fragment SacII XbaI sticky ends                                          internal PmlI, KpnI, NheI restriction sites. fragment digested KpnI.                                          generate pUC-AAG-SK pUC-AAG-SK dummy vectors, digestion products                                          ligated (T4 DNA ligase, Invitrogen) pUC-AAG pre-cut SacII KpnI,                                          KpnI XbaI, respectively. constructs confirmed sequencing. SK <br />                                          random oligonucleotide cassette (Fig. S1B) constructed annealing single- <br />                                          stranded oligomers SK sense: 5&#226;&#8364;&#178;-ACA GAA CTC CGC GGC CGC ATC GTG GAG ACC <br />                                          GAG GCA TAC CTG GGT CCA GAG GAT GAA GCC GCC CAC TCA CGC GGT GGC <br />                                          CGG CAG ACG-3&#226;&#8364;&#178; SK antisense: 5&#226;&#8364;&#178;-CAC GTA CAG GGT ACC CGG CTT CAT GAA <br />                                          CAT GCC TCG GTT GCG AGG CGT CTG CCG GCC ACC GCG-3&#226;&#8364;&#178;. KX random <br />                                          oligonucleotide cassette (Fig. S1B) constructed annealing KX sense: 5&#226;&#8364;&#178;-G AAG <br />                                          CCG GGT ACC CTG TAC GTG TAC ATC ATT TAC GGC ATG TAC TTC TGC ATG <br />                                          AAC ATC TCC AGC CAG GGC GAC GGG-3&#226;&#8364;&#178; KX antisense: 5&#226;&#8364;&#178;-CAG CGG CTC <br />                                          TAG AGC TCG CAG CAA GAC GCA AGC CCC GTC GCC CTG GCT GGA-3&#226;&#8364;&#178;. <br />                                          Oligomers synthesized contain 88% wild-type nucleotide 4% NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          nucleotides underlined position; 18 bp regions hybridization                                          italicized. Annealed oligomers extended Klenow fragment, fully double- <br />                                          stranded SK random oligonucleotides incubated SacII KpnI, KX <br />                                          random oligonucleotides KpnI XbaI. resulting SK KX random <br />                                          oligonucleotide cassettes ligated correspondingly pre-cut pUC-AAG-SK                                          pUC-AAG-KX dummy constructs. Ligation products cut restriction enzymes <br />                                          (PmlI SK dummy vector NheI KX dummy vector) linearize <br />                                          recircularized dummy constructs lacking randomized oligonucleotide inserts,                                          mutant AAG constructs inserted cassettes; subsequent sequencing unselected                                          selected mutants revealed carry dummy constructs. estimate library size, e.,                                          number variants subsequently screened MMS survival, aliquots plasmid DNA <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 5 <br />  <br />  <br />                                           harboring randomized oligonucleotide inserts electroporated E. coli XL1-Blue <br />                                           cells number transformants determined plating LB-carbenicillin agar. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2.3. Construction YH library <br />                                           Tyr127 His136 targeted mutagenesis generation YH library <br />                                           containing fully randomized nucleotides codons amino acids. Procedures <br />                                           followed construction SK KX libraries. YH sense oligonucleotide <br />                                           [5&#226;&#8364;&#178;-ACA GAA CTC CGC GGC CGC ATC GTG GAG ACC GAG GCA NNN CTG GGT <br />                                           CCA GAG GAT GAA GCC GCC NNN TCA CGC GGT GGC CGG CAG ACG-3&#226;&#8364;&#178;] <br />                                           contained 25% nucleotide positions underlined. YH anti-sense <br />                                           oligonucleotide 5&#226;&#8364;&#178;-CAC GTA CAG GGT ACG CGG CTT CAT GAA CAT GCC TCG <br />                                           GTT GCG AGG CGT CTG CCG GCC ACC GCG-3&#226;&#8364;&#178;. annealed oligomers cut <br />                                           SacII KpnI ligated correspondingly cut SK dummy vector. <br />  <br />                               2.4. Selection characterization active AAG mutant libraries <br />                                           MV1932 cells transformed pUC-AAG-SK, pUC-AAG-KX pUC-AAG-YH <br />                                           mutant libraries dummy vectors, grown mid-log phase LB-carbenicillin <br />                                           medium. Aliquots cultures plated LB-carbenicillin agar, clones NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           subsequently isolated order characterize unselected libraries. Mid-log phase cells <br />                                           incubated 0.2% (w/v) MMS l hr 37 &#194;&#176;C, serially diluted M9 minimal <br />                                           medium spread LB-carbenicillin plates. Survival estimated counting <br />                                           surviving colonies incubation 37 &#194;&#176;C 24 hr. Plasmids total 72 unselected <br />                                           118 selected clones SK KX libraries, 15 unselected 27 selected <br />                                           clones YH libraries isolated. 198 bp SacII-XbaI region AAG                                           sequenced using primer 5&#226;&#8364;&#178;-ACTGGG GTTGGAGTTCTTCG-3&#226;&#8364;&#178; BigDye 3.0 (ABI). <br />                                           Mutant sequences aligned identified Sequencher 3.0 (Gene Codes <br />                                           Corporation, Ann Arbor, MI). <br />  <br />                                           Mutants resistant lex isolated growing cells mid-logarithmic phase LB- <br />                                           carbenicillin medium. Prior lex selection, library aliquots plated LB- <br />                                           carbenicillin, unselected clones isolated sequenced order characterize                                           composition unselected SK KX libraries. MV2157 cells (1 mL) expressing wild- <br />                                           type pUC-AAG, SK KX library, SK dummy washed phosphate-buffered <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           saline, resuspended citrate buffer, treated 200 &#206;&#188;M lex 1 hr. Cells                                           diluted M9 minimal media, spread LB-carbenicillin plates incubated 24 <br />                                           hr 37 &#194;&#176;C. Plasmids surviving colonies isolated sequenced. <br />  <br />                               2.5. Purification wild-type L180F AAG <br />                                           length wild-type AAG mutant L180F cloned expression vector <br />                                           pET101/D-TOPO directional TOPO cloning (Invitrogen). resulting AAGs fused <br />                                           C-terminal hexahistidine tag V5 epitope site expressed control                                           T7 promoter. pET-AAGs transformed E. coli BL21(DE3) grown 30 &#194;&#176;C <br />                                           6 L 2&#195;&#8212;YT medium containing 0.5 % glucose 50 &#206;&#188;g/ml carbenicillin; IPTG (1 mM <br />                                           final concentration) added OD600 = &#226;&#710;&#188;0.6. incubation 16 h 30 &#194;&#176;C, <br />                                           cells harvested centrifugation resuspended 120 ml buffer SB [50 mM Tris- <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                          Page 6 <br />  <br />  <br />                                          HCl (pH 8.0), 10 mM NaCl, 1 mM PMSF, 1 mM DTT 10 % (w/v) glycerol],                                          adjusted 500 &#206;&#188;g/ml lysozyme prior storage -80 &#194;&#176;C. cell suspension thawed, <br />                                          immediately supplemented 1&#195;&#8212; protease inhibitor cocktail (EMD Biosciences) NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          sonicated ice. following procedures carried 4&#194;&#176;C. centrifugation                                          20,000 &#195;&#8212;g 30 min, supernatant fraction reserved, cell pellet resuspended <br />                                          30 ml buffer SB; sonification centrifugation steps repeated                                          supernatant fractions combined (Fraction . Fraction loaded DEAE <br />                                          cellulose (Sigma) column (4.9 cm2 &#195;&#8212; 4 cm) equilibrated buffer [30 mM Tris-HCl (pH <br />                                          8.0), 10 mM NaCl, 1 mM DTT, 1 mM EDTA, 2 mM benzamidine (EMD Biosciences)                                          10 % (w/v) glycerol] column washed 60 ml equilibration buffer. <br />                                          Fractions containing AAG activity pooled dialyzed buffer (Fraction II). <br />                                          Fraction II applied SP-Sepharose column (1.8 cm2 &#195;&#8212; 4.2 cm) equilibrated buffer <br />                                           column washed 40 ml buffer  followed 45 ml linear gradient                                          10-500 mM NaCl buffer  fractions analyzed 12.5% SDS-polyacrylamide <br />                                          gels, fractions containing AAG pooled dialyzed buffer B [50 mM <br />                                          sodium phosphate (pH 8.0), 300 mM NaCl, 10 mM imidazole, 1 mM &#206;&#178;-mercatoethanol, 2 <br />                                          mM benzamidine 5 % (w/v) glycerol] (Fraction III). Fraction III loaded                                          Qiagen nickel-nitrilotriacetic acid (Ni-NTA) affinity column (1.77 cm2 &#195;&#8212; 3 cm) equilibrated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          buffer B. column washed 20 ml buffer B 30 ml linear <br />                                          gradient 10-300 mM imidazole buffer SB. Fractions containing AAG, identified                                          electrophoresis 12.5 % SDS-polyacrylamide gels, pooled (fraction IV).                                          dialysis storage buffer [30 mM Tris-HCl (pH 7.4), 1 mM DTT, 1 mM EDTA, 2 mM <br />                                          benzamidine 10 % (w/v) glycerol], preparation concentrated using Amicon <br />                                          filter unit (MW cut 30,000), aliquoted stored -80 &#194;&#176;C. Protein concentration                                          determined Bradford reaction [26] using Bio-Rad Protein Assay bovine <br />                                          serum albumin standard. Equivalent amounts wild-type mutant enzymes,                                          assessed Bradford assay, exhibited equivalent absorbance A280. <br />  <br />                               2.6. DNA glycosylase assays <br />                                          DNA glycosylase activity measured quantifying release [3H]-labeled N- <br />                                          methylpurines calf thymus DNA methylated [3H]-methylnitrosourea (1 mCi, GE <br />                                          Healthcare) previously described [27]. Reaction mixtures (100 &#206;&#188;l) contained [3H]-calf <br />                                          thymus DNA (20,000 cpm), 50 nM 500 nM AAG L180F mutant enzyme specified, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          20 mM Tris-HCl (pH 7.6), 100 mM KCl, 5 mM EDTA, 2 mM EGTA, 5 mM &#206;&#178;- <br />                                          mercaptoethanol; protein concentrations equalized addition bovine serum <br />                                          albumin. Following incubation 37 &#194;&#176;C 0, 1, 2, 4, 8, 16 min, reactions stopped <br />                                          addition 0.1 volumes 3 M sodium acetate (pH 5.2) 3 volumes ethanol <br />                                          followed incubation &#226;&#710;&#8217;20 &#194;&#176;C 1 hr precipitate DNA. Non-radioactive 3-meA                                          7-meG standards added supernatant containing released 3H-methylated bases, <br />                                          solution taken dryness vacuum desiccation. Samples dissolved 15 <br />                                          ml 0.1 M HCl spotted Whatman 3MM paper. Descending paper chromatography                                          performed 18 hr 70% isopropanol, 20% water 10% concentrated ammonium <br />                                          hydroxide separate 3-meA 7-meG. Standards visualized UV light,                                          paper cut 1 cm strips. Radioactivity measured scintillation counting. <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                           Page 7 <br />  <br />  <br />                                           DNA glycosylase activity measured using double-stranded 5&#226;&#8364;&#178;-32P end-labeled <br />                                           36-mer DNA substrate containing &#206;&#181;A&#226;&#8364;&#162;T Hx&#226;&#8364;&#162;T base pair [15]. lesion- <br />                                           containing strand 5&#226;&#8364;&#178; TTGGCAGCAGAATATTGCTXGCGGG-AATTCGGCGCG-3&#226;&#8364;&#178; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           X= &#206;&#181;A Hx. Reaction mixtures (10 &#206;&#188;l) contained 1 pmol 32P-labeled DNA substrate, <br />                                           62.5 fmol 1.6 pmol AAG L180F mutant enzyme 30 mM Tris-Hcl (pH 7.4), 1 mM <br />                                           EDTA, 1 mM DTT, 100 mM KCl. Following incubation 37 &#194;&#176;C 30 min, reactions                                           stopped heating 70 &#194;&#176;C 5 min, brought 50 mM KCl addition 1.2 &#206;&#188;l 0.5 M <br />                                           KCl, incubated E. coli endonuclease IV (1 unit) 37 &#194;&#176;C 30 min. Reactions <br />                                           terminated heating 70 &#194;&#176;C 5 min placed ice 3 min. Samples                                           combined equal volume 2X denaturing buffer containing 95% ionized <br />                                           formamide, 1 mM EDTA, 0.05% bromophenol blue, heated 97 &#194;&#176;C 3 min,                                           resolved electrophoresis 12% polyacrylamide, 8.3 M <span id='am-64' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-65' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span>urea</span> gels. Gels dried                                           visualized phosphor imaging, reaction products analyzed ImageQuant <br />                                           software (GE Healthcare). <br />  <br />                               2.7. DNA binding assay <br />                                           relative binding affinity lesion-containing oligonucleotides evaluated using <br />                                           electrophoretic mobility shift assay previously described [25]. 5&#226;&#8364;&#178;-32P end-labeled <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           36-mer double-stranded DNA ligand contained lesion opposite T. lesion-containing <br />                                           strand 5&#226;&#8364;&#178;TTGGCAGCAGAATA-TTGCTXGCGGGAATTCGGCGCG-3&#226;&#8364;&#178; X                                           &#206;&#181;A, Hx tetrahydrofuran abasic site analog. Briefly, reaction mixtures (10 &#206;&#188;l) <br />                                           contained 1 pmol DNA ligand 1-15 pmol AAG 50 mM Tris-HCl (pH 7.4), 1 mM <br />                                           EDTA, 1 mM DTT, 100 &#206;&#188;g/ml acetylated bovine serum albumin. Following incubation <br />                                           ice 30 min, samples analyzed electrophoresis 7.5% polyacrylamide gels. <br />                                           Gels dried, visualized phosphor imaging, reaction products analyzed <br />                                           ImageQuant software (GE Healthcare). <br />  <br />                               2.8. Fluorescence polarization binding assay <br />                                           Steady-state fluorescence polarization assays performed compare dissociation <br />                                           constants binding wild-type AAG vs. L180F mutant enzyme double-stranded <br />                                           DNA containing single abasic (AP) site. AP site-containing strand                                           DNA binding assay, DNA 5&#226;&#8364;&#178;-end labeled 6-carboxyfluorescein <br />                                           (FAM). fluorescence polarization FAM measured 25 &#194;&#176;C SpectraMax <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           M5 Microplate Reader (Molecular Devices , Sunnyvale, CA) exciting 495 nm                                           monitoring emission 518 nm. Binding reactions (150 &#206;&#188;l) carried binding <br />                                           buffer (30 mM Tris-HCl [pH 7.4], 1 mM EDTA, 1 mM DTT, 25 mg/ml acetylated bovine <br />                                           serum albumin) contained 100 nM FAM-labeled DNA increasing concentrations                                           enzyme. Reaction mixtures equilibrated 25 &#194;&#176;C 15 min prior fluorescence <br />                                           measurements. Assays performed triplicate. subtracting background DNA <br />                                           fluorescence polarization, measurements fit following equation determine                                           apparent dissociation constant Kd (41). <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 8 <br />  <br />  <br />  <br />  <br />                                                                                                                        Eq. (1) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Eq. (1), P represents intensity fluorescence polarization enhancement, c                                            maximum value fluorescence polarization, Lo Ro total concentrations DNA <br />                                            substrate wild-type AAG L180F mutant protein, respectively, Kd overall <br />                                            dissociation constant binding wild-type mutant protein FAM-AP&#194;&#183;T-36-mer. <br />  <br />                               3. Results <br />                               3.1. Creation AAG substrate binding pocket mutant libraries <br />                                            Crystal structures molecular modeling AAG complexed lesion-containing <br />                                            oligonucleotides revealed active site pocket binds substrate bases                                            discriminating normal purines. model AAG bound DNA oligomer <br />                                            containing &#206;&#181;A shown Fig. 1A, detailed view substrate binding pocket <br />                                            illustrated Fig. 1B. proposed pocket formed mainly aromatic hydrophobic <br />                                            residues, believed bind substrate bases stacking interactions involving <br />                                            conjugated bases Tyr127, His136, Tyr159 [22,23]. order provide evidence NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            vivo function binding pocket residues, assess essentiality methylating <br />                                            agent resistance, created libraries mutants containing random substitutions 19 <br />                                            wild-type amino acids. mutated residues contact &#206;&#181;A substrate  <br />                                            crystal complex [Tyr127, Ala134, Ala135, His136, Tyr159, Cys167, Asn169, Leu180 [23]] (Fig. <br />                                            1B), flanking residues primary sequence. used random <br />                                            oligonucleotide replacement mutagenesis create libraries, designated SK KX. <br />                                            SK library targets residues half model Fig. 1B, including <br />                                            Tyr127, Ala134, Ala135, His136; KX library targets residues half <br />                                            including Tyr159, Cys167, Asn169, Leu180. Construction mutant libraries                                            described Experimental Procedures represented schematically Supplementary <br />                                            Fig. S1. Oligonucleotide mixtures contained 12% random nucleotides position <br />                                            encoding targeted <span id='am-29' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-30' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-31' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> replaced wild-type codon libraries; e.,                                            targeted position contained 88% wild-type nucleotide 3%                                            nucleotides. unselected SK KX libraries contained 2.8 &#195;&#8212; 106 1.7 &#195;&#8212; 106 mutant <br />                                            AAGs, respectively. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            sequenced randomized regions 37 unselected clones SK library 35 <br />                                            KX library. majority clones contained multiple nucleotide substitutions, <br />                                            yielding average 2.5 amino acid changes SK mutant cDNA 2.3 amino acid <br />                                            changes KX mutant cDNA (Fig. 2A,B). Multiple nucleotide substitutions observed <br />                                            targeted codon libraries (Fig. 3B). Approximately 15% 4%                                            unselected SK KX libraries harbored frameshift mutations, respectively, presumably <br />                                            derived incomplete extension random oligonucleotide synthesis. mutant <br />                                            cDNA observed  <br />  <br />  <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                           Page 9 <br />  <br />  <br />                               3.2. Genetic complementation yields mutant libraries confer MMS resistance <br />                                           characterized DNA repair-deficient E. coli strain MV1932 lacks 3-meA DNA <br />                                           glycosylase AlkA O6-methylguanine-DNA methyltransferase Ada [12], NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           highly sensitive methylating agent MMS. Expression human AAG rescues <br />                                           MV1932 cells MMS cytoxicity [6], basis functional complementation <br />                                           isolation active AAG mutants. AAG excises 3-meA 7-meG                                           double-stranded DNA [13]. 3-meA comprises 10% MMS-induced base lesions,                                           highly cytotoxic, efficiently cleaved; 7-meG comprises 70% base lesions, <br />                                           appears relatively innocuous, efficiently cleaved. incubation <br />                                           mid-logarithmic phase MV1932 cells 0.2% MMS 1 hr 37&#194;&#176; C yielded 2.5 % <br />                                           survival cells expressing wild-type AAG, 4.6 &#195;&#8212; 10-5 % survival cells expressing                                           inactive construct internal deletion AAG cDNA, 54,000-fold difference                                           resistance (Fig. 2A). select active AAG mutants, MV1932 cells transformed <br />                                           SK KX libraries, incubated 0.2 % MMS 1 hr plated                                           medium lacking MMS. surviving fraction 3.6 &#195;&#8212; 10-3 % 2.2 &#195;&#8212; 10-2 % SK <br />                                           KX libraries, respectively, (Fig. 2A), indicating small proportion mutants <br />                                           able confer MMS resistance. Surviving colonies picked, plasmids harboring <br />                                           mutant AAG genes sequenced. independent libraries comparable NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           respects (Fig. 2A,B). Selected mutants contained 3- 4-fold fewer mutations                                           unselected mutants. selected SK KX libraries averaged 0.6 0.7 amino acid <br />                                           substitutions mutant, respectively, compared 2.5 2.3 unselected <br />                                           libraries. frameshifts, nonsense mutations deletions observed                                           survivors. AAG zero amino acid changes enriched libraries, initial <br />                                           5% 11% unselected SK KX libraries, respectively, 49% 44%                                           selected libraries. large majority mutants wild-type amino acid level <br />                                           harbored different silent mutations nucleotide level. post-selection shift                                           lower mutational load, noted previous studies enzymes, e.g., [28], observed <br />                                           absence frameshifts, nonsense mutations deletions, enrichment wild-type <br />                                           amino acid sequence indicative strong selective pressure active AAG. <br />  <br />                               3.3. Mutability AAG substrate binding pocket residues <br />                                           Selection MMS resistance cross-species functional complementation                                           revealed mutability substrate binding pocket showed striking parallels NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           natural evolution (Fig. 3A), differences. DNA sequencing 116 mutants                                           active SK KX libraries yielded spectrum amino acid substitutions shown Fig. <br />                                           3C,D. average, strictly conserved residues showed fewer replacements relatively <br />                                           conserved residues,  turn, relatively conserved residues showed fewer replacements <br />                                           non-conserved residues. finding holds number different replacements, <br />                                           total replacements, single position. notably, replacements                                           strictly conserved residues Tyr127, His136, Asn169. absence                                           substitutions single mutants double mutants  potential                                           compensatory substitutions) emphasizes functional importance respective                                           chains implicit evolutionary conservation (Fig. 3A) X-ray structures [[22,23] <br />                                           Fig. 1]. results, proposed key roles Tyr127 His136  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 10 <br />  <br />  <br />                                           substrate binding [23], led examine residues greater depth. partially <br />                                           random oligonucleotide mutagenesis described represents incomplete search                                           amino acid substitutions. order test possible substitutions Tyr127 His136, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           constructed YH library, wild-type nucleotide encoding residues                                           synthesized mixture 25%  T, G, C. 16,000 mutants                                           totally random library transformed MV1932 cells subjected MMS selection. <br />                                           Sequencing surviving colonies revealed wild-type amino acid <br />                                           level, harbored silent mutations (data shown). results                                           document critical importance Tyr127 His136 MMS resistance. <br />  <br />                                           noted interesting potentially informative exceptions general evolutionary <br />                                           parallels noted  striking Leu180, strictly conserved residue                                           chain lies adjacent N3 position &#206;&#181;A substrate [23] expected pack <br />                                           closely methyl adduct 3-meA [22]. Leu180 Phe substitutions                                           fully 33% sequenced mutants selected KX library, representing 6-fold <br />                                           enrichment 3% sequenced clones unselected library 19% selected <br />                                           library. comparison, enrichment wild-type AAG sequence 4-fold, 11% <br />                                           sequenced clones unselected library 44% selected library (Fig. 2B). NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           observed selection L180F AAG suggests Leu Phe replacement confers                                           great greater MMS resistance wild-type complementation  <br />                                            contrasting, exception evolutionary conservation parallel                                           results Cys167. observed replacements amino acid single double <br />                                           mutants, conserved natural evolution (Fig. 3A). <br />  <br />                               3.4. lex MMS survivors share similar spectrum substitutions <br />                                           subjected SK KX libraries selection resistance site-specific <br />                                           methylating agent lex. lex selectively binds rich sequences preferentially <br />                                           generates 3-meA lesions (93% base adducts) [29-31]. Selection lex survival <br />                                           enriched wild-type AAG amino acid sequence, comprised 63% 54%                                           selected SK KX libraries, respectively. Fig. 3E shows sequence 15 SK 27 KX <br />                                           library mutants. overall pattern comparable MMS selection (Fig. <br />                                           3C,D).  substitutions Tyr127 His136, Cys167.                                           substitution observed Asn169, I160S, N169D, V179I triple mutant. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Interestingly, Leu180 Phe substitutions highly enriched, appearing 4                                           19 mutants selected KX library. similarity spectrum substitutions <br />                                           produced MMS lex selection suggests protection lethality 3- <br />                                           meA key factor governing MMS resistance complementation  <br />  <br />                               3.5. Effect L180F substitution MMS survival <br />                                           accord notable enrichment L180F mutant MMS- lex- <br />                                           selected libraries (Fig. 3B-E), expression L180F mutant conferred <br />                                           improved MMS protection relative wild-type AAG (Fig. 4A). LD50 <br />                                           essentially unchanged, observed statistically significant 4.4-fold increase                                           survival (P &lt; 0.005) high concentration MMS used selection active mutants <br />                                           (0.2% 1 hr 37 &#194;&#176;C); overall survival reduced orders magnitude <br />  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                          Page 11 <br />  <br />  <br />                                           stringent conditions. difference mutant wild-type viability                                           greatest high MMS concentrations, presumably compensatory survival <br />                                           mechanisms, translesion DNA synthesis recombination, saturated NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           approaching saturation. Immunoblotting showed expression L180F mutant <br />                                           protein comparable wild-type start MMS exposure, 4 hr <br />                                           (Fig. 4B,C). <br />  <br />                               3.6. Effect L180F substitution excision methyl adducts <br />                                           examine effect L180F substitution removal methylated bases, purified <br />                                           wild-type L180F AAG proteins. final preparations essentially <br />                                           homogeneous (&gt;98% pure) SDS-polyacrylamide gels exhibited mobility consistent <br />                                           predicted molecular weight (Fig. 5A). wild-type mutant preparations <br />                                           free contaminating nuclease activity, measured assays utilizing 5&#226;&#8364;&#178;-end 32P-labeled <br />                                           single- double-stranded oligonucleotides (data shown). DNA glycosylase activity <br />                                           assayed quantifying release chromatographically separated radiolabelled N- <br />                                           methylpurines calf thymus DNA methylated [3H]-methylnitrosourea [27]. <br />                                           Previous measurements shown potentially lethal lesion 3-meA preferred <br />                                           substrate wild-type human AAG, presumptively benign lesion 7-meG NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           removed slower rate [32]. illustrated Fig. 5B,C, L180F mutant enzyme <br />                                           displayed reduced activity, rate release [3H]-labeled 3-meA 7-meG lower <br />                                           observed wild type AAG. rate excision 3-meA mutant enzyme <br />                                           reduced 2.2-fold relative wild-type AAG (0.82 vs. 0.38 fmol excised/min/ <br />                                           pmol enzyme, respectively), rate excision 7-meG reduced 7.3-fold <br />                                           (0.049 vs. 0.0068 fmol excised/min/pmol enzyme, respectively).  mutant enzyme <br />                                           exhibited modestly increased discrimination 7-meG. Decreased generation                                           cytotoxic abasic sites resulting reduced excision 3-meA 7-meG factor <br />                                           promoting survival high MMS concentrations (Fig. 4A), downstream enzymes                                           BER pathways saturated; factor particularly relevant case 7- <br />                                           meG, believed toxic abasic site reaction product. Possible <br />                                           structural determinants altered substrate preference presented Discussion. <br />  <br />                               3.7. Effect L180F substitution excision binding neutral adducted bases <br />                                           methylated bases derived MMS lex electron deficient, carry NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           delocalized positive charge presumably important binding excision AAG <br />                                           [23]. adducted bases net charge efficiently cleaved.                                           examine effect L180F substitution activity neutral adducted bases,                                           measured excision &#206;&#181;A oligonucleotide substrate coupled assay                                           excision damaged base followed endonuclease-mediated cleavage                                           abasic site product (Fig. 6A). Quantification resulting, cleaved oligonucleotide                                           denaturing polyacrylamide gel (Fig. 6B) revealed 6.6-fold decrease activity                                           L180F mutant enzyme, calculated using regression analysis 5 points initial, <br />                                           linear portion curves. specific activity wild-type AAG 68 fmol &#206;&#181;A <br />                                           excised/min/fmol enzyme L180F mutant 10 fmol &#206;&#181;A excised/min/fmol <br />                                           enzyme. Similarly, observed 3.0-fold decrease excision Hx mutant enzyme <br />                                           (Fig. 6C,D); oligomer &#206;&#181;A assays, lesion.  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 12 <br />  <br />  <br />                                           specific activity wild-type AAG 111 fmol Hx excised/min/fmol enzyme                                           L180F mutant 37 fmol Hx excised/min/fmol enzyme. examined binding                                           wild-type L180F mutant enzymes lesion-containing oligonucleotides NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           electrophoretic mobility shift assays (Fig. S2). case &#206;&#181;A (Fig. S2A), quantification <br />                                           bound enzyme (Fig. S2B) revealed 1.2-fold increase relative binding affinity                                           mutant enzyme. Relative binding affinity Hx-containing oligonucleotide (Figs. S2C,D) <br />                                           enzymes. <br />  <br />                               3.8. Effect L180F substitution binding abasic site-containing DNA <br />                                           abasic site product AAG cleavage highly cytotoxic. enhanced <br />                                           survival methylation damage mediated L180F mutant tighter <br />                                           binding retarded release potentially lethal intermediate. compare relative <br />                                           binding affinity wild-type mutant proteins AP-DNA, performed <br />                                           electrophoretic mobility shift assays using double-stranded 5&#226;&#8364;&#178;-32P end-labeled 36 base-pair <br />                                           oligonucleotide containing abasic site analog opposite <span id='am-24' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Thymine" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>thymine</span>; oligomer                                           &#206;&#181;A Hx assays, lesion. observed affinity                                           L180F mutant AP-DNA slightly (&#226;&#710;&#188;1.5 fold) higher wild-type AAG <br />                                           (data shown). order compare difference wild-type NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           mutant proteins, performed sensitive fluorescence polarization assays determine                                           apparent steady-state equilibrilium dissociation constant binding duplex 36-mer <br />                                           containing abasic site opposite T 5&#226;&#8364;&#178;-end labeled fluorescent dye FAM.                                           protein binds fluorescently-labeled DNA, rotational freedom DNA usually <br />                                           decreased, leading increased polarization. shown Fig. 7A, duplex, FAM- <br />                                           labeled AP&#194;&#183;T-36-mer titrated increasing amounts wild-type AAG,                                           concentration-dependent increase fluorescence polarization observed saturated                                           approximately 1 &#206;&#188;M AAG. net polarization enhancement concentration fit <br />                                           Eq. (1), yielding apparent Kd 824 &#194;&#177; 104 nM binding assay conditions. <br />                                           magnitude polarization enhancement saturation observed F180L mutant <br />                                           enzyme (Fig. 7B), comparable wild-type AAG, yielding apparent Kd                                           479&#194;&#177; 38 nM.  binding affinity L180F duplex AP-DNA, measured                                           1/Kd, &#226;&#710;&#188; 1.7 fold greater wild-type AAG. <br />  <br />                                           explore possible structural basis enhanced abasic site binding, modeled NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Leu180 Phe substitution crystal structure wild-type human AAG complexed                                           pyrrolidine abasic nucleotide-containing DNA [22]. model derived replacing <br />                                           Leu180 phenylalanine energy-minimizing, using Amber parameters chiral <br />                                           restraints Molecular Operating Environment package programs <br />                                           [<span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> Computing Group, Montreal (www.chemcorp.com)]. conformation                                           adopted Phe chain simply replaced Leu chain; total potential <br />                                           energy calculated conformer -7197 kcal mol-1. n energetically equivalent <br />                                           conformation (-7287 kcal mol-1) adopted choosing alternate rotamer,                                           resulted Phe chain projecting abasic site. model, superposed                                           mutated protein, illustrated Fig. 7C. shown, aromatic chain                                           Phe180 rotates abasic residue position Leu180 wild-type AAG,                                           partially fills space occupied substrate base. Greater proximity Phe180  <br />  <br />                                              DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                           Page 13 <br />  <br />  <br />                                          chain baseless site provide favorable geometry interaction, e.g., van der <br />                                          Waals contact epsilon zeta carbons Phe deoxyribose,                                          increase binding affinity L180F protein AP-DNA. Modeling shows NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          abasic residue displaced outward mutant, causing loss possible, weak H bond <br />                                          O2P 5&#226;&#8364;&#178; phosphate hydroxyl Tyr159  shown). <br />  <br />                               4. Discussion <br />                                          Human AAG broad substrate specificity, acting variously modified purines arising <br />                                          methylation, oxidation lipid peroxidation. Substrate discrimination wild- <br />                                          type enzyme likely represents compromise promotes protection differing <br />                                          sources DNA damage, governed,  detailed architecture                                          substrate binding pocket.  used oligonucleotide-directed mutagenesis                                          simultaneously randomize residues contribute substrate binding pocket                                          shown crystal structures contact substrate bases. mutagenesis strategy <br />                                          affords experimental opportunity remodel active site, optimized                                          natural evolution defense diverse damaging agents optimized defense <br />                                          specific agent. used bacterial complementation select variant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          AAGs protect killing methylating agent MMS. selection                                          methylation-resistant mutants recapitulated conservation certain key amino acids                                          natural evolution, providing vivo evidence support active site model deduced <br />                                          crystal structures [22,23]. observed departures natural evolution                                          reflect optimization methylation survival complementation  <br />  <br />                                          Crystallographic data indicated requirement strictly conserved aromatic <br />                                          residues Tyr127, His136, Tyr159 appear provide crucial stacking interactions                                          AAG &#206;&#181;A substrate [23]. Consistent data, replacements                                          Tyr127 single double MMS- lex-resistant mutants, suggesting Tyr127 <br />                                          essential interactions methylated bases  replacements                                          His136, suggested His136 dispensable excision 3- <br />                                          meA, based model methylated base does reach far                                          substrate binding cavity stack His136 [22]. data indicate His136                                          required survival methylation damage complementation  modeling <br />                                          suggests functions impede accommodation 7-meG binding pocket, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          limiting conversion relatively innocuous base lesion potentially lethal abasic site. <br />                                          Superimposing 7-meG crystal structures AAG complexed abasic site-                                          &#206;&#181;A-containing oligonucleotide [22,23] reveals steric interference His136 imidazole <br />                                          chain exocyclic N7 methyl group (Fig. 8C). position His136 fixed                                          numerous H bonds main chain chain atoms (e.g., OH group                                          Tyr157 chain phosphate upstream baseless deoxyribose), limiting                                          size binding pocket disfavoring accommodation 7-meG. Interestingly,                                          H136Q mutant allow survival, albeit diminished relative wild-type AAG,                                          yeast complementation [42]; difference result likely <br />                                          reflects differences species employed expression levels AAG relative                                          downstream BER enzymes. observed non-conservative (non-planar) substitutions <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                             Page 14 <br />  <br />  <br />                                          Tyr159, Asn Cys, MMS-resistant single mutants, indicating                                          function(s) Tyr159 essential selection conditions. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          substitution highly conserved Asn169 single double mutants <br />                                          Asp, replacement observed lex survivors. Preservation Asn                                          concordant proposed role chain excluding normal guanine                                          active site steric clash exocyclic N2 amino group [23]. Interestingly,                                          observed substitutions Cys167 single double methylation-resistant mutants, <br />                                          indicating required excision methylated bases complementation <br />                                           <br />  <br />                                          unexpected finding work remarkable selection Phe place strictly <br />                                          conserved Leu180. L180F mutant enriched random library &#226;&#710;&#188;50% <br />                                          greater extent wild-type AAG, dominating spectrum MMS-resistant mutants. <br />                                          Expression L180F mutant yielded survival equivalent wild-type AAG low <br />                                          doses MMS, 4.4-fold better survival high doses viability severely <br />                                          compromised. Purification L180F mutant protein examination catalytic <br />                                          properties support conclusion increased survival mediated reduction NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          excision activity, change substrate discrimination, slight increase binding <br />                                          affinity abasic sites. alterations consistent idea  relative human <br />                                          cells, bacterial host cells reduced capacity repair potentially lethal BER <br />                                          intermediates arising glycosylase action, better protected methylators                                          variant AAG generates abasic sites slower wild-type rate. observed                                          overall rate removal adducted bases methylated DNA homogenous <br />                                          L180F protein reduced relative wild type AAG. rate excision 3-meA <br />                                          calf thymus DNA decreased 2.2-fold, rate 7-meG decreased 7.3- <br />                                          fold. 7-meG represents &#226;&#710;&#188;70% - 80% base lesions generated MMS, <br />                                          increased discrimination 7-meG contribute enhanced survival high levels <br />                                          methylation abasic sites potentially cytotoxic BER intermediates                                          saturate downstream enzymes, leading lethal double strand breaks cell death [33]. <br />                                          possible importance survival, apparent binding affinity                                          L180F mutant abasic-site-containing DNA increased 70%, assessed                                          fluorescence polarization; reduced rate release relieve toxicity associated                                          unrepaired abasic sites. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          previous studies suggested imbalanced BER caused excess DNA <br />                                          glycosylase activity increase sensitivity alkylating agents. genetic study yeast <br />                                          [11] demonstrated spontaneous mutation rate apurinic endonuclease-deficient <br />                                          apn1 cells increased 600-fold simple imbalance 3-methyladenine <br />                                          DNA glycosylase Mag1 apurinic endonuclease. increased spontaneous mutation                                          generation abasic sites resulting mainly Mag1 glycosylase activity,                                          exogenous Apn1 expression dramatically suppresses elevation mutation.                                          report [34] suggested repair damaged lesions regulated DNA <br />                                          glycosylase activity glycosylase reaction rate-limiting step BER. Importantly, <br />                                          recent work suggests  human cells, coordination activities AAG APE1 <br />                                          depends,  slow excision AAG rapid incision AP sites  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                            Page 15 <br />  <br />  <br />                                          abundant APE1 (34). balance BER activities, reached natural <br />                                          evolution, lost cross-species complementation systems  presumably <br />                                          leading selection variant AAGs better matched capacity host cells <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          repair potentially lethal BER intermediates arising glycosylase action. <br />  <br />                                          addition alteration rates hydrolysis methylated bases L180F mutant, <br />                                          observed 3- 7-fold decrease rate excision neutral adducted bases <br />                                          ethenoA, arises lipid peroxidation, hypoxanthine, arises oxidative <br />                                          deamination adenine. Overall, data indicate L180F substitution alters <br />                                          discrimination methylated bases better protect MMS complementation <br />                                           concomitant reduction activity damage sources. <br />  <br />                                          does L180F substitution mediate altered substrate specificity observed? <br />                                          Introduction bulky aromatic Phe chain decreases space available                                          substrate binding pocket, limit accommodation modified bases. Fig. 8 <br />                                          shows model wild-type mutant substrate binding pocket, occupied 3- <br />                                          meA 7-meG. case 3-meA, wild-type Leu180 chain makes hydrophobic <br />                                          contact methyl adduct (Fig. 8A, left panel). L180F mutant (Fig. 8B,                                          right panel), modeled Phe180 benzyl ring located 4.2-4.6 &#195;&#8230; methyl adduct, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          favorable range &#207;&#8364;-cation stacking interaction [35,36]. addition, orientation <br />                                          Glu125 chain perpendicular wild-type enzyme. Glu125                                          function deprotonate water molecule  shown) nucleophilic attack                                          glycosydic bond [22], orientation residue suggests possible mechanism                                          slight reduction rate excision 3-meA. important case 7-meG,                                          mutant Phe180 decrease excision mechanisms. orientation                                          Glu125 chain, described (Fig. 8B, compare panels B). Second, <br />                                          Phe180 decrease accommodation, adducted substrate tightly boxed                                          mutant binding pocket, His136 imidazole planar benzyl ring <br />                                           pronounced steric effects described excluding 7-meG <br />                                          substrate binding pocket highly specific 3-methyladenine DNA glycosylase <br />                                          H. pylori, glycosylase belongs different superfamily different <br />                                          overall architecture [37].  substitution Phe Leu180 increases surface area <br />                                          van der Waal&apos;s interactions, apparently resulting increased contact deoxyribose <br />                                          mediate tighter binding slower release abasic site product. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          illustrating consequences replacement Phe, work illuminates roles                                          Leu180 wild-type human AAG. expected residue lines substrate <br />                                          binding pocket contacts substrate bases, Leu contributes, albeit modestly, wild-type <br />                                          excision activity, pattern substrate discrimination, overall compromise <br />                                          protection diverse sources DNA damage. particular  Leu180 appears <br />                                          modulate excision 7-meG, contributing balance BER pathways                                          remove methylation damage [33,38], raising question wild-type rate <br />                                          excision 7-meG, relatively benign lesion, necessary structural consequence                                          preserving activity deleterious adducts. <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                                         Page 16 <br />  <br />  <br />                               Supplementary Material <br />                                          Refer Web version PubMed Central supplementary material. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Acknowledgments <br />                                          thank Dr. Elinor Adman molecular modeling valuable discussions Yunny Yunita, K. Marinov <br />                                          Georgi Dr. Eric Althoff technical assistance. work supported NIH grants CA78885, CA77852, <br />                                          ES07033 L L. CA55042 L.D.S. H.H.G supported Medical Scientist Training Program. <br />                                          L.D.S. American Cancer Society Research Professor. <br />  <br />  <br />                               References <br />                                          1. Sedgwick B. Repairing DNA-methylation damage. Nat Rev Mol Cell Biol. 2004; 5:148&#226;&#8364;&#8220;157. <br />                                             [PubMed: 15040447] <br />                                          2. Wyatt MD, Pittman DL. Methylating agents DNA repair responses: Methylated bases                                             sources strand breaks. Chem Res Toxicol. 2006; 19:1580&#226;&#8364;&#8220;1594. [PubMed: 17173371] <br />                                          3. Friedberg, EC.; Walker, GC.; Siede, W.; Wood, RD.; Schultz, RA.; Ellenberger, T. DNA Repair                                             Mutagenesis. American Society Microbiology; Washington, D.C.: 2005. <br />                                          4. Doublie S, Tabor S, Long  Richardson CC, Ellenberger T. Crystal structure bacteriophage <br />                                             T7 DNA replication complex 2.2 resolution. Nature. 1998; 391:251&#226;&#8364;&#8220;258. [PubMed: 9440688] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          5. Beard WA, Osheroff WP, Prasad R, Sawaya MR, Jaju M, Wood TG, Kraut J, Kunkel TA, Wilson <br />                                             SH. Enzyme-DNA interactions required efficient nucleotide incorporation discrimination                                             human DNA polymerase beta. J Biol Chem. 1996; 271:12141&#226;&#8364;&#8220;12144. [PubMed: 8647805] <br />                                          6. Samson LD, Derfler B, Boosalis M, K. Cloning characterization 3-methyladenine <br />                                             DNA glycosylase cDNA human cells gene maps chromosome 16. Proc Natl Acad <br />                                             Sci U S  1991; 88:9127&#226;&#8364;&#8220;9131. [PubMed: 1924375] <br />                                          7. Engelward BP, Dreslin  Christensen J, Huszar D, Kurahara C, Samson LD. Repair-deficient 3- <br />                                             methyladenine DNA glycosylase homozygous mutant mouse cells increased sensitivity                                             alkylation-induced chromosome damage cell killing. EMBO J. 1996; 15:945&#226;&#8364;&#8220;952. [PubMed: <br />                                             8631315] <br />                                          8. Engelward BP, Allan JM, Dreslin AJ, Kelly JD, Wu MM, Gold B, Samson LD. chemical                                             genetic approach define biological consequences 3-methyladenine lesions                                             mammalian genome. J Biol Chem. 1998; 273:5412&#226;&#8364;&#8220;5418. [PubMed: 9479003] <br />                                          9. Scharer OD, Jiricny J. Recent progress biology, chemistry structural biology DNA <br />                                             glycosylases. Bioessays. 2001; 23:270&#226;&#8364;&#8220;281. [PubMed: 11223884] <br />                                          10. Hosfield DJ, Daniels DS, Mol CD, Putnam CD, Parikh SS, Tainer JA. DNA damage recognition <br />                                              repair pathway coordination revealed structural biochemistry DNA repair enzymes. <br />                                              Prog Nucleic Acid Res Mol Biol. 2001; 68:315&#226;&#8364;&#8220;347. [PubMed: 11554309] <br />                                          11. Glassner BJ, Rasmussen LJ, Najarian MT, Posnick LM, Samson LD. Generation strong <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              mutator phenotype yeast imbalanced base excision repair. Proc Natl Acad Sci U S  1998; <br />                                              95:9997&#226;&#8364;&#8220;10002. [PubMed: 9707589] <br />                                          12. Posnick LM, Samson LD. Imbalanced base excision repair increases spontaneous mutation                                              alkylation sensitivity Escherichia coli. J Bacteriol. 1999; 181:6763&#226;&#8364;&#8220;6771. [PubMed: 10542179] <br />                                          13. Wyatt MD, Allan JM, Lau AY, Ellenberger TE, Samson LD. 3-methyladenine DNA glycosylases: <br />                                              structure, function, biological importance. Bioessays. 1999; 21:668&#226;&#8364;&#8220;676. [PubMed: 10440863] <br />                                          14. Wuenschell GE, O&apos;Connor TR, Termini J. Stability, miscoding potential, repair 2&#226;&#8364;&#178;- <br />                                              deoxyxanthosine DNA: implications <span id='am-36' about='obo:IMR_0100128' typeof='owl:Thing obo:IMR_0001720 obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001'><span id='am-37' property="oboInOwl:hasExactSynonym" content="Nitrogen monoxide" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00533" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="IMR:0200407" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Nitrogen monoxide" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0100128" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="nitric oxide" datatype="xsd:string"></span>nitric oxide</span>-induced mutagenesis. Biochemistry. 2003; <br />                                              42:3608&#226;&#8364;&#8220;3616. [PubMed: 12653565] <br />                                          15. Hitchcock TM, Dong L, Connor EE, Meira LB, Samson LD, Wyatt MD, Cao W. Oxanine DNA <br />                                              glycosylase activity Mammalian alkyladenine glycosylase. J Biol Chem. 2004; 279:38177&#226;&#8364;&#8220; <br />                                              38183. [PubMed: 15247209] <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                                 Page 17 <br />  <br />  <br />                                          16. Bessho T, Roy R, Yamamoto K, Kasai H, Nishimura S, Tano K, Mitra S. Repair 8- <br />                                              hydroxyguanine DNA mammalian N-methylpurine-DNA glycosylase. Proc Natl Acad Sci U <br />                                              S  1993; 90:8901&#226;&#8364;&#8220;8904. [PubMed: 8415629] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          17. Dosanjh MK, Roy R, Mitra S, Singer B. 1,N6-ethenoadenine preferred 3-methyladenine                                              substrate cloned human N-methylpurine-DNA glycosylase (3-methyladenine-DNA <br />                                              glycosylase). Biochemistry. 1994; 33:1624&#226;&#8364;&#8220;1628. [PubMed: 8110764] <br />                                          18. Saparbaev M, Langouet S, Privezentzev CV, Guengerich FP, Cai H, Elder RH, Laval J. 1,N(2)- <br />                                              ethenoguanine, mutagenic DNA adduct, primary substrate Escherichia coli mismatch- <br />                                              specific <span id='am-59' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span>-DNA glycosylase human alkylpurine-DNA-N-glycosylase. J Biol Chem. 2002; <br />                                              277:26987&#226;&#8364;&#8220;26993. [PubMed: 12016206] <br />                                          19. Berdal KG, Johansen RF, Seeberg E. Release normal bases intact DNA native DNA <br />                                              repair enzyme. EMBO J. 1998; 17:363&#226;&#8364;&#8220;367. [PubMed: 9430628] <br />                                          20. O&apos;Brien PJ, Ellenberger T. Dissecting broad substrate specificity human 3-methyladenine- <br />                                              DNA glycosylase. J Biol Chem. 2004; 279:9750&#226;&#8364;&#8220;9757. [PubMed: 14688248] <br />                                          21. Aamodt RM, Falnes PO, Johansen RF, Seeberg E, Bjoras M. Bacillus subtilis counterpart                                              mammalian 3-methyladenine DNA glycosylase hypoxanthine 1,N6-ethenoadenine                                              preferred substrates. J Biol Chem. 2004; 279:13601&#226;&#8364;&#8220;13606. [PubMed: 14729667] <br />                                          22. Lau AY, Scharer OD, Samson L, Verdine GL, Ellenberger T. Crystal structure human <br />                                              alkylbase-DNA repair enzyme complexed DNA: mechanisms nucleotide flipping base <br />                                              excision. Cell. 1998; 95:249&#226;&#8364;&#8220;258. [PubMed: 9790531] <br />                                          23. Lau AY, Wyatt MD, Glassner BJ, Samson LD, Ellenberger T. Molecular basis discriminating <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              normal damaged bases human alkyladenine glycosylase, AAG. Proc Natl <br />                                              Acad Sci U S  2000; 97:13573&#226;&#8364;&#8220;13578. [PubMed: 11106395] <br />                                          24. Guo HH, Choe J, Loeb LA. Protein tolerance random amino acid change. Proc Natl Acad Sci U <br />                                              S  2004; 101:9205&#226;&#8364;&#8220;9210. [PubMed: 15197260] <br />                                          25. Connor EE, Wyatt MD. Active-site clashes prevent human 3-methyladenine DNA glycosylase <br />                                              improperly removing bases. Chem Biol. 2002; 9:1033&#226;&#8364;&#8220;1041. [PubMed: 12323378] <br />                                          26. Bradford MM. rapid sensitive method quantitation microgram quantities protein <br />                                              utilizing principle protein-dye binding. Anal Biochem. 1976; 72:248&#226;&#8364;&#8220;254. [PubMed: <br />                                              942051] <br />                                          27. Samson LD, Linn S. DNA alkylation repair induction cell death sister chromatid <br />                                              exchange human cells. Carcinogenesis. 1987; 8:227&#226;&#8364;&#8220;230. [PubMed: 3802405] <br />                                          28. Encell LP, Landis DM, Loeb LA. Improving enzymes cancer gene therapy. Nat Biotechnol. <br />                                              1999; 17:143&#226;&#8364;&#8220;147. [PubMed: 10052349] <br />                                          29. Fronza G, Gold B. biological effects N3-methyladenine. J Cell Biochem. 2004; 91:250&#226;&#8364;&#8220; <br />                                              257. [PubMed: 14743385] <br />                                          30. Encell L, Shuker  Foiles PG, Gold B. vitro methylation DNA minor groove <br />                                              binding methyl sulfonate ester. Chem Res Toxicol. 1996; 9:563&#226;&#8364;&#8220;567. [PubMed: 8728498] <br />                                          31. Kelly JD, Inga  Chen FX, Dande P, Shah D, Monti P, Aprile  Burns PA, Scott G, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Abbondandolo  Gold B, Fronza G. Relationship DNA methylation mutational <br />                                              patterns induced sequence selective minor groove methylating agent. J Biol Chem. 1999; <br />                                              274:18327&#226;&#8364;&#8220;18334. [PubMed: 10373436] <br />                                          32. O&apos;Connor TR. Purification characterization human 3-methyladenine-DNA glycosylase. <br />                                              Nucleic Acids Res. 1993; 21:5561&#226;&#8364;&#8220;5569. [PubMed: 8284199] <br />                                          33. Rinne ML, Y, Pachkowski BF, Nakamura J, Kelley MR. N-methylpurine DNA glycosylase <br />                                              overexpression increases alkylation sensitivity rapidly removing non-toxic 7-methylguanine <br />                                              adducts. Nucleic Acids Res. 2005; 33:2859&#226;&#8364;&#8220;2867. [PubMed: 15905475] <br />                                          34. Allinson SL, Sleeth KM, Matthewman GE, Dianov GL. Orchestration base excision repair                                              controlling rates enzymatic activities. DNA Repair (Amst). 2004; 3:23&#226;&#8364;&#8220;31. [PubMed: <br />                                              14697756] <br />                                          35. Gallivan JP, Dougherty DA. Cation-pi interactions structural biology. Proc Natl Acad Sci U S <br />                                               1999; 96:9459&#226;&#8364;&#8220;9464. [PubMed: 10449714] <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                                  Page 18 <br />  <br />  <br />                                          36. Hu G, Gershon PD, Hodel AE, Quiocho FA. mRNA cap recognition: dominant role enhanced <br />                                              stacking interactions methylated bases protein aromatic chains. Proc Natl Acad <br />                                              Sci U S  1999; 96:7149&#226;&#8364;&#8220;7154. [PubMed: 10377383] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          37. Eichman BF, O&apos;Rourke EJ, Radicella JP, Ellenberger T. Crystal structures 3-methyladenine <br />                                              DNA glycosylase MagIII recognition alkylated bases. EMBO J. 2003; 22:4898&#226;&#8364;&#8220;4909. <br />                                              [PubMed: 14517230] <br />                                          38. Maher RL, Vallur AC, Feller JA, Bloom LB. Slow base excision human alkyladenine DNA <br />                                              glycosylase limits rate formation AP sites AP endonuclease 1 does stimulate base <br />                                              excision. DNA Repair (Amst). 2007; 6:71&#226;&#8364;&#8220;81. [PubMed: 17018265] <br />                                          39. Fishel ML, Seo YR, Smith ML, Kelley MR. Imbalancing DNA base excision repair pathway                                              mitochondria; targeting overexpressing N-methylpurine DNA glycosylase mitochondria <br />                                              leads enhanced cell killing. Cancer Res. 2003; 63:608&#226;&#8364;&#8220;615. [PubMed: 12566303] <br />                                          40. Berman HM, Westbrook J, Feng  Gilliland G, Bhat TN, Weissig H, Shindyalov  Bourne PE. <br />                                              Protein Data Bank. Nucleic Acids Res. 2000; 28:235&#226;&#8364;&#8220;242. [PubMed: 10592235] <br />                                          41. Yang Y, Sass LE, Du C, Hsieh P, Erie DA. Determination protein-DNA binding constants                                              specificities statistical analyses single molecules: MutS-DNA interactions. Nucleic Acids <br />                                              Res. 2005; 33:4322&#226;&#8364;&#8220;4334. [PubMed: 16061937] <br />                                          42. O&apos;Brien PJ, Ellenberger T. Human alkyladenine DNA glycosylase uses acid-base catalysis                                              selective excision damaged purines. Biochemistry. 2003; 42:12418&#226;&#8364;&#8220;12429. [PubMed: <br />                                              14567703] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                          Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 1. Modelled AAG bound eA-containing DNA close view substrate <br />                                          binding pocket randomized amino acids <br />                                           ribbon drawing AAG illustrating context residues mutated                                          work surround flipped base [coordinate set 1F4R (Lau et al (23)                                          Protein Data Bank (40)]. flipped base (&#206;&#181;A, 1F4R) depicted space-filling <br />                                          (colored red), His136 Tyr127 (colored yellow), Leu180 (colored green).                                          flipped base fits closely binding pocket stacked Tyr127  <br />                                          His136 Tyr159  B, substrate binding pocket randomized <br />                                          residues. randomized residues contacting &#206;&#181;A substrate (Tyr127, Ala134, Ala135, <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                         Page 20 <br />  <br />  <br />                                          His136, Tyr159, Cys167, Asn169, Leu180), flanking amino acids primary <br />                                          sequence, using oligonucleotide-directed mutagenesis. Residues randomized SK <br />                                          library shown yellow, residues randomized KX library green. Ala134 NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          shown. figure drawn using PyMOL (DeLano, W.L. PyMOL Molecular <br />                                          Graphics (2002) DeLano Scientific, Palo Alto, CA, USA. http://www.pymol.org). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                       Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 2. Characterization unselected selected AAG mutant libraries <br />                                           unselected SK KX libraries. Library size measured plating LB- <br />                                          carbenicillin medium counting colonies; nucleotide amino acid changes                                          determined sequencing 72 individual mutants. assessment MMS resistance, <br />                                          cultures log phase MV1932 cells expressing wild type AAG, SK KX library, <br />                                          inactive AAG (SK dummy vector) treated 0.2% MMS 1 hr 37 &#194;&#176;C,                                          surviving colonies counted. B, distribution amino acid changes SK                                          KX libraries assessed sequences 73 unselected mutants 118 selected <br />                                          mutants survived MMS treatment described  distribution shows post- <br />                                          selection enrichment wild-type AAG sequence shift fewer substitutions <br />                                          clone. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                        Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 3. Amino acid substitutions unselected selected AAG mutants <br />                                           Conservation AAG primary sequence natural evolution. Aligned sequences                                          human AAG homologs shown; numbering refers human sequence. Absolutely <br />                                          conserved residues indicated dark gray numbers boxed. Residues showing <br />                                          conservative substitutions indicated light gray numbers underlined. <br />                                          B-E, Plasmid libraries harboring mutant AAGs created random mutagenesis                                          expressed glycosylase-deficient E. coli MV1932. Mutants complemented                                          methylating agent sensitivity MV1932 cells selected plasmid sequences <br />  <br />  <br />                                             DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                      Page 23 <br />  <br />  <br />                                          determined. Amino acid substitutions shown observed 72 mutants                                          unselected AAG SK KX libraries (B), 89 single mutants selected MMS resistance <br />                                          (C), 25 double mutants selected MMS resistance (D), 42 mutants selected lex <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          resistance (E). Mutations 4 MMS-resistant triple mutants shown. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                        Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 4. MMS resistance conferred wild-type L180F mutant AAG <br />                                           MMS survival repair-proficienct E. coli AB1157, glycosylase-deficient MV1932,                                          MV1932 expressing wild-type L180F mutant AAG. AAG proteins                                          expressed vector pTrcHis2 (Invitrogen) control T7 promoter. Cells <br />                                          grown OD600 0.6-0.85, adjusted &#226;&#710;&#188;108 cells/ml LB-carbenicillin medium                                          treated various concentrations MMS 1 hr 37 &#194;&#176;C. Cells diluted plated <br />                                          triplicate, colonies counted 24 hr. results average                                          different experiments, carried different clone wild-type L180F AAG, <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                         Page 25 <br />  <br />  <br />                                          yielded similar results. Error bars shown standard deviations <br />                                          small indicated y-axis ranges 5 orders magnitude. 0.2% <br />                                          MMS, concentration used selection, difference surviving fraction wild- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          type strain (0.0064% &#194;&#177; 0.0003%; mean standard deviation) mutant strain <br />                                          (0.0283% &#194;&#177; 0.0011%) highly significant (P &lt; 0.005, sample t-test, calculated using <br />                                          OriginPro software, OriginLab Corporation, Northampton, MA). B, Western blot. Aliquots <br />                                          cells taken various times MMS treatment subjected immunoblotting <br />                                          using anti-AAG polyclonal antibody previously described [39]. C, Expression wild- <br />                                          type AAG (closed circles) L180F mutant protein (open circles) quantified                                          phosphor imaging using ImageQuant software (GE Healthcare). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                      Page 26 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Figure 5. Excision 3-meA 7-meG DNA homogeneous wild-type L180F AAG <br />                                           wild-type AAG L180F mutant protein purified described Experimental <br />                                          Procedures. Samples (4 &#206;&#188;g) wild-type fractions IV L180F fraction IV                                          subjected electrophoresis 12.5% SDS-polyacrylamide gel protein visualized <br />                                          staining Coomassie Brilliant Blue G-250. mobility molecular weight <br />                                          standards tracking dye indicated arrows. B,C, DNA glycosylase activity wild- <br />                                          type (closed circles) L180F (open circles). AAG assayed function time                                          measuring release [3H]-labelled 3-meA 7-meG methylated calf thymus <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                   Page 27 <br />  <br />  <br />                                          DNA, described Experimental Procedures. Excision 3-meA 7-meG                                          measured reactions containing 50 500 nM enzyme, respectively. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            DNA Repair (Amst). Author manuscript; available PMC 2014 June 27. <br />                             Chen et al.                                                                                        <br /> </body></html>