<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Cell. 2008 April 25; 30(2): 248&#226;&#8364;&#8220;258. doi:10.1016/j.molcel.2008.02.021. <br />  <br />  <br />  <br />                            Comprehensive Temperature-Sensitive Mutant <br />                            Repository Essential Genes Saccharomyces cerevisiae <br />                            Shay Ben-Aroya1, Candice Coombes2, Teresa Kwok1, Kathryn  O&#226;&#8364;&#8482;Donnell2, Jef D. <br />                            Boeke2,*, Philip Hieter1,* <br />                            1Michael    Smith Laboratories, University British Columbia, Vancouver, BC V6T 1Z4, Canada <br />                            2HighThroughput Biology Center, Johns Hopkins University School Medicine, Baltimore, <br />                            MD 21205, USA <br />  <br />                            SUMMARY <br />                                 Saccharomyces cereivisiae gene deletion project revealed approximately 20% yeast <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 genes required viability. analysis essential genes traditionally relies conditional <br />                                 mutants, typically temperature-sensitive (ts) alleles. developed systematic approach (termed <br />                                 &#226;&#8364;&#339;diploid shuffle&#226;&#8364;?) useful generating ts allele essential gene S. cerevisiae                                 improved genetic manipulation mutant alleles gene constructs general. Importantly,                                 ts allele resides normal genomic locus, flanked specific cognate UPTAG DNTAG bar <br />                                 codes. subset 250 ts mutants, including ts alleles uncharacterized essential genes                                 prioritized genes human counterparts, ready distribution. importance                                 collection demonstrated biochemical genetic screens reveal essential genes <br />                                 involved RNA processing maintenance chromosomal stability. <br />  <br />  <br />                            INTRODUCTION <br />                                              Deciphering functional roles encoded gene products genome central <br />                                              challenge understanding biology organism. study mutant phenotypes                                              proven fundamental approach obtaining detailed understanding gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              function. genetic screens based random mutagenesis seminal                                              identifying genes involved biological processes, random mutagenesis approach rarely <br />                                              achieves saturation, mutability, resulting haploid viable mutants, varies widely <br />                                              genes. <br />  <br />                                              efficiency mutant phenotype studies greatly accelerated yeast <br />                                              Saccharomyces cerevisiae following creation comprehensive Saccharomyces gene <br />                                              deletion (YKO) collection consortium laboratories. collection includes isogenic <br />                                              knockouts virtually annotated yeast ORFs, replaced kanMX4 <br />  <br />  <br />  <br />                            &#194;&#169; 2008 Elsevier  <br />                            * <br />                              Correspondence: jboeke@jhmi.edu (J.D.B.), hieter@msl.ubc.ca (P.H.). <br />                            SUPPLEMENTAL DATA <br />                            Supplemental Data include Supplemental Experimental Procedures, figures, tables article <br />                            online http://www.molecule.org/cgi/content 30/2/248/DC1/. <br />                             Ben-Aroya et al.                                                                                              Page 2 <br />  <br />  <br />                                               cassette confers resistance G418 (Giaever et al., 2002). powerful <br />                                               features YKO collection systematic incorporation 20-mer molecular bar <br />                                               codes (UPTAG DNTAG) unique YKO strain (Shoemaker et al., 1996; Winzeler <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               et al., 1999). Functional profiling populations using TAG microarrays greatly <br />                                               expedited genetic screens quantitative (Birrell et al., 2001; Lum et al., <br />                                               2004; Ooi et al., 2001). <br />  <br />                                               YKO mutants valuable resource genome-wide functional analysis 4700 <br />                                               nonessential genes;  equivalent systematic bar-coded mutant collection exists <br />                                               1107 essential genes. collection essential gene haploid mutants                                               scientifically valuable, enabling systematic genome-wide genetic screens, study                                               individual genes, comprehensive view genetic interaction networks. <br />  <br />                                               Genetic analysis essential genes typically relies conditional mutants carrying point <br />                                               mutations, retain function specific gene set conditions <br />                                               (permissive), lack function different set conditions (nonpermissive),                                               exhibit partial (hypomorphic) function semipermissive conditions. Conditional alleles <br />                                               generated replacement native promoter gene NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               rapidly repressed, tetracycline (tet)-regulatable promoter. approach                                               recently used generate collection promoter-shutoff strains thirds                                               essential yeast gene (Mnaimneh et al., 2004). approach technique termed <br />                                               decreased abundance mRNA perturbation (<span id='am-2' about='obo:IMR_0200160' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-3' property="oboInOwl:id" content="IMR:0200160" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00360" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="dAMP" datatype="xsd:string"></span>DAmP</span>); method, 3&#226;&#8364;&#178;UTR                                               essential gene disrupted insertion antibiotic-resistance marker, destabilizing                                               target mRNA. result protein potentially natural transcriptional regulation <br />                                               reduced levels (Schuldiner et al., 2005). <br />  <br />                                               Traditionally, temperature-sensitive (ts) alleles used great success                                               genetic analysis essential proteins, remain mainstay genetic analysis <br />                                               (Hartwell et al., 1970; Simchen, 1978). vitro mutagenesis method known                                               &#226;&#8364;&#339;plasmid shuffling&#226;&#8364;? widely used generate ts mutations individual genes                                               (Sikorski Boeke, 1991);  method easily scaled encompass                                               genome, generated alleles integrated genome                                               capitalize bar code strategy. alternative method constructing ts alleles involves <br />                                               engineering chromosomal locus gene amino terminus encoded <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               protein fused &#226;&#8364;&#339;heat-inducible degron.&#226;&#8364;? fusion protein targeted degradation <br />                                               37&#194;&#176;C, modulates stability protein (Kanemaki et al., 2003; Labib et al., <br />                                               2000). corresponding mutants, termed td  temperature-activated degron) (Dohmen <br />                                               Varshavsky, 2005), exhibit lethality high temperature approximately 50%                                               essential genes tested. <br />  <br />                                               paper, efficient, scalable method generate ts alleles                                               essential genes S. cerevisiae. resultant strains isogenic YKO deletion <br />                                               collection. ts alleles present normal genomic locus flanked                                               specific cognate UPTAG DNTAG bar codes. addition, basic principles                                               diploid shuffle method corollaries broadly applicable yeast molecular genetic <br />                                               analysis. used method generate ts alleles 250 essential genes, 45  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                           Page 3 <br />  <br />  <br />                                               encode proteins relatively unknown function remaining alleles <br />                                               prioritized based sequence similarity human proteins. biochemical <br />                                               genetic screens ts mutants corresponding 45 genes unknown function <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               used. screens reveal new mutants affect RNA processing chromosome <br />                                               transmission strongly validate use ordered array resource biological <br />                                               discovery. <br />  <br />  <br />                               RESULTS <br />                               Diploid Shuffle Systematic Method Generating ts Alleles Essential Genes                               S. cereviviae <br />                                               goal develop systematic approach screen missense mutations result <br />                                               ts alleles essential gene endogenous chromosomal location flanked                                               appropriate bar codes. <br />  <br />                                               step generating ts allele Favorite Essential Gene (YFEG) use                                               genomic DNA template amplify open reading frame (ORF) including 5&#226;&#8364;&#178; <br />                                               promoter 3&#226;&#8364;&#178; terminator regions using mutagenic amplification (Figure 1A). NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               mutagenized PCR product step 1 cloned SB221+Topo-TA. plasmid <br />                                               contains URA3 gene flanked 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; regions KanMX. Topo-TA cloning <br />                                               site (Invitrogen) inserted KanMX 5&#226;&#8364;&#178; end URA3 gene.                                               site allows direct cloning PCR products need                                               modifications. result cloning step library mutagenized YFEG,                                               transformed E. coli digested release linear fragments (following DNA <br />                                               purification). linear fragments directly transformed corresponding strain <br />                                               haploid convertible &#226;&#8364;&#339;heterozygous&#226;&#8364;? diploid YKO collection (Figures 1B 1C). <br />                                               collection individually incorporating modified SGA reporter <br />                                               can1&#206;&#8221;::LEU2-MFA1pr-HIS3 set heterozygous diploid YKOs originally <br />                                               constructed Saccharomyces Genome Deletion project (available Open <br />                                               Biosystems, J.D.B., S. Sookhai-Mahadeo, X. Wang, C. Tiffany, O. Chen, F  Spencer, D.S. <br />                                               Yuan, P.B. Meluh, unpublished data). ~700 bp KanMX5&#226;&#8364;&#178; KanMX3&#226;&#8364;&#178; fragments <br />                                               direct mutagenized YFEG library yfeg&#206;&#8221;::KanMX genomic locus homologous <br />                                               recombination (Figure 1C). Ura+ transformants selected. Pools Ura+ cells containing <br />                                               mutant alleles sporulated (Figure 1D). Spores spread haploid-selective <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               medium incubated 25&#194;&#176;C 5 days colony formation (Figure 1D). haploid <br />                                               MATa Ura+ spores containing integrated mutagenized allele YFEG grow                                               medium. Colonies formed selective medium replica plated incubated 37&#194;&#176;C                                               2 days (Figure 1E). Colonies growing 25&#194;&#176;C, 37&#194;&#176;C, selected potential ts <br />                                               alleles restreaked temperatures retesting. Retesting backcrossing <br />                                               potential ts allele donor strain  Experimental Procedures). Following <br />                                               sporulation tetrad dissection, confirmed (1) ts phenotype segregates                                               Mendelian manner (2:2), indicative depends single mutated gene; (2) linked <br />                                               URA3 cosegregates mutated PCR product; (3) mutagenized <br />                                               PCR product integrated correct genomic locus, rendering cells G418 sensitive <br />                                               (5&#226;&#8364;&#178;kanMX::yfeg-ts-URA3::3&#226;&#8364;&#178;kanMX). <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                             Page 4 <br />  <br />  <br />                                               summary, final product method described confirmed MATa <br />                                               strain YKO collection genetic background containing URA3-marked ts allele                                               specific essential gene integrated endogenous locus flanked bar codes. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               addition, diploid shuffle method variations  Figures S3 S4 available <br />                                               online Discussion) represent significantly improved methods generating strains <br />                                               carrying specific alleles, transferring ts alleles variety strain backgrounds                                               genetic contexts, introduction gene constructs fusion proteins, <br />                                               heterozygous expression cassettes,  <br />  <br />                               Construction ts Strains 45 Essential Genes Unknown Function <br />                                               generated initial collection ts strains provide early-stage resource value                                               community genetic phenotypic screening. chose 45 yeast ORFs known time <br />                                               started essential viability defined S. cerevisiae Genome Data <br />                                               Base (SGD) uncharacterized ORFs, proteins unknown function unknown <br />                                               molecular function. Thirty ORFs (67%) significant human homolog                                               E value ranges 3.00E-05 1.00E-261 (Table S1). <br />  <br />                                               gene, ~15,000 independent PCR-mutagenized clones screened potential ts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               phenotypes. 42 45 cases, successfully isolated ts allele round                                               screening. remaining cases, second round mutagenesis required, <br />                                               resulting successful isolation ts mutants genes attempted.                                               sequencing 12 independent ts alleles, average mutation frequency 0.54 <br />                                               base substitution 100 bp, translates average 0.9 mutations 100 amino <br />                                               acids (Figure S1). demonstrate potential new discovery, screened                                               collection 45 ts mutants using established biochemical genetic screens used <br />                                               extensively past. <br />  <br />                               Screen RNA-Processing Defects Reveals Previously Unidentified Mutants <br />                                               Production ribosomal RNA eukaryotic cells considerably complex                                               prokaryotes, single enzyme, RNase III, makes needed cleavages ribosome <br />                                               assembly cotranscriptional. eukaryotic rRNA organized similarly, 35S <br />                                               precursor, cleavages carried protein ribonucleoprotein complexes <br />                                               (including RNase III [Kufel et al., 1999]). eukaryotes, molecularly intertwined steps <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               rRNA processing ribosome assembly slower, steps                                               posttranscriptional. combined molecular weight small nucleolar ribonucloprotein <br />                                               particle (snoRNP) complexes involved ribosome assembly  present prokaryotes) <br />                                               exceeds molecular weight ribosome &gt;15-fold (Venema Tollervey, 1999). <br />                                               Production rRNA nucleolus, transport r proteins                                                 nucleus, imposes requirements assembly absent prokaryotes (Hage Tollervey, <br />                                               2004). 170 proteins 100 small RNAs estimated participate eukaryotic <br />                                               rRNA processing ribosome assembly (Andersen et al., 2002; Fromont-Racine et al., <br />                                               2003; Venema Tollervey, 1999) addition 79 proteins RNAs make <br />                                               ribosome. <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                         Page 5 <br />  <br />  <br />                                               complexity RNA-processing defects easily identified                                               simple RNA-blotting techniques (Lindahl et al., 1992; Peng et al., 2003), hypothesized <br />                                               unknown essential genes contribute RNA processing. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               performed specific assays detect defects rRNA, mRNA, snoRNA, snRNA <br />                                               processing collection 45 ts mutants (relatively) unknown function (Table S1) <br />                                               positive control nme1 ts mutant  kind gift Mark Schmitt). Mutants                                               grown permissive temperature shifted nonpermissive temperature 2 hr,                                               total RNA harvested, resolved appropriate gels, blotted series                                               hybridization probes detect appropriate RNA precursors. Processing defects                                               readily revealed buildup intermediates larger mature RNA species                                               study (Table S2). Consistent large number known genes affecting rRNA <br />                                               processing, identified new ts mutants obvious rRNA-processing defects (Figure <br />                                               4). revealed probing internal transcribed spacer 1 (ITS1) sequences <br />                                               (Figure 2) mature rRNA probes (Figure S2, Table 1). ts mutants severely affecting <br />                                               pre-mRNA, snoRNA, snRNA-processing pathways detected set                                               mutants. <br />  <br />                                               apparent Figure 2 Table 1, rRNA-processing mutants ranging nearly NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               severe control  nme1 mutant) milder defects observed                                               hyperaccumulation 35S, 33S,  cases, reduced accumulation 27S precursor <br />                                               rRNA species. mutants significant deficit mature 18S 25S <br />                                               rRNA, suggesting direct involvement rRNA processing (Figure S2 Table S3).                                               proteins corresponding mutants, ydr339c yor004w, biochemically <br />                                               identified important components, Utp23p 24p, small-subunit rRNA-processing <br />                                               machinery (Bleichert et al., 2006; Rempola et al., 2006) course                                               experiments, providing critical internal validation assay approach. <br />                                               clearly integral parts rRNA-processing machinery, mutants                                               affect rRNA processing directly. mechanisms; half                                               yeast&#226;&#8364;&#8482;s energy goes ribosomes. example, defects secretion downregulate <br />                                               ribosome biogenesis (Mizuta Warner, 1994). <br />  <br />                               Crosstalk RNA-Processing Pathways <br />                                               searched pathway connections newly identified ts mutants. YJL069C <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               (UTP18) previously predicted function processing U3 snoRNA (Bernstein et al., <br />                                               2004; Samanta Liang, 2003). rRNA-processing defect yjl069-ts provides genetic <br />                                               evidence supporting hypothesis, U3 snoRNA essential rRNA processing. <br />                                               Similarly, YGR145W (ENP2) gene encodes protein known interact MPP10p <br />                                               component SSU processome (SGD); SSU processome responsible                                               processing 18S rRNA. protein encoded YKR022C (NTR2) reportedly                                               component Prp43 helicase spliceosome disassembly complex (Tsai et al., 2005);                                               data suggest facilitate rRNA processing, accord recent studies <br />                                               (Combs et al., 2006; Lebaron et al., 2005; Leeds et al., 2006). Interestingly, YKR022C <br />                                               gene identified  YLR132C, predicted pre-mRNA splicing factors                                               computational analysis (Hazbun et al., 2003), observed defects.                                               YAL043C (PTA1) gene encodes subunit holo-CPF complex involved mRNA 3&#226;&#8364;&#178; <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                            Page 6 <br />  <br />  <br />                                               end formation/polyadenylation (Zhao et al., 1999) tRNA biogenesis (O&#226;&#8364;&#8482;Connor                                               Peebles, 1992). discovery set proteins appear function multiple RNA- <br />                                               processing networks exciting, consistent prior studies (Burckin et al., 2005; Leeds et <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               al., 2006), provides new evidence cross-RNA-processing pathway interactions. <br />  <br />                               Connection Coenzyme Biosynthesis rRNA Processing? <br />                                               rRNA-processing mutants ts genes predicted roles <span id='am-7' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span> <br />                                               biosynthesis (YDR196C YGR277C), based homology prokaryotic CoA <br />                                               biosynthesis proteins. observed defects relatively specific rRNA processing                                               represent general pleiotropic defect. likely CoA derivative,                                               proteins se, affects (directly indirectly) rRNA processing. <br />  <br />                               Screens Chromosome Instability <br />                                               S. cerevisiae attractive identifying characterizing genes involved chromosome <br />                                               segregation. Recently, systematic screening 4700 nonessential gene deletion mutants <br />                                               identified 200 nonessential genes contributing faithful chromosome <br />                                               segregation (Yuen et al., 2007), systematic screening essential gene mutants                                               possible. decided search novel CIN genes screening 45 ts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               mutant collection CIN phenotype. different tests used (Figure 3):                                               chromosome transmission fidelity (CTF) (Spencer et al., 1990) like faker (ALF) <br />                                               (Gerring et al., 1990) assays haploids diploid bimater test (BiM) (Kouprina et al., <br />                                               1988) MATa/MAT&#206;&#177; diploids. Screening different assays broadens                                               spectrum identifiable genes allowing various defects chromosome transmission                                               detected (Yuen et al., 2007). <br />  <br />                                               CIN assays, specific ts alleles tested wide range semi-                                               nonpermissive temperatures (25&#194;&#176;C, 30&#194;&#176;C, 32&#194;&#176;C, 34&#194;&#176;C, 37&#194;&#176;C). total, 45 ts <br />                                               mutants (20%) exhibited increased chromosome instability (CIN) defined new <br />                                               CIN genes (Table 2 Figure 4). Typical results presented mutants <br />                                               (Figure 3). severity CIN phenotypes quantified Table 2. mutants <br />                                               (ynl152w, yhr020w, keg1, yhr122w, dre2, yor262w) define uncharacterized ORFs. <br />                                               additional mutants, swc4, trs23, kae1, previously identified coding                                               components Swr1 (Krogan et al., 2003), TRAPP (Sacher et al., 2000), KEOPS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               (Bianchi Shore, 2006) complexes, respectively, molecular function                                               unknown. <br />  <br />                                               wanted estimate frequently expect ts mutant (isolated random <br />                                               mutagenesis procedure) subsequently reveal detectable hypomorphic CIN phenotype                                               reduced (semipermissive) temperature. addresses issue false negatives                                               phenotypic screening hypomorphs. accomplish  generated ts alleles PSF2 <br />                                               NSE4 tested CIN phenotype range reduced temperatures. Psf2 <br />                                               represents known subunit GINS complex (Sld5p, Psf1p, Psf2p, Psf3p) (Takayama et <br />                                               al., 2003). Loss Psf2p function depletion deletion shown cause <br />                                               chromosome missegregation Schizosaccharomyces pombe human cells (Huang et <br />                                               al., 2005). Nse4 known interact Nse3 Nse5 components Mms21- <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                            Page 7 <br />  <br />  <br />                                               Smc5-Smc6 DNA repair complex. suggested Nse4 plays role                                               maintenance higher-order chromosome structure (Hu et al., 2005). shown Figure 4 <br />                                               Table 2, newly constructed ts alleles psf2 nse4 expected CIN <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               phenotype semipermissive temperatures. addition, similar phenotypes caused                                               newly constructed ts alleles components GINS complex (Psf1 Sld5) <br />                                               Mms21-Smc5-Smc6 complex (Nse3 Nse5).  able isolate new ts <br />                                               alleles subsequently detect CIN phenotype semipermissive temperatures                                               alleles genes tested. <br />  <br />                               Previously Unidentified Essential Genes Involved Sister Chromatid Cohesion <br />                                               test sister chromatid cohesion defects specific CIN mutants, mutant                                               crossed tet repressor-GFP/tet operator repeat strain (Michaelis et al., 1997).                                               markers strain allow visualization specific chromosomal locus assess                                               sister chromatids juxtaposition separated. resulting diploids                                               sporulated, haploid strains containing appropriate markers identified                                               analyzed cohesion defects. <br />  <br />                                               wild-type, 8% cells arrested G2/M GFP signals, indicating sister <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               chromatids efficiently paired prior anaphase. expected (Mayer et al., 2001), ctf18 <br />                                               control strain, 35% cells arrested G2/M GFP signals, indicating sister <br />                                               chromatid cohesion defect. new ts mutants exhibiting CIN phenotype,                                               (yor262w, dre2, swc4, yhr122w) associated sister chromatid cohesion defects <br />                                               values ranging 23% 44% (Figure 5A). defects sister <br />                                               chromatid cohesion defects opposed aneuploidy; 4%&#226;&#8364;&#8220;7% wild-type ts <br />                                               mutant cells arrested G1 GFP signals. <br />  <br />                               YHR122W, SWC4, YOR262W Required Establishment Sister Chromatid <br />                               Cohesion <br />                                               Sister chromatid cohesion established S phase maintained anaphase onset. <br />                                               mutants sister chromatid cohesion defects (yhr122w, swc4, yor262w)                                               used temperature-shift experiments assess roles establishment maintenance                                               cohesion. experimental design described Figure 5B. gene                                               affects establishment, results obtained sample 3 resemble sample 1, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                maintenance defects, results sample 3 resemble sample 2.                                               ts mutants, values GFP signals sample 3 similar obtained                                               sample 1, sample 2 (Figure 5C). conclude essential genes                                               play role establishment sister chromatid cohesion. <br />  <br />                               Resource 250 Essential Gene ts Mutants <br />                                               expanded initial collection 45 ts mutants additional 205 essential genes.                                               decided focus essential genes ts alleles available                                               yeast community. assess ts allele available individual ORF,                                               referred updated list ts alleles collected Boone laboratory (C. Boone, personal <br />                                               communication). collection represents ~40% essential gene spectrum <br />                                               corresponding 450 essential genes. extant alleles generated yeast <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                            Page 8 <br />  <br />  <br />                                               community past 40 years transferred YKO genetic <br />                                               background non-bar-coded format (C. Boone, personal communication).                                               remaining 657 essential genes ts alleles currently available, chose 131 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               genes homology human sequence (E value range 3.00E-05 1.00E-261) 74 <br />                                               additional genes (chosen randomly) add collection  Table S4) generated ts <br />                                               alleles  total, 250 ts mutants array format available                                               community request. ts allele, matched MATa MAT&#206;&#177; strains available. <br />  <br />  <br />                               DISCUSSION <br />                                               method making comprehensive set ts conditional mutants                                               essential genes Saccharomyces cerevisiae. demonstrate feasibility create                                               initial community resource, generated subset ts alleles essential genes                                               unknown function. used biochemical (RNA processing) genetic (chromosomal <br />                                               stability sister chromatid cohesion) screens identify new genes important                                               essential functions. initial collection 45 ts mutants expanded 250 strains,                                               essential genes significant human homologs prioritized. collection                                               ready distribution. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Generating ts alleles individual genes conventional methods tedious readily <br />                                               scalable genome-wide. approach described exploits features &#226;&#8364;&#339;haploid- <br />                                               convertible&#226;&#8364;? heterozygous diploid collection, using SGA reporter select                                               integrated ts-allele flanked corresponding bar codes. method diploid <br />                                               shuffle distinguish standard plasmid shuffle method. Topo-TA modified <br />                                               plasmid saves time effort cloning step mutagenized library. approach <br />                                               validated generating ts alleles;  used applications. Using <br />                                               Topo-TA plasmid protocol described (Figure 1), extant ts allele easily <br />                                               transferred deletion collection genetic background (&#226;&#8364;&#339;allele transfer ?). result                                               integrated allele, marked URA3, flanked appropriate bar codes  Figure <br />                                               S3 details).  using Topo-TA plasmid, PCR product (mutant allele, <br />                                               fusion protein, heterologous gene expression cassette, ) introduced                                               6000 genomic sites carrying KanMX replacement cassette integration site, <br />                                               depending specific heterozygote chosen recipient strain. addition, using <br />                                               primers external primers used original mutagenesis, URA3-marked ts allele <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               gene construct easily transferred deletion set genetic background                                               strain (&#226;&#8364;&#339;allele transfer &#226;&#8364;? Figure S4).  ts mutation gene <br />                                               construct analyzed specific phenotypes various genetic <br />                                               contexts. <br />  <br />                                               model organisms, function known gene predicted ORF analyzed                                               assessing specific phenotype gene ORF deleted inactivated.                                               important milestone yeast functional genomics complete set deletion mutants <br />                                               (Giaever et al., 2002). collection widely used direct screening recessive <br />                                               phenotypes haploids establishing genetic interactions haploid double mutants. <br />                                                systematic screens currently possible ~4700 nonessential <br />                                               genes. essential genes, haploids carrying conditional hypomorphic alleles  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                Page 9 <br />  <br />  <br />                                               required. screens release ts allele resource described paper <br />                                               demonstrate value systematic ts mutant resource. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               screen ribosomal RNA processing netted 11 mutants affecting process relatively <br />                                               severely specifically. considerably expands list approximately 110 genes <br />                                               previously identified using tet-regulated essential gene constructs microarray approach <br />                                               detecting rRNA precursors displaying increases rRNA precursor abundance                                               10-fold (Peng et al., 2003). addition genes previously identified <br />                                               biochemically components processome, additional genes identified. <br />                                               gene products implicated forms RNA processing, implying <br />                                               crosstalk central rRNA, pre-mRNA, snoRNA, tRNA-processing pathways. <br />                                               identified genes products localized nucleus nucleolus expected <br />                                               proteins directly involved rRNA processing, showed significant defect                                               mature rRNA.  mutants, reduction activity affected <br />                                               protein affect rRNA processing indirectly, e.g., affect processing snoRNAs, <br />                                               direct rRNA processing. <br />  <br />                                               mutants predicted affect CoA biosynthesis strongly affected rRNA processing, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               suggesting requirement CoA derivative rRNA processing. possible <br />                                               explanations specific protein acetylation regulates rRNA processing,  <br />                                               alternatively, free CoA derivative serve signal metabolic <br />                                               state cell. make sense decrease readily usable cellular carbon sources <br />                                               downregulated ribosomes/translation machinery. <br />  <br />                                               paper, CIN screening ts mutant collection resource.                                               past 20 years, phenotype-based screens chromosome missegregation generated                                               number mutant collections (CTF, CHL, MCM, CIN), identifying genes                                               functionally conserved eukaryotes (Hoyt et al., 1990; Kouprina et al., 1988; Maine <br />                                               et al., 1984; Meeks-Wagner et al., 1986; Ouspenski et al., 1999; Spencer et al., 1990; Yuen <br />                                               et al., 2007). Results extended effort isolate mutants affect chromosome <br />                                               stability suggest yeast S. cerevisiae close saturation respect. Using <br />                                               subset ts alleles new collection, isolated new CIN mutants.                                               original ctf collection identified ~50 genes necessary high-fidelity segregation                                               chromosomes (Spencer et al., 1990). <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, 25 genes represented original ctf <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               collection cloned characterized, encode functions important                                               sister chromatid cohesion (40%). connection sister chromatid cohesion <br />                                               function CIN emphasized secondary screen,                                               new CIN mutants associated cohesion defect (Figure 5A).                                               potential role cohesion mutants establishment stage (Skibbens et al., <br />                                               1999). Ctf7 shown involved establishment sister chromatid <br />                                               cohesion, essential genes role described. Based                                               results, believe early release subset 250 ts mutants provide valuable <br />                                               resource community enrich spectrum mutants affecting various cellular <br />                                               functions. <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                            Page 10 <br />  <br />  <br />                                               obvious use resource study function individual genes, including <br />                                               poorly characterized ORFs. broader view, collection used directly,                                               modifications, complete genome-wide studies used nonessential gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               deletion collection resource. applications include incorporation hypomorphic <br />                                               alleles synthetic genetic network. MATa version collection contains                                               uniform genetic background  Experimental Procedures), strains <br />                                               contains LEU2-marked SGA reporter.  ts collection used directly <br />                                               synthetic genetic array (SGA) (Tong et al., 2001) epistatic miniarray profiles (eMAPs) <br />                                               (Schuldiner et al., 2005). cases, MATa strains provided array form                                               enrich deletion set. addition, individual mutants used query                                               SGA analysis. alternative approach SGA SLAM (Ooi et al., 2003).                                               unique features collection integrated alleles flanked bar <br />                                               codes. growth rates ts mutants differ nonpermissive temperatures                                               different alleles vary, believe diploid version SLAM, dSLAM, preferable <br />                                               (Pan et al., 2004). addition, ts mutants introduced query mutation <br />                                               transformation assess synthetic lethal profile (using allele transfer method, <br />                                               Figure S4). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               approaches developed yeast community study essential <br />                                               genes, resulted collections available community: <br />                                               DAmP alleles (Schuldiner et al., 2005), N-degron (Kanemaki et al., 2003; Labib et al., <br />                                               2000), tet promoter collections (Mnaimneh et al., 2004). main drawback                                               approach fact  following integration mutagenized PCR product,                                               relatively laborious screening process needed isolate ts allele. methods, <br />                                               final product ready use following simple step homologous recombination, <br />                                               avoiding screening.  approach important advantage promises                                               high rate success. certain gene ts isolated round, second round <br />                                               screening, changing mutagenesis conditions, usually result positive isolate. <br />                                               criterion defining mutant ts clear (inability grow high <br />                                               temperatures), ts mutations isolated gene                                                approaches, step needed modified gene, resultant allele                                               associated clear phenotype. tet promoter collection, rapid                                               sufficient elimination protein function problematic large fraction proteins <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               (Aparicio, 2003). nearly half cases tested, shutting promoter did affect <br />                                               phenotypes gave slight phenotype (Mnaimneh et al., 2004). 104 <br />                                               essential genes fused N-degron sequences, nearly 40% did result                                               inviability nonpermissive temperature (Kanemaki et al., 2003), indicating                                               rapid protein depletion N-degron uniform proteome. Similarly,                                               DAmP approach works approximately essential genes                                               majority. cases effect observed following                                               modification, experimental options limited, rate success                                               methods lower obtained ts alleles. <br />  <br />                                               Genetic analysis essential genes typically relies conditional mutants. cases, ts <br />                                               alleles provide wide spectrum activity ranging permissive nonpermissive <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                               Page 11 <br />  <br />  <br />                                               temperatures, providing flexibility phenotype assessment. tet promoter strains,                                               promoter state, making difficult achieve intermediate expression <br />                                               levels (Schuldiner et al., 2005). DAmP alleles conditional alleles&#226;&#8364;&#8221;reduced <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               expression constitutive. Finally, method, integrated allele controlled                                               natural promoter terminator protein wild-type structure lacking                                               extensions tags. tet DAmP strains, promoter terminator modified. <br />                                               degron  amino terminus encoded protein fused 20 KD <br />                                               fragment, affect function targeted protein. <br />  <br />                                               Taking considerations  comprehensive collection ts missense mutations, <br />                                               integrated, bar-coded format, represents excellent resource analyzing essential <br />                                               gene function significantly enhance discovery, genome crossreferencing,                                               functional analysis. <br />  <br />  <br />                               EXPERIMENTAL PROCEDURES <br />                                               details, Supplemental Experimental Procedures. <br />  <br />                               Yeast Strains <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               ts strains derived yeast heterozygous gene deletion collection carrying                                               haploid selection marker can1&#206;&#8221;::LEU2-MFA1pr-HIS3 (Open Biosystems catalogue number <br />                                               YSC4428). <br />  <br />                               Plasmids <br />                                               SB221 modified starting plasmid M4758 (Voth et al., 2003) allow                                               efficient targeted integration KanMX locus heterozygous deletion strains.                                               BamHII site M4758 modified Topo-TA site easier recombinant cloning. <br />  <br />                               RNA-Processing Mutant Screens <br />                                               independent clones mutant strain grown permissive                                               nonpermissive temperature prior harvesting pellets. RNA prepared, standard <br />                                               northern blot procedure preformed. RNA blots probed biotin-labeled <br />                                               riboprobes. Subsequently, blots reprobed probe containing entire unit NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               yeast rDNA. <br />  <br />                               Construction ts Mutants <br />                                               ts strains constructed described Results; detailed procedures,                                               Supplemental Experimental Procedures. <br />  <br />                               CIN Assays <br />                                               CTF, genomic BiM, ALF performed previously described (Yuen et al., 2007). <br />  <br />  <br />                               Supplementary Material <br />                                               Refer Web version PubMed Central supplementary material. <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                          Page 12 <br />  <br />  <br />                               Acknowledgments <br />                                               thank  Carter, M. Tao, M. Kofoed, B. Le, L. Tang technical assistance; M. Kupiec, K. McManus, G. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Simchen, X. Pan helpful discussions; J. Woolford helpful comments earlier version                                               manuscript. Supported National Institutes Health (NIH) grant P01 CA16519 P.H. J.D.B., <br />                                               Canadian Institute Health Research (CIHR) grant MOP-38096 P.H., NIH roadmap grant U54 RR020839 <br />                                                J.D.B.). S.B.  supported Human Frontier Science Program (HFSP) long-term fellowship                                               research grant Killam trust. <br />  <br />  <br />                               References <br />                                               Andersen JS, Lyon CE, Fox AH, Leung AK, Lam YW, Steen H, Mann M, Lamond AI. Directed <br />                                                 proteomic analysis human nucleolus. Curr Biol. 2002; 12:1&#226;&#8364;&#8220;11. [PubMed: 11790298] <br />                                               Aparicio OM. Tackling essential problem functional proteomics Saccharomyces cerevisiae. <br />                                                 Genome Biol. 2003; 4:230. Published online September 24, 2003. 10.1186/gb-2003-4-10-230 <br />                                                 [PubMed: 14519191] <br />                                               Bernstein KA, Gallagher JE, Mitchell BM, Granneman S, Baserga SJ. small-subunit processome <br />                                                 ribosome assembly intermediate. Eukaryot Cell. 2004; 3:1619&#226;&#8364;&#8220;1626. [PubMed: 15590835] <br />                                               Bianchi  Shore D. KEOPS complex: rosetta stone telomere regulation? Cell. 2006; <br />                                                 124:1125&#226;&#8364;&#8220;1128. [PubMed: 16564005] <br />                                               Birrell GW, Giaever G, Chu  Davis RW, Brown JM. genome-wide screen Saccharomyces <br />                                                 cerevisiae genes affecting UV radiation sensitivity. Proc Natl Acad Sci USA. 2001; 98:12608&#226;&#8364;&#8220; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 12613. [PubMed: 11606770] <br />                                               Bleichert F, Granneman S, Osheim YN, Beyer AL, Baserga SJ. PINc domain protein Utp24,                                                 putative nuclease, required early cleavage steps 18S rRNA maturation. Proc Natl Acad <br />                                                 Sci USA. 2006; 103:9464&#226;&#8364;&#8220;9469. [PubMed: 16769905] <br />                                               Burckin T, Nagel R, Mandel-Gutfreund Y, Shiue L, Clark TA, Chong JL, Chang TH, Squazzo S, <br />                                                 Hartzog G, Ares M Jr. Exploring functional relationships components gene <br />                                                 expression machinery. Nat Struct Mol Biol. 2005; 12:175&#226;&#8364;&#8220;182. [PubMed: 15702072] <br />                                               Combs DJ, Nagel RJ, Ares M Jr, Stevens SW. Prp43p DEAH-box spliceosome disassembly factor <br />                                                 essential ribosome biogenesis. Mol Cell Biol. 2006; 26:523&#226;&#8364;&#8220;534. [PubMed: 16382144] <br />                                               Dohmen RJ, Varshavsky  Heat-inducible degron making conditional mutants. Methods <br />                                                 Enzymol. 2005; 399:799&#226;&#8364;&#8220;822. [PubMed: 16338396] <br />                                               Fromont-Racine M, Senger B, Saveanu C, Fasiolo F. Ribosome assembly eukaryotes. Gene. 2003; <br />                                                   313:17&#226;&#8364;&#8220;42. [PubMed: 12957375] <br />                                               Gerring SL, Spencer F, Hieter P. CHL 1 (CTF 1) gene product Saccharomyces cerevisiae                                                   important chromosome transmission normal cell cycle progression G2/M. EMBO J. <br />                                                   1990; 9:4347&#226;&#8364;&#8220;4358. [PubMed: 2265610] <br />                                               Giaever G, Chu  Ni L, Connelly C, Riles L, Veronneau S, Dow S, Lucau-Danila  Anderson K, <br />                                                   Andre B, et al. Functional profiling Saccharomyces cerevisiae genome. Nature. 2002; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   418:387&#226;&#8364;&#8220;391. [PubMed: 12140549] <br />                                               Hage AE, Tollervey D. surfeit factors: ribosome assembly complicated                                                   eukaryotes bacteria? RNA Biol. 2004; 1:10&#226;&#8364;&#8220;15. [PubMed: 17194932] <br />                                               Hartwell LH, Culotti J, Reid B. Genetic control cell-division cycle yeast. Detection                                                   mutants. Proc Natl Acad Sci USA. 1970; 66:352&#226;&#8364;&#8220;359. [PubMed: 5271168] <br />                                               Hazbun TR, Malmstrom L, Anderson S, Graczyk BJ, Fox B, Riffle M, Sundin BA, Aranda JD, <br />                                                   McDonald WH, Chiu CH, et al. Assigning function yeast proteins integration                                                   technologies. Mol Cell. 2003; 12:1353&#226;&#8364;&#8220;1365. [PubMed: 14690591] <br />                                               Hoyt MA, Stearns T, Botstein D. Chromosome instability mutants Saccharomyces cerevisiae                                                   defective microtubule-mediated processes. Mol Cell Biol. 1990; 10:223&#226;&#8364;&#8220;234. [PubMed: <br />                                                   2403635] <br />                                               Hu B, Liao C, Millson SH, Mollapour M, Prodromou C, Pearl LH, Piper PW, Panaretou B. Qri2/Nse4, <br />                                                   component essential Smc5/6 DNA repair complex. Mol Microbiol. 2005; 55:1735&#226;&#8364;&#8220;1750. <br />                                                   [PubMed: 15752197] <br />  <br />  <br />  <br />                                                      Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                   Page 13 <br />  <br />  <br />                                               Huang HK, Bailis JM, Leverson JD, Gomez EB, Forsburg SL, Hunter T. Suppressors Bir1p <br />                                                   (Survivin) identify roles chromosomal passenger protein Pic1p (INCENP)                                                   replication initiation factor Psf2p chromosome segregation. Mol Cell Biol. 2005; 25:9000&#226;&#8364;&#8220;9015. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   [PubMed: 16199877] <br />                                               Kanemaki M, Sanchez-Diaz  Gambus  Labib K. Functional proteomic identification DNA <br />                                                   replication proteins induced proteolysis vivo. Nature. 2003; 423:720&#226;&#8364;&#8220;724. [PubMed: <br />                                                   12768207] <br />                                               Kouprina N, Pashina OB, Nikolaishwili NT, tsouladze  Larionov VL. Genetic control                                                   chromosome stability yeast Saccharomyces cerevisiae. Yeast. 1988; 4:257&#226;&#8364;&#8220;269. [PubMed: <br />                                                   3064490] <br />                                               Krogan NJ, Keogh MC, Datta N, Sawa C, Ryan OW, Ding H, Haw RA, Pootoolal J, Tong  <br />                                                   Canadien V, et al. Snf2 family ATPase complex required recruitment histone H2A <br />                                                   variant Htz1. Mol Cell. 2003; 12:1565&#226;&#8364;&#8220;1576. [PubMed: 14690608] <br />                                               Kufel J, Dichtl B, Tollervey D. Yeast Rnt1p required cleavage pre-ribosomal RNA                                                   3&#226;&#8364;&#178; ETS 5&#226;&#8364;&#178; ETS. RNA. 1999; 5:909&#226;&#8364;&#8220;917. [PubMed: 10411134] <br />                                               Labib K, Tercero JA, Diffley JF. Uninterrupted MCM2&#226;&#8364;&#8220;7 function required DNA replication fork <br />                                                   progression. Science. 2000; 288:1643&#226;&#8364;&#8220;1647. [PubMed: 10834843] <br />                                               Lebaron S, Froment C, Fromont-Racine M, Rain JC, Monsarrat B, Caizergues-Ferrer M, Henry Y.                                                   splicing ATPase prp43p component multiple preribosomal particles. Mol Cell Biol. 2005; <br />                                                   25:9269&#226;&#8364;&#8220;9282. [PubMed: 16227579] <br />                                               Leeds NB, Small EC, Hiley SL, Hughes TR, Staley JP. splicing factor Prp43p, DEAH box <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   ATPase, functions ribosome biogenesis. Mol Cell Biol. 2006; 26:513&#226;&#8364;&#8220;522. [PubMed: <br />                                                   16382143] <br />                                               Lindahl L, Archer RH, Zengel JM. new rRNA processing mutant Saccharomyces cerevisiae. <br />                                                   Nucleic Acids Res. 1992; 20:295&#226;&#8364;&#8220;301. [PubMed: 1741255] <br />                                               Lum PY, Armour CD, Stepaniants SB, Cavet G, Wolf MK, Butler JS, Hinshaw JC, Garnier P, <br />                                                   Prestwich GD, Leonardson  et al. Discovering modes action therapeutic compounds using <br />                                                   genome-wide screen yeast heterozygotes. Cell. 2004; 116:121&#226;&#8364;&#8220;137. [PubMed: 14718172] <br />                                               Maine GT, Sinha P, Tye BK. Mutants S. cerevisiae defective maintenance                                                   minichromosomes. Genetics. 1984; 106:365&#226;&#8364;&#8220;385. [PubMed: 6323245] <br />                                               Mayer ML, Gygi SP, Aebersold R, Hieter P. Identification RFC(Ctf18p, Ctf8p, Dcc1p):                                                   alternative RFC complex required sister chromatid cohesion S. cerevisiae. Mol Cell. 2001; <br />                                                   7:959&#226;&#8364;&#8220;970. [PubMed: 11389843] <br />                                               Meeks-Wagner D, Wood JS, Garvik B, Hartwell LH. Isolation genes affect mitotic <br />                                                   chromosome transmission S. cerevisiae. Cell. 1986; 44:53&#226;&#8364;&#8220;63. [PubMed: 3510080] <br />                                               Michaelis C, Ciosk R, Nasmyth K. Cohesins: chromosomal proteins prevent premature separation <br />                                                   sister chromatids. Cell. 1997; 91:35&#226;&#8364;&#8220;45. [PubMed: 9335333] <br />                                               Mizuta K, Warner JR. Continued functioning secretory pathway essential ribosome <br />                                                   synthesis. Mol Cell Biol. 1994; 14:2493&#226;&#8364;&#8220;2502. [PubMed: 8139552] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Mnaimneh S, Davierwala AP, Haynes J, Moffat J, Peng WT, Zhang W, Yang X, Pootoolal J, Chua G, <br />                                                   Lopez  et al. Exploration essential gene functions titratable promoter alleles. Cell. 2004; <br />                                                   118:31&#226;&#8364;&#8220;44. [PubMed: 15242642] <br />                                               O&#226;&#8364;&#8482;Connor JP, Peebles CL. PTA1, essential gene Saccharomyces cerevisiae affecting pre-tRNA <br />                                                   processing. Mol Cell Biol. 1992; 12:3843&#226;&#8364;&#8220;3856. [PubMed: 1508188] <br />                                               Ooi SL, Shoemaker DD, Boeke JD. DNA microarray-based genetic screen nonhomologous end- <br />                                                   joining mutants Saccharomyces cerevisiae. Science. 2001; 294:2552&#226;&#8364;&#8220;2556. [PubMed: <br />                                                   11701889] <br />                                               Ooi SL, Shoemaker DD, Boeke JD. DNA helicase gene interaction network defined using synthetic <br />                                                   lethality analyzed microarray. Nat Genet. 2003; 35:277&#226;&#8364;&#8220;286. [PubMed: 14566339] <br />                                               Ouspenski II, Elledge SJ, Brinkley BR. New yeast genes important chromosome integrity                                                   segregation identified dosage effects genome stability. Nucleic Acids Res. 1999; 27:3001&#226;&#8364;&#8220; <br />                                                   3008. [PubMed: 10454593] <br />  <br />  <br />  <br />  <br />                                                      Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                 Page 14 <br />  <br />  <br />                                               Pan X, Yuan DS, Xiang D, Wang X, Sookhai-Mahadeo S, Bader JS, Hieter P, Spencer F, Boeke JD.                                                   robust toolkit functional profiling yeast genome. Mol Cell. 2004; 16:487&#226;&#8364;&#8220;496. [PubMed: <br />                                                   15525520] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Peng WT, Robinson MD, Mnaimneh S, Krogan NJ, Cagney G, Morris Q, Davierwala AP, Grigull J, <br />                                                   Yang X, Zhang W, et al. panoramic view yeast noncoding RNA processing. Cell. 2003; <br />                                                   113:919&#226;&#8364;&#8220;933. [PubMed: 12837249] <br />                                               Rempola B, Karkusiewicz  Piekarska  Rytka J. Fcf1p Fcf2p novel nucleolar Saccharomyces <br />                                                   cerevisiae proteins involved pre-rRNA processing. Biochem Biophys Res Commun. 2006; <br />                                                   346:546&#226;&#8364;&#8220;554. [PubMed: 16762320] <br />                                               Sacher M, Barrowman J, Schieltz D, Yates JR 3rd, Ferro-Novick S. Identification characterization <br />                                                   new subunits TRAPP. Eur J Cell Biol. 2000; 79:71&#226;&#8364;&#8220;80. [PubMed: 10727015] <br />                                               Samanta MP, Liang S. Predicting protein functions redundancies large-scale protein <br />                                                   interaction networks. Proc Natl Acad Sci USA. 2003; 100:12579&#226;&#8364;&#8220;12583. [PubMed: 14566057] <br />                                               Schuldiner M, Collins SR, Thompson NJ, Denic V, Bhamidipati  Punna T, Ihmels J, Andrews B, <br />                                                   Boone C, Greenblatt JF, et al. Exploration function organization yeast early <br />                                                   secretory pathway epistatic miniarray profile. Cell. 2005; 123:507&#226;&#8364;&#8220;519. [PubMed: <br />                                                   16269340] <br />                                               Shoemaker DD, Lashkari DA, Morris D, Mittmann M, Davis RW. Quantitative phenotypic analysis                                                   yeast deletion mutants using highly parallel molecular bar-coding strategy. Nat Genet. 1996; <br />                                                   14:450&#226;&#8364;&#8220;456. [PubMed: 8944025] <br />                                               Sikorski RS, Boeke JD. vitro mutagenesis plasmid shuffling: cloned gene mutant yeast. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Methods Enzymol. 1991; 194:302&#226;&#8364;&#8220;318. [PubMed: 2005795] <br />                                               Simchen G. Cell cycle mutants. Annu Rev Genet. 1978; 12:161&#226;&#8364;&#8220;191. [PubMed: 106766] <br />                                               Skibbens RV, Corson LB, Koshland D, Hieter P. Ctf7p essential sister chromatid cohesion                                                   links mitotic chromosome structure DNA replication machinery. Genes Dev. 1999; 13:307&#226;&#8364;&#8220; <br />                                                   319. [PubMed: 9990855] <br />                                               Spencer F, Gerring SL, Connelly C, Hieter P. Mitotic chromosome transmission fidelity mutants                                                   Saccharomyces cerevisiae. Genetics. 1990; 124:237&#226;&#8364;&#8220;249. [PubMed: 2407610] <br />                                               Takayama Y, Kamimura Y, Okawa M, Muramatsu S, Sugino  Araki H. GINS, novel multiprotein <br />                                                   complex required chromosomal DNA replication budding yeast. Genes Dev. 2003; 17:1153&#226;&#8364;&#8220; <br />                                                   1165. [PubMed: 12730134] <br />                                               Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Page N, Robinson M, Raghibizadeh S, <br />                                                   Hogue CW, Bussey H, et al. Systematic genetic analysis ordered arrays yeast deletion <br />                                                   mutants. Science. 2001; 294:2364&#226;&#8364;&#8220;2368. [PubMed: 11743205] <br />                                               Tsai RT, Fu RH, Yeh FL, Tseng CK, Lin YC, Huang YH, Cheng SC. Spliceosome disassembly <br />                                                   catalyzed Prp43 associated components Ntr1 Ntr2. Genes Dev. 2005; 19:2991&#226;&#8364;&#8220; <br />                                                   3003. [PubMed: 16357217] <br />                                               Venema J, Tollervey D. Ribosome synthesis Saccharomyces cerevisiae. Annu Rev Genet. 1999; <br />                                                   33:261&#226;&#8364;&#8220;311. [PubMed: 10690410] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Voth WP, Jiang YW, Stillman DJ. New &#226;&#8364;&#732;marker swap&#226;&#8364;&#8482; plasmids converting selectable markers                                                   budding yeast gene disruptions plasmids. Yeast. 2003; 20:985&#226;&#8364;&#8220;993. [PubMed: 12898713] <br />                                               Winzeler EA, Lee B, McCusker JH, Davis RW. genome genetic-typing yeast using high- <br />                                                   density oligonucleotide arrays. Parasitology (Suppl). 1999; 118:S73&#226;&#8364;&#8220;S80. <br />                                               Yuen KW, Warren CD, Chen O, Kwok T, Hieter P, Spencer FA. Systematic genome instability <br />                                                   screens yeast potential relevance cancer. Proc Natl Acad Sci USA. 2007; 104:3925&#226;&#8364;&#8220; <br />                                                   3930. [PubMed: 17360454] <br />                                               Zhao J, Kessler M, Helmling S, O&#226;&#8364;&#8482;Connor JP, Moore C. Pta1, component yeast CF II, required <br />                                                   cleavage poly  addition mRNA precursor. Mol Cell Biol. 1999; 19:7733&#226;&#8364;&#8220;7740. <br />                                                   [PubMed: 10523662] <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                             Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 1. Diploid Shuffle Method Generating ts Alleles Essential Genes                                               Saccharomyces cereviviae <br />                                                 Genomic DNA containing YFEG 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; regions used template                                               PCR mutagenesis. black horizontal arrows represent gene-specific primers used. <br />                                               mutagenized PCR product cloned vector SB221+ Topo-TA (mutations                                               represented black stars). Topo-TA cloning site represented red T, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               overhang protruding PCR product represented red  Left beige bar <br />                                               represents 5&#226;&#8364;&#178; half KanMX selectable marker (Kan), right beige bar <br />                                               represents half KanMX selectable marker (MX). NotI restriction sites <br />                                               indicated diagonal black arrows. <br />                                               (B) product cloning step library mutagenized YFEG. library                                               transformed E. coli digested NotI release linear fragments (following DNA <br />                                               purification). <br />                                               (C) linearized library transformed corresponding heterozygous diploid strain. <br />                                               Blue red bars flank KanMX knockout represent bar codes. <br />                                               (D) Heterozygous diploid transformants sporulated (following meiosis), MATa Ura+ <br />                                               haploids spores selected haploid-selective medium 25&#194;&#176;C. <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                         Page 16 <br />  <br />  <br />                                               (E) Selection ts candidates following replica plating incubating 25&#194;&#176;C 37&#194;&#176;C. <br />                                               Black arrows identify potential ts allele. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                          Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 2. Ribosomal RNA-Processing Screen <br />                                                 Scheme rRNA processing. Ribosomal RNA encoded ~9 kb tandem <br />                                               repeat yeast processed 35S precursor multistep pathway outlined  <br />                                               cartoon indicates rDNA repeat structure positions mature 25S, 5.8S, 18S <br />                                               rRNAs (black bars DNA precursor forms, gray bars represent mature forms) derived <br />                                               precursor, separately transcribed 5S rRNA. 3&#226;&#8364;&#178; ends indicated                                               arrowheads. Intermediates pathways rRNA processing identified                                               shown panel. Dotted lines indicate effects processing;                                               alternate pathway formation mature 5.8S 25S rRNAs topmost 27S <br />                                               precursor indicated square bracket. Note multiple species mobilities <br />                                               27S 7S; experiments distinguish  internal <br />                                               transcribed spacer   regions used probes detect precursor forms indicated                                               inset. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               (B) Isolation ribosomal RNA-processing-defective mutants. RNA preparations                                               resolved 1% agarose <span id='am-12' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>formaldehyde</span> gels blotted nylon membranes probed                                               ITS2 probe. Duplicate isolates wild-type identified new mutants shown. <br />                                               Mutant YAL043 set 45 unknown mutants studied. <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                         Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 3. Screening Mutants Defective Faithful Chromosome Segregation <br />                                                 CTF assay: (Aa) supernumerary linear artificial chromosome fragment (CF) <br />                                               containing SUP11 gene serves sensitive indicator chromosome stability                                               ability suppress ade2-101 ochre mutation (resulting white cells). (Ab) Chromosome <br />                                               loss indicated streaking cells media non-selective artificial chromosome <br />                                               examination appearance white/red sectored colonies. sectoring phenotype <br />                                               dre2 starts appear 32&#194;&#176;C; higher temperature (34&#194;&#176;C) phenotype                                               severe. <br />                                               (B) ALF assay. (Ba) ALF based fact default mating type yeast                                               MATa. MAT&#206;&#177; cells ts mutants lose MAT&#206;&#177; locus  loss                                               chromosome III), mate MAT&#206;&#177; tester MATa called  like fakers.&#226;&#8364;? <br />                                               (Bb) patches MAT&#206;&#177; ts isolate control strains (WT Alpha bim1)                                               replica plated lawn MAT&#206;&#177; tester strain. Growing colonies indication NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               ability mate tester strains. shown, ALF swi4 appears 32&#194;&#176;C compared <br />                                               controls. <br />                                               (C) BiM assay. (Ca) Normally, diploid cells heterozygous mating type locus <br />                                               (MATa/MAT&#206;&#177;) mate MATa MAT&#206;&#177; haploids.  loss                                               MATa MAT&#206;&#177; mating type locus generates higher frequency mating-competent <br />                                               cells population cells, exhibits bimating phenotype. (Cb)                                               independent isolates homozygous diploid ts isolate plus control strains (WT <br />                                               homozygous diploid homozygous diploid chl1 deletion) patched replica <br />                                               plated MATa   MAT&#206;&#177;   testers. ability mate diploid                                               assessed number colonies tester strains. BiM phenotype ynl152w <br />                                               appears 34&#194;&#176;C, compared control strains. mechanisms lead CF <br />                                               loss, ALF, BiM shown (Ac), (Bc), (Cc), respectively. <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                           Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 4. Summary ts Mutant Phenotypes, Showing CIN RNA-Processing Defects <br />                                               Central nodes colored red, green, blue, light purple represent CIN (BiM, ALF, <br />                                               CTF) RNA-processing screens, respectively. Arrows project central nodes <br />                                               point mutants positive specific screen. Small circles colored blue <br />                                               represent ts mutants unknown function defective CIN RNA processing <br />                                               semipermissive temperature. ORFs used controls CIN assay                                               colored red. SSU components isolated bicohemically work progress                                               colored purple. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                         Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Figure 5. Identification Genes Critical Proper Sister Chromatid Cohesion <br />                                                 secondary screen ts mutants defective sister chromatid cohesion. Cells                                               arrested G1 G2/M. number GFP signals scored wild-type, ctf18 <br />                                               mutant, CIN mutants. data shown represent percentage cells                                               GFP signals. cells counted G1-arrested cells 300&#226;&#8364;&#8220;400 <br />                                               cells G2/M-arrested cells. <br />                                               (B C) Sister chromatid cohesion mutants required establishment sister <br />                                               chromatids: samples released &#206;&#177; factor arrest nocodazole-containing <br />                                               media allow arrest G2/M (labeled stop sign). Samples 1 (black arrow), 2 (red <br />                                               arrow) released arrested 25&#194;&#176;C semipermissive temperature, respectively. <br />                                               Sample 3 (blue arrow) completed S phase 25&#194;&#176;C;  completion S <br />                                               phase, shifted semipermissive temperature, arrest G2/M. hours                                               arrest, percentage cells GFP signals counted (100 cells). results <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               presented (C). cases, numbers obtained samples 1 3 similar, <br />                                               indicative potential role sister chromatid establishment. <br />  <br />  <br />  <br />  <br />                                                     Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                             Ben-Aroya et al.                                                                                                                Page 21 <br />  <br />  <br />  <br />  <br />                                                                                              Table 1 <br />  <br />                            Genes Identified ts Mutants Affect rRNA Processing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                ORF    Gene s)      Description                                                    rRNA Defect       Percent Reduction <br />                                YAL043C     PTA1, FUN39       Subunit holo-CPF, mRNA, snoRNA pre-tRNA processing; binds   18Sa                    22.4 <br />                                                              phospho-CTD RNAPII <br />  <br />                                YDR196C                       Coenzyme biosynthesis; mitochondrial                         18S                     11.3 <br />  <br />                                YDR339C     FCF1, UTP24       SSU processome component                                       18S                      11 <br />  <br />                                YGR145W     ENP2              Nucleolar                                                      18S                     23.5 <br />  <br />                                YGR277C                       Coenzyme biosynthesis                                        significant <br />  <br />                                YJL069C     UTP18             Possible U3 snoRNP protein                                     25S                      5.9 <br />  <br />                                YKL033W     FMP47             Mitochondrial                                                  18S                     16.9 <br />  <br />                                YKR022C     NTR2              Spliceosome disassembly                                        18S                      9 <br />  <br />                                YLR132C                       Mitochondria nucleus                                       significant <br />  <br />                                YOL022C                       Cytoplasmic                                                    18S                      9.1 <br />  <br />                                YOR004W     UTP23             SSU processome component                                       18S                      8.1 <br />  <br />                                                        Underrepresented mature rRNA indicated; Figure S2 Table S3 data. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                           Mol Cell. Author manuscript; available PMC 2014 August 12. <br />                                                                  NIH-PA Author Manuscript                                                     NIH-PA Author Manuscript                                            NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                     Table 2 <br />  <br />                                                                 Genes Identified ts Mutants Affect CIN, Quantification CIN Phenotype <br />  <br />                                                                     ORF        Gene       Description                                                     ALF     BiM     CTF <br />                                                                     YKR038C         KAE1            component KEOPS protein complex                         0b     0c      3d <br />                                                                                                                                                                                                                                                                                Ben-Aroya et al. <br />  <br />  <br />  <br />  <br />                                                                     YHR020W                         Protein unknown function                                      32     4       3 <br />                                                                     YOR262W                         Cytoplasmic protein unknown function                          0      0       3 <br />                                                                     YDR246W         TRS23           subunits transport protein particle (TRAPP)    6      2       2 <br />                                                                     YHR122W                         Putative protein unknown function                             23     3       3 <br />                                                                     YGR002C         SWC4            Component Swr1p complex                                   19     4       2 <br />                                                                     YFR042W         KEG1            Protein unknown function                                      34     6       2 <br />                                                                     YNL152W                         Protein unknown function                                      59     6       0 <br />  <br />                                                                     YKR071Ca        DRE2            Protein unknown function                                      0      0       3 <br />  <br />                                                                     YJL072Ca        PSF2            Subunit GINS complex                                      61     0       2 <br />  <br />                                                                     YDR013Wa        PSF1            Subunit GINS complex                                      52     6       3 <br />  <br />                                                                     YDR489Wa        SLD5            Subunit GINS complex                                      0      5       3 <br />  <br />                                                                     YDR288Wa        NSE3            Subunit Mms21-Smc5-Smc6 complex                           52     3       1 <br />  <br />                                                                     YML023Ca        NSE5            Subunit Mms21-Smc5-Smc6 complex                           0      6       0 <br />  <br />                                                                     YDL105Wa        NSE4            Subunit Mms21-Smc5-Smc6 complex                           18     4       1 <br />  <br />                                                                                                                  <br /> </body></html>