<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 75,  2, pp. 774-778, February 1978 <br /> Biochemistry <br />  <br />  <br /> Construction characterization Escherichia coli plasmid <br /> bearing functional gene G bacteriophage 4X174* <br />      vitro recombination/transformation/gene-specific suppressor/spike protein) <br /> M. ZAFRI HUMAYUN ROBERT W. CHAMBERS <br /> New York University School Medicine, Department Biochemistry, 550 Avenue, New York, New York 10016 <br /> Communicated Severo Ochoa, December 1, 1977 <br />  <br /> ABSTRACT           order study mutagenic effects                   gene G product, virus spike protein, constitutively site-specific, covalent modifications biologically active DNA,               host cell. communication, construction need host cells permissive type mutation                 characterization ColEl type plasmid containing produced vivo modified DNA. Spe-                      segment kX174 replicative form (RF) DNA carrying gene <br /> cifically, require general, vivo complementation                 G. plasmid, p4XG, renders previously bacteriophage OX174 gene G, essential gene chosen initial studies <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> mutagenesis.                    nonpermissive Eschertchia coli strains (H514, HF4704, end, constructed plasmid (pOXG)                      HF4740) permissive OX174 amn9, phage mutant defective <br /> carries functional copy gbX174 gene G. different                     gene G (5). <br /> bacterial strains nonpermissive am9, gene G <br /> amber mutant, transformed pOXG. trans- <br /> formants permissive gene G mutant,                               METHODS MATERIALS gene E mutants tested. paper  <br /> scribes construction biochemical characterization                      jpX174  Strains.  Wild type R. C. Warner; am9  plasmid, p$XG, describes biological                     gene G mutant), am3  gene E mutant), am86  gene properties exhibited pOXG-bearing strains.                               mutant) R. L. Sinsheimer (5). <br />                                                                                    E. coli Strains. E. coli C (wild type) R. C. Warner. large class mutagens, including ultimate carcinogens,                 C/K12 hybrids, H514 Su-, 4pXs, end, uvrA, thyA, arg, rK, reactive electrophiles produce different kinds                    mK (6), HF4704 F-, OXs, Su-, thy, uvrA (7), HF4714 F-, covalent modifications react DNA (1).                      Su+, OX%, arg,  leu, thr, pro (5, 7) obtained J. <br /> present, direct method determining                     Hurwitz. HF4740 F-, Su-, OXs, recA13,uvrA103, tlr ob- modifications produce mutations, direct                  tained P. Howard Flanders. HB129 Srm, rg, mB, gal, laC, <br /> way showing type mutation (base substitution,                        leu, pro, B1, end carrying pMB9 (8, 9) M. Ptashne&apos;s <br /> frameshift, large deletion) produced different kinds                laboratory. <br /> premutational lesions DNA. laboratory begun                          Growth Bacteria Virus. Luria broth L plates <br /> intensive effort develop methodology enable                  used rich media (10). L-tet medium plates explore problem directly.                                                  Luria broth  L plates) containing 15 mg tetracycline    availability extensive information regarding                      liter. Virus prepared single plaques grown HF4714 <br /> structure biology bacteriophage OX174 (2-4) makes                   Su+ using defined medium, modified 3XD (11). Titers virus especially attractive model initial studies.                   measured soft agar techniques R plates (10). <br /> Briefly, approach follows:  characterized                   Enzymes. Restriction endonuclease EcoRI Miles. <br /> modification introduced DNA single, preselected                    Hae III Hpa II gifts R. Roberts; Hha site combination chemical enzymatic synthesis.                       Jeffrey. Lambda exonuclease gift C. Radding&apos;s <br /> Second, biological response site-altered DNA ex-                 laboratory. Deoxynucleotidyltransferase PL Bio- <br /> amined isolating mutant phage produced transfection                       chemicals. RNase Worthington Biochemicals. permissive cells.  precise change produced                     DNA Preparations. OX174 RF DNA prepared site-specific modification determined determining                     modification Godson Boyer&apos;s method (12). Plasmid DNA <br /> sequence region DNA isolated                     amplified presence chloramphenicol (200 ,ug/ml) <br /> mutant phage. principle, covalent modifications DNA,                  isolated minor modifications &quot;cleared lysate&quot; <br /> including produced carcinogens, examined                     method Clewell (13). electrophoretic identification procedure, molecular expression mutagenic                   recombinant clones, plasmid DNA partially purified activity displayed premutational lesion                        follows: Lysates prepared 20 ml chlorampheni- <br /> fined.                                                                          col-treated cultures essentially according Clewell    goal, need develop bacterial strains ca-                  20 ,ug RNase added detergent solution. <br /> pable suppressing base substitutions,                      chromosomal material pelleted, clear lysates <br /> frameshifts large deletions gene G, essential gene                 extracted twice phenol times OX174 chosen initial studies. principle,                ether. DNA recovered ethanol precipitation. complementation provides general solution problem,                    detergent coprecipitated DNA, caused prob- technical difficulties using helper DNA              lems electrophoretic assay. transfection process.  investigated possibility using plasmid carrying OX174 gene G produce                 Abbreviations: RF, replicative form; TE buffer, 10 mM Tris-HCI, pH <br />                                                                                 8.0/1 mM EDTA; bp, base pairs; restriction fragments named ac- costs publication article defrayed           cording number base pairs contain. <br /> payment page charges. article marked           * paper  1 series: &quot;Molecular mechanisms mutation <br /> &quot;advertisement&quot; accordance 18 U. S. C. &#194;&#167;1734 solely indicate           carcinogens.&quot; presented &quot;Single Stranded DNA fact.                                                                        Phage&quot; meeting Cold Spring Harbor, NY, August 29, 1977. <br />                                                                           774 <br />                Biochemistry: Humay Chambers                                                      Proc. Natl. Acad. Sci. USA 75(1978)           775 <br />  <br />                                Hae III <br />  <br />                                           1        1 Hae III           N.T <br />                                  G <br />                                                   2. Isolate          <span id='am-12' about='obo:IMR_0200084' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00459" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="dTTP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200084" datatype="xsd:string"></span>dTTP</span> <br />                                                                       <span id='am-17' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span>Co2</span> + <br />                                Hae.11 <br />  <br />  <br />  <br />  <br />                                                                               N.T <br />                                                                           <span id='am-22' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span>dATP</span> <br />                                                                           Co2+ <br />     FIG. 1. Experimental protocol biochemical construction hybrid molecules pMB9 DNA segment OX174 RF DNA. X <br />  Exo X exonuclease; N.T. terminal deoxynucleotidyltransferase (20). Heavy dots symbolize 5&apos; ends.   &quot;B&quot; possible orientations <br />  inserted viral DNA fragment relative HindIII restriction site shown open triangle (9). 5&apos; 3&apos; direction viral plus <br />  strand (+) indicated arrowhead. <br />      Gel Electrophoresis Electron Microscopy. Gel elec-                          pmol) annealed incubation 200 Ml buffer (100 mM <br />   trophoretic techniques used described (14). Electron                     CaCl2 TE buffer) 500 30 min followed slow cooling <br />   microscopy formamide (50%) aqueous                              220 hours. control experiments, tailed <br />   techniques described Davis et al. (15). restriction                         pMB9 DNA tailed viral DNA fragment   fragment Hae 1342f OX RF DNA used internal                          identical annealing procedures. Electron <br />   size marker.                                                                       microscopy revealed types DNA present <br />      Biochemical Construction Plasmid-Viral DNA Hybrid                            annealing, 10% linear DNA converted <br />   Molecules. general procedure Jackson et al. (16)                        circular form. circles detectable control ex- <br />  used DNA ligase step omitted (17).                          periments. <br />  pMB9 DNA digested EcoRI buffer (80 mM Tris-                               Transformation. Calcium-shocked cells prepared  HCl, pH 8.0/50 mM NaCI/10 mM MgCI2/1 mM dithiothreitol)                            exponentially growing cultures according Cohen et al. (21), <br />  conditions 95% circles                        CaCI2 concentration increased 50 mM.  <br />  converted linear forms determined electron mi-                          nealed DNA (0.15 pmol) 200 Ml buffer (50 mM CaCl2  croscopy. linear DNA (20 ,g) taken 200 ,l                         10 mM MgCl2 TE buffer) 00 mixed 400,Ml  buffer (67 mM <span id='am-27' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span>glycine</span>-KOH, pH 9.4/4 mM MgCl2)                              calcium-shocked cell suspension following <br />  digested 65 units X exonuclease (18) 1 hr 0&#194;&#176;. DNA                   sequential steps (22): incubation 00 15 min, incubation <br />  recovered following steps sequence: phenol                       420 2 min, incubation 220 10 min, addition 0.3 <br />  extraction, ether extractions, ethanol precipitation.                    ml 3 X concentrated L broth, incubation 370 30 min, <br />  Exonuclease-treated linear pMB9 DNA (10.ug) dissolved                          plating 0.1 ml L-tet plates. Tetracycline-resistant <br />  100 MAl transferase buffer (140 mM Tris-cacodylate adjusted                  colonies visible 12 hr incubation 370. <br />  pH 6.9 KOH/1 mM CoCl2/1 mM 2-mercaptoethanol)                                 recombinant DNA experiments P 1 <br />  (19) containing 10 nmol [3H]dATP (1.1 Ci/mmol)                          containment outlined &quot;National Institutes Health <br />  incubated 66 units deoxynucleotidyltransferase (20)                    Recombinant DNA Research Guidelines, II.&quot; <br />  160 min 370. reaction terminated phenol ether extractions, DNA solution dialyzed <br /> overnight changes (2 liters  TE buffer (10                                                   RESULTS <br /> mM Tris*HCl, pH 8.0/1 mM EDTA). Deoxyadenylate incor- <br /> poration estimated trichloroacetic acid-precipitable                          Cloning Segment joX174 DNA Bearing Gene G. Hae <br /> radioactivity 100 residues 3&apos; end pMB9.                       1342t largest fragment produced Hae III digestion viral fragment, Hae 1342, prepared digesting qX174                            OX174 RF DNA contains entire structural gene (gene <br /> RF DNA Hae III fractionating products 35%                          G) spike protein parts genes F H (23, 24). <br /> preparative polyacrylamide slab gel procedures similar                        experimental procedures schematized Fig. 1, described (14). picomoles Hae 1342                          cloned viral fragment E. coli plasmid, pMB9, incubated  X exonuclease) 55 units transferase                      convenient vehicle carrying ColEl linear DNA EcoRI. 100 Mul transferase buffer containing 10 nmol [3H]dTTP                     poly(dA) tails added. Poly(dT) tails added (1.8 Ci/mmol) 370 60 min. Poly(dT)-tailed DNA                            viral fragment. Equimolar amounts molecular <br /> isolated  Incorporation thymidylate estimated                       species annealed, resulting mixture used 80 residues 3&apos; end Hae 1342.                                     transform E. coli host, H514. Transformants selected <br />    microgram poly(dA)-tailed, linear pMB9 DNA                                plating presence tetracycline. DNA 11 (&apos;-0.28 pmol) 0.25 ,g poly(dT)-tailed Hae 1342 (&apos;-0.3                        346 tetracycline-resistant colonies obtained isolated                                                                                     examined agarose gel electrophoresis. 11 transfor- <br /> t fragment Z1   according widely used, descriptive,         mants examined, plasmid DNA mobility <br />  nomenclature (2).                                                                  identical parental plasmid, pMB9; slowly <br />  776         Biochemistry: Humayun Chambers                                                                        Proc. Natl. Acad. Sci. USA 75 (1978) <br />                                                                                                                                   y <br />                             708.*        (dA)n       (dT)nflJ                                                                                        560   +    (dA)n (dT)n <br />                                                                                                                                                                                        H                                            H                 H              H                                H                                           H               1   270         1         350           msl                201   1        289                614                 1150                          410           19ol1 <br />                                                                                                                                 T- <br />                                                                                                                   Y               Y <br />                                                                                   Hae III    2192 * (dA)n * (dT)n <br />                   C                       H                                                    H                                H                H           H                                       H    <br /> B            1 <br />                270         1           350            1501 <br />                                                      --.;                           614            1I     289       1 201           II 8 <br />                                                                                                                                         --                  410           1901 <br />                                                                                                                                                                                                                                                                                                                     0 0 0 0 <br />                                                      -                                                                                                                                                                      y                     y <br />                        770       *   (dA)n * (dT)n                                                                                            4968 t (dA)         (dT)n <br />         b    F <br />                                                             Y <br />  <br />    FIG. 2. partial restriction cleavage map pOXG105.   p4XG105   orientation  Fig. 1). (B) pOXG105 &quot;B&quot; orientation. <br />  b) Predicted Hha bridge fragments orientations   &quot;B,&quot; respectively. (c) Predicted Hae III fragment pkXG. H, Y, refer cleavage sites restriction endonucleases Hha  Hpa II, Hae III, respectively; v marks HindIII site used orient <br /> pMB9 relative EcoRI site (9). arrowhead gives orientation (5&apos;         3&apos;) OX174 (+) strand; dots represent (dA)n.(dT)n <br />                                                                                                     - <br />  <br />  <br />  <br /> bridges. <br />  <br /> moving DNA, suggesting larger molecule. slowly                                       data Fig. 3 b OX Hae 1342 bearing <br /> moving plasmids similar mobilities, indicating similar                                       gene   G inserted EcoRI site pMB9. size.                                                                                            different orientations  Fig. 1) possible, giving <br />    order gene G functional                                      slightly different bridge fragments (left: 708 + tail 770 + tail; <br /> transformants, screened plaque formation                                        right: 560 + tail 498 + tail; Fig. 2). values OX174 am9, gene G mutant incapable growth                                            similar different orientations length <br /> parent bacterial strain (H514) transformants                                    tails known precision, difficult deter- derived. results showed tetracycline-                                      orientation estimated size bridge <br /> resistant transformants plasmid DNA                                           fragments Hha digest.  map predicts mobility pMB9 marker did form plaques.                                        left Hha bridge fragment change double di- <br /> remaining transformants (harboring larger, apparently                                       gest pOXG Hha Hpa II OX Hae 1342 hybrid, plasmids) gave plaques,                                       orientation shown Fig. 1, right Hha bridge plating efficiency. transformants                                     fragment shortened 40 bp. reverse true <br /> (named H514/pOXG105 H514/pOXG109) selected                                              opposite orientation (&quot;B&quot;, Fig. 1). cleavage pattern analysis.                                                                            Fig. Sc shows left Hha bridge fragment remains <br />    Characterization poXG105 p4XG109. addition                                          constant 830 bp. right bridge (600 bp) disappears electrophoretic mobilities mentioned  pOXG105                                       new fragment 540 bp appears. Hha 613 disappears, poXG109 characterized electron microscopy                                         orientation) contains Hpa II site (2). <br /> restriction mapping. Electron microscopy gave average contour                                    data suggest strongly OX Hae 1342 DNA lengths    poly(dA)-tailed, linear, pMB9 DNA, 1.68 ,um;                                     orientation shown   Fig. 1. Similar results  shown) <br /> (b) poly(dT)-tailed OX Hae 1342, 0.35 ,um; (c) circular mole-                                    obtained pOXG109. <br /> cules produced annealing   (b), 1.98 ,um; (d)                                          Biological Properties Bacterial Hosts Carrying <br /> pOXG105 p4XG109, 2.07 Aim.                                                                   p4$XG105 p4XG109. Table 1 summarizes results ex- <br />    Fig. 2 shows partial restriction map p4XG constructed                                 periments tested ability nonpermissive available information (2, 9). Fig. 3a shows Hae III                                     host, H514, form plaques kX174 amber mutants  <br /> cleavage pattern purified pOXGl05 DNA OX                                      fective genes  E, G transformation <br /> RF DNA pMB9 DNA. map predicts insertion                                          pOXG105 p4XG109. limits data, <br /> OX Hae 1342 single EcoRI site pMB9 cause                                        suppression specific gene G mutants. <br /> pMB9 Hae 850 (9) replaced new fragment, pOXG                                             gene G nonsense mutant, am9, grown Hae 2192 + (dA)n-(dT)n. Lanes 2 3 Fig. 3a shows                                      plasmid-bearing strain, H514/pbXG105, soft agar, small new fragment estimated 2400 base pairs (bp)                                   turbid plaques barely visible 3 hr 370 compared mobility.                                                                               large (-2 mm), clear plaques produced wild-type phage. <br />    map predicts Hha digest pOXG                                    individual plaques picked examined lawns OX fragments (201, 289, 614 bp)                                    permissive nonpermissive test strains (HF4714 Su+ new bridge fragments join pMB9 OX Hae 1342 DNA.                                         C Su-), carry plasmid. typical ex- exact size bridge fragments depend                                      periment, 1/3 virus single plaque H514/ <br /> orientation OX Hae 1342 relative pMB9 length                                       p4XG105 wild type. Plaques picked longer incubation (dA)n- (dT)n tails. map predicts pMB9 Hha                                   greater wild-type content. peculiar 1030, containing EcoRI site (9), disappear. Fig. 3b                                 H514/pOXG105 lawns am9 stock used ex- <br /> verifies predictions. mobility, larger bridge                                 periments gives plaques containing &lt;10-5 wild-type virus fragment estimated 830 bp pOXG105                                           permissive host, HF4714. Results similar pOXG109; fragment appears pMB9 OX                                       H514/pOXG105 obtained H514/pOXG109. <br /> RF DNA digests  smaller bridge fragment                                          investigate production wild-type phage plas- estimated 600 bp pOXG105 625                                          mid-bearing cells infected am9, recombination pOXG109.                                                                                         measured infective center assay. H514/pOXG105 cells <br />               Biochemistry: Humayun Chambers                                                 Proc. Natl. Acad. Sci. USA 75 (1978)            777 <br />                             1   2 3                       1 2 34                            1 2 3 4 <br />  <br />  <br />                                          -   2400 <br />                                                                                                              -   1030(1,4) <br />                                          - 850(2)                                                            -   830(2,3) <br />                                                                                                                  614(2) <br />                                                                                                              ={600(2) <br />                                                                                                              =~580(1,2) <br />                                                                                                                  560(3) <br />  <br />  <br />  <br />  <br />                                                            b                                   c <br />    FIG. 3. Fractionation restriction endonuclease cleavage products pkXG105 pOXG109. DNA (2-10 Mg) digested appropriate enzyme(s) 100 ml buffer (60 mM Tris.HCl, pH 7.5/6 mM MgCl2/6 mM 2-mercaptoethanol) 3 hr 37&#194;&#176;. Reaction terminated phenol ether extractions. microliters dye mix (0.5% bromophenol blue 0.5% xylene cyanol 50% glycerol) added. Electrophoresis vertical 5% polyacrylamide slab gels (20 X 40 X 0.15 cm) 90 mM Tris-borate/2.5 mM EDTA (pH 8.3) 400 <br /> V bromophenol blue marker reached just ran (10-12 hr). Gels stained 30 min 0.5-1 ,ug ethidium bromide ml photographed UV light. Sizes certain relevant restriction fragments indicated. easier identification certain bands, slots occur given parentheses.   Hae III cleavage patterns OX174 RF DNA (slot 1), pMB9 DNA (slot 2), pOXG105 <br /> DNA (slot 3). (b) Hha cleavage patterns OX174 RF DNA (slot 1), pMB9 DNA (slot 2), pOXG105 (slot 3), pOXG109 (slot 4). (c) Hha Hpa II double-digestion patterns pMB9 pOXG105 DNA: pMB9 cleaved Hha (slot 1), pOXG105 cleaved Hha (slot <br /> 2), pOXG105 cleaved Hha Hpa II (slot 3), pMB9 cleaved Hha Hpa II (slot 4). <br />  infected am9 diluted plated lawns                  cycles infection lysis occur plaque  permissive nonpermissive test strains. Wild-type phage             visible. period, probability recombination <br />  0.03% infected cells. Clearly recombination              plasmid DNA mutant viral DNA  itself explain plaques examined lawns             wild-type viral DNA high. occurs, wild-type phage HF514/pOXG105 mixtures mutant wild-type                      produced rapidly burst size 30 times am9 <br /> phage.                                                                      plasmid-bearing cell. test hypothesis, lowered <br />     important clue plaque purity puzzle came                 recombination frequency p0XG am9 DNA single cell, single burst experiments (25) carried according            transforming recombination-defective, nonpermissive host Benbow et al. (7). 200-tube experiment, average burst            (HF4740 recA13) pOXG105. tetracycline-resistant <br /> size 10 obtained H514/p4XG105 infected                      transformant, permissive am9, obtained dif- am9 compared 300 plasmid-                       ficulty. types plaques observed infection bearing host infected wild-type phage.                   recA-, poXG-bearing host am9. plaques <br /> recombination small burst size involved             small regular edge. plaques production mixed plaques am9 grown H514/                      0.1% wild-type phage. 2% plaques large poXG105.                                                                   irregular edge. plaques contained large   suggested small burst size,                wild-type phage.  reduction recombination fre- <br />                                                                            quency recA- mutation makes possible isolate <br />     Table 1. Plaque formation OX174 mutants E. coli                  plaques essentially pure am9. 2% plaques <br />                   containing pOXG plasmids                                 mixed clear  <br />                                     Plaque formation                          biological effects peculiar H514/p4XG                Location              H514/       H514/                  similar effects observed bacterial <br />    4X174        mutation       H514 poXG105 pOXG109                        strain, HF4704, transformed pOXGl05 poXGl09. <br />   Wild type                     +         +           + <br />   am9            GeneG          -         +           +                                            DISCUSSION <br />   am3            Gene E                                                    Gene G, essential gene bacteriophage kX174, required <br />   am86           GeneA          -         -           -                    synthesis single-stranded viral DNA RF. gene <br />  778       Biochemistry: Humayun Chambers                                               Proc. Natl. Acad. Sci. USA 75 (1978) <br />  <br /> product spike protein virion (3, 4). wishes         initial isolation pMB9- DNA gift EcoRI enzyme; R. <br /> isolate propagate conditional unconditional lethal           Roberts  Jeffrey restriction enzymes; W. Holloman X <br /> mutants arise covalent modification              exonuclease; P. Howard-Flanders, J. Hurwitz, R. Sinsheimer, R. <br /> DNA virulant phage fX174,             Warner bacterial phage strains; M. Schneider techrical permissive base substitutions,          assistance; H. Annus photography. work supported <br /> frameshifts large deletions. Gene-specific suppression             Grant 2 ROI CA 16319-03 BIO VO 2 (M77) National  <br />                                                                           stitutes Health. <br /> complementation using plasmid vehicle  <br /> scribed appear provide general                 1. Miller, J.  <span id='am-5' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; Miller, E. (1974) Chemical Carcinogenesis, eds. <br />    anticipated number potential difficulties                   Ts&apos;o, P. &amp; Dipaulo, J.  (Marcel Dekker, , New York), structing functional strain carrying pOXG. example,                  pp. 61-63. <br /> restriction fragment OX Hae III 1342 does contain known              2. Sanger, F., Air, G. M., Barrell, B. G., Brown, N. L., Coulson,  <br /> promoter (2, 26, 27).  production viral spike protein              R., Fiddes, J. C., Hutchison, C. , III, Slocombe, P. M. &amp; Smith, uninfected cell lethal. materialized,               M. (1977) Nature 265,687-695. problem did arise. burst size am9, gene C                 3. Sinsheimer, R. L. (1968) Progress Nucleic Acid Research <br /> nonsense mutant, pOXG-bearing host, H514,                      Molecular Biology, eds. Davidson, J. N. &amp; Cohn, W. E. <br /> 5-10% wild-type phage H514/pOXG. small burst                   (Academic Press, New York), Vol. 8, pp. 115-169. <br /> size apparently known polar effect am9                4. Denhardt, D. (1975) Crit. Rev. Microbiol. 4,161-223. <br /> mutation gene H (28). recent experiments                    5. Benbow, R. M., Hutchison, C. , III, Fabricant, J. D. &amp; Sin- <br /> published) shown minicells carrying p4XG produce                    sheimer, R. L. (1971) J. Virol. 7,549-558. <br />                                                                            6. Vosberg, H. P. &amp; Hoffmann-Berling, H. (1971) J. Mol. Biol. 58, large gene G spike protein. gene H product,                   739-753. spike protein, furnished infecting mu-              7. Benbow, R. M., Zuccarelli,  J., Davis, G. C. &amp; Sinsheimer, R. <br /> tant phage. severe polarity am9 mutation,               L. (1974) J. Virol. 13, 898-907. small H protein formed (28),              8. Rodriguez, R. L., Bolivar, F., Goodman, H. M., Boyer, H. W. &amp; limits burst size. support comes               Betlach, M. (1976) Molecular Mechanisms Control experiments (unpublished) transformed                     Gene Expression, eds. Nierlich, D. P., Rutter, W. J. &amp; Fox, C. F. <br /> amber suppressor strain nonpermissive am9                    (Academic Press, New York), Vol. V, pp. 471-477. <br /> pOXG. transformant permissive am9,                9. Maniatis, T., Kee, S. G., Efstratiadis,  &amp; Kafatos, F. C. (1976) burst size increases 7-fold compared transfor-                 Cell 8, 163-182. <br />                                                                           10. Miller, J. H. (1972) Experiments Molecular Genetics (Cold <br /> mants described paper carry suppressor                  Spring Harbor Laboratory, Cold Spring Harbor, NY), pp. <br /> tRNA.                                                                         433-434. <br />    small burst size combined recombination leads              11. Guthrie, G. D. &amp; Sinsheimer, R. L. (1960) J. Mol. Biol. 2,297- <br /> plaques containing mixture wild-type am9 phage. Pure                 305. <br /> am9 isolated plaques grown recombination-                12. Godson, G. N. &amp; Boyer, H. (1974) Virology 62,270-275. <br /> deficient, poXG-bearing host HF4740/pOXG                     13. Clewell, D. B. (1972) J. Bacteriol. 110,667-676. burst size small. generally useful        14. Humayun, , Jeffrey,  &amp; Ptashne, M. (1977) J. Mol. Biol. 112, proposed recA function                   265-277. <br /> required interesting, carcinogen-induced mutations               15. Davis, R. W., Simon, M. &amp; Davidson, N. (1971) Methods (29, 30). does appear propose              Enzymology, eds. Grossman, L. &amp; Moldave, K. (Academic Press, <br />                                                                               New York), Vol. XXI, pp. 413-418. <br /> produce mutants transfection spheroplasts site-              16. Jackson, D. , Symons, R. H. &amp; Berg, P. (1972) Proc. Natl. Acad. <br /> specific, covalently modified DNA. Transfection spheroplasts               Sci. USA 69,2904-2909. <br /> derived cells resistant infection intact          17. Wensink, P. C., Finnegan, D. J., Donelson, J. E. &amp; Hogness, D. <br /> virus produces single burst virus DNA.                S. (1974) Cell 3,315-325. 0.03% wild-type virus single cells H514/pOXG                 18. Little, J. W., Lehman,  R. &amp; Kaiser,  D. (1967) J. Biol. Chem. <br /> infected am9 plated, expect                242,672-678. single burst transfected spheroplasts.                               19. Roychoudhury, R., Jay, E. &amp; Wu, R. (1976) Nucleic Acids Res. <br />    terms developing general detecting                  3, 101-116. <br /> propagating type mu.tation gene G, polar effects                20. Bollum, F. J. (1969) J. Biol. Chem. 234,2733-2734 <br /> </body></html>