<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol. 12  1 pp. 177 - 186, 1993 <br />  <br />  <br /> CDC39, essential nuclear protein negatively <br /> regulates transcription differentially affects constitutive inducible HIS3 promoters <br />  <br />  <br /> Martine Collart Kevin Struhl                                  HIS3 TATA region (Struhl, 1984). transcriptional <br />                                                                     initiation patterns arise TR respond Department Biological Chemistry Molecular Pharmacology,      transcriptional stimulation GCN4 GAL4, Harvard Medical School, Boston, MA 02115, USA <br /> Communicated R.Treisman <br />                                                                     Tc  <br />                                                                        distinct responses acidic activator proteins, <br />                                                                     TR Tc differ ways. TR element defined yeast HIS3 promoter region contains functionally            primarily canonical TATA sequence, TATAAA <br /> distinct TATA elements, Tc TR, responsible             (Chen Struhl, 1988; Harbury Struhl, 1989), respectively initiation + 1 + 13 sites.            functional activity strongly correlates ability bind Tc TR support basal HIS3 transcription                 TFIID (Strubin Struhl, 1992) support TFIID- <br /> require TATA binding protein TFIID, TR                 dependent transcription vitro (Ponticelli Struhl, 1990; <br /> responds transcriptional activation GCN4                  Wobbe Struhl, 1990). contrast, Tc does contain <br /> GAL4. selecting yeast strains increase                  sequence fits TATAAA consensus, surprisingly <br /> transcription GCN4 derivative defective                 tolerant mutations (Mahadevan Struhl, 1990), does <br /> activation domain, isolated temperature-                  support transcription yeast nuclear extracts (Ponticelli <br /> sensitive mutation CDC39, previously defined gene              Struhl, 1990), does yield DNase footprints <br /> implicated cell-cycle control pheromone                  TFIID  S.Ponticelli K.Struhl, unpublished <br /> response. cdc39-2 mutation causes increased basal              results).  Tc-dependent transcription requires <br /> transcription  genes,                TFIID temperature proteolytic inactivation increased transcriptional activation GCN4 GAL4.              TFIID vivo results equally rapid decline + 1 <br /> Surprisingly, basal HIS3 transcription + 1                 + 13 transcripts (Cormack Struhl, 1992). Finally, <br /> initiation site strongly increased, initiation        micrococcal nuclease sensitivity his3 TATA region + 13 site barely affected.  unlike acidic               associated Tc- TR-dependent transcription <br /> activator proteins function TR, CDC39                  (Oettinger Struhl, 1985; Struhl, 1986). <br /> preferentially affects transcription mediated Tc.                   begun genetic approach investigate CDC39 essential gene encodes large                mechanism GCN4 activation distinction nuclear protein (2108 amino acids) containing                   TR- Tc-mediated transcription. Starting strain <br /> glutamine-rich regions. observations suggest             containing GCN4 derivative partially defective acidic <br /> CDC39 negatively regulates transcription                  activation domain, isolated unlinked suppressor <br /> affecting general RNA polymerase II machinery                mutations increase his3 transcription. study, altering chromatin structure.                                    mutation CDC39 causes novel <br /> Key words: eukaryotic promoters/GCN4 activation/Sac-                phenotype increasing his3 transcription + 1 site <br /> charomyces cerevisiaelTATA element/transcriptional                  hardly affecting initiation + 13 site. CDC39 <br /> repression                                                          originally identified temperature-sensitive mutation <br />                                                                     arrests cells GI inappropriately activates                                                                     pheromone response pathway (Reed, 1980; Barros Lopes <br />                                                                     et al., 1990).  molecular genetic analyses <br /> Introduction                                                        presented suggest CDC39 directly involved <br /> Constitutive GCN4-inducible expression yeast              cell-cycle control, general negative regulator <br /> HIS3 gene involves TATA elements, Tc TR,               transcription differentially affects constitutive promote transcription different molecular mechanisms              inducible his3 promoters. <br /> (Struhl, 1986; Chen Struhl, 1988; Harbury Struhl, <br /> 1989; Mahadevan Struhl, 1990; Ponticelli Struhl,            Results <br /> 1990). Tc responsible transcription initiated +1 site, TR-dependent transcription initiated            Isolation temperature-sensitive suppressor exclusively +13 site. absence             mutation <br /> GCN4, HIS3 transcription initiated equal efficiency         obtain suppressor mutations increase activation + 1 + 13 Tc TR act                     GCN4 derivatives partially defective activation <br /> conjunction upstream poly(dA)-poly(dT) sequence             domain, utilized genetic approach based fact <br /> (Struhl, 1985).  GCN4-activated                      degree GCN4 function correlated cell <br /> transcription, initiation + 13 site increased 5-fold,   growth medium containing aminotriazole  , initiation + 1 remains essentially unaffected.         competitive inhibitor his3 gene product (Hope Preferential initiation + 13 observed GAL4          Struhl, 1986). starting strain, KY1600, contains activates transcription hybrid promoters          chromosomal deletion GCN4, plasmid expressing GAL enhancer fused various positions upstream          gcn4-C163 derivative (amino acids 119-281)    Oxford University Press                                                                                            177 <br />  M Collart K.Struhl <br />  <br />  <br />           dA-dT <br />  <br />  <br />  <br />  <br />              HIS3 <br />            gcn74- 1I <br />                          &lt;~ ~ ~ ~ ~ ~ <br />                         GCN4 <br />  <br />                         GCN4 <br />  <br />  <br />  <br />  <br />                                -    _ <br />                                      r&quot; C.. <br />                                      Tc <br />  <br />  <br />  <br />  <br />                                acti vatio&apos; r <br />                                                 <br />  <br />  <br />  <br />  <br />                                                     --_ <br />                                                           HIS3 <br />  <br />                                                           2 <br />                                                               v <br />  <br />  <br />  <br />  <br />                                                eIec v r2C n!N                                                                   -W- <br />  <br />  <br />  <br />  <br />                                                                    b^ <br />                                                                         HIS3 <br />  <br />  <br />  <br />  <br />                                                               Tiabi r ri;r <br />                                                                              lacZ <br />  <br />                                                                               2 <br />  <br />  <br />  <br />  <br />                                                                               / <br />                                                                                     O <br />  <br />  <br />  <br />                                                                                     gcnv <br />                                                                                          <br />                                                                                      H &apos; <br />                                                                                                   c <br />  <br />  <br />                                                                                                                                                                                                         86   ;; <br />                                                                                                                      S                                                                                                                        11 <br />                                                                                                                         &lt; <br />                                                                                                                       Sc3861 <br />                                                                                                                       Sc3862 <br />                                                                                                                       Sc3863 <br />                                                                                                                       Sc3864 <br />  <br />                                                                                                                       Sc3865 <br />                                                                                                                                 \ <br />                                                                                                                                     RR G HB H ScH <br />                                                                                                                                     II 1 <br />  <br />  <br />  <br />  <br />                                                                                                                                     RRGH <br />                                                                                                                                       _ <br />                                                                                                                                              ~oRF-o <br />  <br />  <br />                                                                                                                                                N <br />                                                                                                                                                 iw <br />                                                                                                                                                       H <br />                                                                                                                                                               G <br />                                                                                                                                                                <br />  <br />  <br />  <br />  <br />                                                                                                                                                               G <br />                                                                                                                                                                         B <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         B <br />                                                                                                                                                                              <br />  <br />  <br />                                                                                                                                                                             +- <br />                                                                                                                                                                             + <br />  <br />                                                                                                                                                                             + <br />                                                                                                                                                                                  AIs <br />  <br />  <br />  <br />                                                                                                                                                                                  + <br />  <br />                                                                                                                                                                                  + <br />  <br />  <br />  <br />  <br />                                                                                                                 Fig. 2. CDC39 locus. structures phenotypes (determined <br />                                                                                                                 complementation) indicated DNA fragments shown                                                                                                                 restriction map containing CDC39 gene region (broken lines <br />           (-&#194;&#167;c3866 )                          ;Si Cernlotj X g                    Sc3AI83L6                   shown Sc3860 indicate uncertainty lengths). location <br />               1 6_ <br />                           ~~~~~~~                                                 ,Ct1-67-                    transcriptional direction CDC39 indicated open box <br />                                                                                                                 arrow. S, Sall; B, BamHI; N, NcoI; R, EcoRI; G, BglII; H, <br />            faint blue                                                                                                                                                                                        HindIII; Sc, SaM. <br />            5 niM  <br />                                                                                                                 protein shows    significant similarities proteins Fig. 1. Genetic selection. structures HIS3 his3-lacZ                                              NBRF database.  sequence regions <br /> reporter genes (promoter elements transcriptional initiation sites <br /> indicated) GCN4 gcn4-C163 proteins (transcriptional                                                  amino acids 1009-1046 1300-1329 <br /> activation DNA binding domains indicated) shown.                                                     respectively contain 63 52% glutamine residues. <br /> parental strain (relevant chromosomal alleles indicated lines                                                Genomic hybridization indicates suppressor gene <br /> plasmid alleles indicated circles) subjected step genetic                                        present single copy patterns observed selection screen obtain unlinked suppressor mutations (X). <br />                                                                                                                  wild-type suppressor strains indistinguishable <br />                                                                                                                  (Figure 4A). Hybridization probe collection <br /> constitutive DED] promoter (Hope Struhl, 1986),                                                          X bacteriophages covering yeast genome reporter plasmid his3-lacZ fusion controlled                                                     (Olson et al., 1986) reveals suppressor gene maps promoter optimal GCN4 binding site                                                                 close genetically defined CDC39 locus right <br /> (Figure 1). KY 1600 grows 5 mM faintly blue                                                         arm chromosome III (L.Riles, personal communication). X-gal indicator plates. selecting derivatives                                                       CDC39 initially identified temperature-sensitive <br /> KY 1600 grow 20 mM darker blue,                                                            mutation (cdc39-1) causes cell-cycle arrest GI, <br /> isolated 13 independent suppressor mutations.                                                                   pachytene arrest effects meiosis,    Genetic analysis indicated suppressor mutations                                                     constitutively activates pheromone response pathway recessive define complementation groups.                                                       (Reed, 1980; Shuster Byers, 1989; Barros Lopes complementation groups defined single                                                          et al., 1990). Sequence comparisons mutation prevents growth 37&#194;&#176;C rich medium.                                                       suppressor gene CDC39 identical (S Reed, personal <br /> Diploids mutant wild-type strains grow                                                       communication) correspond YCR93W chromo- <br /> 37&#194;&#176;C, indicating temperature-sensitive growth                                                          III (Oliver, 1992).  cdc39-l1cdc39-2 <br /> phenotype recessive. temperature-sensitive                                                      heterozygotes grow high temperature indicating resistant phenotypes cosegregate 2:2 tetrads, indicating                                                  alleles fail complement.  refer single suppressor mutation confers phenotypes.                                                      suppressor mutation cdc39-2 corresponding suppressor mutation does confer                                                             gene CDC39. <br /> phenotype absence GCN4, mutant                                                                 parental cdc39-2 strains, CDC39 expressed parental strains grow indistinguishably 2 mM                                                              equivalent levels single 7 kb RNA (Figure 4B).                                                                                                                 transcript long encompass putative CDC39 <br /> Cloning molecular analysis CDC39                                                                         protein 2108 residues. 5&apos; end RNA gene corresponding suppressor mutation                                                           mapped primer extension analysis initiation sites, <br /> cloned complementation temperature-sensitive                                                      major 30 minor 34 nucleotides upstream resistant phenotypes. plasmid containing 20 kb yeast                                                    ATG putative CDC39 protein (Figure 4C). <br /> DNA (Sc3860) recovered transformant                                                           rich region containing sequence GATAAA shown complement phenotypes conferred                                                        centered 51 bp upstream RNA start sites likely <br /> suppressor mutation. Genomic integration genetic                                                            serve TATA element CDC39 transcription. <br /> mapping indicate Sc3860 tightly linked                                                           50 bp upstream putative TATA element suppressor mutation. phenotypically testing restriction                                                      poly(dA) poly(dT) stretch, yeast <br /> fragments, complementing gene delimited                                                            promoters utilized upstream promoter element <br /> 7.4 kb region (Figure 2).                                                                                       (Struhl, 1985). Interestingly, Sc3861 (Figure 2), DNA <br />   <span id='am-8' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-9' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> sequence EcoRI-BglIl fragment                                                           fragment contains putative TATA element entire <br /> (Sc3864) encompassing suppressor gene reveals single                                                      protein coding region lacks poly(dA) * poly(dT) stretch <br /> significant open reading frame 2108 amino acids                                                              sequences upstream, complements (Figure 3). Replacement 3276 bp coding sequence                                                        temperature-sensitive resistant phenotype. URA3 gene (Figure 2) eliminates complementation                                                          suggests elements required CDC39 phenotypes, indicating functional importance                                                    expression lower levels CDC39 sufficient <br /> putative protein. Despite large size, putative                                                     viability suppression. <br /> 178 <br />                                                                                                                       Role CDC39 transcription regulation <br />  <br />     1 gaattcttggaagaaacactgcctttgagtggaaagctatcattagt tactgttgtgcttgcaacagaagataaattgocctcttcatcagtgtcaaggc 100 <br />   101 agtgtccacaccatcacccatcttctgcttatgcgtcagctctgat tttaccgccttcttgaaaaacctgcttattgt 200 <br />   201 ggtgcccattgtatgtcaaataacatgatctttggctctcaaaagtaccagtttaggttcattacctaocacc 300 <br />   301   atgtcgtatactgaataggtgattcgtagggtatccaatttttttcggcac                                                                                            400 <br />   401   aataactgatgaagtacggcagtatgggacca                                                        gtaaaagfttaaagcccgg                                    500 <br />     501 caattgattac ATO crA TCG GCC ACA TAC CGT OAT TIC MC ACA GCA TCT MAT TTA GMA ACA TCA MAG GMA MAA CAG                                              577 <br />       1               M   L    S     T  Y   R    D    L  N   T      S   N    L  E  T   S   K    E   K   Q                                          22 <br />     578 GCC OCT CMA ATC OTC ATT GCA CMA ATT AGT TTA TTA TTC ACG ACT CTT MAC MAC GAC MAT TTT GMA TCC GTG GMA                                            652 <br />      23      Q      V        Q    S    L    L  F   T   T   L    N   N   D N    F   E    S   V   E                                          47- <br />     653 AGA GM. ATT AGA CAT ATT&apos; TTA GAC AGG TCG TCC GTA OAT ATT TAC ATA MAA GTT GMA CGA T! T! ACC TTA                                         727 <br />      48 R    E      R   H      L  D   R   S    S   V    D    Y      K   V   W  E  R    L    L   T   L                                          72 <br />     728 MGT TCT CGG OAT ATT TTA CMA 003 GGA AMA TT!&apos; TTA CTT CAA GMA MAT CTA CTA CAC AGA CTA CTA &apos;ITA GMA TTT                                          802 <br />      73 s    S   R   D       L  Q     0   K    F   L   L   Q   E   N    L1 L N R L L L E F                                                         97 <br />     803 GCO MOG OAT TTA CCG MOG MAA AGC ACA GAC CTT ATT GAG CTT TTG MAA GMA CGA ACC TTC MAT MAC CAG GAG TTT                                            877 <br />      98    K   D   L    P   K   K   S   T   D    L     E   L   1L K E R T F N N Q E F                                                              122 <br />    878 CMA MAA CMA ACA GGA &apos; ACA TTA TCA CTT TTC Aw! OAT CTA TTT OAT MAA TCT GCA MAC AMG OAC ATT ATA GAG                                            952 <br />    123 Q     K   Q   T   0      T   L   S   L    F     D   L. F D K S N K D I E                                                                147 <br />    953 TCA CTT GAC CGC TCC TCT CAG ATr MAC OAT TTC AMG ACA &apos; MG ATG MAT CAT ACA MAT TAT TTA AGG MAT TTT                                             1027 <br />    148 S     L  D    R   S    S   Q     N   D    F   K   T     K   M   N    N   T  N  Y    L   R   N    F                                          172 <br />  1028 TTT CTT CMA ACC ACA CCA GMA ACA CTA GAG TCC MAT CTA CGC GAC TTA rTG CAT TCC TrIO GMA GOT GMA AGT CTA                                             1102 <br />    173 F     L  Q    T   T     P  E   T   L   E    S   N   L   R   D   L   L    N   S  L  E    G   E    S   L                                          197 <br />  1103 MAT GAC TTA TTA OCT CTT&apos; TTA CTG TCC GMA ATA COT TCA CCT GGG TCT CMG MAT TTA CMA MAT OAT CCC ACA CCGG                                            1177 <br />    198 N     D  L    L      L   L   L   S   E      L   S   P   G   S   Q    N   L  Q  N    D   P   T   R                                           222 <br />  1178 AIT TOGG T`T0 ACA CCT CCG ATG OTT TTA GAC GCA ACO MAC COT 000 MAC OTT ATA GCA AGA TCT ATA MGT TCT CTG                                            1252 <br />    223 S    W   L    T   P    P   M   V   L   D      T   N   R 0     N   V       R  S      S   S   L                                           247 <br />  1253 CMA 0CC MAC CM ATA MAT TOG MAT COT GTG TOT MAT TTA ATG TCA ACA AMG TAT TTC TTG AGC GCA CCA ITOG ATC                                              1327 <br />    248 Q      N    Q      N   W N&apos; R V         F   N   L   M   S  T    K    Y   F  L  S      P   L   M                                           272 <br />  1328 CCT ACT ACA GCA TC TTG AGT TiGC TTA TTT GCA GCA TTGI CAC OAT GGT CCA OTT ATT OAT GMA lO TTC AGT TGC                                              1402 <br />    273 P    T   T      S    L   S   C   L   F        L   H   D  G    P    V    D  E    F   F   S   C                                           297 <br />  1403 GAC TOOG MAA OTT ATT TTC MAA CTA OAT TTo 0CC AlIT CMA COT CAT MAG TGG TCG GTA CAG MAT GGT CCC TTT GAC                                            1477 <br />    298 D W      K    V      F   K   L   D   L        Q   L   H  K    W    S   V  Q N    0   C    F   D                                           322 <br />  1478 ITA TTA MAT GCA GMA GOT ACC AGO MAA OTT TCT GMA ACC ATC CCA MAC ACA MAG CMA TCT TTA CTC TAC TTA ITA                                              1552 <br />    323 L    L   N      E    0   T  R    K  V     S   E   T     P  N    T    K   Q  S  L    L  Y    L   L                                           347 <br />  1553 TCCC ATT GCA WCA TTG MAT TTA GMA TTG TTC CTA CMA MOG GAG GMA 10G TOT OAT GOT CCT ATG CTA OCT TAT lT7                                              1627 <br />    348 S         S   L   N   L    E   L  F     L  Q    R   E   E  L    S   DOG     P M    L      Y   F                                           372 <br />  1628 CMA GAG TGC TTC &apos; GMA OAT TIC MAC TAC GCC CCT&apos; GMA TAT CTT ATT TTA GCA TTA OTC MAA GMA ATG MOG COG                                             1702 <br />    373 Q    E   C    F   F    E  D   F   N   Y       P   E  Y    L     L     L  V  K    E   M    K   R                                           3 97 <br />  1703 TTC OTT TTA rIO ATA GMA MAC AGO ACA GTC ATA GAC GMA ATA CTT ATT ACC TTA TTG ATT CMA OTG CAT MAT MAA                                              1777 <br />    398 F    V   L   L       E N    R   T V        D   E      L    T    L   L   Q V      N    N   K                                           422 <br />  1778 TCA CCO TCA TCG TT&apos;C MAG GAC OTT ATT TCT ACA ATA ACC OAT OAT TCT AMA ATC OTA OAT GCA GCA MAA ATC ATA                                             1852 <br />    423 S    P   S    S   F   K   D   V     S    T     T   D    D  S   K      V  D       K                                                447 <br />  1853 ATC MAC TCG OAT GAC GCA COT ATIT 0CC MAC TT!&apos; TTA AMA TCG&apos; rio TTA OAT AGO GOA AGA TTrA OAT ACG GTC ATT                                          1927 <br />    4481I N      S   D   D      P     N      F   L   K   S   L   L   D    T   0  R  L    D   T    V                                             472 <br />  1928 MAT MAA CTT ccT TTc MAT GMA GOT TV!&apos; AMA ATT Tio CCA TCC OCA AGA CMA ATT GOT GMG 000 TTC OAT ACT                                             2002 <br />   473 N     K   L   P   F    N   E     F   K      L    P  C     R   Q      0  W  E    0   F    D   T                                           497 <br /> 2003 TTC TTA AMA ACA MAA GTT TOT CCA TOT MAT OWC OAT OTA GTG CTG GMA WCA CTA GAO OTT CMA ACO MAA ACC ACT                                               2077 <br />   498 F     L   K   T   K    V   S   P   S   N    V   D   V   V   L   E   S    L   E  V  Q    T   K    M   T                                           522 <br /> 2078 OAT ACA MAC ACT CCA TV!&apos; AGO WCA TTA MOG ACA nTT GAC rIA TTc OCT OT7 CAT WCA TTA ATT GMA OTA CCC MAC                                              2152 <br />   523 D    T   N    T    P   F   R   S   L   K    T   F   D   L   F     F    N   S  L     E   V    L  N                                            547 <br /> 2153 MAA CCGC CCA CTA OAT OTT CTC CMA TTA CMA AGG Tyr GMA TCC TCC GMA rTr CCC TTA TTA ATT GCA TTT COT AGA                                              2227 <br />   548 K    C    P   L   D    V   L   Q   L   Q    R   F   E   S   L   E   F    S   L  L       F    P  R                                            572 <br /> 2228 rio ATC MAT TOT GOT TIT! GOA CAC OAT GMA OCT ATT TTA GCC MAT GGT GAC ATC GCA 000 ATT MAT MAT OAT ATT                                              2302 <br />   573 L      N    F   0    F   0   N   D   E        L     N   0   D      G      N   N    D                                             597 <br /> 2303 GMA MOG GMG ACC CAG MAC TAT TTA CAG MAA ACC TAT AIT GOT GAO TTA 0CC ATT AMA OAT OTA AWC GMA CTT CCC                                               2377 <br />   5985E K E M Q N Y L Q K M Y S 0 E L K D V E L L                                                                                                622 <br /> 2378 AGA AGO TTA AGA OAT AGC GAC TIO CCA AGO GAC CMG GMA OTC TTC ACA TOGT ATT ACC CAT 0CC GOT ATA GCA GMA                                              2452 <br />   623KR R      L    R   D    S   D   L   P R      D   Q   E   V   F   T   C      T  N     V        E                                           647 <br /> 2453 TCG ACA TTC TTC CAA OAT TAT CCA TM GOAT GCA rTo OCT ACT ACA TOT OTT CTT! rr GOA TCC ACC ATTcr    CTC rr                                           2527 <br />   648 S    T   F    F   Q    D   Y   P   L   D      L     T   T   S   V    L   F  0  S    M      L  F                                            672 <br /> 2528 CMA CCC TTA COT GGA TTC GTA TTA OAC OWC GCA TTT MOG ATA AWC ACC AGO TIT! 0CC MOG GAG COT CCA GAO TCC                                              2602 <br />   673 Q    L   L    K   0    F   V   L   D V        F   K       M   R    F    K  E    P   P   E   5                                            697 <br /> 2603 MOG ACCG r     AMG TIT! GOT GTA CMA OCT T TAT GCA T MGG ATA COT Tio 0CC GMA TAT CCA CAG TAT CCT AAG                                            2677 <br />   698 K    M   F    K   F      V   Q        Y     F   K     R   L      E  Y  P    Q   Y   C   K                                            722 <br /> 2678 OAC CTC rTG AOA OAT OTT CCG GOT rio MOG TOT CMG OCT CMA OTT TAC CMA TOT ATC OTC GMA GOT OCT ACC CTA                                               2752 <br />   723 D    L   L    R   D    V   P     L   K    S   Q     Q V     Y   Q S       V  E        T   L                                            747 <br /> 2753 GCA MAT GOT CCA MOG GMA AGO WCA MGA CCC OWC CMG GMA ACC ATc CCA TTA AMA TTT TT!&apos; GOT OTA OAT GMA OTT                                              2827 <br />   748 N         P   K    E   R   S   R   P    V   Q   E   M     P   L    K   F  F  V      D   E   V                                            772 <br /> 2828 WCA TOGT CMG ATC MAT CMA GMA GOT GOT COT AMA OAT OTC OTA GMA AMA OTT cT&apos;r rTTT OTT CTC MAC MAC OTT ACT                                            2902 <br />   773 S    C   Q      N    Q   E   0     P    K   D V     V   E   K   V    L   F  V  L N      N   V   T                                            797 <br /> 2903 CTOG OCT MAC TT MAT MAT MOG OTT OAT GMA Tio MAA MAA NIT TCCG ACA CCA MAT TAT TTT TOT TG0 TTT &#194;&#182;1PCC ACA                                            2977 <br />   798 L      N    L   N    N   K   V   D   E    L   K   K   S   L   T   P    N   Y  F  S    W   F   S   T                                            822 <br /> 2978 TAT TTA OTT ACGO CMA AGO GOT AMA ACA GMA COT MAC TAT CAT OAT COT TAT MGC MOG OTT ATA GTT GOT ACC 000                                              3052 <br />   823 Y    L   V   T    Q    R     K   T   E    P   N   Y   H   D   L   Y    S   K  V     V     M   0                                            847 <br /> 3053 WCA 000 rio OTA CAT CAG TTC ATG OTC MAC OTT ACT rio NIA CMA TTA TTT OTC OTA CTA TOT ACA MAA GAC GMG                                               3127 <br />   848 S 0      L   L    N    Q   F   M   V N      V   T   L   R   Q   L   F    V   L  L  S    T   K   D   K                                            872 <br /> 3128 CMA 0CC ATC OAT MAA MOG CAC OTA MOG MAT TTG GOT WCA TOG TTA GGA TOGT ATC ACA TTA GOT Tio MAT MAA CCA                                              3202 <br />   873 Q        D    K    K   N   L   K N      L     S   W   L   0   C      T  L     L   N   K   P                                            897 <br /> 3203 ATT MAA CAC AMG MAT AWC GCA TTC MGG GMA ACC TTA ATC GMA OCT TAT AMG GMA MAT AGA CTT GMA ATA OTT G0CC                                              3277 <br />   8981I    K   N   K    N        F   R   K    M   L     E     Y   K    E   N  R  L    E     V   V                                            922 <br /> 3278 COT TOT GTA ACA MOG ATT TTA CMA MOG OCT TOT GMA WCA MAA ATT TTC AAO COT CCA MAT CCC &#194;&#182;100 ACT 0GTT GGOC                                            3352 <br />   923 P    F   V   T    K      L   Q   R      S   E   S   K     F   K    P   P  N  P    W   T   V   G                                            947 <br /> 3353 ATA TTA AAG CCC rio ATT GAG Tio MAC GMA MAA GCA MAC T00 MAA TTA AOT TTo ACT TTC GMA (GTT GAG 0G02 TTA                                             3427 <br />   9481I L K        L    L      E   L   N   E    K     N   W   K   L   S    L   T  F  K    V   E   V   L                                            972 <br /> 3428 TrA MAA TOT TTT MAT Tio ACC ACC MAA TOT CTC AMG CCC TCG MAT TTC ACC MAT ACT CCO GMA OTT ATA GMA ACTr                                              3502 <br />   973 L    K   S   F    N    L   T   T   K   S    L   K   P   S   N   F      N   T  P  E    V     E   T                                            997 <br />  103 TIA TCC GGT CCL` CCC GGA TCA ATC ACT CTG GAG CACACAAAGG                                        C.AAG&apos;<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>;AVATCIACPAA37 <br />  998 L  S G L G S   T   L E(Q                                         )()T            E                         0  1022, <br />   359~ACAC AG_QGAGCA  T    P  AG                                          PAkA                     -AAA; C VA T 3652 <br /> 3 653 &apos;CTATT <br /> 1048 H                     ~ GCP&apos;0~T    ACA ATC GCA GAC &apos;VAA&apos;cAA          GCACATTTAC <br />                                                                         3TCTCTC1727 <br />                                                                                 AC <br />                 S       N     T       D    Q   Q                     DF     G   G     E     G      S       S       H           N     P     1072 <br /> 3 728 TIT MAC   MAC TTA CIT GGT TCT ACT ATT TTT GTA ACC CAC CCT GAC TTG P.AG AGG GTA TTT CPAI ATG GCT TTA GCC                                          3902 <br /> 1073 F    N     N   L     L     0   S     T         F     V     T       H     P     D   L     K     R V        F     Q       H         L         1097 <br /> 3803 MAG TCA    GTT COG   GMA   ATT TTG   TCC GMA OTA GTC GMA MOG WCA                 TCA GGA   ATT   GCT <br />                                                                                                         GTT      GTT   ACI&apos;,   ACG   ACA   MAA   ATA   3877 <br /> 1098 K    S     V   K     E       L     L     E     V   E V             K   s       s   G         V        v     T       T     T     K         1122 <br /> 3878 ATA CIT    MAA GAC   TTT   0CC ACT   GMA OTT OAT GAO TCT             MOG TTG     MOG ACG   OCT     ATC <br />                                                                                                       GCA        ATT   A7N7    GTA   AGG   CAT   TCC   39 52 <br /> 1123    L     K   D     F       T     E   V   D   81  5               K   L       K   T                    H       V     R     H     L     1147 <br /> 3 953 OCA CMA   AGT TTA   OCT   CGA OCT   ACT WCA ATT GMA CCA             rio MAA     GMA GGC   ATA     TCI <br />                                                                                                        CGT       ACT   ATW     CM.   WCA   CTA   GCA   4027 <br /> 1148    Q     S   L         R       T   S     K   P               L   K       E   0         R S        T     H       Q     S     L         1172 <br /> 4028 CCG MAT    TTA ACC   WCT   CIT TCT   TCT WCA CCI GCA GAO GAOGI                   GAC ACG   GCA     MAT <br />                                                                                                       ATA        GMA   MAT     ATT   GGC   ATT   OCT   4102 <br /> 1 1 73 P1 NI    L   H     S     L                                             S                                               T         N0       S     N0  T <br />                                                                                                                                      0     T         1   1907 <br /> 4103 CTA GTT    TTG ATT   GAG   AAA GCA   TCT   ATG   GAC   AAG TCT       ACT   CAA   GAT TTA   GCA     CAA <br />                                                                                                       GAC        TTG   ATG CAA       GCG   ATT GCT     4177 <br /> 1198 L V        L       E     K       S     M     D     K   S         T     Q     D   L         D Q        L     M       Q                 1222 <br /> 4178 ATT CGT    CGT TAT   CAC   AAG GAA   AGA   AGG   GCA   GAC CAA       CCA   TTT   ATT ACG   CAA AAT ACC      AAT   CCA     TAT   TCA   CTG TCT     4252 <br /> 1223    R     R   Y     H     K   E     R     R         D   Q         P     F       T     Q   N   T        N     P       Y     S     L   S       1247 <br /> 4253 TTA CCA    GAA CCT   CTT   GGT TTG   AAA   AAC   ACT   GGT GTT       ACT   CCT   CAA CAA   TWC AGG GTA      TAC   GAA     GAA   TTT   GGT AAG     43 27 <br /> 1248 L    P     E   P     L     G   L     K     N     T     G   V         T     P     Q   Q     F   R   V        Y     E       E     F     G   K       1272 <br /> 4328 AAT ATT    CCA AAC   TTG   GAT GTT   ATT   CCG   TTT   GCA GGA       TTG   CCC   GCT CAC   GCT CCA CCG      ATG   ACT     CAA   AAT   GTG GGT     44 02 <br /> 1273 N        P   N     L     D   V         P     F       G         L     P       H       P   P        M     T       Q     N     V   G       1297 <br />  <br />                                                                                                                                                                   179 <br />  M Collart K.Struhl <br />  <br />                 4403 TCA ACT fCAG CCT CAG c  AACAA GCG CAA ATG CCT ACC,CAA ATC CTA ACC TCC GAA,CxAA ATA AGA GCT,CAA         4477 <br />                 1298 S T (Q) P (Q) (Q) (Q)(Q) (Q) M             P  T (Q       L T     Q) R (Q) (Q)              1322 <br />                 4478 CAA aAG&#194;&#167;*G 4A:tA G&gt;AAGC CGT TTG AAT $:G CCA &apos;C AG TCG GCTr CCT CA GGA GTG AATGTC tCA                 4552 <br />                 1323(Q) (Q)(Q)(Q)     L (Q)K     S   R    L  N (Q) P      S (Q) S     (Q) P     P   G   V   N   V   P     1347 <br />                 4553 (LAT (TDTiA <br />                              (Q     T GGG TT GCT GCA GTT CAA TCA TTG GAAAG AAT CAA &apos;T GTT CTC GTT CAC CTC ATG GAC        4627 <br />                 1348 N    P  Q    G   G        V   Q    S   D   L   E   Q N     Q R V       L   V   H   L   M   D     1372 <br />                 4628 ATT TTA GTT TCT CAA ATT AAA GAA AAT GCT ACG AAG AAT AAC TTA GCT GAA TA GGC GAT CAA AAC CAA ATT AAA     4702 <br />                 1373    L  V    S   Q    K   E   N     T    K  N    N   L     E   L G     D   Q N     Q     K     1397 <br />                 4703 ACC ATC ATT TTT CAA ATT TTG ACA TTC ATT GCA AAA AGC GCA CAA AAG GAT CAA TTA GCT TTA AAG GTA TCC CAA    4777 <br />                 1398 T        F   Q    L   T    F       K   S     Q   K   D   Q   L     L   K   V   S   Q     1422 <br />                 4778 GCT GTC GTT AAT AGC CTT TTT GCC ACT AGT GAG AGT CCT CTC TGC AGA GAA GTT TTG TCC CTA CTT TTG GAA AAG    4852 <br />                 1423 V V N          S   L  F     T    S   E   S   P   L   C   R E V L         S   L   L   L   E   K     1447 <br />                 4853 TTA TGT TCT TTA TCC CTC GTT GCT AGA AAA GAC GTT GTC TGG TGG TTA GTT TAT GCC TTG GAC AGT AGG AAA TTC    4927 <br />                 1448 L C     S    L   S   L  V     R   K    D  V V      W   W   L V Y         L   D   S   R   K   F     1472 <br />                 4928 AAT GTT CCC GTT ATC AGA TCC CTT CTA GAA GTT AAT TTA ATT GAT GCT ACA GAA TTA GAT AAC GTT TTA GTT ACT    5002 <br />                 1473 N V     P    V     R  S   L    L   E  V   N    L     D     T   E   L   D   N   V   L   V   T     1497 <br />                 5003 GCA ATG AAA AAT AAA ATG GAG AAC TCA ACT GAA TTT GCT ATG AAA TTA ATT CAG AAT ACT GTC TTG TCT GAT GAT    5077 <br />                 1498 M K N K M             E   N    S   T   E   F     M   K   L     Q   N   T   V   L   S   D   D     1522 <br />                 5078 CCA ATT TTG AMG AGA ATG GAC TTC ATT AAA ACC TTA GAA CAC TTG GCC TCT TCG GAA GAT GAA AAT GTA AAG AAA    5152 <br />                 1523 P     L    M   R   M  D   F      K  T    L   E   H   L     S   S E     D   E N V K         K     1547 <br />                 5153 TTC ATC AAA GAG TTC GAA GAT ACT AAG ATA ATG CCA GTG AGG AAA GGT ACC AAA ACC ACA AGA ACA GAA AAG CTT    5227 <br />                 1548 F K        E   F   E  D   T   K     M    P  V    R   K   G   T   K   T   T   R   T   E   K   L     1572 <br />                 5228 TAC TTA GTA TmT ACG GAA TGG GTA AAA TTA CTT CAA AGA GTT GAG AAT AAC GAC GTA ATC ACA ACT GTT TTT ATC    53 02 <br />                 1573 Y L V F T E W V K                    L   L  Q    R   V   E   N   N   D V       T   T   V   F       1597 <br />                 5303 AAG CAA TTA GTC GAA AAG GGT GTT ATC AGC GAT ACT GAT AAT TTA CTT ACA TTT GTC AAA AGT TCT CTT GAG CTA    5377 <br />                 1598 K    Q  L    V   E   K  G   V      S  D   T   D    N   L   L   T   F   V   K   S   S   L   E   L     1622 <br />                 5378 TCA GTT TCT TCA TTC AAA GAA AGT GAC CCG ACT GAT GAG GTT TTC ATC GCT ATT GAT GCT CTA GGA TCG CTA ATT    5452 <br />                 1623 S V S S F K E S D P T D E V F I D L G S L                                                      1647 <br />                 5453 ATA AAA TTG TTG ATT TTA CAG GGT TTC AAA GAT GAT ACA AGA AGA GAT TAC ATA AAT GCA ATA TTT TCT GTG ATC    5527 <br />                 1648    K  L    L     L  Q   G   F   K   D   D   T    R   R   D   Y     N       F   S   V       1672 <br />                 5528 GTT TTA GTG TTT GCT AAG GAT CAT AGC CAA GAG GGT ACC ACA TTC AAT GAA CGA CCA TAT TTC AGA CTA TTT TCT    5602 <br />                 1673 V L V        F     K  D   H    S  Q    E  G   T    T   F   N   E   R   P   Y   F   R   L   F   S     1697 <br />                 5603 AAC ATC TTA TAC GAA TGG GCT ACC ATC AGG ACG CAC AAT TTT GTT AGA ATA TCT GAT TCC AGC ACT AGG CAG GAG    5677 <br />                 1698 N L Y E W T                   R   T    H  N    F   V   R     S   D   S   S   T   R   Q   E     1722 <br />                 5678 CTG ATC GAA TTT GAT TCT GTA TTT TAC AAC ACT TTC TCA GGA TAT TTG CAC GCT CTG CAA CCA TTT GCC TTC CCT    5752 <br />                 1723 L     E    F   D   S  V   F   Y   N   T    F   S   G   Y   L   H     L   Q   P   F     F   P     1747 <br />                 5753 GGA TTC TCA TTT GCA TGG GTG ACA CTA TTA TCA CAC AGA ATG TTA TTA CCA ATT ATG CTA AGA ITA CCC AAT AAA    5827 <br />                 1748 G F     S    F   W    V   T   L    L   S   H   R   M   L   L   P     M   L   R   L   P   N   K     1772 <br />                 5828 ATA GGT TGG GAA AAG TTA ATG CTT TTG ATT ATC GAT TTIG mTT AAA TTT TTG GAC CAA TAC ACA AGT AAA CAT GCA   5902 <br />                 1773 G W E K L M L L                       D    L   F   K   F   L   D   Q   Y   T   S   K   H       1797 <br />                 5903 GTC TCT GAC GCT GTT TCG GTT GTT TAT AAG GGA ACA CTG CGT GTT ATT TTA GGC ATT TCG AAT GAT ATG CCA TCC    5977 <br />                 1798 V S D V S V V Y K G T L R V                                  L   G     S   N   D   M   P   S     1822 <br />                 5978 TTT TTG ATT GAA AAT CAC TAT GAA TTA ATG AAC AAT CTA CCT CCA ACA TAT TTC CAA CTA AAG AAT GTT ATT TTA    6052 <br />                 1823 F L        E   N   H  Y   E   L   M   N   N    L   P   P   T   Y   F   Q   L   K   N   V     L     1847 <br />                 6053 TCT GCT ATT CCT AAG AAT ATG ACC GTT CCC AAC CCA TAT GAC GTG GAT CTT AAT ATG GAG GAT ATT CCA GCA TGT    6127 <br />                 1848 S        P   K   N  M   T   V    P  N    P  Y    D   V   D   L   N   M   E   D     P     C     1872 <br />                 6128 AAA GAA CTA CCT GAA GTC TTC TTT GAT CCT GTA ATT GAT TTA CAC TCA TTG AAA AAG CCA GTT GAC AAC TAC CTA    6202 <br />                 1873 K E L P E V F F D P V D L H S L K K P V D N Y L                                                      1897 <br />                 6203 CGT ATT CCC TCA AAT TCA TTA TTA AGA ACA ATA CTA AGC GCT ATT TAC AAG GAT ACC TAT GAC ATA AAA AAG GGC    6277 <br />                 1898 R     P    S   N   S  L   L   R   T      L   S       Y   K   D   T   Y   D     K   K   G     1922 <br />                 6278 GTA GGC TAC GAC ITT TTA TCT GTT GAT AGT AAA TTA ATT CGC GCT ATT GTA TTA CAT GTG GGC ATT GAA GCT GGA    6352 <br />                 1923 V G Y D F L             S   V   D   S   K   L      R       V   L   H   V   G     E     G     1947 <br />                 6353 ATA GAG TAT AAG AGA ACT TCT TCA AAT GCG GTA TTT AAT ACG AAG TCT TCT TAT TAT ACT TTA ITG TTC AAT CTG    6427 <br />                 1948    E  Y    K   R   T  S   S   N     V   F   N    T   K   S   S   Y   Y   T   L   L   F   N   L     1972 <br />                 6428 ATT CAA AAT GGT AGC ATC GAA ATG AAA TAT CAA ATT ATT CTG TCT ATT GTG GAA CAA TTG CGG TAT CCA AAC ATC    6502 <br />                 1973 1    Q  N    G   S    E   M   K   Y   Q        L   S     V   E   Q   L   R   Y   P   N       1997 <br />                 6503 CAC ACC TAT TGG TTC AGC TTT GTG TTA ATG AAT ATG TTC AAA AGT GAC GAA TGG AAT GAT CAA AAA CTT GAA GTC    6577 <br />                 1998 H T Y W F S             F   V   L   M   N   M   F    K   S   D   E   W N     D   Q   K   L   E   V     2022 <br />                 6578 CAA GAA ATT AIT TTA AGA AAC TTr TTA AAA AGA ATT ATT GTT AAC AAA CCA CAT ACC TGG GGT GTT TCA GTT TTC    6652 <br />                 2023 Q E          L   R  N   F   L   K   R        V   N   K   P   H   T   W   G   V   S   V   F     2047 <br />                 6653 ITT ACT CAG TTG ATA AAC AAT AAC GAT ATT AAT CTT TTA GAC CTG CCC ITT GTA CAA AGT GTT CCC GAA ATT AAA    6727 <br />                 2048 F T Q L            N  N   N   D     N   L   L    D   L   P   F   V   Q   S   V   P   E     K     2072 <br />                 6728 CTA ATT TTA CAA CAA TTA GTA AAA TAT TCC AAA AAA TAC ACA ACC AGT GAA CAA GAT GAC CAA TCC GCC ACC ATC    6802 <br />                 2073 L     L    Q   Q   L  V   K   Y   S   K   K   Y    T   T   S   E   Q   D   D   Q   S     T       2097 <br />                 6803 AAT AGA AGG CAA ACC CCT CTA CAA TCC AAC GCA TAA aaaaactgcatttatttatatacgaaaatcatgatttcagaaaaaaaaatat   6890 <br />                 2098 N    R  R    Q   T   P  L   Q   S   N                                                                2109 <br />                 6891 aacttgtataatactatatttcagagaatactattttgaataagaatagaagcaatttcgattaacttttcttactgtatatgggtcgctttcacttttg   6990 <br />                 6991 aactactgaaaattagtgttacccgtattattgttttggagaataataataataatgagcatcaatttatatcaaacaagagctgcaattaccttatgtc   7090 <br />  <br />  <br /> Fig. 3. CDC39 sequence. nucleotide sequence Sc3864 putative <span id='am-16' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> sequence CDC39 shown major RNA <br /> start sites (arrows), potential promoter elements (underlined) glutamine-rich region (glutamine residues circled). <br />  <br />  <br /> CDC39 essential viability                                          Suppression independent mating pheromone <br /> Strains containing original (cdc39-1) present               response pathway <br /> (cdc39-2) allele unable grow 37&#194;&#176;C, suggesting              initial cdc39-1 allele constitutively activates mating <br /> CDC39 essential gene. determine phenotype                    pheromone response pathway acting level cdc39 null mutation, disrupted CDC39 locus                 transducing G protein  Barros Lopes et al., 1990). diploid strain (KY805) cdc39:: URA3 substitutution             deduced cell-cycle defect consequence allele described  Genomic hybridization confirms               cause pheromone pathway induction. resulting strain (KY1640) heterozygous wild-               cdc39-2 mutant strain does exhibit GI arrest <br /> type disrupted CDC39 alleles (Figure 4A). RNA analysis                phenotype restrictive temperature arrests heterozygous diploid shows disrupted copy                 asynchronously, remained possible cdc39-2 CDC39 expressed small transcripts ending                  mutation affected pheromone response pathway URA3 inserted gene (Figure 4B). Sporulation                       indirectly caused suppression phenotype. <br /> dissection KY1640 resulted tetrads viable                address issue, constructed ste4A::URA3 inviable progeny. viable progeny Ura-                        stel2A:: URA3 derivatives KY 1603 lack functional <br /> indicating presence CDC39 allele absence               pheromone response pathway. strains mate cdc39:: URA3 allele. Microscopic examination                       STE4, : subunit transducing G protein <br /> indicated inviable spores germinated yielded             (Whiteway et al., 1989) STEJ2, pheromone-responsive <br /> microcolonies consisting 10-20 normal looking cells.                   transcription factor (Dolan et al., 1989; Errede  CDC39 encodes essential gene required cell                   Ammerer, 1989), essential components pheromone <br /> growth.                                                                   response pathway.  strains grow  <br /> 180 <br />                                                                                                    Role CDC39 transcription regulation <br />  <br />                                          B                                            control       352     \34     A44 AC163 GCN4 <br />                                                                                          Wt 39      Vt39 Wt 39 Wt 39 Wt39 Wt 39 <br />               ; ca      .c  <br />                                                                           +1 --*_ <br />                                                            4-       : <br />     .*. <br />      -N.&apos; <br />                                                                                                                                          -pw <br />                                                                          +13 -l- <br />  <br />                                                                         DED11 -*                                      *        * <br />  <br />                                                            4  . <br />  <br />  <br />  <br />                                                                         Fig. 5. Transcriptional activation indicated GCN4 derivatives         C                                                               wild-type cdc39-2 strains grown permissive temperature. <br />                                                                         Total RNA samples (25 Ag) hybridized completion                                   G      T-C- <br />                                            av&gt; <br />                                             s                           excess HIS3 DED] oligonucleotide probes digested SI <br />                                                                         nuclease. positions DEDI H1S3 + 1 + 13 <br />                                                                         transcripts indicated. described previously (Hope Struhl, <br />                                                                         1986; Hope et al., 1988), gcn4-52 contains residues 107-121 fused                                                   / <br />                                                                         179-281, gcn4-A34 contains residues 1-95 fused 144-281, <br />                                                                         gcn4-zA44 contains residues 107-137 fused 179-183,                                                                         gcn4-C163 residues 119-281. Hybridization reactions carried                                                                         parallel, products analyzed gel;  <br />                                                                         autoradiographic exposure times detecting various <br />                                                                         transcripts equivalent. <br />  <br />  <br />                                                                         Table  Effect cdc39-2 resistance HIS3 transcript levels <br />                                                                         strains containing various GCN4 derivatives <br />  <br /> Fig. 4. Genomic transcriptional analysis CDC39, cdc39-2                                               HIS3 transcriptionc <br /> CDC39/cdc39::URA3 strains.   Genomic DNAs wild-type <br /> KY803  , mutant KY1617 (b), heterozygous disruption KY1640        Genotypea                     Growthb    +1        +13     ratio +1/+13 <br /> (c) strains hybridized radiolabeled CDC39 probe. positions molecular weight markers shown left.        gcn4 1       CDC39             &lt;2      20       21      0.95 <br /> CDC39 cdc39-2 alleles restriction fragments      gcn4       cdc39-2           &lt;2      21       22      0.95 <br /> following sizes: BamHI, 6 kb; Sall, 20 kb; NcoI, 20 kb; EcoRI, 10       gcn4-A52d      CDC39             &lt;2      22       23      0.95 <br /> kb. fragment sizes cdc39::URA3 allele           gcn4-A52d      cdc39-2           &lt;2      19       20      0.95 <br /> follows: BamHI, 11 kb site lost; SaII, 18 kb; NcoI, 4 kb   gcn4-A34d      CDC39              2      19       27      0.70 17 kb site URA3; EcoRI, 8 kb. (B) Total RNAs strains electrophoretically separated,                                                                         gcn4-A34d      cdc39-2           10      23       29      0.79 <br /> hybridized CDC39 probe. positions 26S, 17S rRNAs      gcn4-A44d      CDC39              5      20       26      0.77 7 kb CDC39 RNA indicated right. (C) Primer               gcn4-44d       cdc39-2           40      25       29      0.86 <br /> extension product (PE), generated extending 5&apos;-end-labeled CDC39     gcn4-C 163     CDC39              5      19       28      0.68 <br /> oligonucleotide primer RNA KY803, compared              gcn4-C 163     cdc39-2           40      26       32      0.81 <br /> products DNA sequencing using primer. Arrows indicate       GCN4           CDC39          &gt; 120      19       60      0.32 5&apos; end CDC39 RNA.                                            GCN4           cdc39-2        &gt; 120      30       63      0.48 <br />  <br /> 40 mM KY 1603, indicating cdc39-2 suppresses                 aStrains contain chromosomal gcn4-A1 allele (lacks GCN4 coding gcn4-C 163 activation mutant independently                   sequences) plasmid expressing indicated   gcn4 <br />                                                                         derivative. <br /> pheromone response pathway. Conversely, parental                    bMaximal concentration (mM) supports cell growth. <br /> CDC39 strain stel2A:: URA3 ste4A:: URA3                      CLevels + + 13 transcripts (arbitrary units) normalized derivatives indistinguishable resistance phenotypes             level DEDI internal control hybridization reaction. <br /> indicating elimination mating response pathway              dDescribed Hope et al. (1988). <br /> does lead suppression. <br />                                                                         gcn4-A52, derivative lacks functional transcriptional <br />                                                                         activation region. <br /> Effect cdc39-2 transcriptional activation                         examine directly effect cdc39-2 GCN4- <br /> varous GCN4 dervatives                                                  activated HIS3 transcription, levels + 1 + 13 order test suppression cdc39-2 specific            transcripts quantitated respect internal DEDJ gcn4-C 163 derivative used isolation,                 control (Figure 5; Table . expected, transcriptional <br /> introduced number different GCN4 derivatives wild-            activation GCN4 partially functional derivatives type cdc39-2 strains tested resistance                wild-type strains increases transcription + 13 site <br /> resulting transformants. shown Table  cdc39-2                   + 1 site. accord growth phenotypes, <br /> increases resistance strains containing gcn4-A34              cdc39-2 slightly increases overall HIS3 transcription gcn4-A44, partially functional derivatives different               strains containing GCN4 derivatives capable partial <br /> portions activation domain. contrast, cdc39-2 does            activation (A34, A44, C 163), strains containing appear increase transcription strain containing           non-activating derivative gcn4-A52. Surprisingly,  <br />                                                                                                                                            181 <br />  M Collart K.Struhl <br />  <br />  <br />          _cn4 <br />           6W3 <br />           1 <br />                                   uCfl <br />                                    1 63 <br />          Wt     y3(         Wth%c <br />  <br />  <br />  <br />  <br />                                                                                                DED    .r&apos; <br />  <br />  <br />  <br />  <br />               :....   ::: <br />  <br />                             *w                      .4---            &quot;C.. <br />  <br />                 .40- <br />                             .* ;&apos;10                                                                               cdt- <br />                                                                                              tk.v         cdc <br />                                                                                                             39     26 <br />                                                                                          ;:     1. <br />                                                                                        rm.                    3   0 3        4--    hrs 38.5 <br />                                                     .4--    r4J. <br />                                                                                                                                                10V <br />                                                                                                                                       tRNA <br />                             ,     :..,   .... <br />  <br />  <br />  <br />  <br />                                                   .4---   H h&apos;.v                                                              ... <br />                                                                                                                                   HIS4 <br />                              .ii. <br />                                                               t                                                              .4--     UEDIE <br />  <br />  <br /> Fig. 6. Transcription various genes wild-type cdc39-2 strains <br /> grown permissive temperature. Total RNA samples (25 itg)                                  :FI.....-D <br />                                                                                                  .- <br />                                                                                                   S. <br />                                                                                                      &apos; <br />  <br />  <br />  hybridized completion excess indicated probes digested SI nuclease. Hybridization reactions carried parallel, products analyzed gel;                                                         *             ilRP3  autoradiographic exposure times detecting various <br /> transcripts equivalent. <br /> effect cdc39-2 GCN4-dependent activation                      Fig. 7. Transcription various genes wild-type, cdc39-2 cdc28 <br /> pronounced + 1 initiation site.  cdc39-2 causes               strains shifting restrictive temperature. Total RNA samples <br /> stronger activators produce + 1 transcripts, unlike                 (25 /%g) hybridized completion excess indicated <br />                                                                             probes digested SI nuclease. Hybridization reactions situation wild-type strains + 1 transcription               carried parallel, products analyzed unaffected GCN4.  suppression cdc39-2 appears                    gel;  autoradiographic exposure times detecting require acidic activation region, specific             various transcripts equivalent. strains deleted particular GCN4 proteins.                                                   chromosomal gcn4 locus. <br />  <br /> Effect cdc39-2 transcription genes                            stimulated different activator proteins, lacZ reporter small effects cdc39-2 his3 transcription                      plasmids control GAL] PH05 promoter <br /> unlikely account fully significant differences              region introduced mutant wild-type strains resistance.  resistance determined                  (Table II). Transcriptional induction GAL] promoter <br />                                                                             galactose medium, mediated GAL4 his3 enzyme levels flux                     activator, increased -3-fold cdc39-2 strain. histidine biosynthetic pathway ATRJ (Kanazawa <br /> et al., 1988), suggesting cdc39-2 increase <br />                                                                             glycerol/ethanol medium, cdc39-2 appears minimal <br />                                                                             effect low activity GAL] promoter <br /> transcription GCN4-activated genes. examined RNA levels genes cdc39-2 wild-                   makes determination difficult. contrast, PH05 <br /> type strains containing control gcn4-C 163 expressing                transcription unaffected cdc39-2 normal <br /> plasmid (Figure 6). presence gcn4-C 163, GCN4-                    conditions medium depleted inorganic phosphate, <br /> regulated genes (HIS3, TRP3 probably HIS4)                         results activation PH02 PHO4. Taken <br /> increased transcription cdc39-2 background.                        results suggest cdc39-2 does affect <br /> contrast, cdc39-2 does affect basal level                                                                             basal transcription, increase transcriptional <br /> genes, does affect transcription unrelated genes <br />                                                                             activation  genes. DED], ribosomal RNA tryptophan tRNA.                      Transcription cdc39-2 strains shifted results suggest cdc39-2 increases activation                        restrictive temperature <br /> gcn4-C163 does affect basal transcription.                          results presented far transcriptional <br />   determine cdc39-2 increases transcription                 alterations growing cdc39-2 cells, conditions  <br /> 182 <br />                                                                                                          Role CDC39 transcription regulation <br />  <br />  <br /> Table II. Effect cdc39-2 expression GAL] -lacZ                       wild-type strain, presumably cell growth <br /> PHO5-lacZa                                                                        inhibited. results confirm preferential effects <br />                                                                                   initiation HIS3 +1 site, demonstrate <br />                                  GAL] - lacZP                   PH05-lacZc        CDC39 negatively regulates basal transcription HIS3 <br />                                                                                   genes.  appears GCN4-activated <br /> Genotype            Glu          <span id='am-23' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-24' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>Gly</span> +Eth Gly +Eth+Gal          +Pi         -P.   transcription sensitive CDC39 function basal <br />                                                                                   transcription preferentially affected CDC39               &lt;0.01        0.1             80             8       95        cdc39-2 cells grown permissive temperature. <br /> cdc39-2             &lt;0.01        0.15           270             5       97 <br />  3-galactosidase   activities (Miller units). <br />                                                                                   Effect cdc39-2 various his3 promoter <br /> bCells grown 2% indicated carbon sources (glucose, glycerol,            derivatives <br /> ethanol, galactose).                                                              determine element(s) HIS3 promoter region <br /> CCells grown phosphate-depleted medium presence                  responsible negative regulation CDC39, assayed <br /> absence inorganic phosphate.                                                   resistance  shown) basal HIS3 transcription                                                                                   cdc39-2 wild-type strains containing his3 <br />              dA-d T         + <br />                                                                                   promoter alleles (Figure 8). cases, resistance              GCN4           -      -                                              HIS3 transcription higher cdc39-2 strain,              TC                                                                   magnitude increase varies depending allele. <br />              TR                                                                   Deletion poly(dA) * poly(dT) sequence GCN4 <br />                           :; <br />                            1. .,           .;                                     binding site does significantly reduce magnitude      -                                                    . <br />                                                             &apos; <br />                                                                                   cdc39-2 effect, indicating HIS3 promoter sequences <br />                                                                                   upstream TATA region required negative <br />          1 <br />                 o <br />                                                                                   regulation CDC39. cdc39-2 effect appears somewhat <br />      -   j <br />  <br />  <br />  <br />  <br />                                                 : <br />                                                                                   reduced strains containing deletion Tc                                                                                   point mutation TR (TGTAAA); expected, increase <br />   DEDI                      j@                                                    + 1 transcription depends Tc increase                                                                                   + 13 transcription depends TR. suggests                           HIS3     A90      A84       \74   93        205 <br />                                                                                   CDC39 does affect inherent properties + 1 <br />                                                                                   + 13 initiator elements, negatively regulates <br /> Fig. 8. Effect loss CDC39 function transcription various            transcription mediated Tc TR.  HIS3 promoters. Wild-type cdc39-2 strains (deleted GCN4) <br /> containing indicated his3 alleles shifted 2 h                 wild-type HIS3 promoter, loss CDC39 function leads restrictive temperature, total RNAs hybridized completion             preferential increase + 1 transcription. suggests excess HIS3 DEDI probes digested SI                       HIS3 TATA elements present, CDC39 affects <br /> nuclease. described previously (Struhl Hill, 1987; Harbury             Tc-dependent transcription strongly TR- <br /> Struhl, 1989; Mahadevan Struhl, 1990), his3-A90 deletes                   dependent transcription. <br /> GCN4 binding site (-109 -80), his3-A84 deletes sequences <br /> upstream Tc (-447 -94), his3 74 retains known <br /> promoter elements lacks region -447 -156,                     CDC39 nuclear protein <br /> his3-A93 deletes Tc (-83 -46) his3-205 point mutation                determine cellular localization CDC39 protein, (TGTAAA) TR.                                                                   expressed CDC39, N-terminal tag                                                                                    flu epitope), constitutive DED] promoter. <br /> CDC39 function completely eliminated.                             Expression protein complemented investigated role wild-type CDC39                                 temperature-sensitive resistant phenotypes definitively analyzing transcription genes                    cdc39-2 mutation. Proteins total cellular, cytoplasmic <br /> shifting cdc39-2 cells restrictive temperature (38.5&#194;&#176;C).                   nuclear extracts electrophoretically separated distinguish direct effects CDC39 inactivation                          probed antibodies flu epitope (Figure 9). <br /> indirect effects growth arrest, analyzed strain                      protein migrating apparent molecular weight temperature-sensitive mutation CDC28, gene                             -200 kDa detected total nuclear extracts encoding cyclin-associated protein kinase mediates                       transformants containing flu-tagged CDC39  <br /> entry cell cycle. temperature-shift experiments                    indicating CDC39 nuclear protein. carried strains deleted GCN4, measure basal  e. non-GCN4 activated) transcription. <br />    strong increase HIS3 transcription, predominantly                         Discussion + 1 initiation site, observed cdc39-2 strain                   CDC39 transcriptional regulatory protein wild-type cdc28 strain (Figure 7). effect                  work, isolated temperature-sensitive mutation <br /> occurs 30 min 38.5&#194;&#176;C  1-2 h 37&#194;&#176;C)                          CDC39 selecting yeast strains increase HIS3 maintained 4 h  shown). Similarly,                        transcription GCN4 derivative defective <br /> basal transcription HIS4, TRP3 TFIID increased                          activation domain. CDC39 originally identified specifically temperature-shifted cdc39-2 strain.                        temperature-sensitive mutation causes cell-cycle arrest <br /> contrast, DEDJ transcript levels strains                         GI pachytene arrest meiosis, inappropriately <br /> indistinguishable time points indicating CDC39                        activates mating pheromone response (Reed, 1980; essential transcription RNA polymerase II.                      Shuster Byers, 1989; Barros Lopes et al., 1990). <br /> levels rRNA  shown) tryptophan tRNA similar                        phenotypes conferred original cdc39-1 cdc39 cdc28 strains; decreased relative                       mutation, including transcriptional induction pheromone <br />  <br />                                                                                                                                             183 <br />  M Collart K.Struhl <br />  <br />  <br />                                         <br />                                         Nuc <br />  <br />  <br />  <br />  <br />                                         .... <br />                                                =*.. <br />  <br />  <br />  <br />  <br />                                                ... <br />                                                      . _. . <br />                                                         +-     u <br />                                                                     B <br />  <br />                                                                       Nuc <br />  <br />                                                                           + <br />                                                                                Cyet   Uot <br />                                                                                       -   _    M <br />  <br />  <br />  <br />  <br />                                                                                                ,   - <br />                                                                                                          OOK <br />                                                                                                        -97.4K <br />                                                                                                        -69K <br />                                                                                                          46K <br />  <br />  <br />  <br />  <br /> Fig. 9. Subcellular localization CDC39. Protein (50 yg) total (Tot), nuclear (Nuc) cytoplasmic (Cyt) extracts prepared strains <br /> expressing CDC39 (+) (-) flu epitope tag analyzed SDS-PAGE assayed   protein Ponceau staining (B) epitope-tagged CDC39 Western blotting using specific antibodies epitope. positions corresponding molecular weight markers tagged CDC39 protein (arrow) indicated. <br />  <br />  <br /> responsive genes (FUS], GPAI, STE4), interpreted                      tryptophan tRNA RNA polymerase III rRNA occurring consequence activating pheromone                        RNA polymerase  shown) significantly affected. <br /> response  Barros Lopes et al., 1990). hypothesized                     permissive temperature, cdc39-2 increases GCN4- CDC39, concert genes (CDC36, CDC72,                        activated, basal, transcription HIS3, HIS4 CDC73 SRMJ) mutated confer                             TRP3.  cdc39-2 affects activation GAL4 phenotypes (Reed et al., 1988; Clark Sprague, 1989),                      GCN4 derivatives different portions acidic regulate activity G proteins mating                       activation region;  does increase <br /> pheromone response signaling pathways  Barros                    PH02/PH04-mediated activation PHO5. Lopes et al., 1990).                                                          conditions, likely cdc39-2 protein retains <br />    lines evidence presented strongly argue                    sufficient function repress basal transcription CDC39 primarily transcriptional regulatory protein.                 activation GCN4 derivatives GAL4. regard,  cdc39-2 mutation described increases                          Sc3861, fragment lacks sequences upstream transcription unrelated genes involved                   putative TATA element presumably expresses pheromone response pathway.                               CDC39 protein, permits growth high temperature does <br /> quantitative effects individual genes modest,                negatively regulate GCN4-activated transcription <br /> phenotypic consequences, depend activity                      (assayed resistance). exclude genes, marked. Second, cdc39-2 strains arrest                        possibility repression basal activated transcription <br /> asynchronously cell cycle, transcriptional                   involves distinct functions, simplest interpretation effects (assayed resistance) occur                        results negative regulation GCN4  GALA) <br /> pheromone response pathway inactivated. regard,                    activation requires common CDC39 function previously noted CDC39 function essential                 negative regulation basal transcription. pheromone response pathway inactivated                            Yeast cells contain number negative regulatory <br /> mutation diploids  Barros Lopes et al., 1990).                   proteins affect transcription genes  loss CDC39 function affects Tc-dependent                            HMG1-like protein SINI (Kruger Herskowitz, 1991), <br /> transcription strongly TR-dependent transcription                   SIN3 (Wang et al., 1990; Vidal et al., 1991), putative wild-type HIS3 gene, leading preferential                    helicase MOTI (Davis et al., 1992) SSN6-TUP1 <br /> increase initiation +1 site. Fourth, CDC39                        complex (Schultz et al., 1990; Williams Trumbly, 1990; <br /> encodes nuclear protein. reasons, suggest                      Williams et al., 1991). proteins function cell-cycle pheromone response phenotypes                         distinct molecular mechanisms. suggested caused original cdc39-1 mutation arise indirectly                   SINI functions C-terminal tail largest consequence increased transcription genes responsible                 subunit RNA polymerase II (Peterson et al., 1991) functions.                                                          SSN6-TUP1 complex functions directly <br />                                                                               interacting specific DNA binding proteins (Keleher <br /> CDC39 negatively regulates basal activated                                et al., 1992). CDC39 functionally related MOTI <br /> transcription genes                                                   proteins essential cell growth basic observation cdc39-2, recessive                     constitutively activate pheromone response pathway. <br /> presumably loss function mutation, causes increased RNA                    Given similarities GI-arrest mating response <br /> levels formally defines CDC39 transcriptional inhibitory                 phenotypes mutant strains, possible CDC36, <br /> protein. restrictive temperature, cdc39-2 increases                CDC72, CDC73 SRM1 (Reed et al., 1988; Clark basal  e. non-GCN4-activated) transcription HIS3, HIS4,                  Sprague, 1989; Barros Lopes et al., 1990) TRP3 TFIID. presume, tested,                         negative regulators transcription. <br /> GCN4-activated transcription HIS3, HIS4, TRP3 increased absence CDC39 function.                           Potential molecular mechanisms <br /> DED] transcription appears unaffected, suggesting                        general classes model explain negative regulation CDC39 universal genes                    CDC39 negatively regulates basal acidic activator- <br /> transcribed RNA polymerase II. addition, synthesis                      mediated transcription unrelated genes. 184 <br />                                                                                                Role CDC39 transcription regulation <br />  <br /> model, CDC39 involved maintaining normal                 1992).  CDC39 affect general <br /> chromatin structure highly repressive transcription.   transcription factors (e.g. TFIIA, TFIIB, RNA polymerase  mutations reduce eliminate function         II) influence chromatin structure manner subtle <br /> histones result increased transcription genes           nucleosome disruption. <br /> (Clark-Adams et al., 1988; Han Grunstein, 1988; Han <br /> et al., 1988; Durrin et al., 1992). alternative model, <br /> CDC39 inhibits activity general component RNA polymerase II transcription machinery TFIID,           Materials methods <br /> TFIIB, TFIIA enzyme itself. inhibition           Yeast strains mediated direct interaction               starting strain used isolate suppressor mutations KY1600, <br /> component(s) indirectly intermediary proteins.          derivative KY803  ura3-52 trpl 1leu2-PET56 gal2 gcn4 1; Hope regard, human Dr, NC 1 NC2 factors interact        Struhl, 1986) carries YCp88-Sc4363, URA3 centromeric vector <br />                                                                    expresses gcn4-C163 derivative constitutive DEDI promoter TFIID repress basal activated transcription        (Hope Struhl, 1986), YCp87-Sc3866, LEU2 centromeric vector <br /> vitro (Meisterernst Roeder, 1991; Inostroza et al.,            containing his3-lacZ fusion protein control promoter <br /> 1992). classes explanation              containing optimal AP-1 site fused upstream HIS3 TATA region <br /> compatible non-equivalent effects CDC39                 (position -86). KY 1603, initial strain containing cdc39-2, generated <br />                                                                    spontaneous revertant KY 1600 grew glucose minimal plates <br /> different genes distinction basal          containing 20 mM aminotriazole   blue X-gal indicator plates. <br /> activated transcription observed permissive                 test effect cdc39-2 various GCN4 proteins, plasmids temperature.                                                       cured KY1603 generate strain KY1617 transformed <br />    striking mechanistically informative               YCp88 plasmids expressing various GCN4 derivatives (Hope observation CDC39 preferential repression          Struhl, 1986; Hope et al., 1988). determine phenotype loss                                                                    CDC39 function, cdc39:: URA3 substitution allele (Sc3865) Tc-mediated transcription wild-type HIS3                  introduced step gene replacement (Rothstein, 1983) KY805, <br /> promoter. stronger increase + 1 + 13                   ci diploid strain generated HO-mediated diploidization KY803 <br /> transcription novel, having observed          (Herskowitz Jensen, 1991). eliminate mating pheromone response <br /> wide variety his3 promoter derivatives analyzed        pathway, ste4:: URA3 (Whiteway et al., 1989) stel2:: URA3 (Errede past decade. CDC39 effect clearly distinct             Ammerer, 1989) substitution alleles introduced KY803                                                                    KY 1617 step gene replacement. resulting strains transcriptional induction acidic activator proteins       transformed pRS314-Sc4363  TRPI centromeric vector expressing mediated exclusively TR results           gcn4-C 163 DED1 promoter). Isogenic derivatives KY1600 preferential initiation + 13 (Struhl, 1986); loss       KY 1603 containing various his3 promoter alleles obtained CDC39 function lead increased transcription           successive gene replacement events (Struhl, 1987). unmasking potential acidic activator proteins.           DNA manipulations <br /> Consistent view, GCN4 derivatives                   YCp87-Sc3866 generated substituting EcoRI-HindIlI fragment <br /> increasingly strong activation regions stimulate increasingly      Sc4099 (Hill et al., 1986) corresponding region Sc3802 (Singer transcription + 13, cdc39-2 strains         et al., 1990) YCp87 vector  LEU2 derivative YCp86 constructed <br /> GCN4-dependent activation + 1 site. Tc           Joan Sellers). pRS314-Sc4363 contains SmaI-EcoRI fragment                                                                    YCp88-Sc4363 (Hope Struhl, 1986) cloned pRS314 (Sikorski does normally respond transcriptional activators            Hieter, 1989). <br /> (Struhl, 1986; Ponticelli Struhl, 1990), present results      CDC39 cloned complementation cdc39-2 allele. Specifically, <br /> suggest reflect inherent defect Tc       YCp5O library containing partial Sau3A fragments yeast genomic DNA inhibition proteins CDC39.                   (obtained Mark Rose) introduced KY1617 containing <br />    considering classes model CDC39               pRS314-Sc4363 screened sensitive colonies. plasmid <br />                                                                    containing 20 kb yeast DNA (Sc3860) recovered function, disfavor view CDC39 involved           transformant shown complement phenotypes conferred maintaining chromatin normally repressive state.            suppressor mutation. YCp50-Sc3861 generated religating largest TR Tc support comparable levels basal              EcoRI fragment YCp5O-Sc3860. YCp5O-Sc3862 obtained partially <br /> HIS3 TFIID-dependent transcription vivo (Cormack            digesting YCp5O-Sc3860 EcoRI digestion religating. Sc3863                                                                    generated ligating partial EcoRI-SacI fragment YCp5O-Sc3862 Struhl, 1992), TR efficiently binds TFHID supports        pRS316 (Sikorski Hieter, 1989). Sc3864 obtained ligating TFIID-dependent transcription vitro (Ponticelli Struhl,     SacI-BglII fragment Sc3861 Sall-SacI fragment Sc3863 1990).  loss CDC39 activity alleviated nucleosome       pRS316. sequencing CDC39, deleted derivatives pRS316-Sc3864 <br />                                                                    generated subcloning restriction fragments digestion <br /> repression, expect observe transcriptional            exonuclease III S1 nuclease. Sc3865 obtained replacing <br /> patterns resembling obtained naked DNA templates          HindlIl fragments Sc3862 1.1 kb HindIII URA3 fragment. vitro.  nucleosome loss vivo, obtained                construct plasmid expressing flu-tagged CDC39 derivative, <br /> repressing histone H4 mRNA synthesis, leads increased           GCCATCGATAATGCTATCGGCCACACATACCG CGGGGTAC- <br /> HIS3 transcription exclusively mediated             CTTCTGCATTTAATAAGTCAAAGC used primers amplify <br />                                                                    CDC39 sequences ATG initiation codon proximal KpnI TR (Durrin et al., 1992). contrast, CDC39               site. amplified DNA digested ClaI KpnI cloned preferentially mediates repressive effects Tc.         pBS29 (constructed Alice Vincent), molecule containing NdeI site favor idea CDC39 directly indirectly      upstream flu epitope. NdeI blunted KpnI fragment inhibits component(s) general transcription               resulting molecule KpnI-SacI fragment clone Sc3863                                                                    cloned HindIII blunted SacI cleaved YCp92, derivative machinery. TFIID attractive candidate             YCp88-GCN4 (Hope Struhl, 1986) constructed Brendan Cormack <br /> component functional distinctions           coding region replaced oligonucleotide containing <br /> TR Tc (Struhl, 1986; Mahadevan Struhl, 1990;               Sacl site stop codons frames. SacI-EcoRI fragment <br /> Ponticelli Struhl, 1990) numerous           containing 3&apos; region CDC39 substituted 3&apos; region <br />                                                                    GCN4 complete gene encoding desired protein. control <br /> proteins TFIID physically interacts (Dynlacht           construct flu tag N-terminus cloning EcoRT <br /> et al., 1991; Lee et al., 1991; Lieberman Berk, 1991;          blunted Sacl fragment Sc3861 SacI-EcoRI fragment Meisterernst Roeder, 1991; Timmers Sharp, 1991;            CDC39 derivative containing flu tag HindUI blunted EcoRI cleaved <br /> Comai et al., 1992; Eisenmann et al., 1992; Inostroza et al.,      YCp92. <br />                                                                                                                                           185 <br />  M Collart K.Struhl <br />  <br /> Phenotypic analyses                                                                 References <br /> GCN4 function assayed ability cells grow glucose minimal <br /> medium containing various concentrations described previously             Chen,W. Struhl,K. (1988) Proc. Natl. Acad. Sci. USA, 85, 2691-2695. <br /> (Hope Struhl, 1986; Hope et al., 1988). degree resistance            Chen,W., Tabor,S. Struhl,K. (1987) Cell, 50, 1047-1055. related ability GCN4 activate transcription HIS3          Clark,K.L. Sprague,G.F.J. (1989) Mol. Cell. Biol., 9, 2682-2694. <br /> encodes enzyme competitively inhibited , histidine         Clark-Adams,C.D., Norris,D., Osley,M , Fassler,J.S. Winston,F. <br /> biosynthetic genes  affect level substrate utilized           (1988) Genes Dev., 2, 150-159. <br /> HIS3 enzyme), ATRJ  encodes membrane associated protein                Comai,L., Tanese,N. Tjian,R. (1992) Cell, 68, 965-976. lowers intracellular levels) (Kanazawa et al., 1988). strains          Cormack,B.P. Struhl,K. (1992) Cell, 69, 685-696. <br /> containing particular GCN4 derivative, degree resistance             Davis,J.L., Kunisawa,R. Thorner,J. (1992) Mol. Cell. Biol., 12, directly related HIS3 RNA levels (Hill et al., 1986; Chen Struhl,           1879-1892. <br /> 1988). assay GAL] expression, strains transformed 131 (Ma             Barros Lopes,M., Ho,J.-Y. Reed,S  (1990) Mol. Cell. Biol., 10, Ptashne, 1987), centromeric plasmid containing GAL] -lacZ fusion              2966-2972. grown supplemented minimal medium containing 2 % glucose 2 %             Dolan,J.W., Kirkman,C. Fields,S. (1989) Proc. Natl. Acad. Sci. USA, glycerol ethanol + 2 % galactose. PH05 expression assayed                86, 5703-5707. transforming cells pMH313 (Han Grunstein, 1988), high copy           Durrin,L.K., Mann,R.K. Grunstein,M. (1992) Mol. Cell. Biol., 12, <br /> PH05-lacZ fusion plasmid, growing resulting strains                        1621-1629. <br /> supplemented glucose minimal medium depleted               Dynlacht,B.D., Hoey,T. Tjian,R. (1991) Cell, 66, 563-576. <br /> inorganic phosphate.                                                               Eisenmann,D.M., Arndt,K.M., Ricupero,S.L., Rooney,J.W.                                                                                      Winston,F. (1992) Genes Dev., 6, 1319-1331. <br />                                                                                    Errede,B. Ammerer,G. (1989) Genes Dev., 3, 1349-1361. <br /> RNA analysis                                                                       Han,M. Grunstein,M. (1988) Cell, 55, 1137-1145. experiments carried permissive growth conditions, strains            Han,M., Kim,U.J., Kayne,P. Grunstein,M. (1988) EMBO J., 7, grown overnight glucose minimal medium containing 1 % casamino               2221 -2228. <br /> acids selective plasmid maintenance, diluted A6 = 0.2 YPD,           Harbury,P B. Struhl,K. (1989) Mol. Cell. Biol., 9, 5298-5304. grown 4 h. Temperature-shift experiments performed                 Herskowitz  Jensen,R.E. (1991) Methods Enzymol., 194, 132-146. <br /> described previously (Cormack Struhl, 1992). CeHls grown YPD           Hill,D.E., Hope , Macke,J.P. Struhl,K. (1986) Science, 234, 30&#194;&#176;C A6W = 0.3, shifted 37&#194;&#176;C 20 min, returned 30&#194;&#176;C                451 -457. <br /> 1 h, shifted restrictive temperature (typically 38.5&#194;&#176;C).          Hope  Struhl,K. (1986) Cell, 46, 885-894. <br /> Aliquots (20 ml) taken shift, second shift,       Hope , Mahadevan,S. Struhl,K. (1988) Nature, 333, 635-640. various times second shift, cells quickly pelleted   Inostroza,J , Mermelstein,F.H., Ha , Lane,W.S. Reinberg,D. frozen -80&#194;&#176;C. protocol avoids looking consequences                (1992) Cel <br /> </body></html>