<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMB <br />    Reports <br />  <br />  <br />  <br /> Partial AUC maximization essential gene prediction using <br /> genetic algorithms <br />                                                                   <span id='am-2' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; Sangsoo Kim * <br />                      1                     1                1,&#226;&#8364;&#160;                2, <br /> Kyu-Baek Hwang , Beom-Yong Ha , Sanghun Ju <br /> 1 <br />  School Science Engineering, Soongsil University, 2School Systems Biomedical Science, Soongsil University, Seoul <br /> 156-743, Korea <br />  <br />  <br />  <br /> Identifying genes indispensable organism&#226;&#8364;&#732;s life                 screening. example, computational prediction essential <br /> characteristics central questions current                       genes pathogenic microbes implication prioritizing <br /> biological research, helpful develop                    antimicrobial drug targets (2). addition candidate list, <br /> computational approaches prediction essential                     computational predictions identify sequence features genes. performance predictor usually measured                     related properties relevant essentiality. area receiver operating characteristic curve                       &#227;&#8364;&#8364;Computationally, predicting essential genes amounts (AUC). propose novel method implementing genetic                         problem classifying genes binary classes essen- <br /> algorithms maximize partial AUC restricted                   tial nonessential ones based number features specific interval lower false positive rate (FPR), region                 calculated gene. features relevant follow experimental validation. predictor                     relevance essentiality. Previous studies used various <br /> uses various features based sequence information, protein-                    types features, comparative genomic properties, se- <br /> protein interaction network topology, gene expression                        quence-based features, functional genomic properties. profiles. feature selection wrapper developed                           established essential genes conserved alleviate fitting problem weigh feature&#226;&#8364;&#8482;s                   nonessential ones (3). Phyletic retention, number organ- <br /> relevance prediction. evaluated method using                       isms ortholog conserved, used widely <br /> proteome budding yeast. implementation genetic                         prediction essential genes (4). comparative ge- <br /> algorithms maximizing partial AUC 0.05 0.10                      nomic features, number sequence-based features FPR outperformed popular classification methods. [BMB                      directly calculated sequenced genome em- <br /> Reports 2013; 46(1): 41-46]                                                      ployed predicting essential genes (4, 5). newly se- <br />                                                                                  quenced genome, sequence-based features typi- <br />                                                                                  cally available,  essentiality prediction solely based INTRODUCTION                                                                     features important applications. <br />                                                                                  &#227;&#8364;&#8364;Proliferation high-throughput functional genomics data <br /> Identifying genes indispensable organism&#226;&#8364;&#732;s life,               provided opportunities explore properties relation identifying characteristics genes, cen-               essentiality. example, protein-protein interaction network <br /> tral questions biology. essential genes number                 crucial biological processes. Essential genes tend ganisms identified catalogued ge-                          play special roles, like hubs network,  topo- <br /> nome-wide knock experiments (1).  identification                     logical properties used predict gene essentiality essential genes wet experiments time-consuming                     (6-8). fact, Hwang et al. combined protein-protein inter- labor-intensive.  helpful develop com-                  action network properties sequence features predict- <br /> putational approaches essential gene prediction candidate                 ing essential genes, demonstrated essentiality pre- <br />                                                                                  diction improved adding network properties (9). Gene ex- <br />                                                                                  pression fundamental process tightly regulated sys- <br /> *Corresponding author. Tel: +82-2-820-0457; FAX: +82-2-820-0816, <br />                                                                                  tem level, related gene essentiality. shown <br /> E-mail: sskimb@ssu.ac.kr <br /> &#226;&#8364;&#160; <br />  Present address: Platform R&amp;D Team, Technical Group, Daum                       gene expression variation related gene essentiality (7, <br /> Commnucations, Seoul 140-894, Korea.                                             10, 11). <br /> http://dx.doi.org/10.5483/BMBRep.2013.46.1.159                                    integrated features, based sequence information <br />                                                                                  protein-protein interaction network topology, Received 2 August 2012, Revised 8 August 2012,                                   used methods. addition, included gene ex- <br /> Accepted 8 August 2012                                                           pression properties transcriptional activity variation, <br /> Keywords: AUC, Classification, Essential genes, Genetic algo- <br />                                                                                  phyletic retention. inclusion fea- <br /> rithms, Partial AUC                                                              tures improve prediction power, irrelevant  <br />  <br /> ISSN: 1976-6696 (print edition) / 1976-670X (electronic edition) <br /> Copyright &#226;&#8220;&#8217; 2013 Korean Society Biochemistry Molecular Biology <br />    open-access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/li- <br /> censes nc/3.0) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />     PAUC maximization using genetic algorithms&#226;&#8364;&#402; <br />    Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br />    distracting features confuse machine learning         ture selection wrapper training set fold.    cause fitting. Pruning features improve         fold, feature selection procedure repeated 50 times  <br />    performance predictor, help assess rela-       duce variability  MATERIALS METHODS). doing  <br />    tive importance feature essentiality prediction.        identify features consistently selected. <br />     various techniques used. example, corre-       features relevant    lation coefficient feature essentiality, n&#195;&#164;ive    essentiality. Table 1 summarizes number times fea- <br />    Bayes technique used assess relative importance         ture selected fold. <br />    feature essentiality predictor (4, 12). Wilcoxon rank        31 features, following selected <br />    sum tests used evaluate statistical difference essen-   50 times fold: nucleus, phyletic retention, essentiality <br />    tial nonessential genes feature (9). receiver op-      index (EI), clique level (KL), variance gene expression. <br />    erating characteristic (ROC) curves evaluated         findings highly important features essentiality <br />    dividual features assess performances (9). employed        concordant previous studies follows.  essential <br />    rigorous robust method avoids fitting.            genes tend localized nucleus, essential bio- <br />    Using backward greedy search elimination, features se-            logical processes information storage processing <br />    lected based ROC evaluation, followed 10-fold                place (4, 16). Second, essential genes known    cross-validation.                                                      evolutionarily-conserved nonessential genes bacteria <br />     area ROC curve  k , AUC) common                (3) eukaryotes (17). regard, Gustafson et <br />    formance measure evaluate classification method.             al. (5) showed phyletic retention, surrogate evolu- <br />    applied binary classification problem, conventional         tionary conservation, highly correlated gene essentiality <br />    AUC maximization concerns range true pos-            yeast E. coli.  essential genes likely  <br />    itive rate false positive rate (FPR). practical purpose    teract essential genes nonessential genes. ex- <br />    prioritizing candidates undergo experimental vali-       ample, Hwang et al. (9) showed essential genes higher <br />    dation, important produce short list depleted    values essentiality index  proportion essential genes <br />    false positives. Seringhaus et al. (4) approached problem      interacting partners) nonessential genes. Fourth, <br />    assigning relatively higher costs false positives false     clique level  size largest cliques protein belongs <br />    negatives.  propose partial AUC maximization fo-          protein-protein interaction network) related  <br />    cuses scoring candidates, entire       nectedness gene, known related gene es- <br />    set. Partial AUC used medical diagnostic screening,        sentiality (18). outperformed network-based features    high true-positive rate fixed lower FPR pref-        Hwang et al.&#226;&#8364;&#8482;s study (9). Finally, variance gene expression    erable (13). developed method maximized partial          known negatively correlated gene essentiality    AUC directly given FPR 0.05 0.1 using genetic      evolutionary conservation rate eukaryotes, including yeast (7, <br />    algorithm. heuristic search techniques      10). <br />    inspired evolutionary biology, used wide-          transcriptional variability, transcriptional activity    ly solve complicated problems various fields. evaluated       gene expression level observed related es- <br />    method using proteome Saccharomyces cerevisiae,             sentiality, highly expressed genes evolve slowly bac- <br />    comprehensive set experimentally validated es-          teria mammals (10). study, mean gene expression    sential genes available. addition, high throughput pro-          selected essentiality features time <br />    tein-protein interaction data gene expression profiles     (&#239;&#188;&#382;90% average). sequence-based features, e., <br />    model organism readily available DIP (14) GEO             A3s, Gravy, CLOSE STOP RATIO, chosen    (15), respectively. comparisons machine              Gravy, general average hydropathicity,    learning algorithms.                                                   TM_HELIX, number transmembrane helices,                                                                           shown negatively correlated essentiality (4).    RESULTS                                                                correlated hydropathy im- <br />                                                                           portant factors contributing prediction transmembrane <br />    compiled total 31 features 5825 S. cerevisiae genes        proteins. transmembrane proteins function transporters <br />    various sources. removing genes missing val-       participate metabolic nonessential roles. yeast, high- <br />    ues, final dataset comprised 3979 genes (including 940 es-      ly expressed  likely essential nonessential) genes pre- <br />    sential genes). fold cross validation used dataset     fer pyrimidine bases (C T) synonymous codon po- <br />    performance comparison.                                            sition prefer codons T richness,                                                                           characteristics yeast genome (38% G+C content) (19). <br />    Features relevant gene essentiality                                 fact A3s, frequency synonymous po- <br />    Pruning irrelevant distracting features important step   sition codons, relatively frequently chosen feature <br />    machine learning  maintaining highly-relevant           selection experiments high frequency    features, simple fast model built. applied fea-      nonessential genes. essential genes yeast prefer pyr- <br />  <br />  <br /> 42 BMB Reports                                                                                                             http://bmbreports.org <br />                                                                                                             PAUC maximization using genetic algorithms <br />                                                                                                                               Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br /> imidine bases synonymous codon position,                   G3s showed negative correlation (C.C. = &#226;&#710;&#8217;0.78) expect C3s T3s, frequencies C T,               fold feature selection experiments (Table 1). <br /> spectively, synonymous codon position, similar <br /> prediction powers.  number times C3s se-                  Prediction performance gene essentiality <br /> lected showed high correlation T3s (C.C. = 0.88),            Genetic algorithms different objective functions:  <br />  <br /> Table 1. number timesa feature selected fold <br />  <br />                                                                             Fold number <br />                                 Features                                                                                              Average <br />                                                         1    2    3    4     5       6        7       8           9       10 <br />                             1 <br />        Cytosol                                          17   20   23   3     23      6        7       13         14        5            13.1 <br />                                         1 <br />        Extracellular                                    0    2    0    0     0       0        1       0           0        0             0.3 <br />                                                 1 <br />        Plasma membrane                                  2    5    6    1     14      4       21       3          16       12             8.4 <br />                                             1 <br />        Mitochondria                                     0    0    0    0     0       1        0       0           0        0             0.1 <br />                             1 <br />        Nucleus                                          50   50   50   50    50      50      50       50         50       50             50 <br />                                     2 <br />        TM_HELIX                                         44   44   43   16    47      43      38       45         50       18            37.8 <br />                 3 <br />        T3s                                              38   21   23   28    38      10       7       19         16       25            22.5 <br />        C3s3                                             29   14   14   24    23      14       7       13         15       23            17.6 <br />                 3 <br />        A3s                                              50   50   50   50    50      50      50       50         50       49            49.9 <br />                  3 <br />        G3s                                              19   29   32   22    21      49      43       41         33       49            33.8 <br />        CAI3                                             2    0    1    6     1       0        3       2           1        2             1.8 <br />                 3 <br />        CBI                                              34   10   24   27    43      19      38       22         18       30            26.5 <br />                 3 <br />        Fop                                              12   42   18   22    10      32      11       25         29       19             22 <br />              3 <br />        Nc                                               15   0    5    1     8       2        7       3           1        1             4.3 <br />                         3 <br />        GC3s                                             39   24   19   27    36      12       8       16         17       18            21.6 <br />              3 <br />        GC                                               2    2    17   0     11      2        2       10          7        9             6.2 <br />                         3 <br />        L_sym                                            4    6    16   9     2       9        4       9          17        7             8.3 <br />                     3 <br />        L_aa                                             4    6    15   9     2       9        4       9          17        6             8.1 <br />                         3 <br />        Gravy                                            50   49   49   49    50      50      48       49         48       47            48.9 <br />        Aromo3                                           28   3    7    0     6       34       5       16          8        2            10.9 <br />                                                     4 <br />        RAREAMINO_ACID                                   9    4    2    0     6       6        0       5           2        0             3.4 <br />                                                     4 <br />        CLOSE STOP RAIO                                  46   44   46   50    50      48      50       45         46       47            47.2 <br />        Phyletic retention5                              50   50   50   50    50      50      50       50         50       50             50 <br />                                 6 <br />        DegreeK                                          2    5    0    6     3       7        2       29          8        1             6.3 <br />                     6 <br />        CCo                                              9    0    0    0     0       0        0       3           3        2             1.7 <br />              6 <br />        BC                                               5    4    3    43    4       3        0       5           0        2             6.9 <br />              6 <br />        CC                                               33   26   34   38    25      37      16       50         41       25            32.5 <br />             6 <br />        KL                                               50   50   50   50    50      50      50       50         50       50             50 <br />          6 <br />        EI                                               50   50   50   50    50      50      50       50         50       50             50 <br />                         7 <br />        Mean                                             49   48   50   47    45      41      47       50         49       47            47.3 <br />                                 7 <br />        Variance                                         50   50   50   50    50      50      50       50         50       50             50  number times feature selected. colored background highlights cells selected time (dark blue),                                                                1                                                              2 90% (puple), 80% (cyan) 50 trials fold. subcellular localization probabilities calculated ConLoc. Number trans- <br /> membrane helices calculated TMHMM. 3Condon usage freqeuncies calculated CondonW. 4Ratios derived results CondonW. <br /> 5                                        6                                             7 <br />  Number organisms having ortholog. Protein-protein interaction network features. Gene expression features. <br />  <br /> http://bmbreports.org                                                                                                                   BMB Reports     43 <br />     PAUC maximization using genetic algorithms&#226;&#8364;&#402; <br />    Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br />    fold, applied criteria feature selection: filtering   genetic algorithm-based methods generally useful    (Entire), &#226;&#8240;&#165;80% (FS_80), &#226;&#8240;&#165;90% (FS_90) 50 trials         classification methods task performance <br />    Table 1 MATERIALS METHODS). selected fea-            measure partial AUC. methods ob- <br />    ture subset, linear discriminant function trained ge-      jective functions. Multilayer perceptrons usually trained    netic algorithm. objective functions genetic algorithm,     minimizing squared error maximizing cross entropy <br />    employed performance measures suitable task         function. Logistic regression performed maximizing    essential gene prediction, e., partial AUC 5% FPR (GA_           likelihood. Support vector machines trained maximizing <br />    PAUC_0.05) partial AUC 10% FPR (GA_PAUC_0.10)               distance decision boundary closest data <br />    AUC (GA_ AUC). Supplementary Fig. 1 shows perform-            point. objective functions calculated train- <br />    ance comparisons different objective functions fea-         ing data examples.  partial AUC calculated    ture selection criteria. Supplementary Figs. 1A 1B compare        subset data examples, e., examples classified positive <br />    performance, measured partial AUC 5% 10% FPR,           specific value FPR.  optimal classifier  <br />    respectively. Regardless feature selection criteria,           spect objective functions achieve suboptimal partial <br />    GA_PAUC_ 0.05 GA_PAUC_0.10 performed significantly                AUC values. contrary, methods trained di- <br />    better GA_AUC terms partial AUC 5% 10%              rectly maximizing performance measure, e., partial AUC <br />    FPR, respectively. Supplementary Fig. 1C shows GA_AUC            likely achieve better performance terms <br />    achieved higher AUC values GA_ PAUC_0.05                measure. <br />    GA_PAUC_0.10. results, conclude            &#227;&#8364;&#8364;Feature selection different impact classification <br />    objective function used genetic algorithms highly influen-     method. feature selection methods greatly improved    tial performance.  objective function     performance multilayer perceptrons, regardless  <br />    matched performance measure.                                   formance measure feature selection criteria (Supple- <br />     feature selection method improved prediction perform-        mentary Fig. 2).  feature selection did strong <br />    ance general, performance GA_PAUC_0.05 sig-         influence performance classification methods.    nificantly enhanced feature selection (Supplementary Figs. 1A      experiments, multilayer perceptrons severely overfitted    1C). GA_PAUC_0.10 showed performance enhancement               training dataset fold  Supplementary Tables 4, 5,    feature selection terms AUC (Supplementary Fig. 1C).            6). Overfitting reduced eliminating relevant <br />     performance measured partial AUC 10% FPR           features. explains feature selection improved  <br />    degraded feature selection (Supplementary Fig. 1B).        formance multilayer perceptrons, methods. <br />    performance GA_AUC improved feature selection <br />    measured partial AUC 10% FPR (Supplementary Fig.           DISCUSSION <br />    1B). measured partial AUC 5% FPR AUC,  <br />    formance did significant difference feature se-       ultimate goal developing computational methods    lection criteria (Supplementary Figs. 1A 1C). ob-         prediction essential genes predictability stud- <br />    served variance performance reduced feature        ied organisms based universal set features devel- <br />    selection (Entire vs. FS_80 FS_90; Supplementary Tables       oped studied model organisms. prediction    1, 2 3). reducing number features, feature se-      robust applicable distantly  <br />    lection method effectively enhances generalization perform-       lated organisms. Deng et al. investigated issue dis- <br />    ance genetic algorithm cases.                          tantly related bacteria (12). report, did address    &#227;&#8364;&#8364;Comparison classification methods: compared            transferability issue, focused core issues, feature se- <br />    genetic algorithm-based methods (GA_PAUC_0.05, GA_                lection classification algorithms. <br />    PAUC_0.10, GA_AUC) popular classification               newly sequenced genome, sequence-based fea- <br />    methods multilayer perceptrons (multiLayer), logistic      tures usually available.  advent    gression (logistic), support vector machines RBF (radial     generation sequencing technology, gene expression profil- <br />    basis function) polynomial kernels (smoRBF smoPoly,           ing using RNA-seq technology performed concomitantly <br />    respectively) (Supplementary Fig. 2). parameters        independent genome sequencing. Considering    methods set default Weka  MATERIALS                gene expression properties closely related evolu- <br />    METHODS). genetic algorithm-based methods outperfor-              tionary conservation essentiality, appropriate include <br />    med case, e., performance meas-      features prediction models. Unlike gene expression <br />    ured partial AUC 5% FPR using entire feature set,       profiles, high throughput protein-protein interaction data usu- <br />    GA_PAUC_0.05 achieved second highest perform-               ally require lengthy experiments. gene coexpression    ance (Supplementary Fig. 2A). Furthermore, methods                observed permanent, transient, protein com- <br />    showed lower variance performance  regardless          plex partners (20), gene coexpression network properties <br />    performance measure feature selection criteria         compensate protein-protein interaction network properties. <br />    Supplementary Tables 1, 2, 3).  conclude          power predict essential genes varies various <br />  <br />  <br /> 44 BMB Reports                                                                                                            http://bmbreports.org <br />                                                                                                          PAUC maximization using genetic algorithms <br />                                                                                                                            Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br /> types features. Unlike previous studies meas-        DEG (16) used calculating EI 4662 genes. <br /> ured feature&#226;&#8364;&#8482;s power independently, using statistical meas-         &#227;&#8364;&#8364;Gene expression features: downloaded gene expression <br /> ures, employed feature selection wrapper technique,               profiles yeast obtained Affymetrix Yeast Genome S98 evaluated feature based classification models           Array (9335 probes) GEO (http://www.ncbi.nlm.nih.gov/ <br /> volving features. measure relevance es-       geo/) (15). Total number samples 2015. raw CEL <br /> sentiality, used number times feature selected.          files, expression level probe calculated RMA majority consistently selected features           (21).  calculated mean variance expression study previously reported powerful pre-             5885 genes (10). multiple probes mapped dictors studies. possible rationalize se-   gene, used average values. <br /> lection terms biological knowledge.  feature se-           &#227;&#8364;&#8364;Phyletic retention features: obtained phyletic retention <br /> lection essential gene prediction essen-       data 4728 genes S. cerevisiae (5), phyletic <br /> tial steps machine learning strong biological          retention defined number organisms having foundation.                                                              ortholog. <br />  computational prediction biological effects            &#227;&#8364;&#8364;Final dataset: Finally, obtained dataset consisting shed insight features play major roles, results        3979 genes 31 features excluding genes having missing bigger impact experimental validations.          value. 940 essential genes. experiments, better suggest candidate list short          10-fold cross validation used. <br /> pleted false positives order minimize labor  <br /> tensive time-consuming wet experiments. cases,               Feature selection <br /> partial AUC, measured fixed lower FPR,          employed backward search-based wrapper feature se- <br /> important AUC. implemented genetic algorithms            lection (22). Feature selection wrappers known produce <br /> maximize directly partial AUC. classification            better results filter methods, information gain,  <br /> methods objective functions, genetic algorithms           cause consider classification models involved. AUC (Area <br /> allow flexibility formulating s objective                 ROC Curve) 3-fold cross validation adopted <br /> functions. knowledge, implementation         performance measure wrapper. classification <br /> genetic algorithms maximizing partial AUC biomedical               model wrapper, used logistic regression. applied <br /> applications. believe flexibility robustness         wrapper 50 times selected features based number <br /> stimulate wide acceptance.                                               times selected. According result                                                                          50 trials, different feature groups: filtering, <br /> MATERIALS METHODS                                                    80% selected, 90% selected. <br />  <br /> Data preparation                                                         Partial AUC maximization genetic algorithms <br /> Sequence-based features: downloaded 5885 ORF se-                      partial AUC (area ROC curve) sub-area quences S. cerevisiae ftp://genome-ftp.stanford.edu.             AUC measured range false positive rates  CodonW (http://codonw.sourceforge.net/) used calculate            Supplementary Fig. 1). order maximize partial AUC 14 features related codon usage  Supplementary Table          training dataset, used genetic algorithms. linear discrim- <br /> 7). ConLoc (http://sbi.postech.ac.kr/conloc/) used predict        inant function used scoring essential gene candi- subcellular localizations: cytosol, extracellular matrix, plas-      <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, c, follows. <br /> ma membrane, mitochondria, nucleus. probabilities locations estimated used features,                                         score(c) = w&#227;&#8224;? f, <br /> respectively. TMHMM Server (http://www.cbs.dtu.dk/ services/ <br /> TMHMM/) predicted number transmembrane helices                 w weight vector f denotes set feature values 5825 <span id='am-9' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> sequences. Finally, ratios rare           c. feature value fi normalized [0, 1] follows: <br /> amino acids amino acids close stop codons calcu- <br /> lated suggested (4).                                                              fi = (fi - min(fi)) / (max(fi) - min(fi)). <br /> &#227;&#8364;&#8364;Protein-protein interaction network features: followed protocol established Hwang et al. (9). Briefly, protein-pro-         genetic algorithm used finding w maximizes <br /> tein interaction data S. cerevisiae downloaded DIP           partial AUC. Supplementary Table 8 parameter setting (version Scere20091230), contained 5033 proteins               genetic algorithm. experiments, used partial AUCs <br /> 22,118 interactions. yeast protein-protein interaction         FPRs (false positive rates) 5% 10% examples ac- <br /> network, calculated following topological properties           ceptable FPRs subsequent validation. <br /> suggested (9): degree (degreeK), clustering coefficient (CCo), <br /> betweenness centrality (BC), closeness centrality (CC), clique           classification methods <br /> level (KL), essentiality index (EI). list essential genes       used Weka package (http://www.cs.waikato.ac.nz/ml/ <br />  <br />  <br /> http://bmbreports.org                                                                                                                BMB Reports     45 <br />     PAUC maximization using genetic algorithms&#226;&#8364;&#402; <br />    Kyu-Baek Hwang, et al. <br />  <br />  <br />  <br />  <br />    weka/) comparing performance GA-based methods                network sequence analysis. Mol. Biosyst. 5, 1672-1678. <br />    following popular classification methods: multilayer           10. Choi, J. K., Kim, S. C., Seo, J., Kim, S. Bhak, J. (2007) Impact <br />    perceptrons, logistic regression, support vector machines.              transcriptional properties essentiality evolutionary <br />    Default settings models Weka used. multi-             rate. Genetics. 175, 199-206. <br />    layer perceptrons, number hidden layers          11. Zhou, L., Ma, X. Sun, F. (2008) effects protein inter- <br />                                                                                actions, gene essentiality regulatory regions ex- <br />    number hidden nodes 16. backpropagation algo- <br />                                                                                pression variation. BMC Syst. Biol. 2, 54. <br />    rithm used training multilayer perceptrons. ridge esti-       12. Deng, J., Deng, L., Su, S., Zhang, M., Lin, X., Wei, L., Minai, <br />    mator used l <br /> </body></html>