<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                          RESEARCH ARTICLE <br />  <br />  <br />                                                           55.2, Phage T4 ORFan Gene, Encodes                                                           Inhibitor Escherichia coli Topoisomerase                                                           Increases Phage Fitness <br />                                                           Yves Mattenberger*, Filo Silva, Dominique Belin <br />                                                           Department Pathology Immunology, University Geneva, Geneva, Switzerland <br />  <br />                                                           Academic Editor Eric Cascales <br />                                                           Centre National la Recherche Scientifique, Aix-Marseille Universit&#195;&#169;, FRANCE <br />  <br />                                                           * yves.mattenberger@unige.ch <br />  <br />  <br />  <br />  <br />                                                           Abstract <br />                                                           Topoisomerases enzymes alter topological properties DNA. Phage T4  <br />                                                           codes topoisomerase utilize host-encoded topoisomerases.                                                           characterized 55.2, phage T4 predicted ORF unknown function. High levels expres- <br />     OPEN ACCESS <br />                                                           sion cloned 55.2 gene toxic E. coli. toxicity suppressed  <br />                                                           creased topoisomerase expression partial inactivation ATPase subunit Citation: Mattenberger Y, Silva F, Belin D (2015) <br /> 55.2, Phage T4 ORFan Gene, Encodes Inhibitor         DNA gyrase. Interestingly, low-level expression 55.2, non-lethal wild type Escherichia coli Topoisomerase Increases         E. coli, prevents growth deletion mutant topoisomerase (topA) gene. vitro, <br /> Phage Fitness. PLoS 10(4): e0124309.                  gp55.2 binds DNA blocks specifically relaxation negatively supercoiled DNA doi:10.1371/journal.pone.0124309 <br />                                                           topoisomerase  vivo, expression gp55.2 low non-toxic levels alters steady state <br /> Academic Editor: Eric Cascales, Centre National        DNA supercoiling reporter plasmid. 55.2 essential gene, competition <br /> la Recherche Scientifique, Aix-Marseille Universit&#195;&#169;, <br />                                                           experiments indicate required optimal phage growth. propose role FRANCE <br />                                                           gp55.2 subtly modulate host topoisomerase activity infection insure optimal <br /> Received: October 3, 2014 <br />                                                           T4 phage yield. <br /> Accepted: March 11, 2015 <br />  <br /> Published: April 14, 2015 <br />  <br /> Copyright: &#194;&#169; 2015 Mattenberger et al. open access article distributed terms Creative Commons Attribution License, permits <br /> unrestricted use, distribution, reproduction medium, provided original author source       Introduction <br /> credited.                                                 various DNA manipulations occur normal life cycle cell result Data Availability Statement: relevant data        topologically altered forms DNA: knots, catenanes, supercoils [1]. Supercoils  paper Supporting Information files.    scribe DNA state number times strands cross differs Funding: Fondation Schmidheiny DB Geneva <br />                                                           unconstrained DNA molecule. number lower higher expected, DNA Department Education funders role        said negatively positively supercoiled. Processes require unwinding study design, data collection analysis, decision   rotation topologically constrained DNA double helix, replication transcrip- <br /> publish, preparation manuscript                 tion, lead accumulation compensatory supercoils hinder polymerase  <br /> Competing Interests: authors declared            ment [2,3]. torsional stress problem, supercoiling important competing interests exist.                        physiological functions. Bacteria, normally maintain chromosome plasmids  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                           1 / 23 <br />                                                                                   Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           negatively supercoiled state important factor chromosome compaction <br />                                           transcriptional regulation [4,5]. <br />                                               DNA&#226;&#8364;&#8482;s diverse topological problems resolved topoisomerases, family enzymes <br />                                           allow DNA strands DNA double helices pass  Type topoisomer- <br />                                           ases transiently cut DNA strand, releasing superhelical tension, type II enzymes <br />                                           cut DNA strands allowing <span id='am-6' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span>-dependent transport DNA double helix <br />                                           [6,7]. Given universality DNA topological constraints, surpris- <br />                                           ing topoisomerases domains life. addition, number viruses, <br />                                           eukaryotic bacterial, encode topoisomerases, [8,9]. critical <br />                                           role determining DNA topology, topoisomerases target choice  <br />                                           velopment antibiotics anticancer drugs. Consequently, type II topoisomerase  <br />                                           hibitors employed clinically potent antibacterial antitumor agents [10,11]. <br />                                               Escherichia coli type topoisomerases (Topo III), type II topoisomer- <br />                                           ases (DNA gyrase Topo IV). negative supercoiling bacterial chromosome                                           plasmids homeostatically regulated counterbalancing activities Topo Topo <br />                                           IV, remove negative supercoils, DNA gyrase, adds negative super- <br />                                           coils [12]. Topo III Topo IV primarily responsible unlinking sister chromo- <br />                                           somes replication [13]. Phage T4, infects E. coli, expresses                                           topoisomerase subunits encoded genes 39, 52, 60 [14,15]. type II <br />                                           topoisomerase absolutely essential T4 growth, topoisomerase mutants display                                           called DNA-delay phenotype [16&#226;&#8364;&#8220;18].  phage topoisomerase absolutely  <br />                                           quired early mode phage DNA replication. topoisomerase mutant phages, DNA <br />                                           replication depends instead late, recombination-dependent, join-cut-copy replication <br />                                           pathway (reviewed [19&#226;&#8364;&#8220;21]). absence topoisomerase, T4 growth                                           completely dependent host DNA gyrase [22]. <br />                                               isolation large collections nonsense temperature-sensitive mutants, <br />                                           phage T4 organism nearly essential genes identified [16,23]. <br />                                           addition 62 essential genes, genome known encode auxiliary proteins, <br />                                           increasing total number characterized genes 156.  T4 genome pre- <br />                                           dicted 289 protein-coding genes (open reading frames ORF); nearly addi- <br />                                           tional 126 genes known function [24]. new unknown T4 genes <br />                                           detectable homology known proteins, considered databases orphans <br />                                           (ORFans).  present genomes related T4-like <br />                                           phages [25]. lytic phages abundant organisms planet [26,27], <br />                                           numerous diverse ORFans represent enormous reservoir unchartered genetic <br />                                           information, called &#226;&#8364;&#339;dark matter biosphere&#226;&#8364;? [26,28]. <br />                                               previously identified T4 ORFans ectopic expression significantly inhib- <br />                                           E. coli growth [29]. described  DNA transactions require resolution                                           DNA topological issues.  considered possibility T4 ORFans <br />                                           involved control phage DNA topology possibly conjunction                                           host topoisomerases. report characterization gene 55.2, toxic T4 ORFan <br />                                           affects supercoiling regulation E. coli inactivation reduces phage T4 proge- <br />                                           ny production. <br />  <br />  <br />                                           Materials Methods <br />                                           E. coli phage strains, plasmids growth conditions <br />                                           bacterial strains phage strains used study listed Table 1; plasmids                                           listed S1 File. construction strains, plasmids genomic DNA library  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                               2 / 23 <br />                                                                                       Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           Table 1. E. colia T4 strains. <br />  <br />                                           Strain          Genotype                                                                 Reference <br />                                           AS1047          MG1655 &#206;&#8221; lacIZYA::frt pAST111                                            [31] <br />                                           AS1050          MG1655 &#206;&#8221; lacIZYA::frt &#206;&#8221; topA::apra pAST111                               [31] <br />                                                        Prototrophic                                                             [32] <br />                                           BL21(DE3)       F- ompT gal dcm lon hsdSB &#206;&#187;(DE3)                                         [32] <br />                                           BP199           W3110 gyrB221(CouR) gyrB203(ts)                                          [13] <br />                                           BW25113         F- &#206;&#8221;(araD-araB)567 &#206;&#8221;lacZ4787(::rrnB-3) rph-1 &#206;&#8221;(rhaD-rhaB)568             CGSC <br />                                                           hsdR514 <br />                                           CL001           W3110 parE10(ts)                                                         [13] <br />                                           CR63            F+, supD, lamB63                                                         lab collection <br />                                           DHB3            MC1000 malF&#206;&#8221; 3 phoA&#206;&#8221; (PvuII) phoR                                        [33] <br />                                           DB503           MC4100 malE16-1 &#206;&#8221; ara714                                                 lab collection <br />                                           DB870           MG1655 &#206;&#8221; ara714, ydeA::kan                                               study <br />                                           DH5&#206;&#177;            F- &#206;&#8221;(argF-lac)169, &#207;&#8224;80dlacZ58(&#206;&#8221;M15), &#206;&#8221;phoA8, glnV44, deoR481,            CGSC <br />                                                           rfbC1, gyrA96(NalR), recA1, endA1, thiE1, hsdR17 <br />                                           JW0204-2        BW25113 &#206;&#8221;rnhA733::kan                                                    [34] <br />                                           MC1000          F araD139 &#206;&#8221; (araA-leu)7697 &#206;&#8221; (lac)X74 rpsL150 galE15 galK16 relA1 <br />                                                            - <br />                                                                                                                                    [35] <br />                                                           thi <br />                                           MC4100          F- araD139 &#206;&#8221; (argF-lac)U169 &#239;&#172;&#8218;hD5301 fruA25 relA1 rpsL150 rbsR22 &#206;&#8221;        [36] <br />                                                           (&#239;&#172;?mB-&#239;&#172; 632 deoC1 thi <br />                                           MG1655          F- rph-1                                                                 CGSC <br />                                           W3110           F- rrnD-rrnE)1 rph-1                                                  CGSC <br />                                           YM63            BW25113 PlacZ-topA76(ts) zci-2234::cat                                   study <br />                                           YM64            BW25113 PlacZ-topA76(ts) zci-2234::cat &#206;&#8221;topB761::kan                     study <br />                                           T4+             T4D                                                                      [16] <br />                                           T4 K10          38amB262 51amS29 nd28 (denA) rIIPT8 (&#206;&#8221;denB-rII)                          [37] <br />                                           T4 K10          K10 55.2 (ATG-&gt; ACA)                                                     study <br />                                           55.2 <br />                                           T4 39           T4D 39amEA29                                                             R.H. Epstein&#226;&#8364;&#8482;s <br />                                                                                                                                    collection <br />                                           T4 55.2         T4D 55.2 (ATG-&gt; ACA)                                                     study <br />                                           T4 39 55.2      T4D 39amEA29 55.2 (ATG-&gt; ACA)                                            study <br />                                                                                      strains E. coli K-12 derivatives BL21(DE3), E. coli B strains. <br />  <br />                                           doi:10.1371/journal.pone.0124309.t001 <br />  <br />  <br />  <br />                                           described S1 File. Unless stated, E. coli strains cultivated 37&#194;&#176;C LB, M9 <br />                                           minimal (M9), M63 minimal (M63) medium [30] supplemented amino acids car- <br />                                           bon source indicated. growth solid medium, 1.5% bacteriological agar included. <br />                                           Antibiotics used following concentrations: ampicillin (Ap), 200 &#206;&#188;g ml-1; chloram- <br />                                           phenicol (Cm), 30 &#206;&#188;g ml-1; kanamycin (Kn), 40 &#206;&#188;g ml-1; spectinomycin (Sp), 50 &#206;&#188;g ml-1; tet- <br />                                           racycline (Tc), 7.5 &#206;&#188;g ml-1. <br />  <br />  <br />  <br />                                           Plasmid based lethality assay <br />                                           Plasmid based lethality assays performed previously described [31]. Briefly, bacteria <br />                                           cultivated overnight LB Kn Ap maintain 55.2 topA plas- <br />                                           mids. morning, saturated cultures diluted 1/80 M63 medium supplemented <br />                                           0.2% glucose Kn Ap grown A600 nm = 0.4, spreading <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                          3 / 23 <br />                                                                                  Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           dilutions M63 agar plates supplemented 0.2% glucose, Kn, 600 &#206;&#188;M IPTG, 80 &#206;&#188;g <br />                                           ml-1 X-Gal. Colonies counted photographed 36 h 37&#194;&#176;C. <br />  <br />  <br />                                           Purification gp55.2-His6 <br />                                           protein expressed pMCN1 BL21(DE3) cells. Exponentially growing bacterial <br />                                           cultures induced 0.2% arabinose 2 hours 37&#194;&#176;C. protein purified                                           frozen cell pellets Ni-NTA agarose (#30210, Quiagen) according manufacturer instruc- <br />                                           tions. Eluted fractions containing tagged protein desalted PD-10 column <br />                                           (GE Healthcare) equilibrated TKDG buffer (50mM Tris pH 7.4, 100mM KCl, 1mM DTT, <br />                                           10% glycerol) concentrated centricon microconcentrator (3 kDa molecular weight <br />                                           cut  #4202, Amicon). Protein concentration determined Bradford assay (Bio-Rad <br />                                           Protein Assay) purity (&gt;99%) assessed Coomassie brilliant blue staining proteins <br />                                           separated SDS-polyacrylamide gel electrophoresis. <br />  <br />  <br />                                           Electrophoretic Mobility Shift Assay (EMSA) <br />                                           EMSA performed form form  DNA plasmid pDB29 prepared described <br />                                            linear DNA obtained restriction digestion EcoRV. Plasmid DNA (0.3 &#206;&#188;g, 79 <br />                                           fmol) incubated indicated amounts gp55.2-His6 EMSA Buffer (50 mM Tris, <br />                                           100 mM KCl, 1 mM &#206;&#178;-mercaptoethanol, 1 mg ml-1 Bovine serum albumin (BSA), 10% glycerol, <br />                                           pH 7.5 25&#194;&#176;C) 30 min 37&#194;&#176;C. Samples electrophoresed 3V cm-1                                           0.8% agarose gels 1x TBE (90 mM Tris, 90 mM boric acid, 2.5 mM EDTA) 20 h 4&#194;&#176;C <br />                                           constant buffer recirculation. Gels stained ethidium bromide photographed <br />                                           UV light using E-BOX VX5 (Vilber Lourmat, France) Chemi Doc MP <br />                                           (Bio-Rad Laboratories) <br />  <br />  <br />                                           DNA relaxation supercoiling assays <br />                                           Covalently closed, negatively supercoiled circular DNA (form  plasmid pDB29 [38]                                           obtained standard alkaline lysis procedure, followed cesium chloride density gradient <br />                                           centrifugation [39]. Covalently closed, relaxed circular DNA (form ) prepared incubat- <br />                                           ing form DNA wheat germ topoisomerase (Promega, #M285) according manufac- <br />                                           turer instruction. plasmid DNAs extracted phenol/CHCl3 ethanol <br />                                           precipitated used substrates supercoiling reactions. relaxation assays, E. <br />                                           coli Topo purchased New England Biolabs (#M0301) reactions performed <br />                                           1x CutSmart buffer 0.6 &#206;&#188;g form pDB29 DNA (158 fmol). indicated amounts                                           gp55.2-His6 incubated DNA 15 min 37&#194;&#176;C, indicated amounts                                           Topo added samples incubated 37&#194;&#176;C 15 min. Reactions terminated <br />                                           described [40]. Briefly, EDTA added 25mM samples incubated 2 min                                           37&#194;&#176;C;  SDS proteinase K added 1% 100 &#206;&#188;g/ml, respectively, sam- <br />                                           ples incubated additional 15 min 37&#194;&#176;C. DNA products extracted phe- <br />                                           nol/CHCl3 electrophoresed 2V cm-1 0.8% agarose gels 1x TBE buffer 48 h <br />                                           4&#194;&#176;C constant buffer recirculation. Gels stained photographed explained <br />                                            supercoiling assays, E. coli DNA gyrase purchased New England Biolabs <br />                                           (#M0306) reactions performed 1x Gyrase buffer 0.6 &#206;&#188;g form  pDB29 <br />                                           (158 fmol). indicated amounts gp55.2-His6 incubated DNA 15 min <br />                                           37&#194;&#176;C, indicated amounts DNA gyrase added reactions                                           incubated 37&#194;&#176;C 30 min. Reactions terminated analyzed described                                           relaxation assays. <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             4 / 23 <br />                                                                                  Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           vivo analysis plasmid topoisomers <br />                                           obtain monomeric molecules, plasmids linearized self-circularized large liga- <br />                                           tion volumes. ligations transformed appropriate strains independent <br />                                           clones purified used subsequent analyses. Equal amounts (A600nm) exponential- <br />                                           ly growing bacterial cultures quickly chilled mixing equal volumes ice- <br />                                           cold growth medium transferred ice. DNA extracted standard alkaline lysis pro- <br />                                           cedure, followed phenol/CHCl3 extraction ethanol precipitation. nucleic acid pel- <br />                                           lets dissolved TE buffer 2 &#206;&#188;g ml-1 RNAse analyzed 0.8% agarose/ TBE <br />                                           gels 1 1.5 &#206;&#188;g ml-1 chloroquine (CLQ) described previous section. Loaded vol- <br />                                           umes adjusted account difference plasmid copy number 55.2-ex- <br />                                           pressing control strains. electrophoresis, gels washed thrice 15 min                                           0.5x TBE, stained ethidium bromide (1 &#206;&#188;g ml-1) <span id='am-17' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span>H2O</span> 1 hour, destained 20 min                                           1mM MgSO4, photographed described  migration plasmid DNA <br />                                           samples gels different CLQ concentrations allowed determine bands <br />                                           correspond negatively supercoiled plasmids [41]. Densitometric analyses  <br />                                           formed unsaturated images using MultiGauge (v.3) software (Fujifilm LifeScience).                                           data normalized total DNA lane. dimensional gel (2D) gel <br />                                           analyses performed previously described [42] using 0.8% agarose/ TBE gels 1.5 &#206;&#188;g <br />                                           ml-1 25 &#206;&#188;g ml-1 CLQ second dimension, respectively. <br />  <br />                                           Plasmid copy number quantification <br />                                           Plasmid DNA extracted equal amounts bacteria (A600nm) standard alkaline lysis <br />                                           procedure resuspended TE buffer. Equal volumes linearized appropriate <br />                                           restriction enzyme, ran standard agarose TAE gels, visualized ethidium bromide. <br />                                           Unsaturated images analyzed using MultiGauge (v.3) software (Fujifilm LifeScience) <br />                                           signals calibrated using standard curve diluted High Mass DNA ladder (Invitro- <br />                                           gen) loaded gel. <br />  <br />                                           Intracellular phage growth phage competition assays <br />                                           Intracellular phage growth assays performed previously described [29] follow- <br />                                           ing modifications. CR63 bacteria grown M9 medium supplemented 50 &#206;&#188;g ml-1 <br />                                           tryptophan 0.4% glucose (M9) 1% casmino acids (M9S) concentration <br />                                           1 x 108 cfu ml-1 placed ice. infection, bacteria prewarmed 10 min                                           infection (t = 0) started mixing bacteria equal volume phages diluted                                           corresponding growth media. Phage growth carried indicated temperature vig- <br />                                           orous agitation. Competition assay performed CR63 cells grown M9S.                                           growth cycle, aliquot bacteria prewarmed 10 min 37&#194;&#176;C, mixed equal vol- <br />                                           ume phage diluted M9S (1 x 106 pfu ml-1, multiplicity infection (moi) = 0.01),                                           growth carried 37&#194;&#176;C vigorous agitation. 45 min, cultures lyzed                                           drop CHCl3 placed ice 10 min, transferred tube, kept 4&#194;&#176;C.  <br />                                           cause average burst size conditions 130, lysates diluted 65 times <br />                                           M9S directly used carry growth cycle. titer lysate determined <br />                                           ensure moi growth cycle exceeded 0.1. <br />  <br />                                           Determination ratio 55.2+ total phage <br />                                           Aliquots phage lysates ( 1 x 105 pfu) used template amplification                                           primers 55.3up (5&#226;&#8364;&#8482;-GGAAATTTAAATGAATCCTGAATC) 55.1dn (5&#226;&#8364;&#8482;-AGACCTATCTTAAAGCATAGAG) <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             5 / 23 <br />                                                                                   Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           using Taq DNA polymerase (Invitrogen) following manufacturer&#226;&#8364;&#8482;s instruction. Aliquots                                           PCR digested BsrGI, cuts mutated 55.2 initiation codon                                           amplified fragment. Restriction digests run 1% agarose/TAE gels, stained ethid- <br />                                           ium bromide, imaged UV light, intensity values DNA bands deter- <br />                                           mined unsaturated images using Multi Gauge 3.0 software (FujiFilm). K10/ (K10- <br />                                           55.2 + K10) ratios calculated dividing intensity value undigested band                                           cumulated intensity values undigested band bands resulting BsrGI di- <br />                                           gestion. confirmed accuracy PCR-restriction determination testing standard <br />                                           curve prepared known amounts K10 K10-55.2 phage. addition, ratio se- <br />                                           lected growth cycle lysates confirmed determination genotype individual <br />                                           plaques (n = 20). <br />  <br />  <br />                                           Results <br />                                           Ectopic expression cloned phage T4 gene 55.2 inhibits irreversibly E. <br />                                           coli growth <br />                                           Using vectors expressing T4 genes control inducible arabinose PBAD promoter, <br />                                           previously demonstrated ectopic expression 55.2 ORFan deleterious                                           E. coli growth [29]. plasmids confer arabinose sensitive phenotype (AraS) carri- <br />                                           er host strain. experiments, performed medium copy number plasmid derived <br />                                           ColEI, residual expression 55.2 PBAD promoter absence arabinose <br />                                           induction sufficient affect bacterial growth. Initial growth dilution stationary <br />                                           phase cultures delayed compared bacteria harboring vector control <br />                                           (S1A Fig). Furthermore, colony forming units (cfu) saturated cultures 10-fold <br />                                           lower controls (S1B Fig). minimize effects residual expression  <br />                                           pressed PBAD promoter, 55.2 ORFan sequence cloned pBAD101, low-copy  <br />                                           ducible vector pSC101 derived replicon [43]. essentially abolished phenotypes <br />                                           observed original construct absence arabinose.  arabinose  <br />                                           duced conditions, 55.2 expression prevented bacterial colony formation. Growth curve  <br />                                           says showed addition arabinose led slow growth arrest complete                                           3 h (Fig 1A).  reversibility assay demonstrated hour                                           55.2 induction, ~100 fold decrease capacity induced cells form colonies <br />                                           absence arabinose (Fig 1B). 55.2 irreversible bactericidal action. <br />  <br />  <br />                                           Overexpression topA, gene encoding E. coli topoisomerase  <br />                                           suppresses 55.2 toxicity <br />                                           searched E. coli genes  present multiple copies, suppressed AraS pheno- <br />                                           type induced 55.2 expression (multicopy suppressors). Initial attempts using standard E. coli <br />                                           genomic libraries identified non-specific suppressors interfere PBAD induction. <br />                                           non-specific suppressors included genes araBAD operon, encodes enzymes <br />                                           degrade arabinose, ydeA, encodes transporter facilitates arabinose export <br />                                           [44].  prepared new library strain harbors araBAD ydeA dele- <br />                                           tions. plasmid library introduced electroporation cells carrying pBAD101 <br />                                           plasmid 55.2 insert transformants directly selected LB plates containing <br />                                           0.2% arabinose. Plasmids suppressor strains purified suppressor activi- <br />                                           ty library plasmids verified. Non-specific suppressors identified eliminated                                           previously described [29]. testing equivalent   180 E. coli genomes, isolated <br />                                           plasmid conferred specific, albeit partial, resistance 55.2 expression. 4259 bp <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                              6 / 23 <br />                                                                                            Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />  <br /> Fig 1. toxicity 55.2 E. coli suppressed increase copy number topA.   Liquid growth assay. Overnight cultures DB503 <br /> cells harboring pBAD101 (ctr) pDB2114-101 (55.2) plasmids diluted fresh LB medium optical density (A600nm) measured indicated <br /> times. A600nm = 0.3, cultures split half induced 0.2% arabinose (vertical arrow). A600nm reached   1, cultures diluted 10-fold prewarmed medium plus minus arabinose. graph depicts data representative experiment. (B) Reversibility assay. growth curve assay, aliquots arabinose-induced DB503 cultures, harboring pBAD101 (55.2 &#226;&#8364;&#8220;) pDB2114-101 (55.2 +) plasmids, withdrawn indicated times, washed cold media arabinose, adjusted A600nm. Serial 10-fold dilutions spotted LB plates arabinose. (C) DB503 cells transformed pBAD101 (vector) pDB2114-101 (55.2) compatible plasmids, pDB868-2 (vector, 1&#226;&#8364;&#8220;2), pDB34- <br /> 8-4 (topA, 3&#226;&#8364;&#8220;4), pDB34-8 (topA, 5&#226;&#8364;&#8220;6) streaked LB plates 0.2% arabinose. (D) Plasmid based lethality assay. Overnight cultures AS1047 (topA +) AS1050 (topA-) transformed pBAD33-K (55.2 &#226;&#8364;&#8220;) pDB2114-33-K (55.2 +) diluted outgrown indicated Materials <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                            7 / 23 <br />                                                                                              Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  Methods section. Aliquots diluted, plated M63 plates supplemented glucose X-gal. number blue white colonies scored 36h 37&#194;&#176;C. Representative photographs shown upper panel  position rare white colony inset IV indicated arrow). lower panel depicts percentage white colonies; average standard deviation independent experiments. <br /> doi:10.1371/journal.pone.0124309.g001 <br />  <br />                                                 insert plasmid contained yciN, topA cysB. Subsequent subcloning revealed <br />                                                 region containing entire topA coding sequence 272 bp sequence upstream <br />                                                 initiation codon sufficed suppression (Fig 1C, streak 4). product topA                                                 DNA topoisomerase (Topo , type IA topoisomerase removes excess negative super- <br />                                                 coils introduced DNA gyrase E. coli chromosome plasmids [45,46]. <br />  <br />                                                 Incompatibility low-level expression 55.2 reduced expression <br />                                                 Topo Topo III <br />                                                 high-level expression topA partially suppressed 55.2-induced toxicity, examined <br />                                                 effect E. coli varying levels expression 55.2 topA. viability topA <br />                                                 null mutants absence compensatory DNA gyrase mutations long debated <br />                                                 [47,48]. Recently, shown Rudolph coworkers loss topA results se- <br />                                                 vere growth defect [31]. assay, copy topA gene cloned unstable mini-F <br />                                                 lac+ plasmid complemented chromosomal topA deletion. lac&#226;&#8364;&#8221;strain, loss plasmid <br />                                                 followed colony color X-gal plates. Using  determined effect <br />                                                 low-level expression 55.2 topA loss. TopA+ &#206;&#8221;topA bacteria harboring topA lac+ <br />                                                 mini-F plasmid transformed pBAD vector 55.2 insert.  <br />                                                 growth absence antibiotic selection topA mini-F plasmid, bacteria plated <br />                                                 minimal X-gal glucose plates selection pBAD plasmid (Fig 1D). presence <br />                                                 topA+ chromosomal copy  , colonies white indicating loss                                                 topA+ mini-F plasmid effect growth. absence topA chromosomal copy <br />                                                 (III), 65% colonies retained topA+ mini-F plasmid; importantly, white colo- <br />                                                 nies notably smaller blue. results confirm topA mutants severe <br />                                                 growth defect. 55.2-coding plasmid present, white colonies observed <br />                                                 (5.5%) size significantly smaller white colonies obtained absence <br />                                                 55.2-coding plasmid (compare insets IV III).  topA function apparently nec- <br />                                                 essary tolerate low-level expression 55.2 non-toxic wild type cells.  <br />                                                 sults obtained using 55.2 expressing plasmid  30 copies cell                                                 presence glucose, represses expression PBAD promoter [49]. Consequently, <br />                                                 level 55.2 expression manifests incompatibility topA deletion strain low. <br />                                                 Significantly, presence 55.2-coding plasmid reduced frequency topA+ mini- <br />                                                 F plasmid loss topA+ genetic background (II). consistent low levels                                                 55.2 expression having effect growth wild-type bacteria (S1A S1B Fig)                                                 suggests mere duplication topA locus suffices partially overcome effect. <br />                                                     mentioned introduction, E. coli possesses type IA topoisomerase, Topo III <br />                                                 (topB). asked  context low-level expression 55.2, Topo III partial- <br />                                                 ly compensate decrease Topo activity. medium copy pBAD plasmid expressing 55.2 <br />                                                 transformed E. coli strains harboring temperature sensitive (ts) allele topA driven <br />                                                 lacZ promoter wild-type disrupted topB gene. assessed  <br />                                                 bility strains presence various concentration IPTG 37&#194;&#176;C, temperature <br />                                                 Topo produced topAts allele active wild-type enzyme [50]. <br />                                                 results, presented S1C Fig, confirm incompatibility reduced Topo activity                                                 residual expression 55.2 PBAD promoter (upper panel). incapacity PlacZ- <br />                                                 topA &#206;&#8221;topB bacteria grow absence IPTG repeated known non-viability topA <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                                8 / 23 <br />                                                                                   Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           topB double mutants [48,51]. Importantly, reduced viability phenotype caused low-level <br />                                           expression 55.2 stronger absence functional Topo III (lower panel). fur- <br />                                           ther decrease viability caused 55.2 &#206;&#8221;topB strain evident presence <br />                                           1mM IPTG, concentration topB+ &#206;&#8221;topB strains harboring control vec- <br />                                           tor difference viabilty. results indicate E. coli requires minimal type IA <br />                                           topoisomerase activity tolerate low-level expression 55.2. <br />  <br />                                           Inhibiting GyrB ATPase subunit gyrase reduces 55.2-induced <br />                                           toxicity <br />                                           supercoiling level E. coli chromosome plasmids primarily determined                                           opposing activities Topo  removes negative supercoils DNA gyrase (gyrAB), <br />                                           adds negative supercoils (Fig 2A).  inhibition DNA gyrase expected                                           similar effect increasing Topo activity. Consequently, likely                                           inhibition counteract 55.2-induced toxicity. tested sensitivity 55.2-express- <br />                                           ing E. coli cells novobiocin, aminocoumarin type antibiotic inhibits gyrase&#226;&#8364;&#8482;s DNA su- <br />                                           percoiling competing ATP binding GyrB subunit [52&#226;&#8364;&#8220;54].  <br />                                           arabinose inhibited growth bacteria harboring 55.2 coding pBAD101 plasmid. Howev- <br />                                           er, low concentrations novobiocin  16 64 &#206;&#188;g ml-1), effect                                           growth bacteria carrying control plasmid, partially reduced toxicity associated                                           55.2 expression (Fig 2B). Furthermore, 55.2 expression permitted bacterial growth bacteria <br />                                           128 &#206;&#188;g ml-1 novobiocin, concentration inhibited growth absence arabinose <br />                                           harbored pBAD101 plasmid  arrowhead Fig 2B). <br />                                               suggested limiting DNA gyrase function alleviated, partially, growth  <br />                                           hibition mediated 55.2.  novobiocin inhibit topoisomerase IV (Topo <br />                                           IV) E. coli binding ParE subunit, homologous GyrB [55].  asked <br />                                           partial inactivation DNA gyrase Topo IV sufficed suppress 55.2 toxic- <br />                                           ity. transformed gyrB parE ts mutant strains 55.2 coding plasmid, tested <br />                                           sensitivity arabinose different temperatures. 55.2 expression blocked bacterial <br />                                           growth strains permissive temperature (34&#194;&#176;C), intermediate temperature <br />                                           (37&#194;&#176;C), permitted growth bacteria carrying ts alleles, led <br />                                           partial suppression 55.2-induced toxicity gyrBts parEts mutant strain <br />                                           (Fig 2C). Complete inhibition DNA gyrase 42&#194;&#176;C abolished growth independently 55.2 <br />                                           expression.  conclude partial inhibition GyrB sufficient offset toxicity <br />                                           high-level expression 55.2. <br />  <br />                                           Gp55.2 binds DNA vitro <br />                                            asked gp55.2 bind DNA. vitro binding gp55.2 DNA                                           demonstrated electrophoretic mobility shift assays (EMSA) employing purified gp552-His6 <br />                                           protein plasmid DNA pDB29  pBR322 derivative containing mouse urokinase <br />                                           cDNA insert; [38]). shown Fig 3A, gp55.2 caused linear pDB29 DNA (upper panel)                                           migrate slowly. shift easily detectable ratio gp55.2 molecule 16.4 <br />                                           bp DNA (380 ng gp55.2 300 ng DNA). protein DNA ratio, gp55.2  <br />                                           tarded migration covalently closed, relaxed circular DNA (form , middle panel),  <br />                                           valently closed, negatively supercoiled circular DNA (form  lower panel), small <br />                                           nicked circular DNA (form II, lower panel, arrow) contaminated form                                           DNA. Additional EMSA using M13 phage DNA showed gp55.2 retarded migration                                           double-stranded replicative form circular single-stranded virion DNA (S2A <br />                                           Fig).  gp55.2 appears bind negatively supercoiled, relaxed, single-stranded DNA. <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                              9 / 23 <br />                                                                                           Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />  <br /> Fig 2. Partial inhibition DNA gyrase ATPase activity protects 55.2-induced toxicity.   Outline regulation steady-state level DNA <br /> supercoiling opposite actions DNA gyrase Topo  (B) Novobiocin sensitivity assay. Overnight cultures DB503 bacteria transformed pBAD101 (55.2 &#226;&#8364;&#8220;) pDB2114-101 (55.2 +) spotted 24 wells LB agar plates 0.2% arabinose containing indicated amounts novobiocin, incubated overnight 37&#194;&#176;C. (C) W3110 (wt), CL001 (parEts), BP199 (gyrBts) transformed pBAD101 (55.2 &#226;&#8364;&#8220;) pDB2114-101 <br /> (55.2 +). Dilution overnight cultures spotted LB agar plates 0.2% arabinose incubated overnight indicated temperatures. <br /> doi:10.1371/journal.pone.0124309.g002 <br />  <br />                                                vitro gp55.2 inhibits Topo does stimulate gyrase activity <br />                                                genetic evidence combined capacity gp55.2 bind DNA suggests                                                protein inhibit Topo activity stimulate DNA gyrase. directly <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                          10 / 23 <br />                                                                                             Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />  <br /> Fig 3. Gp55.2 binds DNA, inhibits Topo relaxation activity does affect DNA gyrase supercoiling activity.   Electrophoretic mobility shift <br /> assays (EMSA). Mixtures containing 300 ng (= 79 fmol) linear, form , form pDB29 DNA indicated gp55.2-His6 (95 ng = 6.9 pmol, gp55.2 molecule 65.6 bp DNA used assay) incubated analyzed described Materials Methods. arrow indicates migration position form II DNA contaminating form DNA. (B) Relaxation assays mixtures containing 600 ng (= 158 fmol) form pDB29 DNA indicated units Topo (0.47 U = 415 fmol) incubated presence (+) absence (&#226;&#8364;&#8220;) 855 ng gp55.2-His6 (= 63 pmol, gp55.2 molecule 14.5 bp DNA), DNA products analyzed described Materials Methods. migration positions form  form  form II DNA indicated left; linear DNA migration position indicated arrowhead. Results representative independent experiments shown. (C) Supercoiling assays mixtures containing 158 fmol form  pDB29 DNA indicated units DNA gyrase (0.42 U = 196 fmol) incubated presence (+) absence (&#226;&#8364;&#8220;) 855 ng gp55.2-His6 DNA products analyzed described Materials Methods <br /> section. Results representative independent experiments shown. (D) Representation form  form  form II plasmid DNA. Note treatment form plasmid DNA eukaryotic Topo obtain form  DNA results, equilibrium, distribution relaxed DNA topoisomers   middle <br /> panel) maximum corresponds fully relaxed form  plasmid illustrated D. <br /> doi:10.1371/journal.pone.0124309.g003 <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                            11 / 23 <br />                                                                                  Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           examined vitro effect gp55.2 E. coli Topo DNA gyrase activities. relaxing <br />                                           activity Topo measured presence absence purified gp552-His6 concen- <br />                                           tration slightly higher lowest concentration band shift observed                                           EMSA assays. shown Fig 3B, gp55.2 strongly inhibited relaxation activity Topo  <br />                                           lowest molar ratio gp55.2 Topo tested (150:1; lane 10). Based                                           enzyme required obtain similar relaxation levels absence presence gp55.2, <br />                                           estimated relaxation activity inhibited 10 30 fold (compare lanes 2 3 <br />                                           lane 10). experiments, Topo fixed gp55.2 <br />                                           varied, indicated ratio gp55.2 molecule 7 bp DNA required com- <br />                                           plete inhibition (S3A Fig, lane 8).  lower gp55.2 DNA ratio, super- <br />                                           coiled relaxed topoisomers observed presence absence                                           gp55.2 (S3A Fig, lane 4 7), suggesting gp55.2 stimulate Topo activity low <br />                                           protein DNA ratio. stark contrast, gp55.2 effect negative supercoiling activ- <br />                                           ity DNA gyrase (Fig 3C). tested gp55.2 inhibit relaxation su- <br />                                           percoiled DNA wheat germ Topo  type 1B topoisomerase structure                                           mechanism relaxation unrelated E. coli Topo [6]. enzymatic activity                                           wheat germ Topo largely insensitive gp55.2; furthermore, observed limited inhibi- <br />                                           tion relaxation (  20%) independent concentration gp55.2 used, suggesting <br />                                           reached saturation (S3B Fig). Altogether, results gp55.2 inhibits specifi- <br />                                           cally E. coli Topo vitro. <br />  <br />  <br />                                           55.2 affects DNA topology control vivo <br />                                           investigated effect vivo low, non-toxic levels 55.2 expression supercoiling <br />                                           plasmid DNA.  used pDB2114, pBAD-55.2 plasmid intermediate <br />                                           copy number, source 55.2 expression, residual expression PBAD ab- <br />                                           sence arabinose. plasmid served reporter DNA supercoiling.                                           control, used closely related plasmid 55.2 coding sequence dis- <br />                                           rupted 5bp insertion ATG initiation codon producing frame-shifted 18 amino <br />                                           acid <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> instead gp55.2. frame shift abolished gp55.2 phenotypes parent <br />                                           plasmid presence absence arabinose. Plasmid DNA extracted bac- <br />                                           teria growing exponentially absence arabinose analyzed dimension agarose <br />                                           gel electrophoresis presence chloroquine, intercalating ligand adds positive su- <br />                                           percoils plasmid DNA allows separation supercoiled topoisomers [56]. seen                                           Fig 4A, distribution topoisomers slightly different plasmids carrying  <br />                                           tact mutated copy 55.2. Changes distribution slow migrating, partially  <br />                                           laxed topoisomers evident gel containing 1 &#206;&#188;g ml-1 CLQ, fast <br />                                           migrating, negatively supercoiled topoisomers better resolved gel containing 1.5 &#206;&#188;g <br />                                           ml-1 CLQ. densitometry profiles right panel summarize analysis indepen- <br />                                           dent DNA samples plasmid  S4A Fig). principal differences ob- <br />                                           served. slight shift profiles 5 bp difference size                                           plasmids. theoretically predicted, experimentally demonstrated, small <br />                                           changes size (&lt;10bp), affect perceptibly mobility linear form II <br />                                           (nicked circle) DNA, cause size-dependent decrease gel mobility topoisomers <br />                                           linking number. half turn shift observed perfectly accordance                                           published results [57]. Aside size-dependent change, relevant differences <br />                                           observed.  55.2-coding plasmid broader distribution topoisomers com- <br />                                           pared control plasmid. addition, peak topoisomer distribution 55.2- <br />                                           coding plasmid shifted negative supercoiling half helical turn.  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                            12 / 23 <br />                                                                                        Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />  <br /> Fig 4. 55.2 expression affects control DNA topology plasmid copy number E. coli.   Plasmid topoisomers analysis. Left panel: plasmid <br /> DNA extracted exponentially growing DH5&#206;&#177; harboring pYM58 (55.2IS) pDB2114 (55.2) plasmids. Plasmid topoisomers resolved TBE <br /> agarose gels containing indicated CLQ. position migration relaxed nicked circular DNA indicated (R). Right panel: <br /> Densitometry analysis topoisomer distribution 1.5 &#206;&#188;g ml-1 CLQ gel independent samples pYM58 pDB2114 plasmid DNA (gel images <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                     13 / 23 <br />                                                                                               Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  depicted S4A Fig). Plotted average (lines) standard deviation (shaded area) relative plasmid density function negative supercoiling. <br /> (B) 2D electrophoretic separation plasmid topoisomers. Left panel: schematic representation 2D gel. migration positions negatively supercoiled <br /> (&#226;&#8364;&#8220;), positively supercoiled (+), form II topoisomers (N) indicated. Right panel: gel images 2D topoisomer distribution plasmid DNA <br /> samples prepared  Chloroquine concentration 1.5 &#206;&#188;g ml-1 25 &#206;&#188;g ml-1 second dimension, respectively. (C) Plasmid copy <br /> number analysis. Linearized plasmid DNA samples experiment shown quantified normalized (A600nm) bacteria used extract plasmids. data represents means standard error independent cultures. (D) Plasmid DNA extracted overnight cultures DHB3, DB503, DH5&#206;&#177; transformed pBAD101 (55.2 &#226;&#8364;&#8220;) pDB2114-101 (55.2 +), reporter plasmid (pDB868-2). Linearized plasmids analyzed agarose gel electrophoresis. <br /> doi:10.1371/journal.pone.0124309.g004 <br />  <br />                                                 obtain better resolution relaxed topoisomers, supercoiling state plasmid <br />                                                 analyzed using dimensional chloroquine gels. results presented Fig 4B <br />                                                 clearly low-level expression 55.2 caused appearance relaxed                                                 negatively supercoiled topoisomers. early stop codon 55.2 mutant plasmid <br />                                                 used control cause premature Rho-dependent transcription termination [58].  <br />                                                 cause transcription elongation affects vivo supercoiling level plasmids [59],  <br />                                                 formed control experiment determine potential effect transcriptional polarity. <br />                                                 compared pBAD plasmids containing wild type 55.1 gene, T4 ORFan adja- <br />                                                 cent functionally unrelated 55.2, mutated 55.1 version ATG start <br />                                                 codon changed AGG. plasmid DNAs displayed identical topoisomer distri- <br />                                                 butions; difference slight increase negatively super- <br />                                                 coiled topoisomers plasmid wild-type 55.1 sequence (S4B S4C Fig). <br />                                                  increase marked case wild-type 55.2 plasmid. <br />                                                  conclude low-level non-toxic expression 55.2 effect regulation                                                 plasmid DNA supercoiling E. coli. <br />                                                    determine chromosomal DNA supercoiling affected 55.2 expression,                                                 measured level topA gyrA mRNA transcription genes deter- <br />                                                 mined,  supercoiling promoter regions. transcription rate topA  <br />                                                 creases increased negative supercoiling gyrA increases DNA <br />                                                 relaxation [60&#226;&#8364;&#8220;62]. Total RNA extracted bacteria harboring 55.2-coding <br />                                                 pBAD101 plasmid, vector, cultures growing exponentially ab- <br />                                                 sence arabinose. RNAse protection assays performed using topA gyrA probes (S4D <br />                                                 Fig), relative ratio topA gyrA-protected fragments determined densi- <br />                                                 tometric analysis. shown S4E Fig, topA gyrA ratio slightly lower bacteria <br />                                                 harboring 55.2 plasmid indicating low-level expression 55.2 did increase super- <br />                                                 coiling promoters somewhat reduced  <br />  <br />  <br />                                                 Low non-toxic level 55.2 increases copy number plasmids <br />                                                 replication controlled antisense RNA <br />                                                 analyzing effects DNA supercoiling mediated gp55.2, noticed                                                 DNA gp55.2 encoding plasmid systematically higher present vari- <br />                                                 ous control plasmids. pBAD vector origin replication derived ColEI,                                                 replication requires formation R-loop plasmid origin comple- <br />                                                 mentary RNAII serves primer DNA replication [63]. formation R-loops                                                 increased negative supercoiling inhibited action RNAse H1 Topo [59]. <br />                                                 quantify effect described  aliquots plasmid DNA prepared supercoil- <br />                                                 ing experiment described Fig 4A linearized restriction digestion, quantified aga- <br />                                                 rose gel comparison DNA standard, normalized total bacteria <br />                                                 used extract plasmid. results 55.2-coding plasmid                                                 seven times greater 55.2-mutant plasmid (Fig 4C). Similarly, overnight <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                                14 / 23 <br />                                                                                   Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           cultures DH5&#206;&#177; DB503 (S4F Fig), copy number 55.2-coding plasmid  <br />                                           creased fold compared control plasmid. determined                                           presence gp55.2 coding sequence operated cis gp55.2 acted                                           trans. Bacteria transformed 55.2-coding pBAD101 plasmid reporter plasmid <br />                                           p15A-derived replication origin, related ColEI replicon [64]. Plasmid <br />                                           DNA extracted equal amounts overnight cultures linearized restriction di- <br />                                           gestion. Compared control, 55.2-coding plasmid increased quantity                                           reporter plasmid strains tested (Fig 4D). demonstrates                                           low levels gp55.2 expression  arabinose present cultures 55.2 gene                                           low copy plasmid) affect plasmid copy number regulation trans. Interestingly, 55.2 <br />                                           did noticeably affect copy number pBAD101 (upper band Fig 4D), replica- <br />                                           tion RepA dependent [65,66]. results confirm interesting unexpected observa- <br />                                           tion gp55.2 significantly increases copy number plasmids replication                                           dependent R-loop formation. effect entirely compatible observed increased <br />                                           plasmid supercoiling. <br />  <br />  <br />                                           null mutation gene 55.2 decreases fitness phage T4 <br />                                           T4 ORFans probably &#226;&#8364;&#339;non-essential genes&#226;&#8364;? lie genomic regions                                           deleted noticeably affecting phage yield [24].  excluded <br />                                           absence gp55.2 function influence production T4 progeny stan- <br />                                           dard laboratory conditions. used T4 S [37] replace ATG initiation <br />                                           codon 55.2 ACA (threonine) codon genome K10 T4 strain; analo- <br />                                           gous substitution plasmid completely abolished 55.2-induced AraS phenotypes (data <br />                                           shown). plaque morphology resulting 55.2 mutant indistinguishable                                           K10 parent strain (data shown).  55.2 mutant phage exhibited                                           small reproducible minutes delay accumulation intracellular viral particles                                           minimal media. Nonetheless, eventually achieved burst size similar parental <br />                                           strain (Fig 5A). comparable delay observed complete media 37&#194;&#176;C (S5A Fig).                                           order determine small difference change overall fitness 55.2 <br />                                           mutant strain, compared growth parental strain successive growth <br />                                           cycles. experiments, mixture control mutant phages grown E. coli bac- <br />                                           teria; progeny initial growth cycle used perform second growth cycle                                           forth. Importantly, infections performed low moi (&lt; 0.1) minimize infection <br />                                           complementation wild type phage. initial phage mix contained 90% K10-55.2 <br />                                           mutant 10% K10 wild-type phages decreased fitness 55.2 mutant  <br />                                           ticipated. 19 growth cycles, 55.2 mutant phage represented 30% popu- <br />                                           lation (Fig 5B). simulation showed results obtained mutant phage <br />                                           16% growth disadvantage growth cycle (dotted line Fig 5B).  conclude <br />                                            55.2 &#226;&#8364;&#339;non-essential gene&#226;&#8364;?, confers non-trivial growth advan- <br />                                           tage T4 standard laboratory conditions. <br />  <br />  <br />                                           Discussion <br />                                           previously identified 55.2 T4 ORFan ectopic expression toxic unin- <br />                                           fected E. coli [29]. cloned medium copy pBAD plasmid (ColEI replicon), low- <br />                                           level &#226;&#8364;&#339;leaky&#226;&#8364;? expression 55.2 PBAD promoter absence arabinose caused                                           increased lag phase growth diluted liquid cultures decreased bacterial titer sat- <br />                                           uration. effects detected lower copy plasmid (pSC101 replicon)  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             15 / 23 <br />                                                                                       Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />  <br />                                           Fig 5. Loss gene 55.2 function reduces T4 phage fitness.   CR63 cells grown M9 medium                                           infected T4 K10 (wt) T4 K10-55.2 (55.2) moi 6 30&#194;&#176;C. Intracellular phage accumulation                                           followed indicated time points; free phages 25 min represented &lt; 12% total infective centers. <br />                                           Data represents mean ranges (wt) (55.2) independent experiments. (B) Competition <br />                                           experiment. mix T4 K10 T4 K10-55.2 initial ratio 1:9 grown E. coli CR63 M9S <br />                                           medium 37&#194;&#176;C low moi (&lt; 0.1) successive growth cycles. percentage 55.2+ phages                                           determined PCR digestion described Materials Methods. Data represent mean                                           standard deviation independent experiments; dotted line represents result simulation                                           55.2 mutant 16% disadvantage growth cycle. intracellular phage accumulation                                           single growth cycle conditions shown S5A Fig. <br />                                           doi:10.1371/journal.pone.0124309.g005 <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                       16 / 23 <br />                                                                                   Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           absence arabinose. Induced expression gp55.2 presence arabinose led irrevers- <br />                                           ible growth arrest plasmid backgrounds. <br />                                               analysis identified ways partially suppress 55.2-mediated bacterial kill- <br />                                           ing: multicopy expression Topo partial inhibition GyrB activity. results sug- <br />                                           gested gp55.2 acting inhibiting Topo activity increasing DNA gyrase <br />                                           activity. plasmid-based lethality assay clearly demonstrated low-level, normally non- <br />                                           toxic, expression gp55.2 compatible reduced Topo activity. incompatibility <br />                                           independently confirmed using strain topA expression regulated                                           IPTG. Interestingly, absence Topo III, second type IA topoisomerase E. coli, <br />                                           bacteria expressing low levels gp55.2 required higher expression Topo grow. Taken  <br />                                           gether, results support notion 55.2 inhibits DNA relaxation stimulates <br />                                           DNA supercoiling. <br />                                               vitro experiments demonstrated gp55.2 inhibits relaxing activity E. coli Topo                                           does affect introduction negative supercoils E. coli gyrase. inhibi- <br />                                           tion Topo activity gp55.2 result direct protein-protein interaction. Alterna- <br />                                           tively, gp55.2 binding DNA alter double helix conformation occlude                                           sites Topo binds. Increased Topo expression shown suppress toxici- <br />                                           ty associated Tn5 transposase (Tnp) overproduction [67,68]. direct interaction                                           proteins responsible vitro inhibition Topo activity                                           vivo suppression Tnp toxicity. failed detect interaction gp55.2 Topo                                           using vivo pull assays (data shown). opti- <br />                                           mal condition detect interaction, nonetheless arguments disfa- <br />                                           vor hypothesis. Firstly, alteration DNA topology required suppression <br />                                           Tnp toxicity overexpression partially defective mutant Topo suppress Tnp <br />                                           toxicity. Conversely, showed partial inhibition gyrase partly suppressed gp55.2 le- <br />                                           thality, reduce topA expression [60,62]. Second, molar ratio pro- <br />                                           tein Topo required observe vitro inhibition far lower Tnp gp55.2 (1&#226;&#8364;&#8220; <br />                                           10:1 vs. &gt;150:1). high concentration gp55.2 required Topo inhibition supports                                           alternative site occlusion/DNA conformation hypothesis. mechanism explains inhi- <br />                                           bition Topo nucleoid-associated protein (NAP) HU high protein DNA ratio <br />                                           [69]. EMSA showed gp55.2 Topo  35:1 molar ratio, bind simultaneously                                           plasmid DNA (S2B Fig).  occlusion place higher gp55.2 Topo <br />                                           ratio required detect inhibition (  150:1). tested gp55.2 alter DNA <br />                                           topology  incubation gp55.2 relaxed DNA (form ) presence                                           wheat germ Topo showed  studied conditions, phage protein constrain <br />                                           negative positive supercoils (S3C Fig). studies needed determine <br />                                           precise mechanism inhibition. possibility gp55.2 binds specific topo- <br />                                           logical features favored negative supercoiling, like single-stranded DNA regions                                           required DNA supercoil relaxation Topo [70]. hypothesis, inferred ca- <br />                                           pacity gp55.2 bind single-stranded M13 DNA, supported observations. <br />                                           Firstly, gp55.2 marginally affected activity wheat germ Topo does require <br />                                           single-stranded DNA substrate [6]. Secondly, low protein DNA ratio, gp55.2                                           stimulate inhibit Topo activity. ambivalent effect Topo activity previ- <br />                                           ously reported E. coli single-stranded DNA binding protein (SSB) stimulates non- <br />                                           covalent interaction Topo DNA stabilizing single-stranded region low protein                                           DNA ratio inhibits DNA relaxation higher ratio [71]. <br />                                               vivo distribution plasmid topoisomers indicates low-level expression 55.2 <br />                                           caused appearance negatively supercoiled relaxed topoisomers,  <br />                                           sulting broader distribution absence gp55.2. measurement  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             17 / 23 <br />                                                                                   Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           relative expression topA gyrA compatible unchanged slightly reduced chro- <br />                                           mosome superhelical density bacteria expressing gp55.2. results appear counterintui- <br />                                           tive complete defect Topo results increased negative supercoiling                                           plasmids bacterial chromosome [45].  limited gp55.2 produced <br />                                           uninduced PBAD promoter unlikely completely inhibit Topo activity. Actually, <br />                                           low-level 55.2 expression did block rapid relaxation induced inhibition DNA gyr- <br />                                           ase norfloxacin (S4G Fig), process entirely dependent Topo activity <br />                                           [72]. Furthermore, partial inhibition Topo compensated homeostatic <br />                                           regulation bacterial DNA supercoiling takes place enzyme activity gene <br />                                           expression levels [73]. note vitro Topo activity inhibited <br />                                           stimulated low gp55.2 DNA ratio resulting broader topoisomer distribution                                           absence gp55.2. vivo observations reconciled genetic                                           vitro data inhibition Topo gp55.2 stronger DNA substrates                                            DNA containing R-loops potential candidate.  low-level expression 55.2 <br />                                           caused strong increase copy number plasmids use R-loop replication, R- <br />                                           looped supercoiled plasmids better substrate Topo supercoiled plasmids                                           R-loops [74]. Topo prevents excessive R-loop formation topA null alleles incompati- <br />                                           ble deletions rnhA, encodes RNAse HI, enzyme degrades RNA strand <br />                                           R-loops [47,59,75].  severe phenotype induced low levels gp55.2                                           absence RNAse H1 (S1C Fig) consistent partial inhibition Topo reflect <br />                                           preferential inhibition Topo activity supercoiled DNA containing R-loops. <br />                                               vitro experiments suggest high levels gp55.2 completely inhibit Topo                                           vivo. supported suppression 55.2 lethality topA overexpression partial <br />                                           inactivation DNA gyrase.  thermosensitive mutation gyrB (gyrB203) used                                           study suppress growth defect topA null mutation 37&#194;&#176;C [47].  Topo inhibi- <br />                                           tion entirely explain gp55.2 killing.  consensus fact                                           Topo absolutely required growth [31,48]. Furthermore, plasmid-based le- <br />                                           thality assay experiments tunable topA allele showed low-level expression <br />                                           55.2 diminished growth capacity bacteria devoid Topo  interesting pos- <br />                                           sibility gp55.2 inhibits Topo Topo III. mentioned  topA topB <br />                                           double mutants non-viable. topoisomerase different physiological <br />                                           role, Topo III structurally functionally related Topo relax negatively super- <br />                                           coiled DNA vitro [76].  showed bacteria required Topo III tolerate low <br />                                           level gp55.2 Topo activity reduced. <br />                                               Altogether, simplest hypothesis explain data presented study gp55.2 <br />                                           inhibits Type IA topoisomerases.  rule reduction                                           DNA negative supercoiling required accommodate suppress consequence                                           gp55.2 interaction DNA, conjunction NAP. case, vitro  <br />                                           hibition Topo gp55.2 incidental consequence gp55.2 binding inter- <br />                                           mediate DNA structure required supercoil relaxation Type IA topoisomerase. <br />                                               Disruption 55.2 T4 phage showed gene bona fide non-essential gene,                                           intracellular accumulation virion particles slightly retarded null mutant. <br />                                            gp55.2 conserved T4-related phages infect different bacterial genera <br />                                           [25]. Competition assays showed  standard laboratory conditions, 55.2 provided <br />                                           easily detectable growth advantage.  55.2 pathways  <br />                                           quired optimal viral replication cycle. toxicity 55.2 characterization                                           mRNA early transcript [77] suggests role transition host phage metab- <br />                                           olism phage DNA replication. gp55.2 protein play role regulation su- <br />                                           percoiling phage. physically linear, phage genome shows transient <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                             18 / 23 <br />                                                                                  Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           acquisition negative superhelicity precedes T4 DNA synthesis requires host gyr- <br />                                           ase [78]. function DNA supercoiling T4 infection clear: hand, <br />                                           gyrase activity required T4 DNA synthesis [22,79], hand, burst <br />                                           size T4 phage reduced gyrB mutant host [22] release superhelical tension <br />                                           slightly reduces rate T4 DNA synthesis [72]. investigation required  <br />                                           termine fitness provided gp55.2 directly linked modulation host Topo ac- <br />                                           tivity infection. examined importance gp55.2 function absence                                           phage topoisomerase. intracellular growth curve 39am, 55.2 double mutant                                           similar 39 single mutant, showing extended eclipse period reduced burst size <br />                                           characteristic DNA delay mutants (S5B Fig).  mature phage particle accumulation <br />                                           slightly delayed double mutant, observed comparison 55.2 single mu- <br />                                           tant T4+. additive effect suggests gp55.2 T4 topoisomerase act indepen- <br />                                           dent biochemical pathways. <br />                                               Despite T4 studied phages, genome codes &gt;100 ORFans                                           biological functions,  largely unknown [24]. genes generally thought                                           non-essential largely regrouped called hyperplastic regions (HPR)                                           T4 genome. hypotheses proposed explain maintenance                                           evolution. genes contribute &#226;&#8364;&#339;molecular arms race&#226;&#8364;? resistance <br />                                           mechanisms new ways circumvent constantly evolving bacteria phages <br />                                           [80,81]. existence numerous ORFans far unique branch T4-like <br />                                           phage tree [25,82] suggests HPR serve breeding ground novo phage gene <br />                                           birth [26,83]. present work, recent characterization ORFans 55.1 [29]                                           39.2 [84], fit hypothesis: T4 ORFans modify host protein functions                                           fine tune bacterial metabolism permit optimal virion production  &#226;&#8364;&#339;molecular <br />                                           splint&#226;&#8364;? hypothesis [26]). Gp39.2 shifts open/close equilibrium host GroEL chaper- <br />                                            Gp55.1 impacts host folate metabolism blocks <span id='am-22' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-23' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-25' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> excision repair inter- <br />                                           acting FolD UvrA respectively, gp55.2 affects host control DNA <br />                                           supercoiling. deletion mutants genes perfectly viable E. <br />                                           coli K12 strains, Gp39.2 permits T4 growth E. coli groEL mutants  <br />                                           wise yield productive infections. Furthermore, demonstrated 55.2 55.1 mutants <br />                                           (Fig 5B data shown) detectably reduced fitness. conclude T4 ORFans <br />                                           major evolutionary role providing small evolutionary significant growth <br />                                           advantages non-optimal hosts environmental conditions. <br />  <br />  <br />  <br />  <br />                                           Supporting Information <br />                                           S1 Fig. Supplementary growth curves viability assays. <br />                                           (TIFF) <br />                                           S2 Fig. Supplementary EMSA. <br />                                           (TIFF) <br />                                           S3 Fig. Supplementary topoisomerase assays. <br />                                           (TIFF) <br />                                           S4 Fig. Supplementary analyses vivo DNA supercoiling plasmid copy number. <br />                                           (TIFF) <br />                                           S5 Fig. Supplementary analyses intracellular phage accumulation. <br />                                           (TIFF) <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                            19 / 23 <br />                                                                                       Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           S1 File. Supporting Materials Methods. Construction E. coli strains, T4 phage strains, <br />                                           plasmids, genomic library; methods used supporting figures; list plasmids <br />                                           used study. <br />                                           (DOCX) <br />  <br />  <br />                                           Acknowledgments <br />                                           thank Drs. C. Georgopoulos G. Panis, members laboratory, anonymous  <br />                                           viewers helpful discussions suggestions; Dr. H. Krisch critical readings manu- <br />                                           script Dr. J. Curran English editing; Marie Claire Teuscher L&#195;&#169;o Zangelmi                                           plasmid T4 strain construction, respectively; Dr  Stasiak help set                                           analysis chloroquine gel electrophoresis; Dr K. Kreuzer sharing knowledge                                           phage T4 topoisomerase. grateful Dr C.J. Rudolph Dr K.J. Marians plas- <br />                                           mids strains. <br />  <br />                                           Author Contributions <br />                                           Conceived designed experiments: YM DB. Performed experiments: YM FS. Ana- <br />                                           lyzed data: YM FS DB. Wrote paper: YM DB. <br />  <br />                                           References <br />                                            1.   Schvartzman JB, Stasiak (2004) topological view replicon. EMBO Rep 5: 256&#226;&#8364;&#8220;261. PMID: <br />                                                 14993926 <br />                                            2.   Liu LF, Wang JC (1987) Supercoiling DNA template transcription. Proc Natl Acad Sci U S <br />                                                 84: 7024&#226;&#8364;&#8220;7027. PMID: 2823250 <br />                                            3.   Higgins NP, Deng S, Pang  Stein R, Champion K, Manna D (2005) Domain behavior supercoil <br />                                                 dynamics bacterial chromosomes  Higgins NP, editor. Bacterial Chromosome. Washington, <br />                                                 DC: American Society Microbiology Press. pp. 133&#226;&#8364;&#8220;153. <br />                                            4.   Travers  Muskhelishvili G (2005) DNA supercoiling global transcriptional regulator entero- <br />                                                 bacterial growth? Nat Rev Microbiol 3: 157&#226;&#8364;&#8220;169. PMID: 15685225 <br />                                            5.   Stavans J, Oppenheim (2006) DNA-protein interactions bacterial chromosome architecture. <br />                                                 Phys Biol 3: R1&#226;&#8364;&#8220;10. PMID: 17200598 <br />                                            6.   Champoux JJ (2001) DNA topoisomerases: structure, function, mechanism. Annu Rev Biochem <br />                                                 70: 369&#226;&#8364;&#8220;413. PMID: 11395412 <br />                                            7.   Wang JC (2002) Cellular roles DNA topoisomerases: molecular perspective. Nat Rev Mol Cell Biol <br />                                                 3: 430&#226;&#8364;&#8220;440. PMID: 12042765 <br />                                            8.   Forterre P, Gadelle D (2009) Phylogenomics DNA topoisomerases: origin putative roles                                                 emergence modern organisms. Nucleic Acids Res 37: 679&#226;&#8364;&#8220;692. doi: 10.1093/nar/gkp032 PMID: <br />                                                 19208647 <br />                                            9.   Forterre P, Gribaldo S, Gadelle D, Serre MC (2007) Origin evolution DNA topoisomerases. Bio- <br />                                                 chimie 89: 427&#226;&#8364;&#8220;446. PMID: 17293019 <br />                                           10.   Kreuzer KN (1998) Bacteriophage T4, model understanding mechanism type II topo- <br />                                                 isomerase inhibitors. Biochim Biophys Acta 1400: 339&#226;&#8364;&#8220;347. PMID: 9748648 <br />                                           11.   Drlica K (1999) Mechanism fluoroquinolone action. Curr Opin Microbiol 2: 504&#226;&#8364;&#8220;508. PMID: <br />                                                 10508721 <br />                                           12.   Zechiedrich EL, Khodursky AB, Bachellier S, Schneider R, Chen D, Lilley DM, et al. (2000) Roles                                                 topoisomerases maintaining steady-state DNA supercoiling Escherichia coli. J Biol Chem 275: <br />                                                 8103&#226;&#8364;&#8220;8113. PMID: 10713132 <br />                                           13.   Perez-Cheeks BA, Lee C, Hayama R, Marians KJ (2012) role topoisomerase III Escherichia coli <br />                                                 chromosome segregation. Mol Microbiol 86: 1007&#226;&#8364;&#8220;1022. doi: 10.1111/mmi.12039 PMID: 23066834 <br />                                           14.   Liu LF, Liu CC, Alberts BM (1979) T4 DNA topoisomerase: new ATP-dependent enzyme essential                                                 initiation T4 bacteriophage DNA replication. Nature 281: 456&#226;&#8364;&#8220;461. PMID: 226889 <br />                                           15.   Stetler GL, King GJ, Huang WM (1979) T4 DNA-delay proteins, required specific DNA replication, <br />                                                 form complex ATP-dependent DNA topoisomerase activity. Proc Natl Acad Sci U S 76: <br />                                                 3737&#226;&#8364;&#8220;3741. PMID: 226976 <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                      20 / 23 <br />                                                                                        Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           16.   Epstein RH, Bolle  Steinberg CM, Kellenberger E, Boy la Tour E, Edgar RS, et al. (1964) Physio- <br />                                                 logical studies conditional lethal mutants bacteriophage T4D. Cold Spring Harbor Symp Quant <br />                                                 Biol 28: 375&#226;&#8364;&#8220;394. <br />                                           17.   Mufti S, Bernstein H (1974) DNA-delay mutants bacteriophage T4. J Virol 14: 860&#226;&#8364;&#8220;871. PMID: <br />                                                 4609406 <br />                                           18.   McCarthy D, Minner C, Bernstein H, Bernstein C (1976) DNA elongation rates growing point distri- <br />                                                 butions wild-type phage T4 DNA-delay amber mutant. J Mol Biol 106: 963&#226;&#8364;&#8220;981. PMID: <br />                                                 789903 <br />                                           19.   Mosig G (1994) Homologous recombination.  Karam JD, editor. Molecular biology bacteriophage <br />                                                 T4. Washington, D. C.: ASM press. pp. 54&#226;&#8364;&#8220;82. <br />                                           20.   Mosig G, Gewin J, Luder  Colowick N, Vo D (2001) recombination-dependent DNA replication <br />                                                 pathways bacteriophage T4, roles mutagenesis horizontal gene transfer. Proc Natl <br />                                                 Acad Sci U S 98: 8306&#226;&#8364;&#8220;8311. PMID: 11459968 <br />                                           21.   Kreuzer KN, Brister JR (2010) Initiation bacteriophage T4 DNA replication replication fork dy- <br />                                                 namics: review Virology Journal series bacteriophage T4 relatives. Virol J 7: 358. <br />                                                 doi: 10.1186/1743-422X-7-358 PMID: 21129203 <br />                                           22.   McCarthy D (1979) Gyrase-dependent initiation bacteriophage T4 DNA replication: interactions                                                 Escherichia coli gyrase novobiocin, coumermycin phage DNA-delay gene products. J Mol Biol <br />                                                 127: 265&#226;&#8364;&#8220;283. PMID: 372540 <br />                                           23.   Epstein RH, Bolle  Steinberg CM (2012) Amber mutants bacteriophage T4D: isolation ge- <br />                                                 netic characterization. Genetics 190: 833&#226;&#8364;&#8220;840. doi: 10.1534/genetics.112.138438 PMID: 22419076 <br />                                           24.   Miller ES, Kutter E, Mosig G, Arisaka F, Kunisawa T, Ruger W (2003) Bacteriophage T4 genome. <br />                                                 Microbiol Mol Biol Rev 67: 86&#226;&#8364;&#8220;156, table contents. PMID: 12626685 <br />                                           25.   Petrov VM, Ratnayaka S, Nolan JM, Miller ES, Karam JD (2010) Genomes T4-related bacterio- <br />                                                 phages windows microbial genome evolution. Virol J 7: 292. doi: 10.1186/1743-422X-7-292 <br />                                                 PMID: 21029436 <br />                                           26.   Comeau  Hatfull GF, Krisch HM, Lindell D, Mann NH, Prangishvili D (2008) Exploring prokaryot- <br />                                                 ic virosphere. Res Microbiol 159: 306&#226;&#8364;&#8220;313. doi: 10.1016/j.resmic.2008.05.001 PMID: 18639443 <br />                                           27.   Brussow H, Hendrix RW (2002) Phage genomics: small beautiful. Cell 108: 13&#226;&#8364;&#8220;16. PMID: 11792317 <br />                                           28.   Comeau  Tremblay D, Moineau S, Rattei T, Kushkina AI, Tovkach FI, et al. (2012) Phage morpholo- <br />                                                 gy recapitulates phylogeny: comparative genomics new group myoviruses. PLoS 7: <br />                                                 e40102. doi: 10.1371/journal.pone.0040102 PMID: 22792219 <br />                                           29.   Mattenberger Y, Mattson S, Metrailler J, Silva F, Belin D (2011) 55.1, gene unknown function                                                 phage T4, impacts Escherichia coli folate metabolism blocks DNA repair NER. Mol Micro- <br />                                                 biol 82: 1406&#226;&#8364;&#8220;1421. doi: 10.1111/j.1365-2958.2011.07897.x PMID: 22029793 <br />                                           30.   Miller JH (1992) short course bacterial genetics. Cold Spring Harbor, NY: Cold Spring Harbor Lab- <br />                                                 oratory Press. <br />                                           31.   Stockum  Lloyd RG, Rudolph CJ (2012) viability Escherichia coli cells lacking DNA topo- <br />                                                 isomerase  BMC Microbiol 12: 26. doi: 10.1186/1471-2180-12-26 PMID: 22373098 <br />                                           32.   Daegelen P, Studier FW, Lenski  Cure S, Kim JF (2009) Tracing ancestors relatives Escheri- <br />                                                 chia coli B, derivation B strains REL606 BL21(DE3). J Mol Biol 394: 634&#226;&#8364;&#8220;643. doi: 10. <br />                                                 1016/j.jmb.2009.09.022 PMID: 19765591 <br />                                           33.   Boyd D, Manoil C, Beckwith J (1987) Determinants membrane protein topology. Proc Natl Acad Sci <br />                                                 USA 84: 8525&#226;&#8364;&#8220;8529. PMID: 3317413 <br />                                           34.   Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, et al. (2006) Construction Escherichia <br />                                                 coli K-12 frame, single-gene knockout mutants: Keio collection. Mol Syst Biol 2: 2006 0008. <br />                                           35.   Casadaban MJ, Cohen SN (1980) Analysis gene control signals DNA fusion cloning                                                 Escherichia coli. J Mol Biol 138: 179&#226;&#8364;&#8220;207. PMID: 6997493 <br />                                           36.   Casadaban MJ (1976) Transposition fusion lac genes selected promoters Escherichia <br />                                                 coli using bacteriophage lambda Mu. J Mol Biol 104: 541&#226;&#8364;&#8220;555. PMID: 781293 <br />                                           37.   Selick  Kreuzer KN, Alberts BM (1988) bacteriophage T4 insertion/substitution vector  <br />                                                 method introducing site-specific mutations virus chromosome. J Biol Chem 263: 11336&#226;&#8364;&#8220; <br />                                                 11347. PMID: 3403529 <br />                                           38.   Belin D, Vassalli J-D, Combepine C, Godeau F, Nagamine Y, Reich E, et al. (1985) Cloning, nucleotide <br />                                                 sequencing expression cDNAs encoding mouse urokinase-type plasminogen activator. Eur- <br />                                                 JBiochem 148: 225&#226;&#8364;&#8220;232. <br />                                           39.   Sambrook J, Fritsch EF, Maniatis T (1990) Molecular cloning. laboratory manual.: Cold Spring Har- <br />                                                 bor Laboratory Press. <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                        21 / 23 <br />                                                                                        Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           40.   Reckinger AR, Jeong KS, Khodursky AB, Hiasa H (2007) RecA stimulate relaxation activity                                                 topoisomerase  Molecular basis topoisomerase-mediated genome-wide transcriptional responses <br />                                                 Escherichia coli. Nucleic Acids Res 35: 79&#226;&#8364;&#8220;86. PMID: 17151069 <br />                                           41.   Clark DJ, Leblanc B (2009) Analysis DNA supercoiling induced DNA-protein Interaction.  Moss <br />                                                 T, Leblanc B, editors. Methods Molecular Biology: Humana Press. pp. 523&#226;&#8364;&#8220;536. doi: 10.1007/978-1- <br />                                                 60327-015-1_30 PMID: 19378184 <br />                                           42.   Cameron AD, Stoebel DM, Dorman CJ (2011) DNA supercoiling differentially regulated environ- <br />                                                 mental factors FIS Escherichia coli Salmonella enterica. Mol Microbiol 80: 85&#226;&#8364;&#8220;101. doi: 10. <br />                                                 1111/j.1365-2958.2011.07560.x PMID: 21276095 <br />                                           43.   Bieler S, Silva F, Soto C, Belin D (2006) Bactericidal activity secreted nonsecreted microcin <br />                                                 E492 requires mannose permease. J Bacteriol 188: 7049&#226;&#8364;&#8220;7061. PMID: 17015644 <br />                                           44.   Bost S, Silva F, Belin D (1999) Transcriptional activation ydeA, encodes member                                                 major facilitator superfamily, interferes arabinose accumulation induction Escherichia <br />                                                 coli arabinose PBAD promoter. J Bacteriol 181: 2185&#226;&#8364;&#8220;2191. PMID: 10094697 <br />                                           45.   Pruss GJ, Manes SH, Drlica K (1982) Escherichia coli DNA topoisomerase mutants: increased super- <br />                                                 coiling corrected mutations near gyrase genes. Cell 31: 35&#226;&#8364;&#8220;42. PMID: 6297751 <br />                                           46.   DiNardo S, Voelkel KA, Sternglanz R, Reynolds AE, Wright (1982) Escherichia coli DNA topoisomer- <br />                                                 ase mutants compensatory mutations DNA gyrase genes. Cell 31: 43&#226;&#8364;&#8220;51. PMID: 6297752 <br />                                           47.   Drolet M, Phoenix P, Menzel R, Masse E, Liu LF, Crouch RJ (1995) Overexpression RNase H partial- <br />                                                 ly complements growth defect Escherichia coli &#206;&#8221;topA mutant: R-loop formation major <br />                                                 problem absence DNA topoisomerase  Proc Natl Acad Sci U S 92: 3526&#226;&#8364;&#8220;3530. PMID: <br />                                                 7536935 <br />                                           48.   Stupina VA, Wang JC (2005) Viability Escherichia coli topA mutants lacking DNA topoisomerase  J <br />                                                 Biol Chem 280: 355&#226;&#8364;&#8220;360. PMID: 15522872 <br />                                           49.   Guzman LM, Belin D, Carson MJ, Beckwith J (1995) Tight regulation, modulation, high-level ex- <br />                                                 pression vectors containing arabinose PBAD promoter. J Bacteriol 177: 4121&#226;&#8364;&#8220;4130. PMID: <br />                                                 7608087 <br />                                           50.   Zhu CX, Qi HY, Tse-Dinh YC (1995) Mutation Cys662 Escherichia coli DNA topoisomerase  <br />                                                 fers temperature sensitivity change DNA cleavage selectivity. J Mol Biol 250: 609&#226;&#8364;&#8220;616. PMID: <br />                                                 7623379 <br />                                           51.   Zhu Q, Pongpech P, DiGate RJ (2001) Type topoisomerase activity required proper chromosom- <br />                                                 al segregation Escherichia coli. Proc Natl Acad Sci U S 98: 9766&#226;&#8364;&#8220;9771. PMID: 11493711 <br />                                           52.   Gellert M, O&apos;Dea MH, Itoh T, Tomizawa J (1976) Novobiocin coumermycin inhibit DNA supercoiling <br />                                                 catalyzed DNA gyrase. Proc Natl Acad Sci U S 73: 4474&#226;&#8364;&#8220;4478. PMID: 794878 <br />                                           53.   Drlica K, Snyder M (1978) Superhelical Escherichia coli DNA: relaxation coumermycin. J Mol Biol <br />                                                 120: 145&#226;&#8364;&#8220;154. PMID: 347091 <br />                                           54.   Gormley NA, Orphanides G, Meyer  Cullis PM, Maxwell (1996) interaction coumarin antibiot- <br />                                                 ics fragments DNA gyrase B protein. Biochemistry 35: 5083&#226;&#8364;&#8220;5092. PMID: 8664301 <br />                                           55.   Hardy CD, Cozzarelli NR (2003) Alteration Escherichia coli topoisomerase IV novobiocin resis- <br />                                                 tance. Antimicrob Agents Chemother 47: 941&#226;&#8364;&#8220;947. PMID: 12604525 <br />                                           56.   Shure M, Pulleyblank  Vinograd J (1977) problems eukaryotic prokaryotic DNA packag- <br />                                                 ing vivo conformation posed superhelix density heterogeneity. Nucleic Acids Res 4: 1183&#226;&#8364;&#8220; <br />                                                 1205. PMID: 197488 <br />                                           57.   Vetcher AA, McEwen AE, Abujarour R, Hanke  Levene SD (2010) Gel mobilities linking-number <br />                                                 topoisomers dependence DNA helical repeat elasticity. Biophys Chem 148: 104&#226;&#8364;&#8220;111. <br />                                                 doi: 10.1016/j.bpc.2010.02.016 PMID: 20346570 <br />                                           58.   Richardson JP (1991) Preventing synthesis unused transcripts Rho factor. Cell 64: 1047&#226;&#8364;&#8220; <br />                                                 1049. PMID: 2004415 <br />                                           59.   Drolet M (2006) Growth inhibition mediated excess negative supercoiling: interplay                                                 transcription elongation, R-loop formation DNA topology. Mol Microbiol 59: 723&#226;&#8364;&#8220;730. PMID: <br />                                                 16420346 <br />                                           60.   Tse-Dinh YC, Beran RK (1988) Multiple promoters transcription Escherichia coli DNA topo- <br />                                                 isomerase gene regulation DNA supercoiling. J Mol Biol 202: 735&#226;&#8364;&#8220;742. PMID: 2845101 <br />                                           61.   Menzel R, Gellert M (1983) Regulation genes E. coli DNA gyrase: homeostatic control DNA <br />                                                 supercoiling. Cell 34: 105&#226;&#8364;&#8220;113. PMID: 6309403 <br />                                           62.   Peter BJ, Arsuaga J, Breier  Khodursky AB, Brown PO, Cozzarelli NR (2004) Genomic transcrip- <br />                                                 tional response loss chromosomal supercoiling Escherichia coli. Genome Biol 5: R87. PMID: <br />                                                 15535863 <br />  <br />  <br />  <br />  <br /> PLOS | DOI:10.1371/journal.pone.0124309 April 14, 2015                                                                                        22 / 23 <br />                                                                                       Toxic T4 ORFan Gene Encodes Inhibitor Topoisomerase  <br />  <br />  <br />                                           63.   del Solar G, Giraldo R, Ruiz-Echevarria MJ, Espinosa M, Diaz-Orejas R (1998) Replication control <br />                                                 circular bacterial plasmids. Microbiol Mol Biol Rev 62: 434&#226;&#8364;&#8220;464. PMID: 9618448 <br />                                           64.   Selzer G, Som T, Itoh T, Tomizawa J (1983) origin replication plasmid p15A comparative <br />                                                 studies nucleotide sequences origin related plasmids. Cell 32: 119&#226;&#8364;&#8220;129. PMID: <br />                                                 6186390 <br />                                           65.   Armstrong KA, Acosta R, Ledner E, Machida Y, Pancotto M, McCormick M, et al. (1984) 37 X 10(3) <br />                                                 molecular weight plasmid-encoded protein required replication copy number control                                                 plasmid pSC101 temperature-sensitive derivative pHS1. J Mol Biol 175: 331&#226;&#8364;&#8220;348. PMID: <br />                                                 6327996 <br />                                           66.   Churchward G, Linder P, Caro L (1983) nucleotide sequence replication maintenance func- <br />                                                 tions encoded plasmid pSC101. Nucleic Acids Res 11: 5645&#226;&#8364;&#8220;5659. PMID: 6310509 <br />                                           67.   Yigit H, Reznikoff WS (1998) Escherichia coli DNA topoisomerase suppression killing Tn5 <br />                                                 transposase overproduction: topoisomerase modulates Tn5 transposition. J Bacteriol 180: 5866&#226;&#8364;&#8220; <br />                                                 5874. PMID: 9811643 <br />                                           68.   Yigit H, Reznikoff WS (1999) Escherichia coli DNA topoisomerase copurifies Tn5 transposase, <br />                                                 Tn5 transposase inhibits topoisomerase  J Bacteriol 181: 3185&#226;&#8364;&#8220;3192. PMID: 10322021 <br />                                           69.   Ghosh S, Mallick B, Nagaraja V (2014) Direct regulation topoisomerase activity nucleoid-associ- <br />                                                 ated protein. Nucleic Acids Res 42: 11156&#226;&#8364;&#8220;11165. doi: 10.1093/nar/gku804 PMID: 25200077 <br />                                           70.   Kirkegaard K, Wang JC (1985) Bacterial DNA topoisomerase relax positively supercoiled DNA <br />                               <br /> </body></html>