<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Ecosal Plus. 2014 ; 2014: . doi:10.1128/ecosalplus.ESP-0009-2013. <br />  <br />  <br />  <br />                            EcoCyc Database <br />                            Peter D. Karp1, Daniel Weaver1, Suzanne Paley1, Carol Fulcher1, Aya Kubo1, Anamika <br />                            Kothari1, Markus Krummenacker1, Pallavi Subhraveti1, Deepika Weerasinghe1, Socorro <br />                            Gama-Castro2, Araceli M. Huerta2, Luis Mu&#195;&#177;iz-Rascado2, C&#195;&#169;sar Bonavides-Martinez2, <br />                            Verena Weiss2, Martin Peralta-Gil2, Alberto Santos-Zavaleta2, Imke Schr&#195;&#182;der3,4, Amanda <br />                            Mackie5, Robert Gunsalus3,4, Julio Collado-Vides2, Ingrid M. Keseler1, Ian Paulsen5 <br />                            1Bioinformatics     Research Group, SRI International, Menlo Park, USA <br />                            2Programade Gen&#195;&#179;mica Computacional, Centro Ciencias Gen&#195;&#179;micas, Universidad Nacional <br />                            Aut&#195;&#179;noma M&#195;&#169;xico, P. 565  Cuernavaca, Morelos 62100, M&#195;&#169;xico <br />                            3Department Microbiology, Immunology, Molecular Genetics, University California, Los <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Angeles, CA 90095, USA <br />                            4UCLA     Institute Genomics Proteomics, University California, Los Angeles, CA 90095, <br />                            USA <br />                            5Department Chemistry Biomolecular Sciences, Macquarie University, Sydney, New South <br />                            Wales 2109, Australia <br />  <br />                            Abstract <br />                                 EcoCyc bioinformatics database available EcoCyc.org describes genome                                 biochemical machinery Escherichia coli K-12 MG1655. long-term goal project                                 complete molecular catalog E. coli cell, functions                                 molecular parts, facilitate level understanding E. coli. EcoCyc electronic <br />                                 reference source E. coli biologists, biologists work related microorganisms. <br />                                 database includes information pages E. coli gene, metabolite, reaction, operon,                                 metabolic pathway. database includes information E. coli gene essentiality, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 nutrient conditions support growth E. coli. web site downloadable <br />                                 software contain tools analysis high-throughput datasets. addition, steady-state <br />                                 metabolic flux model generated new version EcoCyc. model predict <br />                                 metabolic flux rates, nutrient uptake rates, growth rates different gene knockouts                                 nutrient conditions. chapter provides detailed description data content EcoCyc, <br />                                 procedures content generated. <br />  <br />  <br />                            1 EcoCyc Overview <br />                                              EcoCyc1 bioinformatics database describes genome biochemical <br />                                              machinery E. coli K-12 MG1655. project&#226;&#8364;&#8482;s long-term goal describing complete <br />                                              molecular catalog E. coli cell, functions molecular parts, <br />  <br />  <br />  <br />                            1&#226;&#8364;&#339;EcoCyc&#226;&#8364;? pronounced &#226;&#8364;&#339;eeko-sike.&#226;&#8364;? sounds like &#226;&#8364;&#339;ecology&#226;&#8364;? like &#226;&#8364;&#339;encyclopedia&#226;&#8364;?. <br />                             Karp et al.                                                                                             Page 2 <br />  <br />  <br />                                          facilitate level understanding E. coli. EcoCyc electronic reference source <br />                                          E. coli biologists researchers work E. coli related <br />                                          microorganisms. addition database, steady-state metabolic flux model NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          available, generated new version EcoCyc. <br />  <br />                                          chapter provides overview EcoCyc&#226;&#8364;&#8482;s data content procedures                                          data enter EcoCyc. <br />  <br />                                          EcoCyc accelerates science. EcoCyc designed different modes interactive <br />                                          use EcoCyc.org web site conjunction downloadable Pathway <br />                                          Tools [1] software (Section 13 lists resources available assist users learning web <br />                                          site software)): <br />  <br />                                              &#226;&#8364;&#162;    EcoCyc encyclopedic reference providing information biological <br />                                                   roles E. coli genes, metabolites, pathways. Visualization tools,                                                   genome browser, metabolic map display, regulatory network diagram, aid                                                   comprehension complex data. <br />  <br />                                              &#226;&#8364;&#162;    EcoCyc facilitates analysis high-throughput data gene-expression                                                   metabolomics data tools enrichment analysis, visualizing omics data <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   metabolic map diagram, complete genome diagram, regulatory network <br />                                                   diagram. <br />  <br />                                              &#226;&#8364;&#162;    EcoCyc metabolic flux model predict growth growth wildtype                                                   knock E. coli strains different nutrient conditions. <br />  <br />                                          Users EcoCyc fall different groups. Experimental biologists use EcoCyc                                          encyclopedic reference genes, pathways, regulation, use omics-data <br />                                          analysis tools analyze gene-expression metabolomics data. Examples papers citing <br />                                          EcoCyc analysis functional genomics data include: [2, 3, 4, 5, 6]. <br />  <br />                                          EcoCyc data structured sophisticated ontology amenable                                          computational analyses, EcoCyc enables scientists ask computational questions spanning <br />                                          entire genome E. coli, known metabolic network E. coli, known transport <br />                                          complement E. coli, known genetic regulatory network E. coli, combinations <br />                                          thereof. Past work includes use EcoCyc develop methods studying path lengths <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          metabolic networks [7, 8, 9]; studies relating protein structure metabolic <br />                                          network [10, 11]; analysis E. coli regulatory network [12, 13]. <br />  <br />                                          development new bioinformatics methods requires high-quality, gold-standard <br />                                          datasets training validation methods. EcoCyc used gold- <br />                                          standard dataset development genome-context methods predicting gene <br />                                          function [14, 15], operon-prediction methods [16, 17], prediction promoters                                          transcription start sites [18, 19], regulatory network reconstruction [20], prediction <br />                                          functional direct protein-protein interactions [21, 22, 23]. EcoCyc metabolic data <br />                                          used studies concerning predicted metabolic networks growth prediction <br />                                          [24, 25], model checking symbiotic bacteria&#226;&#8364;&#8482;s metabolic network [26]. <br />  <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                             Page 3 <br />  <br />  <br />                                          Metabolic engineers alter microbes produce biofuels, industrial chemicals,                                          pharmaceuticals; grade toxic pollutants; sequester carbon [27, 28, 29]. <br />                                          Metabolic engineers use E. coli host organism consult EcoCyc aid NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          optimizing production end product better standing                                          metabolic network regulation, predict undesirable effects metabolic <br />                                          alteration. Metabolic engineering studies using EcoCyc include [30, 31, 32]. <br />  <br />                                          According Thomson Reuters Web Knowledge citation index, August 2013, <br />                                          23 EcoCyc RegulonDB papers authored 1997 cited 2,395 publications <br />                                          1997&#226;&#8364;&#8220;2013. According Google Analytics, approximately 100,000 visitors query                                          EcoCyc website year, generating 177,000 object page views month average                                          2012. <br />  <br />                                          EcoCyc data available download multiple file formats  http://biocyc.org/ <br />                                          download.shtml) queried programmatically web services  http:// <br />                                          biocyc.org/web-services.shtml). <br />  <br />                                          Pathway Tools software underlies EcoCyc [1] specific E. coli,                                          applied manage genomic biochemical data thousands organisms. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2 Overview EcoCyc Data Content <br />                                          EcoCyc covers broad array data types. Key understanding EcoCyc data                                          presentation EcoCyc website Pathway Tools notion database class, <br />                                          describes specific type data. example, class Genes provides database <br />                                          <span id='am-27' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> gene, including attributes (e.g., starting <span id='am-33' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-34' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-35' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> position                                          genome) relationships (e.g., linkage gene gene product) class. <br />                                          specific gene EcoCyc stored single database object, frame,                                          instance class Genes. <br />  <br />                                          mapping exists EcoCyc classes data pages                                          EcoCyc website, data page typically integrates information multiple <br />                                          classes. example, pathway data page integrates information objects                                          classes Pathways, Reactions, Genes, Proteins, Chemicals. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Genome <br />                                          EcoCyc contains complete genome sequence E. coli describes nucleotide <br />                                          position function known protein-coding RNA-coding E. coli genes. Genome- <br />                                          related classes populated EcoCyc include Genes, Pseudo-Genes, Promoters, <br />                                          DNA-Binding-Sites, REP-Elements. Gene Ontology   terms assigned genes <br />                                          EcoCyc curators import terms UniProt [33]. EcoCyc data                                          essentiality E. coli genes described Section 6. <br />  <br />                               Proteome <br />                                          EcoCyc describes known monomers multimeric protein complexes E. coli. <br />                                          EcoCyc contains extensive annotation features E. coli proteins,  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                Page 4 <br />  <br />  <br />                                            phosphorylation sites, metal-<span id='am-9' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-10' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> binding sites, enzyme active sites, assigned EcoCyc <br />                                            curators imported UniProt. Relevant classes EcoCyc include Polypeptides <br />                                            Protein-Complexes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               RNAome <br />                                            EcoCyc describes known RNAs protein-RNA complexes E. coli. Relevant classes <br />                                            EcoCyc include RNAs, rRNAs, Regulatory-RNAs. Note EcoCyc does                                            explicitly represent messenger RNAs. <br />  <br />                               Regulation <br />                                            EcoCyc contains complete description regulatory network organism. <br />                                            covers E. coli operons, promoters, transcription factors, transcription-factor binding sites, <br />                                            attenuators, small-RNA regulators, substrate-level regulation E. coli <br />                                            enzymes. molecular regulatory interaction described instance class <br />                                            Regulation, subclasses different types regulation. <br />  <br />                               Metabolism <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            EcoCyc describes known metabolic signal-transduction pathways E. coli.                                            describes metabolic enzyme E. coli including cofactors, activators, inhibitors,                                            subunit structure. <br />  <br />                               Membrane Transporters <br />                                            EcoCyc annotates E. coli transport proteins associated transport reactions                                            mediate. <br />  <br />                               Growth Observations <br />                                            EcoCyc integrates data growth E. coli different growth conditions,                                            described Section 5. <br />  <br />                               Database Links <br />                                            EcoCyc linked biological databases containing protein nucleic acid sequence <br />                                            data, bibliographic data, protein structures, descriptions different E. coli strains. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3 Literature-Based Curation <br />                                            Curation process manually refining updating bioinformatics database.                                            EcoCyc project uses literature-based curation approach database updates                                            based evidence experimental literature. EcoCyc largely <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> respect                                            curation activities. October 2013, EcoCyc encodes information                                            25,000 publications. staff time curators updates annotation E. coli <br />                                            genome ongoing basis. <br />  <br />                                            transcriptional regulatory information EcoCyc RegulonDB curated                                            group Dr. Julio Collado-Vides UNAM;  databases include  <br />  <br />  <br />                                                Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                            Page 5 <br />  <br />  <br />                                          data content transcriptional regulation gene expression. actual data curation occurs <br />                                          EcoCyc, information periodically propagated RegulonDB. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Curators collect gene, protein, pathway, compound names synonyms. classify <br />                                          genes gene products using Gene Ontology [34] MultiFun [35] ontologies,                                          classify pathways Pathway Tools pathway ontology. Protein complex <br />                                          components stoichiometry subunits captured; cellular localization                                          polypeptides protein complexes entered, experimentally determined protein <br />                                          molecular weights; enzyme activities enzyme prosthetic groups, cofactors, <br />                                          activators, inhibitors captured. Operon structure gene regulation information                                          encoded. <br />  <br />                                          Curators author textual summaries extensive citations. summaries                                          proteins, RNAs, pathways, operons, curators capture additional information                                          captured highly structured database fields EcoCyc. example, curators <br />                                          use free-text summary sections overall function gene product,                                          phenotypes caused mutation, depletion, overproduction gene product;                                          known genetic interactions; protein domain architecture structural studies; similarity <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          proteins; functional complementation experiments described. <br />                                          Summaries used note cases published reports present <br />                                          contradictory results. cases, viewpoints presented proper <br />                                          attribution. approach strives ensure information lost. <br />  <br />                                          EcoCyc entries generally updated new literature available. Regular <br />                                          PubMed searches used generate lists potentially curatable publications,                                          evaluated prioritized curation. Papers containing newly identified functions                                          gene products, substantial advances understanding functions known gene <br />                                          products, given highest priority curation. Pathway Tools software <br />                                          continues evolve enable addition new data types, older entries                                          updated systematic fashion (e.g., enzyme metabolic pathway) time allows. <br />  <br />                               4 Statistics EcoCyc Content <br />                                          Tables 1, 2, 3, 4 present statistics EcoCyc content. listed numbers current                                          version 17.5, released October 2013. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               5 Conditions E. coli Growth Non-Growth <br />                                          2011, EcoCyc incorporates media shown experimentally support                                          support growth wildtype knock strains E. coli K-12. work                                          goals. assemble comprehensive encyclopedia E. coli growth conditions <br />                                          experimentalists. spectrum environmental conditions supporting growth                                          bacterium important phenotypic traits. expect understand                                          functions genes organism unless understand range environments                                          cell grow. Second, comprehensive collection E. coli growth media                                          drive accurate systems biology modeling E. coli. larger set growth media <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                          Page 6 <br />  <br />  <br />                                          computational models validated, accurate                                          comprehensive models  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          EcoCyc captures approximately 20 media commonly used E. coli laboratories; <br />                                          growth data provided media. EcoCyc records results high- <br />                                          throughput experiments using Biolog Phenotype Microarrays (PMs), measure cell <br />                                          respiration sensitive indicator microbial growth [36]. commercially available PM <br />                                          microorganisms provides comprehensive set phenotype tests including <br />                                          information ability metabolize 190 carbon (C) compounds, 95 nitrogen (N) <br />                                          compounds, 59 phosphorus (P) compounds, 35 sulfur (S) compounds. EcoCyc currently <br />                                          documents sets PM data following sources: <br />  <br />                                              &#226;&#8364;&#162;    B. Bochner X. Lei, personal communication, 2012. <br />  <br />                                                   Strain: E. coli K&#226;&#8364;&#8220;12 BW30270 (rph+ (RNase PH) derivative MG1655;                                                   strains PyrE deficiency. fnr+  according                                                   Datsenko Wanner, unpublished results.) <br />  <br />                                                   dataset includes aerobic growth observations complement C, N, <br />                                                   P, S compounds included PM plus growth observations <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   95 C sources anaerobic conditions. <br />  <br />                                              &#226;&#8364;&#162;    &#226;&#8364;&#339;Genome Scale Reconstruction Salmonella Metabolic Model&#226;&#8364;? <br />  <br />                                                   AbuOun et al. 2009 [37] <br />  <br />                                                   Strain: E. coli K&#226;&#8364;&#8220;12 MG1655 (American Type Culture Collection 700926) <br />  <br />                                                   dataset includes growth observations complement C, N, P, S <br />                                                   compounds aerobic conditions. Bacteria pre-grown LB agar prior                                                   inoculation Biolog plates incubation 37&#194;&#176;C 26 hours. Omnilog <br />                                                   instrument  specialized incubator plus reader) used data collection                                                   analysis. <br />  <br />                                              &#226;&#8364;&#162;     Evolution Metabolic Networks E. coli&#226;&#8364;? <br />  <br />                                                   Baumler et al. 2011 [38] <br />  <br />                                                   Strain: E. coli K&#226;&#8364;&#8220;12 MG1655 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   dataset consists growth observations 95 C compounds aerobic                                                   anaerobic conditions. Bacteria pre-grown Biolog Universal Growth Agar <br />                                                   plus sheep blood (BUG-S), prior inoculation Biolog plates incubation                                                   37&#194;&#176;C. Growth monitored measuring optical density 600nm readings <br />                                                   taken 3, 6, 12, 24, 48 hours (D. Baumler, personal communication). <br />  <br />                                              &#226;&#8364;&#162;    Mackie et al. 2013 [39] <br />  <br />                                                   Strain: E. coli K&#226;&#8364;&#8220;12 MG1655 (Coli Genetic Stock Center 7740). <br />  <br />                                                   data set consists growth observations complement C, N, P <br />                                                   S compounds aerobic conditions. Bacteria pre-grown LB <br />  <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                           Page 7 <br />  <br />  <br />                                                   R2A agar prior inoculation Biolog plates incubation 37&#194;&#176;C 48 <br />                                                   hours. Omnilog instrument used data collection analysis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              &#226;&#8364;&#162;    &#226;&#8364;&#339;Comparative Multi-Omics Systems Analysis Escherichia coli strains B                                                   K-12&#226;&#8364;? <br />  <br />                                                   Yoon et al. 2012 [40] <br />  <br />                                                   Strain: E. coli K&#226;&#8364;&#8220;12 MG1655 <br />  <br />                                                   data set consists growth observations complement C, N, P <br />                                                   S compounds aerobic conditions. Bacteria pregrown BUG-S <br />                                                   agar prior inoculation Biolog plates incubation 37&#194;&#176;C 48 hours.                                                   Omnilog instrument used data collection analysis. <br />  <br />                                          Data growth conditions accessed EcoCyc website invoking menu <br />                                          command Search &#226;&#8224;&#8217; Growth Media clicking button  Growth Media                                          Organism.&#226;&#8364;? Individual media shown initial table; PM data shown                                          following tables. coloring box indicates degree growth observed                                          condition. levels growth recorded: growth, low growth, growth                                           legend indicates colors associated level growth). Click growth <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          medium request page describing composition, genes essential                                          essential growth condition. <br />  <br />                               6 Essential Gene Information <br />                                          2011, EcoCyc incorporates large-scale datasets gene essentiality E. coli. <br />                                          Gene essentiality information useful  <br />                                              &#226;&#8364;&#162;    Predicting antibiotic targets pathogenic bacteria. <br />  <br />                                              &#226;&#8364;&#162;    Guiding design minimal genomes. <br />  <br />                                              &#226;&#8364;&#162;    Validating genome-scale metabolic flux models. Model predictions                                                   compared experimental data recorded EcoCyc assess model accuracy. <br />  <br />                                              &#226;&#8364;&#162;    Providing clues regarding functions genes unknown function,                                                   essentiality varies depending conditions growth. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          EcoCyc incorporates data essentiality following publications: <br />  <br />                                              &#226;&#8364;&#162;    &#226;&#8364;&#339;Experimental Determination Level Analysis Essential Genes                                                   Escherichia coli MG1655&#226;&#8364;? <br />  <br />                                                   Gerdes et al. [41] <br />  <br />                                                   Strain: E. coli K&#226;&#8364;&#8220;12 MG1655 (F - &#206;&#187; - ilvG rfb-50 rph-1) <br />  <br />                                                   study used genetic footprinting technique Tn5 based transposome <br />                                                   reported unambiguous assessment approximately 87% E. coli <br />                                                   ORFs essentiality. 626 genes identified essential aerobic growth                                                   rich media 3126 genes dispensable. Note inability obtain                                                   insertion mutant using cases reflection non- <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                           Page 8 <br />  <br />  <br />                                                targeted nature transposon insertion reflection gene essentiality. <br />                                                technical reasons 327 genes classified study                                                ambiguous regard essentiality. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          &#226;&#8364;&#162;     Construction Escherichia coli K-12 Frame, Single-Gene Knockout Mutants: <br />                                                Keio Collection Baba et al. [42] corrections [43] <br />  <br />                                                Strain: E. coli K&#226;&#8364;&#8220;12 BW25113 (rpoS  rph-1 &#206;&#187; - rrnB3 &#206;&#8221;lacZ4787 hsdR514 <br />                                                &#206;&#8221;(araBAD)567 &#206;&#8221;(rhaBAD)568 rph-1) <br />  <br />                                                study created 3985 frame, single gene deletion mutants using lambda <br />                                                RED recombinase  303 genes unable disrupted                                                predicted essential growth rich media 37&#194;&#176;C. Note cases <br />                                                secondary impacts single-gene deletions, compensating <br />                                                suppressor mutations. errors mutants described                                                paper, later corrected [43]. study profiled growth                                                mutants minimal glucose MOPS media identify genes                                                conditionally essential conditions. <br />  <br />                                          &#226;&#8364;&#162;     &#226;&#8364;&#339;Experimental Computational Assessment Conditionally Essential Genes NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Escherichia coli&#226;&#8364;? Joyce et al. [44] <br />  <br />                                                Strain: E. coli K&#226;&#8364;&#8220;12 BW25113 (rpoS  rph-1 &#206;&#187; - rrnB3 &#206;&#8221;lacZ4787 hsdR514 <br />                                                &#206;&#8221;(araBAD)567 &#206;&#8221;(rhaBAD)568 rph-1)  [42]) <br />  <br />                                                study used Keio collection single gene knockout mutants profiled <br />                                                growth glycerol supplemented minimal medium. 119 genes                                                identified essential growth glycerol. combined                                                observations Baba et al [42] regarding conditional <br />                                                essentiality mutants grown glucose supplemented minimal media <br />                                                able identify conserved conditionally essential core 94 genes <br />                                                required E. coli K&#226;&#8364;&#8220;12 grow minimal nutritional <br />                                                supplementation essential growth rich conditions. <br />  <br />                                          &#226;&#8364;&#162;      Genome-Scale Metabolic Reconstruction Escherichia coli K-12 MG1655 <br />                                                Accounts 1260 ORFs Thermodynamic Information&#226;&#8364;? <br />  <br />                                                Feist et al. [45] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                publication used experimental data regarding conditional gene essentiality <br />                                                Joyce et al. [44] Baba et al. [42] compared                                                computationally predicted essential genes genome scale metabolic <br />                                                reconstruction E. coli. dataset included EcoCyc facilitate <br />                                                benchmarking computational predictions essentiality EcoCyc model <br />                                                computations model Feist et al. Multicopy suppression underpins <br />                                                metabolic evolvability. <br />  <br />                                          &#226;&#8364;&#162;     &#226;&#8364;&#339;Multicopy suppression Underpins Metabolic Evolvability&#226;&#8364;? <br />  <br />                                                Patrick et al. 2007 [46] <br />  <br />  <br />  <br />  <br />                                          Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                             Page 9 <br />  <br />  <br />                                                   Strain: E. coli BW25113 (rpoS  rph-1 &#206;&#187; - rrnB3 &#206;&#8221;lacZ4787 hsdR514 <br />                                                   &#206;&#8221;(araBAD)567 &#206;&#8221;(rhaBAD)568 rph-1) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   study used conditionally essential gene sets identified Baba et al                                                   Joyce et al tested ability form colonies glucose M9 agar. <br />                                                   identified 107 genes conditionally essential conditions. <br />  <br />                                          essentiality data available given gene, EcoCyc gene page includes table <br />                                          conditions gene essential essential <br />                                          growth. Clicking condition navigate growth-medium page lists                                          essentiality information growth condition. <br />  <br />                               7 EcoCyc Metabolic Flux Model <br />                                          quantitative steady-state metabolic flux model derived EcoCyc using flux <br />                                          balance analysis (FBA) [47, 48]. running model different parameters, scientists <br />                                          model growth E. coli different nutrient conditions different gene <br />                                          knock-outs. time model executed, freshly generated EcoCyc, meaning <br />                                          reactions EcoCyc updated curation, model automatically reflects <br />                                          changes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          EcoCyc FBA model distinct E. coli FBA models derived Palsson <br />                                          group [49, 45, 50], models common EcoCyc                                          iAF1260 model partially unified 2007 [45], groups consult s <br />                                          work updating models. <br />  <br />                                          Supplementary Information, provided accompanying paper-suppl-info.xlsx file, <br />                                          details E. coli biomass metabolite set used model biomass production metabolite <br />                                          requirements EcoCyc FBA. metabolite set derived iJO1366 model WT <br />                                          biomass reaction Orth et al. [50]. Supplementary Information contains                                          description nutrient secretion metabolite sets supply inputs outputs                                          FBA model, description differences EcoCyc FBA biomass <br />                                          metabolite set iJO1366 WT biomass reaction. <br />  <br />                                          run EcoCyc FBA model, download install Pathway Tools software <br />                                          configuration includes EcoCyc, invoke MetaFlux modeling component NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          Pathway Tools  Chapter 8 Pathway Tools User&#226;&#8364;&#8482;s Guide). <br />  <br />                                          EcoCyc provides example files describing invocations FBA model                                          different nutrient conditions. files installed Pathway Tools <br />                                          directory tree pathway-tools/aic-export/pgdbs/biocyc/ecocyc/VERSION/data/fba/. Output <br />                                          files produced result successful FBA runs supplied. fba files included. <br />                                          supplied input files  <br />  <br />                                              1.   GlucoseAer.fba: 10 mmol/gCDW/hr glucose uptake, minimal media, aerobic <br />                                                   conditions <br />  <br />                                              2.   GlucoseAnaer.fba: 10 mmol/gCDW/hr glucose uptake, minimal media, anaerobic <br />                                                   conditions <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                            Page 10 <br />  <br />  <br />                                               3.   GlycerolAer.fba: 10 mmol/gCDW/hr glycerol uptake, minimal media, aerobic <br />                                                    conditions <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               4.   GlycerolAnaer.fba: 10 mmol/gCDW/hr glycerol uptake, minimal media, anaerobic <br />                                                    conditions <br />  <br />                               7.1 External Flux Predictions <br />                                           MetaFlux metabolic flux predictions EcoCyc version 17.5 aerobic growth                                           glucose glycerol given Tables 5 6. Model predictions anaerobic growth                                           glucose glycerol given Tables 7 8. cases, uptake rate carbon <br />                                           source set upper bound reflecting experimental uptake rates mmol/gCDW/hr. <span id='am-47' about='obo:IMR_0200217' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-48' property="oboInOwl:id" content="IMR:0200217" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="O2" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00007" datatype="xsd:string"></span>O2</span> <br />                                           uptake rates set upper bound 0.00 mmol/gCDW/hr anaerobic conditions. <br />                                           nutrient sources left free. <br />  <br />                               7.2 Improvement Metabolic Model <br />                                           EcoCyc release plan include improved version EcoCyc metabolic <br />                                           flux model reflects recent improvements knowledge E. coli metabolic <br />                                           network. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Model predictions differ experimental measurements number reasons <br />                                           including operation additional, unmodeled reactions metabolites; existing <br />                                           reactions operating different fashion model (e.g. model contains &#226;&#8364;&#339;perfect&#226;&#8364;? <br />                                           respiratory electron-transfer chain possibility reactive oxygen-species <br />                                           generation); presence regulation product inhibition deactivates reactions                                           limits throughput; differences optimization objective functions depending                                           specified feed source. <br />  <br />                               8 Update Frequency <br />                                           EcoCyc.org BioCyc.org websites downloadable files updated 3&#226;&#8364;&#8220;4 times                                           year. faster, powerful version EcoCyc install locally                                           (Macintosh, PC/Windows, PC/Linux) released semiannually. <br />  <br />                               9 Data Sources Incorporated EcoCyc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           EcoCyc includes data imported following bioinformatics databases. cases, <br />                                           data imported twice year. note literature references <br />                                           EcoCyc obtained PubMed. <br />  <br />                               9.1 UniProt Features <br />                                           UniProt protein features  UniProt KB term sequence annotations) complete <br />                                           proteome E. coli K-12 MG1655 SwissProt imported EcoCyc EcoCyc <br />                                           release. import protein features experimental non-experimental evidence <br />                                           qualifiers following types: turn, helix, beta strand, coiled-coil. chain <br />                                           type imported does span entire length protein. Examples                                           imported feature types include catalytic domains, phosphorylation sites, metal ion <br />  <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                          Page 11 <br />  <br />  <br />                                          binding sites. import citations associated UniProt protein features include                                          associated PubMed <span id='am-46' about='oboInOwl:id' typeof='owl:Thing'>ID</span>. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          import protein features EcoCyc UniProt Feature Importer tool <br />                                          Pathway Tools software. <br />  <br />                               9.2 Gene Ontology <br />                                          years, EcoCyc EcoliWiki/PortEco collaborating improving <br />                                          maintaining annotations E. coli. applications described                                          [56]. summer 2008, periodically generating file <br />                                          containing E. coli K-12 term annotations, called gene_association.ecocyc,                                          obtained Gene Ontology Consortium. <br />  <br />                                          annotation standard EcoCyc&#226;&#8364;&#8482;s manual literature-based curation process.                                          annotations added database objects represent functional gene products <br />                                          multimers, directly gene objects. approach models biology                                          accurately indicates exactly form gene product specified                                          function. parallel, manual annotation E. coli genes ongoing EcoliWiki. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          regular basis, annotations merged. latest UniProt EcoliWiki <br />                                          annotations imported EcoCyc. consortium does accept <br />                                          electronic annotations gene association file annotations                                          year old, UniProt annotations reimported EcoCyc regular basis. <br />  <br />                                          EcoCyc incorporates electronic experimental term annotations E. coli K-12 <br />                                          gene products obtained &#226;&#8364;&#339;UniProt [multispecies] Annotations @ EBI&#226;&#8364;? file <br />                                          downloaded Gene Ontology Consortium. import performed                                          2007, approximately 30,000 new IEA (&#226;&#8364;&#339;Inferred Electronic Annotation&#226;&#8364;?) term <br />                                          assignments added EcoCyc, approximately 1,000 assignments                                          experimental evidence codes including assignments high-throughput protein- <br />                                          interaction studies. import terms UniProt EcoCyc, filtering <br />                                          operation applied prune term annotations based solely computational (IEA) <br />                                          evidence EcoCyc gene product specific annotations                                           words, terms children term imported) experimental <br />                                          evidence available. example, gene product contained experimental <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          annotation term &#226;&#8364;&#339;galactose kinase,&#226;&#8364;? software add computational <br />                                          annotation &#226;&#8364;&#339;<span id='am-2' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-3' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span>carbohydrate</span> kinase.&#226;&#8364;? filtering leads removal approximately 1,000 <br />                                          specific redundant annotations. <br />  <br />                                          gene association file generated quarterly EcoCyc releases. file sent                                          EcoliWiki team Texas M processing. EcoliWiki, annotations                                          wiki-based community annotation EcoCyc update added                                          file, annotations containing qualifiers (mainly contributes_to)                                          supported EcoCyc. annotations complete consortium standards <br />                                          extracted EcoliWiki; incomplete annotations left hope community <br />                                          members eventually complete  EcoliWiki runs consortium validation <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                             Page 12 <br />  <br />  <br />                                             scripts deposits file consortium Concurrent Versioning <br />                                              <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               9.3 Genbank <br />                                             Genbank record U00096, produced Blattner laboratory October 1997,                                             source original E. coli MG1655 genome sequence annotation incorporated                                             EcoCyc. corrected nucleotide sequence deposited GenBank U00096.2 2004, <br />                                             revised sequence incorporated EcoCyc version 8.6 (November 2004). <br />                                             revised genome annotation published [57] incorporated EcoCyc version <br />                                             10.0 (March 2006). <br />  <br />                               9.4 RefSeq Collaboration <br />                                             EcoCyc involved collaboration update genome annotation GenBank <br />                                             (U00096.2) RefSeq (NC 000913.2) entries E. coli K-12 MG1655 ongoing <br />                                             basis. primary collaborators include EcoCyc, EcoGene, UniProtKB/Swiss-Prot,                                             NCBI. collaborators routinely share data resolve data conflicts. updates                                             gene names, gene positions, gene product names shared partners. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               9.5 MetaCyc <br />                                             EcoCyc MetaCyc databases exchange data release processes                                             databases. updates occurred enzymes, genes, pathways, reactions,                                             metabolites exchanged databases based automated comparisons                                             update dates ensure latest information corrections propagated                                             databases. <br />  <br />                               10 EcoCyc Accession Numbers <br />                               10.1 Gene Accession Numbers <br />                                             systems accession numbers typically available genes EcoCyc.                                             accession numbers used querying EcoCyc genes  &#226;&#8364;?                                             website Quick Search. <br />  <br />                                                 &#226;&#8364;&#162;    EcoCyc ID: EcoCyc project assigns unique identifiers gene                                                      historical reasons variable syntax, form &#226;&#8364;&#339;Gnnnn,&#226;&#8364;? &#226;&#8364;&#339;EGnnnnn,&#226;&#8364;? <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      &#226;&#8364;&#339;G0-nnnnn.&#226;&#8364;? EcoCyc IDs stored frame id EcoCyc gene object. <br />  <br />                                                 &#226;&#8364;&#162;    B-numbers: Originally assigned Blattner laboratory E. coli <br />                                                      genome project, b-number identifiers form &#226;&#8364;&#339;bnnnn.&#226;&#8364;? B-numbers                                                      originally assigned sequentially genome. gene object removed <br />                                                      genome decision insufficient evidence existence <br />                                                      gene available, b-number retired reused. new <br />                                                      genes added genome, assigned highest available b- <br />                                                      number.  b-numbers longer purely sequential genome. B- <br />                                                      numbers stored EcoCyc slot Accession-1. <br />  <br />  <br />  <br />  <br />                                                Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                          Page 13 <br />  <br />  <br />                                              &#226;&#8364;&#162;    ECK numbers: ECK numbers assigned E. coli K-12 MG1655                                                   W3110 genomes 2005 attempt provide shared accession numbers                                                   genes common genomes [57]. ECK numbers stored EcoCyc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   slot Accession-2. 18 genes E. coli K-12 MG1655 <br />                                                   genome b-number ECK number number; subsequent genes <br />                                                   numbers diverged. <br />  <br />                               11 E. coli Shigella PGDBs BioCyc <br />                                          EcoCyc larger BioCyc collection Pathway/Genome Databases (PGDBs). <br />                                          BioCyc version 17.5 (2013) includes 160 E. coli Shigella PGDBs. PGDBs <br />                                          generated computationally lack extensive manual literature-based curation                                          EcoCyc K-12 database. E. coli genomes BioCyc focused complete <br />                                          genomes, include draft genomes. <br />  <br />                                          PGDBs undergone additional curation: BioCyc PGDBs strains E. <br />                                          coli W3110 E. coli B str. REL606. strains underwent computational <br />                                          annotation-normalization procedure gene names, product names, heteromultimeric <br />                                          protein complexes, Gene Ontology terms propagated EcoCyc NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          orthologous genes strains  orthologs computed SRI                                          bidirectional best-BLAST hits additional manual review curation). procedure <br />                                          performed assumption genome-annotation pipelines typically introduce <br />                                          syntactically large semantically insignificant variation naming genes gene <br />                                          products. addition, E. coli B str. REL606 underwent literature-based curation SRI                                          incorporate experimental information regarding genes pathways present strain <br />                                          EcoCyc strain MG1655. curation supported PortEco project. <br />  <br />                                          select given genome querying BioCyc website, click words &#226;&#8364;&#339;change <br />                                          organism database&#226;&#8364;? Quick Search Gene Search buttons upper right <br />                                          corner EcoCyc webpages. <br />  <br />                               12 Encourage Feedback <br />                                          Feedback scientific community proved invaluable improving EcoCyc                                          years development. strongly encourage comments suggestions NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          improvements areas, including: <br />  <br />                                              &#226;&#8364;&#162;    database content EcoCyc <br />  <br />                                              &#226;&#8364;&#162;    presentation information EcoCyc website <br />  <br />                                              &#226;&#8364;&#162;    analysis tools provided conjunction EcoCyc <br />  <br />                                              &#226;&#8364;&#162;    performance EcoCyc website <br />  <br />                                          error omission EcoCyc, report using &#226;&#8364;&#339;Report Errors                                          Provide Feedback&#226;&#8364;? link data page. email suggestions                                          questions biocyc-support ai dot sri dot com. <br />  <br />  <br />  <br />  <br />                                             Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                               Page 14 <br />  <br />  <br />                                          EcoCyc release email summary new developments biocyc-users <br />                                          mailing list. subscribe mailing list, http://biocyc.org/subscribe.shtml. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               13 Learn                                               &#226;&#8364;&#162;     use Pathway Tools website EcoCyc <br />  <br />                                               &#226;&#8364;&#162;     Downloadable instructional videos use EcoCyc <br />  <br />                                               &#226;&#8364;&#162;     Pathway/Genome Database Concepts Guide <br />  <br />                                               &#226;&#8364;&#162;     Publications EcoCyc: [58, 59, 60, 61, 62, 63, 64, 65, 66, 67, 68, 69, 70] <br />  <br />                                               &#226;&#8364;&#162;     BioCyc User&#226;&#8364;&#8482;s Guide <br />  <br />                                               &#226;&#8364;&#162;     MetaCyc User&#226;&#8364;&#8482;s Guide <br />  <br />                                               &#226;&#8364;&#162;     Guide Pathway Tools Schema <br />  <br />                                               &#226;&#8364;&#162;     download Pathway Tools organism flat-file databases <br />  <br />                               14 Cite EcoCyc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          cite EcoCyc publications benefit use EcoCyc database                                          website. cite EcoCyc recent Nucleic Acids Research Database issue <br />                                          article, currently: <br />  <br />                                          Keseler et al., Nuc Acids Res, 41:D605&#226;&#8364;&#8220;12 2013. <br />  <br />  <br />                               Supplementary Material <br />                                          Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                          Monica Riley led curation EcoCyc years, inception. efforts created content                                          organism-scale metabolic database. John Ingraham valued advisor EcoCyc years. thank <br />                                          scientists contributed corrections suggestions EcoCyc years, thank                                          scientists served EcoCyc Steering Committee. contributors EcoCyc listed                                          EcoCyc credits page. <br />  <br />                                          development EcoCyc funded NIH grants GM77678 GM71962 NIH National Institute NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          General Medical Sciences. <br />  <br />  <br />                               References <br />                                          1. Karp PD, Paley SM, Krummenacker M, Latendresse M, Dale JM, Lee T, et al. Pathway Tools <br />                                             version 13.0: Integrated Software Pathway/Genome Informatics Systems Biology. Brief <br />                                             Bioinform. 2010; 11:40&#226;&#8364;&#8220;79. Available  http://bib.oxfordjournals.org/cgi/content/abstract/ <br />                                             bbp043. 10.1093/bib/bbp043 [PubMed: 19955237] <br />                                          2. Kim KS, Lee S, Ryu CM. Interspecific bacterial sensing airborne signals modulates <br />                                             locomotion drug resistance. Nat Commun. 2013; 4:1809. [PubMed: 23651997] <br />                                          3. Bower JM, Gordon-Raagas HB, Mulvey MA. Conditioning uropathogenic Escherichia coli                                             enhanced colonization host. Infect Immun. 2009  77(5):2104&#226;&#8364;&#8220;2112. [PubMed: 19255192] <br />                                          4. Rhodius V, Dyk TKV, Gross C, LaRossa RA. Impact genomic technologies studies                                             bacterial gene expression. Annu Rev Microbiol. 2002; 56:599&#226;&#8364;&#8220;624. [PubMed: 12142487] <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                  Page 15 <br />  <br />  <br />                                          5. Gonzalez R, Tao H, Purvis JE, York SW, Shanmugam KT, Ingram LO. Gene array-based <br />                                             identification changes contribute ethanol tolerance ethanologenic Escherichia coli: <br />                                             Comparison KO11 (parent) LY01 (resistant mutant). Biotechnol Prog. 2003 Mar; 19(2):612&#226;&#8364;&#8220; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             623. [PubMed: 12675606] <br />                                          6. Taoka M, Yamauchi Y, Shinkawa T, Kaji H, Motohashi W, Nakayama H, et al. small subset <br />                                             horizontally transferred chromosomal genes Escherichia coli translated proteins. <br />                                             Mol Cell Proteomics. 2004 Aug; 3(8):780&#226;&#8364;&#8220;787. [PubMed: 15115803] <br />                                          7. Ravasz E, Somera AL, Mongru DA, Oltvai ZN, Barabasi AL. Hierarchical organization                                             modularity metabolic networks. Science. 2002 Aug; 297(5586):1551&#226;&#8364;&#8220;1555. [PubMed: 12202830] <br />                                          8. Simeonidis E, Rison SC, Thornton JM, Bogle ID, Papageorgiou LG. Analysis metabolic <br />                                             networks using pathway distance metric linear programming. Metab Eng. 2003 Jul; 5(3): <br />                                             211&#226;&#8364;&#8220;219. [PubMed: 12948755] <br />                                          9. Arita M. metabolic world Escherichia coli small. Proc Natl Acad Sci USA. 2004 Feb; <br />                                             101(6):1543&#226;&#8364;&#8220;1547. [PubMed: 14757824] <br />                                          10. Jardine O, Gough J, Chothia C, Teichmann SA. Comparison small molecule metabolic <br />                                               enzymes Escherichia coli Saccharomyces cerevisiae. Genome Res. 2002 Jun; 12(6):916&#226;&#8364;&#8220; <br />                                               929. [PubMed: 12045145] <br />                                          11. Rison SC, Thornton JM. Pathway evolution, structurally speaking. Curr Opin Struct Biol. 2002 <br />                                               Jun; 12(3):374&#226;&#8364;&#8220;382. [PubMed: 12127458] <br />                                          12. Ma HW, Kumar B, Ditges U, Gunzer F, Buer J, Zeng AP. extended transcriptional regulatory <br />                                               network Escherichia coli analysis hierarchical structure network motifs. Nuc <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Acids Res. 2004; 32(22):6643&#226;&#8364;&#8220;6649. <br />                                          13. Shen-Orr SS, Milo R, Mangan S, Alon U. Network motifs transcriptional regulation network <br />                                               Escherichia coli. Nat Genet. 2002  31(1):64&#226;&#8364;&#8220;68. [PubMed: 11967538] <br />                                          14. Karimpour-Fard  Leach SM, Gill RT, Hunter LE. Predicting protein linkages bacteria:                                               method best depends task. BMC Bioinformatics. 2008; 9:397. [PubMed: 18816389] <br />                                          15. Bowers PM, Pellegrini M, Thompson MJ, Fierro J, Yeates  Eisenberg D. Prolinks: database <br />                                               protein functional linkages derived coevolution. Genome Biol. 2004; 5(5):R35. [PubMed: <br />                                               15128449] <br />                                          16. Price MN, Huang KH, Alm EJ, Arkin AP. novel method accurate operon predictions                                               sequenced prokaryotes. Nuc Acids Res. 2005; 33(3):880&#226;&#8364;&#8220;892. <br />                                          17. Steinhauser D, Junker BH, Luedemann  Selbig J, Kopka J. Hypothesis-driven approach                                               predict transcriptional units gene expression data. Bioinformatics. 2004 Aug; 20(12):1928&#226;&#8364;&#8220; <br />                                               1939. [PubMed: 15044239] <br />                                          18. Burden S, Lin YX, Zhang R. Improving promoter prediction NNPP2.2 algorithm: case <br />                                               study using Escherichia coli DNA sequences. Bioinformatics. 2005 Mar; 21(5):601&#226;&#8364;&#8220;607. <br />                                               [PubMed: 15454410] <br />                                          19. Gordon L, Chervonenkis AY, Gammerman AJ, Shahmuradov IA, Solovyev VV. Sequence <br />                                               alignment kernel recognition promoter regions. Bioinformatics. 2003 Oct; 19(15):1964&#226;&#8364;&#8220; <br />                                               1971. [PubMed: 14555630] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          20. Fu Y, Jarboe LR, Dickerson JA. Reconstructing genome-wide regulatory network E. coli using <br />                                               transcriptome data predicted transcription factor activities. BMC Bioinformatics. 2011; <br />                                               12:233. [PubMed: 21668997] <br />                                          21. Watanabe RL, Morett E, Vallejo EE. Inferring modules functionally interacting proteins using <br />                                               Bond Energy Algorithm. BMC Bioinformatics. 2008; 9:285. [PubMed: 18559112] <br />                                          22. Muley VY, Ranjan  Effect reference genome selection performance computational <br />                                               methods genome-wide protein-protein interaction prediction. PLoS  2012; 7(7):e42057. <br />                                               [PubMed: 22844541] <br />                                          23. Moreno-Hagelsieb G, Jokic P. evolutionary dynamics functional modules                                               extraordinary plasticity regulons: Escherichia coli perspective. Nucleic Acids Res. 2012; <br />                                               40(15):7104&#226;&#8364;&#8220;12. [PubMed: 22618875] <br />                                          24. Kastenmuller G, Schenk  Gasteiger J, Mewes HW. Uncovering metabolic pathways relevant                                               phenotypic traits microbial genomes. Genome Biol. 2009; 10(3):R28. [PubMed: 19284550] <br />  <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                 Page 16 <br />  <br />  <br />                                          25. Kumar VS, Maranas CD. GrowMatch: automated method reconciling silico vivo <br />                                              growth predictions. PLoS Comput Biol. 2009 Mar.5(3):e1000308. [PubMed: 19282964] <br />                                          26. Thomas GH, Zucker J, Macdonald SJ, Sorokin  Goryanin  Douglas AE. fragile metabolic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              network adapted cooperation symbiotic bacterium Buchnera aphidicola. BMC Syst Biol. <br />                                              2009; 3:24. [PubMed: 19232131] <br />                                          27. Frazier  Johnson GM, Thomassen DG, Oliver CE, Patrinos  Realizing potential                                              genome revolution: Genomes Life program. Science. 2003 Apr; 300(5617):290&#226;&#8364;&#8220;293. <br />                                              [PubMed: 12690188] <br />                                          28. Bailey JE. Science Metabolic Engineering. Science. 1991 Jun.252:1668&#226;&#8364;&#8220;1675. <br />                                              [PubMed: 2047876] <br />                                          29. Stephanopoulos G, Vallino JJ. Network rigidity metabolic engineering metabolite <br />                                              overproduction. Science. 1991 Jun.252:1675&#226;&#8364;&#8220;1681. [PubMed: 1904627] <br />                                          30. Arense P, Bernal V, Charlier D, Iborra JL, Foulquie-Moreno MR, Canovas M. Metabolic <br />                                              engineering high yielding L(&#226;&#710;&#8217;)-<span id='am-41' about='obo:IMR_0200035' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00487" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0200035" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Carnitine" datatype="xsd:string"></span>carnitine</span> production Escherichia coli. Microb Cell Fact. <br />                                              2013; 12(1):56. (ENG). [PubMed: 23718679] <br />                                          31. Jantama K, Zhang X, Moore JC, Shanmugam KT, Svoronos SA, Ingram LO. Eliminating                                              products increasing succinate yields engineered strains Escherichia coli C. Biotechnol <br />                                              Bioeng. 2008 Dec; 101(5):881&#226;&#8364;&#8220;893. [PubMed: 18781696] <br />                                          32. Weber J, Hoffmann F, Rinas U. Metabolic adaptation Escherichia coli temperature- <br />                                              induced recombinant protein production: 2. Redirection metabolic fluxes. Biotechnol Bioeng. <br />                                              2002 Nov; 80(3):320&#226;&#8364;&#8220;330. [PubMed: 12226865] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          33. Update activities Universal Protein Resource (UniProt) 2013. Nuc Acids Res. 2013; <br />                                              41(Database issue):D43&#226;&#8364;&#8220;7. <br />                                          34. Ashburner M, Ball CA, Blake JA, Botstein D, Butler H, Cherry JM, et al. Gene Ontology: Tool                                              unification biology. Nature Genetics. 2000; 25:25&#226;&#8364;&#8220;29. [PubMed: 10802651] <br />                                          35. Serres MH, Riley M. MultiFun, multifunctional classification scheme Escherichia coli K&#226;&#8364;&#8220;12 <br />                                              gene products. Genome Biol. 2000; 5(4):205&#226;&#8364;&#8220;222. <br />                                          36. Bochner BR, Gadzinski P, Panomitros E. Phenotype Microarrays High-Throughput Phenotypic <br />                                              Testing Assay Gene Function. Genome Res. 2001 Jul; 11(7):1246&#226;&#8364;&#8220;1255. [PubMed: <br />                                              11435407] <br />                                          37. Oun MA, Suthers PF, Jones GI, Carter BR, Saunders MP, Maranas CD, et al. Genome scale <br />                                              reconstruction Salmonella metabolic model: comparison similarity differences                                              commensal Escherichia coli strain. J Biol Chem. 2009; 284(43):29480&#226;&#8364;&#8220;8. [PubMed: 19690172] <br />                                          38. Baumler DJ, Peplinski RG, Reed JL, Glasner JD, Perna NT. evolution metabolic networks <br />                                              E. coli. BMC Syst Biol. 2011; 5:182. [PubMed: 22044664] <br />                                          39. Mackie  Paley S, Keseler IM, Shearer  Paulsen  Karp PD. Addition Escherichia coli K&#226;&#8364;&#8220; <br />                                              12 Growth-Observation Gene Essentiality Data EcoCyc database. J Bacteriol. 2013 <br />                                          40. Yoon SH, Han MJ, Jeong H, Lee CH, Xia XX, Lee DH, et al. Comparative multi-omics systems <br />                                              analysis Escherichia coli strains B K&#226;&#8364;&#8220;12. Genome Biol. 2012; 13(5):R37. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              22632713] <br />                                          41. Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty MD, et al. Experimental <br />                                              determination level analysis essential genes Escherichia coli MG1655. J <br />                                              Bacteriol. 2003 Oct; 185(19):5673&#226;&#8364;&#8220;5684. [PubMed: 13129938] <br />                                          42. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, et al. Construction Escherichia <br />                                              coli K&#226;&#8364;&#8220;12 frame, single-gene knockout mutants: Keio collection. Mol Syst Biol. 2006; <br />                                              2:2006.0008. <br />                                          43. Yamamoto N, Nakahigashi K, Nakamichi T, Yoshino M, Takai Y, Touda Y, et al. Update                                              collection Escherichia coli single-gene deletion mutants. Mol Syst Biol. 2009; 5:335. [PubMed: <br />                                              20029369] <br />                                          44. Joyce AR, Reed JL, White  Edwards R, Osterman  Baba T, et al. Experimental                                              computational assessment conditionally essential genes Escherichia coli. J Bacteriol. 2006 <br />                                              Dec; 188(23):8259&#226;&#8364;&#8220;8271. [PubMed: 17012394] <br />                                          45. Feist  Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, et al. genome-scale <br />                                              metabolic reconstruction Escherichia coli K&#226;&#8364;&#8220;12 MG1655 accounts 1260 ORFs  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                Page 17 <br />  <br />  <br />                                              thermodynamic information. Mol Syst Biol. 2007; 3:121&#226;&#8364;&#8220;38. Available  http:// <br />                                              www.nature.com/doifinder/10.1038/msb4100155. [PubMed: 17593909] <br />                                          46. Patrick WM, Quandt EM, Swartzlander DB, Matsumura  Multicopy suppression underpins <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              metabolic evolvability. Mol Biol Evol. 2007; 24(12):2716&#226;&#8364;&#8220;22. [PubMed: 17884825] <br />                                          47. Orth JD, Thiele  Palsson BO. flux balance analysis? Nat Biotechnol. 2010; 28(3):245&#226;&#8364;&#8220;8. <br />                                              [PubMed: 20212490] <br />                                          48. Thiele  Palsson BO. protocol generating high-quality genome-scale metabolic <br />                                              reconstruction. Nat Protoc. 2010; 5(1):93&#226;&#8364;&#8220;121. [PubMed: 20057383] <br />                                          49. Reed JL, Vo TD, Schilling CH, Palsson BO. expanded genome-scale model Escherichia coli <br />                                              K-12 (iJR904 GSM/GPR). Genome Biol. 2003; 4(9):R54. [PubMed: 12952533] <br />                                          50. Orth JD, Conrad TM, Na J, Lerman JA, Nam H, Feist  et al. comprehensive genome-scale <br />                                              reconstruction Escherichia coli metabolism &#226;&#8364;&#8221; 2011. Mol Syst Biol. 2011; 7:535. [PubMed: <br />                                              21988831] <br />                                          51. Kayser  Weber J, Hecht V, Rinas U. Metabolic flux analysis Escherichia coli glucose- <br />                                              limited continuous culture.  Growth-rate-dependent metabolic efficiency steady state. <br />                                              Microbiology. 2005; 151:693&#226;&#8364;&#8220;706. [PubMed: 15758216] <br />                                          52. Ibarra RU, Edwards JS, Palsson BO. Escherichia coli K-12 undergoes adaptive evolution                                              achieve silico predicted optimal growth. Nature. 2002; 420:186&#226;&#8364;&#8220;189. [PubMed: 12432395] <br />                                          53. Belaich  Belaich JP. Microcalorimetric study anaerobic growth Escherichia coli: <br />                                              growth thermograms synthetic medium. J Bacteriol. 1976; 72:497&#226;&#8364;&#8220;9. <br />                                          54. Varma  Boesch BW, Palsson BO. Stoichiometric interpretation Escherichia coli glucose <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              catabolism various oxygenation rates. Applied Environmental Microbiology. 1993; <br />                                              59(8):2465&#226;&#8364;&#8220;73. [PubMed: 8368835] <br />                                          55. Dharmadi Y, Murarka  Gonzalez R. Anaerobic fermentation glycerol Escherichia coli: <br />                                              anew platform metabolic engineering. Biotechnology Bioengineering. 2006; 94(5):821&#226;&#8364;&#8220;9. <br />                                              [PubMed: 16715533] <br />                                          56. Hu JC, Karp PD, Keseler IM, Krummenacker M, Siegele DA. learn                                              Escherichia coli application Gene Ontology. Trends Microbiol. 2009 Jul; 17(7):269&#226;&#8364;&#8220; <br />                                              278. [PubMed: 19576778] <br />                                          57. Riley M, Abe T, Arnaud MB, Berlyn MK, Blattner FR, Chaudhuri RR, et al. Escherichia coli <br />                                              K-12: cooperatively developed annotation snapshot&#226;&#8364;&#8220;2005. Nuc Acids Res. 2006; 34(1):1&#226;&#8364;&#8220;9. <br />                                          58. Keseler IM, Collado-Vides J, Santos-Zavaleta  Peralta-Gil M, Gama-Castro S, Muniz-Rascado <br />                                              L, et al. EcoCyc: Comprehensive Database Escherichia coli Biology. Nuc Acids Res. 2011; <br />                                              39:D583&#226;&#8364;&#8220;90. <br />                                          59. Keseler IM, Bonavides-Martinez C, Collado-Vides J, Gama-Castro S, Gunsalus RP, Johnson DA, <br />                                              et al. EcoCyc: comprehensive view E. coli biology. Nuc Acids Res. 2009; 37:D464&#226;&#8364;&#8220;70. <br />                                              Available  http://nar.oxfordjournals.org/cgi/reprint/gkn751? <br />                                              ijkey=7epgizfnGFYQHCe<span id='am-15' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-16' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>;keytype=ref. <br />                                          60. Karp PD, Keseler IM, Shearer  Latendresse M, Krummenacker M, Paley SM, et al. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Multidimensional annotation Escherichia coli K-12 genome. Nuc Acids Res. 2007; <br />                                              35:7577&#226;&#8364;&#8220;90. Available  http://nar.oxfordjournals.org/cgi/content 35/22/7577. <br />                                          61. Keseler IM, Collado-Vides J, Gama-Castro S, Ingraham J, Paley S, Paulsen  et al. EcoCyc:                                              comprehensive database resource E. coli. Nuc Acids Res. 2005; 33:D334&#226;&#8364;&#8220;7. Available  <br />                                              http://nar.oupjournals.org/cgi/content 33/suppl_1/D334? <br />                                              ijkey=80p4BbGpEFjLQ&amp;keytype=ref. <br />                                          62. Karp PD, Arnaud M, Collado-Vides J, Ingraham J, Paulsen  Saier MHJ. E. coli EcoCyc <br />                                              Database: Longer Just Metabolic Pathway Database. ASM News. 2004; 70(1):25&#226;&#8364;&#8220;30. <br />                                          63. Karp PD, Riley M, Saier M, Paulsen  Paley S, Pellegrini-Toole  EcoCyc Database. Nuc <br />                                              Acids Res. 2002; 30(1):56&#226;&#8364;&#8220;8. <br />                                          64. Karp PD, Riley M, Saier M, Paulsen  Paley S, Pellegrini-Toole  EcoCyc MetaCyc <br />                                              Databases. Nuc Acids Res. 2000; 28(1):56&#226;&#8364;&#8220;59. <br />                                          65. Karp, PD. Nucleic Acid Protein Databases Use  London: Academic Press; <br />                                              1999. Using EcoCyc Database; p. 269-280. <br />  <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                Page 18 <br />  <br />  <br />                                          66. Karp, PD.; Riley, M. Bioinformatics Databases Systems. Norwell, MA: Kluwer Academic <br />                                              Publishers; 1999. EcoCyc: resource lessons learned; p. 47-62. <br />                                          67. Karp P, Riley M, Paley S, Pellegrini-Toole  Krummenacker M. EcoCyc: Electronic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Encyclopedia E. coli Genes Metabolism. Nuc Acids Res. 1999; 27(1):55&#226;&#8364;&#8220;58. <br />                                          68. Karp P, Riley M, Paley S, Pellegrini-Toole  Krummenacker M. EcoCyc: Electronic <br />                                              Encyclopedia E. coli Genes Metabolism. Nuc Acids Res. 1998; 26(1):50&#226;&#8364;&#8220;53. <br />                                          69. Karp P, Riley M, Paley S, Pellegrini-Toole  Krummenacker M. EcoCyc: Electronic <br />                                              Encyclopedia E. coli Genes Metabolism. Nuc Acids Res. 1997; 25(1):43&#226;&#8364;&#8220;50. <br />                                          70. Karp P, Riley M, Paley S, Pellegrini-Toole  EcoCyc: Electronic Encyclopedia E. coli Genes <br />                                              Metabolism. Nuc Acids Res. 1996; 24(1):32&#226;&#8364;&#8220;40. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                          Page 19 <br />  <br />  <br />  <br />  <br />                                                                                         Table 1 <br />  <br />                            Genes gene products EcoCyc. Protein features annotations protein sites regions NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            enzyme active sites, metal ion binding sites, transmembrane domains. small number elements                            included count Genes included sub-categories genes. <br />  <br />                             Genes/Proteins <br />  <br />                             Genes                                    4501 <br />                                Protein-Coding Genes                  4284 <br />                                tRNA Genes                               86 <br />                                rRNA Genes                               22 <br />                                Regulatory RNA Genes                     41 <br />                                RNA Genes                          56 <br />                             Pseudogenes                                133 <br />                             Polypeptides                             4393 <br />                             Protein Complexes                          995 <br />                                Heteromultimeric Protein Complexes      290 <br />                                Homomultimeric Protein Complexes        705 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Protein Features                        23,114 <br />                             Enzymes (excluding transporters)         1245 <br />                             Transporters                               267 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                       Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                         Page 20 <br />  <br />  <br />  <br />  <br />                                                                                         Table 2 <br />  <br />                            Gene annotation status EcoCyc. Genes known molecular function experimental evidence NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            assigned function, genes predicted molecular function function predicted <br />                            computationally. <br />  <br />                             Gene Annotation Status <br />  <br />                             Genes known predicted molecular function   3127 <br />                                Genes known molecular function             2710 <br />                                Genes predicted molecular function          417 <br />                             Genes unknown molecular function              1374 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                       Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                        Page 21 <br />  <br />  <br />  <br />  <br />                                                                                                Table 3 <br />  <br />                            Reactions, compounds pathways EcoCyc. Superpathways connected sets base metabolic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            pathways (connected shared substrates.) <br />  <br />                             Reactions/Metabolites/Pathways <br />  <br />                             Metabolic Reactions                                                1443 <br />                             Transport (including Electron Transfer) Reactions                   379 <br />                             Pathways                                                            401 <br />                                Small-molecule metabolism base pathways                          291 <br />                                Signaling pathways                                                29 <br />                                Superpathways                                                     81 <br />                             Metabolites                                                        2466 <br />                                Metabolites substrates enzyme-catalyzed reactions   1331 <br />                                Metabolites physiological enzyme regulators             121 <br />                                Metabolites cofactors prosthetic groups               56 <br />                                Transported metabolites                                          274 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                          Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                                Page 22 <br />  <br />  <br />  <br />  <br />                                                                                             Table 4 <br />  <br />                            Regulation-related objects interactions EcoCyc. member &#226;&#8364;&#339;Instances Regulation NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Transcription Initiation&#226;&#8364;? describes single regulatory interaction transcription factor binding <br />                            site. <br />  <br />                             Transcriptional/Translational Regulation <br />  <br />                             Transcription Units                                      4510 <br />                             Promoters                                                3777 <br />                             Terminators                                               259 <br />                             Transcription Factors                                     194 <br />                             Transcription Factor Binding Sites                       2773 <br />                             Instances Regulation Transcription Initiation      3293 <br />                             Instances Regulation Transcriptional Attenuation     20 <br />                             Instances Regulation Translation                    146 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                           Page 23 <br />  <br />  <br />  <br />  <br />                                                                                      Table 5 <br />  <br />                            Comparison experimental aerobic glucose-limited chemostat growth data EcoCyc iJO1366 FBA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            model predictions (389 reactions active EcoCyc). Metabolite uptake production rates units mmol/ <br />                            gCDW/hr. Growth units hr&#226;&#710;&#8217;1. Experimental data Kayser et al. [51]. <br />  <br />                                                Experimental   EcoCyc <br />                             Glucose uptake        3.008       3.008 <br />  <br />                             Growth rate           0.300       0.276 <br />  <br />                             O2 uptake             7.413       4.472 <br />  <br />                             NH4 uptake            2.367       3.026 <br />  <br />                             Sulfate uptake                    0.068 <br />  <br />                             Phosphate uptake                  0.288 <br />  <br />                             <span id='am-22' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CO2</span> production         7.38       5.480 <br />  <br />                             <span id='am-28' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="H2O" datatype="xsd:string"></span>H2O</span> production                    13.026 <br />  <br />                             H+ production                     2.582 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                           Page 24 <br />  <br />  <br />  <br />  <br />                                                                                      Table 6 <br />  <br />                            EcoCyc FBA model performance aerobic glycerol-limited growth (385 reactions active Eco-Cyc). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Uptake production rates units mmol/gCDW/hr. Growth units hr&#226;&#710;&#8217;1. Experimental G1-UR <br />                            estimated graphically Ibarra et al. [52]. <br />  <br />                                                Experimental   EcoCyc <br />                             Glycerol uptake        10         10.00 <br />  <br />                             Growth rate                        0.53 <br />  <br />                             O2 uptake              11          8.96 <br />  <br />                             NH4 uptake                         5.80 <br />  <br />                             Sulfate uptake                     0.13 <br />  <br />                             Phosphate uptake                   0.55 <br />  <br />                             CO2 production                     5.74 <br />  <br />                             H2O production                    30.36 <br />  <br />                             H+ production                      4.95 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Ecosal Plus. Author manuscript; available PMC 2014 November 25. <br />                             Karp et al.                                                                                         Page 25 <br />  <br />  <br />  <br />  <br />                                                                                       Table 7 <br />  <br />                            EcoCyc FBA model performance anaerobic glucose-limited growth (383 reactions active EcoCyc). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Uptake production rates units mmol/gCDW/hr. Growth units hr&#226;&#710;&#8217;1. Experimental data                            Belaich Belaich [53] Varma et al. [54]. FHL set inactive purposes comparison. <br />  <br />                                                    Experimental   EcoCyc <br />                             Glucose uptake             10.0       10.00 <br />  <br />                             Growth rate                0.30        0.25 <br />  <br />                             O2 uptake                  0.00        0.00 <br />  <br />                             NH4 uptake                             2.76 <br />  <br />                             Sulfate uptake                         0.06 <br />  <br />                             Phosphate uptake                       0.26 <br />  <br />                             CO2 production                         0.27 <br />  <br />                             H2O production                         0.57 <br />  <br />                             H+ production                         27.14 <br />  <br />                             Acetate production         7.5         8.03 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Formate production         11.3       16.76 <br />  <br />                             Succinate production       1.2         0.00 <br />  <br />                             Ethanol production         8.7      <br /> </body></html>