<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />  <br />  <br /> let-65 cytoplasmic methionyl tRNA synthetase C. elegans <br />  <br />  <br /> Article history:                                         Cytoplasmic methionyl tRNA synthetase (MetRS) 20 <br /> Received 13 September 2013                               cytoplasmic aminoacyl tRNA synthetase enzymes (ARS). family 8 August 2014                                            enzymes catalyzes process fundamental protein translation. <br /> Accepted 11 August 2014 <br />                                                          Using combination genetic mapping, oligonucleotide array compar- <br /> Available online xxxx <br />                                                          ative genomic hybridization, phenotypic correlation,                                                          mutations essential gene, let-65, reside predicted <br /> Keywords: <br />                                                          Caenorhabditis elegans homologue MetRS, named <br /> Caenorhabditis elegans <br /> let-65 <br />                                                          mars-1. demonstrate lethality associated alleles Aminoacyl tRNA synthetase, methionyl-tRNA                let-65 fully rescued transgenic array spans mars-1 geno- <br /> synthetase                                               mic region. Furthermore, sequence analysis reveals let-65 alleles <br /> Oligonucleotide array comparative genomic                lead alteration highly conserved amino acids. <br /> hybridization                                              &#194;&#169; 2014 Authors. Published Elsevier B.V. open access <br /> Essential genes                                                                              article CC NC-ND license <br />                                                                              (http://creativecommons.org/licenses nc-nd/4.0/). <br />  <br />  <br /> Introduction <br />  <br />    Analysis mutations affecting genes essential survival model organisms, nematode Caenorhabditis elegans, instrumental developing understanding fundamental <br /> biological processes, example cell division morphology (Green et al., 2011). processes direct relevance understanding human diseases metabolic disorders obesity (Hashmi et al., 2013). generation mutations essential genes correlation speci&#239;&#172;?c genetic locus   &#239;&#172;?rst step understanding nature gene&apos;s function promoting <br /> survival. easily identify genes essential survival model organism C. elegans, created physical de&#239;&#172;?ciency map encompasses portion Chromosome IV, allowing correlation <br />  <br />  &#226;?&#381; Corresponding author  Department Biochemistry, College Medicine Health Sciences, Sultan Qaboos University, Oman, 35, <br /> Al-Khod 123, Oman. Tel.: +968 24141152; fax: +968 2441 3880. <br />     E-mail address: mahazr@squ.edu.om (M  Alriyami) <br />  <br />  <br /> http://dx.doi.org/10.1016/j.mgene.2014.08.006 <br /> 2214-5400 &#194;&#169; 2014 Authors. Published Elsevier B.V. open access article CC NC-ND license <br /> (http://creativecommons.org/licenses nc-nd/4.0/). <br />  820                                  M  Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  genetically de&#239;&#172;?ned mutations physically de&#239;&#172;?ned candidate gene lists. study, use physical de&#239;&#172;?ciency map identify essential gene (let-65), previously isolated ethyl <br /> methanesulfonate (EMS) genetic screens (Clark Baillie, 1992; Clark et al., 1988; Moerman Baillie, <br /> 1981; Rogalski Baillie, 1985; Rogalski et al., 1982). analysis determined mutations let-65 correspond methionyl tRNA synthetase gene, mars-1. <br />     Organisms utilize key components translate genetic information proteins: ribosomes, messenger <br /> RNAs (mRNA), transfer RNAs (tRNA), aminoacyl-tRNA synthetases (ARS), reviewed Brown et al. (2010). <br /> Ribosomes subunit organelles direct synthesis proteins translating nucleic acid triplets, carried mRNAs, amino acids (Crick et al., 1961). mRNAs, generated DNA transcription, act trans- <br /> lation templates, tRNAs act interfaces mRNA strands amino acids added poly- <br /> <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> chain transferring required amino acids growing chain (Crick et al., 1961). <br />     ARS family proteins encompasses 20 multi-domain enzymes. proteins <br /> speci&#239;&#172;?cally recognizes <span id='am-25' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-26' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-29' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> corresponding subset cognate tRNA molecules catalyze aminoacylation step reaction. &#239;&#172;?rst step speci&#239;&#172;?c amino acid activated <span id='am-6' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span>ATP</span> produce aminoacyl adenylate; second step amino acid transferred 3&#226;&#8364;&#178;-end tRNA (Jakubowski, 2001). ARS categorized classes  II) based distinct core domains (Wolf et al., 1999). catalytic domain Class synthetases exhibit typical Rossman <br /> fold, layer topology (&#206;&#177;/&#206;&#178;/&#206;&#177;) inner core &#239;&#172;?ve parallel beta strands.  Class <br /> II synthetases generally consist &#206;&#178; strands (Berg et al., 2001). Class II synthetases recognize <br /> different faces tRNA molecules (Berg et al., 2001). tRNA CCA terminus (containing amino acid <br /> attachment site) adopts different conformations tRNA-synthetase complexes class. CCA termini Class enzymes hairpin conformations,  class II enzymes helical con&#239;&#172;?rmations <br /> observed free tRNAs (Berg et al., 2001). intriguing enzyme belongs family Class synthetase, methionyl tRNA synthetase (MetRS), recognizes initiator tRNA methionine carrying tRNA functions elongation peptide chains (Deniziak Barciszewski, <br /> 2001). <br />     study, use oligonucleotide array Comparative Genomic Hybridization (aCGH) de&#239;&#172;?ne phys- <br /> ical de&#239;&#172;?ciency map region Chromosome IV (LGIV) nematode C. elegans. use map assign precisely de&#239;&#172;?ned lists candidate genes previously generated genetically mapped mutations, confer lethal phenotype organism. Using approach, demonstrate lethality <br /> associated alleles let-65 fully rescued transgenic array spans mars-1 genomic region. <br />  <br /> Materials methods <br />  <br /> Growth handling C. elegans <br />  <br />    Nematode strains cultured previously described (Brenner, 1974). Strains used study: N2 <br /> Bristol (BC00049), dpy-5(e907) (CB00907), let-65(s254); nT1(IV) (BC00962), let-65(s694); nT1(IV) <br /> (BC1121), let-65(s1154); nT1(IV) (BC2028), let-65(s1222); nT1(IV) (BC2116), let-65(s1083); nT1(IV) <br /> (BC1909), let-65(s1084); nT1(IV) (BC1910), let-65(s1730); nT1(IV) (BC3258), let-65(s1777); nT1(IV) <br /> (BC3305), let-65(s1083), unc-22(s7)(IV) [WRM0615dH10 + myo-2::GFP + pCeh361] (BC8656), let-65(s1222), <br /> unc-22(s7) (IV) [WRM0615dH10 + myo-2::GFP + pCeh361] (BC8696). <br />  <br /> Oligonucleotide array comparative genomic hybridization (aCGH) data analysis <br />  <br />       aCGH experiments analysis performed previously described (Jones et al., 2007). <br />  <br /> DNA sequence analysis <br />  <br />    Genomic DNA extracted let-65 homozygous animals alleles (s1730, s1083, s694, s1084, <br /> s254, s1154, s1777 s1222). PCR fragments spanning mars-1 ORF ampli&#239;&#172;?ed genomic DNA <br /> templates primers designed using Primer 3 (Rozen Skaletsky, 2000) (primer sequences available request). ampli&#239;&#172;?ed templates subsequently sequenced (Macrogen, Korea) appropriate <br /> sequencing primers (Supplementary Table 1). <br />                                            M  Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830                                  821 <br />  <br />  <br /> Transgenic rescue <br />  <br />    fosmid WRM0615dH10, spans mars-1 genomic region (11,631,182 11,665,626 bp LGIV), injected syncytial gonads dpy-5 (e907) animals &#239;&#172;?nal concentration 3 ng/&#206;&#188;l, pharyngeal targeted green &#239;&#172;&#8218;uorescent protein (GFP) reporter myo-2::GFP 10 ng/&#206;&#188;l plasmid pCeh361 contains wild-type dpy-5 rescues dpy-5 phenotype, 100 ng/&#206;&#188;l <br /> (Thacker et al., 2006). Injected worms plated (&#239;&#172;?ve P0s plate) phenotypically wild-type animals F1s generation expressed pharyngeal GFP reporter individually isolated propagated. <br /> GFP positive wild-type F2 animals transmitting transgenic array selected PCR speci&#239;&#172;?c fosmid backbone performed con&#239;&#172;?rm presence fosmid (Supplementary Table 1). single, stably <br /> transmitting, animal selected create transgenic lines. conduct transgenic rescue experiments indi- <br /> vidual hermaphrodites carrying WRM0615dH10 + myo-2::GFP + pCeh361 crossed +/+ N2 males obtain dpy-5 (e907)/+[WRM0615dH10 + myo-2::GFP + pCeh361] males. GFP-expressing males crossed let-65 (s1222); unc-22 (s7) let-65 (s1803); unc-22 (s7) heterozygously maintained <br /> hermaphrodites. F1 progeny screened presence males ensure cross successful F1 wild-type hermaphrodites plated individually. F2 animals individually plated screened unconditional twitchers, indicative viable homozygous animals genotype let-65 (s1083 s1222) unc-22 (s7)/let-65 (s1083 s1222) unc-22 (s7)(IV)[WRM0615dH10 + myo2::GFP + pCeh361]. <br /> Unconditional twitchers individually plated incubated 20 &#194;&#176;C assess rescue. <br />  <br /> Microscopy <br />  <br />    Screening transgenic animals performed using Zeiss Axioscope (Quorum Technologies) set QImaging Camera appropriate &#239;&#172;?lter sets used GFP expression analysis. Animals immobilized 100 mM sodium azide  water) immediately prior imaging. <br />  <br /> Bioinformatics studies <br />  <br />    reference DNA sequences derived WormBase release WS208. Analysis sequence data image processing performed using BioEdit (http://www.mbio.ncsu.edu/BioEdit/BioEdit.html), <br /> WolFPSORT (Horton et al., 2007) (http://wolfpsort.org/), BLASTP. <br />  <br /> Results <br />  <br /> aCGH mapped de&#239;&#172;?ciencies physically de&#239;&#172;?ne seven zones unc-22 region LGIV <br />  <br />     effort characterize genes essential survival C. elegans previously generated mapped mutations confer lethal phenotype unc-22 region C. elegans genome (Rogalski <br /> et al., 1982). unc-22 region  unc-43 unc-31) represents approximately map units (mu) chromosome IV, spanned 35 de&#239;&#172;?ned genetic de&#239;&#172;?ciencies (Clark Baillie, 1992; Clark <br />  <br /> Table 1 <br /> aCGH de&#239;&#172;?ned de&#239;&#172;?ciency breakpoints gene complement. <br />  <br />   Zone        Left breakpoint                      Right breakpoint                     Genes zones         Size kb <br />  <br />               Left (bp)         Right (bp)         Left (bp)          Right (bp)        Leftmost       Rightmost <br />  <br />   1           10,942,926        10,947,094         11,070,201         11,076,154        Y69E1A.1       Y5F2A.3     123&#226;&#8364;&#8220;134 <br />   2           11,070,201        11,076,154         11,629,649         11,629,799        Y5F2A.4        lys-6       553&#226;&#8364;&#8220;560 <br />   3           11,629,649        11,629,799         11,639,511         11,639,570        F58B3.4        F58B3.7a     10&#226;&#8364;&#8220;11 <br />   4           11,639,511        11,639,570         11,983,621         11,983,655        F58B3.7a       unc-22a     343&#226;&#8364;&#8220;344 <br />   5           11,983,621        11,983,655         12,243,159         12,251,414        unc-22a        sre-16      252&#226;&#8364;&#8220;253 <br />   6           12,243,159        12,251,414         12,551,447         12,553,668        sre-19         ZK795.1a    315&#226;&#8364;&#8220;317 <br />   7           12,551,447        12,553,668         12,757,572         12,776,111        ZK795.1a       inx-6       196&#226;&#8364;&#8220;204 <br />  <br /> Regions annotated based probe sequence falling adjacent breakpoint position. <br />      Breakpoints falling single gene. <br />  822                                  M  Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br /> et al., 1988; Marra Baillie, 1994; Moerman Baillie, 1981; Rogalski Baillie, 1985; Rogalski et al., <br /> 1982; Schein et al., 1993). Previously, genetic analysis used identify position 31 lethal gene loci genetic de&#239;&#172;?ciency map (Clark et al., 1988).  precise molecular extent de&#239;&#172;?ciencies known construction de&#239;&#172;?nitive candidate gene lists corresponded mutated loci possible. <br />     physically de&#239;&#172;?ne de&#239;&#172;?ciency map unc-22 region analyzed informative <br /> de&#239;&#172;?ciencies using array comparative genomic hybridization (aCGH). aCGH method used compare DNA ratio individual samples organism order determine copy number var- <br /> iations chromosomal genome-wide scale (Dhami et al., 2005). method used determine precise physical extent genetic de&#239;&#172;?ciencies C. elegans (Jones et al., 2007). cases, array data suf&#239;&#172;?cient position de&#239;&#172;?ciency deletion breakpoints single-gene resolution approximately <br /> 6 kb (Table 1). Deletion breakpoints fell single genes genes. Using data seven-zone physical de&#239;&#172;?ciency map constructed spanning approximately 800 kb unc-22 region <br /> (Fig. 1A Table 1). Furthermore, incorporating previously generated genetic mapping data mo- <br /> lecularly unidenti&#239;&#172;?ed lethal loci known map region, able assign locus precisely <br /> de&#239;&#172;?ned list candidate genes. largest number lethal loci map zone de&#239;&#172;?ciency map <br />  genes). Zone contains seven lethal loci, zones seven contain lethal loci  <br /> Zones &#239;&#172;?ve contain lethal loci  Finally, single lethal locus positioned zone (Fig. 1A). <br />  <br /> Sequence analysis let-65 <br />  <br />     let-65 single lethal loci mapped zone de&#239;&#172;?ciency map (Fig. 1A). Based physical extent zone annotated genes, F58B3.4, F58B3.5, F58B3.6 F58B3.7 candi- <br /> dates let-65. let-65 represented alleles, isolated EMS screens larval lethal- <br /> ity (Clark Baillie, 1992; Clark et al., 1988; Rogalski Baillie, 1985; Rogalski et al., 1982). mutations let-65 confer lethal phenotype correlated available RNAi data order rank candidate genes <br /> based severity reported RNAi phenotype (Kamath et al., 2003; Maeda et al., 2001; Piano <br /> et al., 2002; Simmer et al., 2003; Sonnichsen et al., 2005) (Fig. 1B). Using approach identi&#239;&#172;?ed <br /> F58B3.5 F58B3.4 strongest candidate genes. identify mutations F58B3.5 F58B3.4 genomic regions ampli&#239;&#172;?ed PCR let-65 (s254) homozygous animals sequenced. comparison sequence data reference C. elegans genome (WormBase release WS208) revealed mutations associated F58B3.4.  point mutation identi&#239;&#172;?ed coding region F58B3.5. mutation resulted <span id='am-32' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span>glutamic acid</span> lysine substitution (Glu364Lys) highly conserved <br /> amino acid (Fig. 2 Table 2). provide additional con&#239;&#172;?rmation mutation F58B3.5 associated let-65 sequenced F58B3.5 ORF remaining let-65 alleles, detecting presence additional <br /> mutations seven alleles tested. mutations identi&#239;&#172;?ed G &#226;&#8224;&#8217; C &#226;&#8224;&#8217; T transitions,  <br /> sistent known mutational spectrum ethane methylsulfonate (EMS) mutagenesis (Fig. 2, Table 2) <br /> (Maple Moller, 2007). case, exception s1777 s1222, identi&#239;&#172;?ed mutations substitutions highly conserved amino acids predicted MetRS domain F58B3.5 (Figs. 1C 2). lesion s1777 resulted 508 amino acid truncation, s1222 contained mutation &#239;&#172;?rst base &#239;&#172;?rst intron. s1222 potentially prevents splicing &#239;&#172;?rst intron, result premature trunca- <br /> tion protein amino acid 28 (Fig. 1C). data demonstrate let-65 F58B3.5 repre- <br /> sent gene. F58B3.5 encodes cytoplasmic Class synthetase, methionyl tRNA synthetase (MetRS) evolutionary conserved, 58% identity 71% similarity human homologue (BlastP <br /> alignment Altschul et al., 1997, 2005) (Fig. 2). renamed F58B3.5 mars-1 (Methionyl Amino-acyl <br /> tRNA Synthetase-1). <br />  <br /> Rescue let-65 fosmid containing mars-1 <br />  <br />    shown mutations mars-1 present let-65 animals. provide evidence mutations caused lethality associated let-65, performed transgenic rescue experiments. fosmid clone WRM0615dH10 spans entire mars-1 genomic region, excludes F58B3.4 ORF <br /> (Fig. 1B). transgenic animals carrying WRM0615dH10 extra-chromosomal array <br />                                              M  Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830                                         823 <br />  <br />   <br />  <br />  <br />  <br /> B <br />  <br />  <br />  <br />  <br /> C <br />  <br />  <br />  <br />  <br /> Fig. 1. Physical map unc-22 region LGIV including aCGH de&#239;&#172;?ciency mapping data.  Schematic LGIV showing regions <br /> uncovered de&#239;&#172;?ciencies analyzed study shown &#239;&#172;?gure (dark gray). aCGH data region analyzed de&#239;&#172;?ciencies shown. Commonly used genetic markers physical positions de&#239;&#172;?ciency breakpoints (estimated 0.5 kb) shown array data. schematic describing seven zones de&#239;&#172;?ned breakpoints de&#239;&#172;?ciencies shown array data. let-56 zone &#239;&#172;?ve zone  B. expansion zone showing candidate genes let-65. region covered fosmid used complementation tests depicted gene models. C. Schematic showing protein <br /> structures human MetRS C. elegans MARS-1. Conserved domains locations indicated. positions identi&#239;&#172;?ed <br /> mars-1 mutations shown. Figure scale. <br />  824                                        M  Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br />  <br />  <br /> Fig. 2. C. elegans MARS-1 alignment human (GenBank accession  NP_004981), D. melanogaster (GenBank accession  <br /> NP_611382), S. cerevisiae (GenBank accession  CAA97293) orthologs showing conserved amino acids identi&#239;&#172;?ed let-65 <br /> mutations. MARS-1 evolutionarily conserved 58% identity 71% similarity human ortholog. Red lines indicate positions amino acid changes let-65 alleles. s1222 prevents splicing &#239;&#172;?rst intron. s1777 induces stop codon prema- <br /> turely truncates protein. <br />                                             M  Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830                        825 <br />  <br />  <br /> Table 2 <br /> Mutations identi&#239;&#172;?ed let-65 sequencing BioEdit analysis. <br />  <br />   Allele                       <span id='am-17' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>Nucleotide</span> change                         Codon mutation            Amino acid change <br />  <br />   s174                         ND                                        ND                        NDa <br />   s254                         G1188A                                    gag &#226;&#8224;&#8217; aag                 Glu-364 &#226;&#8224;&#8217; Lys <br />   s694                         G655A                                     gga &#226;&#8224;&#8217; gaa                 <span id='am-44' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-45' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>Gly</span>-186 &#226;&#8224;&#8217; Glu <br />   s1083                        G429A                                     gga &#226;&#8224;&#8217; gaa                 Gly-126 &#226;&#8224;&#8217; Glu <br />   s1084                        G921A                                     gga &#226;&#8224;&#8217; aga                 Gly-275 &#226;&#8224;&#8217; Arg <br />   s1154                        C1297T                                    ccc &#226;&#8224;&#8217; ctc                 Pro-400 &#226;&#8224;&#8217; Leu <br />   s1222                        G81A                                      (&#226;&#710;&#8217;) &#226;&#8224;&#8217; aga                 (&#226;&#710;&#8217;)27 &#226;&#8224;&#8217; Arg <br />   s1730                        C174T                                     gct &#226;&#8224;&#8217; gtt                 Ala-41 &#226;&#8224;&#8217; Val <br />   s1777                        G1324A                                    tgg &#226;&#8224;&#8217; tag                 Trp-409 &#226;&#8224;&#8217; Stop <br />       determined &#226;&#8364;&#8221; potentially complex lesion. <br />  <br />  <br /> successfully speci&#239;&#172;?cally rescued lethality associated alleles let-65 tested (s1083 s1222). <br /> Furthermore transgenic animals super&#239;&#172;?cially wild type appearance twitching pheno- <br /> type presence unc-22 (s7) homozygous mutation, used visible phenotypic marker. <br /> Individual viable animals isolated start strains BC8656 BC8696 alleles s1083 s1222 respec- <br /> tively, cultured multiple generations con&#239;&#172;?rm continued rescue lethal phenotype. sequence rescue data con&#239;&#172;?rm mutation mars-1 gives rise lethal phenotype let-65 animals. <br />  <br /> Mitochondrial methionyl tRNA synthetase candidate <br />  <br />     MARS-1 predicted encode cytoplasmic form MetRS (Havrylenko et al., 2010). Computa- <br /> tional analysis predicted MARS-1 protein (WormBase WS209) demonstrated mars-1 encodes protein localizes subcellularly cytoplasm. BLASTP analysis Y105E8A.20 (Altschul et al., 1997), second gene predicted encode methionyl tRNA synthetase C. elegans (WormBase WS211), implies <br /> mitochondrial localization encoded protein. <br />  <br /> Discussion <br />  <br />     study used aCGH de&#239;&#172;?ne physical de&#239;&#172;?ciency map 2 Mb portion LGIV C. elegans. <br /> Using de&#239;&#172;?ciency map correlated physical candidate gene lists mutations number previously <br /> identi&#239;&#172;?ed essential genes (Clark, 1990; Clark Lyckegaard, 1988; Moerman, 1980; Rogalski Baillie, <br /> 1985; Rogalski et al., 1982). analysis able determine genetically identi&#239;&#172;?ed <br /> locus let-65 corresponds mars-1, encodes C. elegans homologue MetRS. let-65 number lethal genes previously identi&#239;&#172;?ed EMS screens essential genes right end LGIV. screens <br /> generated seven alleles let-65 resulted lethality (s253, s1777, s1083, s1154, s1222 s1730), additional allele causing sterility (s694). Previously reported inactivation MARS-1 RNAi leads embryonic lethality (Kamath et al., 2003; Sonnichsen et al., 2005).  loss MARS-1 <br /> function lead sterility (Havrylenko et al., 2011). Furthermore, correlation available phenotypic  <br /> formation ARS family members deleted targeted mutations demonstrates complete correlation lethal sterile phenotypes, highlighting essential nature family enzymes (Table 3). data indicate mars-1 essential processes require large scale protein biosynthesis, embryogenesis (Havrylenko et al., 2011). <br />     C. elegans cytoplasmic MetRS (MARS-1) 917 amino acid protein multiple predicted func- <br /> tional domains. minimal core enzyme, bacteria Aquifex aeolicus, consists core catalytic <br /> domain (CAT), distinguished Rossmann fold characteristic domain Class aminoacyl <br /> tRNA synthetase enzymes (catalyzes aminoacylation reaction), anticodon-binding domain (ABD) <br /> (promotes tRNA&#226;&#8364;&#8220;protein association) (Nakanishi et al., 2005). Additional domains, eukaryotes, appended core domain enzyme believed increase stability tRNA&#226;&#8364;&#8220;protein complexes (Havrylenko et al., 2010). humans, MARS carries C-terminal domain causes slow release aminoacyl-tRNA provides aminoacylation reaction limiting step (Havrylenko <br /> et al., 2010). C. elegans, MARS-1 protein model contains t-RNA binding domain (tRBD) 170 <br />  826                                        M  Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br /> Table 3 <br /> Correlation known viability phenotypes ARS-family genes. <br /> Data source: WormBase, http://www.wormbase.org, release WS243, 2014. <br />  <br />   Gene                                   RNAi phenotype                                    Mutation phenotype [allele] <br />  <br />   aars-1                                      ste <br />   aars-2                                      emb; lvl; ste <br />   cars-1                                      emb; let; lvl; ste <br />   cars-2                                      ste <br />   dars-1                                      emb; lvl; ste <br />   dars-2                                      emb; lvl; ste                                     let; ste [tm3799] <br />   ears-1                                      emb; lvl <br />   ears-2                                      emb; lvl; ste <br />   fars-1                                      emb; lvl; ste <br />   fars-2                                      ND <br />   fars-3                                      emb; lvl; ste <br />   gars-1                                      emb; lvl; ste <br />   hars-1                                      emb; let; lvl; ste                                let; ste [tm4074] (Pierce et al., 2011) <br />   iars-1                                      emb; ste <br />   iars-1                                      emb; let <br />   kars-1                                      emb; lvl; ste <br />   lars-1                                      emb; let; lvl; ste <br />   lars-2                                      emb                                               let; ste [tm5774] <br />   mars-1                                      emb; lvl <br />   nars-1                                      emb; let; lvl; ste                                let; ste [tm524] <br />   nars-2                                      ND <br />   pars-1                                      emb; let; lvl; ste <br />   pars-2                                      ND <br />   qars-1                                      emb; let; lvl; ste <br />   rars-1                                      emb; lvl; ste                                     emb; lvl; ste [gc47] <br />   rars-2                                      emb; ste <br />   sars-1                                      emb; lvl; ste <br />   sars-2                                      emb <br />   tars-1                                      emb; let; lvl; ste <br />   vars-1                                      emb; lvl; ste <br />   vars-2                                      emb; let; lvl; ste                                let; ste [tm3947] <br />   wars-1                                      emb; lvl; ste <br />   yars-1                                      emb; lvl; ste <br />  <br /> emb = embryonic lethality, lvl = larval lethality, let = lethal, ste = sterile, ND = determined. <br />  <br />  <br /> residues appended cis C-terminus enzyme, homologous tRBD domain appended C-terminus MetRS Oriza sativa (rice) (Havrylenko et al., 2010).  C. elegans, domain separated ABD domain putative protein-binding domain approximately 150 residues <br /> (Havrylenko et al., 2010). domain similar human P43 protein, non-synthetase compo- <br /> nent aminoacyl tRNA synthetase complex required assembly protein complex <br /> (Havrylenko et al., 2011). predicted C. elegans MARS-1 catalytic core domain comprises amino acids 36 407 (Marchler-Bauer et al., 2009). let-65 mutations reside domain (s1730, s1083, <br /> s694, s1084, s254 s1154) suggesting disrupt function catalytic core. EMS induced <br /> mutation let-65(s1777) creates truncated protein lacks anticodon-binding domain (AA416 AA545) (Marchler-Bauer et al., 2009) tRNA binding domain (AA755 AA857) (Marchler-Bauer <br /> et al., 2009). let-65(s1222) contains mutation &#239;&#172;?rst base &#239;&#172;?rst intron putatively preventing splic- <br /> ing &#239;&#172;?rst intron resulting putative premature truncation amino acid 28. <br />     mars-1 highly conserved encodes protein unique signi&#239;&#172;?cance translational control. Struc- <br /> tures MARS-1 Escherichia coli (Mechulam et al., 1999) Thermus thermophiles (Sugiura et al., 2000) reported. MARS-1 E. coli greater identity T. thermophiles protein C. elegans. <br /> Amino acids altered C. elegans missense mutations located conserved regions E. coli T. thermophiles: let-65(s1083): &#206;&#178;C; let-65(s694): turn &#206;&#178;2 &#206;&#178;3 (zinc-binding domain); <br /> let-65(s254): turn &#206;&#178;7 &#206;&#177;6; let-65(s1154): turn &#206;&#177;7 &#206;&#178;8. let-65(s1730) turn &#206;&#178;A &#206; let-65(s1084) turn &#206;&#178;5 &#206;&#178;6. identi&#239;&#172;?ed let-65 changes located E. coli Rossman-like fold. <br />                                      M  Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830                         827 <br />  <br />  <br />     let-65(s1777) contains nonsense mutation tryptophan codon amino acid 408 amber stop codon. let-65(s1222) mutation putatively prevents splicing &#239;&#172;?rst intron resulting premature truncation, truncated protein lacks majority synthetase including catalytic <br /> domain, binding domains, anticodon-binding domain, tRNA binding domain. s1777 s1222   putatively null alleles. phenotypes (mid-larval arrest) (Clark Baillie, 1992; <br /> Clark et al., 1988) mutants indistinguishable let-65 alleles. Based obser- <br /> vation predict alleles null MARS-1 function, reduce MARS-1 activity threshold <br /> required survival.  survival nematodes, containing alleles mid-larva, suggests early developmental stages, mother supplies MARS-1 eggs oogenesis. <br />     let-65(s694) let-65(s1083) contain mutations change <span id='am-56' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-57' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>glycine</span> glutamic acid amino acids 186 126 respectively. amino acids sequence blocks conserved human yeast. <br /> let-65(s1084) contains mutation changes <span id='am-68' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-69' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>glycine</span> arginine amino acid 275. Generally, change small non-charged non-hydrophilic glycine large positively charged hydrophilic arginine disrupt structure protein render non-functional, case let-65 cause lethality. let-65 (s1730) contains mutation changes alanine valine position 41. humans, <br /> Drosophila melanogaster Saccharomyces cerevisiae, serine codon located position highly conserved sequence block. change polar amino acid (serine) non-polar <br /> amino acid (valine) disrupt protein&apos;s structure replacing alanine valine analyzed available sequences con&#239;&#172;?rm C. elegans wild type reference sequence position 41. analyzing sequence stretch let-65 alleles concluded de&#239;&#172;?nitively alanine present position 41. mutation let-65(s254) replaces conserved acidic amino acid (glutamic acid) basic amino acid (lysine) position 364. position catalytic domain MARS-1. let- <br /> 65(s1154) polar amino acid (proline) instead non-polar amino acid (leucine) position 400 catalytic domain. Additionally, alterations proline &#239;&#172;&#8218;anking regions active <br /> sites known change activity protein (Proline bracket hypothesis) (Kini et al., 1998), mutation exhibit drastic effects MARS-1 activity. <br />     Typically, eukaryotic cells forms ARS: cytoplasmic mitochondrial (Walker et al., 1983). <br /> MARS-1 localizes cytoplasm (Havrylenko et al., 2010).  attempted &#239;&#172;?nd candidate <br /> gene coding mitochondrial form enzyme. C. elegans genome contains second ORF, <br /> Y105E8A.20 predicted encode methionyl tRNA synthetase (WormBase WS211). Y105E8A.20 1221 bp gene encodes 406 amino acid protein. BlastP used search Swiss-Prot database MARS-1 amino acid sequences queries (Altschul et al., 1997, 2005). BlastP searches <br /> S. cerevisiae used canonical genome identify protein localizations C. elegans. S. cerevisiae used s simplest regard identifying open reading frames (ORFs), highly characterized eukaryotic genome (Fisk et al., 2006). BlastP localization, based S. cerevisiae <br /> matches, predicted mitochondrial localization protein encoded Y105E8A.20. <br />     C. elegans million mutations project (MMP) comprises library 2007 mutagenized ho- <br /> mozygous C. elegans strains sequenced identify resulting mutations. aim project generate mutant alleles gene C. elegans genome.  limitation approach severely detrimental mutations genes required viability select- <br /> ed strains viable fertile. 14 MMP mutations recorded mars-1, missense mutations single exception mutation predicted affect <br /> splicing intron C-terminus protein. intron conserved species <br /> (Table 4). remaining 13 alleles 10 affect residues conserved. Interestingly mutations reside catalytic core enzyme. strains viable    unlikely lesions severely affect function protein. Alternatively secondary <br /> mutations strains present compensate detrimental effect enzyme function. studies required ascertain case. <br />     study isolated identi&#239;&#172;?ed mars-1 alleles lead loss function pheno- <br /> type. Furthermore, number alleles isolated mars-1 greater average number alleles <br /> isolated genes screens type (Berg et al., 2001; Brenner, 1974; Deniziak Barciszewski, <br /> 2001; Horton et al., 2007; Jones et al., 2007; Rozen Skaletsky, 2000; Thacker et al., 2006). indicates let-65 (mars-1) large mutagenic target EMS. availability loss function alleles allows exploration gene&apos;s structure function tractable model    828                                        M  Alriyami et al. / Meta Gene 2 (2014) 819&#226;&#8364;&#8220;830 <br />  <br />  <br /> Table 4 <br /> mars-1 MMP mutations. <br />  <br />   Allele            Mutation           Effect                  Domain affected            Type                    Conserved residue <br />  <br />   gk213073          C&#226;&#8224;&#8217;T                A887T                                         Missense                  gk410589          G&#226;&#8224;                P865S                                         Missense                  gk445673          C&#226;&#8224;&#8217;T                Affects splicing                              Intron, splicing          gk485164          C&#226;&#8224;&#8217;T                A453T                   Anticodon_1                Missense                  gk509072          C&#226;&#8224;&#8217;T                D190N                   tRNA-synt_1g               Missense                Yes <br />   gk515000          C&#226;&#8224;&#8217;T                G668E                   coiled_coil_region         Missense                  gk541772          G&#226;&#8224;                P265L                   tRNA-synt_1g               Missense                Yes <br />   gk550543          C&#226;&#8224;&#8217;T                A613T                   Anticodon_1                Missense                  gk636296          G&#226;&#8224;                S837L                   tRNA_bind                  Missense                  gk656240          T&#226;&#8224;&#8217;C                M844V                   tRNA_bind                  Missense                  gk719138          C&#226;&#8224;&#8217;T                A431T                   tRNA-synt_1g               Missense                Yes <br />   gk780068          C&#226;&#8224;&#8217;T                D752N                                         Missense                  gk793812          A&#226;&#8224;&#8217;T                F412L                   tRNA-synt_1g               Missense                  gk889082          G&#226;&#8224;                A520V                   Anticodon_1                Missense                 <br />  <br /> facilitate better understanding ARS, lead deeper understanding complete protein <br /> synthesis machinery organisms including humans. <br />     Supplementary data article online http://dx.doi.org/10.1016/j.mgene.2014.08.006. <br />  <br />  <br /> Abbreviations <br /> mRNA      messenger RNAs <br /> tRNA      transfer RNAs <br /> ARS       aminoacyl-tRNA synthetases <br /> METRS methionyl tRNA synthetase <br /> aCGH      oligonucleotide array comparative genomic hybridization <br /> EMS       ethyl methanesulfonate <br /> mars-1 C. elegans methionyl tRNA synthetase encoding gene <br /> GFP       green &#239;&#172;&#8218;uorescent protein <br /> NLS       nuclear localization sequence <br /> MARS-1 C. elegans methionyl tRNA synthetase <br /> </body></html>