<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natl. Acad. Sci. USA <br /> Vol. 91, pp. 12696-12700, December 1994 <br /> Genetics <br />  <br />  <br /> Allele-specific quantification Drosophila engrailed invected transcripts <br /> ANDREW S. GOLDSBOROUGH THOMAS B. KORNBERG <br /> Department Biochemistry Biophysics, University California, San Francisco, CA 94143 <br /> Communicated Edward B. Lewis, September 26, 1994 (receivedfor review August 2, 1994) <br />  <br /> ABSTRACT           Changes levels transcription                                                                    Tissue dissection _                        mRNA  purification <br /> difficult gauge animals lethal mutations. exam-                                                                                                             cDNA synthesis _ <br /> ple, mutations regulatory region essential gene                                                                 PCR amplification                         _       Sequencing secondary consequences complicate attempts                                                                                                                                       PhosphorImager analysis <br /> quantify transcripts produced mutant gene.                                                                                                    B method circumvents problem revealing                          ... ....:......... <br />                                                                                        ddG ddG <br />                                                                                         ..................... <br />                                                                                       .....................                        ddG............... <br />                                                                                                                                             .....             .ddA ddG                                   ..................... <br />                                                                                                                                                                                                                                          dG                     ddA ddG relative transcript produced allele                                                ... . <br />                                                                                           . . . . ........... . <br />                                                                                                               .   .                           _....                ...    ......... <br />                                                                                                                                                                           ::.:.:... <br />                                                                                                                                                                                                     .... .............. <br />                                                                                                                                                                                                                                        ...... <br />                                                                                                                                                                                                                                            _.... <br />  <br />  <br />  <br />  heterozygote. method, recessive lethal mutations                                           :......                                          ... <br />                                                                                                                                                                                                                                         ........ <br />  <br />                                                                                                                                                                                                                                    .....:::. analyzed animals phenotypically wild type.                               .... ........                                                                ... <br />                                                                                                                                                                            ..                     ....                    .... .                      ....... <br />                                                                                                                                ......................                                                                              .... <br />                                                                                                                                                                                                                                      ........ <br />                                                                                                                                                                                                                                      ........ . <br />                                                                                             .....    ......        ...   ...                  ...       .:...........                         .    ..            .                 .    ... <br />  <br />                                                                                                                                                                           ..................................                                       ......... used technique analyze mutations regulatory                                                                  .................. <br />                                                                                                               ..                                              ..                                                                                                  .. <br />  <br />  <br />  <br />  <br />                                                                                       :.:           ::.::....... :..            ......... <br /> region Drosophila engrailed gene trun-                                           :::.. <br />                                                                                                                                                                                       ..... <br />  <br />  <br />  <br /> cations reduce transcription levels depend tissue location chromosomal break. mutations affect expression linked                                                                      ........... <br />                                                                                                                                                                                                                                                                        .....   .......... <br />  <br />  <br />  <br />  <br /> invected gene, suggesting engrailed invected share complex set regulatory elements operate 85                            Uk Uk                                                                  B3razil                                                                            Brazil kb. suggest technique general utility quantitation allele-specific transcripts,                              FIG. 1. Procedure allele-specific quantification transcripts. <br /> amounts tissue limiting.                                                      Tissue dissected mRNA purified strand cDNA <br />                                                                                      synthesis PCR amplification engrailed invected exons.                                                                                      coamplification cDNA derived mutant (M) wild-type <br /> Genes present copies diploid organisms                        (WT) alleles removes need carefully calibrate PCR amplifica- <br /> assumed contribute equally RNA production.                             tions reference transcripts. PCR primers span polymor- <br /> exceptions arise course normal development                             phism intron allow discrimination transcripts [e.g., allelic exclusion maturation immune (1)                     avoid genomic DNA amplification. Polymorphisms revealed olfactory (2) systems parental imprinting (3)]                       sequencing PCR product representing transcript mutations alter promoter utilization RNA stability. Al-                           quantified following gel electrophoresis PhosphorImager analy- <br />                                                                                      sis.   Hypothetical representation gel bands obtained increasing documenting                        single sequencing reaction chain termiinator dideoxygua- <br /> variations, available methods accurate sensitive                         nosine <span id='am-1' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span>triphosphate</span> (ddG) engrailed cDNA derived Ukraine <br /> quantitation detect subtle changes. Northern blotting,                        (Uk) Oregon-R   homozygotes heterozygotes (Uk . <br /> RNase protection, quantitative PCR measure total                             insertion polymorphism Ukraine cDNA produces shift <br /> transcript levels, crude comparisons provide                            indicated arrows. (B) Hypothetical representation depend normalizing data unrelated reference tran-                            invected cDNA sequence Brazil Oregon-R homozygotes script abundance (4).  reference transcripts vary                        heterozygote (Brazil  containing single base change poly- different tissues changing growth conditions. Fur-                        morphism (arrow). sequencing lanes (ddG) (ddA) thermore, Northern blotting RNase protection                           cDNA preparation illustrate polymorphism.  general, distinguish transcripts contributed gene heterozygote, use methods limited                        gous genes standardization reference transcript. studies homozygous mutants. consequence,                           rationale ofthe method follows. cell lethal mutations largely precluded analysis.                          different copies actively transcribed gene, RNA <br />    measuring relative abundance transcripts                       pool contain transcripts copy proportion heterozygous human tissues cells increased                        relative rate synthesis half-life. tran- <br /> mutant genes identified isolated. example                        scripts different copies distinguished myotonin-protein kinase gene, expansions                           size sequence, resolving differences provides repetitive trinucleotide sequence associated myotonic                       measure relative abundance. method, termed <br /> muscular dystrophy (5-7). Efforts determine                           allele-specific amplification (ASA), uses simple sequence <br /> level myotonin-protein kinase gene transcripts altered                      polymorphisms monitor relative amounts transcript <br /> heterozygotes carrying autosomal dominant mutation                           produced alleles heterozygote. mRNA frustrated inadequacy available                              isolated small amounts tissue used template <br /> methods analysis.  use quantitative reverse                        reverse transcription provided template PCR <br /> transcription-PCR compare expression levels                                amplification sequences chosen include relevant <br /> groups led contradictory conclusions (8-11).                                      polymorphism span intron. Sequence analysis    report, new method yields relative measure transcript accumulation heterozy-                           classes DNA product yielded quantitative <br />                                                                                      measure relative transcript abundance (Fig. 1). publication costs article defrayed page charge <br /> payment. article marked &quot;advertisement&quot;                Abbreviations: ASA, allele-specific amplification; M, mutant; WT, accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                     wild type; ddN, dideoxynucleoside triphosphate. <br />  <br />                                                                              12696 <br />           Genetics:   Goldsborough Kornberg                                  Proc. Natl. Acad. Sci. USA 91 (1994)         12697 <br />  <br />              MATERIALS METHODS                                Brazil WT flies sequenced primer 5&apos;-ACCGGAT- <br />                                                                   GCTCCTGTGGCACC-3&apos; (nt 1486-1506). <br />    Genomic DNA Amplification. Individual flies homog-           Transcript Quantification. quantify relative abun- <br /> enized 100 ,l 6 M NaI (Bio 101) 1.5-ml Teflon        dance transcripts, amplified products M WT mRNA <br /> pestle, DNA purified binding glass particles        preparations fractionated sequenced standard <br /> (Geneclean, Bio 101). DNA eluted glass 20       sequencing gels. Gels fixed exposed overnight <br /> ,l water, 1 ,l4 used template PCR        Molecular Dynamics Phosphor screen. Data ob- <br /> amplification exon-specific oligonucleotides.                tained scanning exposed screen Molecular <br />    Drosophila Crosses, mRNA Purification, cDNA Synthe-        Dynamics Phosphorlmager equipped IMAGEQUANT <br /> sis. Heterozygote male flies engrailed   break-       Version 3.3 software. <br /> point/Black cells (Bc) genotype crossed Ukrainian            ddG-terminated sequence position 2017 engrailed <br /> WT females breakpoint/Ukraine larvae identified          quantified follows: position <span id='am-6' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> 2017 absence Bc phenotype. Approximately 1-3 ,ug         derived M Ukraine (WT) mRNA separated tissue dissected late instar larvae lysed   bases sequenced 3&apos; cDNA poly(U) paper (2 mm2; Hybond-mAP, Amersham)             presence additional nucleotides WT tran- <br /> presence 4 M guanidinium isothiocyanate/10 mM                  script. relative abundance transcripts Tris HCl, pH 7.5/1 mM EDTA/0.5 M LiCl. mRNA stable             determined directly specific bands DNA- paper 2 days 4&#194;&#176;C.       sequencing gel calculating average pixel value washed times 200 /, 0.5 M LiCl times          bands, subtracting background, determining area 70% EtOH, poly(U) paper briefly air dried,           bands integration. Expression M transcript added 10 Al water, heated 10 min 70&#194;&#176;C. cDNA      calculated percentage WT ratio M prepared using Superscript II according manufac-       WT bands. values M WT reflect turer&apos;s (BRL) instructions. Reverse-transcription reactions       synthetic products single sequencing reaction performed addition oligo(dT)           amplification RNA preparation, controlling poly(U) bound Hybond-mAP paper served                variations reaction conditions loading efficiency effective primer. Incubations 30 min 42&#194;&#176;C, 30 min     unnecessary. 45&#194;&#176;C, 10 min 75&#194;&#176;C.                                         analysis invected transcripts, assay  <br />    PCR Amplification. Amplified products engrailed       volved sequencing lanes, necessary com- <br /> invected cDNAs prepared follows. lower         pensate variations loading follows. Corrections efficiency engrailed cDNA amplified,         differences loading normalizing values <br /> second amplification step (semi-nested PCR) performed.        experimental bands lane selected reference <br /> engrailed cDNA (1 /4) amplified 10-,ul PCR reaction      bands. crosses en49, en63, en59, en2, poly- <br /> mixture containing final concentrations 15 mM Tris HCl      morphisms caused C -- T transitions positions 826 pH 8.8, 2.5 mM MgCl2, 60 mM KCl, 0.4 mM dNTPs, 0.4 mM             834 gave rise reciprocal changes DNA primer oligonucledotides [5&apos;-ATCGTCCCAGCTCAGGAC-                  ddG ddA lanes positions. Reference bands <br /> 3&apos; 5&apos;-GTTGCTCGTTAACATTTCGC-3&apos; (nt 1151-1528                   821 823 ddG lane 820 829 ddA 2332-2351, respectively)], 0.4 unit Taq DNA            lane. crosses en52, en61, en67, polymor- <br /> polymerase. Conditions PCR follows: 30 cycles         phisms caused C -- T transitions positions 1341 30 sec 94&#194;&#176;C, 30 sec 65&#194;&#176;C, 60 sec 72&#194;&#176;C. 1-,ul    1347 scored ddG ddA sequencing <br /> aliquot primary amplification product added       reactions. Reference bands nt 1348 1349 ddG <br /> 25-,ul PCR reaction mixture primers 5&apos;-TTGGC-            lane 1355 ddA lane. Normalized pixel values CCGCCTCAAGCGG-3&apos; 5&apos;-GTTGCTCGTTAACATT-                         polymorphic bands expressed percentage WT. <br /> TCGC-3&apos; (nt 1609-1626 2332-2351, respectively). forward PCR primers designed span introns avoid                                    RESULTS <br /> genomic DNA amplification. <br />    invected cDNA amplified 25-,ul reaction mixture       demonstrate sensitivity validity ASA  Fig. 1 <br /> containing final concentrations 20 mM Tris HCl pH 8.3,      Materials Methods), studied expression 1 mM MgCl2, 50 mM KCl, 0.4 mM dNTPs, 0.4 mM                engrailed invected transcripts genetic contexts. <br /> primer oligonucleotides [5&apos;-CAACTGCCTCCCTCCAC-                    Fig. 2 depicts linked genes, opposing transcrip- <br /> TGG-3&apos; 5&apos;-TGGACGTCGCCGGCTTCTTGG-3&apos; (nt                        tion units, positions 11 mutations chromo- <br /> 772-791 1613-1633, respectively)]. PCR cycles 1          somal rearrangements affect engrailed function various ex- <br /> min 94&#194;&#176;C, 1 min 55&#194;&#176;C, 2 min 72&#194;&#176;C. invected       tents. Polymorphic changes engrailed invected se- <br /> PCR products spanned intron 2 sequenced             quences used distinguish allelic <br /> ddG ddA reveal polymorphisms allow quanti-          transcripts sought laboratory strains tative analysis transcripts.                               seven WT strains isolated geographically <br />    DNA Sequencing. engrailed PCR product purified         distant regions (e.g., Oregon-R, Canton-S, Brazil, Ukraine, <br /> (Geneclean) following agarose gel electrophoresis directly    Siberia, Australia, Serbia). Sequencing engrailed  <br /> sequenced. Template DNA (200 ng) primer [200 ng; 5&apos;-          vected exons amplified strains revealed differences <br /> TACTTTCGGCTATCCAGG-3&apos; (nt 2094-2111)] mixed               engrailed 3 &apos;-untranslated regions CyO chromosome total volume 4.5 ,ul H20 containing 10%o (vol/vol)        [C -* (2236),  (2262),  (2265), +T (2268)], dimethyl sulfoxide. DNA denatured 94&#194;&#176;C 2 min       Ukraine chromosome [+TA (2015), +ATCCTA (2032), C quickly frozen dry ice. Labeling reactions       -* T (2041)], en1 chromosome [+TA (2015) C -&gt; formed Sequenase according manufacturer&apos;s (Unit-      (2236)]. sequence differences engrailed 3&apos; untranslated <br /> ed States Biochemical) instructions following modifi-    region WT strains. sequence <br /> cations. Labeling reactions included 10% dimethyl sulfoxide       differences 200-residue 3&apos; untranslated region terminated 2 min 5 ,ul ofddC containing 10o   invected strains.  comparisons se- <br /> dimethyl sulfoxide. invected PCR product crosses      quences invected exon 1 revealed differences Brazil <br /> en49, en63, en59, en2 WT sequenced     chromosome [T -- C (1161), C -* T (1341), C -) T (1347)] <br /> primer 5&apos;-TCCCAGACCCAATGCCGTGGC-3&apos; (nt 1088-                      CyO, Bc, en49, en63, en59, en2 chromosomes [C <br /> 1108). PCR products crosses 52, en61, en67    -- T (826 834)]. <br />  12698     Genetics: Goldsborough Kornberg                                                     Proc. Natl. Acad. Sci. USA 91              (1994) <br />                                                                                   c                                         r- <br />                                                                                                                                                                                                                 Ul~                                         c <br />                                                                                        CI               <br />                                                              CIA <br />                                                                                           r_                                                                                             -&apos;1 <br />           -60 kb                 -40                     -20                      0                           20                             40 <br />  <br />                                                           <br />                                 invected                                  engrailed <br />    FIG. 2. Map engrailed/invected region. Genomic map (horizontal line) engrailed invected transcription units indicated arrows. Intron/exon organization engrailed invected depicted, approximate locations 11 engrailed mutations (23). Scale kilobases start site engrailed transcription unit designated 0. Sequence polymorphisms engrailed invected transcripts: Ukraine <br /> DNA additional base pairs (ATCCTA) 3&apos;-untranslated region nt 2032 2033 engrailed transcript. M <br /> chromosomes sequence Canton-S region. single base pair polymorphisms (C -- T) exon invected en19, en63, en9, en2 positions 826 834 allowed identification transcripts derived chromosomes. Transcripts Brazil chromosome identified single polymorphisms (C -* T) nt 1341 1347. <br />     gauge proportion input mRNA correlates               genes heterozygous animals. engrailed mRNA isolated sequenced product ASA, separate preparations              tissues instar larva engrailed RNA isolated Oregon-R Ukraine flies copied                    expressed, including wing imaginal discs brains. tissue cDNA, amplified, spectrophotometrically quantified.                M/Ukraine (WT) heterozygous larvae, proportion DNA preparations mixed various propor-                   product derived M chromosome varied consid- <br /> tions, amplified, sequenced. ratio DNA ddG                erably manner dependent bands position 2017, distinguishes Oregon-R-                   particular mutant tissue type (Fig. 4). wing discs, <br /> Ukraine-derived transcripts, revealed excellent correlation              mutants lethal breakpoints 20 kb immediately ratio input cDNA ratio amplified                  upstream engrailed transcription start (en42, en49, en63, <br /> product (Fig. 3). Furthermore, genomic DNA isolated Oregon-R/Ukraine heterozygotes yielded equivalent amounts                                                      1        2            3   4       5 sequenced products PCR amplification  shown). <br /> Additional controls exploited polymorphic differences                                             Uk    .   _             _ <br /> tween 3&apos; untranslated regions CyO, Oregon-R,                                             O-R    _                     _   -   - <br /> Ukraine engrailed transcripts demonstrate propor-                                                      *O~ .:III <br />  <br />  <br />  <br /> tion amplified product equivalent type heterozy-                           1.5 <br /> gote fly  Oregon-R/Ukraine, Fig. 4; shown). conclude WT engrailed genes Oregon-R, CyO,                           U Ukraine flies produced equivalent amounts transcript                                1.0 ASA method accurately assesses relative                              - <br />  <br />  <br />  <br />  <br /> abundance initial template.                                                        demonstrate utility ASA quantifying relative                              0.5 <br /> levels transcripts, various mutations regulatory                        v <br />                                                                                   ce <br /> region engrailed analyzed. engrailed                         0) <br />                                                                                   u <br /> transcription unit relatively small (approximately 3.7 kb),                     _g. <br />                                                                                              0 contained large regulatory region                     54 <br /> defined mutations truncate gene far 15 kb                       C4 <br /> downstream 45 kb upstream transcription unit (12).                               0.5 mutants recessive  exception en61, <br /> en67, en1, en3, embryonic lethal hemizygous homozygous. developmental program                                      -1.0 *- <br /> grailed expression includes spatially restricted domains                                  -1.0    -0.5         0          0.5                 -1 .0 <br /> developmental stages, know                                                   log (ORIUK) imput template <br /> various gene truncations alter spatially temporally reg- <br /> ulated expression. early developmental lethality associ-                  FIG. 3. Quantitative ASA different ratios Ukraine ated mutants,  precluded anal-                 Oregon-R engrailed cDNAs. cDNA prepared homozygous <br /> ysis later stages. ASA provides convenient way study                Ukraine Oregon-R larval tissue amplified, quantified lethal mutations, M transcripts distin-                absorbance 260 nm, mnixed various proportions, reamplified, guished tissue obtained phenotypically normal, viable               sequenced. (Inset) region sequencing gel includes ddG <br /> heterozygotes.                                                              2017 reveals 6-bp polymorphic difference    representative set engrailed mutants (en52, en42, en49,             strains  legend Fig. 2). Lane 1, 10%o Ukraine:90%o Oregon-R; <br />                                                                             lane 2, 30% Ukraine:70%o Oregon-R; lane 3, 50% Ukraine:50% <br /> en63, en50, , en2, en24, en61, en67, en3) selected           Oregon-R; lane 4, 70%6 Ukraine:30%o Oregon-R; lane 5, 90%o <br /> analysis. mutants lack 6-bp insertion present              Ukraine:10% Oregon-R. Graphical representation data shows <br /> Ukraine WT engrailed gene, ASA used compare                   linear relationship ratio input cDNA  UK) relative proportion engrailed RNA produced M                      ratio UK product. <br />                Genetics: Goldsborough Komberg                                                      Proc. Natl. Acad. Sci. USA 91 (1994)          12699 <br />  <br />   WT                                                                      B     WT             -.m. <br />  <br />          M      };   -                                                                 M                                                         *=::.qft <br />  <br />  <br />  <br />  <br />         250                                                                       150 <br />  <br />  <br />   (1) 200 <br />                                                                                     100 <br />   -o <br />   .D 150 <br />  <br />         100 <br />   0)                                                                               50 <br />   c) <br />     <br />   0-    50 <br />  <br />          0                                                                             0 <br />                en52 en42 en49 en63 en5O en2 en24 en67 en3          O-R                          en52 en42 en49 en63 en5O en2 en24 en67 en3 O-R <br />                               Chromosome                                                                         Chromosome <br />   FIG. 4. Quantitative analysis engrailed expression heterozygous larval tissues. relevant portion sequencing gel pictured graphic representation data. lanes arranged left right proximal distal order positions mutant <br /> breakpoints. Relative levels expression shown percentage WT (100%).   Wing imaginal discs (approximately discs sample). (B) Larval brain (approximately brains sample). bar represents average independent preparations. values obtained multiple preparations highly reproducible. Expression engrailed WT Oregon-R (O-R) chromosome shown equal Ukraine (100%o 6%). horizontal broken lines graph represent WT levels engrailed expression. <br />                                                  &#194;&#177; <br />  <br />  <br />  <br /> en50, en2, en24) reduced expression M chromo-                      changing number engrailed invected transcripts 30% WT. contrast, nonlethal                         produced mutant chromosomes illustrates breakpoint mutation, en3, located 45 kb upstream,                    utility sensitivity. Unlike quantitative PCR meth- <br /> reduced expression wing disc expression                      ods, ASA requires careful calibration PCR <br /> 10%. Curiously, en52, lethal breakpoint 15 kb                     amplification reaction quantitative restriction enzyme <br /> downstream engrailed transcription unit, 2-fold                    digestion (14, 15).  method, WT transcripts <br /> greater expression M chromosome. Values obtained                     serve internal reference expression M multiple preparations highly reproducible.                          gene. ASA amplifies transcripts WT    Preparations larval brain (Fig. 4B) yielded quantita-                 M chromosomes  using set primers, tively different results. alleles breakpoints                 calibration standard necessary. simplifies region immediately upstream engrailed transcription                    improves reproducibility accuracy analysis. start site (en42, en49, en63) marginal effects            practice, necessary calibrate sequencing <br /> brain. contrast, distal breakpoint mutations en50, <br /> en2, en24 increasingly severe effects expression                                100 - <br /> levels. inverse relationship length 5&apos; <br /> regulatory region level expression suggests sequence elements regulating transcription larval brain                     0L)      80    - <br />  numerous varied function. <br />   Located approximately 20 kb downstream engrailed <br /> transcription unit gene, invected, developmental                        CL <br />                                                                                             60    - <br />                                                                                    30 <br /> program expression virtually identical  <br /> grailed (Fig. 2A). invected encodes protein closely <br /> related engrailed protein (13). analyzed invected                                 40 - <br />                                                                                             j <br />                                                                                    cJ <br /> transcripts heterozygous engrailed mutants determine gene truncations engrailed regulatory region <br />                                                                                   0~ <br />                                                                                     <br />                                                                                             0 <br />                                                                                             9 <br />                                                                                                              T affect invected expression. RNA isolated hetero- <br /> zygous larval wing imaginal discs used ASA taking <br /> advantage polymorphisms exon invected distinguish WT M chromosome transcripts (Fig. 2C).                                               en52        en63   en59   en2   en6l   en67 <br /> Quantitative sequence analysis revealed engrailed                                                          Chromosome <br /> mutants substantially alter invected expression (Fig. 5). mutants reduced expression, 50% WT levels en52                      FIG. 5. Quantitative analysis invected expression hetero- en67 heterozygotes, 15% WT en63                    zygous   wing imaginal discs. Graphic representation ASA en59 heterozygotes.                                                       heterozygous engrailed Ms reveals invected expression depends <br />                                                                               integrity regulatory region surrounding engrailed <br />                                                                               gene. exon invected amplified PCR primers <br />                           DISCUSSION                                          flank sequence polymorphisms. lanes arranged left <br />                                                                               right proximal distal order positions mutant shown ASA used accurately quantify                     breakpoints. Relative levels expression shown percentage relative abundance particular transcripts derived                 WT (100%). Expression invected WT Oregon-R <br /> multiple gene copies diploid nucleus. Application ASA                 chromo <br /> </body></html>