<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                       <br />  <br /> Essentiality centrality protein interaction <br /> networks revisited <br />  <br />  <br />  <br />   Abstract <br />   Background: Minimum dominating sets (MDSet) protein interaction networks allow control underlying <br />   protein interaction networks topological placement. essential proteins enriched MDSets, <br />   hypothesize statistical properties biological functions essential genes enhanced   focus essential MDSet proteins (e-MDSet). <br />   Results:  determined minimum dominating sets proteins (MDSet) interaction networks E. coli, S. <br />   cerevisiae H. sapiens, defined subsets proteins remaining protein reached single <br />   interaction. compared topological functional parameters essential, MDSet, essential MDSet <br />   (e-MDSet) proteins. particular, observed topological placement allowed e-MDSet proteins provide <br />   positive correlation degree lethality, connect protein complexes, stronger impact <br />   network resilience essential proteins  comparison essential proteins interactions <br />   e-MDSet proteins appeared frequently complexes, interactions e-MDSet proteins   complexes depleted. Finally, e-MDSet proteins classified functional groupings play central role   survival adaptability. <br />   Conclusions: determination e-MDSet organism highlights set proteins enhances enrichment <br />   signals biological functions essential proteins. consequence, surmise e-MDSets provide new <br />   method evaluating core proteins organism. <br />   Keywords: PPI, Essential proteins, Protein network, Protein function <br />  <br />  <br /> Background                                                                           prompting Nacher Akutsu determine minimum biological importance protein frequently                             dominating sets (MDSet) nodes, defined set cen- <br /> sidered question number interactions given                             trally located nodes provide control undirected net- <br /> protein involved [1-3], suggesting high topo-                             works [16]. Answering question nodes logical centrality indicator protein&#226;&#8364;&#8482;s importance                         predicted important control interaction <br /> [4-9]. addition, proteins involved                            networks translate directly functional sets proteins, <br /> large number protein complexes [10], signifying                              minimum dominating sets enriched dis- essentiality consequence complex                              ease related essential genes protein interaction net- <br /> volvement [5,9,11-14].                                                               works [17,18]. <br />   Focusing determination nodes control                                hypothesize essential proteins appear <br /> entire network, Liu et al. introduced maximum matching                             MDSets (e-MDSet) enhance enrichment sig- <br /> approach predict nodes allowed control vari-                          nals biological functions compared essential proteins <br /> ous technical, social biological networks [15].                           Specifically, considered high-quality protein  approach applicable directed networks,                             teractions S. cerevisiae H. sapiens  <br />                                                                                      termined large-scale yeast hybrid approaches, * Correspondence: wuchtys@cs.miami.edu                                               recently released high-throughput binary inter- <br /> 1 <br />  Department Science, University Miami, Coral Gables, FL               action set E. coli [19]. highly connected proteins <br /> 33146, USA                                                                           showed weak enrichment essential proteins recov- <br /> 2 <br />  Center Computational Science, University Miami, Coral Gables, FL <br /> 33146, USA                                                                           ered strong correlation protein&#226;&#8364;&#8482;s degree  <br />                                         &#194;&#169; 2015 Khuri Wuchty; licensee BioMed Central. Open Access article distributed terms Creative <br />                                         Commons Attribution License (http://creativecommons.org/licenses 4.0), permits unrestricted use, distribution,                                         reproduction medium, provided original work properly credited. Creative Commons Public Domain <br />                                         Dedication waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, <br />                                         unless stated. <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                                                                                 Page 2 8 <br />  <br />  <br />  <br />  <br /> tendency essential focused essential pro-                                                                 1,203 proteins experimentally obtained <br /> teins appeared corresponding MDSet                                                                     yeast hybrid approach [19]. source infor- <br /> (e-MDSet). impact combination essentiality                                                                   mation essential genes, collected 712 essential protein presence corresponding MDSet                                                                   genes E. coli 1,110 essential genes S. cerevisiae evidenced observation e-MDSet                                                                      DEG database [21], obtained 2,708 essential genes proteins likely connect protein complexes                                                                 H. sapiens online gene essentiality database (OGEE) <br /> essential proteins organisms.                                                                   [22]. defined MDSet optimized subset proteins  e-MDSet proteins predominantly oc-                                                                     interaction network remaining  e. <br /> cupied central positions networks connected                                                                    non-MDSet) protein reached interaction. protein complexes. corollary, e-MDSet proteins                                                                    non-MDSet protein connected enriched interactions occured com-                                                                     MDSet protein (Figure 1A). protein interaction <br /> plexes, vice versa. Taking closer look functional                                                                 networks determined corresponding minimum dominat- <br /> classifications e-MDSet proteins enhance                                                                     ing sets solving integer-based linear programming enrichment signals functional groups play role                                                               problem  Materials Methods). particular, responses external stimuli physiological                                                                 569 MDSet proteins yeast interactions network <br /> dition cells.                                                                                                           (21.26%), 352 proteins MDSet E. <br />                                                                                                                            coli interactions (29.2%) 940 MDSet proteins (15.9%) Results                                                                                                                    human interaction network. comparison pro- utilized network 6,225 high-quality interactions                                                                teins (&lt;kall &gt; = 3.9), mean degree MDSet proteins tween 2,640 proteins S. cerevisiae 17,523 high-quality                                                              yeast interaction network significantly increased <br /> links 5,926 human proteins entirely deter-                                                               (&lt;kMDSet &gt; = 8.5), observation held E. coli <br /> mined yeast hybrid approaches [20]. E. coli,                                                              (&lt;kall &gt; = 3.2, &lt;kMDSet &gt; = 5.6) human networks used recently released, map 1,938 interactions                                                                  (&lt;kall &gt; = 5.3, &lt;kMDSet &gt; = 15.5). turn, observed  <br />  <br />                                                                                                          B <br />                                                                                                                                        1.5 <br />                                                                                                            lg2 [fess. MDSet /fess.] <br />  <br />  <br />  <br />  <br />                                                                                                                                               1 <br />  <br />                                                                                                                                        0.5 <br />  <br />                                                                                                                                               0 <br />                                                                                                                                                                             E. coli <br />                                                                                                                                       -0.5                                  S. cerevisiae <br />                                                                                                                                                                             H. sapiens <br />                                                                                                                                               -1 <br />                                                                                                   MDS                                              1               10                       100 <br />                                                                                                                                                                degree <br />                       C7                                                                                   D <br />                                                                  7                                                                            7 <br />                                                  enr.essential <br />  <br />  <br />  <br />  <br />                                                                  6 <br />                                                                                                                           enrichmente-MDSet <br />  <br />  <br />  <br />  <br />                                          6                       5 <br />                       enrichment MDSet <br />  <br />  <br />  <br />  <br />                                                                  4                                                                            6 <br />                                                                  3 <br />                                          5                       2                                                                            5 <br />                                                                  1 <br />                                          4                       0 <br />                                                                      1   3   5   7   9 11 13 15 <br />                                                                                                                                               4 <br />                                          3                                                                                                    3 <br />                                          2                                                                                                    2 <br />                                          1                                                                                                    1 <br />                                          0                                                                                                    0 <br />                                              1                                       6              11                                             1   3   5   7        9    11    13       15 <br />                                                                                                          degree k <br />  Figure 1 e-MDSet proteins follow centrality-lethality rule.   toy network defined minimum dominating set (MDSet)  optimized subset nodes (red square symbol) remaining  e. non-MDSet) node (gray circle symbol) immediately <br />  reached step.  non-MDSet protein connected MDSet protein. (B) calculated corresponding <br />  MDSets binary interaction networks E. coli, S. cerevisiae H. sapiens, grouped proteins bins logarithmically increasing degree.  bin determined fraction essential proteins participated underlying MDSet  allowing observe essential <br />  MDSet proteins (e-MDSet) preferably enriched highly connected essential proteins organisms. inset (C) calculated <br />  enrichment essential proteins function degree binary protein interaction networks E. coli, S. cerevisiae H. sapiens. <br />  Generally, essential proteins binary interactions failed produce significant trend organism. Focusing enrichment MDSet <br />  proteins,  recovered strong increasing trend (D) reinforced focusing e-MDSet proteins. <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                                                Page 3 8 <br />  <br />  <br />  <br />  <br /> essential genes slightly increased mean degree (E. coli                                  H. sapiens (Figure 1C). Notably, trends rein- <br /> &lt; kess. &gt; = 3.4, S. cerevisiae &lt; kess. &gt; = 4.8, H. sapiens &lt; kess. &gt; =                         forced focused e-MDSet proteins, confirming <br /> 6.7) compared corresponding values proteins                                   hypothesis (Figure 1D). underlying networks. particular, 91 essential                                     different measure central placement essen- <br /> proteins participated underlying MDSet proteins                                    tial proteins, calculated betweeness centrality E. coli (e-MDSet), number statistically significant                               proteins underlying networks. defined set applying Fisher&#226;&#8364;&#8482;s exact test (P = 9.2 &#195;&#8212; 10&#226;&#710;&#8217;4). Notably,                                bottleneck proteins 20% proteins highest <br /> enirchment pattern applied remaining organisms                                       centrality [23]. null-model, randomly picked essen-  179 e-MDSet proteins S. cerevisiae                                     tial proteins 10,000 times determined enrichment (P = 2.3 &#195;&#8212; 10&#226;&#710;&#8217;7) 209 H. sapiens (P = 5.3 &#195;&#8212; 10&#226;&#710;&#8217;3).                                       essential proteins corresponding sets bottleneck <br />                                                                                                proteins. Figure 2A indicates essential proteins Centrality lethality                                                                       weakly enriched bottleneck proteins. MDSet direct comparison, grouped proteins                                      proteins strongly enriched bottleneck proteins <br /> bins logarithmically increasing degree calculated                                   organisms, observed reinforcement fraction essential genes group. Determining                                     trends considered e-MDSet proteins, suggesting <br /> fraction e-MDSet proteins bin, Figure 1B clearly                                    topological placement MDSet proteins enhanced <br /> shows e-MDSet proteins predominantly enriched                                        enrichment signals essential proteins (Figure 2A). groups essential proteins increased num-                                        measure protein&#226;&#8364;&#8482;s impact interaction net- <br /> ber interaction partners. Considering enrichment                                     work&#226;&#8364;&#8482;s resilience, performed robustness analysis. essential genes function degree                                       sorted e-MDSet proteins according degree lying interaction networks, grouped proteins                                        interaction networks organisms. Starting certain number interactions counted                                         connected protein gradually deleted proteins <br /> number essential proteins bin. provide                                         calculated number connected components control, randomly sampled sets essential genes                                      deletion step. comparison, considered sets null-model defined ratio observed ex-                                       equal size connected, essential proteins. Figure 2B <br /> pected number enrichment essential genes                                          indicates successive deletion e-MDSet proteins group. inset Figure 1C indicates binary                                       higher impact network topology producing <br /> interaction networks organisms failed produce                                      connected components removing fewer inter- <br /> viable trend. Hypothesizing proteins                                          actions connected essential proteins E. <br /> enriched highly connected proteins repeated                                           coli, S. cerevisiae H. sapiens. initial enrichment analysis considering MDSet e-MDSet proteins.  increasing correl-                                       Protein complexes <br /> ation elevated degree presence                                            Moving higher level cellular organization, calcu- <br /> MDSets interaction networks E. coli, S. cerevisiae                                   lated complex participation coefficients proteins,  <br />  <br />                                                       essential <br />                                                                                        B <br />                                           3                              ** P &lt; 10-4                   700 <br />                                                         MDSet                                                    essential <br />                                                                           * P &lt; 10-2                   600 <br />                                          2.5            E-MDSet                                                  e-MDSet <br />                                                                              ** <br />                      lg2 [fobs./fexp.] <br />  <br />  <br />  <br />  <br />                                                                                        N conn. comp. <br />  <br />  <br />  <br />  <br />                                                                     **                                 500                                       yeast     500 <br />                                           2        ** <br />                                                                                                        400                                                 400 <br />                                          1.5                                                                                                               300 <br />                                                                                                        300 <br />                                                                                                                                             E. coli        200 <br />                                           1 <br />                                                                                                        200                                                 100 <br />                                                * <br />                                          0.5                                                           100                                                  0 <br />                                                                                                                          0     200   400   600   800     1000 <br />                                           0                                                             0 <br />                                                E. coli      S. cervisiae H. sapiens                          0   1000   2000     3000       4000         5000 <br />                                                                                                                         N deleted PPI <br />  <br />  Figure 2 Centrality e-MDSet proteins.   determined betweeness centrality protein interaction network E. <br />  coli, S. cerevisiae H. sapiens chose 20% proteins highest betweeness centrality sets bottleneck proteins. <br />  Randomly sampling sets 10,000 times, observed bottleneck proteins weakly enriched essential proteins  organisms. turn, MDSet e-MDSet proteins significantly enriched bottleneck proteins. (B) Utilizing subset e-MDSet <br />  proteins E. coli, S. cerevisiae  inset) H. sapiens, sorted proteins according degree. Starting highest <br />  connected protein, successively deleted proteins calculated number connected components. compare, applied  procedure set highest connected essential proteins size organism. results suggest removal  e-MDSet proteins led lower number deleted interactions higher number connected components. <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                                              Page 4 8 <br />  <br />  <br />  <br />  <br /> value indicates protein&#226;&#8364;&#8482;s tendency interact                         Functional classes <br /> different complexes interactions. com-                           striking network patterns consistent proteins <br /> plex participation coefficient tends 1 given                         high level regulatory functions.  took <br /> protein predominantly interacts proteins                          logical step grouping protein sets according <br /> complex vice versa. particular, utilized set                        broad functional classes defined clusters 517 protein complexes E. coli [24], 430 protein com-                            orthologous groups (COGs) [27,28]. Specifically, counted <br /> plexes S. cerevisiae [25] 1,843 protein complexes                        occurrence essential, MDSet, e-MDSet pro- <br /> H. sapiens [26]. essential proteins tend connect                          teins groups (Figure 5) randomly assigned complexes non-essential proteins, hypothe-                            number functional classes protein <br /> sized topological placement MDSet proteins                        10,000 times null model. organism specific level, <br /> enhance statistical characteristic essential proteins.                     essential, MDSet, e-MDSets proteins roughly appeared comparison frequency distributions                               enriched/depleted functional groups  <br /> corresponding participation coefficients Figure 3A                              ganisms, apart certain striking cases enrichment <br /> clearly confirmed assumption.  e-MDSet                                  patterns different organisms largely incongruent. <br /> proteins interaction networks largely reached                               Notably, enrichment signals essential proteins higher number different protein complexes                                involved RNA processing (group  cell cycle control <br /> compared essential proteins  basis                             (D), transcription (K), post-translational modification (O), finding, wondered interactions                               intracellular transport (U) cytoskeleton    <br /> tween essential proteins enriched single                                hanced considering e-MDSet proteins S. cerevisiae. <br /> complexes complexes (Figure 4A). Specif-                                Conversely, e-MDSet proteins exceedingly depleted ically, counted number inter- intra-                                 aminoacid  , <span id='am-7' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span>carbohydrate</span> (G) <span id='am-1' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span>lipid</span>   transport complex interactions randomly assigned                                metabolism chaperone turnover activities (P) <br /> number proteins corresponding complex                                   compared essential proteins  enrich- <br /> 10,000 times random null model. Generally,                                 ment patterns largely differ E. coli H. sapiens, observed interactions essential proteins                              <span id='am-35' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-36' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> transporations metabolism (E) <br /> connecting complexes appeared frequently                                 chaperones turnover functions (P) depleted expected organisms (P &lt; 10&#226;&#710;&#8217;4, Figure 4B).                                organisms. comparison essential proteins turn, interactions proteins                                MDSet proteins, e-MDSet proteins E. coli enhanced complex occurred frequently expected                                  enrichment signal proteins play role translation organisms (P &lt; 10&#226;&#710;&#8217;4, Figure 4B). Focusing inter-                            (J). turn, cell cycle control proteins enriched actions MDSet proteins, similar,                                essential proteins (D), signal mitigated e- <br /> albeit slightly weaker, signal. turn, trends                         MDSet proteins. H. sapiens, e-MDSet pro- largely enhanced considered e-MDSet                                   teins enhance enrichment signals essential genes proteins organisms (P &lt; 10&#226;&#710;&#8217;4, Figure 4B).                                   transcription (K), replication (L) signal transduction <br />  <br />  <br />                                                        1                                  1                                      1 <br />                                                        0.9                                0.9                                    0.9 <br />                                       E. coli          0.8         S. cerevisiae          0.8                H. sapiens          0.8 <br />                                                                                                                                        complex participation <br />  <br />  <br />  <br />  <br />                                                        0.7                                0.7                                    0.7 <br />                                                        0.6                                0.6                                    0.6 <br />                                                        0.5                                0.5                                    0.5 <br />                                                                                                     essential <br />                                                        0.4                                0.4       e-MDSet                      0.4 <br />                                                        0.3                                0.3                                    0.3 <br />                                                        0.2                                0.2                                    0.2 <br />                                                        0.1                                0.1                                    0.1 <br />                                                        0                                  0                                      0 <br />  <br />                       0.6      0.4      0.2        0         0.6    0.4      0.2      0          0.6      0.4      0.2       0 <br />                                                                    frequency <br />  Figure 3 Essential MDSet proteins protein complexes. calculated complex participation coefficients proteins binary <br />  interaction networks E. coli, S. cerevisiae H. sapiens. Specifically, e-MDSet proteins reached complexes compared essential proteins  E. coli (P = 1.9 &#195;&#8212; 10&#226;&#710;&#8217;4, Wilcoxon test), S. cerevisiae (P = 3.5 &#195;&#8212; 10&#226;&#710;&#8217;6,) H. sapiens (P = 9.6 &#195;&#8212; 10&#226;&#710;&#8217;11). <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                           Page 5 8 <br />  <br />  <br />  <br />  <br />                                                     B <br />                                                                          2 <br />  <br />  <br />  <br />  <br />                                                                                                                             E. coli <br />                                                                          1 <br />                               complex <br />                                                                        0 <br />                                                                          -1 <br />  <br />  <br />  <br />  <br />                                                       lg2 [Nobs/Nexp] <br />                                                                          2 <br />  <br />  <br />  <br />  <br />                                                                                                                             S. cerevisiae <br />                                                                          1 <br />                                                                          0 <br />                                         B <br />                                                                          -1 <br />  <br />                               C                                          1 <br />  <br />  <br />  <br />  <br />                                                                                                                             H. sapiens <br />                                                                         0.5 <br />                                                                                                  0 <br />                                                                   -0.5 <br />                           complexes                                           MDSet        essential         e-MDSet <br />  Figure 4 Enrichment interactions protein complexes.   Schematic illustration interactions appear <br />  complexes. (B) determined number interactions MDSet, essential e-MDSet proteins  complex complexes E. coli, S. cerevisiae H. sapiens. random null model, resampled proteins complexes <br />  10,000 times. Generally, interactions complexes appear diluted interactions complexes  enriched. appeared significant sets interactions essential proteins observed signals enhanced <br />  interaction sets e-MDSet proteins (P &lt; 10&#226;&#710;&#8217;4). <br />  <br />  <br />  <br /> functions, transportation (E,H,P,U) translation (J)                      ability MDSet &#226;&#8364;&#732;cover&#226;&#8364;&#8482; proteins network <br /> functions depleted.                                                        just question finding connected pro- <br />                                                                                 teins, necessitates determination lowest num- <br /> Discussion conclusions                                                      ber strategically placed proteins [17]. concept  determined minimum domainting sets (MDSets)                            highlights idea MDSets capture proteins proteins underlying binary interaction networks                       involved adaptability lethality. E. coli, S. cerevisiae H. sapiens                      essential MDset proteins appear enriched bottleneck <br /> tirely determined yeast hybrid approaches.                         protein sets, respectively, topological <br /> fined MDSets smallest group strategically placed                      placement e-MDSet enhanced initial trends.  <br /> proteins remaining protein  e. non-                            e-MDSet proteins predominantly connect MDSet protein) immediately reached                             protein complexes essential MDSet proteins  <br /> single interaction.  non-MDSet protein                        confirming hypothesis. functional aspects, teracts MDSet protein. words,                      analysis revealed e-MDSet proteins broadly fell MDSet smallest possible set proteins allows                      functional classes vital survival reproduction &#226;&#8364;&#732;cover&#226;&#8364;&#8482; remaining proteins underlying net-                     cell cycle control, trafficking, cytoskeletal, transla- <br /> works. hypothesized topological placement                        tional posttranslational modifiying functions. MDSet proteins correlate presence                          functional groupings highly dynamic responses <br /> essential proteins underlying protein interactions                       external stimuli physiological conditions cells. <br /> networks.  MDSet proteins                              MDSet provides optimized set topologic- <br /> enriched essential proteins interaction net-                        ally central proteins contributing essenti- <br /> works. Despite absence correlation                          ality genes, including necessary continued <br /> number interaction protein&#226;&#8364;&#8482;s tendency es-                        survival changing environment. <br /> sential, anticipated ability cover pro-                       Notably, results organism-independent teins allow highly connected MDSet                          strongly suggested MDSet proteins provide statis- <br /> proteins increasingly essential survival                     tical enhancement topological functional charac- <br /> organism adaptability [17].  recovered                      teristics essential genes. think MDSet proteins <br /> strongly ascending correlation protein&#226;&#8364;&#8482;s lethality                    &#226;&#8364;&#339;vital essential&#226;&#8364;? set, scoring significantly higher es- number interactions organisms                         sential proteins  enhancing topological parameters <br /> considered e-MDSet proteins.                                                    underlying networks. Furthermore, network <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                               Page 6 8 <br />  <br />  <br />  <br />  <br />                     4 <br />  <br />                     2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                        E. coli <br />                     0 <br />  <br />                     -2 <br />  <br />                     -4 <br />                             B    C    D    E    F    G     H      J    K    L    M    N    O     P    Q RS T           U    V    W     Y                         4 <br />   lg2 [Nobs/Nexp] <br />  <br />  <br />  <br />  <br />                                                                                                                                                        S. cerevisiae <br />                     2 <br />  <br />                     0 <br />                                                                                                                                     MDSet <br />                     -2                                                                                                              essential <br />                                                                                                                                     e-MDSet <br />                     -4 <br />                             B    C    D    E    F    G     H      J    K    L    M    N    O     P    Q RS T           U    V    W     Y                         4 <br />  <br />                     2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                        H. sapiens <br />                     0 <br />  <br />                     -2 <br />  <br />                     -4 <br />                             B    C    D    E    F    G     H      J    K    L    M    N    O     P    Q RS T           U    V     W     Y                       RNA processing modification         J Translation, ribosomal structure biogenesis   RS General Function/unknown <br />               B      Chromatin structure dynamics        K Transcription                                     T  Signal transduction mechanisms <br />               C      Energy production conversion        L Replication, recombination repair             U  Intracellular trafficking, secretion <span id='am-20' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-21' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; <br />               D      Cell cycle control, cell division,      M Cell wall/membrane/envelope biogenesis               vesicular transport <br />                      chromosome partitioning                 N Cell motility                                     V  Defense mechanisms <br />               E      Amino acid transport metabolism     O Posttranslational modification, protein           W Extracellular structures <br />               F      <span id='am-27' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-31' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>Nucleotide</span> transport metabolism     P turnover, chaperones                              Y  Nuclear structure <br />               G      Carbohydrate transport metabolism     Inorganic <span id='am-14' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-15' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> transport metabolism             Cytoskeleton <br />               H      Coenzyme transport metabolism       Q Sec. metabolites biosynthesis, transport <br />                    Lipid transport metabolism            catabolism <br />  Figure 5 Enrichment functional classes. determined number proteins MDSet, essential e-MDSet sets E. coli, S. cerevisiae <br />  H. sapiens belong underlying functional classes. random null model resampled proteins classes 10,000 times. <br />  <br />  <br /> resilience aspect, directly compared e-MDSets sets                          context crucial elucidating possible roles essential protein hubs equal size. Notably,                     certain genes pathway caus- deletion e-MDSet proteins higher disruptive ef-                        ing slight perturbations. <br /> fect underlying networks organisms deletion essential hub proteins  Taken results demonstrate topological central relevance                        Methods <br /> proteins involved MDSets essen-                        Determination minimum dominating set (MDSet) <br /> tial protein interaction networks, characteristc                   defined set S &#226;&#352;&#8224; V nodes network G = (V, E) <br /> just matter highest connectivity.                                              minimum dominating set node v &#226;&#710;&#710; V ei- <br />    research protein interaction networks seeks                            ther element S adjacent element S better understand disease mechanism evolution-                              (inset, Figure 1A). binary integer-programming <br /> ary development question. data collecting tech-                              problem assigned binary variable xv =1 niques sophisticated, ask                                   protein v &#226;&#710;&#710; V participates interactions E pro- <br /> intricate questions data collected.                             tein interaction network G = (V, E) element clude identification MDSet proteins                             MDSet, xv =0  smallest set MDSet <br />  Khuri Wuchty BMC Bioinformatics (2015) 16:109                                                                                  Page 7 8 <br />  <br />  <br />  <br />  <br />                               X <br /> nodes obtained min v&#226;&#710;&#710;V xv , subject            enrichment essential proteins Ni,&#226;&#8240;&#165; k              X <br /> straint xv &#195;&#190; w&#226;&#710;&#710;&#206;&#8220; &#195;&#176;v&#195;&#382; xw &#226;&#8240;&#165;1 &#206;&#8220;(v) set            corresponding random number essential pro- <br />                                                                 teins N&#226;&#8240;&#165; k proteins corresponding bin. <br /> interaction partners protein v. domination <br />                                                                 averaging Ei 10,000 randomizations Ei &gt;1 <br /> problem graphs NP-complete algorithm ne- <br />                                                                 pointed enrichment vice versa, Ei ~ 1 <br /> cessarily exists allows determination                                                                 indicated random process [40]. <br /> minimum dominating set arbitrary graphs poly- <br /> nomial time [29]. utilized branch bound al- <br /> gorithm [30] implemented library lpSolve           Bottleneck proteins <br /> R programming language solve binary integer-             global measure nodes centrality, calculated <br /> programming problem.                                            betweenness centrality, indicating proteins appear- <br />                                                                 ance shortest paths network.                                                                 particular, defined betweeness centrality cB pro- <br /> Protein-protein interactions E. coli, S. cerevisiae H.                      P <br /> sapeins                                                         tein v cB &#195;&#176;v&#195;&#382; &#194;&#188; s&#226;&#8240;&#160;t&#226;&#8240;&#160;v&#226;&#710;&#710;V &#207;&#402; st&#207;&#402; &#195;&#176;stv&#195;&#382; , &#207;&#402;st num- collected 1,938 interactions 1,203 proteins          ber shortest paths proteins s t &#207;&#402;st experimentally determined using yeast           (v) number shortest paths running hybrid approach E. coli [19]. S. cerevisiae        node v. defined set bottleneck nodes collected 6,225 high-quality interactions 2,640         20% interactions highest node betweeness cen- <br /> proteins entirely determined large-scale           trality [23]. <br /> yeast hybrid approaches HINT database <br /> [20], including [2,31,32]. Furthermore, assembled          Functional classes <br /> network 17,523 high-quality interactions H. sapiens       Proteins grouped according broad functional 5,926 proteins HINT database [20].             classes defined clusters orthologous <br /> Specifically, set protein interactions      groups (COGs) [27,28]. COGs provide consistent clas- <br /> tirely determined large-scale yeast hybrid ap-           sification bacterial eukaryotic species based proaches, including [33-36].                                    orthologous groups. <br />  <br />                                                                 Competing interests <br /> Protein complexes E. coli, S. cerevisiae H. sapiens      authors declare competing interests. utilized set 517 protein complexes E. coli affinity purification study followed mass      Authors&#226;&#8364;&#8482; contributions <br />                                                                 SW designed concept, SK SW analyzed data wrote spectrometry analyses [24]. S. cerevisiae, utilized   manuscript. authors read approved final manuscript. <br /> 430 protein complexes compiled [25], including SGD Macromolecular Complex standard [37],                 <br /> CYC2008 protein complex catalogue [38] set              <br /> manually curated protein complexes. Furthermore, uti-         <br /> lized 1,843 protein complexes H. sapiens CORUM database [26]. <br />                                                                 <br />                                                                <br /> Protein complex participation coefficient                          protein protein com-       <br />                                                                     <br /> plex, defined protein complex participation coef- <br />   <br /> ficient protein              topology protein interaction networks. Proc Biol Sci/Roy Soc. <br />                                                            <br />                                             <br />                                                                 number links protein proteins complex             <br /> s N total complexes. protein predominantly           <br /> interacts partners complex, P tends              <br />  <br /> 1 vice versa [39].                              <br /> Enrichment analysis function degree <br />                                                                    binned proteins groups N&#226;&#8240;&#165; k protein               k interactions calculated corre-         <br /> sponding number essential proteins  Ni,&#226;&#8240;&#165; k. Ran-            <br />                                                                  <br />                                                <br /> domly picking essential genes defined E &#226;&#8240;&#165;k &#194;&#188; N &#226;&#8240;&#165;k <br />                                                      <br /> </body></html>