<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Proc. Natd. Acad. Sci. USA <br /> Vol. 88, pp. 11373-11377, December 1991 <br /> Biochemistry <br />  <br />  <br /> RAM2, essential gene yeast, RAMI encode polypeptide components farnesyltransferase prenylates factor Ras proteins <br />      (CAAX motifs/membrane targeting/post-                tioal     <span id='am-11' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span>/extr                c suppressors) <br /> BIN , PENG CHENt, SEI-YU CHEN*, KATHLEEN L. VANCURAt, SUSAN MICHAELISt, SCOTT POWERS*t <br /> *Department Biochemistry, Robert Wood Johnson Medical School, University Medicine Dentistry New Jersey, Piscataway, NJ 08854; tDepartment Cell Biology Anatomy, Johns Hopkins University School Medicine, Baltimore, MD 21205 <br /> Communicated Leland Hartwell, September 23, 1991 <br />  <br /> ABSTRACT           yeast Saccharomyccs cerevwise, qiuta-                      CAAX sequence noted (22, 23). Geranylgeranyltrans- <br /> tions unlinked genes, RAM)-1or RAM2, abolish                        ferase type (GGTase , modifies proteins farnesyltrnsferae activity responsible prenylation                        terminate second type CAAX sequence (23-25), <br /> Ras proteins factor mating pheromone.                              heterodimeric enzyme appears share report thefunction RAM) antd RAM2 genme required                          common subunit p21&quot; protein farnesyltransferase membrane laliation ofRas proteins factor.                          (21, 23). enzyme identified, GGTase II, <br /> RAM2 gene sequenced encode 38-kDa protein.                             geranylgeranylates Yptl/Sec4 class proteins examined tie functional interaction RAM2 andRAMf                            (21). <br /> expressing genes Escherchia colt. Extracts derived                          Genetic studies yeast identified genes, E. coil strain coexpressed RAM] RAM2 efficiently                         uniquely affects type prenyltransferase <br /> farnesylated factor <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> Ras protein substrates.                         reaction. genes, RAM)  called DPRI), contrast, extracts derived E. colN strains expressed                       identified suppressor RAs2ra19, mutationally acti- RAM gene devoid activity;                          vated RAS allele (26, 27). gene identified extracts mixed, protein farnesyltransferaSe                          based involvement factor production (26, 28, 29) <br /> activity reconstituted. results indicate yeast                    suppressor G protein function (30). ram) extracts <br /> farnesyl-protein transferase comprised Raml Ram2                           defective factor Ras protein farnesyltransferase <br /> polypeptides. Raml component enzyme,                            activity prenyltransferase activities (21, 23, <br /> disruption RAM) gene yeast lethal indicating                       31, 32). Furthermore, Raml protein produced Escherichia Raml-Ram2 farnesyltserase essential                              coli, lacking enzyme activity  biochem- <br /> viability. contrast, disruption RAM2 lethal, suggest-                      ically complements ram] extracts factor peptide farne- <br /> ing Ram2 essential function addition role                      syltransferase activity (31). result suggests Raml protein farnesylation.                                                  RAM] gene product necessary insufficient compo- <br />                                                                                      nent protein farnesyltransferase. Mutants harboring post-translational modification proteins cova-                         lesions CDC43/CALI, gene shows homology lent attachment isoprenoid groups plays important role                         RAM) (33), specifically defective GGTase activity membrane targeting various proteins (1-5).                           (21, 23, 34). yeast gene homologous RAM), <br /> classes prenylated proteins eukaryotic cells                         called BET2, affects membrane localization Yptl described. class, represented certain fungal                            Sec4 proteins (35) specifically affects yeast GGTase II <br /> mating pheromones, Ras proteins, nuclear lamins,                              activity (21). <br /> initially synthesized C-terminal CAAX sequence,                                  report characterize yeast gene, RAM2, X = Ser, Cys, Met, Ala, modified thioether                        identified extragenic suppressor RAS2val&apos;9. <br /> linkage C15 farnesyl group cysteine residue (6-9).                       Previous reports shown ram2 mutants, like ram] second class prenylated proteins, represented                          mutants, defective Ras protein farnesyltransferase <br /> y subunits certain heterotrimeric G proteins various                          activity (23, 32). report DNA sequence RAM2 <br /> Ras-like proteins, initially synthesized different C-ter-                    present direct evidence RAM] RAM2 encode <br /> minal CAAX sequences, X = Leu Phe,                                essential subunits yeast farnesyl-protein transfer- <br /> modified attachment C20 geranylgeranyl group                          ase. cysteine residue (10-12). Members classes prenylated proteins subsequently processed                                      MATERIALS METHODS <br /> proteolytic removal terminal amino acids                               Yeast Strains Materials. ram2-1 mutants used methylation newly exposed carboxyl group                               study derived original mutant STS39 prenyl-cysteine (6, 10-15). class prenylated                            backcrosses RS16-4C (26). yeast strains proteins, represented Yptl/Sec4 (Rab) family                               described (26, 29, 36). Recombinant p21&quot;5 purified Ras-like GTPases terminate sequence CC                              described (37). Biotinylated KTSCVIM peptide (38) CXC, geranylgeranylated C-terminal cysteine                             obtained Multiple Peptide Systems (San Diego). residue(s) (16-18).                                                                  factor peptides YIIKGVFWDPAC YIIKGVFWD- <br />   mammalian p21&quot;5 protein farnesyltransferase                               PACVIA synthesized P. Shenbagamurthi (Johns <br /> heterodimeric enzyme (19-21) appears responsible modification proteins type                                                                                      Abbreviations: CAAX, protein sequence C = cysteine, = <br />                                                                                      usually aliphatic <span id='am-32' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-36' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span>, X = amino acid; FPP, publication costs article defrayed page charge           farnesyl <span id='am-6' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span>pyrophosphate</span>; GGTase  geranylgeranyltransferase type- <br /> payment. article marked &quot;advertisement&quot;                 GGTase II, GGTase type II. accordance 18 U.S.C. &#194;&#167;1734 solely indicate fact.                     ITo reprint requests addressed. <br />                                                                              11373 <br />  11374    Biochemistry: et al.                                                   Proc. Natl. Acad. Sci. USA 88 (1991) <br /> Hopkins). [3H]Farnesyl pyrophosphate ([3H]FPP; 20 Ci/             ysis 10 complete tetrads, phenotypes resulted <br /> mmol; 1 Ci = 37 GBq) purchased NEN.                      single mutation (ram2-1). ram2-1 previously <br />    Plasmid Constructions Gene Disruptions. integra-       demonstrated unlinked ram) (32). <br /> tive mapping plasmid pBH43 contains the1l.85-kilobase-pair           ru2 Mutants Defettive Farnesylatlon Mea. <br /> (kbp) Sac Pst RAM2 fragment inserted pRS305 (39).        brane Targeting -Ras Proteins Factor Ma <br /> pram2::LEU2 constructed insertion ofthe 1.5-kbp Pst        Pheromone. shown extracts prepared Bgl II RAM2 fragment pBH43 gene replacements           ram2-1 mutants fail farnesylate bacterially produced Ras generated digestion Pst transformation         proteins vitro (23, 32). determined ram2-) extracts, diploid strain TTSD1 (40). pram):: URA3 (pSM353)         like ram] extracts, defective farnesylating factor <br /> contains 4.6-kbp BamHI-Bgl II fragment RAM)            vitro (data shown). determine lack locus inserted vector pUC-4k, URA3              farnesylation measured vitro reflected vivo HindIII fragment place RAM) HindIII fragment,           defect membrane targeting, radiolabeled wild-type codons 1-322 deleted (41). Gene replacements        mutant yeast cells examined soluble particulate generated digestion EcoRI transformation         fractions immunoprecipitation using antibodies rec- diploid strain SM1060 (36). express RAM2 E.        ognized Ras proteins factor (Fig. 1). wild-type <br /> coli, RAM2 coding sequence cassette generated            strain, Ras2 localized particulate fraction PCR utilizing oligonucleotide primers overlapped         apparent 40-kDa species. contrast, ram2-1 initiation codon termination codon shown     ram) mutants, Ras2 localized soluble fraction Fig. 3. relevant sequence 5&apos; portion cassette   migrated slowly (Fig. lA). Fig. 1B shows factor CTCGAGGAAAAM, underlined ATG corre-                  detected particulate fraction wild-type cells sponds initiation codon RAM2. 3&apos; portion     precursor species (P1 P2), factor synthe- <br /> cassette contained artificial BamHI site following         sized ram) ram2-1 mutants predominantly localized <br /> termination codon. cassette inserted chlor-     soluble fraction slower migrating form (PO).  <br /> amphenicol-resistance plasmid pBC-KS+ (Stratagene)           ram2-1, like mutations RAM), blocks membrane cleaved Xho BamHI, generate pBH57,         targeting Ras proteins factor. utilizes lac promoter direct synthesis        Cloing Sequence Analysis RAM2. utilized N-terminal lacZ-RAM2 fusion protein. express RAM)           Ts- phenotype ram2-1 mutants clone wild-type <br /> E. coli, plasmid pBH63 constructed inserting Sal     RAM2 gene complementation. ram2-1 mutant strain Kpn RAM] coding sequence cassette ampicillin-        transformed yeast genonuc library resistance plasmid pUC119. RAM) cassette gener-           transformants exhibited plasmid-dependent <br /> ated PCR utilizing oligonucleotide primers       Ts&apos; growth. Plasmids recovered transform- <br /> lapped initiation codon termination codon       ants restored normal growth mating properties ram2 <br /> reported DPRI (41). relevant sequence 5&apos;           mutant strains. Restriction enzyme mapping revealed portion cassette GTCGACTAIG,            plasmids contained overlapping inserts common <br /> lined ATG indicates initiation codon RAM].                 region DNA &gt;11 kbp (Fig. 2). determine    Preparation E. cofl Extracts Farnesyltrnsferase      gene cloned corresponded locus identified says. E. coli extracts prepared culturing JM110           ram2-1 mutation, integrated LEU2 &apos;-marked plasmid <br /> double transformants mid-logarithmic phase followed         (pBH43) containing fragment cloned DNA induction 0.5 mM isopropyl fB-D-thiogalactoside          RAM2 + strain. Integration targeted chromosomal <br /> incubation 1 hr. Cells collected, washed extrac-      locus corresponding cloned fragment digestion tion buffer (50 mM sodium Mes, pH 6.5/0.1 mM EGTA/0.1             <span id='am-39' about='obo:IMR_0100178' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Sphingoid" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="sphingenine" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="sph" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="Sphingoid" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Sphingenine" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="sphingosine" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="sph" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="Sphingenine" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00319" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="sphingenine" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0100178" datatype="xsd:string"></span>Sph</span>  Fig. 2), correctly targeted integration mM MgC12/5% glycerol/2 mM phenylmethylsulfonyl fluo- <br /> ride), transferred 0.5-ml tubes, pelleted, lysed 200                                                2 <br /> ,l extraction buffer glass beads Vortex mixing.                                      --       S       P       S    P <br /> Soluble fractions prepared centrifugation 50,000 x                                                                      P~~~~~~~~~~~~~~~~~~~ <br />                                                                                    41    _ <br /> g 1 hr.                                                                        40&gt; <br />    Methods. Fractionation yeast detection factor Ras2 proteins immunoprecipitation performed described (26, 36). Yeast. extracts pre- <br /> pared described (32). Farnesyltransferase assays using <br /> recombinant p21lS substrate performed analyzed                   B                   &apos; <br />                                                                                                  1              2                3 described (32). Assays using synthetic factor peptides                                  7   D      S       P       3        P   S carried 250 pmol peptide essentially                       p <br /> described (31).                                                                pp2.,Et   46  <br />                                                                                                             ,       -w * <br />  <br />               EXPERIMENTAL RESULTS <br />   Identification Characterization ram2 Mutation.         FIG. 1. Localization Ras proteins factor ram mutants. yeast mutations suppress RAS2vall9, including ram),         SDS/PAGE analysis immunoprecipitated Ras2 proteins cyr), capl/srv2 (26, 27, 42, 43). set            soluble (S) particulate (P) fractions extracts prepared RAS2v8119 extragenic suppressor mutants derived heat-          [35S]methionine-labeled wild-type mutant yeast strains, shock selection MATa rasf RAS2va&quot;l9 strain PT1-6         contained high-copy RAS2 plasmid (26). Lanes: 1, wild-type <br /> (26), recessive mutations fall       (SP1); 2, ram2-1 (STS39); 3, ram)-) (STS11). soluble precursor <br /> known complementation groups. suppressors,           form Ras2 (41 kDa) particulate mature form Ras2 (40 <br /> STS39, displayed nonselected phenotypes: inability      kDa) marked arrows. (B) SDS/PAGE analysis immuno- <br /> grow high temperatures (370C) significant mating    precipitated factor total extracts (T) particulate (P)                                                                   soluble (S) fractions total extracts prepared [35Slcysteine- <br /> defect. mating defect determined MATa-              labeled wild-type mutant yeast strains, contained <br /> specific correlated defective factor production.       high-copy MFal plasmid (29). Lanes: 1, wild-type (SM1229); 2, <br /> Based cosegregation phenotypes           ram)::URA3 (SM1866); 3, ram2-) (SM1863). intracel- <br /> following cross wild-type strain subsequent anal-      lular forms factor marked arrows (PO, P1, P2). <br />             Biochemistry: et al.                                                                Proc. Natl. Acad. Sci. USA 88 (1991)                               11375 <br />                                                                               -444    TCTAGAGTATAATTATTTTGAATGAGTTTCGTTGACATTTTCAGTTCATGCGATCATCTT <br />                                                                               -384    ACTTTCATGGTTGCTCCTTGCCGTTAAAATTGAAACGAAAAAGCGGGAATACAACAATG <br />                                                                               -324    TCACCTTACACGAGCAGAGAAAAGGGCAGCAATAGCGACAATAACCGATAATGCCAGGCC <br />                                                                               -264    TTCCAGAGCCCTATGCTGACCTATATTCATCATGACGCTATCCATATGCAACTAGAGACC <br />                                                                               -204    TGCTGGGAAGCGCTCTCTCATTTTTGAAGGTTGCAGCAGCTTGCGCGCCGGTTTTCGATT <br />                                                                               -144    GGGCCTCGTGATAAAACACATCAGCAAAGTTTCGCTTTTGACATTCTGTTAAATTGGTCC <br />                                                                                -84    TTAACTGTTGGTGCAGACCTACTGC.AAGACTGTTAATTATGGTATATTATATATCAAGTG <br />                                                                                -24    CTGATACAACGTACACCATCGAAAATGGAGGAGTACGATTATTCAGACGTTAAACCTTTG <br />                                                                                                                    1       M   E   E    Y   D   Y   S   D    V   K   P   L <br />                                                                                 36    CCCATTGAGACAGACTTGCAGGATGAACTGTGCAGGATTATGTATACCGAGGATTATAAG <br />                                                                                 12    P     E   T   D   L    Q       D   E   L   C    R     M   Y   T    E   D   Y   K <br />                                                                                 96    CGGTTGATGGGACTCGCAAGGGCTCTGATCAGCCTTAACGAACTGTCACCCAGGGCACTA <br />                                        clo   CL                                 32    R   L   M   G   L      R         L     S    L   N   E   L   S    P   R     L <br />            enuJL           xen x       = = <br />    E                            ~ ~ gU (1          G <br />                                                                                                                                               Eco     156    CAGCTAACAGCCGAAATTATCGACGTGGCGCCAGCCTTCTACACCATATGGAACTACCGA <br />                                                               0. <br />                                                                                 52    Q   L   T     E        D       V     P      F   Y   T      W   N   Y   R <br />                                                                 L-                 -   .     L                          . <br />                                                                         s5     216    TTCAATATCGTCAGGCACATGATGAGTGAATCCGAAGACACTGTCTTGTACCTGAACAAG <br />    0           1           2       3              4       5        6            72    F   N     V   R   H    M   M       S   E   S    E   D   T   V   L    Y   L   N   K <br />                                    RAM2                                        276 <br />                                                                                 92 <br />                                                                                       GAATTAGACTGGCTAGATGAAGTTACGCTGAATAATCCAAAGAACTATCAGATCTGGTCC <br />                                                                                       E L D       W   L   D    E   V       T   L   N    N   P   K   N   Y    Q     W   S <br />    FIG. 2. Restriction map RAM2 locus ram2::LEU2                336 <br />                                                                                112 <br />                                                                                       TATAGACAGTCTCTTTTGAAGCTACATCCGTCTCCTTCCTTCAAAAGAGAGCTGCCTATC <br /> gene disruption construct. restriction map RAM2 locus                   Y   R   Q S     L   L    K       L   H   P   S    P   S   F   K   R    E   L   P   shown. original library plasmid isolates contained                 396    TTAAAACTGATGATTGATGATGATTCCAAGAATTATCACGTTTGGTCGTACAGAAAGTGG <br />                                                                                132    L       L   M     D    D   D       S            Y <br /> additional regions DNA (-6 kbp) shown map.                        K                                    K   N        H   V   W   S    Y   R   K   W <br /> Shown map -y-disruption vector pram2::LEU2,                   456    TGCTGTTTGTTCTTCAGTGACTTTCAACATGAGCTCGCCTACGCCAGCGACCTCATCGAG <br />                                                                                152    C   C   L   F   F   S    D   F       Q contains noncontiguous fragments RAM2 region                                                                H   E    L     Y     S    D   L     E <br /> inserted head tail orientation yeast integration vector       516 <br />                                                                                172    T   D     Y   N   N    S         W       H <br /> pRS305 (39). Integration plasmid RAM2 locus results                                                            T        R   M   F   Y   W    V   N     K deletion RAM2 coding region resides             576    GATGTCATTTCAAAAGTGGAATTGGCCGACGAGCTCCAGTTCATTATGGACAAGATTCAA <br />                                                                                192    D V       S   K   V    E   L                        Q <br /> indicated Bgl II Sac restriction sites.                                                                                D   E    L       F     M    D   K     Q <br />                                                                               636     TTGGTTCCGCAGAACATCAGTCCGTGGACCTACCTCCGTGGTTTCCAAGAGCTATTCCAT <br />                                                                               212     L   V   P   Q   N      S <br /> confirmed DNA-blot hybridization transformant                                                           P       W   T   Y    L   R   G   F   Q    E   L   F   H <br /> BHYDC1. cross ram2-1 ieu2 strain BHYDC1,                        696 <br />                                                                               232 <br />                                                                                       GATAGGCTACAGTGGGATAGCAAAGTAGTCGACTTCGCCACAACCTTCATCGGTGACGTA <br />                                                                                       D R     L   Q   W   D    S   K       V   V   D    F     T   T   F      G   D   V segregation pattern LEU2 + ram2-1 indicated <br />                                                                               756     TTGTCACTTCCAATTGGCTCACCAGAGGATTTGCCCGAGATCGAGTCCTCATATGCCCTG <br /> complete linkage  recombinants 25 tetrads),                      252     L   S   L   P     G    S   P       E   D   L    P   E     E   S    S   Y     L conclude cloned gene RAM2.                       816     GAATTCCTGGCATATCACTGGGGGGCAGACCCTTGTACCCGAGACAACGCTGTTAAGGCC <br />    Subclones original plasmids constructed                    272     E   F   L     Y   H    W   G         D   P    C   T   R   D   N      V   K   based ability complement ram2 mutation                    876     TATAGCTTGCTAGCAATCAAATACGATCCTATTAGAAAAAACTTGTGGCACCACAAAATA determined entire RAM2 gene                         292     Y   S   L   L        K   Y       D   P      R   K   N   L   W    H   H   K   functional segment) resided 1.4-kbp Xba Sph                    936 <br />                                                                               312 <br />                                                                                       AATAATCTGAACTGGAATATATGTAAGCACTCTATTTGTACGAATATATTTACGTGTGTAC <br />                                                                                       N   N   L   N   * <br /> fragment shown Fig. 2. DNA sequence region <br />                                                                               996     GTGCATGC determined strands contains open <br /> reading frame encode protein 316 amino acids ATG used initiate translation (Fig. 3).                     FIG. 3. <span id='am-24' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-25' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>Nucleotide</span> sequence RAM2 region predicted <br /> predicted molecular mass Ram2 protein 38 kDa.                       amino acid sequence Ram2 protein. <br />    Reconstitution Farnesyltransferase Activity. investi-                  tested farnesyltransferase activity gate possibility Raml Ram2 proteins                   reconstituted mixing E. coli extracts containing Raml <br /> structural subunits yeast farnesyl-protein transferase,                extracts containing Ram2  results expressed RAM] RAM2 genes E. coli assayed extracts farnesyltransferase activity. Using                    shown Table 1 indicate extracts, mixed, substrate peptide corresponding C-terminal sequence                    exhibit farnesyltransferase activity. activity pro- K-rasB, control extracts extracts prepared                        duced mixed extract -25-fold lower bacterial strains expressed RAM] RAM2 gene                     observed E. coli extracts cells coexpressed completely inactive (Table 1). contrast, ex-                    Table 1. Farnesyltransferase activity extracts E. coli <br /> tracts prepared strain coexpressed RAM) RAM2 exhibited high levels famesyltransferase activity <br />                                                                                     E. coli                   RAM gene expressed                            [3H]Farnesyl <br /> (Table 1). determined specific activity                                                                                                  transferred, E. coli extracts 100-fold higher                transformant                    RAM]                     RAM2                 pmol tube <br /> similarly prepared wild-type yeast extracts (data shown).                 BHE-1                             -                        - <br />                                                                                                                                          &lt;0.006 <br />    Extracts prepared E. coli cells coexpressed                      BHE-2                     +                                - <br />                                                                                                                                          &lt;0.004 <br /> RAM) RAM2 efficiently farnesylated synthetic                         BHE-3                     -              +                 &lt;0.002 factor substrate. observed SDS/PAGE analysis                     BHE-4                     +              +              5.5 &#194;&#177; 0.1 <br /> control extracts derived E. coli carrying plasmids                  Mixed extract             +              +              0.19 &#194;&#177; 0.02 RAM inserts produced unidentified product                           E. coli host JM110 transformed sequentially [3H]FPP (Fig. 4A, lane 3) related factor               different plasmids generate strains used experiment. observed reactions peptide                    assay (25 jl) performed duplicate contained 50 mM <br /> absent (Fig. 4A, lane 9). contrast, using extracts cells              potassium phosphate (pH 7.4), 10 mM MgCl2, 5 mM dithiothreitol, coexpressed RAM] RAM2, reaction product                        0.2% Tween 20, 50 ,ug soluble extract indicated E. coil <br /> observed (Fig. 4A, lane 1) reactions                    transformant, 90 pmol biotinylated KTSCVIM peptide, 25 <br /> utilizing extracts prepared E. coli expressed                pmol [3H]FPP. incubation 30 min 300C, RAM) RAM2 gene (Fig. 4A, lanes 4 5).                    [3H]farnesyl transferred peptide measured described (38). <br /> product comigrated authentic mature factor produced                    mixed extract, 50 zg BHE-2 BHE-3 E. coli extracts <br />                                                                               mixed microcentrifuge tube left ice 30 min vivo (Fig. 4B) detected control reactions                   performing assay. separate experiment, BHE-4 extract <br /> containing synthetic factor peptide ter-                   transferred 1.6 &#194;&#177; 0.1 pmol [3H]farnesyl bacterially produced <br /> minal-AAX residues  CAAX sequence) missing                        p211, control extracts listed transferred &lt;6 fmol Ras <br /> (Fig. 4A, lane 8).                                                            substrate. <br />  11376         Biochemistry: et al.                                                        Proc. -Natl. Acad. Sci. USA 88 (1991) <br />  <br />                                               B                              significantly defective growth low temperatures <br />           1     2   3   4   5   6   7   8   9     1   2                        completely unable grow 37TC (Fig. 5). <br />    2.3- <br />                                                                                   viability ram) null mutants marked contrast                                                     .,   --ii-- factor   lethal effects disrupting RAM2 gene.                                                                                genetic variations strains employed disruption <br />                                                                                RAM] lethal strains used test effects <br />                                                                                RAM2 disruption (data shown). explore pos- <br />    FIG. 4. SDS/PAGE analysis factor famesyltransferase ac-                sibility residual protein farnesyltransferase activity tivity E. coli extracts.   Farnesylation assays factor         ram) null mutants, prepared soluble extract ram) <br /> quindecapeptide substrate performed  text) analyzed               null mutant assayed farnesyltransferase activity. SDS/PAGE fluorography. Unless stated, reac-               reaction product observed using factor quinde- <br /> tion mixtures contained buffer, [3H]FPP, factor 15-mer sub-              capeptide substrate, long exposures fluo- <br /> strate, E. coli extract. Mock reactions shown          rogram (data shown). [3H]farnesyl trans- <br /> E. coli extract peptide substrate (lane 1) E. coli extract         ferred K-rasB peptide using ram] null mutant <br /> (lane 2) absent. Lanes 3-7 display complete reactions containing <br /> extract control E. coli strain BHE-1 (lane 3), extract           extract significantly higher blank values (&lt;2 <br /> BHE-2, expresses RAM] (lane 4), extract BHE-3,                fmol), parallel reactions extract prepared expresses RAM2 (lane 5), mixture extracts                wild-type strain SP1 transferred 284 + 50 fmol. BHE-2 BHE-3 (lane 6), extract BHE-4, coex-                  did detect residual protein farnesyltransferase activity presses RAM] RAM2 (lane 7). Lanes 8 9 represent control                ram] null mutants, conclude activity reactions identical shown lane 7 substitution         required yeast viability. raises paradox, RAS 12-mer factor substrate (lane 8) peptide substrate (lane 9).       function essential yeast. Possible resolution (B) Comigration vitro farnesylated factor authentic                paradox, explanations disparate effects dis- factor. Lane 1, reaction mixture lane 7  Lane 2,             rupting RAM) RAM2, discussed  factor immunoprecipitated culture supernatant MATa strain radiolabeled [35S]cysteine (36, 44). <br /> RAM) RAM2 genes (Table 1). lower activity                                                                                                     DISCUSSION <br /> mixed extracts observed experiments utilizing                      characterized newly identified yeast gene, RAM2, <br /> Ras proteins substrate (data shown) reactions                    shown expression E. coli RAM) utilizing quindecapeptide factor substrate (Fig. 4A,                  RAM2 encode required components yeast <br /> lane 6).                                                                       enzyme responsible farnesylation factor Ras <br />    Effect Disruption RAM) RAM2 Genes Yeast.                    proteins. finding Raml Ram2 proteins examine Raml-Ram2 farnesyl-protein transfer-                       required reconstitution protein farnesyltransferase <br /> ase essential yeast viability, plasmids                   activity parallels finding mammalian Ras protein <br /> contained deletion alleles RAM) RAM2. DNA                            farnesyltransferase comprised different polypeptide <br />                                                                                subunits (19-21). noteworthy components <br /> transformation, constructed diploids heterozy-                    mammalian enzyme, f3 subunit subunit, <br /> gous disrupted RAM alleles. Analysis oftetrads                  structurally homologous Raml Ram2, respectively ram2/RAM2+ diploid indicated ram2 null mu-                         (21, 45, 46). <br /> tants nonviable; 2:2 segregation viability ob-                       RAS function essential yeast, mutant Ras pro- <br /> served 30 tetrads, 60 viable colonies Leu-,                    teins farnesylated supply essential RAS <br /> indicating contained wild-type RAM2 gene.                       function unless significantly overexpressed (47-49). microscopic examination, observed ram2- spores                         somewhat surprising RAM), abso- <br /> occasionally germinated divided times                  lutely required farnesyl-protein transferase activity, ceased growing.                                                                essential gene. possible explanation, based    test effects disrupting RAM) gene,                     ability CAAX peptides efficient substrates analyzed tetrads derived ram) -/RAM) + diploid.                      farnesyl-protein transferase serve efficient sub- <br /> observed 300C ram) spores germinated slowly                        strates GGTase  ram) - mutant cells small formed colonies, indicating ram) null mutants viable                  sufficient Ras proteins geranylgeranylated (Fig. 5). analysis indicated ram) null mutants                    yeast GGTase (22, 23, 25). Accordingly, propose  <br />  <br />  <br />  <br />    FIG. 5. Tetrad analysis raml-/RAM)Idiploids growth phenotype segregants.   Tetrads sporulation dissection diploid SM1589 (ram:: URA3/RAMI), derived SM1060 (36), grown YEPD plates 6 days 30&apos;C. 10 tetrads <br /> displayed 2:2 segregation slow growth. slow-growing colonies determined Ura&apos;, indicating contained ram):: URA3 <br /> allele. (B) colonies single tetrad streaked YEPD plate incubated 300C 3 days. ramP:: URA3 <br /> segregants grew slower wild-type segregants. (C) colonies single tetrad streaked YEPD plate incubated 370C 3 days. ramP:: URA3 segregants did grow  <br />            Biochemistry: et al.                                                      Proc. Natl. Acad. Sci. USA 88 (1991)            11377 <br /> ram2- null mutants nonviable result                 18. Kinsella, B. T. <span id='am-17' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; Maltese, W.  (1991) J. Biol. Chem. 266, <br /> deficient farnesyltransferase activity deficient                     8540-8544. <br /> GGTase activity. proposal based                 19. Reiss, Y., Goldstein, J. L., Seabra, M. C., Casey, P. J. &amp; <br /> finding mammalian subunit farnesyl-protein                  Brown, M. S. (1990) Cell 62, 81-88. <br />                                                                          20. Reiss, Y., Seabra, M. C., Armstrong, S. , Slaughter, C. , <br /> transferase, homologous Ram2, appears                  Goldstein, J. L. &amp; Brown, M. S. (1991) J. Biol. Chem. 266, structural subunit GGTase (21, 24). addition, ram2-1                10672-10677. <br /> mutants shown somewhat defective                      21. Kohl, N. E., Diehl, R. E., Schaber, M. D., Rands, E., Soder- <br /> GGTase activity (23). analysis required                  man, D. D.,  B., Moores, S. L., Pompliano, D. L., Ferro- <br /> fully understand role Ram2 prenyltransferase                  Novick, S., Powers, S., Thomas, K.  &amp; Gibbs, J. B. (1991) <br /> reactions.                                                                   J. Biol. Chem., press. <br />    yeast enzymes postulated involved              22. Reiss, Y., Stradley, S. J., Gierasch, L. M., Brown, M. S. &amp; post-translational modification Ras proteins                    Goldstein, J. L. (1991) Proc. Nail. Acad. Sci. USA 88, 732-736. <br />                                                                          23. Moores, S. L., Schaber, M. D., Mosser, S. D., Rands, E., factor definitively assigned particular                    O&apos;Hara, M. B., Garsky, V. M., Marshall, M. S., Pompliano, <br /> genes Saccharomyces cerevisiae reconsti-                     D. L. &amp; Gibbs, J. B. (1991) J. Biol. Chem. 266, 18884-18888. <br /> tuted expression E. coli. addition Raml-Ram2             24. Seabra, M. C., Reiss, Y., Casey, P. J., Brown, M. S. &amp; <br /> farnesyl-protein transferase, shown STEJ4                    Goldstein, J. L. (1991) Cell 65, 429-434. <br /> encodes methyltransferase modifies factor Ras             25. Yokoyama, K., Goodwin, G. W., Ghomshchi, F., Glomset, <br />                                                                              J.  &amp; Gelb, M. H. (1991) Proc. Nail. Acad. Sci. USA 88, <br /> proteins (36). absence STEJ4 function, prior steps                  5302-5306. Ras protein processing, presumably prenylation prote-              26. Powers, S., Michaelis, S., Broek, D., Santa Anna , S., Field, <br /> olytic removal C-terminal amino acids,                     J., Herskowitz,  &amp; Wigler, M. (1986) Cell 47, 413-422. <br /> significantly delayed. finding suggests different              27. Fujiyama, , Matsumoto, K. &amp; Tamanoi, F. (1987) EMBO J. <br /> steps modification Ras proteins, biochem-                 6, 223-228. <br /> ically separable, coupled vivo.                        28. Wilson, K. L. &amp; Herskowitz,  (1987) Genetics 155, 441-449. <br />                                                                         29. Michaelis, S. &amp; Herskowitz,  (1988) Mol. Cell. Biol. 8, <br />   S.P. thanks Michael Wigler, laboratory STS39 strain            1309-1318. isolated, continuing encouragement. grateful N.   30. Nakayama, N., Arai, K.  &amp; Matsumoto, K. (1988) Mol. Cell. <br /> Kohl J. Gibbs communicating results publication, M.           Biol. 8, 5410-5416. <br /> Kozak critical reading manuscript, S. Ferro-Novick   31. Schafer, W. R., Trueblood, C. E., Yang, C., Mayer, M. P., <br /> discussions. work supported National Institutes               Rosenberg, S., Poulter, C. D., Kim, S.-H. &amp; Rine, J. (1990) <br /> Health Grants GM41258 (S.P.) GM41223 (S.M.), American                    Science 249, 1133-1139. <br /> Cancer Society Grant MG-1 (S.P.), American Heart Asso-           32. Goodman, L. E., Judd, S. R., Farnsworth, C. C., Powers, S., <br /> ciation-Maryland Affiliate Grant Aid (S.M.).                              Gelb, M. H., Glomset, J.  &amp; Tamanoi, F. (1990) Proc. Nail. <br />                                                                              Acad. Sci. USA 87, 9665-9669. <br />                                                                         33. Ohya, Y., Goebl, M., Goodman, L. E., Petersen-Bjorn, S., <br />   1. Gibbs, J. G. (1991) Cell 65, 1-4.                                       Friesen, J. D., Tamanoi, F. &amp; Ankura, Y. (1991) J. Biol. Chem. <br />   2. Glomset, J. , Gelb, M. H. &amp; Farnsworth, C. C. (1990)                  266, 12356-12360. <br />       Trends Biochem. Sci. 15, 139-142.                                 34. Finegold,  , Johnson, D. , Farnsworth, C. C., Gelb, <br />   3. Rine, J. &amp; Kim, S. (1990) New Biol. 2, 219-226.                         M. H., Glomset, J.  &amp; Tamanoi, F. (1991) Proc. Natl. Acad. <br />   4. Goldstein, J. L. &amp; Brown, M. S. (1990) Nature (London) 343,             Sci. USA 88, 4448-4452. <br />      425-430.                                                           35. Rossi, G., Jiang, Y., Newman,  P. &amp; Ferro-Novick, S. (1991) <br />   5. Der, C. J. &amp; Cox,  D. (1991) Cancer Cells 3, 331-340.                 Nature (London) 351, 159-161. <br />   6. Anderegg, R. J., Betz, R., Carr, S. , Crabb, J. W. &amp; Duntze,     36. Hrycyna, C. , Sapperstein, S. K., Clarke, S. &amp; Michaelis, S. <br />      W. (1988) J. Biol. Chem. 263, 18236-18240.                              (1991) EMBO J. 10, 1699-1709. <br />  7. Casey, P. J., Solski, P. , Der, C. J. &amp; Buss, J. E. (1989)        37. Broek, D., Samiy, N., Fasano, O., Fujiyama, , Tamanoi, F., <br />      Proc. Natl. Acad. Sci. USA 86, 8323-8327.                               Northup, J. &amp; Wigler, M. (1985) Cell 41, 763-769. <br />  8. Ishibashi, Y., Sakagami, Y., Isogai,  &amp; Suzuki,  (1984)         38. Reiss, Y., Seabra, M. C., Goldstein, J. L. &amp; Brown, M. S. <br />      Biochemistry 23, 1399-1404.                                             (1990) Meth. Compan. Meth. Enzymol. 1, 241-245. <br />  9. Farnsworth, C. C., Wolda, S. L., Gelb, M. H. &amp; Glomset, J.          39. Sikorski, R. S. &amp; Hieter, P. (1989) Genetics 122, 19-27. <br />      (1989) J. Biol. Chem. 264, 20422-20429.                            40. Broek, D., Toda, T., Michaeli, T., Levin, L., Birchmeier, C., <br /> 10. Yamane, H. K., Farnsworth, C. C., Xie, H., Howald, W.,                   Zoller, M., Powers, S. &amp; Wigler, M. (1987) Cell 48, 789-799. <br />      Fung, B. K.-K., Clarke, S., Gelb, M. &amp; Glomset, J.  (1990)       41. Goodman, L. E., Perou, C. M., Fujiyama,  &amp; Tamanoi, F. <br />      Proc. Natl. Acad. Sci. USA 87, 5868-5872.                               (1988) Yeast 4, 271-281. <br /> 11. Yamane, H. K., Farnsworth, C. C., Xie, H., Evans, T., Gelb,         42. Fields, J., Vojtek, , Ballester, R., Bolger, G., Collicelli, J., <br />      M., Glomset, J. , Clarke, S. &amp; Fung, B. K.-K. (1991) Proc.            Ferguson, K., Gerst, J., Kataoka, T., Michaeli, T., Powers, S., <br />      Natl. Acad. Sci. USA 88, 286-290.                                       Riggs, M., Rodgers, L., Weiland, , Wheland, B. &amp; Wigler, M. <br /> 12. Kawata, M., Farnsworth, C. C., Yoshida, Y., Gelb, M. H.,                 (1990) Cell 61, 319-327. <br />      Glomset, J.  &amp; Takai, Y. (1990) Proc. Natl. Acad. Sci. USA       43. Fedor-Chaiken, M., Deschennes, R. J. &amp; Broach, J. R. (1990) <br />      87, 8960-8964.                                                          Cell 61, 329-340. <br /> 13. Clarke, S., Vogel, J. P., Deschennes, R. J. &amp; Stock, J. (1988)      44. Berkower, C. &amp; Michaelis, S. (1991) EMBO J., press. <br />      Proc. Natl. Acad. Sci. USA 85, 4643-4647.                   <br /> </body></html>