<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Plant Cell, Vol. 16, 3400&#226;&#8364;&#8220;3412, December 2004, www.plantcell.org &#194;&#170; 2004 American Society Plant Biologists <br />  <br />  <br />  <br />  Essential Basic Helix-Loop-Helix Protein FIT1 Required <span id='am-47' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-48' property="rdfs:label" content="Iron" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span>Iron</span> Deficiency Response <br />  <br /> Elizabeth P. Colangelo Mary Lou Guerinot1 <br /> Department Biological Sciences, Dartmouth College, Hanover, New Hampshire 03755 <br />  <br />  <br /> Regulation iron uptake critical plant survival. activities responsible reduction transport iron plant root surface described, genes controlling activities largely unknown. report identification essential gene Fe-deficiency Induced Transcription Factor 1 (FIT1), encodes putative <br /> transcription factor regulates iron uptake responses Arabidopsis thaliana. Like Fe(III) chelate reductase FRO2 high affinity Fe(II) transporter IRT1, FIT1 mRNA detected outer cell layers root accumulates response iron deficiency. fit1 mutant plants chlorotic die seedlings rescued addition supplemental iron, pointing defect iron uptake. fit1 mutant plants accumulate iron wild-type plants root shoot tissues. Microarray analysis shows expression (72 179) iron-regulated genes dependent FIT1. demonstrate FIT1 regulates FRO2 level mRNA accumulation IRT1 level protein <br /> accumulation. propose new model iron uptake Arabidopsis FRO2 IRT1 differentially regulated FIT1. <br />  <br />  <br />  <br />  <br /> INTRODUCTION                                                                 grasses (Takagi et al., 1984), species bacteria                                                                              fungi (Guerinot, 1994), use Strategy II response, relies <br /> Iron essential element organisms, including plants,           chelation Fe(III) reduction. Phytosiderophores required cellular functions including photosynthe-           released soil chelate Fe(III) sis respiration. Iron deficiency poses agricultural chal-             internalized iron-bound state specific transporters <br /> lenge iron nutrients limits            (Curie et al., 2001). <br /> plant growth. Iron deficiency compromises human health                     Reduction transport iron plant root final leading human nutritional disorder worldwide,              steps iron acquisition Strategy plants. Transcripts plants common source dietary iron.                 genes responsible activities induced fore, improving iron content plants benefit                 iron deficiency, providing primary level regulation. FRO2 agriculture human health. Engineering plants increased              IRT1 transcript levels undetectable RNA gel blot analysis <br /> iron levels requires understanding plants cope            plants grown iron sufficient conditions challenges acquiring iron soil pro-                greatly induced 24 h transfer iron-deficient medium <br /> cesses controlled.                                                       (Connolly et al., 2002, 2003). Protein levels tight <br />    Plants overcome iron-deficient growth conditions            control FRO2 IRT1 subject posttran- <br /> ways. Nongraminaceous plants, including Arabidopsis thaliana,                scriptional control (Connolly et al., 2002, 2003), signifying use Strategy response, consists induction                 importance controlling uptake essential poten- activities low iron conditions (Ro&#194;&#168;mheld, 1987).            tially toxic metal. addition local induction iron, signals <br /> H&#195;&#190;-ATPase extrudes protons rhizosphere lower                 induce iron deficiency responses include shoot-derived <br /> pH soil, making Fe(III) soluble. inducible              signals shown reciprocal grafting experiments (Grusak ferric chelate reductase activity FRO2 reduces Fe(III) Fe(II)          Pezeshgi, 1996) split-root experiments (Schmidt et al., 1996; <br /> (Robinson et al., 1999), recently shown rate             Schikora Schmidt, 2001; Vert et al., 2003). nature limiting step iron acquisition soil (Connolly et al.,           shoot-derived signal understood. Iron transport, <br /> 2003). Fe(II) transported plant IRT1 (Eide et al.,       homeostasis, signaling recently reviewed (Curie <br /> 1996), major iron transporter plant root                 Briat, 2003; Hell Stephan, 2003). <br /> (Henriques et al., 2002; Varotto et al., 2002; Vert et al., 2002).          <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span>, description putative transcription factor <br />                                                                              involved iron acquisition plants fer protein 1  correspondence addressed. E-mail guerinot@              tomato (Lycopersicon esculentum). FER mutant initially <br /> dartmouth.edu; fax 603-646-1347.                                             characterized unable induce Strategy responses author responsible distribution materials integral         iron deficiency (Brown et al., 1971; Brown Ambler, <br /> findings presented article accordance policy described   1974). Grafting experiments shown fer gene Instructions Authors (www.plantcell.org)  Mary Lou Guerinot <br /> (guerinot@dartmouth.edu). <br />                                                                              required roots shoots (Brown et al., 1971). Recent <br /> Article, publication date, citation information          cloning fer gene reveals encodes basic helix-loop- <br /> www.plantcell.org/cgi/doi/10.1105/tpc.104.024315.                            helix (bHLH) putative transcription factor (Ling et al., 2002). <br />                                                                                                    FIT1 Regulates Iron Uptake Genes            3401 <br />  <br />  <br />  <br />    Using microarray analysis, identified putative bHLH <br /> transcription factor, Fe-deficiency Induced Transcription Factor <br /> 1 (FIT1), regulates iron deficiency responses Arabidop- <br /> sis. 161 predicted bHLH proteins Arabidopsis (Heim <br /> et al., 2003; Toledo-Ortiz et al., 2003), FIT1 closest homolog tomato fer gene. genes appear play similar, identical, roles respective systems (Ling et al., 2002). superfamily bHLH transcription factors conserved yeast mammals. bHLHs second largest transcription <br /> factor family plants govern wide range biological <br /> processes (Riechmann et al., 2000). conserved bHLH <br /> domain consists ;18 hydrophilic basic amino acids <br /> comprising basic region, permits binding DNA hexanucleotide E-box sequence 59-CANNTG-39. stretches hydrophobic residues separated loop region form amphipathic helices allow proteins form homo- <br /> dimers heterodimers (Voronova Baltimore, 1990; <br /> Toledo-Ortiz et al., 2003). <br />     report characterization essential gene FIT1 role relates iron deficiency responses Arabidopsis. FIT1 required proper regulation ferric <br /> chelate reductase activity iron transport plant root. achieved regulating Fe(III) chelate reductase FRO2 level steady state mRNA accumulation control- <br /> ling protein accumulation Fe(II) transporter IRT1. FIT1 controls genes implicated iron homeostasis novel genes, demonstrate microarray analysis fit1 mutant. <br />  <br /> RESULTS <br />                                                                          Figure 1. Steady State Levels mRNA Protein Iron Uptake <br /> Identification FIT1                                                   Genes fit1-1 Plants. <br />  <br />                                                                          Wild-type fit1-1 plants grown B5 plates 12 d, Microarray analysis used compare transcript abundance             transferred iron sufficient (&#195;&#190;) iron deficient ( ) media 3 d. RNA varying iron conditions wild-type frd3 mutant plants        total protein samples prepared root (R) shoot (S) identify novel iron-regulated genes involved uptake        tissues. <br /> distribution iron. frd3 plants exhibit constitutive iron deficiency     FIT1, FRO2, IRT1 cDNAs used probe individual RNA gel <br /> responses independent iron supply defective             blots, corresponding ethidium bromide&#226;&#8364;&#8220;stained rRNA shown <br /> iron distribution signaling (Rogers Guerinot, 2002).          loading control. <br /> putative bHLH transcription factor FIT1 genes             (B) IRT1 affinity-purified <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> antibody used detect IRT1 <br /> identified message iron regulated wild-type roots        protein ;35 kD. <br /> deregulated frd3 mutant  unpublished data). confirm microarray data, RNA gel blot analysis performed <br />                                                                             FIT1 Arabidopsis bHLH shares homology <br /> showing FIT1 message highly expressed iron- <br />                                                                          outside bHLH domain fer gene tomato, deficient roots iron-sufficient roots undetectable                                                                          recently cloned described playing role regulating <br /> shoots wild-type plants (Figure 1A). frd3 plants, levels                                                                          iron uptake (Ling et al., 2002). FER FIT1 proteins share <br /> FIT1 message equivalent regardless iron supply roots <br />                                                                          42.5% identity 72% similarity (Ling et al., 2002).  absent shoots RNA gel blot analysis (data shown). <br />                                                                          genes appear related functions mutations <br />    FIT1 putative transcription factor, predictions                                                                          genes result similar growth phenotypes planta. <br /> DNA recognition sequence  identity non- <br />                                                                           striking differences conserved amino acids basic region bHLH transcription <br />                                                                          genes, expression pattern, localization, effect factors determine affinity specific E-box sequences                                                                          expression iron uptake genes,  <br /> promoters regulated genes, common G-box 59-CACGTG-39 (Robinson et al., 2000). examination specific residues bHLH domain, FIT1 predicted            Localization FIT1 Iron-Deficient Roots <br /> belong subgroup bHLHs recognize E-box <br /> 59-CANNTG-39, G-box promoters target               identify regions root FIT1 mRNA genes (Toledo-Ortiz et al., 2003).                                       expressed, transgenic plants expressing b-glucuronidase <br />  3402      Plant Cell <br />  <br />  <br />  <br /> (GUS) reporter gene fused 59 end FIT1 control               antisense probe day 7 iron-deficient roots showed FIT1 endogenous FIT1 promoter (FIT1-GUS) analyzed                      message present outer cell layers (Figure 2D). signal <br /> GUS staining. Seedlings T3 transgenic lines                      observed sections day 7 iron deficient roots hybridized <br /> examined showed GUS staining outer cell layers                   FIT1 sense probe (Figure 2E). signal observed <br /> root early day 2 plants germinated iron-deficient                  sections iron-sufficient roots hybridized plates (Figure 2A). GUS staining observed differenti-               sense antisense probes (data shown). FIT1 message, <br /> ation zone absent elongation meristematic                 like Fe(III) chelate reductase FRO2 (Connolly et al., <br /> zones plants germinated iron-deficient plates days 2, 3,              2003) Fe(II) transporter IRT1 (Vert et al., 2002), localized 4, 5, 7, 9 post-germination (Figure 2B). Strong GUS staining               outer cell layer iron-deficient roots, making iron detected lateral roots day 9 plants (Figure 2C)               deficiency response genes potential targets FIT1 regulation. similar pattern observed main root (Figure 2B). <br /> Staining root hairs (Figure 2B) indicates FIT1 expression            FIT1 T-DNA Insertion Lines Indicate Role FIT1 <br /> epidermis. plants germinated iron-sufficient                      Iron Uptake <br /> plates, weak GUS staining observed day 7 day 9, primarily lateral roots (data shown).                 T-DNA insertion line, fit1-1, identified PCR-based localize FIT1 expression, RNA situ hybridization                screening approach Arabidopsis Knockout Facility&#226;&#8364;&#8482;s <br /> performed longitudinal root sections. Hybridization FIT1               collection 60,480 insertion lines (Krysan et al., 1999). second <br />  <br />  <br />  <br />  <br /> Figure 2. Localization FIT1 Iron-Deficient Roots. <br />  <br />   (C) GUS staining representative T3 transgenic line expressing FIT1-GUS translational fusion protein. Plants germinated directly iron-deficient plates containing hygromycin (25 mg/mL). <br />   Staining day 2 seedling. <br /> (B) main root day 9 seedling. <br /> (C) Main lateral roots day 9 seedling. <br /> (D) (E) situ hybridization performed 10-mm longitudinal root sections day 7 plants grown iron-deficient plates using FIT1 antisense <br /> probe (D) FIT1 sense probe (E). <br />                                                                                                   FIT1 Regulates Iron Uptake Genes        3403 <br />  <br />  <br />  <br /> T-DNA insertion line, fit1-2, obtained Salk collection insertion lines (Alonso et al., 2003). T-DNA inserted 106 <br /> bp upstream FIT1 start codon fit1-1 allele 70 bp <br /> downstream start exon fit1-2 allele. RNA <br /> gel blot analysis wild-type Wassilewskija (Ws) fit1-1 plants <br /> shows FIT1 mRNA greatly reduced fit1-1 plants. wild- <br /> type plants, FIT1 mRNA abundant iron-deficient roots, low <br /> levels iron-sufficient roots, absent shoots regardless iron supply (Figure 1A). fit1-1 background, low FIT1 mRNA detectable iron-deficient roots <br /> (Figure 1A). RNA gel blot analysis FIT1 wild-type <br /> Columbia fit1-2 backgrounds shows reduced transcript <br /> accumulation mutant, transcript appears slightly <br /> larger wild type (data shown). <br />    heterozygous fit1 insertion lines visible <br /> phenotype, mutations FIT1 recessive loss function <br /> alleles. Homozygous insertion lines fit1 alleles severe growth phenotype, indicating FIT1 essential survival. Disruption FIT1 gene results lethality seedling stage (Figure 3A). fit1-1 fit1-2 seedlings chlo- <br /> rotic, consistent iron starvation, smaller wild- <br /> type counterparts, die 2 3 weeks post-germination. <br /> Watering fit1 plants supplemental iron overcomes lethality permits fit1 plants reach reproductive stage (Figure <br /> 3B), suggesting FIT1 required iron uptake. genomic <br /> fragment consisting FIT1 coding sequence flanking 59 39 regions used complement growth phenotype fit1 insertion alleles (fit1-1:FIT1 fit1-2:FIT1) lethality result disruption FIT1 gene. Seedling <br /> lethality completely reversed 11 independent fit1-1: <br /> FIT1 (Figure 3A) fit1-2:FIT1 T2 transgenic lines. plants appeared slightly chlorotic compared wild <br /> type, plants healthier fit1 plants, indicating disruption FIT1 responsible observed pheno- <br /> types. <br />  <br />                                                                          Figure 3. Growth Phenotype, Rescue, Complementation fit1-1 <br /> Altered Iron Deficiency Responses fit1                            Plants. <br /> Mutant Background <br />                                                                          Wild-type fit1-1 plants germinated B5 plates. T2 fit1-1 determine FIT1 regulates iron deficiency response genes        transgenic plants complemented genomic fragment containing <br /> FRO2 IRT1, transcripts genes analyzed           FIT1 (fit1-1:FIT1) selected B5 medium containing hygromycin <br /> fit1-1 background RNA gel blot analysis. FRO2 mRNA,             (25 mg/mL). 11 d, wild-type, fit1-1, fit1-1:FIT1 plants                                                                          transferred soil 3 weeks. detected iron-deficient roots wild-type plants,                                                                            added iron. <br /> detectable fit1-1 plants (Figure 1A), indicating FIT1 directly <br />                                                                          (B) Plants watered times week 0.5 g/L Sequestrene indirectly regulates FRO2 level mRNA accumulation.          additional iron source. ferric chelate reductase activity FRO2 measured <br /> using ferrozine assay. Ferric chelate reductase activity, highly induced iron-deficient roots wild-type plants,       controls iron deficiency responses: FRO2 level induced iron deficiency fit1-1 plants (Figure 4).      mRNA accumulation IRT1 level protein accumu- <br /> indicates FRO2 protein activity abolished fit1-1 plants,      lation. expected lack detectable FRO2 mRNA. hand, IRT1 transcript detectable fit1-1           Elemental Analysis fit1-1 Mutant Plants <br /> background RNA gel blot analysis (Figure 1A). IRT1 protein <br /> levels examined using IRT1-specific peptide                 determine fit1 plants display altered iron accumulation, antibody determine IRT1 protein abundance affected           iron content biological replicates wild-type fit1-1 fit1-1 background. IRT1 protein accumulates iron-deficient        mutant 15-d-old seedlings measured inductively coupled <br /> roots wild-type plants, detectable IRT1              plasma&#226;&#8364;&#8220;mass spectrometry. roots fit1-1 plants protein roots fit1-1 plants (Figure 1B).  FIT1       average 43% iron roots wild-type plants  3404      Plant Cell <br />  <br />  <br />  <br />                                                                                conclusion iron deficiency responses upregulated                                                                                35S:FIT1 plants. 35S:FIT1 plants showed obvious growth <br />                                                                                phenotype grown standard B5, iron-deficient, iron- <br />                                                                                sufficient conditions grown soil. <br />  <br />  <br />                                                                                Identification Genes FIT1 Regulation                                                                                Microarray Analysis <br />  <br />                                                                                expression analysis identified FRO2 potential direct <br />                                                                                target FIT1 regulation. identify additional FIT1 targets,                                                                                performed microarray analysis compare expression levels                                                                                roots wild-type fit1-1 plants grown iron- <br />                                                                                sufficient iron-deficient conditions. particular                                                                                genes iron regulated wild type                                                                                deregulated fit1-1 mutant. conditions                                                                                tested statistical significance, genes Bayesian P <br />                                                                                value &lt; 0.05 studied. 179 genes twofold <br />                                                                                upregulated response iron deficiency wild-type roots, 72 <br />                                                                                deregulated twofold fit1-1 iron-deficient roots <br /> Figure 4. fit1-1 Plants Lack Inducible Fe(III) Chelate Reductase Activity.     compared wild-type iron-deficient roots. 72 genes <br /> Wild-type fit1-1 plants grown B5 plates 12 d,                                                                                grouped based regulation FIT1 transferred iron-sufficient (Fe&#195;&#190;) iron-deficient (Fe ) plates 3 d.   predicted function. signal intensities fold changes Fe(III) chelate reductase activity pool plant roots           response iron supply average replicates measured, triplicate, using ferrozine assay (Yi Guerinot,           presented Table 1. genes group previously <br /> 1996). Error bars indicate standard deviation.                                 shown upregulated response iron deficiency, <br />                                                                                including IRT1 (Eide et al., 1996; Connolly et al., 2002), IRT2 (Vert <br />                                                                                et al., 2001; Wintz et al., 2003), NAS1 (Wintz et al., 2003), grown standard B5 conditions, iron-deficient conditions,                 NRAMP1 (Curie et al., 2000; Thomine et al., 2000). iron-sufficient conditions (Figure 5). result                  genes, 59 72, showed near complete loss iron <br /> pronounced standard B5 growth conditions fit1-1                    regulation fit1-1 mutant. conclude increase roots 51% iron wild-type roots. Altered iron <br /> accumulation evident shoots, fit1-1 plants 42% iron grown B5 conditions 21% iron grown iron-sufficient conditions compared wild type (Figure 5). differences statistically significant.  significant differ- <br /> ence shoot iron content plants grown iron- <br /> deficient conditions (Figure 5). <br />  <br />  <br /> 35S:FIT1 Transgenic Plants Obvious Phenotype <br />  decreasing FIT1 mRNA dramatic <br /> effects plant survival, wanted determine increasing <br /> FIT1 copy number effect plant growth expres- <br /> sion iron deficiency response genes. Transgenic plants <br /> expressing FIT1 cDNA control strong, constitutive <br /> 35S promoter generated, independent homozy- <br /> gous single insertion lines examined T4 generation. <br /> RNA gel blot analysis shows FIT1 mRNA highly ex- <br /> pressed regardless iron supply roots shoots transgenic lines (Figure 6A). expression patterns FRO2 IRT1 unchanged compared wild type                   Figure 5. Iron Content Wild-Type fit1-1 Plants. 35S:FIT1 transgenic lines studied (Figure 6A). IRT1 <br />                                                                                Wild-type fit1-1 plants grown B5 plates 12 d, protein accumulation unchanged 35S:FIT1 plants                     harvested transferred iron-sufficient (&#195;&#190;Fe) iron-deficient ( Fe) <br /> compared wild type (Figure 6B). significant              media 3 d. Plants pooled harvested root shoot <br /> difference iron content wild-type plants (average Fe                samples, biological sets tissue subjected elemental <br /> ppm 77.2) independent 35S:FIT1 transgenic lines                      analysis. Standard deviations calculated, statistically signifi- <br /> (average Fe ppm 79.0, 78.3, 76.7), supporting                     differences indicated (*). <br />                                                                                                      FIT1 Regulates Iron Uptake Genes         3405 <br />  <br />  <br />  <br />                                                                             compared wild type iron-sufficient conditions). <br />                                                                             IRT1 mRNA known regulated iron (Eide et al., 1996; <br />                                                                             Connolly et al., 2002; Vert et al., 2003), shown                                                                             IRT1 transcript responds iron deficiency fit1-1 <br />                                                                             mutant (Figure 1).  IRT1 predicted fall                                                                             category genes responding iron-deficient conditions <br />                                                                             fit1-1 mutant loss FIT1. <br />                                                                                Notably, genes known iron regulated                                                                             affected loss FIT1. RNA gel blot analysis NRAMP3 <br />                                                                             (Thomine et al., 2000) recent microarray analysis FRO3 <br />                                                                             (Wintz et al., 2003) shown genes upregulated <br />                                                                             response iron deficiency, transcripts                                                                             affected loss FIT1 (data shown). Similarly, Ferritin1                                                                             Ferritin4 shown downregulated response <br />                                                                             iron deficiency (Petit et al., 2001a; Wintz et al., 2003).                                                                             analysis demonstrates regulation genes iron,                                                                             expression pattern independent FIT1 (data shown). <br />                                                                             results demonstrate portion iron-regulated <br />                                                                             genes FIT1 regulation. FRO6 FRO8 showed                                                                             low expression iron-sufficient iron-deficient condi- <br />                                                                             tions wild-type fit1-1 plants. FRO2, clearly iron <br />                                                                             regulated deregulated fit1-1 (Figure 1), represented <br />                                                                             ATH1 chip. remaining members FRO family <br />                                                                             Arabidopsis represented ATH1 chip. <br />                                                                                possible groups genes similar expression <br />                                                                             patterns controlled regulator(s). ana- <br />                                                                             lyzed 1000 bp sequence upstream translational start <br /> Figure 6. Expression Iron Uptake Genes 35S:FIT1 Plants.               selected FIT1-regulated genes. promoter regions Wild-type plants independent homozygous 35S:FIT1 trans-           searched occurrences bHLH recognition sequence <br /> genic lines grown B5 plates 12 d, transferred iron-     59-CANNTG-39 determine sequences overrepre- <br /> sufficient (&#195;&#190;) iron-deficient ( ) media 3 d.                         sented compared genome-wide noncoding sequences. <br />   RNA prepared root (R) shoot (S) tissues. FIT1, FRO2,    used calculation noncoding sequences IRT1 cDNAs used probe individual RNA gel blots,             Arabidopsis genome GC content 33% determine corresponding ethidium bromide&#226;&#8364;&#8220;stained rRNA shown loading <br />                                                                             recognition sequences occurred frequently control. <br />                                                                             predicted chance (Arabidopsis Genome Initiative, <br /> (B) Total protein prepared, IRT1 affinity-purified peptide <br /> antibody used detect IRT1 protein wild-type plants single   2000). examined upstream regions 10 transporters, <br /> 35S-FIT1 line.                                                              seven transcription factors, 20 iron-regulated genes <br />                                                                             showing greatest deregulation fit1-1 list 72 <br />                                                                             genes. bHLH recognition sequence expression genes iron deficiency wild-type                significantly overrepresented upstream regions plants requires presence FIT1. 72 transcripts           transcription factor group, transporter group, 20 <br /> retain partial iron regulation fit1-1 plants. propose FIT1       genes showing greatest deregulation fit1-1.  35 involved regulation genes             37 genes group contain E-box 1000 <br /> regulatory factor. category genes                bp sequence upstream translational start (Figure 7). <br /> expression level elevated iron-sufficient conditions fit1-1 plants compared wild type. genes, including            DISCUSSION <br /> IRT1, fit description. explanation genes experiencing relief negative regulation fit1-1 mutant.        Homozygous insertion alleles fit1 seedling lethal, FIT1 normally activate negative regulator,               plants rescued watering supplemental iron (Figure <br /> absence FIT1, repression removed expression                  3A). similar growth phenotype previously reported levels elevated fit1-1 plants.  favor ex-             irt1 mutant (Henriques et al., 2002; Varotto et al., 2002; Vert et al., <br /> planation fit1-1 plants experiencing iron deficiency,         2002), suggesting FIT1 regulate IRT1 activity grown iron-sufficient conditions, defect            mutants share growth phenotype. iron uptake. supported data fit1-1 roots             fit1-1 plants unable induce Fe(III) chelate reductase <br /> contain iron wild-type roots (Figure 5).          Fe(II) transport activities (Figures 1 4). fit1-1 plants <br /> genes responding iron-deficient conditions             accumulate iron wild-type counterparts (Figure 5). <br /> fit1-1 mutant loss FIT1. IRT1 shows                   FIT1 copy number increased 35S:FIT1 <br /> expression pattern (elevated transcript levels fit1-1 plants             transgenic plants, obvious alteration FRO2 IRT1 expression <br />  3406      Plant Cell <br />  <br />  <br />  <br />  <br /> Table 1. Summary Microarray Analysis <br />  <br />                      Wild Type &#195;&#190;Fe Wild Type  Fe Fold   fit1-1 &#195;&#190;Fe      fit1-1 -Fe  Fold <br /> Locus Identifier     Baseline Signal Chip Signal Change Baseline Signal Chip Signal Change Annotation <br />  <br /> Iron Regulation Depends Mainly FIT1 <br /> Transporter <br />    At5g38820          194           1300          6.8     189            218        1.2    <span id='am-40' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-41' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-44' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>Amino acid</span> transporter family protein <br />    At4g30120          144            794          5.5     162            173        1.1    HMA3, cadmium-transporting ATPase <br />    At4g21680          122            570          4.7     120            111        1.1    Oligopeptide transporter (POT) family <br />    At1g80830         1531           6465          4.2    1592           2339        1.5    NRAMP1, metal <span id='am-11' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> transporter <br />    At3g53480         1740           5282          3.1    1512           1464        1.0    ABC transporter-like protein <br />    At3g60330          261            780          3.0     313            125        2.5    AHA7, plasma membrane H&#195;&#190;-ATPase <br />    At4g33020          120            299          2.6      97             68        1.5    ZIP9, Fe(II) Zn transport protein <br /> Transcription Factor <br />    At4g09110           50            308          6.2      46             48        1.0    Putative RING-H2 zinc finger protein <br />    At3g13610         2131           8476          4.0    2355           2066        1.2    Similarity DNA binding protein zyxin <br />    At2g28160          636           2244          3.5     251            280        1.1    bHLH29/FIT1 <br />    At5g06490           43            129          3.0      46             53        1.1    C3HC4 RING zinc finger protein-like <br />    At2g20030           89            261          3.0     106             97        1.1    Putative RING zinc finger protein <br /> Signaling <br />    At1g34760          128           1097          8.4     153            131        1.2    14-3-3 Protein <br />    At3g61410          197            710          3.6     267            247        1.1    Putative protein protein kinase <br />    At2g19410           44            137          3.1      48             50        1.0    Putative protein kinase, Class 1 <br />    At1g77280          375           1125          3.0     376            427        1.1    Receptor-like protein kinase, Class 1 <br />    At1g51860          308            847          2.8     329            206        1.6    Receptor-like protein kinase, Class 1 <br />    At5g35580          412            983          2.3     431            329        1.3    Ser/Thr protein kinase-like, Class 1 <br />    At5g01060          130            278          2.1      91             79        1.2    Putative protein kinase, Class 1 <br /> Metabolism <br />    At4g31940           32           1727         55.0      22             24        1.1    Cytochrome P450-like monooxygenase <br />    At3g53280           27            519         19.4      27             27        1.0    Cytochrome P450 71B5 <br />    At5g02780          222           4245         19.1     227            415        1.8    Putative protein In2 <br />    At3g12900           96           1730         18.5      71             66        1.1    Hypothetical, similar oxidoreductases <br />    At3g11750           41            433         10.5      44             56        1.3    Putative dihydroneopterin aldolase <br />    At4g02330           64            398          6.3      50             50        1.0    Hypothetical, similar pectinesterase <br />    At5g36890          468           2784          6.2     373            369        1.0    b-Glucosidase <br />    At4g31950           25            118          4.7      23             24        1.0    Cytochrome P450-like monooxygenase <br />    At2g01880          268           1041          4.0     358            385        1.1    Putative purple acid phosphatase <br />    At2g02310           45            171          3.9      49             45        1.1    Putative phloem-specific lectin <br />    At1g09790          117            441          3.8     115            131        1.1    Putative phytochelatin synthetase <br />    At3g47040          247            934          3.7     307            329        1.1    b-D-Glucan exohydrolase-like protein <br />    At5g04950         1425           4951          3.5    1147            980        1.2    NAS1, nicotianamine synthase <br />    At3g47420          223            740          3.3     307            145        2.1    Putative sn-glycerol-3-phosphate permease <br />    At3g21240         1562           4646          3.0    1789           2142        1.2    Putative 4-coumarate:<span id='am-35' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-36' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CoA</span> ligase 2 <br />    At1g18910          663           1841          2.8     698            716        1.0    Similar flavonol-induced pollen germ. <br />    At1g60610          371           1025          2.8     361            493        1.4    Similar S-ribonuclease binding protein <br />    At4g29220          754           2027          2.7     871            840        1.0    Phosphofructo-1-kinase-like <br />    At3g31415          164            419          2.6     178            168        1.1    Vetispiradiene synthase, putative <br />    At2g05830          698           1761          2.5     693            666        1.0    Putative translation initiation factor eIF-2B <br />    At2g30670           74            181          2.4      57             74        1.3    Putative tropinone reductase <br />    At2g40000          414            991          2.4     428            468        1.1    Putative nematode-resistance protein <br />    At1g05530           57            133          2.3      53             50        1.1    Indole-3-acetate b-D-glucosyltransferase <br />    At4g12910          206            483          2.3     250            236        1.1    Ser carboxypeptidase precursor-like <br />    At3g54580          876           2049          2.3    1084            592        1.8    Extensin precursor-like protein <br />    At2g37040         2089           4272          2.1    2145           2021        1.1    Phe ammonia lyase (PAL1) <br />    At4g38950          119            243          2.1     117            115        1.0    Kinesin-like protein <br />    At4g14680          288            595          2.1     243            222        1.1    <span id='am-24' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-25' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>-sulfurylase <br />    At4g10510           75            154          2.1      59             72        1.2    Subtilisin-like Ser protease <br /> Unknown <br />    At3g61930           69           2229         32.5      55             54        1.0    Hypothetical protein <br />    At4g19370          117           1157          9.7     148            199        1.3    Hypothetical protein <br />    At3g58060           68            605          8.9      74             77        1.0    Putative protein <br />    At3g51200           37            274          7.4      34             43        1.3    Putative protein <br />                                                                                                                            (Continued) <br />                                                                                                          FIT1 Regulates Iron Uptake Genes           3407 <br />  <br />  <br />  <br />  <br /> Table 1. (continued). <br />  <br />                         Wild Type &#195;&#190;Fe Wild Type  Fe Fold   fit1-1 &#195;&#190;Fe      fit1-1 -Fe  Fold <br /> Locus Identifier        Baseline Signal Chip Signal Change Baseline Signal Chip Signal Change Annotation <br />  <br />   At1g49820             603              2160             3.6      743              761           1.0     Unknown protein <br />   At5g54790              76               261             3.4       97               72           1.3     Unknown protein <br />   At3g48450             274               790             2.9      307              375           1.2     Hypothetical protein <br />   At3g06890             261               713             2.8      241              265           1.1     Hypothetical protein <br />   At5g40590             777              1958             2.5      804              577           1.4     Putative protein <br />   At3g18560             393               907             2.3      473              346           1.4     Unknown protein <br />   At2g46740             822              1825             2.2      840              768           1.1     Unknown protein <br /> Iron Regulation Partially Dependent FIT1 <br /> Transporter <br />    At4g19680          103               2156            21.0       149              335           2.3     IRT2, Fe(II) transport protein <br />    At3g46900          133                766             5.8       141              261           1.9     COPT2, copper transport protein <br /> Transcription Factor <br />    At1g56160            75              1756            23.4        92              582           6.3     MYB72 <br />    At3g12820            70               790            11.7        94              222           2.4     MYB10 <br /> Signaling <br />    At1g05700          232                687              3.1      231              317           1.4     Putative light repressible receptor kinase, <br /> Metabolism <br />    At1g14190          115                639              5.6      134              251           1.9     Putative mandelonitrile lyase <br />    At5g47910          733               2763              3.8      785             1091           1.4     RbohD, respiratory burst oxidase protein <br /> Unknown <br />    At1g73120          148               1548            10.5        78              391           5.0     Hypothetical protein <br /> Baseline Expression   Levels Higher fit1-1 &#195;&#190;Fe versus Wild-Type &#195;&#190;Fe <br /> Transporter <br />   At4g19690             368              6142           16.3      1224             2516           2.1     IRT1, Fe(II) transport protein <br />   At3g58810             355              4257           11.9      1080             1864           1.7     MTPa2, CDF family <br />   At5g03570             778              5097            6.6      1441             2041           1.4     FERROPORTIN2, putative Fe transporter <br /> Metabolism <br />   At3g50740             437              3871             8.6      781              907           1.2     <span id='am-52' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span>UTP</span>-glucose glucosyltransferase-like <br /> Unknown <br />   At3g07720             904            11666            12.9      3525             4078           1.2     Unknown protein locus identifiers given genes twofold upregulated response iron deficiency wild type twofold downregulated fit1-1 iron-deficient roots compared wild-type iron-deficient roots. Seventy genes met requirements statistically significant. Wild-type fit1-1 chip signal intensities fold changes presented average biological <br /> replicates. Genes grouped based regulation FIT1 secondly predicted function. Genes known involved metal transport homeostasis shown bold. <br />  <br />  <br />  <br />  IRT1 protein accumulation observed (Figure 6),                   needs regulated iron, interacting partners involved growth phenotypes revealed. demonstrate                       iron homeostasis identified. FIT1 message overexpressed 35S:FIT1 plants,                   sequence similarity outside bHLH motif, FIT1 possible FIT1 protein levels altered trans-             Arabidopsis protein closely related FER protein <br /> genic lines. Alternatively, overexpression FIT1               tomato. described  chlorotic lethal phenotype sufficient alter expression target genes. likely              fit1 plants correlates description previously given bHLH transcription factors shown di-                      fer mutant (Brown et al., 1971; Ling et al., 2002). merize bHLHs members transcrip-                   shown FIT1 message accumulates higher levels tion factor families, MYB family (Goff et al., 1992; Abe           iron-deficient growth conditions iron-sufficient <br /> et al., 1997; Grotewold et al., 2000), partners                growth conditions roots (Figure 1A). contrast required affect transcription target genes. identified                expression pattern fer, reported  iron-regulated transcription factors representing variety            dependent iron supply, like FIT1, fer families, including bHLH MYB families, microarray                expressed root-specific manner (Ling et al., 2002). fit1 fer <br /> analysis. investigate possibility FIT1 interacts              mutants display inability induce ferric chelate  potential binding partners regulate               ductase activity iron deficiency.  shown <br /> iron uptake.  partner dimer                    RNA gel blot analysis unlike FRO2, IRT1 message does <br />  3408      Plant Cell <br />  <br />  <br />  <br />  <br /> Figure 7. Promoter Analysis Potential FIT1 Binding Sites. <br />  promoter regions 37 iron-regulated genes deregulated fit1-1 analyzed occurrence E-box motif 59-CANNTG-39, potential FIT1 binding site. number sites upstream region indicated left schematic, represents 1000 <br /> bp sequence upstream translational start. Approximate positions E-box sites indicated filled squares, drawn scale. <br />  <br />  <br /> accumulate fit1-1 mutant FIT1 controls IRT1             outer layers root (Vert et al., 2002; Connolly et al., 2003), level protein accumulation (Figure 1). contrast             possible FIT1 regulating activities direct <br /> reports Leirt1 mRNA abundance dependent fer                     manner, FER govern activities indirectly, <br /> gene (Ling et al., 2002; Bereczky et al., 2003). FIT1 fer              regulator(s). demonstrated <br /> genes share similarities differences terms mRNA                FIT1 expressed outer cell layers differentiation <br /> localization pattern. fer message detectable                    zone wild-type roots (Figures 1 2). data supported <br /> epidermal cells mature, root hair zone tomato (Ling                  large-scale microarray analysis (AtGenExpress, http:// <br /> et al., 2002). shown Arabidopsis FIT1 localizes             www.cbs.umn.edu/arabidopsis/) expression analysis (Mas- <br /> root hairs differentiation zone histochemical staining            sively Parallel Signature Sequencing database, http://mpss. <br /> FIT1-GUS transgenic plants (Figure 2B). transcripts                 udel.edu java.html), indicating FIT1 expression highest iron deficiency response genes FRO2 IRT1 localize               roots compared tissues. Birnbaum et al. (2003) <br />                                                                                                     FIT1 Regulates Iron Uptake Genes            3409 <br />  <br />  <br />  localization gene expression Arabi-           analyzed microarray analysis, LeIRT2, LeNAS, dopsis root report FIT1 highly expressed         gene encoding 14-3-3 protein respond Pi, K, <br /> outer cell layers (epidermis lateral root cap) stele,    Fe deficiencies (Wang et al., 2002). mentioned Results, <br /> endodermis, cortex (Birnbaum et al., 2003). Expression            Wintz et al. reported IRT2 NAS1 iron regulated using lowest stage 1  root tip) highest stage 3          Affymetrix DNA chip representing 8300 Arabidopsis genes <br /> (higher root root hairs present) (Birnbaum et al.,      (Wintz et al., 2003). <br /> 2003), similar results.                                              FRO2 IRT1 messages coordinately regulated  <br />    Previous studies plants identified promoter elements          sponse iron. plants transferred iron-sufficient iron- <br /> involved regulation iron. cis-regulatory element,          deficient conditions, transcripts detectable RNA gel <br /> iron-dependent regulatory sequence (IDRS), identified         blot 24 h, transcript levels peaking 3 d transfer <br /> promoter region ZmFerritin1 (Petit et al., 2001b). IDRS        (Connolly et al., 2002, 2003). plants grown iron- <br /> conserved AtFer1 permits induction ferritin iron        deficient conditions transferred iron-sufficient conditions, repressing ZmFer1 AtFer1 iron levels low (Petit          FRO2 IRT1 mRNA levels quickly decrease undetect- <br /> et al., 2001b; Tarantino et al., 2003). IDRS       able RNA gel blot 24 h transfer (Connolly et al., 2002, <br /> shown control genes respond presence iron,       2003). genes related activities likely distinct mechanism exists controlling genes       similar expression patterns, likely respond iron deficiency. Recently, iron deficiency&#226;&#8364;&#8220;               activated regulator.  findings FIT1 <br /> responsive elements (IDE1 IDE2) identified               regulates FRO2 level mRNA accumulation IRT1 promoter IDS2 gene barley (Hordeum vulgare)                level protein accumulation suggests regulation shown required iron deficiency&#226;&#8364;&#8220;inducible expression         FRO2 IRT1 complicated previously thought. <br /> HvIDS2 tobacco roots (Kobayashi et al., 2003). iron-          fit1-1 mutants die seedling stage lack <br /> inducible genes barley, rice (Oryza sativa), Arabidopsis          FRO2 mRNA IRT1 protein, conclude FIT1 plays <br /> contain sequences homologous IDE1, including AtFRO2               significant role iron homeostasis. FRO2 demon- <br /> AtIRT1 (Kobayashi et al., 2003). FIT1 belongs             strated serve rate limiting step iron acquisition <br /> bHLH family predicted recognize specific sequence         (Connolly et al., 2003).  ability regulate FRO2 makes promoters regulated genes, focused occurrence           FIT1 important factor iron uptake. possible E-box motif genes identified potential FIT1 targets      FIT1 binds directly FRO2 promoter regulate <br /> microarray analysis. promoter analysis 10 trans-         transcription, likely additional regulatory step(s) <br /> porters, seven transcription factors, 20 iron-regulated          FIT1 IRT1. need determine genes showing greatest deregulation fit1-1 revealed          regulators IRT1 exist downstream FIT1. Previous studies E-box sequence overrepresented            shown IRT1 subject posttranscriptional control. groups, possible FIT1 direct regulator   Transgenic plants expressing IRT1 control 35S cases. Thirty 37 promoters analyzed, including       promoter accumulate IRT1 mRNA roots shoots FRO2, contain E-box sequence (Figure 7),            iron-sufficient -deficient conditions, IRT1 protein supporting possibility direct targets FIT1.         accumulates iron-deficient roots (Connolly et al., 2002). IRT1 E-box sequence occur random,              protein controlled posttranslationally ubiquitination anticipate E-boxes identified Figure 7 serve     endocytosis. Ubiquitin-mediated protein turnover ZRT1, <br /> FIT1 DNA binding sites.                                                  metal transporter belonging family IRT1,    analysis addresses plant iron deficiency responses         demonstrated yeast dependent critical Lys <br /> scale genome using ATH1 Affymetrix chip, represents ;24,000 genes. microarray <br /> studies addressed iron deficiency induced changes gene expression.  studies difficult use direct comparison data different experimental <br /> growth conditions employed  example, Thimm <br /> et al. used 6000 cDNA chip studied expression older <br /> plants different ecotype (Landsberg erecta) grown <br /> hydroponically. identified set genes induced iron <br /> deficiency, including encoding cytochrome P450-like <br /> proteins encoding zinc finger proteins (Thimm et al., <br /> 2001). did identify exact genes microarray analysis, reported cytochrome P450 proteins zinc finger transcription factors transcripts accumu- <br /> late response iron deficiency. chip representing 8987 rice       Figure 8. Model IRT1 Protein Regulation FIT1. <br /> clones used analyze genes responding iron deficiency           FIT1 prevent IRT1 protein turnover inhibition unknown barley roots (Negishi et al., 2002). OsNAS1 gene zinc       factor (product Gene X), allowing IRT1 protein accumulate finger protein identified way, consistent     iron-deficient conditions. Failure activate Gene X fit1-1 plants findings. tomato, 1280 mineral nutrition-related genes            prevent IRT1 protein accumulation. <br />  3410      Plant Cell <br />  <br />  <br />  <br /> residue variable loop region (Gitan Eide, 2000). IRT1                Plasmid Construction Plant Transformation contains Lys residues analogous region,                                                                                 35S:FIT1 Fusion <br /> serve ubiquitination sites mediate protein degradation <br /> (Connolly et al., 2002). explanation loss IRT1 fit1-1         FIT1 cDNA, clone RZ108e05 (Asamizu et al., 2000), subcloned <br /> plants FIT1 regulates factor(s) involved IRT1 protein              pGEM-TEasy (Promega, Madison, WI). cDNA excised using <br /> turnover. model depicting IRT1 protein regulation FIT1                  BamHI cloned BamHI site pCGN18. pCGN18 gift <br />                                                                                 T. Jack (Department Biological Sciences, Dartmouth College, <br /> presented Figure 8. FIT1 negatively regulate IRT1 protein <br />                                                                                 Hanover, NH; Connolly et al., 2002). construct moved turnover. wild-type roots iron-deficient conditions, IRT1 <br />                                                                                 Agrobacterium tumefaciens strain ASE transformed wild-type <br /> protein turnover inhibited Gene X, positively              Columbia plants. plant transformations floral dip <br /> regulated FIT1.  fit1-1 mutant, Gene X               method (Clough Bent, 1998). <br /> activated FIT1, IRT1 protein turnover ensues. <br />   Finally, FIT1 itself iron regulated, need look           FIT1-GUS Fusion iron-regulated transcription factors control FIT1                                                                                 1924-bp PCR fragment amplified Ws genomic DNA using directly regulate IRT1. Upstream FIT1,                                                                                 59-CGGGATCCCAACACCTAGATGGAATC-39 59-AACACTGCATC- <br /> expect iron sensor itself affected iron <br />                                                                                 TCCAACAATCCATGC-39 primers subcloned pGEM-T Easy <br /> status sense iron levels communicate message <br />                                                                                 (Promega). fragment containing 1333 bp sequence upstream activation repression downstream targets.                     FIT1 translational start 59 435 bp coding sequence excised <br />                                                                                 HindIII BamHI digestion cloned pCAMBIA1381Xa <br />                                                                                 (GenBank accession number AF234303) HindIII BamHI sites <br /> METHODS                                                                         creating frame translational fusion gusA gene                                                                                 confirmed DNA sequencing. construct moved Agro- <br />                                                                                 bacterium strain GV3101 transformed wild-type Ws plants. <br /> Identification FIT1 Loss Function Mutants <br />  FIT1 locus identifier At2g28160 named AtbHLH029               fit1 Complementation <br /> reported (Heim et al., 2003). FIT1-specific primer 59-CAACAA- <br /> TCTCGGTTACATCATCACTAGAA-39 T-DNA&#226;&#8364;&#8220;specific primer                          2722-bp fragment containing FIT1 coding sequence, 1140-bp <br />                                                                                 sequence upstream translational start, 386-bp downstream 59-CATTTTATAATAACGCTGCGGACATCTAC-39 used screen <br />                                                                                 stop codon amplified PCR engineered XbaI sites Arabidopsis Knockout Facility&#226;&#8364;&#8482;s collection T-DNA insertion lines <br />                                                                                 cloned XbaI site pCAMBIA1300 (GenBank accession number <br /> (Ws ecotype) PCR (Krysan et al., 1999). fit1-1 allele identified <br />                                                                                 AF234296). plasmid moved Agrobacterium strain GV3101 confirmed DNA gel blot hybridization. insertion site                                                                                 transformed fit1-1 fit1-2 plants. <br /> confirmed DNA sequencing using T-DNA&#226;&#8364;&#8220;specific primer. fit1-2 <br /> insertion mutant (Columbia-0 ecotype) obtained Salk <br /> collection (Alonso et al., 2003). seed stocks obtained           Gel Blot Hybridization ABRC (http://www.biosci.ohio-state.edu/;plantbio/Facilities/abrc/ <br />                                                                                 Root shoot tissues harvested plants grown iron- <br /> abrchome.htm). <br />                                                                                 sufficient iron-deficient conditions. Total RNA prepared hot <br />                                                                                 phenol extraction treated glyoxal (McMaster Carmichael, <br />                                                                                 1977). micrograms total RNA separated 1.2% agarose gel <br /> Plant Materials Growth Conditions <br />                                                                                 10 mM NaPO4, transferred nylon membrane, UV cross-linked. <br /> Seeds surface sterilized 95% ethanol followed gentle shaking         Hybridizations performed 428C 50% formamide according 25% bleach/0.2% SDS 20 min. Seeds suspended 0.15% agar                standard procedures (Ausubel et al., 2004). Individual blots probed placed dark 48C 2 4 d, plated Gamborg&#226;&#8364;&#8482;s B5        32P-labeled FIT1, FRO2, IRT1 cDNAs. Blots visualized medium (Sigma, St. Louis, MO) 2% sucrose, 1 mM Mes, 0.7%               2 16 h exposure Typhoon Phosphorimager screen (Molecular <br /> agar, pH 5.8.  true-leaf stage, seedlings transferred   Dynamics, Sunnyvale, CA). iron-sufficient plates containing 50 mM Fe(III)-EDTA iron-deficient <br /> plates containing 300 mM ferrozine [3-(2-pyridyl)-5,6-diphenyl-1,2,4- <br />                                                                                 Immunoblot Analysis <br /> triazine sulfonate] (HACH <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span>, Ames, IA) 3 d. media contain macronutrients micronutrients (Marschner et al., 1982), 0.7%        Immunoblot analysis performed previously described (Connolly <br /> agar 1 mM Mes, pH 6.0. Plants grown 218C constant             et al., 2002). Briefly, total protein prepared plants grown light (;90 m 2 s 1) yellow filter (acrylic yellow-2208; Cadillac     iron-sufficient iron-deficient conditions described  Approx- <br /> Plastic Chemical, Pittsburgh, PA) protect Fe(III)-EDTA          imately 10 mg protein separated SDS-PAGE transferred photochemical degradation (Hangarter Stasinopoulos, 1991). Plants           polyvinylidene fluoride membrane. Membranes blocked, incu- <br /> used situ hybridization studies germinated directly iron-        bated overnight affinity-purified IRT1 peptide antibody, washed, deficient iron-sufficient plates grown vertically. Plants used       incubated goat anti-rabbit IgG conjugated horseradish peroxidase <br /> GUS histochemical staining germinated directly iron-deficient        1 h. Chemiluminescence performed using Western Lightning kit <br /> iron-sufficient plates. Ferric chelate reductase assays performed       (Perkin-Elmer, Boston, MA). <br /> previously described (Yi Guerinot, 1996). Pools plants analyzed triplicate, standard deviations calculated. Soil- <br />                                                                                 Elemental Analysis <br /> grown fit1 plants watered 0.5 g/L Sequestrene (Helena <br /> Chemical, Spartanburg, SC) times week seedling               Wild-type fit1-1 mutant plants grown standard B5, iron- <br /> stage. reproductive stage, watering supplemented         sufficient, iron-deficient plates. Root shoot tissues harvested 0.5 g/L Sequestrene.                                                    dried overnight 658C oven. Elemental analysis performed <br />                                                                                                          FIT1 Regulates Iron Uptake Genes            3411 <br />  <br />  <br />  <br /> using inductively coupled plasma spectroscopy Purdue University          Asamizu, E., Nakamura, Y., Sato, S., Tabata, S. (2000). large <br /> described (Lahner et al., 2003). Standard deviations calculated         scale analysis cDNA Arabidopsis thaliana: Generation 12,028 biological replicates.                                                       non-redundant expressed sequence tags normalized size- <br />                                                                                  selected cDNA libraries. DNA Res. 7, 175&#226;&#8364;&#8220;180. Situ Hybridization                                                          Ausubel, R.M., Brent, R., Kingston, R.E., Moore, D.D., Seidman, <br />                                                                                  J.G., Smith, J , Struhl, K. (2004). Current Protocols FIT1 cDNA clone RZ108e05 (Asamizu et al., 2000) cloned                  Molecular Biology. (New York: John Wiley <span id='am-17' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Sons). <br /> pBluescript II SK  EcoRI XhoI sites (FIT1-SK) used           Bereczky, , Wang, H.Y., Schubert, V., Ganal, M., Bauer, P. <br /> generate sense antisense probes situ hybridization. Probes            (2003). Differential regulation nramp irt metal transporter genes labeled digoxigenin-11-UTP (Roche Diagnostics, Indianapolis,           wild type iron uptake mutants tomato. J. Biol. Chem. 278, . Linearization FIT1-SK performed ApaI digestion followed        24697&#226;&#8364;&#8220;24704. <br /> transcription T3 polymerase obtain sense probe. Linearization      Birnbaum, K., Shasha, D.E., Wang, J.Y., Jung, J.W., Lambert, G.M., SmaI followed transcription T7 polymerase used               Galbraith, D.W., Benfey, P.N. (2003). gene expression map create antisense probe. Tissue samples fixed embedded            Arabidopsis root. Science 302, 1956&#226;&#8364;&#8220;1960. <br /> described (Di Laurenzio et al., 1996), exception 70%   Brown, J.C., Ambler, J.E. (1974). Iron-stress response tomato <br /> ethanol dehydration step, tissue embedded 1% agarose.              (Lycopersicon esculentum). Physiol. Plant 31, 221&#226;&#8364;&#8220;224. <br /> situ hybridization performed previously described (Long et al.,         Brown, J.C., Chaney, R.L., Ambler, J.E. (1971). new tomato <br /> 1996; Long Barton, 1998; Vert et al., 2002; Connolly et al., 2003).          mutant inefficient transport iron. Physiol. Plant. 25, 48&#226;&#8364;&#8220;53. <br />                                                                                Clough, S.J., Bent, F. (1998). Floral dip: simplified method <br /> GUS Histochemical Staining                                                       Agrobacterium-mediated transformation Arabidopsis thaliana. <br />                                                                                  Plant J. 16, 735&#226;&#8364;&#8220;743. <br /> GUS histochemical staining performed independent T3 <br />                                                                                Connolly, E.L., Campbell, N.H., Grotz, N., Prichard, C.L., transgenic lines day 4 day 7 growth iron-deficient iron- <br />                                                                                  Guerinot, M.L. (2003). Overexpression FRO2 ferric chelate <br /> sufficient plates containing hygromycin (25 mg/mL). representative <br />                                                                                  reductase confers tolerance growth low iron uncovers <br /> T3 line examined conditions time points <br />                                                                                  posttranscriptional control. Plant Physiol. 133, 1102&#226;&#8364;&#8220;1110. day 1 day 9. Seedlings incubated substrate <br />                                                                                Connolly, E.L., Fett, J.P., Guerinot, M.L. (2002). Expression 5-bromo-4-chloro-3-indolyl b-D-glucuronide described (Jefferson et <br />                                                                                  IRT1 metal transporter controlled metals levels al., 1987). <br />                                                                                  transcript protein accumulation. Plant Cell 14, 1347&#226;&#8364;&#8220;1357. <br />                                                                                Curie, C., Alonso, J.M., Le Jean, M., Ecker, J.R., Briat, J.F. <br /> Microarray Analysis                                                              (2000). Involvement NRAMP1 Arabidopsis thaliana iron <br />                                                                                  transport. Biochem. J. 347, 749&#226;&#8364;&#8220;755. <br /> Total RNA biological replicates reverse transcribed, <br />                                                                                Curie, C., Briat, J.F. (2003). Iron transport signaling plants. <br /> labeled, hybridized individual ATH1 Affymetrix chips UCI <br />                                                                                  Annu. Rev. Plant Biol. 54, 183&#226;&#8364;&#8220;206. <br /> DNA Array Core Facility (Irvine, CA). Expression data analyzed using <br /> GeneTraffic (Iobion Informatics, La Jolla, CA). Determination statistical   Curie, C., Panaviene, , Loulergue, C., Dellaporta, S.L., Briat, J.F., <br /> significance performed using Cyber-T statistics program (http://         Walker, E.L. (2001). Maize yellow stripe1 encodes membrane <br /> visitor.ics. uci.edu/genex/cybert/).                                             protein directly involved Fe(III) uptake. Nature 409, 346&#226;&#8364;&#8220;349. <br />                                                                                Di Laurenzio, L., Wysocka-Diller, J., Malamy, J.E., Pysh, L., <br />                                                                                  Helariutta, Y., Freshour, G., Hahn, M.G., Feldmann, K ,                                                                                  Benfey, P.N. (1996). SCARECROW gene regulates asymmetric <br /> ACKNOWLEDGMENTS                                                                  cell division essential generating radial organization                                                                                  Arabidopsis root. Cell 86, 423&#226;&#8364;&#8220;433. authors thank Brett Lahner David Salt performing elemen-           Eide, D., Broderius, M., Fett, J., Guerinot, M.L. (1996). novel <br /> tal analysis, Carol Ringelberg help microarray analysis, Arijit         iron-regulated metal transporter plants identified functional <br /> Chakravarty discussion promoter analysis, Sara Thiebaud               expression yeast. Proc. Natl. Acad. Sci. USA 93, 5624&#226;&#8364;&#8220;5628. <br /> experimental assistance, Rob McClung critical reading           Gitan, R.S., Eide, D.J. (2000). Zinc-regulated ubiquitin conjugation <br /> manuscript. work supported National Science Foundation <br />                                                                                  signals endocytosis yeast ZRT1 zinc transporter. Biochem. J. <br /> Plant Genome grant (0077378-DBI) M.L.G. <br />                                                                                  346, 329&#226;&#8364;&#8220;336. <br />                                                                                Goff, S , Cone, K.C., Chandler, V.L. (1992). Functional analysis <br />                                                                                  transcriptional activator encoded maize B gene: <br /> Received 18, 2004; accepted September 18, 2004. <br />                                                                                  Evidence direct functional interaction classes                                                                                  regulatory proteins. Genes Dev. 6, 864&#226;&#8364;&#8220;875. <br />                                                                                Grotewold, E., Sainz, M.B., Tagliani, L., Hernandez, J.M., Bowen, B., <br /> REFERENCES <br />                                                                                  Chandler, V.L. (2000). Identification residues Myb <br /> Abe, H., Yamaguchi-Shinozaki, K., Urao, T., Iwasaki, T., Hosokawa,               domain maize C1 specify interaction bHLH <br />                                                                                  cofactor R. Proc. Natl. Acad. Sci. USA 97, 13579&#226;&#8364;&#8220;13584. <br />   D., Shinozaki, K. (1997). Role Arabidopsis MYC MYB <br />                                                                                Grusak, M , Pezeshgi, S. (1996). Shoot root signal trans- <br />   homologs drought- abscisic acid-regulated gene expression. <br />                                                                                  mission regulates root Fe(III) reductase activity dgl mutant   Plant Cell 9, 1859&#226;&#8364;&#8220;1868.                                                       pea. Plant Physiol. 110, 329&#226;&#8364;&#8220;334. <br /> Alonso, J.M., et al. (2003). Genome-wide insertional mutagenesis            Guerinot, M.L. (1994). Microbial iron transport. Annu. Rev. Microbiol. <br />   Arabidopsis thaliana. Science 301, 653&#226;&#8364;&#8220;657.                                    48, 743&#226;&#8364;&#8220;772. <br /> Arabidopsis Genome Initiative (2000). Analysis genome                   Hangarter, R.P., Stasinopoulos, T.C. (1991). Effect Fe- <br />   sequence flowering plant Arabidopsis thaliana. Nature 408,              catalyzed photooxidation EDTA root growth plant culture <br />   796&#226;&#8364;&#8220;815.                                                                       media. Plant Physiol. 96, 843&#226;&#8364;&#8220;847. <br />  3412      Plant Cell <br />  <br />  <br />  <br /> Heim, M , Jakoby, M., Werber, M., Martin, C., Weisshaar, B.,            Robinson, N.J., Procter, C.M., Connolly, E.L., Guerinot, M.L. <br />   Bailey, P.C. (2003). basic helix-loop-helix transcription factor          (1999). ferric-chelate reductase iron uptake soils. Nature <br />   family plants: genome-wide study protein structure                397, 694&#226;&#8364;&#8220;697. <br />   functional diversity. Mol. Biol. Evol. 20, 735&#226;&#8364;&#8220;747.                         Rogers, E.E., Guerinot, M.L. (2002). FRD3, member Hell, R., Stephan, U.W. (2003). Iron uptake, trafficking                multidrug toxin efflux family, controls iron deficiency responses   homeostasis plants. Planta 216, 541&#226;&#8364;&#8220;551.                                   Arabidopsis. Plant Cell 14, 1787&#226;&#8364;&#8220;1799. <br /> Henriques, R., Jasik, J., Klein, M., Martinoia, E., Feller, U., Schell, J.,   Ro&#194;&#168;mheld, V. (1987). Different strategies iron acquisition higher <br />   Pais, M.S., Koncz, C. (2002). Knock Arabidopsis metal              plants. Physiol. Plant. 70, 231&#226;&#8364;&#8220;234. <br />   transporter gene IRT1 results iron deficiency accompanied cell        Schikora, , Schmidt, W. (2001). Iron stress-induced changes   differentiation defects. Plant Mol. Biol. 50, 587&#226;&#8364;&#8220;597.                        root epidermal cell fate regulated independently physiolog- <br /> Jefferson, R , Kavanagh, T , Bevan, M.W. (1987). GUS                    ical responses low iron availability. Plant Physiol. 125, 1679&#226;&#8364;&#8220;1687. <br />   fusions: Beta-glucuronidase sensitive versatile gene fusion        Schmidt, W., Boomgaarden, B., Ahrens, V. (1996). Reduction   marker higher plants. EMBO J. 6, 3901&#226;&#8364;&#8220;3907.                                root iron Plantago lanceolata recovery Fe deficiency. <br /> Kobayashi, T., Nakayama, Y., Itai, R.N., Nakanishi, H., Yoshihara, T.,          Physiol. Plant. 98, 587&#226;&#8364;&#8220;593. <br />   Mori, S., Nishizawa, N.K. (2003). Identification novel cis-          Takagi, S., Nomoto, K., Takemoto, T. (1984). Physiological aspect <br />   acting elements, IDE1 IDE2, barley IDS2 gene promoter              mugineic acid, possible phytosiderophore graminaceous <br />   conferring iron-deficiency-inducible, root-specific expression het-        plants. J. Plant Nutr. 7, 469&#226;&#8364;&#8220;477. <br />   erogeneous tobacco plants. Plant J. 36, 780&#226;&#8364;&#8220;793.                            Tarantino, D., Petit, J.M., Lobreaux, S., Briat, J.F., Soave, C., Krysan, P.J., Young, J.C., Sussman, M.R. (1999). T-DNA                Murgia,  (2003). Differential involvement IDRS cis-element   insertional mutagen Arabidopsis. Plant Cell 11, 2283&#226;&#8364;&#8220;2290.                 developmental environmental regulation AtFer1 ferritin <br /> Lahner, B., Gong, J., Mahmoudian, M., Smith, E.L., Abid, K.B.,                  gene Arabidopsis. Planta 217, 709&#226;&#8364;&#8220;716. <br />   Rogers, E.E., Guerinot, M.L., Harper, J.F., Ward, J.M., McIntyre,           Thimm, O., Essigmann, B., Kloska, S., Altmann, T., Buckhout, <br />   L., Schroeder, J , Salt, D.E. (2003). Genomic scale profiling        T.J. (2001). Response Arabidopsis iron deficiency stress   nutrient trace elements Arabidopsis thaliana. Nat. Biotechnol.         revealed microarray analysis. Plant Physiol. 127, 1030&#226;&#8364;&#8220;1043. <br />   21, 1215&#226;&#8364;&#8220;1221.                                                              Thomine, S., Wang, R., Ward, J.M., Crawford, N.M., Schroeder, <br /> Ling, H.-Q., Bauer, P., Bereczky, , Keller, B., Ganal, M. (2002).         J  (2000). Cadmium iron transport members plant metal <br />   tomato fer gene encoding bHLH protein controls iron-uptake              transporter family Arabidopsis homology Nramp genes. <br />   responses roots. Proc. Natl. Acad. Sci. USA 99, 13938&#226;&#8364;&#8220;13943.               Proc. Natl. Acad. Sci. USA 97, 4991&#226;&#8364;&#8220;4996. <br /> Long, J , Barton, M.K. (1998). development apical                Toledo-Ortiz, G., Huq, E., Quail, P.H. (2003). Arabidopsis <br />   embryonic pattern Arabidopsis. Development 125, 3027&#226;&#8364;&#8220;3035.                 basic/helix-loop-helix transcription factor family. Plant Cell 15, 1749&#226;&#8364;&#8220; <br /> Long, J , Moan, E , Medford, J , Barton, M.K. (1996).               1770. <br />   member KNOTTED class homeodomain proteins encoded              Varotto, C., Maiwald, D., Pesaresi, P., Jahns, P., Salamini, F.,   STM gene Arabidopsis. Nature 379, 66&#226;&#8364;&#8220;69.                               Leister, D. (2002). metal ion transporter IRT1 necessary iron <br /> Marschner, H., Romheld, V., Ossenberg-Neuhaus, H. (1982).                   homeostasis efficient photosynthesis Arabidopsis thaliana. <br />   Rapid methods measuring changes pH reducing processes              Plant J. 31, 589&#226;&#8364;&#8220;599. <br />   roots intact plants.  Pflanzenphysiol. 105, 407&#226;&#8364;&#8220;416.             Vert, G., Briat, J.F., Curie, C. (2001). Arabidopsis IRT2 gene <br /> McMaster, G.K., Carmichael, G.G. (1977). Analysis single-            encodes root-periphery iron transporter. Plant J. 26, 181&#226;&#8364;&#8220;189. <br />   double-stranded nucleic acids polyacrylamide agarose gels            Vert, G., Grotz, N., Dedaldechamp, F., Gaymard, F., Guerinot, M.L., <br />   using glyoxal acridine orange. Proc. Natl. Acad. Sci. USA 74,             Briat, J.F., Curie, C. (2002). IRT1, Arabidopsis transporter <br />   4835&#226;&#8364;&#8220;4839.                                                                    essential iron uptake soil plant growth. Plant Cell <br /> Negishi, T., Nakanishi, H., Yazaki, J., Kishimoto, N., Fujii, F.,               14, 1223&#226;&#8364;&#8220;1233. <br />   Shimbo, K., Yamamoto, K., Sakata, K., Sasaki, T., Kikuchi, S.,              Vert, G , Briat, J.F., Curie, C. (2003). Dual regulation   Mori, S., Nishizawa, N.K. (2002). cDNA microarray analysis             Arabidopsis high-affinity root iron uptake local long- <br />   gene expression Fe-deficiency stress barley suggests           distance signals. Plant Physiol. 132, 796&#226;&#8364;&#8220;804. <br />   polar transport vesicles implicated phytosiderophore secretion     Voronova, , Baltimore, D. (1990). Mutations disrupt DNA <br />   Fe-deficient barley roots. Plant J. 30, 83&#226;&#8364;&#8220;94.                             binding dimer formation E47 helix- loop-helix protein map <br /> Petit, J.M., Briat, J.F., Lobreaux, S. (2001a). Structure               distinct domains. Proc. Natl. Acad. Sci. USA 87, 4722&#226;&#8364;&#8220;4726. <br />   differential expression members Arabidopsis thaliana     Wang, Y.H., Garvin, D.F., Kochian, L.V. (2002). Rapid induction   ferritin gene family. Biochem. J. 359, 575&#226;&#8364;&#8220;582.                               regulatory transporter genes response phosphorus, potas- <br /> Petit, J.M., van Wuytswinkel, O., Briat, J.F., Lobreaux, S.                 sium, iron deficiencies tomato roots. Evidence cross talk <br />   (2001b). Characterization iron-dependent regulatory sequence            root/rhizosphere-mediated signals. Plant Physiol. 130, 1361&#226;&#8364;&#8220; <br />   involved transcriptional control AtFer1 ZmFer1 plant            1370. <br />   ferritin genes iron. J. Biol. Chem. 276, 5584&#226;&#8364;&#8220;5590.                      Wintz, H., Fox, T., Wu, Y.Y., Feng, V., Chen, W., Chang, H.S., Zhu, T., <br /> Riechmann, J.L., et al. (2000). Arabidopsis transcription factors:              Vulpe, C. (2003). Ex <br /> </body></html>