<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Jan. 1999, p. 342&#226;&#8364;&#8220;352                                                                          Vol. 19,  1 <br /> 0270-7306/99/$04.0010 <br /> Copyright &#194;&#169; 1999, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />             Functional Domains Rsp5 Ubiquitin-Protein Ligase <br />                                GUANGLI WANG, JOYCE YANG,                       JON M. HUIBREGTSE* <br />              Department Molecular Biology Biochemistry, Rutgers University, Piscataway, New Jersey 08855-1059 <br />                     Received 18 August 1998/Returned modification 16 September 1998/Accepted 23 September 1998 <br />  <br />              RSP5, essential gene Saccharomyces cerevisiae, encodes hect domain E3 ubiquitin-protein ligase. Hect <br />           E3 proteins proposed consist broad functional domains: conserved catalytic carboxyl- <br />           terminal domain approximately 350 amino acids  hect domain) large, nonconserved amino- <br />           terminal domain containing determinants substrate specificity. report mapping minimal <br />           region Rsp5 necessary essential vivo function, minimal region necessary stably interact           substrate Rsp5 (Rpb1, large subunit RNA polymerase II), finding hect domain,           itself, sufficient formation ubiquitin-thioester intermediate. Mutations hect domain           affect ability form ubiquitin-thioester catalyze substrate ubiquitination abrogate vivo <br />           function, strongly suggesting ubiquitin-protein ligase activity Rsp5 intrinsically linked           essential function. amino-terminal region Rsp5 contains WW domains C2 calcium-binding <br />           domain. WW domains required essential vivo function, C2 domain            requirements Rpb1 binding ubiquitination lie region required vivo function. <br />            results support domain model hect E3 function indicate WW domains <br />           play role recognition substrates Rsp5, including related essential <br />           function. addition, haploid yeast strains bearing complete disruptions           hect E3 genes yeast, designated HUL4 (YJR036C) HUL5 (YGL141W), viable. <br />  <br />  <br />    number proteins recognized substrates modi-                yl-terminal hect domain (homologous E6-AP carboxyl <br /> fication ubiquitination increased dramatically recent             terminus) approximately 350 amino acids (14, 37). Hect E3s <br /> years, relatively poor understanding               appear present eukaryotes, exactly 5 encoded <br /> components ubiquitin function defining                 Saccharomyces cerevisiae genome 30 far <br /> controlling substrate specificity. classes pro-           identified mammalian species. Based biochemical char- <br /> teins recognized cooperate catalyzing protein ubiq-             acterization E6-AP, proposed obligatory <br /> uitination,  E1 ubiquitin-activating enzyme                intermediate ubiquitination reaction catalyzed hect family E2 ubiquitin-conjugating enzymes,                E3s ubiquitin-thioester formed thiol characterized (37). group activities, E3 ubiq-             absolutely conserved cysteine hect domain uitin-protein ligases, characterized, diver-          terminal carboxyl group ubiquitin (36). E3 sity activities, terms composition mechanisms            &#226;&#8364;&#339;charged&#226;&#8364;? ubiquitin cascade ubiquitin-thioester action, unknown. clear,  E3 activities             transfers, ubiquitin transferred active-site <br /> play major role defining substrate specificity, E1             cysteine E1 enzyme active-site cysteine E2 E2 activities,  generally sufficient direct             finally, hect E3, catalyzes isopeptide bond <br /> ubiquitination biologically relevant substrates.                         formation ubiquitin substrate. E3    largest known family E3 proteins, hect E3s,              thought recharged ubiquitin bound discovered characterization human E6-AP (15, 35,                 substrate catalyze ligation multiple ubiq- <br /> 36). interaction E6-AP human papillomavirus                 uitin moieties substrate conjugation (HPV) E6 protein cervical cancer-associated HPV types                lysines substrate lysine residues previ- <br /> causes E6-AP associate ubiquitinate p53 tumor               ously conjugated ubiquitin molecules. resulting multiubiq- <br /> suppressor, lines evidence suggest           uitinated substrate recognized degraded 26S <br /> important component cell-immortalizing activity               proteasome. <br /> cancer-associated HPVs (13). natural (HPV E6-                        hect E3s range mass 92 500 kDa independent) substrates E6-AP known,                 generally hect domain common. pattern <br /> proposed lack expression maternal allele                  similarity, structure-function analyses human <br /> E6-AP  known UBE3A) human brain likely                E6-AP (16), suggested large highly variable ami- <br /> cause Angelman syndrome, severe neurologic disorder                    terminal domains proteins function defining <br /> (20, 26). suggests E6-AP-dependent ubiquitination              substrate specificity.  simple model func- proteins brain, particular, hip-               tion hect E3s consist broad functional <br /> pocampal Purkinje neurons (1), critical normal brain             domains: large amino-terminal domain contains deter- <br /> function.                                                                   minants substrate specificity carboxyl-terminal hect <br />    family E6-AP-related proteins defined carbox-             domain, catalyzes multiubiquitination associated sub- <br />                                                                             strates. test refine model, explored <br />                                                                             function substrate specificity hect E3 encoded   * Corresponding author. Mailing address: Department Molecular          RSP5 gene S. cerevisiae. <br /> Biology Biochemistry, Rutgers University, Piscataway, NJ 08855-            RSP5  known NPI1 MDP1) essential gene <br /> 1059. Phone: (732) 445-0938. Fax: (732) 445-4213. E-mail: huibregt          isolated types genetic screens, <br /> @waksman.rutgers.edu.                                                       including suppressor mutations SPT3 (46b). SPT3 <br />  <br />                                                                       342 <br />  VOL. 19, 1999                                                                                   FUNCTIONAL DOMAINS Rsp5                         343 <br />  <br />  <br /> encodes component SAGA (Spt/Ada/Gcn5/acetyl-                                         MATERIALS METHODS <br /> transferase) complex (8, 34), proposed affect minimum aspects RNA polymerase II (pol II)                   Yeast strains plasmids. Yeast strain FY56 (MATa his4-912 DR5 lys2-128D <br />                                                                       ura3-52) isogenic rsp5-1 mutant strain FW1808 obtained Fred <br /> transcription: function TATA binding protein               Winston (Harvard Medical School). RSP5 rsp5-1 open reading frames <br /> (Spt15) activity Gcn5 histone acetyltransferase.       (ORFs) amplified PCR genomic DNAs FY56 FW1808, previously reported largest subunit RNA polII,         respectively, cloned pGEM-1 vector. independent clones                                                                       sequenced compared. rsp5-1 ORF subcloned Rpb1, substrate Rsp5 (17). biological rele-         pYES2 pGEX-6p-1 vectors, previously described wild-type <br /> vance Rpb1 ubiquitination clear, related       (wt) RSP5 ORF, active-site Cys Ala (C  mutant (<span id='am-42' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-43' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-45' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> 777), SPT3/RSP5 genetic interaction. addition, study         CD6 mutant. Additional mutated RSP5 genes  Fig. 3) generated                                                                       PCR using gene-specific primers. 59 PCR primers, cases, encoded showed large subunit human polII (polII-LS)            hemagglutinin (HA) epitope, cases, PCR products subcloned <br /> subject ubiquitination degradation response UV           version pYES2 initiating ATG HA epitope cloned <br /> irradiation, suggesting ubiquitin-mediated degradation        upstream multicloning sites (pYES-HA). Mutants M, N, O utilized <br />                                                                       primers encoding indicated mutations, introduced SalI restriction polII-LS involved response DNA damage (4,            site. Multiple clones construct analyzed vitro vivo <br /> 32).                                                                  assays, clones partially sequenced confirm introduction    RSP5 shown genetically affect activity     mutations cloning junctions. constructs shown Fig. 3,                                                                       pYES2 vector, transformed FY56 FW1808 strains. Plasmids plasma membrane-associated permeases, including               encoding Rpb1 <span id='am-11' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-12' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span>glutathione</span> S-transferase (GST)-carboxy-terminal  <br /> Gap1 (general amino acid permease), Fur4 (<span id='am-54' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> permease),           main (CTD) fusion described previously (17). <br /> maltose permease, plasma membrane H1-ATPase (6,                  rsp5D::LEU2 mutant strains derived W303 diploid strain <br /> 11, 23), ubiquitination Fur4 Gap1 results         (leu2-3112 ade2-1 his3-11 trp1-1 can1-100). 4.5-kbp fragment chromosome <br />                                                                       V spanning RSP5 locus amplified PCR cloned pBlue- regulation (9, 42). Mutations RSP5             script II KS1 vector. Plasmid pBlueDR5 generated replacing com- <br /> shown alter mitochondrial-cytoplasmic protein distribu-        plete RSP5 ORF plasmid LEU2 gene. linear 3.5-kbp disrupting <br /> tion Mod5 protein, tRNA modification enzyme             fragment isolated pBlueDR5 digestion HindIII NotI.                                                                       DNA fragment transformed diploid W303 cells, Leu1 transfor- <br /> functions cellular compartments (49). Al-            mants isolated. deletion chromosomal copy RSP5 gene effect Rsp5 pathways            introduction LEU2 RSP5 locus confirmed PCR. <br /> indirect, broad range cellular processes affected RSP5      Sporulation heterozygous diploid (GW001) gave expected 2:2 ratio                                                                       viable inviable spores essential gene. GW001 transformed suggests Rsp5 protein multiple diverse substrates.       centromere- URA3-based plasmid, pRS416-RSP5, harboring 4.5-kbp <br />    primary structure Rsp5 reveals types domains         genomic RSP5 fragment, selecting growth plates lacking uracil leu- amino-terminal region protein  Fig. 1).         cine. Haploid spores bearing rsp5D::LEU2 allele pRS416-RSP5 <br />                                                                       plasmid isolated yield strains GW003 (MATa rsp5D::LEU2 ade2-1 his3-11 C2 domain (amino acids 3 140),       trp1-1 can1-100[pRS416-RSP5]) GW004, equivalent MATa strain. variety proteins, including protein kinase C, synapto-           Centromere- TRP1-based plasmids pRS414-Gal  -B, -D gen- <br /> tagmin, phospholipase C, p120ras-GAP (33). C2 domains             erated subcloning NgoM1-NotI fragment pYES2-HA-Rsp5 (construct shown interact membrane phospholipids,               Fig. 3), pYES2-HA-Rsp5C (construct B), pYES2-HA-Rsp5DC2 <br />                                                                       (construct D) sites pRS414, yielding pRS414-Gal  -B, -D. <br /> inositol polyphosphates, proteins. interaction C2          Expression respective Rsp5 proteins plasmids control <br /> domains ligands  cases, dependent         GAL1 promoter. pRS414-RSP5 plasmid contained 4.5-kbp <br /> regulated Ca21. membrane-associated sub-             genomic RSP5 fragment, expression Rsp5 control                                                                       natural RSP5 promoter. Plasmids pRS414-Gal  pRS414-Gal-B, pRS414- <br /> strates Rsp5 identified, conceivable      Gal-D, pRS414-RSP5 transformed yeast strains GW003 C2 domain involved targeting proteins,          GW004, followed 5-fluoroorotic acid (5-FOA) counterselection localizing Rsp5 plasma membrane directly medi-           pRS416-RSP5 plasmid. GW015 GW016 MATa   respectively, <br />                                                                       containing pRS414-Gal plasmid. GW017 GW018 MATa   <br /> ating interaction membrane-associated substrates,            respectively, containing pRS414-Gal-D plasmid, GW013 GW014 Fur4 Gap1.                                                MATa   respectively, containing pRS414-RSP5 plasmid. 5-FOA-resis- <br />    Rsp5 contains WW domains, spanning amino                tant colonies obtained pRS414-Gal-B  C mutant) <br />                                                                       transformant. <br /> acids 231 418. WW domains protein-protein interaction             HUL4 (YJR036C) HUL5 (YGL141W) ORFs, located chromo- <br /> modules affinity proline-rich sequences,        somes X VII, respectively, isolated PCR FY56 genomic DNA consensus binding site containing PPxY sequence (PY             cloned pBluescript. Heterozygous chromosomal disruptions motif) (7, 45). WW domains consist 38 40 amino acids,           ORFs individually W303 diploid background, described <br />                                                                       RSP5. Haploid spores containing hul4D::LEU2 hul5D:: absolutely conserved tryptophan residues, form         LEU2 allele fully viable. stranded antiparallel b-sheet hydrophobic binding           Protein expression vitro biochemical assays. GST fusion proteins pocket PPxY ligand (24). Functionally, WW              expressed Escherichia coli standard methods affinity purified gluta- <br />                                                                       thione-Sepharose (Pharmacia). Rsp5 proteins expressed pGEX-6p-1 <br /> mains somewhat analogous SH3 domains              vector cleaved GST using PreScission protease (Pharmacia) affinities proline-rich ligands, cases,     manufacturer-recommended conditions, proteins used ubiq- <br /> WW SH3 domains compete binding                uitin-thioester ubiquitination reactions. Ubiquitin-thioester reactions                                                                       performed using vitro-translated proteins  Fig. 7), described <br /> ligands vitro (3, 44). Given proposed function pro-       previously (14). E1 ubiquitin-activating enzyme Arabidopsis thaliana Ubc8 <br /> tein-protein interaction modules, WW domains clearly          protein prepared described previously (35). vitro ubiquitin-thioester <br /> strong candidates sites Rsp5 substrate interaction.             assays utilized 32P-labeled ubiquitin, prepared pGEX-2TK-ubiquitin (35) <br />                                                                       labeled vitro [g-32P]<span id='am-23' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-24' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span>ATP</span> heart muscle kinase (Sigma).    performed structure-function analysis Rsp5         ubiquitin cleaved GST thrombin, thrombin heat <br /> order understand domain structure vitro             inactivated incubation 65&#194;&#176;C 10 min. vitro translation reactions biochemical activities Rsp5 related essential vivo   carried coupled transcription-translation reactions using Promega TNT <br />                                                                       reagents [35S]methionine. Rpb1-DCTD coupled vitro tran- <br /> function. determined minimal region                scription-translation using template pYES2-Rbp1 plasmid DNA mains Rsp5 necessary complementation rsp5-1             treated BsiWI restriction endonuclease described previously <br /> temperature-sensitive mutant vivo regions necessary        (17). interaction ubiquitination Rpb1 vitro. addition,        cell yeast cell extracts immunoblotting analyses  Fig. 4A B) <br />                                                                       prepared method Silver et al. (41). Primary antibodies hect domain, itself, sufficient forma-     anti-HA rabbit polyclonal antibody (Santa Cruz Biotechnology) anti- <br /> tion ubiquitin-thioester intermediate.                         Rsp5 mouse monoclonal antibody (17). Horseradish peroxidase-linked second- <br />  344       WANG ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  <br /> ary antibodies chemiluminescent reagents obtained Du Pont <br /> NEN. <br />   Rsp5/Rpb1 binding assays  Fig. 6) conducted previously described <br /> (17). Briefly, reaction mixture contained 125 ml 25 mM Tris (pH 8.0), 125 <br /> mM NaCl, 0.1% Nonidet P-40, 100 ng GST fusion protein bound 10 ml <span id='am-49' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-50' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glutathione</span>-Sepharose. microliters vitro-translated protein added reaction mixture. Reaction mixtures rotated 2 h 4&#194;&#176;C, Sepharose beads washed times 500 ml buffer contain- <br /> ing 100 mM Tris, 100 mM NaCl, 1% Nonidet P-40. Sodium dodecyl sulfate- <br /> polyacrylamide gel electrophoresis (SDS-PAGE) loading buffer added di- <br /> rectly beads heated 95&#194;&#176;C 5 min, proteins analyzed SDS-PAGE autoradiography. <br />  <br />  <br />                                  RESULTS <br />  <br />    protein encoded rsp5-1 allele impaired ubiquitin-thioester formation catalysis substrate ubiq- <br /> uitination. S. cerevisiae rsp5-1 mutant (strain FW1808),                           FIG. 1. Schematic Rsp5 protein C2 domain, WW domains 1 <br /> isolated F. Winston coworkers (cited references 14                        3, hect domain indicated. sequence carboxyl-terminal 100 17), grows 30&#194;&#176;C rich media doubling time                      amino acid residues Rsp5 (amino acids 710 809) shown. Amino acids <br />                                                                                     marked asterisks residues highly conserved hect domain <br /> slightly longer isogenic RSP5 mutant strain                        family proteins. rsp5-1 mutation (L S, amino acid 733), active-site <br /> (FY56) severely impaired growth elevated tem-                         mutation (C  amino acid 777), residues deleted CD6 mutation <br /> perature. Characterization rsp5-1 allele protein                     indicated. <br /> product undertaken order understand biochem- <br /> ical basis mutant phenotype. rsp5-1 RSP5 ORFs amplified PCR genomic DNAs respec- <br /> tive strains, multiple independent clones isolated                     abrogates ubiquitination p53 (14). rsp5-1-encoded <br /> sequenced. difference rsp5-1 sequence                          protein impaired formation ubiquitin-thioester wild-type RSP5 sequence T C transition                          intermediate relative CD6 mutant. small <span id='am-34' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-35' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> 2198 rsp5-1 ORF, resulting Leu Ser                        monoubiquitinated rsp5-1-encoded protein detected alteration amino acid 733 (Fig. 1). Leu733 hect                    DTT heat treatment, suggesting retains ca- <br /> domain, amino terminal active-site cysteine residue                       pacity catalyze isopeptide-linked ubiquitination. temper- <br /> 777. Leu733 represents conserved hydrophobic residue                              ature dependence thioester formation observed hect domains, leucine phenylalanine                       rsp5-1-encoded protein vitro, impaired nearly cases.                                                                temperatures tested, 4 37&#194;&#176;C (data shown). <br />    protein encoded rsp5-1 allele expressed                         previously reported largest subunit RNA <br /> GST fusion protein pGEX-6p vector, allows                          polII, Rpb1, substrate Rsp5 (17). purified Rsp5 <br /> cleavage GST portion molecule highly                           proteins assayed ability catalyze ubiq- <br /> specific protease (PreScission Protease; Pharmacia). Wt Rsp5,                       uitination Rpb1 vitro (Fig. 2B). wt Rsp5 protein active-site Cys Ala (C  mutant, previously                         efficiently ubiquitinated wt Rpb1 did ubiquitinate Rpb1 <br /> described CD6 mutant, carboxyl-terminal                            CTD, binding site Rsp5, deleted. amino acids truncated, expressed manner.                       C protein, CD6 protein, rsp5-1-encoded protein <br /> Figure 2A shows result vitro ubiquitin-thioester                       inactive catalyzing ubiquitination Rpb1.  <br /> assay utilizing purified Rsp5 proteins partially                   termine inability mutant protein ubiquitinate <br /> purified recombinant E1 (human) E2   thaliana Ubc8)                           Rpb1 inability bind Rpb1, GST-Rsp5 fusion <br /> proteins 32P-labeled ubiquitin. basis assay                       proteins assayed ability bind Rpb1 present thioester adducts formed radiolabeled ubiquitin                        crude yeast cell extracts. Figure 2C shows GST enzyme E1, E2, E3 stable analyzed                            fusion proteins (wt, C  CD6, Rsp5-1) equally ca- <br /> SDS-PAGE performed absence reducing agent,                          pable binding Rpb1.  results suggest adducts disrupted analyzed standard                        biochemical basis rsp5-1-encoded phenotype SDS-PAGE conditions (high reducing agent concentration).                            defect ability enzyme form ubiquitin-thioester reported previously (14), difficult detect ubiquitin-                 intermediate catalyze substrate ubiquitination, thioester intermediate wild-type Rsp5 protein                      general defect recognition binding sub- propensity protein catalyze isopeptide-linked                     strates. <br /> (dithiothreitol [DTT] insensitive) ubiquitination itself                     hect domain WW domains 2 3 required intramolecular transfer cysteine lysine residues                        essential vivo function Rsp5. order determine protein. unclear                     minimal domain requirements complementation lying biological relevance vitro self-ubiquitination                      rsp5-1 allele, set mutated Rsp5 proteins expressed reaction, dependent active-site cysteine                        rsp5-1 mutant yeast strain FW1808, isogenic RSP5 <br /> shown  correlates ability catalyze intermo-                       mutant strain FY56, control GAL1 promoter. <br /> lecular ubiquitination substrates.                                               Figure 3 illustrates set truncated mutated Rsp5 <br />    CD6 mutant forms DTT-sensitive ubiquitin-thioester                         proteins generated. constructs encoded HA  contrast wild-type protein, does catalyze                        antibody epitope 59 end, construct E. Mutations <br /> isopeptide-linked self-ubiquitination, suggesting ex-                      disrupted individual WW domains altered second  <br /> treme carboxyl terminus protein important                         served tryptophan nearby conserved proline residue <br /> ability catalyze isopeptide-linked ubiquitin conjugates.                    WxxP FVDA, shown Fig. 3. substitutions supported observation analogous mutation                      originally suggested based nuclear magnetic res- <br /> human E6-AP does affect ubiquitin-thioester formation                           onance structure YAP WW domain, substitu- <br />  VOL. 19, 1999                                                                                                    FUNCTIONAL DOMAINS Rsp5                        345 <br />  <br />  <br />  <br />  <br />    FIG. 2.   Ubiquitin-thioester assay using 32P-labeled ubiquitin (ub) purified Rsp5 proteins  wt, C mutant protein; rsp5-1-encoded [ts] mutant <br /> protein, CD6 mutant protein). presence absence recombinant E1 E2 proteins indicated, migration positions ubiquitin adducts indicated. samples left (lanes 1 7) analyzed SDS-PAGE using loading buffer lacked DTT, samples right analyzed DTT-containing loading buffer (lanes 8 14). Migration positions molecular mass standards  kilodaltons) indicated right. (B) Rpb1 <br /> ubiquitination assay using purified Rsp5 proteins vitro-translated, 35S-labeled Rpb1 (lanes 1 5) Rpb1 CTD deleted (Rpb1-DCTD, lanes 6 10). <br /> (C) Binding Rpb1, present total yeast cell extract (prepared glass bead lysis), GST-Rsp5 fusion proteins (lanes 2 5) GST&#226;&#8364;&#8220;E6-AP negative control <br /> (lane 6). Lane 1 represents 20% input used binding reaction mixture. Rpb1 detected immunoblotting anti-CTD antibody. <br />  <br />  <br />  <br />  <br /> tions, individually, shown render domain                             medium, total cell extracts analyzed protein ex- <br /> completely inactive terms ligand (5, 24).                                       pression immunoblotting using anti-HA anti- <br />    confirm vivo expression altered Rsp5 proteins,                        Rsp5 antibodies (Fig. 4A B). proteins expressed FY56 transformants grown galactose-containing                             detectable levels, level expression exogenous <br />  346     WANG ET AL.                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br /> HA-tagged wt Rsp5 roughly equivalent level endogenous Rsp5 protein (protein C, CD6 mutant form, shown results experiment expressed level similar wt Rsp5). Expression protein E, does contain HA epitope, confirmed using anti-Rsp5 antibody (Fig. 4B). Proteins K L detected anti-Rsp5 antibody, HA epitope encoded ORF, suggesting epitope removed proteolysis posttranslation- <br /> ally. Protein did react anti-Rsp5 antibody,  <br /> sistent epitope mapping experiments indicated monoclonal antibody recognizes epitope spanning region containing WW domains 2 3. level expres- <br /> sion WW domain mutants M, N, O similar wt Rsp5;  proteins N O consistently <br /> showed pattern higher-molecular-weight species represent ubiquitinated intermediates proteins. expression pattern mutant proteins rsp5-1 <br /> transformants identical seen RSP5  <br /> ground (data shown). <br />    expression exogenous RSP5 genes  <br /> pressed  growth dextrose), FY56 (RSP5) FW1808 (rsp5-1) transformants grew similarly respec- <br /> tive untransformed strains;  FY56 transformants <br /> grew 30 37&#194;&#176;C, FW1808 transformants <br /> grew 30&#194;&#176;C 37&#194;&#176;C. expression induced  growth galactose-containing medium), FY56 trans- <br /> formants, K L (Discussed , grew similarly control strain (vector  30 37&#194;&#176;C. ability various constructs complement rsp5-1 tem- <br /> perature-sensitive growth defect tested assaying growth rsp5-1 transformants 37&#194;&#176;C galactose  <br /> taining agar plates. Expression wild-type HA-tagged             FIG. 3. Schematic mutated Rsp5 proteins  O; numbering corre- <br /> Rsp5 protein   fully complemented temperature-sensi-          sponds amino acids retained mutated proteins) summary tive growth defect, vector did        abilities complement rsp5-1-encoded growth defect 37&#194;&#176;C ga- active-site C mutant (B) CD6 mutant (C)           <span id='am-6' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Lactose" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lactose</span>. proteins E contained HA epitope amino- <br />                                                                     terminal end. WW consensus sequence (7) shown relative se- <br /> able complement growth defect. results strongly        quences WW domains 1, 2, 3. lowercase h consensus sequence <br /> suggest ubiquitin-protein ligase activity Rsp5       represents hydrophobic residues, lowercase t represents turn-like intrinsically linked essential vivo function.             polar position (45). WW domain mutant proteins (M, N, O) <br />    Constructs D, E, F, G, deletions increasing          contained sequence FVDA place WxxP sequence, indicated. <br /> portions amino-terminal domain, complemented rsp5-1 mutation, variable efficiency. Protein D, entire C2 domain deleted, complemented growth          L, share characteristics: disrupted hect  <br /> defect efficiency similar length Rsp5;      main intact WW domains. Expression proteins K L,  protein E, sequences  including,        contain amino-terminal domain sequences, inhib- <br /> WW domain 1 deleted grew significantly slower 37&#194;&#176;C. Trun-        ited growth severely inhibited growth cation WW domain 1 (F) restored nearly comple-          RSP5 strain. speculate growth-inhibitory effect mentation activity, truncation WW domain 1        mutant proteins titration substrates away spacer region domains 1 2 supported              endogenous rsp5-1-encoded protein, resulting growth reduced efficiency (G). Proteins amino-       lack degradation substrate(s) related essential <br /> terminal truncations included WW domain 2 (proteins           function Rsp5. fact region spanning WW <br /> H  did support growth 37&#194;&#176;C, did         domains required rsp5-1 complementation carboxyl-terminally truncated proteins (C, J, K, L). ad-      dominant-inhibitory effect consistent hypothesis. <br /> dition, WW domain 1 mutant fully complemented                  C2 domain required essential vivo <br /> rsp5-1 growth defect (protein M), mutation        function Rsp5. somewhat surprising C2  <br /> WW domain 2 3 completely abrogated complementation               main dispensable complementation rsp5-1 mu- <br /> activity (proteins N O, respectively). results indicate   tation, plasma membrane-associated substrates region amino-terminal domain includes          Rsp5 identified. led question WW domains 2 3, complete functional            protein encoded rsp5-1 allele playing role carboxyl-terminal hect domain, required essential       supporting function proteins C2 deleted (proteins <br /> function Rsp5.                                                   D, E, F, G) complementation assay described  <br />    Interestingly, subgroup noncomplementing mutant         example, conceivable 37&#194;&#176;C temperature- <br /> proteins actually inhibited growth rsp5-1 transformants      inactivated rsp5-1 mutant protein, heterodimerization galactose-containing medium permissive temperature        protein C2 deleted, serve localize (data shown) dominant-negative             protein C2 deleted plasma membrane, mutant proteins. set included mutant proteins B, C, J, K,      contact ubiquitinate substrates. idea  VOL. 19, 1999                                                                                           FUNCTIONAL DOMAINS Rsp5                     347 <br />  <br />  <br />  <br />  <br />    FIG. 4.   Anti-HA immunoblot total cell extracts indicated FY56 transformants grown galactose-containing medium. vec., vector. (B) <br /> Anti-Rsp5 immunoblot extracts. migration position endogenous Rsp5 protein indicated. (C) Growth FW1808 transformants galactose-containing agar plates 37&#194;&#176;C. <br />  <br />  <br />  <br /> suggested experiments indicate Rsp5,                    tion results, 5-FOA-resistant transformants obtained human E6-AP, form homodimers certain condi-                           galactose-containing plates plasmids expressing protein tions (2a). determined Rsp5 protein                     D, wild-type RSP5 genomic fragment, C2 deleted (protein D) function sole source                   construct B, C mutant (Fig. 5A). expected, Rsp5 protein cell.                                                    D transformants mating type grow <br />    chromosomal deletion complete RSP5 ORF                          galactose glucose carbon source (Fig. 5B). diploid strain (W303). Tetrad dissections                    Immunoblotting confirmed transformant D rsp5D <br /> sistent RSP5 essential gene, previously                     background expresses form Rsp5 C2  <br /> ported (48, 49). Viable rsp5D::LEU2 haploids obtained                   main deleted (Fig. 5C). conclude C2 <br /> sporulation heterozygous diploid transformation                    domain required essential Rsp5 function URA3-based centromere-containing plasmid expressing                      standard growth conditions. <br /> wild-type RSP5 promoter. strain                         WW domain 2 essential mediating interaction transformed TRP1-based plasmid expressing                     Rsp5 Rpb1. previously showed Rsp5 binds struct (wt HA-Rsp5), B  C mutant), D  C2                    ubiquitinates Rpb1, largest subunit RNA polII domain deleted) control galactose-inducible pro-                     CTD Rpb1 necessary sufficient stable binding <br /> moter wild-type RSP5 expressed pro-                        Rsp5 (17). determine regions Rsp5 critical <br /> moter. URA3-based RSP5 plasmid selected                            binding Rpb1, subset Rsp5 mutant proteins   5-FOA. accord rsp5-1 complementa-                      D, E, G  L O) translated vitro rabbit <br />  348      WANG ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5.   Drsp5::LEU2 haploid cells transformed URA3-based plasmid expressing wt RSP5 TRP1-based plasmid expressing HA-tagged <br /> wt Rsp5 (Gal , C mutant protein (Gal-B), protein C2 domain deleted (Gal-D) control GAL1 promoter wt Rsp5 control promoter (RSP5). transformants streaked galactose-containing agar plates containing uracil 5-FOA select loss URA3 plasmid. TRP1-based plasmids, expressing C mutant protein (B), support continued growth cells conditions. (B) Galactose dextrose plates RSP5 Gal Gal-D transformants panel  upper lower sectors mating types, respectively. (C) Total cell <br /> extracts genomic RSP5 Gal Gal-D transformants, grown galactose, analyzed immunoblotting using anti-Rsp5   anti-HA   antibody. mating types analyzed. <br />  <br />  <br />  <br />  <br /> reticulocyte lysate (Fig. 6A) assayed ability                     deletion hect domain (L) resulted binding GST- bind GST-CTD (Fig. 6B). Paralleling importance                        CTD efficiency similar wild-type Rsp5  . vivo rsp5-1 complementation assay, deletion C2                            Binding Rpb1 Rsp5 analyzed expressing domain WW domain 1 did affect CTD binding (D, E, G,                           subset Rsp5 mutants GST fusion proteins (Fig. 6C) L), deletion WW domain 2 (H                             assaying binding vitro-translated Rpb1 (Fig. 6D). <br /> abolished CTD binding. Likewise, disruption WW domain 1                           results similar described  mutation did affect CTD binding (M), disruption                         assay, disruption WW domain 3 did significantly WW domain 2 completely abolished binding (N). Mutation                            affect Rpb1 binding. results  regardless of WW domain 3 (protein O) consistently reduced binding                           Rpb1 vitro translated present crude yeast <br /> approximately 50%. shown previously (17), binding Rsp5                         cell extracts (data shown), consistent previous CTD independent hect domain, complete                           findings Rsp5 bind Rpb1 free  VOL. 19, 1999                                                                                             FUNCTIONAL DOMAINS Rsp5                        349 <br />  <br />  <br />  <br />  <br />                                                                                     FIG. 7. Ubiquitin-thioester assay. wt C mutant hect domains                                                                                  Rsp5 (amino acids 456 809) vitro translated rabbit reticulocyte <br />                                                                                  lysate incubated partially purified recombinant E1 (human) E2   <br />                                                                                  thaliana UBC8) proteins. Products subjected SDS&#226;&#8364;&#8220;10% PAGE using <br />                                                                                  loading buffer lacked contained DTT. <br />  <br />  <br />  <br />                                                                                  length protein (Fig. 2A) lie outside hect <br />                                                                                  domain. <br />                                                                                     yeast hect E3 genes, HUL4 HUL5, non- <br />                                                                                  essential genes. Database searches indicate S. cerevisiae <br />                                                                                  genes encode apparent hect E3 proteins.                                                                                  RSP5, UFD4, TOM1, HUL4, HUL5 (HUL stands                                                                                  hect ubiquitin ligase). RSP5 essential (49), UFD4                                                                                  essential (19), TOM1 (Trigger Mitosis) presumed                                                                                  essential (required G2/M cell cycle transition;  <br />                                                                                  published data; reference 46). determine phenotype <br />                                                                                  haploid strains bearing disruption previous- <br />                                                                                  ly uncharacterized hect E3 genes, designated HUL4 <br />                                                                                  (YJR036C) HUL5 (YGL141W), completely deleted <br />                                                                                  ORF individually W303 diploid background, <br />                                                                                  creating HUL4/hul4D::LEU2 HUL5/hul5D::LEU2 <br />                                                                                  heterozygous strains. created RSP5/rsp5D::LEU2 <br />    FIG. 6.   35S-labeled vitro translation products Rsp5 mutants,   diploid manner. Tetrad analysis designations corresponding proteins depicted Fig. 2, human      RSP5/rsp5D::LEU2 diploid showed expected 2:2 ratio E6-AP. (B) Translation products panel assayed binding         viable inviable spores essential gene, vi- <br /> GST-CTD immobilized glutathione-Sepharose. (C) Coomassie-stained SDS- <br /> PAGE GST-Rsp5 fusion proteins, designations corresponding proteins    able spores leucine auxotrophs. contrast, 90% <br /> depicted Fig. 2. (D) GST-Rsp5 fusion proteins (100 ng lane)      tetrads HUL4/hul4D::LEU2 panel C assayed binding 35S-labeled, vitro-translated Rpb1. GST   HUL5/hul5D::LEU2 diploids yielded viable spores <br />  fusion) GST&#226;&#8364;&#8220;E6-AP served negative controls binding.        (Fig. 8), 2:2 segregation LEU2 allele. hap- <br /> percent input translation reaction mixture used binding reaction shown.                                                       loid disruptants grew similarly parental strain 37&#194;&#176;C,                                                                                    HUL4 HUL5 essential genes  <br />                                                                                  der standard growth conditions. <br />  <br /> present multisubunit polII enzyme.                                                    DISCUSSION <br /> sults indicate C2 domain, WW domain 1, hect <br /> domain required Rpb1 binding, WW domain                           hect E3 proteins proposed consist 2 absolutely required. WW domain 3 contribute                     functional domains: large amino-terminal domain func- <br /> Rpb1 binding required.                                                tions defining substrate specificity approximately <br />    hect domain, itself, form ubiquitin-thioester                    40-kDa carboxyl-terminal domain catalyzes ubiquitination <br /> intermediate. determine hect domain, itself,                     bound substrates. results Rsp5 structure-function <br /> sufficient recognition activation E1 E2                       analysis support domain model point enzymes ubiquitin  isolated hect domain                     determinants amino-terminal domain crit- <br /> Rsp5 (amino acids 456 809) assayed ubiquitin-                         ical essential vivo function Rsp5. addition, thioester formation, equivalent active-site (C                  results strongly suggest ubiquitin-protein ligase activity <br /> mutant. Figure 7 shows hect domain sufficient                    Rsp5 intrinsically linked essential vivo function. <br /> ubiquitin-thioester formation mutation                       based finding mutations hect <br /> served cysteine hect domain abolishes activity.                      domain disrupt impair formation ubiquitin-thio- results indicate hect domain separable                       ester ability protein catalyze transfer functional domain determinants necessary inter-                     ubiquitin substrates  C777  rsp5-1, CD6 muta- <br /> action upstream components ubiquitin-thio-                       tions) abrogate vivo function Rsp5. rsp5 <br /> ester cascade  E1 E2 enzymes) lie completely                      temperature-sensitive alleles isolated Zolladek et al. hect domain. hect domain, unlike length pro-                    (49), like rsp5-1, mutations lie hect <br /> tein, did catalyze isopeptide-linked self-ubiquitination, sug-               domain disrupt conserved amino acids. speculate gesting lysines sites self-ubiquitination              mutations result defect Rsp5 ubiquitination <br />  350     WANG ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />                                                                        tion clear;  human polII-LS ubiquitinated                                                                        response UV irradiation (32), suggesting possible role                                                                        response DNA damage. Preliminary results shown <br />                                                                        DNA damage induces degradation Rpb1 yeast,                                                                         dependent Rsp5 (2a). Apparent ho- <br />                                                                        mologs Rsp5 exist mammalian cells  ,                                                                        possible mediate UV- <br />                                                                        dependent ubiquitination human polII. <br />                                                                           majority hect E3s unique amino-terminal <br />                                                                        domain sequences recognizable characterized se- <br />                                                                        quence motifs, hect E3s mamma- <br />                                                                        lian nonmammalian species  like Rsp5, contain mul- <br />                                                                        tiple   WW domains (18, 21, 29, 30, 47).                                                                        cDNAs encoding proteins remain                                                                        sequenced entirety, proteins appear                                                                        contain C2 domain extreme amino terminus. Schizo- <br />                                                                        saccharomyces pombe encodes hect E3 genes                                                                        WW domains C2 domain, pub1,   FIG. 8. Tetrad analysis RSP5/rsp5D::LEU2, HUL4/hul4D::LEU2,                                                                        targets cdc25 protein phosphatase (27). mouse-human <br /> HUL5/hul5D::LEU2 W303 diploids leucine-containing agar plates.      Nedd4 protein (human Rpf1) C2/WW-hect E3 protein                                                                        proposed regulate epithelial Na1 chan- <br />                                                                        nel ubiquitinating b g subunits (31, 43).                                                                         WW- C2/WW-hect E3s appear form distinct subgroup <br /> activity, general defect binding associa-       hect E3s. diversity functions substrates tion substrates.                                                  WW hect E3s suggests proteins perform    respect specific determinants amino-         functions diverse intracellular locations. Studies terminal region Rsp5, results clearly indicate C2      mammalian C2/WW-hect E3, Nedd4, shown pri- <br /> domain required essential function Rsp5.             marily cytoplasmic perinuclear calcium influx membrane-associated permeases shown               mediates localization plasma membrane (10, 21, 31). regulated ubiquitination Rsp5-dependent manner,            Preliminary localization studies yeast suggest Rsp5 <br /> including Fur4 (uracil permease) Gap1 (general amino               cytoplasmic nuclear (46a). <br /> acid permease) (9, 42). Based fact C2 domains            yeast hect E3s, Rsp5 contains WW domains, shown associate membrane phospholipids                 Tom1, Ufd4, Hul4, Hul5 contains previ- <br /> membrane-associated proteins, C2 domain                ously identified protein motifs amino-terminal  <br /> volved targeting group substrates, affecting      gion. speculate individual WW domains intracellular localization Rsp5 directly contacting sub-      Rsp5 involved mediating interaction strates. case, targeting membrane-       specific substrates. yeast genes encode proteins associated substrates essential function            apparent WW domains (ESS1, PRP40, TIN1, YPR152C, Rsp5. experiments necessary determine               YFL010C), contain single WW domain, <br /> effect specific Rsp5 mutations ubiquitination plasma          PRP40, contains  subject inves- <br /> membrane-associated substrates.                                        tigation WW domain proteins    contrast C2 domain, subset WW domains            hect E3s compete Rsp5 binding sub- essential Rsp5 function. Disruption WW domain 2          strates, basis regulated ubiquitination. <br /> 3 abrogates Rsp5 function vivo, WW domain 1               hect domain Rsp5, itself, sufficient <br /> disrupted significant effect rsp5-1 complementation         formation ubiquitin-thioester intermediate, indicat- <br /> activity. WW domains proposed protein-pro-             ing hect domain biochemical activity independent <br /> tein interaction modules affinity proline-rich se-         amino-terminal domain. Schwarz et al. recently reported <br /> quences containing PPxY motif (45). WW domains 2 3               similar results human E6-AP (38). hect domains likely involved mediating interaction       E3s shown interact specific, substrate(s) Rsp5 related essential function.       cases multiple, E2 enzymes (10, 22, 28). specific yeast Ubc <br /> WW domain 2 absolutely required Rpb1 binding,             proteins capable activating Rsp5 known, WW domain 3 contribute, C2 domain                 homology Arabidopsis Ubc8, efficiently activates <br /> WW domain 1 disrupted affecting Rpb1 bind-              Rsp5 vitro, yeast Ubc4 Ub5 best candidates, <br /> ing.  determinants Rpb1 binding ubiquitina-           followed Ubc1 Ubc13. UBC1, UBC4, UBC5  <br /> tion correspond required rsp5-1 complementation,          stitute essential subset UBC genes (39, 40). fact Rpb1 candidate        hect domain possesses determinants ubiq- <br /> substrates related essential function Rsp5.                  uitin-thioester formation supports domain model    CTD Rpb1, consisting 26 copies heptapeptide          hect E3 function suggests hect domains,  repeat, binding site Rsp5 (17). CTD essential        average, share approximately 40% similarity, inter- Rpb1 activity vivo site interaction      changeable hect E3 proteins. gener- <br /> components transcriptional machinery. consensus             ally case,  Schwarz et al. showed binding site WW domains PPxY sequence,               E6-AP/hectH6 chimera, capable forming ubiq- <br /> certain substitutions proline permissible (5).    uitin-thioester binding HPV E6 protein p53, did <br /> CTD heptapeptide sequence SPTSPSY                 ubiquitinate p53 (38). addition, structure-function anal- <br /> direct recognition sequence WW domains Rsp5.            yses E6-AP demonstrated determinants E6  <br /> function Rpb1 ubiquitination respect polII transcrip-       pendent p53 binding lie partially hect domain (16). <br />  VOL. 19, 1999                                                                                                  FUNCTIONAL DOMAINS Rsp5                          351 <br />  <br />   results obtained Rsp5 support simple                              uitin-protein ligase. Proc. Natl. Acad. Sci. USA 92:2563&#226;&#8364;&#8220;2567. domain model hect E3 function, results obtained                         15. Huibregtse, J. M., M. Scheffner, P. M. Howley. 1993. Cloning                                                                                          expression cDNA E6-AP, protein mediates interaction <br /> E6-AP suggest complex picture substrate recogni-                               human papillomavirus E6 oncoprotein p53. Mol. Cell. Biol. <br /> tion. Combined genetic biochemical analyses                                 13:775&#226;&#8364;&#8220;784. <br /> yeast hect E3s provide opportunities address                          16. Huibregtse, J. M., M. Scheffner, P. M. Howley. 1993. Localization question substrate recognition specificity.                                   E6-AP regions direct human papillomavirus E6 binding, association <br />                                                                                          p53, ubiquitination associated proteins. Mol. Cell. Biol. 13:4918&#226;&#8364;&#8220; <br />    results presented support idea ubiq-                            4927. <br /> uitin-thioester intermediate requisite intermediate ubiq-                    17. Huibregtse, J. M., J. C. Yang, S. L. Beaudenon. 1997. large subunit <br /> uitination reactions catalyzed hect E3 proteins.                          RNA polymerase II substrate Rsp5 ubiquitin-protein ligase. <br /> classes E3 activities identified, repre-                          Proc. Natl. Acad. Sci. USA 94:3656&#226;&#8364;&#8220;3661. <br />                                                                                      18. Imhof, M. O., D. P. McDonnell. 1996. Yeast RSP5 human <br /> sented Ubr1 yeast (2, 25), multimeric anaphase-                                homolog hRPF1 potentiate hormone-dependent activation transcription <br /> promoting complex  cyclosome) (12), Cdc53/Cdc4/                               human <span id='am-69' about='obo:IMR_0100034' typeof='owl:Thing obo:IMR_0100033 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000966 obo:IMR_0000001'><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00410" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="progesterone" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:id" content="IMR:0100034" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="4-Pregnene-3,20-dione" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="4-Pregnene-3,20-dione" datatype="xsd:string"></span>progesterone</span> <span id='am-59' about='obo:IMR_0100036' typeof='owl:Thing obo:IMR_0100033 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000966 obo:IMR_0000001'><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00735" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="stress hormone" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00762" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:id" content="IMR:0100036" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="stress hormone" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="KEGG:C02140" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="glucocorticoid" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="KEGG:C05490" datatype="xsd:string"></span>glucocorticoid</span> receptors. Mol. Cell. Biol. 16: <br /> Skp1 complex (12). types E3 activities                             2594&#226;&#8364;&#8220;2605. shown function ubiquitin-thioester intermediate,                       19. Johnson, E. S., P. C. M. Ma,  M. Ota,  Varshavsky. 1995. proteo- <br />                                                                                          lytic pathway recognizes ubiquitin degradation signal. J. Biol. Chem. presumed,  E2 directly catalyzes sub-                           270:17442&#226;&#8364;&#8220;17456. <br /> strate ubiquitination. E3 activities func-                       20. Kishino, T., M. Lalande, J. Wagstaff. 1997. UBE3A/E6-AP mutations <br /> tion providing surface simultaneous interaction E2                           cause Angelman syndrome. Nat. Genet. 15:70&#226;&#8364;&#8220;73. substrate(s). mechanism action                            21. Kumar, S., K. F. Harvey, M. Kinoshita, N. G. Copeland, M. Noda, N.  <br />                                                                                          Jenkins. 1997. cDNA cloning, expression analysis, mapping mouse <br /> hect E3s rule exception E3 activities                             Nedd4 gene. Genomics 40:435&#226;&#8364;&#8220;443. <br /> awaits characterization potentially                       22. Kumar, S., W. H. Kao, P. M. Howley. 1997. Physical interaction types E3 activities substrates.                                             specific E2 Hect E3 enzymes determines functional cooperativity. J. <br />                                                                                          Biol. Chem. 272:13548&#226;&#8364;&#8220;13554. <br />                                                                                      23. Lucero, P., R. Lagunas. 1997. Catabolite inactivation yeast mal- <br />                           ACKNOWLEDGMENTS                                                tose transporter requires ubiquitin-ligase npi1/rsp5 ubiquitin-hydrolase <br />   work supported Public Health Service grant                                 npi2/doa4. FEMS Microbiol. Lett. 147:273&#226;&#8364;&#8220;277. <br /> CA72943-01 National Cancer Institute.                                       24. Macias, M. J., M. Hyvonen, E. Baralk, J. Schultz, M. Sudol, M. Saraste,                                                                                          H. Oschkinat. 1996. Structure WW domain kinase-associated <br />   thank Sylvie Beaudenon helpful discussions, Mike                           protein complexed proline-rich <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span>. Nature 382:646&#226;&#8364;&#8220;649. <br /> Kiledjian critical reading manuscript, Xin Li assis-              25. Madura, K.,  Varshavsky. 1994. Degradation Ga N-end rule <br /> tance tetrad dissections.                                                           pathway. Science 265:1454&#226;&#8364;&#8220;1458. <br />                                                                                      26. Matsuura, T., J. S. Sutcliffe, P. Fang, R.-J. Galjaard, Y.-H. Jiang, C. S. <br />                                    REFERENCES                                            Benton, J. M. Rommens,  L. Beaudet. 1997. novo truncating <br />  1. Albrecht, U., J. S. Sutcliffe, B. M. Cattanach, C. V. Beechey, D. Armstrong,         mutations E6-AP ubiquitin-protein ligase gene (UBE3A) Angelman <br />     G. Eichele,  L. Beaudet. 1997. Imprinted expression murine              syndrome. Nat. Genet. 15:74&#226;&#8364;&#8220;77. <br />     Angelman syndrome gene, Ube3a, hippocampal Purkinje neurons.              27. Nefsky, B., D. Beach. 1996. Pub1 acts E6-AP-like protein ubiquitin <br />     Nat. Genet. 17:75&#226;&#8364;&#8220;78.                                                                ligase degradation cdc25. EMBO J. 15:1301&#226;&#8364;&#8220;1312. <br />  2. Bartel, B.,  Wu <br />                     &#194;&#168;nning,  Varshavsky. 1990. recognition component       28. Nuber, U., S. Schwarz, P. Kaiser, R. Schneider, M. Scheffner. 1996. <br />     N-end rule pathway. EMBO J. 9:3179&#226;&#8364;&#8220;3189.                                      Cloning human ubiquitin-conjugating enzymes UbcH6 UbcH7 (E2- <br />  2a.Beaudenon, S., J. M. Huibregtse. Unpublished data.                               F1) characterization interaction E6-AP RSP5. J. Biol. <br />  3. Bedford, M. T., D. C. Chan, P. Leder. 1997. FBP WW domains               Chem. 271:2795&#226;&#8364;&#8220;2800. <br />     Abl SH3 domain bind specific class proline-rich ligands. EMBO J.         29. Perry, W. L., C. M. Hustad, D.  Swing, T. N. O&#226;&#8364;&#8482;Sullivan, N.  Jenkins,     16:2376&#226;&#8364;&#8220;2383.                                                                        N. G. Copeland. 1998. itchy locus encodes novel ubiquitin protein <br />  4. Bregman, D. B., R. Halaban,  J. van Gool, K.  Henning, E. C. Friedberg,          ligase disrupted a18H mice. Nat. Genet. 18:143&#226;&#8364;&#8220;146. <br />     S. L. Warren. 1996. UV-induced ubiquitination RNA polymerase II:          30. Pirozzi, G., S. J. McConnell,  J. Uveges, J. M. Carter,  B. Sparks, B. K. <br />     novel modification deficient Cockayne syndrome cells. Proc. Natl. Acad.         Kay, D. M. Fowlkes. 1997. Identification novel human WW domain- <br />     Sci. USA 93:11586&#226;&#8364;&#8220;11590.                                                             containing proteins cloning ligand targets. J. Biol. Chem. 272:14611&#226;&#8364;&#8220; <br />  5. Chen, H. ,  Einbond, S.-J. Kwak, H. Linn, E. Koepf, S. Peterson, J. W.           14666. <br />     Kelly, M. Sudol. 1997. Characterization WW domain human            31. Plant, P. J., H. Yeger, O. Staub, P. Howard, D. Rotin. 1997. C2 <br />     Yes-associated protein polyproline-containing ligands. J. Biol. Chem.        domain ubiquitin protein ligase Nedd4 mediates Ca21-dependent <br />     272:17070&#226;&#8364;&#8220;17077.                                                                     plasma membrane localization. J. Biol. Chem. 272:32329&#226;&#8364;&#8220;32336. <br />  6. la Fuente, N.,  M. Maldonado, F. Portillo. 1997. Glucose activation     32. Ratner, J. N., B. Balasubramanian, J. Corden, S. L. Warren, D. B. <br />     yeast plasma membrane H1-ATPase requires ubiquitin-protea-                Bregman. 1998. Ultraviolet radiation-induced ubiquitination proteaso- <br />     proteolytic pathway. FEBS Lett. 411:308&#226;&#8364;&#8220;312.                                    mal degradation large subunit RNA polymerase II. Implications  7. Einbond, , M. Sudol. 1996. prediction cognate complexes             transcription-coupled DNA repair. J. Biol. Chem. 273:5184&#226;&#8364;&#8220;5189. <br />     WW domain proline-rich ligands. FEBS Lett. 384:1&#226;&#8364;&#8220;8.              33. Rizo, J., T. C. Sudhof. 1998. C2-d <br /> </body></html>