<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Jan. 2000, p. 1&#226;&#8364;&#8220;11                                                                             Vol. 20,  1 <br /> 0270-7306/00/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 2000, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br /> Scanning Mutagenesis Mcm1: Residues Required DNA Binding, <br />           DNA Bending, Transcriptional Activation                        MADS-Box Protein <br />               THOMAS B. ACTON, JANET MEAD, ANDREW M. STEINER,                                ANDREW K. VERSHON* <br />                      Waksman Institute Microbiology Department Molecular Biology Biochemistry, <br />                                       Rutgers University, Piscataway, New Jersey 08854-8020 <br />                         Received 11 June 1999/Returned modification 22 July 1999/Accepted 23 September 1999 <br />  <br />              MCM1 essential gene yeast Saccharomyces cerevisiae member MADS-box family <br />           transcriptional regulatory factors. understand nature protein-DNA interactions class <br />           proteins, series alanine substitutions DNA-binding domain Mcm1 examined <br />           effects mutations vivo vitro. results indicate residues Mcm1 important <br />           viability, transcriptional activation, DNA binding bending. Substitution residues Mcm1           highly conserved MADS-box proteins lethal cell abolish DNA binding vitro. <br />           positions identical interactions DNA serum response factor-DNA           &#226;?&#163;2-Mcm1-DNA crystal structures, suggesting residues make conserved core protein-DNA <br />           interactions responsible docking MADS-box proteins DNA. Substitution residues           conserved members MADS-box family play important roles contributing specificity           DNA binding. results suggest general model MADS-box proteins recognize bind DNA.           provide evidence N-terminal extension Mcm1 considerable conformational freedom, <br />           possibly allow binding different DNA sites. Finally, identified mutants positions           critical Mcm1-mediated DNA bending slow-growth phenotype. finding consistent           earlier results, indicating DNA bending role Mcm1 function cell. <br />  <br />  <br />    yeast Mcm1 protein member MADS-box                      middle layer consists hydrophobic stranded  <br /> family transcriptional regulatory factors. growing class           tiparallel &#226;?&#164;-sheet, formed &#226;?&#164;-strands mono- proteins named Mcm1, Arg80, Agamous, Deficiens,                mer aligned roughly parallel coiled-coil alpha- serum response factor (SRF),                helices. &#226;?&#164;-loop linking strands monomer proteins isolated containing DNA-binding dimeriza-                involved phosphate contact position outside tion motif (5, 16, 18, 23, 32). class proteins        conserved CArG-box [CC T)6GG] binding site. 60 members organisms range                  layer protein consists extended coil region, fol- single-cell yeast Saccharomyces cerevisiae (MCM1, ARG80,               lowed alpha-helix turns. second alpha-helix RLM1) Xenopus (SRF), Drosophila (DRSF MEF2),                    monomer packs corresponding face plants (Ag, Def, Glo) humans (SRF) (2, 5, 13, 15, 16,              monomer, adding dimerization interface. addi- <br /> 18, 20, 23, 28, 32). DNA-binding domains                  tion layers protein, N- <br /> highly conserved family proteins, function             terminal extension alpha-helix monomer <br /> varies greatly, ranging regulatory factors control               makes base-specific phosphate backbone  <br /> basic cellular processes required development             tacts positions center recognition site. Inter- determination cell type (2, 11, 27, 28). Understanding              estingly, position extension different class proteins functions important            proteins. SRF structure region follows path <br /> understanding mechanism regulation cellular                 minor groove center site. developmental processes.                                               Mcm1 structure, region crosses minor groove    cocrystal SRF protein complex DNA revealed               makes base-specific contacts major groove center structure MADS domain time (19).                  site. recently, structure member MADS-                   Biochemical structural studies shown box family, yeast Mcm1 protein, indicated overall             MADS-box proteins produce large bend DNA fold protein DNA contacts remark-               binding recognition sites (1, 8, 19&#226;&#8364;&#8220;22, 24, 30). Circular <br /> ably conserved yeast human proteins (24).                  permutation assays suggest SRF produces significant structures MADS-box domain forms protein                  bend DNA binding site. results agree <br /> dimer composed layers (Fig. 1). layer consists <br />                                                                            SRF crystal structure, DNA bent antiparallel coiled-coil alpha-helices, mono- <br />                                                                            CArG recognition site protein mer, positioned minor groove center                                                                            produce overall bend angle 72&#194;&#176; (19). Mcm1 struc- <br /> recognition site. Residues helices make numerous <br />                                                                            ture, DNA bent protein, edge <br /> phosphate base-specific contacts major groove. <br />                                                                            site second DNA bend away protein <br />                                                                            plane perpendicular bend (24). unlike <br />   * Corresponding author. Mailing address: Waksman Institute, 190          DNA SRF structure, lies predominantly <br /> Frelinghuysen Rd., Piscataway, NJ 08854-8020. Phone: (732) 445-2905.       plane. Mutations positions outside  <br /> Fax: (732) 445-5735. E-mail: vershon@mbcl.rutgers.edu.                     served CArG site affect degree bending Mcm1 (1). <br />  <br />                                                                        1 <br />  2      ACTON ET AL.                                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Mcm1 contacts consensus binding site DNA.   contacts shown predicted observed contacts &#226;?&#163;2-Mcm1-DNA ternary complex consensus symmetric Mcm1 binding site used studies described  half site shown axis symmetry &#226;&#171;&#185;1 &#226;&#171;&#186;1 <br /> bp. contacts shown Mcm1 monomer, Lys46, adjacent monomer complex. arrows indicate base-specific contacts; circles indicate phosphate sugar backbone contacts. (B C) Models Mcm1 dimer binding site derived coordinates crystal structure Mcm1 ternary complex (24). different views complex shown, positions different layers protein. chains residues <br /> required DNA bending, K40, S37, V34  , shown right monomer panel C. <br />  <br />  <br />  base-pair substitutions large effect tran-                       DNA bending Mcm1. does appear <br /> scriptional activation, suggesting possible link                       strict correlation simple DNA bending DNA-bending activity Mcm1 transcrip-                               transcriptional activation Mcm1   mu- <br /> tional regulation MADS-box protein.                                           tants altered DNA bending slow growth phe- <br />    obtain deeper understanding protein-DNA                             notype, indicating important role Mcm1 <br /> teractions requirements DNA-bending                                function cell. <br /> Mcm1 protein, constructed series alanine substi- <br /> tutions DNA-binding domain assayed effects                                                   MATERIALS METHODS mutations vivo vitro. results identify <br />                                                                                          Plasmids. Individual alanine substitutions position DNA-bind- <br /> residues critical supporting viability cell                      ing region Mcm1 constructed cloning double-stranded oligonucleo- measure contributions chain DNA                           tides containing desired codon changes pJM231, derivative binding transcriptional activation. results presented                         pRS413 (CEN HIS3) vector contains engineered MCM1 gene expressed  SRF-DNA &#226;?&#163;2-Mcm1-DNA crystal                               native promoter. Unique restriction sites, KpnI (bp 24), SacI (bp 51), <br />                                                                                       MluI (bp 91), SpeI (bp 100), MfeI (bp 140), StuI (bp 170), engineered <br /> structures, help define conserved set protein-DNA inter-                         region coding DNA-binding domain Mcm1 introducing <br /> actions critical DNA binding MADS-box proteins.                            silent mutations indicated <span id='am-12' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> positions using recursive PCR results identify residues required sequence-specific                     described previously (14). sequences inserts adjacent regions <br />  VOL. 20, 2000                                                                                           SCANNING MUTAGENESIS Mcm1                  3 <br />  <br />  determined verify presence specific base-pair substitutions   assayed ability support growth cell (18). ensure mutations introduced cloning process.       order accomplish  plasmid shuffle technique engineered MCM1 construct complements mcm1 null mutant viability activates transcription level unmodified gene, indicating     used introduce mcm1 mutants yeast remove codon changes did significantly alter expression activity      wild-type copy gene. yeast strain YY2052  <br /> protein.                                                                             tains chromosomal mcm1::LEU2 null mutation main- <br />    Strains vivo assays. ability MCM1 mutants maintain cell      tained plasmid containing wild-type MCM1 (4). plas- <br /> viability activate transcription measured strain YY2052 [MATa <br /> P(PAL)-lacZ::FUS1 leu2-3,112 trp1-1 ura3 his3-11,15 ade2-1 can1-100 mcm1::           mid contains URA3, cells lost LEU2/pSL1574-CEN/ARS MCM1 URA3] provided G. Sprague (4).                plasmid selected plating media containing Cells transformed mutant derivatives pJM231 (CEN HIS3               drug 5-FOA. Yeast cells transformed derivatives MCM1) colonies lost wild-type copy MCM1 plasmid          plasmid pJM231 containing mcm1 alanine substitution mu- <br /> pSL1574 selected plating synthetic medium containing 5-fluoroorotic <br /> acid (5-FOA) absence histidine leucine. Mcm1-dependent tran-           tants tested ability support growth plating <br /> scriptional activation measured strain YY2052 assaying lacZ ex-        5-FOA (3). Transformants fail grow wild- <br /> pression integrated CYC1-lacZ reporter promoter P(PAL)               type MCM1 lost indicates alanine substitu- <br /> Mcm1 binding site liquid &#226;?&#164;-galactosidase assays (4). assay effects      tion significant effect Mcm1 activity mutations Mcm1 protein combination mutant binding sites, mutant <br /> derivatives pJM231 transformed strain JM02 (MATa leu2-3,112             mutant unable complement mcm1 null mutation. <br /> trp1-1 ura3 his3-11,15 ade2-1 can1-100 mcm1::LEU2/pSL1574-CEN/ARS MCM1                  results alanine scanning mutagenesis indicate URA3), loss plasmid pSL1574, containing wild-type gene,       14 40 chains DNA-binding region selected medium containing 5-FOA. strains transformed             Mcm1 MADS-box domain essential ability sup- derivatives pTBA43, URA3, 2&#226;?&#174;m, CYC1-lacZ transcription reporter <br /> vector CYC1 upstream activation sequence UAS elements              port growth cell (Table 1). lethal muta- removed replaced consensus Mcm1 binding site containing              tions alpha-helix forms coiled-coil interface <br /> symmetric base-pair substitutions (1). activation results shown tables    monomers contacts DNA. based averages independent transformants mutant         residues contact DNA SRF Mcm1 crystal presented percent activity wild-type protein  Table 1) units &#226;?&#164;-galactosidase activity  Table 2 Fig. 6).                       structures highly conserved MADS-box pro- <br />    Protein purification. Mutant Mcm11&#226;&#8364;&#8220;96 proteins used vitro DNA-          teins (19, 24) (Fig. 1). Alanine substitutions residues K38, <br /> binding assays purified Escherichia coli BL21 cells trans-      R39, K40, G42, K45, K46, make direct contacts formed pHA227, protein expression vector sequences coding        DNA Mcm1 crystal structure, support cell  Mcm1 residues 1 96 fused frame gene coding maltose-binding protein. mutant fusion proteins purified 100-ml        bility. Substitutions conserved positions alpha- <br /> cultures described earlier (1). proteins isolated procedure      helices contact DNA, R32, F36, <br /> &#226;&#172;&#381;95% homogenous consisted nonnative N-terminal amino acids,               I43, M44, L53, T54, fail complement mcm1 null <br /> <span id='am-32' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-33' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>Gly</span>-Ser, followed native Mcm1 residues 1 96. protein concentra-        mutant lethal cell. residues positioned <br /> tion mutant determined Bradford assays (Bio-Rad), normalized, verified sodium dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-      dimer interface, possible explanation PAGE) gels stained Coomassie blue.                                              inability support growth cell    EMSAs. DNA fragments utilized electrophoretic mobility shift           chains critical protein stability proper folding. <br /> assays (EMSAs)  Fig. 2 Fig. 6) synthesized PCR amplifi-         mutations N-terminal extension, F25A cation wild-type mutant &#226;?&#163;2/Mcm1 operator sites transcription  <br /> porter plasmids. described previously, oligonucleotides W340 (5&#226;&#172;&#732;-CACGCC           I26A, failed complement null mutant. TGGCGGATCTGC-3&#226;&#172;&#732;), [&#226;?&#165;-32P]<span id='am-1' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> end labeled purified                 positions located beginning N-terminal exten- <br /> Sephadex G-25 spin column, W341 (5&#226;&#172;&#732;-GCCCACGCTAGGCAATC-3&#226;&#172;&#732;)                        sion, folds coiled-coil alpha-helices annealed CYC1 promoter region Mcm1 binding         forms dimer interface. substitutions site used amplify 120-bp fragment containing Mcm1 recogni- <br /> tion site (1, 33). PCR-generated fragments purified native     affect protein stability. mutations, V34A, S37A, PAGE. circular permutation assays  Fig. 4 Fig. 5) performed         E49A, viable slow-growth phenotype, suggesting BamHI-, NheI-, HindIII-, EcoRI-generated end-labeled fragments          weakly complement mcm1 null mutant. <br /> derivatives pGD579, contains tandem repeat 375-bp fragment              Effects alanine substitutions Mcm1-mediated tran- <br /> separated 24-bp polylinker containing wild-type mutant Mcm1 binding <br /> sites (1).                                                                           scriptional activation. mutants fail    purified mutant Mcm11&#226;&#8364;&#8220;96 proteins incubated 1 h room              complement mcm1 null mutation lethal <br /> temperature labeled DNA fragment 20 mM Tris (pH 7.6), 200 mM             cell, large number alanine substitutions NaCl, 5 mM MgCl2, 0.1 mM EDTA, 0.1% NP-40, 10% glycerol, 10 &#226;?&#174;g                viable. determine substitutions affect Mcm1 <br /> herring sperm DNA ml. circular permutation EMSA performed 6% polyacrylamide gel 1&#226;&#171;&#187; Tris-borate-EDTA (TBE) 200 V,        activity, viable mutants assayed ability EMSAs performed 6% polyacrylamide gel 0.5&#226;&#171;&#187; TBE. EMSA              activate transcription vivo. assay used integrated <br /> gels dried visualized using Molecular Dynamics 425E Phosphor-          lacZ reporter gene control P(PAL) Mcm1 bind- <br /> Imager. relative binding affinity Mcm1 mutants calculated          ing site (Table 1) (4, 10). viable mutants linear regression compared wild-type protein. <br />    Cross-linking assays. Oligonucleotides used cross-linking experiments      wild-type near-wild-type levels transcriptional activation synthesized using photoreactive reagent 5-bromo-2&#226;&#172;&#732;-deoxyuridine-         reporter promoter, indicating dramat- <br /> &#226;?&#164;-cyanoethylphosphoramidite (Glen Research, ) place <span id='am-20' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Thymine" datatype="xsd:string"></span>thymine</span> phos-          ically alter structure function protein. phoramadite position &#226;&#171;&#185;1 DNA strand Mcm1 symmetric               substitutions residues surface protein <br /> consensus site (5&#226;&#172;&#732;-GCCCTACCTAABrUTAGGTACCCG-3&#226;&#172;&#732;). complemen- <br /> tary strand (5&#226;&#172;&#732;-CGGGTACCTAATTAGGTAGGGC-3&#226;&#172;&#732;), did                     contact DNA. expected, substitutions contain 5-bromouracil (BrU), annealed BrU containing oligonucleotides         residues contact DNA, R19, I21, S37, produce double-stranded recognition site. Cross-linking experiments        T66, reduced levels activation. Interestingly, performed purified proteins described earlier (1).                           slow-growth mutant V34A, contacts DNA,                                                                                      lowest level activation viable mutant. Alanine substi- <br />                                   RESULTS                                            tutions residues I23, F48, E49 reduced levels <br />                                                                                      activation. residues near dimer interface,    Conserved residues Mcm1 essential viability.                        possible alanine substitutions positions alter <br /> determine residues Mcm1 important activ-                        stability protein, causing decrease transcrip- <br /> ity, residue DNA-binding region MADS-box                          tional activation. results correlate domain individually substituted alanine. MCM1                         predictions based Mcm1 crystal structure, essential gene yeast, alanine mutants                      results unexpected (24). example, residue R18 <br />  4       ACTON ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />          TABLE 1. Activity Mcm1 alanine substitutions                       expressed purified manner determine                                                                                    reductions Mcm1-mediated activation vivo function <br />                                                     Relative decrease DNA- <br />     Mutanta      Growthb        % Activationc <br />                                                       binding affinityd (fold)     decreased DNA-binding affinity. example assays                                                                                    shown Fig. 2, relative binding affinity calculated Wild type           &#226;&#171;&#185;                100                         1                 mutant listed Table 1. <br /> Q14A                &#226;&#171;&#185;                104                         1.5                  general, results DNA-binding affinity analysis <br /> Q15A                &#226;&#171;&#185;                103                         1.4               good agreement vivo viability activation <br /> K16A                &#226;&#171;&#185;                 80                         2.5 <br /> E17A                &#226;&#171;&#185;                107                         1.5 <br />                                                                                    data correlate predictions based Mcm1 crystal <br /> R18A                &#226;&#171;&#185;                100                         1.6               structure (24). number alanine substitutions R19A                &#226;&#171;&#185;                 21                         4                 support viability, including F25A, I26A, R32A, F36A, <br /> K20A                &#226;&#171;&#185;                 80                         3                 I43A, M44A, L53A, T54A, residues form I21A                &#226;&#171;&#185;                 53                         4.5               hydrophobic core protein dimer interface. E22A                &#226;&#171;&#185;                 87                         1.4               purification mutants bacteria, I23A                &#226;&#171;&#185;                 34                         4.3               proteins present lower levels lysates, exhibited <br /> K24A                &#226;&#171;&#185;                110                         0.8               decreased affinity heparin column, sus- <br /> F25A                &#226;&#171;&#186;                                          NDB                 ceptible degradation. finding consistent model <br /> I26A                &#226;&#171;&#186;                                          NDB <br /> E27A                &#226;&#171;&#185;                 98                         2 <br />                                                                                    substitutions affect folding protein, N28A                &#226;&#171;&#185;                 87                        20                 making stable susceptible proteolysis. <br /> K29A                &#226;&#171;&#185;                100                         0.8               mutants &#226;&#172;&#381;100-fold decrease DNA- <br /> T30A                &#226;&#171;&#185;                100                                           binding affinity. results suggest loss viability <br /> R31A                &#226;&#171;&#185;                 70                                           mutants function inability produce <br /> R32A                &#226;&#171;&#186;                                          110                 proteins competent DNA binding. <br /> H33A                &#226;&#171;&#185;                120                         0.25                 remaining nonviable substitutions, K38A, R39A, K40A, <br /> V34A                S                  9                         2.7               G42A, K45A, K46A, binding face coiled- <br /> T35A                &#226;&#171;&#185;                 82                         3                 coil alpha-helices, residues contact DNA F36A                &#226;&#171;&#186;                                        NDB <br /> S37A                S                 66                         2.1 <br />                                                                                    crystal structure (24). mutants &#226;&#172;&#381;1,000- <br /> K38A                &#226;&#171;&#186;                                        NDB                   fold decrease DNA-binding affinity vitro, indicating R39A                &#226;&#171;&#186;                                        NDB                   residues essential role DNA binding Mcm1. <br /> K40A                &#226;&#171;&#186;                                          125                 results fact residues highly  <br /> H41A                &#226;&#171;&#185;                100                         0.12              served MADS-box proteins underscores im- <br /> G42A                &#226;&#171;&#186;                                        NDB                   portance DNA binding supports model  <br /> I43A                &#226;&#171;&#186;                                          250                 served mechanism DNA recognition family M44A                &#226;&#171;&#186;                                       &#226;&#172;&#381;1,000                 proteins. <br /> K45A                &#226;&#171;&#186;                                       &#226;&#172;&#381;1,000                    viable mutants decreases DNA- <br /> K46A                &#226;&#171;&#186;                                        NDB <br /> A47                 &#226;&#171;&#185;                100                         1 <br />                                                                                    binding affinity, general effect F48A                &#226;&#171;&#185;                 34                         0.8               DNA-binding affinity nonviable mutants. Alanine sub- <br /> E49A                S                 24                        48                 stitutions residues contact DNA S51A                &#226;&#171;&#185;                 99                         1.0               crystal structure, R18, R19, I21, T35, S37, T66, <br /> V52A                &#226;&#171;&#185;                 91                         1.2               modest decreases DNA-binding affinity vitro, L53A                &#226;&#171;&#186;                                        NDB                   correlate observed decreases activation T54A                &#226;&#171;&#186;                                        NDB                   mutants vivo. residues somewhat  <br /> T66A                &#226;&#171;&#185;                 77                         3.1               served MADS-box proteins K38, R39, G42, <br />         DNA contacts boldface (based &#226;?&#163;2/Mcm1/DNA crystal struc-          K45, K46, essential viability ture).                                                                             residues clearly contribute DNA-binding affinity   b <br />     &#226;&#171;&#185;, Normal growth swapping wild-type copy MCM1; &#226;&#171;&#186;,          transcriptional activation. Additionally, residues <br /> growth 5 days; S, slow-growth phenotype. <br />   c <br />     level activation based lacZ expression comparison                                                                                    role determining specificity DNA binding. <br /> wild type.                                                                            mutants slow-growth phenotype, V34A <br />   d <br />     relative decrease DNA-binding affinity comparison wild-type   S37A, produce protein-DNA complexes higher mo- <br /> protein. NDB, detectable binding observed EMSA.                      bility wild-type protein  Fig. 6). mutations <br />                                                                                    cause significant decreases transcriptional activation, <br />                                                                                    V34A resulting lowest &#226;?&#164;-galactosidase activity makes base-specific phosphate backbone contacts                        viable mutants. contrast, mutations, N28A base pairs center site                  I21A, greater decreases binding affinity V34A expected important role function                       similar decreases levels transcrip- <br /> protein.  alanine substitution position                     tional activation. results suggest V34A S37A <br /> viable activates transcription wild-type levels, indicating                 affect aspects Mcm1 function. R18 chain critical Mcm1 activity                     Loss base-specific contacts mutations N-termi- <br /> site.                                                                              nal extension. striking differences    Effects alanine substitutions DNA-binding affinity                    SRF Mcm1 crystal structures location vitro. significant number mutants alanine substi-                     N-terminal extension protein. Mcm1 structure, <br /> tutions residues involved protein-DNA contacts fail                       R18 chain making set contacts major <br /> complement mcm1 null mutant cell viability. deter-                      groove, SRF structure analogous residue substitutions affect DNA-binding activity                     making series contacts minor groove DNA protein, mutants cloned bacterial expres-                      (19, 24). difference predicted analysis sion vector purified, DNA-binding affinity                       SRF structure supported differences protein-DNA <br /> assayed EMSAs. addition, viable mutants                      cross-linking SRF Mcm1 base pairs center  VOL. 20, 2000                                                                                                  SCANNING MUTAGENESIS Mcm1                           5 <br />  <br />  <br />  <br />  <br />    FIG. 2. EMSA analysis Mcm1 substitutions. DNA binding symmetric consensus site wild-type Mcm1 (lanes 2 6 47 50), K40A (lanes 7 11), <br /> K45A (lanes 12 16), I43A (lanes 17 21), F48A (lanes 22 26), M44A (lanes 27 30), H41A (lanes 31 34), I21A (lanes 35 38), R19A (lanes 39 42), R18A (lanes 43 46). proteins span residues 1 98 Mcm1, contains entire MADS-box domain. Fivefold titrations Mcm1 performed, <br /> increasing concentration 2.2 &#226;&#171;&#187; 10&#226;&#171;&#186;10 M (lanes 2, 7, 12, 17, 22) 2.0 &#226;&#171;&#187; 10&#226;&#171;&#186;9 (lanes 27, 31, 35, 39, 43, 47). relative binding affinity calculated mutant comparison wild-type shown Table 1. <br />  <br />  <br />  recognition site (1, 19). Additionally, mutational analysis                     shows decrease activity mutant site com- DNA site shows substitutions central base pairs                        parison wild-type site, suggest produce large decrease Mcm1-mediated activation, indi-                            base-specific contacts positions mediated cating sequence-specific contacts posi-                        residues. R18A mutation shows decrease activation <br /> tions (1). Residue R18 makes extensive set base-specific                         A1C binding site mutation comparison wild-type phosphate backbone contacts center site                           site (Table 2). reduction activation R18A mutant crystal structure (24).  alanine substitution                        reporter A1C mutation similar position R18 does significantly alter transcriptional activa-                      reduction wild-type protein, suggesting resi- <br /> tion vivo DNA-binding affinity vitro. result                         dues contacting base pairs. E17A mutant unexpected, positions involved base-                          displays similar activity. hand, R19A muta- <br /> specific contacts significantly larger effects viability                   tion does reduction activation A1C com- <br /> activation. comparison, substitution adjacent resi-                        pared wild-type binding site. fact activation  R19, significantly larger effect transcriptional                         mutant A1C binding site slightly increased, <br /> activation DNA-binding affinity.                                                   suggesting steric interference Arg <br />    investigate contributions residues                       chain mutant DNA site. results raise N-terminal extension specificity activation,                         possibility R19 contact central base pairs formed loss contact experiments (6) E17A,                              consensus symmetric binding site used experiments. <br /> R18A, R19A mutations assayed ability                                <span id='am-25' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-26' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>Amino acid</span> residues Mcm1 required BrU-mediated <br /> activate transcription reporter promoter containing                            cross-linking center site. BrU photoactivat- <br /> A1C base-pair substitution Mcm1 site. comparison                          able, isosteric analog thymine. Irradiation BrU UV wild-type site, A1C substitution causes fourfold                          light leads reactive species form covalent <br /> duction level transcriptional activation wild-                        bond groups close proximity (17). type Mcm1 protein (1). R18A mutant shows similar                                previously used property BrU transcriptional activation wild-type mutant                       Mcm1 protein contacts thymine methyl groups major <br /> sites, indicate alanine substitution                        groove base pair position center DNA <br /> removed base-specific contacts position                             recognition site (1). Mcm1 crystal structure NH1 <br /> DNA, protein unable discriminate                           group R18 close proximity C-5 methyl group wild-type mutant sites.  mutant protein                             T&#226;&#171;&#186;1 likely target cross-linking (24). test  6           ACTON ET AL.                                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />            TABLE 2. Activation N-terminal extension mutants <br />                    wild-type mutant DNA sitesa <br />                                            &#226;?&#164;-Gal activity (U) <br />     Site <br />                      WT            E17A                  R18A            R19A <br />                   0.4            0.3                  0.4              0.4 <br /> WT                    5.5            4.6                  4.9              0.3 <br /> A1C                   1.4            0.9                  0.9              0.9 <br />       Values shown units &#226;?&#164;-galactosidase activity (&#226;?&#164;-Gal) standard <br /> deviations &#226;&#172;?10%. WT, wild type. <br />  <br />  <br />  <br />  <br /> amino acid N-terminal extension involved base-specific contact thymine center recognition site, BrU cross-linking assays performed Mcm1 alanine substitution mutants. E17A R18A <br /> proteins form cross-link DNA (Fig. 3). result <br /> suggests chains involved direct contact center bases site used experi- <br />                                                                                       FIG. 3. Formation cross-linked complexes Mcm1 mutants. <span id='am-44' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-45' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Urea" datatype="xsd:string"></span>Urea</span>- <br /> ments.                                                                             denaturing PAGE BrU-mediated photo-cross-linking reactions    R19A mutant,  did form significant level                      E17A, R18A, R19A mutants labeled site containing BrU substitution BrU-mediated cross-linked band, suggesting resi-                      thymine base &#226;&#171;&#186;1. Lanes 1, 6, 11 control reactions omitting UV required interaction thymine methyl group                      radiation. Lanes 2, 7, 12 control reactions omitting Mcm1 protein. <br />                                                                                    Lanes 3, 8, 13 control reactions labeled Mcm1 binding site position &#226;&#171;&#186;1. small cross-linking po-                      BrU (denoted T figure). Lanes 4, 9, 14 control reactions sition   result background level                      10-fold excess unlabeled competitor added. Lanes 5, 10, 15 <br /> Mcm1 photo-cross-linking observed absence                           cross-linking reactions. arrow indicates position cross- <br /> BrU-substituted DNA (Fig. 3, lanes 3, 8, 13).                       linked Mcm1-DNA products. <br /> possible R19A substitution alters structure N-terminal extension, allowing residue interact BrU functional group.                            bending wild-type protein T7G site. result <br /> previously shown Mcm1 fragment missing 17                        suggests large drop bending observed T7G <br /> residues protein, Mcm118&#226;&#8364;&#8220;96, forms cross-linked prod-                     base-pair substitution similar loss contacts res- <br /> uct DNA position (1). result indicates                   idues V34 S37. <br /> 17 residues N-terminal extension involved                           addition base-specific contacts V34 S37, cross-link. cross-linking results loss contact                  residues K40, T66, lesser degree K29 H33, data agreement suggest R19 involved                      close region DNA possibly contribute major groove contact thymine methyl group                          DNA bending (24, 29). K40A mutation able position &#226;&#171;&#185;1 &#226;&#171;&#186;1 site used experiments.                     support viability cell, DNA-binding affinity    Mcm1 mutants altered bending activity. Base-pair sub-                      mutant decreased 125-fold compared wild- <br /> stitutions positions &#226;&#171;&#185;7 &#226;&#171;&#186;7 Mcm1 DNA-binding site                      type protein. circular permutation analysis shows cause large decrease DNA bending correspond-                        DNA bending decreased 36&#194;&#176; (Fig. 4B). decrease ing effect DNA-binding affinity (1). Interestingly, mutations                   equivalent decrease bending observed V34A- position result significant decrease transcrip-               S37A double mutant indicates residue tional activation. Mcm1 crystal structure, residues V34                     critical role DNA bending. contrast, ala- K38 make base-specific contacts position                        substitutions K29, H33, T66 little effect <br /> DNA site (24). residue S37, V34 makes base-                        DNA binding, bending, transcriptional activation (Table <br /> specific contacts position 8 site. investigate                     1, Fig. 4B; DNA bending data shown K29A mutations residues affect DNA-bending activity                     H33A). results residues make Mcm1, circular permutation EMSAs performed                               large contribution DNA binding bending protein. mutants apparent bend angles calculated                            determine residues DNA-binding region (Fig. 4). V34A S37A mutants significant                            Mcm1 required DNA bending, assayed alanine <br /> creases DNA bending compared wild-type protein.                          substitution EMSA NheI fragment used results support predictions based crystal structure                   circular permutation experiments. fragment shows residues important producing                       largest differences shift mobility mutations affect <br /> Mcm1-dependent DNA bend. Interestingly, decrease                            DNA bending serves sensitive diagnostic assay bending mutants half decrease                      identifying mutants altered bending. exception bending resulting T7G mutant DNA (1).                       mutants described  mutant <br /> possibility intermediate effect protein mutants                  proteins bind DNA appear alter DNA bending (data chain contributing bending. test                  shown). possible chains, K38, model, V34A-S37A double mutant constructed                             contribute DNA bending.  failed assayed viability bending. Unlike single mutants,                      bind DNA assay conditions, unable V34A-S37A double mutant unable complement                            measure effect bending. <br /> mcm1 null mutant. V34A-S37A mutant apparent                                Sequence specificity DNA bending Mcm1. crystal <br /> bend angle 61&#194;&#176;, double observed decrease                          structure, residues V34A, S37A, K40A make base-specific single mutants, comparable                       phosphate backbone contacts positions 7 8  VOL. 20, 2000                                                                                          SCANNING MUTAGENESIS Mcm1                7 <br />  <br />  <br />                                                                                     DNA site, shown alanine substitutions                                                                                     residues cause decreases degree DNA bending <br />                                                                                     protein (24). examine specificity required                                                                                     bending, tested ability Mcm1 mutants bend <br />                                                                                     DNA mutant binding sites. V34A S37A mu- <br />                                                                                     tants degree bending mutant site                                                                                     wild-type site, suggest residues                                                                                     contacting particular base pair additional <br />                                                                                     base-specific contacts position lost.                                                                                      reduction bending mutant protein                                                                                     mutant site, indicate base-specific <br />                                                                                     contacts base pair. V34A substi- <br />                                                                                     tution apparent degree DNA bending                                                                                     T7G wild-type sites, providing evidence decrease <br />                                                                                     bending mutant site caused loss contact                                                                                     residue V34 (Fig. 4C). result suggests K38 contact <br />                                                                                     thymine base pair critical bending.                                                                                     hand, T7G mutation causes significant decrease <br />                                                                                     DNA bending wild type (35&#194;&#176;) R18A (29&#194;&#176;), S37A <br />                                                                                     (25&#194;&#176;), T66A (32&#194;&#176;) mutants (Fig. 4C). result indicates <br />                                                                                     residues contact base pair.                                                                                     expected, V34A-S37A double mutant does                                                                                     decrease bending T7G mutation. <br />                                                                                        SRF protein binds Mcm1 site moderate <br />                                                                                     affinity produces significant bend DNA.  <br />                                                                                     unlike Mcm1, DNA bending SRF affected T7G <br />                                                                                     base-pair substitution (1). finding suggests                                                                                     residues contact DNA identical <br />                                                                                     Mcm1 SRF, mechanism bending signif- <br />                                                                                     icantly different. difference proteins                                                                                     SRF contains threonine position analogous <br />                                                                                     V34 Mcm1, chain makes contacts position <br />                                                                                     8 DNA position 7 (19). test differ- <br />                                                                                     ences specificity bending proteins                                                                                     presence threonine position 34, constructed <br />                                                                                     V34T substitution Mcm1 assayed effects                                                                                     transcription vivo DNA binding bending vitro. <br />                                                                                     V34T mutant complements mcm1 null mutant                                                                                     near-wild-type levels transcriptional activation                                                                                     P(PAL) site vivo, showing threonine functionally <br />                                                                                     substitute valine position Mcm1 (data shown). <br />                                                                                     V34T mutant binds wild-type affinity                                                                                     produces bend wild-type site similar                                                                                     wild-type protein (Fig. 4B).  unlike SRF, V34T <br />                                                                                     mutation exhibits decrease apparent bend angle                                                                                     T7G substitution (Fig. 4C). results indicate                                                                                     difference specificity bending pro- <br />                                                                                     teins solely different amino acids position <br />                                                                                     34 MADS-box domain mechanism bend- <br />                                                                                     ing mutant different wild-type Mcm1                                                                                     SRF. Furthermore, results suggest mechanisms <br />                                                                                     used proteins bend DNA differs significantly <br />                                                                                     expected proteins near identity                                                                                     region protein. Interestingly, magnitude                                                                                     bending V34T T7G site similar V34A binding <br />                                                                                     mutant DNA site. mutants severe <br />                                                                                     decrease bending wild-type protein site.                                                                                     T7G substitution sterically interfere wild-type <br />                                                                                     valine chain residue 34, substitution residue <br />                                                                                     alanine relieve interference, permitting increased <br />    FIG. 4. Substitutions Mcm1 reduce DNA bending wild-type           bending comparison wild-type protein. unlikely T7G sites. EMSAs performed utilizing position-permuted frag- <br /> ments shown panel containing wild-type (B) T7G (C) binding sites   V34T substitution relieves steric interference, wild-type, V34A, S37A, V34A-S37A, K40A, V34T, T66A, R18A purified pro-          residue make alternative set contacts <br /> teins indicated. Lanes labeled B, N, H, R denote labeled fragments      partially restore bending. generated BamHI, NheI, HindIII, EcoRI enzymes, respectively, shown panel  bend angle protein-DNA complex shown                 Residues V34 S37 contact C-5 methyl group calculated based Thompson Landy relationship (26).         thymine position 8 DNA recognition site,  8      ACTON ET AL.                                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />                                                                                          Mcm1 binding site causes modest reduction level <br />                                                                                          activation wild-type protein (1). Interestingly,                                                                                          thymine contacted V34 S37 residues                                                                                          removed T8C substitution, site shows normal <br />                                                                                          levels DNA bending wild-type protein (Fig. 5B).                                                                                          result suggests alternate contacts protein <br />                                                                                          mutant site restore bending. alternate  <br />                                                                                          tacts dependent V34 S37 chains  <br />                                                                                          cause alanine substitutions residues cause decrease <br />                                                                                          DNA bending T8C site (Fig. 5B). <br />                                                                                             vivo vitro DNA sequence specificity altered <br />                                                                                          bending mutants. V34A mutant decreased levels                                                                                          DNA bending transcriptional activation, suggesting                                                                                          DNA bending important transcriptional acti- <br />                                                                                          vation protein. examine possible role                                                                                          sequence specificity DNA bending, wild type V34A <br />                                                                                          S37A mutants assayed combination T7G <br />                                                                                          T8C DNA sites transcriptional activation vivo                                                                                          DNA binding vitro (Fig. 6). wild-type protein shows <br />                                                                                          similar decreases transcriptional activation DNA-bind- <br />                                                                                          ing affinity T7G T8C DNA sites.  <br />                                                                                          T7G mutant causes large decrease DNA bending, <br />                                                                                          T8C mutant does  results consis- <br />                                                                                          tent model directly linking degree DNA bending <br />                                                                                          activation. <br />   FIG. 5. effects Mcm1 mutants DNA bending T8G T8C                         S37A protein decreased levels DNA bending mutant sites. EMSAs performed utilizing position-permuted fragments              transcriptional activation binding sites.  <br /> containing T8G   T8C (B) binding sites wild-type, V34A, V34T,             decreases activation appear correspond changes <br /> S37A, V34A-S37A purified proteins described Fig. 4. <br />                                                                                          binding affinity protein. Interestingly,                                                                                          binding affinity wild-type S37A proteins interactions contribute Mcm1-dependent DNA                                   mutant DNA sites identical, activation levels bending (24). T8G substitution Mcm1 binding site                                mutant reduced compared wild-type protein. results complete loss transcriptional activation vivo                         V34A protein, like S37A protein, decreased <br /> (1). base-pair substitution causes large decrease DNA                          bending activation sites, levels bending wild-type mutant proteins, indicating                          activation roughly correspond changes binding affinity. sequence specificity position DNA bending (Fig.                           DNA-binding affinity V34A greater 5A). contrast T8G substitution, T8C mutation                          wild-type protein mutant DNA sites, levels ac- <br />  <br />  <br />  <br />  <br />    FIG. 6. DNA-binding affinity, DNA bending, transcriptional activation wild-type, V34A, S37A proteins wild-type mutant binding sites. DNA <br /> binding symmetric consensus site wild-type Mcm1 (lanes 2 13), V34A (lanes 14 25), S37A (lanes 26 37) wild-type site (lanes 2 5, 14 17, 26 29), T7G site (lanes 6 9, 18 21, 30 33), T8C site (lanes 10 13, 22 25, 34 37). DNA-binding affinity protein site calculated binding curves shown fold decrease affinity relative wild-type protein. apparent bend angle proteins different sites derived data shown Fig. 4 5. levels transcriptional activation vivo proteins sites <br /> driving expression lacZ measured described Materials Methods shown. Values shown units &#226;?&#164;-galactosidase activity average independent isolates. <br />  VOL. 20, 2000                                                                               SCANNING MUTAGENESIS Mcm1                 9 <br />  <br />  <br /> tivation V34A sites equal           observed homeodomain proteins (7). homeodo- wild-type protein. Taken  results indicate            main proteins, majority contacts DNA DNA-binding affinity important component tran-             highly conserved alpha-helix lies major groove. <br /> scriptional activation Mcm1 simple            N51 residue, Lys Arg residues correlation absolute degree DNA bending               helix, conserved virtually homeodomain pro- <br /> activation.  fact  comparison wild-           teins, chains make set base-specific type protein, V34A S37A decreased activation               phosphate backbone contacts nearly identical mutant sites binding sites better          homeodomain-DNA structure determined. Dif- wild-type protein suggests mutations              ferences sequence-specific binding homeo- <br /> residues affect characteristic Mcm1 important           domains appear result base-specific contacts transcriptional activation does strictly correlate         nonconserved residues helix contacts DNA-binding affinity.                                                    N-terminal arm minor groove DNA. Based <br />                                                                          similarity contacts SRF Mcm1 <br />                           DISCUSSION                                     crystal structures, high degree conservation <br />                                                                          residues shown essential DNA binding, <br />    determination crystal structures SRF                appears MADS-box proteins use conserved <br /> Mcm1 bound DNA identified residues MADS-box pro-                   mechanism bind DNA. Residue K38 Mcm1, teins make base-specific phosphate backbone contacts            absolutely conserved MADS-box proteins, DNA (19, 24). performed alanine scanning                function manner similar N51 homeodomain proteins <br /> mutagenesis DNA-binding region Mcm1 examine                 making conserved base-specific contacts DNA. relative importance contacts transcrip-          chains highly conserved MADS- <br /> tional activation Mcm1 vivo DNA binding vitro.              box family proteins, residues R39, G42, K45, Alanine substitution 14 40 residues mutated          K46, make similar contacts DNA phosphate backbone results loss cell viability, indicating chains   SRF Mcm1 crystal structures essential crucial proper function Mcm1 cell.              DNA binding. residues similar role dock- <br /> expected, number chains, residues F25, I26,              ing MADS-box proteins DNA conserved lysines R32, F36, I43, M44, buried hydrophobic core            recognition helix homeodomain proteins, appear protein dimer interface. res-           position anchor helix base-specific recognition. <br /> idues directly involved protein-DNA contacts              MADS-box proteins contain conserved <br /> crystal structure, detectable levels DNA             mechanism bind DNA, site selection mutagenesis ex- <br /> binding vitro, suggests important          periments shown significant differences stability protein folding.                                            sequence specificity (9, 31). differences specificity  <br />    lethal mutations positions               tween different MADS-box proteins appear largely <br /> highly conserved MADS-box proteins contact                 different contacts N-terminal extension DNA SRF Mcm1 crystal structures (19,                 center recognition site residues coiled-coil <br /> 24). residues, K38, R39, G42, K45, K46, unable             alpha-helices interact base pairs outside  complement mcm1 null mutant detect-               served CArG site. example, position DNA  <br /> able DNA-binding activity vitro. residues             tacts N-terminal extension center site highly conserved MADS-box proteins               different SRF Mcm1 crystal structures make similar contacts DNA Mcm1 SRF                     contribute differences specificity (19, 24). crystal structures.  results mutagenesis            shown chains residues 34 37 Mcm1 contrib- residues suggest chains make             ute sequence specificity positions outside  <br /> critical contributions function protein. exam-         served CArG site. addition, residues <br /> ple, residue T35 Mcm1 position conserved           contacting DNA outside CArG box involved <br /> nearly MADS-box proteins, residue makes identi-             contacts phosphate backbone, contacts cal contacts phosphate backbone G4 DNA               contribute sequence specificity binding. example, Mcm1 SRF crystal structures (19, 24).               phosphate backbone contacts Mcm1 alanine substitution residue Mcm1 complements              occur DNA bent, results mcm1 null mutant near-wild-type levels tran-              shown ability bend sequence specific. Differences <br /> scriptional activation;  DNA-binding affinity            sequence specificity responsible broad range <br /> mutant threefold wild-type protein.        DNA bending observed various MADS-box pro- <br /> Likewise, residue T66 highly conserved MADS-               teins (29). <br /> box proteins, alanine substitution residue              effects mutations correlate <br /> little effect viability, activation, DNA-binding af-         Mcm1 crystal structure (24).  results finity. results suggest contacts           effects mutations N-terminal extension  <br /> highly conserved relative importance great        expected. Mcm1 crystal structure residue R18 making contacts common MADS-box proteins.                       base-specific contact center site. vivo <br />    results alanine scanning mutagenesis Mcm1               transcription data vitro cross-linking results suggest structural determination SRF Mcm1 indicates               R18 limited role contacting base pair. contrast, protein-DNA interactions essential roles recog-            data R19 required  making, <br /> nition binding MADS-box protein. conserved                  contact major groove center site. base-specific contacts K38, phosphate             possible explanation results structural <br /> backbone contacts K39, G42, K45, K46,                    determination Mcm1 MADS-box solved complex <br /> provide proper general framework DNA binding                  &#226;?&#163;2 protein. DNA contacts N-terminal <br /> MADS-box proteins. conserved mechanism DNA bind-                  extensions parts proteins altered ing MADS-box proteins similar             presence &#226;?&#163;2.  &#226;?&#163;2-Mcm1-DNA structure, <br />  10     ACTON ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br />  N-terminal extension trans Mcm1 monomer               DNA bend angle circular permutation assay teracting &#226;?&#163;2 monomer DNA strand)                 based assumption protein produces single <br /> makes minor groove contacts consequence inter-            bend plane.  assay, sensitive actions trans &#226;?&#163;2 monomer (24). possible expla-         relative conformational changes DNA, able nation result site used           distinguish contributions type bend experiments varies bases center site         calculated angle. V34A S37A mutants clearly alter used crystal structure. possible   conformation Mcm1-DNA complex, N-terminal extension adopts different conformations make           assay determine affect bend <br /> optimal contacts different sites. fact posi-      protein, away protein, combination tion contacts N-terminal extensions SRF Mcm1              mutants clearly DNA- <br /> differ greatly position extensions     binding affinity sole determinant Mcm1- <br /> Mcm1 monomers crystal structure significantly         mediated transcriptional activation. <br /> different suggests region considerable            alternative explanation effects V34A formational freedom. conformational freedom ex-            S37A mutants DNA contacts, tension important allowing MADS-box proteins            DNA bend, affect position conformation recognize different sites complex cofac-       chains. shown Mcm1 alters conforma- <br /> tors. Interestingly, amino acid chains 17 resi-     tion binding DNA complex &#226;?&#163;1 protein (25). <br /> dues Mcm1 phosphorylated specific conditions          Mutations residues H33 H41 little effect vivo, hypothesized different isoforms        DNA-binding affinity complex &#226;?&#163;1 result protein recognize different target sites (12).                decreased levels &#226;?&#163;1-Mcm1-dependent transcriptional acti- <br />    Alanine scanning mutagenesis identified mutants         vation (4). observations led suggestion altered DNA-bending activity: V34A, S37A, K40A.              region protein role &#226;?&#163;1-induced mutants used study mechanism            conformational change Mcm1 (4). possible V34 <br /> sequence-specific DNA bending. results examining DNA          S37 contacts DNA provide similar type bending V34A mutant combination T7G site           change structure protein altered <br /> suggests V34 contacts position 7 interaction       conformation important transcriptional activation contributes Mcm1-dependent DNA bending. DNA-                   protein. <br /> bending activity S37A mutant decreased          important mind address T8C mutation, result consistent model         activation DNA-binding activity Mcm1  interaction S37 T8 contributes DNA-bend-           cell, Mcm1 interacts different proteins regulate <br /> ing activity Mcm1. results correlate perfectly      diverse pathways. results suggest bending crystal structure Mcm1 reveal relative importance          DNA overriding factor transcriptional activa- interactions producing Mcm1-dependent             tion Mcm1  DNA bending play DNA bending. Interestingly, bending associated           important role complex factors reg- <br /> K40A mutant does appear sequence specificity,             ulate different pathways. observed SRF, bend- bending wild-type mutant sites identical.        ing Mcm1 needed recruitment finding supported Mcm1 crystal structure,        protein factors promoter regions target genes (21). <br /> shows chain involved phosphate contact           Future studies utilizing mutants help provide deep- <br /> position T8. possible V34 S37 dictate speci-       er understanding importance DNA bending tran- <br /> ficity bending K40 helps provide necessary            scriptional activation functions Mcm1 cell. <br /> change free energy stabilize DNA altered  <br /> formation. contributions chains                                  ACKNOWLEDGMENTS <br /> wild-type levels DNA bending occur. Predictions               thank George Sprague providing strains plasmids based Mcm1 crystal structure work               Tim Richmond Song Tan providing coordinates MADS-box proteins suggest K29 T66               &#226;?&#163;2-Mcm1-DNA crystal structure helpful discussions. <br /> tribute DNA bending (24, 29).  results indicate          M.S. recipient Howard Hughes Medical Institute  residues contribute little Mcm1 function        graduate Summer Research Fellowship. research supported <br /> vivo Mcm1-dependent DNA bending vitro.                          grant National Institutes Health (GM49265)    results wild-type protein ma-        K.V. <br /> jority mutants, good correlation                                                                                                        REFERENCES <br /> levels transcriptional activation DNA-binding affinity. <br />                                                                       1. Acton, T. B., H. Zhong,  K. Vershon. 1997. DNA-binding specificity  observation V34A S37A, mutants               Mcm1: operator mutations alter DNA bending transcriptional ac- <br /> alter DNA bending little effect DNA-binding                tivities MADS box protein. Mol. Cell. Biol. 17:1881&#226;&#8364;&#8220;1889. <br /> affinity, slow-growth phenotypes suggests DNA               2. Affolter, M., J. Montagne, U. Walldorf, J. Groppe, U. Kolter, M. LaRosa, <br /> bending Mcm1 important essential function               W. J. Gehring. 1994. Drosophila SRF homolog expressed                                                                          subset tracheal cells maps genomic region required protein. fact mutations Mcm1              tracheal development. Development 120:743&#226;&#8364;&#8220;753. <br /> similar greater decreases DNA-binding affinity           3. Boeke, J. D., J. Trueheart, G. Natsoulis, G. R. Fink. 1987. 5-Fluoroorotic <br /> result slow growth similar reductions transcrip-           acid selective agent yeast molecular genetics. Methods Enzymol. 154: <br /> tional activation supports idea.  results          164&#226;&#8364;&#8220;175. <br />       <br /> </body></html>