<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Dec. 1999, p. 8042&#226;&#8364;&#8220;8051                                                                                    Vol. 19,  12 <br /> 0270-7306/99/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 1999, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />            Subunit Yeast TFIIIC Participates Recruitment <br />                           TATA-Binding Protein <br />                ERIC DEPREZ,&#226;&#8364;&#160; ROSALI ARREBOLA, CHRISTINE CONESA,                                          &#194;&#180; SENTENAC* <br />                                                                                                            ANDRE <br />                Service Biochimie et Ge&#194;&#180;ne&#194;&#180;tique Mole&#194;&#180;culaire, CEA/Saclay, F-91191 Gif-sur-Yvette Cedex, France <br />                        Received 29 June 1999/Returned modification 5 August 1999/Accepted 7 September 1999 <br />  <br />            TFIIIC plays key role nucleating assembly initiation factor TFIIIB class III genes.          characterized essential gene, TFC8, encoding 60-kDa polypeptide, &#226;?&#182;60, present affinity-purified <br />          TFIIIC. Hemagglutinin-tagged variants &#226;?&#182;60 TFIIIC-tDNA complexes reside <br />          downstream DNA-binding domain &#226;?&#182;B. Unexpectedly, thermosensitive phenotype          N-terminally tagged &#226;?&#182;60 suppressed overexpression &#226;?&#182;95, belongs &#226;? domain,          TFIIIB components, TATA-binding protein (TBP) B&#226;&#180;&#8211;/TFIIIB90  TFIIIB70). Mutant TFIIIC          deficient activation certain tRNA genes vitro, transcription defect selectively alleviated <br />          increasing TBP concentration. Coimmunoprecipitation experiments support direct interaction          TBP &#226;?&#182;60. suggested &#226;?&#182;60 links &#226;? &#226;?&#182;B domains participates TFIIIB assembly          interaction TBP. <br />  <br />  <br />    primary step tRNA gene activation binding               gel retardation DNA photo-cross-linking (31). TBP stabi- <br /> TFIIIC B blocks intragenic promoter.                lizes weak interaction TFIIIB70 TFIIIC- <br /> main function assemble initiation factor TFIIIB              DNA complex complete upstream footprint upstream transcription start site (30). Yeast TFIIIC (&#226;?&#182;              characteristic stability TFIIIB-DNA complex requires <br /> factor) multisubunit protein ca. 600 kDa organized               recruitment B&#226;&#172;&#8482;/TFIIIB90 (29, 31). cascade confor- large globular domains &#226;? &#226;?&#182;B similar size mass               mational rearrangements protein DNA levels (10-nm diameter ca. 300 kDa), interacting                 accompanying assembly steps, evidenced successive <br /> promoter element, visualized electron microscopy (17,                 changes accessibility TFIIIB70, TBP, &#226;?&#182;131 53). Binding &#226;?&#182;B B block predominant favors             site-specific DNA cross-linking (31), DNA bending  <br /> binding &#226;? block (4). TFIIIC-DNA interaction                    duced TFIIIB binding (11, 38, 46), presence displays remarkable adaptability variable B dis-                 cryptic DNA binding domain TFIIIB70 (24). &#226;?&#182;131 appears <br /> tances different tRNA genes (3). Affinity-purified                 play major role positioning TFIIIB Saccharomyces cerevisiae TFIIIC comprises polypeptides               TFIIIC subunit accessible DNA cross-linking upstream 138, 131, 95, 91, 60, 55 kDa (6, 19, 47, 59). Gene cloning,             start site (5, 7) interact TFIIIB70 (14, 33) <br /> protein-DNA cross-linking, mutagenesis, protein-protein                 TFIIIB90 (51). TFIIIB effect assembly interaction studies provided global view location              TATA-containing SNR6 gene interaction role TFIIIC subunits TFIIIC-DNA com-                  TBP strong TATA box (27, 43, 45). Interestingly, <br /> plex. &#226;?&#182;138 &#226;?&#182;91 subunits reside &#226;?&#182;B domain                  Whitehall et al. (60) TBP discern operate downstream DNA binding (1, 10, 36, 37); &#226;?&#182;95                  polarity TATA element directed TFIIIB assembly &#226;?&#182;55 interact physically, belong &#226;? domain,                   orientations.  contrast TATA-depen- <br /> thought participate block binding (7, 17, 40, 48, 59).              dent assembly, TFIIIC placed TFIIIB correct orienta- <br /> &#226;?&#182;131 (42) stands TFIIIC subunit responsible TFIIIB               tion. TFIIIC component known interact assembly based genetic evidence (49, 61), upstream lo-               TBP, presumed unidirectional binding TBP <br /> cation (7), interaction TFIIIB components (14,             TATA box dictated oriented interaction 33, 51).                                                                    TFIIIB70 &#226;?&#182;131 (60). <br />    TFIIIB stable molecular entity like TFIIIC.              present work completed characterization <br /> resolved chromatographically fractions named B&#226;&#172;&#732;                TFIIIC components cloning yeast gene, named TFC8, <br /> B&#226;&#172;&#8482; (29). B&#226;&#172;&#732; comprises TATA-binding protein (TBP)                    encodes 60-kDa polypeptide. present genetic TFIIB-related factor TFIIIB70/Brf1 (12, 16, 31, 39), B&#226;&#172;&#8482;               biochemical evidence component, named &#226;?&#182;60, resides <br /> contains TFIIIB90 (32, 50, 51). TFIIIC-dependent TFIIIB                 &#226;?&#182;B domain participates TFIIIB <br /> assembly TATA class III genes multistep pathway                recruitment TBP binding. decomposed vitro (29, 31) reconstituted recombinant TFIIIB components (32, 51). order                                           MATERIALS METHODS <br /> interaction TFIIIB70, TBP, B&#226;&#172;&#8482;, shown                    Yeast strains, plasmids, genetic techniques. YNN281&#226;&#171;&#187;YNN282 (21)                                                                             used gene disruption. Cells grown standard rich medium (YPD)                                                                             minimal medium (Casamino acid medium). plasmids used suppression <br />   * Corresponding author. Mailing address: Service Biochimie et          studies described Lefebvre et al. (37). Preparation media, tetrad dis- Ge&#194;&#180;ne&#194;&#180;tique Mole&#194;&#180;culaire, CEA/Saclay, F-91191 Gif-sur-Yvette,            section, transformation lithium acetate-treated cells, plasmid shuffling                                                                             using 5-fluoro-<span id='am-21' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span>orotic acid</span> performed standard techniques (2). <br /> France. Phone: 33-1-69-08-22-36. Fax: 33-1-69-08-47-12. E-mail: <br />                                                                                <span id='am-56' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-57' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>Amino acid</span> sequence determination. polypeptide components purified <br /> sentenac@dsvidf.cea.fr.                                                     TFIIIC preparation separated preparative electrophoresis 6% <br />   &#226;&#8364;&#160; Present address: Laboratoire Physicochimie et Pharmacologie          polyacrylamide&#226;&#8364;&#8220;sodium dodecyl sulfate (SDS) gel (36). gel lightly <br /> des Macromole&#194;&#180;cules Biologiques (UMR8532), ENS Cachan,                   stained Coomassie blue. gel slice containing 60-kDa polypeptide F-94235 Cachan Cedex, France.                                               excised, crushed, subjected trypsin digestion. Preparation analysis  <br />                                                                      8042 <br />  VOL. 19, 1999                                                                                 &#226;?&#182;60 PARTICIPATES TFIIIB RECRUITMENT                              8043 <br />  <br />  <br /> tryptic peptides previously described (36, 59). <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> sequence        Amersham ECL Kit. polypeptides revealed antibodies directed TBP partially determined (??TLYLTT[T/F]PT).                                           &#226;?&#182;60 identified independent experiments. <br />    Cloning, construction plasmids, disruption TFC8. 32-mer                 DNA binding vitro transcription assays. TFIIIC-tDNA &#226;?&#182;B-tDNA <br /> oligonucleotides used amplify open reading (ORF) frame TFC8         interaction monitored gel retardation analysis described previously (25, <br /> PCR yeast genomic DNA. resulting DNA fragment labeled            37). 32P-labeled DNA fragment (3 10 fmol; 4,000 10,000 cpm) carrying <br /> [&#226;?&#163;-32P]<span id='am-68' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-69' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="dCTP" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span>dCTP</span> used screen FL100 library (57) containing yeast genomic        tRNA3Glu (198 bp) SUP4tRNATyr (375 bp) genes incubated 10 <br /> DNA fragments inserted pFL44L (2&#226;?&#174;, URA3) plasmid (9). pFL44L             min 25&#194;&#176;C 15-&#226;?&#174;l reaction mixture containing 10 mM Tris-HCl (pH 8.0), <br /> plasmid isolated hybridizing clones contain large      10% glycerol, bovine serum albumin, DNA competitor, TFIIIC (MonoQ <br /> DNA insert comprising TFC8 gene named pCC12 (pFL44L-TFC8).                fraction). final ammonium sulfate concentration 75 mM. &#226;?&#182;B-tDNA 2.3-kb SalI-FspI DNA fragment pCC12 harboring TFC8 gene              complexes obtained 10 min incubation 25&#194;&#176;C TFIIIC- <br /> cloned pUN45 creating pYED1 plasmid. 69-mer oligonucleotide            tDNA complexes 10 ng &#226;?&#163;-chymotrypsin (Sigma). Digestion stopped <br /> used introduce NdeI restriction site initiation codon TFC8, followed   addition 1 ng aprotinin (Sigma). Complexes analyzed nonde- sequence encoding hemagglutinin (HA) epitope (YPYDVPDYA)                   naturing gel electrophoresis (5% polyacrylamide). <br /> derived influenza virus HA protein (62), oligonucleotide-mediated            Transcription mixtures (40 &#226;?&#174;l) contained 20 mM HEPES (pH 8.0); 5 mM <br /> mutagenesis single-stranded pYED1 DNA using Muta-Gene kit (Bio-               MgCl2; 1 mM DTT; 0.1 mM EDTA; 5% glycerol; 8 U RNasin (Amersham); <br /> Rad), yielding pYED2 plasmid (encoding HA Nter-&#226;?&#182;60). sequence                 0.6 mM <span id='am-32' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-33' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span>ATP</span>, <span id='am-6' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>GTP</span>, <span id='am-78' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-79' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="CTP" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CTP</span>; 0.03 mM <span id='am-63' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span>UTP</span> 10 &#226;?&#174;Ci [32P]UTP; <br /> encoding HA epitope introduced stop codon TFC8          TFIIIB (370 ng Cibacron-blue fraction) rTBP (250 ng); rTFIIIB70 (1.2 &#226;?&#174;g) <br /> PCR-mediated mutagenesis oligonucleotides. contained SalI              partially purified B&#226;&#172;&#8482; fraction (1.8 &#226;?&#174;g); RNA polymerase III (Pol III; 50 ng); <br /> restriction site nucleotides complementary upstream region          TFIIIC (MonoQ fraction) indicated. final KCl concentration 110 <br /> TFC8 promoter, harbored sequence encoding HA epitope,           mM. 10 min preincubation 25&#194;&#176;C, transcription reaction initiated NotI restriction site, nucleotides complementary stop codon region       addition 130 ng plasmid DNA harboring different tRNA genes TFC8. PCR amplification, SalI-NotI DNA fragment inserted             allowed proceed 45 min 25&#194;&#176;C. Transcripts analyzed electro- corresponding sites pYED1 pYED3 (encoding HA Cter-            phoresis 8 M <span id='am-94' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-95' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>urea</span> gel (6% polyacrylamide). <br /> &#226;?&#182;60). NdeI-BamHI DNA fragment pYED2 inserted                       vivo labeling RNAs Northern blot analysis. RNA labeling corresponding sites pET28c vector (Novagen), yielding pET60.                       [3H]<span id='am-73' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-74' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span> strains ura3 mutation complemented    Disruption TFC8 gene performed previously described (8, 40).         URA3 plasmid pFL44L. Cells exponentially grown uracil-free Casamino <br />                                                                                       acid medium supplemented <span id='am-26' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span> (Casa&#226;&#171;&#185;Ade) optical density 55-mer oligonucleotides harboring sequences complementary TFC8                                                                                       0.4 600 nm.  150 &#226;?&#174;Ci [3H]uracil added 10 ml culture 15 yeast HIS3 selectable marker used amplify PCR &#226;&#172;&#402;1.1-kb <br />                                                                                       min. cells harvested chilled 10 ml ice-cold sterile <br /> DNA fragment containing HIS3 gene flanked TFC8 promoter termi- <br />                                                                                       water. RNAs extracted previously described (22, 52). Small RNA species <br /> nator sequences. PCR-amplified DNA fragment used transform                                                                                       analyzed loading separating equal amounts RNA (6 &#226;?&#174;g lane) <br /> strain YNN281&#226;&#171;&#187;YNN282. structure  diploids verified                                                                                       7 M urea gel electrophoresis (6% polyacrylamide). <br /> PCR analysis. determine TFC8 essential growth, sporulation <br />                                                                                          RNA extraction gel electrophoresis Northern blot analysis  dissection analysis performed. diploid  strain trans- <br />                                                                                       formed described previous section. Electrophoretic transfer nylon <br /> formed pCC12 plasmid (pFL44L-TFC8) sporulated.  spore              membrane (Bio-Rad apparatus), UV cross-linking DNA (Stratalinker appara- <br /> containing pCC12 plasmid chosen strain YCC8 used plasmid          tus), hybridization DNA probe sodium phosphate buffer carried <br /> shuffling.                                                                            previously described (15). DNA probe used Northern blot <br />    Purification TFIIIC. TFIIIC purified starting &#226;&#172;&#402;12 g S. cerevisiae   experiment shown Fig. 2B 327-bp 32P-labeled PCR fragment encom- <br /> cells, using fast-protein liquid chromatography-grade resins. preparation      passing yeast tRNA3Leu gene amplified pGE2 plasmid. cell extract described Huet et al. (25). Crude extract diluted 0.25 M ammonium sulfate   buffer (20 mM Tris-HCl, pH 8.0; <br /> 0.5 mM EDTA; 10 mM &#226;?&#164;-mercaptoethanol; 10% [vol/vol] glycerol)                                                  RESULTS <br /> loaded 2.5 ml/min 25-ml heparin Hyper D (BioSepra) column previously <br /> equilibrated buffer (0.25 M . resin washed 5 ml/min            TFC8 encodes &#226;?&#182;60, 60-kDa subunit yeast TFIIIC. 250 ml buffer (0.35 M . linear gradient 0.35 0.70        TFIIIC S. cerevisiae comprises polypeptides 138, <br /> M 180 ml buffer applied 2.5 ml/min. Fractions (2 ml)         131, 95, 91, 60, 55 kDa. order clone gene encod- <br /> collected assayed TFIIIC-DNA binding activity. TFIIIC-containing frac- <br /> tions (0.45 0.55 M  pooled dialyzed buffer (0.07 M .      ing hypothetical &#226;?&#182;60 subunit, polypeptide purified <br /> Proteins loaded 0.5 ml/min 1-ml MonoQ column (Pharmacia,            SDS-PAGE subjected tryptic digestion. se- <br /> Piscataway, N.J.) previously equilibrated buffer (0.07 M . column       quence peptide determined (??TLYLTT washed 0.5 ml/min 20 ml buffer (0.07 M . linear gradient        [T/F]PT). compared DNA sequences databases 0.07 0.4 M 15 ml buffer applied 0.5 ml/min. <br /> Fractions (200 &#226;?&#174;l) containing TFIIIC-DNA binding activity eluted                                                                                       using BLAST program, peptide sequence 0.24 0.30 M  Based Western blot experiments anti-&#226;?&#182;55 anti-&#226;?&#182;60       slight variation (DGTLYLTTFPD) unique yeast <br /> antibodies, TFIIIC preparation Ntag-&#226;?&#182;60 mutant cells            hypothetical protein 588 residues, theoretical molec- <br /> contain half factor wild type.                                         ular weight 67,640 isoelectric point 5.87. gene <br />    Expression purification &#226;?&#182;60 Escherichia coli. Recombinant Tfc8p          encoding protein unique, maps chromosome XVI <br /> tagged N-terminal end histidines HA epitope (HA- <br /> r&#226;?&#182;60) expressed plasmid pET60 E. coli BL21 (pLysS). Cell culture,         (13), shows similarity sequences EMBL/ <br /> protein induction, extract preparation performed described earlier        GenBank data bank (S. cerevisiae, Caenorhabditis elegans, (1). Crude cell extract loaded 1 ml/min 5 ml Ni2&#226;&#171;&#185;-charged HiTrap       current versions Arabidopsis thaliana, Homo sapiens, chelating column (Pharmacia) previously equilibrated 20 mM HEPES (pH             Schizosaccharomyces pombe genomic sequences). gene, <br /> 7.8), 300 mM NaCl, 10% glycerol containing 10 mM imidazole. Proteins eluted linear gradient 60 ml buffer containing 20 270 <br />                                                                                       previously designated YPL007C, named TFC8. <br /> mM imidazole 1 ml/min. Fractions (1 ml) assayed HA-r&#226;?&#182;60                   investigate TFC8 essential growth, Western blotting anti-&#226;?&#182;60 antibodies. Fractions containing HA-r&#226;?&#182;60          chromosomal copy TFC8 gene deleted diploid <br /> eluted concentrations 160 190 mM imidazole.                            strain YNN281&#226;&#171;&#187;YNN282 replacing ORF yeast <br />    Anti-&#226;?&#182;60 polyclonal antibodies. 14-mer peptide (N-12-M) encompassing C-terminal amino acid residues &#226;?&#182;60 synthesized (Neosystem, Strasbourg, <br />                                                                                       HIS3 selectable marker. Sporulation tetrad analysis  <br /> France) conjugated maleimide-activated keyhole limpet hemocyanin               vealed viable  spores nonviable spores, sug- <br /> (KLH). N-12-M&#226;&#8364;&#8220;KLH conjugate injected rabbits antibody               gesting TFC8 essential gene. confirm  <br /> production injections week intervals.                         clusion, diploid  strain  contains copy    Immunoprecipitation experiments. rTBP (200 ng) rTBP (200             TFC8 gene) transformed 2&#226;?&#174; plasmid, pCC12, har- <br /> ng) HA-r&#226;?&#182;60 (650 ng) preincubated 90 min 25&#194;&#176;C 40 &#226;?&#174;l buffer containing 20 mM HEPES (pH 8.0), 5 mM MgCl2, 1 mM dithiothreitol (DTT),             boring TFC8 gene, sporulated. resulting  <br /> 0.1 mM EDTA, 5% glycerol, 0.05% NP-40, 110 mM KCl. Magnetic beads                 haploid strains containing pCC12 plasmid TFC8 <br /> (Dynabeads/Dynal) coated 12CA5 antibody added mixtures.              gene able grow. results demonstrated  like beads incubated gentle shaking 3 h 10&#194;&#176;C washed          previously characterized genes encoding times buffer  Bound proteins eluted boiling Laemmli <br /> sample buffer, separated SDS-polyacrylamide gel electrophoresis (PAGE) <br />                                                                                       TFIIIC subunits (1, 36, 40, 42, 48, 59), TFC8 required cell analyzed Western blotting using mixture polyclonal antibodies          viability. <br /> directed TBP &#226;?&#182;60. Bound antibodies revealed using                  demonstrate presence TFC8 gene product  8044     DEPREZ ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />  <br /> TFIIIC factor, sequence encoding HA epitope fused 5&#226;&#172;&#732; 3&#226;&#172;&#732; end TFC8. haploid strain lacked chromosomal TFC8 gene expressed HA <br /> C-terminally tagged &#226;?&#182;60 (Ctag-&#226;?&#182;60) grew normally. sequence encoding HA epitope fused just initiation codon, growth haploid strain expressing N-terminally tagged &#226;?&#182;60 (Ntag-&#226;?&#182;60) slightly affected 30&#194;&#176;C  mutant cells having doubling time 130 min liquid medium, instead 110 min wild-type strain). <br /> Furthermore, Ntag-&#226;?&#182;60 strain temperature sensitive. 37&#194;&#176;C, cells grew apparent doubling time 190 <br /> min approximately 16 18 h cell death occurred. <br />    TFIIIC factor purified strains, strain expressing HA-tagged version &#226;?&#182;138 (36). Pre- <br /> formed TFIIIC-tDNA3Glu complexes incubated 30 <br /> min 25&#194;&#176;C increasing concentrations anti-HA mono- <br /> clonal antibody analyzed gel retardation experiments <br /> (Fig. 1A). anti-HA antibody clearly altered migration <br /> rate Ctag-&#226;?&#182;60 TFIIIC-tDNA3Glu complexes (compare <br /> lanes 6 7) extent HA-&#226;?&#182;138-containing com- <br /> plexes (lanes 11 12). hand, migration rate Ntag-&#226;?&#182;60 TFIIIC-tDNA3Glu complexes altered <br /> (lanes 9 10), presence 1 &#226;?&#174;g anti-HA antibody (data shown), HA epitope buried inaccessible antibody. Note  contrast previous results TFIIIC containing HA-tagged <br /> &#226;?&#182;131 (42) &#226;?&#182;138 subunits  lanes 11 12 reference <br /> 36), binding anti-HA antibody Ctag-&#226;?&#182;60 sub- <br /> unit appeared cause dissociation TFIIIC- <br /> tDNA3Glu complexes. <br />    gain insight localization &#226;?&#182;60 TFIIIC, performed limited proteolysis TFIIIC-tDNA <br /> complexes &#226;?&#163;-chymotrypsin. Limited proteolysis yeast <br /> TFIIIC causes separation transcription factor          FIG. 1. TFC8 encodes 60-kDa subunit yeast TFIIIC.   TFIIIC domains ca. 300 kDa  called &#226;? &#226;?&#182;B (44).         purified haploid strains expressing HA-tagged versions TFC3 (HA-&#226;?&#182;138) <br /> &#226;?&#182;B domain generated proteolysis forms stable complex           TFC8 (Ctag-&#226;?&#182;60 Ntag-&#226;?&#182;60, tagged C-terminal N-terminal end B block visualized gel shift assays     Tfc8p, respectively). TFIIIC preincubated 32P-labeled DNA fragment <br />                                                                     containing tRNA3Glu gene 10 min 25&#194;&#176;C incubated <br /> supershifted addition anti-&#226;?&#182;138 polyclonal antibodies     various amounts 12CA5 monoclonal antibody directed HA epitope <br /> (19). similar result obtained present study         30 min 25&#194;&#176;C. Protein-tDNA complexes analyzed gel retardation <br /> anti-HA antibodies neatly supershifted tagged-&#226;?&#182;138 &#226;?&#182;B-tDNA          assay revealed autoradiography. Lanes 1 8, TFIIIC, antibody; <br /> complexes (Fig. 1B, lanes 7 8). &#226;?&#182;B-tDNA complexes          lanes 2, 7, 9, 11, antibody; lanes 3 6, 10, 12, addition 12CA5 <br />                                                                     monoclonal antibody, indicated. (B) &#226;?&#182;B, protease-resistant domain obtained limited proteolysis Ctag-&#226;?&#182;60 TFIIIC-tDNA          TFIIIC, HA-tagged &#226;?&#182;138 &#226;?&#182;60, obtained digestion TFIIIC- <br /> complexes incubated increasing amounts anti-HA         tDNA complex 10 ng &#226;?&#163;-chymotrypsin 10 min 25&#194;&#176;C. Digestion antibodies, defined supershifted band complex ob-         stopped 1 ng aprotinin addition 12CA5 antibody. Left <br /> served, antibodies reduced migration rate &#226;?&#182;B-        panel, C-terminally HA-tagged TFC8 gene (Ctag-&#226;?&#182;60); right panel, HA-tagged <br />                                                                     TFC3 gene (HA-&#226;?&#182;138) N-terminally HA-tagged TFC8 gene (Ntag-&#226;?&#182;60). Lanes <br /> tDNA complexes, causing marked trailing band (Fig.         1 9, control TFIIIC-tDNA complex; lanes 2, 7, 10, &#226;?&#182;B generated 1B, lanes 2 6). weak accessibility partial proteolytic    proteolysis; lanes 3 6, 8, 11, incubation &#226;?&#182;B-tDNA complex various <br /> degradation HA epitope possibly caused dissoci-         amounts 12CA5 monoclonal antibody, indicated. &#226;?&#182;, TFIIIC-tDNA complex; <br /> ation immune complex electrophoresis.          &#226;?&#182;-IgG, immunoglobulin G (IgG)-TFIIIC-tDNA ternary complex; &#226;?&#182;B, &#226;?&#182;B-tDNA <br />                                                                     complex; &#226;?&#182;B-IgG, IgG-&#226;?&#182;B-tDNA ternary complex. <br /> phenomenon observed Ntag-&#226;?&#182;60 &#226;?&#182;B-tDNA com- <br /> plexes (lanes 10 11), untagged &#226;?&#182;B-tDNA com- <br /> plexes (data shown), yield migration rate unaffected monoclonal antibody. Altogether, gel        served RNA labeling pattern wild-type mu- <br /> shift experiments indicated polypeptide encoded         tant cells grown 30&#194;&#176;C (lanes 1 2). contrast, TFC8 TFIIIC C-terminal end      shifted 6 12 h 37&#194;&#176;C, mutation altered labeling <br /> located &#226;?&#182;B domain.                                           pattern tRNAs. smear radioactivity detected just <br />    vivo characterization Ntag-&#226;?&#182;60 mutant. took         labeled tRNA bands (lanes 4 5), advantage thermosensitive phenotype Ntag-&#226;?&#182;60          correspond partially matured tRNAs, 12 h mutant explore role TFC8 gene product              labeling tRNA bands selectively reduced (lane 5). <br /> TFIIIC. effect mutation Pol III transcripts vivo   Clearly, fusion HA epitope &#226;?&#182;60 N-terminal shown Fig. 2A. wild-type mutant cells grown      extremity disturbed Pol III transcription  Like 30&#194;&#176;C shifted nonpermissive temperature        previous studies different Pol III mutants, syn- <br /> (37&#194;&#176;C) 6 12 h. cells grown 30 37&#194;&#176;C            thesis 5S RNA did decrease significantly (20, 41). <br /> incubated temperature tritiated uracil 15        similar tRNA maturation defect previously ob- <br /> min; RNAs extracted, separated 7 M urea          served thermosensitive mutants TFIIIC subunits, <br /> gel, analyzed autoradiography. difference ob-         &#226;?&#182;138 (37), &#226;?&#182;131 (56)  recently, &#226;?&#182;91 (1a).  VOL. 19, 1999                                                                              &#226;?&#182;60 PARTICIPATES TFIIIB RECRUITMENT                             8045 <br />  <br />  <br />  <br />  <br />                                                                                       FIG. 3. Suppressibility Ntag-&#226;?&#182;60 mutation overexpression Pol III- <br />                                                                                    related genes. Stationary-phase cultures mutant cells harboring different mul- <br />                                                                                    ticopy plasmids diluted 10-, 102-, 103-fold water spotted (5 &#226;?&#174;l) <br />                                                                                    solid minimum medium (Casa&#226;&#171;&#185;Ade plates). plates incubated                                                                                    4 days 37&#194;&#176;C. different genes harbored high-copy-number vector <br />                                                                                    pFL44L indicated. &#226;&#171;&#186;, plasmid insert. RPR1 encodes RNA  <br />                                                                                    factor RNase P. <br />  <br />  <br />  <br />  <br />                                                                                    explore possible relationship maturation defect, <br />                                                                                    TFIIIC function cell lethality phenotype,  <br />                                                                                    formed Northern blot analysis various tRNA precursors. <br />                                                                                    Total RNA extracted wild-type mutant strains <br />                                                                                    grown 8 h 37&#194;&#176;C, separated electrophoresis                                                                                    denaturing conditions, transferred membrane,                                                                                    probed various labeled tRNA genes (Fig. 2B). wild- <br />                                                                                    type mutant strain contained multicopy plasmid <br />                                                                                    pFL44L, mutant strains contained pFL44L <br />                                                                                    harboring RPR1 SPT15 (TBP). precursor pattern <br />                                                                                    tRNA3Leu mutant characterized presence <br />                                                                                    extra RNA band absent wild-type extract (compare <br />                                                                                    lanes 1 2). similar result obtained labeled- <br />                                                                                    SUP4tDNATyr tDNA3Glu (data shown). addi- <br />                                                                                    tional RNA species previously observed RNase P <br />                                                                                    mutants shown correspond unmatured spliced <br />                                                                                    tRNA (18, 35, 58). RNase P endonuclease cleaves <br />                                                                                    pre-tRNA substrates yield mature 5&#226;&#172;&#732; extremity.                                                                                    RNA component RNaseP, RPR1, overexpressed                                                                                    mutant strain, anomalous RNA species disappeared (lane <br />                                                                                    3).  mutant cells overexpressing RPR1 retained <br />                                                                                    thermosensitive lethal phenotype  Fig. 3). contrast <br />                                                                                    RPR1, overexpression TBP did change tRNA <br />                                                                                    maturation pattern mutant (Fig. 2, lane 4) sup- <br />                                                                                    pressed lethality  . direct relationship                                                                                    maturation defect thermosensitive phenotype                                                                                    Ntag-&#226;?&#182;60 mutant excluded. tRNA <br />                                                                                    maturation problem likely consequence reduced <br />    FIG. 2. Comparison vivo tRNA synthesis wild-type mutant            synthesis class III RPR1 RNA, class III <br /> (Ntag-&#226;?&#182;60) yeast strains.   Autoradiogram vivo pulse-labeling exper- <br /> iments. [3H]uracil incorporation wild-type mutant strains    transcripts (34). <br /> 15 min 30&#194;&#176;C (lane 1, wild-type; lane 2, mutant) 37&#194;&#176;C 6 h (lane 4,      looked multicopy suppressors compensate <br /> mutant) 12 h (lane 3, wild-type; lane 5, mutant) growth 37&#194;&#176;C. RNA         interaction defects TFIIIC preinitiation com- <br /> species analyzed electrophoresis 7 M urea gel (6% polyacrylamide)     plex. Ntag-&#226;?&#182;60 mutant cells transformed high- using equal amounts RNA (6 &#226;?&#174;g lane). asterisk indicates trailing radioactive material tRNA species, suggesting defect    copy-number plasmids harboring different genes Pol III <br /> tRNA maturation. (B) Northern blot analysis tRNA precursors wild-type      transcription  series cell dilutions plated mutant cells labeled tDNA3Leu probe. RNA extraction      minimal medium incubated permissive nonpermis- <br /> 8 h growth 37&#194;&#176;C. Gel electrophoresis, electrophoretic transfer nylon      sive temperature. TFIIIC genes, TFC8, <br /> membrane, hybridization DNA probe described Mate- <br /> rials Methods. positions primary transcript tRNA3Leu, 5&#226;&#172;&#732;-    TFC1 (encoding &#226;?&#182;95) acted multicopy suppressor 3&#226;&#172;&#732;-processed nonspliced precursor, spliced 5&#226;&#172;&#732;- 3&#226;&#172;&#732;-unproc-      (Fig. 3 Table 1). suppressors tsv115 muta- <br /> essed precursor, mature tRNA3Leu indicated right. Lanes 1       tion TFC3, RPR1, NOP1, FHL1 (37), involved 2, wild-type mutant cells, respectively, containing high-copy-number   maturation stable RNAs, able suppress plasmid pFL44L; lane 3, mutant cells containing high-copy-number plasmid <br /> pFL44L harboring RPR1 gene; lane 4, mutant cells containing pFL44L harboring       thermosensitive defect mutant cells (Table 1). Figure 3 <br /> SPT15 gene (TBP).                                                                  shows high dosage TFIIIB genes, TFC5 <br />                                                                                    (TFIIIB90), SPT15 (TBP), BRF1 (TFIIIB70), sup- <br />  8046        DEPREZ ET AL.                                                                                                      MOL. CELL. BIOL. <br />  <br />  <br />        TABLE 1. Suppression &#226;?&#182;60 mutation gene dosage <br />                                                  Suppression  <br /> Multicopy <br />                       Gene product            Minimal       Rich <br />  gene <br />                                               medium       medium <br />  <br />  TFC8         &#226;?&#182;60                             &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;           &#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185; <br />  RPR1         RNA component RNase P         &#226;&#171;&#186;             &#226;&#171;&#186; <br />                 (tRNA maturation) <br />  NOP1         Fibrillarin homolog (rRNA         &#226;&#171;&#186;             &#226;&#171;&#186; <br />                 maturation) <br />  FHL1         Forkhead-like protein (rRNA       &#226;&#171;&#186;             &#226;&#171;&#186; <br />                 maturation) <br />  TFC3         &#226;?&#182;138, subunit TFIIIC          &#226;&#171;&#186;              &#226;&#171;&#186; <br />  TFC4         &#226;?&#182;131, subunit TFIIIC          &#226;&#171;&#186;              &#226;&#171;&#186; <br />  TFC1         &#226;?&#182;95, subunit TFIIIC           &#226;&#171;&#185;&#226;&#171;&#185;             &#226;&#171;&#186; <br />  TFC6         &#226;?&#182;91, subunit TFIIIC           &#226;&#171;&#186;              &#226;&#171;&#186; <br />  TFC7         &#226;?&#182;55, subunit TFIIIC           &#226;&#171;&#186;              &#226;&#171;&#186; <br />  BRF1         Brf1/TFIIIB70, subunit        &#226;&#171;&#186;              &#226;&#171;&#186; <br />                 TFIIIB <br />  SPT15        TBP (TATA-binding                &#226;&#171;&#185;&#226;&#171;&#185;             &#226;&#171;&#185; <br />                 protein), subunit TFIIIB <br />  TFC5         B&#226;&#172;&#8482;/TFIIIB90, subunit          &#226;&#171;&#185;&#226;&#171;&#185;             &#226;&#171;&#186; <br />                 TFIIIB <br />  <br />  <br />  <br />  <br /> pressed lethal phenotype Ntag-&#226;?&#182;60 mutant cells min- <br /> imal medium 37&#194;&#176;C. Remarkably, overproduction SPT15 (TBP) restore cell growth rich YPD medium nonpermissive temperature  suppression level weaker conditions observed min-            FIG. 4. DNA-binding properties wild-type mutant TFIIIC.   <br /> imal medium, Table 1).                                          TFIIIC-tDNA complex formation tRNA3Glu SUP4tRNATyr genes. <br />    vitro characterization Ntag-&#226;?&#182;60 mutant. TFIIIC         TFIIIC-DNA complexes formed Ntag-&#226;?&#182;60 wild-type TFIIIC prep- <br />                                                                     arations (MonoQ fraction) analyzed electrophoresis described factor partially purified wild-type strain      Materials Methods. add similar amounts wild-type mutant TFIIIC strain expressing HA N-terminal version &#226;?&#182;60           (based Western blot experiments), twice mutant protein fraction compare DNA binding properties. TFIIIC fractions pu-          added follows: 25 ng (lanes 2 7) 100 ng (lanes 3 8) wild-type <br /> rified mutant strain reproducibly West-      TFIIIC 50 ng (lanes 4 9) 200 ng (lanes 5 10) mutant TFIIIC <br />                                                                     fraction. Lanes 1 5, tRNA3Glu gene; lanes 6 10, SUP4tRNATyr gene. (B) <br /> ern blotting anti-&#226;?&#182;60 anti-&#226;?&#182;55 polyclonal antibodies       Effect temperature &#226;?&#182;B-tDNA complex stability. Preformed TFIIIC- <br /> (40) contain half TFIIIC wild-type fractions.         tDNA3Glu complexes digested &#226;?&#163;-chymotrypsin generate &#226;?&#182;B-DNA  care taken use wild-type       complexes, proteolytic digestion stopped addition aprotinin, mutant TFIIIC following gel retardation experiments.      mixtures incubated 10 min various temperatures                                                                     indicated. Lanes 1 8, wild-type mutant TFIIIC-tDNA complexes,  labeled probes, tDNA3Glu SUP4tDNATyr, incu-             spectively; lanes 2 7, wild-type &#226;?&#182;B; lanes 9 14, mutant &#226;?&#182;B. &#226;?&#182;, TFIIIC-tDNA <br /> bated limiting amounts wild-type mutant TFIIIC.          complex; &#226;?&#182;B, &#226;?&#182;B-tDNA complex. <br /> TFIIIC-tDNA complexes analyzed electrophoresis  <br /> der nondenaturing conditions revealed autoradiogra- <br /> phy. Figure 4A shows yields TFIIIC-DNA com- <br /> plexes formed wild-type mutant TFIIIC factor                Wild-type mutant TFIIIC compared tDNA3Glu SUP4tDNATyr similar. addition,             ability direct specific transcription various tRNA genes wild-type mutant TFIIIC-tDNA complex stabilities var-        vitro. Identical amounts TFIIIC preparations, cali- <br /> ious temperatures ionic strengths indistinguish-       brated Western blot analysis anti-&#226;?&#182;60 anti-&#226;?&#182;55 <br /> able (data shown). contrast, TFIIIC-tDNA3Glu            polyclonal antibodies gel retardation analysis complexes subjected limited proteolysis, significantly      tDNA3Glu SUP4tDNATyr probes, used reconstituted <br /> fewer mutant &#226;?&#182;B-tDNA complexes obtained compared            transcription assays presence B&#226;&#172;&#8482; fraction, recombinant wild type (Fig. 4B, compare lanes 2 9).            TBP (rTBP), TFIIIB70 (rTFIIIB70), purified RNA Pol <br /> mutant &#226;?&#182;B-tDNA complexes stable wild type         III. shown Fig. 5, transcriptional activity mutant <br /> increasing temperatures. mutant &#226;?&#182;B-tDNA               TFIIIC similar wild-type factor complexes decreased drastically 10 min 55&#194;&#176;C 5%       tDNA3Glu, tDNA2Lys, tDNA1Ile templates (lanes 1 2, initial control level 25&#194;&#176;C, 60% wild-type &#226;?&#182;B-       7 8, 11 12). Surprisingly, transcriptional ac- <br /> tDNA complexes resisted conditions (lanes 6          tivity mutant factor impaired tem- 13), determined using PhosphorImager Image-        plates, SUP4tDNATyr, tDNA1Phe, tDNA1Pro (lanes <br /> Quant software (Molecular Dynamics). struc-            3 6 9 10).  shown previously, mutant <br /> tural defect causing increased thermolability        wild-type factor preparations bound similarly tDNA3Glu <br /> presence HA epitope subtle difference pro-        SUP4tDNATyr probes  . difference teolysis indirectly caused presence epitope),   DNA-binding properties account differ- <br /> observations clearly revealed defect &#226;?&#182;B level, enforcing   ence transcription efficiency, explored ability mu- idea N-terminal &#226;?&#182;60 belongs B           tant TFIIIC recruit TFIIIB. used tDNA3Leu,  <br /> block-binding domain.                                               like SUP4tDNATyr, poor template, mutant TFIIIC. <br />  VOL. 19, 1999                                                                             &#226;?&#182;60 PARTICIPATES TFIIIB RECRUITMENT                  8047 <br />  <br />  <br />                                                                                    tDNA3Glu template (Fig. 6E).  inferred                                                                                    TBP concentration standard transcription assay                                                                                    optimal tDNA3Glu, tDNA2Lys, tDNA1Ile limiting <br />                                                                                    SUP4tDNATyr, tDNA1Phe, tDNA1Pro, tDNA3Leu.                                                                                    analysis promoter region genes did disclose <br />                                                                                    feature TFIIIB binding region account <br />                                                                                    differential TBP requirement. <br />                                                                                       &#226;?&#182;60 interacts TBP. lines evidence indicated                                                                                    mutant TFIIIC defective TBP recruitment, vivo <br />                                                                                     multicopy suppressor studies) vitro  transcrip- <br />                                                                                    tion experiments). explored possibility                                                                                    &#226;?&#182;60 subunit directly participate TBP binding.                                                                                    physical interaction &#226;?&#182;60 TBP investigated                                                                                    coimmunoprecipitation experiments recombinant pro- <br />                                                                                    teins. rTBP (200 ng) preincubated 90 min 25&#194;&#176;C    FIG. 5. Specific transcription various tRNA genes presence         HA-r&#226;?&#182;60 (650 ng) transcription buffer. protein samples <br /> wild-type mutant TFIIIC. vitro transcription performed described Materials Methods using 100 200 ng wild-type Ntag-&#226;?&#182;60            added magnetic beads precoated anti-HA <br /> MonoQ-TFIIIC fraction (respectively), rTBP, rTFIIIB70, partially purified B&#226;&#172;&#8482;       antibodies, beads washed buffer, fraction, RNA Pol III. different plasmid templates indicated. Odd      bound proteins eluted SDS competition lanes, wild-type TFIIIC. lanes, mutant TFIIIC.                                HA peptide. input eluted proteins subse- <br />                                                                                    quently analyzed SDS-PAGE immunoblotting  <br />                                                                                    ti-TBP anti-&#226;?&#182;60 antibodies (Fig. 7). Compared low <br /> Wild-type mutant TFIIIC similar DNA-binding                            background level TBP retained beads absence <br /> properties tDNA3Leu (data shown). Transcription                         &#226;?&#182;60, sevenfold-higher TBP bound performed presence increasing amounts TFIIIB                          preincubation &#226;?&#182;60, quantified using ImageQuant wild-type mutant TFIIIC. Remark-                        software. similar result obtained bound proteins <br /> ably, shown Fig. 6A, high doses TFIIIB fraction                     eluted competition HA peptide compensate transcription defect mutant TFIIIC.                      beads washed extensively transcription buffer <br /> relative transcriptional activity mutant TFIIIC reached 92%                     (data shown). background-corrected TBP/Tfc8 ratio wild-type level 19 ng TFIIIB &#226;?&#174;l, concentration                  eluted fraction quantified corresponded TFIIIB longer limiting wild-type TFIIIC                       0.56 (compared 0.9 input), showing mole- <br />  maximum tRNA synthesis presence wild-type                        cule bound &#226;?&#182;60 specifically associated 0.5 molecule TFIIIC reached ca. 15 ng TFIIIB &#226;?&#174;l).                                 TBP. result, tentatively estimate Kd TBP- <br />    explore TFIIIB component critical TFIIIB                       &#226;?&#182;60 interaction ca. 100 nM, indicative assembly mutant TFIIIC, performed multiple-round                             reasonably strong interaction. Kd reported transcriptions tDNA3Leu template various                              TFIIIB70-TBP interaction Sethy-Coraci et al. (55). amounts rTFIIIB70 (Fig. 6B), B&#226;&#172;&#8482; (Fig. 6C), rTBP (Fig.                        observations strongly suggested direct interaction 6D). Increasing concentration rTFIIIB70 did correct                     &#226;?&#182;60 subunit TBP. similar experiment, physical transcription defect mutant TFIIIC (Fig. 6B). 12-                   interaction detected r&#226;?&#182;60 rTFIIIB70: <br /> 48-ng/&#226;?&#174;l concentration range rTFIIIB70 factor, tRNA syn-                        Western blot analysis revealed significant difference thesis increased similarly wild-type mutant                           amounts rTFIIIB70 bound beads presence TFIIIC, relative transcriptional efficiency mutant                      absence HA-r&#226;?&#182;60 (data shown). <br /> TFIIIC remained low constant (8 16%). concen- <br /> tration 48 ng/&#226;?&#174;l, rTFIIIB70 appeared titrate component transcription  causing dramatic                                                    DISCUSSION <br /> decrease tRNA synthesis directed wild-type mutant TFIIIC. shown Fig. 6C, increasing concen-                            report characterization 60-kDa polypep- <br /> tration B&#226;&#172;&#8482; 11- 45-ng/&#226;?&#174;l concentration range did                    tide present affinity-purified TFIIIC. polypeptide, correct transcription B&#226;&#172;&#8482; 11- 45-ng/&#226;?&#174;l concen-                   named &#226;?&#182;60, shown essential subunit TFIIIC. tration range did correct transcription defect                     yeast TFIIIC factor,  comprises subunits, &#226;?&#182;138, <br /> mutant TFIIIC. concentration B&#226;&#172;&#8482; &#226;&#172;&#381;45 ng/&#226;?&#174;l,                          &#226;?&#182;131, &#226;?&#182;95, &#226;?&#182;91, &#226;?&#182;60, &#226;?&#182;55, essential cell <br /> transcription efficiency wild-type mutant TFIIIC                       viability (Table 2). &#226;?&#182;60 appears reside,  &#226;?&#182;B, <br /> decreased similarly (Fig. 6C). dosage-dependent effect                      downstream DNA-binding domain. Paradoxically,  <br /> rTBP transcriptional efficiency totally different (Fig.                      analysis &#226;?&#182;60 mutant indicated &#226;?&#182;60 plays role 6D). concentration range 2.5 8 ng rTBP factor                    TFIIIB assembly interaction TBP. results sug- <br /> &#226;?&#174;l, transcriptional efficiency optimal wild-type                       gest network interactions linking TFIIIC TFIIIB com- <br /> TFIIIC. contrast, mutant TFIIIC, tRNA synthesis                        ponents preinitiation complex formation. <br /> rate increased rTBP concentration reach 97%                            demonstration &#226;?&#182;60 integral subunit TFIIIC <br /> wild-type level. result indicated rTBP limiting                  rests biochemical genetic evidence.  antibodies <br /> component TFIIIB assembly directed mutant TFIIIC,                        directed C-terminally HA-tagged &#226;?&#182;60 altered migration tDNA3Leu.                                                                 tagged TFIIIC-tDNA &#226;?&#182;B-tDNA complexes. Second,    observation tRNA genes normally                           thermosensitive mutant TFIIIC (harboring N-terminally HA- <br /> transcribed presence mutant TFIIIC (Fig. 5) sug-                         tagged &#226;?&#182;60) affected tRNA gene transcription vivo <br /> gested templates different TBP require-                      vitro.  vivo phenotype suppressed ments.  decreasing TBP concentra-                         overexpression components class III transcription <br /> tion revealed defect mutant TFIIIC transcription                    machinery  TFIIIC subunit, &#226;?&#182;95, components  8048       DEPREZ ET AL.                                                                                                                            MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 6. Effects TFIIIB, TFIIIB70, B&#226;&#172;&#8482;, TBP concentration transcription tRNA3Leu tRNA3Glu genes presence wild-type mutant TFIIIC. <br />   Effect TFIIIB. Transcription mixtures (40 &#226;?&#174;l) contained 100 ng wild-type 200 ng Ntag-&#226;?&#182;60 TFIIIC, 220 ng plasmid DNA harboring tRNA3Leu <br /> gene, RNA polymerase, various concentrations TFIIIB (Cibacron blue fraction) indicated. (B) Effect TFIIIB70. Transcription mixtures contained wild-type mutant TFIIIC, plasmid DNA, RNA polymerase panel  2.5 ng rTBP &#226;?&#174;l, 45 ng partially purified B&#226;&#172;&#8482; fraction &#226;?&#174;l, various concentrations rTFIIIB70 indicated. (C) Effect B&#226;&#172;&#8482;. Transcription mixtures contained wild-type mutant TFIIIC, plasmid DNA, RNA polymerase panel  4 ng rTBP &#226;?&#174;l, 35 ng rTFIIIB70 &#226;?&#174;l, various concentrations partially purified B&#226;&#172;&#8482; fraction indicated. (D) Effect TBP. Lanes panels, 35 ng rTFIIIB70 &#226;?&#174;l, 45 ng partially purified B&#226;&#172;&#8482; fraction &#226;?&#174;l, various concentrations rTBP indicated. (E) Effect TBP. Lanes described panel D, plasmid DNA harboring tRNA3Glu gene instead tRNA3Leu gene used template. Transcripts analyzed electrophoresis autoradiography (left panels). Odd lanes, wild-type TFIIIC; lanes, mutant TFIIIC. Transcripts quantified (right panels) using PhosphorImager ImageQuant software (Molecular Dynamics). u., arbitrary units. Gray bars, wild-type TFIIIC; black bars, mutant TFIIIC. relative <br /> transcription efficiencies mutant versus wild-type TFIIIC indicated bar graphs. ND, determined. independent experiments compiled quantification. <br />  <br />  <br />  <br /> TFIIIB factor, TFIIIB90/B&#226;&#172;&#8482; TBP). Fourth, vitro                              coding &#226;?&#182;60 essential cell viability. Altogether,  <br /> transcription defect strongly alleviated increasing                         sults indicate &#226;?&#182;60 plays important role TFIIIC. <br /> concentration TFIIIB TBP. Finally, like sub-                             location TFIIIC subunits site-specific DNA cross- <br /> units TFIIIC previously characterized, TFC8 gene                             linking provided broad view possible functions  VOL. 19, 1999                                                                              &#226;?&#182;60 PARTICIPATES TFIIIB RECRUITMENT                8049 <br />  <br />  <br />                                                                                  weakened stability &#226;?&#182;B-DNA complex.  mutant <br />                                                                                  TFIIIC-DNA complex formation notably affected <br />                                                                                  compared wild-type factor, unfavorable  <br />                                                                                  ditions, high ionic strengths, high temperatures.  <br />                                                                                  fore, role &#226;?&#182;60 DNA binding excluded                                                                                  strongly supported present data, keeping lack <br />                                                                                  detectable &#226;?&#182;60-DNA cross-linking (28). Schematic models                                                                                  TFIIIC-DNA complexes based site-specific cross-linking <br />                                                                                  studies tentatively visualized 60-kDa polypeptide                                                                                  unbound DNA, connecting &#226;?&#182;95 &#226;?&#182;138 subunits,                                                                                  overlapping block-B block interval (10). proposal <br />                                                                                  fits remarkably present observations, &#226;?&#182;60 <br />                                                                                  appears reside partly &#226;?&#182;B possibly contact <br />                                                                                  &#226;?&#182;95, suggested vivo suppression &#226;?&#182;60 mutant <br />                                                                                  lethality overexpression &#226;?&#182;95, hybrid exper- <br />                                                                                  iments did confirm interaction (data shown). <br />     FIG. 7. Coimmunoprecipitation experiment. HA-tagged r&#226;?&#182;60 (650 ng)           view position &#226;?&#182;60 TFIIIC-DNA complex, <br /> preincubated rTBP (200 ng) 1.5 h 25&#194;&#176;C purified magnetic       effect &#226;?&#182;60 mutation level TFIIIB recruitment <br /> beads coated anti-HA antibodies. Bound proteins eluted analyzed Western blotting anti-&#226;?&#182;60 anti-TBP polyclonal antibodies. po- <br />                                                                                  unexpected. &#226;?&#182;131 subunit sitions rTBP native HA-r&#226;?&#182;60 shown. intermediate bands      TFIIIC extending upstream transcriptional start site (7), <br /> specifically revealed anti-&#226;?&#182;60 antibodies. Lane 1, rTBP&#226;&#171;&#185;HA-r&#226;?&#182;60; lane 2,      assumed entirely responsible TFIIIB <br /> rTBP  lane 3, 25% input rTBP&#226;&#171;&#185;HA-r&#226;?&#182;60 proteins.                          assembly.  &#226;?&#182;131 TFIIIC subunit                                                                                  interact TFIIIB70/Brf1 TFIIIB90/B&#226;&#172;&#8482; (14, 33, 51). <br />                                                                                  lines evidence,  indicate &#226;?&#182;60 likely different polypeptides TFIIIC-DNA complex                         participates TFIIIB recruitment:   lethality (Table 2), good agreement                         Ntag-&#226;?&#182;60 variant nonpermissive temperature sup- <br /> biochemical genetic data (28). Unfortunately, &#226;?&#182;60                     pressed overexpression TFIIIB components, TFIIIC polypeptide cross-linked                       TFIIIB90/B&#226;&#172;&#8482; TBP  TFIIIB70 ; (ii) located elegant powerful approach  subunit                       vitro transcription defect mutant TFIIIC strongly <br /> status questioned reason). inde-                    alleviated increasing TFIIIB concentration spe- <br /> pendent experiments suggested &#226;?&#182;60 resides &#226;?&#182;B                      cifically increasing TBP concentration  TFIIIB70 <br /> main: &#226;?&#182;B-DNA complexes obtained proteolysis TFIIIC                         B&#226;&#172;&#8482;); (iii) &#226;?&#182;60 interact directly TBP. <br /> harboring HA-tagged &#226;?&#182;60 thermosen-                         possibility indirect role &#226;?&#182;60 TFIIIB recruitment <br /> sitive untagged complexes recognized antibod-                    envisioned view possible interaction &#226;?&#182;95 <br /> ies directed HA epitope.  experiments,                 belongs &#226;? domain. &#226;?&#182;60 mutation perturb <br /> HA tag placed N-terminal C-termi-                    &#226;?&#182;95 block binding affect indirectly, &#226;?&#182;95, TFIIIB assem- <br /> nal end polypeptide, inferred extremities                 bly &#226;?&#182;131. case, expect suppres- <br /> &#226;?&#182;60 resided &#226;?&#182;B, &#226;?&#182;138 (19) &#226;?&#182;91 (1).                      sion vivo vitro defects increased TFIIIB70 <br /> total mass polypeptides, 275 kDa, corre-                    dosage.  mutations TFIIIC (37), TBP (12, 16), spond reasonably mass &#226;?&#182;B (&#226;&#172;&#402;300 kDa) visualized                    promoter elements decreasing factor-DNA interaction (39) scanning transmission electron microscopy B block                    best suppressed vivo overexpres- <br /> (53). Note <span id='am-31' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> &#226;?&#182;B electron microscopy                  sion TFIIIB70. makes sense TFIIIB70 interacts limited proteolysis necessarily equivalent. pro-                  TFIIIC (14, 33), initiates TFIIIB assembly (31), ap- <br /> tease-resistant form &#226;?&#182;B observed gel retardation assay                     pears limiting transcription cell extracts contain pieces TFIIIC unrelated B-block-binding                    vivo (39, 54, 55). increasing TFIIIB70  <br /> function. participation &#226;?&#182;60 &#226;?&#182;B  defined limited                  centration did correct TFIIIC mutant phenotype proteolysis gel retardation assay) points role                  vivo vitro, rate-limiting step caused &#226;?&#182;60 <br /> subunit DNA binding.  presence HA                           mutation expected follow TFIIIB70 recruitment step, <br /> epitope N-terminal end &#226;?&#182;60 decreased yield                     e., TBP recruitment step (31). appears unlikely <br />  <br />  <br />                                                            TABLE 2. Subunits yeast TFIIIC <br />             Calculated                    Null             Location proposed role                                                  Source(s) Subunit                      Gene                                                                              Features <br />             mass (kDa)                  phenotype             TFIIIC-DNA complex                                                           reference <br />  <br />  &#226;?&#182;138           132         TFC3         Lethal        &#226;?&#182;B; B block binding                  HMG domains <br />  &#226;?&#182;131           120         TFC4         Lethal        5&#226;&#172;&#732;; TFIIIB assembly;               11 TPR motifs; phosphorylated;               23, 41a <br />                                                          interacts TFIIIB70             ortholog H. sapiens, C. elegans, <br />                                                          TFIIIB90                                K. lactis <br />  &#226;?&#182;95             74         TFC1         Lethal        &#226;?  block binding; interacts       Ortholog H. sapiens, D.m., C. elegans     23, 27a <br />                                                          &#226;?&#182;55                                S. pombe <br />  &#226;?&#182;91             75         TFC6         Lethal        3&#226;&#172;&#732;; cooperates &#226;?&#182;138        (Cys -rich domain; ortholog H.        1a <br />                                                          DNA binding                             sapiens S. pombe <br />  &#226;?&#182;60             68         TFC8          Lethal       &#226;?&#182;B; TFIIIB assembly; interacts <br />                                                          TBP <br />  &#226;?&#182;55             49         TFC7          Lethal       &#226;?  interacts &#226;?&#182;95               Chimeric protein <br />  8050       DEPREZ ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  &#226;?&#182;60 mutation indirect effect                                 Araujo,  Aparicio, B. Barrell, K. Badcock, V. Benes, D. Botstein, S. Bow- <br /> recruitment TBP TFIIIB70, favor idea                              man, M. Bruckner, J. Carpenter, J. M. Cherry, E. Chung, C. Churcher, F. <br />                                                                                            Coster, K. Davis, R. W. Davis, F. S. Dietrich, H. Delius, T. DiPaolo, J. Hani, <br /> &#226;?&#182;60 directly involved TFIIIB recruitment contacting                               et al. 1997. <span id='am-43' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-44' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> sequence Saccharomyces cerevisiae chromo- <br /> TBP. coimmunoprecipitation TBP &#226;?&#182;60 comes                                     XVI. Nature 387:103&#226;&#8364;&#8220;105. <br /> support proposal. interaction TBP TFIIIC                           14. Chaussivert, N., C. Conesa, S. Shaaban,  Sentenac. 1995. Complex <br /> subunits previously reported.  Huet et                                interactions yeast TFIIIB TFIIIC. J. Biol. Chem. 270:15353&#226;&#8364;&#8220; <br />                                                                                            15358. <br /> al. (26) noted TBP slightly retarded migration                        15. Church, G. M., W. Gilbert. 1984. Genomic sequencing. Proc. Natl. Acad. <br /> TFIIIC-DNA complexes TATA gene. TFIIIC                                     Sci. USA 81:1991&#226;&#8364;&#8220;1995. <br /> specifies orientation TFIIIB DNA                             16. Colbert, T., S. Hahn. 1992. yeast TFIIB-related factor involved direction transcription) (60), direct interaction                            RNA polymerase III transcription. Genes Dev. 6:1940&#226;&#8364;&#8220;1949. <br />                                                                                        17. Conesa, C., R. N. Swanson, P. Schultz, P. Oudet,  Sentenac. 1993. TFIIIC TBP, &#226;?&#182;60, contribute fix                                 subunit composition, stoichiometry, phosphorylation yeast <br /> correct orientation DNA. Alternatively, &#226;?&#182;60-TBP inter-                              transcription factor TFIIIC/&#226;?&#182;. J. Biol. Chem. 268:18047&#226;&#8364;&#8220;18052. <br /> action involved positioning TBP DNA                           18. Dichtl, B., D. Tollervey. 1997. Pop3p essential activity represent transient step facilitating delivery TBP                           RNase MRP RNase P ribonucleoproteins vivo. EMBO J. 16:417&#226;&#8364;&#8220;429. <br />                                                                                        19. Gabrielsen, O. S., N. Marzouki,  Ruet,  Sentenac, P. Fromageot. TFIIIB70-TFIIIC-DNA complex.                                                        1989. polypeptide chains yeast transcription factor &#226;?&#182; interact                                                                                            DNA. J. Biol. Chem. 264:7505&#226;&#8364;&#8220;7511. <br />                           ACKNOWLEDGMENTS                                              20. Gudenus, R., S. Mariotte,  Moenne,  Ruet, S. met, J.-M. Buhler,  <br />                                                                                            Sentenac, P. Thuriaux. 1988. Conditional mutants RPC160, gene <br />    grateful Christophe Carles, Michel Riva, Olivier Lefebvre,                    encoding largest subunit RNA polymerase C Saccharomyces <span id='am-83' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-84' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span>cer</span>- Franc&#194;&#184;oise Bouet help peptide microsequence deter-                        evisiae. Genetics 119:517&#226;&#8364;&#8220;526. <br /> mination Jacques Grassi Christophe Creminon                       21. Heinemeyer, W.,  Gruhler, V. Mohrle, Y. Mahe, D. H. Wolf. 1993. <br /> help raising polyclonal antibodies. thank Emmanuel Favry                         PRE2, highly homologous human major histocompatibility complex- <br />                                                                                            linked RING10 gene, codes yeast proteasome subunit necessary protein preparations Olivier Lefebvre advice TFIIIC puri-                       chymotryptic activity degradation ubiquinated proteins. J. Biol. <br /> fication immunoprecipitation experiments. thank Chris-                         Chem. 268:5115&#226;&#8364;&#8220;5120. <br /> tian Marck le`ne Dumay providing plasmids harboring dif-                    22. Hermann-Le Denmat, S., M. Werner,  Sentenac, P. Thuriaux. 1994. <br /> ferent tRNA genes helpful discussions Olivier Lefebvre,                        Suppression yeast RNA polymerase III mutations FHL1, gene coding <br /> Michel Riva, Michel Werner improving manuscript.                               fork head protein involved rRNA processing. Mol. Cell. Biol. 14: <br />    E.D. supported postdoctoral fellowships Associa-                        2905&#226;&#8364;&#8220;2913. <br /> tion pour la Recherche contre le Cancer, Ligue Nationale contre le                 23. Hsieh, Y.-J.,  Wang, R. Kovelman, R. G. Roeder. 1999. Cloning Cancer, Fondation pour la Recherche dicale, R                         characterization evolutionarily conserved subunits (TFIIIC102                                                                                            TFIIIC63) human TFIIIC involvement functional interactions <br /> supported postdoctoral fellowship European Union <br />                                                                                            TFIIIB RNA polymerase III. Mol. Cell. Biol. 19:4944&#226;&#8364;&#8220;4952. <br /> (Marie Curie Training Mobility Researchers).                                    24. Huet, J., C. Conesa, C. Carles,  Sentenac. 1997. cryptic DNA <br />                                                                                            binding domain COOH terminus TFIIIB70 affects formation, sta- <br />                                  REFERENCES                                                bility, function preinitiation complexes. J. Biol. Chem. 272:18341&#226;&#8364;&#8220; <br />  1. Arrebola, R., N. Manaud, S. Rozenfeld, M. C. Marsolier, O. Lefebvre, C.                18349. <br />     Carles, P. Thuriaux, C. Conesa,  Sentenac. 1998. &#226;?&#182;91, essential           25. Huet, J., N. Manaud, G. Dieci, G. Peyroche, C. Conesa, O. Lefebvre,  Ruet, <br />     subunit yeast TFIIIC, cooperates &#226;?&#182;138 DNA binding. Mol. Cell.               M. Riva,  Sentenac. 1996. RNA polymerase III class III transcrip- <br />     Biol. 18:1&#226;&#8364;&#8220;9.                                                                          tion factors Saccharomyces cerevisiae. Methods Enzymol. 273:249&#226;&#8364;&#8220;267. <br />  1a.Arrebola, R. Unpublished data.                                                     26. Huet, J.,  Sentenac. 1992. TATA-binding protein participates  2. Ausubel, F. M., R. Brent, R. E. Kingston, D. D. Moore, J. G. Seidman, J.             TFIIIB assembly tRNA genes. Nucleic Acids Res. 20:6451&#226;&#8364;&#8220;6454. <br />     Smith, K. Struhl. 1994. Current protocols molecular biology. John           27. Joazeiro, C.  P., G.  Kassavetis, E. P. Geiduschek. 1994. Identical <br />     Wiley <span id='am-14' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; Sons, , New York, N.Y.                                                     components yeast transcription factor IIIB required sufficient  3. Baker, R. E., S. Camier,  Sentenac, B. D. Hall. 1987. Gene size                  transcription TATA box-containing TATA genes. Mol. Cell. Biol. <br />     differentially affects binding yeast transcription factor &#226;?&#182; intra-       14:2798&#226;&#8364;&#8220;2808. <br />     genic regions. Proc. Natl. Acad. Sci. USA 84:8768&#226;&#8364;&#8220;8772.                            27a.Jourdain, S. Personal communication. <br />  4. Baker, R. E., O. S. Gabrielsen, B. D. Hall. 1986. Effects tRNATyr           28. Kassavetis, G. , C. Bardeleben, B. Bartholomew, B. R. Braun, C.  P. <br />     point mutations binding yeast RNA polymerase III transcription               Joazeiro, M. Pisano, E. P. Geiduschek. 1994. Transcription RNA <br />     factor C. J. Biol. Chem. 261:5275&#226;&#8364;&#8220;5282.                                                polymerase III, p. 107&#226;&#8364;&#8220;126. R. C. Conaway J. W. Conaway (ed.), <br />  5. Bartholomew, B., B. R. Braun, G.  Kassavetis, E. P. Geiduschek. 1994.            Transcription: mechanisms regulation. Raven Press, , New York, <br />     Probing close DNA contacts RNA polymerase III transcription complexes               N.Y. <br />     photoactive <span id='am-51' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-52' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span>nucleoside</span> 4-thiodeoxythymidine. J. Biol. Chem. 269:          29. Kassavetis, G. , B. Bartholomew, J.  Blanco, T. E. Johnson, E. P. <br />     18090&#226;&#8364;&#8220;18095.                                                                           Geiduschek. 1991. essential components Saccharomyces cerevisiae <br />  6. Bartholomew, B., G.  Kassavetis, B. R. Braun, E. P. Geiduschek. 1990.            transcription factor TFIIIB: transcription DNA-binding properties. <br />     subunit structure Saccharomyces cerevisiae transcription factor IIIC            Proc. Natl. Acad. Sci. USA 88:7308&#226;&#8364;&#8220;7312. <br />     probed novel photocrosslinking reagent. EMBO J. 9:2197&#226;&#8364;&#8220;2205.                30. Kassavetis, G. , B. R. Braun, L. H. Nguyen, E. P. Geiduschek. 1990. S. <br />  7. Bartholomew, B., G.  Kassavetis, E. P. Geiduschek. 1991. com-                cerevisiae TFIIIB transcription initiation factor proper RNA poly- <br />     ponents Saccharomyces cerevisiae transcription factor IIIB (TFIIIB)             merase III, TFIIIA TFIIIC assembly factors. Cell 60:235&#226;&#8364;&#8220;245. <br />     stereospecifically located upstream tRNA gene interact           31. Kassavetis, G. , C.  P. Joazeiro, M. Pisano, E. P. Geiduschek, T. Colbert, <br />     second-largest subunit TFIIIC. Mol. Cell. Biol. 11:5181&#226;&#8364;&#8220;5189.                       S. Hahn, J.  Blanco. 1992. role TATA-binding protein  8. Baudin, , O. Ozier-Kalogeropoulos,  Denouel, F. Lacroute, C. Cul-              assembly function multisubunit yeast RNA polymerase III <br />     lin. 1993. simple efficient method direct gene deletion Saccha-           transcription factor, TFIIIB. Cell 71:1055&#226;&#8364;&#8220;1064. <br />     romyces cerevisiae. Nucleic Acids Res. 21:3329&#226;&#8364;&#8220;3330.                               32. Kassavetis, G. , S. T. Nguyen, R. Kobayashi,  Kumar, E. P. Geiduschek, <br />  9. Bonneaud, N., O. Ozier-Kalogeropoulos, G. Li, M. Labouesse, L. Minvielle-              M. Pisano. 1995. Cloning, expression, function TFC5, gene <br />     Sebastia, F. Lacroute. 1991. family low high copy replicative,            encoding B&#226;&#172;&#8482; component Saccharomyces cerevisiae RNA polymer- <br />     integrative single-stranded S. cerevisiae/E. coli shuttle vectors. Yeast           ase III transcription factor TFIIIB. Proc. Natl. Acad. Sci. USA 92:9786&#226;&#8364;&#8220;9790. <br />     7:609&#226;&#8364;&#8220;615.                                                                         33. Khoo, B., B. Brophy, S. P. Jackson. 1994. Conserved functional domains <br /> 10. Braun, B. R., B. Bartholomew, G.  Kassavetis, E. P. Geiduschek. 1992.            RNA polymerase III general transcription factor BRF. Genes Dev. <br />     Topography transcription factor complexes Saccharomyces cerevi-              8:2879&#226;&#8364;&#8220;2890. <br />     siae 5 S RNA gene. J. Mol. Biol. 228:1063&#226;&#8364;&#8220;1077.                                    34. Lee, J. Y., C. F. Evans, D. R. Engelke. 1991. Expression RNase P RNA <br /> 11. Braun, B. R., G.  Kassavetis, E. P. Geiduschek. 1992. Bending             Saccharomyces cerevisiae controlled unusual RNA polymerase III <br />     Saccharomyces cerevisiae 5S rRNA gene transcription factor complexes.               promoter. Proc. Natl. Acad. Sci. USA 88:6986&#226;&#8364;&#8220;6990. <br />     J. Biol. Chem. 267:22562&#226;&#8364;&#8220;22569.                                                    35. Lee, J. Y., C. E. Rohlman, L.  Molony, D. R. Engelke. 1991. Charac- <br /> 12. Buratowski, S., H. Zhou. 1992. suppressor TBP mutations encodes               terization RPR1, essential gene encoding RNA component     RNA polymerase III transcription factor homology TFIIB. Cell                Saccharomyces cerevisiae nuclear RNase P. Mol. Cell. Biol. 11:721&#226;&#8364;&#8220;730. <br />     71:221&#226;&#8364;&#8220;230.                                                                        36. Lefebvre, O., C. Carles, C. Conesa, R. N. Swanson, F. Bouet, M. Riva, 13. Bussey, H., R. K. Storms,  Ahmed, K. Albermann, E. Allen, W. Ansorge, R.              Sentenac. 1992. TFC3: Gene encoding B-block binding subunit  VOL. 19, 1999                                                                                &#226;?&#182;60 PARTICIPATES TFIIIB RECRUITMENT                                 8051 <br />  <br />  <br />     yeast transcription factor TFIIIC. Proc. Natl. Acad. Sci. USA 89:10512&#226;&#8364;&#8220;          50. Roberts, S., S. J. Miller, W. S. Lane, S. Lee, S. Hahn. 1996. Cloning     10516.                                                                               functional characterization gene encoding TFIIIB90 subunit 37. Lefebvre, O., J. Ru  &#194;&#168;th,  Sentenac. 1994. mutation largest           RNA polymerase III transcription factor TFIIIB. J. Biol. Chem. 271:14903&#226;&#8364;&#8220; <br />     subunit yeast TFIIIC affects tRNA 5 S RNA synthesis. Identification           14909. <br />     classes suppressors. J. Biol. Chem. 269:23374&#226;&#8364;&#8220;23381.                   51. Ru&#194;&#168;th, J., C. Conesa, G. Dieci, O. Lefebvre,  Du  &#194;&#168;sterho&#194;&#168;ft, S. Ottonello, 38. Le&#194;&#180;veillard, T., G.  Kassavetis, E. P. Geiduschek. 1991. Saccharomyces          Sentenac. 1996. suppressor mutations class III transcription <br />     cerevisiae transcription factors IIIB IIIC bend DNA tRNAGln             encodes component yeast TFIIIB. EMBO J. 15:1941&#226;&#8364;&#8220;1949. <br />     gene. J. Biol. Chem. 266:5162&#226;&#8364;&#8220;5168.                                              52. Schmitt, M. E., T.  Brown, B. L. Trumpower. 1990. rapid simple <br /> 39. Lo&#194;&#180;pez Leo&#194;&#180;n, , M. Librizzi, K. Puglia,  Willis. 1992. PCF4 encodes       method preparation RNA Saccharomyces cerevisiae. Nucleic <br />     RNA polymerase III transcription factor homology TFIIB. Cell              Acids Res. 18:3091&#226;&#8364;&#8220;3092. <br />     71:211&#226;&#8364;&#8220;220.                                                                      53. Schultz, P., N. Marzouki, C. Marck,  Ruet, P. Oudet,  Sentenac. <br /> 40. Manaud, N., R. Arrebola, B. Buffin-Meyer, O. Lefebvre, H. Voss, M. Riva, C.          1989. DNA-binding domains yeast transcription factor &#226;?&#182;     Conesa,  Sentenac. 1998. chimeric subunit TFIIIC forms                  observed scanning transmission electron microscopy. EMBO J. 8:3815&#226;&#8364;&#8220; <br />     subcomplex &#226;?&#182;95. Mol. Cell. Biol. 18:3191&#226;&#8364;&#8220;3200.                                  3824. <br /> 41. Mann, C., J. Y. Micouin, N. Chiannilkulchai,  Treich, J.-M. Buhler,      54. Sethy, ,  M. Willis. 1995. Recessive mutations second largest <br />     Sentenac. 1992. RPC53 encodes subunit Saccharomyces cerevisiae RNA              subunit TFIIIC suggest new step RNA polymerase III transcription. <br />     polymerase C (III) inactivation leads predominantly G1 arrest.            Gene Expr. 5:35&#226;&#8364;&#8220;47. <br />     Mol. Cell. Biol. 12:4314&#226;&#8364;&#8220;4326.                                                   55. Sethy-Coraci, , R. D. Moir,  Lo   &#194;&#180;pez Leo&#194;&#180;n,  M. Willis. 1998. 41a.Marck, C. Personal communication.                                                    differential response wild type mutant promoters TFIIIB70  <br /> 42. Marck, C., O. Lefebvre, C. Carles, M. Riva, N. Chaussivert,  Ruet,  <br />                                                                                          expression vivo vitro. Nucleic Acids Res. 26:2344&#226;&#8364;&#8220;52. <br />     Sentenac. 1993. TFIIIB-assembling subunit yeast transcription factor <br />                                                                                      56. Simos, G., H. Tekotte, H. Grosjean,  Segref, K. Sharma, D. Tollervey,     TFIIIC tetratricopeptide repeat basic-helix-loop-helix motifs. <br />                                                                                          E. C. Hurt. 1996. Nuclear pore proteins involved biogenesis     Proc. Natl. Acad. Sci. USA 90:4027&#226;&#8364;&#8220;4031. <br /> 43. Margottin, F., G. Dujardin, M. Ge&#194;&#180;rard, J. M. Egly, J. Huet,  Sentenac.        functional tRNA. EMBO J. 15:2270&#226;&#8364;&#8220;2284. <br />     1991. Participation TATA factor transcription yeast U6 gene     57. Stettler, S., N. Chiannilkulchai, S. Hermann-Le Denmat, D. Lalo, F. La- <br />     RNA polymerase C. Science 251:424&#226;&#8364;&#8220;426.                                            croute,  Sentenac, P. Thuriaux. 1993. general suppressor RNA <br /> 44. Marzouki, N., S. Camier,  Ruet,  Moenne,  Sentenac. 1986.                  polymerase  II III mutations Saccharomyces cerevisiae. Mol. Gen. <br />     Selective proteolysis defines DNA binding domains yeast transcription         Genet. 239:169&#226;&#8364;&#8220;176. <br />     factor &#226;?&#182;. Nature 323:176&#226;&#8364;&#8220;178.                                                    58. Stolc, V., S. Altman. 1997. Rpp1, essential protein subunit nuclear <br /> 45. Moenne, , S. Camier, G. Anderson, F. Margottin, J. Beggs,  Sente-            RNase P required processing precursor tRNA 35S precursor <br />     nac. 1990. U6 gene Saccharomyces cerevisiae transcribed RNA             rRNA Saccharomyces cerevisiae. Genes Dev. 11:2414&#226;&#8364;&#8220;2425. (Corrected <br />     polymerase C (III) vivo vitro. EMBO J. 9:271&#226;&#8364;&#8220;277.                          republished, 11:2926&#226;&#8364;&#8220;2937.) <br /> 46. Pardee, T. S., C. S. Bangur,  S. Ponticelli. 1998. N-terminal region   59. Swanson, R. N., C. Conesa, O. Lefebvre, C. Carles,  Ruet, E. Quemeneur, <br />     yeast TFIIB contains adjacent functional domains involved stable           J. Gagnon,  Sentenac. 1991. Isolation TFC1, gene encoding     RNA polymerase II binding transcription start site selection. J. Biol.           DNA-binding subunits yeast transcription factor &#226;?&#182; (TFIIIC). Proc. <br />     Chem. 273:17859&#226;&#8364;&#8220;64.                                                                  Natl. Acad. Sci. USA 88:4887&#226;&#8364;&#8220;4891. <br /> 47. Parsons, M. C., P.  Weil. 1990. Purification characterization       60. Whitehall, S. K., G.  Kassavetis, E. P. Geiduschek. 1995. sym- <br />     Saccharomyces cerevisiae transcription factor TFIIIC. Polypeptide composi-           metry yeast U6 RNA gene&#226;&#8364;&#8482;s TATA box orientation     tion defined polyclonal antibodies. J. Biol. Chem. 265:5095&#226;&#8364;&#8220;5103.               TATA-binding protein yeast TFIIIB. Genes Dev. 9:2974&#226;&#8364;&#8220;2985. <br /> 48. Parsons, M. C., P.  Weil. 1992. Cloning TFC1, Saccharomyces         61. Willis, , P. Schmidt, D. ll. 1989. selection mutants RNA <br />     cerevisiae gene encoding 95-kDa subunit transcription factor                  polymerase III transcription apparatus: PCF1 stimulates transcription     TFIIIC. J. Biol. Chem. 267:2894&#226;&#8364;&#8220;2901.                                                tRNA 5S RNA genes. EMBO J. 8:4281&#226;&#8364;&#8220;4288. <br /> 49. Rameau, R., K. Puglia,  Crowe,  Sethy,  M. Willis. 1994. mutation    62. Wilson,  , H. L. Niman, R.  <br /> </body></html>