<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Sept. 2000, p. 5147&#226;&#8364;&#8220;5152                                                                                  Vol. 182,  18 <br /> 0021-9193/00/$04.00&#226;&#171;&#185;0 <br /> Copyright &#194;&#169; 2000, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />           Essentiality, Expression, Characterization Class II <br />              3-Hydroxy-3-Methylglutaryl Coenzyme Reductase                               Staphylococcus aureus <br />           E. IMOGEN WILDING,1 DONG-YUL KIM,2 ALEXANDER P. BRYANT,1 MICHAEL N. GWYNN,1* <br />         R. DWAYNE LUNSFORD,1 DAMIEN MCDEVITT,1 JOSEPH E. MYERS, JR.,1 MARTIN ROSENBERG,1 <br />                DANIEL SYLVESTER,1 CYNTHIA V. STAUFFACHER,3 VICTOR W. RODWELL2 <br />              Department Microbiology, SmithKline Beecham Pharmaceuticals, Collegeville, Pennsylvania 19426,1              Departments Biochemistry2 Biological Sciences,3 Purdue University, West Lafayette, Indiana 47907 <br />                                               Received 21 April 2000/Accepted 12 June 2000 <br />  <br />              Sequence comparisons implied presence genes encoding enzymes mevalonate pathway           <span id='am-22' about='obo:IMR_0200287' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00129" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200287" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Isopentenyl diphosphate" datatype="xsd:string"></span>isopentenyl diphosphate</span> biosynthesis gram-positive pathogen Staphylococcus aureus. study           showed genetic disruption experiments mvaA, encodes putative class II 3-hydroxy-3- <br />           methylglutaryl coenzyme (HMG-<span id='am-42' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span>) reductase, essential vitro growth S. aureus. Supplementa- <br />           tion media mevalonate permitted isolation auxotrophic mvaA null mutant attenuated           virulence murine hematogenous pyelonephritis infection model. mvaA gene cloned S. aureus <br />           DNA expressed N-terminal tag Escherichia coli. encoded protein affinity purified <br />           apparent homogeneity shown class II HMG-CoA reductase, class II eubacterial <br />           biosynthetic enzyme isolated. Unlike HMG-CoA reductases, S. aureus enzyme exhibits dual <br />           coenzyme specificity NADP(H) NAD(H), NADP(H) preferred coenzyme. Kinetic param- <br />           eters determined substrates catalyzed reactions using NADP(H) NAD(H).           instances optimal activity using NAD(H) occurred pH units acidic using <br />           NADP(H). pH profiles suggested His378 Lys263, apparent cognates active-site histidine           lysine Pseudomonas mevalonii HMG-CoA reductase, function catalysis general catalytic <br />           mechanism valid S. aureus enzyme. Fluvastatin inhibited competitively HMG-CoA, Ki           320 &#226;?&#174;M, 104 higher class HMG-CoA reductase. Bacterial class II HMG-CoA reductases <br />           potential targets antibacterial agents directed multidrug-resistant gram-positive cocci. <br />  <br />  <br />    Isoprenoids, ubiquitous nature, comprise fam-              distinct classes HMG-CoA reductase. Genes appear ily 23,000 products, composed repeating                 encode class HMG-CoA reductases present eu- <br /> carbon, isopentenyl diphosphate (IPP) subunits. principal                karyotes, archaea, streptomycetes. products IPP bacteria include <span id='am-5' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> carrier undeca-                contrast, genes encode class II forms enzyme prenol, involved cell wall biosynthesis (33), mena-              present eubacteria archaeon Archaeoglobus <br /> quinones ubiquinones involved electron transport (29),                fulgidus (41). Previously characterized class HMG-CoA  carotenoids (19). pathways biosynthesis IPP               ductases include archaea Haloferax volcanii (6) described, mevalonate pathway (16)                     Sulfolobus solfataricus (7, 22), gram-positive eubacte- <br /> glyceraldehyde-3-phosphate (GAP)-pyruvate pathway (36, 37).                  rium Streptomyces sp. strain CL190 (40), numerous eu- <br /> Analysis distribution genes encoding enzymes                   karyotes. Characterized class II HMG-CoA reductases include <br /> involved pathways revealed Bacillus subtilis              fulgidus (23) biodegradative Pseudomonas gram-negative bacteria, including Escherichia coli, Hae-                mevalonii enzyme (4, 20), structure deter- <br /> mophilus influenzae, Helicobacter pylori, possess genes             mined (24, 39) converts mevalonate HMG-CoA, encode GAP-pyruvate pathway, low-G&#226;&#171;&#185;C                      permitting growth mevalonate (15). biosynthetic eubac- <br /> gram-positive cocci Borrelia burgdorferi possess genes              terial class II HMG-CoA reductase   char- encode mevalonate pathway (41).  func-                  acterized. <br /> tionality implied polypeptides mevalonate path-                   Biosynthetic HMG-CoA reductases catalyze reaction 1, way bacteria demonstrated.                                   reductive deacylation thioester group HMG-CoA    Enzymes involved mevalonate pathway number                 primary alcohol mevalonate using 2 mol <span id='am-37' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span>NADPH</span> <br /> organisms, including humans, isolated studied.                 (35). putative intermediates, mevaldyl-CoA mevalde- <br /> 3-Hydroxy-3-methylglutaryl coenzyme (HMG-CoA) reduc-                       hyde, remain bound course reaction. HMG- <br /> tase (EC 1.1.1.34) best-characterized rate-limiting               CoA reductases catalyze additional reactions.  <br /> enzyme pathway (9) target statin class              actions 2 3 free mevaldehyde appear model <span id='am-79' about='obo:IMR_0200486' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00187" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Cholesterol" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:id" content="IMR:0200486" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cholesterol</span>-lowering drugs (1). Based <span id='am-55' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-56' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-57' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-58' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> se-                   second reductive stage reverse reductive <br /> quence analysis, Bochar et al. (8) suggested              stage reaction 1, respectively. Reaction 4, oxidative ac- <br />                                                                              ylation (R)-mevalonate (S)-HMG-CoA, reverse                                                                              reaction 1. <br />   * Corresponding author. Mailing address: Department Microbi- <br /> ology, SmithKline Beecham Pharmaceuticals, 1250 South College Rd.,                &#229;&#8230;&#177;S&#229;&#8230;&#178;-HMG-CoA &#226;&#171;&#185; 2 NADPH &#226;&#171;&#185; 2 H&#226;&#171;&#185;3&#229;&#8230;&#177;R&#229;&#8230;&#178;-mevalonate &#226;&#171;&#185; CoASH &#226;&#171;&#185; 2 NADP&#226;&#171;&#185;   (1) <br /> Collegeville, PA 19426. Phone: (610) 917-7749. Fax: (610) 917-4989. <br /> E-mail: Mick_Gwynn-1@sbphrd.com.                                                      &#229;&#8230;&#177;R&#229;&#8230;&#178;-mevaldehyde &#226;&#171;&#185; NADPH &#226;&#171;&#185; H&#226;&#171;&#185;3&#229;&#8230;&#177;R&#229;&#8230;&#178;-mevalonate &#226;&#171;&#185; NADP&#226;&#171;&#185;         (2) <br />  <br />                                                                       5147 <br />  5148       WILDING ET AL.                                                                                                                               J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Chromosomal analysis S. aureus mvaA null mutant.   chromosome mvaA null mutant contains allelic replacement mvaA (dotted <br /> block arrow) ermC gene (gray block arrow). PCR primers (small arrows) designed anneal regions flanking mvaA. (B) Products expected size amplified using chromosomal DNA mutant (2,288 bp [lane 3]) wild type (2,371 bp [lane 8]). Replacement mvaA ermC resulted elimination HindIII sites introduction SspI sites chromosome. Amplified fragments digested HindIII SspI, products expected size obtained. Lane 1, mutant PCR product incubated HindIII (undigested); lane 2, mutant PCR product cut SspI (1,518 659 bp smaller bands); lane 6, wild-type PCR product cut HindIII (894, 776, 618 bp); lane 7, wild-type PCR product incubated SspI (undigested); <br /> lanes 4 5, 100-bp 1-kb DNA markers, respectively. <br />  <br />  <br />         &#229;&#8230;&#177;R&#229;&#8230;&#178;-mevaldehyde &#226;&#171;&#185; NADP&#226;&#171;&#185; &#226;&#171;&#185; CoASH3&#229;&#8230;&#177;S&#229;&#8230;&#178;-HMG-CoA &#226;&#171;&#185; NADPH &#226;&#171;&#185; H&#226;&#171;&#185;             (3)          Determination essentiality mvaA gene. DNA construct allelic <br />                                                                                        replacement S. aureus mvaA gene generated overlap piece <br />        &#229;&#8230;&#177;R&#229;&#8230;&#178;-mevalonate &#226;&#171;&#185; 2 NADP&#226;&#171;&#185; &#226;&#171;&#185; CoASH3&#229;&#8230;&#177;S&#229;&#8230;&#178;-HMG-CoA &#226;&#171;&#185; 2 NADPH &#226;&#171;&#185; 2 H&#226;&#171;&#185;         (4)       PCR technique. technique gene fusion procedure (42) extended                                                                                        include separate long flanking sequences (547 bp directly upstream 591 CoASH reduced coenzyme                                                      bp directly downstream mvaA gene) surrounding central selectable <br />    report mvaA gene gram-positive                              cassette containing 1,234-bp ermC gene pE194 (17). final  <br /> pathogen Staphylococcus aureus essential growth,                         length products digested cloned BamHI site pBluescript <br /> report cloning, purification, characterization                          II KS(&#226;&#171;&#185;) derivative containing tetracycline resistance gene (tetK) pT181 <br /> mvaA gene product S. aureus, truly biosynthetic                           (21). resulting plasmids introduced S. aureus RN4220 electro- <br />                                                                                        poration selection resistance erythromycin. colonies iso- <br /> class II HMG-CoA reductase characterized eubacterium.                           lated, resistant tetracycline (cointegrants), examined                                                                                        target-specific plasmid integration diagnostic PCR primer pairs based                         MATERIALS METHODS                                          plasmid- locus-specific sequences. Bona fide plasmid cointegrants                                                                                        transferred S. aureus RN4220 NCTC 8325-4 phage &#226;?&#190;11 trans- <br />    Reagents. Purchased reagents included restriction enzymes, calf intestinal          duction allow second recombination event potentially resolve <br /> alkaline phosphatase, T4 DNA ligase, protein DNA molecular weight mark-            generate allelic-replacement mutation. Transductants resistant erythromy- <br /> ers (Gibco BRL Life Technologies), Pfu Turbo DNA polymerase (Stratagene),              cin sensitive tetracycline examined allelic replacement PCR. lysostaphin (Applied Microbiology, ). Fluvastatin gift Novar-          Murine hematogenous pyelonephritis infection model. Cells overnight <br /> tis. Unless specified, chemicals Sigma.                  cultures S. aureus NCTC 8325-4 mvaA null mutant grown TSB    Plasmid, bacterial strains, culture media. Expression plasmid pET28         10 mM mevalonate required centrifuged, washed twice sterile phos- Novagen. Bacterial strains used included E. coli BL21(DE3) DH5&#226;?&#163;           phate-buffered saline, adjusted A600 0.2. Female CD-1 mice weigh- <br /> S. aureus strains RN4220 (31), WCUH29 (NCIMB 40771), NCTC 8325-4.                  ing 18 20 g (Charles River, Quebec, Canada) inoculated tail vein <br /> Luria-Bertani broth agar (38) served growth E. coli, tryptone soy       injection 0.2 ml suspension containing approximately 107 bacteria. Mice <br /> broth (TSB) tryptone soy agar (TSA) (Oxoid) served growth S. aureus.        monitored twice daily signs illness. animals euthanatized required, E. coli medium supplemented 50 &#226;?&#174;g kanamycin            carbon dioxide overdose 5 days inoculation. kidneys removed ml 1% (wt/vol) glucose S. aureus medium supplemented          using aseptic technique homogenized 1 ml PBS, 0.5 &#226;?&#174;g erythromycin 5 &#226;?&#174;g tetracycline ml.                                number viable bacteria determined plating TSA, sup- <br />    DNA techniques. Plasmid DNA isolated using RPM kit (Bio 101 )           plemented 10 mM mevalonate mvaA null mutant. Wizard Midiprep DNA purification (Promega). PCR products                    Construction HMG-CoA reductase expression plasmid. mvaA gene isolated horizontal agarose gel electrophoresis purified using         amplified S. aureus WCUH29 chromosomal DNA using primers GENECLEAN II kit (Bio 101 ). Chromosomal DNA isolated S.                  introduced NheI BamHI sites ends amplified fragment. aureus using standard procedures (27). Incubation 0.1 &#226;?&#174;g lysostaphin       1,278-bp PCR fragment cloned using pCR-Zero Blunt cloning kit   <br /> ml included preparation plasmid chromosomal DNA             vitrogen)  using introduced restriction sites, subcloned pET28 S. aureus facilitate cell lysis. Procedures DNA restriction, dephosphoryla-     frame N-terminal histidine tag thrombin cleavage site form pJO1. <br /> tion ligation, agarose gel electrophoresis, PCR, transformation CaCl2-          insert sequenced verify errors introduced competent E. coli, phage transduction performed described               PCR amplification. <br /> Sambrook et al. (38) recommended manufacturers&#226;&#8364;&#8482; instructions. PCR            Expression purification HMG-CoA reductase. E. coli BL21(DE3) cells <br /> employed RoboCycler gradient temperature cycler (Stratagene). Synthetic              containing pJO1 grown mid-log phase, induced addition 1 mM <br /> oligonucleotides prepared, automated DNA sequencing                   isopropyl-&#226;?&#164;-D-thiogalactopyranoside (IPTG), harvested 2.5 h postinduction. <br /> formed house SmithKline Beecham.                                                 confirm expression, total cell lysates prepared outlined Sambrook <br />  VOL. 182, 2000                                                                        STAPHYLOCOCCUS AUREUS HMG-CoA REDUCTASE                                       5149 <br />  <br />  <br />                                                                                          Reaction 1, reductive deacylation HMG-CoA mevalonate. Assays using <br />                                                                                       NADPH coenzyme contained 0.25 mM NADPH, 0.25 mM (R,S)-HMG- <br />                                                                                       CoA, 50 mM NaCl, 1 mM EDTA, 5 mM DTT, 25 mM KH2PO4 (pH 7.5). <br />                                                                                       Assays using <span id='am-27' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="NADH" datatype="xsd:string"></span>NADH</span> coenzyme contained 0.25 mM NADH, 0.25 mM <br />                                                                                       (R,S)-HMG-CoA, 50 mM NaCl, 1 mM EDTA, 5 mM DTT, 25 mM KH2PO4 <br />                                                                                       (pH 6.0). <br />                                                                                          Reaction 2, reduction mevaldehyde mevalonate. Assays contained 0.5 <br />                                                                                       mM NADPH, 8.0 mM (R,S)-mevaldehyde, 50 mM NaCl, 1 mM EDTA, 5 mM <br />                                                                                       DTT, 25 mM KH2PO4 (pH 7.0). <br />                                                                                          Reaction 3, oxidative acylation mevaldehyde HMG-CoA. Assays  <br />                                                                                       tained 0.5 mM NADP&#226;&#171;&#185;, 5.0 mM coenzyme  8.0 mM (R,S)-mevaldehyde, 50 <br />                                                                                       mM NaCl, 1 mM EDTA, 5 mM DTT, 100 mM Tris-HCl (pH 9.0). <br />                                                                                          Reaction 4, oxidative acylation mevalonate HMG-CoA. Assays contained <br />                                                                                       5 mM NADP&#226;&#171;&#185;, 5.0 mM coenzyme  6.0 mM (R,S)-mevalonate, 50 mM NaCl, <br />                                                                                       1 mM EDTA, 5 mM DTT, 100 mM Tris-HCl (pH 9.0). <br />                                                                                          Unless stated, reactions initiated adding HMG-CoA, <br />    FIG. 2. Expression S. aureus mvaA purification gene product.       mevaldehyde, mevalonate. assays, enzyme unit represents protein anticipated size absent induced E. coli BL21(DE3) cells       turnover, 1 min, 1 &#226;?&#174;mol <span id='am-74' about='obo:IMR_0200025' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-75' property="rdfs:label" content="Nicotinamide" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:id" content="IMR:0200025" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="KEGG:C00153" datatype="xsd:string"></span>nicotinamide</span> <span id='am-47' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-48' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-50' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> coenzyme. corre- <br /> containing parent vector (lane 1) expressed cells containing pJO1      sponds turnover 1 &#226;?&#174;mol mevaldehyde 0.5 &#226;?&#174;mol HMG-CoA <br /> (lane 2). Lane 3, crude cytosol; lane 4, material failed bind nickel   mevalonate. Reported data mean values triplicate determinations. <br /> column. Bound protein eluted imidazole concentrations 10 mM <br /> (lane 5), 50 mM (lane 6), 100 mM (lane 7), 500 mM (lane 8). Lanes 9 10 contain 20 4 &#226;?&#174;g purified protein, respectively.                                                               RESULTS <br />  <br />                                                                                          Essentiality mvaA gene S. aureus. genes  <br /> et al. (38) subjected sodium dodecyl sulfate-polyacrylamide gel electro-       volved synthesis IPP mevalonate pathway phoresis. Purification HMG-CoA reductase employed 500-ml culture              essential vitro growth Streptococcus pneumoniae (41). <br /> E. coli BL21(DE3) transformed pJO1. Following induction, cells harvested centrifugation HMG-CoA reductase purified using                                                                                        determine S. pneu- <br /> HisTrap kit (Amersham Pharmacia Biotech) essentially described              moniae representative gram-positive cocci manufacturer&#226;&#8364;&#8482;s instructions, following modifications. Cell lysis     HMG-CoA reductase essential pathogens. Attempts <br /> performed addition 0.2 mg lysozyme ml incubation 20 min          delete mvaA chromosome S. aureus 37&#194;&#176;C. Lysates frozen dry ice-ethanol bath, thawed, sonicated times 10 s, 10 s bursts. freezing sonication <br />                                                                                       using allelic-replacement mutagenesis strategy. non- <br /> procedure repeated times. Protein eluted column 10,        essential genes cointegrant readily resolved using <br /> 50, 100, 500 mM imidazole. Purified HMG-CoA reductase solution            phage transduction isolate mutant fre- <br /> 50 mM KH2PO4 (pH 7.5), 200 mM NaCl, 1 mM dithiothreitol (DTT), 1 mM                   quency 0.5 5%. case,  proved impossible <br /> EDTA, 50% (wt/vol) glycerol stored &#226;&#171;&#186;20&#194;&#176;C. Protein concentrations determined using Bradford assay (10) using bovine serum albumin                                                                                       resolve cointegrant structure isolate mutant standard.                                                                           standard growth media, strongly suggesting mvaA es- <br />   Assays HMG-CoA reductase activities. Spectrophotometric assays HMG-           sential vitro growth. <br /> CoA reductase activity employed Hewlett-Packard model 8452 diode array                 growth media supplemented meval- <br /> spectrophotometer cell compartment maintained 37&#194;&#176;C mea- <br /> surements 340 nm oxidation reduction NAD(P)H. Assays                                                                                       onate, mvaA auxotrophic mutants readily obtained conducted final volume 200 &#226;?&#174;l. Standard assay conditions             allelic replacement confirmed using diagnostic PCR <br /> reaction studied follows.                                                     restriction analysis. Oligonucleotide primers designed  <br />  <br />  <br />  <br />    FIG. 3. Effect hydrogen <span id='am-16' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> concentration activity. effects indicated concentrations hydrogen ion rates indicated reactions assayed standard conditions pH using NADP(H) (F) NAD(H) (E) coenzyme. Assays conducted solution 50 mM KCl, <br /> 25 mM K-PO4, 100 mM Tris-HCl adjusted indicated pH values (x axis).  left) Reaction 1, reductive deacylation HMG-CoA mevalonate;  right) <br /> reaction 2, reduction mevaldehyde mevalonate; (lower left) reaction 3, oxidative acylation mevaldehyde HMG-CoA; (lower right) reaction 4, oxidative <br /> acylation mevalonate HMG-CoA. eu, enzyme units. <br />  5150         WILDING ET AL.                                                                                                                           J. BACTERIOL. <br />  <br />  <br />       TABLE 1. Comparison Km values class II S. aureus <br />       HMG-CoA reductase class II biodegradative <br />          enzyme P. mevalonii class biosynthetic <br />                          hamster enzymea <br />                                                        Km (&#226;?&#174;M) enzyme  <br />              Reaction substrates                   S.         P. <br />                                                                              Hamster <br />                                                      aureus    mevalonii <br />  <br /> Reaction 1 (HMG-CoA3mevalonate) <br />  HMG-CoA                                                40         20            20 <br />  NADPH                                                  70                       80 <br />  NADH                                                  100         80 <br />  <br /> Reaction 2 (mevaldehyde3mevalonate) <br />  Mevaldehyde                                         6,900      8,000         1,600 <br />  NADPH                                                 240                      160 <br />                                                                                           FIG. 4. Inhibition statin drug. Double-reciprocal plot inhibition Reaction 3 (mevaldehyde3HMG-CoA)                                                       reaction 1, reductive deacylation HMG-CoA mevalonate, using <br />  Mevaldehyde                                           570         80            90    NADPH coenzyme. Analyses conducted pH 7.5 37&#194;&#176;C                                                                                        indicated concentrations HMG-CoA presence 0 &#226;?&#174;M (E) 500 &#226;?&#174;M <br />  CoASH                                                 490        110            50    (F) fluvastatin. reactions initiated adding NADPH. eu, enzyme units. <br />  NADP&#226;&#171;&#185;                                                  40                      600 <br />  <br /> Reaction 4 (mevalonate3HMG-CoA) <br />  Mevalonate                                            670        260            20 <br />  CoASH                                                 390         60            10    mevalonate concentration reduced 1 nM,  <br />  NADP&#226;&#171;&#185;                                                 580                      510    mvaA null mutant unable grow, viability <br />                                                                                        unaffected (data shown). <br />         Data P. mevalonii hamster HMG-CoA reductase reference              Virulence attenuation mvaA null mutant. mvaA <br /> 34. <br />                                                                                        mutation transferred pathogenic strain S. aureus <br />                                                                                        NCTC 8325-4 phage &#226;?&#190;11 transduction. resulting mu- <br />                                                                                        tants auxotrophic mevalonate. mice inocu- <br /> anneal regions flanking mvaA (Fig. 1) internal                         lated intravenously S. aureus NCTC 8325-4 regions mvaA ermC (data shown) used                               mvaA null mutant sacrificed 5 days postinfection  plify chromosomal DNA templates prepared wild-                                mice died infection). Viable S. aureus NCTC <br /> type mutant strains. expected products obtained,                          8325-4 harboring deletion recovered confirming mvaA replaced ermC auxo-                            kidneys  e., level limit detection 1.6 <br /> trophic mutant.                                                                        log10 CFU/mouse) plated presence mevalonate, <br />    Auxotrophy mvaA null mutants. minimal concentra-                             contrast wild type, S. aureus NCTC 8325-4, tion mevalonate required growth S. aureus                            present 5.17 &#226;&#171;&#190; 0.17 log10 CFU/mouse (mean &#226;&#171;&#190; standard <br /> mvaA null mutant investigated TSA containing meval-                             deviation mice). results indicate auxotro- <br /> onate 0.01, 0.1, 1 mM TSA supplemen-                             phic mvaA null mutant greatly reduced virulence tation. Addition mevalonate medium enabled                               hematogenous pyelonephritis infection model. <br /> mutant grow, rate growth lower lower                            Expression mvaA purification S. aureus HMG-CoA <br /> concentrations mevalonate. absence added                               reductase. mvaA gene S. aureus cloned pET28 <br /> valonate, mutant showed growth 60 h rich                               frame N-terminal histidine tag thrombin cleav- <br /> medium (TSA).                                                                          age site. soluble protein expressed E. coli BL21(DE3) <br />    minimum concentration mevalonate required                             high levels following induction 1 mM IPTG mvaA null mutant overnight growth TSB 1 mM (data                            readily purified nickel-chelating column, majority shown). effect removing mevalonate                              eluting 500 mM imidazole fraction (Fig. 2). Typical <br /> dium viability mvaA null mutants investigated.                           yields 30 mg 90% homogenous protein liter overnight culture mvaA null mutant grown                              induced culture. mvaA gene encodes putative protein presence 1 mM mevalonate diluted 106-fold fresh                            425 residues calculated molecular weight 46,180. <br /> media mevalonate. number viable                                Matrix-assisted laser desorption ionization mass spectrometry <br /> cells determined plating presence 1 mM                                analysis indicated purified protein molecular <br /> mevalonate. presence mevalonate mutant grew                              weight 48,503, corresponds length HMG-  viable count increased 4 logs 24 h.                         CoA reductase fusion protein minus terminal methionine <br />                                                                                        residue. N-terminal sequencing purified protein  <br />                                                                                        firmed residue removed N- <br /> TABLE 2. Relative abilities class II HMG-CoA reductases                       terminal sequence intact. <br />                use NADP(H) NAD(H)                                                  Biochemical characterization. enzymatic activity                                                                                        purified protein determined measuring oxidation  Source                     kcat(NADPH)/kcat(NADH) fora: <br />  HMG-CoA                                                                   Reference <br />                                                                                        reduction NAD(P)H 340 nm. Firstly, temperature <br />   reductase              Reaction 1               Reaction 4                           dependency S. aureus HMG-CoA reductase deacy- <br />                                                                                        lation HMG-CoA investigated. Optimal activity S. aureus                      25                   25                  study     catalysis reaction 1 observed 35 45&#194;&#176;C  fulgidus                     0.5                  0.05               23             decreased precipitously 45&#194;&#176;C (data shown). sub- <br /> P. mevalonii                                     4.4 &#226;&#171;&#187; 10&#226;&#171;&#186;4             14 <br />                                                                                        sequent assays conducted 37&#194;&#176;C. <br />         kcat, turnover number.                                                              Coenzyme specificity S. aureus HMG-CoA reductase  VOL. 182, 2000                                                                    STAPHYLOCOCCUS AUREUS HMG-CoA REDUCTASE                                 5151 <br />  <br />  <br />  <br />  <br />   FIG. 5. Alignment selected amino acid sequences class II HMG-CoA reductases S. aureus (Sa) P. mevalonii (Pm). Conserved paired residues <br /> E81S-E83P, K263S-K267P, D279S-D283P, H378S-H381P, bold, represent active-site residues known function catalysis P. mevalonii enzyme. conserved ENVIG, DAMGXN, GTVGG sequences shaded. Asp146P Leu148P, marked asterisks, contribute ability P. mevalonii HMG-CoA <br /> reductase discriminate NADP(H). Additional residues conserved sequences boxed. sequences S. aureus P. mevalonii <br /> HMG-CoA reductases 39% identical overall. Alignment performed using MegAlign sequence analysis software DNASTAR , Madison, Wis. <br />  <br />  <br />  <br /> investigated range pH values. Figure 3 illustrates                  reductase established functions catalysis (9). pH profiles catalysis reactions 1 4 using                        infer mechanism proposed catalysis NADP(H) NAD(H) coenzyme. Unlike                              P. mevalonii HMG-CoA reductase (39) valid S. aureus <br /> characterized HMG-CoA reductases, S. aureus enzyme                        HMG-CoA reductase. <br /> use NADP(H) NAD(H) coenzymes catalysis                            characterized class II HMG-CoA reductases use reactions. significant activity observed using                 NAD(H) (Table 2), S. aureus  fulgidus <br /> NAD(H), NADP(H) instances preferred coen-                          enzymes use NAD(P)H NAD(H). respect, zyme. reactions, optimal activity using NAD(H)                       S. aureus enzyme closely resembles archaeal enzyme <br /> occurred pH units acidic using                     mesophilic, biodegradative counterpart. ability NADP(H).                                                                          S. aureus HMG-CoA reductase use coenzyme    Km values determined substrates                      predicted, Asp146P Leu148P major determinants <br /> reactions catalyzed HMG-CoA reductases. Table 1 summa-                         nucleotide coenzyme specificity P. mevalonii HMG-CoA <br /> rizes data includes comparison Km values                   reductase (14). Crystal structures P. mevalonii enzyme class II HMG-CoA reductase P. mevalonii                         revealed Asp146P interacts 2&#226;&#172;&#732;-hydroxyl class Syrian hamster enzyme. Table 2 coenzyme spec-                      adenosyl ribose NAD(H), discriminating NADP(H) <br /> ificities characterized class II HMG-CoA reduc-                      (24, 39). Consistent ability use NADP(H), sequence <br /> tases compared.                                                               alignments revealed clear cognates Asp146P Leu148P <br />    effect statin drug fluvastatin S. aureus HMG-                    S. aureus HMG-CoA reductase. Despite significant similar- <br /> CoA reductase investigated. Statin drugs competitive                      ity (39% identity 57% similarity), ability S. aureus <br /> inhibitors HMG-CoA reductase activity (9), ex-                          HMG-CoA reductase accommodate coenzymes sug- <br /> pected, catalysis reaction 1 inhibited fluvastatin.                     gests significant differences coenzyme binding site <br /> Inhibition competitive respect HMG-CoA,                        apparent crystal structure S. aureus <br /> Ki 320 &#226;?&#174;M (Fig. 4).                                                            HMG-CoA reductase determined compared                                                                                   P. mevalonii enzyme. Finally, suggest ability                          DISCUSSION                                               use coenzymes turn general property    demonstrated mvaA gene essential                        biosynthetic class II forms enzyme. <br /> growth S. aureus mvaA null mutants                              Despite ability use NAD(H), S. aureus HMG-CoA <br /> readily isolated media supplemented mevalonate.                           reductase true biosynthetic enzyme. inferred results support previous observations mvaA es-                      preference NADPH presence S. aureus sential S. pneumoniae (41) strongly suggest gene                  genes encode putative HMG-CoA synthase, mevalonate essential gram-positive cocci use meval-                    kinase, phosphomevalonate kinase, mevalonate decarbox- <br /> onate pathway synthesis IPP, GAP-pyruvate                       ylase (41). oxidoreductases exhibit high order <br /> pathway organisms. S. aureus                      coenzyme specificity, oxidoreductases use mvaA null mutant severely attenuated mouse                             NADP(H) NAD(H) include <span id='am-62' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-63' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span>L-glutamate</span> dehydrogenase <br /> hematogenous pyelonephritis infection model.                         (11, 12, 28), isocitrate dehydrogenase (13, 25), glucose-6-phos- <br /> plasma mevalonate concentration mice mea-                         phate dehydrogenase (3, 5), 6-phosphogluconate dehydroge- <br /> sured, concentrations humans rats, 0.02 0.08 &#226;?&#174;M                     nase (5), aldose reductase (30), biliverdin IX&#226;?&#163; reductase 0.08 0.50 &#226;?&#174;M, respectively (32),                       (26). exception liver alcohol dehydrogenase (2), <br /> required support growth mvaA null mutant.                           coenzyme Km values 1 orders magnitude lower <br />    Derived sequence similarities conservation known                        NADP(H) NAD(H), suggesting preference active-site residues motifs (Fig. 5) suggested mvaA                  NADPH. HMG-CoA reductases  fulgidus S. <br /> gene S. aureus encodes class II HMG-CoA reductase (41).                      aureus mere handful oxidoreductases, Expression E. coli mvaA-polyhistidine tag construct                      HMG-CoA reductases, Km essentially subsequent purification enzymatic characteriza-                       coenzyme. <br /> tion gene product revealed case.                     Ki inhibition S. aureus HMG-CoA reductase Inspection pH profiles catalysis reactions                 statin drug 4 orders magnitude higher Ki revealed functional groups pKa values 7                      class eukaryotic enzyme, suggesting classes 9 protonated states activity. pro-                    discriminated chemically. recent publication pose His378 S. aureus (His378S) Lys263S                     crystal structure catalytic portion human HMG- <br /> (Fig. 5), apparent cognates active-site residues His381                 CoA reductase (18) confirms large differences architec- <br /> P. mevalonii (His381P) Lys267P P. mevalonii HMG-CoA                        ture active sites human P. mevalonii enzymes. <br />  5152        WILDING ET AL.                                                                                                                                 J. BACTERIOL. <br />  <br />   agents selective inhibition class II bacterial                           map pE194, plasmid specifies inducible resistance macrolide, <br /> enzymes achievable. essential nature                                  lincosamide, streptogramin type B antibiotics. J. Bacteriol. 150:804&#226;&#8364;&#8220;814. <br />                                                                                         18.   Istvan, E. S., M. Palnitkar, S. K. Buchannan, J. Deisenhofer. 2000. <br /> zyme suggests class II HMG-CoA reductases repre-                                     Crystal structure catalytic portion human HMG-CoA reductase: <br /> sent promising target antibacterial agents directed                             insights regulation activity catalysis. EMBO J. 19:819&#226;&#8364;&#8220;830. <br /> multidrug-resistant gram-positive cocci.                                                19.   Johnson, E. , W.  Schroeder. 1996. Microbial carotenoids. Adv. <br />                                                                                               Biochem. Eng. Biotechnol. 53:119&#226;&#8364;&#8220;178. <br />                                                                                         20.   Jordan-Starck, T. C., V. W. Rodwell. 1989. Pseudomonas mevalonii <br />                            ACKNOWLEDGMENTS                                                    3-hydroxy-3-methylglutaryl-CoA reductase characterization <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> <br />                                                                                               modification. J. Biol. Chem. 264:17913&#226;&#8364;&#8220;17918. <br />    E. Imogen Wilding Dong-Yul Kim contributed equally                                                                                         21.   Khan, S., R. P. Novick. 1983. Complete nucleotide sequence pT181, <br /> work.                                                                                         tetracycline-resistance plasmid Staphylococcus aureus. Plasmid 10: <br />    Purdue contribution funded National Institutes                               251&#226;&#8364;&#8220;259. <br /> Health grants HL 47113 (V.W.R.) 52115 (C.V.S.). SmithKline                      22.   Kim, D.-Y., D.  Bochar, C. V. Stauffacher, V. W. Rodwell. 1999. <br /> Beecham contribution funded DARPA grant N65236-                                Expression characterization HMG-CoA reductase thermo- <br /> 97-1-5810.                                                                                    philic archaeon Sulfolobus solfataricus. Protein Exp. Purif. 17:435&#226;&#8364;&#8220;442. <br />    SmithKline Beecham thank Chantal Petit Howard                            23.   Kim, D.-Y., C. V. Stauffacher, V. W. Rodwell. 2000. Expression Kallender helpful suggestions manuscript, Christopher                        characterization class II HMG-CoA reductase hyperthermo- <br /> Traini, Thomas Mathie, Stephanie van Horn synthesis                            philic archaeon Archaeoglobus fulgidus. Protein Sci. 9:1226&#226;&#8364;&#8220;1234. <br />                                                                                         24.   Lawrence, C. M., V. W. Rodwell, C. V. Stauffacher. 1995. Crystal struc- <br /> oligonucleotides sequencing plasmid constructs, Nicole                             ture Pseudomonas mevalonii HMG-CoA reductase 3.0 angstrom reso- <br /> Robertson Dean McNulty matrix-assisted laser desorption                               lution. Science 268:1758&#226;&#8364;&#8220;1762. <br /> ionization mass spectral N-terminal sequence analysis.                              25.   Leyland, M. L., D. J. Kelly. 1991. Purification characterization                                                                                               monomeric isocitrate dehydrogenase dual coenzyme specificity                                   REFERENCES                                                  photosynthetic bacterium Rhodomicrobium vannielii. Eur. J. Biochem. 202: <br />  1. Alberts,  W., J. Chen, G. Kuron, V. Hunt, J. Huff, C. Hoffman, J. Rothrock,             85&#226;&#8364;&#8220;93. <br />     M. Lopez, H. Joshua, E. Harris,  Patchett, R. Monaghan, S. Currie, E.             26.   Maines, M. D., B. V. Polevoda, T. J. Huang, W. K. McCoubrey, Jr. 1996. <br />     Stapley, G. Albers-Schonberg, O. Hensens, J. Hirshfield, K. Hoogsteen, J.                 Human biliverdin IX&#226;?&#163; reductase zinc-metalloprotein. Characterization     Liesch, J. Springer. 1980. Mevinolin: highly potent competitive inhib-              purified Escherichia coli expressed enzymes. Eur. J. Biochem. 235:372&#226;&#8364;&#8220;381. <br />     itor hydroxymethylglutaryl-coenzyme reductase cholesterol-low-           27.   Marmur, J. 1961. procedure isolation deoxyribonucleic acid <br />     ering agent. Proc. Natl. Acad. Sci. USA 77:3957&#226;&#8364;&#8220;3961.                                     micro-organisms. J. Mol. Biol. 3:208&#226;&#8364;&#8220;218. <br />  2. al-Kassim, L. S., C. S. Tsai. 1993. Purification kinetic characteriza-      28.   Maulik, P., S. Ghosh. 1986. NADPH/NADH-dependent cold-labile <br />     tion pickerel liver alcohol dehydrogenase dual coenzyme specificity.              glutamate dehydrogenase Azospirillum brasilense. Purification prop- <br />     Biochem. Cell Biol. 71:421&#226;&#8364;&#8220;426.                                                           erties. Eur. J. Biochem. 155:595&#226;&#8364;&#8220;602. <br />  3. Anderson, B. M., C. D. Anderson. 1995. Purification characteriza-           29.   Meganathan, R. 1996. Biosynthesis isoprenoid quinones menaquinone <br />     tion Azotobacter vinelandii glucose-6-phosphate dehydrogenase: dual                 (<span id='am-11' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span>vitamin</span> K2) <span id='am-32' about='obo:IMR_0200483' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-33' property="rdfs:label" content="Ubiquinone" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0200483" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00399" datatype="xsd:string"></span>ubiquinone</span> (coenzyme Q), p. 642&#226;&#8364;&#8220;656. F. C. Neidhardt, <br />     enzyme specificity. Arch. Biochem. Biophys. 321:94&#226;&#8364;&#8220;100.                                   R. Curtiss III, J. L. Ingraham, E. C. C. Lin, K. B. Low, B. Magasanik, W. S. <br />  4. Beach, M. J., V. W. Rodwell. 1989. Cloning, sequencing, overex-                   Reznikoff, M. Riley, M. Schaechter, H. E. Umbarger (ed.), Escherichia <br />     pression mvaA, encodes Pseudomonas mevalonii 3-hydroxy-3-meth-                   coli Salmonella: cellular molecular biology, 2nd ed., vol. 1. American <br />     ylglutaryl coenzyme reductase. J. Bacteriol. 171:2994&#226;&#8364;&#8220;3001.                             Society Microbiology, Washington, D.C. <br />  5. Ben-Bassat, ,  Goldberg. 1980. Purification properties glu-          30.   Neuhauser, W. D., D. Haltrich, K. D. Kulbe, B. Nidetzky. 1997. <br />     cose-6-phosphate dehydrogenase (NADP&#226;&#171;&#185;/NAD&#226;&#171;&#185;) 6-phosphogluco-                           NAD(P)H-dependent aldose reductase xylose-assimilating yeast <br />     nate dehydrogenase (NADP&#226;&#171;&#185;/NAD&#226;&#171;&#185;) methanol-grown Pseudomonas                           Candida tenuis. Isolation, characterization biochemical properties     C. Biochim. Biophys. Acta 611:1&#226;&#8364;&#8220;10.                                                       enzyme. <br /> </body></html>