<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>2304                                                                       Biophysical Journal   Volume 91   September 2006      2304 8211;2311 <br />  <br />  <br /> Using Topology Metabolic Networks Predict Viability Mutant Strains <br />  <br /> Zeba Wunderlich* Leonid  Mirnyy <br /> *Biophysics Program, Harvard University, Cambridge, Massachusetts; yHarvard-MIT Division Health Sciences Technology, <br /> Cambridge, Massachusetts <br />  <br />  <br /> ABSTRACT Understanding relationships structure (topology) function biological networks central <br /> question systems biology. idea topology major determinant systems function attractive highly disputed hypothesis. structural analysis interaction networks demonstrates correlation topological properties node (<span id='am-617' about='obo:IMR_0000949' typeof='owl:Thing obo:IMR_0000001'><span id='am-618' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-619' property="oboInOwl:id" content="IMR:0000949" datatype="xsd:string"></span><span id='am-620' property="rdfs:label" content="protein" datatype="xsd:string"></span>protein</span>, gene) network functional essentiality, analysis metabolic networks <br /> fails  65533;?nd correlations. contrast, approaches utilizing topology biochemical parameters metabolic <br /> networks, e.g.,  64258;ux balance analysis, successful predicting phenotypes knockout strains. reconcile seemingly con 64258;icting results showing topology metabolic networks Escherichia coli Saccharomyces cerevisiae  fact, suf 65533;?cient predict viability knockout strains accuracy comparable  64258;ux <br /> balance analysis large, unbiased mutant data sets. surprising result obtained introducing novel topology-based <br /> measure network transport: synthetic accessibility. popular topology-based characteristics node degree, graph diameter, node usage (betweenness) fail predict viability E. coli mutant strains. success synthetic accessibility demonstrates ability capture essential properties metabolic network, branching chemical reactions directed transport material inputs outputs. results strongly support link topology function biological networks  agreement recent genetic studies, emphasize minimal <br /> role  64258;ux rerouting providing robustness mutant strains. <br />  <br />  <br /> INTRODUCTION suggested debated idea topology                    single-gene knockout strains. <span id='am-432' about='obo:IMR_0701861' typeof='owl:Thing'><span id='am-433' property="oboInOwl:hasExactSynonym" content="E2F-binding site-modulating activity protein" datatype="xsd:string"></span><span id='am-434' property="oboInOwl:hasExactSynonym" content="Transcription factor E2F6" datatype="xsd:string"></span><span id='am-435' property="rdfs:label" content="E2F transcription factor 6" datatype="xsd:string"></span><span id='am-436' property="rdfs:label" content="Transcription factor E2F6" datatype="xsd:string"></span><span id='am-437' property="oboInOwl:id" content="IMR:0701861" datatype="xsd:string"></span><span id='am-438' property="rdfs:label" content="E2F-6" datatype="xsd:string"></span><span id='am-439' property="oboInOwl:hasExactSynonym" content="E2f6" datatype="xsd:string"></span><span id='am-440' property="rdfs:label" content="EMA" datatype="xsd:string"></span><span id='am-441' property="oboInOwl:hasExactSynonym" content="E2F-6" datatype="xsd:string"></span><span id='am-442' property="oboInOwl:hasExactSynonym" content="EMA" datatype="xsd:string"></span><span id='am-443' property="oboInOwl:hasDbXref" content="UniProt:O54917" datatype="xsd:string"></span><span id='am-444' property="rdfs:label" content="E2F6_MOUSE" datatype="xsd:string"></span><span id='am-445' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-446' property="rdfs:label" content="E2F-binding site-modulating activity protein" datatype="xsd:string"></span><span id='am-447' property="oboInOwl:hasExactSynonym" content="E2F transcription factor 6" datatype="xsd:string"></span><span id='am-448' property="rdfs:label" content="E2f6" datatype="xsd:string"></span>EMA</span> predict viability <br /> determines network function. structures               mutant strains comparable accuracy. biological networks available, remains hard separate            methods use network topology, stoichiometry contributions topology contributions                   metabolic chemical transport reactions, cases, <br /> kinetic equilibrium parameters.                 maximal rates reactions, established structures wealth related experimental             separate role topology role played data, Escherichia coli Saccharomyces cerevisiae                   parameters network function. addition, metabolic networks perfect model systems explore               complexity method results, EMA techniques <br /> role network topology. topology metabolic network              computationally expensive (12) provide little insight <br /> sufficient predict viability knockout mutants?                  certain mutations lethal,    Metabolic networks modeled extensively using                 tolerated. <br /> steady-state flux balance approaches (1 8211;8). test                      untangle topology stoichiometry capabilities metabolic network models, groups                metabolic network topology sufficient <br /> compared predicted experimentally measured effects                 predict viability E. coli S. cerevisiae <br /> gene deletions cell growth. effective                   mutant strains accurately FBA large, unbiased sets <br /> methods flux balance analysis (FBA) (3,4,6 8211;9),                 mutants (9,13,14). result supports claim lated minimization metabolic adjustment (MOMA) method                  topology plays central role determining network func- <br /> (10), elementary mode analysis (EMA) (11).                   tion malfunction (15,16). employ novel network methods shown useful understand-                 property, synthetic accessibility, intuitive trans- <br /> ing structure dynamics metabolic fluxes,                  parent way understanding effects metabolic muta- <br /> deliver different experimentally testable predictions. FBA                tion (Fig. 1). define synthetic accessibility, S, total accurately predict fluxes individual reactions             number reactions needed transform given set input wild-type mutant strains viability              metabolites set output metabolites predict                                                                           increases S result alterations topology                                                                           metabolic network adversely affect growth. term <br /> Submitted December 30, 2005, accepted publication 24, 2006.    8216; 8216;synthetic accessibility 8217; 8217; borrowed field drug <br /> Address reprint requests Leonid Mirny, Harvard-MIT Div. Health      design, defined smallest number chem- <br /> Sciences <span id='am-365' about='obo:IMR_0702349' typeof='owl:Thing'><span id='am-366' property="oboInOwl:hasExactSynonym" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-367' property="rdfs:label" content="MGC125856" datatype="xsd:string"></span><span id='am-368' property="rdfs:label" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-369' property="rdfs:label" content="MGC125857" datatype="xsd:string"></span><span id='am-370' property="oboInOwl:hasExactSynonym" content="AMP" datatype="xsd:string"></span><span id='am-371' property="rdfs:label" content="APRT" datatype="xsd:string"></span><span id='am-372' property="oboInOwl:hasExactSynonym" content="MGC125857" datatype="xsd:string"></span><span id='am-373' property="oboInOwl:id" content="IMR:0702349" datatype="xsd:string"></span><span id='am-374' property="rdfs:label" content="DKFZp686D13177" datatype="xsd:string"></span><span id='am-375' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-376' property="rdfs:label" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-377' property="rdfs:label" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-378' property="oboInOwl:hasExactSynonym" content="APRT" datatype="xsd:string"></span><span id='am-379' property="rdfs:label" content="MGC129961" datatype="xsd:string"></span><span id='am-380' property="oboInOwl:hasExactSynonym" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-381' property="oboInOwl:hasDbXref" content="UniProt:P07741" datatype="xsd:string"></span><span id='am-382' property="rdfs:label" content="APT_HUMAN" datatype="xsd:string"></span><span id='am-383' property="oboInOwl:hasExactSynonym" content="DKFZp686D13177" datatype="xsd:string"></span><span id='am-384' property="oboInOwl:hasExactSynonym" content="MGC125856" datatype="xsd:string"></span><span id='am-385' property="oboInOwl:hasExactSynonym" content="MGC129961" datatype="xsd:string"></span><span id='am-386' property="oboInOwl:hasExactSynonym" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-387' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Technology, Massachusetts Institute Technology, 77          ical steps needed synthesize drug common labo- <br /> Massachusetts Ave., Cambridge, MA 02139. <span id='am-983' about='obo:IMR_0002980' typeof='owl:Thing obo:IMR_0000397 obo:IMR_0000949 obo:IMR_0000369 obo:IMR_0000368 obo:IMR_0000001'><span id='am-984' property="rdfs:label" content="ETV6" datatype="xsd:string"></span><span id='am-985' property="oboInOwl:hasExactSynonym" content="ETV6" datatype="xsd:string"></span><span id='am-986' property="rdfs:label" content="TEL" datatype="xsd:string"></span><span id='am-987' property="rdfs:label" content="TEL1" datatype="xsd:string"></span><span id='am-988' property="oboInOwl:hasExactSynonym" content="TEL" datatype="xsd:string"></span><span id='am-989' property="oboInOwl:id" content="IMR:0002980" datatype="xsd:string"></span><span id='am-990' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Tel</span>.: 617-452-4862; Fax:         ratory reactants similar spirit  8216; 8216;scope 8217; 8217; 617-253-2514; E-mail: leonid@mit.edu. <br />  211; 2006 Biophysical Society <br /> 0006-3495/06/09/2304/08 $2.00                                                                                doi: 10.1529/biophysj.105.080572 <br />  Topology-Based Metabolic Predictions                                                                                                                         2305 <br />  <br />  <br />                                                                                    Construction graphic metabolism model <br />                                                                                    reactions included E. coli metabolic network taken                                                                                    Edwards Palsson (4), reactions included yeast metabolic <br />                                                                                    network taken Duarte et al. (8). updated <br />                                                                                    version E. coli metabolic network available (6), chose use                                                                                    previous version enable comparison synthetic accessibility perfor- <br />                                                                                    mance previous studies (4,9 8211;11). reaction metabolite repre- <br />                                                                                    sented node, directed edges connect reactants reactions                                                                                    reactions products, accounting reversibility reactions. <br />  <br />  <br />                                                                                    Selection input output metabolite sets <br />                                                                                    input metabolites E. coli minimal medium, E. coli rich medium,                                                                                    various yeast medium conditions listed Supplementary Material, <br />                                                                                    Tables <span id='am-759' about='obo:IMR_0702938' typeof='owl:Thing'><span id='am-760' property="oboInOwl:hasExactSynonym" content="26S proteasome subunit p112" datatype="xsd:string"></span><span id='am-761' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-762' property="oboInOwl:hasExactSynonym" content="MGC133041" datatype="xsd:string"></span><span id='am-763' property="rdfs:label" content="Rpn2" datatype="xsd:string"></span><span id='am-764' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 1" datatype="xsd:string"></span><span id='am-765' property="rdfs:label" content="MGC133041" datatype="xsd:string"></span><span id='am-766' property="rdfs:label" content="S1" datatype="xsd:string"></span><span id='am-767' property="oboInOwl:hasExactSynonym" content="PSMD1" datatype="xsd:string"></span><span id='am-768' property="oboInOwl:hasExactSynonym" content="Rpn2" datatype="xsd:string"></span><span id='am-769' property="rdfs:label" content="PSMD1_HUMAN" datatype="xsd:string"></span><span id='am-770' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 1" datatype="xsd:string"></span><span id='am-771' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN2" datatype="xsd:string"></span><span id='am-772' property="rdfs:label" content="26S proteasome subunit p112" datatype="xsd:string"></span><span id='am-773' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 1" datatype="xsd:string"></span><span id='am-774' property="rdfs:label" content="26S proteasome regulatory subunit RPN2" datatype="xsd:string"></span><span id='am-775' property="oboInOwl:hasExactSynonym" content="S1" datatype="xsd:string"></span><span id='am-776' property="rdfs:label" content="P112" datatype="xsd:string"></span><span id='am-777' property="oboInOwl:hasDbXref" content="UniProt:Q99460" datatype="xsd:string"></span><span id='am-778' property="rdfs:label" content="MGC133040" datatype="xsd:string"></span><span id='am-779' property="oboInOwl:hasExactSynonym" content="P112" datatype="xsd:string"></span><span id='am-780' property="rdfs:label" content="26S proteasome regulatory subunit S1" datatype="xsd:string"></span><span id='am-781' property="oboInOwl:hasExactSynonym" content="MGC133040" datatype="xsd:string"></span><span id='am-782' property="rdfs:label" content="PSMD1" datatype="xsd:string"></span><span id='am-783' property="oboInOwl:id" content="IMR:0702938" datatype="xsd:string"></span><span id='am-784' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S1" datatype="xsd:string"></span><span id='am-785' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 1" datatype="xsd:string"></span>S1</span> 8211;<span id='am-597' about='obo:IMR_0010555' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000896 obo:IMR_0100710 obo:IMR_0000001 obo:IMR_0000950'><span id='am-598' property="oboInOwl:hasExactSynonym" content="LLREP3 protein" datatype="xsd:string"></span><span id='am-599' property="rdfs:label" content="LLREP3 protein" datatype="xsd:string"></span><span id='am-600' property="oboInOwl:id" content="IMR:0010555" datatype="xsd:string"></span><span id='am-601' property="oboInOwl:hasExactSynonym" content="S4" datatype="xsd:string"></span><span id='am-602' property="rdfs:label" content="40S ribosomal protein S2" datatype="xsd:string"></span><span id='am-603' property="rdfs:label" content="S4" datatype="xsd:string"></span><span id='am-604' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-605' property="rdfs:label" content="40S small ribosomal protein 2" datatype="xsd:string"></span><span id='am-606' property="oboInOwl:hasExactSynonym" content="40S small ribosomal protein 2" datatype="xsd:string"></span>S4</span>. E. coli minimal medium consists energy source <br />                                                                                    (glucose, acetate, glycerol, succinate), components minimal medium, <br />                                                                                    sulfur source, carbon dioxide oxygen, nicotinamide mononucleotide, <br />                                                                                    regulatory protein thioredoxin (Supplementary Material, Table S1). <br />                                                                                    input metabolites chosen match real composition minimal <br /> FIGURE 1 illustration synthetic accessibility approach.          medium closely possible. Nicotinamide mononucleotide thiore- <br /> representation metabolic network, circular nodes represent metabo-          doxin included ensure  wild-type network, components <br /> lites, rectangular nodes represent reactions, directed edges indicate    output biomass accessible. chosen specifically relationships. Nodes outline (green blue) synthetically        upstream metabolites biomass synthesis pathways. <br /> accessible, nodes outline (red orange) accessible.      E. coli rich medium consists metabolites minimal medium algorithm begins identifying reactions neighbor input      biotin, riboflavin, pantoate, pyridoxine, thiamin, dihydrofolate, <br /> metabolites (nodes A 8211;D) marking reactions                p-aminobenzoic acid, 20 amino acids, nucleotide bases <br /> reactants available accessible (reactions 1 2). products     included external metabolites metabolic network (external thymine reactions marked accessible (nodes F 8211;H). algorithm              metabolic network). Rich medium difficult model <br /> examines neighboring reactions newly marked metabolites           accurately, using slightly different input metabolite sets <span id='am-245' about='obo:IMR_0703879' typeof='owl:Thing'><span id='am-246' property="oboInOwl:hasExactSynonym" content="STEP" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Ptpn5" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasExactSynonym" content="Ptpn5" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:id" content="IMR:0703879" datatype="xsd:string"></span><span id='am-251' property="oboInOwl:hasExactSynonym" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-252' property="rdfs:label" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-253' property="rdfs:label" content="PTN5_RAT" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-255' property="rdfs:label" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-256' property="rdfs:label" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasExactSynonym" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-258' property="rdfs:label" content="STEP" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-260' property="rdfs:label" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-261' property="rdfs:label" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-262' property="oboInOwl:hasExactSynonym" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-263' property="oboInOwl:hasDbXref" content="UniProt:P35234" datatype="xsd:string"></span>step</span> continues new metabolites marked accessible.       significant effect results (results shown). inset demonstrates happens gene produces <span id='am-154' about='obo:IMR_0000207' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001'><span id='am-155' property="rdfs:label" content="enzyme" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:id" content="IMR:0000207" datatype="xsd:string"></span><span id='am-158' property="obo:IAO_0000115" content="Protein that catalyze a specific chemical reaction." datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasDbXref" content="GO:0003824" datatype="xsd:string"></span>enzyme</span>               input metabolites yeast based descriptions Duarte catalyzes reaction 2 deleted: metabolites H K reaction 5         et al. (8) include histidine, leucine, uracil compensate accessible  define synthetic accessibility, S,         deletions 3/Leu-2/Ura-3 mutant strains. Additionally, number reactions required transform set inputs set        thioredoxin (oxidized), H1  endoplasmic reticulum), <span id='am-222' about='obo:IMR_0702411' typeof='owl:Thing'><span id='am-223' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-224' property="rdfs:label" content="DECR_HUMAN" datatype="xsd:string"></span><span id='am-225' property="rdfs:label" content="SDR18C1" datatype="xsd:string"></span><span id='am-226' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasDbXref" content="UniProt:Q16698" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasExactSynonym" content="2,4-dienoyl-CoA reductase [NADPH]" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasExactSynonym" content="4-enoyl-CoA reductase [NADPH]" datatype="xsd:string"></span><span id='am-230' property="rdfs:label" content="2,4-dienoyl-CoA reductase [NADPH]" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:id" content="IMR:0702411" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasExactSynonym" content="NADPH" datatype="xsd:string"></span><span id='am-233' property="rdfs:label" content="4-enoyl-CoA reductase [NADPH]" datatype="xsd:string"></span><span id='am-234' property="rdfs:label" content="DECR1" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasExactSynonym" content="SDR18C1" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasExactSynonym" content="2,4-dienoyl-CoA reductase, mitochondrial" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasExactSynonym" content="2,4-dienoyl CoA reductase 1, mitochondrial" datatype="xsd:string"></span><span id='am-238' property="rdfs:label" content="2,4-dienoyl CoA reductase 1, mitochondrial" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasExactSynonym" content="DECR1" datatype="xsd:string"></span><span id='am-240' property="rdfs:label" content="EC 1.3.1.34" datatype="xsd:string"></span><span id='am-241' property="oboInOwl:hasExactSynonym" content="EC 1.3.1.34" datatype="xsd:string"></span><span id='am-242' property="rdfs:label" content="2,4-dienoyl-CoA reductase, mitochondrial" datatype="xsd:string"></span><span id='am-243' property="oboInOwl:hasExactSynonym" content="DECR" datatype="xsd:string"></span><span id='am-244' property="rdfs:label" content="DECR" datatype="xsd:string"></span>NADPH</span>  outputs. Synthetic accessibility analogous diameter directed        endoplasmic reticulum), dolichol included inputs, graph, contrast graph diameters, synthetic accessibility takes       components biomass producible, account branching nature chemical reactions purpose              wild-type network. <br /> metabolic networks, produce outputs inputs.                                    E. coli output metabolites taken components E. coli <br />                                                                                    biomass (Supplementary Material, Table S5) (19). yeast output metabo- <br /> metabolites (17,18). demonstrate network                        lites components biomass reaction reported (8). <br /> characteristics node degree change graph <br /> diameter unable predict viability E. coli mutant <br />                                                                                    Synthetic accessibility algorithm <br /> strains better random predictions, suggesting synthetic <br /> accessibility appropriate characteristic networks                    determine synthetic accessibility outputs given inputs, directed transport, metabolic networks.                               use type iterative breadth search, similar previously described <br />                                                                                     8216; 8216;forward-firing 8217; 8217; (Fig. 1) (20). algorithm starts examining                                                                                    reactions require given input metabolites reactant. MATERIALS METHODS                                                              marks reactions reactants available  8216; 8216;accessible 8217; 8217; <br />                                                                                    marks metabolites produced reactions  8216; 8216;accessible 8217; 8217; De 65533;?nition synthetic accessibility                                                algorithm examines reactions require newly <br />                                                                                    marked metabolites starting material, determines reaction <br /> Consider metabolic network access certain inputs: substrates         accessible based availability reactants, consumed environment (e.g., sugars, oxygen, nitrogen),           new metabolites marked accessible. Concurrently, number aim producing certain outputs amino acids, nucleotides,         steps needed reach accessible metabolite j, synthetic accessibility components collectively called biomass. define synthetic          Sj, recorded; synthetic accessibility network S calculated accessibility Sj output j minimal number metabolic reactions       summing synthetic accessibilities outputs. <br /> needed produce j network inputs (Fig. 1). Sj set infinity j synthesized network inputs. Summing synthetic <br /> accessibility components biomass, obtain total              Comparison experimental results <br /> synthetic accessibility S  188;  Si biomass. propose enzyme knockout does change S, e., biomass produced               compare results approach experimental data sets, extra metabolic cost, mutant viable. S  188; N,         create adjacency matrix, represents wild-type metabolic <span id='am-449' about='obo:IMR_0702135' typeof='owl:Thing'><span id='am-450' property="oboInOwl:hasExactSynonym" content="ELK3, ETS-DOMAIN PROTEIN" datatype="xsd:string"></span><span id='am-451' property="rdfs:label" content="SAP2" datatype="xsd:string"></span><span id='am-452' property="oboInOwl:hasExactSynonym" content="SAP-2" datatype="xsd:string"></span><span id='am-453' property="oboInOwl:hasExactSynonym" content="SRF accessory protein 2" datatype="xsd:string"></span><span id='am-454' property="rdfs:label" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-455' property="oboInOwl:hasExactSynonym" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-456' property="rdfs:label" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-457' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-458' property="oboInOwl:id" content="IMR:0702135" datatype="xsd:string"></span><span id='am-459' property="rdfs:label" content="NET" datatype="xsd:string"></span><span id='am-460' property="rdfs:label" content="ELK3_HUMAN" datatype="xsd:string"></span><span id='am-461' property="rdfs:label" content="ETS-RELATED PROTEIN" datatype="xsd:string"></span><span id='am-462' property="rdfs:label" content="ELK3" datatype="xsd:string"></span><span id='am-463' property="oboInOwl:hasExactSynonym" content="NET" datatype="xsd:string"></span><span id='am-464' property="oboInOwl:hasExactSynonym" content="ELK3" datatype="xsd:string"></span><span id='am-465' property="rdfs:label" content="ERP" datatype="xsd:string"></span><span id='am-466' property="rdfs:label" content="ELK3, ETS-domain protein (SRF accessory protein 2)" datatype="xsd:string"></span><span id='am-467' property="oboInOwl:hasExactSynonym" content="SAP2" datatype="xsd:string"></span><span id='am-468' property="rdfs:label" content="Serum response factor accessory protein 2" datatype="xsd:string"></span><span id='am-469' property="oboInOwl:hasDbXref" content="UniProt:P41970" datatype="xsd:string"></span><span id='am-470' property="oboInOwl:hasExactSynonym" content="ERP" datatype="xsd:string"></span><span id='am-471' property="rdfs:label" content="SAP-2" datatype="xsd:string"></span><span id='am-472' property="oboInOwl:hasExactSynonym" content="Serum response factor accessory protein 2" datatype="xsd:string"></span><span id='am-473' property="oboInOwl:hasExactSynonym" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-474' property="oboInOwl:hasExactSynonym" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-475' property="oboInOwl:hasExactSynonym" content="ELK3, ETS-domain protein (SRF accessory protein 2)" datatype="xsd:string"></span><span id='am-476' property="rdfs:label" content="ELK3, ETS-DOMAIN PROTEIN" datatype="xsd:string"></span><span id='am-477' property="rdfs:label" content="SRF accessory protein 2" datatype="xsd:string"></span><span id='am-478' property="oboInOwl:hasExactSynonym" content="ETS-RELATED PROTEIN" datatype="xsd:string"></span><span id='am-479' property="rdfs:label" content="ETS-related protein ERP" datatype="xsd:string"></span>net</span>- <br /> essential component biomass produced network                 work topology.  mutant strain, create  8216; 8216;mutated 8217; 8217; adja- <br /> inputs, predict lethal phenotype.                               cency matrix removing reactions catalyzed gene.  <br />                                                                                                                       Biophysical Journal 91(6) 2304 8211;2311 <br />  2306                                                                                                                                       Wunderlich Mirny <br />  <br /> E. coli predictions, previous papers, delete corresponding           unrelated reactions. measure performance synthetic acces- <br /> genes reactions catalyzed multiple isozymes. calculate              sibility erroneous network plotting accuracy percentage <br /> viability mutant compare results experimental data          shuffled assignments. <br /> Supplementary Material). Smutant  188; Swild-type, predict mutant viable; predict inviable. E. coli insertional mutant data set, <br /> phenotype data given competitive growth rates. mutant                      RESULTS <br /> considered negatively selected  inviable) twofold decrease growth rates 30 generations (9). Gerdes et al. data set (21),       Performance synthetic accessibility create mutated adjacency matrices genes included metabolic             E. coli metabolic network <br /> network model, resulting 598 mutated adjacency matrices. yeast <br /> experimental data, use preprocessed data set created Duarte et al.           study performance synthetic accessibility (8) simultaneously delete isozymes.                                         predicting viability knockout strains compare                                                                                        previous studies, applied method E. coli <br />                                                                                        metabolic network. initially tested data sets <br /> Calculation topology-based predictions <br />                                                                                        used previous studies: large, unbiased data set explore number topology-based measures predictors                  insertional (transposon-induced) mutants (9) smaller <br /> E. coli mutant viability, including node degree, diameter, node usage. <br />                                                                                        data set collected FBA analysis (4), mainly degree enzyme calculated summing degrees reactions catalyzed enzyme isozymes. define network                  contained knockouts enzymes involved central metab- <br /> diameter sum metabolites versus metabolites 8217; shortest paths,         olism. mutants grown minimal medium. mutant, calculate change network diameter wild-            used data sets specifically used type. define node usage enzyme number times                  previous studies (4,9 8211;11), compared results. <br /> reactions catalyzed enzyme used produce biomass wild- <br />                                                                                        used union data sets refer type strain, according synthetic accessibility approach, essentially analogous betweenness (22,23). measure, degree,                combined data set. applied combined data <br /> diameter, usage, predict enzyme essential                set, approach performed (62% accuracy, p  188; 6 3 corresponding mutant stain inviable), measure greater            10 8) FBA approach (62%, p  188; 3 3 10 8)  Table given cutoff. vary cutoff entire range possible         1, Fig. 2 details). large data set 487 insertional <br /> values value gives optimal performance, measured                                                                                        mutants (9), synthetic accessibility approach performed accuracy significance x 2 statistic. <br />                                                                                        (60% accuracy, p  188; 3 3 10 5) FBA                                                                                        MOMA approaches (58% 59% accuracy, p  188; 1 3 10 3 <br /> Quantitative analysis performance                                                   1 3 10 4, respectively), somewhat higher <br />                                                                                        statistical significance. smaller data set 79 mutants assess performance synthetic accessibility methods predicting phenotype mutant stains, use measures: accuracy,             (4), FBA correctly predicted 86% cases, sensitivity, specificity, p-value x 2 statistic. define accuracy     topology-based synthetic accessibility approach 71% (<span id='am-621' about='obo:IMR_0200780' typeof='owl:Thing'><span id='am-622' property="oboInOwl:hasExactSynonym" content="Platelet-derived endothelial cell growth factor" datatype="xsd:string"></span><span id='am-623' property="oboInOwl:hasExactSynonym" content="PDECGF" datatype="xsd:string"></span><span id='am-624' property="oboInOwl:hasExactSynonym" content="TYMP" datatype="xsd:string"></span><span id='am-625' property="rdfs:label" content="MEDPS1" datatype="xsd:string"></span><span id='am-626' property="oboInOwl:hasExactSynonym" content="thymidine phosphorylase" datatype="xsd:string"></span><span id='am-627' property="rdfs:label" content="EC 2.4.2.4" datatype="xsd:string"></span><span id='am-628' property="oboInOwl:hasExactSynonym" content="THYMIDINE PHOSPHORYLASE: TYMP" datatype="xsd:string"></span><span id='am-629' property="rdfs:label" content="MTDPS1" datatype="xsd:string"></span><span id='am-630' property="rdfs:label" content="PDECGF" datatype="xsd:string"></span><span id='am-631' property="rdfs:label" content="THYMIDINE PHOSPHORYLASE: TYMP" datatype="xsd:string"></span><span id='am-632' property="rdfs:label" content="ECGF1" datatype="xsd:string"></span><span id='am-633' property="oboInOwl:hasExactSynonym" content="ENDOTHELIAL CELL GROWTH FACTOR, PLATELET-DERIVED" datatype="xsd:string"></span><span id='am-634' property="rdfs:label" content="ECGF" datatype="xsd:string"></span><span id='am-635' property="oboInOwl:id" content="IMR:0200780" datatype="xsd:string"></span><span id='am-636' property="rdfs:label" content="TdRPase" datatype="xsd:string"></span><span id='am-637' property="rdfs:label" content="Platelet-derived endothelial cell growth factor" datatype="xsd:string"></span><span id='am-638' property="rdfs:label" content="Thymidine phosphorylase" datatype="xsd:string"></span><span id='am-639' property="oboInOwl:hasExactSynonym" content="MEDPS1" datatype="xsd:string"></span><span id='am-640' property="oboInOwl:hasDbXref" content="UniProt:P19971" datatype="xsd:string"></span><span id='am-641' property="oboInOwl:hasExactSynonym" content="PD-ECGF" datatype="xsd:string"></span><span id='am-642' property="rdfs:label" content="Gliostatin" datatype="xsd:string"></span><span id='am-643' property="oboInOwl:hasExactSynonym" content="EC 2.4.2.4" datatype="xsd:string"></span><span id='am-644' property="oboInOwl:hasExactSynonym" content="TP" datatype="xsd:string"></span><span id='am-645' property="rdfs:label" content="TP" datatype="xsd:string"></span><span id='am-646' property="rdfs:label" content="thymidine phosphorylase" datatype="xsd:string"></span><span id='am-647' property="rdfs:label" content="TYMP" datatype="xsd:string"></span><span id='am-648' property="rdfs:label" content="TYPH_HUMAN" datatype="xsd:string"></span><span id='am-649' property="rdfs:label" content="ENDOTHELIAL CELL GROWTH FACTOR, PLATELET-DERIVED" datatype="xsd:string"></span><span id='am-650' property="oboInOwl:hasExactSynonym" content="Thymidine phosphorylase" datatype="xsd:string"></span><span id='am-651' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-652' property="rdfs:label" content="hPD-ECGF" datatype="xsd:string"></span><span id='am-653' property="oboInOwl:hasExactSynonym" content="Gliostatin" datatype="xsd:string"></span><span id='am-654' property="oboInOwl:hasExactSynonym" content="ECGF" datatype="xsd:string"></span><span id='am-655' property="oboInOwl:hasExactSynonym" content="MNGIE" datatype="xsd:string"></span><span id='am-656' property="rdfs:label" content="MNGIE" datatype="xsd:string"></span><span id='am-657' property="oboInOwl:hasExactSynonym" content="MTDPS1" datatype="xsd:string"></span><span id='am-658' property="oboInOwl:hasAlternativeId" content="IMR:0702894" datatype="xsd:string"></span><span id='am-659' property="oboInOwl:hasExactSynonym" content="hPD-ECGF" datatype="xsd:string"></span><span id='am-660' property="oboInOwl:hasExactSynonym" content="ECGF1" datatype="xsd:string"></span><span id='am-661' property="rdfs:label" content="PD-ECGF" datatype="xsd:string"></span><span id='am-662' property="oboInOwl:hasExactSynonym" content="TdRPase" datatype="xsd:string"></span>TP</span> 1 <span id='am-915' about='obo:IMR_0701306' typeof='owl:Thing'><span id='am-916' property="oboInOwl:hasExactSynonym" content="150-225" datatype="xsd:string"></span><span id='am-917' property="rdfs:label" content="Glioma-associated-extracellular matrix antigen" datatype="xsd:string"></span><span id='am-918' property="rdfs:label" content="GP 150-225" datatype="xsd:string"></span><span id='am-919' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-920' property="oboInOwl:hasExactSynonym" content="HXB" datatype="xsd:string"></span><span id='am-921' property="oboInOwl:hasExactSynonym" content="TN" datatype="xsd:string"></span><span id='am-922' property="oboInOwl:hasExactSynonym" content="Tenascin-C" datatype="xsd:string"></span><span id='am-923' property="rdfs:label" content="MGC167029" datatype="xsd:string"></span><span id='am-924' property="oboInOwl:hasExactSynonym" content="Cytotactin" datatype="xsd:string"></span><span id='am-925' property="oboInOwl:hasExactSynonym" content="GP" datatype="xsd:string"></span><span id='am-926' property="oboInOwl:hasExactSynonym" content="GP 150-225" datatype="xsd:string"></span><span id='am-927' property="oboInOwl:hasExactSynonym" content="TNC" datatype="xsd:string"></span><span id='am-928' property="oboInOwl:hasExactSynonym" content="JI" datatype="xsd:string"></span><span id='am-929' property="oboInOwl:hasExactSynonym" content="GMEM" datatype="xsd:string"></span><span id='am-930' property="oboInOwl:hasExactSynonym" content="tenascin C" datatype="xsd:string"></span><span id='am-931' property="rdfs:label" content="150-225" datatype="xsd:string"></span><span id='am-932' property="rdfs:label" content="Neuronectin" datatype="xsd:string"></span><span id='am-933' property="rdfs:label" content="TN-C" datatype="xsd:string"></span><span id='am-934' property="oboInOwl:hasExactSynonym" content="Hexabrachion" datatype="xsd:string"></span><span id='am-935' property="rdfs:label" content="TENA_HUMAN" datatype="xsd:string"></span><span id='am-936' property="rdfs:label" content="TN" datatype="xsd:string"></span><span id='am-937' property="rdfs:label" content="Hexabrachion" datatype="xsd:string"></span><span id='am-938' property="oboInOwl:hasDbXref" content="UniProt:P24821" datatype="xsd:string"></span><span id='am-939' property="rdfs:label" content="HXB" datatype="xsd:string"></span><span id='am-940' property="rdfs:label" content="Tenascin" datatype="xsd:string"></span><span id='am-941' property="oboInOwl:hasExactSynonym" content="Glioma-associated-extracellular matrix antigen" datatype="xsd:string"></span><span id='am-942' property="oboInOwl:hasExactSynonym" content="MGC167029" datatype="xsd:string"></span><span id='am-943' property="rdfs:label" content="Cytotactin" datatype="xsd:string"></span><span id='am-944' property="rdfs:label" content="JI" datatype="xsd:string"></span><span id='am-945' property="rdfs:label" content="GMEM" datatype="xsd:string"></span><span id='am-946' property="oboInOwl:hasExactSynonym" content="Myotendinous antigen" datatype="xsd:string"></span><span id='am-947' property="rdfs:label" content="tenascin C" datatype="xsd:string"></span><span id='am-948' property="oboInOwl:hasExactSynonym" content="Tenascin" datatype="xsd:string"></span><span id='am-949' property="oboInOwl:id" content="IMR:0701306" datatype="xsd:string"></span><span id='am-950' property="rdfs:label" content="Myotendinous antigen" datatype="xsd:string"></span><span id='am-951' property="rdfs:label" content="GP" datatype="xsd:string"></span><span id='am-952' property="rdfs:label" content="Tenascin-C" datatype="xsd:string"></span><span id='am-953' property="oboInOwl:hasExactSynonym" content="TN-C" datatype="xsd:string"></span><span id='am-954' property="oboInOwl:hasExactSynonym" content="Neuronectin" datatype="xsd:string"></span><span id='am-955' property="rdfs:label" content="TNC" datatype="xsd:string"></span>TN</span>)/(TP 1 <span id='am-18' about='obo:IMR_0700460' typeof='owl:Thing'><span id='am-19' property="oboInOwl:hasExactSynonym" content="troponin I, cardiac 3" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="TNNI3_MOUSE" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Tn1" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="cTnI" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Tnni3" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="Troponin I, cardiac muscle" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Troponin I, cardiac muscle" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="cTnI" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="UniProt:P48787" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="Cardiac troponin I" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Cardiac troponin I" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="Tn1" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="cardiac troponin I" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="cardiac troponin I" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0700460" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Tnni3" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="troponin I, cardiac 3" datatype="xsd:string"></span>TN 1</span> <span id='am-1' about='obo:IMR_0703622' typeof='owl:Thing'><span id='am-2' property="rdfs:label" content="Sdha" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="DHSA_RAT" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Flavoprotein subunit of complex II" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="EC 1.3.5.1" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="UniProt:Q920L2" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Fp" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Sdha" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="succinate dehydrogenase complex, subunit A, flavoprotein (Fp)" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0703622" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="succinate dehydrogenase complex, subunit A, flavoprotein (Fp)" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="Flavoprotein subunit of complex II" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Succinate dehydrogenase [ubiquinone] flavoprotein subunit, mitochondrial" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="Fp" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="EC 1.3.5.1" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="Succinate dehydrogenase [ubiquinone] flavoprotein subunit, mitochondrial" datatype="xsd:string"></span>FP</span> 1 <span id='am-836' about='obo:IMR_0701299' typeof='owl:Thing'><span id='am-837' property="rdfs:label" content="DKFZp686O13149" datatype="xsd:string"></span><span id='am-838' property="rdfs:label" content="Fibronectin" datatype="xsd:string"></span><span id='am-839' property="oboInOwl:hasExactSynonym" content="DKFZp686O13149" datatype="xsd:string"></span><span id='am-840' property="oboInOwl:hasExactSynonym" content="DKFZp686H0342" datatype="xsd:string"></span><span id='am-841' property="rdfs:label" content="MSF" datatype="xsd:string"></span><span id='am-842' property="rdfs:label" content="Ugl-Y2" datatype="xsd:string"></span><span id='am-843' property="oboInOwl:hasExactSynonym" content="DKFZp686F10164" datatype="xsd:string"></span><span id='am-844' property="rdfs:label" content="FINC_HUMAN" datatype="xsd:string"></span><span id='am-845' property="oboInOwl:id" content="IMR:0701299" datatype="xsd:string"></span><span id='am-846' property="oboInOwl:hasExactSynonym" content="Ugl-Y3" datatype="xsd:string"></span><span id='am-847' property="rdfs:label" content="Cold-insoluble globulin" datatype="xsd:string"></span><span id='am-848' property="oboInOwl:hasExactSynonym" content="FINC" datatype="xsd:string"></span><span id='am-849' property="rdfs:label" content="FNZ" datatype="xsd:string"></span><span id='am-850' property="rdfs:label" content="GFND2" datatype="xsd:string"></span><span id='am-851' property="oboInOwl:hasExactSynonym" content="Cold-insoluble globulin" datatype="xsd:string"></span><span id='am-852' property="oboInOwl:hasExactSynonym" content="LETS" datatype="xsd:string"></span><span id='am-853' property="rdfs:label" content="FN1" datatype="xsd:string"></span><span id='am-854' property="oboInOwl:hasExactSynonym" content="CIG" datatype="xsd:string"></span><span id='am-855' property="oboInOwl:hasExactSynonym" content="DKFZp686I1370" datatype="xsd:string"></span><span id='am-856' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-857' property="rdfs:label" content="FN" datatype="xsd:string"></span><span id='am-858' property="rdfs:label" content="Anastellin" datatype="xsd:string"></span><span id='am-859' property="oboInOwl:hasExactSynonym" content="Fibronectin" datatype="xsd:string"></span><span id='am-860' property="rdfs:label" content="DKFZp686F10164" datatype="xsd:string"></span><span id='am-861' property="oboInOwl:hasExactSynonym" content="FN" datatype="xsd:string"></span><span id='am-862' property="rdfs:label" content="FINC" datatype="xsd:string"></span><span id='am-863' property="oboInOwl:hasExactSynonym" content="FN1" datatype="xsd:string"></span><span id='am-864' property="oboInOwl:hasExactSynonym" content="GFND" datatype="xsd:string"></span><span id='am-865' property="rdfs:label" content="ED-B" datatype="xsd:string"></span><span id='am-866' property="rdfs:label" content="fibronectin 1" datatype="xsd:string"></span><span id='am-867' property="rdfs:label" content="GFND" datatype="xsd:string"></span><span id='am-868' property="oboInOwl:hasExactSynonym" content="Anastellin" datatype="xsd:string"></span><span id='am-869' property="rdfs:label" content="LETS" datatype="xsd:string"></span><span id='am-870' property="oboInOwl:hasExactSynonym" content="Ugl-Y1" datatype="xsd:string"></span><span id='am-871' property="rdfs:label" content="Ugl-Y3" datatype="xsd:string"></span><span id='am-872' property="rdfs:label" content="CIG" datatype="xsd:string"></span><span id='am-873' property="oboInOwl:hasExactSynonym" content="GFND2" datatype="xsd:string"></span><span id='am-874' property="oboInOwl:hasDbXref" content="UniProt:P02751" datatype="xsd:string"></span><span id='am-875' property="oboInOwl:hasExactSynonym" content="ED-B" datatype="xsd:string"></span><span id='am-876' property="rdfs:label" content="DKFZp686H0342" datatype="xsd:string"></span><span id='am-877' property="oboInOwl:hasExactSynonym" content="MSF" datatype="xsd:string"></span><span id='am-878' property="rdfs:label" content="Ugl-Y1" datatype="xsd:string"></span><span id='am-879' property="oboInOwl:hasExactSynonym" content="fibronectin 1" datatype="xsd:string"></span><span id='am-880' property="oboInOwl:hasExactSynonym" content="Ugl-Y2" datatype="xsd:string"></span><span id='am-881' property="rdfs:label" content="DKFZp686I1370" datatype="xsd:string"></span><span id='am-882' property="oboInOwl:hasExactSynonym" content="FNZ" datatype="xsd:string"></span>FN</span>), TP number true                       accuracy, providing correct predictions 53/68  188; 78% positives, TN number true negatives, <span id='am-38' about='obo:IMR_0702418' typeof='owl:Thing'><span id='am-39' property="oboInOwl:hasExactSynonym" content="SUCCINATE DEHYDROGENASE COMPLEX, SUBUNIT A, FLAVOPROTEIN" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="SUCCINATE DEHYDROGENASE COMPLEX, SUBUNIT A, FLAVOPROTEIN" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="SDH2" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Flavoprotein subunit of complex II" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="CMD1GG" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="SDH2" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Succinate dehydrogenase [ubiquinone] flavoprotein subunit, mitochondrial" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="SDHA" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="FP" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="succinate dehydrogenase complex, subunit A, flavoprotein (Fp)" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="Flavoprotein subunit of complex II" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="DHSA_HUMAN" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="SDH1, HOMOLOG OF" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="UniProt:P31040" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="EC 1.3.5.1" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="SDHF" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Fp" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="SDH1" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="FP" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="SUCCINATE DEHYDROGENASE 1, S. CEREVISIAE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasExactSynonym" content="succinate dehydrogenase complex, subunit A, flavoprotein (Fp)" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="SDHF" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="Fp" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="Succinate dehydrogenase [ubiquinone] flavoprotein subunit, mitochondrial" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="SDH1" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:id" content="IMR:0702418" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="EC 1.3.5.1" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="SUCCINATE DEHYDROGENASE 1, S. CEREVISIAE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="CMD1GG" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="SDH1, HOMOLOG OF" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="SDHA" datatype="xsd:string"></span>FP</span> number false               cases predicted correctly FBA. <br /> positives, <span id='am-883' about='obo:IMR_0701301' typeof='owl:Thing'><span id='am-884' property="rdfs:label" content="fibronectin 1" datatype="xsd:string"></span><span id='am-885' property="oboInOwl:hasDbXref" content="UniProt:P11276" datatype="xsd:string"></span><span id='am-886' property="oboInOwl:hasExactSynonym" content="FN" datatype="xsd:string"></span><span id='am-887' property="rdfs:label" content="FN" datatype="xsd:string"></span><span id='am-888' property="oboInOwl:hasExactSynonym" content="Fn1" datatype="xsd:string"></span><span id='am-889' property="oboInOwl:id" content="IMR:0701301" datatype="xsd:string"></span><span id='am-890' property="rdfs:label" content="Fn-1" datatype="xsd:string"></span><span id='am-891' property="oboInOwl:hasExactSynonym" content="Fn-1" datatype="xsd:string"></span><span id='am-892' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-893' property="rdfs:label" content="Fibronectin" datatype="xsd:string"></span><span id='am-894' property="rdfs:label" content="Anastellin" datatype="xsd:string"></span><span id='am-895' property="oboInOwl:hasExactSynonym" content="fibronectin 1" datatype="xsd:string"></span><span id='am-896' property="rdfs:label" content="Fn1" datatype="xsd:string"></span><span id='am-897' property="rdfs:label" content="FINC_MOUSE" datatype="xsd:string"></span><span id='am-898' property="oboInOwl:hasExactSynonym" content="Fibronectin" datatype="xsd:string"></span><span id='am-899' property="oboInOwl:hasExactSynonym" content="Anastellin" datatype="xsd:string"></span>FN</span> number false negatives. define positives negatives terms experimentally measured phenotypes,                                                                                           difference performance synthetic <br /> positives viable strains negatives inviable strains,            accessibility approach compared FBA assignment arbitrary reversed. similar fashion, define           data sets probably related way data sets <br /> sensitivity TP/(TP 1 <span id='am-71' about='obo:IMR_0702418' typeof='owl:Thing'><span id='am-72' property="oboInOwl:hasExactSynonym" content="SDH1, HOMOLOG OF" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="succinate dehydrogenase complex, subunit A, flavoprotein (Fp)" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="SDHA" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="Succinate dehydrogenase [ubiquinone] flavoprotein subunit, mitochondrial" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="SDH2" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Flavoprotein subunit of complex II" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasExactSynonym" content="SUCCINATE DEHYDROGENASE COMPLEX, SUBUNIT A, FLAVOPROTEIN" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="SDH2" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:id" content="IMR:0702418" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="SDH1, HOMOLOG OF" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Fp" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="DHSA_HUMAN" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="SDHF" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="CMD1GG" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="SDHA" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="SUCCINATE DEHYDROGENASE 1, S. CEREVISIAE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="SUCCINATE DEHYDROGENASE 1, S. CEREVISIAE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="Flavoprotein subunit of complex II" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="SDHF" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="SDH1" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="Fp" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasExactSynonym" content="FP" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="SUCCINATE DEHYDROGENASE COMPLEX, SUBUNIT A, FLAVOPROTEIN" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasExactSynonym" content="succinate dehydrogenase complex, subunit A, flavoprotein (Fp)" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasExactSynonym" content="SDH1" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="Succinate dehydrogenase [ubiquinone] flavoprotein subunit, mitochondrial" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="EC 1.3.5.1" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="EC 1.3.5.1" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="UniProt:P31040" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="FP" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="CMD1GG" datatype="xsd:string"></span>FP</span>) specificity TN/(TN 1 <span id='am-900' about='obo:IMR_0701300' typeof='owl:Thing'><span id='am-901' property="oboInOwl:hasExactSynonym" content="FN" datatype="xsd:string"></span><span id='am-902' property="oboInOwl:hasExactSynonym" content="fibronectin 1" datatype="xsd:string"></span><span id='am-903' property="oboInOwl:id" content="IMR:0701300" datatype="xsd:string"></span><span id='am-904' property="oboInOwl:hasExactSynonym" content="Anastellin" datatype="xsd:string"></span><span id='am-905' property="rdfs:label" content="fibronectin 1" datatype="xsd:string"></span><span id='am-906' property="rdfs:label" content="Fn1" datatype="xsd:string"></span><span id='am-907' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-908' property="rdfs:label" content="Anastellin" datatype="xsd:string"></span><span id='am-909' property="oboInOwl:hasExactSynonym" content="Fibronectin" datatype="xsd:string"></span><span id='am-910' property="oboInOwl:hasDbXref" content="UniProt:P04937" datatype="xsd:string"></span><span id='am-911' property="rdfs:label" content="FINC_RAT" datatype="xsd:string"></span><span id='am-912' property="oboInOwl:hasExactSynonym" content="Fn1" datatype="xsd:string"></span><span id='am-913' property="rdfs:label" content="Fibronectin" datatype="xsd:string"></span><span id='am-914' property="rdfs:label" content="FN" datatype="xsd:string"></span>FN</span>). calculate              interpreted cases included data sets. x 2 statistic, use two contingency tables sort mutant          smaller data set, mutant strains classified viable strain based silico vivo phenotypes calculate appropriate p-value. <br />                                                                                        inviable, insertional data set, mutants                                                                                        labeled negatively selected  population mutant <br />                                                                                        strain half wild-type population 30 <br /> Assessment synthetic accessibility robustness                                       generations competitive growth) negatively selected. test robustness approach, introduce random mistakes              synthetic accessibility approach deems mutant E. coli network randomly reassigning certain fraction enzymes           strain inviable negatively selected based path <br />  <br /> TABLE 1 Comparison accuracy statistical signi 65533;?cance FBA, MOMA, EMA, synthetic accessibility methods <br /> applied E. coli metabolic network <br />                                                                                                           Method <br />  <br />   Mutant data source               cases         Synthetic accessibility               FBA                        MOMA                         EMA <br />                                                                                                     11                       <br /> Collected literature               79                 71%, 8 3 10 *  -5 <br />                                                                                       86%, 7 3 10    (4)                                     90%, 3 3 10 14 (11) <br /> Insertional mutants                    481                 60%, 3 3 10 5              58%, 1 3 10 3 (9)          59%, 1 3 10    4 <br />                                                                                                                                     (10)              8211; <br /> Combined data sets                     560                 62%, 6 3 10 8              62%, 3 3 10 8                       8211;                            8211; <br /> Gerdes data set                        598                 74%, 1 3 10 5                       8211;                           8211;                            8211; <br /> *Accuracy, p-value x 2 statistic. <br />  <br /> Biophysical Journal 91(6) 2304 8211;2311 <br />  Topology-Based Metabolic Predictions                                                                                                                            2307 <br />  <br />  <br />  <br />  <br /> FIGURE 2 Performance synthetic accessibility compared FBA, MOMA, EMA, topology-based measures using E. coli metabolic <br /> network. graphs illustrate relative performance techniques using measures, accuracy, (TP 1 TN)/(TP 1 TN 1 <span id='am-607' about='obo:IMR_0010210' typeof='owl:Thing obo:IMR_0010047 obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0000952'><span id='am-608' property="rdfs:label" content="Fp" datatype="xsd:string"></span><span id='am-609' property="rdfs:label" content="succinate dehydrogenase subunit A" datatype="xsd:string"></span><span id='am-610' property="oboInOwl:hasExactSynonym" content="flavoprotein" datatype="xsd:string"></span><span id='am-611' property="oboInOwl:id" content="IMR:0010210" datatype="xsd:string"></span><span id='am-612' property="oboInOwl:hasExactSynonym" content="succinate dehydrogenase subunit A" datatype="xsd:string"></span><span id='am-613' property="rdfs:label" content="SDHA" datatype="xsd:string"></span><span id='am-614' property="oboInOwl:hasExactSynonym" content="Fp" datatype="xsd:string"></span><span id='am-615' property="rdfs:label" content="flavoprotein" datatype="xsd:string"></span><span id='am-616' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>FP</span> 1 FN), negative <br /> log x 2 statistic 8217;s p-value, indicates correlation silico predictions vivo observations E. coli mutant strain viability. x2 statistic calculated using contingency table like ones Fig. 3 smaller data set (79 data points, 90 data points EMA), insertional <br /> mutant data set (487 data points), combined data set (560 data points) (4,9,11). larger, representative insertional mutant data set combined data set used, synthetic accessibility accurate statistically significant FBA.  synthetic accessibility performs poorly smaller data set, probably data set data points covers central metabolism, small fraction metabolic network. topology-based measures, degree diameter, perform worse FBA, MOMA, EMA, synthetic accessibility, indicating poorly <br /> characterize functioning metabolic network. random predictions using expected values produced FBA x 2 test represent expected performance correlation silico vivo predictions. vary little expected values x 2 tests used. <br />  <br />  <br /> lengths inputs outputs accessibility outputs,                     2. approach correctly predicted strains classification scheme correspond closely                            longer rerouted pathways inviable, suggesting synthetic accessibility approach: longer path lengths                            rerouting metabolic fluxes plays small role rescuing correspond reduced growth rates invia-                               mutant strains. result consistent results bility.                                                                                 FBA analysis yeast mutants (24). <br />    number type data points included data                           3. viable mutants untouched primary <br /> sets different. insertional data set larger                        synthetic pathways short rerouting (e.g., (487 vs. 79 data points) includes fairly random col-                              isozymes). <br /> lection insertions metabolic genes, smaller <br /> data set contains data enzymes used                               used previous FBA studies, central metabolism (glycolysis, pentose phosphate pathway,                           applied synthetic accessibility approach large- <br /> citric acid cycle, respiration processes) (4).                           scale knockout study Gerdes et al. (21), identified <br /> central metabolism contains number alternate pathways,                          genes essential robust growth rich medium using require fewer steps commonly                              genetic footprinting technique based transposon-based <br /> used pathways, surprising synthetic acces-                        mutagenesis. synthetic accessibility approach performed <br /> sibility approach performs poorly FBA                                 data set (74% accuracy, p  188; 1 3 10 5). <br /> applied smaller data sets. <br />    regard combined data set, synthetic accessibility greater sensitivity, indicating better FBA                      Performance synthetic accessibility MOMA predicting strains viable,                            yeast metabolic network <br /> lower specificity, indicating good                                                                                      ensure success synthetic accessibility <br /> predicting inviable strains (Figs. 3 4). success                                                                                      method limited E. coli metabolic network, synthetic accessibility combined data set demonstrates <br />                                                                                      tested method metabolic network S. cerevisiae, important results, making transparent difference <br />                                                                                      metabolic network reconstructed viable nonviable strains. <br />                                                                                      hand (8). reconstruction extensively validated <br /> 1. nonviable mutants simply lack pathway                                   use FBA predict phenotypes large <br />    synthesize biomass components (S  188; N),                              number single-gene knockout yeast strains grown    e., essential metabolites produced                        variety conditions (13,14). conditions include glucose <br />    network inputs (Table 2).                                                     minimal medium (MMD) rich medium defined <br />  <br />                                                                                                                          Biophysical Journal 91(6) 2304 8211;2311 <br />  2308                                                                                                                                   Wunderlich Mirny <br />  <br />  <br />  <br />  <br /> FIGURE 3 Results synthetic accessibility approach, divided type prediction. contingency graphs allow exploration types errors common. Results reported   combined E. coli data set, (B) Gerdes et al. (21) E. coli data set, (C) combined yeast <br /> data set (metabolic genes . x axis represents phenotypes predicted synthetic accessibility method, y axis represents experimental <br /> phenotypes. green blocks correspond cases prediction matches experiment, red, hashed blocks correspond errors. area box proportional number cases category. diagram, common type error synthetic accessibility <br /> approach predicts mutant viable actually inviable. <br />  <br />  <br /> carbon source (YPGal, galactose; YPD, glucose; YPDGE,                              essential slow sets, synthetic accessibility performs <br /> glucose-ethanol-glycerol, YPG, glycerol; YPE, ethanol;                         comparably FBA. conditions considered <br /> YPL, lactate). Sets essential slow-growth genes                        simultaneously, synthetic accessibility predicts phenotype identified experimentally genes                           83.7% accuracy, compared FBA 82.6% <br /> mutant strains constructed genes pro-                         accuracy. <br /> duced slow-growing mutant strains rich (YPD) medium.                               believe higher overall accuracy synthetic results (Table 3) gene sets                        accessibility FBA applied yeast metabolic <br />                                                                                    network probably largely result way data sets <br />                                                                                    used. E. coli data sets, predictions                                                                                    knockout strains involved genes                                                                                    included metabolic network model. yeast data <br />                                                                                    sets, following protocol previous FBA study (8), <br />                                                                                    predictions strains, gene                                                                                    included metabolic network model                                                                                     genes nonessential, predict knockouts                                                                                    genes absent metabolic network model viable, <br />                                                                                    inflates accuracy. report accuracies                                                                                    predictions metabolic gene knockouts Table 3,                                                                                    accuracies higher cases. <br />  <br />                                                                                    Performance topology-based measures <br />                                                                                    E. coli metabolic network <br />  <br /> FIGURE 4 Plot sensitivity specificity synthetic accessibility           tested ability topology-based graph charac- <br /> (SA) prediction methods. combined E. coli data set (560          teristics, node degree, graph diameter, node <br /> data points), sensitivity, TP/(TP 1 <span id='am-663' about='obo:IMR_0002647' typeof='owl:Thing'><span id='am-664' property="oboInOwl:id" content="IMR:0002647" datatype="xsd:string"></span><span id='am-665' property="oboInOwl:hasExactSynonym" content="SDHF" datatype="xsd:string"></span><span id='am-666' property="oboInOwl:hasExactSynonym" content="Succinate dehydrogenase [ubiquinone] flavoprotein subunit, mitochondrial" datatype="xsd:string"></span><span id='am-667' property="oboInOwl:hasExactSynonym" content="succinate dehydrogenase complex, subunit A, flavoprotein (Fp)" datatype="xsd:string"></span><span id='am-668' property="rdfs:label" content="Fp" datatype="xsd:string"></span><span id='am-669' property="rdfs:label" content="Sdha" datatype="xsd:string"></span><span id='am-670' property="oboInOwl:hasDbXref" content="UniProt:Q8K2B3" datatype="xsd:string"></span><span id='am-671' property="rdfs:label" content="DHSA_MOUSE" datatype="xsd:string"></span><span id='am-672' property="oboInOwl:hasExactSynonym" content="EC 1.3.5.1" datatype="xsd:string"></span><span id='am-673' property="oboInOwl:hasExactSynonym" content="FP" datatype="xsd:string"></span><span id='am-674' property="rdfs:label" content="Flavoprotein subunit of complex II" datatype="xsd:string"></span><span id='am-675' property="rdfs:label" content="FP" datatype="xsd:string"></span><span id='am-676' property="rdfs:label" content="EC 1.3.5.1" datatype="xsd:string"></span><span id='am-677' property="oboInOwl:hasExactSynonym" content="Sdha" datatype="xsd:string"></span><span id='am-678' property="rdfs:label" content="SDHF" datatype="xsd:string"></span><span id='am-679' property="rdfs:label" content="succinate dehydrogenase complex, subunit A, flavoprotein (Fp)" datatype="xsd:string"></span><span id='am-680' property="oboInOwl:hasExactSynonym" content="Flavoprotein subunit of complex II" datatype="xsd:string"></span><span id='am-681' property="oboInOwl:hasExactSynonym" content="SDH2" datatype="xsd:string"></span><span id='am-682' property="oboInOwl:hasExactSynonym" content="Fp" datatype="xsd:string"></span><span id='am-683' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-684' property="rdfs:label" content="Succinate dehydrogenase [ubiquinone] flavoprotein subunit, mitochondrial" datatype="xsd:string"></span><span id='am-685' property="rdfs:label" content="SDH2" datatype="xsd:string"></span>FP</span>), specificity, TN/(TN 1 FN),        usage  Materials Methods), predict viability calculated predictions using synthetic accessibility, FBA, <br /> degree, diameter. cutoff values degree diameter selected <br />                                                                                    TABLE 2 E. coli mutants predicted inviable minimize x 2 test p-value. random values calculated using                                                                                    synthetic accessibility approach combined data set, <br /> expected values calculated x2 test FBA essentially                                                                                    divided reason predicting inviability values synthetic accessibility used instead. degree diameter good sensitivity, specificity quite low.    Reason predicting inviability    Correct (percent)    Incorrect (percent) <br /> synthetic accessibility FBA moderate values sensitivity <br />                                                                                     accessible outputs ,              89 (59%)              63 (41%) specificity. cases, sensitivity greater, implying                                                                                      wild-type (S  188; N) <br /> viable predictions reliable inviable predictions,                                                                                    S . wild-type                            10 (67%)               5 (33%) seen Fig. 3. <br />  <br /> Biophysical Journal 91(6) 2304 8211;2311 <br />  Topology-Based Metabolic Predictions                                                                                                           2309 <br />  <br /> TABLE 3 Accuracy synthetic accessibility FBA methods predicting viability yeast deletion strains <br />                                                                                       Data set <br />  <br />                                         Essential    Slow    MMD      YPGal      YPD        YPDGE        YPG         YPE         YPL           cases                            118         83       564      564       565         565         565         565        565         4154 <br /> FBA (8)                                  31.4%      19.3%     84.0%    85.6%     84.4%       85.3%       86.5%       85.7%      86.4%        82.6% <br /> Synthetic accessibility                  11.9%       1.20%    94.0%    97.2%     85.3%       84.4%       83.7%       84.1%      84.1%        83.7%  cases  metabolic enzymes)   100         45       459      459       462         462         462         462        462         3373 <br /> FBA                                      33.0%       4.44%    95.0%    97.6%     87.9%       87.9%       89.2%       87.4%      88.3%        87.6% <br /> Synthetic accessibility                  14.0%       2.22%    94.3%    96.9%     88.5%       88.3%       89.0%       88.7%      88.7%        87.1% <br />  <br />  <br />  <br /> E. coli mutant strains. studies suggested            topological characteristics node degree diameter <br /> nodes higher degrees important              predictive information-carrying networks (e.g., network, removal nodes biological networks              Internet, protein 8211;protein interaction networks), results likely lead lethal phenotype (15,16). test          suggest characteristics synthetic accessi- hypothesis, computed degree enzyme              bility appropriate transport directed networks, number metabolites participating reactions cata-             metabolic networks. <br /> lyzed enzyme. strain predicted inviable degree knocked enzyme certain <br />                                                                        Robustness synthetic accessibility <br /> cutoff. Fig. 2 shows optimized cutoff value, procedure predicts viability worse random prediction.           Metabolic networks incomplete    theoretical studies focused graph diameter          contain errors. study predictions using measure network performance, defining graph                  synthetic accessibility depend errors network, <br /> diameter mean shortest paths pair             performed robustness analysis using E. coli meta- <br /> nodes (15,25,26). test graph diameter predictor             bolic network. Errors modeled random reassignment <br /> viability, predicted mutant inviable increase         certain percentages enzymes different reactions. Fig. <br /> graph diameter exceeded cutoff. Fig. 2 shows  similar           5 shows accuracy prediction decreased node degree, graph diameter did perform better              increased fraction introduced mistakes. method random predictions.                                               tolerated assignment error rates 5 8211;10%, accuracy <br />    Similarly, tested topology-based measure, <br /> enzyme usage, defined number times reactions catalyzed enzyme used produce <br /> biomass wild-type strain according synthetic <br /> accessibility approach. Enzyme usage analogous node <br /> betweenness, number shortest paths pairs nodes node (22,23). Enzyme <br /> usage performed somewhat better random predictions worse synthetic accessibility, surprising basically used subset data produced synthetic accessibility approach. <br />    summary, popular topology-based measures performed poorly synthetic accessibility.  node <br /> degree diameter accurate simply pre- <br /> dicting mutants viable, gives accu- <br /> racy 53.8%, node usage performed better                FIGURE 5 Accuracy synthetic accessibility approach node degree diameter, worse predictor           percentage enzyme-reaction assignments shuffled. assess robust- <br />                                                                        ness synthetic accessibility method errors topology synthetic accessibility  Supplementary Material). <br />                                                                        E. coli metabolic network, randomly shuffle given percentage    characteristics ignore essential properties              assignments enzymes reactions calculate accuracy metabolic network, directionality branching reactions,          synthetic accessibility method 10 trials. plot average accuracy directed transport material cellular substrates            percentage assignments shuffled, error bars noting (sugars, oxygen, ) products (biomass). Synthetic                minimum maximum observed accuracy. horizontal line denotes                                                                        accuracy predicting mutants viable, best expected result accessibility, contrast, takes account properties <br />                                                                        random network. approach relatively robust random errors metabolic network.  synthetic accessibility             enzyme-reaction assignments, clear expected trend thought generalization concept graph          lower accuracy great variability accuracy number diameter directed transport networks. certain             shuffled assignments increases. <br />  <br />                                                                                                          Biophysical Journal 91(6) 2304 8211;2311 <br />  2310                                                                                                              Wunderlich Mirny <br />  <br />  <br /> dropped level random predictions ;50%               tary Material) demonstrates known complexities metab- <br /> enzyme-reaction assignments shuffled.                            olism: metabolic pathway used produce specific <br />                                                                       product shortest                                                                        completely characterized sets input output meta- <br /> DISCUSSION <br />                                                                       bolites. Similar rates failure flux balance techniques study, topology function          suggest importance regulation adaptation muta- <br /> metabolic network intimately related. introducing            tions possible role undiscovered metabolic novel topology-based measure, synthetic accessibility,             transport reactions. able correctly predict viability 443 598 mutant            explore popular network characteristics <br /> E. coli strains comprehensive, reliable data set (21)          graph diameter, node degree, betweenness (usage) 3477 4154 mutant yeast strains grown             predictors mutant viability. results demonstrate conditions (13,14). Synthetic accessibility, S, essentially      characteristics fail predict mutants 8217; viability. network diameter specifically tailored transport networks,        conclude, agreement recent similar study (28), increase S correlated inviable        node degree used predict viability metabolic <br /> phenotype. significant increase S mutation suggests           knockout strains. <br /> increased metabolic costs, leading reduction growth            lack predictive utility node degree graph <br /> rate death. apparent success synthetic accessibility        diameter metabolic networks easy understand. attributed contribution network                 concepts widely applied information exchange <br /> topology information used                networks, Internet social networks, predictions.                                                    pair nodes potentially interact. contrary,    Synthetic accessibility rapidly computed given        metabolic network transport network products network, adjustable parameters,  contrast            synthesized set initial substrates. Performance FBA, MOMA, EMA, does require knowledge                 network determined ability synthesize pro- <br /> stoichiometry maximal uptake rates metabolic               ducts,  paths inputs final products transport reactions. E. coli insertional data set,         central importance, contrast diameter, pair accuracy synthetic accessibility approach comparable        nodes considered. chemical reactions require FBA MOMA. performance synthetic                substrate yield product, linear path used accessibility compared FBA EMA smaller E.            information networks needs replaced tree coli data set worse, smaller data set data   required substrates. Considering aspects naturally leads mutants affecting central metabolism            concept synthetic accessibility study metabolic biased, large data set insertional mutants       similar transport networks, e.g., signaling networks, fairly unbiased representative. Synthetic accessibility           webs reactions, input chemical performs comparably FBA yeast data sets.               physical stimulus output group chemical  <br />    Unlike FBA, synthetic accessibility does assume           sponses stimulus. Synthetic accessibility defined way <br /> optimality regard biomass production. model           generalization graph diameter directed, branching <br /> assumes long rerouted fluxes efficient native      chemical reactions input 8211;output transport network. <br /> ones, predicting mutants longer fluxes (larger synthetic            summary, topology metabolic <br /> accessibility) inviable. assumption fails         network central determining viability mutant <br /> certain cases, similar success rates FBA ap-           strains, success widely used flux balance techniques <br /> proach suggest assumption holds true vast ma-           predicting viability primarily attributed jority mutant strains. conclude, agreement recent       topology. addition stoichiometric parameters <br /> studies (24,27), rerouting does contribute signifi-          does significantly improve accuracy predictions, <br /> cantly robustness knockout mutants.                             used FBA predict fluxes  <br />    Similar accuracy achieved techniques based flux              dividual reactions. introduce concept synthetic <br /> balance synthetic accessibility points network topol-      accessibility, allows fast, accurate, easily inter- <br /> ogy primary determinant viability predictions          pretable analysis metabolic networks. results suggest <br /> FBA MOMA. results suggest network               rerouting metabolic fluxes plays minimal role topology sufficient predict strain viability use   providing viability mutant strains. Importantly, results stoichiometric coefficients flux balances does im-         strongly support central role network topology prove prediction accuracy, detailed prediction fluxes     determining phenotypes biological systems. individual reactions FBA/MOMA does require knowl- <br /> edge stoichiometric coefficients maximal uptake rates. <br />                                                                       SUPPLEMENTARY MATERIAL <br />    Importantly, flux balance synthetic accessibility <br /> fail predict viability significant number mutants.         online supplement article visiting <br /> Analysis incorrect predictions E. coli  Supplemen-          BJ Online http://www.biophysj.org. <br />  <br /> Biophysical Journal 91(6) 2304 8211;2311 <br />  Topology-Based Metabolic Predictions                                                                                                                    2311 <br />  thank V. Spirin help processing <span id='am-956' about='obo:IMR_0701312' typeof='owl:Thing'><span id='am-957' property="oboInOwl:hasExactSynonym" content="collagen, type IV, alpha 1" datatype="xsd:string"></span><span id='am-958' property="rdfs:label" content="Collagen alpha-1(IV) chain" datatype="xsd:string"></span><span id='am-959' property="oboInOwl:hasExactSynonym" content="Collagen alpha-1(IV) chain" datatype="xsd:string"></span><span id='am-960' property="rdfs:label" content="Bru" datatype="xsd:string"></span><span id='am-961' property="rdfs:label" content="Svc" datatype="xsd:string"></span><span id='am-962' property="rdfs:label" content="Del(8)Bru44H" datatype="xsd:string"></span><span id='am-963' property="oboInOwl:hasExactSynonym" content="Raw" datatype="xsd:string"></span><span id='am-964' property="rdfs:label" content="Arresten" datatype="xsd:string"></span><span id='am-965' property="oboInOwl:hasExactSynonym" content="Bru" datatype="xsd:string"></span><span id='am-966' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-967' property="oboInOwl:hasExactSynonym" content="Del(8)Bru44H" datatype="xsd:string"></span><span id='am-968' property="oboInOwl:hasExactSynonym" content="Svc" datatype="xsd:string"></span><span id='am-969' property="rdfs:label" content="collagen, type IV, alpha 1" datatype="xsd:string"></span><span id='am-970' property="oboInOwl:hasExactSynonym" content="Col4a1" datatype="xsd:string"></span><span id='am-971' property="oboInOwl:hasExactSynonym" content="alpha1(IV) collagen" datatype="xsd:string"></span><span id='am-972' property="oboInOwl:hasExactSynonym" content="Col4a-1" datatype="xsd:string"></span><span id='am-973' property="rdfs:label" content="CO4A1_MOUSE" datatype="xsd:string"></span><span id='am-974' property="oboInOwl:id" content="IMR:0701312" datatype="xsd:string"></span><span id='am-975' property="rdfs:label" content="alpha1(IV) collagen" datatype="xsd:string"></span><span id='am-976' property="rdfs:label" content="Col4a1" datatype="xsd:string"></span><span id='am-977' property="oboInOwl:hasDbXref" content="UniProt:P02463" datatype="xsd:string"></span><span id='am-978' property="rdfs:label" content="Del(8)44H" datatype="xsd:string"></span><span id='am-979' property="rdfs:label" content="Col4a-1" datatype="xsd:string"></span><span id='am-980' property="rdfs:label" content="Raw" datatype="xsd:string"></span><span id='am-981' property="oboInOwl:hasExactSynonym" content="Arresten" datatype="xsd:string"></span><span id='am-982' property="oboInOwl:hasExactSynonym" content="Del(8)44H" datatype="xsd:string"></span>raw</span> data comments        13. Giaever, G.,  M. Chu, L. Ni, C. Connelly, L. Riles, S. Veronneau, manuscript, M. Kardar suggestions testing robustness,           S. Dow,  Lucau-Danila, K. Anderson, B. Andre,  P. Arkin, V. Berube M. Slutsky comments manuscript.                   Astromoff, et al. 2002. Functional profiling Saccharomyces <br /> thank  Trusina K. Sneppen useful discussions.                               cerevisiae genome. Nature. 418:387 8211;391. <br />                                                                                   14. Steinmetz, L. M., C. Scharfe,  M. Deutschbauer, D. Mokranjac, W. recipient Howard Hughes Medical Institute Predoctoral <br />                                                                                        S. Herman, T. Jones,  M. Chu, G. Giaever, H. Prokisch, <br /> Fellowship. L M. Alfred P. Sloan Research Fellow.                             P. J. Oefner, R. W. Davis. 2002. Systematic screen human <br />                                                                                       disease genes yeast. <span id='am-160' about='obo:IMR_0701753' typeof='owl:Thing'><span id='am-161' property="rdfs:label" content="SCNBA_MOUSE" datatype="xsd:string"></span><span id='am-162' property="rdfs:label" content="Scn11a" datatype="xsd:string"></span><span id='am-163' property="rdfs:label" content="Nan" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasExactSynonym" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-165' property="rdfs:label" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasExactSynonym" content="SNS2" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasExactSynonym" content="Nat" datatype="xsd:string"></span><span id='am-168' property="rdfs:label" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-169' property="rdfs:label" content="NSS2" datatype="xsd:string"></span><span id='am-170' property="rdfs:label" content="NaT" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-172' property="rdfs:label" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasExactSynonym" content="Scn11a" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasExactSynonym" content="NaT" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasDbXref" content="UniProt:Q9R053" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasExactSynonym" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasExactSynonym" content="Nan" datatype="xsd:string"></span><span id='am-178' property="rdfs:label" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-180' property="rdfs:label" content="Nat" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasExactSynonym" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-182' property="rdfs:label" content="NaN" datatype="xsd:string"></span><span id='am-183' property="rdfs:label" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-184' property="rdfs:label" content="SNS2" datatype="xsd:string"></span><span id='am-185' property="rdfs:label" content="Sns2" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasExactSynonym" content="NaN" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:id" content="IMR:0701753" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasExactSynonym" content="NSS2" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasExactSynonym" content="Sns2" datatype="xsd:string"></span>Nat</span>. Genet. 31:400 8211;404. <br />                                                                                   15. Albert, R., H. Jeong,  L. Barabasi. 2000. Error attack <br />                                                                                       tolerance complex networks. Nature. 406:378 8211;382. <br /> REFERENCES                                                                        16. Jeong, H., S. P. Mason,  L. Barabasi,  N. Oltvai. 2001. <br />                                                                                       Lethality centrality protein networks. Nature. 411:41 8211;42. <br />  1. Varma, , B. O. Palsson. 1994. Metabolic flux balancing 8212;basic <br />     concepts, scientific practical use. Biotechnology. 12:994 8211;998.            17. Myatt, G. J. 1994. aided estimation synthetic accessibility. <br />                                                                                       PhD Thesis. University Leeds, Leeds, <span id='am-333' about='obo:IMR_0200676' typeof='owl:Thing'><span id='am-334' property="oboInOwl:id" content="IMR:0200676" datatype="xsd:string"></span><span id='am-335' property="rdfs:label" content="DKFZp686M04245" datatype="xsd:string"></span><span id='am-336' property="rdfs:label" content="UCK2_HUMAN" datatype="xsd:string"></span><span id='am-337' property="oboInOwl:hasDbXref" content="UniProt:Q9BZX2" datatype="xsd:string"></span><span id='am-338' property="oboInOwl:hasExactSynonym" content="UCK 2" datatype="xsd:string"></span><span id='am-339' property="rdfs:label" content="Uridine-cytidine kinase 2" datatype="xsd:string"></span><span id='am-340' property="rdfs:label" content="URIDINE/CYTIDINE KINASE 2" datatype="xsd:string"></span><span id='am-341' property="oboInOwl:hasExactSynonym" content="Uridine monophosphokinase 2" datatype="xsd:string"></span><span id='am-342' property="rdfs:label" content="Testis-specific protein TSA903" datatype="xsd:string"></span><span id='am-343' property="rdfs:label" content="TSA903" datatype="xsd:string"></span><span id='am-344' property="oboInOwl:hasExactSynonym" content="Testis-specific protein TSA903" datatype="xsd:string"></span><span id='am-345' property="rdfs:label" content="UMPK" datatype="xsd:string"></span><span id='am-346' property="oboInOwl:hasExactSynonym" content="TSA903" datatype="xsd:string"></span><span id='am-347' property="oboInOwl:hasExactSynonym" content="UMPK" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasExactSynonym" content="Cytidine monophosphokinase 2" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasExactSynonym" content="Uridine-cytidine kinase 2" datatype="xsd:string"></span><span id='am-350' property="rdfs:label" content="UCK 2" datatype="xsd:string"></span><span id='am-351' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-352' property="rdfs:label" content="Cytidine monophosphokinase 2" datatype="xsd:string"></span><span id='am-353' property="rdfs:label" content="UCK2" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasExactSynonym" content="EC 2.7.1.48" datatype="xsd:string"></span><span id='am-355' property="rdfs:label" content="EC 2.7.1.48" datatype="xsd:string"></span><span id='am-356' property="oboInOwl:hasExactSynonym" content="URIDINE/CYTIDINE KINASE 2" datatype="xsd:string"></span><span id='am-357' property="oboInOwl:hasExactSynonym" content="UCK2" datatype="xsd:string"></span><span id='am-358' property="oboInOwl:hasExactSynonym" content="uridine-cytidine kinase 2" datatype="xsd:string"></span><span id='am-359' property="rdfs:label" content="uridine-cytidine kinase 2" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasExactSynonym" content="DKFZp686M04245" datatype="xsd:string"></span><span id='am-361' property="rdfs:label" content="Uridine monophosphokinase 2" datatype="xsd:string"></span><span id='am-362' property="rdfs:label" content="UK" datatype="xsd:string"></span><span id='am-363' property="oboInOwl:hasAlternativeId" content="IMR:0702901" datatype="xsd:string"></span><span id='am-364' property="oboInOwl:hasExactSynonym" content="UK" datatype="xsd:string"></span>UK</span>. <br />  2. Edwards, J. S., B. O. Palsson. 1999. Systems properties     Haemophilus influenzae <span id='am-104' about='obo:IMR_0702640' typeof='owl:Thing'><span id='am-105' property="oboInOwl:hasExactSynonym" content="RDP" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasExactSynonym" content="RNA-binding protein RD" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasExactSynonym" content="RDBP" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="RD RNA-BINDING PROTEIN" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasExactSynonym" content="Negative elongation factor E" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="NELFE" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="RDBP" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="RD" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:id" content="IMR:0702640" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="RDP" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="MAJOR HISTOCOMPATIBILITY COMPLEX GENE RD" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="NELFE_HUMAN" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasExactSynonym" content="D6S45" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="Negative elongation factor E" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="NEGATIVE ELONGATION FACTOR POLYPEPTIDE E" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="D6S45" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="NELF-E" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasExactSynonym" content="MAJOR HISTOCOMPATIBILITY COMPLEX GENE RD" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="RD RNA binding protein" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="UniProt:P18615" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasExactSynonym" content="NELF-E" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasExactSynonym" content="NEGATIVE ELONGATION FACTOR POLYPEPTIDE E" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasExactSynonym" content="RD" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasExactSynonym" content="RD RNA binding protein" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="NELFE" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasExactSynonym" content="RD RNA-BINDING PROTEIN" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="RNA-binding protein RD" datatype="xsd:string"></span>Rd</span> metabolic genotype. J. Biol. Chem.                  18. Handorf, T., O. Ebenhoh, R. Heinrich. 2005. Expanding metabolic <br />     274:17410 8211;17416.                                                                  networks: scopes compounds, robustness, evolution. J. Mol. <br />                                                                                       Evol. 61:498 8211;512. <br />  3. Edwards, J. S., B. O. Palsson. 2000. Metabolic flux balance <br />     analysis silico analysis Escherichia coli K-12 gene             19. Neidhardt, F. C., H. E. Umbarger. 1996. Chemical composition     deletions. BMC Bioinformatics. 1:1.                                               Escherichia coli. Escherichia coli Salmonella: Cellular                                                                                       Molecular Biology. F. C. Neidhardt R. Curtis, editors. ASM Press, <br />  4. Edwards, J. S., B. O. Palsson. 2000. Escherichia coli MG1655              Washington, <span id='am-501' about='obo:IMR_0702190' typeof='owl:Thing'><span id='am-502' property="rdfs:label" content="Decorin" datatype="xsd:string"></span><span id='am-503' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-504' property="rdfs:label" content="decorin" datatype="xsd:string"></span><span id='am-505' property="oboInOwl:hasDbXref" content="UniProt:P28654" datatype="xsd:string"></span><span id='am-506' property="rdfs:label" content="PG40" datatype="xsd:string"></span><span id='am-507' property="oboInOwl:hasExactSynonym" content="PG-S2" datatype="xsd:string"></span><span id='am-508' property="oboInOwl:hasExactSynonym" content="decorin" datatype="xsd:string"></span><span id='am-509' property="oboInOwl:hasExactSynonym" content="Decorin" datatype="xsd:string"></span><span id='am-510' property="rdfs:label" content="PG-S2" datatype="xsd:string"></span><span id='am-511' property="oboInOwl:hasExactSynonym" content="Dcn" datatype="xsd:string"></span><span id='am-512' property="oboInOwl:hasExactSynonym" content="Bone proteoglycan II" datatype="xsd:string"></span><span id='am-513' property="rdfs:label" content="SLRR1B" datatype="xsd:string"></span><span id='am-514' property="rdfs:label" content="Bone proteoglycan II" datatype="xsd:string"></span><span id='am-515' property="oboInOwl:hasExactSynonym" content="PG40" datatype="xsd:string"></span><span id='am-516' property="rdfs:label" content="DC" datatype="xsd:string"></span><span id='am-517' property="oboInOwl:hasExactSynonym" content="SLRR1B" datatype="xsd:string"></span><span id='am-518' property="oboInOwl:hasExactSynonym" content="DC" datatype="xsd:string"></span><span id='am-519' property="oboInOwl:id" content="IMR:0702190" datatype="xsd:string"></span><span id='am-520' property="rdfs:label" content="Dcn" datatype="xsd:string"></span><span id='am-521' property="rdfs:label" content="PGS2_MOUSE" datatype="xsd:string"></span>DC</span>. 13 8211;16. <br />     silico metabolic genotype: <span id='am-37' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, characteristics, capabil- <br />     ities. <span id='am-584' about='obo:IMR_0010425' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010045 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0000952'><span id='am-585' property="oboInOwl:id" content="IMR:0010425" datatype="xsd:string"></span><span id='am-586' property="rdfs:label" content="Pyrroline-5-carboxylate reductase" datatype="xsd:string"></span><span id='am-587' property="rdfs:label" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-588' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-589' property="oboInOwl:hasExactSynonym" content="P5C reductase" datatype="xsd:string"></span><span id='am-590' property="oboInOwl:hasExactSynonym" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-591' property="oboInOwl:hasDbXref" content="EC:1.5.1.2" datatype="xsd:string"></span><span id='am-592' property="rdfs:label" content="P5C reductase" datatype="xsd:string"></span><span id='am-593' property="rdfs:label" content="P5CR" datatype="xsd:string"></span><span id='am-594' property="rdfs:label" content="PROC" datatype="xsd:string"></span><span id='am-595' property="oboInOwl:hasExactSynonym" content="PROC" datatype="xsd:string"></span><span id='am-596' property="oboInOwl:hasExactSynonym" content="P5CR" datatype="xsd:string"></span>Proc</span>. Natl. Acad. Sci. USA. 97:5528 8211;5533.                              20. Romero, P. R., P. Karp. 2001. Nutrient-related analysis                                                                                       pathway/genome databases. Pac. Symp. Biocomput. 471 8211;482. <br />  5. Forster, J.,  Famili, P. <span id='am-388' about='obo:IMR_0702301' typeof='owl:Thing'><span id='am-389' property="rdfs:label" content="Axin1" datatype="xsd:string"></span><span id='am-390' property="oboInOwl:hasExactSynonym" content="Axin" datatype="xsd:string"></span><span id='am-391' property="rdfs:label" content="Axin" datatype="xsd:string"></span><span id='am-392' property="rdfs:label" content="axin 1" datatype="xsd:string"></span><span id='am-393' property="oboInOwl:hasDbXref" content="UniProt:O35625" datatype="xsd:string"></span><span id='am-394' property="oboInOwl:hasExactSynonym" content="Protein Fused" datatype="xsd:string"></span><span id='am-395' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-396' property="oboInOwl:hasExactSynonym" content="Fu" datatype="xsd:string"></span><span id='am-397' property="rdfs:label" content="Axin-1" datatype="xsd:string"></span><span id='am-398' property="oboInOwl:hasExactSynonym" content="Axin1" datatype="xsd:string"></span><span id='am-399' property="rdfs:label" content="Protein Fused" datatype="xsd:string"></span><span id='am-400' property="oboInOwl:hasExactSynonym" content="axin 1" datatype="xsd:string"></span><span id='am-401' property="oboInOwl:id" content="IMR:0702301" datatype="xsd:string"></span><span id='am-402' property="rdfs:label" content="AXN1_MOUSE" datatype="xsd:string"></span><span id='am-403' property="rdfs:label" content="Axis inhibition protein 1" datatype="xsd:string"></span><span id='am-404' property="oboInOwl:hasExactSynonym" content="Axin-1" datatype="xsd:string"></span><span id='am-405' property="oboInOwl:hasExactSynonym" content="Axis inhibition protein 1" datatype="xsd:string"></span><span id='am-406' property="rdfs:label" content="Fu" datatype="xsd:string"></span>Fu</span>, B. O. Palsson, J. Nielsen. 2003. <br />     Genome-scale reconstruction Saccharomyces cerevisiae meta-             21. Gerdes, S. Y., M. D. Scholle, J. W. Campbell, G. Balazsi, E. Ravasz, <br />     bolic network. Genome Res. 13:244 8211;253.                                            M. D. Daugherty,  L. Somera, N. C. Kyrpides,  Anderson, M. S. <br />                                                                                       Gelfand,  Bhattacharya, V. Kapatral, et al. 2003. Experimental deter- <br />  6. Reed, J. L., T. D. Vo, C. H. Schilling, B. O. Palsson. 2003.               mination level analysis essential genes Escherichia <br />     expanded genome-scale model Escherichia coli K-12 (iJR904 GSM/ <br />                                                                                       coli MG1655. J. Bacteriol. 185:5673 8211;5684. <br />     GPR). Genome Biol. 4:R54. <br />                                                                                   22. Newman, M. E. 2001. Scientific collaboration networks.  Network <br />  7. Burgard,  P., C. D. Maranas. 2001. Probing performance <br />                                                                                       construction fundamental results. Phys. Rev. E. 64:016131. <br />     limits Escherichia coli metabolic network subject gene addi- <br />     tions deletions. Biotechnol. Bioeng. 74:364 8211;375.                           23. Newman, M. E. 2001. Scientific collaboration networks. II. Shortest <br />                                                                                       paths, weighted networks, centrality. Phys. Rev. E. 64:016132. <br />  8. Duarte, N. C., M. J. Herrgard, B. O. Palsson. 2004. Reconstruction <br />     validation Saccharomyces cerevisiae iND750, fully compart-           24. Papp, B., C. Pal, L. D. Hurst. 2004. Metabolic network analysis     mentalized genome-scale metabolic model. Genome Res. 14:1298 8211;1309.                causes evolution enzyme dispensability yeast. Nature. <br />                                                                                       429:661 8211;664. <br />  9. Badarinarayana, V., P. W. Estep 3rd, J. Shendure, J. Edwards, S. Tavazoie, <br />     F. <span id='am-786' about='obo:IMR_0002765' typeof='owl:Thing'><span id='am-787' property="rdfs:label" content="KIAA0243" datatype="xsd:string"></span><span id='am-788' property="rdfs:label" content="TSC1 GENE" datatype="xsd:string"></span><span id='am-789' property="rdfs:label" content="Hamartin" datatype="xsd:string"></span><span id='am-790' property="oboInOwl:hasExactSynonym" content="Tuberous sclerosis 1 protein" datatype="xsd:string"></span><span id='am-791' property="oboInOwl:hasExactSynonym" content="tuberous sclerosis 1" datatype="xsd:string"></span><span id='am-792' property="rdfs:label" content="tuberous sclerosis 1" datatype="xsd:string"></span><span id='am-793' property="oboInOwl:id" content="IMR:0002765" datatype="xsd:string"></span><span id='am-794' property="rdfs:label" content="TSC1" datatype="xsd:string"></span><span id='am-795' property="oboInOwl:hasExactSynonym" content="KIAA0243" datatype="xsd:string"></span><span id='am-796' property="rdfs:label" content="hamartin" datatype="xsd:string"></span><span id='am-797' property="rdfs:label" content="Tuberous sclerosis 1 protein" datatype="xsd:string"></span><span id='am-798' property="rdfs:label" content="TSC1_HUMAN" datatype="xsd:string"></span><span id='am-799' property="oboInOwl:hasDbXref" content="UniProt:Q92574" datatype="xsd:string"></span><span id='am-800' property="oboInOwl:hasExactSynonym" content="TSC1" datatype="xsd:string"></span><span id='am-801' property="oboInOwl:hasExactSynonym" content="hamartin" datatype="xsd:string"></span><span id='am-802' property="rdfs:label" content="TSC" datatype="xsd:string"></span><span id='am-803' property="oboInOwl:hasExactSynonym" content="Hamartin" datatype="xsd:string"></span><span id='am-804' property="oboInOwl:hasExactSynonym" content="TSC" datatype="xsd:string"></span><span id='am-805' property="rdfs:label" content="LAM" datatype="xsd:string"></span><span id='am-806' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-807' property="oboInOwl:hasExactSynonym" content="TSC1 GENE" datatype="xsd:string"></span><span id='am-808' property="oboInOwl:hasExactSynonym" content="LAM" datatype="xsd:string"></span>Lam</span>, G. M. Church. 2001. Selection analyses insertional             25. Jeong, H., B. Tombor, R. Albert,  N. Oltvai,  L. Barabasi. <br />     mutants using subgenic-resolution arrays. <span id='am-191' about='obo:IMR_0701753' typeof='owl:Thing'><span id='am-192' property="rdfs:label" content="NSS2" datatype="xsd:string"></span><span id='am-193' property="rdfs:label" content="SNS2" datatype="xsd:string"></span><span id='am-194' property="rdfs:label" content="Nat" datatype="xsd:string"></span><span id='am-195' property="rdfs:label" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasExactSynonym" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-198' property="rdfs:label" content="NaT" datatype="xsd:string"></span><span id='am-199' property="rdfs:label" content="Nan" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasExactSynonym" content="Scn11a" datatype="xsd:string"></span><span id='am-201' property="rdfs:label" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-202' property="rdfs:label" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasExactSynonym" content="NaT" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasExactSynonym" content="NSS2" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasExactSynonym" content="NaN" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasExactSynonym" content="Sns2" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-208' property="rdfs:label" content="Sns2" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasDbXref" content="UniProt:Q9R053" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:id" content="IMR:0701753" datatype="xsd:string"></span><span id='am-211' property="rdfs:label" content="NaN" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasExactSynonym" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasExactSynonym" content="Nat" datatype="xsd:string"></span><span id='am-214' property="rdfs:label" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-216' property="rdfs:label" content="Scn11a" datatype="xsd:string"></span><span id='am-217' property="rdfs:label" content="SCNBA_MOUSE" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasExactSynonym" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-219' property="rdfs:label" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasExactSynonym" content="Nan" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasExactSynonym" content="SNS2" datatype="xsd:string"></span>Nat</span>. Biotechnol. 19:1060 8211;               2000. large-scale organization metabolic networks. Nature. <br />     1065.                                                                             407:651 8211;654. <br /> 10. Segre, D., D. Vitkup, G. M. Church. 2002. Analysis optimality          26. Ma, H.,  P <br /> </body></html>