<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>et al. <br /> Vitkup <br /> 2006 <br /> Volume 7, Issue 5, Article R39                                                                                               Open Access <br /> Research <br /> Influence metabolic network structure function <span id='am-67' about='obo:IMR_0000207' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001'><span id='am-68' property="rdfs:label" content="enzyme" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="GO:0003824" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-71' property="obo:IAO_0000115" content="Protein that catalyze a specific chemical reaction." datatype="xsd:string"></span><span id='am-72' property="oboInOwl:id" content="IMR:0000207" datatype="xsd:string"></span>enzyme</span> <br />  <br />  <br />  <br />  <br />                                                                                                                                                              <span id='am-528' about='rdfs:comment' typeof='owl:Thing'>comment</span> <br /> evolution <br /> Dennis Vitkup*, Peter Kharchenko 8224; Andreas Wagner 8225; <br /> Addresses: *Center Computational Biology Bioinformatics, Department Biomedical Informatics, Columbia University, Russ Berrie <br /> Pavilion, St Nicholas Avenue, New York, NY 10032, USA.  8224;Department Genetics, New Research Building, Ave Louis Pasteur, Harvard Medical <br /> School, Boston, MA 02115, USA.  8225;Department Biology, Castetter Hall, University New Mexico, Albuquerque, NM 87131, USA. <br />  <br /> Correspondence: Dennis Vitkup. Email: dv2121@columbia.edu <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reviews <br /> Published: 9 2006                                                         Received: 6 September 2005 <br />                                                                               Revised: 9 January 2006 <br /> Genome Biology 2006, 7:R39 (doi:10.1186/<span id='am-406' about='obo:IMR_0700031' typeof='owl:Thing'><span id='am-407' property="rdfs:label" content="c-Kit ligand" datatype="xsd:string"></span><span id='am-408' property="rdfs:label" content="SCF_MOUSE" datatype="xsd:string"></span><span id='am-409' property="rdfs:label" content="SLF" datatype="xsd:string"></span><span id='am-410' property="oboInOwl:hasDbXref" content="UniProt:P20826" datatype="xsd:string"></span><span id='am-411' property="rdfs:label" content="Gb" datatype="xsd:string"></span><span id='am-412' property="rdfs:label" content="kit ligand" datatype="xsd:string"></span><span id='am-413' property="oboInOwl:hasExactSynonym" content="MGF" datatype="xsd:string"></span><span id='am-414' property="rdfs:label" content="SCF" datatype="xsd:string"></span><span id='am-415' property="oboInOwl:hasExactSynonym" content="Slf" datatype="xsd:string"></span><span id='am-416' property="rdfs:label" content="Slf" datatype="xsd:string"></span><span id='am-417' property="oboInOwl:hasExactSynonym" content="SLF" datatype="xsd:string"></span><span id='am-418' property="oboInOwl:hasExactSynonym" content="SF" datatype="xsd:string"></span><span id='am-419' property="oboInOwl:hasExactSynonym" content="Stem cell factor" datatype="xsd:string"></span><span id='am-420' property="rdfs:label" content="SF" datatype="xsd:string"></span><span id='am-421' property="oboInOwl:hasExactSynonym" content="Mast cell growth factor" datatype="xsd:string"></span><span id='am-422' property="rdfs:label" content="grizzle-belly" datatype="xsd:string"></span><span id='am-423' property="rdfs:label" content="stem cell factor" datatype="xsd:string"></span><span id='am-424' property="oboInOwl:hasExactSynonym" content="grizzle-belly" datatype="xsd:string"></span><span id='am-425' property="rdfs:label" content="Mast cell growth factor" datatype="xsd:string"></span><span id='am-426' property="oboInOwl:hasExactSynonym" content="SCF" datatype="xsd:string"></span><span id='am-427' property="oboInOwl:hasExactSynonym" content="Kitlg" datatype="xsd:string"></span><span id='am-428' property="rdfs:label" content="Soluble KIT ligand" datatype="xsd:string"></span><span id='am-429' property="rdfs:label" content="sKITLG" datatype="xsd:string"></span><span id='am-430' property="oboInOwl:id" content="IMR:0700031" datatype="xsd:string"></span><span id='am-431' property="oboInOwl:hasExactSynonym" content="kit ligand" datatype="xsd:string"></span><span id='am-432' property="rdfs:label" content="Hematopoietic growth factor KL" datatype="xsd:string"></span><span id='am-433' property="oboInOwl:hasExactSynonym" content="Mgf" datatype="xsd:string"></span><span id='am-434' property="oboInOwl:hasExactSynonym" content="Kit ligand" datatype="xsd:string"></span><span id='am-435' property="oboInOwl:hasExactSynonym" content="stem cell factor" datatype="xsd:string"></span><span id='am-436' property="oboInOwl:hasExactSynonym" content="Soluble KIT ligand" datatype="xsd:string"></span><span id='am-437' property="rdfs:label" content="Kitlg" datatype="xsd:string"></span><span id='am-438' property="rdfs:label" content="Steel" datatype="xsd:string"></span><span id='am-439' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-440' property="oboInOwl:hasExactSynonym" content="Hematopoietic growth factor KL" datatype="xsd:string"></span><span id='am-441' property="oboInOwl:hasExactSynonym" content="sKITLG" datatype="xsd:string"></span><span id='am-442' property="oboInOwl:hasExactSynonym" content="Gb" datatype="xsd:string"></span><span id='am-443' property="rdfs:label" content="Kitl" datatype="xsd:string"></span><span id='am-444' property="rdfs:label" content="Kit ligand" datatype="xsd:string"></span><span id='am-445' property="oboInOwl:hasExactSynonym" content="Steel factor" datatype="xsd:string"></span><span id='am-446' property="oboInOwl:hasExactSynonym" content="Steel" datatype="xsd:string"></span><span id='am-447' property="rdfs:label" content="Steel factor" datatype="xsd:string"></span><span id='am-448' property="oboInOwl:hasExactSynonym" content="Kitl" datatype="xsd:string"></span><span id='am-449' property="rdfs:label" content="MGF" datatype="xsd:string"></span><span id='am-450' property="oboInOwl:hasExactSynonym" content="Sl" datatype="xsd:string"></span><span id='am-451' property="rdfs:label" content="Sl" datatype="xsd:string"></span><span id='am-452' property="oboInOwl:hasExactSynonym" content="c-Kit ligand" datatype="xsd:string"></span><span id='am-453' property="rdfs:label" content="Mgf" datatype="xsd:string"></span><span id='am-454' property="rdfs:label" content="Stem cell factor" datatype="xsd:string"></span>gb</span>-2006-7-5-r39) <br />                                                                               Accepted: 7 April 2006 electronic version article complete online http://genomebiology.com/2006/7/5/R39 <br />  <br />  169; 2006 Vitkup et al.; licensee BioMed Central  <br />  <br />  <br />  <br />  <br />                                                                                                                                                              reports open access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br /> ture <br /> &lt;p      function <br />              enzyme <br />       andanalysis <br /> Metabolic          ofof  metabolicconstraints, <br />                      evolutionary <br />                       evolution         network shapes <br />                                                     gene                                                          duplication <br />                                                               evolutionand <br />                                                                          essentiability <br />                                                                             enzymes.&lt;/p&gt; <br />                                                                                            yeast metabolic network demonstrates struc- <br />  <br />  <br />  <br />  <br />                  Abstract <br />  <br />  <br />  <br />  <br />                                                                                                                                                              deposited research <br />                  Background: studies molecular evolution focused individual genes proteins. <br />                   understanding design principles evolutionary properties molecular networks <br />                  requires wide perspective. present work connect molecular evolution                  gene level properties cellular metabolic network. contrast <span id='am-854' about='obo:IMR_0000949' typeof='owl:Thing obo:IMR_0000001'><span id='am-855' property="oboInOwl:id" content="IMR:0000949" datatype="xsd:string"></span><span id='am-856' property="rdfs:label" content="protein" datatype="xsd:string"></span><span id='am-857' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>protein</span> interaction <br />                  networks, previous studies investigated molecular evolution proteins, <br />                  metabolic networks relatively defined global function. ability consider fluxes                  metabolic network allows relate functional role enzyme network rate <br />  <br />  <br />  <br />  <br />                                                                                                                                                              refereed research <br />                  evolution. <br />  <br />                  Results: results, based yeast metabolic network, demonstrate important <br />                  evolutionary processes, fixation single nucleotide mutations, gene duplications,                  gene deletions, influenced structure function network. Specifically, central                  highly connected enzymes evolve slowly connected enzymes.  enzymes <br />                  carrying high metabolic fluxes natural biological conditions experience higher evolutionary <br />                  constraints. Genes encoding enzymes high connectivity high metabolic flux higher <br />                  chances retain duplicates evolution. contrast protein interaction networks, highly <br />                                                                                                                                                              interactions <br />                  connected enzymes likely essential compared connected enzymes. <br />  <br />                  Conclusion: presented analysis evolutionary constraints, gene duplication, essentiality <br />                  demonstrates structure function metabolic network shapes evolution                  enzymes. results underscore need systems-based approaches studies molecular <br />                  evolution. <br />                                                                                                                                                              information <br />  <br />  <br />  <br />  <br /> Background                                                                     duplications influence structure evolution Molecular networks genes encoding building                       networks. Conversely, network function constrain blocks represent different levels biological organiza-                  kinds mutations tolerated, genes <br /> tion interact evolution. hand, genetic                      evolve. Existing work structure evolution molec- <br /> changes <span id='am-973' about='obo:IMR_0002979' typeof='owl:Thing'><span id='am-974' property="oboInOwl:hasExactSynonym" content="0123/09" datatype="xsd:string"></span><span id='am-975' property="rdfs:label" content="0608/07" datatype="xsd:string"></span><span id='am-976' property="oboInOwl:hasExactSynonym" content="Ets2" datatype="xsd:string"></span><span id='am-977' property="rdfs:label" content="Ets at 58AB" datatype="xsd:string"></span><span id='am-978' property="oboInOwl:hasExactSynonym" content="ETS-like protein pointed, isoform P2/D" datatype="xsd:string"></span><span id='am-979' property="oboInOwl:hasExactSynonym" content="Ets at 58AB" datatype="xsd:string"></span><span id='am-980' property="rdfs:label" content="pnt" datatype="xsd:string"></span><span id='am-981' property="obo:IAO_0000115" content="pnt gene transcription generates alternatively spliced mRNA isoforms that encode two proteins, PntP1 and PntP2.
PntP2 is the homolog of the human Ets-1 transcription factor and functions as a transcriptional activator when phosphorylated by MAPK." datatype="xsd:string"></span><span id='am-982' property="rdfs:label" content="D-ets-2" datatype="xsd:string"></span><span id='am-983' property="oboInOwl:hasExactSynonym" content="D-ETS-2" datatype="xsd:string"></span><span id='am-984' property="rdfs:label" content="Ets2" datatype="xsd:string"></span><span id='am-985' property="oboInOwl:hasExactSynonym" content="EK3-2" datatype="xsd:string"></span><span id='am-986' property="rdfs:label" content="CG17077" datatype="xsd:string"></span><span id='am-987' property="oboInOwl:hasExactSynonym" content="ETS2" datatype="xsd:string"></span><span id='am-988' property="oboInOwl:hasExactSynonym" content="PNTP1" datatype="xsd:string"></span><span id='am-989' property="rdfs:label" content="PNTP2" datatype="xsd:string"></span><span id='am-990' property="rdfs:label" content="l(3)07825" datatype="xsd:string"></span><span id='am-991' property="oboInOwl:hasExactSynonym" content="pointed-P1" datatype="xsd:string"></span><span id='am-992' property="oboInOwl:hasExactSynonym" content="D-ets-2" datatype="xsd:string"></span><span id='am-993' property="oboInOwl:hasExactSynonym" content="0608/07" datatype="xsd:string"></span><span id='am-994' property="rdfs:label" content="PNTP1" datatype="xsd:string"></span><span id='am-995' property="rdfs:label" content="POINT" datatype="xsd:string"></span><span id='am-996' property="rdfs:label" content="pointed-RC" datatype="xsd:string"></span><span id='am-997' property="rdfs:label" content="PNT2_DROME" datatype="xsd:string"></span><span id='am-998' property="oboInOwl:hasExactSynonym" content="0998/12" datatype="xsd:string"></span><span id='am-999' property="oboInOwl:hasExactSynonym" content="ets94F" datatype="xsd:string"></span><span id='am-1000' property="rdfs:label" content="pointed-P1" datatype="xsd:string"></span><span id='am-1001' property="rdfs:label" content="3520" datatype="xsd:string"></span><span id='am-1002' property="rdfs:label" content="0123/09" datatype="xsd:string"></span><span id='am-1003' property="rdfs:label" content="Ets protein Pointed P2" datatype="xsd:string"></span><span id='am-1004' property="oboInOwl:hasExactSynonym" content="pointed-RC" datatype="xsd:string"></span><span id='am-1005' property="oboInOwl:hasExactSynonym" content="E(E2F)3D" datatype="xsd:string"></span><span id='am-1006' property="oboInOwl:hasExactSynonym" content="pointedP2" datatype="xsd:string"></span><span id='am-1007' property="oboInOwl:hasExactSynonym" content="POINT" datatype="xsd:string"></span><span id='am-1008' property="rdfs:label" content="pointed" datatype="xsd:string"></span><span id='am-1009' property="rdfs:label" content="D-Ets-2" datatype="xsd:string"></span><span id='am-1010' property="oboInOwl:hasExactSynonym" content="PNTP2" datatype="xsd:string"></span><span id='am-1011' property="rdfs:label" content="D-ETS-2" datatype="xsd:string"></span><span id='am-1012' property="rdfs:label" content="ETS2" datatype="xsd:string"></span><span id='am-1013' property="rdfs:label" content="E(E2F)3D" datatype="xsd:string"></span><span id='am-1014' property="oboInOwl:hasExactSynonym" content="l(3)07825" datatype="xsd:string"></span><span id='am-1015' property="oboInOwl:hasExactSynonym" content="pnt" datatype="xsd:string"></span><span id='am-1016' property="rdfs:label" content="EY3-1" datatype="xsd:string"></span><span id='am-1017' property="rdfs:label" content="l(3)j1B7" datatype="xsd:string"></span><span id='am-1018' property="rdfs:label" content="0998/12" datatype="xsd:string"></span><span id='am-1019' property="oboInOwl:hasExactSynonym" content="l(3)j1B7" datatype="xsd:string"></span><span id='am-1020' property="oboInOwl:hasExactSynonym" content="Ets protein Pointed P2" datatype="xsd:string"></span><span id='am-1021' property="rdfs:label" content="ETS-like protein pointed, isoform P2/D" datatype="xsd:string"></span><span id='am-1022' property="oboInOwl:hasDbXref" content="UniProt:P51023" datatype="xsd:string"></span><span id='am-1023' property="rdfs:label" content="Ets94F" datatype="xsd:string"></span><span id='am-1024' property="rdfs:label" content="Ets58AB" datatype="xsd:string"></span><span id='am-1025' property="rdfs:label" content="pointed-P2" datatype="xsd:string"></span><span id='am-1026' property="oboInOwl:hasExactSynonym" content="pointed-P2" datatype="xsd:string"></span><span id='am-1027' property="oboInOwl:hasExactSynonym" content="Ets94F" datatype="xsd:string"></span><span id='am-1028' property="rdfs:label" content="ets94F" datatype="xsd:string"></span><span id='am-1029' property="oboInOwl:hasExactSynonym" content="EY3-1" datatype="xsd:string"></span><span id='am-1030' property="oboInOwl:hasExactSynonym" content="DMPOINT1A" datatype="xsd:string"></span><span id='am-1031' property="oboInOwl:hasExactSynonym" content="CG17077" datatype="xsd:string"></span><span id='am-1032' property="oboInOwl:hasExactSynonym" content="D-Ets-2" datatype="xsd:string"></span><span id='am-1033' property="rdfs:label" content="pointedP2" datatype="xsd:string"></span><span id='am-1034' property="oboInOwl:hasExactSynonym" content="Ets58AB" datatype="xsd:string"></span><span id='am-1035' property="rdfs:label" content="EK3-2" datatype="xsd:string"></span><span id='am-1036' property="oboInOwl:hasExactSynonym" content="3520" datatype="xsd:string"></span><span id='am-1037' property="oboInOwl:hasExactSynonym" content="ptd" datatype="xsd:string"></span><span id='am-1038' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1039' property="oboInOwl:hasExactSynonym" content="pointed" datatype="xsd:string"></span><span id='am-1040' property="oboInOwl:id" content="IMR:0002979" datatype="xsd:string"></span><span id='am-1041' property="rdfs:label" content="DMPOINT1A" datatype="xsd:string"></span><span id='am-1042' property="rdfs:label" content="ptd" datatype="xsd:string"></span>point</span> mutations, gene deletions, gene                      ular networks mainly focused protein interaction <br />  <br />                                                               Genome Biology 2006, 7:R39 <br />  R39.2 Genome Biology 2006,   Volume 7, Issue 5, Article R39   Vitkup et al.                                          http://genomebiology.com/2006/7/5/R39 <br />  <br />  <br />  <br />  <br />        networks [1-6]. networks heterogeneous:                                6 <br />        contain large macromolecular complexes, regulatory interac- <br />        tions, signaling interactions, interactions proteins        provide structural support cell. result, difficult <br />                                                                                                  5 <br />        ascertain network structure reflects network function. <br />        large fraction false positives false negatives pro- <br />  <br />  <br />  <br />  <br />                                                                                Mean centrality <br />        tein interaction networks [7,8] complicates struc- <br />        ture function analysis. contrast, cellular metabolic                                4 <br />        networks relatively characterized model <br />        organisms Saccharomyces cerevisiae [9,10]        Escherichia coli [11]. function - biosynthesis energy                           3 <br />        production - understood, relationship        network structure network function. <br />  <br />        present study, <span id='am-162' about='obo:IMR_0703847' typeof='owl:Thing'><span id='am-163' property="oboInOwl:hasExactSynonym" content="ASK-1" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasExactSynonym" content="mitogen-activated protein kinase kinase kinase 5" datatype="xsd:string"></span><span id='am-165' property="rdfs:label" content="ASK" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasExactSynonym" content="ASK1" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasExactSynonym" content="MEK kinase 5" datatype="xsd:string"></span><span id='am-168' property="rdfs:label" content="Ask1" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasDbXref" content="UniProt:O35099" datatype="xsd:string"></span><span id='am-170' property="rdfs:label" content="mitogen-activated protein kinase kinase kinase 5" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasExactSynonym" content="Mekk5" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasExactSynonym" content="ASK" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasExactSynonym" content="Mitogen-activated protein kinase kinase kinase 5" datatype="xsd:string"></span><span id='am-174' property="rdfs:label" content="Map3k5" datatype="xsd:string"></span><span id='am-175' property="rdfs:label" content="ASK-1" datatype="xsd:string"></span><span id='am-176' property="rdfs:label" content="MEK kinase 5" datatype="xsd:string"></span><span id='am-177' property="rdfs:label" content="Apoptosis signal-regulating kinase 1" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:id" content="IMR:0703847" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasExactSynonym" content="MAPK/ERK kinase kinase 5" datatype="xsd:string"></span><span id='am-181' property="rdfs:label" content="Mitogen-activated protein kinase kinase kinase 5" datatype="xsd:string"></span><span id='am-182' property="rdfs:label" content="MEKK 5" datatype="xsd:string"></span><span id='am-183' property="rdfs:label" content="EC 2.7.11.25" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasExactSynonym" content="Map3k5" datatype="xsd:string"></span><span id='am-185' property="rdfs:label" content="ASK1" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasExactSynonym" content="MEKK 5" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasExactSynonym" content="Ask1" datatype="xsd:string"></span><span id='am-188' property="rdfs:label" content="MAPK/ERK kinase kinase 5" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasExactSynonym" content="EC 2.7.11.25" datatype="xsd:string"></span><span id='am-190' property="rdfs:label" content="M3K5_MOUSE" datatype="xsd:string"></span><span id='am-191' property="rdfs:label" content="Mekk5" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasExactSynonym" content="Apoptosis signal-regulating kinase 1" datatype="xsd:string"></span>ask</span> topology metabolic                              2 <br />                                                                                                      0   10     20           30         40        50 <br />        network metabolic fluxes  metabolic flux rate <br />                                                                                                               Enzyme connectivity <br />        chemical reaction converts reactants <span id='am-808' about='obo:IMR_0010426' typeof='owl:Thing obo:IMR_0010047 obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0000952'><span id='am-809' property="oboInOwl:hasExactSynonym" content="Proline dehydrogenase" datatype="xsd:string"></span><span id='am-810' property="rdfs:label" content="PROD" datatype="xsd:string"></span><span id='am-811' property="oboInOwl:hasExactSynonym" content="proline oxidase, mitochondrial" datatype="xsd:string"></span><span id='am-812' property="oboInOwl:hasExactSynonym" content="PROD" datatype="xsd:string"></span><span id='am-813' property="rdfs:label" content="proline oxidase, mitochondrial" datatype="xsd:string"></span><span id='am-814' property="oboInOwl:id" content="IMR:0010426" datatype="xsd:string"></span><span id='am-815' property="rdfs:label" content="Proline oxidase, mitochondrial precursor" datatype="xsd:string"></span><span id='am-816' property="oboInOwl:hasDbXref" content="EC:1.5.3.-" datatype="xsd:string"></span><span id='am-817' property="rdfs:label" content="Proline dehydrogenase" datatype="xsd:string"></span><span id='am-818' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>prod</span>- <br />        ucts) reactions network influence evolution                                                                                         Figure <br />                                                                                    correlation <br />                                                                                metabolic1 network <br />                                                                                                enzyme connectivity centrality yeast <br />        metabolic network genes point mutations                  correlation enzyme connectivity centrality yeast <br />        gene duplication. results suggest network                 metabolic network. Spearman&apos;s rank correlation r = -0.74, P &lt; 0.0001; <br />        structure function need understood fully appre-            Pearson&apos;s correlation r = -0.67, P &lt; 0.0001. centrality enzyme                                                                                equal mean length network distances enzyme        ciate metabolic networks constrain evolution           enzymes networks (pairs enzymes connected        parts. present study possible recent            path network excluded calculation). <br />        publication comprehensive compendium metabolic <br />        reactions yeast Saccharomyces cerevisiae [10].        compendium comprises 1,175 metabolic reactions 584                  connected enzymes direct access network <br />        metabolites, involves 16% yeast genes.                 nodes consequently shorter path lengths                                                                                enzymes network. evolutionary constraints        Using stoichiometric equations chemical               metabolic enzyme estimated normalized <br />        reactions, calculate connectivity enzyme            ratio non-synonymous synonymous substitutions        number metabolic enzymes produce consume               nucleotide site (Ka/<span id='am-906' about='obo:IMR_0701288' typeof='owl:Thing'><span id='am-907' property="rdfs:label" content="MGC74745" datatype="xsd:string"></span><span id='am-908' property="rdfs:label" content="collagen, type XVIII, alpha 1" datatype="xsd:string"></span><span id='am-909' property="oboInOwl:hasExactSynonym" content="KNO1" datatype="xsd:string"></span><span id='am-910' property="oboInOwl:hasExactSynonym" content="FLJ27325" datatype="xsd:string"></span><span id='am-911' property="oboInOwl:hasExactSynonym" content="KNO" datatype="xsd:string"></span><span id='am-912' property="oboInOwl:hasExactSynonym" content="Collagen alpha-1(XVIII) chain" datatype="xsd:string"></span><span id='am-913' property="oboInOwl:hasExactSynonym" content="MGC74745" datatype="xsd:string"></span><span id='am-914' property="oboInOwl:hasExactSynonym" content="KS" datatype="xsd:string"></span><span id='am-915' property="oboInOwl:id" content="IMR:0701288" datatype="xsd:string"></span><span id='am-916' property="rdfs:label" content="COLLAGEN, TYPE XVIII, ALPHA-1" datatype="xsd:string"></span><span id='am-917' property="rdfs:label" content="FLJ27325" datatype="xsd:string"></span><span id='am-918' property="rdfs:label" content="Collagen alpha-1(XVIII) chain" datatype="xsd:string"></span><span id='am-919' property="rdfs:label" content="KNO" datatype="xsd:string"></span><span id='am-920' property="rdfs:label" content="KS" datatype="xsd:string"></span><span id='am-921' property="oboInOwl:hasExactSynonym" content="COL18A1" datatype="xsd:string"></span><span id='am-922' property="oboInOwl:hasExactSynonym" content="COLLAGEN, TYPE XVIII, ALPHA-1" datatype="xsd:string"></span><span id='am-923' property="rdfs:label" content="COIA1_HUMAN" datatype="xsd:string"></span><span id='am-924' property="rdfs:label" content="Endostatin" datatype="xsd:string"></span><span id='am-925' property="rdfs:label" content="FLJ34914" datatype="xsd:string"></span><span id='am-926' property="rdfs:label" content="KNO1" datatype="xsd:string"></span><span id='am-927' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-928' property="oboInOwl:hasExactSynonym" content="Endostatin" datatype="xsd:string"></span><span id='am-929' property="oboInOwl:hasExactSynonym" content="collagen, type XVIII, alpha 1" datatype="xsd:string"></span><span id='am-930' property="oboInOwl:hasDbXref" content="UniProt:P39060" datatype="xsd:string"></span><span id='am-931' property="oboInOwl:hasExactSynonym" content="FLJ34914" datatype="xsd:string"></span><span id='am-932' property="rdfs:label" content="COL18A1" datatype="xsd:string"></span>Ks</span>) occurred gene coding        enzyme&apos;s products reactants  Materials meth-             enzyme [13]. small Ka/Ks ratio suggests higher evolu- <br />        ods Additional data file 1). words, metabolic            tionary constraints enzyme,  smaller fraction <br />        enzyme metabolic enzyme B connected                 accepted amino acid substitutions. analysis, used <br />        share metabolite product reactant.              average ratio Ka/Ks unambiguous orthologs        Highly connected enzymes representation enzymes             sequenced Saccharomyces species: S. cerevisiae, S. para- <br />        share metabolites enzymes. Including               doxus, S. bayanus, S. mikatae [14]. average Ka/Ks <br />        highly connected metabolites cofactors             values used main analysis taken study        ATP hydrogen network representation render                Kellis et al. [14]. recalculated average ratios using <br />        network structure dominated nodes,                 maximum-likelihood method Yang Nielsen [15] <br />        obscure functional relationships enzymes.              obtained qualitatively similar results. <br />        excluded 14 highly connected metabolites: <br />        ATP, H, ADP, pyrophosphate, orthophosphate, CO2, NAD,                   Figure 2 demonstrates statistically significant negative cor- <br />        glutamate, NADP, NADH, <span id='am-383' about='obo:IMR_0702411' typeof='owl:Thing'><span id='am-384' property="rdfs:label" content="SDR18C1" datatype="xsd:string"></span><span id='am-385' property="oboInOwl:hasExactSynonym" content="EC 1.3.1.34" datatype="xsd:string"></span><span id='am-386' property="oboInOwl:hasExactSynonym" content="2,4-dienoyl-CoA reductase, mitochondrial" datatype="xsd:string"></span><span id='am-387' property="rdfs:label" content="2,4-dienoyl-CoA reductase, mitochondrial" datatype="xsd:string"></span><span id='am-388' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-389' property="oboInOwl:id" content="IMR:0702411" datatype="xsd:string"></span><span id='am-390' property="oboInOwl:hasExactSynonym" content="2,4-dienoyl-CoA reductase [NADPH]" datatype="xsd:string"></span><span id='am-391' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-392' property="oboInOwl:hasExactSynonym" content="4-enoyl-CoA reductase [NADPH]" datatype="xsd:string"></span><span id='am-393' property="oboInOwl:hasExactSynonym" content="DECR1" datatype="xsd:string"></span><span id='am-394' property="rdfs:label" content="DECR1" datatype="xsd:string"></span><span id='am-395' property="rdfs:label" content="DECR_HUMAN" datatype="xsd:string"></span><span id='am-396' property="rdfs:label" content="4-enoyl-CoA reductase [NADPH]" datatype="xsd:string"></span><span id='am-397' property="rdfs:label" content="2,4-dienoyl CoA reductase 1, mitochondrial" datatype="xsd:string"></span><span id='am-398' property="rdfs:label" content="2,4-dienoyl-CoA reductase [NADPH]" datatype="xsd:string"></span><span id='am-399' property="rdfs:label" content="EC 1.3.1.34" datatype="xsd:string"></span><span id='am-400' property="oboInOwl:hasExactSynonym" content="2,4-dienoyl CoA reductase 1, mitochondrial" datatype="xsd:string"></span><span id='am-401' property="oboInOwl:hasDbXref" content="UniProt:Q16698" datatype="xsd:string"></span><span id='am-402' property="oboInOwl:hasExactSynonym" content="NADPH" datatype="xsd:string"></span><span id='am-403' property="oboInOwl:hasExactSynonym" content="SDR18C1" datatype="xsd:string"></span><span id='am-404' property="rdfs:label" content="DECR" datatype="xsd:string"></span><span id='am-405' property="oboInOwl:hasExactSynonym" content="DECR" datatype="xsd:string"></span>NADPH</span>, <span id='am-455' about='obo:IMR_0702349' typeof='owl:Thing'><span id='am-456' property="oboInOwl:hasExactSynonym" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-457' property="rdfs:label" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-458' property="oboInOwl:hasExactSynonym" content="DKFZp686D13177" datatype="xsd:string"></span><span id='am-459' property="oboInOwl:hasExactSynonym" content="AMP" datatype="xsd:string"></span><span id='am-460' property="rdfs:label" content="MGC125857" datatype="xsd:string"></span><span id='am-461' property="rdfs:label" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-462' property="rdfs:label" content="MGC125856" datatype="xsd:string"></span><span id='am-463' property="oboInOwl:hasExactSynonym" content="MGC129961" datatype="xsd:string"></span><span id='am-464' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-465' property="rdfs:label" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-466' property="rdfs:label" content="APT_HUMAN" datatype="xsd:string"></span><span id='am-467' property="rdfs:label" content="APRT" datatype="xsd:string"></span><span id='am-468' property="rdfs:label" content="DKFZp686D13177" datatype="xsd:string"></span><span id='am-469' property="oboInOwl:hasExactSynonym" content="MGC125856" datatype="xsd:string"></span><span id='am-470' property="oboInOwl:hasExactSynonym" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-471' property="oboInOwl:hasExactSynonym" content="MGC125857" datatype="xsd:string"></span><span id='am-472' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-473' property="oboInOwl:hasExactSynonym" content="APRT" datatype="xsd:string"></span><span id='am-474' property="oboInOwl:hasDbXref" content="UniProt:P07741" datatype="xsd:string"></span><span id='am-475' property="oboInOwl:hasExactSynonym" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-476' property="rdfs:label" content="MGC129961" datatype="xsd:string"></span><span id='am-477' property="oboInOwl:id" content="IMR:0702349" datatype="xsd:string"></span>AMP</span>, NH3, CoA [12].                   relation metabolic connectivity enzyme        results report qualitatively insensitive        ratio Ka/Ks (Spearman&apos;s rank correlation r = -0.20, P = 1.1 <br />        exact number removed metabolites.                                     215; 10-4; Pearson&apos;s correlation r = -0.18, P = 7  215; 10-4). inset <br />                                                                                Figure 2 shows negative association holds                                                                                broad range connectivities, caused        Results                                                                 small number highly connected proteins. Additional data <br />        Highly connected enzymes evolve slowly                                  file 2 demonstrates weaker negative correlation        discuss network structure - specifically,          non-synonymous (amino acid changing) substitutions Ka        enzyme&apos;s position network - influences enzyme evolu-             gene connectivity (Spearman&apos;s rank correlation r = -0.13, P = <br />        tion. Generally, enzymes central parts metabolism            1.6  215; 10-2). reason using Ka, instead        tricarboxylic acid cycle neighbors           preferable Ka/Ks, measure evolutionary constraints <br />        enzymes peripheral metabolic pathways (Figure 1).                does compensate gene-specific differences synony- <br />        correlation shown Figure 1 arises fact            mous substitution rates introduces additional noise <br />  <br />                                                                Genome Biology 2006, 7:R39 <br />  http://genomebiology.com/2006/7/5/R39                                             Genome Biology 2006,              Volume 7, Issue 5, Article R39   Vitkup et al. R39.3 <br />  <br />  <br />  <br />  <br />                                                                                                      0.075 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            comment <br />               0.40 <br />  <br />  <br />  <br />  <br />                                                                                     Ka/Ks (median) <br />               0.35                                                                                   0.050 <br />  <br />  <br />  <br />  <br />               0.30                                                                                   0.025 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            reviews <br />               0.25 <br />                                                                                                      0.000 <br />    Ka/Ks <br />  <br />  <br />  <br />  <br />                                                                                                              0-10        10-20      20-30        30-40 <br />  <br />               0.20                                                                                             Enzyme connectivity <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            reports <br />               0.15 <br />  <br />  <br />               0.10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            deposited research <br />               0.05 <br />  <br />  <br />               0.00 <br />                          0               10               20               30                          40           50              60               70 <br />  <br />                                                              Enzyme connectivity <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            refereed research Figure <br />     relationship <br />         2        enzyme connectivity yeast metabolic network evolutionary constraint quantified Ka/Ks ratio relationship enzyme connectivity yeast metabolic network evolutionary constraint quantified Ka/Ks ratio. Spearman&apos;s rank <br /> correlation r = -0.20, P = 1.1  215; 10-4; Pearson&apos;s correlation r = -0.18, P = 7  215; 10-4. connectivity metabolic enzyme equal total number network enzymes producing consuming enzyme&apos;s reactants products. Ka fraction amino acid replacement substitutions amino acid replacement site DNA; Ks fraction silent substitutions silent site DNA. inset shows histogram binned enzyme <br /> connectivity versus median evolutionary constraint Ka/Ks (using data main figure). standard errors bin shown. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            interactions data. Additional data file 3 shows synonymous                        ratio Ka/Ks mRNA expression levels (Spearman&apos;s <br /> (silent) substitutions Ks enzyme connectivity sig-                   rank correlation r = -0.33, P = 5.5  215; 10-10; Pearson&apos;s correla- <br /> nificantly correlated (Spearman&apos;s rank correlation r = 0.056,                    tion r = -0.30, P = 3.6  215; 10-8). Information mRNA expres- <br /> P = 0.30). expected, synonymous substitutions                   sion metabolic genes obtained study cause amino acid changes selectively neu-                    Holstege et al. [16] number mRNA molecules <br /> tral purpose analysis.                                            cell estimated based microarray data.                                                                                  relatively weak correlation connectivity                                                                                                                                                                            information <br />  <br />  <br />  <br />  highly connected enzymes greater evolutionary                        expression levels (Spearman&apos;s rank correlation r = 0.11, P = <br /> constraint (smaller Ka/Ks)? possibility corre-                  4.6  215; 10-2).  partial correlation analysis -  <br /> lation primarily mediated corresponding gene                           trolling mRNA expression levels - gene connec- <br /> expression level [3].  confirming previous observa-                       tivity evolutionary constraint Ka/Ks shows enzymes <br /> tions [3], significant negative correlation                   highly connected parts network evolve slowly <br />  <br />                                                                Genome Biology 2006, 7:R39 <br />  R39.4 Genome Biology 2006,    Volume 7, Issue 5, Article R39    Vitkup et al.                                              http://genomebiology.com/2006/7/5/R39 <br />  <br />  <br />  <br />  <br />        Table 1 <br />  <br />        Correlation enzymatic flux magnitude evolutionary constraint Ka/Ks <br />  <br />        Uptake                                 Maximum uptake rates       Spearman&apos;s rank correlation (P value) zero      Spearman&apos;s rank correlation (P value) <br />                                                  (mmol/gDW/h)                                fluxes                                  zero fluxes <br />  <br />        Glucose/oxygen                               15.3/2.4                          -0.28 (P = 3.8  215; 10-3)                          -0.25 (P = 3.6  215; 10-6) <br />  <br />        Glucose/oxygen                               15.3/0.2                          -0.31 (P = 1.7  215; 10-3)                          -0.22 (P = 5.7  215; 10-5) <br />        Glucose/oxygen                               15.3/0.01                         -0.26 (P = 9.3  215; 10-3)                          -0.21 (P = 1.2  215; 10-4) <br />        Fructose/oxygen                              15.3/6.0                          -0.27 (P = 6.4  215;   10-3)                        -0.20 (P = 2.5  215; 10-4) <br />        Fructose/oxygen                              15.3/0.2                          -0.25 (P = 1.3  215;   10-2)                        -0.20 (P = 1.8  215; 10-6) <br />        Acetate/oxygen                               10.0/2.4                            -0.08 (P = 0.45)                              -0.21 (P = 9.2  215; 10-5) <br />        Acetate/oxygen                                5.0/5.0                            -0.010 (P = 0.39)                             -0.19 (P = 3.7  215; 10-4) <br />  <br />        correlation enzymatic flux magnitude evolutionary constraint Ka/Ks calculated enzymes carrying zero fluxes. <br />        gDW, grams dry weight. <br />        independent expression levels (Spearman&apos;s partial correla-                 tive correlation flux individual enzymatic <br />        tion r = -0.18, P = 1.4  215; 10-3; P value Spearman&apos;s partial            reactions ratio Ka/Ks (Spearman&apos;s rank correlation r <br />        correlation estimated randomization).                                  = -0.31; P = 1.7  215; 10-3; Pearson&apos;s correlation r = -0.24, P = 1.7 <br />                                                                                       215; 10-2). correlation clearly non-linear, expo- <br />        Enzymes carry large metabolic fluxes evolve                              nential shape. results summarized Table 1        slowly                                                                        similar associations exist flux evolutionary  <br />        metabolic network supports cell growth                      straints Ka/Ks growth conditions glucose fruc- <br />        computationally quantified apparatus meta-                     tose - natural carbon sources yeast. Interestingly,        bolic flux analysis [17]. flux balance analysis,                   correlations evolutionary constraint Ka/Ks flux <br />        straints imposed stoichiometry reversibility                        substantially lower, statistically insignificant, <span id='am-626' about='obo:IMR_0003025' typeof='owl:Thing'><span id='am-627' property="oboInOwl:hasExactSynonym" content="AnCE" datatype="xsd:string"></span><span id='am-628' property="oboInOwl:hasExactSynonym" content="Angiotensin-converting enzyme" datatype="xsd:string"></span><span id='am-629' property="oboInOwl:hasExactSynonym" content="anon-fast-evolving-3D10" datatype="xsd:string"></span><span id='am-630' property="rdfs:label" content="br31" datatype="xsd:string"></span><span id='am-631' property="rdfs:label" content="Angiotensin-converting enzyme" datatype="xsd:string"></span><span id='am-632' property="oboInOwl:hasExactSynonym" content="CG8827" datatype="xsd:string"></span><span id='am-633' property="oboInOwl:hasExactSynonym" content="br31" datatype="xsd:string"></span><span id='am-634' property="oboInOwl:hasExactSynonym" content="Related to angiotensin converting enzyme" datatype="xsd:string"></span><span id='am-635' property="rdfs:label" content="BG:DS08220.3" datatype="xsd:string"></span><span id='am-636' property="oboInOwl:hasExactSynonym" content="l(2)br31" datatype="xsd:string"></span><span id='am-637' property="oboInOwl:hasExactSynonym" content="EC 3.4.15.1" datatype="xsd:string"></span><span id='am-638' property="rdfs:label" content="l(2)34Eb" datatype="xsd:string"></span><span id='am-639' property="oboInOwl:hasExactSynonym" content="RACE" datatype="xsd:string"></span><span id='am-640' property="oboInOwl:hasExactSynonym" content="anon-EST:fe3D10" datatype="xsd:string"></span><span id='am-641' property="oboInOwl:hasExactSynonym" content="race" datatype="xsd:string"></span><span id='am-642' property="rdfs:label" content="CG8827" datatype="xsd:string"></span><span id='am-643' property="rdfs:label" content="EC 3.4.15.1" datatype="xsd:string"></span><span id='am-644' property="rdfs:label" content="Race" datatype="xsd:string"></span><span id='am-645' property="oboInOwl:hasExactSynonym" content="BG:DS08220.3" datatype="xsd:string"></span><span id='am-646' property="rdfs:label" content="Related to angiotensin converting enzyme" datatype="xsd:string"></span><span id='am-647' property="oboInOwl:hasExactSynonym" content="l34Eb" datatype="xsd:string"></span><span id='am-648' property="rdfs:label" content="Kininase II" datatype="xsd:string"></span><span id='am-649' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-650' property="rdfs:label" content="Ance" datatype="xsd:string"></span><span id='am-651' property="oboInOwl:hasExactSynonym" content="Race" datatype="xsd:string"></span><span id='am-652' property="rdfs:label" content="AnCE" datatype="xsd:string"></span><span id='am-653' property="rdfs:label" content="ACE" datatype="xsd:string"></span><span id='am-654' property="rdfs:label" content="ACE_DROME" datatype="xsd:string"></span><span id='am-655' property="rdfs:label" content="Dipeptidyl carboxypeptidase I" datatype="xsd:string"></span><span id='am-656' property="rdfs:label" content="anon-fast-evolving-3D10" datatype="xsd:string"></span><span id='am-657' property="rdfs:label" content="l(2)br31" datatype="xsd:string"></span><span id='am-658' property="rdfs:label" content="race" datatype="xsd:string"></span><span id='am-659' property="rdfs:label" content="RACE" datatype="xsd:string"></span><span id='am-660' property="oboInOwl:id" content="IMR:0003025" datatype="xsd:string"></span><span id='am-661' property="oboInOwl:hasExactSynonym" content="l(2)34Eb" datatype="xsd:string"></span><span id='am-662' property="oboInOwl:hasExactSynonym" content="Kininase II" datatype="xsd:string"></span><span id='am-663' property="oboInOwl:hasExactSynonym" content="Dipeptidyl carboxypeptidase I" datatype="xsd:string"></span><span id='am-664' property="rdfs:label" content="l34Eb" datatype="xsd:string"></span><span id='am-665' property="rdfs:label" content="anon-EST:fe3D10" datatype="xsd:string"></span><span id='am-666' property="oboInOwl:hasExactSynonym" content="ACE" datatype="xsd:string"></span><span id='am-667' property="oboInOwl:hasDbXref" content="UniProt:Q10714" datatype="xsd:string"></span><span id='am-668' property="rdfs:label" content="Angiotensin converting enzyme" datatype="xsd:string"></span><span id='am-669' property="oboInOwl:hasExactSynonym" content="Angiotensin converting enzyme" datatype="xsd:string"></span><span id='am-670' property="oboInOwl:hasExactSynonym" content="Ance" datatype="xsd:string"></span>ace</span>- <br />        chemical reactions used restrict space feasible                 tate, carbon source dominate natural yeast <br />        metabolic fluxes. constrained subjected                  environment [21]. correlation        optimization procedure obtain flux distribution                  flux magnitude connectivity (results shown), evo- <br />        maximizes desirable metabolic property. cellu-                   lutionary constraints high fluxes complementary        lar growth-rate important component fitness                 connectivity constraints described (Figure 2). <br />        single-cell organism, biomass production used        property optimized. predictions flux balance                     Gene duplication correlation connectivity flux <br />        analysis good agreement experimental                        Gene duplications effects opposite        results E. coli [18,19] S. cerevisiae [20].                           amino acid changes: increase reduce <br />                                                                                      flux enzymatic reaction. established        relate metabolic flux rate enzyme evolution,                 highly connected enzymes enzymes high associated <br />        used flux balance analysis calculate metabolic fluxes               flux especially sensitive amino acid changes (Figures 2 <br />        yeast metabolic network [10], maximizing growth                    3). enzyme-coding genes, conversely,        different carbon sources (Table 1). Specifically, asked                    likely undergo duplication? Figure 4 shows        flux enzymatic reactions associated                   case enzyme connectivity. figure demon- <br />        evolutionary constraint Ka/Ks corresponding                        strates association enzyme-coding gene&apos;s <br />        enzyme-coding genes. analysis, enzymes catalyzing                 number duplicates enzyme connectivity  enzymes <br />        different chemical reactions, used reaction                   sequence identity higher 40% considered        largest flux; enzyme isoenzymes (enzymes                    duplicates). Mean connectivity genes duplicates        catalyzing reaction), used isoenzyme                 15.0, genes duplicates 19.2 (non-parametric <br />        smallest ratio Ka/Ks. growth conditions used vary                   Wilcoxon test, P = 1.4  215; 10-4). result suggests dupli- <br />        available carbon sources different uptake rates                cates enzymes producing consuming widely used metab- <br />        oxygen (Table 1). calculated distribution flux values               olites likely retained evolution. Figure 5        metabolic network highly non-uniform [11]                 Additional data file 5 demonstrate similar association <br />        fluxes - usually representing glycolytic enzymes -                  exists non-zero enzymatic flux reaction <br />        orders magnitude larger rest (Additional data                 number duplicates respective enzyme&apos;s <span id='am-946' about='obo:IMR_0702791' typeof='owl:Thing'><span id='am-947' property="oboInOwl:hasExactSynonym" content="SmB/SmB&apos;" datatype="xsd:string"></span><span id='am-948' property="oboInOwl:hasExactSynonym" content="Sm-B/B&apos;" datatype="xsd:string"></span><span id='am-949' property="rdfs:label" content="Sm protein B/B&apos;" datatype="xsd:string"></span><span id='am-950' property="oboInOwl:hasDbXref" content="UniProt:P14678" datatype="xsd:string"></span><span id='am-951' property="rdfs:label" content="small nuclear ribonucleoprotein polypeptides B and B1" datatype="xsd:string"></span><span id='am-952' property="oboInOwl:id" content="IMR:0702791" datatype="xsd:string"></span><span id='am-953' property="rdfs:label" content="SNRPB1" datatype="xsd:string"></span><span id='am-954' property="rdfs:label" content="SmB/B&apos;" datatype="xsd:string"></span><span id='am-955' property="rdfs:label" content="Sm-B/B&apos;" datatype="xsd:string"></span><span id='am-956' property="oboInOwl:hasExactSynonym" content="SNRPB1" datatype="xsd:string"></span><span id='am-957' property="oboInOwl:hasExactSynonym" content="SMALL NUCLEAR RIBONUCLEOPROTEIN POLYPEPTIDES B AND B-PRIME" datatype="xsd:string"></span><span id='am-958' property="rdfs:label" content="SmB/SmB&apos;" datatype="xsd:string"></span><span id='am-959' property="rdfs:label" content="COD" datatype="xsd:string"></span><span id='am-960' property="rdfs:label" content="SMALL NUCLEAR RIBONUCLEOPROTEIN POLYPEPTIDES B AND B-PRIME" datatype="xsd:string"></span><span id='am-961' property="rdfs:label" content="RSMB_HUMAN" datatype="xsd:string"></span><span id='am-962' property="rdfs:label" content="SNRPB" datatype="xsd:string"></span><span id='am-963' property="rdfs:label" content="snRNP-B" datatype="xsd:string"></span><span id='am-964' property="rdfs:label" content="Small nuclear ribonucleoprotein-associated proteins B and B&apos;" datatype="xsd:string"></span><span id='am-965' property="oboInOwl:hasExactSynonym" content="Sm protein B/B&apos;" datatype="xsd:string"></span><span id='am-966' property="oboInOwl:hasExactSynonym" content="SNRPB" datatype="xsd:string"></span><span id='am-967' property="oboInOwl:hasExactSynonym" content="snRNP-B" datatype="xsd:string"></span><span id='am-968' property="oboInOwl:hasExactSynonym" content="COD" datatype="xsd:string"></span><span id='am-969' property="oboInOwl:hasExactSynonym" content="SmB/B&apos;" datatype="xsd:string"></span><span id='am-970' property="oboInOwl:hasExactSynonym" content="small nuclear ribonucleoprotein polypeptides B and B1" datatype="xsd:string"></span><span id='am-971' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-972' property="oboInOwl:hasExactSynonym" content="Small nuclear ribonucleoprotein-associated proteins B and B&apos;" datatype="xsd:string"></span>cod</span>- <br />        file 4). eliminate disproportionate effect large              ing gene. Specifically, higher flux reaction, <br />        fluxes, removed fluxes orders magni-                   duplicates enzyme-coding gene  Qualitative <br />        tude larger median metabolic flux network                     association enzymatic flux gene duplication        (similar results obtained fluxes). Figure 2 demon-               recently shown Papp et al. [22]. <br />        strates, aerobic growth glucose, significant nega- <br />  <br />                                                                  Genome Biology 2006, 7:R39 <br />  http://genomebiology.com/2006/7/5/R39                                                                            Genome Biology 2006,                                               Volume 7, Issue 5, Article R39            Vitkup et al. R39.5 <br />  <br />  <br />  <br />  <br />                                                                                                                                                             (b) <br />                                              1.2                                                                                                                1 <br />  <br />  <br />  <br />  <br />                                                                                                                    Metabolic flux magnitude (mmol/gDW/hour) <br />   Metabolic flux magnitude (mmol/gDW/hour) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     comment <br />                                              1.0 <br />  <br />  <br />  <br />                                              0.8                                                                                                               0.1 <br />  <br />  <br />                                              0.6 <br />  <br />  <br />  <br />                                              0.4 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     reviews <br />                                                                                                                                                               0.01 <br />  <br />  <br />                                              0.2 <br />  <br />  <br />  <br />                                              0.0 <br />                                                                                                                                                                  0.00       0.05         0.10           0.15           0.20         0.25 <br />                                                0.00      0.05      0.10           0.15   0.20         0.25 <br />  <br />                                                                           Ka/Ks                                                                                                                 Ka/Ks <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     reports <br /> Figure  relationship <br />          3        metabolic flux evolutionary constraint relationship metabolic flux evolutionary constraint.  relationship metabolic flux values evolutionary constraint Ka/Ks aerobic growth glucose. (maximal uptake rate glucose 15.3 mmol/g dry weight (DW)/h; maximal oxygen uptake 0.2 mmol/gDW/h). Spearman&apos;s <br /> rank correlation r = -0.30; P = 2.7  215; 10-3; Pearson&apos;s correlation r = -0.24, P = 1.7  215; 10-2. metabolic fluxes calculated using flux balance analysis maximize cell growth rate. Fluxes orders magnitude larger median non-zero flux - representing large glycolytic fluxes - excluded analysis. (b)   using log coordinates metabolic flux magnitude. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     deposited research <br />                                                                                                                                                                      1.6 <br />                                                    1.4                                          &gt;30 <br /> Average number gene <br />  <br />  <br />  <br />  <br />                                                                                                                Average number gene <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      1.4 <br />                                                                                                                  duplications  bin) <br />  <br />  <br />  <br />  <br />                                                    1.2 <br />                                                                                                                                                                      1.2 <br />                                                    1.0 <br />       duplications <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      1.0 <br />                                                                               15-30 <br />                                                    0.8 <br />                                                            0-15                                                                                                      0.8 <br />                                                    0.6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                     refereed research <br />                                                                                                                                                                      0.6 <br />  <br />                                                    0.4 <br />                                                                                                                                                                      0.4 <br />  <br />                                                    0.2                                                                                                               0.2 <br />  <br />                                                    0.0                                                                                                               0.0 <br />                                                             1                 2                 3 <br />                                                                                                                                                                            0.001-0.01     0.01-0.1             0.1-1          &gt;1 <br />                                                                   Enzyme connectivity                                                                                       1      Metabolic <br />                                                                                                                                                                                          2   flux magnitude <br />                                                                                                                                                                                                      3                         4 <br />  Figure <br />     relationship  enzyme <br />          4 corresponding <br /> duplications                  enzyme-coding <br />                                   connectivitygenes <br />                                                average number relationship enzyme connectivity average number                                         reaction <br />                                                                                                                                                                                                                               gene <br />                                                                                                                Figure <br />                                                                                                                    relationship betweenofthe <br />                                                                                                                     and5 magnitude     thenumber <br />                                                                                                                                                 metabolic <br />                                                                                                                                                        duplicates <br />                                                                                                                                                           flux                                                                                                                                                                                                                                                                                                                                              enzyme-coding <br />                                                                                                                                                                             enzymatic <br />                                                                                                                                                                                                                                                     interactions <br /> duplications corresponding enzyme-coding genes. Enzymes                                                relationship number duplicates enzyme-coding <br /> sequence identity larger 40% 100 aligned amino acids                                         gene magnitude metabolic flux enzymatic considered duplicates.                                                                                 reaction. results shown aerobic growth glucose (maximal <br />                                                                                                                uptake rate glucose 15.3 mmol/gDW/h; oxygen 0.2 mmol/gDW/h). <br />                                                                                                                Putative duplicate pairs 40% amino acid similarity Connectivity, essentiality, metabolic robustness                                                           100 aligned amino acid residues excluded. <br /> Evolutionary constraints enzymes indirect indicators metabolic robustness amino acid changes, changes                                                    fitness studied [23,24]. studies determine essen- metabolic network tolerated millions years                                                  tial genes,  genes elimination lab- <br />                                                                                                                                                                                                                                                     information <br />  <br />  <br />  <br />  evolution. type biological robustness                                                    oratory environments effectively lethal. use available complete gene deletions. Robustness gene                                                       essentiality data motivated observation highly <br /> deletions derived laboratory studies                                                  connected proteins protein interaction networks effects gene deletions growth rate indicators                                                  likely essential cell [1]. carried analyses <br />  <br />  <br />                                                                                                 Genome Biology 2006, 7:R39 <br />  R39.6 Genome Biology 2006,                         Volume 7, Issue 5, Article R39   Vitkup et al.                                http://genomebiology.com/2006/7/5/R39 <br />  <br />  <br />  <br />  <br />                                                                                                      essential. highly connected parts metabolic network, <br />                                                                                                      metabolic fluxes rerouted alternative path- <br />        Fraction essential genes <br />  <br />  <br />  <br />                                      0.25 <br />                                                                                                      ways loss function mutation [19,28,29]. does <br />                                                                                                      hold linear metabolic pathways metabolic <span id='am-574' about='obo:IMR_0702135' typeof='owl:Thing'><span id='am-575' property="rdfs:label" content="ELK3, ETS-domain protein (SRF accessory protein 2)" datatype="xsd:string"></span><span id='am-576' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-577' property="oboInOwl:hasExactSynonym" content="ETS-RELATED PROTEIN" datatype="xsd:string"></span><span id='am-578' property="oboInOwl:hasExactSynonym" content="ELK3" datatype="xsd:string"></span><span id='am-579' property="rdfs:label" content="ERP" datatype="xsd:string"></span><span id='am-580' property="rdfs:label" content="NET" datatype="xsd:string"></span><span id='am-581' property="rdfs:label" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-582' property="oboInOwl:hasExactSynonym" content="NET" datatype="xsd:string"></span><span id='am-583' property="oboInOwl:hasExactSynonym" content="ERP" datatype="xsd:string"></span><span id='am-584' property="oboInOwl:hasExactSynonym" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-585' property="oboInOwl:hasExactSynonym" content="ELK3, ETS-domain protein (SRF accessory protein 2)" datatype="xsd:string"></span><span id='am-586' property="oboInOwl:hasExactSynonym" content="SAP2" datatype="xsd:string"></span><span id='am-587' property="rdfs:label" content="ELK3" datatype="xsd:string"></span><span id='am-588' property="oboInOwl:hasExactSynonym" content="SAP-2" datatype="xsd:string"></span><span id='am-589' property="oboInOwl:hasExactSynonym" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-590' property="rdfs:label" content="ETS-RELATED PROTEIN" datatype="xsd:string"></span><span id='am-591' property="rdfs:label" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-592' property="rdfs:label" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-593' property="oboInOwl:hasDbXref" content="UniProt:P41970" datatype="xsd:string"></span><span id='am-594' property="oboInOwl:id" content="IMR:0702135" datatype="xsd:string"></span><span id='am-595' property="oboInOwl:hasExactSynonym" content="Serum response factor accessory protein 2" datatype="xsd:string"></span><span id='am-596' property="rdfs:label" content="SAP2" datatype="xsd:string"></span><span id='am-597' property="rdfs:label" content="SAP-2" datatype="xsd:string"></span><span id='am-598' property="rdfs:label" content="ELK3_HUMAN" datatype="xsd:string"></span><span id='am-599' property="rdfs:label" content="Serum response factor accessory protein 2" datatype="xsd:string"></span><span id='am-600' property="rdfs:label" content="ELK3, ETS-DOMAIN PROTEIN" datatype="xsd:string"></span><span id='am-601' property="oboInOwl:hasExactSynonym" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-602' property="rdfs:label" content="SRF accessory protein 2" datatype="xsd:string"></span><span id='am-603' property="oboInOwl:hasExactSynonym" content="ELK3, ETS-DOMAIN PROTEIN" datatype="xsd:string"></span><span id='am-604' property="oboInOwl:hasExactSynonym" content="SRF accessory protein 2" datatype="xsd:string"></span>net</span>- <br />                                      0.20 <br />                                                                                                      work&apos;s periphery, loss function mutation                                                                                                      fatal rerouting possible. <br />                                      0.15 <br />  <br />  <br />  <br />                                      0.10 <br />                                                                                                      Discussion <br />                                                                                                      sum, demonstrate highly connected enzymes <br />                                      0.05 <br />                                                                                                      enzymes carry high metabolic fluxes yeast <br />                                                                                                      metabolic network tolerated fewer amino acid substitu- <br />                                      0.00                                                            tions evolutionary history. enzymes carrying <br />                                             0-12                  12-24              &gt;24 <br />                                                                                                      larger fluxes constrained? likely answer comes <br />                                                      Enzyme connectivity <br />                                                              2                        3              observation mutations affecting enzy- <br />                                                                                                      matic activity reduce increase flux. Enzymes <br />        Figure <br />        relationship <br />                6         enzyme connectivity gene essentiality                           carrying high fluxes tend reaction products enter <br />        relationship enzyme connectivity gene essentiality.                       large number metabolic pathways. Consequently, muta- <br />        connectivity metabolic enzyme equal total number        network enzymes producing consuming enzyme&apos;s reactants                             tional reduction activity enzymes        products. information gene essentiality obtained                          detrimental reduction activity enzymes <br />        systematic gene deletion study Giaever et al. [23] using SGD                           lower flux. <br />        database [25]. <br />                                                                                                      genes encoding enzymes high flux <br />        using data essential genes derived large scale gene                                 duplicates. Importantly, argue        deletion study Giaever et al. [23], used Saccharo-                                 duplications arise frequently genes products <br />        myces genome database (SGD) [25] collect essentiality                                  carry high flux, duplications likely        data.                                                                                         preserved evolution, advantage - higher <br />                                                                                                      flux - provide. gene&apos;s duplicates initially                                                                                                      preserved advantageous increase metabolic <br />        analyses essential enzyme-coding genes                                                                                                      flux, divergence provide functional ben- <br />        clearest deviations behavior observed protein interac- <br />                                                                                                      efits [30]. Divergence metabolic genes expression <br />        tion networks. metabolic network                                                                                                      regulation established gene intensely stud- <br />        highly connected metabolites  ATP hydrogen)                                                                                                      ied parts metabolism, tricarboxylic acid cycle <br />        excluded, mean connectivity essential enzymes                                                                                                      enzymes [31]. <br />        significantly smaller mean connectivity non- <br />        essential enzymes (Figure 6). yeast network, mean <br />                                                                                                      association predicted enzymatic <br />        connectivity essential enzymes 13.2, non-essen- <br />                                                                                                      flux evolutionary rate pronounced carbon <br />        tial enzymes 17.5 (non-parametric Wilcoxon test, P = 4.0  215; 10- <br />        4). statistically significant difference connectivity                                sources dominate natural environment yeast.                                                                                                      suggests use association flux        essential non-essential enzymes observed                                                                                                      evolutionary constraint search conditions domi- <br />        metabolites used establish network connections.  <br />                                                                                                      nated evolution metabolic networks. Similar analyses, <br />        sequently, highly connected metabolic enzymes                                                                                                      use genomic data infer environment        likely essential low connected enzymes. Similarly, <br />                                                                                                      shaped organism&apos;s evolution, used        Mahadevan et al. [26] demonstrated recently, removal                                                                                                      carbon limitation influenced evolu- <br />        highly connected metabolites essential                                                                                                      tion E. coli metabolic network strongly        removal low connected metabolites.  <br />                                                                                                      nitrogen limitation [19], yeast evolution <br />        highly connected enzymes enzymes carrying high fluxes <br />                                                                                                      favored fermentation respiration [32]. <br />        likely duplicates  sim- <br />        ilar biochemical function). suggests highly  <br />                                                                                                      previous study Hahn et al. [6] reported  based        nected enzymes likely essential                                                                                                      amino acid divergence, E. coli metabolic network        duplicates compensate loss  <br />                                                                                                      exists statistically significant association enzyme <br />        function mutations [27].  average <br />                                                                                                      connectivity evolutionary constraint. emphasize        number duplicates essential metabolic enzymes 0.19 <br />                                                                                                      contradiction earlier work results        average number duplicates non-essential <br />                                                                                                      apparent.  earlier study based        enzymes 0.8 (non-parametric Wilcoxon test, P = 8  215; 10-49). <br />                                                                                                      smaller set enzymes (n = 108 opposed n = 350 , <br />        addition gene duplications, flux rerouting provide <br />                                                                                                      statistical power.  differ- <br />        mechanism make highly connected genes  <br />                                                                                      Genome Biology 2006, 7:R39 <br />  http://genomebiology.com/2006/7/5/R39                                 Genome Biology 2006,   Volume 7, Issue 5, Article R39   Vitkup et al. R39.7 <br />  <br />  <br />  <br />  <br /> <span id='am-478' about='obo:IMR_0702309' typeof='owl:Thing'><span id='am-479' property="oboInOwl:hasDbXref" content="UniProt:P21589" datatype="xsd:string"></span><span id='am-480' property="rdfs:label" content="Ecto-5&apos;-nucleotidase" datatype="xsd:string"></span><span id='am-481' property="rdfs:label" content="NUCLEOTIDASE, ECTO-5-PRIME" datatype="xsd:string"></span><span id='am-482' property="oboInOwl:hasExactSynonym" content="NUCLEOTIDASE, ECTO-5-PRIME" datatype="xsd:string"></span><span id='am-483' property="rdfs:label" content="CD73" datatype="xsd:string"></span><span id='am-484' property="rdfs:label" content="NT5" datatype="xsd:string"></span><span id='am-485' property="oboInOwl:hasExactSynonym" content="NT5E" datatype="xsd:string"></span><span id='am-486' property="rdfs:label" content="5&apos;-NT" datatype="xsd:string"></span><span id='am-487' property="rdfs:label" content="NTE" datatype="xsd:string"></span><span id='am-488' property="rdfs:label" content="5NTD_HUMAN" datatype="xsd:string"></span><span id='am-489' property="oboInOwl:hasExactSynonym" content="Ecto-5&apos;-nucleotidase" datatype="xsd:string"></span><span id='am-490' property="oboInOwl:hasExactSynonym" content="5&apos;-nucleotidase, ecto (CD73)" datatype="xsd:string"></span><span id='am-491' property="rdfs:label" content="5&apos;-nucleotidase" datatype="xsd:string"></span><span id='am-492' property="oboInOwl:hasExactSynonym" content="NTE" datatype="xsd:string"></span><span id='am-493' property="rdfs:label" content="EC 3.1.3.5" datatype="xsd:string"></span><span id='am-494' property="oboInOwl:hasExactSynonym" content="5&apos;-NT" datatype="xsd:string"></span><span id='am-495' property="oboInOwl:id" content="IMR:0702309" datatype="xsd:string"></span><span id='am-496' property="oboInOwl:hasExactSynonym" content="NT5" datatype="xsd:string"></span><span id='am-497' property="rdfs:label" content="eNT" datatype="xsd:string"></span><span id='am-498' property="oboInOwl:hasExactSynonym" content="ECTO-5-PRIME NUCLEOTIDASE" datatype="xsd:string"></span><span id='am-499' property="rdfs:label" content="NT5E" datatype="xsd:string"></span><span id='am-500' property="rdfs:label" content="ECTO-5-PRIME NUCLEOTIDASE" datatype="xsd:string"></span><span id='am-501' property="rdfs:label" content="5&apos;-nucleotidase, ecto (CD73)" datatype="xsd:string"></span><span id='am-502' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-503' property="rdfs:label" content="E5NT" datatype="xsd:string"></span><span id='am-504' property="oboInOwl:hasExactSynonym" content="E5NT" datatype="xsd:string"></span><span id='am-505' property="oboInOwl:hasExactSynonym" content="eNT" datatype="xsd:string"></span><span id='am-506' property="oboInOwl:hasExactSynonym" content="5&apos;-nucleotidase" datatype="xsd:string"></span><span id='am-507' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.5" datatype="xsd:string"></span><span id='am-508' property="oboInOwl:hasExactSynonym" content="CD73" datatype="xsd:string"></span>ent</span> statistical measures previous study showed, like       Materials methods  negative association connectivity evo-        Metabolic network <br /> lutionary constraint, albeit P &lt; 0.05. Second,        used comprehensive collection yeast S. cerevisiae <br />  <br />  <br />  <br />  <br />                                                                                                                                                     comment lack sufficient sequence information closely         metabolic reactions Foster et al. [10] calculate metabolic <br /> related sister species E. coli, previous study used      enzyme connectivities. addition enzymatic reactions <br /> amino acid divergence Ka preferable Ka/Ks             assigned 671 open reading frames (ORFs), collection <br /> indicate evolutionary constraint. fact, correlation           contains reactions unassigned known ORFs, transport connectivity Ka similar              reactions, reactions represented large macromolecu- <br /> present study previous work (Spearman&apos;s rank corre-          lar complexes. reactions used calculate lation r = -0.13, P = 1.2  215; 10-2 versus Spearman&apos;s rank         enzyme connectivities excluded main anal- <br /> correlation r = -0.15, P = 7  215; 10-2 study Hahn et al.).    ysis. Large macromolecular complexes (containing  <br />  <br />  <br />  <br />                                                                                                                                                     reviews <br />                                                                      ORFs) represented single enzymatic nodes cal- surprising observed associations       culation connectivities metabolic enzymes. weak magnitude. reason low magnitude          order include functional relationships calcula- factors influence evolution enzyme-coding          tion enzyme connectivities, excluded 14 highly <br /> genes. factors gene expression levels (dis-         connected metabolites factors  described cussed paper) constraints stemming ter-          main text). result exclusion, small fraction (5%) <br /> tiary quaternary structure enzymes, differ          network enzymes disconnected network enzymes (little known constraints).           zero connectivity). enzymes  <br />  <br />  <br />  <br />                                                                                                                                                     reports <br /> key point factors, metabolic         included analysis. <br /> network function structure clear influence protein evolution.                                                   Flux balance analysis <br />                                                                      Flux balance analysis (FBA) used obtain metabolic flux results yeast metabolic network relate          distribution described previously [10,17,19]. network <br /> earlier work protein interaction networks?  similar       Forster et al. [10] used flux balance calculations. <br /> relationship protein connectivity evolutionary           silico network yeast metabolism includes central <br />  <br />  <br />  <br />  <br />                                                                                                                                                     deposited research <br /> constraint suggested [4,5];  association       carbon metabolism, transmembrane transport reactions, <br /> exists different reasons. Highly connected proteins pro-      pathways responsible synthesis degradation tein interaction networks evolve slowly larger         amino acids, nucleic acids, vitamins, cofactors, lipids. fraction highly connected protein&apos;s sequence involved        total, network consists 733 metabolites 1,175 <span id='am-144' about='obo:IMR_0000071' typeof='owl:Thing obo:IMR_0000210 obo:IMR_0000054 obo:IMR_0000063 obo:IMR_0000949 obo:IMR_0000512 obo:IMR_0000209 obo:IMR_0010050 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0001012'><span id='am-145' property="rdfs:label" content="hepatocyte growth factor receptor" datatype="xsd:string"></span><span id='am-146' property="rdfs:label" content="Met" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasExactSynonym" content="hepatocyte growth factor receptor" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasExactSynonym" content="HGF receptor" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasDbXref" content="GO:0005008" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:id" content="IMR:0000071" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="HGF receptor" datatype="xsd:string"></span>met</span>- protein interactions evolutionarily           abolic reactions. flux-balance analysis, constraints <br /> strained [4]. contrast, high protein connectivity <span id='am-296' about='obo:IMR_0700167' typeof='owl:Thing'><span id='am-297' property="oboInOwl:hasExactSynonym" content="HGF/SF receptor" datatype="xsd:string"></span><span id='am-298' property="oboInOwl:hasExactSynonym" content="HGF receptor" datatype="xsd:string"></span><span id='am-299' property="rdfs:label" content="HGF/SF receptor" datatype="xsd:string"></span><span id='am-300' property="rdfs:label" content="met proto-oncogene (hepatocyte growth factor receptor)" datatype="xsd:string"></span><span id='am-301' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-302' property="rdfs:label" content="MET" datatype="xsd:string"></span><span id='am-303' property="rdfs:label" content="Scatter factor receptor" datatype="xsd:string"></span><span id='am-304' property="rdfs:label" content="SF receptor" datatype="xsd:string"></span><span id='am-305' property="oboInOwl:hasDbXref" content="UniProt:P08581" datatype="xsd:string"></span><span id='am-306' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein kinase Met" datatype="xsd:string"></span><span id='am-307' property="oboInOwl:hasExactSynonym" content="ONCOGENE MET" datatype="xsd:string"></span><span id='am-308' property="oboInOwl:hasExactSynonym" content="RCCP2" datatype="xsd:string"></span><span id='am-309' property="oboInOwl:hasExactSynonym" content="Hepatocyte growth factor receptor" datatype="xsd:string"></span><span id='am-310' property="oboInOwl:hasExactSynonym" content="Proto-oncogene c-Met" datatype="xsd:string"></span><span id='am-311' property="oboInOwl:hasExactSynonym" content="Scatter factor receptor" datatype="xsd:string"></span><span id='am-312' property="rdfs:label" content="MET_HUMAN" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:id" content="IMR:0700167" datatype="xsd:string"></span><span id='am-314' property="rdfs:label" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-315' property="oboInOwl:hasExactSynonym" content="SF receptor" datatype="xsd:string"></span><span id='am-316' property="oboInOwl:hasExactSynonym" content="MET PROTOONCOGENE" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasExactSynonym" content="MET" datatype="xsd:string"></span><span id='am-318' property="rdfs:label" content="RCCP2" datatype="xsd:string"></span><span id='am-319' property="rdfs:label" content="Hepatocyte growth factor receptor" datatype="xsd:string"></span><span id='am-320' property="rdfs:label" content="Proto-oncogene c-Met" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasExactSynonym" content="HGFR" datatype="xsd:string"></span><span id='am-322' property="rdfs:label" content="ONCOGENE MET" datatype="xsd:string"></span><span id='am-323' property="rdfs:label" content="MET PROTOONCOGENE" datatype="xsd:string"></span><span id='am-324' property="rdfs:label" content="HGFR" datatype="xsd:string"></span><span id='am-325' property="oboInOwl:hasExactSynonym" content="met proto-oncogene (hepatocyte growth factor receptor)" datatype="xsd:string"></span><span id='am-326' property="rdfs:label" content="Tyrosine-protein kinase Met" datatype="xsd:string"></span><span id='am-327' property="oboInOwl:hasExactSynonym" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-328' property="rdfs:label" content="HGF receptor" datatype="xsd:string"></span>met</span>-     limiting nutrient uptake, reaction irreversibility, steady- <br /> abolic network established protein-protein            state conservation metabolite concentrations applied. <br /> interactions, consumption production               fluxes optimal growth obtained maximi- <br />  <br />  <br />  <br />  <br />                                                                                                                                                     refereed research <br /> widely used metabolites. metabolic networks, mutations         zation biomass production using linear optimization. <span id='am-671' about='obo:IMR_0001155' typeof='owl:Thing'><span id='am-672' property="rdfs:label" content="lio/drl" datatype="xsd:string"></span><span id='am-673' property="oboInOwl:hasDbXref" content="UniProt:Q27324" datatype="xsd:string"></span><span id='am-674' property="oboInOwl:hasExactSynonym" content="lin" datatype="xsd:string"></span><span id='am-675' property="oboInOwl:hasExactSynonym" content="drl" datatype="xsd:string"></span><span id='am-676' property="rdfs:label" content="lin" datatype="xsd:string"></span><span id='am-677' property="oboInOwl:hasExactSynonym" content="Protein derailed" datatype="xsd:string"></span><span id='am-678' property="rdfs:label" content="RYK1_DROME" datatype="xsd:string"></span><span id='am-679' property="rdfs:label" content="linotte" datatype="xsd:string"></span><span id='am-680' property="rdfs:label" content="CG17348" datatype="xsd:string"></span><span id='am-681' property="rdfs:label" content="DRL" datatype="xsd:string"></span><span id='am-682' property="oboInOwl:hasExactSynonym" content="derailed" datatype="xsd:string"></span><span id='am-683' property="oboInOwl:hasExactSynonym" content="linotte" datatype="xsd:string"></span><span id='am-684' property="oboInOwl:hasExactSynonym" content="DRL" datatype="xsd:string"></span><span id='am-685' property="rdfs:label" content="derailed" datatype="xsd:string"></span><span id='am-686' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-687' property="oboInOwl:hasExactSynonym" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-688' property="oboInOwl:hasExactSynonym" content="lionette" datatype="xsd:string"></span><span id='am-689' property="rdfs:label" content="lio" datatype="xsd:string"></span><span id='am-690' property="rdfs:label" content="drl" datatype="xsd:string"></span><span id='am-691' property="oboInOwl:hasExactSynonym" content="lio" datatype="xsd:string"></span><span id='am-692' property="rdfs:label" content="lionette" datatype="xsd:string"></span><span id='am-693' property="oboInOwl:hasExactSynonym" content="CG10758" datatype="xsd:string"></span><span id='am-694' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein kinase Drl" datatype="xsd:string"></span><span id='am-695' property="rdfs:label" content="CG10758" datatype="xsd:string"></span><span id='am-696' property="oboInOwl:hasExactSynonym" content="CG17348" datatype="xsd:string"></span><span id='am-697' property="rdfs:label" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-698' property="rdfs:label" content="Tyrosine-protein kinase Drl" datatype="xsd:string"></span><span id='am-699' property="oboInOwl:id" content="IMR:0001155" datatype="xsd:string"></span><span id='am-700' property="rdfs:label" content="Protein derailed" datatype="xsd:string"></span><span id='am-701' property="oboInOwl:hasExactSynonym" content="lio/drl" datatype="xsd:string"></span>Lin</span>- <br /> enzyme-coding genes - changing reaction rates concen-            <span id='am-933' about='obo:IMR_0701164' typeof='owl:Thing'><span id='am-934' property="oboInOwl:hasExactSynonym" content="Arrestin-E" datatype="xsd:string"></span><span id='am-935' property="rdfs:label" content="Arrestin-E" datatype="xsd:string"></span><span id='am-936' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-937' property="rdfs:label" content="arrestin-E" datatype="xsd:string"></span><span id='am-938' property="oboInOwl:hasExactSynonym" content="LOC246218" datatype="xsd:string"></span><span id='am-939' property="rdfs:label" content="Ear" datatype="xsd:string"></span><span id='am-940' property="rdfs:label" content="ARRE_RAT" datatype="xsd:string"></span><span id='am-941' property="oboInOwl:id" content="IMR:0701164" datatype="xsd:string"></span><span id='am-942' property="rdfs:label" content="LOC246218" datatype="xsd:string"></span><span id='am-943' property="oboInOwl:hasDbXref" content="UniProt:P37200" datatype="xsd:string"></span><span id='am-944' property="oboInOwl:hasExactSynonym" content="Ear" datatype="xsd:string"></span><span id='am-945' property="oboInOwl:hasExactSynonym" content="arrestin-E" datatype="xsd:string"></span>ear</span> optimization performed using GNU Linear <br /> trations - especially deleterious consequences          Programming <span id='am-153' about='obo:IMR_0000074' typeof='owl:Thing obo:IMR_0000210 obo:IMR_0000054 obo:IMR_0000063 obo:IMR_0000949 obo:IMR_0000512 obo:IMR_0000209 obo:IMR_0010050 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0001012'><span id='am-154' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasExactSynonym" content="c-Kit" datatype="xsd:string"></span><span id='am-156' property="rdfs:label" content="mast/stem cell growth factor receptor" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasDbXref" content="GO:0005020" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:id" content="IMR:0000074" datatype="xsd:string"></span><span id='am-159' property="rdfs:label" content="c-Kit" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasExactSynonym" content="mast/stem cell growth factor receptor" datatype="xsd:string"></span><span id='am-161' property="rdfs:label" content="Kit" datatype="xsd:string"></span>Kit</span> [33]. <br /> widely used metabolites. Consequently, highly connected <br /> metabolic enzymes evolve slowly functional             Molecular evolution <br /> opposed structural constraints. ability consider           identified duplicates S. cerecisae genome using fluxes enzymes metabolic network allows           previously described genome analysis tool [34]. <br /> relate functional role enzyme network        Briefly, tool locates gene duplicates genome using <br /> rate evolution. functional analysis genome-scale      BLASTP [35] aligns globally Needleman <br /> network counterpart genome-scale <span id='am-605' about='obo:IMR_0702136' typeof='owl:Thing'><span id='am-606' property="oboInOwl:hasExactSynonym" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-607' property="rdfs:label" content="Elk3" datatype="xsd:string"></span><span id='am-608' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-609' property="oboInOwl:hasExactSynonym" content="Sap-2" datatype="xsd:string"></span><span id='am-610' property="rdfs:label" content="Net" datatype="xsd:string"></span><span id='am-611' property="rdfs:label" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-612' property="rdfs:label" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-613' property="rdfs:label" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-614' property="oboInOwl:hasExactSynonym" content="Elk3" datatype="xsd:string"></span><span id='am-615' property="rdfs:label" content="Erp" datatype="xsd:string"></span><span id='am-616' property="oboInOwl:hasExactSynonym" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-617' property="oboInOwl:hasDbXref" content="UniProt:P41971" datatype="xsd:string"></span><span id='am-618' property="rdfs:label" content="Sap-2" datatype="xsd:string"></span><span id='am-619' property="oboInOwl:hasExactSynonym" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-620' property="oboInOwl:hasExactSynonym" content="Erp" datatype="xsd:string"></span><span id='am-621' property="oboInOwl:id" content="IMR:0702136" datatype="xsd:string"></span><span id='am-622' property="oboInOwl:hasExactSynonym" content="Net" datatype="xsd:string"></span><span id='am-623' property="rdfs:label" content="ELK3_MOUSE" datatype="xsd:string"></span><span id='am-624' property="rdfs:label" content="ELK3, member of ETS oncogene family" datatype="xsd:string"></span><span id='am-625' property="oboInOwl:hasExactSynonym" content="ELK3, member of ETS oncogene family" datatype="xsd:string"></span>net</span>-            Wunsch dynamic programming alignment algorithm                             interactions <br /> work studied far.                                               [36]. Putative duplicate pairs 40% amino acid <br />                                                                      similarity 100 aligned amino acid residues conclusion, analysis evolutionary constraints, gene        excluded; remaining pairs calculated number duplication, essentiality demonstrates structure        substitutions synonymous site (Ks) number function metabolic network shapes evolution          substitutions non-synonymous site (Ka) using maxi- enzymes. long run, analyses biological          mum likelyhood models Muse Gaut [37] Goldman <br /> networks allow increasingly place evolution        Yang [38]. <br />                                                                                                                                                     information <br />  <br />  <br />  <br />  <br /> genes larger context operate, building <br /> blocks cellular networks.                                         average Ka/Ks, Ka, Ks values used analysis                                                                      obtained study Kellis et al. [14]. complemen- <br />                                                                      tary approach, recalculated average ratios using <br />  <br />  <br />  <br />                                                       Genome Biology 2006, 7:R39 <br />  R39.8 Genome Biology 2006,             Volume 7, Issue 5, Article R39               Vitkup et al.                                          http://genomebiology.com/2006/7/5/R39 <br />  <br />  <br />  <br />  <br />        maximum-likelihood method Yang Nielsen [15]                                                   Comparative assessment large-scale data sets protein- <br />        obtained qualitatively similar results.                                                              protein interactions. Nature 2002, 417:399-403. <br />                                                                                                           8.    Spinzak E, Sattah S, Margalit H: reliable experimental <br />                                                                                                                 protein-protein interaction data ? J Mol Biol 2003, 327:919-923. <br />                                                                                                           9.    Karp PD, Paley S, Romero P: Pathway Tools software. Bioin- <br />                                                                                                                 formatics 2002, 18:S225-S232. <br />        Additional data files                                                                              10.   Forster J, Famili  <span id='am-509' about='obo:IMR_0702301' typeof='owl:Thing'><span id='am-510' property="rdfs:label" content="Axis inhibition protein 1" datatype="xsd:string"></span><span id='am-511' property="rdfs:label" content="Protein Fused" datatype="xsd:string"></span><span id='am-512' property="oboInOwl:hasExactSynonym" content="Axin-1" datatype="xsd:string"></span><span id='am-513' property="oboInOwl:hasExactSynonym" content="axin 1" datatype="xsd:string"></span><span id='am-514' property="rdfs:label" content="AXN1_MOUSE" datatype="xsd:string"></span><span id='am-515' property="rdfs:label" content="Axin-1" datatype="xsd:string"></span><span id='am-516' property="oboInOwl:id" content="IMR:0702301" datatype="xsd:string"></span><span id='am-517' property="rdfs:label" content="Fu" datatype="xsd:string"></span><span id='am-518' property="oboInOwl:hasExactSynonym" content="Axis inhibition protein 1" datatype="xsd:string"></span><span id='am-519' property="rdfs:label" content="axin 1" datatype="xsd:string"></span><span id='am-520' property="oboInOwl:hasExactSynonym" content="Protein Fused" datatype="xsd:string"></span><span id='am-521' property="oboInOwl:hasExactSynonym" content="Fu" datatype="xsd:string"></span><span id='am-522' property="oboInOwl:hasExactSynonym" content="Axin" datatype="xsd:string"></span><span id='am-523' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-524' property="oboInOwl:hasExactSynonym" content="Axin1" datatype="xsd:string"></span><span id='am-525' property="oboInOwl:hasDbXref" content="UniProt:O35625" datatype="xsd:string"></span><span id='am-526' property="rdfs:label" content="Axin1" datatype="xsd:string"></span><span id='am-527' property="rdfs:label" content="Axin" datatype="xsd:string"></span>Fu</span> P, Palsson BO, Nielsen J: Genome-scale <br />        following additional data available online                                              reconstruction Saccharomyces cerevisiae metabolic <br />        version paper. Additional data file 1 figure showing                                        network. Genome Res 2003, 13:244-253. <br />                                                                                                           11.   Edwards JS, Palsson BO: Escherichia coli MG1655 silico <br />        examples metabolic connectivity.   example                                                metabolic genotype: <span id='am-66' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, characteristics,        metabolic reaction network sphingoglycolipid metabo-                                                capabilities. <span id='am-795' about='obo:IMR_0010425' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010045 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0000952'><span id='am-796' property="rdfs:label" content="P5CR" datatype="xsd:string"></span><span id='am-797' property="oboInOwl:id" content="IMR:0010425" datatype="xsd:string"></span><span id='am-798' property="oboInOwl:hasExactSynonym" content="PROC" datatype="xsd:string"></span><span id='am-799' property="rdfs:label" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-800' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-801' property="rdfs:label" content="PROC" datatype="xsd:string"></span><span id='am-802' property="rdfs:label" content="Pyrroline-5-carboxylate reductase" datatype="xsd:string"></span><span id='am-803' property="oboInOwl:hasExactSynonym" content="P5C reductase" datatype="xsd:string"></span><span id='am-804' property="oboInOwl:hasExactSynonym" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-805' property="oboInOwl:hasDbXref" content="EC:1.5.1.2" datatype="xsd:string"></span><span id='am-806' property="oboInOwl:hasExactSynonym" content="P5CR" datatype="xsd:string"></span><span id='am-807' property="rdfs:label" content="P5C reductase" datatype="xsd:string"></span>Proc</span> Natl Acad Sci USA 2000, 97:5528-5533. <br />                                                                                                           12.   Kharchenko P, Vitkup D, Church GM: Filling <span id='am-529' about='obo:IMR_0703252' typeof='owl:Thing'><span id='am-530' property="rdfs:label" content="Phosphoribosylglycinamide formyltransferase" datatype="xsd:string"></span><span id='am-531' property="oboInOwl:hasExactSynonym" content="GART" datatype="xsd:string"></span><span id='am-532' property="oboInOwl:hasExactSynonym" content="EC 2.1.2.2" datatype="xsd:string"></span><span id='am-533' property="rdfs:label" content="5&apos;-phosphoribosylglycinamide transformylase" datatype="xsd:string"></span><span id='am-534' property="rdfs:label" content="PUR2_MOUSE" datatype="xsd:string"></span><span id='am-535' property="oboInOwl:hasExactSynonym" content="phosphoribosylglycinamide formyltransferase" datatype="xsd:string"></span><span id='am-536' property="oboInOwl:hasExactSynonym" content="Glycinamide ribonucleotide synthetase" datatype="xsd:string"></span><span id='am-537' property="oboInOwl:hasExactSynonym" content="EC 6.3.3.1" datatype="xsd:string"></span><span id='am-538' property="oboInOwl:id" content="IMR:0703252" datatype="xsd:string"></span><span id='am-539' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-540' property="rdfs:label" content="GART" datatype="xsd:string"></span><span id='am-541' property="rdfs:label" content="Trifunctional purine biosynthetic protein adenosine-3" datatype="xsd:string"></span><span id='am-542' property="rdfs:label" content="GAR transformylase" datatype="xsd:string"></span><span id='am-543' property="oboInOwl:hasExactSynonym" content="Phosphoribosyl-aminoimidazole synthetase" datatype="xsd:string"></span><span id='am-544' property="oboInOwl:hasExactSynonym" content="GARS" datatype="xsd:string"></span><span id='am-545' property="rdfs:label" content="GARS" datatype="xsd:string"></span><span id='am-546' property="oboInOwl:hasExactSynonym" content="Phosphoribosylformylglycinamidine cyclo-ligase" datatype="xsd:string"></span><span id='am-547' property="oboInOwl:hasExactSynonym" content="EC 6.3.4.13" datatype="xsd:string"></span><span id='am-548' property="rdfs:label" content="EC 2.1.2.2" datatype="xsd:string"></span><span id='am-549' property="oboInOwl:hasExactSynonym" content="Gaps" datatype="xsd:string"></span><span id='am-550' property="rdfs:label" content="Phosphoribosylglycinamide synthetase" datatype="xsd:string"></span><span id='am-551' property="rdfs:label" content="Phosphoribosylformylglycinamidine cyclo-ligase" datatype="xsd:string"></span><span id='am-552' property="rdfs:label" content="Phosphoribosyl-aminoimidazole synthetase" datatype="xsd:string"></span><span id='am-553' property="oboInOwl:hasExactSynonym" content="AIR synthase" datatype="xsd:string"></span><span id='am-554' property="rdfs:label" content="Gaps" datatype="xsd:string"></span><span id='am-555' property="oboInOwl:hasExactSynonym" content="Gart" datatype="xsd:string"></span><span id='am-556' property="rdfs:label" content="Phosphoribosylamine--glycine ligase" datatype="xsd:string"></span><span id='am-557' property="oboInOwl:hasExactSynonym" content="Trifunctional purine biosynthetic protein adenosine-3" datatype="xsd:string"></span><span id='am-558' property="oboInOwl:hasExactSynonym" content="Phosphoribosylglycinamide synthetase" datatype="xsd:string"></span><span id='am-559' property="rdfs:label" content="phosphoribosylglycinamide formyltransferase" datatype="xsd:string"></span><span id='am-560' property="oboInOwl:hasExactSynonym" content="GAR transformylase" datatype="xsd:string"></span><span id='am-561' property="rdfs:label" content="AIRS" datatype="xsd:string"></span><span id='am-562' property="oboInOwl:hasDbXref" content="UniProt:Q64737" datatype="xsd:string"></span><span id='am-563' property="rdfs:label" content="Glycinamide ribonucleotide synthetase" datatype="xsd:string"></span><span id='am-564' property="oboInOwl:hasExactSynonym" content="Prgs" datatype="xsd:string"></span><span id='am-565' property="rdfs:label" content="Prgs" datatype="xsd:string"></span><span id='am-566' property="rdfs:label" content="EC 6.3.4.13" datatype="xsd:string"></span><span id='am-567' property="oboInOwl:hasExactSynonym" content="Phosphoribosylglycinamide formyltransferase" datatype="xsd:string"></span><span id='am-568' property="oboInOwl:hasExactSynonym" content="5&apos;-phosphoribosylglycinamide transformylase" datatype="xsd:string"></span><span id='am-569' property="oboInOwl:hasExactSynonym" content="Phosphoribosylamine--glycine ligase" datatype="xsd:string"></span><span id='am-570' property="rdfs:label" content="AIR synthase" datatype="xsd:string"></span><span id='am-571' property="rdfs:label" content="Gart" datatype="xsd:string"></span><span id='am-572' property="rdfs:label" content="EC 6.3.3.1" datatype="xsd:string"></span><span id='am-573' property="oboInOwl:hasExactSynonym" content="AIRS" datatype="xsd:string"></span>gaps</span> metabolic <br />        lism; metabolites drawn small circles (DHSP, sphinga-                                             network using expression information. Bioinformatics 2004, <br />        1-phosphate; PETHM, ethanolamine phosphate; SPH,                                                    20:I178-I185. <br />        sphinganine; CDPETN, CDPethanolamine; ETHM, eth-                                                   13.   Li W-H: Molecular Evolution Sunderland: Sinauer Associates; 1997. <br />                                                                                                           14.   Kellis M, Patterson N, Endrizzi M, Birren B, Lander ES: Sequencing <br />        anolamine) enzyme-encoding genes shown rectan-                                                comparison yeast species identify genes regu- <br />        gles. (b) Metabolic connectivity dpl1 gene (solid edges),                                         latory elements. Nature 2003, 423:241-254. <br />        defined reactions shown  . dpl1 gene                                      15.   Yang  Nielsen R: Estimating synonymous nonsynony- <br />                                                                                                                 mous substitution rates realistic evolutionary models. <br />        total metabolic connections: established                                              Mol Biol Evol 2000, 17:32-43. <br />        ethanolamine phosphate (red edges); sph-                                          16.   Holstege FC, Jennings  Wyrick JJ, Lee TI, Hengartner CJ, Green <br />                                                                                                                 MR, Golub <span id='am-135' about='obo:IMR_0000086' typeof='owl:Thing obo:IMR_0000054 obo:IMR_0000949 obo:IMR_0000369 obo:IMR_0000368 obo:IMR_0000001 obo:IMR_0001012 obo:IMR_0000085'><span id='am-136' property="rdfs:label" content="c-erbA" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasExactSynonym" content="thyroid hormone receptor" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasDbXref" content="GO:0004887" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="thyroid hormone receptor" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:id" content="IMR:0000086" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="TR" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasExactSynonym" content="c-erbA" datatype="xsd:string"></span>TR</span>, Lander ES, Young RA: Dissecting regulatory <br />        inganine 1-phosphate (blue edges). Metabolic connections                                                 circuitry eukaryotic genome. Cell 1998, 95:717-728. <br />        enzymes dashed edges. Additional                                         17.   Varma  Boesch BW, Palsson BO: Biochemical production capa- <br />        data file 2 demonstrates relationship enzyme                                                 bilites Escherichia coli. Biotech Bioeng 1993, 42:59-73. <br />                                                                                                           18.   Edwards JS, Ibarra RU, Palsson BO: silico predictions        connectivity average amino acid divergence Ka.                                                   Escherichia coli metabolic capabilites consistent        Spearman&apos;s rank correlation r = -0.13, P = 1.6  215; 10-2. Addi-                                             experimental data. <span id='am-73' about='obo:IMR_0701753' typeof='owl:Thing'><span id='am-74' property="rdfs:label" content="Scn11a" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="NaN" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="NSS2" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="SCNBA_MOUSE" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Nan" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasExactSynonym" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="SNS2" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="Nan" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Nat" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="NSS2" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="NaN" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="UniProt:Q9R053" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="NaT" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Sns2" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasExactSynonym" content="Sns2" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasExactSynonym" content="Nat" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="NaT" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasExactSynonym" content="SNS2" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="Scn11a" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:id" content="IMR:0701753" datatype="xsd:string"></span>Nat</span> Biotechnol 2001, 19:125-130. <br />                                                                                                           19.   Segre D, Vitkup D, Church GM: Analysis optimality natural <br />        tional data file 3 shows relationship enzyme                                             perturbed metabolic networks. Proc Natl Acad Sci USA 2002, <br />        nectivity average silent divergence Ks. Spearman&apos;s                                               99:15112-12117. <br />        rank correlation r = -0.056, P = 0.30. Additional data file 4                                   20.   Foster J, Famili  Palsson BO, Nielsen J: Large-scale evaluation                                                                                                                 silico gene deletions Saccharomyces cerevisiae. OMICS <br />        histogram calculated metabolic fluxes yeast                                              2003, 7:193-202. <br />        network aerobic growth glucose (maximal uptake rate                                         21.   Strathern JN, Jones EW, Broach JR: Molecular Biology        glucose 15.3 mmol/g dry weight/h; oxygen 0.2 mmol/g                                                  Yeast Saccharomyces. Metabolism Gene Expression Cold <br />                                                                                                                 Spring Harbor Press, NY; 1982. <br />        dry weight/h). Note small number fluxes - representing                                      22.   Papp B, Pal C, Hurst LD: Metabolic network analysis        glycolysis - disproportionately large magnitudes. Similar                                           causes evolution enzyme dispensability yeast. <br />                                                                                                                 Nature 2004, 429:661-664. <br />        flux distributions obtained growth condi-                                      23.   Giaever G, Chu  Ni L, Connelly C, Riles L, Veronneau S, Dow S, <br />        tions. Additional data file 5 shows correlation                                              Lucau-Danila  Anderson K, Andre B, et al.: Functional profiling <br />        non-zero enzymatic flux reaction number                                                Saccharomyces cerevisiae genome.                 Nature 2002, <br />                                                                                                                 418:387-391. <br />        duplicates respective enzyme&apos;s coding gene. Addi-                                        24.   Steinmetz LM, Scharfe C, Deutschbauer  Mokranjac D, Herman <br />        tional data file 6 provides connectivity evolutionary                                                ZS, Jones T, Chu  Giaever G, Prokisch H, Oefner PJ, Davis RW: <br />        parameters (Ka/Ks, Ka, Ks) yeast metabolic enzymes.                                                  Systematic screen human disease genes yeast. <span id='am-104' about='obo:IMR_0701753' typeof='owl:Thing'><span id='am-105' property="oboInOwl:hasExactSynonym" content="Nat" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasExactSynonym" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Nat" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasExactSynonym" content="NSS2" datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="SCNBA_MOUSE" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasExactSynonym" content="Sns2" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasExactSynonym" content="NaT" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="UniProt:Q9R053" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasExactSynonym" content="SNS2" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasExactSynonym" content="NaN" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasExactSynonym" content="Nan" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="Sns2" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-125' property="rdfs:label" content="NaT" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasExactSynonym" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-127' property="rdfs:label" content="Scn11a" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="Nan" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasExactSynonym" content="Scn11a" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:id" content="IMR:0701753" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="NSS2" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="SNS2" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="NaN" datatype="xsd:string"></span>Nat</span> <br />                                                                                                                 Genet 2002, 31:400-404. <br />        Histogram <br />        colipid <br />         growth <br />        Connectivity <br />         gene. <br />         tion <br />        0.30. <br />               Maximal <br />        -oxygen <br />         tudes. <br />         gene <br />         metabolic <br />         edges. <br />         nections: <br />         edges); <br />         Metabolic <br />           representing <br />            1.6 <br />                           aerobic <br />               enzyme-encoding <br />               correlation <br />               relationship <br />                               215; <br />                shown <br />                         <br />        sphinganine <br />        silent <br />               Additional <br />        Click <br />        amino <br />        tions <br />        connectivity <br />               sphinganine; <br />        Examples <br />        =        example <br />                  acid                   10 <br />                 Similar <br />                   0.2 <br />                                                                               -2     file <br />                   conditions. <br />                     uptake <br />                         growth <br />                         mmol/g <br />                                                                               number <br />                                                                                                divergence <br />                  metabolism; <br />                           flux <br />                       enzymes. <br />                       enzymes <br />                       connections <br />                                              data                                 theevolutionary <br />                         divergence <br />                           1-phosphate; <br />                           metabolic <br />                            CDPETN, <br />                             established <br />                              glycolysis <br />                                . <br />                                                               file <br />                                 rate <br />                                                                                                  K <br />                                  calculated <br />                                                                      distributions <br />                                                                       dpl1 <br />                                    sThe <br />                                     .3 <br />                                      16 <br />                                       5 <br />                                       4 <br />                                       2dry <br />                                        metabolites <br />                                                                                Spearman&apos;s <br />                                                                                  metabolic <br />                                          glucose <br />                                          genes <br />                                           duplicates <br />                                           K <br />                                                                                     connectivity <br />                                            dpl1 <br />                                            CDPethanolamine; <br />                                            gene <br />                                            weight/h. <br />                                             aglucose <br />                                                    gene <br />                                                                                                        (solid <br />                                                       parameters <br />                                                                                                .sphinganine <br />                                                  enzyme <br />                                                  PETHM, <br />                                                                                                   Spearman&apos;s <br />                                                    metabolic <br />                                              non-zerowere <br />                                                                                                                  rank <br />                                                                                                                                                                             Note <br />                                                           enzymatic <br />                                                                                                               reaction <br />                                                        shown <br />                                                        15.3                                                            edges), <br />                                                             drawn <br />                                                               aalso <br />                                                              mmol/g <br />                                                                                                                                  rank         sphosphate <br />                                                                               number <br />                                                                               ,weight/h <br />                                                                     rectangles. <br />                                                       disproportionately <br />                                                           connectivity <br />                                                           ethanolamine <br />                                                            ethanolamine <br />                                                               enzymes <br />                                                                 1-phosphate <br />                                                                 correlation <br />                                                                 fluxes <br />                                                                  total <br />                                                                   respective <br />                                                                                                                                      network <br />                                                                      obtained <br />                                                                      (K <br />                                                                                                                                             small <br />                                                                       ETHM, <br />                                                                        correlation <br />                                                                                                                                                 ain <br />                                                                          flux <br />                                                                          dry <br />                                                                           small <br />                                                                           defined <br />                                                                           /K <br />                                                                                                                                                                                                                                                                                                              <span id='am-1' about='obo:IMR_0701539' typeof='owl:Thing'><span id='am-2' property="oboInOwl:hasExactSynonym" content="CSAID-binding protein" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="EXIP" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="p38-ALPHA" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Mxi2" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="MK14_HUMAN" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="MAPK 14" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="EXIP" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="MAP kinase 14" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="MAX-interacting protein 2" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="MAP kinase p38 alpha" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="CSBP1" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="PRKM15" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Mitogen-activated protein kinase p38 alpha" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="CSPB1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="EC 2.7.11.24" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="PRKM15" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="p38" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="SAPK2A" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="CSPB1" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="CYTOKINE-SUPPRESSIVE ANTIINFLAMMATORY DRUG-BINDING PROTEIN 1" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="EC 2.7.11.24" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="MAPK14" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="p38ALPHA" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="PRKM14" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="p38" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="MAP kinase 14" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="CSAID-BINDING PROTEIN 1" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="CSBP" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="p38ALPHA" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="CSBP2" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="p38 MAP KINASE" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="MAP kinase MXI2" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="MXI2" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="RK" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="CSBP1" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0701539" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Mitogen-activated protein kinase 14" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="MAX-interacting protein 2" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="MAP kinase p38 alpha" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="CYTOKINE-SUPPRESSIVE ANTIINFLAMMATORY DRUG-BINDING PROTEIN 1" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="STRESS-ACTIVATED PROTEIN KINASE 2A" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="CSAID-BINDING PROTEIN 1" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="UniProt:Q16539" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="CSBP" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Mxi2" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="MXI2" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="mitogen-activated protein kinase 14" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="MAPK14" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Cytokine suppressive anti-inflammatory drug-binding protein" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="Mitogen-activated protein kinase p38 alpha" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="SAPK2A" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="MAP kinase MXI2" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="STRESS-ACTIVATED PROTEIN KINASE 2A" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="mitogen-activated protein kinase 14" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="p38 MAP KINASE" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="Cytokine suppressive anti-inflammatory drug-binding protein" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="p38-ALPHA" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="PRKM14" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasExactSynonym" content="Mitogen-activated protein kinase 14" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="CSBP2" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="RK" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="MAPK 14" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="CSAID-binding protein" datatype="xsd:string"></span>rK</span> <br />                                                                                                                                                                   large <br />                                                                                   (blue <br />                                                                                   = <br />                                                                                  metabolic <br />                                                                                phosphate; <br />                                                                                   circles <br />                                                                                                                                                                enzyme&apos;s <br />                                                                                                                                                                                                                                                    yeast <br />                                                                                    (b) <br />                                                                                    forby   = <br />                                                                                           byof <br />                                                                                   ethanolamine) <br />                                                                                       -0.056, <br />                                                                                       Ksphingogly- <br />                                                                                         average <br />                                                                                         rother <br />                                                                                          smagni- <br />                                                                                           )Metabolic <br />                                                                                                                                                                                       edges). <br />                                                                                             network <br />                                                                                                                                                                                          DHSP, <br />                                                                                                                                                                                               dashed <br />                                                                                                coding <br />                                                                                                 (red <br />                                                                                                 reac- <br />                                                                                                 fluxes <br />                                                                                                  reac- <br />                                                                                                   <br />                                                                                                  yeast <br />                                                                                               -0.13, <br />                                                                                                   P                                                                                                     =P <br />                                                                                                   SPH,    25.   Dwight <span id='am-193' about='obo:IMR_0200626' typeof='owl:Thing'><span id='am-194' property="oboInOwl:hasExactSynonym" content="EC 2.5.1.21" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasExactSynonym" content="FDFT1" datatype="xsd:string"></span><span id='am-196' property="rdfs:label" content="ERG9" datatype="xsd:string"></span><span id='am-197' property="rdfs:label" content="FDFT1" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasExactSynonym" content="DGPT" datatype="xsd:string"></span><span id='am-199' property="rdfs:label" content="FPP:FPP farnesyltransferase" datatype="xsd:string"></span><span id='am-200' property="rdfs:label" content="Squalene synthase" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-202' property="rdfs:label" content="EC 2.5.1.21" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasExactSynonym" content="FARNESYLDIPHOSPHATE FARNESYLTRANSFERASE 1" datatype="xsd:string"></span><span id='am-204' property="rdfs:label" content="farnesyl-diphosphate farnesyltransferase 1" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasDbXref" content="UniProt:P37268" datatype="xsd:string"></span><span id='am-206' property="rdfs:label" content="SS" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasExactSynonym" content="Squalene synthase" datatype="xsd:string"></span><span id='am-208' property="oboInOwl:hasExactSynonym" content="SQS" datatype="xsd:string"></span><span id='am-209' property="rdfs:label" content="Farnesyl-diphosphate farnesyltransferase" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasExactSynonym" content="farnesyl-diphosphate farnesyltransferase 1" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:id" content="IMR:0200626" datatype="xsd:string"></span><span id='am-212' property="rdfs:label" content="DGPT" datatype="xsd:string"></span><span id='am-213' property="rdfs:label" content="SQS" datatype="xsd:string"></span><span id='am-214' property="rdfs:label" content="FARNESYLDIPHOSPHATE FARNESYLTRANSFERASE 1" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasExactSynonym" content="ERG9" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasExactSynonym" content="Farnesyl-diphosphate farnesyltransferase" datatype="xsd:string"></span><span id='am-217' property="rdfs:label" content="FDFT_HUMAN" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasExactSynonym" content="FPP:FPP farnesyltransferase" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasExactSynonym" content="SS" datatype="xsd:string"></span>SS</span>, Balakrishnan R, Christie KR, Costanzo MC, Dolinski K, <br />                                                                                                                 Engel SR, Feierboch B, Fisk DG, Hirchman J, Hong EL, et al.: Saccha- <br />                                                                                                                 romyces genome database: underlying principles        Acknowledgements                                                                                         organisation. Brief Bioinform 2004, 5:9-22. <br />        thank Dr Andrey Rzhetsky, Dr Uwe Sauer, Dr Eugene Koonin                                26.   Mahadevan R, Palsson BO: Properties metabolic networks: <br />        valuable discussions. thank anonymous reviewers                                  structure versus function. Biophys J 2005, 88:L07-L09. <br />        helpful suggestions.                                                                          27.   Gu  Steinmetz LM, Gu X, Scharfe C, Davis RW, Li WH: Role                                                                                                                 duplicate genes genetic robustness null mutations. <br />                                                                                                                 Nature 2003, 421:63-66. <br />                                                                                                           28.   Wagner  Robustness mutations genetics <span id='am-748' about='obo:IMR_0001104' typeof='owl:Thing obo:IMR_0000397 obo:IMR_0000949 obo:IMR_0000369 obo:IMR_0000368 obo:IMR_0000001'><span id='am-749' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-750' property="oboInOwl:id" content="IMR:0001104" datatype="xsd:string"></span><span id='am-751' property="rdfs:label" content="Net" datatype="xsd:string"></span><span id='am-752' property="rdfs:label" content="Sap-2" datatype="xsd:string"></span><span id='am-753' property="rdfs:label" content="Elk-3" datatype="xsd:string"></span><span id='am-754' property="oboInOwl:hasExactSynonym" content="Sap-2" datatype="xsd:string"></span><span id='am-755' property="oboInOwl:hasExactSynonym" content="Elk-3" datatype="xsd:string"></span>net</span>- <br />        References                                                                                               works yeast. Nat Genet 2000, 24:355-361. <br />        1.     Jeong H, Mason SP, Barabasi AL, Oltvai ZN: Lethality central-                           29.   Edwards JS, Palsson BO: Robustness analysis Esherichia <br />               ity protein networks. Nature 2001, 411:41-42.                                                  coli metabolic network. Biotechnol Prog 2000, 16:927-939. <br />        2.     Hirsh AE, Fraser HB: Protein dispensability rate                                     30.   Kuepfer L, Sauer U, Blank LM: Metabolic functions duplicate <br />               evolution. Nature 2001, 411:1046-104 <br /> </body></html>