<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Activity Reaction Core Plasticity Metabolic Networks <br /> Eivind Almaas1,2, Zolta 180;n N. Oltvai3*, Albert-La 180;szlo 180; Baraba 180;si2,4 <br /> 1 Microbial Systems Division, Biosciences Directorate, Lawrence Livermore National Laboratory, Livermore, California, United States America, 2 Center Complex <br /> Network Research Department Physics, University Notre Dame, Notre Dame, Indiana, United States America, 3 Department Pathology, University Pittsburgh <br /> School Medicine, Pittsburgh, Pennsylvania, United States America, 4 Center Cancer Systems Biology, Dana-Farber Cancer Institute, Harvard University, Boston, <br /> Massachusetts, United States America <br />  <br />  <br /> Understanding level adaptive changes taking place organism response variations environment key issue contemporary biology. Current modeling approaches, constraint-based flux- <br /> balance analysis, proved highly successful analyzing capabilities cellular metabolism, including capacity predict deletion phenotypes, ability calculate relative flux values metabolic reactions, capability identify properties optimal growth states.  use flux-balance analysis thoroughly assess activity Escherichia coli, Helicobacter pylori, Saccharomyces cerevisiae metabolism 30,000 diverse simulated <br /> environments. identify set metabolic reactions forming connected metabolic core carry non-zero fluxes growth conditions, flux variations highly correlated. Furthermore, enzymes <br /> catalyzing core reactions display considerably higher fraction phenotypic essentiality evolutionary <br /> conservation catalyzing noncore reactions. Cellular metabolism characterized large number species-specific conditionally active reactions organized evolutionary conserved, active, <br /> metabolic core. Finally, current antibiotics interfering bacterial metabolism target core <br /> enzymes, indicating findings important implications antimicrobial drug-target discovery. <br /> Citation: Almaas E, Oltvai ZN, Baraba 180;si (2005) activity reaction core plasticity metabolic networks. PLoS Comput Biol 1(7): e68. <br />  <br />  <br />                                                                                           changes induce structural plasticity, resulting Introduction <br />                                                                                           changes metabolism 8217;s active wiring diagram, turning <br />   Constraint-based modeling approaches,  64258;ux-bal-                                  previously zero- 64258;ux reactions inhibiting previously <br /> <span id='am-244' about='obo:IMR_0003025' typeof='owl:Thing'><span id='am-245' property="oboInOwl:hasExactSynonym" content="BG:DS08220.3" datatype="xsd:string"></span><span id='am-246' property="rdfs:label" content="l(2)br31" datatype="xsd:string"></span><span id='am-247' property="oboInOwl:hasExactSynonym" content="Race" datatype="xsd:string"></span><span id='am-248' property="rdfs:label" content="l(2)34Eb" datatype="xsd:string"></span><span id='am-249' property="rdfs:label" content="AnCE" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasExactSynonym" content="Ance" datatype="xsd:string"></span><span id='am-251' property="oboInOwl:hasExactSynonym" content="Angiotensin converting enzyme" datatype="xsd:string"></span><span id='am-252' property="oboInOwl:hasExactSynonym" content="l34Eb" datatype="xsd:string"></span><span id='am-253' property="rdfs:label" content="BG:DS08220.3" datatype="xsd:string"></span><span id='am-254' property="rdfs:label" content="Related to angiotensin converting enzyme" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasExactSynonym" content="EC 3.4.15.1" datatype="xsd:string"></span><span id='am-256' property="oboInOwl:hasExactSynonym" content="CG8827" datatype="xsd:string"></span><span id='am-257' property="rdfs:label" content="anon-EST:fe3D10" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasExactSynonym" content="ACE" datatype="xsd:string"></span><span id='am-259' property="rdfs:label" content="Angiotensin converting enzyme" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasExactSynonym" content="AnCE" datatype="xsd:string"></span><span id='am-261' property="oboInOwl:id" content="IMR:0003025" datatype="xsd:string"></span><span id='am-262' property="rdfs:label" content="Race" datatype="xsd:string"></span><span id='am-263' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-264' property="rdfs:label" content="Ance" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:hasExactSynonym" content="Kininase II" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasExactSynonym" content="br31" datatype="xsd:string"></span><span id='am-267' property="rdfs:label" content="CG8827" datatype="xsd:string"></span><span id='am-268' property="rdfs:label" content="Angiotensin-converting enzyme" datatype="xsd:string"></span><span id='am-269' property="rdfs:label" content="anon-fast-evolving-3D10" datatype="xsd:string"></span><span id='am-270' property="oboInOwl:hasExactSynonym" content="Angiotensin-converting enzyme" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasExactSynonym" content="RACE" datatype="xsd:string"></span><span id='am-272' property="oboInOwl:hasExactSynonym" content="Dipeptidyl carboxypeptidase I" datatype="xsd:string"></span><span id='am-273' property="oboInOwl:hasExactSynonym" content="anon-EST:fe3D10" datatype="xsd:string"></span><span id='am-274' property="rdfs:label" content="race" datatype="xsd:string"></span><span id='am-275' property="rdfs:label" content="l34Eb" datatype="xsd:string"></span><span id='am-276' property="oboInOwl:hasExactSynonym" content="anon-fast-evolving-3D10" datatype="xsd:string"></span><span id='am-277' property="rdfs:label" content="ACE" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasDbXref" content="UniProt:Q10714" datatype="xsd:string"></span><span id='am-279' property="rdfs:label" content="Dipeptidyl carboxypeptidase I" datatype="xsd:string"></span><span id='am-280' property="rdfs:label" content="ACE_DROME" datatype="xsd:string"></span><span id='am-281' property="rdfs:label" content="Kininase II" datatype="xsd:string"></span><span id='am-282' property="oboInOwl:hasExactSynonym" content="Related to angiotensin converting enzyme" datatype="xsd:string"></span><span id='am-283' property="rdfs:label" content="RACE" datatype="xsd:string"></span><span id='am-284' property="oboInOwl:hasExactSynonym" content="l(2)br31" datatype="xsd:string"></span><span id='am-285' property="rdfs:label" content="EC 3.4.15.1" datatype="xsd:string"></span><span id='am-286' property="oboInOwl:hasExactSynonym" content="l(2)34Eb" datatype="xsd:string"></span><span id='am-287' property="rdfs:label" content="br31" datatype="xsd:string"></span><span id='am-288' property="oboInOwl:hasExactSynonym" content="race" datatype="xsd:string"></span>ance</span> analysis (FBA) [1,2], proved highly successful                               active pathways. example, shifting E. coli cells analyzing capabilities cellular metabolism, including                          glucose- succinate-rich media, 11 previously active <br /> capacity predict deletion phenotypes, ability                                   reactions turned completely, previously <br /> calculate relative  64258;ux values metabolic reactions,                             inactive reactions turned  capability identify properties alternate optimal                                   types response mechanisms described  64258;ux <br /> growth states wide range simulated single-carbon-                                 structural plasticity imply possible existence source environmental conditions [3 8211;5]. Recent analyses                               group reactions subject structural plasticity, <br /> indicate deletion phenotype substantial subset                           active environmental conditions.  <br /> metabolic enzymes growth-condition-dependent [6], argu-                                metabolic reactions carry non-zero  64258;uxes <br /> ing selective use metabolic reactions distinct                                S. cerevisiae different growth conditions [6]. environments.                                                                             itself,   65533;?nding compatible scenario                                                                                           active reactions randomly distributed: Results                                                                                   typically q fraction metabolic reactions active <br />                                                                                           speci 65533;?c growth condition, n distinct conditions <br />    thoroughly examine utilization relative  64258;ux                                 predict non-zero overlap encompassing qn <br /> rates metabolic reaction wide range simulated                             fraction reactions.   65533;?nd increase <br /> environmental conditions, sampled 30,000 randomly                                  number inspected conditions, n, number uniformly chosen optimal growth conditions, single-carbon-source minimal medium conditions suf 65533;?cient <br />                                                                                           Received June 27, 2005; Accepted November 2, 2005; Published December 16, growth, using FBA [1,2] reconstructed metabolic                                2005 <br /> networks Helicobacter pylori, Escherichia coli, Saccharomyces                      DOI: 10.1371/journal.pcbi.0010068 <br /> cerevisiae [7 8211;9]  Materials Methods).                               Copyright:  211; 2005 Almaas et al. open-access article distributed assuming optimal growth microorganisms,                                 terms Creative Commons Attribution License, permits unrestricted <br />                                                                                           use, distribution, reproduction medium, provided original author metabolism adapts environmental changes                                  source credited. distinct mechanisms. common mechanism                                                                                           Abbreviation: FBA, flux-balance analysis <br />  64258;ux plasticity, involving changes  64258;uxes active reactions organism shifted                                    Editor: Daniel Segre, Boston University, United States America <br /> growth condition  example, changing                                   * correspondence addressed. E-mail: oltvai@pitt.edu <br /> glucose- succinate-rich media alters  64258;ux 264 E. coli                            previous version article appeared Early Online Release November <br /> reactions 20%. commonly, environmental                                  2, 2005 (DOI: 10.1371/journal.pcbi.0010068.eor). <br />  <br />  <br />         PLoS Computational Biology | www.ploscompbiol.org                          0557                                     December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                      Metabolic Network Core <br />  <br />  <br /> Synopsis                                                                                64258;ux-based plasticity, rest reactions                                                                                        conditionally active, turned speci 65533;?c cellular metabolism investigated                            growth conditions subject structural- <br /> cellular functions, understood changes                        64258;ux-based plasticity. <br /> adapts systems level response environmental variations.                     Figure 2 displays metabolic reactions E. coli study, authors advantage reconstructed                            remain active 30,000 simulated growth conditions. metabolic networks Helicobacter pylori, Escherichia coli,                       striking feature diagram fact reactions <br /> Saccharomyces cerevisiae computational method flux- <br />                                                                                        form single connected cluster, encompassing 90 <br /> balance analysis investigate functional plasticity metabolic networks large number simulated growth <br />                                                                                        reactions. unique feature E. coli; H. pylori, conditions.                                                                            138 reactions form single cluster, S. cerevisiae 33 <br />                                                                                        reactions connected. Given relatively low number approach, authors identify metabolic cores                      reactions active, likelihood form organisms. metabolic cores represent connected sets                     single large cluster chance negligible, p , <br /> reactions used tested environments. cross-                        1e 6 H. pylori E. coli, p ; 2e 6 S. cerevisiae. Given <br /> correlating predicted cores experimental data, authors                    compact clustered nature group reactions cores display significantly higher fraction,                                                                                        active, refer collectively essential evolutionary conserved enzymes, noncore <br /> counterparts. Additionally, extended mRNA half-lives core <br />                                                                                        metabolic core. <br /> enzymes support notion core reactions                            metabolic core contains types reactions: represent main integrators metabolic activity.                                       65533;?rst type consists essential biomass <br />                                                                                        formation environmental conditions (81 90 metabolic core reactions indispensable                   reactions E. coli), second type reaction growth microorganism, existing antibiotics target                   required assure optimal metabolic performance <br /> select bacterial enzymes core, authors suggest                     (Table <span id='am-645' about='obo:IMR_0702938' typeof='owl:Thing'><span id='am-646' property="rdfs:label" content="Rpn2" datatype="xsd:string"></span><span id='am-647' property="oboInOwl:hasExactSynonym" content="26S proteasome subunit p112" datatype="xsd:string"></span><span id='am-648' property="oboInOwl:id" content="IMR:0702938" datatype="xsd:string"></span><span id='am-649' property="rdfs:label" content="PSMD1" datatype="xsd:string"></span><span id='am-650' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN2" datatype="xsd:string"></span><span id='am-651' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 1" datatype="xsd:string"></span><span id='am-652' property="rdfs:label" content="PSMD1_HUMAN" datatype="xsd:string"></span><span id='am-653' property="rdfs:label" content="MGC133040" datatype="xsd:string"></span><span id='am-654' property="rdfs:label" content="26S proteasome regulatory subunit RPN2" datatype="xsd:string"></span><span id='am-655' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 1" datatype="xsd:string"></span><span id='am-656' property="oboInOwl:hasExactSynonym" content="S1" datatype="xsd:string"></span><span id='am-657' property="rdfs:label" content="MGC133041" datatype="xsd:string"></span><span id='am-658' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 1" datatype="xsd:string"></span><span id='am-659' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-660' property="oboInOwl:hasExactSynonym" content="PSMD1" datatype="xsd:string"></span><span id='am-661' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 1" datatype="xsd:string"></span><span id='am-662' property="rdfs:label" content="26S proteasome regulatory subunit S1" datatype="xsd:string"></span><span id='am-663' property="oboInOwl:hasExactSynonym" content="Rpn2" datatype="xsd:string"></span><span id='am-664' property="rdfs:label" content="P112" datatype="xsd:string"></span><span id='am-665' property="oboInOwl:hasExactSynonym" content="MGC133041" datatype="xsd:string"></span><span id='am-666' property="oboInOwl:hasExactSynonym" content="P112" datatype="xsd:string"></span><span id='am-667' property="rdfs:label" content="26S proteasome subunit p112" datatype="xsd:string"></span><span id='am-668' property="oboInOwl:hasDbXref" content="UniProt:Q99460" datatype="xsd:string"></span><span id='am-669' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S1" datatype="xsd:string"></span><span id='am-670' property="rdfs:label" content="S1" datatype="xsd:string"></span><span id='am-671' property="oboInOwl:hasExactSynonym" content="MGC133040" datatype="xsd:string"></span>S1</span>). case inactivation second type, <br /> metabolic core important applications antimicrobial <br />                                                                                        alternative sub-optimal pathways used ensure <br /> drug-target discovery. <br />                                                                                        cellular survival. Interestingly, compact core S. <br />                                                                                        cerevisiae, 33 reactions predicted indispensable <br />                                                                                        biomass formation growth conditions.  <br /> reactions carrying non-zero  64258;ux condition decreases                             assuming 10% reduction growth rate <br /> rapidly.  eventually saturates constant value                          (compared optimal growth), size identity E. <br /> (Figure 1A 8211;C), identifying group reactions non-                         coli metabolic core remains largely unchanged, retaining 83 zero  64258;ux 30,000 simulated growth conditions.                                 original 90 reactions, nonessential. Speci 65533;?cally,  65533;?nd 138 381 H. pylori (36.2%), 90                           indicates concept metabolic core valid 758 E. coli (11.9%), 33 1,172 S. cerevisiae (2.8%),                          optimal suboptimal growth, metabolic reactions active (Figure 1D 1E).                              difference identity individual reactions. note,                                                                                        metabolic core represents subset minimal reaction reactions respond environmental changes                                                                                        sets [10,11] overlap alternative  e., degenerate) <br />                                                                                        minimal reaction sets [4,5]. minimal reaction set [11] <br />                                                                                        contains metabolic core addition,                                                                                         reactions necessary sustained growth chosen <br />                                                                                        substrate, minimal reaction set [5] consists                                                                                        201 reactions active E. coli 136 <br />                                                                                        aerobic anaerobic single-carbon-source minimal envi- <br />                                                                                        ronments capable sustaining optimal growth.                                                                                         65533;?nding excellent agreement Figure 1B (upper <br />                                                                                        curve), demonstrates analysis low number                                                                                        growth conditions signi 65533;?cantly overestimate set                                                                                        reactions active. <br />                                                                                           identify factors determine size                                                                                        metabolic core, note number core <br />                                                                                        metabolic reactions systematically decreases                                                                                        H. pylori S. cerevisiae (Figures 3A 1E), relative size                                                                                        core decreasing 36.2% reactions H. pylori Figure 1. Emergence Metabolic Core <br />                                                                                        11.9% E. coli 2.8% S. cerevisiae. trend  C) average relative size number reactions active function number sampled conditions (black                 explained collective network effect: relatively small <br /> line)   H. pylori, (B) E. coli, (C) S. cerevisiae. number          size H. pylori metabolic network leaves little  64258;exibility <br /> conditions increases, curve converges constant denoted                 biomass production, requiring continuous activity dashed line, identifying metabolic core organism. red line <br /> denotes number reactions active activity                                                                                        high fraction available metabolic reactions.  <br /> randomly distributed metabolic network. fact                         65533;?nd average approximately 61% H. pylori <br /> converges zero indicates real core represents collective                 reactions active given environmental condition. network effect, forcing group reactions active conditions.           larger number metabolic reactions present E. coli offers <br /> (D E) number metabolic reactions (D) number metabolic core reactions (E) studied organisms.                           higher degree metabolic  64258;exibility, allowing DOI: 10.1371/journal.pcbi.0010068.g001                                                 signi 65533; fraction biomass produced  <br />        PLoS Computational Biology | www.ploscompbiol.org                        0558                               December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                    Metabolic Network Core <br />  <br />  <br />  <br />  <br /> Figure 2. Metabolic Core E. coli reactions active 30,000 investigated external conditions shown. Metabolites contribute directly biomass formation [8] colored blue, core reactions (links) catalyzed essential  nonessential) enzymes [13] colored red  green). <br /> (Black-colored links denote enzymes unknown deletion phenotype.) blue dashed lines indicate multiple appearances metabolite, links arrows denote unidirectional reactions. Note 20 51 metabolites necessary biomass synthesis present core, <br /> indicating produced  consumed) growth-condition-specific manner. Protocol S1 Table S1 abbreviations metabolites list core reactions E. coli, H. pylori, S. cerevisiae. folate peptidoglycan biosynthesis pathways indicated blue brown shading, respectively, white numbered arrows denote current antibiotic targets inhibited  (1) sulfonamides, (2) trimethoprim, (3) <br /> cycloserine, (4) fosfomycin. Note reactions appear disconnected omitted drawing cofactors. <br /> DOI: 10.1371/journal.pcbi.0010068.g002 <br />  <br />  <br />  <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                   0559                                December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                Metabolic Network Core <br />  <br />  <br />                                                                                    alternative pathways.  average utilization E. <br />                                                                                    coli metabolic network given growth condition                                                                                    35.3%. S. cerevisiae, metabolic network size <br />                                                                                    signi 65533;?cantly exceeds H. pylori E. coli,                                                                                    higher metabolic  64258;exibility, activity                                                                                    19.7% reactions required typical environment. <br />                                                                                       fact core reactions active                                                                                    investigated environmental conditions suggests                                                                                    play key role maintaining metabolism 8217;s overall <br />                                                                                    functional integrity.  absence individual <br />                                                                                    reactions metabolic core lead                                                                                    signi 65533; metabolic disruptions.  using genome-scale <br />                                                                                    deletion-phenotype data obtained rich growth media <br />                                                                                    [12,13],  65533;?nd 74.7% E. coli enzymes                                                                                    catalyze core metabolic reactions  e., core enzymes)                                                                                    essential, compared 19.6% lethality fraction character- <br />                                                                                    izing noncore enzymes. <span id='am-357' about='obo:IMR_0704195' typeof='owl:Thing'><span id='am-358' property="oboInOwl:hasExactSynonym" content="Rasa1" datatype="xsd:string"></span><span id='am-359' property="oboInOwl:hasExactSynonym" content="p120GAP" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasExactSynonym" content="Ras p21 protein activator" datatype="xsd:string"></span><span id='am-361' property="rdfs:label" content="GAP" datatype="xsd:string"></span><span id='am-362' property="rdfs:label" content="RAS p21 protein activator (GTPase activating protein) 1" datatype="xsd:string"></span><span id='am-363' property="rdfs:label" content="Ras GTPase-activating protein 1" datatype="xsd:string"></span><span id='am-364' property="oboInOwl:hasExactSynonym" content="RasGAP" datatype="xsd:string"></span><span id='am-365' property="oboInOwl:id" content="IMR:0704195" datatype="xsd:string"></span><span id='am-366' property="rdfs:label" content="RasGAP" datatype="xsd:string"></span><span id='am-367' property="oboInOwl:hasExactSynonym" content="GTPase-activating protein" datatype="xsd:string"></span><span id='am-368' property="rdfs:label" content="GTPase-activating protein" datatype="xsd:string"></span><span id='am-369' property="oboInOwl:hasExactSynonym" content="Rasa" datatype="xsd:string"></span><span id='am-370' property="oboInOwl:hasExactSynonym" content="RAS p21 protein activator (GTPase activating protein) 1" datatype="xsd:string"></span><span id='am-371' property="rdfs:label" content="Rasa1" datatype="xsd:string"></span><span id='am-372' property="oboInOwl:hasExactSynonym" content="Ras GTPase-activating protein 1" datatype="xsd:string"></span><span id='am-373' property="rdfs:label" content="RASA1_RAT" datatype="xsd:string"></span><span id='am-374' property="rdfs:label" content="Ras p21 protein activator" datatype="xsd:string"></span><span id='am-375' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-376' property="oboInOwl:hasExactSynonym" content="GAP" datatype="xsd:string"></span><span id='am-377' property="rdfs:label" content="p120GAP" datatype="xsd:string"></span><span id='am-378' property="oboInOwl:hasDbXref" content="UniProt:P50904" datatype="xsd:string"></span><span id='am-379' property="rdfs:label" content="Rasa" datatype="xsd:string"></span>gap</span> core                                                                                    noncore enzymes signi 65533; S. cerevisiae,                                                                                    essential enzymes catalyze 84% core reactions,                                                                                    average essentiality conditionally active enzymes                                                                                    15.6% (Figure 3B). Note likelihood random <br />                                                                                    concentration essential enzymes core                                                                                    extremely small, p-values 3.3e 23 9.0e 13 E. coli <br />                                                                                    yeast, respectively, presume presence                                                                                    errors deletion-phenotype data (Protocol S1). Taken <br />                                                                                     results indicate organism 8217;s ability                                                                                    adapt changing environmental conditions rests, large <br />                                                                                    extent, continuous activity metabolic core, <br />                                                                                    regardless environmental conditions. <br />                                                                                       Intuitively, assume core represents                                                                                    subset high- 64258;ux reactions characterizing activity                                                                                    metabolic networks [3].  measurements indicate <br />                                                                                     average,  64258;uxes core noncore reactions <br />                                                                                    highly comparable (Figure 3C). Alternatively,                                                                                    assume main biological role metabolic <br />                                                                                    core ensure continuous production biomass                                                                                    growth conditions. contrast,  65533;?nd 20                                                                                    51 metabolites (39%) considered necessary biomass <br />                                                                                    production E. coli [8] produced core <br />                                                                                    reactions; instead growth-condition-dependent fash- <br />                                                                                    ion produced various alternative metabolic <br />                                                                                    pathways. core,  contains large number                                                                                    reactions selected anabolic pathways, including                                                                                    membrane lipid-, cell envelope-, peptidoglycan-biosyn- <br />                                                                                    thesis pathways (Table S1). core reactions represent <br />                                                                                    network bottlenecks, paths synthesis                                                                                    certain biomass components.  appears                                                                                    composition metabolic core determined                                                                                    factors.  metabolic reactions directly contrib- <br />                                                                                    ute biomass production tend metabolic <br />                                                                                    core organism. Second,  tendency offset <br />                                                                                    network-induced redundancy: reactions pathways <br />                                                                                    end-products synthesized                                                                                    alternative pathways environmental redundancy Figure 3. Characterizing Metabolic Cores                                       structural plasticity, eliminated core. <br />   number overlapping metabolic reactions metabolic core             reactions metabolic network possesses H. pylori, E. coli, S. cerevisiae.                                           Figure 1D), stronger network-induced <br /> (B) fraction metabolic reactions catalyzed essential enzymes          redundancy, smaller core (Figure 1E). cores (black) outside core E. coli S. cerevisiae. <br /> (C) distribution average metabolic fluxes core                                                                                       Given important functional role played noncore reactions E. coli.                                                      metabolic core given organism, expect <br /> DOI: 10.1371/journal.pcbi.0010068.g003                                             signi 65533; parts core conserved different <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                    0560                             December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                      Metabolic Network Core <br />  <br />  <br /> organisms.  E. coli H. pylori cores 63            redundant core, regulatory mechanisms shifted reactions common, 18 33 core reactions S.            mRNA decay rate nature regulatory <br /> cerevisiae present E. coli H. pylori          interactions, offering higher regulatory stability metabolic cores  Figure 3A).  considering               core enzymes. <br /> <span id='am-44' about='obo:IMR_0000207' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001'><span id='am-45' property="oboInOwl:hasDbXref" content="GO:0003824" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="enzyme" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0000207" datatype="xsd:string"></span><span id='am-48' property="obo:IAO_0000115" content="Protein that catalyze a specific chemical reaction." datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>enzyme</span> orthologs 32 divergent bacteria [13],  65533;?nd                 65533;?nding core reactions form single cluster metabolic core enzymes E. coli display high             suggest activity participating reactions <br /> degree evolutionary conservation, average core enzyme          highly synchronized, changes  64258;ux having orthologs 71.7% reference bacteria (p ,              reaction affecting  64258;ux reactions   1e 6). contrast, conditionally active noncore enzymes          able discover core inspecting evolutionary retention 47.7% [13]. Given          correlations activities metabolic reactions. <br /> observed correlation evolutionary retention           test hypothesis, calculated  64258;ux correlation <br /> essentiality gene products [13], difference          coef 65533;?cient, Cij, reaction pair E. coli metabolism <br /> simple consequence high-lethality fraction core         [17] inspecting  64258;ux values reactions enzymes.  analysis indicates         30,000 simulated growth conditions. Using Cij case: random selection 90 enzymes 74.7%                 matrix metric hierarchical clustering algorithm <br /> lethality ratio average evolutionary retention         [18], observed emergence group reactions <br /> 63.4%. Taken  results indicate major               64258;uxes change simultaneously environmental <br /> portions metabolic core conserved,                   shifts. Interestingly, highly correlated reactions signi 65533;?- <br /> displaying higher evolutionary retention individual        cantly overlap metabolic core (Figure 4A), indicating <br /> essentiality participating enzymes indicate,             environmental conditions altered, suggesting maintaining core 8217;s integrity collective       64258;uxes core reactions increased decreased need organism.                                                 synchronized fashion. <br />    requirement continuous activity core               synchrony observed inspect exper- <br /> reactions impact regulation catalytic        imental mRNA data core reactions changes <br /> enzymes. expect activity core                    environmental conditions. Calculating correlations <br /> reactions  64258;ux magnitude needs                core noncore enzymes using correspond- <br /> modulated display higher degree stability           ing mRNA copy number data  Materials Methods) different regulatory control reactions outside        41 experiments [19],  65533;?nd correlations core. Evidence regulatory effects provided         mRNA copy numbers core enzymes systematically <br /> measurements.  inspected experimentally deter-           higher observed correlations noncore <br /> mined mRNA half-lives E. coli metabolic enzymes           counterparts (Figure 4B), average correlation ,C. <br /> cells grown Luria-Bertani medium [14].                188; 0.23 core enzymes ,C.  188; 0.07 noncore enzymes distribution mRNA half-life times core            (p , 1e 4). Notably,  65533;?nding different average <br /> enzymes displays shift higher values, average           correlation mRNA expression 66  8216; 8216;correlated <br /> half-life time mRNAs 14.0 min compared            reaction sets 8217; 8217; (groups reactions 10.5 min noncore mRNAs (p  188; 0.016). Furthermore,              turned , frequency 2% core enzymes corresponding mRNA half-             correlated reaction set used does affect life times shorter 5 min, compared 23% ones           mRNA correlation [5]. <br /> catalyzing conditionally active noncore reactions  Materi- <br /> als Methods).  continuous metabolic need                Discussion reactions apparently affected mechanisms <br /> responsible mRNA decay catalytic enzymes,                       Previous studies  65533;?rmly established environmental <br /> providing higher dynamic inertia environmental                    changes induce  64258;ux plasticity, altering  64258;ux levels changes.                                                                  individual reactions (reviewed [20]). Similarly, fact    Additionally, enzyme-encoding operon,                      metabolism displays structural plasticity, turning counted number activating repressive regulatory                reactions growth conditions altered, links E. coli transcriptional regulatory network [15,16].          observed [3,4,6].  demonstration regulatory interactions currently known 13             group reactions predicted active environmental core enzyme-encoding operons, considered                       conditions forming connected metabolic core extended core, representing set 234 reactions                 substantially improve understanding organization active 90% 30,000 simulated growth                utilization metabolic networks. emergence conditions. results indicate fraction repressive          core represents collective network effect, channeling regulatory links extended core 52.3%,                 production indispensable biomass components fraction activating interactions 35.7%,                key reactions replaced alternative <br /> remaining 12% represents regulatory links                 pathways environmental conditions. collective <br /> activate suppress enzyme 8217;s mRNA synthesis rate.                 origin core supported observed changes contrast, noncore enzyme-encoding operons,                core size. number biomass components difference fraction activating repressing              metabolism needs produce comparable links, representing 45% regulatory interactions.              organisms (49 H. pylori, 51 E. coli, 44 S. results offer evidence evolution core             cerevisiae), number metabolic reactions contributing metabolic network corresponding regulatory network:               differs. larger size metabolic network <br /> given requirement continuously active non-                    signi 65533;?cantly increases capacity redundancy, decreas- <br />  <br />       PLoS Computational Biology | www.ploscompbiol.org            0561                             December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                          Metabolic Network Core <br />  <br />  <br />                                                                                       bacterial yeast gene products essential,                                                                                       high fraction essential speci 65533;?c environ- <br />                                                                                       ments. example, recent measurements indicate 76% <br />                                                                                       S. cerevisiae genes inactive nutrient-rich <br />                                                                                       conditions fact active, essential                                                                                       growth conditions [6].  effective <br />                                                                                       antimicrobial drug needs able kill target organism <br />                                                                                       physiological conditions exist.                                                                                       pharmacological interventions targeting speci 65533;?c pathway <br />                                                                                       effective environments pathway                                                                                       needed corresponding enzyme turned  Instead, <br />                                                                                       effective antimicrobials target activity                                                                                       core reactions, disruption impact                                                                                       microorganism 8217;s ability function environ- <br />                                                                                       mental conditions.  currently used anti- <br />                                                                                       biotics, fosfomycin cycloserine act inhibiting cell-wall <br />                                                                                       peptidoglycan, sulfonamides trimethoprim inhibit <br />                                                                                       tetrahydrofolate biosynthesis, pathways present                                                                                       H. pylori E. coli core  Figure 2). addition, <br />                                                                                       core pathways involved synthesis  64258;avin adenine <br />                                                                                       dinucleotide, 2-dehydro-3-deoxy-D-octonate, lipopolysac- <br />                                                                                       charide potential antibiotic targets [21]. <br />                                                                                          indicates core enzymes essential biomass <br />                                                                                       formation, optimal suboptimal growth (Table <br />                                                                                       S1), prove effective antibiotic targets given cell 8217;s <br />                                                                                       need maintain activity enzymes                                                                                       conditions. fact core reactions                                                                                       shared bacteria offers possibility identify <br />                                                                                       bacterium-speci 65533;?c drug targets. Finally, results pertain- <br />                                                                                       ing existence core characteristics                                                                                       means limited studied organisms,                                                                                       analysis carried organisms high- <br />                                                                                       quality metabolic reconstructions available. Given                                                                                       large number sequenced bacterial genomes, studies <br />                                                                                       open new avenues rapid silico antimicrobial <br />                                                                                       drug-target identi 65533;?cation. <br />  <br />                                                                                       Materials Methods <br /> Figure 4. Correlations E. coli Metabolic Reactions                                FBA. Starting published stoichiometric matrices   calculated Pearson correlation using flux values 30,000               reconstructed E. coli MG1655, H. pylori, S. cerevisiae metabolic <br /> conditions reaction pair grouping reactions                       networks [7 8211;9], steady-state concentrations internal <br /> according hierarchical average-linkage clustering algorithm.                 metabolites organism satisfy <br /> values flux-correlation matrix range  1 (red) 0                                           d         X <br /> (white) unity (blue). horizontal color <span id='am-187' about='obo:IMR_0100740' typeof='owl:Thing'><span id='am-188' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasExactSynonym" content="Mir" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasExactSynonym" content="Protein hedgehog" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasExactSynonym" content="Hh-Np" datatype="xsd:string"></span><span id='am-192' property="rdfs:label" content="hg" datatype="xsd:string"></span><span id='am-193' property="oboInOwl:hasExactSynonym" content="anon-WO0134654.19" datatype="xsd:string"></span><span id='am-194' property="rdfs:label" content="Mrt" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasExactSynonym" content="l(3)neo56" datatype="xsd:string"></span><span id='am-196' property="rdfs:label" content="hedgehog" datatype="xsd:string"></span><span id='am-197' property="rdfs:label" content="Mirabile" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasExactSynonym" content="hh" datatype="xsd:string"></span><span id='am-199' property="rdfs:label" content="bar-on-3" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasExactSynonym" content="hg" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasExactSynonym" content="l(3)neo57" datatype="xsd:string"></span><span id='am-202' property="rdfs:label" content="Hh-Cp" datatype="xsd:string"></span><span id='am-203' property="rdfs:label" content="Protein hedgehog" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasExactSynonym" content="CG4637" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasExactSynonym" content="hedgehog" datatype="xsd:string"></span><span id='am-206' property="rdfs:label" content="CG4637" datatype="xsd:string"></span><span id='am-207' property="rdfs:label" content="Protein hedgehog C-product" datatype="xsd:string"></span><span id='am-208' property="rdfs:label" content="C-Hh" datatype="xsd:string"></span><span id='am-209' property="rdfs:label" content="N-Hh" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasExactSynonym" content="HH" datatype="xsd:string"></span><span id='am-211' property="rdfs:label" content="bar3" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasExactSynonym" content="bar3" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:hasExactSynonym" content="Protein hedgehog C-product" datatype="xsd:string"></span><span id='am-214' property="rdfs:label" content="Moonrat" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasExactSynonym" content="Hh-Cp" datatype="xsd:string"></span><span id='am-216' property="rdfs:label" content="hh" datatype="xsd:string"></span><span id='am-217' property="rdfs:label" content="HH" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasDbXref" content="UniProt:Q02936" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasExactSynonym" content="C-Hh" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasExactSynonym" content="Mirabile" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasExactSynonym" content="Mrt" datatype="xsd:string"></span><span id='am-222' property="oboInOwl:hasExactSynonym" content="bar-3" datatype="xsd:string"></span><span id='am-223' property="rdfs:label" content="anon-WO0134654.19" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasExactSynonym" content="bar" datatype="xsd:string"></span><span id='am-225' property="rdfs:label" content="Hh" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasExactSynonym" content="N-Hh" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasExactSynonym" content="Hh" datatype="xsd:string"></span><span id='am-228' property="rdfs:label" content="l(3)neo57" datatype="xsd:string"></span><span id='am-229' property="rdfs:label" content="bar-3" datatype="xsd:string"></span><span id='am-230' property="rdfs:label" content="anon-WO0182946.19" datatype="xsd:string"></span><span id='am-231' property="rdfs:label" content="Hh-Np" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasExactSynonym" content="l(3)hh" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasExactSynonym" content="Protein hedgehog N-product" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasExactSynonym" content="Moonrat" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasExactSynonym" content="anon-WO0182946.19" datatype="xsd:string"></span><span id='am-236' property="rdfs:label" content="l(3)hh" datatype="xsd:string"></span><span id='am-237' property="rdfs:label" content="bar" datatype="xsd:string"></span><span id='am-238' property="rdfs:label" content="l(3)neo56" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasExactSynonym" content="bar-on-3" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:id" content="IMR:0100740" datatype="xsd:string"></span><span id='am-241' property="rdfs:label" content="HH_DROME" datatype="xsd:string"></span><span id='am-242' property="rdfs:label" content="Mir" datatype="xsd:string"></span><span id='am-243' property="rdfs:label" content="Protein hedgehog N-product" datatype="xsd:string"></span>bar</span> denotes reaction                                     189;<span id='am-334' about='obo:IMR_0703320' typeof='owl:Thing'><span id='am-335' property="rdfs:label" content="Arg1" datatype="xsd:string"></span><span id='am-336' property="rdfs:label" content="Liver-type arginase" datatype="xsd:string"></span><span id='am-337' property="rdfs:label" content="PGIF" datatype="xsd:string"></span><span id='am-338' property="oboInOwl:hasExactSynonym" content="EC 3.5.3.1" datatype="xsd:string"></span><span id='am-339' property="rdfs:label" content="EC 3.5.3.1" datatype="xsd:string"></span><span id='am-340' property="rdfs:label" content="Arg-1" datatype="xsd:string"></span><span id='am-341' property="oboInOwl:hasExactSynonym" content="PGIF" datatype="xsd:string"></span><span id='am-342' property="rdfs:label" content="Type I arginase" datatype="xsd:string"></span><span id='am-343' property="oboInOwl:hasExactSynonym" content="Liver-type arginase" datatype="xsd:string"></span><span id='am-344' property="rdfs:label" content="AI" datatype="xsd:string"></span><span id='am-345' property="oboInOwl:hasExactSynonym" content="Arg1" datatype="xsd:string"></span><span id='am-346' property="rdfs:label" content="Arginase-1" datatype="xsd:string"></span><span id='am-347' property="rdfs:label" content="ARGI1_MOUSE" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasExactSynonym" content="arginase, liver" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasExactSynonym" content="Arginase-1" datatype="xsd:string"></span><span id='am-350' property="oboInOwl:hasDbXref" content="UniProt:Q61176" datatype="xsd:string"></span><span id='am-351' property="oboInOwl:id" content="IMR:0703320" datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasExactSynonym" content="Arg-1" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasExactSynonym" content="Type I arginase" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-355' property="rdfs:label" content="arginase, liver" datatype="xsd:string"></span><span id='am-356' property="oboInOwl:hasExactSynonym" content="AI" datatype="xsd:string"></span>Ai</span>    188;    Sij mj  188; 0                   240;1 222; <br />                                                                                                              dt          j <br /> member core (green), vertical color bar denotes enzymes catalyzing reaction essential (red).                              Sij stoichiometric coef 65533;?cient metabolite Ai reaction <br /> (B) Distribution Pearson correlation mRNA copy numbers 41                  j, mj  64258;ux reaction j. use convention experiments [14]. correlations core reactions clearly shifted          metabolite Ai substrate  product) reaction j, Sij , 0 (Sij . 0). higher values, average correlation coefficient ,C.  188;               vector  64258;uxes fmjg satis 65533;?es equation 1 corresponds 0.23 compared average noncore coefficient ,C.  188; 0.07                      state metabolic network,  potential state DOI: 10.1371/journal.pcbi.0010068.g004                                                operation cell. Using linear programming, choose                                                                                       solution maximizes biomass production respective <br />                                                                                       organisms. Reactions active, likely re 64258;ecting annota- <br />                                                                                       tion errors incomplete data, ignored  64258;ux analysis. <br /> ing size metabolic core.  metabolic                             Metabolic core identi 65533;?cation. model possible cellular <br /> core contains reactions necessary optimal cellular                       environments investigated FBA models, described performance regardless environmental conditions,                               [3]. Brie 64258;y, metabolic-uptake reaction,  65533;?x uptake rate <br />                                                                                       randomly chosen value 0 20 mmol/g/h (dry conditionally active metabolic reactions represent                          weight) calculating optimal  64258;uxes con 65533;?guration different ways cell capable optimally                          using linear programming. reaction subsequently tagged utilizing substrates environment.                                            active (non-zero  64258;ux) inactive (zero  64258;ux).                                                                                       large number possible combinations selected uptake <br />   identi 65533;?cation metabolic core important                          rates, repeat process 30,000 times. addition, calculate <br /> practical implications: given continuous activity high                        optimal  64258;uxes single-carbon-source con 65533;?gurations                                                                                       minimal-uptake medium consisting unlimited ammonia, sulfate, <br /> degree essentiality core enzymes, represent                            phosphate, carbon dioxide, potassium, restricted oxygen E. <br /> potential targets antimicrobial intervention. Speci 65533;?cally,                        coli, H. pylori, S. cerevisiae, plus unlimited ergosterol zymosterol <br />  <br />        PLoS Computational Biology | www.ploscompbiol.org                       0562                                  December 2005 | Volume 1 | Issue 7 | e68 <br />                                                                                                                                               Metabolic Network Core <br />  S. cerevisiae. reaction, assign value, qi, zero             database (http://www.shigen.nig.ac.jp/ecoli/pec) given Table <span id='am-606' about='obo:IMR_0702943' typeof='owl:Thing'><span id='am-607' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 2" datatype="xsd:string"></span><span id='am-608' property="rdfs:label" content="P97" datatype="xsd:string"></span><span id='am-609' property="oboInOwl:hasExactSynonym" content="S2" datatype="xsd:string"></span><span id='am-610' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 2" datatype="xsd:string"></span><span id='am-611' property="rdfs:label" content="PROTEASOME 26S SUBUNIT, NON-ATPASE, 2" datatype="xsd:string"></span><span id='am-612' property="oboInOwl:hasExactSynonym" content="P97" datatype="xsd:string"></span><span id='am-613' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S SUBUNIT, NON-ATPASE, 2" datatype="xsd:string"></span><span id='am-614' property="oboInOwl:hasExactSynonym" content="TRAP2" datatype="xsd:string"></span><span id='am-615' property="rdfs:label" content="TRAP2" datatype="xsd:string"></span><span id='am-616' property="oboInOwl:hasExactSynonym" content="TNFR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-617' property="rdfs:label" content="TUMOR NECROSIS FACTOR RECEPTOR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-618' property="rdfs:label" content="26S proteasome regulatory subunit S2" datatype="xsd:string"></span><span id='am-619' property="oboInOwl:hasDbXref" content="UniProt:Q13200" datatype="xsd:string"></span><span id='am-620' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S2" datatype="xsd:string"></span><span id='am-621' property="oboInOwl:hasExactSynonym" content="MGC14274" datatype="xsd:string"></span><span id='am-622' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor type 1 receptor-associated protein 2" datatype="xsd:string"></span><span id='am-623' property="oboInOwl:hasExactSynonym" content="Rpn1" datatype="xsd:string"></span><span id='am-624' property="rdfs:label" content="PROTEASOME 26S, SUBUNIT 2" datatype="xsd:string"></span><span id='am-625' property="oboInOwl:hasExactSynonym" content="Protein 55.11" datatype="xsd:string"></span><span id='am-626' property="rdfs:label" content="MGC14274" datatype="xsd:string"></span><span id='am-627' property="oboInOwl:hasExactSynonym" content="TUMOR NECROSIS FACTOR RECEPTOR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-628' property="oboInOwl:hasExactSynonym" content="26S proteasome subunit p97" datatype="xsd:string"></span><span id='am-629' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S, SUBUNIT 2" datatype="xsd:string"></span><span id='am-630' property="rdfs:label" content="26S proteasome regulatory subunit RPN1" datatype="xsd:string"></span><span id='am-631' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN1" datatype="xsd:string"></span><span id='am-632' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 2" datatype="xsd:string"></span><span id='am-633' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 2" datatype="xsd:string"></span><span id='am-634' property="rdfs:label" content="Rpn1" datatype="xsd:string"></span><span id='am-635' property="oboInOwl:hasExactSynonym" content="PSMD2" datatype="xsd:string"></span><span id='am-636' property="rdfs:label" content="26S proteasome subunit p97" datatype="xsd:string"></span><span id='am-637' property="oboInOwl:id" content="IMR:0702943" datatype="xsd:string"></span><span id='am-638' property="rdfs:label" content="Tumor necrosis factor type 1 receptor-associated protein 2" datatype="xsd:string"></span><span id='am-639' property="rdfs:label" content="Protein 55.11" datatype="xsd:string"></span><span id='am-640' property="rdfs:label" content="PSMD2_HUMAN" datatype="xsd:string"></span><span id='am-641' property="rdfs:label" content="S2" datatype="xsd:string"></span><span id='am-642' property="rdfs:label" content="TNFR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-643' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-644' property="rdfs:label" content="PSMD2" datatype="xsd:string"></span>S2</span> unity, representing relative number conditions                      Gerdes et al. [13], resulting total 62 essential enzymes reaction active. preliminary metabolic core de 65533;?ned                    seven enzymes determine deletion <br /> set reactions active sampled conditions (qi  188; 1).                  phenotype. deletion phenotype yeast enzymes taken <br /> determine  65533;?nal metabolic core removing preliminary                       <span id='am-534' about='obo:IMR_0200844' typeof='owl:Thing'><span id='am-535' property="oboInOwl:hasExactSynonym" content="Retinal guanylyl cyclase 1" datatype="xsd:string"></span><span id='am-536' property="oboInOwl:hasDbXref" content="UniProt:Q02846" datatype="xsd:string"></span><span id='am-537' property="rdfs:label" content="Rod outer segment membrane guanylate cyclase" datatype="xsd:string"></span><span id='am-538' property="rdfs:label" content="ROS-GC" datatype="xsd:string"></span><span id='am-539' property="rdfs:label" content="retGC" datatype="xsd:string"></span><span id='am-540' property="oboInOwl:hasExactSynonym" content="GUANYLATE CYCLASE 2D, MEMBRANE" datatype="xsd:string"></span><span id='am-541' property="rdfs:label" content="GUC2D_HUMAN" datatype="xsd:string"></span><span id='am-542' property="rdfs:label" content="guanylate cyclase 2D, membrane (retina-specific)" datatype="xsd:string"></span><span id='am-543' property="oboInOwl:id" content="IMR:0200844" datatype="xsd:string"></span><span id='am-544' property="rdfs:label" content="LCA1" datatype="xsd:string"></span><span id='am-545' property="rdfs:label" content="GUC1A4" datatype="xsd:string"></span><span id='am-546' property="rdfs:label" content="ROS-GC1" datatype="xsd:string"></span><span id='am-547' property="oboInOwl:hasExactSynonym" content="Guanylate cyclase 2D, retinal" datatype="xsd:string"></span><span id='am-548' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-549' property="rdfs:label" content="RETGC" datatype="xsd:string"></span><span id='am-550' property="oboInOwl:hasExactSynonym" content="GUCY2D" datatype="xsd:string"></span><span id='am-551' property="oboInOwl:hasExactSynonym" content="GUC2D" datatype="xsd:string"></span><span id='am-552' property="rdfs:label" content="RETGC1" datatype="xsd:string"></span><span id='am-553' property="oboInOwl:hasExactSynonym" content="RETGC-1" datatype="xsd:string"></span><span id='am-554' property="oboInOwl:hasExactSynonym" content="LCA1" datatype="xsd:string"></span><span id='am-555' property="rdfs:label" content="CYGD" datatype="xsd:string"></span><span id='am-556' property="rdfs:label" content="ROSGC" datatype="xsd:string"></span><span id='am-557' property="oboInOwl:hasExactSynonym" content="guanylate cyclase 2D, membrane (retina-specific)" datatype="xsd:string"></span><span id='am-558' property="rdfs:label" content="GUCY2D" datatype="xsd:string"></span><span id='am-559' property="rdfs:label" content="EC 4.6.1.2" datatype="xsd:string"></span><span id='am-560' property="rdfs:label" content="Retinal guanylyl cyclase 1" datatype="xsd:string"></span><span id='am-561' property="oboInOwl:hasExactSynonym" content="GUC1A4" datatype="xsd:string"></span><span id='am-562' property="oboInOwl:hasExactSynonym" content="LCA" datatype="xsd:string"></span><span id='am-563' property="oboInOwl:hasExactSynonym" content="Rod outer segment membrane guanylate cyclase" datatype="xsd:string"></span><span id='am-564' property="rdfs:label" content="GUANYLATE CYCLASE 2D, MEMBRANE" datatype="xsd:string"></span><span id='am-565' property="rdfs:label" content="LCA" datatype="xsd:string"></span><span id='am-566' property="oboInOwl:hasExactSynonym" content="RETGC" datatype="xsd:string"></span><span id='am-567' property="oboInOwl:hasExactSynonym" content="CORD5" datatype="xsd:string"></span><span id='am-568' property="oboInOwl:hasExactSynonym" content="EC 4.6.1.2" datatype="xsd:string"></span><span id='am-569' property="oboInOwl:hasExactSynonym" content="RCD2" datatype="xsd:string"></span><span id='am-570' property="oboInOwl:hasExactSynonym" content="RETGC1" datatype="xsd:string"></span><span id='am-571' property="rdfs:label" content="GUCY2E, MOUSE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-572' property="oboInOwl:hasExactSynonym" content="ROSGC" datatype="xsd:string"></span><span id='am-573' property="oboInOwl:hasExactSynonym" content="ROS-GC" datatype="xsd:string"></span><span id='am-574' property="rdfs:label" content="RCD2" datatype="xsd:string"></span><span id='am-575' property="rdfs:label" content="CORD5" datatype="xsd:string"></span><span id='am-576' property="oboInOwl:hasExactSynonym" content="CORD6" datatype="xsd:string"></span><span id='am-577' property="rdfs:label" content="GUC2D" datatype="xsd:string"></span><span id='am-578' property="oboInOwl:hasExactSynonym" content="retGC" datatype="xsd:string"></span><span id='am-579' property="rdfs:label" content="Guanylate cyclase 2D, retinal" datatype="xsd:string"></span><span id='am-580' property="oboInOwl:hasExactSynonym" content="CYGD" datatype="xsd:string"></span><span id='am-581' property="rdfs:label" content="RETGC-1" datatype="xsd:string"></span><span id='am-582' property="oboInOwl:hasExactSynonym" content="ROS-GC1" datatype="xsd:string"></span><span id='am-583' property="oboInOwl:hasExactSynonym" content="GUCY2E, MOUSE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-584' property="rdfs:label" content="CORD6" datatype="xsd:string"></span>CYGD</span> database [12]. <br /> core reactions   64258;ux rate constrained zero total set initial conditions, leave growth rate <br /> unchanged.                                                                               Supporting Information <br />    Correlations clustering. evaluate FBA  64258;ux correlations calculating Pearson coef 65533;?cient possible reaction pair                     Protocol S1. Supplementary Analyses List Metabolite <br /> combinations 30,000 different simulated growth conditions.                        Abbreviations subsequently group reactions employing correlation <br /> values metric standard hierarchical average-linkage                          DOI: 10.1371/journal.pcbi.0010068.sd001 (628 KB DOC). <br /> clustering algorithm [22].                                                               Table S1. Metabolic Core Reactions <br />    Data analysis. annotated metabolic FBA models specify enzymes genes catalyzing various metabolic reactions.                         DOI: 10.1371/journal.pcbi.0010068.st001 (57 KB XLS). <br /> represent mRNA activity level half-life time metabolic <br /> enzyme averaging available experimental values, Ek, catalyzing gene products,                                                             Acknowledgments <br />                                     1 X <br />                                <span id='am-1' about='obo:IMR_0001390' typeof='owl:Thing'><span id='am-2' property="oboInOwl:hasExactSynonym" content="DRI" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="PKC" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="PKA" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="PKA-RI" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="PKAr" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="PKA-RI" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="dPKA-RI" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="PKC" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="18304" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="cAMP-dependent protein kinase R1" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="pkA" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="CG18677" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="cAMP-dependent protein kinase R1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="pka-R1" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="cAMP-dependent protein kinase type I regulatory subunit" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="PKAr" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="dPKA-RI" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="dr[[1]]" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="DRI class I to class IV" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="cAMP-dependent protein kinase type I regulatory subunit" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Pka-RI" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Pka-R1" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="CdkR" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="CG3263" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="RI" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="PKA" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="DRI" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="DRI class I to class IV" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="Pka-RI" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="CG18677" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="CG3263" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="UniProt:P16905" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="KAPR1_DROME" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="Pka-R1" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="pka-R1" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="CdkR" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="RI" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0001390" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="dr[[1]]" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="pkA" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="18304" datatype="xsd:string"></span>ri</span>  188;           Ek                          240;2 222;             thank S. Mobashery S. Vakulenko discussions. Work                                     Ni k 188;1;Ni                                            University Notre Dame University Pittsburgh                                                                                          supported Department Energy, National Institutes Ni number genes. assess essentiality                         Health, National Science Foundation. <br /> reaction considering essential catalyzing <br /> enzymes  gene products) essential survival                             Competing interests. authors declared competing <br /> organism  Protocol S1 error analysis).                                           interests exist. <br />   Core noncore essentiality. deletion phenotype E. coli                          Author contributions. EA, ZNO, ALB conceived study. EA <br /> enzymes taken Gerdes et al. [13]. nonlethal core                            ALB designed study. EA performed computational <br /> reactions, cross-checked deletion phenotype PEC                          work. EA, ZNO ALB wrote paper.                             <span id='am-149' about='obo:IMR_0702349' typeof='owl:Thing'><span id='am-150' property="oboInOwl:hasDbXref" content="UniProt:P07741" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasExactSynonym" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasExactSynonym" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasExactSynonym" content="AMP" datatype="xsd:string"></span><span id='am-155' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-156' property="rdfs:label" content="APT_HUMAN" datatype="xsd:string"></span><span id='am-157' property="rdfs:label" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-158' property="rdfs:label" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasExactSynonym" content="MGC125857" datatype="xsd:string"></span><span id='am-160' property="rdfs:label" content="MGC125856" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasExactSynonym" content="APRT" datatype="xsd:string"></span><span id='am-162' property="rdfs:label" content="MGC125857" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:id" content="IMR:0702349" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasExactSynonym" content="DKFZp686D13177" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasExactSynonym" content="MGC125856" datatype="xsd:string"></span><span id='am-166' property="rdfs:label" content="MGC129961" datatype="xsd:string"></span><span id='am-167' property="oboInOwl:hasExactSynonym" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-168' property="rdfs:label" content="APRT" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasExactSynonym" content="MGC129961" datatype="xsd:string"></span><span id='am-170' property="rdfs:label" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-171' property="rdfs:label" content="DKFZp686D13177" datatype="xsd:string"></span>amp</span>; <br />  <br />  <br />  <br />  <br /> References                                                                               12. Gu 168;ldener U, Mu 168;nsterko 168;tter M, Kastenmu 168;ller G, Strack N, van Helden J, et <br /> 1. Edwards JS, Palsson BO (2000) Escherichia coli MG1655 silico                       al. (2005) CYGD: Comprehensive Yeast Genome Database. Nucleic <br />     metabolic genotype: de 65533;?nition, characteristics, capabilities. <span id='am-419' about='obo:IMR_0010425' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010045 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0000952'><span id='am-420' property="rdfs:label" content="Pyrroline-5-carboxylate reductase" datatype="xsd:string"></span><span id='am-421' property="rdfs:label" content="P5CR" datatype="xsd:string"></span><span id='am-422' property="rdfs:label" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-423' property="oboInOwl:hasExactSynonym" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-424' property="oboInOwl:hasExactSynonym" content="P5C reductase" datatype="xsd:string"></span><span id='am-425' property="oboInOwl:hasExactSynonym" content="PROC" datatype="xsd:string"></span><span id='am-426' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-427' property="rdfs:label" content="PROC" datatype="xsd:string"></span><span id='am-428' property="oboInOwl:id" content="IMR:0010425" datatype="xsd:string"></span><span id='am-429' property="rdfs:label" content="P5C reductase" datatype="xsd:string"></span><span id='am-430' property="oboInOwl:hasExactSynonym" content="P5CR" datatype="xsd:string"></span><span id='am-431' property="oboInOwl:hasDbXref" content="EC:1.5.1.2" datatype="xsd:string"></span>Proc</span>               Acids Res 33: D364 8211;D368. <br />     Natl Acad Sci U S 97: 5528 8211;5533.                                                   13. Gerdes <span id='am-679' about='obo:IMR_0002908' typeof='owl:Thing'><span id='am-680' property="rdfs:label" content="sy" datatype="xsd:string"></span><span id='am-681' property="rdfs:label" content="sisterless c" datatype="xsd:string"></span><span id='am-682' property="oboInOwl:hasExactSynonym" content="sis-C" datatype="xsd:string"></span><span id='am-683' property="oboInOwl:hasExactSynonym" content="Unpaired" datatype="xsd:string"></span><span id='am-684' property="oboInOwl:hasExactSynonym" content="unpaired" datatype="xsd:string"></span><span id='am-685' property="rdfs:label" content="unpaired" datatype="xsd:string"></span><span id='am-686' property="oboInOwl:id" content="IMR:0002908" datatype="xsd:string"></span><span id='am-687' property="rdfs:label" content="bdw" datatype="xsd:string"></span><span id='am-688' property="rdfs:label" content="Unpaired protein" datatype="xsd:string"></span><span id='am-689' property="oboInOwl:hasExactSynonym" content="sisC" datatype="xsd:string"></span><span id='am-690' property="oboInOwl:hasExactSynonym" content="unp" datatype="xsd:string"></span><span id='am-691' property="rdfs:label" content="O96525_DROME" datatype="xsd:string"></span><span id='am-692' property="rdfs:label" content="sis-c" datatype="xsd:string"></span><span id='am-693' property="rdfs:label" content="Unpaired" datatype="xsd:string"></span><span id='am-694' property="oboInOwl:hasExactSynonym" content="sis-c" datatype="xsd:string"></span><span id='am-695' property="oboInOwl:hasExactSynonym" content="outstretched-smalleye" datatype="xsd:string"></span><span id='am-696' property="rdfs:label" content="UPD" datatype="xsd:string"></span><span id='am-697' property="oboInOwl:hasExactSynonym" content="upa" datatype="xsd:string"></span><span id='am-698' property="oboInOwl:hasExactSynonym" content="odsy" datatype="xsd:string"></span><span id='am-699' property="oboInOwl:hasExactSynonym" content="bdw" datatype="xsd:string"></span><span id='am-700' property="oboInOwl:hasExactSynonym" content="CG5993" datatype="xsd:string"></span><span id='am-701' property="rdfs:label" content="sis-C" datatype="xsd:string"></span><span id='am-702' property="rdfs:label" content="upa" datatype="xsd:string"></span><span id='am-703' property="rdfs:label" content="outstretched" datatype="xsd:string"></span><span id='am-704' property="oboInOwl:hasExactSynonym" content="Unpaired protein" datatype="xsd:string"></span><span id='am-705' property="oboInOwl:hasDbXref" content="UniProt:O96525" datatype="xsd:string"></span><span id='am-706' property="oboInOwl:hasExactSynonym" content="sy" datatype="xsd:string"></span><span id='am-707' property="rdfs:label" content="sisC" datatype="xsd:string"></span><span id='am-708' property="oboInOwl:hasExactSynonym" content="l(1)YC43" datatype="xsd:string"></span><span id='am-709' property="rdfs:label" content="Upd" datatype="xsd:string"></span><span id='am-710' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-711' property="rdfs:label" content="outstretched-smalleye" datatype="xsd:string"></span><span id='am-712' property="rdfs:label" content="CG5993" datatype="xsd:string"></span><span id='am-713' property="rdfs:label" content="l(1)YM55" datatype="xsd:string"></span><span id='am-714' property="oboInOwl:hasExactSynonym" content="outstretched" datatype="xsd:string"></span><span id='am-715' property="rdfs:label" content="odsy" datatype="xsd:string"></span><span id='am-716' property="oboInOwl:hasExactSynonym" content="sisterless c" datatype="xsd:string"></span><span id='am-717' property="oboInOwl:hasExactSynonym" content="UPD" datatype="xsd:string"></span><span id='am-718' property="rdfs:label" content="upd" datatype="xsd:string"></span><span id='am-719' property="rdfs:label" content="l(1)YC43" datatype="xsd:string"></span><span id='am-720' property="oboInOwl:hasExactSynonym" content="Upd" datatype="xsd:string"></span><span id='am-721' property="rdfs:label" content="small-eye" datatype="xsd:string"></span><span id='am-722' property="oboInOwl:hasExactSynonym" content="small-eye" datatype="xsd:string"></span><span id='am-723' property="rdfs:label" content="unp" datatype="xsd:string"></span><span id='am-724' property="rdfs:label" content="bending-wings" datatype="xsd:string"></span><span id='am-725' property="oboInOwl:hasExactSynonym" content="upd" datatype="xsd:string"></span><span id='am-726' property="oboInOwl:hasExactSynonym" content="l(1)YM55" datatype="xsd:string"></span><span id='am-727' property="oboInOwl:hasExactSynonym" content="bending-wings" datatype="xsd:string"></span>SY</span>, Scholle MD, Campbell JW, Balazsi G, Ravasz E, et al. (2003) <br /> 2. Schilling CH, Letscher D, Palsson BO (2000) Theory systemic                       Experimental determination level analysis essential genes     de 65533;?nition metabolic pathways use interpreting metabolic                  Escherichia coli MG1655. J Bacteriol 185: 5673 8211;5684. <br />     function pathway-oriented perspective. J Theor Biol 203: 229 8211;248.             14. Selinger DW, Saxena RM, Cheung KJ, Church GM, Rosenow C (2003) Global <br /> 3. Almaas E, Kova 180;<span id='am-50' about='obo:IMR_0704812' typeof='owl:Thing'><span id='am-51' property="oboInOwl:hasExactSynonym" content="PKA C-alpha" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="cAMP-dependent protein kinase catalytic subunit alpha" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="Pkaca" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="protein kinase, cAMP dependent, catalytic, alpha" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Prkaca" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="EC 2.7.11.11" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="C alpha" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="UniProt:P05132" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Prkaca" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="PKA C-alpha" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="C alpha" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="protein kinase, cAMP dependent, catalytic, alpha" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="PKA" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="KAPCA_MOUSE" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="Cs" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="EC 2.7.11.11" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0704812" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Pkaca" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="PKA" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Cs" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="cAMP-dependent protein kinase catalytic subunit alpha" datatype="xsd:string"></span>cs</span> B, Vicsek T, Oltvai ZN, Baraba 180;si AL (2004) Global                      RNA half-life analysis Escherichia coli reveals positional patterns     organization metabolic  64258;uxes bacterium Escherichia coli. Nature                transcript degradation. Genome Res 13: 216 8211;223. <br />     427: 839 8211;843.                                                                        15. Shen-Orr S, Milo R, Mangan S, Alon U (2002) Network motifs 4. Burgard AP, Nikolaev EV, Schilling CH, Maranas <span id='am-172' about='obo:IMR_0701943' typeof='owl:Thing'><span id='am-173' property="rdfs:label" content="Cd" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasExactSynonym" content="Cd" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasDbXref" content="UniProt:O88572" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasExactSynonym" content="Low-density lipoprotein receptor-related protein 6" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:id" content="IMR:0701943" datatype="xsd:string"></span><span id='am-179' property="rdfs:label" content="low density lipoprotein receptor-related protein 6" datatype="xsd:string"></span><span id='am-180' property="rdfs:label" content="Low-density lipoprotein receptor-related protein 6" datatype="xsd:string"></span><span id='am-181' property="rdfs:label" content="Lrp6" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasExactSynonym" content="LRP-6" datatype="xsd:string"></span><span id='am-183' property="rdfs:label" content="LRP-6" datatype="xsd:string"></span><span id='am-184' property="rdfs:label" content="LRP6_MOUSE" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasExactSynonym" content="low density lipoprotein receptor-related protein 6" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasExactSynonym" content="Lrp6" datatype="xsd:string"></span>CD</span> (2004) Flux coupling                    transcriptional regulation network Escherichia coli. <span id='am-73' about='obo:IMR_0701753' typeof='owl:Thing'><span id='am-74' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="NaT" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasExactSynonym" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="Sns2" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="NaN" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="Scn11a" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="SNS2" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="Nat" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="NaN" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="SNS2" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="SCNBA_MOUSE" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="NSS2" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Sns2" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="Nan" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="NaT" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="Nan" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="UniProt:Q9R053" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Scn11a" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="NSS2" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:id" content="IMR:0701753" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="Nat" datatype="xsd:string"></span>Nat</span> Genet 31: 64 8211;68. <br />     analysis genome-scale metabolic reconstructions. Genome Res 14: 301 8211;              16. Salgado H, Gama-Castro S, Martinez-Antonio  Diaz-Peredo E, Sanchez- <br />     312.                                                                                     Solano F, et al. (2004) RegulonDB (version 4.0): Transcriptional regulation, <br /> 5. Reed JL, Palsson BO (2004) Genome-scale silico models E. coli                  operon organization growth conditions Escherichia coli K-12. Nucleic <br />     multiple equivalent phenotypic states: Assessment correlated reaction                 Acids Res 32: D303 8211;D306. <br />     subsets comprise network states. Genome Res 14: 1797 8211;1805.                      17. Wiback SJ, Famili  Greenberg HJ, Palsson BO (2004) Monte Carlo <br /> 6. Papp B, Pal C, Hurst LD (2004) Metabolic network analysis causes <br />                                                                                              sampling used determine size shape steady-state <br />     evolution enzyme dispensability yeast. Nature 429: 661 8211;664. <br />                                                                                               64258;ux space. J Theor Biol 228: 437 8211;447. <br /> 7. Schilling CH, Covert MW, Famili  Church GM, Edwards JS, et al. (2002) <br />                                                                                          18. Eisen <span id='am-501' about='obo:IMR_0002532' typeof='owl:Thing'><span id='am-502' property="rdfs:label" content="H2-Mb1" datatype="xsd:string"></span><span id='am-503' property="oboInOwl:hasExactSynonym" content="H-2Mb1" datatype="xsd:string"></span><span id='am-504' property="rdfs:label" content="Mb" datatype="xsd:string"></span><span id='am-505' property="oboInOwl:hasExactSynonym" content="H2-DMb1" datatype="xsd:string"></span><span id='am-506' property="oboInOwl:hasExactSynonym" content="H2-M beta1" datatype="xsd:string"></span><span id='am-507' property="rdfs:label" content="DMB_MOUSE" datatype="xsd:string"></span><span id='am-508' property="oboInOwl:hasExactSynonym" content="histocompatibility 2, class II, locus Mb1" datatype="xsd:string"></span><span id='am-509' property="rdfs:label" content="H2-M beta 1 chain" datatype="xsd:string"></span><span id='am-510' property="oboInOwl:hasDbXref" content="UniProt:P35737" datatype="xsd:string"></span><span id='am-511' property="oboInOwl:hasExactSynonym" content="Mb" datatype="xsd:string"></span><span id='am-512' property="rdfs:label" content="H2-M beta1" datatype="xsd:string"></span><span id='am-513' property="rdfs:label" content="H-2Mb1" datatype="xsd:string"></span><span id='am-514' property="oboInOwl:id" content="IMR:0002532" datatype="xsd:string"></span><span id='am-515' property="rdfs:label" content="histocompatibility 2, class II, locus Mb1" datatype="xsd:string"></span><span id='am-516' property="oboInOwl:hasExactSynonym" content="H2-Mb1" datatype="xsd:string"></span><span id='am-517' property="oboInOwl:hasExactSynonym" content="Class II histocompatibility antigen, M beta 1 chain" datatype="xsd:string"></span><span id='am-518' property="oboInOwl:hasExactSynonym" content="H2-M beta 1 chain" datatype="xsd:string"></span><span id='am-519' property="rdfs:label" content="H2-DMb1" datatype="xsd:string"></span><span id='am-520' property="rdfs:label" content="Class II histocompatibility antigen, M beta 1 chain" datatype="xsd:string"></span><span id='am-521' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>MB</span>, Spellman PT, Brown PO, Botstein D (1998) Cluster analysis     Genome-scale metabolic model Helicobacter pylori 26695. J Bacteriol 184: <br />                                                                                              display genome-wide expression patterns. Proc Natl Acad Sci U S 95: <br />     4582 8211;4593. <br /> 8. Reed JL, Vo TD, Schilling CH, Palsson BO (2003) expanded genome-                       14863 8211;14868. <br />     scale model Escherichia coli K-12 (iJR904 GSM/GPR). Genome Biol 4:                19. Allen TE, Herrgard MJ, Liu MZ, Qiu Y, Glasner JD, et al. (2003) Genome- <br />     R54.1 8211;R54.12.                                                                            scale analysis uses Escherichia coli genome: Model-driven <br /> 9. Duarte NC, Herrgard MJ, Palsson BO (2004) Reconstruction validation                   analysis heterogeneous data sets. J Bacteriol 185: 6392 8211;6399. <br />     Saccharomyces cerevisiae iND750, fully compartmentalized genome-scale           20. Price ND, Reed JL, Palsson BO (2004) Genome-scale models microbial <br />     metabolic model. Genome Res 14: 1298 8211;1309.                                               cells: Evaluating consequences constraints. <span id='am-104' about='obo:IMR_0701753' typeof='owl:Thing'><span id='am-105' property="oboInOwl:hasDbXref" content="UniProt:Q9R053" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasExactSynonym" content="NaN" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasExactSynonym" content="Scn11a" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Nat" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="NSS2" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="NaT" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Nan" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="SCNBA_MOUSE" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasExactSynonym" content="Nat" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:id" content="IMR:0701753" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="Sns2" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasExactSynonym" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="NaT" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasExactSynonym" content="Nan" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="NaN" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasExactSynonym" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasExactSynonym" content="NSS2" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasExactSynonym" content="SNS2" datatype="xsd:string"></span><span id='am-129' property="rdfs:label" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasExactSynonym" content="Sns2" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasExactSynonym" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="SNS2" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="Scn11a" datatype="xsd:string"></span>Nat</span> Rev Microbiol 2: 886 8211; <br /> 10. Burgard AP, Maranas <span id='am-455' about='obo:IMR_0002520' typeof='owl:Thing'><span id='am-456' property="oboInOwl:hasExactSynonym" content="FLJ27328" datatype="xsd:string"></span><span id='am-457' property="oboInOwl:hasAlternativeId" content="IMR:0002522" datatype="xsd:string"></span><span id='am-458' property="rdfs:label" content="HLADC HISTOCOMPATIBILITY TYPE" datatype="xsd:string"></span><span id='am-459' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-460' property="oboInOwl:hasExactSynonym" content="FLJ27088" datatype="xsd:string"></span><span id='am-461' property="oboInOwl:hasExactSynonym" content="HLA-DCA" datatype="xsd:string"></span><span id='am-462' property="oboInOwl:hasExactSynonym" content="DQ-A1" datatype="xsd:string"></span><span id='am-463' property="oboInOwl:hasAlternativeId" content="IMR:0002523" datatype="xsd:string"></span><span id='am-464' property="oboInOwl:hasExactSynonym" content="HLADC HISTOCOMPATIBILITY TYPE" datatype="xsd:string"></span><span id='am-465' property="oboInOwl:hasExactSynonym" content="HLA-DQA" datatype="xsd:string"></span><span id='am-466' property="oboInOwl:hasExactSynonym" content="HLA-DQ" datatype="xsd:string"></span><span id='am-467' property="rdfs:label" content="CD" datatype="xsd:string"></span><span id='am-468' property="oboInOwl:hasExactSynonym" content="DC-alpha" datatype="xsd:string"></span><span id='am-469' property="rdfs:label" content="DC-1 alpha chain" datatype="xsd:string"></span><span id='am-470' property="rdfs:label" content="MHC class II DQA1" datatype="xsd:string"></span><span id='am-471' property="rdfs:label" content="CELIAC1" datatype="xsd:string"></span><span id='am-472' property="rdfs:label" content="DQ-A1" datatype="xsd:string"></span><span id='am-473' property="oboInOwl:id" content="IMR:0002520" datatype="xsd:string"></span><span id='am-474' property="oboInOwl:hasAlternativeId" content="IMR:0002525" datatype="xsd:string"></span><span id='am-475' property="oboInOwl:hasExactSynonym" content="major histocompatibility complex, class II, DQ alpha 1" datatype="xsd:string"></span><span id='am-476' property="rdfs:label" content="MGC149527" datatype="xsd:string"></span><span id='am-477' property="rdfs:label" content="GSE" datatype="xsd:string"></span><span id='am-478' property="oboInOwl:hasExactSynonym" content="MHC class II DQA1" datatype="xsd:string"></span><span id='am-479' property="rdfs:label" content="HLA-DQA" datatype="xsd:string"></span><span id='am-480' property="rdfs:label" content="HLA class II histocompatibility antigen, DQ alpha 1 chain" datatype="xsd:string"></span><span id='am-481' property="rdfs:label" content="DQA1_HUMAN" datatype="xsd:string"></span><span id='am-482' property="oboInOwl:hasExactSynonym" content="CELIAC1" datatype="xsd:string"></span><span id='am-483' property="rdfs:label" content="DC-alpha" datatype="xsd:string"></span><span id='am-484' property="oboInOwl:hasAlternativeId" content="IMR:0002521" datatype="xsd:string"></span><span id='am-485' property="oboInOwl:hasAlternativeId" content="IMR:0002519" datatype="xsd:string"></span><span id='am-486' property="rdfs:label" content="major histocompatibility complex, class II, DQ alpha 1" datatype="xsd:string"></span><span id='am-487' property="oboInOwl:hasExactSynonym" content="MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DQ ALPHA-1" datatype="xsd:string"></span><span id='am-488' property="oboInOwl:hasDbXref" content="UniProt:P01909" datatype="xsd:string"></span><span id='am-489' property="rdfs:label" content="HLA-DQ" datatype="xsd:string"></span><span id='am-490' property="oboInOwl:hasExactSynonym" content="HLA class II histocompatibility antigen, DQ alpha 1 chain" datatype="xsd:string"></span><span id='am-491' property="rdfs:label" content="FLJ27088" datatype="xsd:string"></span><span id='am-492' property="rdfs:label" content="FLJ27328" datatype="xsd:string"></span><span id='am-493' property="oboInOwl:hasExactSynonym" content="MGC149527" datatype="xsd:string"></span><span id='am-494' property="oboInOwl:hasExactSynonym" content="HLA-DQA1" datatype="xsd:string"></span><span id='am-495' property="oboInOwl:hasExactSynonym" content="CD" datatype="xsd:string"></span><span id='am-496' property="rdfs:label" content="MAJOR HISTOCOMPATIBILITY COMPLEX, CLASS II, DQ ALPHA-1" datatype="xsd:string"></span><span id='am-497' property="oboInOwl:hasExactSynonym" content="GSE" datatype="xsd:string"></span><span id='am-498' property="rdfs:label" content="HLA-DQA1" datatype="xsd:string"></span><span id='am-499' property="rdfs:label" content="HLA-DCA" datatype="xsd:string"></span><span id='am-500' property="oboInOwl:hasExactSynonym" content="DC-1 alpha chain" datatype="xsd:string"></span>CD</span> (2001) Probing performance limits                      897. <br />     Escherichia coli metabolic network subject gene additions deletions.           21. Gerdes SY, Scholle MD, D 8217;Souza M, Bernal  Baev MV, et al. (2002)     Biotechnol Bioeng 74: 364 8211;375.                                                           genetic footprinting antimicrobial drug targets: Examples cofactor <br /> 11. Burgard AP, Vaidyraman S, Maranas CD (2001) Minimal reaction sets                    biosynthetic pathways. J Bacteriol 184: 4555 8211;4572. <br />     Escherichia coli metabolism different growth requirements uptake           22. Dunn G, Everitt BS (1982) introduction mathematical taxonomy. <br />     environments. Biotechnol Prog 17: 791 8211;797.                                               Cambridge: Cambridge University Press. 152 p. <br />  <br />  <br />  <br />  <br />        PLoS Computational Bio <br /> </body></html>