<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Oct. 1996, p. 5712 8211;5718                                                                            Vol. 178,  19 <br /> 0021-9193/96/$04.0010 <br /> Copyright q 1996, American Society Microbiology <br />  <br />  <br />  <br />        Molecular Cloning Expression spsB Gene Encoding <br />       Essential Type Signal <span id='am-41' about='obo:IMR_0000304' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010044 obo:IMR_0000001 obo:IMR_0000207'><span id='am-42' property="oboInOwl:hasDbXref" content="GO:0008233" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="EC:3.4.-.-" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="peptidase" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="protease" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0000304" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="peptidase" datatype="xsd:string"></span>Peptidase</span> Staphylococcus aureus <br />                           KAREN M. CREGG, E. IMOGEN WILDING,                          MICHAEL T. BLACK* <br />                  Biotechnology, SmithKline Beecham Pharmaceuticals, Epsom, Surrey KT18 5XQ, United Kingdom <br />                                               Received 6 June 1996/Accepted 29 July 1996 <br />  <br />            gene, spsB, encoding type signal peptidase cloned gram-positive eubacterium <br />          Staphylococcus aureus. gene encodes <span id='am-952' about='obo:IMR_0000949' typeof='owl:Thing obo:IMR_0000001'><span id='am-953' property="rdfs:label" content="protein" datatype="xsd:string"></span><span id='am-954' property="oboInOwl:id" content="IMR:0000949" datatype="xsd:string"></span><span id='am-955' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>protein</span> 191 amino acid residues calculated molecular mass <br />          21,692 Da. Comparison protein sequence known type signal peptidases indicates <br />          conservation amino acid residues known important essential catalytic activity. <span id='am-107' about='obo:IMR_0000207' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001'><span id='am-108' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="enzyme" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="GO:0003824" datatype="xsd:string"></span><span id='am-111' property="obo:IAO_0000115" content="Protein that catalyze a specific chemical reaction." datatype="xsd:string"></span><span id='am-112' property="oboInOwl:id" content="IMR:0000207" datatype="xsd:string"></span>enzyme</span>          expressed high levels Escherichia coli demonstrated possess enzymatic activity <br />          E. coli preproteins vivo. Experiments spsB gene transferred plasmid          temperature sensitive replication indicate spsB essential gene. identified open reading <br />          frame immediately upstream spsB gene encodes type signal peptidase homolog 174 amino <br />          acid residues calculated molecular mass 20,146 Da predicted devoid catalytic activity. <br />  <br />  <br />    majority proteins translocated             lepB single-copy gene E. coli chromosome.  membranes site synthesis initially <span id='am-887' about='obo:IMR_0704059' typeof='owl:Thing'><span id='am-888' property="rdfs:label" content="Proto-oncogene c-Fyn" datatype="xsd:string"></span><span id='am-889' property="oboInOwl:hasExactSynonym" content="p59-Fyn" datatype="xsd:string"></span><span id='am-890' property="rdfs:label" content="SLK" datatype="xsd:string"></span><span id='am-891' property="oboInOwl:hasExactSynonym" content="EC 2.7.10.2" datatype="xsd:string"></span><span id='am-892' property="oboInOwl:hasExactSynonym" content="SYN" datatype="xsd:string"></span><span id='am-893' property="oboInOwl:id" content="IMR:0704059" datatype="xsd:string"></span><span id='am-894' property="rdfs:label" content="MGC45350" datatype="xsd:string"></span><span id='am-895' property="oboInOwl:hasDbXref" content="UniProt:P06241" datatype="xsd:string"></span><span id='am-896' property="rdfs:label" content="Proto-oncogene Syn" datatype="xsd:string"></span><span id='am-897' property="rdfs:label" content="Src-like kinase" datatype="xsd:string"></span><span id='am-898' property="oboInOwl:hasExactSynonym" content="FYN" datatype="xsd:string"></span><span id='am-899' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-900' property="oboInOwl:hasExactSynonym" content="FYN oncogene related to SRC, FGR, YES" datatype="xsd:string"></span><span id='am-901' property="oboInOwl:hasExactSynonym" content="FYN TYROSINE KINASE PROTOONCOGENE" datatype="xsd:string"></span><span id='am-902' property="rdfs:label" content="FYN oncogene related to SRC, FGR, YES" datatype="xsd:string"></span><span id='am-903' property="oboInOwl:hasExactSynonym" content="Proto-oncogene c-Fyn" datatype="xsd:string"></span><span id='am-904' property="oboInOwl:hasExactSynonym" content="MGC45350" datatype="xsd:string"></span><span id='am-905' property="oboInOwl:hasExactSynonym" content="Proto-oncogene Syn" datatype="xsd:string"></span><span id='am-906' property="rdfs:label" content="FYN TYROSINE KINASE PROTOONCOGENE" datatype="xsd:string"></span><span id='am-907' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein kinase Fyn" datatype="xsd:string"></span><span id='am-908' property="rdfs:label" content="EC 2.7.10.2" datatype="xsd:string"></span><span id='am-909' property="oboInOwl:hasExactSynonym" content="SLK" datatype="xsd:string"></span><span id='am-910' property="rdfs:label" content="p59-Fyn" datatype="xsd:string"></span><span id='am-911' property="rdfs:label" content="SYN" datatype="xsd:string"></span><span id='am-912' property="oboInOwl:hasExactSynonym" content="Src-like kinase" datatype="xsd:string"></span><span id='am-913' property="rdfs:label" content="FYN_HUMAN" datatype="xsd:string"></span><span id='am-914' property="rdfs:label" content="FYN" datatype="xsd:string"></span><span id='am-915' property="rdfs:label" content="Tyrosine-protein kinase Fyn" datatype="xsd:string"></span>syn</span>-                trast, species G1 Bacillus <br /> thesized N-terminal extension known signal,                 genus, B. subtilis B. amyloliquefaciens. known leader, peptide (42). Proteolytic cleavage signal se-                homologous SPase genes quence yield mature protein occurs shortly                 Bacillus species. sipS gene deleted chro- translocation event catalyzed pro-                 mosome B. subtilis 168 affecting cell growth rate karyotes eukaryotes enzymes known signal, leader,              viability laboratory conditions yield mutant strain <br /> peptidases (SPases). bacterial SPases membrane pro-                 process pre amylase amylase (4a).  <br /> teins consisting single polypeptide anchored mem-               cently reported putative SPase sequence (1) gene <br /> brane (gram-positive [G1] gram-negative <br />                                                                             product responsible activity, B. subtilis [<span id='am-5' about='obo:IMR_0700368' typeof='owl:Thing'><span id='am-6' property="rdfs:label" content="myosin light chain, phosphorylatable, fast skeletal muscle" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="UniProt:P04466" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="Mylpf" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Fast skeletal myosin light chain 2" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0700368" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Myosin regulatory light chain 2, skeletal muscle isoform" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Mylpf" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="DTNB" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="Myosin regulatory light chain 2, skeletal muscle isoform" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="myosin light chain, phosphorylatable, fast skeletal muscle" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="MLRS_RAT" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Fast skeletal myosin light chain 2" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="MLC-2" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="G2" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="G2" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="DTNB" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="MLC-2" datatype="xsd:string"></span>G2</span>] bacteria) (G2 bacteria) transmembrane sections. <br />                                                                             harbor SPase genes. genes  <br /> Predicted amino acid sequences bacterial SPases high level similarity known Escherichia coli (43),           coding distinct SPase homologs reside chromosome Pseudomonas fluorescens (4), Salmonella typhimurium (39),                   closely related species B. amyloliquefaciens (12), Haemophilus influenzae (8), Phormidium laminosum (25), Bra-                 evidence suggest B. japonicum possess dyrhizobium japonicum (22), Rhodobacter capsulatus (17), Ba-                SPase (22, 23). SPase sequences seven <br /> cillus subtilis  chromosomal plasmid origin [1,              genera G2 bacteria known, single Bacillus <br /> 21, 37]), Bacillus licheniformis (11), Bacillus caldolyticus (34),          genus G1 eubacteria investigated Bacillus amyloliquefaciens  chromosomal genes) (12,                 respect SPase characteristics. considered 35), partial sequence reported Bacillus                  determine G1 eubacterium  unlike <br /> pumilis (12). enzymes collectively defined               B. subtilis B. amyloliquefaciens, known excep- <br /> type signal peptidases (37). amino acid se-                  tional secretion activity genes encoding quences 15 bacterial SPases  16th partial sequence)                SPase overlapping substrate specificities reported, best-studied examples leader                resembles E. coli H. influenzae  possibly G2 <br /> peptidase (LPase LepB) E. coli SPase B.                 eubacteria) closely single SPase gene. subtilis (SipS).                                                            recent publication entire genome sequence obli- <br />    demonstrated LPase activity essential            gate G1-like intracellular bacterium Mycoplasma genitalium <br /> cell growth E. coli. Experiments expression               reveals interesting feature relating heterogeneity <br /> lepB gene, encoding LPase, regulated control-               SPases (9). homologs known <br /> lable ara promoter (5) partial deletion natural                components bacterial protein translocation machinery promoter (6) indicated minimization LPase production <br />                                                                             present (secA, secY, ), reported gene associated cessation cell growth division.                                                                             encoding type SPase detected. implies addition, E. coli strain possessing mutated lepB gene (E. <br /> coli IT41) shown drastically reduced growth              non-type SPase exist organism. Finally, SPases <br /> rate display rapid pronounced accumulation pre-                 components secretory apparatus) proteins temperature growth medium ele-                  represent novel targets chemical intervention attempts vated 428C (14). results imply              derive new classes antibacterial agents clinical utility <br /> gene product E. coli substitute LPase               potent inhibitors E. coli LPase recently                                                                             discovered (2), determine type                                                                             SPases play essential role pathogenic G1 eubacterium. <br />   * Corresponding author. Mailing address: SmithKline Beecham               Experiments performed order clone Pharmaceuticals, Yew Tree Road, Epsom, Surrey KT18 5XQ,              gene(s) encodes SPase  determine essen- <br /> United Kingdom. Phone: 44 1737 364000. Fax: 44 1737 364073. <br /> Electronic mail address: Mick_Black-1@sbphrd.com.                           tiality gene product  Staphylococcus aureus. <br />  <br />                                                                      5712 <br />  VOL. 178, 1996                                                                                           SIGNAL PEPTIDASE S. AUREUS                             5713 <br />  <br />  <br />                                                                 TABLE 1. Plasmids used study <br />                             Size <br />        Plasmid                                                                     Description                                                           Source <br />                            (kbp) <br />  <br /> pBluescriptII <span id='am-209' about='obo:IMR_0200531' typeof='owl:Thing'><span id='am-210' property="rdfs:label" content="PAPS SYNTHETASE 2" datatype="xsd:string"></span><span id='am-211' property="rdfs:label" content="ATP SULFURYLASE/ADENOSINE 5-PRIME PHOSPHOSULFATE KINASE 2" datatype="xsd:string"></span><span id='am-212' property="rdfs:label" content="SK2" datatype="xsd:string"></span><span id='am-213' property="rdfs:label" content="PAPS2_HUMAN" datatype="xsd:string"></span><span id='am-214' property="rdfs:label" content="APS kinase" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasExactSynonym" content="Adenylylsulfate 3&apos;-phosphotransferase" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasExactSynonym" content="ATP-sulfurylase" datatype="xsd:string"></span><span id='am-217' property="rdfs:label" content="3&apos;-phosphoadenosine 5&apos;-phosphosulfate synthase 2" datatype="xsd:string"></span><span id='am-218' property="rdfs:label" content="ATPSK2" datatype="xsd:string"></span><span id='am-219' property="rdfs:label" content="PAPSS2" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasExactSynonym" content="ATP SULFURYLASE/APS KINASE 2" datatype="xsd:string"></span><span id='am-221' property="rdfs:label" content="Adenylyl-sulfate kinase" datatype="xsd:string"></span><span id='am-222' property="rdfs:label" content="3-PRIME-@PHOSPHOADENOSINE 5-PRIME-PHOSPHOSULFATE SYNTHASE 2" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasExactSynonym" content="Adenylyl-sulfate kinase" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasExactSynonym" content="PAPS synthase 2" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasExactSynonym" content="SK2" datatype="xsd:string"></span><span id='am-226' property="rdfs:label" content="Sulfate adenylate transferase" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasExactSynonym" content="SK 2" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasExactSynonym" content="APS kinase" datatype="xsd:string"></span><span id='am-229' property="rdfs:label" content="ATP-sulfurylase" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasExactSynonym" content="Adenosine-5&apos;-phosphosulfate 3&apos;-phosphotransferase" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasExactSynonym" content="Bifunctional 3&apos;-phosphoadenosine 5&apos;-phosphosulfate synthase 2" datatype="xsd:string"></span><span id='am-232' property="rdfs:label" content="EC 2.7.7.4" datatype="xsd:string"></span><span id='am-233' property="rdfs:label" content="EC 2.7.1.25" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasExactSynonym" content="3&apos;-phosphoadenosine-5&apos;-phosphosulfate synthase" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasDbXref" content="UniProt:O95340" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasExactSynonym" content="PAPS SYNTHETASE 2" datatype="xsd:string"></span><span id='am-237' property="rdfs:label" content="ATP SULFURYLASE/APS KINASE 2" datatype="xsd:string"></span><span id='am-238' property="oboInOwl:hasExactSynonym" content="ATP SULFURYLASE/ADENOSINE 5-PRIME PHOSPHOSULFATE KINASE 2" datatype="xsd:string"></span><span id='am-239' property="rdfs:label" content="SK 2" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:hasExactSynonym" content="Sulfate adenylyltransferase" datatype="xsd:string"></span><span id='am-241' property="rdfs:label" content="3&apos;-phosphoadenosine-5&apos;-phosphosulfate synthase" datatype="xsd:string"></span><span id='am-242' property="oboInOwl:hasExactSynonym" content="EC 2.7.7.4" datatype="xsd:string"></span><span id='am-243' property="oboInOwl:hasExactSynonym" content="PAPSS 2" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasExactSynonym" content="3-PRIME-@PHOSPHOADENOSINE 5-PRIME-PHOSPHOSULFATE SYNTHASE 2" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasExactSynonym" content="SAT" datatype="xsd:string"></span><span id='am-246' property="rdfs:label" content="SAT" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Bifunctional 3&apos;-phosphoadenosine 5&apos;-phosphosulfate synthase 2" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasExactSynonym" content="PAPSS2" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasExactSynonym" content="Sulfate adenylate transferase" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasExactSynonym" content="PHOSPHOADENOSINE-PHOSPHOSULFATE SYNTHETASE 2" datatype="xsd:string"></span><span id='am-251' property="oboInOwl:hasExactSynonym" content="3&apos;-phosphoadenosine 5&apos;-phosphosulfate synthase 2" datatype="xsd:string"></span><span id='am-252' property="oboInOwl:hasExactSynonym" content="ATPSK2" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:id" content="IMR:0200531" datatype="xsd:string"></span><span id='am-254' property="rdfs:label" content="PHOSPHOADENOSINE-PHOSPHOSULFATE SYNTHETASE 2" datatype="xsd:string"></span><span id='am-255' property="rdfs:label" content="PAPSS 2" datatype="xsd:string"></span><span id='am-256' property="rdfs:label" content="Adenylylsulfate 3&apos;-phosphotransferase" datatype="xsd:string"></span><span id='am-257' property="rdfs:label" content="Adenosine-5&apos;-phosphosulfate 3&apos;-phosphotransferase" datatype="xsd:string"></span><span id='am-258' property="rdfs:label" content="Sulfate adenylyltransferase" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasExactSynonym" content="Sulfurylase kinase 2" datatype="xsd:string"></span><span id='am-260' property="rdfs:label" content="Sulfurylase kinase 2" datatype="xsd:string"></span><span id='am-261' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-262' property="rdfs:label" content="PAPS synthase 2" datatype="xsd:string"></span><span id='am-263' property="oboInOwl:hasExactSynonym" content="EC 2.7.1.25" datatype="xsd:string"></span>SK2</span>           3.0      Phagemid derived pUC19; Ampr                                                                        Stratagene <br /> pGEM-T                      3.0      PCR cloning vector; Ampr                                                                                 Promega <br /> pKC09                       3.2      S. aureus Oxford PCR product (primers 1 B) cloned pGEM-T                                          work <br /> pKC10                      10.7      Excised clone S. aureus WCUH29 lZapII library; 7.7-kb insert containing spsA                        work <br />                                         spsB <br /> pKC11                        7.5     Large fragment SalI-cut pKC10 religated form pKC11; 4.5-kb insert containing                       work <br />                                         spsA spsB <br /> pDB575                       5.4     lacIq Ptac expression vector; Ampr                                                                       Gift M. Lawrence <br />                                                                                                                                                 D. <span id='am-24' about='obo:IMR_0703532' typeof='owl:Thing'><span id='am-25' property="oboInOwl:hasExactSynonym" content="UDP-galactose 4-epimerase" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="galactose-4-epimerase, UDP" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="Galactowaldenase" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="EC 5.1.3.2" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0703532" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="UniProt:Q8R059" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Gale" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="UDP-glucose 4-epimerase" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="UDP-glucose 4-epimerase" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="GALE_MOUSE" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="UDP-galactose 4-epimerase" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="galactose-4-epimerase, UDP" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Gale" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Galactowaldenase" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="EC 5.1.3.2" datatype="xsd:string"></span>Gale</span> <br /> pKC16                        6.3     SpsB expression construct; spsB PCR product (primers C 1 D pKC11                                    work <br />                                        template) cloned pDB575 <br /> pUC18                        2.7     E. coli plasmid; Ampr                                                                                    Stratagene <br /> pTS1                         6.9     E. coli-S. aureus shuttle vector; Ts G1 origin; Ampr Cmr                                                 Gift T. Foster <br /> pIM04                        4.4     Regions flanking spsB (850 bp  inserted pUC18                                                  work <br /> pIM05                        8.6     Regions flanking spsB (850 bp  inserted pTS1                                                   work <br /> pIM11                        9.2     pIM05 containing spsB                                                                                    work <br />  <br />  <br />  <br />                          MATERIALS METHODS                                           min 608C, 2 min 728C, 5 min 728C; G plus H, 5 min 948C, 15 <br />                                                                                          cycles 1 min 948C, 1 min 458C, 2 min 728C, 5 min 728C;    Bacterial strains, plasmids, media. following bacterial strains          plus J, 5 min 948C, 30 cycles 1 min 948C, 1 min 428C, 3 min used study: E. coli XL1-Blue, JM109, IT41 S. aureus RN4220, H            728C, 5 min 728C. <br /> (ATCC 13801), Oxford (ATCC 9144), WCUH29 (NCIMB 40771). S. aureus                       DNA Southern blotted 0.7% (wt/vol) agarose gels nylon mem- <br /> strains grown tryptone soya broth (Oxoid) Luria-Bertani broth (28).        branes (Hybond-N1; Amersham) instructed manufacturer. library <br /> E. coli cultured Luria-Bertani broth. solid medium, 1.5% (wt/vol) agar        screening purposes, plaques transferred nylon membranes (Hybond-N1; added. appropriate, medium supplemented 100 mg                 Amersham) instructed manufacturer. Membranes hybridized ampicillin ml E. coli 5 mg chloramphenicol ml S. aureus.          oligonucleotide gene probes labeled using ECL (enhanced S. aureus WCUH29 genomic library constructed Stratagene                chemiluminescence) 39-oligolabeling <span id='am-183' about='obo:IMR_0000074' typeof='owl:Thing obo:IMR_0000210 obo:IMR_0000054 obo:IMR_0000063 obo:IMR_0000949 obo:IMR_0000512 obo:IMR_0000209 obo:IMR_0010050 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0001012'><span id='am-184' property="oboInOwl:hasExactSynonym" content="c-Kit" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:id" content="IMR:0000074" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasDbXref" content="GO:0005020" datatype="xsd:string"></span><span id='am-187' property="rdfs:label" content="Kit" datatype="xsd:string"></span><span id='am-188' property="rdfs:label" content="c-Kit" datatype="xsd:string"></span><span id='am-189' property="rdfs:label" content="mast/stem cell growth factor receptor" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasExactSynonym" content="mast/stem cell growth factor receptor" datatype="xsd:string"></span>kit</span> ECL random prime labeling <span id='am-264' about='obo:IMR_0700172' typeof='owl:Thing'><span id='am-265' property="rdfs:label" content="SCO5" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasExactSynonym" content="SOW3" datatype="xsd:string"></span><span id='am-267' property="oboInOwl:hasExactSynonym" content="Sl" datatype="xsd:string"></span><span id='am-268' property="rdfs:label" content="SCFR" datatype="xsd:string"></span><span id='am-269' property="oboInOwl:hasExactSynonym" content="c-KIT" datatype="xsd:string"></span><span id='am-270' property="oboInOwl:hasExactSynonym" content="belly-spot" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasExactSynonym" content="Gsfsow3" datatype="xsd:string"></span><span id='am-272' property="rdfs:label" content="Gsfsow3" datatype="xsd:string"></span><span id='am-273' property="rdfs:label" content="Sl" datatype="xsd:string"></span><span id='am-274' property="rdfs:label" content="Dominant white spotting" datatype="xsd:string"></span><span id='am-275' property="rdfs:label" content="CD117" datatype="xsd:string"></span><span id='am-276' property="rdfs:label" content="Gsfsco1" datatype="xsd:string"></span><span id='am-277' property="oboInOwl:id" content="IMR:0700172" datatype="xsd:string"></span><span id='am-278' property="rdfs:label" content="kit oncogene" datatype="xsd:string"></span><span id='am-279' property="rdfs:label" content="Proto-oncogene c-Kit" datatype="xsd:string"></span><span id='am-280' property="rdfs:label" content="KIT_MOUSE" datatype="xsd:string"></span><span id='am-281' property="oboInOwl:hasExactSynonym" content="kit oncogene" datatype="xsd:string"></span><span id='am-282' property="oboInOwl:hasExactSynonym" content="CD117" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasExactSynonym" content="Mast/stem cell growth factor receptor" datatype="xsd:string"></span><span id='am-284' property="rdfs:label" content="Gsfsco5" datatype="xsd:string"></span><span id='am-285' property="oboInOwl:hasExactSynonym" content="Gsfsco1" datatype="xsd:string"></span><span id='am-286' property="rdfs:label" content="belly-spot" datatype="xsd:string"></span><span id='am-287' property="rdfs:label" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-288' property="oboInOwl:hasExactSynonym" content="Steel Factor Receptor" datatype="xsd:string"></span><span id='am-289' property="oboInOwl:hasExactSynonym" content="SCFR" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein kinase Kit" datatype="xsd:string"></span><span id='am-291' property="oboInOwl:hasExactSynonym" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-292' property="rdfs:label" content="Kit" datatype="xsd:string"></span><span id='am-293' property="rdfs:label" content="Steel Factor Receptor" datatype="xsd:string"></span><span id='am-294' property="rdfs:label" content="SCO1" datatype="xsd:string"></span><span id='am-295' property="rdfs:label" content="Tyrosine-protein kinase Kit" datatype="xsd:string"></span><span id='am-296' property="oboInOwl:hasExactSynonym" content="SCO5" datatype="xsd:string"></span><span id='am-297' property="rdfs:label" content="Tr-kit" datatype="xsd:string"></span><span id='am-298' property="rdfs:label" content="Mast/stem cell growth factor receptor" datatype="xsd:string"></span><span id='am-299' property="oboInOwl:hasExactSynonym" content="Kit" datatype="xsd:string"></span><span id='am-300' property="oboInOwl:hasExactSynonym" content="Dominant white spotting" datatype="xsd:string"></span><span id='am-301' property="oboInOwl:hasExactSynonym" content="Gsfsco5" datatype="xsd:string"></span><span id='am-302' property="oboInOwl:hasDbXref" content="UniProt:P05532" datatype="xsd:string"></span><span id='am-303' property="rdfs:label" content="c-KIT" datatype="xsd:string"></span><span id='am-304' property="oboInOwl:hasExactSynonym" content="SCO1" datatype="xsd:string"></span><span id='am-305' property="rdfs:label" content="SOW3" datatype="xsd:string"></span><span id='am-306' property="oboInOwl:hasExactSynonym" content="Proto-oncogene c-Kit" datatype="xsd:string"></span><span id='am-307' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-308' property="oboInOwl:hasExactSynonym" content="Tr-kit" datatype="xsd:string"></span>kit</span> <br /> vector lZapII. lZapII S. aureus H library constructed laboratory         (Amersham) appropriate. Washing detection steps performed instructed manufacturer lZapII (Stratagene). shuttle vector             instructed manufacturer. sequence data manipulation performed <br /> constructs transferred E. coli S. aureus WCUH29             Wisconsin package (7). <br /> used transform restriction-minus S. aureus strain RN4220, plasmid                Expression studies. Sodium dodecyl sulfate (<span id='am-503' about='obo:IMR_0200659' typeof='owl:Thing'><span id='am-504' property="oboInOwl:hasExactSynonym" content="L-serine deaminase" datatype="xsd:string"></span><span id='am-505' property="rdfs:label" content="SDH" datatype="xsd:string"></span><span id='am-506' property="oboInOwl:hasExactSynonym" content="EC 4.3.1.19" datatype="xsd:string"></span><span id='am-507' property="oboInOwl:hasExactSynonym" content="SDS" datatype="xsd:string"></span><span id='am-508' property="rdfs:label" content="serine dehydratase" datatype="xsd:string"></span><span id='am-509' property="oboInOwl:hasExactSynonym" content="EC 4.3.1.17" datatype="xsd:string"></span><span id='am-510' property="oboInOwl:hasExactSynonym" content="SDH" datatype="xsd:string"></span><span id='am-511' property="oboInOwl:hasDbXref" content="UniProt:P20132" datatype="xsd:string"></span><span id='am-512' property="oboInOwl:hasExactSynonym" content="TDH" datatype="xsd:string"></span><span id='am-513' property="rdfs:label" content="L-serine dehydratase/L-threonine deaminase" datatype="xsd:string"></span><span id='am-514' property="oboInOwl:hasExactSynonym" content="serine dehydratase" datatype="xsd:string"></span><span id='am-515' property="rdfs:label" content="L-threonine dehydratase" datatype="xsd:string"></span><span id='am-516' property="rdfs:label" content="SDHL_HUMAN" datatype="xsd:string"></span><span id='am-517' property="oboInOwl:id" content="IMR:0200659" datatype="xsd:string"></span><span id='am-518' property="rdfs:label" content="L-serine deaminase" datatype="xsd:string"></span><span id='am-519' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-520' property="rdfs:label" content="TDH" datatype="xsd:string"></span><span id='am-521' property="oboInOwl:hasExactSynonym" content="L-threonine dehydratase" datatype="xsd:string"></span><span id='am-522' property="oboInOwl:hasExactSynonym" content="L-serine dehydratase/L-threonine deaminase" datatype="xsd:string"></span><span id='am-523' property="rdfs:label" content="EC 4.3.1.17" datatype="xsd:string"></span><span id='am-524' property="rdfs:label" content="SDS" datatype="xsd:string"></span><span id='am-525' property="rdfs:label" content="EC 4.3.1.19" datatype="xsd:string"></span>SDS</span>)-polyacrylamide gel electro- <br /> purified therefrom used transform S. aureus WCUH29. Plasmids used              phoresis (PAGE) carried essentially described Laemmli (19), using study described Table 1.                                                     Mini Protean II apparatus (Bio-Rad). Cultures grown mid-log phase <br />    DNA techniques. chemicals purchased Sigma unless              induced addition 1 mM isopropylthiogalactopyranoside (IPTG). <br /> specified. Plasmid DNA isolated using RPM <span id='am-309' about='obo:IMR_0700173' typeof='owl:Thing'><span id='am-310' property="oboInOwl:hasExactSynonym" content="STEM CELL FACTOR RECEPTOR" datatype="xsd:string"></span><span id='am-311' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein kinase Kit" datatype="xsd:string"></span><span id='am-312' property="rdfs:label" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasExactSynonym" content="Proto-oncogene c-Kit" datatype="xsd:string"></span><span id='am-314' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-315' property="oboInOwl:id" content="IMR:0700173" datatype="xsd:string"></span><span id='am-316' property="oboInOwl:hasExactSynonym" content="v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasExactSynonym" content="MAST CELL GROWTH FACTOR RECEPTOR" datatype="xsd:string"></span><span id='am-318' property="rdfs:label" content="KIT_HUMAN" datatype="xsd:string"></span><span id='am-319' property="rdfs:label" content="SCFR" datatype="xsd:string"></span><span id='am-320' property="rdfs:label" content="Tyrosine-protein kinase Kit" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasExactSynonym" content="CD117" datatype="xsd:string"></span><span id='am-322' property="oboInOwl:hasExactSynonym" content="C-Kit" datatype="xsd:string"></span><span id='am-323' property="oboInOwl:hasExactSynonym" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-324' property="rdfs:label" content="v-kit Hardy-Zuckerman 4 feline sarcoma viral oncogene homolog" datatype="xsd:string"></span><span id='am-325' property="rdfs:label" content="MAST CELL GROWTH FACTOR RECEPTOR" datatype="xsd:string"></span><span id='am-326' property="rdfs:label" content="KIT ONCOGENE" datatype="xsd:string"></span><span id='am-327' property="rdfs:label" content="KIT" datatype="xsd:string"></span><span id='am-328' property="rdfs:label" content="CD117" datatype="xsd:string"></span><span id='am-329' property="rdfs:label" content="PBT" datatype="xsd:string"></span><span id='am-330' property="rdfs:label" content="C-Kit" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasExactSynonym" content="KIT ONCOGENE" datatype="xsd:string"></span><span id='am-332' property="rdfs:label" content="Proto-oncogene c-Kit" datatype="xsd:string"></span><span id='am-333' property="oboInOwl:hasExactSynonym" content="Mast/stem cell growth factor receptor" datatype="xsd:string"></span><span id='am-334' property="rdfs:label" content="Mast/stem cell growth factor receptor" datatype="xsd:string"></span><span id='am-335' property="oboInOwl:hasDbXref" content="UniProt:P10721" datatype="xsd:string"></span><span id='am-336' property="oboInOwl:hasExactSynonym" content="KIT" datatype="xsd:string"></span><span id='am-337' property="oboInOwl:hasExactSynonym" content="SCFR" datatype="xsd:string"></span><span id='am-338' property="oboInOwl:hasExactSynonym" content="PBT" datatype="xsd:string"></span><span id='am-339' property="rdfs:label" content="STEM CELL FACTOR RECEPTOR" datatype="xsd:string"></span>kit</span> (Bio 101 )            Cells harvested 2.5 h postinduction, total cell lysates prepared Wizard Midiprep DNA purification (Promega). PCR products iso-                outlined Sambrook et al. (28), proteins separated electrophoresis <br /> lated horizontal agarose gel electrophoresis, treated Agarase (Boehr-            15% (wt/vol) acrylamide resolving gel. <br /> inger Mannheim), purified Wizard DNA Clean (Pro-                     complementation studies, E. coli IT41 transformed appropriate <br /> mega). Chromosomal DNA isolated E. coli S. aureus using                  plasmids, ampicillin-resistant transformants picked agar plates grown genomic DNA purification kit (bacterial) (Advanced Genetic Technol-             288C incubated broth overnight permissive temperature 288C. <br /> ogies Corp.) published procedures (20). incubation appropriate            Cultures diluted 100-<span id='am-721' about='obo:IMR_0703347' typeof='owl:Thing'><span id='am-722' property="rdfs:label" content="Argininosuccinate synthase" datatype="xsd:string"></span><span id='am-723' property="rdfs:label" content="Ass1" datatype="xsd:string"></span><span id='am-724' property="oboInOwl:hasExactSynonym" content="Ass-1" datatype="xsd:string"></span><span id='am-725' property="oboInOwl:hasExactSynonym" content="argininosuccinate synthetase 1" datatype="xsd:string"></span><span id='am-726' property="rdfs:label" content="fold" datatype="xsd:string"></span><span id='am-727' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-728' property="oboInOwl:hasExactSynonym" content="fold" datatype="xsd:string"></span><span id='am-729' property="rdfs:label" content="EC 6.3.4.5" datatype="xsd:string"></span><span id='am-730' property="rdfs:label" content="Citrulline--aspartate ligase" datatype="xsd:string"></span><span id='am-731' property="oboInOwl:id" content="IMR:0703347" datatype="xsd:string"></span><span id='am-732' property="oboInOwl:hasExactSynonym" content="Ass1" datatype="xsd:string"></span><span id='am-733' property="rdfs:label" content="argininosuccinate synthetase 1" datatype="xsd:string"></span><span id='am-734' property="oboInOwl:hasDbXref" content="UniProt:P16460" datatype="xsd:string"></span><span id='am-735' property="rdfs:label" content="Ass-1" datatype="xsd:string"></span><span id='am-736' property="oboInOwl:hasExactSynonym" content="Argininosuccinate synthase" datatype="xsd:string"></span><span id='am-737' property="oboInOwl:hasExactSynonym" content="EC 6.3.4.5" datatype="xsd:string"></span><span id='am-738' property="rdfs:label" content="ASSY_MOUSE" datatype="xsd:string"></span><span id='am-739' property="oboInOwl:hasExactSynonym" content="Citrulline--aspartate ligase" datatype="xsd:string"></span>fold</span> fresh medium incubated shaking quantity lysostaphin (Applied Microbiology ) included              nonpermissive temperature 428C. optical density 540 nm (OD540) <br /> preparation plasmid chromosomal DNA S. aureus facilitate cell             recorded 30-min intervals period 8 h. <br /> lysis. lZapII library clones excised recircularized form recombinant            Selection double-crossover plasmid integration/resolution events plas- <br /> phagemid instructed manufacturer. Procedures DNA restriction               mid curing. S. aureus WCUH29(pIM05) WCUH29(pIM11) grown dephosphorylation, agarose gel electrophoresis, PCR, transformation           overnight 288C presence chloramphenicol. cultures diluted <br /> competent E. coli cells performed essentially described Sambrook et           100-fold fresh medium chloramphenicol incubated 438C. <br /> al. (28). Electrocompetent S. aureus cells prepared described Schenk          Cultures grown 438C absence chloramphenicol 56 h Laddaga (29), following modifications: bacteria grown           reduce plasmid copy number, time cultures subinocu- <br /> tryptone soya broth washed 2500 3 g 5 min 208C. Competent cells            lated times (twice daily) fresh medium. Samples diluted plasmid DNA electroporated 1-mm-<span id='am-769' about='obo:IMR_0704195' typeof='owl:Thing'><span id='am-770' property="oboInOwl:hasExactSynonym" content="Rasa1" datatype="xsd:string"></span><span id='am-771' property="oboInOwl:hasExactSynonym" content="GTPase-activating protein" datatype="xsd:string"></span><span id='am-772' property="rdfs:label" content="GAP" datatype="xsd:string"></span><span id='am-773' property="oboInOwl:hasExactSynonym" content="Ras GTPase-activating protein 1" datatype="xsd:string"></span><span id='am-774' property="oboInOwl:id" content="IMR:0704195" datatype="xsd:string"></span><span id='am-775' property="oboInOwl:hasDbXref" content="UniProt:P50904" datatype="xsd:string"></span><span id='am-776' property="rdfs:label" content="GTPase-activating protein" datatype="xsd:string"></span><span id='am-777' property="rdfs:label" content="Ras p21 protein activator" datatype="xsd:string"></span><span id='am-778' property="oboInOwl:hasExactSynonym" content="RAS p21 protein activator (GTPase activating protein) 1" datatype="xsd:string"></span><span id='am-779' property="oboInOwl:hasExactSynonym" content="Ras p21 protein activator" datatype="xsd:string"></span><span id='am-780' property="rdfs:label" content="p120GAP" datatype="xsd:string"></span><span id='am-781' property="oboInOwl:hasExactSynonym" content="Rasa" datatype="xsd:string"></span><span id='am-782' property="rdfs:label" content="Rasa" datatype="xsd:string"></span><span id='am-783' property="rdfs:label" content="RasGAP" datatype="xsd:string"></span><span id='am-784' property="oboInOwl:hasExactSynonym" content="RasGAP" datatype="xsd:string"></span><span id='am-785' property="rdfs:label" content="RASA1_RAT" datatype="xsd:string"></span><span id='am-786' property="rdfs:label" content="RAS p21 protein activator (GTPase activating protein) 1" datatype="xsd:string"></span><span id='am-787' property="oboInOwl:hasExactSynonym" content="GAP" datatype="xsd:string"></span><span id='am-788' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-789' property="oboInOwl:hasExactSynonym" content="p120GAP" datatype="xsd:string"></span><span id='am-790' property="rdfs:label" content="Rasa1" datatype="xsd:string"></span><span id='am-791' property="rdfs:label" content="Ras GTPase-activating protein 1" datatype="xsd:string"></span>gap</span> electroporation cuvettes              plated 288C presence (integration/resolution) absence (plasmid <br /> 208C, 100 V, 25 mF, <span id='am-1165' about='obo:IMR_0002971' typeof='owl:Thing'><span id='am-1166' property="rdfs:label" content="sos" datatype="xsd:string"></span><span id='am-1167' property="rdfs:label" content="dme-SOS" datatype="xsd:string"></span><span id='am-1168' property="rdfs:label" content="l(2)Sos" datatype="xsd:string"></span><span id='am-1169' property="oboInOwl:hasExactSynonym" content="Sos" datatype="xsd:string"></span><span id='am-1170' property="rdfs:label" content="2.3" datatype="xsd:string"></span><span id='am-1171' property="oboInOwl:hasExactSynonym" content="Son of sevenless" datatype="xsd:string"></span><span id='am-1172' property="oboInOwl:hasExactSynonym" content="E(sev)2A" datatype="xsd:string"></span><span id='am-1173' property="oboInOwl:hasExactSynonym" content="l(2)Sos" datatype="xsd:string"></span><span id='am-1174' property="oboInOwl:hasExactSynonym" content="EY2-3" datatype="xsd:string"></span><span id='am-1175' property="oboInOwl:hasExactSynonym" content="br24" datatype="xsd:string"></span><span id='am-1176' property="oboInOwl:hasExactSynonym" content="EK2-8" datatype="xsd:string"></span><span id='am-1177' property="rdfs:label" content="Son of sevenless" datatype="xsd:string"></span><span id='am-1178' property="oboInOwl:id" content="IMR:0002971" datatype="xsd:string"></span><span id='am-1179' property="rdfs:label" content="br24" datatype="xsd:string"></span><span id='am-1180' property="oboInOwl:hasExactSynonym" content="34Ea" datatype="xsd:string"></span><span id='am-1181' property="oboInOwl:hasExactSynonym" content="dSos" datatype="xsd:string"></span><span id='am-1182' property="rdfs:label" content="l(2)34Ea" datatype="xsd:string"></span><span id='am-1183' property="oboInOwl:hasExactSynonym" content="Su(tor)2-2" datatype="xsd:string"></span><span id='am-1184' property="oboInOwl:hasExactSynonym" content="lethal 25 in the black-reduced region" datatype="xsd:string"></span><span id='am-1185' property="oboInOwl:hasExactSynonym" content="CG7793" datatype="xsd:string"></span><span id='am-1186' property="rdfs:label" content="Protein son of sevenless" datatype="xsd:string"></span><span id='am-1187' property="rdfs:label" content="E(var)189" datatype="xsd:string"></span><span id='am-1188' property="rdfs:label" content="EY2-3" datatype="xsd:string"></span><span id='am-1189' property="rdfs:label" content="34Ea" datatype="xsd:string"></span><span id='am-1190' property="rdfs:label" content="lethal 24 in the black-reduced region" datatype="xsd:string"></span><span id='am-1191' property="oboInOwl:hasExactSynonym" content="l(2)k06321" datatype="xsd:string"></span><span id='am-1192' property="oboInOwl:hasExactSynonym" content="l(2)br24" datatype="xsd:string"></span><span id='am-1193' property="oboInOwl:hasExactSynonym" content="lethal 24 in the black-reduced region" datatype="xsd:string"></span><span id='am-1194' property="oboInOwl:hasExactSynonym" content="sos" datatype="xsd:string"></span><span id='am-1195' property="rdfs:label" content="lethal 25 in the black-reduced region" datatype="xsd:string"></span><span id='am-1196' property="rdfs:label" content="l(2)br25" datatype="xsd:string"></span><span id='am-1197' property="rdfs:label" content="Su(tor)2-2" datatype="xsd:string"></span><span id='am-1198' property="oboInOwl:hasExactSynonym" content="SOS" datatype="xsd:string"></span><span id='am-1199' property="oboInOwl:hasExactSynonym" content="br25" datatype="xsd:string"></span><span id='am-1200' property="rdfs:label" content="l(2)k05224" datatype="xsd:string"></span><span id='am-1201' property="oboInOwl:hasExactSynonym" content="Protein son of sevenless" datatype="xsd:string"></span><span id='am-1202' property="rdfs:label" content="E(sev)2A" datatype="xsd:string"></span><span id='am-1203' property="oboInOwl:hasDbXref" content="UniProt:P26675" datatype="xsd:string"></span><span id='am-1204' property="rdfs:label" content="SOS" datatype="xsd:string"></span><span id='am-1205' property="rdfs:label" content="BG:DS00941.4" datatype="xsd:string"></span><span id='am-1206' property="oboInOwl:hasExactSynonym" content="2.3" datatype="xsd:string"></span><span id='am-1207' property="rdfs:label" content="EK2-8" datatype="xsd:string"></span><span id='am-1208' property="oboInOwl:hasExactSynonym" content="E(var)189" datatype="xsd:string"></span><span id='am-1209' property="oboInOwl:hasExactSynonym" content="Sos1" datatype="xsd:string"></span><span id='am-1210' property="rdfs:label" content="Sos" datatype="xsd:string"></span><span id='am-1211' property="rdfs:label" content="dSos" datatype="xsd:string"></span><span id='am-1212' property="oboInOwl:hasExactSynonym" content="BG:DS00941.4" datatype="xsd:string"></span><span id='am-1213' property="rdfs:label" content="CG7793" datatype="xsd:string"></span><span id='am-1214' property="oboInOwl:hasExactSynonym" content="l(2)br25" datatype="xsd:string"></span><span id='am-1215' property="rdfs:label" content="SOS_DROME" datatype="xsd:string"></span><span id='am-1216' property="rdfs:label" content="l(2)br24" datatype="xsd:string"></span><span id='am-1217' property="oboInOwl:hasExactSynonym" content="l(2)k05224" datatype="xsd:string"></span><span id='am-1218' property="rdfs:label" content="Sos1" datatype="xsd:string"></span><span id='am-1219' property="oboInOwl:hasExactSynonym" content="l(2)34Ea" datatype="xsd:string"></span><span id='am-1220' property="rdfs:label" content="l(2)k06321" datatype="xsd:string"></span><span id='am-1221' property="oboInOwl:hasExactSynonym" content="dme-SOS" datatype="xsd:string"></span><span id='am-1222' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1223' property="rdfs:label" content="br25" datatype="xsd:string"></span>2.3</span> <span id='am-1403' about='obo:IMR_0000519' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000481 obo:IMR_0001049 obo:IMR_0000513 obo:IMR_0000500 obo:IMR_0000655 obo:IMR_0000001 obo:IMR_0000498 obo:IMR_0100584'><span id='am-1404' property="oboInOwl:hasDbXref" content="IPR:003968" datatype="xsd:string"></span><span id='am-1405' property="obo:IAO_0000115" content="The Kv family can be divided into 4 subfamilies on the basis of sequence similarity and function: Shaker (Kv1), Shab (Kv2), Shaw (Kv3) and Shal (Kv4)." datatype="xsd:string"></span><span id='am-1406' property="rdfs:label" content="Kv" datatype="xsd:string"></span><span id='am-1407' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1408' property="oboInOwl:id" content="IMR:0000519" datatype="xsd:string"></span>kV</span> Gene Pulser apparatus pulse controller          curing) chloramphenicol. <br /> (Bio-Rad Laboratories ). Restriction enzymes calf intestine alkaline                Identification colonies plasmid-borne copy spsB derived <span id='am-1395' about='obo:IMR_0000517' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010044 obo:IMR_0000001 obo:IMR_0000207'><span id='am-1396' property="rdfs:label" content="phosphoric monoester hydrolase activity" datatype="xsd:string"></span><span id='am-1397' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1398' property="oboInOwl:hasDbXref" content="GO:0016791" datatype="xsd:string"></span><span id='am-1399' property="oboInOwl:hasDbXref" content="EC:3.1.3.-" datatype="xsd:string"></span><span id='am-1400' property="oboInOwl:id" content="IMR:0000517" datatype="xsd:string"></span><span id='am-1401' property="rdfs:label" content="phosphatase" datatype="xsd:string"></span><span id='am-1402' property="oboInOwl:hasExactSynonym" content="phosphoric monoester hydrolase activity" datatype="xsd:string"></span>phosphatase</span> purchased Promega, ligations performed                 chromosome. Colonies seeded 100 ml tryptone soya broth containing <br /> DNA ligation kit (Amersham) instructed manufacturer, sequencing            5 mg chloramphenicol ml wells microtiter plates incubated performed using Sequenase version 2.0 DNA sequencing kit (Amersham).               overnight 288C (27). microtiter plates centrifuged 1,000 3 g 10 <br /> PCRs, optimized using Opti-Prime kit (Stratagene), performed             20 min, supernatant discarded. Pellets resuspended 20 ml <br /> Hybaid OmniGene thermal cycler, using Pfu <span id='am-1062' about='obo:IMR_0010027' typeof='owl:Thing obo:IMR_0000954 obo:IMR_0000949 obo:IMR_0010050 obo:IMR_0000001 obo:IMR_0000207'><span id='am-1063' property="oboInOwl:hasDbXref" content="EC:2.7.7.7" datatype="xsd:string"></span><span id='am-1064' property="rdfs:label" content="DNA polymerase" datatype="xsd:string"></span><span id='am-1065' property="oboInOwl:id" content="IMR:0010027" datatype="xsd:string"></span><span id='am-1066' property="oboInOwl:hasExactSynonym" content="DNA-directed DNA polymerase" datatype="xsd:string"></span><span id='am-1067' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1068' property="oboInOwl:hasExactSynonym" content="DNA nucleotidyltransferase (DNA-directed)" datatype="xsd:string"></span><span id='am-1069' property="oboInOwl:hasDbXref" content="GO:0003887" datatype="xsd:string"></span><span id='am-1070' property="rdfs:label" content="DNA nucleotidyltransferase (DNA-directed)" datatype="xsd:string"></span><span id='am-1071' property="rdfs:label" content="DNA-directed DNA polymerase" datatype="xsd:string"></span>DNA polymerase</span> (Strat-                  50 mM Tris-HCl (pH 7.5) 8211;145 mM NaCl containing 3 U lysostaphin agene) Taq DNA polymerase (Promega), products cloned                  incubated 30 45 min 378C. Cell lysates (2 ml) spotted nylon <br /> using TA-Cloning kit (Invitrogen), pGEM-T cloning kit (Promega),                membranes (Hybond-N1; Amersham), DNA denatured fixed DNA ligation kit (Amersham). Synthetic oligonucleotides Cru-                membrane instructed manufacturer. Membranes hybridized <br /> achem  R<span id='am-526' about='obo:IMR_0702349' typeof='owl:Thing'><span id='am-527' property="rdfs:label" content="MGC125856" datatype="xsd:string"></span><span id='am-528' property="oboInOwl:hasExactSynonym" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-529' property="oboInOwl:hasExactSynonym" content="DKFZp686D13177" datatype="xsd:string"></span><span id='am-530' property="rdfs:label" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-531' property="oboInOwl:hasExactSynonym" content="MGC125857" datatype="xsd:string"></span><span id='am-532' property="oboInOwl:hasExactSynonym" content="AMP" datatype="xsd:string"></span><span id='am-533' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-534' property="oboInOwl:hasExactSynonym" content="MGC125856" datatype="xsd:string"></span><span id='am-535' property="oboInOwl:hasDbXref" content="UniProt:P07741" datatype="xsd:string"></span><span id='am-536' property="oboInOwl:hasExactSynonym" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-537' property="oboInOwl:hasExactSynonym" content="APRT" datatype="xsd:string"></span><span id='am-538' property="oboInOwl:id" content="IMR:0702349" datatype="xsd:string"></span><span id='am-539' property="rdfs:label" content="APT_HUMAN" datatype="xsd:string"></span><span id='am-540' property="oboInOwl:hasExactSynonym" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-541' property="rdfs:label" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-542' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-543' property="oboInOwl:hasExactSynonym" content="MGC129961" datatype="xsd:string"></span><span id='am-544' property="rdfs:label" content="DKFZp686D13177" datatype="xsd:string"></span><span id='am-545' property="rdfs:label" content="MGC129961" datatype="xsd:string"></span><span id='am-546' property="rdfs:label" content="MGC125857" datatype="xsd:string"></span><span id='am-547' property="rdfs:label" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-548' property="rdfs:label" content="APRT" datatype="xsd:string"></span>amp</span>;D Systems Europe  PCR primer sequences, inten-                  spsB gene downstream flanking region labeled using tional differences template sequence shown uppercase,                ECL random prime labeling kit (Amersham) instructed manufacturer. <br /> follows: primer  (g tIgg t)(c/t)tIcc t)gg t)ga t)<span id='am-740' about='obo:IMR_0703320' typeof='owl:Thing'><span id='am-741' property="oboInOwl:hasExactSynonym" content="PGIF" datatype="xsd:string"></span><span id='am-742' property="oboInOwl:hasExactSynonym" content="Arg1" datatype="xsd:string"></span><span id='am-743' property="oboInOwl:hasDbXref" content="UniProt:Q61176" datatype="xsd:string"></span><span id='am-744' property="rdfs:label" content="arginase, liver" datatype="xsd:string"></span><span id='am-745' property="oboInOwl:hasExactSynonym" content="Arg-1" datatype="xsd:string"></span><span id='am-746' property="rdfs:label" content="PGIF" datatype="xsd:string"></span><span id='am-747' property="oboInOwl:hasExactSynonym" content="Arginase-1" datatype="xsd:string"></span><span id='am-748' property="rdfs:label" content="Type I arginase" datatype="xsd:string"></span><span id='am-749' property="rdfs:label" content="EC 3.5.3.1" datatype="xsd:string"></span><span id='am-750' property="oboInOwl:hasExactSynonym" content="EC 3.5.3.1" datatype="xsd:string"></span><span id='am-751' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-752' property="oboInOwl:hasExactSynonym" content="arginase, liver" datatype="xsd:string"></span><span id='am-753' property="rdfs:label" content="Arg-1" datatype="xsd:string"></span><span id='am-754' property="oboInOwl:hasExactSynonym" content="Liver-type arginase" datatype="xsd:string"></span><span id='am-755' property="rdfs:label" content="Arg1" datatype="xsd:string"></span><span id='am-756' property="oboInOwl:hasExactSynonym" content="AI" datatype="xsd:string"></span><span id='am-757' property="oboInOwl:hasExactSynonym" content="Type I arginase" datatype="xsd:string"></span><span id='am-758' property="rdfs:label" content="Liver-type arginase" datatype="xsd:string"></span><span id='am-759' property="rdfs:label" content="Arginase-1" datatype="xsd:string"></span><span id='am-760' property="rdfs:label" content="ARGI1_MOUSE" datatype="xsd:string"></span><span id='am-761' property="rdfs:label" content="AI" datatype="xsd:string"></span><span id='am-762' property="oboInOwl:id" content="IMR:0703320" datatype="xsd:string"></span>aI</span> t)(g t; primer B,       Nucleotide sequence accession number. GenBank accession number  <br /> c(t/g) g)tt g)tc(t cccatIa(t/c) g)aa g)ta; primer C, tgGAaTTCAtgaaaaa          signed nucleotide sequence reported U65000. <br /> agaaCtGttggaatggattatttc; primer D, atttgtAAGCtTttaGtttttGgtGttttcaggattga <br /> aa; primer E, ctgGATCCcgcttgattagttttattga; primer F, ttGGtACCttttgacacctctt <br /> tttaag; primer G, aaGGtACCtatgaaacaaatacaacatc; primer H, atGAAtTCtcaata                                                   RESULTS <br /> taattgtgacactc; primer  atattagagcgataattcc; primer J, gttcatttgctatttcttc. <br />    PCR cycle conditions primer pairs follows: plus B, 5 min 948C, <br /> 30 cycles 1 min 948C, 1 min 428C, 1 min 728C, 5 min 728C; <br />                                                                                            DNA cloning. strategy devised clone SPase PCR <br /> C plus D, 5 min 948C, 15 cycles 1 min 948C, 1 min 508C, 1 min         using primers B  Materials Methods). frag- <br /> 728C, 5 min 728C; E plus F, 5 min 948C, 15 cycles 1 min 948C, 1          ment DNA expected length, determined  5714       CREGG ET AL.                                                                                                                                 J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. DNA encoded protein sequences spsA spsB genes S. aureus WCUH29. strands entire sequence determined oligonucleotide walking. Putative ribosome-binding sites gene underlined, potential <span id='am-1' about='obo:IMR_0000296' typeof='owl:Thing obo:IMR_0000295 obo:IMR_0000949 obo:IMR_0010044 obo:IMR_0000001 obo:IMR_0000914 obo:IMR_0000207 obo:IMR_0000960 obo:IMR_0000288'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Rho" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000296" datatype="xsd:string"></span>rho</span>-independent terminator sequence distal spsB gene. active-site serine 36 lysine 77 residues boldface. <br />  <br />  <br />  <br />  <br /> known sizes SPases Bacillus species, obtained                       regions highest sequence conservation  ex- preparation genomic DNA S. aureus Oxford. 163                          pected, concentrated close sections C (Fig. 2) <br /> bp sequence derived highly homologous 163 bp                          primary structure.  surprising observation sipS gene B. subtilis, implying fragment                        fact serine lysine residue known gene encoding type SPase cloned. PCR                                essential catalytic activity type SPases conserved product labeled used probe lZapII library S.                         SpsA. concluded SpsA, produced aureus WCUH29 genomic DNA order obtain se-                                cell, catalyze hydrolysis peptide bonds quence information. positive clone identified,                         physiologically significant extent. ascertain exis- <br /> recombinant plasmid (pKC10) excised, cut                            tence spsA peculiar WCUH29 strain S. <br /> restriction enzymes, blotted, probed labeled                       aureus, lZapII library DNA isolated S. aureus H DNA fragment (data shown). SalI-digested                             probed probe originally used identify clones  <br /> pKC10 preparation religated form plasmid pKC11.                           taining spsA spsB genes strain WCUH29. Close <br /> DNA sequence 3,093 bp insert DNA determined                             homologs spsA spsB discovered S. <br /> oligonucleotide walking directions                           aureus H; SpsA lacked active-site serine lysine resi- <br /> sequence originally derived PCR. Figure 1 shows 1,220                             dues (data shown). <br /> nucleotides DNA sequence chromosome S.                                   Expression spsB gene E. coli. Despite fact aureus WCUH29, nucleotides 817 1025 represent                            significant expression levels active LPase lepB cloned probe region. probe sequence comprises                             plasmid obtained E. coli (5, 43), protein <br /> potential open reading frame entirety encodes                         functionally expressed B. subtilis. polypeptide 191 amino acid residues calculated mo-                         gene transcribed translated, LPase did assemble <br /> lecular mass 21,692 Da single section hydro-                        correctly membrane detectable activity phobic residues close N terminus probably form                         vitro assays using cell extracts B. subtilis producing E. coli <br /> single transmembrane anchor. small surface-exposed                            LPase enzyme source (36). Conversely, functional ex- <br /> main single transmembrane anchor typical SPases                           pression B. subtilis sipS-encoded SPase E. coli G1 eubacteria. predicted protein high homology                          demonstrated instrumental successful known G1 SPases (data shown), gene                          cloning sipS gene (37). clear work  named spsB (signal peptidase staphylococcus).                        scribed SipS assembled correct orientation E. <br /> noteworthy highest levels sequence similarity                     coli membrane.  activity monitored solely <br /> regions protein corresponding highly                          cleavage artificial engineered preprotein, property served regions known SPases (sections C Fig. 2). regions B. subtilis SPase, LPase E. coli, contain residue critical catalytic activity (3, 33, 38). <br />    second open reading frame, named spsA, proximal spsB gene (separated 15 nucleotides) potentially encodes protein 174 amino acid residues calculated molecular <br /> mass 20,146 Da. Sequence comparison reveals                                                                                         FIG. 2. Alignment sections primary structure signal peptidases <br /> protein similar known SPases. optimized align-                         E. coli (LepB), B. subtilis (SipS), S. aureus (SpsB), SpsA. active-site <br /> ment SpsB SpsA results 62% similarity 31%                             serine lysine residues conserved aspartate residue shown identity sequences (data shown).                             boldface, SpsA residues corresponding positions underlined. <br />  VOL. 178, 1996                                                                                        SIGNAL PEPTIDASE S. AUREUS                          5715 <br />  <br />  <br />  <br />  <br />    FIG. 3. Expression S. aureus SpsB signal peptidase E. coli. Total cell <br /> proteins E. coli equivalent 175 ml cell culture OD540 1.0 loaded SDS 8211;15% (wt/vol) polyacrylamide gel, electrophoresed 45 <br /> min 200 V, stained Coomassie brilliant blue R. Lanes 1 6, <br /> Bio-Rad low-molecular-mass markers (phosphorylase b, 97.4 kDa; serum albu- <br /> min, 66.2 kDa; ovalbumin, 45 kDa; carbonic anhydrase, 31 kDa; trypsin inhibitor, <br /> 21.5 kDa; lysozyme, 14.4 kDa); lane 2, pDB575 IPTG; lane 3, pKC16 IPTG; lane 4, pDB575 IPTG; lane 5, pKC16 IPTG. <br />                                                                                          FIG. 4. Growth curves E. coli IT41(pKC16) (circles) E. coli <br />                                                                                       IT41(pDB575) (squares) 428C. Bacterial colonies seeded L broth <br />                                                                                       containing ampicillin grown 288C overnight. Cell cultures incubated extension central hydrophobic region                           shaking 428C. OD540 recorded 30-min intervals. signal sequence. determine spsB gene functionally expressed E. <br /> coli SPase produced catalyze cleavage signal sequences natural E. coli preproteins.                        similar observed E. coli IT41 transformed spsB gene amplified plasmid pKC11 PCR                                 plasmid overexpresses authentic E. coli LPase using primers C D containing mismatches template                           grown conditions (3).  E. coli <br /> sequence. PCR primer designed effect major                            IT41 known revert wild-type growth character- <br /> changes: introduction restriction site end                        istics, considered necessary ascertain se- <br /> gene facilitate cloning (EcoRI 59 end HindIII                        quence mutant lepB gene changed 39 end) changes bases produced silent                           course experiment shown Fig. 4. sequence codon changes; changes effected minimize                           E. coli IT41 lepB gene previously determined, <br /> restrictions mRNA translation staphylococcal                                necessary determine sequence codons comparatively rare E. coli change                       stocks cells used complementation studies. <br /> TTG initiation codon ATG codon. PCR product                             E. coli IT41 lepB gene amplified PCR <br /> cloned plasmid pDB575 (Table 1) form pKC16.                               genomic DNA isolated sources E. coli IT41 <br /> correct sequence cloned insert confirmed. pKC16                            [E. coli IT41 grown permissive temperature 288C used transform E. coli XL1-Blue ampicillin resis-                           E. coli IT41(pKC16) growth 428C] cloned tance, cells examined production SpsB                           pBluescriptII <span id='am-982' about='obo:IMR_0701097' typeof='owl:Thing'><span id='am-983' property="oboInOwl:hasExactSynonym" content="KCa2.2" datatype="xsd:string"></span><span id='am-984' property="oboInOwl:hasExactSynonym" content="Kcnn2" datatype="xsd:string"></span><span id='am-985' property="oboInOwl:id" content="IMR:0701097" datatype="xsd:string"></span><span id='am-986' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-987' property="oboInOwl:hasExactSynonym" content="SKCa2" datatype="xsd:string"></span><span id='am-988' property="rdfs:label" content="SK2" datatype="xsd:string"></span><span id='am-989' property="oboInOwl:hasExactSynonym" content="potassium intermediate/small conductance calcium-activated channel, subfamily N, member 2" datatype="xsd:string"></span><span id='am-990' property="rdfs:label" content="SKCa2" datatype="xsd:string"></span><span id='am-991' property="rdfs:label" content="Small conductance calcium-activated potassium channel protein 2" datatype="xsd:string"></span><span id='am-992' property="rdfs:label" content="KCa2.2" datatype="xsd:string"></span><span id='am-993' property="oboInOwl:hasExactSynonym" content="SKCa 2" datatype="xsd:string"></span><span id='am-994' property="oboInOwl:hasDbXref" content="UniProt:P70604" datatype="xsd:string"></span><span id='am-995' property="rdfs:label" content="Kcnn2" datatype="xsd:string"></span><span id='am-996' property="oboInOwl:hasExactSynonym" content="SK2" datatype="xsd:string"></span><span id='am-997' property="rdfs:label" content="SKCa 2" datatype="xsd:string"></span><span id='am-998' property="rdfs:label" content="KCNN2_RAT" datatype="xsd:string"></span><span id='am-999' property="rdfs:label" content="potassium intermediate/small conductance calcium-activated channel, subfamily N, member 2" datatype="xsd:string"></span><span id='am-1000' property="oboInOwl:hasExactSynonym" content="Small conductance calcium-activated potassium channel protein 2" datatype="xsd:string"></span>SK2</span>. sequences PCR products induction IPTG. Cultures grown mid-log phase,                            determined identical. case, induced, harvested 2.5 h postinduction. Cell lysates                         single <span id='am-763' about='obo:IMR_0100817' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000429 obo:IMR_0000428 obo:IMR_0001014'><span id='am-764' property="rdfs:label" content="base" datatype="xsd:string"></span><span id='am-765' property="oboInOwl:hasDbXref" content="GO:0008540" datatype="xsd:string"></span><span id='am-766' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-767' property="oboInOwl:id" content="IMR:0100817" datatype="xsd:string"></span>base</span> change compared wild-type E. coli lepB <br /> analyzed SDS-PAGE (Fig. 3). clear significant                           sequence apparent. base change E. coli IT41 <br /> quantities SpsB produced E. coli(pKC16)                            lepB gene case C3T transition nucleotide absence induction SpsB expression IPTG.                             position 115 lepB coding sequence, mutation increase stain density (lanes 3 5) compared                           changes glutamine codon (Gln-39 LPase) TAG amber <br /> control (lanes 2 4) position corresponding protein                      termination codon. demonstrated se- Mr ;21,000 apparent (lanes 4 5)                           quence E. coli IT41 lepB gene changed (lanes 2 3) IPTG induction.                                           course experiment shown Fig. 4,  <br />    Experiments carried determine SpsB                          cluded SpsB complements growth E. coli pre- <br /> functionally active E. coli. E. coli IT41 single mutant                      protein-processing Ts mutant IT41 vivo. conclusion <br /> copy lepB gene temperature sensitive (Ts)                           infers SpsB assembles correctly E. coli plasma mem- <br /> preprotein processing (14). strain used exten-                           brane catalyzes processing essential E. coli <br /> sively investigations LPase function activity,                       preproteins. <br /> rate preprotein processing rate cell division                              Essentiality spsB gene S. aureus. known severely compromised 428C restored comple-                           B. subtilis B. amyloliquefaciens possess mentation plasmid-borne active LPase (3, 4, 14).                       SPase genes, determine S. aureus noted unambiguous correlation E. coli IT41                         second gene encoding active SPase. Attempts <br /> growth rate rate preprotein processing demon-                           delete spsB gene chro- <br /> strably apparent experimental conditions (3, 4, 31,                       mosome S. aureus WCUH29 double-crossover homolo- <br /> 33).                                                                                  gous recombination using plasmid pTS1, contains    E. coli IT41 transformed pKC16                           chloramphenicol resistance determinant active S. aureus, trol plasmid pDB575. Growth E. coli IT41(pDB575) E.                            Ts G1 replication function derived mutant pE194 <br /> coli IT41(pKC16) 428C monitored OD540                                (41), pUC18, enabling replication selection E. coli <br /> period 8 h (Fig. 4) absence IPTG                             gift T. Foster, Trinity College, Dublin, Ireland). possible toxic effect high level SpsB production).                      general strategy details method described substantial increase growth rate E. coli                               E. coli Hamilton et al. (10) depicted Fig. 5. IT41(pKC16) compared E. coli IT41(pDB575), indicating                            essence method upstream downstream  <br /> functional expression SpsB. generation time 1.5 h                        gions DNA flanking gene cloned  5716       CREGG ET AL.                                                                                                                                J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                                          FIG. 6. Southern blot S. aureus chromosomal DNA fragments isolated                                                                                       PCR probed spsB-specific probe probe specific                                                                                       downstream DNA fragment. Chromosomal DNA prepared S. aureus <br />                                                                                       WCUH29 (lane 1), S. aureus(pIM11) (lane 2), 10 chloramphenicol-sensitive <br />                                                                                       colonies (lanes 3 12) amplified using primers corresponding se- <br />                                                                                       quences upstream downstream flanking regions, products <br />                                                                                       separated agarose gel electrophoresis blotted nylon mem- <br />    FIG. 5. Strategy determination essentiality spsB S. aureus.   branes. identical membranes prepared hybridized labeled <br /> U D, upstream downstream regions, respectively, flanking spsB; Cm,            oligonucleotide corresponding spsB   downstream flanking <br /> chloramphenicol resistance determinant; ori, G2 origin replication; ori-       region (B) described Materials Methods. Expected fragment sizes Ts, G1 Ts origin.                                                                     2,554 bp spsB 1,979 bp spsB. Fluorescein-labeled DNA markers <br />                                                                                       (lDNA-HindIII) 23.1, 9.4, 6.6, 4.4, 2.3, 2.0 kbp loaded outside <br />                                                                                       lanes. <br />  <br /> plasmid Ts replication function. Growth nonper- <br /> missive temperature selects integration homologous                          icol-sensitive colonies, cases plasmid isolated combination (Campbell-type integration), subsequent                               chloramphenicol-resistant colonies possessed wild-type <br /> growth cointegrants permissive temperature leads                         spsB. explanations account appearance second recombination event resulting plasmid resolution.                         chloramphenicol-sensitive colonies derived S. aureus- event takes place upstream region                         (pIM11). spsB essential gene; second event takes place downstream region, vice                            pIM11 recombined chromosome, plasmid <br /> versa, gene transferred                           resolved, depositing spsB chromosome, subse- <br /> plasmid chromosome. gene dis-                          quently lost cell. distinguish pensable, recombinant plasmid subsequently                            possibilities, genomic DNA isolated 10 colonies, <br /> cured cell raising temperature,                        S. aureus(pIM11), S. aureus WCUH29, gene essential, plasmid cured  cured                       oligonucleotide PCR primers J used amplify recombination events reverse overall process                      chromosomal DNA therefrom. primers correspond  <br /> leading replacement gene chromosome).                             gions chromosomal DNA lay upstream  <br /> gions DNA upstream downstream spsB                                 stream sequences cloned pIM11, precluding <br /> gene, 850 bp  amplified PCR plasmid                                 possibility amplification undetected plasmid-borne <br /> pKC11 using primer pairs E-F G-H, respectively.                          target sequences. DNA fragments derived size frac- <br /> pairs generate terminal restriction enzyme sites facilitate                        tionated agarose gel blotted nylon membranes. <br /> subsequent cloning steps. PCR products cloned                            identical membranes prepared probed ligated KpnI site pUC18 (pIM04), DNA se-                             labeled oligonucleotide recognized spsB order <br /> quence ascertained ensure errors                              confirm presence absence spsB labeled <br /> incorporated amplification. 1,700-bp fragment                          oligonucleotide bound downstream flanking  <br /> subcloned multiple cloning site pTS1 using                             gion. results presented Fig. 6 indicate BamHI-EcoRI sites, construct, pIM05, trans-                               cases plasmid pIM11 lost S. aureus formed E. coli S. aureus RN4220 S.                              effecting transfer spsB chromosome, aureus WCUH29 electroporation. appropriate tem-                              amplified fragments size (2,554 bp) perature cycling select integration homologous                           amplified S. aureus WCUH29 DNA products <br /> combination chromosome plasmid resolution                                hybridized probes spsB downstream <br /> therefrom, cell lysates individual colonies prepared                        flanking region. product obtained amplifying chromo- screened presence plasmids bearing copy                           somal DNA S. aureus(pIM11) smaller (1,979 bp) spsB derived chromosome hybridization probe                          did hybridize probe spsB, confirming spsB spsB. clone isolated (pIM11), identity                       deleted chromosome chromosomal confirmed restriction analysis Southern blotting                           plasmid DNA amplified. data imply <br /> (data shown). clone subjected temperature                             type SPase encoded spsB gene indis- <br /> cycling 438C attempt cure plasmid cell.                      pensable role preprotein processing S. aureus Cells plated absence chloramphenicol, 146                          activity essential viability S. aureus. control <br /> colonies replica plated plates chlor-                         experiment, S. aureus WCUH29 transformed plasmid <br /> amphenicol. Seventy colonies grew presence                         pIM11 subjected temperature cycling replica plat- <br /> chloramphenicol, remainder showed poor                           ing described  Plasmid-borne spsB readily growth. plasmid isolated chloramphen-                            removed spsB merodiploids. <br />  VOL. 178, 1996                                                                       SIGNAL PEPTIDASE S. AUREUS                     5717 <br />  <br />  <br />                           DISCUSSION                                   b-lactamase changed glycine codons GGC GGA                                                                        order produce catalytically inactive protein.    Evidence accumulated suggest LPase belongs           case (GGC; base change), small amounts b-lactamase new class serine <span id='am-49' about='obo:IMR_0000304' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010044 obo:IMR_0000001 obo:IMR_0000207'><span id='am-50' property="oboInOwl:hasExactSynonym" content="peptidase" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="GO:0008233" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="EC:3.4.-.-" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="protease" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="peptidase" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0000304" datatype="xsd:string"></span>protease</span> does utilize histidine       activity equivalent ;0.1% wild-type levels detected, catalytic base (4, 31) instead employ lysine       case (GGA; base changes), activity <br /> chain fulfill role (3, 33). observations com-        detected. Evidence presented indicated G wob- <br /> parisons LexA E. coli led proposal             ble seryl-tRNA position codon-anti- <br /> serine-lysine catalytic dyad, similar thought operate       codon interaction responsible limited peptide bond hydrolysis catalyzed LexA (30),             mistranslation, resulting production small pool wild- <br /> operate LPase (3). Similar observations             type serine-containing enzyme. similar phenomenon SPase B. subtilis (38) Tsp periplas-         noted authors (15). data presented mic protease E. coli (16); similarities SipS LexA       strongly suggestive similar mechanism production suggested extend regions primary            LPase activity E. coli IT41 Ts growth pre- <br /> structure (38). serine lysine residues (90 145,            protein processing phenotype, likely mis- <br /> respectively, E. coli LPase numbering) known critical         translation occurs rate higher 0.1% case, catalytic activity E. coli LPase (3, 33) B. subtilis   wild-type levels LPase E. coli estimated SPase (38) thought form catalytic dyad               region 1,000 molecules cell (40). <br /> conserved S. aureus protein SpsB (S-36 K-77).               discovery gene (spsA) encoding homolog type addition, aspartate position 155 (280 E. coli LPase          SPases immediately upstream spsB gene novel  <br /> numbering) conserved  residue appears important           ing bacteria. short spacing (15 bases) 39 activity SipS SPase [38] LPase E.      end spsA 59 end spsB implies cotranscription <br /> coli [31]). observations, described       genes  possible cryptic promoter <br /> Results, indicate gene encoding type SPase, spsB,       spsB transcription resides 39-terminal coding cloned S. aureus.                                            region spsA). characteristic spsA gene reminiscent <br />    expressed plasmid E. coli control           genes eukaryotes encode pro- E. coli tac promoter, gene produces active protein,          teins key catalytic residues replaced, <br /> judged complementation activity E. coli strain        leading catalytically inactive proteins. Examples serine <br /> Ts preprotein processing. known positive correlation           protease homologs catalytically inactive result E. coli IT41 growth rate catalytic activity         amino acid replacements active site include haptoglobin <br /> plasmid-borne SPase nonpermissive temperature (3, 4,            (18), protein (13), <span id='am-192' about='obo:IMR_0000033' typeof='owl:Thing obo:IMR_0000030 obo:IMR_0000949 obo:IMR_0000002 obo:IMR_0000001'><span id='am-193' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-194' property="rdfs:label" content="HGF" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasExactSynonym" content="hepatocyte growth factor" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasExactSynonym" content="scatter factor" datatype="xsd:string"></span><span id='am-197' property="oboInOwl:hasDbXref" content="GO:0005171" datatype="xsd:string"></span><span id='am-198' property="rdfs:label" content="hepatocyte growth factor" datatype="xsd:string"></span><span id='am-199' property="rdfs:label" content="scatter factor" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:id" content="IMR:0000033" datatype="xsd:string"></span>hepatocyte growth factor</span> (24), human <br /> 31, 33) renders direct assay processing activity unnecessary        heparin-binding protein (26). active-site mutations experiments. Sequence determination mutant             proteins S3A, H3K; S3A, H3T; S3Y, H3Q; <br /> lepB gene host strain complemen-           S3G, H3S, respectively. common feature tation experiment demonstrated acceleration E.         proteins appears retained, presumably coli IT41 cell growth rate reversion Ts          result retention overall folding topology, capacity phenotype lepB functional              bind hydrolyze substrates detectable level. expression catalytically active SpsB. established       likely SpsA falls general class vitro assays LPase activity considerably sen-      molecule bind preproteins cleave signal <br /> sitive vivo assay LPase adapted        peptides therefrom. demonstrated SpsB work estimation SpsB activity. data indicate         function effectively SPase absence SpsA E. proteins normally processed LPase,        coli, role   SpsA protein targeting translo- <br /> essential E. coli, processed SpsB. protein          cation S. aureus difficult define stage. likely, <br /> sequence deduced sequencing E. coli IT41 lepB               SpsA persisted nonhydrolyzing enzyme <br /> gene intriguing. clear protein fragment 38          considerable time S. aureus, active-site mutations <br /> amino acid residues (essentially transmembrane anchor)             (S3D K3S) necessarily involve changes predicted arise C3T transition nucleotide po-           nucleotides codon, transitional forms inactive <br /> sition 115 lepB coding sequence responsible           enzyme preceded formation SpsA preprotein processing E. coli IT41 288C, alter-       nonaspartyl replacement active-site serine non- <br /> native explanation sought. likely explanation         seryl replacement active-site lysine. small quantities authentic LPase produced E.            Finally, data presented imply SpsB represents <br /> coli IT41 result misreading amber codon.            essential enzymatic activity S. aureus. possible resultant cellular concentration LPase insufficient sup-      transfer spsB gene chromosome plasmid, port normal rates preprotein processing (preproteins            possible eradicate spsB gene seen accumulate permissive temperature [14]),           cell, possible remove plasmid-borne cell growth rates sufficiently high enable experiments      spsB S. aureus spsB merodiploids. observation performed. likely 428C rate protein         marked contrast situation Bacillus genus: targeting plasma membrane considerably greater          apparent cases, nonidentical rate 288C insufficiency LPase activity       SPase genes coexist B. subtilis  possibly considerably apparent higher temperature;               bacteria SPase), SPase gene noted activity zero 428C           deleted. <br /> (14). E. coli IT41 parent strain E. coli W3110 (14), nonsuppressing strain, behavior E. coli                                 ACKNOWLEDGMENTS <br /> IT41 428C explained presence Ts                 gratefully acknowledge gifts pTS1 T. Foster, <br /> amber suppressor phenotype. Cases mistranslation pro-            pDB575 M. Lawrence D. <span id='am-340' about='obo:IMR_0200755' typeof='owl:Thing'><span id='am-341' property="oboInOwl:hasExactSynonym" content="UDP-galactose-4-epimerase" datatype="xsd:string"></span><span id='am-342' property="rdfs:label" content="UDP-galactose 4-epimerase" datatype="xsd:string"></span><span id='am-343' property="oboInOwl:hasExactSynonym" content="UDP-glucose 4-epimerase" datatype="xsd:string"></span><span id='am-344' property="rdfs:label" content="SDR1E1" datatype="xsd:string"></span><span id='am-345' property="rdfs:label" content="EC 5.1.3.2" datatype="xsd:string"></span><span id='am-346' property="rdfs:label" content="FLJ97302" datatype="xsd:string"></span><span id='am-347' property="rdfs:label" content="GALE_HUMAN" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasDbXref" content="UniProt:Q14376" datatype="xsd:string"></span><span id='am-350' property="rdfs:label" content="GALACTOSE EPIMERASE" datatype="xsd:string"></span><span id='am-351' property="oboInOwl:hasExactSynonym" content="FLJ95174" datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasExactSynonym" content="UDP-galactose 4-epimerase" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasAlternativeId" content="IMR:0702471" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasExactSynonym" content="FLJ97302" datatype="xsd:string"></span><span id='am-355' property="oboInOwl:hasExactSynonym" content="SDR1E1" datatype="xsd:string"></span><span id='am-356' property="rdfs:label" content="GALE" datatype="xsd:string"></span><span id='am-357' property="rdfs:label" content="Galactowaldenase" datatype="xsd:string"></span><span id='am-358' property="oboInOwl:id" content="IMR:0200755" datatype="xsd:string"></span><span id='am-359' property="oboInOwl:hasExactSynonym" content="Galactowaldenase" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasExactSynonym" content="GALE" datatype="xsd:string"></span><span id='am-361' property="rdfs:label" content="FLJ95174" datatype="xsd:string"></span><span id='am-362' property="rdfs:label" content="UDP-glucose 4-epimerase" datatype="xsd:string"></span><span id='am-363' property="oboInOwl:hasExactSynonym" content="GALACTOSE EPIMERASE" datatype="xsd:string"></span><span id='am-364' property="rdfs:label" content="UDP-galactose-4-epimerase" datatype="xsd:string"></span><span id='am-365' property="oboInOwl:hasExactSynonym" content="EC 5.1.3.2" datatype="xsd:string"></span>Gale</span>, E. coli IT41 Y. <br /> tein coding regions mutagenesis experiments               Nakamura. thank M. J. Browne bringing paper hepa- <br /> noted (32). case, serine (<span id='am-574' about='obo:IMR_0702178' typeof='owl:Thing'><span id='am-575' property="oboInOwl:hasExactSynonym" content="Cartilage-specific proteoglycan core protein" datatype="xsd:string"></span><span id='am-576' property="rdfs:label" content="PGCA_RAT" datatype="xsd:string"></span><span id='am-577' property="oboInOwl:hasExactSynonym" content="aggrecan" datatype="xsd:string"></span><span id='am-578' property="oboInOwl:hasExactSynonym" content="Aggrecan core protein" datatype="xsd:string"></span><span id='am-579' property="oboInOwl:hasDbXref" content="UniProt:P07897" datatype="xsd:string"></span><span id='am-580' property="oboInOwl:id" content="IMR:0702178" datatype="xsd:string"></span><span id='am-581' property="rdfs:label" content="Acan" datatype="xsd:string"></span><span id='am-582' property="oboInOwl:hasExactSynonym" content="Agc" datatype="xsd:string"></span><span id='am-583' property="oboInOwl:hasExactSynonym" content="CSPCP" datatype="xsd:string"></span><span id='am-584' property="rdfs:label" content="CSPCP" datatype="xsd:string"></span><span id='am-585' property="rdfs:label" content="Agc1" datatype="xsd:string"></span><span id='am-586' property="oboInOwl:hasExactSynonym" content="Acan" datatype="xsd:string"></span><span id='am-587' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-588' property="oboInOwl:hasExactSynonym" content="Agc1" datatype="xsd:string"></span><span id='am-589' property="rdfs:label" content="Aggrecan core protein" datatype="xsd:string"></span><span id='am-590' property="rdfs:label" content="Cartilage-specific proteoglycan core protein" datatype="xsd:string"></span><span id='am-591' property="rdfs:label" content="aggrecan" datatype="xsd:string"></span><span id='am-592' property="rdfs:label" content="Agc" datatype="xsd:string"></span>AGC</span>) codon             tocyte <span id='am-201' about='obo:IMR_0000030' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000002 obo:IMR_0000001'><span id='am-202' property="obo:IAO_0000115" content="Extracellular polypeptide signal molecule that can stimulate a cell grow or proliferate. Examples are EGF and PDGF. Most growth factors also have other actions." datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:id" content="IMR:0000030" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasDbXref" content="GO:0008083" datatype="xsd:string"></span><span id='am-206' property="rdfs:label" content="growth factor" datatype="xsd:string"></span><span id='am-207' property="oboInOwl:hasExactSynonym" content="Growth Substance" datatype="xsd:string"></span><span id='am-208' property="rdfs:label" content="Growth Substance" datatype="xsd:string"></span>growth factor</span> (24) attention. <br />  5718       CREGG ET AL.                                                                                                                                    J. BACTERIOL. <br />  <br />  <br />                                  REFERENCES                                                 Bron, J. M. van Dijl. 1995. endogenous B. subtilis (natto) plasmids <br />  1. Akagawa, E., K. Kurita, T. Sugawara, K. Nakamura, Y. Kasahara, N. Oga-                  pTA1015 pTA1040 contain signal peptidase-encoding genes: identifica- <br />     sawara, K. Yamane. 1995. Determination 17848 bp nucleotide                     tion new structural module cryptic plasmids. Mol. Microbiol. 17:621 8211; <br />     sequence 398 region Bacillus subtilis chromosome                  631. <br />     similarity analysis products putative ORFs. Microbiology 141:3241 8211;      22.   Mu  168;ller, P., K. Ahrens,  Klaucke. 1995. TnphoA insertion     3245.                                                                                   Bradyrhizobium japonicum sipS gene, homologous prokaryotic signal <span id='am-844' about='obo:IMR_0704046' typeof='owl:Thing'><span id='am-845' property="rdfs:label" content="LyP" datatype="xsd:string"></span><span id='am-846' property="oboInOwl:id" content="IMR:0704046" datatype="xsd:string"></span><span id='am-847' property="oboInOwl:hasExactSynonym" content="LyP" datatype="xsd:string"></span><span id='am-848' property="rdfs:label" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-849' property="rdfs:label" content="PROTEIN-TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE 8, FORMERLY" datatype="xsd:string"></span><span id='am-850' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-851' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein phosphatase non-receptor type 22" datatype="xsd:string"></span><span id='am-852' property="rdfs:label" content="Lyp1" datatype="xsd:string"></span><span id='am-853' property="oboInOwl:hasDbXref" content="UniProt:Q9Y2R2" datatype="xsd:string"></span><span id='am-854' property="oboInOwl:hasExactSynonym" content="LYP2" datatype="xsd:string"></span><span id='am-855' property="oboInOwl:hasExactSynonym" content="Lymphoid phosphatase" datatype="xsd:string"></span><span id='am-856' property="oboInOwl:hasExactSynonym" content="PTPN8" datatype="xsd:string"></span><span id='am-857' property="oboInOwl:hasExactSynonym" content="PTPN8, FORMERLY" datatype="xsd:string"></span><span id='am-858' property="oboInOwl:hasExactSynonym" content="PROTEIN-TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE 8, FORMERLY" datatype="xsd:string"></span><span id='am-859' property="oboInOwl:hasExactSynonym" content="Hematopoietic cell protein-tyrosine phosphatase 70Z-PEP" datatype="xsd:string"></span><span id='am-860' property="rdfs:label" content="PTN22_HUMAN" datatype="xsd:string"></span><span id='am-861' property="rdfs:label" content="Tyrosine-protein phosphatase non-receptor type 22" datatype="xsd:string"></span><span id='am-862' property="rdfs:label" content="protein tyrosine phosphatase, non-receptor type 22 (lymphoid)" datatype="xsd:string"></span><span id='am-863' property="rdfs:label" content="PTPN8" datatype="xsd:string"></span><span id='am-864' property="rdfs:label" content="PTPN22" datatype="xsd:string"></span><span id='am-865' property="oboInOwl:hasExactSynonym" content="Lyp2" datatype="xsd:string"></span><span id='am-866' property="rdfs:label" content="PEST-DOMAIN PHOSPHATASE" datatype="xsd:string"></span><span id='am-867' property="rdfs:label" content="Lymphoid phosphatase" datatype="xsd:string"></span><span id='am-868' property="rdfs:label" content="PROTEIN TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE, 22" datatype="xsd:string"></span><span id='am-869' property="rdfs:label" content="LYP2" datatype="xsd:string"></span><span id='am-870' property="rdfs:label" content="Hematopoietic cell protein-tyrosine phosphatase 70Z-PEP" datatype="xsd:string"></span><span id='am-871' property="rdfs:label" content="PEP" datatype="xsd:string"></span><span id='am-872' property="oboInOwl:hasExactSynonym" content="PEP" datatype="xsd:string"></span><span id='am-873' property="oboInOwl:hasExactSynonym" content="PEST-DOMAIN PHOSPHATASE" datatype="xsd:string"></span><span id='am-874' property="oboInOwl:hasExactSynonym" content="protein tyrosine phosphatase, non-receptor type 22 (lymphoid)" datatype="xsd:string"></span><span id='am-875' property="oboInOwl:hasExactSynonym" content="PROTEIN TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE, 22" datatype="xsd:string"></span><span id='am-876' property="oboInOwl:hasExactSynonym" content="PTPN22" datatype="xsd:string"></span><span id='am-877' property="oboInOwl:hasExactSynonym" content="Lyp1" datatype="xsd:string"></span><span id='am-878' property="rdfs:label" content="Lyp" datatype="xsd:string"></span><span id='am-879' property="oboInOwl:hasExactSynonym" content="LYP" datatype="xsd:string"></span><span id='am-880' property="oboInOwl:hasExactSynonym" content="Lyp" datatype="xsd:string"></span><span id='am-881' property="rdfs:label" content="PTPN8, FORMERLY" datatype="xsd:string"></span><span id='am-882' property="rdfs:label" content="LYP" datatype="xsd:string"></span><span id='am-883' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-884' property="rdfs:label" content="Lyp2" datatype="xsd:string"></span><span id='am-885' property="oboInOwl:hasExactSynonym" content="LYP1" datatype="xsd:string"></span><span id='am-886' property="rdfs:label" content="LYP1" datatype="xsd:string"></span>pep</span>- <br />  2. Allsop,  E., G. Brooks, G. Bruton, S. Coulton, P. D. Edwards,  K. Hatton,           tidases, results extensive changes expression PMB-specific nodu- <br />      C. Kaura, S. D. McLean, N. D. Pearson, T. Smale, R. Southgate. 1995.             lins infected soybean (Glycine max) cells. Mol. Microbiol. 18:831 8211;840. <br />     Penem inhibitors bacterial signal peptidase. Bioorg. <span id='am-113' about='obo:IMR_0701745' typeof='owl:Thing'><span id='am-114' property="rdfs:label" content="sodium channel, voltage-gated, type VIII, alpha" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Nav1.6" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasExactSynonym" content="motor end-plate disease" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasExactSynonym" content="nur14" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasExactSynonym" content="nmf58" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="ataxia 3" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="Sodium channel protein type 8 subunit alpha" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasDbXref" content="UniProt:Q9WTU3" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="SCN8A_MOUSE" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="nmf2" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="mnd2" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasExactSynonym" content="NMF335" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type VIII subunit alpha" datatype="xsd:string"></span><span id='am-127' property="rdfs:label" content="med" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="nur14" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasExactSynonym" content="NaCh6" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="NMF335" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="Voltage-gated sodium channel subunit alpha Nav1.6" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasExactSynonym" content="sodium channel, voltage-gated, type VIII, alpha" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasExactSynonym" content="Voltage-gated sodium channel subunit alpha Nav1.6" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type 8 subunit alpha" datatype="xsd:string"></span><span id='am-135' property="rdfs:label" content="nmf335" datatype="xsd:string"></span><span id='am-136' property="rdfs:label" content="seal" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasExactSynonym" content="Scn8a" datatype="xsd:string"></span><span id='am-138' property="rdfs:label" content="NaCh6" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="motor end-plate disease" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasExactSynonym" content="mnd-2" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="Sodium channel protein type VIII subunit alpha" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:id" content="IMR:0701745" datatype="xsd:string"></span><span id='am-143' property="rdfs:label" content="Scn8a" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasExactSynonym" content="med" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasExactSynonym" content="nmf335" datatype="xsd:string"></span><span id='am-146' property="rdfs:label" content="Nbna1" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasExactSynonym" content="Nbna1" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-149' property="rdfs:label" content="nmf58" datatype="xsd:string"></span><span id='am-150' property="rdfs:label" content="mnd-2" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasExactSynonym" content="Nav1.6" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasExactSynonym" content="seal" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasExactSynonym" content="nmf2" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasExactSynonym" content="ataxia 3" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasExactSynonym" content="mnd2" datatype="xsd:string"></span>Med</span>. Chem. Lett.          23.   Mu  168;ller, P.,  Klaucke, E. Wegel. 1995. TnphoA-induced symbiotic <br />     5:443 8211;448.                                                                              mutants Bradyrhizobium japonicum impair cell tissue differenti- <br />  3. Black, M. T. 1993. Evidence catalytic activity prokaryote leader            ation Glycine max nodules. Planta 197:163 8211;175. <br />     peptidase depends operation serine-lysine catalytic dyad. J.        24.   Nakamura, T., T. Nishizawa, M. Hagiya, T. Seki, M. Shimonishi,  <br />     Bacteriol. 175:4957 8211;4961.                                                               Sugimura, K. Tashire, S. Shimizu. 1989. Molecular cloning expres- <br />  4. Black, M. T., J. G. R. Munn,  E. Allsop. 1992. catalytic                   sion human <span id='am-366' about='obo:IMR_0700057' typeof='owl:Thing'><span id='am-367' property="rdfs:label" content="HGFB" datatype="xsd:string"></span><span id='am-368' property="rdfs:label" content="Hepatocyte growth factor alpha chain" datatype="xsd:string"></span><span id='am-369' property="rdfs:label" content="Scatter factor" datatype="xsd:string"></span><span id='am-370' property="oboInOwl:id" content="IMR:0700057" datatype="xsd:string"></span><span id='am-371' property="rdfs:label" content="HGF" datatype="xsd:string"></span><span id='am-372' property="oboInOwl:hasExactSynonym" content="LUNG FIBROBLAST-DERIVED MITOGEN" datatype="xsd:string"></span><span id='am-373' property="rdfs:label" content="SF" datatype="xsd:string"></span><span id='am-374' property="rdfs:label" content="DFNB39" datatype="xsd:string"></span><span id='am-375' property="oboInOwl:hasExactSynonym" content="Hepatocyte growth factor" datatype="xsd:string"></span><span id='am-376' property="oboInOwl:hasExactSynonym" content="HPTA" datatype="xsd:string"></span><span id='am-377' property="oboInOwl:hasExactSynonym" content="DFNB39" datatype="xsd:string"></span><span id='am-378' property="rdfs:label" content="HEPATOPOIETIN A" datatype="xsd:string"></span><span id='am-379' property="oboInOwl:hasExactSynonym" content="Hepatopoeitin-A" datatype="xsd:string"></span><span id='am-380' property="rdfs:label" content="Hepatocyte growth factor beta chain" datatype="xsd:string"></span><span id='am-381' property="rdfs:label" content="HPTA" datatype="xsd:string"></span><span id='am-382' property="oboInOwl:hasExactSynonym" content="HGF" datatype="xsd:string"></span><span id='am-383' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-384' property="oboInOwl:hasExactSynonym" content="SF" datatype="xsd:string"></span><span id='am-385' property="oboInOwl:hasExactSynonym" content="hepatocyte growth factor (hepapoietin A; scatter factor)" datatype="xsd:string"></span><span id='am-386' property="oboInOwl:hasDbXref" content="UniProt:P14210" datatype="xsd:string"></span><span id='am-387' property="oboInOwl:hasExactSynonym" content="Hepatocyte growth factor alpha chain" datatype="xsd:string"></span><span id='am-388' property="oboInOwl:hasExactSynonym" content="Hepatocyte growth factor beta chain" datatype="xsd:string"></span><span id='am-389' property="oboInOwl:hasExactSynonym" content="HGFB" datatype="xsd:string"></span><span id='am-390' property="rdfs:label" content="LUNG FIBROBLAST-DERIVED MITOGEN" datatype="xsd:string"></span><span id='am-391' property="rdfs:label" content="Hepatocyte growth factor" datatype="xsd:string"></span><span id='am-392' property="rdfs:label" content="Hepatopoeitin-A" datatype="xsd:string"></span><span id='am-393' property="rdfs:label" content="hepatocyte growth factor (hepapoietin A; scatter factor)" datatype="xsd:string"></span><span id='am-394' property="oboInOwl:hasExactSynonym" content="Scatter factor" datatype="xsd:string"></span><span id='am-395' property="rdfs:label" content="HGF_HUMAN" datatype="xsd:string"></span><span id='am-396' property="rdfs:label" content="F-TCF" datatype="xsd:string"></span><span id='am-397' property="oboInOwl:hasExactSynonym" content="F-TCF" datatype="xsd:string"></span><span id='am-398' property="oboInOwl:hasExactSynonym" content="HEPATOPOIETIN A" datatype="xsd:string"></span>hepatocyte growth factor</span>. Nature (London) 342:440 8211;443. <br />     mechanism prokaryotic leader peptidase 1. Biochem. J. 282:539 8211;543.             25.   Packer, J. C., D. Andre, C. J. Howe. 1995. Cloning sequence analysis <br />  4a.Cregg, K. M., M. T. Black. Unpublished data.                                        signal peptidase thermophilic cyanobacterium Phormidium <br />  5. Dalbey, R. E., W. Wickner. 1985. Leader peptidase catalyzes release             laminosum. Plant Mol. Biol. 27:199 8211;204. <br />     exported proteins outer surface Escherichia coli plasma        26.   Petersen, L. C., J. J. Birktoft, H. Flodgaard. 1993. Binding bovine <br />     membrane. J. Biol. Chem. 260:15925 8211;15931.                                               pancreatic trypsin inhibitor heparin binding protein/CAP37/azurocidin. <br />  6. <span id='am-768' about='oboInOwl:date' typeof='owl:Thing'>Date</span>, T. 1983. Demonstration novel genetic technique leader                   Interaction Kunitz-type inhibitor proteolytically inactive <br />     peptidase essential enzyme Escherichia coli. J. Bacteriol. 154:76 8211;83.          serine proteinase homologue. Eur. J. Biochem. 214:271 8211;279. <br />  7. Devereux, J., P. Haeberli, O. Smithies. 1984. comprehensive set          27.   Rifai, S., V. Barbancon, G. Prevost, Y. Piemont. 1989. Synthetic exfoli- <br />     sequence analysis programs VAX. Nucleic Acids Res. 12:387 8211;395.                  ative <span id='am-916' about='obo:IMR_0100467' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000002 obo:IMR_0000001'><span id='am-917' property="obo:IAO_0000115" content="Specific, characterizable, poisonous chemicals, often proteins, with specific biological properties, including immunogenicity, produced by microbes, higher plants, or animals." datatype="xsd:string"></span><span id='am-918' property="rdfs:label" content="toxin" datatype="xsd:string"></span><span id='am-919' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-920' property="oboInOwl:id" content="IMR:0100467" datatype="xsd:string"></span>toxin</span> B DNA probes detection toxigenic Staphylococcus <br />  8. Fleischmann, R. D., M. D. Adams, O. White, R.  Clayton, E. F. Kirkness,               aureus strains. J. Clin. Microbiol. 27:504 8211;506. <br />      R. Kerlavage, C. J. Bult, J.-F. Tomb, B.  Dougherty, J. M. Merrick, K.       28.   Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular cloning:     McKenney, G. Sutton, W. FitzHugh, C. Fields, J. D. Gocayne, J. Scott, R.                laboratory manual, 2nd ed. Cold Spring Harbor Laboratory Press, Cold <br />     Shirley, L.  Liu,  Glodek, J. M. Kelly, J. F. Weidman, C.  Phillips, T.           Spring Harbor, N.Y. <br />     Spriggs, E. Hedblom, M. D. Cotton, T. R. Utterback, M. C. Hanna, D. T.            29.   Schenk, S., R.  Laddaga. 1992. Improved method electroporation <br />     Nguyen, D. M. Saudek, R. C. Brandon, L. D. Fine, J. L. Fritchman, J. L.                 Staphylococcus aureus. FEMS Microbiol. Lett. 94:133 8211;138. <br />     Fuhrmann, N. S. M. Geoghagen, C. L. Gnehm, L.  McDonald, K. V. Small,           30.   Slilaty, S. N., J. Little. 1987. Lysine-156 serine-119 required     C. M. Fraser, H. O. Smith, J. C. Venter. 1995. genome random                  LexA repressor cleavage: possible mechanism. <span id='am-831' about='obo:IMR_0010425' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010045 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0000952'><span id='am-832' property="rdfs:label" content="P5CR" datatype="xsd:string"></span><span id='am-833' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-834' property="oboInOwl:hasExactSynonym" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-835' property="rdfs:label" content="PROC" datatype="xsd:string"></span><span id='am-836' property="rdfs:label" content="P5C reductase" datatype="xsd:string"></span><span id='am-837' property="rdfs:label" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-838' property="oboInOwl:hasExactSynonym" content="P5CR" datatype="xsd:string"></span><span id='am-839' property="rdfs:label" content="Pyrroline-5-carboxylate reductase" datatype="xsd:string"></span><span id='am-840' property="oboInOwl:id" content="IMR:0010425" datatype="xsd:string"></span><span id='am-841' property="oboInOwl:hasExactSynonym" content="P5C reductase" datatype="xsd:string"></span><span id='am-842' property="oboInOwl:hasDbXref" content="EC:1.5.1.2" datatype="xsd:string"></span><span id='am-843' property="oboInOwl:hasExactSynonym" content="PROC" datatype="xsd:string"></span>Proc</span>. Natl. Acad. Sci. USA <br />     sequencing assembly Haemophilus influenzae <span id='am-57' about='obo:IMR_0702640' typeof='owl:Thing'><span id='am-58' property="rdfs:label" content="D6S45" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="NEGATIVE ELONGATION FACTOR POLYPEPTIDE E" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="RDP" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:id" content="IMR:0702640" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="RDP" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="NELF-E" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="RD RNA-BINDING PROTEIN" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="Negative elongation factor E" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="RD RNA-BINDING PROTEIN" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="RD RNA binding protein" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="RDBP" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="NELF-E" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="MAJOR HISTOCOMPATIBILITY COMPLEX GENE RD" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="NELFE" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="NELFE_HUMAN" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="UniProt:P18615" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="RD" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="RDBP" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="RNA-binding protein RD" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="NEGATIVE ELONGATION FACTOR POLYPEPTIDE E" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="RD" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="MAJOR HISTOCOMPATIBILITY COMPLEX GENE RD" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasExactSynonym" content="NELFE" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="RD RNA binding protein" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="RNA-binding protein RD" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="D6S45" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Negative elongation factor E" datatype="xsd:string"></span>Rd</span>. Science 269:496 8211;                  84:3987 8211;3991. <br />     512.                                                                              31.   Sung, M., R. E. Dalbey. 1992. Identification potential active-site <br />  9. Fraser, C. M., J. D. Gocayne, O. White, M. D. Adams, R.  Clayton, R. D.               residues Escherichia coli leader peptidase. J. Biol. Chem. 267:13154 8211; <br />     Fleischmann, C. J. Bult,  R. Kerlavage, G. Sutton, J. M. Kelley, J. L.                13159. <br />     Fritchman, J. F. Weidman, K. V. Small, M. Sandusky, J. Fuhrmann, D.               32.   Toth, M. J., E. J. Murgola, P. Schimmel. 1988. Evidence unique <br />     Nguyen, T. R. Utterback, D. M. Saudek, C.  Phillips, J. M. Merrick, J.-F.             position codon-anticodon mismatch vivo. J. Mol. Biol. 201:451 8211;454. <br />     Tomb, B.  Dougherty, K. F. Bott, P.-C. Hu, T. S. Lucier, S. C. Peterson,        33.   Tschantz, W. R., M. Sung, V. M. Delgado-Partin, R. E. Dalbey. 1993.     H. O. Smith, C.  Hutchison III, J. C. Venter. 1995. minimal gene              serine lysine residue implicated catalytic mechanism     complement Mycoplasma genitalium. Science 270:397 8211;403. <br />                                                                                             Escherichia coli leader peptidase. J. Biol. Chem. 268:27349 8211;27354. <br /> 10. Hamilton, C. M., M. Aldea, B. K. Washburn, P. Babitzke, S. R. Kushner. <br />                                                                                       34.   van Dijl, J. M. 1993. Sequence p41027. Submitted EMBL/GenBank/ <br />     1989. New method generating deletions gene replacements Esch- <br />                                                                                             DDBJ data banks. <br />     erichia coli. J. Bacteriol. 171:4617 8211;4622. <br />                                                                                       35.   van Dijl, J. M. 1993. Sequence p41026. Submitted EMBL/GenBank/ <br /> 11. Hoang, V.,  Birger, J. Hofemeister. 1993. Sequence P42668. Submitted <br />     EMBL/GenBank/DDBJ data banks.                                                        DDBJ data banks. <br /> 12. Hoang, V., J. Hofemeister. 1995. B. amyloliquefaciens possesses second      36.   van Dijl, J. M.,  Jong, H. Smith, S. Bron, G. Venema. 1991. <br />     type signal peptidase extensive sequence similarity Bacillus            Non-functional expression Escherichia coli signal peptidase B. subtilis. <br />     SPases. Biochim. Biophys. <span id='am-1047' about='obo:IMR_0701090' typeof='owl:Thing'><span id='am-1048' property="oboInOwl:hasExactSynonym" content="Acta" datatype="xsd:string"></span><span id='am-1049' property="rdfs:label" content="Acta" datatype="xsd:string"></span><span id='am-1050' property="rdfs:label" content="Actin, alpha skeletal muscle" datatype="xsd:string"></span><span id='am-1051' property="oboInOwl:hasExactSynonym" content="Alpha-actin-1" datatype="xsd:string"></span><span id='am-1052' property="oboInOwl:hasExactSynonym" content="Acta1" datatype="xsd:string"></span><span id='am-1053' property="rdfs:label" content="ACTS_RAT" datatype="xsd:string"></span><span id='am-1054' property="oboInOwl:hasExactSynonym" content="actin, alpha 1, skeletal muscle" datatype="xsd:string"></span><span id='am-1055' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1056' property="rdfs:label" content="Alpha-actin-1" datatype="xsd:string"></span><span id='am-1057' property="oboInOwl:hasDbXref" content="UniProt:P68136" datatype="xsd:string"></span><span id='am-1058' property="rdfs:label" content="actin, alpha 1, skeletal muscle" datatype="xsd:string"></span><span id='am-1059' property="rdfs:label" content="Acta1" datatype="xsd:string"></span><span id='am-1060' property="oboInOwl:hasExactSynonym" content="Actin, alpha skeletal muscle" datatype="xsd:string"></span><span id='am-1061' property="oboInOwl:id" content="IMR:0701090" datatype="xsd:string"></span>Acta</span> 1269:64 8211;68.                                              J. Gen. Microbiol. 137:2073 8211;2083. <br /> 13. Hojrup, P., M. S. Jensen, T. E. Petersen. 1995. Amino acid sequence        37.   van Dijl, J. M.,  Jong, J. Vehmaanpera, G. Venema, S. Bron. 1992. <br />     bovine protein  vitamin K-dependent serine protease homolog. FEBS                   Signal peptidase B. subtilis: patterns conserved amino acids pro- <br />     Lett. 184:333 8211;338.                                                                      karyotic eukaryotic type signal peptidases. EMBO J. 11:2819 8211;2828. <br /> 14. Inada, T., D. L. Court, K. <br /> </body></html>