<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />                               Inherited adaptation genome-rewired cells <br />                               response challenging environment <br />  <br />                               Despite evolutionary significance, little known adaptation <br />                               dynamics genomically rewired cells evolution. confronted yeast <br />                               cells carrying rewired regulatory circuit severe unforeseen <br />                               challenge. essential HIS3 gene histidine biosynthesis pathway                               placed exclusive regulation galactose utilization  <br />                               Glucose containing medium strongly represses <span id='am-129' about='obo:IMR_0703780' typeof='owl:Thing'><span id='am-130' property="oboInOwl:hasExactSynonym" content="Gal" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="Galanin message-associated peptide" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasExactSynonym" content="Galn" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:id" content="IMR:0703780" datatype="xsd:string"></span><span id='am-135' property="rdfs:label" content="galanin" datatype="xsd:string"></span><span id='am-136' property="rdfs:label" content="GMAP" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasExactSynonym" content="galanin" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasExactSynonym" content="Galanin" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="Galanin" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasExactSynonym" content="GMAP" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasDbXref" content="UniProt:P47212" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasExactSynonym" content="Galanin message-associated peptide" datatype="xsd:string"></span><span id='am-143' property="rdfs:label" content="Galn" datatype="xsd:string"></span><span id='am-144' property="rdfs:label" content="GALA_MOUSE" datatype="xsd:string"></span><span id='am-145' property="rdfs:label" content="Gal" datatype="xsd:string"></span>GAL</span> genes including HIS3 <br />                               rewired cells required operate essential gene.                               adapted cells prior encounter glucose,                               large fraction cells adapted grow medium adaptation                               stably inherited. adaptation relied individual cells switched                               adapted state   adaptation response individual <br />                               cells change environment selection rare advantageous <br />                               phenotypes. adaptation numerous individual cells heritable phenotypic <br />                               switching response challenge extends common evolutionary <br />                               framework attests adaptive potential regulatory circuits. <br />                               [DOI: 10.2976/1.3353782] <br />  <br />  <br /> CORRESPONDENCE                    Organisms cells exhibit significant ad-       Delbruck, 1943; Paquin Adams, 1983; <br /> Erez Braun:                   aptation capabilities diverse environmental        Lenski Travisano, 1994; Drake et al., 1998; <br /> erez@physics.technion.ac                               conditions evident realized biodiver-       Marini et al., 1999; Elena Lenski, 2003; <br />                               sity. Darwinian natural selection frame-          Fong et al., 2005; Maharjan et al., 2006; <br />                               work states order steps evolution-      Perfeito et al., 2007). <br />                               ary process leads diversification.          evolutionary significant cases, <br />                               heritable phenotypic variability exists        phenotypes evolve emergence                               population later, change condi-         new <span id='am-710' about='obo:IMR_0000949' typeof='owl:Thing obo:IMR_0000001'><span id='am-711' property="rdfs:label" content="protein" datatype="xsd:string"></span><span id='am-712' property="oboInOwl:id" content="IMR:0000949" datatype="xsd:string"></span><span id='am-713' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>protein</span> modified protein functionality <br />                               tions, environment impose selection         novelty gene regulation <br />                               certain phenotypes change          (King Wilson, 1975). Regulatory modes <br />                               frequency generation         flexible diverse   com- <br />                               neo-Darwinian view extends paradigm           parative studies delineated cases                               maintaining underlying heritable phe-        novel phenotypes emerged modi- <br />                               notypic diversity genes genetic varia-        fication gene regulation created new <br />                               tion, ascribed neutral            functional context existing gene (Carroll <br />                               advantageous mutations occur rarely,             et al., 2001; Wilkins, 2002; Alonso                               spontaneously random locations, inde-          Wilkins, 2005; Carroll, 2005; Davidson, 2006; <br />                               pendently selection processes imposed          Wray, 2007; Tuch et al., 2008a, 2008b).                               environmental conditions.           large variability regulatory circuits wit- <br />                               studies demonstrated importance           ness today shaped past regulatory <br />                               genetic variability confers fitness advan-       challenges successfully resolved                               tage emergence novel functional ele-       evolution.  large ob- <br />                               ments given selective environment (Luria         served variability  represents frac- <br />  <br /> HFSP Journal  169; HFSP Publishing $25.00                                                                                            131 <br /> Vol. 4, <span id='am-231' about='obo:IMR_0200719' typeof='owl:Thing'><span id='am-232' property="oboInOwl:hasExactSynonym" content="bNOS" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasExactSynonym" content="nitric oxide synthase 1 (neuronal)" datatype="xsd:string"></span><span id='am-234' property="rdfs:label" content="Peptidyl-cysteine S-nitrosylase NOS1" datatype="xsd:string"></span><span id='am-235' property="rdfs:label" content="N-NOS" datatype="xsd:string"></span><span id='am-236' property="rdfs:label" content="NOS type I" datatype="xsd:string"></span><span id='am-237' property="rdfs:label" content="nNOS" datatype="xsd:string"></span><span id='am-238' property="oboInOwl:id" content="IMR:0200719" datatype="xsd:string"></span><span id='am-239' property="rdfs:label" content="NITRIC OXIDE SYNTHASE 1" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:hasExactSynonym" content="EC 1.14.13.39" datatype="xsd:string"></span><span id='am-241' property="oboInOwl:hasExactSynonym" content="NOS type I" datatype="xsd:string"></span><span id='am-242' property="oboInOwl:hasExactSynonym" content="N-NOS" datatype="xsd:string"></span><span id='am-243' property="rdfs:label" content="nitric oxide synthase 1 (neuronal)" datatype="xsd:string"></span><span id='am-244' property="rdfs:label" content="Constitutive NOS" datatype="xsd:string"></span><span id='am-245' property="rdfs:label" content="NC-NOS" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-247' property="oboInOwl:hasExactSynonym" content="Nitric oxide synthase, brain" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasExactSynonym" content="NOS" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasExactSynonym" content="IHPS1" datatype="xsd:string"></span><span id='am-250' property="rdfs:label" content="IHPS1" datatype="xsd:string"></span><span id='am-251' property="oboInOwl:hasExactSynonym" content="Peptidyl-cysteine S-nitrosylase NOS1" datatype="xsd:string"></span><span id='am-252' property="rdfs:label" content="NOS" datatype="xsd:string"></span><span id='am-253' property="rdfs:label" content="EC 1.14.13.39" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasExactSynonym" content="Neuronal NOS" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasExactSynonym" content="NITRIC OXIDE SYNTHASE 1" datatype="xsd:string"></span><span id='am-256' property="rdfs:label" content="NOS1_HUMAN" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasExactSynonym" content="nNOS" datatype="xsd:string"></span><span id='am-258' property="rdfs:label" content="Nitric oxide synthase, brain" datatype="xsd:string"></span><span id='am-259' property="rdfs:label" content="bNOS" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasDbXref" content="UniProt:P29475" datatype="xsd:string"></span><span id='am-261' property="oboInOwl:hasExactSynonym" content="NC-NOS" datatype="xsd:string"></span><span id='am-262' property="rdfs:label" content="Neuronal NOS" datatype="xsd:string"></span><span id='am-263' property="rdfs:label" content="NOS1" datatype="xsd:string"></span><span id='am-264' property="oboInOwl:hasExactSynonym" content="NOS1" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:hasExactSynonym" content="Constitutive NOS" datatype="xsd:string"></span>Nos</span>. 3-4, June-August 2010, 131 8211;141   http://hfspj.aip.org <br />  HFSP Journal <br />  <br />  <br /> tion vast combinatorial space possible regulatory            previous study (Stolovicki et al., 2006) focused <br /> modes.  reasonable hypothesize existing     population average adaptation dynamics, did <br /> rare genetic variation provide immediate advanta-        allow follow population structure geous solution possible novel regulatory challenge       process.  impossible conclude ob- <br /> (Gerhart Kirschner, 1997; West-Eberhard, 2003)             served adaptation phenomenon selection  alternative mechanisms adaptation              special subpopulation processes. understand considered.                                                        population achieved rapid adaptation, paper, <br />     knowledge biology based studies        monitored colony counts microscopy measure- comparisons evolutionary end points,              ments growth behavior individual cells current life forms. Little information exists dynamics      encounter challenging environment. processes lead functional biological novelties      spite overall population growth encoun- intermediate states evolving forms (West-Eberhard,          ter liquid glucose medium, observable <br /> 2003). Traditionally, studies experimental evolution fo-        fraction adapted cells, prior exposure <br /> cused evolved organism, characterizing           initial growth phase population.  <br /> higher fitness underlying advantageous mutations           markably, potential cells adapt declined sharply dur- selected (Paquin Adams, 1983; Elena              ing generations exposure glucose, contrary <br /> Lenski, 2003).  detecting alternative adaptation        expected behavior exponentially growing popu- <br /> process requires careful monitoring adaptation process      lation. Plating samples growing population  <br /> itself dynamic trajectory population          firmed numerous individual cells eventually grow merely studying end <span id='am-1288' about='obo:IMR_0002979' typeof='owl:Thing'><span id='am-1289' property="rdfs:label" content="D-Ets-2" datatype="xsd:string"></span><span id='am-1290' property="oboInOwl:hasExactSynonym" content="pnt" datatype="xsd:string"></span><span id='am-1291' property="rdfs:label" content="D-ETS-2" datatype="xsd:string"></span><span id='am-1292' property="rdfs:label" content="0123/09" datatype="xsd:string"></span><span id='am-1293' property="oboInOwl:hasExactSynonym" content="ptd" datatype="xsd:string"></span><span id='am-1294' property="oboInOwl:hasExactSynonym" content="pointedP2" datatype="xsd:string"></span><span id='am-1295' property="rdfs:label" content="Ets2" datatype="xsd:string"></span><span id='am-1296' property="oboInOwl:hasExactSynonym" content="D-ETS-2" datatype="xsd:string"></span><span id='am-1297' property="rdfs:label" content="Ets protein Pointed P2" datatype="xsd:string"></span><span id='am-1298' property="oboInOwl:hasExactSynonym" content="PNTP1" datatype="xsd:string"></span><span id='am-1299' property="rdfs:label" content="pointed-P1" datatype="xsd:string"></span><span id='am-1300' property="oboInOwl:hasExactSynonym" content="l(3)07825" datatype="xsd:string"></span><span id='am-1301' property="rdfs:label" content="EK3-2" datatype="xsd:string"></span><span id='am-1302' property="rdfs:label" content="pointed-RC" datatype="xsd:string"></span><span id='am-1303' property="oboInOwl:hasExactSynonym" content="Ets at 58AB" datatype="xsd:string"></span><span id='am-1304' property="rdfs:label" content="pointed" datatype="xsd:string"></span><span id='am-1305' property="rdfs:label" content="D-ets-2" datatype="xsd:string"></span><span id='am-1306' property="oboInOwl:hasExactSynonym" content="ETS2" datatype="xsd:string"></span><span id='am-1307' property="rdfs:label" content="POINT" datatype="xsd:string"></span><span id='am-1308' property="oboInOwl:hasDbXref" content="UniProt:P51023" datatype="xsd:string"></span><span id='am-1309' property="rdfs:label" content="PNTP1" datatype="xsd:string"></span><span id='am-1310' property="rdfs:label" content="l(3)j1B7" datatype="xsd:string"></span><span id='am-1311' property="oboInOwl:hasExactSynonym" content="ETS-like protein pointed, isoform P2/D" datatype="xsd:string"></span><span id='am-1312' property="oboInOwl:hasExactSynonym" content="EK3-2" datatype="xsd:string"></span><span id='am-1313' property="oboInOwl:hasExactSynonym" content="EY3-1" datatype="xsd:string"></span><span id='am-1314' property="rdfs:label" content="Ets94F" datatype="xsd:string"></span><span id='am-1315' property="rdfs:label" content="ETS2" datatype="xsd:string"></span><span id='am-1316' property="oboInOwl:hasExactSynonym" content="pointed" datatype="xsd:string"></span><span id='am-1317' property="rdfs:label" content="PNT2_DROME" datatype="xsd:string"></span><span id='am-1318' property="rdfs:label" content="0998/12" datatype="xsd:string"></span><span id='am-1319' property="rdfs:label" content="l(3)07825" datatype="xsd:string"></span><span id='am-1320' property="rdfs:label" content="DMPOINT1A" datatype="xsd:string"></span><span id='am-1321' property="oboInOwl:hasExactSynonym" content="D-ets-2" datatype="xsd:string"></span><span id='am-1322' property="obo:IAO_0000115" content="pnt gene transcription generates alternatively spliced mRNA isoforms that encode two proteins, PntP1 and PntP2.
PntP2 is the homolog of the human Ets-1 transcription factor and functions as a transcriptional activator when phosphorylated by MAPK." datatype="xsd:string"></span><span id='am-1323' property="rdfs:label" content="3520" datatype="xsd:string"></span><span id='am-1324' property="rdfs:label" content="pointedP2" datatype="xsd:string"></span><span id='am-1325' property="rdfs:label" content="Ets58AB" datatype="xsd:string"></span><span id='am-1326' property="oboInOwl:hasExactSynonym" content="0608/07" datatype="xsd:string"></span><span id='am-1327' property="oboInOwl:hasExactSynonym" content="CG17077" datatype="xsd:string"></span><span id='am-1328' property="rdfs:label" content="EY3-1" datatype="xsd:string"></span><span id='am-1329' property="rdfs:label" content="0608/07" datatype="xsd:string"></span><span id='am-1330' property="oboInOwl:hasExactSynonym" content="pointed-P1" datatype="xsd:string"></span><span id='am-1331' property="oboInOwl:hasExactSynonym" content="Ets protein Pointed P2" datatype="xsd:string"></span><span id='am-1332' property="oboInOwl:hasExactSynonym" content="pointed-RC" datatype="xsd:string"></span><span id='am-1333' property="oboInOwl:hasExactSynonym" content="ets94F" datatype="xsd:string"></span><span id='am-1334' property="rdfs:label" content="CG17077" datatype="xsd:string"></span><span id='am-1335' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1336' property="oboInOwl:hasExactSynonym" content="POINT" datatype="xsd:string"></span><span id='am-1337' property="oboInOwl:id" content="IMR:0002979" datatype="xsd:string"></span><span id='am-1338' property="oboInOwl:hasExactSynonym" content="DMPOINT1A" datatype="xsd:string"></span><span id='am-1339' property="rdfs:label" content="ETS-like protein pointed, isoform P2/D" datatype="xsd:string"></span><span id='am-1340' property="oboInOwl:hasExactSynonym" content="D-Ets-2" datatype="xsd:string"></span><span id='am-1341' property="rdfs:label" content="Ets at 58AB" datatype="xsd:string"></span><span id='am-1342' property="oboInOwl:hasExactSynonym" content="E(E2F)3D" datatype="xsd:string"></span><span id='am-1343' property="rdfs:label" content="ptd" datatype="xsd:string"></span><span id='am-1344' property="oboInOwl:hasExactSynonym" content="3520" datatype="xsd:string"></span><span id='am-1345' property="oboInOwl:hasExactSynonym" content="Ets94F" datatype="xsd:string"></span><span id='am-1346' property="rdfs:label" content="pnt" datatype="xsd:string"></span><span id='am-1347' property="rdfs:label" content="PNTP2" datatype="xsd:string"></span><span id='am-1348' property="rdfs:label" content="ets94F" datatype="xsd:string"></span><span id='am-1349' property="oboInOwl:hasExactSynonym" content="Ets58AB" datatype="xsd:string"></span><span id='am-1350' property="oboInOwl:hasExactSynonym" content="PNTP2" datatype="xsd:string"></span><span id='am-1351' property="oboInOwl:hasExactSynonym" content="Ets2" datatype="xsd:string"></span><span id='am-1352' property="oboInOwl:hasExactSynonym" content="pointed-P2" datatype="xsd:string"></span><span id='am-1353' property="oboInOwl:hasExactSynonym" content="0123/09" datatype="xsd:string"></span><span id='am-1354' property="rdfs:label" content="E(E2F)3D" datatype="xsd:string"></span><span id='am-1355' property="oboInOwl:hasExactSynonym" content="l(3)j1B7" datatype="xsd:string"></span><span id='am-1356' property="oboInOwl:hasExactSynonym" content="0998/12" datatype="xsd:string"></span><span id='am-1357' property="rdfs:label" content="pointed-P2" datatype="xsd:string"></span>point</span>,       glucose similar wild-type cells capacity long term outcome process.  studying           gained later plates initial <br /> adaptation novel challenge           growth liquid glucose. Cells gained growth capac- <br /> countered cells evolutionary history bares      ity glucose truly adapted propagate <br /> higher chances detect alternative process compared        capacity descendants hundreds genera- <br /> studying repeated challenge cells        tions. adaptive state gained  8220;predesigned 8221; solution.                                     exposure glucose numerous cells simulta- <br />                                                                    neously, heritable adaptation induced     explored possible alternative adaptation pro- <br />                                                                    individual cells environment. study,  <br /> cess placing cells novel rewired genome chal- <br />                                                                    details process different fundamental com- <br /> lenging environment. genomic rewiring event, exist- <br />                                                                    mon view adaptation. adaptation rely ing gene linked foreign  e., previously <br />                                                                    random rare genetic variability accumulated inde- <br /> functionally unrelated) regulatory  events                                                                    pendently selection agent. documented comparative genomic studies reported drive significantly evolution gene regula- <br /> tion [(Carroll et al., 2001; Wilkins, 2002; Davidson, 2006)        RESULTS  discussions Alonso Wilkins (2005); Stern      Adaptation cells <span id='am-867' about='obo:IMR_0002833' typeof='owl:Thing'><span id='am-868' property="oboInOwl:hasDbXref" content="UniProt:Q8IRI6" datatype="xsd:string"></span><span id='am-869' property="oboInOwl:hasExactSynonym" content="glucose transporter 1" datatype="xsd:string"></span><span id='am-870' property="rdfs:label" content="CG1086" datatype="xsd:string"></span><span id='am-871' property="oboInOwl:hasExactSynonym" content="Glut1" datatype="xsd:string"></span><span id='am-872' property="oboInOwl:hasExactSynonym" content="Glucose transporter type 1" datatype="xsd:string"></span><span id='am-873' property="rdfs:label" content="Glucose transporter 1" datatype="xsd:string"></span><span id='am-874' property="rdfs:label" content="Glut1" datatype="xsd:string"></span><span id='am-875' property="rdfs:label" content="glut1" datatype="xsd:string"></span><span id='am-876' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-877' property="oboInOwl:hasExactSynonym" content="Dmglut1" datatype="xsd:string"></span><span id='am-878' property="rdfs:label" content="GTR1_DROME" datatype="xsd:string"></span><span id='am-879' property="oboInOwl:id" content="IMR:0002833" datatype="xsd:string"></span><span id='am-880' property="rdfs:label" content="GLU" datatype="xsd:string"></span><span id='am-881' property="oboInOwl:hasExactSynonym" content="glut1" datatype="xsd:string"></span><span id='am-882' property="rdfs:label" content="Glucose transporter type 1" datatype="xsd:string"></span><span id='am-883' property="oboInOwl:hasExactSynonym" content="Glucose transporter 1" datatype="xsd:string"></span><span id='am-884' property="rdfs:label" content="glucose transporter 1" datatype="xsd:string"></span><span id='am-885' property="oboInOwl:hasExactSynonym" content="CG1086" datatype="xsd:string"></span><span id='am-886' property="rdfs:label" content="Dmglut1" datatype="xsd:string"></span><span id='am-887' property="oboInOwl:hasExactSynonym" content="GLU" datatype="xsd:string"></span>Glu</span> plates Orgogozo (2008); Erwin Davidson (2009)].            adaptation capability rewired cells colony growth <br /> experiments, strain yeast Saccharomyces cerevisiae        assay plates demonstrated Stolovicki et al. engineered recruit gene HIS3, essential <span id='am-90' about='obo:IMR_0000207' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001'><span id='am-91' property="oboInOwl:hasDbXref" content="GO:0003824" datatype="xsd:string"></span><span id='am-92' property="obo:IAO_0000115" content="Protein that catalyze a specific chemical reaction." datatype="xsd:string"></span><span id='am-93' property="oboInOwl:id" content="IMR:0000207" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="enzyme" datatype="xsd:string"></span>enzyme</span>       (2006). study, expanded analysis ex- histidine biosynthesis pathway, exclusively        amine variability process. Naive rewired cells, <br /> <span id='am-167' about='obo:IMR_0703778' typeof='owl:Thing'><span id='am-168' property="oboInOwl:hasDbXref" content="UniProt:P10683" datatype="xsd:string"></span><span id='am-169' property="rdfs:label" content="Galanin message-associated peptide" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasExactSynonym" content="Galanin message-associated peptide" datatype="xsd:string"></span><span id='am-172' property="oboInOwl:hasExactSynonym" content="galanin prepropeptide" datatype="xsd:string"></span><span id='am-173' property="rdfs:label" content="GMAP" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasExactSynonym" content="Galn" datatype="xsd:string"></span><span id='am-175' property="rdfs:label" content="GALA_RAT" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:id" content="IMR:0703778" datatype="xsd:string"></span><span id='am-177' property="rdfs:label" content="galanin prepropeptide" datatype="xsd:string"></span><span id='am-178' property="rdfs:label" content="Galanin" datatype="xsd:string"></span><span id='am-179' property="rdfs:label" content="Gal" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasExactSynonym" content="GMAP" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasExactSynonym" content="Galanin" datatype="xsd:string"></span><span id='am-182' property="rdfs:label" content="Galn" datatype="xsd:string"></span><span id='am-183' property="oboInOwl:hasExactSynonym" content="Gal" datatype="xsd:string"></span>GAL</span> responsible galactose utilization           exposed glucose  sampled <br /> (Stolovicki et al., 2006). arbitrary regulatory linkage     single colony batch cultured histidine-lacking <br /> tween evolutionary conserved highly specific         galactose-based (<span id='am-184' about='obo:IMR_0703779' typeof='owl:Thing'><span id='am-185' property="rdfs:label" content="GAL" datatype="xsd:string"></span><span id='am-186' property="rdfs:label" content="GALA_HUMAN" datatype="xsd:string"></span><span id='am-187' property="oboInOwl:hasExactSynonym" content="Galanin" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasExactSynonym" content="galanin prepropeptide" datatype="xsd:string"></span><span id='am-189' property="rdfs:label" content="Galanin message-associated peptide" datatype="xsd:string"></span><span id='am-190' property="rdfs:label" content="MGC40167" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasExactSynonym" content="GLNN" datatype="xsd:string"></span><span id='am-192' property="rdfs:label" content="Galanin" datatype="xsd:string"></span><span id='am-193' property="rdfs:label" content="GAL1" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasExactSynonym" content="MGC40167" datatype="xsd:string"></span><span id='am-195' property="rdfs:label" content="GMAP" datatype="xsd:string"></span><span id='am-196' property="rdfs:label" content="GALN" datatype="xsd:string"></span><span id='am-197' property="rdfs:label" content="galanin prepropeptide" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasDbXref" content="UniProt:P22466" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:hasExactSynonym" content="GAL" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-201' property="oboInOwl:hasExactSynonym" content="Galanin message-associated peptide" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasExactSynonym" content="GAL1" datatype="xsd:string"></span><span id='am-203' property="rdfs:label" content="GLNN" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasExactSynonym" content="GMAP" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:id" content="IMR:0703779" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasExactSynonym" content="GALN" datatype="xsd:string"></span>Gal</span>  medium. sample cells modules stressful, challenging created incom-              culture washed placed histidine-lacking <span id='am-888' about='obo:IMR_0002834' typeof='owl:Thing'><span id='am-889' property="oboInOwl:hasExactSynonym" content="Glut3" datatype="xsd:string"></span><span id='am-890' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-891' property="oboInOwl:id" content="IMR:0002834" datatype="xsd:string"></span><span id='am-892' property="rdfs:label" content="Glucose transporter-like protein" datatype="xsd:string"></span><span id='am-893' property="oboInOwl:hasExactSynonym" content="CG3853" datatype="xsd:string"></span><span id='am-894' property="oboInOwl:hasExactSynonym" content="glut-l" datatype="xsd:string"></span><span id='am-895' property="oboInOwl:hasExactSynonym" content="GLU" datatype="xsd:string"></span><span id='am-896' property="oboInOwl:hasExactSynonym" content="Glucose transporter-like protein" datatype="xsd:string"></span><span id='am-897' property="rdfs:label" content="Glucose transporter-like" datatype="xsd:string"></span><span id='am-898' property="rdfs:label" content="Glut3" datatype="xsd:string"></span><span id='am-899' property="rdfs:label" content="Glucose transporter type 3" datatype="xsd:string"></span><span id='am-900' property="rdfs:label" content="GTR3_DROME" datatype="xsd:string"></span><span id='am-901' property="oboInOwl:hasExactSynonym" content="Glucose transporter type 3" datatype="xsd:string"></span><span id='am-902' property="rdfs:label" content="GLU" datatype="xsd:string"></span><span id='am-903' property="oboInOwl:hasDbXref" content="UniProt:P53403" datatype="xsd:string"></span><span id='am-904' property="rdfs:label" content="CG3853" datatype="xsd:string"></span><span id='am-905' property="rdfs:label" content="glut-l" datatype="xsd:string"></span><span id='am-906' property="oboInOwl:hasExactSynonym" content="Glucose transporter-like" datatype="xsd:string"></span>glu</span>- <br /> patibilities gene expression (Stolovicki et al., 2006).     cose (Glu  plates test adaptation rich  <br /> cells exercised regulation HIS3 carbon sources       dium plates viability control. rich medium plates, <br /> availability history, substantial adaptive     contained histidine, rewired cells grew wild-type <br /> sponse required survive medium lacking        cells formed mature colonies 2 8211;3 days.  <br /> histidine. Recently, shown cell population <span id='am-96' about='obo:IMR_0704802' typeof='owl:Thing'><span id='am-97' property="rdfs:label" content="MGC17251" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasExactSynonym" content="cAMP-dependent protein kinase type I-alpha regulatory subunit" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="CNC" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="UniProt:P10644" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasExactSynonym" content="DKFZp779L0468" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="TSE1" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasExactSynonym" content="PRKAR1A" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:id" content="IMR:0704802" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasExactSynonym" content="PPNAD1" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="PKR1" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasExactSynonym" content="protein kinase, cAMP-dependent, regulatory, type I, alpha (tissue specific extinguisher 1)" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="cAMP-dependent protein kinase type I-alpha regulatory subunit" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="CAR" datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="Tissue-specific extinguisher 1" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasExactSynonym" content="Tissue-specific extinguisher 1" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasExactSynonym" content="CAR" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="DKFZp779L0468" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="PROTEIN KINASE, cAMP-DEPENDENT, REGULATORY, TYPE I, ALPHA" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasExactSynonym" content="MGC17251" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="CNC1" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasExactSynonym" content="CNC1" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="PRKAR1" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasExactSynonym" content="PROTEIN KINASE, cAMP-DEPENDENT, REGULATORY, TYPE I, ALPHA" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="CNC" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasExactSynonym" content="PKR1" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="PPNAD1" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="PRKAR1A" datatype="xsd:string"></span><span id='am-125' property="rdfs:label" content="KAP0_HUMAN" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasExactSynonym" content="PRKAR1" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasExactSynonym" content="TSE1" datatype="xsd:string"></span><span id='am-128' property="rdfs:label" content="protein kinase, cAMP-dependent, regulatory, type I, alpha (tissue specific extinguisher 1)" datatype="xsd:string"></span>car</span>-     trast, Glu plates, colonies observed day <br /> rying GAL-HIS3 rewired genome rapidly adapt            6 post-plating additional incubation, colonies <br /> grow competitively histidine-lacking glucose medium          started form gradually accumulated maximal <br /> spite strong repression GAL HIS3             number colonies counted day 20 post-plating. Im- <br /> (Stolovicki et al., 2006; Stern et al., 2007). established,   portantly, shown numerous studies used adaptation propagated stably hundreds         deletion HIS3 oxotrophic marker, ex- <br /> generations (Stolovicki et al., 2006).                             periments cells copy HIS3 grow  <br /> 132                                                                      Inherited adaptation genome-rewired . . . | David et al. <br />                                                                                                                                 RT C L E <br />  <br />  <br />  <br />  <br /> Figure 1. Adaptation propagation plates. culture naive cells galactose, equal cells plated Glu  20849;a 20850; rich medium  20849;b 20850; plates. Images captured 12 days  20849;a 20850; 48 h  20849;b 20850; incubation, respectively. Note high proportion cells adapted formed colonies glucose variable sizes colonies. Cells single adapted colony adaptation plate  20849;a 20850; taken equal amounts plated Glu  20849;c 20850; rich medium  20849;d 20850; plates. images  20851; 20849;c 20850;  20849;d 20850; 20852; taken 48 h post-plating. Note similar growth uniform colony size adapted cells medium types. adaptation <br /> experiments Glu plates  20851;as panels  20849;a 20850;  20849;b 20850; 20852;, replicates, carried proportion adaptation determined colony number Glu divided rich medium  20849;e 20850;. Note overall high proportion adaptation  20849;average 0.5 20850; variation, similar replicates experiments. <br />  <br />  <br /> Glu plates longer incubation times. Common                stably inherited generations, allowing sur- experiment described Stolovicki et al. (2006)           vival genome-rewired strain challenging environ- 11 experiments report cells            ment. <br /> incubation repressive Glu environment gradually capable producing histidine, propagate form                population growth dynamics adaptation <br /> mature colonies. average, fraction cells            context evolution, term adaptation used <br /> grow mature colonies Glu 50% total               change population average phenotype <br /> number viable cells plated, estimated colony                  generations result selection favor <br /> numbers rich medium plates [Figs. 1 , 1(b), 1(e)].              certain variants. large fraction cells lead  fraction turned variable                  adapted colonies plates surprising  <br /> experiments started different colonies simi-               derstand large fraction cells adapted, lar variability replicates              process population adaptation studied adding <br /> experiment drawn single starting culture                time dimension; measuring population growth dy- <br /> [Fig. 1(e)].                                                             namics analyzing behavior individual cells     large variable fraction mature colonies               onset exposure glucose final adapted state. Glu plates unexpected raised concern                previous studies, growth dynamics population cell growth mere transient physiological response              exposure Glu studied chemostat (Novick <br /> accommodating stressful condition.                  Szilard, 1950) large populations rewired case cells colonies retained growth          cells grown Gal switched Glu medium <br /> capability hundreds generations Glu               controlled stable environmental condi- <br /> liquid medium plates. Furthermore, cells              tions. chemostat setup allowed monitor online colonies adapted Glu formed mature colonies                  cell density [optical density (OD)] automatically sample 2 8211;3 days respreading new Glu                  population high temporal resolution plates [Figs. 1(c) 1(d)], showing growth rates similar               analyses cells  examples Stolovicki et al. (2006) rich medium plates.  adapted pheno-              Stern et al. (2007)]. study, repeated chemostat ex- <br /> type inherited based normal growth Glu             periments carried identical conditions adaptive phenotype easily distinguishable               reproducible patterns population growth dynamics naive  Furthermore, study corroborated previous             <span id='am-1218' about='obo:IMR_0002979' typeof='owl:Thing'><span id='am-1219' property="oboInOwl:hasExactSynonym" content="ets94F" datatype="xsd:string"></span><span id='am-1220' property="rdfs:label" content="POINT" datatype="xsd:string"></span><span id='am-1221' property="oboInOwl:hasExactSynonym" content="ETS2" datatype="xsd:string"></span><span id='am-1222' property="rdfs:label" content="l(3)j1B7" datatype="xsd:string"></span><span id='am-1223' property="rdfs:label" content="PNT2_DROME" datatype="xsd:string"></span><span id='am-1224' property="oboInOwl:hasExactSynonym" content="EY3-1" datatype="xsd:string"></span><span id='am-1225' property="oboInOwl:hasExactSynonym" content="pointed-P1" datatype="xsd:string"></span><span id='am-1226' property="rdfs:label" content="D-ets-2" datatype="xsd:string"></span><span id='am-1227' property="rdfs:label" content="pnt" datatype="xsd:string"></span><span id='am-1228' property="oboInOwl:hasExactSynonym" content="PNTP1" datatype="xsd:string"></span><span id='am-1229' property="rdfs:label" content="E(E2F)3D" datatype="xsd:string"></span><span id='am-1230' property="rdfs:label" content="pointed" datatype="xsd:string"></span><span id='am-1231' property="oboInOwl:hasExactSynonym" content="ptd" datatype="xsd:string"></span><span id='am-1232' property="rdfs:label" content="EK3-2" datatype="xsd:string"></span><span id='am-1233' property="rdfs:label" content="pointedP2" datatype="xsd:string"></span><span id='am-1234' property="rdfs:label" content="3520" datatype="xsd:string"></span><span id='am-1235' property="oboInOwl:hasExactSynonym" content="Ets at 58AB" datatype="xsd:string"></span><span id='am-1236' property="rdfs:label" content="EY3-1" datatype="xsd:string"></span><span id='am-1237' property="rdfs:label" content="pointed-RC" datatype="xsd:string"></span><span id='am-1238' property="oboInOwl:hasExactSynonym" content="pointedP2" datatype="xsd:string"></span><span id='am-1239' property="rdfs:label" content="Ets at 58AB" datatype="xsd:string"></span><span id='am-1240' property="rdfs:label" content="ptd" datatype="xsd:string"></span><span id='am-1241' property="rdfs:label" content="Ets58AB" datatype="xsd:string"></span><span id='am-1242' property="oboInOwl:hasExactSynonym" content="POINT" datatype="xsd:string"></span><span id='am-1243' property="obo:IAO_0000115" content="pnt gene transcription generates alternatively spliced mRNA isoforms that encode two proteins, PntP1 and PntP2.
PntP2 is the homolog of the human Ets-1 transcription factor and functions as a transcriptional activator when phosphorylated by MAPK." datatype="xsd:string"></span><span id='am-1244' property="oboInOwl:hasExactSynonym" content="EK3-2" datatype="xsd:string"></span><span id='am-1245' property="oboInOwl:hasExactSynonym" content="3520" datatype="xsd:string"></span><span id='am-1246' property="oboInOwl:hasExactSynonym" content="Ets2" datatype="xsd:string"></span><span id='am-1247' property="oboInOwl:hasExactSynonym" content="l(3)j1B7" datatype="xsd:string"></span><span id='am-1248' property="oboInOwl:id" content="IMR:0002979" datatype="xsd:string"></span><span id='am-1249' property="rdfs:label" content="Ets protein Pointed P2" datatype="xsd:string"></span><span id='am-1250' property="oboInOwl:hasExactSynonym" content="0608/07" datatype="xsd:string"></span><span id='am-1251' property="oboInOwl:hasExactSynonym" content="0123/09" datatype="xsd:string"></span><span id='am-1252' property="rdfs:label" content="Ets94F" datatype="xsd:string"></span><span id='am-1253' property="oboInOwl:hasExactSynonym" content="ETS-like protein pointed, isoform P2/D" datatype="xsd:string"></span><span id='am-1254' property="oboInOwl:hasExactSynonym" content="DMPOINT1A" datatype="xsd:string"></span><span id='am-1255' property="rdfs:label" content="CG17077" datatype="xsd:string"></span><span id='am-1256' property="oboInOwl:hasExactSynonym" content="Ets94F" datatype="xsd:string"></span><span id='am-1257' property="oboInOwl:hasExactSynonym" content="pointed-RC" datatype="xsd:string"></span><span id='am-1258' property="rdfs:label" content="0998/12" datatype="xsd:string"></span><span id='am-1259' property="rdfs:label" content="Ets2" datatype="xsd:string"></span><span id='am-1260' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1261' property="oboInOwl:hasExactSynonym" content="Ets protein Pointed P2" datatype="xsd:string"></span><span id='am-1262' property="rdfs:label" content="PNTP1" datatype="xsd:string"></span><span id='am-1263' property="oboInOwl:hasExactSynonym" content="CG17077" datatype="xsd:string"></span><span id='am-1264' property="oboInOwl:hasDbXref" content="UniProt:P51023" datatype="xsd:string"></span><span id='am-1265' property="rdfs:label" content="PNTP2" datatype="xsd:string"></span><span id='am-1266' property="oboInOwl:hasExactSynonym" content="PNTP2" datatype="xsd:string"></span><span id='am-1267' property="oboInOwl:hasExactSynonym" content="D-ETS-2" datatype="xsd:string"></span><span id='am-1268' property="rdfs:label" content="l(3)07825" datatype="xsd:string"></span><span id='am-1269' property="oboInOwl:hasExactSynonym" content="0998/12" datatype="xsd:string"></span><span id='am-1270' property="oboInOwl:hasExactSynonym" content="l(3)07825" datatype="xsd:string"></span><span id='am-1271' property="oboInOwl:hasExactSynonym" content="pointed-P2" datatype="xsd:string"></span><span id='am-1272' property="oboInOwl:hasExactSynonym" content="D-ets-2" datatype="xsd:string"></span><span id='am-1273' property="rdfs:label" content="0608/07" datatype="xsd:string"></span><span id='am-1274' property="oboInOwl:hasExactSynonym" content="pointed" datatype="xsd:string"></span><span id='am-1275' property="rdfs:label" content="0123/09" datatype="xsd:string"></span><span id='am-1276' property="rdfs:label" content="D-ETS-2" datatype="xsd:string"></span><span id='am-1277' property="rdfs:label" content="ETS2" datatype="xsd:string"></span><span id='am-1278' property="rdfs:label" content="ETS-like protein pointed, isoform P2/D" datatype="xsd:string"></span><span id='am-1279' property="oboInOwl:hasExactSynonym" content="Ets58AB" datatype="xsd:string"></span><span id='am-1280' property="oboInOwl:hasExactSynonym" content="E(E2F)3D" datatype="xsd:string"></span><span id='am-1281' property="rdfs:label" content="DMPOINT1A" datatype="xsd:string"></span><span id='am-1282' property="rdfs:label" content="ets94F" datatype="xsd:string"></span><span id='am-1283' property="oboInOwl:hasExactSynonym" content="pnt" datatype="xsd:string"></span><span id='am-1284' property="oboInOwl:hasExactSynonym" content="D-Ets-2" datatype="xsd:string"></span><span id='am-1285' property="rdfs:label" content="D-Ets-2" datatype="xsd:string"></span><span id='am-1286' property="rdfs:label" content="pointed-P1" datatype="xsd:string"></span><span id='am-1287' property="rdfs:label" content="pointed-P2" datatype="xsd:string"></span>pointed</span> robust features (Fig. 2). particular, <br /> experiments (Stolovicki et al., 2006) confirmed                 phase populations, immediately following switch growth phenotype adapted cells stably inherited               Glu medium, characterized prolonged ex- did require repeatable adaptation period                 ponential increase cell density lasting gen- <br /> switching forth Gal Glu envi-               erations  phase II, cultures exhibited ronments.  growth  11011;50% cells mature              exponential collapse. increased density phase colonies glucose plates genuine adaptation process              probably elevated growth rate typical  <br /> HFSP Journal Vol. 4, June-August 2010                                                                                                     133 <br />  HFSP Journal <br />  <br />                                                                          time reached phase IV. understand better adap- <br />                                                                          tive state population time limits,                                                                          phenotype individual cells studied sampling                                                                          consecutive time points chemostat phases                                                                          spreading cells Glu plates. analysis indi- <br />                                                                          cated fraction adapted cells minimal end                                                                          phase increased exponentially phase II  <br />                                                                          came 100% end phase (Fig. 2, inset).                                                                          analysis explained cell density increased exponen- <br />                                                                          tially phase III, turned serve adjust- <br />                                                                          ment period population composed                                                                          100% adapted cells fulfilled maximal growth potential                                                                          Glu  importantly, new time limits placed <br />                                                                          adaptation process shown completed <br />                                                                          end phase II. Repeated chemostat experiment                                                                          precisely conditions revealed variable adaptation <br /> Figure 2. Adaptation dynamics chemostat populations. Cell             time ranged 10 30 generations (Fig. 2). density  20849;OD 600 nm 20850; logarithmic function time repeated    adaptation time surprisingly shorter usually  <br /> chemostat experiments conducted nominal condi-            countered laboratory experiments involving fixation tions. comparison, light dark blue curves corresponds chemostat experiments described  20849;Stolovicki et al., 2006 20850;.                                                                          rare mutations large populations microorganisms, <br /> histidine-lacking medium switched galactose glucose       order hundreds thousands genera- sole carbon source t = 0, leaving nutrients    tions (Luria Delbruck, 1943; Paquin steady-state typical galactose metabolism established prior       Adams, 1983; Cairns et al., 1988; Lenski Travisano, switch glucose. <span id='am-563' about='obo:IMR_0100740' typeof='owl:Thing'><span id='am-564' property="rdfs:label" content="l(3)neo57" datatype="xsd:string"></span><span id='am-565' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-566' property="rdfs:label" content="Hh-Np" datatype="xsd:string"></span><span id='am-567' property="oboInOwl:hasExactSynonym" content="Protein hedgehog C-product" datatype="xsd:string"></span><span id='am-568' property="oboInOwl:hasExactSynonym" content="bar-3" datatype="xsd:string"></span><span id='am-569' property="oboInOwl:hasExactSynonym" content="hh" datatype="xsd:string"></span><span id='am-570' property="rdfs:label" content="Protein hedgehog N-product" datatype="xsd:string"></span><span id='am-571' property="oboInOwl:hasExactSynonym" content="N-Hh" datatype="xsd:string"></span><span id='am-572' property="rdfs:label" content="Mrt" datatype="xsd:string"></span><span id='am-573' property="oboInOwl:hasExactSynonym" content="CG4637" datatype="xsd:string"></span><span id='am-574' property="oboInOwl:hasExactSynonym" content="C-Hh" datatype="xsd:string"></span><span id='am-575' property="rdfs:label" content="l(3)neo56" datatype="xsd:string"></span><span id='am-576' property="rdfs:label" content="Protein hedgehog" datatype="xsd:string"></span><span id='am-577' property="rdfs:label" content="bar-3" datatype="xsd:string"></span><span id='am-578' property="rdfs:label" content="anon-WO0134654.19" datatype="xsd:string"></span><span id='am-579' property="oboInOwl:hasExactSynonym" content="bar" datatype="xsd:string"></span><span id='am-580' property="oboInOwl:hasDbXref" content="UniProt:Q02936" datatype="xsd:string"></span><span id='am-581' property="rdfs:label" content="bar-on-3" datatype="xsd:string"></span><span id='am-582' property="rdfs:label" content="Moonrat" datatype="xsd:string"></span><span id='am-583' property="oboInOwl:hasExactSynonym" content="anon-WO0182946.19" datatype="xsd:string"></span><span id='am-584' property="rdfs:label" content="C-Hh" datatype="xsd:string"></span><span id='am-585' property="rdfs:label" content="Mirabile" datatype="xsd:string"></span><span id='am-586' property="rdfs:label" content="bar3" datatype="xsd:string"></span><span id='am-587' property="oboInOwl:hasExactSynonym" content="hedgehog" datatype="xsd:string"></span><span id='am-588' property="rdfs:label" content="hedgehog" datatype="xsd:string"></span><span id='am-589' property="rdfs:label" content="anon-WO0182946.19" datatype="xsd:string"></span><span id='am-590' property="rdfs:label" content="N-Hh" datatype="xsd:string"></span><span id='am-591' property="oboInOwl:hasExactSynonym" content="hg" datatype="xsd:string"></span><span id='am-592' property="rdfs:label" content="hh" datatype="xsd:string"></span><span id='am-593' property="oboInOwl:hasExactSynonym" content="l(3)neo56" datatype="xsd:string"></span><span id='am-594' property="oboInOwl:hasExactSynonym" content="l(3)neo57" datatype="xsd:string"></span><span id='am-595' property="oboInOwl:hasExactSynonym" content="Mrt" datatype="xsd:string"></span><span id='am-596' property="rdfs:label" content="HH" datatype="xsd:string"></span><span id='am-597' property="oboInOwl:hasExactSynonym" content="Hh" datatype="xsd:string"></span><span id='am-598' property="rdfs:label" content="HH_DROME" datatype="xsd:string"></span><span id='am-599' property="rdfs:label" content="Hh" datatype="xsd:string"></span><span id='am-600' property="rdfs:label" content="hg" datatype="xsd:string"></span><span id='am-601' property="oboInOwl:hasExactSynonym" content="Mir" datatype="xsd:string"></span><span id='am-602' property="rdfs:label" content="CG4637" datatype="xsd:string"></span><span id='am-603' property="rdfs:label" content="bar" datatype="xsd:string"></span><span id='am-604' property="rdfs:label" content="Mir" datatype="xsd:string"></span><span id='am-605' property="rdfs:label" content="l(3)hh" datatype="xsd:string"></span><span id='am-606' property="oboInOwl:hasExactSynonym" content="Protein hedgehog N-product" datatype="xsd:string"></span><span id='am-607' property="oboInOwl:hasExactSynonym" content="Hh-Np" datatype="xsd:string"></span><span id='am-608' property="oboInOwl:hasExactSynonym" content="bar3" datatype="xsd:string"></span><span id='am-609' property="oboInOwl:id" content="IMR:0100740" datatype="xsd:string"></span><span id='am-610' property="oboInOwl:hasExactSynonym" content="anon-WO0134654.19" datatype="xsd:string"></span><span id='am-611' property="oboInOwl:hasExactSynonym" content="l(3)hh" datatype="xsd:string"></span><span id='am-612' property="oboInOwl:hasExactSynonym" content="Mirabile" datatype="xsd:string"></span><span id='am-613' property="oboInOwl:hasExactSynonym" content="Moonrat" datatype="xsd:string"></span><span id='am-614' property="rdfs:label" content="Hh-Cp" datatype="xsd:string"></span><span id='am-615' property="oboInOwl:hasExactSynonym" content="Protein hedgehog" datatype="xsd:string"></span><span id='am-616' property="oboInOwl:hasExactSynonym" content="Hh-Cp" datatype="xsd:string"></span><span id='am-617' property="oboInOwl:hasExactSynonym" content="HH" datatype="xsd:string"></span><span id='am-618' property="rdfs:label" content="Protein hedgehog C-product" datatype="xsd:string"></span><span id='am-619' property="oboInOwl:hasExactSynonym" content="bar-on-3" datatype="xsd:string"></span>Bar</span> generations population       1994; Drake et al., 1998; Marini et al., 1999; Elena growth phases denoted purple  20849;I-IV 20850;. Inset: fraction adapted colonies phase II  20849;blue dots 20850; background                                                                          Lenski, 2003; Fong et al., 2005; Roth et al., 2006; Perfeito population density  20849;gray 20850; showing population         et al., 2007). <br /> fully adapted end phase II. <br />  <br />                                                                          time-dependent adaptation potential glucose <br /> yeast cells switched galactose-based glucose-               Realizing short timeframe population <br /> based medium efficient carbon                 achieved fully adapted state pointed signifi- <br /> source. rewired cells  unexpected                cance generations Glu particular behavior strong repression HIS3               population structure phase   sampled <br /> previously observed (Stolovicki et al., 2006).           population particular phase high temporal <br /> fully understand robust phase fair assume         resolution tested adaptive state individual cells prolonged exponential growth relies existing                   samples Glu plates. rich medium  <br /> sources, reservoirs histidine, HIS3 mRNA His3p              trol plates, cells grew normal rate, forming uniform previous period galactose, allowing limited                colonies 2 8211;3 days exhibiting similar viability <br /> number cell divisions. leave interesting issue            counts time points. Glu plates,  future work concentrate paper growth dy-              growth behavior cells completely different. <br /> namics population individual cells eventu-           Similar plate experiments described  visible <br /> ally lead adaptation.                                                 colonies appeared day 6 post-plating additional <br />     Following initial growth phase  population              colonies plate accumulated incubation time collapsed cell density decreased exponentially, indi-            maximum visible colonies reached 20 days <br /> cating regardless resources allowing initial             post-plating.  adapted cells, form ma- <br /> growth, limited cells eventually faced              ture colonies 2 8211;3 days, detected severe growth challenge phase II. time,            phase time points. Mature colonies detected population reached turning point cell density             day 6 post-plating, initiated single cell, turned <br /> started increase, indicating cells growing             fully adapted cells colonies faster chemostat 8217;s dilution rate (phase III), followed          form uniform new colonies 2 8211;3 days replating stable growth constant cell density stage reflecting          Glu plates. Adapted colonies capable population steady-state (phase IV).                                 propagating stably liquid Glu medium.      Based repeatable growth dynamics chemostat                 analysis cells phase chemostat recapitulates <br /> experiments, clear population adapted            plate experiments terms adaptation dynamics exposure Glu fully adapted              individual cells. <br />  <br /> 134                                                                            Inherited adaptation genome-rewired . . . | David et al. <br />                                                                                                                              RT C L E <br />  <br />  <br />  <br />  <br />                                                                          ports notion adaptation achieved cells                                                                          transition challenging environment. <br />                                                                               validate study adaptation process                                                                          naive cells, switched 15 parallel populations Gal                                                                          Glu batch cultures, sampled cells                                                                          growth, measured adaptation rate Glu plates. <br />                                                                          switching Glu  populations grew exponen- <br />                                                                          tially seven generations, resembling phase                                                                          chemostat (Supplementary Material Fig. S1a ).  <br />                                                                          replicate experiments did notice adapted <br /> Figure 3. Population characteristics exposure glucose.           cells plated samples. Like chemostat experiment, maximal fraction adapted colonies  20849;dots 20850; phase     visible colonies appeared Glu plates day 6 chemostat  20849;switching glucose t = 0 20850;. colonies initiated   20 days post-plating maximal fraction individual cells sampled chemostat grown       adapted colonies Glu plates reached 50% Glu plates 20 days. Note logarithmic scale. chemo- <br />                                                                          samples early time points declined exponentially <br /> <span id='am-16' about='obo:IMR_0000374' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000369 obo:IMR_0000368 obo:IMR_0000001'><span id='am-17' property="oboInOwl:hasExactSynonym" content="signal tranducers and activators of transcription" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="STAT" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="signal tranducers and activators of transcription" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0000374" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>stat</span> population density plotted background  20849;gray line 20850;. <br />                                                                          exposure time glucose (Supplementary Material <br />                                                                          Fig. S1b).  plate, chemostat batch culture experi- <br />     Notably,  experiment showed despite                ments agreement high adaptation poten- initial exponential growth chemostat population,              tial naive cells variable time took individu- observable fraction adapted cells               als achieve adapted state. points stress population encountered challenging environment              hand, wide accessibility adaptation process entire period phase   11011;105 cells              cells emphasize hand, achieving sampled time point phase chemostat ex-             adapted state process cell undergoes individu- <br /> periment normally grow Glu plates expected               ally. chemostat batch culture experiments adapted cell. Instead, numerous individual cells          dynamics adaptation process mea- <br /> chemostat culture clearly carried potential                sured, confirmed potential cells develop <br /> adapted eventually, long variable incubation         adaptation declines time seven time, grew colonies Glu plates. results indicated           generations challenging environment, indicating adaptive phenotype, faithfully inherited         critical dependency cellular status time <br />                                                                          glucose. future generations, developed independently numer- <br /> ous cells following exposure challenging environ- <br /> ment. Clearly, property cells carried        Cell division capacity determines chances Gal growth period, suspected             adaptation short adaptation time chemostat.  adap-             Cells developed adaptive state exposure Glu tation experiments property individual cells             liquid solid media experiments. Measuring property population process led        dynamics population liquid medium experiments cell adapted state induced challenging            indicated population grew exponentially,  <br /> environment.                                                             dividuals lost capability adapted.      fact, findings corroborated striking result.         understand nature adaptation process, conventional adaptation process, fraction adapted            explain counterintuitive observation less <br /> individuals expected increase time selective           cells adapted time Glu  answer, hypoth- <br /> condition difficult rare advan-           esized, following response indi- <br /> tageous mutations established. Surprisingly,                vidual cells switched glucose medium. case cells consecutive time points Glu              plates used estimate adaptation capacity exhibited opposite trend. challenging           analyzed microscopy imaging follow growth environment phase  fewer cells capable            individual colonies. Samples different time points of forming adapted colonies Glu plates (Fig. 3).              liquid culture extracted placed Glu largest fraction population form adapted               plates. single cell plate served seed colonies  50%) sampled onset phase                 growing colony. colonies imaged repeatedly dur- adaptation capacity declined sharply time.            ing long incubation period 20 days. types colo- end phase   11011;1% cells plated               nies observed long incubation time. form visible adapted colonies. Notably, decline             adapted colonies size visible naked adaptive potential time argues existence            eye  containing 106 cells. previously  advantageous subpopulation phase sup-                 scribed, adapted colonies appeared 6 days <br />  <br /> HFSP Journal Vol. 4, June-August 2010                                                                                                 135 <br />  HFSP Journal <br />  <br />  <br /> post-plating joined continuously maximal <br /> fraction adapted colonies reached 20 days <br /> (Supplementary Material Fig. <span id='am-828' about='obo:IMR_0702943' typeof='owl:Thing'><span id='am-829' property="rdfs:label" content="PROTEASOME 26S SUBUNIT, NON-ATPASE, 2" datatype="xsd:string"></span><span id='am-830' property="rdfs:label" content="PSMD2" datatype="xsd:string"></span><span id='am-831' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S, SUBUNIT 2" datatype="xsd:string"></span><span id='am-832' property="oboInOwl:hasExactSynonym" content="S2" datatype="xsd:string"></span><span id='am-833' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-834' property="oboInOwl:hasExactSynonym" content="TRAP2" datatype="xsd:string"></span><span id='am-835' property="rdfs:label" content="PSMD2_HUMAN" datatype="xsd:string"></span><span id='am-836' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 2" datatype="xsd:string"></span><span id='am-837' property="oboInOwl:hasExactSynonym" content="TNFR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-838' property="rdfs:label" content="26S proteasome subunit p97" datatype="xsd:string"></span><span id='am-839' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor type 1 receptor-associated protein 2" datatype="xsd:string"></span><span id='am-840' property="rdfs:label" content="26S proteasome regulatory subunit S2" datatype="xsd:string"></span><span id='am-841' property="oboInOwl:hasExactSynonym" content="Protein 55.11" datatype="xsd:string"></span><span id='am-842' property="rdfs:label" content="PROTEASOME 26S, SUBUNIT 2" datatype="xsd:string"></span><span id='am-843' property="oboInOwl:id" content="IMR:0702943" datatype="xsd:string"></span><span id='am-844' property="oboInOwl:hasExactSynonym" content="MGC14274" datatype="xsd:string"></span><span id='am-845' property="oboInOwl:hasExactSynonym" content="TUMOR NECROSIS FACTOR RECEPTOR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-846' property="oboInOwl:hasExactSynonym" content="26S proteasome subunit p97" datatype="xsd:string"></span><span id='am-847' property="oboInOwl:hasExactSynonym" content="P97" datatype="xsd:string"></span><span id='am-848' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 2" datatype="xsd:string"></span><span id='am-849' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN1" datatype="xsd:string"></span><span id='am-850' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S SUBUNIT, NON-ATPASE, 2" datatype="xsd:string"></span><span id='am-851' property="oboInOwl:hasExactSynonym" content="Rpn1" datatype="xsd:string"></span><span id='am-852' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 2" datatype="xsd:string"></span><span id='am-853' property="rdfs:label" content="Protein 55.11" datatype="xsd:string"></span><span id='am-854' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 2" datatype="xsd:string"></span><span id='am-855' property="rdfs:label" content="TNFR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-856' property="rdfs:label" content="Rpn1" datatype="xsd:string"></span><span id='am-857' property="rdfs:label" content="26S proteasome regulatory subunit RPN1" datatype="xsd:string"></span><span id='am-858' property="rdfs:label" content="Tumor necrosis factor type 1 receptor-associated protein 2" datatype="xsd:string"></span><span id='am-859' property="rdfs:label" content="P97" datatype="xsd:string"></span><span id='am-860' property="oboInOwl:hasDbXref" content="UniProt:Q13200" datatype="xsd:string"></span><span id='am-861' property="rdfs:label" content="TRAP2" datatype="xsd:string"></span><span id='am-862' property="rdfs:label" content="S2" datatype="xsd:string"></span><span id='am-863' property="rdfs:label" content="MGC14274" datatype="xsd:string"></span><span id='am-864' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S2" datatype="xsd:string"></span><span id='am-865' property="rdfs:label" content="TUMOR NECROSIS FACTOR RECEPTOR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-866' property="oboInOwl:hasExactSynonym" content="PSMD2" datatype="xsd:string"></span>S2</span>). variability formation <br /> time adapted colonies plates indicated adaptation independently gained cell lineage. second type colonies microcolonies observed microscope remained entire incubation <br /> period 20 days (Supplementary Material Fig. <span id='am-764' about='obo:IMR_0702945' typeof='owl:Thing'><span id='am-765' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S3" datatype="xsd:string"></span><span id='am-766' property="rdfs:label" content="RPN3" datatype="xsd:string"></span><span id='am-767' property="rdfs:label" content="Proteasome subunit p58" datatype="xsd:string"></span><span id='am-768' property="oboInOwl:hasExactSynonym" content="P58" datatype="xsd:string"></span><span id='am-769' property="oboInOwl:id" content="IMR:0702945" datatype="xsd:string"></span><span id='am-770' property="rdfs:label" content="PSMD3" datatype="xsd:string"></span><span id='am-771' property="oboInOwl:hasExactSynonym" content="Proteasome subunit p58" datatype="xsd:string"></span><span id='am-772' property="oboInOwl:hasExactSynonym" content="TSTA2" datatype="xsd:string"></span><span id='am-773' property="rdfs:label" content="Rpn3" datatype="xsd:string"></span><span id='am-774' property="rdfs:label" content="P58" datatype="xsd:string"></span><span id='am-775' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 3" datatype="xsd:string"></span><span id='am-776' property="oboInOwl:hasExactSynonym" content="Rpn3" datatype="xsd:string"></span><span id='am-777' property="oboInOwl:hasDbXref" content="UniProt:O43242" datatype="xsd:string"></span><span id='am-778' property="oboInOwl:hasExactSynonym" content="S3" datatype="xsd:string"></span><span id='am-779' property="rdfs:label" content="26S proteasome regulatory subunit RPN3" datatype="xsd:string"></span><span id='am-780' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 3" datatype="xsd:string"></span><span id='am-781' property="rdfs:label" content="S3" datatype="xsd:string"></span><span id='am-782' property="rdfs:label" content="26S proteasome regulatory subunit S3" datatype="xsd:string"></span><span id='am-783' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-784' property="rdfs:label" content="TSTA2" datatype="xsd:string"></span><span id='am-785' property="rdfs:label" content="PSMD3_HUMAN" datatype="xsd:string"></span><span id='am-786' property="oboInOwl:hasExactSynonym" content="PSMD3" datatype="xsd:string"></span><span id='am-787' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 3" datatype="xsd:string"></span><span id='am-788' property="oboInOwl:hasExactSynonym" content="RPN3" datatype="xsd:string"></span><span id='am-789' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 3" datatype="xsd:string"></span><span id='am-790' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN3" datatype="xsd:string"></span>S3</span>). <br /> Microcolonies variable sizes, single cells, suggesting cells originated variable, limited, division capacity compared adapted <br /> cells. discrepancy colonies proliferated visible size ones remained aborted small sizes clearly observed later stages incubation period <br /> (Supplementary Material Fig. S3) essentially  <br /> tected days plating adapted colonies large counted eye <br /> [Fig. 4 ]. <br />     quantify variability, sizes colonies grown seed cells extracted different time points batch <br /> cultures measured microscopy days post- <br /> plating. Similar exponential decay fraction adapted colonies,  average colony area decreased exposure time glucose (Supplementary Material <br /> Fig. S1b).  decline adaptation capability        Figure 4. Distribution colony sizes.  20849;a 20850; representative mi- <br /> cells exposure time glucose signature      croscopy image colonies grown Glu plate average distribution colony sizes           days. colonies formed seed cells sampled                                                                     liquid culture onset transition Glu  Note coexist- days post-plating.  average, cells             ence wide range colony sizes single cell. <br /> sampled immediately exposure Glu formed                Scale image  11011;320  9262;m.  20849;b 20850; distributions colonies area  20849;in <br /> larger microcolonies size distribution           arbitrary units 20850; based numerous microscopy images, broader compared cells sampled later batch cul-         panel  20849;a 20850;, including colonies  Curves different <br />                                                                     colors different times transition batch culture <br /> ture [Fig. 4(b)]. addition, time Glu plates, <br />                                                                     Glu  20849;black, t = 0; red, t = 4 h; blue, t = 12 h 20850;. time point batch culture, size distribution tail    curves color different symbols microcolonies shifted larger colonies           plates measured earlier  20849;46 8211;57 h; squares 20850; later <br /> eventually adapted.  wide range                  20849; 11011;120 <span id='am-146' about='obo:IMR_0200533' typeof='owl:Thing'><span id='am-147' property="oboInOwl:hasExactSynonym" content="EC 6.1.1.21" datatype="xsd:string"></span><span id='am-148' property="rdfs:label" content="EC 6.1.1.21" datatype="xsd:string"></span><span id='am-149' property="rdfs:label" content="Histidine--tRNA ligase" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:id" content="IMR:0200533" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="Histidyl-tRNA synthetase, cytoplasmic" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasExactSynonym" content="HisRS" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasExactSynonym" content="FLJ20491" datatype="xsd:string"></span><span id='am-155' property="rdfs:label" content="HRS" datatype="xsd:string"></span><span id='am-156' property="rdfs:label" content="SYHC_HUMAN" datatype="xsd:string"></span><span id='am-157' property="rdfs:label" content="histidyl-tRNA synthetase" datatype="xsd:string"></span><span id='am-158' property="rdfs:label" content="FLJ20491" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasExactSynonym" content="HRS" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasExactSynonym" content="Histidine--tRNA ligase" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasExactSynonym" content="HARS" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasExactSynonym" content="histidyl-tRNA synthetase" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasDbXref" content="UniProt:P12081" datatype="xsd:string"></span><span id='am-164' property="rdfs:label" content="HARS" datatype="xsd:string"></span><span id='am-165' property="rdfs:label" content="HisRS" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasExactSynonym" content="Histidyl-tRNA synthetase, cytoplasmic" datatype="xsd:string"></span>hrs</span>; dots 20850; plating. curves normalized unit <br /> smaller microcolonies adapt remained <span id='am-639' about='obo:IMR_0010753' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000429 obo:IMR_0000428 obo:IMR_0001014'><span id='am-640' property="oboInOwl:hasExactSynonym" content="ASF" datatype="xsd:string"></span><span id='am-641' property="oboInOwl:hasExactSynonym" content="AF" datatype="xsd:string"></span><span id='am-642' property="oboInOwl:hasExactSynonym" content="Multiubiquitin chain binding protein" datatype="xsd:string"></span><span id='am-643' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 4" datatype="xsd:string"></span><span id='am-644' property="oboInOwl:id" content="IMR:0010753" datatype="xsd:string"></span><span id='am-645' property="rdfs:label" content="ASF" datatype="xsd:string"></span><span id='am-646' property="rdfs:label" content="Multiubiquitin chain binding protein" datatype="xsd:string"></span><span id='am-647' property="rdfs:label" content="AF" datatype="xsd:string"></span><span id='am-648' property="rdfs:label" content="Rpn10" datatype="xsd:string"></span><span id='am-649' property="rdfs:label" content="26S proteasome regulatory subunit S5A" datatype="xsd:string"></span><span id='am-650' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-651' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S5A" datatype="xsd:string"></span><span id='am-652' property="oboInOwl:hasExactSynonym" content="Rpn10" datatype="xsd:string"></span><span id='am-653' property="oboInOwl:hasExactSynonym" content="Antisecretory factor-1" datatype="xsd:string"></span><span id='am-654' property="rdfs:label" content="Antisecretory factor-1" datatype="xsd:string"></span>af</span>-         area presented probability density comparison  20849;x-axis <br />                                                                     divided by104 20850;. bar denotes approximate distance ter 20 days. Similar microscopy results obtained           x-axis colony  11011;70 cells. <br /> cells harvested phase chemostat (Supplemen- <br /> tary Material Fig. <span id='am-700' about='obo:IMR_0010555' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000896 obo:IMR_0100710 obo:IMR_0000001 obo:IMR_0000950'><span id='am-701' property="rdfs:label" content="40S small ribosomal protein 2" datatype="xsd:string"></span><span id='am-702' property="rdfs:label" content="40S ribosomal protein S2" datatype="xsd:string"></span><span id='am-703' property="oboInOwl:hasExactSynonym" content="S4" datatype="xsd:string"></span><span id='am-704' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-705' property="oboInOwl:hasExactSynonym" content="40S small ribosomal protein 2" datatype="xsd:string"></span><span id='am-706' property="rdfs:label" content="S4" datatype="xsd:string"></span><span id='am-707' property="rdfs:label" content="LLREP3 protein" datatype="xsd:string"></span><span id='am-708' property="oboInOwl:id" content="IMR:0010555" datatype="xsd:string"></span><span id='am-709' property="oboInOwl:hasExactSynonym" content="LLREP3 protein" datatype="xsd:string"></span>S4</span>).  population structure      explaining collapse chemostat population phase cells, average formed smaller microcolonies suc-        II.  determined growth dynamics indi- <br /> ceeded adapt, longer experienced chal-         vidual cells   population, expo- <br /> lenging environment.                                                sure time new challenging Glu environment. <br />     presence matured colonies alongside aborted          cells lineages actually escape microcolonies likely differences growth      fate division arrest switching adapted state rates. Tiny microcolonies 20 days,            forming mature colonies propagated. time number adapted colonies               essence adaptation switching reached maximal value (Supplementary Material Fig.              cell state allows regular stable metabolism including  <br /> S2).  similar growth rates batch cultures          tidine production   unlimited propagation. measured adapted naive cells gen-         variable cell division capacity, parameter  <br /> erations following initial exposure Glu (Fig. 5).      termines chances adaptation, corroborates notion  limited number cell divisions <span id='am-1141' about='obo:IMR_0701167' typeof='owl:Thing'><span id='am-1142' property="rdfs:label" content="Cone arrestin" datatype="xsd:string"></span><span id='am-1143' property="rdfs:label" content="arrestin 4" datatype="xsd:string"></span><span id='am-1144' property="rdfs:label" content="cArr" datatype="xsd:string"></span><span id='am-1145' property="rdfs:label" content="Arr3" datatype="xsd:string"></span><span id='am-1146' property="oboInOwl:hasExactSynonym" content="CAR" datatype="xsd:string"></span><span id='am-1147' property="rdfs:label" content="CAR" datatype="xsd:string"></span><span id='am-1148' property="oboInOwl:hasDbXref" content="UniProt:Q9EQP6" datatype="xsd:string"></span><span id='am-1149' property="oboInOwl:hasExactSynonym" content="Arr3" datatype="xsd:string"></span><span id='am-1150' property="rdfs:label" content="X-arrestin" datatype="xsd:string"></span><span id='am-1151' property="oboInOwl:hasExactSynonym" content="cone arrestin" datatype="xsd:string"></span><span id='am-1152' property="rdfs:label" content="Retinal cone arrestin-3" datatype="xsd:string"></span><span id='am-1153' property="oboInOwl:hasExactSynonym" content="arrestin 4" datatype="xsd:string"></span><span id='am-1154' property="rdfs:label" content="ARRC_MOUSE" datatype="xsd:string"></span><span id='am-1155' property="rdfs:label" content="mCAR" datatype="xsd:string"></span><span id='am-1156' property="oboInOwl:hasExactSynonym" content="Cone arrestin" datatype="xsd:string"></span><span id='am-1157' property="oboInOwl:hasExactSynonym" content="cArr" datatype="xsd:string"></span><span id='am-1158' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1159' property="rdfs:label" content="mCARFL" datatype="xsd:string"></span><span id='am-1160' property="oboInOwl:hasExactSynonym" content="X-arrestin" datatype="xsd:string"></span><span id='am-1161' property="rdfs:label" content="Arr4" datatype="xsd:string"></span><span id='am-1162' property="oboInOwl:hasExactSynonym" content="Retinal cone arrestin-3" datatype="xsd:string"></span><span id='am-1163' property="rdfs:label" content="cone arrestin" datatype="xsd:string"></span><span id='am-1164' property="rdfs:label" content="Arrestin-C" datatype="xsd:string"></span><span id='am-1165' property="oboInOwl:hasExactSynonym" content="arrestin 3, retinal" datatype="xsd:string"></span><span id='am-1166' property="oboInOwl:id" content="IMR:0701167" datatype="xsd:string"></span><span id='am-1167' property="oboInOwl:hasExactSynonym" content="mCARFL" datatype="xsd:string"></span><span id='am-1168' property="rdfs:label" content="arrestin 3, retinal" datatype="xsd:string"></span><span id='am-1169' property="oboInOwl:hasExactSynonym" content="mCAR" datatype="xsd:string"></span><span id='am-1170' property="oboInOwl:hasExactSynonym" content="Arr4" datatype="xsd:string"></span><span id='am-1171' property="oboInOwl:hasExactSynonym" content="Arrestin-C" datatype="xsd:string"></span>car</span>-       adaptation developed individual lineages ried lineage reason wide size vari-     response challenging environment rare <br /> ability microcolonies. Advancing deeper phase             property carried cells Gal Glu caused cells arrest division fewer generations,       medium switch. <br />  <br /> 136                                                                        Inherited adaptation genome-rewired . . . | David et al. <br />                                                                                                                                      RT C L E <br />  <br />  <br />  <br />  <br /> Figure 5. Growth curves parallel populations batch cul- <br /> tures. Adapted  20849;red 20850; versus naive  20849;nonadapted, black 20850; cells switched Gal Glu control adapted cells <br />  20849;blue 20850; propagated just Glu  strains exhibited simi- <br /> lar growth rates Glu environment. different lines measurements different clones averaged replicates,           Figure 6. Simulated distributions colony size. Monte-Carlo <br /> showing 7% variability growth rates.                    simulations colonies initiated single cells pro- <br />                                                                            liferated opposing processes: decaying probability                                                                            continually divide increasing probability acquire adaptation model adaptation containing competing                             unlimited divisions, exponential function exposure <br /> cellular processes                                                         time Glu  20849;see Materials methods section details 20850;.                                                                            different curves different values parameter tg, control- <br /> Based experimental results presented                   ling processes representing exposure time Glu mulated model population dynamics. previous                  seed cells  20849;see Materials methods section 20850;. colony <br /> population average model (Stolovicki et al., 2006),                     sizes measured number cells colony. curves showed population structure, allowing switching                 normalized unit area presented probability density adapted nonadapted states fitness advan-                comparison. Inset: maximal fraction adapted simulated colo- <br />                                                                            nies  20849;i.e., colonies containing adapted cell   <br /> tage adapted state, inconsistent observed               cell division indefinitely 20850; log scale, function chemostat dynamics large fraction adapted cells al-                exposure time Glu seed cells. <br /> ready exists onset transition glucose.  account fact adapted cells fact               creasing function time (inset Supplementary Mate- fitness advantage model competing cellu-                rial Fig. S5). processes competed rates oc- <br /> lar processes underlying population dynamics: pheno-                 currence depended history exposure glucose. <br /> typic switch limited growth capacity Glu ei-               model computed forms; chemostat-like <br /> ther arrested state adapted state                   setting subpopulations grew limiting <br /> unlimited division capacity. rewired cells initially             resource environment (requiring addition equations <br /> proliferate glucose medium rate similar adapted                 describing dynamics subpopulations (naive cells limited number generations             adapted), equation describing dynamics division arrested unless switched stable             limiting resource), colonies plate limit- <br /> adapted state allowing maintain proliferation.                     ing resource. Solving plate model numerically showed <br /> Switching cell, naive adapted arrested             seed-cell forming larger microcolony state, occurred wide time range assumed               daughter cells ceased dividing, significantly random event. event described prob-              larger overall probability (probability cell multiplied ability unit time simulations following single cells                number cells colony) yield adapted colony. <br /> (Monte-Carlo simulations) rate constant numeri-                model reproduced experimentally observed colony <br /> cal integration average population dynamic equations                size distributions plates (Fig. 6), fast decay chemostat setting. Importantly, long exposure time                 maximal fraction adapted colonies exposure <br /> naive cells Glu medium reduced chance adapt                time glucose chemostat batch culture experiments   interesting ingredient model uti-           (Fig. 6 inset). Similarly, chemostat-like model repro- <br /> lization transition rates  probabilities) depend                duced main features population growth dynamics <br /> strongly integrated exposure time Glu               (Supplementary Material Fig. S5). history cell. probability cell maintain divisions assumed exponentially decreasing                DISCUSSION <br /> function past exposure time Glu  Similarly,              studying cells rewired genome demonstrated <br /> probability adapt assumed exponentially                 alternative adaptation process following charac- <br />  <br /> HFSP Journal Vol. 4, June-August 2010                                                                                                          137 <br />  HFSP Journal <br />  <br />  <br /> teristics. (1) fraction naive cells adapted ex-    noteworthy adaptation phenomenon observed tremely high variable average 50%. (2)        experiments acquisition new adaptation genuine phenotype adapted         metabolism reorganization stabilization <br /> cells distinct naive cells phe-    familiar growth capacity constraints ge- <br /> notype stably inherited. (3) adaptation popu-       nome rewiring event regulatory challenge pre- <br /> lation occurred rapidly 10 8211;30 generations, probably         sented Glu  precisely challenge posed large fraction cells adapted. (4) adap-     genome rewiring events evolution; stabilization famil- <br /> tation population relied individual cells inde-     iar protein functions new contexts.  used pendently developed adaptive phenotype response        term adaptation, usually utilized  challenging environment. Undoubtedly, character-         crease fitness population average means selec- <br /> istics signify adaptation phenomenon does          tion, adjustments regulation enable <br /> form common view evolutionary process            single cells stabilize familiar metabolic state. ad-  extends framework understanding          justments, shown, rapidly gained indi- <br /> new phenotypes evolve.                                             vidual cells stably propagated generations  <br />     Adaptation commonly viewed change pheno-             considered adaptation type gene frequencies response change environ-      significant role evolution regulatory systems. <br /> mental conditions. adaptation carried          doubt fraction cells, adapted familiar glucose-containing environment challenge          plates, solid liquid medium experiments, was satisfy internal cellular requirements, imposed      unprecedentedly high   stands contrast previ- frustrating environmental conditions exclusive      ous evolutionary experiments lower rates <br /> linkage HIS3 repressed GAL  dynamics           105  8211; 109 cells) adaptive phenotypes rose population phases II chemostat         stressful conditions (Paquin Adams, 1983; Lenski somewhat unexpected given observed behavior          Travisano, 1994; Storchov 225; et al., 1998; Elena Lenski, <br /> dividuals. phase population grew exponentially,       2003; Heidenreich, 2007). previous studies adaptive pheno- individuals progressively lost adaptation capa-     types observed rates matched rates genetic <br /> bility phase II population collapsed        mutations   studies, mutations individuals gained adaptation. fact,    causative agents new phenotypes.  indications exposure Glu       rate adapted phenotypes rose does match dium, naive cells similar metabolism         known rates mutations.  mutations,  pro- <br /> adapted cells despite internal challenge imposed        vided mechanism adaptation cell genome rewiring. Naive cells, exposed            experiments arisen follow- <br /> glucose  grew defined single rate       ing processes. random mutations occurred conven- encounter Glu liquid medium evident          tional low rate rare advantageous ones propagated optical density plate counts measurements (Fig. 2         population prior exposure glucose, explaining Supplementary Material Fig. S1a).  similar            high fraction adapted cells plates rapid adaptation <br /> growth rates measured naive adapted cells          chemostat population.  cells acquired adaptive muta- <br /> batch cultures, moved Gal          tions (mutations directed advantageous positions) Glu propagated fresh Glu        high rate exposure glucose. Based plate <br /> medium (Fig. 5). agreement, chemostat measurements          growth patterns experiment types, galactose popula- <br /> showed similar growth exponents phases III,         tions contain detectable fraction cells contained naive fully adapted cells, respectively, (Fig. 2).   advantageous phenotypes.  cells lost ca- <br /> Furthermore, maximal cell density reached non-              pacity adapt exposure time glucose,  overall <br /> adapted cells phase chemostat similar      adaptation process rapid.  option fully adapted cells glucose steady-state (phase IV).     exposure glucose, population density cells chemostat sensitive function   contained substantial fraction advantageous mutated geno- average cellular metabolic state, determines         types facilitated rapid adaptation inconsistent capacity population maintain stable growth rate.       results. hand, adaptive mutations arise similar densities similar growth rates,  indicate    response stressful environments allow wide- cells phases IV similar metabolic states.       spread adaptation individuals rapid adaptation Furthermore, similarity initial growth rate     population.   spite major <br /> naive cells propagation rate adapted cells lends        efforts identify cases support lack advantageous subpopulation       debate, conclusive evidence literature phase   given familiar glucose-             existence adaptive mutations (Cairns et al., 1988; Cairns containing environment similarity metabolism,        Foster, 1991; Steele Jinks-Robertson, 1992; Hall, 1998a, <br />  <br /> 138                                                                      Inherited adaptation genome-rewired . . . | David et al. <br />                                                                                                                          RT C L E <br />  <br />  <br />  <br />  <br /> 1998b; Rosenberg, 2001; Roth et al., 2006). adaptive         mechanism enabling inherited cellular adaptation mutations consensus, previous studies revertants       induced unforeseen challenge cells simulta- <br /> response stressful conditions induced adaptive mutations      neously. regulatory nature challenge, abun- reported occur minute fraction  20849; 11011;10 8722;6 20850;      dance adapted cells, rapid dynamics make cellular <br /> population (Storchov 225; et al., 1998; Heidenreich, 2007)       plasticity supported epigenetic processes reasonable 50% cells experiments. phenom-        candidate mediate induced adaptation process ena resemblance described        stable inheritance (Lachmann Jablonka, 1996; Jablonka <br /> studied. Phenotypic switching cases phenotype           Lamb, 2005; Ahmed Brickner, 2007; Allis et al., unstable changed according environment           2007; Brickner et al., 2007; Jablonka Raz, 2009). studies (Hallet, 2001; Balaban et al., 2004; Levin Rozen,        unforeseen challenges bound encountered 2006). classes phenotypic switching, sug-         organism, accessible rapid adaptation based cell <br /> gested chaperons act capacitors phenotypic variation      plasticity enable cells overcome immediate severe expose hidden genetic variation response environmen-      challenges relying preexisting solutions <span id='am-334' about='obo:IMR_0704587' typeof='owl:Thing'><span id='am-335' property="oboInOwl:hasExactSynonym" content="EC 3.4.21.75" datatype="xsd:string"></span><span id='am-336' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-337' property="oboInOwl:hasExactSynonym" content="FURIN" datatype="xsd:string"></span><span id='am-338' property="rdfs:label" content="Dibasic-processing enzyme" datatype="xsd:string"></span><span id='am-339' property="rdfs:label" content="PCSK3" datatype="xsd:string"></span><span id='am-340' property="oboInOwl:hasExactSynonym" content="SPC1" datatype="xsd:string"></span><span id='am-341' property="rdfs:label" content="PACE" datatype="xsd:string"></span><span id='am-342' property="rdfs:label" content="Paired basic amino acid residue-cleaving enzyme" datatype="xsd:string"></span><span id='am-343' property="rdfs:label" content="PAIRED BASIC AMINO ACID CLEAVING ENZYME" datatype="xsd:string"></span><span id='am-344' property="oboInOwl:hasExactSynonym" content="PAIRED BASIC AMINO ACID CLEAVING ENZYME" datatype="xsd:string"></span><span id='am-345' property="oboInOwl:hasExactSynonym" content="FUR" datatype="xsd:string"></span><span id='am-346' property="oboInOwl:hasExactSynonym" content="furin (paired basic amino acid cleaving enzyme)" datatype="xsd:string"></span><span id='am-347' property="rdfs:label" content="Furin" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasDbXref" content="UniProt:P09958" datatype="xsd:string"></span><span id='am-349' property="rdfs:label" content="EC 3.4.21.75" datatype="xsd:string"></span><span id='am-350' property="oboInOwl:hasExactSynonym" content="Paired basic amino acid residue-cleaving enzyme" datatype="xsd:string"></span><span id='am-351' property="oboInOwl:hasExactSynonym" content="Furin" datatype="xsd:string"></span><span id='am-352' property="rdfs:label" content="FUR" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasExactSynonym" content="Dibasic-processing enzyme" datatype="xsd:string"></span><span id='am-354' property="rdfs:label" content="SPC1" datatype="xsd:string"></span><span id='am-355' property="rdfs:label" content="FURIN" datatype="xsd:string"></span><span id='am-356' property="oboInOwl:id" content="IMR:0704587" datatype="xsd:string"></span><span id='am-357' property="oboInOwl:hasExactSynonym" content="PCSK3" datatype="xsd:string"></span><span id='am-358' property="rdfs:label" content="FURIN_HUMAN" datatype="xsd:string"></span><span id='am-359' property="oboInOwl:hasExactSynonym" content="PACE" datatype="xsd:string"></span><span id='am-360' property="rdfs:label" content="furin (paired basic amino acid cleaving enzyme)" datatype="xsd:string"></span>fur</span>- <br /> <span id='am-207' about='obo:IMR_0200565' typeof='owl:Thing'><span id='am-208' property="oboInOwl:hasAlternativeId" content="IMR:0702871" datatype="xsd:string"></span><span id='am-209' property="rdfs:label" content="TALDO" datatype="xsd:string"></span><span id='am-210' property="rdfs:label" content="TALH" datatype="xsd:string"></span><span id='am-211' property="rdfs:label" content="TAL-H" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasExactSynonym" content="TALDO" datatype="xsd:string"></span><span id='am-213' property="rdfs:label" content="Transaldolase" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasExactSynonym" content="TALDO1" datatype="xsd:string"></span><span id='am-215' property="oboInOwl:hasExactSynonym" content="Transaldolase" datatype="xsd:string"></span><span id='am-216' property="rdfs:label" content="TALDO_HUMAN" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-218' property="rdfs:label" content="EC 2.2.1.2" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasDbXref" content="UniProt:P37837" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasExactSynonym" content="EC 2.2.1.2" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasExactSynonym" content="transaldolase 1" datatype="xsd:string"></span><span id='am-222' property="rdfs:label" content="transaldolase 1" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasExactSynonym" content="TALH" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasExactSynonym" content="TAL" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasExactSynonym" content="TAL-H" datatype="xsd:string"></span><span id='am-226' property="rdfs:label" content="TALDOR" datatype="xsd:string"></span><span id='am-227' property="rdfs:label" content="TAL" datatype="xsd:string"></span><span id='am-228' property="rdfs:label" content="TALDO1" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasExactSynonym" content="TALDOR" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:id" content="IMR:0200565" datatype="xsd:string"></span>tal</span> changes (Rutherford Lindquist, 1998; Queitsch et al.,        ther maybe rare advantageous genotypes real- <br /> 2002). adaptation occurring experiments             ized fixed population (West-Eberhard, 2003). shares features phenomena, fundamen-          implications mechanism far reaching diverse <br /> tally differs  mainly rate   areas biology; behavior single cells tissues adaptive phenotype rose conclusion inherited     health disease dynamics populations nature. <br /> phenotype induced environment selected         particular, adaptation capability supports signifi-  know actual mechanism adaptation       potential genomic rewiring evolution biological genetic mutations evolutionary      networks life itself. <br /> process.  stable advantageous change pheno- <br /> type observed, probable explanations                                                                      MATERIALS METHODS <br /> high rate adaptation adaptive mutations mechanisms mutations, referred epigenetic pro-    Strain growth conditions <br /> cesses (Lachmann Jablonka, 1996; Jablonka Lamb,              Experiments carried haploid yeast strain <br /> 2005; Allis et al., 2007; Jablonka Raz, 2009).                   YPH499 [Mata, ura3 8211;52, <span id='am-655' about='obo:IMR_0002022' typeof='owl:Thing'><span id='am-656' property="oboInOwl:hasExactSynonym" content="AASDPPT" datatype="xsd:string"></span><span id='am-657' property="oboInOwl:hasExactSynonym" content="EC 2.7.8.-" datatype="xsd:string"></span><span id='am-658' property="rdfs:label" content="DKFZp566E2346" datatype="xsd:string"></span><span id='am-659' property="oboInOwl:hasExactSynonym" content="LYS2" datatype="xsd:string"></span><span id='am-660' property="rdfs:label" content="LYS2" datatype="xsd:string"></span><span id='am-661' property="rdfs:label" content="L-aminoadipate-semialdehyde dehydrogenase-phosphopantetheinyl transferase" datatype="xsd:string"></span><span id='am-662' property="oboInOwl:hasDbXref" content="UniProt:Q9NRN7" datatype="xsd:string"></span><span id='am-663' property="oboInOwl:hasExactSynonym" content="AASD-PPT" datatype="xsd:string"></span><span id='am-664' property="oboInOwl:hasExactSynonym" content="AASDHPPT" datatype="xsd:string"></span><span id='am-665' property="oboInOwl:hasExactSynonym" content="L-aminoadipate-semialdehyde dehydrogenase-phosphopantetheinyl transferase" datatype="xsd:string"></span><span id='am-666' property="rdfs:label" content="LYS5" datatype="xsd:string"></span><span id='am-667' property="rdfs:label" content="AASDHPPT" datatype="xsd:string"></span><span id='am-668' property="rdfs:label" content="Alpha-aminoadipic semialdehyde dehydrogenase-phosphopantetheinyl transferase" datatype="xsd:string"></span><span id='am-669' property="rdfs:label" content="AASD-PPT" datatype="xsd:string"></span><span id='am-670' property="rdfs:label" content="ALPHA-AMINOADIPATE SEMIALDEHYDE DEHYDROGENASE-PHOSPHOPANTETHEINYL TRANSFERASE" datatype="xsd:string"></span><span id='am-671' property="rdfs:label" content="aminoadipate-semialdehyde dehydrogenase-phosphopantetheinyl transferase" datatype="xsd:string"></span><span id='am-672' property="oboInOwl:hasExactSynonym" content="ALPHA-AMINOADIPATE SEMIALDEHYDE DEHYDROGENASE-PHOSPHOPANTETHEINYL TRANSFERASE" datatype="xsd:string"></span><span id='am-673' property="oboInOwl:hasExactSynonym" content="LYS5, S. CEREVISIAE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-674' property="rdfs:label" content="LYS5 ortholog" datatype="xsd:string"></span><span id='am-675' property="oboInOwl:hasExactSynonym" content="LYS5 ortholog" datatype="xsd:string"></span><span id='am-676' property="oboInOwl:hasExactSynonym" content="Alpha-aminoadipic semialdehyde dehydrogenase-phosphopantetheinyl transferase" datatype="xsd:string"></span><span id='am-677' property="rdfs:label" content="EC 2.7.8.-" datatype="xsd:string"></span><span id='am-678' property="oboInOwl:hasExactSynonym" content="DKFZP566E2346" datatype="xsd:string"></span><span id='am-679' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-680' property="rdfs:label" content="AASDPPT" datatype="xsd:string"></span><span id='am-681' property="oboInOwl:hasExactSynonym" content="4&apos;-phosphopantetheinyl transferase" datatype="xsd:string"></span><span id='am-682' property="oboInOwl:hasExactSynonym" content="DKFZp566E2346" datatype="xsd:string"></span><span id='am-683' property="rdfs:label" content="LYS5, S. CEREVISIAE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-684' property="rdfs:label" content="ADPPT_HUMAN" datatype="xsd:string"></span><span id='am-685' property="rdfs:label" content="4&apos;-phosphopantetheinyl transferase" datatype="xsd:string"></span><span id='am-686' property="oboInOwl:hasExactSynonym" content="CGI-80" datatype="xsd:string"></span><span id='am-687' property="oboInOwl:hasExactSynonym" content="aminoadipate-semialdehyde dehydrogenase-phosphopantetheinyl transferase" datatype="xsd:string"></span><span id='am-688' property="rdfs:label" content="CGI-80" datatype="xsd:string"></span><span id='am-689' property="rdfs:label" content="DKFZP566E2346" datatype="xsd:string"></span><span id='am-690' property="oboInOwl:hasExactSynonym" content="LYS5" datatype="xsd:string"></span><span id='am-691' property="oboInOwl:id" content="IMR:0002022" datatype="xsd:string"></span>lys2</span> 8211;801, <span id='am-22' about='obo:IMR_0702619' typeof='owl:Thing'><span id='am-23' property="rdfs:label" content="Phosphoribosylaminoimidazole-succinocarboxamide synthase" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="ADE2H1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="Multifunctional protein ADE2" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="UniProt:P22234" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="PAICS" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="EC 6.3.2.6" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="phosphoribosylaminoimidazole carboxylase, phosphoribosylaminoimidazole succinocarboxamide synthetase" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="MGC5024" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="AIR carboxylase" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="MGC1343" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="PUR6_HUMAN" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="Phosphoribosylaminoimidazole carboxylase" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="AIRC" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="AIR carboxylase" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Phosphoribosylaminoimidazole-succinocarboxamide synthase" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Phosphoribosylaminoimidazole carboxylase" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="SAICAR synthetase" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="SAICAR synthetase" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="ADE2H1" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="ADE2" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="DKFZp781N1372" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="PAICS" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="PAIS" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="EC 4.1.1.21" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="PAIS" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="MGC5024" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="phosphoribosylaminoimidazole carboxylase, phosphoribosylaminoimidazole succinocarboxamide synthetase" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="AIRC" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="ADE2" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="MGC1343" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="EC 6.3.2.6" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Multifunctional protein ADE2" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="DKFZp781N1372" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:id" content="IMR:0702619" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="EC 4.1.1.21" datatype="xsd:string"></span>ade2</span> 8211;101, <span id='am-1' about='obo:IMR_0700457' typeof='owl:Thing'><span id='am-2' property="oboInOwl:hasDbXref" content="UniProt:P27768" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Troponin I, fast skeletal muscle" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Troponin I, fast-twitch isoform" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Troponin I, fast skeletal muscle" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="TNNI2_RAT" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Trp1" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="Troponin I, fast-twitch isoform" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0700457" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="Trp1" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="troponin I type 2 (skeletal, fast)" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="troponin I type 2 (skeletal, fast)" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Tnni2" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Tnni2" datatype="xsd:string"></span>trp1</span>- 9004;63, <br />      results presented place environment        his3 9004;200, <span id='am-305' about='obo:IMR_0703972' typeof='owl:Thing'><span id='am-306' property="rdfs:label" content="MAL" datatype="xsd:string"></span><span id='am-307' property="oboInOwl:hasExactSynonym" content="T8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-308' property="rdfs:label" content="CD8A" datatype="xsd:string"></span><span id='am-309' property="rdfs:label" content="OKT8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-310' property="rdfs:label" content="CD8" datatype="xsd:string"></span><span id='am-311' property="oboInOwl:hasExactSynonym" content="CD8" datatype="xsd:string"></span><span id='am-312' property="oboInOwl:hasExactSynonym" content="MAL" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasDbXref" content="UniProt:P01732" datatype="xsd:string"></span><span id='am-314' property="oboInOwl:hasExactSynonym" content="T-lymphocyte differentiation antigen T8/Leu-2" datatype="xsd:string"></span><span id='am-315' property="oboInOwl:id" content="IMR:0703972" datatype="xsd:string"></span><span id='am-316' property="oboInOwl:hasExactSynonym" content="T-cell surface glycoprotein CD8 alpha chain" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasExactSynonym" content="Leu2" datatype="xsd:string"></span><span id='am-318' property="oboInOwl:hasExactSynonym" content="p32" datatype="xsd:string"></span><span id='am-319' property="rdfs:label" content="CD8a molecule" datatype="xsd:string"></span><span id='am-320' property="oboInOwl:hasExactSynonym" content="CD8 ANTIGEN, ALPHA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-321' property="rdfs:label" content="CD8a" datatype="xsd:string"></span><span id='am-322' property="rdfs:label" content="T-lymphocyte differentiation antigen T8/Leu-2" datatype="xsd:string"></span><span id='am-323' property="rdfs:label" content="p32" datatype="xsd:string"></span><span id='am-324' property="rdfs:label" content="CD8A_HUMAN" datatype="xsd:string"></span><span id='am-325' property="oboInOwl:hasExactSynonym" content="CD8A" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasExactSynonym" content="OKT8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-327' property="oboInOwl:hasExactSynonym" content="CD8a" datatype="xsd:string"></span><span id='am-328' property="rdfs:label" content="Leu2" datatype="xsd:string"></span><span id='am-329' property="rdfs:label" content="T8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-330' property="oboInOwl:hasExactSynonym" content="CD8a molecule" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-332' property="rdfs:label" content="CD8 ANTIGEN, ALPHA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-333' property="rdfs:label" content="T-cell surface glycoprotein CD8 alpha chain" datatype="xsd:string"></span>leu2</span> 9004;1] carrying plasmid pESC-LEU (Strat- <br /> history population significant factors determining      agene) containing pGAL1-pGAL10 divergent promoter potential cells adapt evident dynamics        HIS3 pGAL1 (Stolovicki et al., 2006). dele- adaptation process. Pending exposure time            tion his3 9004;200 removed entire HIS3 coding region plus <br /> glucose, seed cells, descendents individually            upstream promoter region. Cells grown cus- <br /> switching phenotype chance stabilize adap-         <span id='am-59' about='obo:IMR_0702614' typeof='owl:Thing'><span id='am-60' property="rdfs:label" content="Pre-mRNA-processing factor 6" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="C20orf14" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="TOM" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:id" content="IMR:0702614" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="hPrp6" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="U5-102 kDa protein" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="PRP6 homolog" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="C20orf14" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="U5 snRNP-associated 102 kDa protein" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="TOM" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="PRP6_HUMAN" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="U5 snRNP-associated 102 kDa protein" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="PRPF6" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="hPrp6" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="U5-102K" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="U5-102 kDa protein" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="Prp6" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasExactSynonym" content="bB152O15.1" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="Pre-mRNA-processing factor 6" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="ANT-1" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasExactSynonym" content="PRP6 homolog" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="PRPF6" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Prp6" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="PRP6 pre-mRNA processing factor 6 homolog (S. cerevisiae)" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="UniProt:O94906" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="PRP6 pre-mRNA processing factor 6 homolog (S. cerevisiae)" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="bB152O15.1" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="ANT-1" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="U5-102K" datatype="xsd:string"></span>tom</span> chemostat (Stolovicki et al., 2006) synthetic <br /> tive metabolic state avoid arresting. phenotypic fea-        dropout medium lacking histidine leucine ap- <br /> tures population dynamics faithfully captured         propriate amino acid supplement 2% high grade apparently simple model involving competing pro-              pure galactose pure glucose sole carbon source cesses, causing switch naive initial state      (Stolovicki et al., 2006). Histidine leucine-lacking ga- arrested state adapted enabled unlim-        lactose glucose medium referred text ited cell division capacity. phenotypic switching event          Gal Glu  respectively. Growth chemostat <br /> depends time spent glucose,            limited concentration amino acid supple- <br /> tory cell. process leads cell arrest       ment. chemostat contains  11011;130 ml (109  8211; 1010 cells experiments presumably different observed            steady-state). generation time equals chemostat dilution starvation imposed            time x ln 2, 5 h. online measurement sys- <br /> cells facilitates generation adapted        tem used measure OD  600 nm) cells state  observation supported          chemostat (Stolovicki et al., 2006). <br /> previous work showing phase                experiments batch cultures, cells genomewide arbitrary transcriptional response involving            grown Gal medium midexponential phase large fraction genome having small overlap         washed switched Glu medium. Parallel popula- known stress responses (Stern et al., 2007).             tions incubated shaken 96 plate (LabNet <br /> dramatic transcriptional response significantly affect         plate shaker 30 176; C). Samples cells population physiology ways hand lead cell        plated agar rich medium colony counts used <br /> arrest open new opportunities           measure growth rates. addition, growth rates adapted <br /> adaptation.                                                          nonadapted cells measured Tecan Infinite-200 <br />       experiments prove existence cellular         plate reader shaking incubation 30 176; C. <br />  <br /> HFSP Journal Vol. 4, June-August 2010                                                                                             139 <br />  HFSP Journal <br />  <br /> Adaptation assays test plates                                                    b                          d <br /> Approximately 2 ml cells harvested chemo-             fs =                   ,   fa =                   . <br />                                                                               c + exp 20849; 9270;s  8722; t 20850;            e + exp 20849;  8722; t 20850; <br /> <span id='am-438' about='obo:IMR_0200736' typeof='owl:Thing'><span id='am-439' property="oboInOwl:hasExactSynonym" content="STAT" datatype="xsd:string"></span><span id='am-440' property="oboInOwl:hasExactSynonym" content="ACAT-1" datatype="xsd:string"></span><span id='am-441' property="rdfs:label" content="sterol O-acyltransferase 1" datatype="xsd:string"></span><span id='am-442' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-443' property="oboInOwl:hasExactSynonym" content="Acyl-coenzyme A:cholesterol acyltransferase 1" datatype="xsd:string"></span><span id='am-444' property="rdfs:label" content="STAT" datatype="xsd:string"></span><span id='am-445' property="rdfs:label" content="SOAT" datatype="xsd:string"></span><span id='am-446' property="oboInOwl:hasExactSynonym" content="Cholesterol acyltransferase 1" datatype="xsd:string"></span><span id='am-447' property="oboInOwl:hasExactSynonym" content="RP11-215I23.2" datatype="xsd:string"></span><span id='am-448' property="rdfs:label" content="Sterol O-acyltransferase 1" datatype="xsd:string"></span><span id='am-449' property="oboInOwl:hasExactSynonym" content="EC 2.3.1.26" datatype="xsd:string"></span><span id='am-450' property="oboInOwl:hasExactSynonym" content="ACAT" datatype="xsd:string"></span><span id='am-451' property="rdfs:label" content="SOAT1_HUMAN" datatype="xsd:string"></span><span id='am-452' property="oboInOwl:hasExactSynonym" content="STEROL ACYLTRANSFERASE" datatype="xsd:string"></span><span id='am-453' property="oboInOwl:hasExactSynonym" content="sterol O-acyltransferase 1" datatype="xsd:string"></span><span id='am-454' property="rdfs:label" content="ACAT" datatype="xsd:string"></span><span id='am-455' property="oboInOwl:hasExactSynonym" content="ACAT1" datatype="xsd:string"></span><span id='am-456' property="oboInOwl:hasExactSynonym" content="ACYL-CoA:CHOLESTEROL ACYLTRANSFERASE" datatype="xsd:string"></span><span id='am-457' property="rdfs:label" content="RP11-215I23.2" datatype="xsd:string"></span><span id='am-458' property="rdfs:label" content="ACACT" datatype="xsd:string"></span><span id='am-459' property="rdfs:label" content="STEROL ACYLTRANSFERASE" datatype="xsd:string"></span><span id='am-460' property="oboInOwl:hasExactSynonym" content="ACACT" datatype="xsd:string"></span><span id='am-461' property="oboInOwl:id" content="IMR:0200736" datatype="xsd:string"></span><span id='am-462' property="rdfs:label" content="ACAT1" datatype="xsd:string"></span><span id='am-463' property="rdfs:label" content="ACACT1" datatype="xsd:string"></span><span id='am-464' property="oboInOwl:hasExactSynonym" content="SOAT1" datatype="xsd:string"></span><span id='am-465' property="rdfs:label" content="ACAT-1" datatype="xsd:string"></span><span id='am-466' property="rdfs:label" content="ACYL-CoA:CHOLESTEROL ACYLTRANSFERASE" datatype="xsd:string"></span><span id='am-467' property="rdfs:label" content="SOAT1" datatype="xsd:string"></span><span id='am-468' property="rdfs:label" content="EC 2.3.1.26" datatype="xsd:string"></span><span id='am-469' property="oboInOwl:hasExactSynonym" content="Sterol O-acyltransferase 1" datatype="xsd:string"></span><span id='am-470' property="rdfs:label" content="Acyl-coenzyme A:cholesterol acyltransferase 1" datatype="xsd:string"></span><span id='am-471' property="oboInOwl:hasDbXref" content="UniProt:P35610" datatype="xsd:string"></span><span id='am-472' property="oboInOwl:hasExactSynonym" content="SOAT" datatype="xsd:string"></span><span id='am-473' property="oboInOwl:hasExactSynonym" content="ACACT1" datatype="xsd:string"></span><span id='am-474' property="rdfs:label" content="Cholesterol acyltransferase 1" datatype="xsd:string"></span>stat</span> precise time points, washed diluted OD= 1. microliters cell suspension diluted 1,000-     x1,2 fractions nonadapted adapted sub- <br /> <span id='am-620' about='obo:IMR_0703347' typeof='owl:Thing'><span id='am-621' property="oboInOwl:hasExactSynonym" content="EC 6.3.4.5" datatype="xsd:string"></span><span id='am-622' property="oboInOwl:hasExactSynonym" content="Citrulline--aspartate ligase" datatype="xsd:string"></span><span id='am-623' property="rdfs:label" content="argininosuccinate synthetase 1" datatype="xsd:string"></span><span id='am-624' property="oboInOwl:hasExactSynonym" content="argininosuccinate synthetase 1" datatype="xsd:string"></span><span id='am-625' property="rdfs:label" content="EC 6.3.4.5" datatype="xsd:string"></span><span id='am-626' property="oboInOwl:hasExactSynonym" content="fold" datatype="xsd:string"></span><span id='am-627' property="rdfs:label" content="Ass1" datatype="xsd:string"></span><span id='am-628' property="rdfs:label" content="fold" datatype="xsd:string"></span><span id='am-629' property="oboInOwl:hasExactSynonym" content="Ass1" datatype="xsd:string"></span><span id='am-630' property="rdfs:label" content="Argininosuccinate synthase" datatype="xsd:string"></span><span id='am-631' property="rdfs:label" content="ASSY_MOUSE" datatype="xsd:string"></span><span id='am-632' property="rdfs:label" content="Citrulline--aspartate ligase" datatype="xsd:string"></span><span id='am-633' property="oboInOwl:hasExactSynonym" content="Ass-1" datatype="xsd:string"></span><span id='am-634' property="oboInOwl:hasExactSynonym" content="Argininosuccinate synthase" datatype="xsd:string"></span><span id='am-635' property="rdfs:label" content="Ass-1" datatype="xsd:string"></span><span id='am-636' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-637' property="oboInOwl:hasDbXref" content="UniProt:P16460" datatype="xsd:string"></span><span id='am-638' property="oboInOwl:id" content="IMR:0703347" datatype="xsd:string"></span>fold</span> spread agar Glu plate monitor fraction   populations, respectively,  181; maximal growth rate, adapted cells parallel rich medium (YPD) plates       Michaelis 8211;Menten (half saturation) constant  viability controls. medium composition plates       growth parameters assumed similar chemostat addition 2%     subpopulations). naive cells population 1 switch <br /> agar. plates incubated 30 176; C 20 days colo-     arrested state rate fs adapted state rate <br /> nies counted day. colony counts            fa, transfers cells population 1 population 2. <br /> malized counts rich medium plates.        b, c, d, e constants transition rates  9270;s <br /> plating procedure used batch culture experi-            response time constants (time units  <br /> ments. study growth dynamics individual cells, agar      malized). S concentration limiting nutrient plates imaged bright field microscopy (Zeiss            chemostat. units normalized S,  x1,2 <br /> Axiovert 135). Colony area distributions measured using       measured units feeding concentration lim- <br /> IMAGEPRO software.                                                 iting nutrient yield normalized  Time                                                                    measured units chemostat dilution time  Smith <br />                                                                    Waltman (1995) details]. Supplementary Material <br />                                                                    Fig. S5 produced following parameters:  181; = 1.8, <br /> Cell growth model <br />                                                                    = 0.2, b = 3, c = 3, d = 1, e = 125,  9270;s = 18,  = 25. equa- computations performed using MATLAB.                  tions integrated initial conditions: x1 20849;t = 0 20850; <br /> model computes dynamics subpopulations; na-           = 0.1, x2 20849;t = 0 20850; = 0, S = 1. <br /> ive supporting limited number cell divisions ending           cells growing colonies (Fig. 6), rate functions <br /> eventually cell arrest, adapted state      interpreted probabilities unit time switch pheno- <br /> supporting indefinite divisions. growth rates          type following probability density functions: Pac <br /> sumed subpopulations. Arrested cells        = 0.02 exp 20849;tcell /  9270;s 20850; Pad = 0.06/  20853;3 + exp 20849; 8722;tcell /  9270;a 20850; 20854;, allowed switch growing phase       Pac probability density remain active  dividing), <br /> adapted state inherited daughter cells. similar           Pad probability density switch adaptive state, <br /> model computed different settings:   cells        tcell = t + tg running time including past exposure growing chemostat  addition           glucose tg liquid medium.  9270;s =  = 1 time  <br /> equations describing dynamics subpopulations        stants related switch arrested adapted states,  equation, describing dynamics limiting         spectively. times units integration time, dt <br /> source added (b) colony growth plates.          = 0.1. models computed wide range pa-  equations solved Monte-Carlo          rameters qualitative change results. <br /> simulations. time <span id='am-475' about='obo:IMR_0703879' typeof='owl:Thing'><span id='am-476' property="oboInOwl:hasExactSynonym" content="Ptpn5" datatype="xsd:string"></span><span id='am-477' property="rdfs:label" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-478' property="oboInOwl:id" content="IMR:0703879" datatype="xsd:string"></span><span id='am-479' property="rdfs:label" content="PTN5_RAT" datatype="xsd:string"></span><span id='am-480' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-481' property="rdfs:label" content="Ptpn5" datatype="xsd:string"></span><span id='am-482' property="oboInOwl:hasExactSynonym" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-483' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-484' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-485' property="rdfs:label" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-486' property="rdfs:label" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-487' property="rdfs:label" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-488' property="rdfs:label" content="STEP" datatype="xsd:string"></span><span id='am-489' property="oboInOwl:hasExactSynonym" content="STEP" datatype="xsd:string"></span><span id='am-490' property="rdfs:label" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-491' property="oboInOwl:hasDbXref" content="UniProt:P35234" datatype="xsd:string"></span><span id='am-492' property="oboInOwl:hasExactSynonym" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-493' property="oboInOwl:hasExactSynonym" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span>step</span> decision divide  threshold cell mass) comparing             <br /> constant probability. resulting cells divi- <br />                                                                     <br /> sion, cell state switched arrested adapted ac- <br />                                                               <br /> cording probabilities computed function integrated <br />                                                                    <br /> time glucose cell lineage  probability func- <br />                                                               <br /> tions .  cell continues grow di- <br />                                                                    <br /> vide  <br />                                                                   <br />     population density chemostat function                                                                     <br /> time computed integrating following chemostat <br />                                                                    <br /> equations: <br />                                                                   <br /> </body></html>