<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Changes transcriptional orientation associated increases evolutionary <br /> rates enterobacterial genes <br />  <br />   Abstract <br />   Background: Changes transcriptional orientation ( 8220;CTOs 8221;) occur frequently prokaryotic genomes.   changes usually result genomic inversions, cause conflict directions replication <br />   transcription increase mutation rate.  CTOs lead replication- <br />   transcription confrontation. Furthermore, CTOs cause deleterious disruptions operon structure gene <br />   regulations. currently existing CTOs indicate relaxation selection pressure.    investigate CTOs independent effect evolutionary rates affected genes,   genes subject type selection pressure prokaryotes. <br />   Methods: closely related enterbacteria, Escherichia coli, Klebsiella pneumoniae Salmonella enterica serovar <br />   Typhimurium, selected comparisons synonymous (dS) nonsynonymous (dN) substitution rate <br />   genes experienced changes transcriptional orientation (changed-orientation genes, <br />    8220;COGs 8221;)  orientation genes,  8220;SOGs 8221;). dN/dS ratio derived evaluate   selection pressure analyzed genes. Confounding factors estimation evolutionary rates,   gene essentiality, gene expression level, replication-transcription confrontation, decreased dS gene terminals <br />   controlled COG-<span id='am-214' about='obo:IMR_0003021' typeof='owl:Thing'><span id='am-215' property="oboInOwl:hasExactSynonym" content="short gastrulation" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasExactSynonym" content="Sog" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasExactSynonym" content="l(1)G0395" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasExactSynonym" content="Short gastrulation protein" datatype="xsd:string"></span><span id='am-219' property="rdfs:label" content="short gastrulation" datatype="xsd:string"></span><span id='am-220' property="rdfs:label" content="sog" datatype="xsd:string"></span><span id='am-221' property="rdfs:label" content="Dorsal-ventral patterning protein Sog" datatype="xsd:string"></span><span id='am-222' property="rdfs:label" content="l(1)G0160" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasExactSynonym" content="Dorsal-ventral patterning protein Sog" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasExactSynonym" content="l(1)G0479" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-226' property="rdfs:label" content="Short gastrulation protein" datatype="xsd:string"></span><span id='am-227' property="rdfs:label" content="Sog" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:id" content="IMR:0003021" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasExactSynonym" content="CG9224" datatype="xsd:string"></span><span id='am-230' property="rdfs:label" content="l(1)G0395" datatype="xsd:string"></span><span id='am-231' property="rdfs:label" content="SOG_DROME" datatype="xsd:string"></span><span id='am-232' property="rdfs:label" content="CG9224" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasDbXref" content="UniProt:Q24025" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasExactSynonym" content="sog" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasExactSynonym" content="l(1)G0160" datatype="xsd:string"></span><span id='am-236' property="rdfs:label" content="l(1)G0479" datatype="xsd:string"></span>SOG</span> comparisons. <br />   Results: demonstrate COGs significantly higher dN dS SOGs series confounding <br />   factors controlled.  dN/dS ratios similar gene groups, suggesting   increase dS sufficiently explain increase dN COGs.  increases evolutionary rates   COGs mainly mutation-driven. <br />   Conclusions: CTOs increase evolutionary rates affected genes. effect   independent replication-transcription confrontation, suggested major cause inversion- <br />   associated evolutionary rate increases. real cause evolutionary rate increases remains unclear   worth explorations. <br />  <br />  <br /> Introduction                                                 major driver changes evolutionary <br /> Genome rearrangements occur frequently evolu-         rates, dN/dS ratios expected differ signifi- <br /> tion prokaryotes. rearrangement events,         cantly gene groups. Clarifying inversions usually occur symmetrically origin     molecular mechanisms prokaryotic genomes <br /> (designated  8220;Ori 8221;) terminus ( 8220;Ter 8221;) replication     evolve help understand prokaryotes develop closely related bacterial genomes [1].         novel functions, turn relevant ecological <br /> rearrangement events lead changes transcrip-     biomedical studies. <br /> tional orientation (designated  8220;CTOs 8221;) [2]              Accordingly, compared genomes clo- <br /> increases mutation pressure affected genes         sely related enterobacteria [5,6]: Escherichia coli (ECO), conflict directions transcrip-   Klebsiella pneumonia (KPN) Salmonella enterica <br /> tion replication [1,3,4].  inver-       serovar Typhimurium (STM). series analyses sion events result flipping Ori Ter,      performed control potential confounding factors, <br /> replication-transcription conflict occur [1].        including gene essentiality, expression level, background <br /> Furthermore, CTOs cause changes homology-             mutation rate, pattern replication-transcription <br /> based recombination impediments DNA replica-           confrontation, codon usage bias. potential <br /> tion  altering, example, DNA <span id='am-323' about='obo:IMR_0000949' typeof='owl:Thing obo:IMR_0000001'><span id='am-324' property="oboInOwl:id" content="IMR:0000949" datatype="xsd:string"></span><span id='am-325' property="rdfs:label" content="protein" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>protein</span> binding          confounding factors reported associated <br /> sites secondary structure), increase muta-      evolutionary rates. example, highly expressed <br /> tion rate.  CTOs lead disruptions        genes, essential genes, genes large codon usage <br /> operon structure transcriptional regulations,      bias tend evolve slowly [7-9].  higher potentially deleterious.  currently existing   mutation rate observed genes near Ter[10] <br /> CTOs signify relaxation selection pressure         genes subject orientation conflicts affected genes.  remains unknown         DNA replication transcription [3]. selection actually plays important role        results suggest COGs significantly higher dN <br /> maintaining CTOs. view potential influences     dS SOGs. increases independent CTOs gene evolution, interested investigat-     analyzed confounding factors, mainly muta- <br /> ing CTO se effect evolution-      tion-driven. <br /> ary rates prokaryotic genes. <br />   Note evolution prokaryotic genes driven     Methods major forces: mutation natural selection.      Data sources <br /> increased mutation rate accelerate short-term            complete genomic sequences gene annotations <br /> nucleotide substitutions, retained         Escherichia coli K-12 MG 1655 (GenBank accession <br /> eliminated natural selection according fitness   number U00096), Klebsiella pneumonia MGH 78578 <br /> effects. evolutionary rates prokaryotic genes         (CP000647) Salmonella enterica serovar Typhimur- <br /> mainly reflect combinatorial effects        ium LT2 (AE006468) retrieved GenBank <br /> forces. general, substitutions cause protein         [11]. species selected genomes <br /> sequence changes (nonsynonymous substitutions)           completely sequenced carefully annotated, <br /> subject stronger selection pressure    genome-scale gene expression data (synonymous substitutions). comparing nonsy-          available species. orthologous gene pairs <br /> nonymous (dN) synonymous substitution rate (dS),         identified according reciprocal BLASTP best evaluate mutation selection plays        matches following parameters: (1) E value &lt; 1.0 important role evolution prokaryotic          x 10 -5 ; (2) &gt; 50% amino acid sequence identity. <br /> genes.                                                       orthologous genes   attempt examine (1) effects CTOs         compared species retained. total 2,574 prokaryotic gene evolution; (2) relative      orthologous gene groups tributions abovementioned driving forces       obtained subsequent analyses. ana- <br /> dS dN genes experienced CTOs.           lyses measured evolutionary rates separately end, compared dS dN           terminal central regions individual genes, genes experienced CTOs (changed-orientation        analyzed genes sufficiently long reduce varia- <br /> genes,  8220;COGs 8221;)  orientation       tions evolutionary rate estimates. previous study <br /> genes,  8220;SOGs 8221;) closed related prokaryotic genomes.        suggested 50 20 codons usually case mutation-driven evolution, COGs           evolved slowly rest [12,13].  SOGs significantly different dS approxi-     took twice length 70 (2*(50+20) = 140) include <br /> mately dN/dS ratios. Alternatively, selection    central region minimal length 70 codons. <br />  <span id='am-264' about='obo:IMR_0001155' typeof='owl:Thing'><span id='am-265' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasExactSynonym" content="DRL" datatype="xsd:string"></span><span id='am-267' property="oboInOwl:hasExactSynonym" content="lionette" datatype="xsd:string"></span><span id='am-268' property="rdfs:label" content="linotte" datatype="xsd:string"></span><span id='am-269' property="rdfs:label" content="lin" datatype="xsd:string"></span><span id='am-270' property="rdfs:label" content="lio/drl" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasExactSynonym" content="linotte" datatype="xsd:string"></span><span id='am-272' property="rdfs:label" content="Tyrosine-protein kinase Drl" datatype="xsd:string"></span><span id='am-273' property="oboInOwl:hasExactSynonym" content="CG10758" datatype="xsd:string"></span><span id='am-274' property="rdfs:label" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-275' property="oboInOwl:hasExactSynonym" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-276' property="oboInOwl:hasExactSynonym" content="derailed" datatype="xsd:string"></span><span id='am-277' property="rdfs:label" content="DRL" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:id" content="IMR:0001155" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:hasExactSynonym" content="Protein derailed" datatype="xsd:string"></span><span id='am-280' property="oboInOwl:hasExactSynonym" content="drl" datatype="xsd:string"></span><span id='am-281' property="oboInOwl:hasExactSynonym" content="lio" datatype="xsd:string"></span><span id='am-282' property="rdfs:label" content="drl" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein kinase Drl" datatype="xsd:string"></span><span id='am-284' property="oboInOwl:hasExactSynonym" content="lin" datatype="xsd:string"></span><span id='am-285' property="oboInOwl:hasDbXref" content="UniProt:Q27324" datatype="xsd:string"></span><span id='am-286' property="rdfs:label" content="CG10758" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:hasExactSynonym" content="CG17348" datatype="xsd:string"></span><span id='am-288' property="rdfs:label" content="derailed" datatype="xsd:string"></span><span id='am-289' property="rdfs:label" content="Protein derailed" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasExactSynonym" content="lio/drl" datatype="xsd:string"></span><span id='am-291' property="rdfs:label" content="CG17348" datatype="xsd:string"></span><span id='am-292' property="rdfs:label" content="lio" datatype="xsd:string"></span><span id='am-293' property="rdfs:label" content="lionette" datatype="xsd:string"></span><span id='am-294' property="rdfs:label" content="RYK1_DROME" datatype="xsd:string"></span>Lin</span> et al. BMC Bioinformatics 2011, 12(Suppl 9):<span id='am-315' about='obo:IMR_0010549' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000896 obo:IMR_0100710 obo:IMR_0000001 obo:IMR_0000950'><span id='am-316' property="rdfs:label" content="40S ribosomal protein S24" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasExactSynonym" content="S19" datatype="xsd:string"></span><span id='am-318' property="oboInOwl:id" content="IMR:0010549" datatype="xsd:string"></span><span id='am-319' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-320' property="rdfs:label" content="S19" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasExactSynonym" content="40S small ribosomal protein 24" datatype="xsd:string"></span><span id='am-322' property="rdfs:label" content="40S small ribosomal protein 24" datatype="xsd:string"></span>S19</span>                                                           Page 3 8 <br /> http://www.biomedcentral.com/1471-2105/12/<span id='am-327' about='obo:IMR_0702940' typeof='owl:Thing'><span id='am-328' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 11" datatype="xsd:string"></span><span id='am-329' property="rdfs:label" content="Rpn6" datatype="xsd:string"></span><span id='am-330' property="oboInOwl:hasExactSynonym" content="S9" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasExactSynonym" content="Rpn6" datatype="xsd:string"></span><span id='am-332' property="rdfs:label" content="26S proteasome regulatory subunit S9" datatype="xsd:string"></span><span id='am-333' property="oboInOwl:hasExactSynonym" content="PSMD11" datatype="xsd:string"></span><span id='am-334' property="rdfs:label" content="PROTEASE 26S, SUBUNIT 9" datatype="xsd:string"></span><span id='am-335' property="oboInOwl:id" content="IMR:0702940" datatype="xsd:string"></span><span id='am-336' property="rdfs:label" content="PSMD11" datatype="xsd:string"></span><span id='am-337' property="oboInOwl:hasExactSynonym" content="p44.5" datatype="xsd:string"></span><span id='am-338' property="rdfs:label" content="26S proteasome regulatory subunit RPN6" datatype="xsd:string"></span><span id='am-339' property="rdfs:label" content="PSD11_HUMAN" datatype="xsd:string"></span><span id='am-340' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 11" datatype="xsd:string"></span><span id='am-341' property="rdfs:label" content="26S proteasome regulatory subunit p44.5" datatype="xsd:string"></span><span id='am-342' property="rdfs:label" content="S9" datatype="xsd:string"></span><span id='am-343' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 11" datatype="xsd:string"></span><span id='am-344' property="oboInOwl:hasExactSynonym" content="PROTEASE 26S, SUBUNIT 9" datatype="xsd:string"></span><span id='am-345' property="rdfs:label" content="PROTEASOME 26S SUBUNIT, NON-ATPase, 11" datatype="xsd:string"></span><span id='am-346' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 11" datatype="xsd:string"></span><span id='am-347' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S SUBUNIT, NON-ATPase, 11" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN6" datatype="xsd:string"></span><span id='am-350' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit p44.5" datatype="xsd:string"></span><span id='am-351' property="rdfs:label" content="p44.5" datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasExactSynonym" content="MGC3844" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasDbXref" content="UniProt:O00231" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S9" datatype="xsd:string"></span><span id='am-355' property="rdfs:label" content="MGC3844" datatype="xsd:string"></span>S9</span>/S19 <br />  <br />  <br />  <br />  <br /> Accordingly, genes shorter 420 bp (140*3)       expression log2-transformed normalized discarded. Note practice lead <span id='am-81' about='obo:IMR_0700040' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="Tnfsf2" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="TNF-alpha" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="TNF alpha" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="TNF alpha" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="tumor necrosis factor" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="TNF-a" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="TNF-alpha" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Tumor necrosis factor, soluble form" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="TNFalpha" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasExactSynonym" content="tumor necrosis factor" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Tnf" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Tnfsf1a" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="DIF" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasExactSynonym" content="Tnfsf1a" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasExactSynonym" content="Tnf" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="Cachectin" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="Cachectin" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor, soluble form" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Tumor necrosis factor ligand superfamily member 2" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="TNFalpha" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasExactSynonym" content="tumor necrosis factor-alpha" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasExactSynonym" content="Tnfsf2" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor ligand superfamily member 2" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasExactSynonym" content="Tnfa" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="TNFA_MOUSE" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor, membrane form" datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="Tumor necrosis factor, membrane form" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Tumor necrosis factor" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="UniProt:P06804" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="Tnfa" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasExactSynonym" content="TNF-a" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="tumor necrosis factor-alpha" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:id" content="IMR:0700040" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasExactSynonym" content="DIF" datatype="xsd:string"></span>dif</span>-     using quantile normalization. cross-species compari- <br /> ferences number analyzed gene pairs         sons, gene expression levels species different pair-<span id='am-68' about='obo:IMR_0702237' typeof='owl:Thing'><span id='am-69' property="oboInOwl:hasExactSynonym" content="Sclerostin domain-containing protein 1" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="SOSTDC1" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="SOSTDC1" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Sclerostin domain-containing protein 1" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="WISE" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Wnt-signaling modulator" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="SOSD1_CHICK" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="UniProt:Q6VYA3" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasExactSynonym" content="Wnt-signaling modulator" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="WISE" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:id" content="IMR:0702237" datatype="xsd:string"></span>wise</span> species comparisons. numbers       standardized median value 0 variance 1. <br /> analyzed gene pairs listed Additional file 1. reciprocal-BLASTP approach optimal          Functional enrichment analysis <br /> solution finding orthologous genes, retrieved     Gene Ontology   enrichment analysis  <br /> orthologous genes analyzed species           formed using DAVID Bioinformatics Resources <br /> OMA database comparison [14]. fact, 98%        [20,21]. genes ECO extensively orthologous gene pairs retrieved OMA            studied species, identical identified using reciprocal BLASTP      COGs ECO-STM ECO-KPN comparisons matches  Additional file 2). actually observed         included analysis. gene clustering similar results 98% orthologous genes           term enrichment assessed reference analyzed  e. COGs significantly higher dS         enrichment score p-values modified Fish- dN similar dN/dS compared SOGs;             er 8217;s Exact test. p&lt;0.05, Additional file 3). <br />   locations Ori Ter determined             Results discussion <br /> using program Oriloc [15].                                COGs higher dN dS similar dN/dS ratio <br />                                                               compared SOGs <br /> Estimation evolutionary rates statistical analyses     pair-<span id='am-260' about='obo:IMR_0001162' typeof='owl:Thing obo:IMR_0001160 obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000428'><span id='am-261' property="oboInOwl:id" content="IMR:0001162" datatype="xsd:string"></span><span id='am-262' property="rdfs:label" content="Wise" datatype="xsd:string"></span><span id='am-263' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>wise</span> species comparisons (ECO-KPN, amino acid sequences orthologous gene pairs            STM-KPN, ECO-STM), COGs significantly aligned using ClustalW 2.0 [16] trans-       higher dS dN SOGs  p&lt;0.05).  lated nucleotide sequences. dS dN esti-           dN/dS ratio does significant difference <br /> mated using CODEML module PAML 4.1                  COGs SOGs pair-wise com- <br /> [17]. orthologous gene pairs dS value &lt; 3      parisons (Figure 1). used way (ECO- considered [18]. Note lead <span id='am-162' about='obo:IMR_0700039' typeof='owl:Thing'><span id='am-163' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-164' property="oboInOwl:hasExactSynonym" content="TNFSF2" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:id" content="IMR:0700039" datatype="xsd:string"></span><span id='am-166' property="rdfs:label" content="TNFA" datatype="xsd:string"></span><span id='am-167' property="rdfs:label" content="Cachectin" datatype="xsd:string"></span><span id='am-168' property="rdfs:label" content="DIF" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor ligand superfamily member 2" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasExactSynonym" content="TNF-a" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasDbXref" content="UniProt:P01375" datatype="xsd:string"></span><span id='am-172' property="rdfs:label" content="Tumor necrosis factor" datatype="xsd:string"></span><span id='am-173' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor" datatype="xsd:string"></span><span id='am-174' property="oboInOwl:hasExactSynonym" content="TNFA" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor, membrane form" datatype="xsd:string"></span><span id='am-176' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor, soluble form" datatype="xsd:string"></span><span id='am-177' property="rdfs:label" content="TNF-alpha" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasExactSynonym" content="TNF" datatype="xsd:string"></span><span id='am-179' property="rdfs:label" content="tumor necrosis factor" datatype="xsd:string"></span><span id='am-180' property="rdfs:label" content="TUMOR NECROSIS FACTOR, ALPHA" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasExactSynonym" content="DIF" datatype="xsd:string"></span><span id='am-182' property="rdfs:label" content="TNF-a" datatype="xsd:string"></span><span id='am-183' property="rdfs:label" content="TNFA_HUMAN" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:hasExactSynonym" content="TNF, MONOCYTE-DERIVED" datatype="xsd:string"></span><span id='am-185' property="rdfs:label" content="TNF" datatype="xsd:string"></span><span id='am-186' property="rdfs:label" content="TNF, MACROPHAGE-DERIVED" datatype="xsd:string"></span><span id='am-187' property="rdfs:label" content="Tumor necrosis factor, membrane form" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasExactSynonym" content="tumor necrosis factor" datatype="xsd:string"></span><span id='am-189' property="rdfs:label" content="Tumor necrosis factor ligand superfamily member 2" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasExactSynonym" content="TNF, MACROPHAGE-DERIVED" datatype="xsd:string"></span><span id='am-191' property="rdfs:label" content="TNFSF2" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:hasExactSynonym" content="Cachectin" datatype="xsd:string"></span><span id='am-193' property="rdfs:label" content="TNF, MONOCYTE-DERIVED" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasExactSynonym" content="TNF-alpha" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasExactSynonym" content="TUMOR NECROSIS FACTOR, ALPHA" datatype="xsd:string"></span><span id='am-196' property="rdfs:label" content="Tumor necrosis factor, soluble form" datatype="xsd:string"></span>dif</span>-   STM-KPN) comparison infer lineage-specific COGs, <br /> ferences number analyzed genes different          8220;completely conserved SOGs 8221;  e. genes pair-wise species comparisons. end, 1,784, 1,809       changed transcriptional orientation 2,069 orthologous gene pairs obtained, respec-       compared species). shown Additional file 5, line- <br /> tively, ECO-KPN, STM-KPN, ECO-STM                 <span id='am-41' about='obo:IMR_0200587' typeof='owl:Thing'><span id='am-42' property="rdfs:label" content="EC 5.1.3.8" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0200587" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="RNBP" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="UniProt:P51606" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="RENBP_HUMAN" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="renin binding protein" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="RENBP" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="RBP" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="AGE" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="RENBP" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="GlcNAc 2-epimerase" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="N-acylglucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="EC 5.1.3.8" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="RNBP" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Renin-binding protein" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="AGE" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="N-acetyl-D-glucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="N-acetyl-D-glucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasExactSynonym" content="RnBP" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="N-acylglucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Renin-binding protein" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="RnBP" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="RBP" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="GlcNAc 2-epimerase" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="renin binding protein" datatype="xsd:string"></span>age</span>-specific (ECO STM  COGs sig- <br /> comparison (Additional file 1). dS &gt; 3 indi-        nificantly higher dS similar dN/dS ratios cate extremely high mutation rate loss func-        compared completely conserved SOGs. tion, investigated COGs tend          observations suggest increases evolutionary <br /> dS &gt; 3.  proportion COGs dS &gt; 3         rates COGs mainly mutation-driven.  higher of analyzed genes  Addi-      number confounding factors need excluded tional file 4). observation fact consistent   confirm results. confounding factors results COGs tend evolve faster SOGs.        controlled include gene essentiality, gene expres- <br />   Spearman 8217;s rank correlations partial correlations       sion level, background mutation rate, transcription- evolutionary rates rate-determining fac-      replication confrontation, codon usage bias. <br /> tors performed using R program (http:// <br /> www.r-project.org). statistical significance       Confounding factor 1: gene essentiality <br /> evolutionary rate differences evaluated using      Essential genes known evolutionarily <br /> Wilcoxon Rank Sum test study.                  conserved nonessential genes [8].                                                               necessary clarify COGs tend non- <br /> Analysis gene expression data                              essential, evolve faster SOGs. genome-scale gene expression data ECO                  retrieved gene essentiality information E.coli[22], <br /> (GSE15534) STM (GSE11486) downloaded                 divided orthologous genes essential Gene Expression Omnibus (GEO) database               nonessential genes ECO-KPN ECO-STM <br /> [19]. gene expression data KPN generated          comparisons. results indicate dN dS using custom NimbleGen tiling array (pro-           significantly higher COGs SOGs nones- <br /> vided Dr. Bernhard O. Palsson University Cali-       sential genes (Table 1). observed similar trends fornia, San Diego). data sets, gene   essential genes, differences dN dS <br /> expression levels measured separately log         statistically insignificant possibly small stationary phase. signal intensity gene            sample sizes reduced variations evolutionary <br />  <br />   Figure 1 evolutionary rates COGs SOGs ECO-KPN STM-KPN comparisons. *: p-value &lt; 0.05; **: p-value &lt; 0.01; ***: <br />   p-value &lt; 0.0001 <br />  <br />  <br />  <br /> rates gene group.  dN/dS ratios                                       evolve slowly lowly expressed genes [23,24]. similar COGs SOGs, consistent                                          question SOGs tend highly previous observations.  difference                                     expressed, evolve slowly COGs. level gene essentiality does affect                                  classified analyzed genes highly  result. expected COG-<span id='am-237' about='obo:IMR_0003021' typeof='owl:Thing'><span id='am-238' property="oboInOwl:hasExactSynonym" content="Sog" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasExactSynonym" content="CG9224" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:hasExactSynonym" content="short gastrulation" datatype="xsd:string"></span><span id='am-241' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-242' property="rdfs:label" content="sog" datatype="xsd:string"></span><span id='am-243' property="oboInOwl:hasExactSynonym" content="sog" datatype="xsd:string"></span><span id='am-244' property="rdfs:label" content="SOG_DROME" datatype="xsd:string"></span><span id='am-245' property="rdfs:label" content="l(1)G0479" datatype="xsd:string"></span><span id='am-246' property="rdfs:label" content="short gastrulation" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Short gastrulation protein" datatype="xsd:string"></span><span id='am-248' property="rdfs:label" content="CG9224" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasExactSynonym" content="l(1)G0160" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:id" content="IMR:0003021" datatype="xsd:string"></span><span id='am-251' property="oboInOwl:hasExactSynonym" content="Short gastrulation protein" datatype="xsd:string"></span><span id='am-252' property="oboInOwl:hasExactSynonym" content="Dorsal-ventral patterning protein Sog" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:hasDbXref" content="UniProt:Q24025" datatype="xsd:string"></span><span id='am-254' property="rdfs:label" content="Sog" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasExactSynonym" content="l(1)G0479" datatype="xsd:string"></span><span id='am-256' property="rdfs:label" content="Dorsal-ventral patterning protein Sog" datatype="xsd:string"></span><span id='am-257' property="rdfs:label" content="l(1)G0395" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasExactSynonym" content="l(1)G0395" datatype="xsd:string"></span><span id='am-259' property="rdfs:label" content="l(1)G0160" datatype="xsd:string"></span>SOG</span> differences                                      20%) non-highly  80%) expressed genes. evolutionary rates mainly mutation-driven.                                              ECO-KPN STM-KPN comparisons, dS                                                                                             COGs significantly larger SOGs highly <br /> Confounding factor 2: gene expression level                                                 non-highly expressed genes growth phases <br /> Highly expressed genes strong selection <span id='am-407' about='obo:IMR_0001902' typeof='owl:Thing'><span id='am-408' property="oboInOwl:hasExactSynonym" content="DmPS" datatype="xsd:string"></span><span id='am-409' property="oboInOwl:hasExactSynonym" content="l(3)77CDb" datatype="xsd:string"></span><span id='am-410' property="rdfs:label" content="Dpresenilin" datatype="xsd:string"></span><span id='am-411' property="rdfs:label" content="DPS" datatype="xsd:string"></span><span id='am-412' property="rdfs:label" content="CG18803" datatype="xsd:string"></span><span id='am-413' property="rdfs:label" content="presenilin" datatype="xsd:string"></span><span id='am-414' property="rdfs:label" content="CG5868" datatype="xsd:string"></span><span id='am-415' property="rdfs:label" content="EC 3.4.23.-" datatype="xsd:string"></span><span id='am-416' property="rdfs:label" content="Psn" datatype="xsd:string"></span><span id='am-417' property="oboInOwl:hasExactSynonym" content="Dps" datatype="xsd:string"></span><span id='am-418' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-419' property="oboInOwl:hasExactSynonym" content="DPS" datatype="xsd:string"></span><span id='am-420' property="rdfs:label" content="psn" datatype="xsd:string"></span><span id='am-421' property="rdfs:label" content="Dps" datatype="xsd:string"></span><span id='am-422' property="rdfs:label" content="DPsn" datatype="xsd:string"></span><span id='am-423' property="oboInOwl:hasExactSynonym" content="CG18803" datatype="xsd:string"></span><span id='am-424' property="oboInOwl:hasExactSynonym" content="Presenilin homolog" datatype="xsd:string"></span><span id='am-425' property="rdfs:label" content="pres" datatype="xsd:string"></span><span id='am-426' property="oboInOwl:hasExactSynonym" content="dPresenilin" datatype="xsd:string"></span><span id='am-427' property="oboInOwl:hasExactSynonym" content="PS1" datatype="xsd:string"></span><span id='am-428' property="oboInOwl:hasDbXref" content="UniProt:O02194" datatype="xsd:string"></span><span id='am-429' property="rdfs:label" content="l(3)77CDb" datatype="xsd:string"></span><span id='am-430' property="rdfs:label" content="PS1" datatype="xsd:string"></span><span id='am-431' property="oboInOwl:hasExactSynonym" content="DrosPS" datatype="xsd:string"></span><span id='am-432' property="oboInOwl:hasExactSynonym" content="psn" datatype="xsd:string"></span><span id='am-433' property="rdfs:label" content="DmPS" datatype="xsd:string"></span><span id='am-434' property="oboInOwl:hasExactSynonym" content="pres" datatype="xsd:string"></span><span id='am-435' property="oboInOwl:id" content="IMR:0001902" datatype="xsd:string"></span><span id='am-436' property="rdfs:label" content="Presenilin homolog" datatype="xsd:string"></span><span id='am-437' property="rdfs:label" content="PSN_DROME" datatype="xsd:string"></span><span id='am-438' property="oboInOwl:hasExactSynonym" content="Dpresenilin" datatype="xsd:string"></span><span id='am-439' property="oboInOwl:hasExactSynonym" content="EC 3.4.23.-" datatype="xsd:string"></span><span id='am-440' property="oboInOwl:hasExactSynonym" content="Presenilin" datatype="xsd:string"></span><span id='am-441' property="rdfs:label" content="DrosPS" datatype="xsd:string"></span><span id='am-442' property="oboInOwl:hasExactSynonym" content="Psn" datatype="xsd:string"></span><span id='am-443' property="rdfs:label" content="dPresenilin" datatype="xsd:string"></span><span id='am-444' property="oboInOwl:hasExactSynonym" content="CG5868" datatype="xsd:string"></span><span id='am-445' property="oboInOwl:hasExactSynonym" content="DPsn" datatype="xsd:string"></span><span id='am-446' property="oboInOwl:hasExactSynonym" content="dPS" datatype="xsd:string"></span><span id='am-447' property="rdfs:label" content="dPS" datatype="xsd:string"></span><span id='am-448' property="rdfs:label" content="Presenilin" datatype="xsd:string"></span><span id='am-449' property="oboInOwl:hasExactSynonym" content="presenilin" datatype="xsd:string"></span>pres</span>-                                     (stationary log phase, Table 1). dN values sure maintain functional stability, usually                                     similar trends.  dN/dS ratios  <br /> Table 1 median evolutionary rates COGs SOGs different confounding factors controlled. <br /> confounding factors                        comparison                     #gene (COG/                dN                                dS                       dN/dS <br />                                                                              SOG) <br />                                                                                              COGs SOGs       p-     COGs              SOGs      p-      COGs    SOGs      p- <br />                                                                                                             value                              value                     value <br /> nonessential gene <br />                                              ECO-KPN                       121/1426          0.076 0.067      *     1.6811 1.466                ***    0.0481 0.0463       - <br />                                              ECO-STM                       240/1582          0.057 0.045     ***    1.5921 1.1926               ***    0.0395 0.0381       - <br /> essential gene <br />                                              ECO-KPN                        8/224            0.045 0.041      -     1.374             0.978      -      0.042   0.046      - <br />                                              ECO-STM                        17/225           0.025 0.025      -     0.785             0.825      -      0.029   0.032      - <br /> non-highly expressed <br /> gene <br />                                       ECO-KPN (log phase)                  106/1240          0.082 0.074      -     1.721             1.561     **      0.049   0,047      - <br />                                   ECO-KPN (stationary phase)               103/1342          0.077 0.066      -     1.686             1.457     ***     0.048   0.046      - <br />                                       STM-KPN (log phase)                   88/1416          0.077 0.068      -     1.845             1,507     ***     0.042   0.045      - <br />                                   STM-KPN (stationary phase)                80/1475          0.077 0.065      -      1.77             1.458     ***     0.042   0.046      - <br /> highly expressed <br />                                       ECO-KPN (log phase)                   23/415           0.045 0.037      -     1.352             0.896     **      0.042   0.044      - <br />                                   ECO-KPN (stationary phase)                26/313           0.064 0.047      -     1.459             1.142      *      0.047   0.045      - <br />                                     STM-KPN (log phase)                     18/287           0.06 0.037       -     1.384             0.895      *      0.046   0.046      - <br />                                   STM-KPN (stationary phase)                26/228           0.075 0.053      *     1.845             1.273     ***     0.042   0.044      - <br /> * : p &lt; 0.05: **: p &lt; 0.01: ***: p &lt; 0.001; -: insignificant <br />  Lin et al. BMC Bioinformatics 2011, 12(Suppl 9):S19                                                                                Page 5 8 <br /> http://www.biomedcentral.com/1471-2105/12/S9/S19 <br />  <br />  <br />  <br />  <br /> approximately COGs SOGs                             location-independent effect dS.  (Table 1). Notably, previously reported                 increase dN COGs likely affected chro- <br /> mutation rate increase gene expression level                    mosomal locations genes.  [25-27].  investigate             COGs close Ter far outnumber increases evolutionary rate COGs asso-                      evidence needed clarify correla- <br /> ciated increased expression levels genes.                  tion dN COG/SOG gene type  results indicate case.                 context background mutation rate. <br /> Highly expressed COGs actually lower dN dS non-highly expressed COGs (Additional file 6).                      Confounding factor 4: replication 8211;transcription  increases evolutionary rates (particu-                 confrontation <br /> larly dS) COGs result increased expres-                   head collision directions tran- <br /> sion levels genes.                                              scription replication reported increase <br />                                                                          mutation rate affected genes [3,10,29]. Confounding factor 3: background mutation rate                           gene involved inversion event, transcrip- gene inversions usually occur vicinity                tional orientation likely change, causing replica- Ori Ter[1,2], actually                       tion-transcription confrontation. investigated <br /> COGs analyzed study located near Ter                        COGs replication-transcription confronta- <br /> (Figure 2). known genes located near Ter                  tion SOGs. Interestingly, increased mutation rate [28]. <span id='am-10' about='obo:IMR_0703847' typeof='owl:Thing'><span id='am-11' property="rdfs:label" content="ASK-1" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Apoptosis signal-regulating kinase 1" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="MEKK 5" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Ask1" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="ASK1" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="ASK-1" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="ASK" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="MEKK 5" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Mitogen-activated protein kinase kinase kinase 5" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="Mitogen-activated protein kinase kinase kinase 5" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="M3K5_MOUSE" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="ASK1" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0703847" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Mekk5" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Map3k5" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="ASK" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="MEK kinase 5" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="UniProt:O35099" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Ask1" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="Mekk5" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="mitogen-activated protein kinase kinase kinase 5" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Apoptosis signal-regulating kinase 1" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="MEK kinase 5" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="MAPK/ERK kinase kinase 5" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="MAPK/ERK kinase kinase 5" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Map3k5" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="EC 2.7.11.25" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="EC 2.7.11.25" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="mitogen-activated protein kinase kinase kinase 5" datatype="xsd:string"></span>ask</span>                        COGs maintain status terms replication-tran- uneven chromosomal distribution COGs                      scription confrontation. example, using KPN contributed COG-SOG differences evolu-                     outgroup, 157 162 COGs ECO- <br /> tionary rates.  performed partial correla-                  STM comparison maintain status head colli- <br /> tion analyses examine relationship                        sion orientation replication transcrip- <br /> evolutionary rates gene type (COG SOG).                       tion occurrences inversion events. <br /> results indicate dS significantly correlated                 ratio head collision orienta- COG/SOG gene type distance effect                      tion genes COGs SOGs 1.1 0.64, respec- controlled (Spearman 8217;s r &gt; 0.05 p &lt; 0.05) (Addi-                  tively ECO 8211;STM comparison.  COGs <br /> tional file 7). positive correlation indicates                  fact likely head collision genes <br /> CTOs location-independent effect                           SOGs. clarify difference increase dS.  distance-controlled cor-                  degree head collision major determinant <br /> relations statistically insignificant dN                 COG-SOG differences evolutionary rate, dN/dS) COG/SOG gene type.                          compared evolutionary rates COGs SOGs results indicate CTOs actually                         controlling replication-transcription confrontation <br />  <br />  <br />  Figure 2 Dot plot analyzed orthologous genes   ECO-KPN; (b) STM-KPN comparison. Y X axis represents, <br />  respectively, chromosomal positions KPN genes  <span id='am-310' about='obo:IMR_0100817' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000429 obo:IMR_0000428 obo:IMR_0001014'><span id='am-311' property="oboInOwl:id" content="IMR:0100817" datatype="xsd:string"></span><span id='am-312' property="rdfs:label" content="base" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-314' property="oboInOwl:hasDbXref" content="GO:0008540" datatype="xsd:string"></span>base</span> pair) orthologous genes ECO (left panel) STM (right <br />  panel). black red dots represent SOGs COGs, respectively. COGs located close Ter. positions Ter  1,892,000, 1,549,000 1,635,000 bp KPN, ECO STM, respectively. <br />  Lin et al. BMC Bioinformatics 2011, 12(Suppl 9):S19                                                                                        Page 6 8 <br /> http://www.biomedcentral.com/1471-2105/12/S9/S19 <br />  <br />  <br />  <br />  <br /> pattern. ECO-STM comparison, rates dN,                         Functional enrichment analysis COGs <br /> dS dN/dS head collision COGs differ                         ask rapidly evolving COGs significantly orientation COGs (Additional File                      enriched certain functional categories. Additional file <br /> 8). contrast, head collision SOGs signifi-                        10 indicates protein products COGs tend cantly higher evolutionary rates orientation                         located membranes cell walls, SOGs. results suggest higher evolutionary                     involved variety metabolic reactions. obser- <br /> rates COGs result higher proportion                      vation biologically sensible (1) proteins head collision genes.  CTO itself                        located cell surface tend evolve faster; major reason increased evolutionary rate.                        (2) different bacterial species different meta- <br />                                                                              bolic capacities.  relationship Confounding factor 5: decreased dS gene terminals                         CTOs functional preferences genes <br /> (codon usage bias)                                                           remains unknown. reported synonymous substitution <br /> rate decreases near 50-100 codons [12]                     Conclusions 20 codons gene [13], possibly codon                        knowledge, study demonstrate <br /> usage biased regions. ask                           changes transcriptional orientation increase SOGs tend lowered dS ends                         evolutionary rates (dN dS) prokaryotic genes. <br /> compared COGs, overall dS                                evolutionary effects CTOs inde- <br /> decreased. Accordingly, divided analyzed                      pendent gene essentiality, gene expression level, replica- <br /> genes terminal  50 20 codons)                        tion-transcriptional confrontation, decrease dS middle region  rest codons). <span id='am-1' about='obo:IMR_0000300' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010044 obo:IMR_0000653 obo:IMR_0100009 obo:IMR_0000001 obo:IMR_0000914 obo:IMR_0000207 obo:IMR_0000960 obo:IMR_0000288'><span id='am-2' property="rdfs:label" content="Ran" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Ras-like nuclear protein" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="TC4" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="TC4" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0000300" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="Ras-like nuclear protein" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="IPR:002041" datatype="xsd:string"></span>ran</span>-                         gene terminals.  increase dN par- <br /> domly sampled 70 codons middle region                           tially related gene locations. Furthermore, results <br /> comparison. differences evolutionary rates ( 916;dS,                      suggest increases evolutionary rates COGs <br />  916;dN,  916;dN/dS) terminal codons                             mainly mutation-driven, dN/dS ratios similar randomly sampled codons calculated com-                         COGs SOGs. real cause increases <br /> pared COGs SOGs. results                           evolutionary rate COGs (particularly dS) remains <br />  916;dS,  916;dN,  916;dN/dS similar COGs                            unclear worth explorations. speculated <br /> SOGs (Figure 3 ECO-KPN comparison Addi-                              CTOs result impediments DNA <br /> tional file 9 STM-KPN comparison; pair-                       replication  altering, example, DNA protein binding <br /> wise differences statistically insignificant).                 sites secondary structure), turn lead difference codon usage bias evolutionary rate                      recruitment error-prone DNA repair polymerases <br /> does appear affect overall results.                               increase mutation rate [30,31]. <br />  <br />                  genes COGs      SOGs                         genes COGs        SOGs                              genes COGs    SOGs <br />  Figure 3 Difference evolutionary rates terminal middle region ECO-KPN orthologous genes. Y axis shows <br />  difference (terminal minus middle) dN, dS, dN/dS ratio, respectively, left, middle, right panel. differences dN, dS  dN/dS ratio statistically insignificant pair-wise gene group comparisons panel. <br />  <br />  <br /> Additional material                                                             Authors 8217; contributions <br />                                                                                 CHL conceived study carried data analyses. CHL, FCC                                                                                 CAH designed conceptual framework study, interpreted     Additional file 1: numbers detailed information              results, drafted manuscript. CYL pre-processed gene expression data <br />     orthologous gene pairs analyzed study.                              assisted statistical analyses. <br />     Additional file 2: numbers ECO-KPN, ECO-STM, STM-KPN <br />     orthologous gene pairs identified reciprocal BLASTP             <br />     OMA database                                                                <br />     Additional file 3: evolutionary rates COGs SOGs                                                                              <br />     ECO-KPN STM-KPN comparisons. genes included     analysis identified reciprocal BLASTP OMA <br />     database orthologous genes. *: p-value &lt; 0.05; **: p-value &lt;             <br />     0.01; ***: p-value &lt; 0.0001                                           <br />     Additional file 4: numbers COGs dS &gt; 3 ECO-KPN, <br />                                                                            <br />     ECO-STM, STM-KPN comparisons. <br />                                                                      <br />     Additional file 5: evolutionary rates   ECO-KPN comparison;         <br />     (b) STM-KPN comparison. compare evolutionary                    <br />     rates genes changed transcriptional orientation      <br />     ECO, STM, KPN, ( 8220;ESK 8221;), genes changed orientation               <br />     ECO-STM lineage KPN lineage ( 8220;ES/K 8221;),           <br />     genes changed orientation species ( 8220;ECO           <br />      8220;STM ).                                                             <br />     Additional file 6: Comparison evolutionary rates highly           <br />     non-highly expressed COGs.   ECO-KPN comparison         <br />     log phase; (b) ECO-KPN comparison stationary phase; (c)                 <br />     STM-KPN comparison log phase; (d) STM-KPN comparison            <br />     stationary phase. *: p-value &lt; 0.05; **: p-value &lt; 0.01             <br />     Additional file 7: Partial correlations evolutionary rates <br />                                                                                  <br />     COG/SOG gene type physical distance Ter                                                                             <br />     controlled. <br />                                                                                     <br />     Additional file 8: evolutionary rates COGs SOGs          <br />     subject head collision orientation DNA       <br />     replication transcription ECO-STM comparison. *                   <br />     represents p value &lt; 0.05.                                                <br />     Additional file 9: Comparison  916;dN,  916;dS  916;dN/dS                <br />     COGs SOGs STM-KPN comparison.  916;dN,  916;dS  916;dN/dS                    <br />     calculated subtracting evolutionary rates middle        <br />     region terminal regions gene. Note          <br />     pair-wise comparisons panels              <br />     statistically significant.                                                 <br />     Additional file 10: Functional enrichment analyses COGs       ECO-KPN (b) ECO-STM comparisons. Note                                                                                    <br />     functional assignments based gene annotations                                                                                     <br />     ECO. <br /> </body></html>