<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 27 November 2013                                              Nucleic Acids Research, 2014, Vol. 42,  4 2405 8211;2414 <br />                                                                                                                 doi:10.1093/nar/gkt1200 <br />  <br />  <br /> Genetic robustness functional evolution gene duplicates <br /> Germa 180;n Plata1,2 Dennis Vitkup1,3,* <br /> 1 <br />  Department Systems Biology, Center Computational Biology Bioinformatics, Columbia University, <br /> New York City, NY 10032, USA, 2Integrated Program Cellular, Molecular, Structural, Genetic Studies, <br /> Columbia University, New York City, NY 10032, USA 3Department Biomedical Informatics, Columbia <br /> University, New York City, NY 10032, USA <br />  <br /> Received September 5, 2013; Revised November 3, 2013; Accepted November 4, 2013 <br />  <br />  <br />  <br /> ABSTRACT                                                                         duplicates (1,6). Gene duplications frequent evolu- <br />                                                                                  tion range size small-scale (SSD)  <br /> Gene duplications major source evolution-                               genome events (WGD) (7,8).  90% cases <br /> ary innovations. Understanding functional diver-                             duplicate eventually lost evolution (6), duplicated <br /> gence duplicates role genetic                                    genes remain genome  partially, <br /> robustness important challenge biology.                                 backup s functions. Importantly, functional <br /> Previously, analyses genetic robustness                                  compensation duplicates plays signi 65533; role buf- <br /> primarily focused duplicates essentiality                                 fering deleterious human mutations (9). <br /> epistasis laboratory conditions.                                 Genetic robustness gene duplicates inherently <br /> study, use quantitative data sets                                  tied functional divergence. Duplicates acquire <br /> understand compensatory interactions                                     distinct molecular functions (MFs) naturally unable Saccharomyces cerevisiae duplicates                                     compensate  addition, MF                                                                                  conserved, incomplete compensation duplicates likely relevant natural biological popula- <br />                                                                                  possible owing different expression patterns dosage <br /> tions.  owing high functional                              effects. Gene duplications major source new <br /> load, close duplicates unlikely provide sub-                              genes (10) conceptual models duplicates 8217; <br /> stantial backup context large natural popu-                            evolution proposed (11,12). neofunctio- <br /> lations. Interestingly, duplicates diverge                               nalization model duplicate gains new functions, e.  overall functional load reduced.                            functions associated ancestral gene, intermediate divergence distances quantita-                               duplicate retains ancestral functions <br /> tive decrease fitness removal dupli-                            (10,13,14). contrast, subfunctionalization model <br /> cate smaller. distances, yeast                                  duplicates indispensable retained duplicates display balanced functional loads                                evolution partitioning ancestral gene functions transcriptional control signifi-                               (15,16). models imply eventual loss                                                                                  ability duplicates fully substitute  cantly complex. yeast duplicates diverge <br />                                                                                  likely signi 65533; fraction duplicates  65533;?xed 70% sequence identity, ability                                   retained genomes owing selective advantages, <br /> compensate similar                                     dosage effects condition-speci 65533;?c expression random pairs singletons.                                              patterns, present moment duplication <br />                                                                                  (17,18). cases  65533;?xation selective advantage, <br />                                                                                  compensation duplicates unlikely. <br /> INTRODUCTION                                                                        compensation duplicates Survival biological systems crucially depends robust-                      expected long term, ability duplicates <br /> ness harmful genetic mutations, e. genetic robustness,                      buffer deleterious mutations paralogs changes environmental conditions (1 8211;3).                            demonstrated independent observa- <br /> distinct mechanisms genetic robustness                              tions. include lower expected fraction previously discussed.  alternative signaling meta-                     essential genes close duplicates (1), paucity bolic pathways provide important mechanism                                pairwise epistatic interactions involving duplicated genes <br /> rerouting molecular networks (4,5). Second,                            (19), excess aggravating genetic interactions <br /> major role genetic robustness attributed gene                           paralogs (20,21). contribution duplicates <br />  <br />  <br />  correspondence addressed. <span id='am-932' about='obo:IMR_0002980' typeof='owl:Thing obo:IMR_0000397 obo:IMR_0000949 obo:IMR_0000369 obo:IMR_0000368 obo:IMR_0000001'><span id='am-933' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-934' property="rdfs:label" content="ETV6" datatype="xsd:string"></span><span id='am-935' property="oboInOwl:hasExactSynonym" content="TEL" datatype="xsd:string"></span><span id='am-936' property="oboInOwl:hasExactSynonym" content="ETV6" datatype="xsd:string"></span><span id='am-937' property="rdfs:label" content="TEL" datatype="xsd:string"></span><span id='am-938' property="oboInOwl:id" content="IMR:0002980" datatype="xsd:string"></span><span id='am-939' property="rdfs:label" content="TEL1" datatype="xsd:string"></span>Tel</span>: +1 212 851 5152; Fax: +1 212 851 5149; Email: dv2121@columbia.edu <br />  <br />  223; Author(s) 2013. Published Oxford University Press. Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ nc/3.0/), permits non-commercial use, distribution, reproduction medium, provided original work properly cited. commercial use, contact journals.permissions@oup.com <br />  2406 Nucleic Acids Research, 2014, Vol. 42,  4 <br />  <br />  robustness primarily considered context      DeLuna et al. (34) Costanzo et al. (35). Gene qualitative quantitative growth phenotypes      essentiality data obtained study Giaever <br /> nutrient rich small number laboratory condi-         et al. (36). <br /> tions (1,22,23). popular experiments,            functionally characterize duplicated genes, Gene <br /> conditions unlikely approximate natural           Ontology   (37) annotations collected  8216;milieu 8217; living systems, constantly bombarded      SGD <span id='am-208' about='obo:IMR_0000207' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001'><span id='am-209' property="rdfs:label" content="enzyme" datatype="xsd:string"></span><span id='am-210' property="obo:IAO_0000115" content="Protein that catalyze a specific chemical reaction." datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasDbXref" content="GO:0003824" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:id" content="IMR:0000207" datatype="xsd:string"></span>Enzyme</span> Commission (EC) annotations diverse array environmental stresses stimuli.       Comprehensive Yeast Genome Database (<span id='am-572' about='obo:IMR_0200844' typeof='owl:Thing'><span id='am-573' property="oboInOwl:hasExactSynonym" content="RCD2" datatype="xsd:string"></span><span id='am-574' property="rdfs:label" content="GUCY2E, MOUSE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-575' property="rdfs:label" content="ROS-GC" datatype="xsd:string"></span><span id='am-576' property="oboInOwl:hasExactSynonym" content="LCA" datatype="xsd:string"></span><span id='am-577' property="oboInOwl:hasExactSynonym" content="EC 4.6.1.2" datatype="xsd:string"></span><span id='am-578' property="oboInOwl:hasExactSynonym" content="Guanylate cyclase 2D, retinal" datatype="xsd:string"></span><span id='am-579' property="oboInOwl:hasExactSynonym" content="CORD5" datatype="xsd:string"></span><span id='am-580' property="oboInOwl:hasDbXref" content="UniProt:Q02846" datatype="xsd:string"></span><span id='am-581' property="oboInOwl:hasExactSynonym" content="Rod outer segment membrane guanylate cyclase" datatype="xsd:string"></span><span id='am-582' property="rdfs:label" content="guanylate cyclase 2D, membrane (retina-specific)" datatype="xsd:string"></span><span id='am-583' property="oboInOwl:hasExactSynonym" content="GUANYLATE CYCLASE 2D, MEMBRANE" datatype="xsd:string"></span><span id='am-584' property="oboInOwl:hasExactSynonym" content="ROSGC" datatype="xsd:string"></span><span id='am-585' property="rdfs:label" content="RCD2" datatype="xsd:string"></span><span id='am-586' property="rdfs:label" content="LCA1" datatype="xsd:string"></span><span id='am-587' property="oboInOwl:hasExactSynonym" content="GUCY2E, MOUSE, HOMOLOG OF" datatype="xsd:string"></span><span id='am-588' property="oboInOwl:hasExactSynonym" content="RETGC" datatype="xsd:string"></span><span id='am-589' property="rdfs:label" content="ROSGC" datatype="xsd:string"></span><span id='am-590' property="oboInOwl:hasExactSynonym" content="ROS-GC1" datatype="xsd:string"></span><span id='am-591' property="rdfs:label" content="GUANYLATE CYCLASE 2D, MEMBRANE" datatype="xsd:string"></span><span id='am-592' property="oboInOwl:hasExactSynonym" content="retGC" datatype="xsd:string"></span><span id='am-593' property="oboInOwl:hasExactSynonym" content="GUC1A4" datatype="xsd:string"></span><span id='am-594' property="rdfs:label" content="RETGC1" datatype="xsd:string"></span><span id='am-595' property="oboInOwl:hasExactSynonym" content="guanylate cyclase 2D, membrane (retina-specific)" datatype="xsd:string"></span><span id='am-596' property="rdfs:label" content="GUC2D" datatype="xsd:string"></span><span id='am-597' property="rdfs:label" content="Retinal guanylyl cyclase 1" datatype="xsd:string"></span><span id='am-598' property="rdfs:label" content="RETGC" datatype="xsd:string"></span><span id='am-599' property="oboInOwl:hasExactSynonym" content="ROS-GC" datatype="xsd:string"></span><span id='am-600' property="oboInOwl:hasExactSynonym" content="GUCY2D" datatype="xsd:string"></span><span id='am-601' property="oboInOwl:hasExactSynonym" content="RETGC-1" datatype="xsd:string"></span><span id='am-602' property="oboInOwl:hasExactSynonym" content="CORD6" datatype="xsd:string"></span><span id='am-603' property="rdfs:label" content="LCA" datatype="xsd:string"></span><span id='am-604' property="oboInOwl:hasExactSynonym" content="Retinal guanylyl cyclase 1" datatype="xsd:string"></span><span id='am-605' property="rdfs:label" content="ROS-GC1" datatype="xsd:string"></span><span id='am-606' property="oboInOwl:hasExactSynonym" content="RETGC1" datatype="xsd:string"></span><span id='am-607' property="rdfs:label" content="GUCY2D" datatype="xsd:string"></span><span id='am-608' property="rdfs:label" content="GUC1A4" datatype="xsd:string"></span><span id='am-609' property="rdfs:label" content="CORD5" datatype="xsd:string"></span><span id='am-610' property="rdfs:label" content="retGC" datatype="xsd:string"></span><span id='am-611' property="rdfs:label" content="GUC2D_HUMAN" datatype="xsd:string"></span><span id='am-612' property="oboInOwl:id" content="IMR:0200844" datatype="xsd:string"></span><span id='am-613' property="rdfs:label" content="Guanylate cyclase 2D, retinal" datatype="xsd:string"></span><span id='am-614' property="rdfs:label" content="RETGC-1" datatype="xsd:string"></span><span id='am-615' property="oboInOwl:hasExactSynonym" content="CYGD" datatype="xsd:string"></span><span id='am-616' property="oboInOwl:hasExactSynonym" content="GUC2D" datatype="xsd:string"></span><span id='am-617' property="rdfs:label" content="CORD6" datatype="xsd:string"></span><span id='am-618' property="rdfs:label" content="EC 4.6.1.2" datatype="xsd:string"></span><span id='am-619' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-620' property="oboInOwl:hasExactSynonym" content="LCA1" datatype="xsd:string"></span><span id='am-621' property="rdfs:label" content="Rod outer segment membrane guanylate cyclase" datatype="xsd:string"></span><span id='am-622' property="rdfs:label" content="CYGD" datatype="xsd:string"></span>CYGD</span>) importantly, strong com-        (38). <span id='am-51' about='obo:IMR_0000368' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001'><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0000368" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="transcription factor" datatype="xsd:string"></span><span id='am-55' property="obo:IAO_0000115" content="Term loosely applied to any protein required to initiate or regulate transcription in eucaryotes. Includes both gene regulatory proteins as well as the general transcription factors." datatype="xsd:string"></span>Transcription factor</span> binding motifs used pensatory interaction pair duplicates, evo-     work compiled Kafri et al. (39) high- <br /> lutionary relevant decrease  65533;?tness persist     con 65533;?dence predictions Kellis et al. (28). used incomplete buffering damaging mutation         <span id='am-568' about='obo:IMR_0000949' typeof='owl:Thing obo:IMR_0000001'><span id='am-569' property="rdfs:label" content="protein" datatype="xsd:string"></span><span id='am-570' property="oboInOwl:id" content="IMR:0000949" datatype="xsd:string"></span><span id='am-571' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>protein</span> localization data Huh et al. (40), Codon duplicates (24). context long-term         Adaptation Index (CAI) calculations based data <br /> evolution, difference mu-         set Lu et al. (41) annotation protein <br /> tations leading lethal phenotype mutations           complexes CYGD. <br /> associated  65533;?tness decrease substantially larger inverse effective population size (25,26). <br /> Given typical population sizes free-living microbial    RESULTS <br /> species large (&gt;106 8211;108) (27), small  65533;?tness <br />                                                                 Hillenmeyer et al. (33) quanti 65533;?ed growth phenotypes decrease effectively lethal organisms. <br />                                                                 single-gene yeast deletion strains large collection Consequently, quantitative analyses growth pheno- <br />                                                                 environmental conditions. assembled data set <br /> types, preferably multiple environmental conditions, <br />                                                                 contains  5.5 million phenotypes heterozygous necessary understand extent compen- <br />                                                                 homozygous mutants  400 conditions. sampled <br /> sation duplicates plays important role                                                                 conditions represent 27 different environmental stresses <br /> natural biological populations. perform                                                                 hundreds perturbations diverse chemical com- <br /> analysis context natural popula- <br />                                                                 pounds. Environmental stresses comprised different <br /> tions, genetic buffering mediated duplicates likely                                                                 growth media, media lacking speci 65533;?c vitamins amino rare  surprisingly, monotonic function                                                                 acids, different pH temperature regimes. <br /> duplicates 8217; divergence. <br />                                                                 comprehensive collection phenotypes allowed                                                                 investigate diversi 65533;?cation duplicates 8217; <br />                                                                 functions contribution genetic robustness MATERIALS METHODS                                           multiple conditions. <br /> Gene protein sequences Saccharomyces cerevisiae,            65533;?rst investigated average number sensi- <br /> Saccharomyces paradoxus, Saccharomyces bayanus,                 tive conditions, e. conditions signi 65533; growth <br /> Saccharomyces      castelli,    Saccharomyces       mikatae,    decrease deletion duplicate, depends Saccharomyces kudriavzevii Saccharomyces kluyveri           sequence divergence (Ka) duplicated genes obtained Saccharomyces Genome                     (Figure 1A B). considered fraction different <br /> Database (SGD; http://downloads.yeastgenome.org/)               conditions growth phenotype quantitative study Kellis et al. (28). Pairs gene duplicates   measure compensation capacity duplicates identi 65533;?ed sequence homology proteins            various divergence distances. close duplicates genome using BLASTP (29). duplicates           average number sensitive conditions signi 65533;?cantly bidirectional best hits aligned       different random pair yeast singletons <br /> &gt;80% open reading frame 8217;s sequence length          (Figure 1B, horizontal line). Importantly, result does <br /> considered analysis (30). Following previous studies     imply random gene pairs close duplicates (1), excluded ribosomal genes analysis owing        equivalent terms similarity MF. high expression, dominant impact growth         demonstrate  observed pattern likely strong codon adaptation bias. Evolutionary distances            higher overall functional load close duplicates. duplicated genes estimated using               article use term  8216;functional <br /> method Yang Nielsen (31) implemented              load 8217; gene characterize average  65533;?tness  <br /> PAML package (32); use methods,            crease considered conditions gene <br /> maximum likelihood, estimate Ka <span id='am-878' about='obo:IMR_0701288' typeof='owl:Thing'><span id='am-879' property="rdfs:label" content="KNO" datatype="xsd:string"></span><span id='am-880' property="oboInOwl:hasExactSynonym" content="KS" datatype="xsd:string"></span><span id='am-881' property="rdfs:label" content="Collagen alpha-1(XVIII) chain" datatype="xsd:string"></span><span id='am-882' property="rdfs:label" content="COLLAGEN, TYPE XVIII, ALPHA-1" datatype="xsd:string"></span><span id='am-883' property="rdfs:label" content="COIA1_HUMAN" datatype="xsd:string"></span><span id='am-884' property="rdfs:label" content="FLJ34914" datatype="xsd:string"></span><span id='am-885' property="oboInOwl:hasExactSynonym" content="KNO" datatype="xsd:string"></span><span id='am-886' property="rdfs:label" content="FLJ27325" datatype="xsd:string"></span><span id='am-887' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-888' property="rdfs:label" content="Endostatin" datatype="xsd:string"></span><span id='am-889' property="oboInOwl:hasDbXref" content="UniProt:P39060" datatype="xsd:string"></span><span id='am-890' property="oboInOwl:hasExactSynonym" content="COLLAGEN, TYPE XVIII, ALPHA-1" datatype="xsd:string"></span><span id='am-891' property="oboInOwl:hasExactSynonym" content="Collagen alpha-1(XVIII) chain" datatype="xsd:string"></span><span id='am-892' property="rdfs:label" content="KNO1" datatype="xsd:string"></span><span id='am-893' property="rdfs:label" content="KS" datatype="xsd:string"></span><span id='am-894' property="oboInOwl:hasExactSynonym" content="MGC74745" datatype="xsd:string"></span><span id='am-895' property="oboInOwl:hasExactSynonym" content="KNO1" datatype="xsd:string"></span><span id='am-896' property="oboInOwl:hasExactSynonym" content="collagen, type XVIII, alpha 1" datatype="xsd:string"></span><span id='am-897' property="oboInOwl:hasExactSynonym" content="COL18A1" datatype="xsd:string"></span><span id='am-898' property="oboInOwl:hasExactSynonym" content="FLJ34914" datatype="xsd:string"></span><span id='am-899' property="rdfs:label" content="MGC74745" datatype="xsd:string"></span><span id='am-900' property="oboInOwl:id" content="IMR:0701288" datatype="xsd:string"></span><span id='am-901' property="rdfs:label" content="collagen, type XVIII, alpha 1" datatype="xsd:string"></span><span id='am-902' property="rdfs:label" content="COL18A1" datatype="xsd:string"></span><span id='am-903' property="oboInOwl:hasExactSynonym" content="Endostatin" datatype="xsd:string"></span><span id='am-904' property="oboInOwl:hasExactSynonym" content="FLJ27325" datatype="xsd:string"></span>Ks</span> did sig-          deletion; note  based de 65533;?nition  ni 65533;?cantly change observed patterns (Supplementary           functional load measure total number Figure S1A).                                                    MFs gene  re 64258;ects gene 8217;s overall  65533;?tness <br />    used data obtained Hillenmeyer et al. (33)      contribution. <br /> measure  65533;?tness contribution duplicates               Interestingly, number sensitive conditions initially <br /> multiple environmental conditions chemical perturb-         drops duplicates diverge, decreasing 30% ations. Using P-value cutoff 0.01, obtained         distances corresponding Ka <span id='am-399' about='obo:IMR_0702349' typeof='owl:Thing'><span id='am-400' property="oboInOwl:hasExactSynonym" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-401' property="rdfs:label" content="MGC125856" datatype="xsd:string"></span><span id='am-402' property="rdfs:label" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-403' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-404' property="rdfs:label" content="DKFZp686D13177" datatype="xsd:string"></span><span id='am-405' property="oboInOwl:hasExactSynonym" content="MGC129961" datatype="xsd:string"></span><span id='am-406' property="rdfs:label" content="APRT" datatype="xsd:string"></span><span id='am-407' property="oboInOwl:hasDbXref" content="UniProt:P07741" datatype="xsd:string"></span><span id='am-408' property="oboInOwl:hasExactSynonym" content="AMP" datatype="xsd:string"></span><span id='am-409' property="oboInOwl:hasExactSynonym" content="EC 2.4.2.7" datatype="xsd:string"></span><span id='am-410' property="rdfs:label" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-411' property="oboInOwl:hasExactSynonym" content="APRT" datatype="xsd:string"></span><span id='am-412' property="oboInOwl:hasExactSynonym" content="MGC125857" datatype="xsd:string"></span><span id='am-413' property="rdfs:label" content="APT_HUMAN" datatype="xsd:string"></span><span id='am-414' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-415' property="oboInOwl:hasExactSynonym" content="DKFZp686D13177" datatype="xsd:string"></span><span id='am-416' property="oboInOwl:hasExactSynonym" content="MGC125856" datatype="xsd:string"></span><span id='am-417' property="rdfs:label" content="Adenine phosphoribosyltransferase" datatype="xsd:string"></span><span id='am-418' property="rdfs:label" content="MGC129961" datatype="xsd:string"></span><span id='am-419' property="rdfs:label" content="MGC125857" datatype="xsd:string"></span><span id='am-420' property="oboInOwl:id" content="IMR:0702349" datatype="xsd:string"></span><span id='am-421' property="oboInOwl:hasExactSynonym" content="adenine phosphoribosyltransferase" datatype="xsd:string"></span>amp</span>; 0.1 (Ks &amp; 1, number experimental conditions growth            Supplementary Figure S2A B). duplicates <br /> defect observed single gene deletion              diverge  average number sensitive conditions <br /> mutant. analyzed quantitative growth measure-           increases  reaching average random pair <br /> ments double single deletion yeast strains obtained     yeast singletons Ka &amp; 0.25. trend shown                                                                                                                                                              Nucleic Acids Research, 2014, Vol. 42,  4 2407 <br />  <br />  <br />       Fraction conditions significant     0.30                                                        B <br />                                                                                                                                                           0.12 <br />  <br />  <br />  <br />  <br />                                                                                                              Fraction conditions significant <br />                                                  0.25 <br />                                                                                                                                                           0.11 <br />                                                  0.20 <br />               growth decrease <br />  <br />  <br />  <br />  <br />                                                                                                                          growth decrease <br />                                                                                                                                                           0.10 <br />                                                  0.15 <br />                                                                                                                                                           0.09 <br />                                                  0.10 <br />                                                                                                                                                           0.08 <br />  <br />                                                  0.05                                                                                                     0.07 <br />                                                                                                                                                                   p=0.01 <br />                                                  0.00                                                                                                     0.06 <br />  <br />                                                          0.0   0.1   0.2     0.3    0.4   0.5    0.6                                                             0.0   0.1   0.2    0.3   0.4    0.5   0.6 <br />                                                                              Ka                                                                                                     Ka <br />  <br />  <br />           C 0.25                                                                                             D                                                                                         SSD <br />                                                                                                                                                          0.14 <br />                                                                                                                                                                                                        WGD <br />  <br />  <br />                                                                                                              Fraction conditions significant <br />             Fraction essential duplicates <br />  <br />  <br />  <br />  <br />                                                0.20 <br />                                                                                                                                                          0.12 <br />  <br />                                                0.15                                                                      growth decrease <br />                                                                                                                                                          0.10 <br />  <br />                                                0.10 <br />                                                                                                                                                          0.08 <br />                                                0.05 <br />                                                                                                                                                          0.06 <br />                                                0.00 <br />                                                                                                                                                          0.04 <br />                                                         0.0    0.2     0.4         0.6     0.8         1.0                                                       0.0   0.1   0. 2   0.3   0. 4   0.5   0.6 <br />                                                                            Ka <br />                                                                                                                                                                                     Ka <br /> Figure 1. Compensation patterns yeast duplicates function evolutionary divergence, Ka, number nonsynonymous <br /> substitutions site.   Scatterplot fraction sensitive conditions, e. conditions detectable growth phenotypes resulting duplicate gene deletion, versus Ka. dot  65533;?gure represents pair yeast duplicates. (B) average fraction sensitive conditions duplicate pair. P-value calculated using Mann 8211;Whitney U test. horizontal lines B indicate average fraction sensitive conditions random pair yeast singletons. (C) average fraction essential duplicates, e. duplicates lethal phenotype deletion, function Ka. horizontal line indicates fraction essential yeast singletons. Gene essentiality data obtained Saccharomyces Genome Deletion Project (36). (D) Fraction conditions signi 65533; growth decrease deletion yeast duplicates arising small-scale (SSD) genome duplications (WGD). duplicates classi 65533;?ed SSD WGD based study Kellis et al. <br /> (8) horizontal line shows average fraction sensitive conditions random pair yeast singletons.  65533;?gures, error bars represent standard error mean (<span id='am-1010' about='obo:IMR_0002978' typeof='owl:Thing'><span id='am-1011' property="rdfs:label" content="Extracellular-signal-related kinase A" datatype="xsd:string"></span><span id='am-1012' property="oboInOwl:hasExactSynonym" content="E(sina)7" datatype="xsd:string"></span><span id='am-1013' property="obo:IAO_0000115" content="The Drosophila genome has representatives of each of the three MAPK subfamilies found in other animals: one extracellular signal-related kinase [ERK; rolled (rl)], one Jun N-terminal kinase [JNK; basket (bsk)], and p38 (encoded by the two genes Mpk2 and p38b)." datatype="xsd:string"></span><span id='am-1014' property="rdfs:label" content="ERKA_DROME" datatype="xsd:string"></span><span id='am-1015' property="oboInOwl:hasExactSynonym" content="Mitogen-activated protein kinase ERK-A" datatype="xsd:string"></span><span id='am-1016' property="rdfs:label" content="ERK" datatype="xsd:string"></span><span id='am-1017' property="oboInOwl:hasExactSynonym" content="DmErk" datatype="xsd:string"></span><span id='am-1018' property="oboInOwl:hasExactSynonym" content="CG12559" datatype="xsd:string"></span><span id='am-1019' property="rdfs:label" content="rolled" datatype="xsd:string"></span><span id='am-1020' property="rdfs:label" content="Sem" datatype="xsd:string"></span><span id='am-1021' property="rdfs:label" content="DmERK-A" datatype="xsd:string"></span><span id='am-1022' property="oboInOwl:hasExactSynonym" content="Erk1" datatype="xsd:string"></span><span id='am-1023' property="rdfs:label" content="DmERKA" datatype="xsd:string"></span><span id='am-1024' property="oboInOwl:hasExactSynonym" content="DmERK-A" datatype="xsd:string"></span><span id='am-1025' property="rdfs:label" content="l(2R)EMS45-39" datatype="xsd:string"></span><span id='am-1026' property="rdfs:label" content="pERK" datatype="xsd:string"></span><span id='am-1027' property="rdfs:label" content="CT39192" datatype="xsd:string"></span><span id='am-1028' property="rdfs:label" content="l(2)41Ac" datatype="xsd:string"></span><span id='am-1029' property="oboInOwl:hasExactSynonym" content="l(2R)EMS45-39" datatype="xsd:string"></span><span id='am-1030' property="rdfs:label" content="SR2-1" datatype="xsd:string"></span><span id='am-1031' property="rdfs:label" content="dpERK" datatype="xsd:string"></span><span id='am-1032' property="rdfs:label" content="Protein rolled" datatype="xsd:string"></span><span id='am-1033' property="oboInOwl:hasExactSynonym" content="BcDNA:RE08694" datatype="xsd:string"></span><span id='am-1034' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1035' property="rdfs:label" content="Sevenmaker" datatype="xsd:string"></span><span id='am-1036' property="rdfs:label" content="Erk MAP kinase" datatype="xsd:string"></span><span id='am-1037' property="oboInOwl:hasExactSynonym" content="rll" datatype="xsd:string"></span><span id='am-1038' property="rdfs:label" content="Erk/Map kinase" datatype="xsd:string"></span><span id='am-1039' property="rdfs:label" content="CG18732" datatype="xsd:string"></span><span id='am-1040' property="oboInOwl:hasExactSynonym" content="CG18732" datatype="xsd:string"></span><span id='am-1041' property="oboInOwl:hasExactSynonym" content="pERK" datatype="xsd:string"></span><span id='am-1042' property="rdfs:label" content="dp-ERK" datatype="xsd:string"></span><span id='am-1043' property="rdfs:label" content="rll" datatype="xsd:string"></span><span id='am-1044' property="oboInOwl:hasExactSynonym" content="Su(Raf)2B" datatype="xsd:string"></span><span id='am-1045' property="oboInOwl:hasExactSynonym" content="CT39192" datatype="xsd:string"></span><span id='am-1046' property="oboInOwl:id" content="IMR:0002978" datatype="xsd:string"></span><span id='am-1047' property="rdfs:label" content="Extracellular-regulated kinase A" datatype="xsd:string"></span><span id='am-1048' property="rdfs:label" content="Erk" datatype="xsd:string"></span><span id='am-1049' property="oboInOwl:hasExactSynonym" content="sevenmaker" datatype="xsd:string"></span><span id='am-1050' property="rdfs:label" content="Enhancer of seven in absentia 7" datatype="xsd:string"></span><span id='am-1051' property="rdfs:label" content="CT34260" datatype="xsd:string"></span><span id='am-1052' property="oboInOwl:hasExactSynonym" content="rl/MAPK" datatype="xsd:string"></span><span id='am-1053' property="rdfs:label" content="erk" datatype="xsd:string"></span><span id='am-1054' property="oboInOwl:hasExactSynonym" content="Extracellular-regulated kinase A" datatype="xsd:string"></span><span id='am-1055' property="oboInOwl:hasExactSynonym" content="DERK" datatype="xsd:string"></span><span id='am-1056' property="rdfs:label" content="DmMAPK" datatype="xsd:string"></span><span id='am-1057' property="rdfs:label" content="ErkA" datatype="xsd:string"></span><span id='am-1058' property="rdfs:label" content="MAP-kinase" datatype="xsd:string"></span><span id='am-1059' property="rdfs:label" content="DERK" datatype="xsd:string"></span><span id='am-1060' property="oboInOwl:hasExactSynonym" content="Mapk" datatype="xsd:string"></span><span id='am-1061' property="oboInOwl:hasExactSynonym" content="EY2-2" datatype="xsd:string"></span><span id='am-1062' property="rdfs:label" content="MAPK" datatype="xsd:string"></span><span id='am-1063' property="oboInOwl:hasExactSynonym" content="Enhancer of seven in absentia 7" datatype="xsd:string"></span><span id='am-1064' property="rdfs:label" content="EK2-1" datatype="xsd:string"></span><span id='am-1065' property="rdfs:label" content="E(sina)7" datatype="xsd:string"></span><span id='am-1066' property="oboInOwl:hasExactSynonym" content="SR2-1" datatype="xsd:string"></span><span id='am-1067' property="oboInOwl:hasExactSynonym" content="MAP kinase" datatype="xsd:string"></span><span id='am-1068' property="rdfs:label" content="dpErk" datatype="xsd:string"></span><span id='am-1069' property="rdfs:label" content="EC 2.7.11.24" datatype="xsd:string"></span><span id='am-1070' property="rdfs:label" content="Mapk" datatype="xsd:string"></span><span id='am-1071' property="oboInOwl:hasExactSynonym" content="CT34260" datatype="xsd:string"></span><span id='am-1072' property="oboInOwl:hasExactSynonym" content="Sem" datatype="xsd:string"></span><span id='am-1073' property="oboInOwl:hasExactSynonym" content="GroupII" datatype="xsd:string"></span><span id='am-1074' property="oboInOwl:hasExactSynonym" content="rolled" datatype="xsd:string"></span><span id='am-1075' property="rdfs:label" content="CG12559" datatype="xsd:string"></span><span id='am-1076' property="oboInOwl:hasExactSynonym" content="Erk/Map kinase" datatype="xsd:string"></span><span id='am-1077' property="oboInOwl:hasExactSynonym" content="EK2-1" datatype="xsd:string"></span><span id='am-1078' property="rdfs:label" content="EY2-2" datatype="xsd:string"></span><span id='am-1079' property="oboInOwl:hasExactSynonym" content="dpMAPK" datatype="xsd:string"></span><span id='am-1080' property="rdfs:label" content="ERK-A" datatype="xsd:string"></span><span id='am-1081' property="rdfs:label" content="rl/MAPK" datatype="xsd:string"></span><span id='am-1082' property="oboInOwl:hasExactSynonym" content="ErkA" datatype="xsd:string"></span><span id='am-1083' property="oboInOwl:hasDbXref" content="UniProt:P40417" datatype="xsd:string"></span><span id='am-1084' property="rdfs:label" content="GroupII" datatype="xsd:string"></span><span id='am-1085' property="oboInOwl:hasExactSynonym" content="ERK-A" datatype="xsd:string"></span><span id='am-1086' property="oboInOwl:hasExactSynonym" content="Extracellular-signal-related kinase A" datatype="xsd:string"></span><span id='am-1087' property="oboInOwl:hasExactSynonym" content="EC 2.7.11.24" datatype="xsd:string"></span><span id='am-1088' property="rdfs:label" content="sevenmaker" datatype="xsd:string"></span><span id='am-1089' property="oboInOwl:hasExactSynonym" content="ERK" datatype="xsd:string"></span><span id='am-1090' property="rdfs:label" content="Erk1" datatype="xsd:string"></span><span id='am-1091' property="rdfs:label" content="Su(Raf)2B" datatype="xsd:string"></span><span id='am-1092' property="rdfs:label" content="ERKa" datatype="xsd:string"></span><span id='am-1093' property="oboInOwl:hasExactSynonym" content="Protein rolled" datatype="xsd:string"></span><span id='am-1094' property="oboInOwl:hasExactSynonym" content="mapk" datatype="xsd:string"></span><span id='am-1095' property="rdfs:label" content="Dsor2" datatype="xsd:string"></span><span id='am-1096' property="oboInOwl:hasExactSynonym" content="Erk MAP kinase" datatype="xsd:string"></span><span id='am-1097' property="oboInOwl:hasExactSynonym" content="l(2)41Ac" datatype="xsd:string"></span><span id='am-1098' property="rdfs:label" content="BcDNA:RE08694" datatype="xsd:string"></span><span id='am-1099' property="oboInOwl:hasExactSynonym" content="Sevenmaker" datatype="xsd:string"></span><span id='am-1100' property="oboInOwl:hasExactSynonym" content="DmMAPK" datatype="xsd:string"></span><span id='am-1101' property="oboInOwl:hasExactSynonym" content="ERKA" datatype="xsd:string"></span><span id='am-1102' property="oboInOwl:hasExactSynonym" content="ERKa" datatype="xsd:string"></span><span id='am-1103' property="oboInOwl:hasExactSynonym" content="Dsor2" datatype="xsd:string"></span><span id='am-1104' property="oboInOwl:hasExactSynonym" content="MAPK" datatype="xsd:string"></span><span id='am-1105' property="rdfs:label" content="Mitogen-activated protein kinase ERK-A" datatype="xsd:string"></span><span id='am-1106' property="rdfs:label" content="DmErk" datatype="xsd:string"></span><span id='am-1107' property="rdfs:label" content="DERK-A" datatype="xsd:string"></span><span id='am-1108' property="rdfs:label" content="MAP kinase" datatype="xsd:string"></span><span id='am-1109' property="oboInOwl:hasExactSynonym" content="DmERKA" datatype="xsd:string"></span><span id='am-1110' property="oboInOwl:hasExactSynonym" content="Erk" datatype="xsd:string"></span><span id='am-1111' property="rdfs:label" content="dpMAPK" datatype="xsd:string"></span><span id='am-1112' property="oboInOwl:hasExactSynonym" content="DERK-A" datatype="xsd:string"></span><span id='am-1113' property="oboInOwl:hasExactSynonym" content="MAP-kinase" datatype="xsd:string"></span><span id='am-1114' property="rdfs:label" content="D-ERK" datatype="xsd:string"></span><span id='am-1115' property="rdfs:label" content="mapk" datatype="xsd:string"></span><span id='am-1116' property="oboInOwl:hasExactSynonym" content="dpERK" datatype="xsd:string"></span><span id='am-1117' property="rdfs:label" content="ERKA" datatype="xsd:string"></span><span id='am-1118' property="oboInOwl:hasExactSynonym" content="erk" datatype="xsd:string"></span><span id='am-1119' property="oboInOwl:hasExactSynonym" content="D-ERK" datatype="xsd:string"></span><span id='am-1120' property="oboInOwl:hasExactSynonym" content="dp-ERK" datatype="xsd:string"></span><span id='am-1121' property="oboInOwl:hasExactSynonym" content="dpErk" datatype="xsd:string"></span>SEM</span>). <br />  <br />  <br />  <br /> Figure 1B sensitive P-value cutoff used                                                     compensation heterozygous homozygous <br /> determine signi 65533;?cance growth decrease observed                                                    mutants similar multiple-drug resistance genes, mutant strains (Supplementary Figure <span id='am-732' about='obo:IMR_0702945' typeof='owl:Thing'><span id='am-733' property="oboInOwl:hasExactSynonym" content="S3" datatype="xsd:string"></span><span id='am-734' property="rdfs:label" content="26S proteasome regulatory subunit S3" datatype="xsd:string"></span><span id='am-735' property="oboInOwl:hasExactSynonym" content="Rpn3" datatype="xsd:string"></span><span id='am-736' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-737' property="rdfs:label" content="TSTA2" datatype="xsd:string"></span><span id='am-738' property="rdfs:label" content="PSMD3" datatype="xsd:string"></span><span id='am-739' property="rdfs:label" content="P58" datatype="xsd:string"></span><span id='am-740' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN3" datatype="xsd:string"></span><span id='am-741' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 3" datatype="xsd:string"></span><span id='am-742' property="rdfs:label" content="Rpn3" datatype="xsd:string"></span><span id='am-743' property="rdfs:label" content="26S proteasome regulatory subunit RPN3" datatype="xsd:string"></span><span id='am-744' property="oboInOwl:hasExactSynonym" content="P58" datatype="xsd:string"></span><span id='am-745' property="rdfs:label" content="S3" datatype="xsd:string"></span><span id='am-746' property="rdfs:label" content="Proteasome subunit p58" datatype="xsd:string"></span><span id='am-747' property="rdfs:label" content="RPN3" datatype="xsd:string"></span><span id='am-748' property="oboInOwl:id" content="IMR:0702945" datatype="xsd:string"></span><span id='am-749' property="rdfs:label" content="PSMD3_HUMAN" datatype="xsd:string"></span><span id='am-750' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 3" datatype="xsd:string"></span><span id='am-751' property="oboInOwl:hasExactSynonym" content="PSMD3" datatype="xsd:string"></span><span id='am-752' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 3" datatype="xsd:string"></span><span id='am-753' property="oboInOwl:hasExactSynonym" content="Proteasome subunit p58" datatype="xsd:string"></span><span id='am-754' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S3" datatype="xsd:string"></span><span id='am-755' property="oboInOwl:hasExactSynonym" content="TSTA2" datatype="xsd:string"></span><span id='am-756' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 3" datatype="xsd:string"></span><span id='am-757' property="oboInOwl:hasDbXref" content="UniProt:O43242" datatype="xsd:string"></span><span id='am-758' property="oboInOwl:hasExactSynonym" content="RPN3" datatype="xsd:string"></span>S3</span>). similar                                                       de 65533;?ned Hillenmeyer et al. (33), considered <br /> trend observed average growth decrease                                                      (Supplementary Figure S4C). trends remain <br /> (functional load), measured log ratios                                                          similar environmental perturbations scores tested conditions (Supplementary                                                         analyzed homozygous experiments (Supplementary <br /> Figure S4A B). Bin-free analyses data                                                             Figure S4D). checked observed compen- <br /> (Supplementary Figures S1B C S2B)                                                               sation patterns closest duplicates signi 65533;?- <br /> revealed smaller  65533;?tness cost loss duplicates                                                 cantly in 64258;uenced additional, e. diverged, intermediate distances (Ka &amp; 0.1).                                                                        paralogs (Supplementary Figure S4E). lack signi 65533;?- <br />    actively growing wild-type yeast popula-                                                     compensation diverged duplicates results tions diploid (42), mainly focused analysis                                                    approximately linear relationship number heterozygous mutant strains. patterns functional                                                      sensitive conditions yeast protein family  2408 Nucleic Acids Research, 2014, Vol. 42,  4 <br />  <br />  <br /> family size (Supplementary Figure S5). Finally,              pairs decreases function sequence divergence <br /> observed compensation patterns affected              (Figure 2E F)  (21)]. <br /> removal gene pairs high CAI (Supplementary                smaller overall functional load, possible <br /> Figure S6A), suggesting observed trend           duplicates intermediate distances explained expression-based constraints rate         properties favor genetic robustness. explore duplicate sequence evolution (Ka) (43) high expres-        possibility, duplicate pair, looked gene <br /> sion levels certain duplicates.                               largest gene smallest number    interesting compare ability duplicates     sensitive conditions (Figure 3A). Notably, dupli- <br /> buffer mutations leading detectable growth                cate conditions (Figure 3A, sensitive <br /> decrease given  65533;?tness threshold (Figure 1B)             duplicate) follows average trend duplicates role protecting growth pheno-        (Figure 1B), duplicate fewer conditions (Figure <br /> type, e. likelihood observe essential genes dupli-   3A, sensitive duplicate) shows steady gain cate pairs. Figure 1C, using data study           number conditions function Ka. Consequently, <br /> Giaever et al. (36), fraction essential du-       functional load close duplicates, measured plicates function divergence. agreement         number sensitive conditions, different, differ- previous studies (1,22,23)                ence signi 65533;?cantly smaller genes diverge <br /> fraction essential genes remains low approximately        (Figure 3B. Pearson 8217;s r = 0.64, P = 7   10 4, constant close duplicates, increases substantially       Supplementary Figure S2C). Close duplicates divergence distances corresponding Ka &gt; 0.4.          larger number sensitive conditions higher <br /> Notably, pattern qualitatively different        evolutionary constraint, evaluated normalized ratio <br /> compensation quantitative growth phenotypes                  nonsynonymous synonymous substitutions nucleo- <br /> (Figure 1B), demonstrating aforementioned impact             tide site, Ka/Ks (Wilcoxon Signed Rank test P = 7   10 3, using quantitative phenotypes assess evolution-        Figure 2C). result agrees previous reports arily relevant consequences mutations. contrast       asymmetric evolution duplicates context ex- patterns obtained studies based essential genes         pression, genetic interaction protein 8211;protein interaction <br /> (22), observed similar compensation pro 65533;?les gene          networks (19,44,45). observed asymmetry func- <br /> pairs originating small-scale genome-wide dupli-        tional load close duplicates make buffering <br /> cations (Figure 1D). WGD duplicates         <span id='am-327' about='obo:IMR_0700040' typeof='owl:Thing'><span id='am-328' property="rdfs:label" content="Tumor necrosis factor, soluble form" datatype="xsd:string"></span><span id='am-329' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor ligand superfamily member 2" datatype="xsd:string"></span><span id='am-330' property="rdfs:label" content="TNF-alpha" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor" datatype="xsd:string"></span><span id='am-332' property="rdfs:label" content="DIF" datatype="xsd:string"></span><span id='am-333' property="oboInOwl:hasExactSynonym" content="tumor necrosis factor-alpha" datatype="xsd:string"></span><span id='am-334' property="rdfs:label" content="TNF-a" datatype="xsd:string"></span><span id='am-335' property="oboInOwl:hasExactSynonym" content="TNFalpha" datatype="xsd:string"></span><span id='am-336' property="rdfs:label" content="TNFalpha" datatype="xsd:string"></span><span id='am-337' property="rdfs:label" content="Tumor necrosis factor ligand superfamily member 2" datatype="xsd:string"></span><span id='am-338' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-339' property="oboInOwl:hasExactSynonym" content="Tnfa" datatype="xsd:string"></span><span id='am-340' property="oboInOwl:hasExactSynonym" content="tumor necrosis factor" datatype="xsd:string"></span><span id='am-341' property="rdfs:label" content="Tumor necrosis factor" datatype="xsd:string"></span><span id='am-342' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor, soluble form" datatype="xsd:string"></span><span id='am-343' property="rdfs:label" content="tumor necrosis factor" datatype="xsd:string"></span><span id='am-344' property="oboInOwl:hasExactSynonym" content="TNF alpha" datatype="xsd:string"></span><span id='am-345' property="oboInOwl:hasExactSynonym" content="TNF-a" datatype="xsd:string"></span><span id='am-346' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor, membrane form" datatype="xsd:string"></span><span id='am-347' property="oboInOwl:hasExactSynonym" content="Cachectin" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasExactSynonym" content="DIF" datatype="xsd:string"></span><span id='am-349' property="rdfs:label" content="Tnf" datatype="xsd:string"></span><span id='am-350' property="rdfs:label" content="Cachectin" datatype="xsd:string"></span><span id='am-351' property="rdfs:label" content="TNFA_MOUSE" datatype="xsd:string"></span><span id='am-352' property="rdfs:label" content="TNF alpha" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:id" content="IMR:0700040" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasExactSynonym" content="Tnfsf1a" datatype="xsd:string"></span><span id='am-355' property="oboInOwl:hasExactSynonym" content="Tnfsf2" datatype="xsd:string"></span><span id='am-356' property="rdfs:label" content="Tnfsf1a" datatype="xsd:string"></span><span id='am-357' property="rdfs:label" content="tumor necrosis factor-alpha" datatype="xsd:string"></span><span id='am-358' property="oboInOwl:hasExactSynonym" content="Tnf" datatype="xsd:string"></span><span id='am-359' property="rdfs:label" content="Tnfsf2" datatype="xsd:string"></span><span id='am-360' property="oboInOwl:hasExactSynonym" content="TNF-alpha" datatype="xsd:string"></span><span id='am-361' property="rdfs:label" content="Tnfa" datatype="xsd:string"></span><span id='am-362' property="rdfs:label" content="Tumor necrosis factor, membrane form" datatype="xsd:string"></span><span id='am-363' property="oboInOwl:hasDbXref" content="UniProt:P06804" datatype="xsd:string"></span>dif</span> 65533;?cult. example, sensitive duplicate                                                                  expressed speci 65533;?c environmental conditions. <span id='am-300' about='obo:IMR_0200587' typeof='owl:Thing'><span id='am-301' property="rdfs:label" content="EC 5.1.3.8" datatype="xsd:string"></span><span id='am-302' property="rdfs:label" content="N-acylglucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-303' property="rdfs:label" content="Renin-binding protein" datatype="xsd:string"></span><span id='am-304' property="rdfs:label" content="RNBP" datatype="xsd:string"></span><span id='am-305' property="rdfs:label" content="RENBP_HUMAN" datatype="xsd:string"></span><span id='am-306' property="rdfs:label" content="N-acetyl-D-glucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-307' property="rdfs:label" content="renin binding protein" datatype="xsd:string"></span><span id='am-308' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-309' property="oboInOwl:hasDbXref" content="UniProt:P51606" datatype="xsd:string"></span><span id='am-310' property="rdfs:label" content="RnBP" datatype="xsd:string"></span><span id='am-311' property="oboInOwl:hasExactSynonym" content="RNBP" datatype="xsd:string"></span><span id='am-312' property="oboInOwl:hasExactSynonym" content="AGE" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasExactSynonym" content="GlcNAc 2-epimerase" datatype="xsd:string"></span><span id='am-314' property="oboInOwl:hasExactSynonym" content="EC 5.1.3.8" datatype="xsd:string"></span><span id='am-315' property="oboInOwl:hasExactSynonym" content="renin binding protein" datatype="xsd:string"></span><span id='am-316' property="oboInOwl:hasExactSynonym" content="RBP" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasExactSynonym" content="Renin-binding protein" datatype="xsd:string"></span><span id='am-318' property="oboInOwl:hasExactSynonym" content="N-acylglucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-319' property="oboInOwl:id" content="IMR:0200587" datatype="xsd:string"></span><span id='am-320' property="rdfs:label" content="AGE" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasExactSynonym" content="RnBP" datatype="xsd:string"></span><span id='am-322' property="rdfs:label" content="RBP" datatype="xsd:string"></span><span id='am-323' property="oboInOwl:hasExactSynonym" content="RENBP" datatype="xsd:string"></span><span id='am-324' property="oboInOwl:hasExactSynonym" content="N-acetyl-D-glucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-325' property="rdfs:label" content="GlcNAc 2-epimerase" datatype="xsd:string"></span><span id='am-326' property="rdfs:label" content="RENBP" datatype="xsd:string"></span>age</span>, result suggests ability duplicates <br />                                                                     explore mechanism observed buffer s function multiple conditions <br />                                                                  backup patterns, analyzed functional diversi 65533;?cation <br /> depends strongly sequence divergence                                                                  yeast duplicates function sequence diver- time duplication. <br />                                                                  gence (Ka).  genes encoding metabolic enzymes <br />    likely observed decrease number                                                                  calculated fraction gene pairs conserved EC <br /> sensitive conditions intermediate divergence distances <br />                                                                  numbers (Figure 4A); conservation EC numbers <br /> (Ka &amp; 0.1) decrease functional load           indicates corresponding proteins catalyze identical <br /> carried distances union duplicate             biochemical reactions. Second, calculated fraction <br /> genes. explore possibility, considered quan-      shared terms describing protein MF dupli- <br /> titative  65533;?tness data DeLuna et al. (34) <span id='am-539' about='obo:IMR_0704059' typeof='owl:Thing'><span id='am-540' property="oboInOwl:hasExactSynonym" content="Src-like kinase" datatype="xsd:string"></span><span id='am-541' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-542' property="rdfs:label" content="EC 2.7.10.2" datatype="xsd:string"></span><span id='am-543' property="rdfs:label" content="Proto-oncogene Syn" datatype="xsd:string"></span><span id='am-544' property="oboInOwl:hasExactSynonym" content="Proto-oncogene Syn" datatype="xsd:string"></span><span id='am-545' property="rdfs:label" content="Tyrosine-protein kinase Fyn" datatype="xsd:string"></span><span id='am-546' property="oboInOwl:id" content="IMR:0704059" datatype="xsd:string"></span><span id='am-547' property="oboInOwl:hasExactSynonym" content="MGC45350" datatype="xsd:string"></span><span id='am-548' property="oboInOwl:hasExactSynonym" content="FYN oncogene related to SRC, FGR, YES" datatype="xsd:string"></span><span id='am-549' property="rdfs:label" content="Proto-oncogene c-Fyn" datatype="xsd:string"></span><span id='am-550' property="oboInOwl:hasDbXref" content="UniProt:P06241" datatype="xsd:string"></span><span id='am-551' property="oboInOwl:hasExactSynonym" content="SYN" datatype="xsd:string"></span><span id='am-552' property="rdfs:label" content="Src-like kinase" datatype="xsd:string"></span><span id='am-553' property="oboInOwl:hasExactSynonym" content="FYN" datatype="xsd:string"></span><span id='am-554' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein kinase Fyn" datatype="xsd:string"></span><span id='am-555' property="rdfs:label" content="SLK" datatype="xsd:string"></span><span id='am-556' property="oboInOwl:hasExactSynonym" content="p59-Fyn" datatype="xsd:string"></span><span id='am-557' property="oboInOwl:hasExactSynonym" content="SLK" datatype="xsd:string"></span><span id='am-558' property="rdfs:label" content="FYN TYROSINE KINASE PROTOONCOGENE" datatype="xsd:string"></span><span id='am-559' property="rdfs:label" content="FYN" datatype="xsd:string"></span><span id='am-560' property="rdfs:label" content="SYN" datatype="xsd:string"></span><span id='am-561' property="oboInOwl:hasExactSynonym" content="Proto-oncogene c-Fyn" datatype="xsd:string"></span><span id='am-562' property="oboInOwl:hasExactSynonym" content="FYN TYROSINE KINASE PROTOONCOGENE" datatype="xsd:string"></span><span id='am-563' property="rdfs:label" content="p59-Fyn" datatype="xsd:string"></span><span id='am-564' property="rdfs:label" content="FYN oncogene related to SRC, FGR, YES" datatype="xsd:string"></span><span id='am-565' property="rdfs:label" content="FYN_HUMAN" datatype="xsd:string"></span><span id='am-566' property="oboInOwl:hasExactSynonym" content="EC 2.7.10.2" datatype="xsd:string"></span><span id='am-567' property="rdfs:label" content="MGC45350" datatype="xsd:string"></span>syn</span>-        cates (Figure 4B). measures showed MF thetic genetic array (SGA) data Costanzo et al. (35).       yeast duplicates typically starts substantially diverge studies, authors performed quantitative             Ka &gt; 0.4. timing divergence ap- <br /> growth measurements yeast strains individual             proximately coincides signi 65533; increase simultaneous deletions duplicates. Using              fraction essential duplicates (Figure 1C). single deletion phenotypes DeLuna et al.                hand, signi 65533; changes number quantitative <br /> (Figure 2A) Costanzo et al. studies (Figure 2B),          growth phenotypes observed MF dupli- <br /> observed  65533;?tness pro 65533;?les similar obtained based        cates usually conserved. data Hillenmeyer et al. (Figure 1B)                complementary analysis transcription factor <br /> function Ka, smaller phenotypic effects inter-        binding sites suggests gene regulation plays import- <br /> mediate distances. Interestingly, overall functional         ant role establishing observed compensation <br /> load duplicate pairs, measured phenotype            patterns. previously demonstrated duplicated <br /> double deletions, substantially decreases            yeast genes  average, higher number <span id='am-843' about='obo:IMR_0000460' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000428'><span id='am-844' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-845' property="oboInOwl:hasExactSynonym" content="CIS" datatype="xsd:string"></span><span id='am-846' property="oboInOwl:id" content="IMR:0000460" datatype="xsd:string"></span><span id='am-847' property="rdfs:label" content="STAT induced STAT inhibitor" datatype="xsd:string"></span><span id='am-848' property="oboInOwl:hasExactSynonym" content="Cytokine-inducible SH2 protein" datatype="xsd:string"></span><span id='am-849' property="oboInOwl:hasExactSynonym" content="Suppressor of cytokine signaling" datatype="xsd:string"></span><span id='am-850' property="rdfs:label" content="CIS" datatype="xsd:string"></span><span id='am-851' property="oboInOwl:hasExactSynonym" content="STAT induced STAT inhibitor" datatype="xsd:string"></span><span id='am-852' property="rdfs:label" content="SOCS" datatype="xsd:string"></span><span id='am-853' property="oboInOwl:hasExactSynonym" content="SSI" datatype="xsd:string"></span><span id='am-854' property="rdfs:label" content="SSI" datatype="xsd:string"></span><span id='am-855' property="rdfs:label" content="Cytokine-inducible SH2 protein" datatype="xsd:string"></span><span id='am-856' property="rdfs:label" content="Suppressor of cytokine signaling" datatype="xsd:string"></span>cis</span>-regu- divergence (Figure 2C D). result suggests         latory motifs singleton genes (46). Using compre- close duplicates likely similar      hensive data set  150 known predicted DNA <br /> functions, higher functional load makes complete           binding motifs yeast (28,39), average <br /> compensation likely. overall functional         number different motifs regulating duplicate pair  <br /> load duplicates remains approximately constant            creases signi 65533;?cantly Ka &amp; 0.1 (Figure 4D, dashed line, <br /> Ka &gt; 0.15, higher fraction detectable growth pheno-       Mann 8211;Whitney U test, P = 0.06). divergence <br /> types distances likely decreased ability    distance, average number different motifs dupli- functional compensation duplicates diverge.               cate pair twice number motifs pair <br /> Compensation duplicates quanti 65533;?ed                 yeast singletons (Figure 4D, dashed horizontal line). <br /> presence aggravating interactions duplicate           number regulatory motifs increases                                                                                                                        Nucleic Acids Research, 2014, Vol. 42,  4 2409 <br />  <br />  <br />                                                0.7 <br />                                                                 p=0.01 <br />                                                                                  B                                  0.20 <br />                                                                                                                                       p=0.08 <br />  <br />  <br />               Fraction single deletions <br />  <br />  <br />  <br />  <br />                                                                                  Fraction single deletions <br />                                                  0.6 <br />  <br />  <br />                growth phenotype <br />  <br />  <br />  <br />  <br />                                                                                   growth phenotype <br />                                                                                                                     0.15 <br />                                                  0.5 <br />  <br />                                                  0.4 <br />                                                                                                                     0.10 <br />                                                  0.3 <br />  <br />                                                  0.2 <br />                                                                                                                     0.05 <br />                                                  0.1 <br />  <br />                                                  0.0                                                                0.00 <br />                                                     0.0   0.1        0.2   0.3                                          0.0     0.1            0.2   0.3 <br />                                                                     Ka                                                                     Ka <br />               C                                                                  D                                   1.1 <br />                                                  1.0 <br />               Fraction double deletions <br />  <br />  <br />  <br />  <br />                                                                                  Fraction double deletions <br />                                                                                                                      1.0 <br />                growth phenotype <br />  <br />  <br />  <br />  <br />                                                                                   growth phenotype <br />                                                  0.8                                                                 0.9 <br />  <br />                                                                                                                      0.8 <br />                                                  0.6 <br />                                                                                                                      0.7 <br />                                                  0.4                                                                 0.6 <br />  <br />                                                  0.2 <br />                                                                                                                      0.5 <br />  <br />                                                                                                                      0.4 <br />                                                     0.0   0.1        0.2   0.3                                          0.0     0.1            0.2   0.3 <br />                                                                     Ka                                                                     Ka <br />               E                                                                  F <br />                                                                                  Fraction buffering duplicates <br />               Fraction buffering duplicates <br />  <br />  <br />  <br />  <br />                                                  1.0                                                                1.0 <br />  <br />                                                  0.8                                                                0.8 <br />  <br />                                                  0.6 <br />                                                                                                                     0.6 <br />  <br />                                                  0.4 <br />                                                                                                                     0.4 <br />                                                  0.2 <br />                                                                                                                     0.2 <br />                                                  0.0 <br />                                                     0.0   0.1        0.2   0.3                                        0.0      0.1         0.2       0.3 <br />                                                                     Ka                                                                    Ka <br /> Figure 2. Growth phenotypes individual simultaneous deletion duplicates function sequence divergence (Ka). results  65533;?rst column   C, E) based competition experiments DeLuna et al. (34), second column (B, D, F) synthetic <br /> genetic arrays (SGA) Costanzo et al. (35).   B) Fractions single duplicate deletions signi 65533; growth decrease. (C, D) Fractions simultaneous (double) duplicate deletions signi 65533; growth decrease. different measurement sensitivities studies, different <br /> cutoffs used determine signi 65533; growth decrease: 1% DeLuna et al.   C) 10% Costanzo et al. (B, D); presented results sensitive exact cutoff values  Supplementary Figure <span id='am-91' about='obo:IMR_0001294' typeof='owl:Thing'><span id='am-92' property="rdfs:label" content="Rps7" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="40S ribosomal protein S7" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="RS7_MOUSE" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="40S ribosomal protein S7" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasExactSynonym" content="Rps7" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:id" content="IMR:0001294" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="S7" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasExactSynonym" content="ribosomal protein S7" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="ribosomal protein S7" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="S7" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="UniProt:P62082" datatype="xsd:string"></span>S7</span>). P-values obtained using Fisher 8217;s exact test. (E, F) Fraction paralogs signi 65533; negative epistatic interaction studies DeLuna et al. Costanzo et al., respectively.  65533;?gures error bars <br /> represent SEM. <br />  <br />  <br />  <br /> duplicate highest duplicate                            shared motifs [Supplementary Figure S9A, (48)], <br /> smallest number sensitive conditions (Supplementary                           overlap terms describing biological processes <br /> Figure S8A B). increase complexity du-                         (Figure 4C) overlap cellular localization <br /> plicates regulation Ka &amp; 0.1 con 65533;?rmed sig-                       observed  64258;uorescence-tagging experiments (40) <br /> ni 65533; increase (Mann 8211;Whitney U test, P = 1   10 3)                          decrease (Supplementary Figure S8B). pattern distances number transcription factor                            suggests increase regulatory complexity <br /> mutants (47) affecting duplicate gene expression (Figure                         allows duplicates specialize different biological <br /> 4D, solid line).                                                                 processes preserving common MFs.    total number DNA motifs regulating du-                           ability duplicates partially diverged regulatory <br /> plicates initially increases divergence, fraction                    regions compensate expression <br />  2410 Nucleic Acids Research, 2014, Vol. 42,  4 <br />  <br />  <br />                                                        0.10                                                                       B <br />                                                                                               sensitive duplicate <br />              Fraction conditions significant <br />  <br />  <br />  <br />  <br />                                                                                                                                     Relative difference number                                                                                               sensitive duplicate                                                     1.0 <br />                                                          0.09 <br />  <br />                                                          0.08                                                                                                              0.8 <br />  <br />  <br />  <br />  <br />                                                                                                                                             sensitive conditions <br />                          growth decrease <br />  <br />  <br />                                                          0.07 <br />                                                                                                                                                                            0.6 <br />                                                          0.06 <br />                                                                                                                                                                            0.4 <br />                                                          0.05 <br />  <br />                                                          0.04                            p=2E-4                                                                            0.2 <br />  <br />                                                          0.03 <br />                                                                                                                                                                            0.0 <br />                                                          0.02 <br />                                                                      0.0            0.1        0.2          0.3   0.4   0.5   0.6                                                0.00   0.02   0.04   0.06   0.08   0.10 <br />                                                                                                       Ka                                                                                          Ka <br />              C                                                                      p=0.007 <br />                                                          0.07 <br />  <br />                                                          0.06 <br />  <br />                                                          0.05 <br />  <br />                                                          0.04 <br />                                              Ka/Ks <br />  <br />  <br />  <br />  <br />                                                          0.03 <br />  <br />                                                          0.02 <br />  <br />                                                          0.01 <br />  <br />                                                          0.00 <br />                                                                                te                      te <br />                                                                        plica                    pli <br />                                                                                                      ca <br />                                                                 e   du                    e   du <br />                                                            itiv                       itiv <br />                                                     e    <span id='am-1' about='obo:IMR_0701568' typeof='owl:Thing'><span id='am-2' property="oboInOwl:hasExactSynonym" content="KIRSTEN MURINE SARCOMA VIRUS 2" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="K-RAS4B" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="IMR:0701571" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="NS" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="GTPase KRas, N-terminally processed" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="c-Ki-ras" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="NS3" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="c-K-ras" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="K-Ras 2" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="RASK2" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Ki-Ras" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0701568" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="C-K-RAS" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="GTPase KRas, N-terminally processed" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="ONCOGENE KRAS2" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="KRAS" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="KRAS" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="c-Ki-ras" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="K-RAS2A" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="GTPase KRas" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="KI-RAS" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="C-K-RAS" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="K-Ras 2" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="K-RAS2B" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="KRAS2" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="K-RAS4A" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="C-KRAS" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="GTPase KRas" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="KRAS2" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="K-RAS4A" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="KRAS1" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="RASK_HUMAN" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="c-K-ras" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="UniProt:P01116" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="KIRSTEN MURINE SARCOMA VIRUS 2" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="KI-RAS" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="ONCOGENE KRAS2" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="RASK2" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="KRAS1" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="Ki-Ras" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="NS3" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="K-RAS2B" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="C-KRAS" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="K-RAS4B" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="K-RAS2A" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="v-Ki-ras2 Kirsten rat sarcoma viral oncogene homolog" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="NS" datatype="xsd:string"></span>ns</span>                    e    ns <br />                                            r     es                        sss <br />                    Mo                                                Le <br /> Figure 3. Differences number sensitive conditions duplicates.   average fraction sensitive conditions duplicates higher lower number sensitive conditions pair; Ka values represent sequence divergence duplicates. P-value Mann 8211;Whitney U test. (B) relative difference number sensitive conditions duplicates function initial <br /> divergence; Ka values represent sequence divergence duplicates. relative difference calculated absolute difference number sensitive conditions duplicates normalized total number sensitive conditions pair (Spearman 8217;s r = 0.60, <br /> P = 2   10 3; Pearson 8217;s r = 0.64, P = 7   10 4). (C) average Ka/Ks ratio paralogs largest  sensitive) smallest <br />  sensitive) number conditions signi 65533; growth decrease. Ka/Ks ratios calculated relative orthologous sequences S. bayanus. duplicates Ka &lt; 0.15 considered. P-value Wilcoxon signed rank test. <br />  <br />  <br />  <br />  <br /> changes intact gene previously described                                                                             Finally, patterns diversi 65533;?cation functional <br /> Kafri et al. (39,49).  recent study DeLuna                                                                          compensation described correlate et al. (50) showed deletion duplicate, ex-                                                                       process duplicate loss evolution. investigated pression changes remaining paralog need-                                                                       retention yeast duplicates using complete genomic <br /> based, e. happen primarily corresponding                                                                        sequences seven species: S. cerevisiae, S. paradoxus, <br /> function required. regulatory backup circuits                                                                           S. bayanus, S. castelli, S. mikatae, S. kudriavzevii  cases, enable functional compen-                                                                       S. kluyveri. calculated number remaining dupli- <br /> sation homologs different expression                                                                               cates function sequence divergence (Figure 5, <br /> patterns wild type. Notably, based data                                                                          Supplementary Figure S10A correspond- <br /> recent study Springer et al. (51), measured                                                                          ing relationships individual yeast species). analysis <br /> expression changes yeast genes                                                                               suggests relatively brief initial period high dupli- <br /> genomic copies deleted diploid cells, observed                                                                        cate loss (6) followed long evolutionary period signi 65533; dosage response genes forming                                                                             (Ka &gt; 0.1) average loss rate decreases <br /> recently duplicated pairs (Ka &lt; 0.15, Figure 4E).                                                                          &gt;10-<span id='am-507' about='obo:IMR_0703347' typeof='owl:Thing'><span id='am-508' property="oboInOwl:id" content="IMR:0703347" datatype="xsd:string"></span><span id='am-509' property="oboInOwl:hasExactSynonym" content="Ass-1" datatype="xsd:string"></span><span id='am-510' property="rdfs:label" content="Ass1" datatype="xsd:string"></span><span id='am-511' property="oboInOwl:hasDbXref" content="UniProt:P16460" datatype="xsd:string"></span><span id='am-512' property="oboInOwl:hasExactSynonym" content="Argininosuccinate synthase" datatype="xsd:string"></span><span id='am-513' property="oboInOwl:hasExactSynonym" content="EC 6.3.4.5" datatype="xsd:string"></span><span id='am-514' property="oboInOwl:hasExactSynonym" content="argininosuccinate synthetase 1" datatype="xsd:string"></span><span id='am-515' property="rdfs:label" content="fold" datatype="xsd:string"></span><span id='am-516' property="oboInOwl:hasExactSynonym" content="Ass1" datatype="xsd:string"></span><span id='am-517' property="oboInOwl:hasExactSynonym" content="fold" datatype="xsd:string"></span><span id='am-518' property="rdfs:label" content="Citrulline--aspartate ligase" datatype="xsd:string"></span><span id='am-519' property="rdfs:label" content="Ass-1" datatype="xsd:string"></span><span id='am-520' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-521' property="rdfs:label" content="EC 6.3.4.5" datatype="xsd:string"></span><span id='am-522' property="rdfs:label" content="ASSY_MOUSE" datatype="xsd:string"></span><span id='am-523' property="oboInOwl:hasExactSynonym" content="Citrulline--aspartate ligase" datatype="xsd:string"></span><span id='am-524' property="rdfs:label" content="Argininosuccinate synthase" datatype="xsd:string"></span><span id='am-525' property="rdfs:label" content="argininosuccinate synthetase 1" datatype="xsd:string"></span>fold</span> (red Figure 5). Interestingly, loss rate sig- <br /> suggests genes close duplicates respon-                                                                      ni 65533;?cantly decreases approximately divergence <br /> sive dosage effects.                                                                                                         distance duplicates similar terms <br />                                                                                                                                                                                        Nucleic Acids Research, 2014, Vol. 42,  4 2411 <br />  <br />  <br />                                                                                                                                       B 1.0 <br />                    1.0 <br />  <br />  <br />  <br />  <br />                                                                                                                                  Fraction shared molecular functions <br />                  Fraction pairs sharing EC numbers <br />                                                                0.8                                                                                                           0.9 <br />  <br />  <br />                                                                0.6                                                                                                           0.8 <br />  <br />                                                                0.4 <br />                                                                                                                                                                              0.7 <br />  <br />                                                                0.2 <br />                                                                                                                                                                              0.6 <br />  <br />                                                                0.0 <br />                                                                                                                                                                              0.5 <br />                                                                  0.0         0.2    0.4         0.6         0.8    1.0                                                          0.0       0.2         0.4          0.6         0.8        1.0 <br />                    C 0.8                                                                  Ka                                                                                                                Ka <br />                   Fraction shared biological processes <br />  <br />  <br />  <br />  <br />                                                                                                                          D                                                                 TF binding motifs <br />                                                                0.7                                                                                                                         TF mutations affecting expression <br />                                                                                                                                                                                        p=0.06 <br />  <br />                                                                0.6                                                                                                                    p=0.001 <br />  <br />                                                                                                                                                                               10 <br />                                                                0.5 <br />                                                                                                                          Number unique TF associations <br />  <br />  <br />  <br />                                                                                                                                                                                9 <br />  <br />                                                                0.4                                                                                                             8 <br />                                                                                                                                 duplicate pair <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                7 <br />                                                                0.3 <br />                                                                                                                                                                                6 <br />                                                                  0.0         0.2    0.4        0.6          0.8    1.0 <br />                                                                                                                                                                                5 <br />                                                                                           Ka <br />                                                                                                                                                                                4 <br />             E                                                                                          p=0.003 <br />                                                                0.25 <br />                                                                                                                                                                                3 <br />            Expression responsiveness <br />  <br />  <br />  <br />  <br />                                                                0.20                                                                                                            2 <br />                                                                                                                                                                                 0.0             0.1         0.2          0.3         &gt;   0.4 <br />                                                                0.15                                                                                                                                           Ka <br />  <br />  <br />                                                                0.10 <br />  <br />                                                                0.05 <br />  <br />                                                                0.00 <br />  <br />                                                            -0.05 <br />                                                                        0.0    0.1   0.2    0.3        0.4    0.5   0.6 <br />                                                                                            Ka <br /> Figure 4. Diversi 65533;?cation duplicates function regulation.   Fraction metabolic duplicates sharing EC numbers; conservation EC numbers indicates catalysis identical biochemical reactions. (B) Fraction MF terms shared duplicates. (C) Fraction Biological Process (BP) terms shared duplicates. panels B C considered terms distance corresponding root hierarchy term. (D) Dashed line, average number different transcription factor binding motifs duplicate pair. <br /> Transcription factor (TF) binding motifs compiled studies Kafri et al. (39) Kellis et al. (28). Solid line, average number transcription factor deletions S. cerevisiae signi 65533;?cantly affect expression duplicate genes. data obtained study Hu <br /> et al. (47). comparison average number motifs TF mutants affecting expression random pairs yeast singletons <br /> (horizontal dashed solid lines); P-values calculated using Mann 8211;Whitney U test. (E) average dosage compensation (respon- <br /> siveness) duplicates function sequence divergence (Ka). data average expression responsiveness obtained work Springer et al. (51). study, responsiveness measured diploid yeast strains Log2 ratio (perturbed versus normal) expression <br /> changes remaining gene copy following deletion equivalent gene copy sister chromosome. P-value calculated using Mann 8211;Whitney U test.  65533;?gures error bars represent SEM. <br />  2412 Nucleic Acids Research, 2014, Vol. 42,  4 <br />  <br />  <br />                                        100 <br />                                                                        Ka &lt; 0.1         et al. (35), duplicates annotated redundant                                         80 <br />                                                                                         signi 65533;?cantly different yeast duplicates <br />       Number remaining duplicates <br />  <br />                                                                        Ka &gt; 0.1 <br />        averaged 7 yeast species <br />  <br />                                        60                                               (Mann 8211;Whitney U, P = 0.13 0.35, respectively, <br />                                                                                         Supplementary Figure <span id='am-759' about='obo:IMR_0702942' typeof='owl:Thing'><span id='am-760' property="rdfs:label" content="26S proteasome regulatory subunit S11" datatype="xsd:string"></span><span id='am-761' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 13" datatype="xsd:string"></span><span id='am-762' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN9" datatype="xsd:string"></span><span id='am-763' property="rdfs:label" content="PSD13_HUMAN" datatype="xsd:string"></span><span id='am-764' property="rdfs:label" content="HSPC027" datatype="xsd:string"></span><span id='am-765' property="rdfs:label" content="PROTEASOME 26S SUBUNIT, NON-ATPase, 13" datatype="xsd:string"></span><span id='am-766' property="oboInOwl:hasExactSynonym" content="HSPC027" datatype="xsd:string"></span><span id='am-767' property="oboInOwl:hasExactSynonym" content="p40.5" datatype="xsd:string"></span><span id='am-768' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 13" datatype="xsd:string"></span><span id='am-769' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S11" datatype="xsd:string"></span><span id='am-770' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-771' property="rdfs:label" content="S11" datatype="xsd:string"></span><span id='am-772' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit p40.5" datatype="xsd:string"></span><span id='am-773' property="oboInOwl:hasExactSynonym" content="Rpn9" datatype="xsd:string"></span><span id='am-774' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S SUBUNIT, NON-ATPase, 13" datatype="xsd:string"></span><span id='am-775' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 13" datatype="xsd:string"></span><span id='am-776' property="rdfs:label" content="26S proteasome regulatory subunit RPN9" datatype="xsd:string"></span><span id='am-777' property="oboInOwl:hasExactSynonym" content="PSMD13" datatype="xsd:string"></span><span id='am-778' property="rdfs:label" content="PROTEASE 26S, SUBUNIT 11" datatype="xsd:string"></span><span id='am-779' property="oboInOwl:hasExactSynonym" content="PROTEASE 26S, SUBUNIT 11" datatype="xsd:string"></span><span id='am-780' property="rdfs:label" content="PSMD13" datatype="xsd:string"></span><span id='am-781' property="oboInOwl:hasExactSynonym" content="S11" datatype="xsd:string"></span><span id='am-782' property="rdfs:label" content="p40.5" datatype="xsd:string"></span><span id='am-783' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 13" datatype="xsd:string"></span><span id='am-784' property="oboInOwl:hasDbXref" content="UniProt:Q9UNM6" datatype="xsd:string"></span><span id='am-785' property="rdfs:label" content="26S proteasome regulatory subunit p40.5" datatype="xsd:string"></span><span id='am-786' property="oboInOwl:id" content="IMR:0702942" datatype="xsd:string"></span><span id='am-787' property="rdfs:label" content="Rpn9" datatype="xsd:string"></span>S11</span>). demonstrates  <br />                                        40 <br />                                                                                         yeast duplicates func- <br />                                                                                         tional overlap laboratory conditions, com- <br />                                                                                         pensation properties probably signi 65533;?cantly                                        20 <br />                                                                                         important large natural populations ability <br />                                                                                         purifying selection <span id='am-459' about='obo:IMR_0702026' typeof='owl:Thing'><span id='am-460' property="rdfs:label" content="Phosphatidylcholine 2-acylhydrolase 2A" datatype="xsd:string"></span><span id='am-461' property="oboInOwl:hasExactSynonym" content="Pla2g2a" datatype="xsd:string"></span><span id='am-462' property="rdfs:label" content="EF" datatype="xsd:string"></span><span id='am-463' property="rdfs:label" content="Enhancing factor" datatype="xsd:string"></span><span id='am-464' property="oboInOwl:id" content="IMR:0702026" datatype="xsd:string"></span><span id='am-465' property="rdfs:label" content="PA2GA_MOUSE" datatype="xsd:string"></span><span id='am-466' property="rdfs:label" content="Pla2g2a" datatype="xsd:string"></span><span id='am-467' property="oboInOwl:hasDbXref" content="UniProt:P31482" datatype="xsd:string"></span><span id='am-468' property="oboInOwl:hasExactSynonym" content="Phospholipase A2, membrane associated" datatype="xsd:string"></span><span id='am-469' property="oboInOwl:hasExactSynonym" content="GIIC sPLA2" datatype="xsd:string"></span><span id='am-470' property="oboInOwl:hasExactSynonym" content="EC 3.1.1.4" datatype="xsd:string"></span><span id='am-471' property="oboInOwl:hasExactSynonym" content="Phosphatidylcholine 2-acylhydrolase 2A" datatype="xsd:string"></span><span id='am-472' property="oboInOwl:hasExactSynonym" content="Group IIA phospholipase A2" datatype="xsd:string"></span><span id='am-473' property="oboInOwl:hasExactSynonym" content="modifier of Min1" datatype="xsd:string"></span><span id='am-474' property="rdfs:label" content="EC 3.1.1.4" datatype="xsd:string"></span><span id='am-475' property="rdfs:label" content="GIIC sPLA2" datatype="xsd:string"></span><span id='am-476' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-477' property="rdfs:label" content="Group IIA phospholipase A2" datatype="xsd:string"></span><span id='am-478' property="rdfs:label" content="Phospholipase A2, membrane associated" datatype="xsd:string"></span><span id='am-479' property="oboInOwl:hasExactSynonym" content="phospholipase A2, group IIA (platelets, synovial fluid)" datatype="xsd:string"></span><span id='am-480' property="oboInOwl:hasExactSynonym" content="Enhancing factor" datatype="xsd:string"></span><span id='am-481' property="oboInOwl:hasExactSynonym" content="EF" datatype="xsd:string"></span><span id='am-482' property="rdfs:label" content="modifier of Min1" datatype="xsd:string"></span><span id='am-483' property="rdfs:label" content="phospholipase A2, group IIA (platelets, synovial fluid)" datatype="xsd:string"></span>ef</span> 65533;?ciently prune mutations <br />                                                                                         causing small  65533;?tness decrease. <br />                                        10 <br />                                                                                            likely different factors contribute                                         8 <br />                                                                                         relative paucity functional compensation                                         6                                               paralogs small divergence distances. signi 65533;                                          0.0   0.1   0.2   0.3   0.4     0.5      0.6   fraction duplications likely  65533;?xed owing                                                                                         dosage effects (17), functional compensation <br />                                                            Ka <br />                                                                                         duplicates context natural popula- <br /> Figure 5. average number duplicates retained genomes                   tions unlikely. example, lack signi 65533; com- <br /> yeast species function duplicates divergence Ka, number                 pensation histone pairs, HTA1-HTA2 nonsynonymous substitutions site. number remaining <br /> duplicates averaged genomes seven yeast species: S.                     HHT1-<span id='am-422' about='obo:IMR_0701596' typeof='owl:Thing'><span id='am-423' property="rdfs:label" content="ORW2" datatype="xsd:string"></span><span id='am-424' property="oboInOwl:hasExactSynonym" content="activin A receptor type II-like 1" datatype="xsd:string"></span><span id='am-425' property="rdfs:label" content="activin A receptor type II-like 1" datatype="xsd:string"></span><span id='am-426' property="rdfs:label" content="ACVRLK1" datatype="xsd:string"></span><span id='am-427' property="oboInOwl:hasExactSynonym" content="HHT2" datatype="xsd:string"></span><span id='am-428' property="oboInOwl:hasExactSynonym" content="EC 2.7.11.30" datatype="xsd:string"></span><span id='am-429' property="oboInOwl:id" content="IMR:0701596" datatype="xsd:string"></span><span id='am-430' property="oboInOwl:hasExactSynonym" content="ALK-1" datatype="xsd:string"></span><span id='am-431' property="rdfs:label" content="ALK-1" datatype="xsd:string"></span><span id='am-432' property="oboInOwl:hasExactSynonym" content="ACVRL1" datatype="xsd:string"></span><span id='am-433' property="rdfs:label" content="HHT2" datatype="xsd:string"></span><span id='am-434' property="oboInOwl:hasExactSynonym" content="ORW2" datatype="xsd:string"></span><span id='am-435' property="rdfs:label" content="EC 2.7.11.30" datatype="xsd:string"></span><span id='am-436' property="rdfs:label" content="ALK1" datatype="xsd:string"></span><span id='am-437' property="oboInOwl:hasExactSynonym" content="HHT" datatype="xsd:string"></span><span id='am-438' property="rdfs:label" content="Activin receptor-like kinase 1" datatype="xsd:string"></span><span id='am-439' property="oboInOwl:hasDbXref" content="UniProt:P37023" datatype="xsd:string"></span><span id='am-440' property="rdfs:label" content="TGF-B superfamily receptor type I" datatype="xsd:string"></span><span id='am-441' property="oboInOwl:hasExactSynonym" content="ACVRLK1" datatype="xsd:string"></span><span id='am-442' property="oboInOwl:hasExactSynonym" content="ACTIVIN A RECEPTOR, TYPE II-LIKE KINASE 1" datatype="xsd:string"></span><span id='am-443' property="rdfs:label" content="ACVL1_HUMAN" datatype="xsd:string"></span><span id='am-444' property="rdfs:label" content="TSR-I" datatype="xsd:string"></span><span id='am-445' property="rdfs:label" content="ACTIVIN A RECEPTOR, TYPE II-LIKE KINASE 1" datatype="xsd:string"></span><span id='am-446' property="rdfs:label" content="SKR3" datatype="xsd:string"></span><span id='am-447' property="oboInOwl:hasExactSynonym" content="Activin receptor-like kinase 1" datatype="xsd:string"></span><span id='am-448' property="oboInOwl:hasExactSynonym" content="ACTIVIN A RECEPTOR, TYPE II-LIKE 1" datatype="xsd:string"></span><span id='am-449' property="rdfs:label" content="Serine/threonine-protein kinase receptor R3" datatype="xsd:string"></span><span id='am-450' property="oboInOwl:hasExactSynonym" content="ALK1" datatype="xsd:string"></span><span id='am-451' property="oboInOwl:hasExactSynonym" content="SKR3" datatype="xsd:string"></span><span id='am-452' property="oboInOwl:hasExactSynonym" content="Serine/threonine-protein kinase receptor R3" datatype="xsd:string"></span><span id='am-453' property="rdfs:label" content="ACVRL1" datatype="xsd:string"></span><span id='am-454' property="oboInOwl:hasExactSynonym" content="TGF-B superfamily receptor type I" datatype="xsd:string"></span><span id='am-455' property="oboInOwl:hasExactSynonym" content="TSR-I" datatype="xsd:string"></span><span id='am-456' property="rdfs:label" content="HHT" datatype="xsd:string"></span><span id='am-457' property="rdfs:label" content="ACTIVIN A RECEPTOR, TYPE II-LIKE 1" datatype="xsd:string"></span><span id='am-458' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>HHT2</span>, likely consequence role <br /> cerevisiae, S. paradoxus, S. bayanus, S. castelli, S. mikatae, S.                       maintaining proper histone levels yeast cells. Gene <br /> kudriavzevii S. kluyveri. Supplementary Figure <span id='am-664' about='obo:IMR_0702948' typeof='owl:Thing'><span id='am-665' property="oboInOwl:hasExactSynonym" content="p44S10" datatype="xsd:string"></span><span id='am-666' property="rdfs:label" content="26S proteasome regulatory subunit S10" datatype="xsd:string"></span><span id='am-667' property="rdfs:label" content="PFAAP4" datatype="xsd:string"></span><span id='am-668' property="oboInOwl:hasExactSynonym" content="p42A" datatype="xsd:string"></span><span id='am-669' property="oboInOwl:hasExactSynonym" content="PFAAP4" datatype="xsd:string"></span><span id='am-670' property="rdfs:label" content="Rpn7" datatype="xsd:string"></span><span id='am-671' property="oboInOwl:hasDbXref" content="UniProt:Q15008" datatype="xsd:string"></span><span id='am-672' property="rdfs:label" content="SGA-113M" datatype="xsd:string"></span><span id='am-673' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 6" datatype="xsd:string"></span><span id='am-674' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 6" datatype="xsd:string"></span><span id='am-675' property="oboInOwl:hasExactSynonym" content="Phosphonoformate immuno-associated protein 4" datatype="xsd:string"></span><span id='am-676' property="oboInOwl:hasExactSynonym" content="PSMD6" datatype="xsd:string"></span><span id='am-677' property="rdfs:label" content="S10" datatype="xsd:string"></span><span id='am-678' property="rdfs:label" content="p42A" datatype="xsd:string"></span><span id='am-679' property="oboInOwl:hasExactSynonym" content="KIAA0107" datatype="xsd:string"></span><span id='am-680' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S10" datatype="xsd:string"></span><span id='am-681' property="rdfs:label" content="p44S10" datatype="xsd:string"></span><span id='am-682' property="rdfs:label" content="PSMD6_HUMAN" datatype="xsd:string"></span><span id='am-683' property="rdfs:label" content="PSMD6" datatype="xsd:string"></span><span id='am-684' property="rdfs:label" content="KIAA0107" datatype="xsd:string"></span><span id='am-685' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 6" datatype="xsd:string"></span><span id='am-686' property="oboInOwl:hasExactSynonym" content="Rpn7" datatype="xsd:string"></span><span id='am-687' property="rdfs:label" content="Proteasome regulatory particle subunit p44S10" datatype="xsd:string"></span><span id='am-688' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 6" datatype="xsd:string"></span><span id='am-689' property="oboInOwl:hasExactSynonym" content="Proteasome regulatory particle subunit p44S10" datatype="xsd:string"></span><span id='am-690' property="rdfs:label" content="Phosphonoformate immuno-associated protein 4" datatype="xsd:string"></span><span id='am-691' property="rdfs:label" content="26S proteasome regulatory subunit RPN7" datatype="xsd:string"></span><span id='am-692' property="rdfs:label" content="Breast cancer-associated protein SGA-113M" datatype="xsd:string"></span><span id='am-693' property="oboInOwl:id" content="IMR:0702948" datatype="xsd:string"></span><span id='am-694' property="oboInOwl:hasExactSynonym" content="S10" datatype="xsd:string"></span><span id='am-695' property="oboInOwl:hasExactSynonym" content="Breast cancer-associated protein SGA-113M" datatype="xsd:string"></span><span id='am-696' property="oboInOwl:hasExactSynonym" content="SGA-113M" datatype="xsd:string"></span><span id='am-697' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN7" datatype="xsd:string"></span><span id='am-698' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>S10</span>                     dosage explain inability duplicates number remaining duplicates individual species                    backup  unlikely explan- number remaining duplicates function Ks. rate duplicate loss evolution &gt;10 times lower distances <br />                                                                                         ation. showed duplicate pairs corresponding Ka &gt; 0.1 compared distances Ka &lt; 0.1.                      high CAI (Supplementary Figure S6A) pairs forming  65533;?gure, error bars represent SEM.                                             known protein complexes (Supplementary Figure S6B)                                                                                         removed analysis, patterns functional com- <br />                                                                                         pensation remain similar. Notably, genes high CAI functional load (Figure 2B) regu-                               associated higher frequencies latory complexity signi 65533;?cantly increases (Figure 5D).                             interlocus gene conversion (IGC) (53,54). IGC <br /> likely duplicates surviving initial loss stage                             slow rate duplicates sequence divergence <br /> develop independent functionalities preserved                                   (55), analyses based WGDs evidence long times genomes yeast species.                                         IGC [using data recently reported Casola et al. (56)] <br />                                                                                         revealed essentially identical compensation patterns <br />                                                                                         (Supplementary Figure <span id='am-699' about='obo:IMR_0702949' typeof='owl:Thing'><span id='am-700' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S12" datatype="xsd:string"></span><span id='am-701' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S SUBUNIT, NON-ATPase, 7" datatype="xsd:string"></span><span id='am-702' property="oboInOwl:hasExactSynonym" content="S12" datatype="xsd:string"></span><span id='am-703' property="rdfs:label" content="PROTEASOME 26S SUBUNIT, NON-ATPase, 7" datatype="xsd:string"></span><span id='am-704' property="rdfs:label" content="P40" datatype="xsd:string"></span><span id='am-705' property="rdfs:label" content="26S proteasome regulatory subunit RPN8" datatype="xsd:string"></span><span id='am-706' property="rdfs:label" content="PSMD7" datatype="xsd:string"></span><span id='am-707' property="rdfs:label" content="Rpn8" datatype="xsd:string"></span><span id='am-708' property="oboInOwl:hasDbXref" content="UniProt:P51665" datatype="xsd:string"></span><span id='am-709' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN8" datatype="xsd:string"></span><span id='am-710' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-711' property="oboInOwl:hasExactSynonym" content="MOV34" datatype="xsd:string"></span><span id='am-712' property="rdfs:label" content="MOV34" datatype="xsd:string"></span><span id='am-713' property="rdfs:label" content="S12" datatype="xsd:string"></span><span id='am-714' property="oboInOwl:hasExactSynonym" content="MOV34L" datatype="xsd:string"></span><span id='am-715' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 7" datatype="xsd:string"></span><span id='am-716' property="rdfs:label" content="MOV34L" datatype="xsd:string"></span><span id='am-717' property="oboInOwl:hasExactSynonym" content="Mov34 protein homolog" datatype="xsd:string"></span><span id='am-718' property="rdfs:label" content="PSMD7 MOLONEY LEUKEMIA VIRUS 34 PROVIRAL INTEGRATION SITE, MOUSE, HOMOLOG" datatype="xsd:string"></span><span id='am-719' property="oboInOwl:hasExactSynonym" content="Proteasome subunit p40" datatype="xsd:string"></span><span id='am-720' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 7" datatype="xsd:string"></span><span id='am-721' property="oboInOwl:id" content="IMR:0702949" datatype="xsd:string"></span><span id='am-722' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 7" datatype="xsd:string"></span><span id='am-723' property="oboInOwl:hasExactSynonym" content="P40" datatype="xsd:string"></span><span id='am-724' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 7" datatype="xsd:string"></span><span id='am-725' property="rdfs:label" content="Proteasome subunit p40" datatype="xsd:string"></span><span id='am-726' property="oboInOwl:hasExactSynonym" content="PSMD7" datatype="xsd:string"></span><span id='am-727' property="rdfs:label" content="PSD7_HUMAN" datatype="xsd:string"></span><span id='am-728' property="rdfs:label" content="26S proteasome regulatory subunit S12" datatype="xsd:string"></span><span id='am-729' property="oboInOwl:hasExactSynonym" content="PSMD7 MOLONEY LEUKEMIA VIRUS 34 PROVIRAL INTEGRATION SITE, MOUSE, HOMOLOG" datatype="xsd:string"></span><span id='am-730' property="rdfs:label" content="Mov34 protein homolog" datatype="xsd:string"></span><span id='am-731' property="oboInOwl:hasExactSynonym" content="Rpn8" datatype="xsd:string"></span>S12</span>). <br /> DISCUSSION                                                                                 Close duplicates likely compensate                                                                                         probably owing aforementioned dichot- present study, analyzed genetic robustness                                omy functional loads (Figure 3A B). duplicates context quantitative growth pheno-                              close duplicates classi 65533;?ed, based activity <br /> types sensitivities gene deletions multiple envir-                            breadth expression, major minor func- <br /> onmental conditions. robustness important                                   tional isoforms. example, glyceraldehyde-3-phos- <br /> understanding buffering deleterious mutations                                 phate <span id='am-788' about='obo:IMR_0010047' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0000952'><span id='am-789' property="rdfs:label" content="dehydrogenase" datatype="xsd:string"></span><span id='am-790' property="oboInOwl:id" content="IMR:0010047" datatype="xsd:string"></span><span id='am-791' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>dehydrogenase</span> TDH1 active various stress <br /> large natural biological populations. results demon-                                conditions, isoenzyme TDH2 used primarily <br /> strate  contrary commonly held view, close gene                                 exponential growth (57). Similarly, <span id='am-56' about='obo:IMR_0702634' typeof='owl:Thing'><span id='am-57' property="rdfs:label" content="CEP80" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="UBA80" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Ubiquitin-40S ribosomal protein S27a" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="UBIQUITIN A-80-RESIDUE RIBOSOMAL PROTEIN FUSION PRODUCT" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="CEP80" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="ribosomal protein S27a" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="HUBCEP80" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="Uba80" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="UniProt:P62979" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="HUMAN UBIQUITIN CARBOXYL EXTENSION PROTEIN, 80-RESIDUE" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="40S ribosomal protein S27a" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0702634" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="UBCEP1" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="RPS27A" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="UBCEP80" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="Ubiquitin-40S ribosomal protein S27a" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="HUMAN UBIQUITIN CARBOXYL EXTENSION PROTEIN, 80-RESIDUE" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="UBCEP1" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Ubiquitin carboxyl extension protein 80" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="UBIQUITIN CARBOXYL EXTENSION PROTEIN 1" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="ribosomal protein S27a" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="UBIQUITIN CARBOXYL EXTENSION PROTEIN 1" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Ubiquitin" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasExactSynonym" content="Ubiquitin carboxyl extension protein 80" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="Ubiquitin" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="UBIQUITIN A-80-RESIDUE RIBOSOMAL PROTEIN FUSION PRODUCT" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="Uba80" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="HUBCEP80" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="40S ribosomal protein S27a" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="RS27A_HUMAN" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="UBA80" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="RPS27A" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="UBCEP80" datatype="xsd:string"></span>ubiquitin</span> <br /> duplicates unlikely provide high level backup                            conjugating enzyme <span id='am-484' about='obo:IMR_0700799' typeof='owl:Thing'><span id='am-485' property="oboInOwl:hasExactSynonym" content="ube2d2" datatype="xsd:string"></span><span id='am-486' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-487' property="rdfs:label" content="ubc4" datatype="xsd:string"></span><span id='am-488' property="rdfs:label" content="ube2d2" datatype="xsd:string"></span><span id='am-489' property="rdfs:label" content="Ubiquitin-conjugating enzyme E2 D2" datatype="xsd:string"></span><span id='am-490' property="rdfs:label" content="Ubiquitin carrier protein D2" datatype="xsd:string"></span><span id='am-491' property="oboInOwl:hasExactSynonym" content="Ubiquitin-conjugating enzyme E2 D2" datatype="xsd:string"></span><span id='am-492' property="oboInOwl:hasExactSynonym" content="Ubiquitin-protein ligase D2" datatype="xsd:string"></span><span id='am-493' property="rdfs:label" content="Ubiquitin-protein ligase D2" datatype="xsd:string"></span><span id='am-494' property="rdfs:label" content="xUBC4" datatype="xsd:string"></span><span id='am-495' property="oboInOwl:hasExactSynonym" content="Ubiquitin carrier protein D2" datatype="xsd:string"></span><span id='am-496' property="oboInOwl:hasExactSynonym" content="Ubiquitin carrier protein 4" datatype="xsd:string"></span><span id='am-497' property="rdfs:label" content="ube2d3.1" datatype="xsd:string"></span><span id='am-498' property="oboInOwl:hasExactSynonym" content="ube2d3.1" datatype="xsd:string"></span><span id='am-499' property="oboInOwl:hasDbXref" content="UniProt:P62840" datatype="xsd:string"></span><span id='am-500' property="rdfs:label" content="UB2D2_XENLA" datatype="xsd:string"></span><span id='am-501' property="oboInOwl:hasExactSynonym" content="ubc4" datatype="xsd:string"></span><span id='am-502' property="oboInOwl:hasExactSynonym" content="xUBC4" datatype="xsd:string"></span><span id='am-503' property="rdfs:label" content="Ubiquitin carrier protein 4" datatype="xsd:string"></span><span id='am-504' property="rdfs:label" content="EC 6.3.2.19" datatype="xsd:string"></span><span id='am-505' property="oboInOwl:hasExactSynonym" content="EC 6.3.2.19" datatype="xsd:string"></span><span id='am-506' property="oboInOwl:id" content="IMR:0700799" datatype="xsd:string"></span>UBC4</span> expressed exponen- context large natural populations. Consequently,                              tial growth, duplicate UBC5 active sta- unlikely duplicates  65533;?xed natural popu-                              tionary phase (58). difference functional load <br /> lations speci 65533;?cally selection robustness.                                    close yeast duplicates consistent    analysis suggests duplicate redundancies                               asymmetric partition functions, interactions gene <br /> described genomics databases, frequently observed                                expression, observed close duplicates  laboratory experiments, considered                                    ganisms, example, Arabidopsis Human (45,59,60). <br /> caution, respect functions                                    suggests duplicate-dependent compensation natural biological populations. investigate <span id='am-940' about='obo:IMR_0002979' typeof='owl:Thing'><span id='am-941' property="rdfs:label" content="Ets94F" datatype="xsd:string"></span><span id='am-942' property="rdfs:label" content="ETS-like protein pointed, isoform P2/D" datatype="xsd:string"></span><span id='am-943' property="oboInOwl:hasExactSynonym" content="ptd" datatype="xsd:string"></span><span id='am-944' property="oboInOwl:hasDbXref" content="UniProt:P51023" datatype="xsd:string"></span><span id='am-945' property="oboInOwl:hasExactSynonym" content="PNTP2" datatype="xsd:string"></span><span id='am-946' property="oboInOwl:hasExactSynonym" content="D-Ets-2" datatype="xsd:string"></span><span id='am-947' property="oboInOwl:hasExactSynonym" content="EY3-1" datatype="xsd:string"></span><span id='am-948' property="rdfs:label" content="3520" datatype="xsd:string"></span><span id='am-949' property="rdfs:label" content="DMPOINT1A" datatype="xsd:string"></span><span id='am-950' property="oboInOwl:hasExactSynonym" content="PNTP1" datatype="xsd:string"></span><span id='am-951' property="rdfs:label" content="PNTP2" datatype="xsd:string"></span><span id='am-952' property="rdfs:label" content="pointedP2" datatype="xsd:string"></span><span id='am-953' property="rdfs:label" content="l(3)07825" datatype="xsd:string"></span><span id='am-954' property="oboInOwl:hasExactSynonym" content="Ets at 58AB" datatype="xsd:string"></span><span id='am-955' property="rdfs:label" content="ets94F" datatype="xsd:string"></span><span id='am-956' property="oboInOwl:hasExactSynonym" content="E(E2F)3D" datatype="xsd:string"></span><span id='am-957' property="rdfs:label" content="pointed-P2" datatype="xsd:string"></span><span id='am-958' property="oboInOwl:hasExactSynonym" content="DMPOINT1A" datatype="xsd:string"></span><span id='am-959' property="rdfs:label" content="ETS2" datatype="xsd:string"></span><span id='am-960' property="rdfs:label" content="Ets protein Pointed P2" datatype="xsd:string"></span><span id='am-961' property="rdfs:label" content="Ets2" datatype="xsd:string"></span><span id='am-962' property="oboInOwl:hasExactSynonym" content="0123/09" datatype="xsd:string"></span><span id='am-963' property="oboInOwl:hasExactSynonym" content="l(3)07825" datatype="xsd:string"></span><span id='am-964' property="oboInOwl:hasExactSynonym" content="Ets protein Pointed P2" datatype="xsd:string"></span><span id='am-965' property="oboInOwl:hasExactSynonym" content="CG17077" datatype="xsd:string"></span><span id='am-966' property="rdfs:label" content="EK3-2" datatype="xsd:string"></span><span id='am-967' property="oboInOwl:hasExactSynonym" content="0608/07" datatype="xsd:string"></span><span id='am-968' property="oboInOwl:hasExactSynonym" content="ETS-like protein pointed, isoform P2/D" datatype="xsd:string"></span><span id='am-969' property="oboInOwl:hasExactSynonym" content="D-ets-2" datatype="xsd:string"></span><span id='am-970' property="oboInOwl:hasExactSynonym" content="D-ETS-2" datatype="xsd:string"></span><span id='am-971' property="rdfs:label" content="l(3)j1B7" datatype="xsd:string"></span><span id='am-972' property="oboInOwl:hasExactSynonym" content="pointed" datatype="xsd:string"></span><span id='am-973' property="rdfs:label" content="D-ETS-2" datatype="xsd:string"></span><span id='am-974' property="rdfs:label" content="0608/07" datatype="xsd:string"></span><span id='am-975' property="rdfs:label" content="pointed-P1" datatype="xsd:string"></span><span id='am-976' property="rdfs:label" content="CG17077" datatype="xsd:string"></span><span id='am-977' property="oboInOwl:hasExactSynonym" content="pointed-P2" datatype="xsd:string"></span><span id='am-978' property="oboInOwl:hasExactSynonym" content="POINT" datatype="xsd:string"></span><span id='am-979' property="rdfs:label" content="ptd" datatype="xsd:string"></span><span id='am-980' property="oboInOwl:hasExactSynonym" content="Ets94F" datatype="xsd:string"></span><span id='am-981' property="rdfs:label" content="Ets58AB" datatype="xsd:string"></span><span id='am-982' property="oboInOwl:hasExactSynonym" content="pnt" datatype="xsd:string"></span><span id='am-983' property="rdfs:label" content="PNTP1" datatype="xsd:string"></span><span id='am-984' property="rdfs:label" content="pointed" datatype="xsd:string"></span><span id='am-985' property="rdfs:label" content="EY3-1" datatype="xsd:string"></span><span id='am-986' property="rdfs:label" content="0998/12" datatype="xsd:string"></span><span id='am-987' property="rdfs:label" content="pointed-RC" datatype="xsd:string"></span><span id='am-988' property="rdfs:label" content="D-Ets-2" datatype="xsd:string"></span><span id='am-989' property="oboInOwl:hasExactSynonym" content="ets94F" datatype="xsd:string"></span><span id='am-990' property="rdfs:label" content="pnt" datatype="xsd:string"></span><span id='am-991' property="rdfs:label" content="E(E2F)3D" datatype="xsd:string"></span><span id='am-992' property="rdfs:label" content="PNT2_DROME" datatype="xsd:string"></span><span id='am-993' property="oboInOwl:hasExactSynonym" content="Ets2" datatype="xsd:string"></span><span id='am-994' property="obo:IAO_0000115" content="pnt gene transcription generates alternatively spliced mRNA isoforms that encode two proteins, PntP1 and PntP2.
PntP2 is the homolog of the human Ets-1 transcription factor and functions as a transcriptional activator when phosphorylated by MAPK." datatype="xsd:string"></span><span id='am-995' property="rdfs:label" content="0123/09" datatype="xsd:string"></span><span id='am-996' property="oboInOwl:hasExactSynonym" content="0998/12" datatype="xsd:string"></span><span id='am-997' property="oboInOwl:hasExactSynonym" content="pointed-P1" datatype="xsd:string"></span><span id='am-998' property="oboInOwl:hasExactSynonym" content="l(3)j1B7" datatype="xsd:string"></span><span id='am-999' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1000' property="oboInOwl:hasExactSynonym" content="pointedP2" datatype="xsd:string"></span><span id='am-1001' property="rdfs:label" content="POINT" datatype="xsd:string"></span><span id='am-1002' property="oboInOwl:id" content="IMR:0002979" datatype="xsd:string"></span><span id='am-1003' property="oboInOwl:hasExactSynonym" content="Ets58AB" datatype="xsd:string"></span><span id='am-1004' property="oboInOwl:hasExactSynonym" content="ETS2" datatype="xsd:string"></span><span id='am-1005' property="oboInOwl:hasExactSynonym" content="pointed-RC" datatype="xsd:string"></span><span id='am-1006' property="rdfs:label" content="D-ets-2" datatype="xsd:string"></span><span id='am-1007' property="oboInOwl:hasExactSynonym" content="3520" datatype="xsd:string"></span><span id='am-1008' property="oboInOwl:hasExactSynonym" content="EK3-2" datatype="xsd:string"></span><span id='am-1009' property="rdfs:label" content="Ets at 58AB" datatype="xsd:string"></span>point</span>                               context natural populations limited  analyzed set, compiled Kafri et al. (52),                              species  112 yeast duplicates reported partially                                  analysis suggests typical lifecycle gene <br /> redundant research publications. duplicates                                    duplicates yeast consists distinct evolutionary described redundant based func-                                   stages (11,12).  65533;?rst stage  duplicate distances cor- <br /> tional overlap compensatory interactions observed                                   responding Ka &lt; 0.05), duplicates tend high small-scale experimental studies. Interestingly, based                               overall functional loads signi 65533; asymmetry number conditions quantitative growth                                    number sensitive conditions; factors make <br /> phenotypes study Hillenmeyer et al. (33),                                   complete compensation unlikely. high functional load quantitative growth measurements Costanzo                                    close duplicates suggests adaptive selection plays <br />                                                                                   Nucleic Acids Research, 2014, Vol. 42,  4 2413 <br />  <br />  important role  65533;?xation. second stage                 10. Ohno,S. (1970) Evolution Gene Duplication. Springer-Verlag, <br /> (0.05 &lt; Ka &lt; 0.25), duplicates diverge                     Berlin, New York. <br />                                                                         11. Conant,G.C. Wolfe,K.H. (2008) Turning hobby job: <br /> overall functional load usually decreases.                         duplicated genes  65533;?nd new functions. <span id='am-214' about='obo:IMR_0701753' typeof='owl:Thing'><span id='am-215' property="rdfs:label" content="NaN" datatype="xsd:string"></span><span id='am-216' property="rdfs:label" content="SNS2" datatype="xsd:string"></span><span id='am-217' property="rdfs:label" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-218' property="rdfs:label" content="Sns2" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasExactSynonym" content="Nan" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasExactSynonym" content="Sns2" datatype="xsd:string"></span><span id='am-221' property="oboInOwl:hasDbXref" content="UniProt:Q9R053" datatype="xsd:string"></span><span id='am-222' property="rdfs:label" content="Nan" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasExactSynonym" content="NaN" datatype="xsd:string"></span><span id='am-224' property="rdfs:label" content="NaT" datatype="xsd:string"></span><span id='am-225' property="oboInOwl:hasExactSynonym" content="SNS2" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-227' property="rdfs:label" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:id" content="IMR:0701753" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasExactSynonym" content="NaT" datatype="xsd:string"></span><span id='am-230' property="rdfs:label" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-231' property="rdfs:label" content="Nat" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasExactSynonym" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasExactSynonym" content="NSS2" datatype="xsd:string"></span><span id='am-235' property="rdfs:label" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasExactSynonym" content="Nat" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasExactSynonym" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-238' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasExactSynonym" content="Scn11a" datatype="xsd:string"></span><span id='am-240' property="rdfs:label" content="SCNBA_MOUSE" datatype="xsd:string"></span><span id='am-241' property="rdfs:label" content="NSS2" datatype="xsd:string"></span><span id='am-242' property="rdfs:label" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-243' property="rdfs:label" content="Scn11a" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasExactSynonym" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span>Nat</span>. Rev. Genet., 9, <br /> happen, example, relaxation environmen-                   938 8211;950. <br /> <span id='am-276' about='obo:IMR_0200565' typeof='owl:Thing'><span id='am-277' property="rdfs:label" content="TALH" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasExactSynonym" content="TALDOR" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:id" content="IMR:0200565" datatype="xsd:string"></span><span id='am-280' property="oboInOwl:hasExactSynonym" content="TALDO1" datatype="xsd:string"></span><span id='am-281' property="oboInOwl:hasExactSynonym" content="TALDO" datatype="xsd:string"></span><span id='am-282' property="oboInOwl:hasExactSynonym" content="TAL" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasExactSynonym" content="TALH" datatype="xsd:string"></span><span id='am-284' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-285' property="rdfs:label" content="transaldolase 1" datatype="xsd:string"></span><span id='am-286' property="rdfs:label" content="TAL-H" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:hasExactSynonym" content="Transaldolase" datatype="xsd:string"></span><span id='am-288' property="rdfs:label" content="TALDO" datatype="xsd:string"></span><span id='am-289' property="oboInOwl:hasAlternativeId" content="IMR:0702871" datatype="xsd:string"></span><span id='am-290' property="oboInOwl:hasExactSynonym" content="transaldolase 1" datatype="xsd:string"></span><span id='am-291' property="oboInOwl:hasExactSynonym" content="TAL-H" datatype="xsd:string"></span><span id='am-292' property="rdfs:label" content="TALDO1" datatype="xsd:string"></span><span id='am-293' property="rdfs:label" content="TALDO_HUMAN" datatype="xsd:string"></span><span id='am-294' property="oboInOwl:hasExactSynonym" content="EC 2.2.1.2" datatype="xsd:string"></span><span id='am-295' property="rdfs:label" content="TALDOR" datatype="xsd:string"></span><span id='am-296' property="rdfs:label" content="Transaldolase" datatype="xsd:string"></span><span id='am-297' property="rdfs:label" content="TAL" datatype="xsd:string"></span><span id='am-298' property="rdfs:label" content="EC 2.2.1.2" datatype="xsd:string"></span><span id='am-299' property="oboInOwl:hasDbXref" content="UniProt:P37837" datatype="xsd:string"></span>tal</span> conditions, facilitated original duplicate                12. Innan,H. Kondrashov,F. (2010) evolution gene <br />  65533;?xation. vast majority duplicates, likely                        duplications: classifying distinguishing models. <span id='am-245' about='obo:IMR_0701753' typeof='owl:Thing'><span id='am-246' property="rdfs:label" content="NaN" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Sns2" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasExactSynonym" content="NSS2" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:id" content="IMR:0701753" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-251' property="rdfs:label" content="SCNBA_MOUSE" datatype="xsd:string"></span><span id='am-252' property="rdfs:label" content="Scn11a" datatype="xsd:string"></span><span id='am-253' property="rdfs:label" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-254' property="rdfs:label" content="Nan" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasExactSynonym" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-256' property="rdfs:label" content="NaT" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasDbXref" content="UniProt:Q9R053" datatype="xsd:string"></span><span id='am-259' property="rdfs:label" content="Sodium channel protein type XI subunit alpha" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasExactSynonym" content="SNS2" datatype="xsd:string"></span><span id='am-261' property="oboInOwl:hasExactSynonym" content="NaT" datatype="xsd:string"></span><span id='am-262' property="oboInOwl:hasExactSynonym" content="Sns2" datatype="xsd:string"></span><span id='am-263' property="rdfs:label" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-264' property="rdfs:label" content="Nat" datatype="xsd:string"></span><span id='am-265' property="rdfs:label" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-266' property="rdfs:label" content="sodium channel, voltage-gated, type XI, alpha" datatype="xsd:string"></span><span id='am-267' property="oboInOwl:hasExactSynonym" content="Scn11a" datatype="xsd:string"></span><span id='am-268' property="oboInOwl:hasExactSynonym" content="Sensory neuron sodium channel 2" datatype="xsd:string"></span><span id='am-269' property="oboInOwl:hasExactSynonym" content="Nan" datatype="xsd:string"></span><span id='am-270' property="oboInOwl:hasExactSynonym" content="Nat" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasExactSynonym" content="Voltage-gated sodium channel subunit alpha Nav1.9" datatype="xsd:string"></span><span id='am-272' property="oboInOwl:hasExactSynonym" content="Sodium channel protein type 11 subunit alpha" datatype="xsd:string"></span><span id='am-273' property="oboInOwl:hasExactSynonym" content="NaN" datatype="xsd:string"></span><span id='am-274' property="rdfs:label" content="SNS2" datatype="xsd:string"></span><span id='am-275' property="rdfs:label" content="NSS2" datatype="xsd:string"></span>Nat</span>. <br />                                                                             Rev. Genet., 11, 97 8211;108. <br /> paralogs relatively smaller functional loads                       13. Nadeau,J.H. Sankoff,D. (1997) Comparable rates gene <br /> (Figure 3C), lost stage (Figure 5). Gene pairs                  loss functional divergence genome duplications early survive period high duplicate loss display                 vertebrate evolution. Genetics, 147, 1259 8211;1266. <br /> balanced functional loads complex regulation;                 14. Sidow  (1996) Genome duplications evolution early <br /> gene pairs usually retained long evolutionary times                 vertebrates. Curr. Opin. Genet. <span id='am-1122' about='obo:IMR_0002991' typeof='owl:Thing'><span id='am-1123' property="oboInOwl:hasExactSynonym" content="FGFR" datatype="xsd:string"></span><span id='am-1124' property="oboInOwl:hasExactSynonym" content="dev" datatype="xsd:string"></span><span id='am-1125' property="oboInOwl:hasExactSynonym" content="FGFR1" datatype="xsd:string"></span><span id='am-1126' property="rdfs:label" content="DFR2" datatype="xsd:string"></span><span id='am-1127' property="rdfs:label" content="D-FGFR" datatype="xsd:string"></span><span id='am-1128' property="rdfs:label" content="Dfr-2" datatype="xsd:string"></span><span id='am-1129' property="rdfs:label" content="0844/01" datatype="xsd:string"></span><span id='am-1130' property="oboInOwl:hasExactSynonym" content="DFR2" datatype="xsd:string"></span><span id='am-1131' property="oboInOwl:hasExactSynonym" content="Protein breathless" datatype="xsd:string"></span><span id='am-1132' property="oboInOwl:id" content="IMR:0002991" datatype="xsd:string"></span><span id='am-1133' property="oboInOwl:hasExactSynonym" content="DFGF-R1" datatype="xsd:string"></span><span id='am-1134' property="rdfs:label" content="Dtk2" datatype="xsd:string"></span><span id='am-1135' property="oboInOwl:hasExactSynonym" content="Dfr-2" datatype="xsd:string"></span><span id='am-1136' property="oboInOwl:hasExactSynonym" content="Fgf-r" datatype="xsd:string"></span><span id='am-1137' property="rdfs:label" content="Fgf-r" datatype="xsd:string"></span><span id='am-1138' property="oboInOwl:hasExactSynonym" content="FR2" datatype="xsd:string"></span><span id='am-1139' property="oboInOwl:hasExactSynonym" content="breathless" datatype="xsd:string"></span><span id='am-1140' property="oboInOwl:hasExactSynonym" content="Fibroblast-growth-factor-receptor" datatype="xsd:string"></span><span id='am-1141' property="oboInOwl:hasExactSynonym" content="Fibroblast growth factor receptor homolog 2" datatype="xsd:string"></span><span id='am-1142' property="oboInOwl:hasExactSynonym" content="dtk2" datatype="xsd:string"></span><span id='am-1143' property="oboInOwl:hasExactSynonym" content="HD-311" datatype="xsd:string"></span><span id='am-1144' property="oboInOwl:hasExactSynonym" content="Tk2" datatype="xsd:string"></span><span id='am-1145' property="rdfs:label" content="CG6714" datatype="xsd:string"></span><span id='am-1146' property="rdfs:label" content="HD-311" datatype="xsd:string"></span><span id='am-1147' property="oboInOwl:hasExactSynonym" content="fgf-r" datatype="xsd:string"></span><span id='am-1148' property="oboInOwl:hasExactSynonym" content="devenir" datatype="xsd:string"></span><span id='am-1149' property="rdfs:label" content="FGFR1" datatype="xsd:string"></span><span id='am-1150' property="oboInOwl:hasExactSynonym" content="dFGF-R1" datatype="xsd:string"></span><span id='am-1151' property="rdfs:label" content="BTL/FGFR2" datatype="xsd:string"></span><span id='am-1152' property="oboInOwl:hasExactSynonym" content="CG6714" datatype="xsd:string"></span><span id='am-1153' property="rdfs:label" content="devenir" datatype="xsd:string"></span><span id='am-1154' property="rdfs:label" content="FGFR" datatype="xsd:string"></span><span id='am-1155' property="rdfs:label" content="Btl" datatype="xsd:string"></span><span id='am-1156' property="rdfs:label" content="dFGF-R1" datatype="xsd:string"></span><span id='am-1157' property="rdfs:label" content="Fibroblast growth factor receptor homolog 2" datatype="xsd:string"></span><span id='am-1158' property="rdfs:label" content="FGF receptor" datatype="xsd:string"></span><span id='am-1159' property="rdfs:label" content="DmHD-311" datatype="xsd:string"></span><span id='am-1160' property="oboInOwl:hasExactSynonym" content="Tyrosine kinase receptor HD-311" datatype="xsd:string"></span><span id='am-1161' property="rdfs:label" content="fgf-r" datatype="xsd:string"></span><span id='am-1162' property="oboInOwl:hasExactSynonym" content="l(3)00208" datatype="xsd:string"></span><span id='am-1163' property="rdfs:label" content="dev" datatype="xsd:string"></span><span id='am-1164' property="rdfs:label" content="btl" datatype="xsd:string"></span><span id='am-1165' property="rdfs:label" content="dtk2" datatype="xsd:string"></span><span id='am-1166' property="oboInOwl:hasExactSynonym" content="CT20816" datatype="xsd:string"></span><span id='am-1167' property="rdfs:label" content="CG32134" datatype="xsd:string"></span><span id='am-1168' property="rdfs:label" content="FR2" datatype="xsd:string"></span><span id='am-1169' property="oboInOwl:hasExactSynonym" content="0844/01" datatype="xsd:string"></span><span id='am-1170' property="rdfs:label" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-1171' property="rdfs:label" content="l(3)00208" datatype="xsd:string"></span><span id='am-1172' property="oboInOwl:hasDbXref" content="UniProt:Q09147" datatype="xsd:string"></span><span id='am-1173' property="rdfs:label" content="Tyrosine kinase 2" datatype="xsd:string"></span><span id='am-1174' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1175' property="rdfs:label" content="Protein breathless" datatype="xsd:string"></span><span id='am-1176' property="obo:IAO_0000115" content="The Drosophila genome includes two homologues of vertebrate FGF-R genes, DFR1and DFR2." datatype="xsd:string"></span><span id='am-1177' property="oboInOwl:hasExactSynonym" content="D-FGFR" datatype="xsd:string"></span><span id='am-1178' property="rdfs:label" content="dTk2" datatype="xsd:string"></span><span id='am-1179' property="oboInOwl:hasExactSynonym" content="CG32134" datatype="xsd:string"></span><span id='am-1180' property="oboInOwl:hasExactSynonym" content="EC 2.7.10.1" datatype="xsd:string"></span><span id='am-1181' property="oboInOwl:hasExactSynonym" content="BTL/FGFR2" datatype="xsd:string"></span><span id='am-1182' property="oboInOwl:hasExactSynonym" content="DmHD-311" datatype="xsd:string"></span><span id='am-1183' property="rdfs:label" content="Tk2" datatype="xsd:string"></span><span id='am-1184' property="rdfs:label" content="DFGF-R1" datatype="xsd:string"></span><span id='am-1185' property="oboInOwl:hasExactSynonym" content="Btl" datatype="xsd:string"></span><span id='am-1186' property="rdfs:label" content="Tyrosine kinase receptor HD-311" datatype="xsd:string"></span><span id='am-1187' property="rdfs:label" content="Fibroblast-growth-factor-receptor" datatype="xsd:string"></span><span id='am-1188' property="rdfs:label" content="FGFR2_DROME" datatype="xsd:string"></span><span id='am-1189' property="oboInOwl:hasExactSynonym" content="FGF receptor" datatype="xsd:string"></span><span id='am-1190' property="rdfs:label" content="CT20816" datatype="xsd:string"></span><span id='am-1191' property="rdfs:label" content="breathless" datatype="xsd:string"></span><span id='am-1192' property="oboInOwl:hasExactSynonym" content="btl" datatype="xsd:string"></span><span id='am-1193' property="oboInOwl:hasExactSynonym" content="Tyrosine kinase 2" datatype="xsd:string"></span><span id='am-1194' property="oboInOwl:hasExactSynonym" content="Dtk2" datatype="xsd:string"></span><span id='am-1195' property="oboInOwl:hasExactSynonym" content="dTk2" datatype="xsd:string"></span>Dev</span>., 6, 715 8211;722. <br />                                                                         15. Force , Lynch,M., Pickett,F.B., Amores , <span id='am-1275' about='obo:IMR_0002983' typeof='owl:Thing'><span id='am-1276' property="oboInOwl:hasExactSynonym" content="SK2-1" datatype="xsd:string"></span><span id='am-1277' property="oboInOwl:hasExactSynonym" content="yan/pok" datatype="xsd:string"></span><span id='am-1278' property="oboInOwl:hasExactSynonym" content="anterior open" datatype="xsd:string"></span><span id='am-1279' property="rdfs:label" content="aop/yan" datatype="xsd:string"></span><span id='am-1280' property="oboInOwl:hasExactSynonym" content="Protein anterior open" datatype="xsd:string"></span><span id='am-1281' property="oboInOwl:hasExactSynonym" content="aop" datatype="xsd:string"></span><span id='am-1282' property="oboInOwl:hasExactSynonym" content="pok" datatype="xsd:string"></span><span id='am-1283' property="rdfs:label" content="Aop" datatype="xsd:string"></span><span id='am-1284' property="rdfs:label" content="Protein yan" datatype="xsd:string"></span><span id='am-1285' property="rdfs:label" content="SK2-1" datatype="xsd:string"></span><span id='am-1286' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1287' property="oboInOwl:hasDbXref" content="UniProt:Q01842" datatype="xsd:string"></span><span id='am-1288' property="rdfs:label" content="anterior open" datatype="xsd:string"></span><span id='am-1289' property="oboInOwl:hasExactSynonym" content="YAN" datatype="xsd:string"></span><span id='am-1290' property="rdfs:label" content="aop" datatype="xsd:string"></span><span id='am-1291' property="oboInOwl:hasExactSynonym" content="DROYANETSB" datatype="xsd:string"></span><span id='am-1292' property="oboInOwl:hasExactSynonym" content="yan" datatype="xsd:string"></span><span id='am-1293' property="oboInOwl:hasExactSynonym" content="Aop/Yan" datatype="xsd:string"></span><span id='am-1294' property="oboInOwl:hasExactSynonym" content="DROYANET" datatype="xsd:string"></span><span id='am-1295' property="rdfs:label" content="DROYANET" datatype="xsd:string"></span><span id='am-1296' property="rdfs:label" content="yan" datatype="xsd:string"></span><span id='am-1297' property="oboInOwl:hasExactSynonym" content="Aop" datatype="xsd:string"></span><span id='am-1298' property="rdfs:label" content="POK_DROME" datatype="xsd:string"></span><span id='am-1299' property="oboInOwl:id" content="IMR:0002983" datatype="xsd:string"></span><span id='am-1300' property="rdfs:label" content="YAN" datatype="xsd:string"></span><span id='am-1301' property="oboInOwl:hasExactSynonym" content="CG3166" datatype="xsd:string"></span><span id='am-1302' property="oboInOwl:hasExactSynonym" content="aop/yan" datatype="xsd:string"></span><span id='am-1303' property="rdfs:label" content="Protein anterior open" datatype="xsd:string"></span><span id='am-1304' property="rdfs:label" content="Aop/Yan" datatype="xsd:string"></span><span id='am-1305' property="rdfs:label" content="yan/pok" datatype="xsd:string"></span><span id='am-1306' property="rdfs:label" content="pokkuri" datatype="xsd:string"></span><span id='am-1307' property="oboInOwl:hasExactSynonym" content="pokkuri" datatype="xsd:string"></span><span id='am-1308' property="rdfs:label" content="DROYANETSB" datatype="xsd:string"></span><span id='am-1309' property="oboInOwl:hasExactSynonym" content="Yan" datatype="xsd:string"></span><span id='am-1310' property="rdfs:label" content="Yan" datatype="xsd:string"></span><span id='am-1311' property="obo:IAO_0000115" content="Yan is the homolog of the human Tel1 protein and is a transcriptional repressor." datatype="xsd:string"></span><span id='am-1312' property="oboInOwl:hasExactSynonym" content="Protein yan" datatype="xsd:string"></span><span id='am-1313' property="rdfs:label" content="Ets DNA-binding protein pokkuri" datatype="xsd:string"></span><span id='am-1314' property="oboInOwl:hasExactSynonym" content="Ets DNA-binding protein pokkuri" datatype="xsd:string"></span><span id='am-1315' property="rdfs:label" content="pok" datatype="xsd:string"></span><span id='am-1316' property="rdfs:label" content="CG3166" datatype="xsd:string"></span>Yan</span>,Y.L. yeast genomes (Figure 5). Surviving duplicates                       Postlethwait,J. (1999) Preservation duplicate genes provide partial compensation intermediate di-                   complementary, degenerative mutations. Genetics, 151, 1531 8211;1545. <br /> vergence distances serve important source                16. Lynch,M. Force  (2000) probability duplicate gene new protein functions. stage (Ka &gt; 0.3                   preservation subfunctionalization. Genetics, 154, 459 8211;473. <br />                                                                         17. Kondrashov,F , Rogozin B., Wolf,Y  Koonin,E.V. <br />  70% sequence identity), lifecycle duplicates                     (2002) Selection evolution gene duplications. Genome <br /> completed functional roles diverge,                    Biol., 3, RESEARCH0008. <br /> quantitative compensation properties indistin-                   18. Bergthorsson,U., Andersson,D  Roth,J.R. (2007) Ohno 8217;s <br /> guishable random pairs yeast singletons.                   dilemma: evolution new genes continuous selection. <br />                                                                             <span id='am-526' about='obo:IMR_0010425' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010045 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0000952'><span id='am-527' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-528' property="oboInOwl:hasExactSynonym" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-529' property="oboInOwl:hasExactSynonym" content="PROC" datatype="xsd:string"></span><span id='am-530' property="rdfs:label" content="pyrroline-5-carboxylate reductase monomer" datatype="xsd:string"></span><span id='am-531' property="oboInOwl:hasExactSynonym" content="P5C reductase" datatype="xsd:string"></span><span id='am-532' property="rdfs:label" content="P5C reductase" datatype="xsd:string"></span><span id='am-533' property="rdfs:label" content="P5CR" datatype="xsd:string"></span><span id='am-534' property="rdfs:label" content="PROC" datatype="xsd:string"></span><span id='am-535' property="rdfs:label" content="Pyrroline-5-carboxylate reductase" datatype="xsd:string"></span><span id='am-536' property="oboInOwl:hasDbXref" content="EC:1.5.1.2" datatype="xsd:string"></span><span id='am-537' property="oboInOwl:hasExactSynonym" content="P5CR" datatype="xsd:string"></span><span id='am-538' property="oboInOwl:id" content="IMR:0010425" datatype="xsd:string"></span>Proc</span>. Natl Acad. Sci. USA, 104, 17004 8211;17009. <br />                                                                         19. VanderSluis,B., Bellay,J., Musso,G., Costanzo,M., Papp,B., <br />                                                                             Vizeacoumar,F.J., Baryshnikova , Andrews,B., Boone,C. SUPPLEMENTARY DATA                                                          Myers,C.L. (2010) Genetic interactions reveal evolutionary <br /> Supplementary Data available NAR Online.                             trajectories duplicate genes. Mol. Syst. Biol., 6, 429. <br />                                                                         20. Ihmels,J., Collins,S.R., Schuldiner,M., Krogan,N.J.                                                                             Weissman,J.S. (2007) Backup redundancy: genetic <br />                                                                             interactions reveal cost duplicate gene loss. Mol. Syst. <br /> ACKNOWLEDGEMENTS                                                            Biol., 3, 86. <br />                                                                         21. Li,J., Yuan  Zhang  (2010) cellular robustness thank Fyodor Kondrashov (Centre Genomic                              genetic redundancy budding yeast. PLoS Genet., 6, <br /> Regulation) members Vitup laboratory                         e1001187. <br /> illuminating discussions regarding manuscript.                      22. Guan,Y., Dunham,M.J. Troyanskaya,O.G. (2007) Functional <br />                                                                             analysis gene duplications Saccharomyces cerevisiae. <br />                                                                             Genetics, 175, 933 8211;943. <br /> FUNDING                                                                 23. Conant,G.C. Wagner  (2004) Duplicate genes                                                                             robustness transient gene knock-downs Caenorhabditis <br /> National Institutes Health (NIH) [GM079759]                      elegans. Proc. Biol. Sci., 271, 89 8211;96. <br /> National    Centers        Biomedical  Computing                    24. Thatcher,J.W., <span id='am-905' about='obo:IMR_0701307' typeof='owl:Thing'><span id='am-906' property="oboInOwl:hasExactSynonym" content="Voltage-gated potassium channel subunit Kv4" datatype="xsd:string"></span><span id='am-907' property="oboInOwl:hasExactSynonym" content="Shaw" datatype="xsd:string"></span><span id='am-908' property="rdfs:label" content="Shaw" datatype="xsd:string"></span><span id='am-909' property="rdfs:label" content="KCNC1_MOUSE" datatype="xsd:string"></span><span id='am-910' property="rdfs:label" content="Kcnc1" datatype="xsd:string"></span><span id='am-911' property="oboInOwl:hasExactSynonym" content="Kcr2-1" datatype="xsd:string"></span><span id='am-912' property="oboInOwl:hasExactSynonym" content="NGK2" datatype="xsd:string"></span><span id='am-913' property="rdfs:label" content="Potassium voltage-gated channel subfamily C member 1" datatype="xsd:string"></span><span id='am-914' property="oboInOwl:hasExactSynonym" content="KV4" datatype="xsd:string"></span><span id='am-915' property="rdfs:label" content="Voltage-gated potassium channel subunit Kv3.1" datatype="xsd:string"></span><span id='am-916' property="rdfs:label" content="Voltage-gated potassium channel subunit Kv4" datatype="xsd:string"></span><span id='am-917' property="oboInOwl:hasDbXref" content="UniProt:P15388" datatype="xsd:string"></span><span id='am-918' property="rdfs:label" content="NGK2" datatype="xsd:string"></span><span id='am-919' property="oboInOwl:hasExactSynonym" content="KShIIIB" datatype="xsd:string"></span><span id='am-920' property="oboInOwl:id" content="IMR:0701307" datatype="xsd:string"></span><span id='am-921' property="oboInOwl:hasExactSynonym" content="Kv3.1" datatype="xsd:string"></span><span id='am-922' property="oboInOwl:hasExactSynonym" content="Kcnc1" datatype="xsd:string"></span><span id='am-923' property="oboInOwl:hasExactSynonym" content="Potassium voltage-gated channel subfamily C member 1" datatype="xsd:string"></span><span id='am-924' property="rdfs:label" content="Kcr2-1" datatype="xsd:string"></span><span id='am-925' property="oboInOwl:hasExactSynonym" content="potassium voltage gated channel, Shaw-related subfamily, member 1" datatype="xsd:string"></span><span id='am-926' property="rdfs:label" content="potassium voltage gated channel, Shaw-related subfamily, member 1" datatype="xsd:string"></span><span id='am-927' property="rdfs:label" content="Kv3.1" datatype="xsd:string"></span><span id='am-928' property="rdfs:label" content="KShIIIB" datatype="xsd:string"></span><span id='am-929' property="rdfs:label" content="KV4" datatype="xsd:string"></span><span id='am-930' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-931' property="oboInOwl:hasExactSynonym" content="Voltage-gated potassium channel subunit Kv3.1" datatype="xsd:string"></span>Shaw</span>,J.M. Dickinson,W.J. (1998) Marginal <br />                                                                              65533;?tness contributions nonessential genes yeast. Proc. Natl <br /> [U54CA121852]. Funding open access charge: NIH                          Acad. Sci. USA, 95, 253 8211;257. <br /> [GM079759].                                                             25. Gillespie,J.H. (1998) Population Genetics, Concise Guide. Johns <br />                                                                             Hopkins Univ. Press, Baltimore. <br /> Con 64258;ict statement. declared.                           26. Hartl,D. Clark  (1997) Principles Population Genetics, <br />                                                                             3 edn. Sinauer Associates, Sunderland. <br />                                                                         27. Lynch,M. (2006) Streamlining simpli 65533;?cation microbial <br /> REFERENCES                                                                  genome architecture. Annu. Rev. Microbiol., 60, 327 8211;349. <br />                                                                         28. Kellis,M., Patterson,N., Endrizzi,M., Birren,B. Lander,E.S. <br />  1. Gu , Steinmetz,L.M., Gu,X., Scharfe,C., Davis,R.W.                (2003) Sequencing comparison yeast species identify <br />     Li,W.H. (2003) Role duplicate genes genetic robustness            genes regulatory elements. Nature, 423, 241 8211;254. <br />     null mutations. Nature, 421, 63 8211;66.                         29. Altschul,S.F., Madden,T.L., Schaffer , Zhang,J., Zhang , <br />  2. Stelling,J., Sauer,U., Szallasi , Doyle,F.J. III Doyle,J.         Miller,W. Lipman,D.J. (1997) Gapped BLAST PSI- <br />     (2004) Robustness cellular functions. Cell, 118, 675 8211;685.            BLAST: new generation protein database search programs. <br />  3. Wagner  (2005) Robustness Evolvability Living Systems.         Nucleic Acids Res., 25, 3389 8211;3402. <br />     Princeton University Press, Princeton.                              30. Gu , Cavalcanti , Chen,F.C., Bouman,P. Li,W.H. (2002) <br />  4. Wagner  (2000) Robustness mutations genetic                Extent gene duplication genomes Drosophila, <br />     networks yeast. Nat. Genet., 24, 355 8211;361.                            nematode, yeast. Mol. Biol. Evol., 19, 256 8211;262. <br />  5. Papp,B., Pal,C. Hurst,L.D. (2004) Metabolic network             31. Yang  Nielsen,R. (2000) Estimating synonymous     analysis causes evolution enzyme dispensability        nonsynonymous substitution rates realistic evolutionary <br />     yeast. Nature, 429, 661 8211;664.                                            models. Mol. Biol. Evol., 17, 32 8211;43. <br />  6. Lynch,M. Conery,J.S. (2000) evolutio <br /> </body></html>