<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Metabolic network prediction pairwise rational kernels <br />  <br />  <br />  <br />   Abstract <br />   Background: Metabolic networks represented set metabolic pathways. Metabolic pathways series <br />   biochemical reactions, product (output) reaction serves substrate (input)   reaction. pathways remain incompletely characterized. major challenges computational biology   obtain better models metabolic pathways. Existing models dependent annotation genes.   propagates error accumulation pathways predicted incorrectly annotated genes. Pairwise <br />   classification methods supervised learning methods used classify new pair entities.   classification methods, e.g., Pairwise Support Vector Machines (SVMs), use pairwise kernels. Pairwise kernels   similarity measures pairs entities. Using pairwise kernels handle sequence data requires long <br />   processing times large storage. Rational kernels kernels based weighted finite-state transducers   represent similarity measures sequences automata. effectively used problems   handle large sequence information <span id='am-1233' about='obo:IMR_0000949' typeof='owl:Thing obo:IMR_0000001'><span id='am-1234' property="oboInOwl:id" content="IMR:0000949" datatype="xsd:string"></span><span id='am-1235' property="rdfs:label" content="protein" datatype="xsd:string"></span><span id='am-1236' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>protein</span> essentiality, natural language processing machine <br />   translations. <br />   Results: create new family pairwise kernels using weighted finite-state transducers (called Pairwise Rational <br />   Kernel (<span id='am-1' about='obo:IMR_0701526' typeof='owl:Thing'><span id='am-2' property="rdfs:label" content="Cytokine-inducible serine/threonine-protein kinase" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="polo-like kinase 3" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Proliferation-related kinase" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Serine/threonine-protein kinase PLK3" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="PLK-3" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="UniProt:Q9H4B4" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="FGF-inducible kinase" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Polo-like kinase 3" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="CYTOKINE-INDUCIBLE KINASE" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="FGF-inducible kinase" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="PRK" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="PLK-3" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Cytokine-inducible serine/threonine-protein kinase" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="CNK" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Serine/threonine-protein kinase PLK3" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="PLK3_HUMAN" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="FNK" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="EC 2.7.11.21" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="CNK" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="polo-like kinase 3" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="PLK3" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Polo-like kinase 3" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="FNK" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="PLK3" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0701526" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="Proliferation-related kinase" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="EC 2.7.11.21" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="CYTOKINE-INDUCIBLE KINASE" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="PRK" datatype="xsd:string"></span>PRK</span>)) predict metabolic pathways variety biological data. PRKs advantage simpler <br />   representations faster algorithms transducers. <span id='am-1411' about='obo:IMR_0701312' typeof='owl:Thing'><span id='am-1412' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1413' property="rdfs:label" content="Raw" datatype="xsd:string"></span><span id='am-1414' property="rdfs:label" content="Arresten" datatype="xsd:string"></span><span id='am-1415' property="oboInOwl:hasExactSynonym" content="alpha1(IV) collagen" datatype="xsd:string"></span><span id='am-1416' property="rdfs:label" content="CO4A1_MOUSE" datatype="xsd:string"></span><span id='am-1417' property="oboInOwl:hasExactSynonym" content="Svc" datatype="xsd:string"></span><span id='am-1418' property="oboInOwl:hasExactSynonym" content="Col4a1" datatype="xsd:string"></span><span id='am-1419' property="rdfs:label" content="Collagen alpha-1(IV) chain" datatype="xsd:string"></span><span id='am-1420' property="oboInOwl:id" content="IMR:0701312" datatype="xsd:string"></span><span id='am-1421' property="oboInOwl:hasDbXref" content="UniProt:P02463" datatype="xsd:string"></span><span id='am-1422' property="rdfs:label" content="collagen, type IV, alpha 1" datatype="xsd:string"></span><span id='am-1423' property="oboInOwl:hasExactSynonym" content="Arresten" datatype="xsd:string"></span><span id='am-1424' property="rdfs:label" content="Bru" datatype="xsd:string"></span><span id='am-1425' property="oboInOwl:hasExactSynonym" content="Col4a-1" datatype="xsd:string"></span><span id='am-1426' property="rdfs:label" content="Col4a-1" datatype="xsd:string"></span><span id='am-1427' property="oboInOwl:hasExactSynonym" content="Del(8)44H" datatype="xsd:string"></span><span id='am-1428' property="oboInOwl:hasExactSynonym" content="collagen, type IV, alpha 1" datatype="xsd:string"></span><span id='am-1429' property="oboInOwl:hasExactSynonym" content="Bru" datatype="xsd:string"></span><span id='am-1430' property="rdfs:label" content="alpha1(IV) collagen" datatype="xsd:string"></span><span id='am-1431' property="oboInOwl:hasExactSynonym" content="Collagen alpha-1(IV) chain" datatype="xsd:string"></span><span id='am-1432' property="rdfs:label" content="Svc" datatype="xsd:string"></span><span id='am-1433' property="rdfs:label" content="Del(8)Bru44H" datatype="xsd:string"></span><span id='am-1434' property="oboInOwl:hasExactSynonym" content="Raw" datatype="xsd:string"></span><span id='am-1435' property="rdfs:label" content="Col4a1" datatype="xsd:string"></span><span id='am-1436' property="oboInOwl:hasExactSynonym" content="Del(8)Bru44H" datatype="xsd:string"></span><span id='am-1437' property="rdfs:label" content="Del(8)44H" datatype="xsd:string"></span>raw</span> sequence data used, predictor model <br />   avoids errors introduced incorrect gene annotations. developed experiments PRKs   Pairwise SVM validate methods using metabolic network Saccharomyces cerevisiae. result, PRKs <br />   used, method executes faster comparison pairwise kernels.  use PRKs combined <br />   simple kernels include evolutionary information, accuracy values improved,   maintaining lower construction execution times. <br />   Conclusions: power using kernels sort data represented using kernels.  <br />   completely disparate types data combined add power kernel-based machine learning methods.   compared proposal using PRKs similar kernel, execution times decreased,   compromise accuracy. proved combining PRKs kernels include evolutionary <br />   information, accuracy improved. proposal use type sequence data, genes   need properly annotated, avoiding accumulation errors incorrect previous annotations. <br />   Keywords: Metabolic network, Pairwise rational kernels, Supervised network inference, Finite-state transducers, <br />   Pairwise support vector machine <br />  <br />  <br />  <br />  <br /> Background                                                       Yamanishi [9] Kotera et al. [11] described Related work                                                   theory implementation GENIES, web applica- <br /> Metabolic networks allow modelling molecular sys-       tion allowed prediction unknown parts tems understand underlying biological mechanisms        metabolic networks using supervised graph inference cell [1]. Metabolic networks represented set   kernel methods. algorithms implemented metabolic pathways. Metabolic pathways series      GENIES decision predictive func- <br /> biochemical reactions, product (output)      tions supervised network inference. reaction serves substrate (input)        algorithms Kernel Canonical Correlation Analysis <br /> reaction. experimental determination metabolic          (KCCA) [13,14], Expectation-Maximization (EM) algo- <br /> networks, based known biological data DNA           rithm [15] Kernel Matrix Regression (KMR) [9]. protein sequences, gene expression data,   authors developed experiments, did challenging [2].  efforts      use sequence data.  motivations develop supervised learning methods determine genes         extend previous research [7] use sequence <br /> coding missing enzymes predict unknown parts        data combined algorithms. noted  metabolic networks [3,4].                                      obtained best accuracy values SVM method <br />   methods predict metabolic networks            combined sequence kernels, high execution <br /> assume genome annotation correct, e.g., Path-      times. <br /> way Tools [4], software application predict metabolic       address high computational costs,  <br /> networks using information BioCyc databases [5].          sider results Allauzen et al. [16], proposed <br /> Pathway Tools uses algorithm,         method predict protein essentiality using SVMs 1 infers reactions catalyzed organism          manipulating sequence data using rational kernels. set enzymes present annotated genome,            authors designed sequence kernels (called general 2 infers metabolic pathways present        domain-based kernels), instances rational <br /> organism reactions 1.           kernels. handle large data (6190 domains <br /> sidering BioCyc MetaCyc huge              3000 protein sequences), automata rep- <br /> available data, application potentially make pre-     resentation used create rational kernels. cise metabolic pathway predictions [6].           results showed final kernels favourably predicted <br /> 2 based annotated genes,            protein essentiality. note,  errors annotation, inferred pathways           previous works using rational kernels bioinformatics correct.  methods intrinsically         [16-18] considered problems related biological <br /> carry error accumulations incorrect genome              network predictions. <br /> annotations.                                                     Based fact rational kernels described   tackle problem, previously proposed          Allauzen et al. [16] extended problems, <br /> using information directly related sequence          define new kernels applied metabolic network primary data (e.g., genomic proteomic data)            predictions. research, represent sequence data <br /> [7]. result, obtained best accuracy values         using rational kernels. Rational kernels advantage using Support Vector Machine (SVM) methods combined            fast algorithms  efficient representation  <span id='am-872' about='xsd:string' typeof='owl:Thing'>string</span> kernels representing sequence data.         transducers sequence manipulations improve  <br /> experimentally demonstrated SVMs supersede          formance. sequence data used, raw genomic <br /> methods, matrix kernel regression, predict-        proteomic information considered, ing metabolic networks. consistent recent         method avoids problems associated incorrect anno- <br /> results showing usefulness SVMs bioinformatics       tation predicting metabolic networks. Additionally, <br /> [8].  solution [7] computationally expen-      current work combine rational kernels <br /> sive terms execution time sequence data       (using finite-state transducers) [17-20] known pair- <br /> manipulation.                                                  <span id='am-364' about='obo:IMR_0702237' typeof='owl:Thing'><span id='am-365' property="oboInOwl:id" content="IMR:0702237" datatype="xsd:string"></span><span id='am-366' property="rdfs:label" content="WISE" datatype="xsd:string"></span><span id='am-367' property="rdfs:label" content="Sclerostin domain-containing protein 1" datatype="xsd:string"></span><span id='am-368' property="oboInOwl:hasExactSynonym" content="Sclerostin domain-containing protein 1" datatype="xsd:string"></span><span id='am-369' property="oboInOwl:hasExactSynonym" content="SOSTDC1" datatype="xsd:string"></span><span id='am-370' property="rdfs:label" content="Wnt-signaling modulator" datatype="xsd:string"></span><span id='am-371' property="rdfs:label" content="SOSTDC1" datatype="xsd:string"></span><span id='am-372' property="oboInOwl:hasDbXref" content="UniProt:Q6VYA3" datatype="xsd:string"></span><span id='am-373' property="oboInOwl:hasExactSynonym" content="Wnt-signaling modulator" datatype="xsd:string"></span><span id='am-374' property="rdfs:label" content="SOSD1_CHICK" datatype="xsd:string"></span><span id='am-375' property="oboInOwl:hasExactSynonym" content="WISE" datatype="xsd:string"></span><span id='am-376' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>wise</span> kernels [10,21-23] obtain pairwise rational kernels. <br />   authors combined SVM               kernel techniques proposed paper supervised learning techniques kernel methods          applied equally machine learning tools predict metabolic networks [9-11]. main advantage          employ kernel methods, KCCA, EM KMR, using kernel methods heterogeneous data         focused SVMs illustration capabil- represented combined simultaneously.            ity reduce computational costs. chosen <br /> disparate types data manipulated kernels,         SVM methods light experimental results data sources contribute uni-          obtained previous works [7], efficiency <br /> formly information training set building    effectiveness SVM methods predict protein <br /> model [12].                                                    essentiality [16]. <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                      Page 3 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />  <br /> Automata transducers                                             example, weighted transducer shown Automata define mathematical formalism analyze            Figure 1 . use delimiters colon sepa- <br /> model real problems useful machines [24].               rate input output labels transitions automaton set states (generally represented            slash separate weight values  e., nota- <br /> circles), transitions (generally represented arrows).       tion input:output/weight). States represented automaton moves state state                circles, set initial states bold circles (makes transition) activated event func-           set final states double circles. ini- <br /> tion. variant automaton called finite state           tial final states associated weighs  notation <br /> machine. finite-state machine used model               state/weight). Example 1 shows compute simple  turnstiles transit lights,          weight transducer T  e., T(x, y)) given <br /> complex systems sophisticated spaceship controls           sequences x y. case, define alphabets <br /> [25].                                                                = {G, C}   = {G, C}. <br />    Automata work sequence symbols,    8727; <br /> denotes finite sequences using symbols          Example 1. weight  value) associated trans- <br /> alphabet  , including   represents sym-             ducer T Figure 1  pair (x, y) = (GGC, CCG)  8712; <br /> bol. order formally define automata transducers,            8727;  215;   8727; computed  follow notations used Cortes et al. [17].             T(GGC, CCG) = 1  8727; 2  8727; 3  8727; 6  8727; 1 + 1  8727; 3  8727; 1  8727; 4  8727; 1 = 48, <br /> automaton 5-tuple ( , Q,  F,  948;) [24]         considering accepting paths labelled <br /> input alphabet set, Q state set,  8834; Q subset        input GCC output CCG. paths  initial states, F  8834; Q subset final states,             Path 1 : State 0    8594; State 0    8594; State 1    8594; State 3, <br />  948;  8838; Q  215; (   8746; { })  215; Q transition set. transition               Path 2 : State 0    8594; State 1    8594; State 2    8594; State 3. <br />  953;  8712;  948; describes actions moving state            initial final values terms T(GGC, CCG) condition (input symbol) encountered.            correspond weights initial final <br />    Similarly, Finite-State Transducer (FST) automa-        states. <br /> ton output <span id='am-995' about='rdfs:label' typeof='owl:Thing'>label</span> included transition addition input label. Based <span id='am-137' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>,          Figure 1(b) shows graph representation weighted FST T 6-tuple ( ,  , Q,  F,  948;) [18], new        automaton. obtained output projection term   output alphabet transition set  948;          transducer T input labels omitted.   948;  8838; Q  215; (   8746; { })  215; (   8746; { })  215; Q. Similar pre-         alphabet     = {G, C} weight computation <br /> vious definition, transition  953;  8712;  948; action moving       automaton given sequences shown state input symbol               Example 2. encountered output   produced. <br />    addition, Automata Finite-State Transducers          Example 2. weight  value) associated weighted, transition labelled              Automaton Figure 1(b) y = CCG  8712;   8727; com- <br /> weight.  Weighted Automaton (WA) 7-tuple               puted  <br /> ( , Q,  F,  948;,  955;,  65533;?) Weighted Finite-State Transducer            CCG) = 1  8727; 2  8727; 3  8727; 6  8727; 1 + 1  8727; 3  8727; 1  8727; 4  8727; 1 = 48 <br /> (WFST) 8-tuple ( ,  , Q,  F,  948;,  955;,  65533;?) [18],       considering accepting paths labelled <br /> new terms  955;  65533;?   955; :  8594; R, initial weight func-         CCG. paths  <br /> tion,  65533;? : F  8594; R, final weight function. new                 Path 1 : State 0    8594; State 0    8594; State 1    8594; State 3, <br /> transitions WFSTs  948;  8838; Q 215;(  8746;{ }) 215;                    Path 2 : State 0    8594; State 1    8594; State 2    8594; State 3. <br /> R 215;Q  948;  8838; Q 215;(  8746;{ }) 215;(  8746;{ }) 215;R 215;Q, respectively,                   initial final values terms CCG) R represents weights real numbers.                    correspond weights initial final states. <br />  <br />  <br />  <br />  <br />  Figure 1 Weighted transducer weighted automaton representing sequences alphabet   =   = {G, C}.   Weighted <br />  Transducer T. (b) Weighted Automaton  obtained projecting output T). <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                      Page 4 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />  <br />   operations defined automata                 Using Algorithm 1, overall complexity com- <br /> transducers, inverse composition. Given             pute value rational kernel O(|U||Mx | |), <br /> transducer T, inverse T  8722;1 transducer obtained           |U| remains constant. practice, complexity input output labels swapped               reduced O(|U| + |Mx | +  |) kernels transition. composition operation transduc-              used areas natural language process- <br /> ers T1 <span id='am-925' about='obo:IMR_0010637' typeof='owl:Thing obo:IMR_0000955 obo:IMR_0000949 obo:IMR_0010050 obo:IMR_0000001 obo:IMR_0000207'><span id='am-926' property="oboInOwl:id" content="IMR:0010637" datatype="xsd:string"></span><span id='am-927' property="oboInOwl:hasExactSynonym" content="acetyl-CoA acetyltransferase monomer" datatype="xsd:string"></span><span id='am-928' property="rdfs:label" content="acetyl-CoA acetyltransferase monomer" datatype="xsd:string"></span><span id='am-929' property="rdfs:label" content="Acetyl-CoA acetyltransferase, mitochondrial precursor" datatype="xsd:string"></span><span id='am-930' property="rdfs:label" content="Acetoacetyl-CoA thiolase" datatype="xsd:string"></span><span id='am-931' property="oboInOwl:hasDbXref" content="EC:2.3.1.9" datatype="xsd:string"></span><span id='am-932' property="oboInOwl:hasExactSynonym" content="T2" datatype="xsd:string"></span><span id='am-933' property="oboInOwl:hasExactSynonym" content="Acetoacetyl-CoA thiolase" datatype="xsd:string"></span><span id='am-934' property="rdfs:label" content="T2" datatype="xsd:string"></span><span id='am-935' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>T2</span> input output alphabets                  ing computational biology. example, Algorithm 1 <br /> equal   weighted transducer, denoted T1  9702;                n-gram kernel linear complexity   2 , provided sum given (T1  9702; <span id='am-1438' about='obo:IMR_0702890' typeof='owl:Thing'><span id='am-1439' property="rdfs:label" content="ACETOACETYL-CoA THIOLASE, MITOCHONDRIAL" datatype="xsd:string"></span><span id='am-1440' property="oboInOwl:hasDbXref" content="UniProt:P24752" datatype="xsd:string"></span><span id='am-1441' property="rdfs:label" content="Acetyl-CoA acetyltransferase, mitochondrial" datatype="xsd:string"></span><span id='am-1442' property="rdfs:label" content="MITOCHONDRIAL ACETOACETYL-CoA THIOLASE" datatype="xsd:string"></span><span id='am-1443' property="rdfs:label" content="Acetoacetyl-CoA thiolase" datatype="xsd:string"></span><span id='am-1444' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1445' property="rdfs:label" content="THIL" datatype="xsd:string"></span><span id='am-1446' property="oboInOwl:hasExactSynonym" content="EC 2.3.1.9" datatype="xsd:string"></span><span id='am-1447' property="rdfs:label" content="T2" datatype="xsd:string"></span><span id='am-1448' property="oboInOwl:id" content="IMR:0702890" datatype="xsd:string"></span><span id='am-1449' property="oboInOwl:hasExactSynonym" content="T2" datatype="xsd:string"></span><span id='am-1450' property="rdfs:label" content="MAT" datatype="xsd:string"></span><span id='am-1451' property="oboInOwl:hasExactSynonym" content="Acetyl-CoA acetyltransferase, mitochondrial" datatype="xsd:string"></span><span id='am-1452' property="rdfs:label" content="EC 2.3.1.9" datatype="xsd:string"></span><span id='am-1453' property="oboInOwl:hasExactSynonym" content="Acetoacetyl-CoA thiolase" datatype="xsd:string"></span><span id='am-1454' property="oboInOwl:hasExactSynonym" content="MITOCHONDRIAL ACETOACETYL-CoA THIOLASE" datatype="xsd:string"></span><span id='am-1455' property="oboInOwl:hasExactSynonym" content="THIL" datatype="xsd:string"></span><span id='am-1456' property="rdfs:label" content="acetyl-CoA acetyltransferase 1" datatype="xsd:string"></span><span id='am-1457' property="rdfs:label" content="THIL_HUMAN" datatype="xsd:string"></span><span id='am-1458' property="oboInOwl:hasExactSynonym" content="MAT" datatype="xsd:string"></span><span id='am-1459' property="oboInOwl:hasExactSynonym" content="ACETOACETYL-CoA THIOLASE, MITOCHONDRIAL" datatype="xsd:string"></span><span id='am-1460' property="oboInOwl:hasExactSynonym" content="ACAT" datatype="xsd:string"></span><span id='am-1461' property="oboInOwl:hasExactSynonym" content="acetyl-CoA acetyltransferase 1" datatype="xsd:string"></span><span id='am-1462' property="rdfs:label" content="ACAT1" datatype="xsd:string"></span><span id='am-1463' property="oboInOwl:hasExactSynonym" content="ACAT1" datatype="xsd:string"></span><span id='am-1464' property="rdfs:label" content="ACAT" datatype="xsd:string"></span>T2</span> )(x, y) = <br /> T                                                                   detailed description n-gram kernel . <br />       8727; T1 (x, T2   y) defined R (x, y)  8712;      Kernels used training methods discriminant clas- <br />   8727;.                                                                 sification algorithms (e.g., SVM) need satisfy Mercer 8217;s <br />                                                                     condition equivalently Positive Definite Sym- <br /> Rational kernels                                                    metric - <span id='am-427' about='obo:IMR_0200723' typeof='owl:Thing'><span id='am-428' property="rdfs:label" content="PROSTAGLANDIN D SYNTHASE, LIPOCALIN-TYPE" datatype="xsd:string"></span><span id='am-429' property="rdfs:label" content="PGDS" datatype="xsd:string"></span><span id='am-430' property="oboInOwl:hasExactSynonym" content="Prostaglandin-H2 D-isomerase" datatype="xsd:string"></span><span id='am-431' property="oboInOwl:hasExactSynonym" content="LPGDS" datatype="xsd:string"></span><span id='am-432' property="oboInOwl:hasExactSynonym" content="Prostaglandin-D2 synthase" datatype="xsd:string"></span><span id='am-433' property="oboInOwl:hasExactSynonym" content="EC 5.3.99.2" datatype="xsd:string"></span><span id='am-434' property="rdfs:label" content="LPGDS" datatype="xsd:string"></span><span id='am-435' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-436' property="rdfs:label" content="PGD2 synthase" datatype="xsd:string"></span><span id='am-437' property="rdfs:label" content="LIPOCALIN-TYPE PROSTAGLANDIN D SYNTHASE" datatype="xsd:string"></span><span id='am-438' property="oboInOwl:hasExactSynonym" content="PGD2 synthase" datatype="xsd:string"></span><span id='am-439' property="oboInOwl:hasExactSynonym" content="PGD2" datatype="xsd:string"></span><span id='am-440' property="oboInOwl:hasExactSynonym" content="L-PGDS" datatype="xsd:string"></span><span id='am-441' property="rdfs:label" content="L-PGDS" datatype="xsd:string"></span><span id='am-442' property="rdfs:label" content="Glutathione-independent PGD synthase" datatype="xsd:string"></span><span id='am-443' property="rdfs:label" content="Beta-trace protein" datatype="xsd:string"></span><span id='am-444' property="oboInOwl:id" content="IMR:0200723" datatype="xsd:string"></span><span id='am-445' property="rdfs:label" content="PROSTAGLANDIN D2 SYNTHASE, BRAIN" datatype="xsd:string"></span><span id='am-446' property="oboInOwl:hasExactSynonym" content="PROSTAGLANDIN D2 SYNTHASE, BRAIN" datatype="xsd:string"></span><span id='am-447' property="rdfs:label" content="Lipocalin-type prostaglandin-D synthase" datatype="xsd:string"></span><span id='am-448' property="rdfs:label" content="PTGDS" datatype="xsd:string"></span><span id='am-449' property="oboInOwl:hasExactSynonym" content="BETA-TRACE" datatype="xsd:string"></span><span id='am-450' property="oboInOwl:hasExactSynonym" content="PTGDS" datatype="xsd:string"></span><span id='am-451' property="oboInOwl:hasExactSynonym" content="PGDS" datatype="xsd:string"></span><span id='am-452' property="oboInOwl:hasExactSynonym" content="PGDS2" datatype="xsd:string"></span><span id='am-453' property="rdfs:label" content="Prostaglandin-D2 synthase" datatype="xsd:string"></span><span id='am-454' property="oboInOwl:hasDbXref" content="UniProt:P41222" datatype="xsd:string"></span><span id='am-455' property="oboInOwl:hasExactSynonym" content="prostaglandin D2 synthase 21kDa (brain)" datatype="xsd:string"></span><span id='am-456' property="oboInOwl:hasExactSynonym" content="PROSTAGLANDIN D SYNTHASE, LIPOCALIN-TYPE" datatype="xsd:string"></span><span id='am-457' property="oboInOwl:hasExactSynonym" content="PDS" datatype="xsd:string"></span><span id='am-458' property="oboInOwl:hasExactSynonym" content="Glutathione-independent PGD synthase" datatype="xsd:string"></span><span id='am-459' property="oboInOwl:hasExactSynonym" content="LIPOCALIN-TYPE PROSTAGLANDIN D SYNTHASE" datatype="xsd:string"></span><span id='am-460' property="rdfs:label" content="PTGDS_HUMAN" datatype="xsd:string"></span><span id='am-461' property="rdfs:label" content="Prostaglandin-H2 D-isomerase" datatype="xsd:string"></span><span id='am-462' property="rdfs:label" content="EC 5.3.99.2" datatype="xsd:string"></span><span id='am-463' property="rdfs:label" content="PDS" datatype="xsd:string"></span><span id='am-464' property="oboInOwl:hasExactSynonym" content="Lipocalin-type prostaglandin-D synthase" datatype="xsd:string"></span><span id='am-465' property="oboInOwl:hasExactSynonym" content="Beta-trace protein" datatype="xsd:string"></span><span id='am-466' property="rdfs:label" content="prostaglandin D2 synthase 21kDa (brain)" datatype="xsd:string"></span><span id='am-467' property="rdfs:label" content="Cerebrin-28" datatype="xsd:string"></span><span id='am-468' property="rdfs:label" content="PGD2" datatype="xsd:string"></span><span id='am-469' property="rdfs:label" content="PGDS2" datatype="xsd:string"></span><span id='am-470' property="oboInOwl:hasExactSynonym" content="Cerebrin-28" datatype="xsd:string"></span><span id='am-471' property="rdfs:label" content="BETA-TRACE" datatype="xsd:string"></span>PDS</span> [18]. Cortes et al. [18] proven result order manipulate sequence data, FSTs provide sim-           gives general method construct PDS rational <br /> ple representation efficient algorithms          kernel using WFSTs. <br /> composition shortest-distance [18]. Rational Kernels, <br /> based Finite-State Transducers, effective ana-           Theorem 1. ([18]). T arbitrary weighted trans- <br /> lyzing sequences variable lengths [17].                        ducer, U = T  9702; T  8722;1 defines PDS rational kernel. <br />   formal definition, function k :    8727;  215;   8727;  8594; R rational kernel exists WFST U k           n-gram kernel rational kernel <br /> coincides function defined U, e., k(x, y) =           Hofmann et al. [26] defined class similarity mea- <br /> U(x, y) sequences x, y  8712;    8727;  215;   8727; [17].            sures biological sequences function  consider input output alphabets             number equal subsequences  symbols  e.,   =  ), terms      8727;            example measures spectrum kernel defined used.                                                       Leslie et al. [27]. Similarity values results   order compute value U(x, y) partic-            summing products counts sub- <br /> ular pair sequences x, y  8712;    8727;  215;    8727; , composition           sequences. referred computational biology <br /> algorithm weighted transducers used [17]:                     k-mer n-gram kernel. rest paper,                                                                     use term n-gram follow notation Hofmann <br />    8226;  Mx , considered trivial weighted <br />                                                                     et al. [26] Cortes et al. [17]. <br />     transducers representing x, y respectively,     Mx (x, x) = 1 Mx (v, w) = 0 v  <br />  = x w  <br />  = x.          n-gram kernel defined kn (x, y)                  = <br />                                                                         =n cx  cy   fixed integer n, represents <br />     Mx obtained using linear finite automata <br />                                                                     subsequences length n.  ca (b) number     representing x augmenting transition                                                                     times subsequence b appears  kn     output label identical input label setting <br />                                                                     represented rational kernel using weighted trans- <br />     transition, initial final weights                                                                      ducer = <span id='am-1370' about='obo:IMR_0701306' typeof='owl:Thing'><span id='am-1371' property="rdfs:label" content="Myotendinous antigen" datatype="xsd:string"></span><span id='am-1372' property="oboInOwl:hasExactSynonym" content="Cytotactin" datatype="xsd:string"></span><span id='am-1373' property="oboInOwl:id" content="IMR:0701306" datatype="xsd:string"></span><span id='am-1374' property="oboInOwl:hasExactSynonym" content="GP" datatype="xsd:string"></span><span id='am-1375' property="oboInOwl:hasExactSynonym" content="Tenascin" datatype="xsd:string"></span><span id='am-1376' property="oboInOwl:hasExactSynonym" content="Hexabrachion" datatype="xsd:string"></span><span id='am-1377' property="rdfs:label" content="TN-C" datatype="xsd:string"></span><span id='am-1378' property="rdfs:label" content="tenascin C" datatype="xsd:string"></span><span id='am-1379' property="rdfs:label" content="Neuronectin" datatype="xsd:string"></span><span id='am-1380' property="oboInOwl:hasExactSynonym" content="MGC167029" datatype="xsd:string"></span><span id='am-1381' property="rdfs:label" content="Glioma-associated-extracellular matrix antigen" datatype="xsd:string"></span><span id='am-1382' property="oboInOwl:hasExactSynonym" content="GMEM" datatype="xsd:string"></span><span id='am-1383' property="oboInOwl:hasExactSynonym" content="Tenascin-C" datatype="xsd:string"></span><span id='am-1384' property="oboInOwl:hasExactSynonym" content="150-225" datatype="xsd:string"></span><span id='am-1385' property="rdfs:label" content="MGC167029" datatype="xsd:string"></span><span id='am-1386' property="rdfs:label" content="TN" datatype="xsd:string"></span><span id='am-1387' property="rdfs:label" content="HXB" datatype="xsd:string"></span><span id='am-1388' property="oboInOwl:hasExactSynonym" content="TNC" datatype="xsd:string"></span><span id='am-1389' property="rdfs:label" content="TENA_HUMAN" datatype="xsd:string"></span><span id='am-1390' property="rdfs:label" content="JI" datatype="xsd:string"></span><span id='am-1391' property="rdfs:label" content="Cytotactin" datatype="xsd:string"></span><span id='am-1392' property="oboInOwl:hasExactSynonym" content="Neuronectin" datatype="xsd:string"></span><span id='am-1393' property="oboInOwl:hasExactSynonym" content="Myotendinous antigen" datatype="xsd:string"></span><span id='am-1394' property="oboInOwl:hasExactSynonym" content="tenascin C" datatype="xsd:string"></span><span id='am-1395' property="rdfs:label" content="GP" datatype="xsd:string"></span><span id='am-1396' property="rdfs:label" content="GP 150-225" datatype="xsd:string"></span><span id='am-1397' property="rdfs:label" content="TNC" datatype="xsd:string"></span><span id='am-1398' property="rdfs:label" content="Hexabrachion" datatype="xsd:string"></span><span id='am-1399' property="oboInOwl:hasExactSynonym" content="TN-C" datatype="xsd:string"></span><span id='am-1400' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1401' property="oboInOwl:hasExactSynonym" content="JI" datatype="xsd:string"></span><span id='am-1402' property="rdfs:label" content="Tenascin-C" datatype="xsd:string"></span><span id='am-1403' property="oboInOwl:hasExactSynonym" content="GP 150-225" datatype="xsd:string"></span><span id='am-1404' property="rdfs:label" content="Tenascin" datatype="xsd:string"></span><span id='am-1405' property="oboInOwl:hasDbXref" content="UniProt:P24821" datatype="xsd:string"></span><span id='am-1406' property="rdfs:label" content="150-225" datatype="xsd:string"></span><span id='am-1407' property="rdfs:label" content="GMEM" datatype="xsd:string"></span><span id='am-1408' property="oboInOwl:hasExactSynonym" content="TN" datatype="xsd:string"></span><span id='am-1409' property="oboInOwl:hasExactSynonym" content="Glioma-associated-extracellular matrix antigen" datatype="xsd:string"></span><span id='am-1410' property="oboInOwl:hasExactSynonym" content="HXB" datatype="xsd:string"></span>Tn</span>  9702; Tn 8722;1 , transducer Tn defined <br />     obtained similar way using y. <br />                                                                     Tn (x,  = cx  , x,  8712;      8727;   = n [18]. <br />    8226;  definition weighted transducer <br />                                                                     example, n = 2, k2 (x, y) =  =2 cx  cy       composition: <br />                                                                     rational kernel represents subsequences <br />     (Mx  9702; U  9702; )(x, y) = Mx (x, x)U(x, y (y, y). <br />                                                                        8727; size 2 T2 (x,  = cx   counts     Considering Mx (x, x) = 1 (y, y) = 1,                                                                     times occurs x. <br />     obtain (Mx  9702; U  9702; )(x, y) = k(x, y), e., sum                                                                       Allauzen et al. [16] extended construction     weights paths Mx  9702; U  9702; exactly <br />                                                                     kernel, kn , measure similarity     U(x, y) = k(x, y). <br />                                                                     sequences represented automata. Firstly, define <br />   Based representation, <span id='am-472' about='obo:IMR_0703879' typeof='owl:Thing'><span id='am-473' property="oboInOwl:id" content="IMR:0703879" datatype="xsd:string"></span><span id='am-474' property="rdfs:label" content="STEP" datatype="xsd:string"></span><span id='am-475' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-476' property="rdfs:label" content="Ptpn5" datatype="xsd:string"></span><span id='am-477' property="oboInOwl:hasExactSynonym" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-478' property="oboInOwl:hasDbXref" content="UniProt:P35234" datatype="xsd:string"></span><span id='am-479' property="oboInOwl:hasExactSynonym" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-480' property="oboInOwl:hasExactSynonym" content="STEP" datatype="xsd:string"></span><span id='am-481' property="rdfs:label" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-482' property="rdfs:label" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-483' property="oboInOwl:hasExactSynonym" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-484' property="rdfs:label" content="PTN5_RAT" datatype="xsd:string"></span><span id='am-485' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-486' property="oboInOwl:hasExactSynonym" content="Ptpn5" datatype="xsd:string"></span><span id='am-487' property="rdfs:label" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-488' property="rdfs:label" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-489' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-490' property="rdfs:label" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span>step</span> algorithm             count      sequence weighted automaton defined Cortes et al. [17] obtain k(x, y) = U(x, y).          cA   =         u 8712;   8727; cu  u), u ranges                                                                     set sequences    8727; represented                                                                     automaton  equation represents sums <br />                                                                     obtained u, times occurs Algorithm 1 Rational Kernel Computation                             u multiplied weight  value) associated INPUT: pair sequences (x, y) WFST U                        sequence u automaton  computed   compute N using composition N = Mx  9702; U  9702;                  Example 2). <br /> (ii) compute sum paths N using                           similarity measure weighted <br /> shortest-distance algorithm, equal U(x, y).             automata A1 <span id='am-716' about='obo:IMR_0702085' typeof='owl:Thing'><span id='am-717' property="oboInOwl:hasExactSynonym" content="ARCL" datatype="xsd:string"></span><span id='am-718' property="oboInOwl:hasExactSynonym" content="VPH1" datatype="xsd:string"></span><span id='am-719' property="rdfs:label" content="ATP6V0A2" datatype="xsd:string"></span><span id='am-720' property="rdfs:label" content="a2" datatype="xsd:string"></span><span id='am-721' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-722' property="rdfs:label" content="A2" datatype="xsd:string"></span><span id='am-723' property="rdfs:label" content="TJ6M" datatype="xsd:string"></span><span id='am-724' property="rdfs:label" content="ATPase, H+ TRANSPORTING, LYSOSOMAL, V0 SUBUNIT A2" datatype="xsd:string"></span><span id='am-725' property="rdfs:label" content="ATP6A2" datatype="xsd:string"></span><span id='am-726' property="oboInOwl:hasExactSynonym" content="Vacuolar proton translocating ATPase 116 kDa subunit a isoform 2" datatype="xsd:string"></span><span id='am-727' property="oboInOwl:hasExactSynonym" content="A2" datatype="xsd:string"></span><span id='am-728' property="rdfs:label" content="ATP6a2" datatype="xsd:string"></span><span id='am-729' property="oboInOwl:hasExactSynonym" content="J6B7" datatype="xsd:string"></span><span id='am-730' property="rdfs:label" content="WSS" datatype="xsd:string"></span><span id='am-731' property="rdfs:label" content="A2V-ATPase" datatype="xsd:string"></span><span id='am-732' property="oboInOwl:hasExactSynonym" content="Lysosomal H(+)-transporting ATPase V0 subunit a2" datatype="xsd:string"></span><span id='am-733' property="oboInOwl:hasDbXref" content="UniProt:Q9Y487" datatype="xsd:string"></span><span id='am-734' property="oboInOwl:hasExactSynonym" content="Stv1" datatype="xsd:string"></span><span id='am-735' property="oboInOwl:hasExactSynonym" content="TJ6M" datatype="xsd:string"></span><span id='am-736' property="oboInOwl:hasExactSynonym" content="ATPase, H+ TRANSPORTING, LYSOSOMAL, V0 SUBUNIT A2" datatype="xsd:string"></span><span id='am-737' property="rdfs:label" content="V-ATPase 116 kDa isoform a2" datatype="xsd:string"></span><span id='am-738' property="rdfs:label" content="ATP6N1D" datatype="xsd:string"></span><span id='am-739' property="oboInOwl:hasExactSynonym" content="ATP6V0A2" datatype="xsd:string"></span><span id='am-740' property="rdfs:label" content="ARCL" datatype="xsd:string"></span><span id='am-741' property="rdfs:label" content="Vacuolar proton translocating ATPase 116 kDa subunit a isoform 2" datatype="xsd:string"></span><span id='am-742' property="oboInOwl:id" content="IMR:0702085" datatype="xsd:string"></span><span id='am-743' property="rdfs:label" content="Stv1" datatype="xsd:string"></span><span id='am-744' property="rdfs:label" content="TJ6s" datatype="xsd:string"></span><span id='am-745' property="rdfs:label" content="V-type proton ATPase 116 kDa subunit a isoform 2" datatype="xsd:string"></span><span id='am-746' property="oboInOwl:hasExactSynonym" content="V-type proton ATPase 116 kDa subunit a isoform 2" datatype="xsd:string"></span><span id='am-747' property="oboInOwl:hasExactSynonym" content="TJ6" datatype="xsd:string"></span><span id='am-748' property="rdfs:label" content="ATPase, H+ transporting, lysosomal V0 subunit a2" datatype="xsd:string"></span><span id='am-749' property="rdfs:label" content="RTF" datatype="xsd:string"></span><span id='am-750' property="oboInOwl:hasExactSynonym" content="ATPase, H+ transporting, lysosomal V0 subunit a2" datatype="xsd:string"></span><span id='am-751' property="oboInOwl:hasExactSynonym" content="V-ATPase 116 kDa isoform a2" datatype="xsd:string"></span><span id='am-752' property="rdfs:label" content="STV1" datatype="xsd:string"></span><span id='am-753' property="rdfs:label" content="Lysosomal H(+)-transporting ATPase V0 subunit a2" datatype="xsd:string"></span><span id='am-754' property="oboInOwl:hasExactSynonym" content="STV1" datatype="xsd:string"></span><span id='am-755' property="rdfs:label" content="Vph1" datatype="xsd:string"></span><span id='am-756' property="rdfs:label" content="VPP2_HUMAN" datatype="xsd:string"></span><span id='am-757' property="oboInOwl:hasExactSynonym" content="ATP6a2" datatype="xsd:string"></span><span id='am-758' property="rdfs:label" content="J6B7" datatype="xsd:string"></span><span id='am-759' property="oboInOwl:hasExactSynonym" content="ATP6N1D" datatype="xsd:string"></span><span id='am-760' property="oboInOwl:hasExactSynonym" content="WSS" datatype="xsd:string"></span><span id='am-761' property="oboInOwl:hasExactSynonym" content="ATP6A2" datatype="xsd:string"></span><span id='am-762' property="oboInOwl:hasExactSynonym" content="RTF" datatype="xsd:string"></span><span id='am-763' property="oboInOwl:hasExactSynonym" content="a2" datatype="xsd:string"></span><span id='am-764' property="oboInOwl:hasExactSynonym" content="Vph1" datatype="xsd:string"></span><span id='am-765' property="rdfs:label" content="VPH1" datatype="xsd:string"></span><span id='am-766' property="oboInOwl:hasExactSynonym" content="TJ6s" datatype="xsd:string"></span><span id='am-767' property="rdfs:label" content="TJ6" datatype="xsd:string"></span><span id='am-768' property="oboInOwl:hasExactSynonym" content="TJ6S" datatype="xsd:string"></span><span id='am-769' property="rdfs:label" content="TJ6S" datatype="xsd:string"></span><span id='am-770' property="oboInOwl:hasExactSynonym" content="A2V-ATPase" datatype="xsd:string"></span>A2</span> , according n-gram kernel kn , RESULTS: value k(x, y) = U(x, y)                                 defined  <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                           Page 5 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />                         <br />     kn (A1 , <span id='am-771' about='obo:IMR_0702085' typeof='owl:Thing'><span id='am-772' property="oboInOwl:hasExactSynonym" content="Stv1" datatype="xsd:string"></span><span id='am-773' property="rdfs:label" content="J6B7" datatype="xsd:string"></span><span id='am-774' property="rdfs:label" content="TJ6s" datatype="xsd:string"></span><span id='am-775' property="rdfs:label" content="Vacuolar proton translocating ATPase 116 kDa subunit a isoform 2" datatype="xsd:string"></span><span id='am-776' property="rdfs:label" content="TJ6S" datatype="xsd:string"></span><span id='am-777' property="rdfs:label" content="ATPase, H+ TRANSPORTING, LYSOSOMAL, V0 SUBUNIT A2" datatype="xsd:string"></span><span id='am-778' property="rdfs:label" content="Stv1" datatype="xsd:string"></span><span id='am-779' property="oboInOwl:hasExactSynonym" content="J6B7" datatype="xsd:string"></span><span id='am-780' property="oboInOwl:hasExactSynonym" content="ATP6V0A2" datatype="xsd:string"></span><span id='am-781' property="oboInOwl:hasExactSynonym" content="a2" datatype="xsd:string"></span><span id='am-782' property="oboInOwl:hasExactSynonym" content="V-type proton ATPase 116 kDa subunit a isoform 2" datatype="xsd:string"></span><span id='am-783' property="oboInOwl:id" content="IMR:0702085" datatype="xsd:string"></span><span id='am-784' property="oboInOwl:hasExactSynonym" content="STV1" datatype="xsd:string"></span><span id='am-785' property="oboInOwl:hasExactSynonym" content="ATP6a2" datatype="xsd:string"></span><span id='am-786' property="oboInOwl:hasExactSynonym" content="A2" datatype="xsd:string"></span><span id='am-787' property="rdfs:label" content="V-type proton ATPase 116 kDa subunit a isoform 2" datatype="xsd:string"></span><span id='am-788' property="oboInOwl:hasExactSynonym" content="Lysosomal H(+)-transporting ATPase V0 subunit a2" datatype="xsd:string"></span><span id='am-789' property="rdfs:label" content="a2" datatype="xsd:string"></span><span id='am-790' property="rdfs:label" content="TJ6M" datatype="xsd:string"></span><span id='am-791' property="oboInOwl:hasExactSynonym" content="TJ6M" datatype="xsd:string"></span><span id='am-792' property="rdfs:label" content="RTF" datatype="xsd:string"></span><span id='am-793' property="oboInOwl:hasExactSynonym" content="ATPase, H+ transporting, lysosomal V0 subunit a2" datatype="xsd:string"></span><span id='am-794' property="oboInOwl:hasExactSynonym" content="ARCL" datatype="xsd:string"></span><span id='am-795' property="rdfs:label" content="TJ6" datatype="xsd:string"></span><span id='am-796' property="oboInOwl:hasExactSynonym" content="ATP6N1D" datatype="xsd:string"></span><span id='am-797' property="oboInOwl:hasExactSynonym" content="TJ6S" datatype="xsd:string"></span><span id='am-798' property="oboInOwl:hasExactSynonym" content="RTF" datatype="xsd:string"></span><span id='am-799' property="oboInOwl:hasExactSynonym" content="A2V-ATPase" datatype="xsd:string"></span><span id='am-800' property="rdfs:label" content="WSS" datatype="xsd:string"></span><span id='am-801' property="rdfs:label" content="ATPase, H+ transporting, lysosomal V0 subunit a2" datatype="xsd:string"></span><span id='am-802' property="oboInOwl:hasExactSynonym" content="TJ6" datatype="xsd:string"></span><span id='am-803' property="rdfs:label" content="ATP6A2" datatype="xsd:string"></span><span id='am-804' property="rdfs:label" content="A2" datatype="xsd:string"></span><span id='am-805' property="oboInOwl:hasExactSynonym" content="V-ATPase 116 kDa isoform a2" datatype="xsd:string"></span><span id='am-806' property="rdfs:label" content="V-ATPase 116 kDa isoform a2" datatype="xsd:string"></span><span id='am-807' property="rdfs:label" content="Vph1" datatype="xsd:string"></span><span id='am-808' property="rdfs:label" content="ATP6a2" datatype="xsd:string"></span><span id='am-809' property="oboInOwl:hasExactSynonym" content="VPH1" datatype="xsd:string"></span><span id='am-810' property="rdfs:label" content="ARCL" datatype="xsd:string"></span><span id='am-811' property="rdfs:label" content="ATP6V0A2" datatype="xsd:string"></span><span id='am-812' property="oboInOwl:hasExactSynonym" content="Vacuolar proton translocating ATPase 116 kDa subunit a isoform 2" datatype="xsd:string"></span><span id='am-813' property="rdfs:label" content="VPP2_HUMAN" datatype="xsd:string"></span><span id='am-814' property="oboInOwl:hasExactSynonym" content="WSS" datatype="xsd:string"></span><span id='am-815' property="oboInOwl:hasDbXref" content="UniProt:Q9Y487" datatype="xsd:string"></span><span id='am-816' property="rdfs:label" content="A2V-ATPase" datatype="xsd:string"></span><span id='am-817' property="rdfs:label" content="STV1" datatype="xsd:string"></span><span id='am-818' property="oboInOwl:hasExactSynonym" content="TJ6s" datatype="xsd:string"></span><span id='am-819' property="rdfs:label" content="ATP6N1D" datatype="xsd:string"></span><span id='am-820' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-821' property="rdfs:label" content="Lysosomal H(+)-transporting ATPase V0 subunit a2" datatype="xsd:string"></span><span id='am-822' property="oboInOwl:hasExactSynonym" content="ATPase, H+ TRANSPORTING, LYSOSOMAL, V0 SUBUNIT A2" datatype="xsd:string"></span><span id='am-823' property="rdfs:label" content="VPH1" datatype="xsd:string"></span><span id='am-824' property="oboInOwl:hasExactSynonym" content="ATP6A2" datatype="xsd:string"></span><span id='am-825' property="oboInOwl:hasExactSynonym" content="Vph1" datatype="xsd:string"></span>A2</span> ) =           (A1  9702; Tn  9702; Tn 8722;1  9702; A2 )(x, y)             di binary values (e.g., pair (xi , yj ) classified                       x,y 8712;X                                            +1  8722;1), = 1, . . . , n, j = 1, . . . , n mapping <br />                                                                        function 	, Pairwise SVM methods opti- <br />                  =            cA1  cA2                      (1) <br />                                                                        <span id='am-398' about='obo:IMR_0703972' typeof='owl:Thing'><span id='am-399' property="oboInOwl:hasExactSynonym" content="OKT8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-400' property="oboInOwl:hasExactSynonym" content="CD8A" datatype="xsd:string"></span><span id='am-401' property="oboInOwl:hasExactSynonym" content="CD8 ANTIGEN, ALPHA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-402' property="oboInOwl:hasExactSynonym" content="CD8a" datatype="xsd:string"></span><span id='am-403' property="rdfs:label" content="CD8A" datatype="xsd:string"></span><span id='am-404' property="oboInOwl:hasExactSynonym" content="T-lymphocyte differentiation antigen T8/Leu-2" datatype="xsd:string"></span><span id='am-405' property="rdfs:label" content="T-cell surface glycoprotein CD8 alpha chain" datatype="xsd:string"></span><span id='am-406' property="oboInOwl:hasExactSynonym" content="Leu2" datatype="xsd:string"></span><span id='am-407' property="rdfs:label" content="Leu2" datatype="xsd:string"></span><span id='am-408' property="oboInOwl:hasExactSynonym" content="CD8" datatype="xsd:string"></span><span id='am-409' property="oboInOwl:id" content="IMR:0703972" datatype="xsd:string"></span><span id='am-410' property="oboInOwl:hasExactSynonym" content="T8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-411' property="rdfs:label" content="CD8" datatype="xsd:string"></span><span id='am-412' property="rdfs:label" content="OKT8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-413' property="rdfs:label" content="CD8a molecule" datatype="xsd:string"></span><span id='am-414' property="rdfs:label" content="CD8A_HUMAN" datatype="xsd:string"></span><span id='am-415' property="oboInOwl:hasExactSynonym" content="MAL" datatype="xsd:string"></span><span id='am-416' property="oboInOwl:hasExactSynonym" content="CD8a molecule" datatype="xsd:string"></span><span id='am-417' property="oboInOwl:hasExactSynonym" content="p32" datatype="xsd:string"></span><span id='am-418' property="rdfs:label" content="CD8 ANTIGEN, ALPHA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-419' property="oboInOwl:hasDbXref" content="UniProt:P01732" datatype="xsd:string"></span><span id='am-420' property="oboInOwl:hasExactSynonym" content="T-cell surface glycoprotein CD8 alpha chain" datatype="xsd:string"></span><span id='am-421' property="rdfs:label" content="CD8a" datatype="xsd:string"></span><span id='am-422' property="rdfs:label" content="T-lymphocyte differentiation antigen T8/Leu-2" datatype="xsd:string"></span><span id='am-423' property="rdfs:label" content="T8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-424' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-425' property="rdfs:label" content="MAL" datatype="xsd:string"></span><span id='am-426' property="rdfs:label" content="p32" datatype="xsd:string"></span>mal</span> hyperplane, wT 	(xi , yi ) + b = 0, separate                        =n <br />                                                                        points categories. solutions based   Based definition using Algorithm 1,                  dual formalism optimization problem described <br /> n-gram rational kernel constructed time                      Cortes et al. [33]. case decision function  <br /> O( | + |Mx | +  |), described Allauzen et al. [16]                           n                              Mohri et al. [28].                                                       f (x, y) =        945;ij K (xi , yj ), (x, y) + b, <br />                                                                                             j <br />   Yu et al. [29] verified n-gram sequence kernels good predict protein interactions. address concerns experiments combin-                K pairwise kernel, (xi , yj ) set train- <br /> ing n-gram kernels include evolutionary                ing examples,  945; obtained Lagrange Multipliers <br /> information.                                                           function w  normal vector) b offset <br />                                                                        hyperplane   Cortes et al. [33] Pairwise kernels <br />                                                                        details). case,  945; b  8220;learned 65533;? parame- apply kernel methods problem predicting <br />                                                                        ters training process.  f classifies new <br /> relationships given entities, e., pairwise pre- <br />                                                                        pairs (x, y). example, f (x, y) &gt;= 0, (x, y) classified <br /> diction. Models solve problem input <br />                                                                        +1, (x, y) classified  8722;1. instances, output relationship  Kernels used models need define simi-                Metabolic networks <br /> larities arbitrary pairs entities. Typically,           work, metabolic network represented construction pairwise kernels K based simple             graph, vertices enzymes, kernels k, k : X  215; X  8594; R. paper differ-             edges <span id='am-331' about='obo:IMR_0000207' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001'><span id='am-332' property="rdfs:label" content="enzyme" datatype="xsd:string"></span><span id='am-333' property="obo:IAO_0000115" content="Protein that catalyze a specific chemical reaction." datatype="xsd:string"></span><span id='am-334' property="oboInOwl:hasDbXref" content="GO:0003824" datatype="xsd:string"></span><span id='am-335' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-336' property="oboInOwl:id" content="IMR:0000207" datatype="xsd:string"></span>enzyme</span>-enzyme relations  proteins <span id='am-685' about='obo:IMR_0702309' typeof='owl:Thing'><span id='am-686' property="oboInOwl:hasExactSynonym" content="CD73" datatype="xsd:string"></span><span id='am-687' property="oboInOwl:hasExactSynonym" content="NT5E" datatype="xsd:string"></span><span id='am-688' property="oboInOwl:hasExactSynonym" content="ECTO-5-PRIME NUCLEOTIDASE" datatype="xsd:string"></span><span id='am-689' property="rdfs:label" content="ECTO-5-PRIME NUCLEOTIDASE" datatype="xsd:string"></span><span id='am-690' property="oboInOwl:hasExactSynonym" content="5&apos;-nucleotidase, ecto (CD73)" datatype="xsd:string"></span><span id='am-691' property="rdfs:label" content="NT5" datatype="xsd:string"></span><span id='am-692' property="rdfs:label" content="5NTD_HUMAN" datatype="xsd:string"></span><span id='am-693' property="rdfs:label" content="E5NT" datatype="xsd:string"></span><span id='am-694' property="rdfs:label" content="5&apos;-nucleotidase" datatype="xsd:string"></span><span id='am-695' property="rdfs:label" content="EC 3.1.3.5" datatype="xsd:string"></span><span id='am-696' property="oboInOwl:id" content="IMR:0702309" datatype="xsd:string"></span><span id='am-697' property="oboInOwl:hasExactSynonym" content="NUCLEOTIDASE, ECTO-5-PRIME" datatype="xsd:string"></span><span id='am-698' property="rdfs:label" content="Ecto-5&apos;-nucleotidase" datatype="xsd:string"></span><span id='am-699' property="rdfs:label" content="NTE" datatype="xsd:string"></span><span id='am-700' property="oboInOwl:hasExactSynonym" content="E5NT" datatype="xsd:string"></span><span id='am-701' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-702' property="oboInOwl:hasExactSynonym" content="Ecto-5&apos;-nucleotidase" datatype="xsd:string"></span><span id='am-703' property="rdfs:label" content="NUCLEOTIDASE, ECTO-5-PRIME" datatype="xsd:string"></span><span id='am-704' property="rdfs:label" content="5&apos;-nucleotidase, ecto (CD73)" datatype="xsd:string"></span><span id='am-705' property="oboInOwl:hasExactSynonym" content="5&apos;-NT" datatype="xsd:string"></span><span id='am-706' property="rdfs:label" content="CD73" datatype="xsd:string"></span><span id='am-707' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.5" datatype="xsd:string"></span><span id='am-708' property="oboInOwl:hasDbXref" content="UniProt:P21589" datatype="xsd:string"></span><span id='am-709' property="oboInOwl:hasExactSynonym" content="eNT" datatype="xsd:string"></span><span id='am-710' property="rdfs:label" content="eNT" datatype="xsd:string"></span><span id='am-711' property="oboInOwl:hasExactSynonym" content="5&apos;-nucleotidase" datatype="xsd:string"></span><span id='am-712' property="oboInOwl:hasExactSynonym" content="NTE" datatype="xsd:string"></span><span id='am-713' property="oboInOwl:hasExactSynonym" content="NT5" datatype="xsd:string"></span><span id='am-714' property="rdfs:label" content="5&apos;-NT" datatype="xsd:string"></span><span id='am-715' property="rdfs:label" content="NT5E" datatype="xsd:string"></span>ent</span> pairwise kernels investigated: Direct Sum Learning             enzymes catalyze successive reactions known path- <br /> Pairwise Kernel [21], Tensor Learning Pairwise Kernel               ways). Figure 2 represents graphical transition Kronecker Kernel) [22,30,31], Metric Learning Pairwise                 metabolic pathway graph. <br /> Kernel [23] Cartesian Pairwise Kernel [10].                           traditional representation metabolic path- <br />   pairwise functions guarantee symmetry                  way, enzymes vertices (nodes), metabolites pairwise kernels K, e., K((x1 , y1 ), (x2 , <span id='am-267' about='obo:IMR_0704767' typeof='owl:Thing'><span id='am-268' property="rdfs:label" content="Really interesting new gene 10 protein" datatype="xsd:string"></span><span id='am-269' property="rdfs:label" content="Low molecular mass protein 7" datatype="xsd:string"></span><span id='am-270' property="rdfs:label" content="Multicatalytic endopeptidase complex subunit C13" datatype="xsd:string"></span><span id='am-271' property="rdfs:label" content="beta5i" datatype="xsd:string"></span><span id='am-272' property="oboInOwl:hasExactSynonym" content="Proteasome subunit beta-5i" datatype="xsd:string"></span><span id='am-273' property="rdfs:label" content="Macropain subunit C13" datatype="xsd:string"></span><span id='am-274' property="rdfs:label" content="PSB8_HUMAN" datatype="xsd:string"></span><span id='am-275' property="rdfs:label" content="PROTEASOME-RELATED GENE 7" datatype="xsd:string"></span><span id='am-276' property="oboInOwl:hasExactSynonym" content="Proteasome component C13" datatype="xsd:string"></span><span id='am-277' property="rdfs:label" content="RING10" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) subunit, beta type, 8 (large multifunctional peptidase 7)" datatype="xsd:string"></span><span id='am-279' property="rdfs:label" content="EC 3.4.25.1" datatype="xsd:string"></span><span id='am-280' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-281' property="oboInOwl:hasExactSynonym" content="PROTEASOME SUBUNIT, BETA-TYPE, 8" datatype="xsd:string"></span><span id='am-282' property="oboInOwl:hasExactSynonym" content="Multicatalytic endopeptidase complex subunit C13" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasExactSynonym" content="Really interesting new gene 10 protein" datatype="xsd:string"></span><span id='am-284' property="rdfs:label" content="MGC1491" datatype="xsd:string"></span><span id='am-285' property="rdfs:label" content="LMP7" datatype="xsd:string"></span><span id='am-286' property="rdfs:label" content="PSMB8" datatype="xsd:string"></span><span id='am-287' property="rdfs:label" content="Proteasome subunit beta type-8" datatype="xsd:string"></span><span id='am-288' property="oboInOwl:id" content="IMR:0704767" datatype="xsd:string"></span><span id='am-289' property="oboInOwl:hasExactSynonym" content="RING10" datatype="xsd:string"></span><span id='am-290' property="rdfs:label" content="Proteasome subunit beta-5i" datatype="xsd:string"></span><span id='am-291' property="oboInOwl:hasExactSynonym" content="PSMB5i" datatype="xsd:string"></span><span id='am-292' property="rdfs:label" content="PROTEASOME SUBUNIT, BETA-TYPE, 8" datatype="xsd:string"></span><span id='am-293' property="oboInOwl:hasExactSynonym" content="PROTEASOME-RELATED GENE 7" datatype="xsd:string"></span><span id='am-294' property="rdfs:label" content="D6S216E" datatype="xsd:string"></span><span id='am-295' property="oboInOwl:hasExactSynonym" content="PSMB8" datatype="xsd:string"></span><span id='am-296' property="rdfs:label" content="proteasome (prosome, macropain) subunit, beta type, 8 (large multifunctional peptidase 7)" datatype="xsd:string"></span><span id='am-297' property="oboInOwl:hasExactSynonym" content="beta5i" datatype="xsd:string"></span><span id='am-298' property="oboInOwl:hasExactSynonym" content="Macropain subunit C13" datatype="xsd:string"></span><span id='am-299' property="oboInOwl:hasExactSynonym" content="D6S216E" datatype="xsd:string"></span><span id='am-300' property="oboInOwl:hasExactSynonym" content="Proteasome subunit beta type-8" datatype="xsd:string"></span><span id='am-301' property="rdfs:label" content="PSMB5i" datatype="xsd:string"></span><span id='am-302' property="rdfs:label" content="Proteasome component C13" datatype="xsd:string"></span><span id='am-303' property="oboInOwl:hasExactSynonym" content="MGC1491" datatype="xsd:string"></span><span id='am-304' property="oboInOwl:hasExactSynonym" content="D6S216" datatype="xsd:string"></span><span id='am-305' property="oboInOwl:hasDbXref" content="UniProt:P28062" datatype="xsd:string"></span><span id='am-306' property="oboInOwl:hasExactSynonym" content="Y2" datatype="xsd:string"></span><span id='am-307' property="rdfs:label" content="LARGE MULTIFUNCTIONAL PROTEASE 7" datatype="xsd:string"></span><span id='am-308' property="oboInOwl:hasExactSynonym" content="Low molecular mass protein 7" datatype="xsd:string"></span><span id='am-309' property="rdfs:label" content="D6S216" datatype="xsd:string"></span><span id='am-310' property="oboInOwl:hasExactSynonym" content="EC 3.4.25.1" datatype="xsd:string"></span><span id='am-311' property="rdfs:label" content="Y2" datatype="xsd:string"></span><span id='am-312' property="oboInOwl:hasExactSynonym" content="LARGE MULTIFUNCTIONAL PROTEASE 7" datatype="xsd:string"></span><span id='am-313' property="oboInOwl:hasExactSynonym" content="LMP7" datatype="xsd:string"></span>y2</span> )) =           edges (branches). Following Yamanishi [9], represent <br /> K((x2 , y2 ), (x1 , y1 )), x1 , x2 , y1 , y2  8712; X.    differently, interactions pairs simple kernel k PDS (satisfies Mercer condition),               enzymes considered discrete data points. exam- resulting pairwise kernel K PDS,               ple, Figure 2 , enzyme numbered <span id='am-226' about='obo:IMR_0702468' typeof='owl:Thing'><span id='am-227' property="rdfs:label" content="glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-228' property="rdfs:label" content="Phosphohexose isomerase" datatype="xsd:string"></span><span id='am-229' property="oboInOwl:hasExactSynonym" content="Phosphohexose isomerase" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:hasExactSynonym" content="AMF" datatype="xsd:string"></span><span id='am-231' property="rdfs:label" content="NLK" datatype="xsd:string"></span><span id='am-232' property="oboInOwl:hasExactSynonym" content="PHI" datatype="xsd:string"></span><span id='am-233' property="rdfs:label" content="Phosphoglucose isomerase" datatype="xsd:string"></span><span id='am-234' property="oboInOwl:hasExactSynonym" content="GPI" datatype="xsd:string"></span><span id='am-235' property="rdfs:label" content="AMF" datatype="xsd:string"></span><span id='am-236' property="rdfs:label" content="DKFZp686C13233" datatype="xsd:string"></span><span id='am-237' property="rdfs:label" content="SA-36" datatype="xsd:string"></span><span id='am-238' property="rdfs:label" content="Glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-239' property="rdfs:label" content="G6PI_HUMAN" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:hasExactSynonym" content="SA36" datatype="xsd:string"></span><span id='am-241' property="rdfs:label" content="SA36" datatype="xsd:string"></span><span id='am-242' property="oboInOwl:hasExactSynonym" content="GLUCOSE PHOSPHATE ISOMERASE" datatype="xsd:string"></span><span id='am-243' property="oboInOwl:hasExactSynonym" content="DKFZp686C13233" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasExactSynonym" content="SA-36" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasExactSynonym" content="glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasExactSynonym" content="GNPI" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Sperm antigen 36" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasExactSynonym" content="NLK" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:hasExactSynonym" content="Autocrine motility factor" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasDbXref" content="UniProt:P06744" datatype="xsd:string"></span><span id='am-251' property="oboInOwl:hasExactSynonym" content="Sperm antigen 36" datatype="xsd:string"></span><span id='am-252' property="rdfs:label" content="GPI" datatype="xsd:string"></span><span id='am-253' property="rdfs:label" content="Neuroleukin" datatype="xsd:string"></span><span id='am-254' property="rdfs:label" content="GNPI" datatype="xsd:string"></span><span id='am-255' property="oboInOwl:hasExactSynonym" content="EC 5.3.1.9" datatype="xsd:string"></span><span id='am-256' property="rdfs:label" content="GLUCOSE PHOSPHATE ISOMERASE" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasExactSynonym" content="Neuroleukin" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:id" content="IMR:0702468" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-260' property="oboInOwl:hasExactSynonym" content="Phosphoglucose isomerase" datatype="xsd:string"></span><span id='am-261' property="rdfs:label" content="Autocrine motility factor" datatype="xsd:string"></span><span id='am-262' property="rdfs:label" content="EC 5.3.1.9" datatype="xsd:string"></span><span id='am-263' property="rdfs:label" content="PHI" datatype="xsd:string"></span><span id='am-264' property="rdfs:label" content="PGI" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:hasExactSynonym" content="PGI" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasExactSynonym" content="Glucose-6-phosphate isomerase" datatype="xsd:string"></span>EC 5.3.1.9</span> pairwise kernels defined [10,32].                            create D-fructose-6-phosphate product,                                                                        turn used substrate enzyme numbered <br /> Pairwise support vector machine <br />                                                                        <span id='am-32' about='obo:IMR_0702517' typeof='owl:Thing'><span id='am-33' property="rdfs:label" content="Phosphohexokinase" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="PFK1" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="Phosphofructo-1-kinase isozyme A" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="PFKA" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="PHOSPHOFRUCTOKINASE, MUSCLE TYPE" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="PFK1" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="PFKA" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="PFKM" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="GSD7" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="PFK, MUSCLE TYPE" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="Phosphofructokinase-M" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="Phosphohexokinase" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="PFK-A" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Phosphofructokinase-M" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="PFK-A" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="PFK-1" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="PFKM" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="phosphofructokinase, muscle" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="K6PF_HUMAN" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="PFK-1" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="phosphofructokinase, muscle" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="6-phosphofructokinase, muscle type" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="PFKX" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="MGC8699" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="GSD7" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="PHOSPHOFRUCTOKINASE, MUSCLE TYPE" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="EC 2.7.1.11" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="MGC8699" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0702517" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="6-phosphofructokinase, muscle type" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="UniProt:P08237" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Phosphofructo-1-kinase isozyme A" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="PFK, MUSCLE TYPE" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Phosphofructokinase 1" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="PFKX" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="Phosphofructokinase 1" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="EC 2.7.1.11" datatype="xsd:string"></span>EC 2.7.1.11</span>. means enzyme-enzyme <span id='am-1155' about='obo:IMR_0704199' typeof='owl:Thing'><span id='am-1156' property="oboInOwl:hasExactSynonym" content="MGC131774" datatype="xsd:string"></span><span id='am-1157' property="oboInOwl:hasExactSynonym" content="V-REL AVIAN RETICULOENDOTHELIOSIS VIRAL ONCOGENE HOMOLOG A" datatype="xsd:string"></span><span id='am-1158' property="oboInOwl:id" content="IMR:0704199" datatype="xsd:string"></span><span id='am-1159' property="oboInOwl:hasExactSynonym" content="NUCLEAR FACTOR KAPPA-B, SUBUNIT 3" datatype="xsd:string"></span><span id='am-1160' property="oboInOwl:hasDbXref" content="UniProt:Q04206" datatype="xsd:string"></span><span id='am-1161' property="oboInOwl:hasExactSynonym" content="v-rel reticuloendotheliosis viral oncogene homolog A (avian)" datatype="xsd:string"></span><span id='am-1162' property="rdfs:label" content="Transcription factor p65" datatype="xsd:string"></span><span id='am-1163' property="rdfs:label" content="NFKB, p65 SUBUNIT" datatype="xsd:string"></span><span id='am-1164' property="oboInOwl:hasExactSynonym" content="NUCLEAR FACTOR OF KAPPA LIGHT CHAIN GENE ENHANCER IN B CELLS 3" datatype="xsd:string"></span><span id='am-1165' property="oboInOwl:hasExactSynonym" content="NFKB, p65 SUBUNIT" datatype="xsd:string"></span><span id='am-1166' property="rdfs:label" content="RELA" datatype="xsd:string"></span><span id='am-1167' property="oboInOwl:hasExactSynonym" content="Transcription factor p65" datatype="xsd:string"></span><span id='am-1168' property="rdfs:label" content="Nuclear factor NF-kappa-B p65 subunit" datatype="xsd:string"></span><span id='am-1169' property="oboInOwl:hasExactSynonym" content="Nuclear factor of kappa light polypeptide gene enhancer in B-cells 3" datatype="xsd:string"></span><span id='am-1170' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1171' property="oboInOwl:hasExactSynonym" content="TRANSCRIPTION FACTOR NFKB3" datatype="xsd:string"></span><span id='am-1172' property="rdfs:label" content="V-REL AVIAN RETICULOENDOTHELIOSIS VIRAL ONCOGENE HOMOLOG A" datatype="xsd:string"></span><span id='am-1173' property="rdfs:label" content="MGC131774" datatype="xsd:string"></span><span id='am-1174' property="oboInOwl:hasExactSynonym" content="NFKB3" datatype="xsd:string"></span><span id='am-1175' property="rdfs:label" content="v-rel reticuloendotheliosis viral oncogene homolog A (avian)" datatype="xsd:string"></span><span id='am-1176' property="rdfs:label" content="TRANSCRIPTION FACTOR NFKB3" datatype="xsd:string"></span><span id='am-1177' property="rdfs:label" content="Nuclear factor of kappa light polypeptide gene enhancer in B-cells 3" datatype="xsd:string"></span><span id='am-1178' property="rdfs:label" content="p65" datatype="xsd:string"></span><span id='am-1179' property="oboInOwl:hasExactSynonym" content="p65" datatype="xsd:string"></span><span id='am-1180' property="rdfs:label" content="NUCLEAR FACTOR OF KAPPA LIGHT CHAIN GENE ENHANCER IN B CELLS 3" datatype="xsd:string"></span><span id='am-1181' property="rdfs:label" content="NFKB3" datatype="xsd:string"></span><span id='am-1182' property="rdfs:label" content="TF65_HUMAN" datatype="xsd:string"></span><span id='am-1183' property="oboInOwl:hasExactSynonym" content="RELA" datatype="xsd:string"></span><span id='am-1184' property="rdfs:label" content="NUCLEAR FACTOR KAPPA-B, SUBUNIT 3" datatype="xsd:string"></span><span id='am-1185' property="oboInOwl:hasExactSynonym" content="Nuclear factor NF-kappa-B p65 subunit" datatype="xsd:string"></span>rela</span>- rationale preceding discussion represent- <br />                                                                        tion EC 5.3.1.9 <span id='am-71' about='obo:IMR_0702518' typeof='owl:Thing'><span id='am-72' property="oboInOwl:hasExactSynonym" content="Phosphohexokinase" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="UniProt:P17858" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="EC 2.7.1.11" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Phosphofructokinase 1" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="PFK-B" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="6-phosphofructokinase, liver type" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasExactSynonym" content="PFKL" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="PFKL" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="EC 2.7.1.11" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasExactSynonym" content="DKFZp686G1648" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="PHOSPHOFRUCTOKINASE, LIVER TYPE" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="6-phosphofructokinase, liver type" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="Phosphofructo-1-kinase isozyme B" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasExactSynonym" content="FLJ30173" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="PHOSPHOFRUCTOKINASE, LIVER TYPE" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="DKFZp686G1648" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="FLJ40909" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="PFK-B" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="PFK, LIVER TYPE" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="Phosphofructokinase 1" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="FLJ30173" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="K6PL_HUMAN" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasExactSynonym" content="Phosphofructo-1-kinase isozyme B" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="FLJ40909" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasExactSynonym" content="DKFZp686L2097" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasExactSynonym" content="phosphofructokinase, liver" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:id" content="IMR:0702518" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="DKFZp686L2097" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="phosphofructokinase, liver" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="Phosphohexokinase" datatype="xsd:string"></span><span id='am-103' property="rdfs:label" content="PFK, LIVER TYPE" datatype="xsd:string"></span>EC 2.7.1.11</span>.  create <br /> ing disparate types data kernels enable                                                                        graph enzyme-enzyme relations edges <br /> use machine learning formalisms Support <br />                                                                        enzymes nodes shown Figure 2(b). Vector Machines (SVMs). SVMs used classifica- <br />                                                                        relation enzymes, relation clas- <br /> tion regression analysis, defined supervised models <br />                                                                        sified +1  e., interacting pair). Enzyme-enzyme pairs associated learning algorithms [33]. research, <br />                                                                        relation exists classified  8722;1 (non- use SVMs classification. SVMs represents data <br />                                                                        interacting pairs). Figure 2(c) describes classifica- vectors vector space  e., input feature space). <br />                                                                        tions, used training set SVM method. training set, entities xi (vectors) classified categories given. SVM trained                Using pairwise kernel SVM predict metabolic <br /> hyperplane separates vector space parts.               networks feature space groups entities                input data, considered training example category.  new entity x classified              dataset ((xi , yi ), di ), set known pairs enzymes <br /> depending location feature space related            genes) classified categories (interacting hyperplane [33].                                                       non-interacting pairs). Figure 3  shows example   Pairwise Support Vector Machines, instead, classify pair             input data, obtained metabolic network entities (x, y) [32]. Let formally define binary             described Figure 2(c). Figure 3 , enzymes rep- <br /> Pairwise Support Vector Machine formulation, following                 resented EC number   gene nomenclature <br /> Brunner et al. [32]: given training data ((xi , yj ), di ),     . <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                                    Page 6 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />  <br />  Figure 2 Conversion metabolic network graph representation.   Glycolysis Pathways, BioCyc Database [5,6]. <br />  (b) resulting graph nodes (enzymes) edges (enzyme-enzyme relations). (c) Table represents known enzymes relations (EC <br />  numbers related classified +1 non-related -1). <br />  <br />  <br />   Figure 3(b) represents example pairwise                         Pairwise SVM based dual formalism kernel (K((x1 , y1 ), (x2 , y2 ))). state art pair-        optimization problem represented Figure 3(c). <span id='am-936' about='obo:IMR_0001162' typeof='owl:Thing obo:IMR_0001160 obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000428'><span id='am-937' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-938' property="oboInOwl:id" content="IMR:0001162" datatype="xsd:string"></span><span id='am-939' property="rdfs:label" content="Wise" datatype="xsd:string"></span>wise</span> kernels mentioned  example,                     parameters  945;ij b learned, using pairwise <br /> consider Tensor Product Pairwise Kernel K [22],                  kernel, K, training dataset, (xi , yi ). Finally, new <br /> K((x1 , y1 ), (x2 , y2 )) computed using simple kernel k             pairs enzymes genes (x, y) classified (e.g., k simple Phylogenetic (PFAM) ker-                     interacting interacting, depending evaluation <br /> nel described Ben-Hur et al. [22]). PFAM kernel                    decision function f  example representation <br /> (kpfam (x, y)) describes similarity measures based                 Figure 3(d)). predicting gene interactions PFAM database [34] gene x gene y.                     unseen examples, metabolic pathways  Tensor Product Pairwise Kernel K, using                    predicted. <br /> simple kernel PFAM Kernel kpfam defined                           pairwise kernel computation     K((x1 , y1 ), (x2 , y2 )) = kpfam (x1 , x2 )  8727; kpfam (y1 , y2 )       expensive tasks prediction metabolic <br />                                                                           networks processing storage. Using sequence data <br />                                 + kpfam (x1 , y2 )  8727; kpfam (y1 , x2 )     causes longer execution times large storage <br />   example, Figure 3(b)  genes                    needs.  mentioned advantages associated variables follow: x1 = YAR071W, y1 =                 using sequence data order avoid error accumu- <br /> YAL002W, x2 = YDR127W, y2 = YAL038W, Tensor                           lation genome annotation dependencies. Product Pairwise Kernel                                                 SVMs guarantee better accuracy values  <br />     K ((x1 , y1 ), (x2 , y2 )) = kpfam (YAR071W, YDR127W)  8727; kpfam (YAL002W, YAL038W) <br />                  + kpfam (YAR071W, YAL038W)  8727; kpfam (YAL002W, YDR127W) = 0.5. <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                                              Page 7 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />  <br />  Figure 3 Diagram pairwise SVM applied metabolic network prediction.   example pairs training set using EC <br />  numbers   gene names  . (b) pairwise kernel matrix, numerical values cell correspond measure  similarities, given pairs EC numbers   pairs gene names  . (c) model trained estimate parameters  945;i,j b  decision function f . (d) Given new pair EC numbers (left) gene names (right) decision function evaluated pair classified  interacting non-interacting. <br />  <br />  <br />  <br />  <br /> supervised learning methods sequence ker-                               8226; Tensor Product Pairwise Rational Kernel <br /> nels metabolic network inference [7].                              (KPRKT ) focus improvement pairwise kernel computa-                                K((x1 , y1 ), (x2 , y2 )) = U(x1 , x2 )  8727; U(y1 , y2 )+ <br /> tions representation, incorporating rational kernels                         U(x1 , y2 )  8727; U(y1 , x2 ) manipulate sequence data. accomplish                             8226; Metric Learning Pairwise Rational Kernel proposed new framework called Pairwise Rational                              (KPRKM ) Kernels.                                                                            K((x1 , y1 ),(x2 , y2 )) = (U(x1 , x2 ) 8722;U(x1 , y2 ) 8722;U(y1 , x2 ) <br />                                                                                                               +U(y1 , y2 ))2 <br /> Methods                                                                            8226; Cartesian Pairwise Rational Kernel (KPRKC ) Pairwise rational kernels <br />                                                                                     K((x1 , y1 ), (x2 , y2 )) = U(x1 , x2 )  8727;  948;(y1 = y2 ) section, propose new pairwise kernels based <br />                                                                                                                 + 948;(x1 = x2 )  8727; U(y1 , y2 ) rational kernels, e., Pairwise Rational Kernels (PRKs). <br />                                                                                                                 +U(x1 , y2 )  8727;  948;(y1 = x2 ) obtained using rational kernels sim- <br />                                                                                                                 + 948;(x1 = y2 )  8727; U(y1 , x2 ) <br /> ple kernels k. defined PRKs, based                                                                                      948;(x = y) = 1 x = y 0  notations definitions Background Section <br />                                                                                      8704;x, y  8712; X.  <br /> Definition 1. Given X  8838;    8727; transducer U,                            Following Theorem 1, construct U using function                                                                        weighted transducer T, U = T  9702; T  8722;1 , K : (X  215; X)  215; (X  215; X)  8594; R                                                    guarantee U Positive Definite Symmetric <br />                                                                                 (PDS) kernel. PDS needed condition use kernels <br />    8226; Direct Sum Pairwise Rational Kernel (KPRKDS )                          training classification algorithms. kernels <br />     K((x1 , y1 ), (x2 , y2 )) = U(x1 , x2 ) + U(y1 , y2 )+                      defined results PDS kernel operations,     U(y1 , x2 ) + U(x1 , y2 )                                                   PRK kernels PDS [35]. <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                      Page 8 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />  <br /> Algorithm                                                          graph species. used SVM methods designed general algorithm, Algorithm 2,              metabolic network inference, prefer balanced <br /> compute kernels, using composition weighted             dataset. dataset, unbalanced pro- <br /> transducers. extension Algorithm 1.             portions interacting (+1) non-interacting ( 8722;1) <br /> uses input transducers Mx1 , My1 , Mx2 , My2 ,           classified pairs (e.g., dataset 282060 represent sequences x1 , y1 , x2 , y2  8712; X         non-interacting pairs). order balance dataset, Weighted Finite-State Transducer U, outputs                followed procedure recommended Yu et al. [29], <br /> value K((x1 , y1 ), (x2 , y2 )).                                using program BRS-noint select non-interacting <br />                                                                    pairs. Yu et al. [29] describes bias non- <br /> Algorithm 2 Pairwise Rational Kernel Computation                   interacting pair selection training process                                                                    accuracy estimation. eliminate bias, BRS- <br /> INPUT: pairs sequences (x1 , y1 ), (x2 , y2 ) WFST U <br />                                                                    noint program used create  8220;balanced 65533;? negative <br />   obtain Mx1 , My1 , Mx2 , My2 use transducer <br />                                                                    set maintain right distribution non-interacting <br /> composition compute: <br />                                                                    interacting pairs. result, obtained 2574 non- <br /> N1 = Mx1  9702; U  9702; Mx2 <br />                                                                    interacting pairs total 5149 pairs training <br /> <span id='am-377' about='obo:IMR_0704569' typeof='owl:Thing'><span id='am-378' property="oboInOwl:hasDbXref" content="UniProt:O35516" datatype="xsd:string"></span><span id='am-379' property="rdfs:label" content="Motch B" datatype="xsd:string"></span><span id='am-380' property="rdfs:label" content="Notch gene homolog 2 (Drosophila)" datatype="xsd:string"></span><span id='am-381' property="rdfs:label" content="Notch 2 intracellular domain" datatype="xsd:string"></span><span id='am-382' property="rdfs:label" content="Neurogenic locus notch homolog protein 2" datatype="xsd:string"></span><span id='am-383' property="rdfs:label" content="Notch 2 extracellular truncation" datatype="xsd:string"></span><span id='am-384' property="rdfs:label" content="N2" datatype="xsd:string"></span><span id='am-385' property="oboInOwl:hasExactSynonym" content="N2" datatype="xsd:string"></span><span id='am-386' property="oboInOwl:hasExactSynonym" content="Notch 2 extracellular truncation" datatype="xsd:string"></span><span id='am-387' property="oboInOwl:hasExactSynonym" content="Notch gene homolog 2 (Drosophila)" datatype="xsd:string"></span><span id='am-388' property="oboInOwl:id" content="IMR:0704569" datatype="xsd:string"></span><span id='am-389' property="oboInOwl:hasExactSynonym" content="Neurogenic locus notch homolog protein 2" datatype="xsd:string"></span><span id='am-390' property="oboInOwl:hasExactSynonym" content="Notch 2" datatype="xsd:string"></span><span id='am-391' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-392' property="rdfs:label" content="NOTC2_MOUSE" datatype="xsd:string"></span><span id='am-393' property="oboInOwl:hasExactSynonym" content="Motch B" datatype="xsd:string"></span><span id='am-394' property="rdfs:label" content="Notch 2" datatype="xsd:string"></span><span id='am-395' property="oboInOwl:hasExactSynonym" content="Notch 2 intracellular domain" datatype="xsd:string"></span><span id='am-396' property="rdfs:label" content="Notch2" datatype="xsd:string"></span><span id='am-397' property="oboInOwl:hasExactSynonym" content="Notch2" datatype="xsd:string"></span>N2</span> = Mx1  9702; U  9702; My2 <br />                                                                    process. <br /> N3 = My1  9702; U  9702; Mx2 <br /> N4 = My1  9702; U  9702; My2 <br /> (ii) compute sum paths N1 , N2 , N3 , N4 using       Training process kernel computation <br /> shortest-distance algorithm                                        known metabolic network converted <br /> (iii) compute formulas Definition 1:                        graph obtained pairs training set, <br /> KPRKDS ((x1 , y1 ), (x2 , y2 )) = N1 + N2 + N3 + N4                corresponding Figure 3 . PRK representation <br /> KPRKT ((x1 , y1 ), (x2 , y2 )) = N1  8727; N4 + N2  8727; N3                 coincides Figure 3.  compu- <br /> KPRKM ((x1 , y1 ), (x2 , y2 )) = (N1  8722; N2  8722; N3 + N4 )2             tation PRKs  main contribution KPRKC ((x1 , y1 ), (x2 , y2 )) = N1  8727; 948;(y1 = y2 )+N2  8727; 948;(y1 = x2 )   research), given data yeast Saccharomyces <br /> +N3  8727;  948;(x1 = y2 ) + N4  8727;  948;(x1 = x2 )                               cerevisiae: <br /> RESULTS: values K((x1 , y1 ), (x2 , y2 )) <br />                                                                       8226; 755 known genes represented                                                                        trivial weighted automaton  e., Ax1 , Ax2 , . . . Ax755 ) <br />   implementation described  use n-                 using nucleotide sequences, <br /> gram rational kernel kernel U  n-gram kernel           8226; n-gram kernel, n = 3, used rational <br />                                                                                                           rational kernel Section details).               kernel, U(Ax1 , Ax2 ) =  =3 cAx1  cAx2    complexity steps   (ii) O(|Mx1 | + |My1 | +               n-gram kernel rational kernel Section |Mx2 | + |My2 |). <span id='am-491' about='obo:IMR_0703880' typeof='owl:Thing'><span id='am-492' property="rdfs:label" content="STEP" datatype="xsd:string"></span><span id='am-493' property="oboInOwl:hasDbXref" content="UniProt:P54829" datatype="xsd:string"></span><span id='am-494' property="rdfs:label" content="PROTEIN-TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE, 5" datatype="xsd:string"></span><span id='am-495' property="rdfs:label" content="PTPN, STRIATUM-ENRICHED" datatype="xsd:string"></span><span id='am-496' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-497' property="oboInOwl:hasExactSynonym" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-498' property="rdfs:label" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-499' property="rdfs:label" content="protein tyrosine phosphatase, non-receptor type 5 (striatum-enriched)" datatype="xsd:string"></span><span id='am-500' property="oboInOwl:hasExactSynonym" content="protein tyrosine phosphatase, non-receptor type 5 (striatum-enriched)" datatype="xsd:string"></span><span id='am-501' property="oboInOwl:hasExactSynonym" content="PROTEIN-TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE, 5" datatype="xsd:string"></span><span id='am-502' property="oboInOwl:hasExactSynonym" content="STEP" datatype="xsd:string"></span><span id='am-503' property="oboInOwl:hasExactSynonym" content="FLJ14427" datatype="xsd:string"></span><span id='am-504' property="rdfs:label" content="PTPN5" datatype="xsd:string"></span><span id='am-505' property="rdfs:label" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-506' property="oboInOwl:hasExactSynonym" content="PTPN, STRIATUM-ENRICHED" datatype="xsd:string"></span><span id='am-507' property="oboInOwl:id" content="IMR:0703880" datatype="xsd:string"></span><span id='am-508' property="rdfs:label" content="FLJ14427" datatype="xsd:string"></span><span id='am-509' property="rdfs:label" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-510' property="rdfs:label" content="PTN5_HUMAN" datatype="xsd:string"></span><span id='am-511' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-512' property="oboInOwl:hasExactSynonym" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-513' property="oboInOwl:hasExactSynonym" content="PTPSTEP" datatype="xsd:string"></span><span id='am-514' property="rdfs:label" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-515' property="oboInOwl:hasExactSynonym" content="PTPN5" datatype="xsd:string"></span><span id='am-516' property="rdfs:label" content="PTPSTEP" datatype="xsd:string"></span><span id='am-517' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.48" datatype="xsd:string"></span>Step</span> (iii) adds constant time complexity.          details), conclude PRKs  based n-gram kernels               8226; Algorithm 2 implemented obtain K values, computed time O |Mx1 | + |My1 | + |Mx2 | + |My2 | .             8226; example, Tensor Product Pairwise Rational <br />                                                                        Kernel Definition 1 obtained  <br /> Experiments <br />                                                                        KPRKT ((x1 , y1 ), (x2 , y2 )) = section experiments predict <br /> metabolic networks using pairwise SVMs combined                   = U(Ax1 , Ax2 )  8727; U(Ay1 , Ay2 ) + U(Ax1 , Ay2 ) <br /> PRKs. aim prove advantage using PRKs                      +U(Ay1 , Ax2 ) <br />                                                                                                          <br /> improve execution time computation                   =  =3 cAx1  cAx2    8727;  =3 cAy1  cAy2  + <br />                                                                                                           <br /> pairwise kernels training process, maintain-                +  =3 cAx1  cAy2    8727;  =3 cAy1  cAx2  . <br /> ing improving accuracy values. <br />                                                                       8226; finally, PRK kernels K positive <br /> Dataset                                                                eigenvalues normalized avoid fact used data yeast Saccharomyces cerevisiae                   longer sequences contain n-grams, <br /> [36]. species selected compare methods,                resulting similarities [16]. <br /> implementations results methods predict biological networks Saccharomyces cerevisiae             implemented method compute PRKs <br /> [9,10,22].                                                         using Open Finite-State Transducer (OpenFST) library <br />   data species taken KEGG               [38] OpenKernel library [39]. input data pathway [37] converted graph described                 nucleotide sequences known genes, outputs previous section  Figure 2 details).           pairwise rational kernel values similarity 755 nodes 2575 interacting pairs             measure pairs. Example 3 shows input  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                     Page 9 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />  <br /> output values method described  equivalent           represent abbreviated nucleotide sequences, inter- Figure 3(b), using sequence data.                           act interact. decision function, f (x, y),                                                                    previously obtained training process  Example 3. Given nucleotide sequences x1 , y1 , x2 , y2 ,          Pairwise support vector machine Section details). represent abbreviated examples known genes             resulting value evaluating decision function dataset,                                                       f (x, y) greater 0, pair (x, y) interact, x1 = GCTAAATTGGACAAATCTCAATGAAATTGTC                               pair (x, y) interact. Suppose evaluation <br /> TTGG                                                               y1 = ATGTCCTCGTCTTCGTCTACCGGGTACAGAA                               f (x, y) = f (CTCAAAGTCTTAATGCTTGGACAAATTGA <br /> AA                                                                 AATTGG . . . , TCTACAGAGTCGTCCTTCGTCTACCGG <br /> x2 = CATGACTAAAGAAACGATTCGGGTAGTTATT                               GAAAAT . . .) = +3. <br /> TGGCGG                                                              predict nucleotide sequences (x, y) <br /> y2 = ATCTACAAGCGAACCAGAGTCTTCTGCAGGC                               interact context metabolic network TTAGAT                                                             yeast Saccharomyces cerevisiae. Tensor Product Pairwise Rational Kernel KPRKT                  case, used 755 genes training pro- <br /> ((x1 , y1 ), (x2 , y2 )) obtained using 3-gram ratio-   cess, species 6000 genes [41]. <br /> nal kernel, e.g., = TCT, values                       rest metabolic pathways predicted <br />                                                                    classifying pairs genes  pairs raw <br />    8226; cAx   = 2  TCT appears twice x1                   nucelotide sequences), interacting non-interacting, <br />        1 <br />     GCTAAATTGGACAAATCT CAATGAAATTG                                 using decision function f . Note decision func- <br />     TCT TGG,                                                       tion obtained training process,    8226; cAy   = 2  TCT appears twice y1                   used needed prediction process. <br />        1 <br />     ATGTCCTCGTCT TCGTCT ACCGGGTACAGA <br />     AAA,                                                             advantage using sequence data nucleotide <br />    8226; cAx   = 1  TCT appears x2                    sequences used, annotated. <br />        2 <br />     CATGACTAAAGAAACGATTCT GGTAGTTATT                                type sequence data, e.g., high- <br />     TGGCGG,                                                    throughput analysis, considered combined, <br />    8226; cAy   = 3  TCT appears times y2             using similar implementation. <br />        2 <br />     ATCT ACAAGCGAACCAGAGTCT TTCT GCAGG <br />     CTTAGAT.                                                       Experiment description performance measures <br />                                                                    used pairwise SVM PRKs metabolic <br />   results values corresponding                network prediction, using data algo- 3-gram rational kernel, KPRKT computed                rithms described  <span id='am-138' about='obo:IMR_0000300' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010044 obo:IMR_0000653 obo:IMR_0100009 obo:IMR_0000001 obo:IMR_0000914 obo:IMR_0000207 obo:IMR_0000960 obo:IMR_0000288'><span id='am-139' property="rdfs:label" content="TC4" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasExactSynonym" content="Ras-like nuclear protein" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:id" content="IMR:0000300" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasExactSynonym" content="TC4" datatype="xsd:string"></span><span id='am-144' property="rdfs:label" content="Ras-like nuclear protein" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasDbXref" content="IPR:002041" datatype="xsd:string"></span><span id='am-146' property="rdfs:label" content="Ran" datatype="xsd:string"></span>ran</span> experiments KPRKT ((x1 , y1 ), (x2 , y2 )) = 0.3, 0.3 measure    different kernels. Firstly, used PRKs <br /> similarity.                                                        described Definition 1 using 3-gram rational <br />                                                                    kernel  e., KPRKDS 8722;3gram , KPRKT 8722;3gram , KPRKM 8722;3gram <br /> SVM predicting process                                         KPRKC 8722;3gram ). addition, combination PRKs implement pairwise SVM method, use                   kernels considered. included sequential minimal optimization (<span id='am-147' about='obo:IMR_0704710' typeof='owl:Thing'><span id='am-148' property="oboInOwl:id" content="IMR:0704710" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasExactSynonym" content="Smoh" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasDbXref" content="UniProt:P56726" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasExactSynonym" content="smoothened homolog (Drosophila)" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="Smo" datatype="xsd:string"></span><span id='am-153' property="rdfs:label" content="Smoh" datatype="xsd:string"></span><span id='am-154' property="rdfs:label" content="SMO_MOUSE" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasExactSynonym" content="Smo" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasExactSynonym" content="Smoothened homolog" datatype="xsd:string"></span><span id='am-157' property="rdfs:label" content="Smoothened homolog" datatype="xsd:string"></span><span id='am-158' property="rdfs:label" content="SMO" datatype="xsd:string"></span><span id='am-159' property="rdfs:label" content="smoothened homolog (Drosophila)" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasExactSynonym" content="SMO" datatype="xsd:string"></span>SMO</span>) technique               phylogenetic kernel (Kphy ) described Yamanishi 2010 package LIBSVM [40] combination OpenKer-               [9] PFAM kernel (Kpfam ) Ben-Hur et al. <br /> nel library [39]. training process, decision        [22].  second set experiments devel- <br /> function obtained estimating parameters,          oped combining PRKs phylogenetic kernel  e., <br /> shown Figure 3(c).  prediction process allows           KPRKDS 8722;3gram + Kphy , KPRKT 8722;3gram + Kphy , KPRKM 8722;3gram + <br /> classification new pairs nucleotide sequences             Kphy KPRKC 8722;3gram + Kphy ). Finally, combined <br /> interacting interacting evaluating decision          PRKs PFAM kernel, obtaining KPRKDS 8722;3gram + <br /> function. Example 4 shows description prediction          Kpfam , KPRKT 8722;3gram + Kpfam , KPRKM 8722;3gram + Kpfam process, similar process described Figure 3(d),          KPRKC 8722;3gram + Kpfam kernels. Considering phy- using nucleotide sequences.                                    logenetic PFAM kernels PDS, resulting <br />                                                                    combinations PDS [35]. <br /> Example 4. example predictor process.              compare advantages PRKs framework, <br /> Suppose want know                                         developed new set experiments x = CTCAAAGTCTTAATGCTTGGACAAATTGAAAT                               dataset, using finite-state transducers. TGG,                                                           considered pairwise (n-gram) kernel, e., KT 8722;3gram . <br /> y=TCTACAGAGTCGTCCTTCGTCTACCGGGAAAAT,                               KT 8722;3gram denoted pairwise tensor product described <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                          Page 10 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />  Pairwise kernels Section. consistent                accuracy comparable Experiments II III. <br /> previous experiments, combined KT 8722;3gram ker-                       Similar results obtained Yu et al. [29] PPI <span id='am-873' about='obo:IMR_0702135' typeof='owl:Thing'><span id='am-874' property="rdfs:label" content="SAP-2" datatype="xsd:string"></span><span id='am-875' property="oboInOwl:hasExactSynonym" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-876' property="rdfs:label" content="ELK3, ETS-domain protein (SRF accessory protein 2)" datatype="xsd:string"></span><span id='am-877' property="oboInOwl:hasExactSynonym" content="ELK3, ETS-DOMAIN PROTEIN" datatype="xsd:string"></span><span id='am-878' property="oboInOwl:hasExactSynonym" content="ELK3, ETS-domain protein (SRF accessory protein 2)" datatype="xsd:string"></span><span id='am-879' property="oboInOwl:hasExactSynonym" content="SAP2" datatype="xsd:string"></span><span id='am-880' property="rdfs:label" content="Serum response factor accessory protein 2" datatype="xsd:string"></span><span id='am-881' property="rdfs:label" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-882' property="rdfs:label" content="ELK3, ETS-DOMAIN PROTEIN" datatype="xsd:string"></span><span id='am-883' property="oboInOwl:hasExactSynonym" content="SAP-2" datatype="xsd:string"></span><span id='am-884' property="rdfs:label" content="ELK3_HUMAN" datatype="xsd:string"></span><span id='am-885' property="rdfs:label" content="ETS-RELATED PROTEIN" datatype="xsd:string"></span><span id='am-886' property="oboInOwl:hasExactSynonym" content="ELK3" datatype="xsd:string"></span><span id='am-887' property="oboInOwl:hasExactSynonym" content="ETS-RELATED PROTEIN" datatype="xsd:string"></span><span id='am-888' property="oboInOwl:hasExactSynonym" content="SRF accessory protein 2" datatype="xsd:string"></span><span id='am-889' property="oboInOwl:hasExactSynonym" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-890' property="oboInOwl:id" content="IMR:0702135" datatype="xsd:string"></span><span id='am-891' property="oboInOwl:hasExactSynonym" content="NET" datatype="xsd:string"></span><span id='am-892' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-893' property="rdfs:label" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-894' property="rdfs:label" content="SRF accessory protein 2" datatype="xsd:string"></span><span id='am-895' property="oboInOwl:hasExactSynonym" content="Serum response factor accessory protein 2" datatype="xsd:string"></span><span id='am-896' property="rdfs:label" content="SAP2" datatype="xsd:string"></span><span id='am-897' property="rdfs:label" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-898' property="rdfs:label" content="ERP" datatype="xsd:string"></span><span id='am-899' property="oboInOwl:hasExactSynonym" content="ERP" datatype="xsd:string"></span><span id='am-900' property="rdfs:label" content="ELK3" datatype="xsd:string"></span><span id='am-901' property="rdfs:label" content="NET" datatype="xsd:string"></span><span id='am-902' property="oboInOwl:hasExactSynonym" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-903' property="oboInOwl:hasDbXref" content="UniProt:P41970" datatype="xsd:string"></span>net</span>- <br /> nel phylogenetic kernel (Kphy ) PFAM kernel                  works. stated simple sequence-based kernels, (Kpfam ), e., KT 8722;3gram + Kphy KT 8722;3gram + Kpfam ker-                 n-gram, properly predict-protein interactions. <br /> nels, respectively. pairwise SVM algorithm used                    Yu et al. [29] combined sequence kernels predict metabolic network using data set                  kernels incorporate evolutionary informa- <br /> described  Table 1 describes groups created                  tion, accuracy model predictor improved. <br /> compare kernels equivalent PRKs.                           obtained similar results applied metabolic <span id='am-904' about='obo:IMR_0702136' typeof='owl:Thing'><span id='am-905' property="rdfs:label" content="Erp" datatype="xsd:string"></span><span id='am-906' property="rdfs:label" content="Net" datatype="xsd:string"></span><span id='am-907' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-908' property="rdfs:label" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-909' property="rdfs:label" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-910' property="oboInOwl:hasExactSynonym" content="ELK3, member of ETS oncogene family" datatype="xsd:string"></span><span id='am-911' property="rdfs:label" content="ELK3, member of ETS oncogene family" datatype="xsd:string"></span><span id='am-912' property="oboInOwl:id" content="IMR:0702136" datatype="xsd:string"></span><span id='am-913' property="rdfs:label" content="Elk3" datatype="xsd:string"></span><span id='am-914' property="rdfs:label" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-915' property="oboInOwl:hasExactSynonym" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-916' property="oboInOwl:hasExactSynonym" content="Net" datatype="xsd:string"></span><span id='am-917' property="oboInOwl:hasExactSynonym" content="Erp" datatype="xsd:string"></span><span id='am-918' property="rdfs:label" content="Sap-2" datatype="xsd:string"></span><span id='am-919' property="oboInOwl:hasExactSynonym" content="Elk3" datatype="xsd:string"></span><span id='am-920' property="rdfs:label" content="ELK3_MOUSE" datatype="xsd:string"></span><span id='am-921' property="oboInOwl:hasDbXref" content="UniProt:P41971" datatype="xsd:string"></span><span id='am-922' property="oboInOwl:hasExactSynonym" content="Sap-2" datatype="xsd:string"></span><span id='am-923' property="oboInOwl:hasExactSynonym" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-924' property="oboInOwl:hasExactSynonym" content="ETS-related protein NET" datatype="xsd:string"></span>net</span>- <br />    experiments executed <span id='am-314' about='obo:IMR_0702625' typeof='owl:Thing'><span id='am-315' property="oboInOwl:hasExactSynonym" content="Pyruvic carboxylase" datatype="xsd:string"></span><span id='am-316' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-317' property="oboInOwl:hasExactSynonym" content="pyruvate carboxylase" datatype="xsd:string"></span><span id='am-318' property="rdfs:label" content="PYC_HUMAN" datatype="xsd:string"></span><span id='am-319' property="rdfs:label" content="EC 6.4.1.1" datatype="xsd:string"></span><span id='am-320' property="rdfs:label" content="PCB" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasExactSynonym" content="PCB" datatype="xsd:string"></span><span id='am-322' property="rdfs:label" content="PC" datatype="xsd:string"></span><span id='am-323' property="oboInOwl:hasExactSynonym" content="Pyruvate carboxylase, mitochondrial" datatype="xsd:string"></span><span id='am-324' property="rdfs:label" content="pyruvate carboxylase" datatype="xsd:string"></span><span id='am-325' property="oboInOwl:id" content="IMR:0702625" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasExactSynonym" content="PC" datatype="xsd:string"></span><span id='am-327' property="rdfs:label" content="Pyruvate carboxylase, mitochondrial" datatype="xsd:string"></span><span id='am-328' property="rdfs:label" content="Pyruvic carboxylase" datatype="xsd:string"></span><span id='am-329' property="oboInOwl:hasExactSynonym" content="EC 6.4.1.1" datatype="xsd:string"></span><span id='am-330' property="oboInOwl:hasDbXref" content="UniProt:P11498" datatype="xsd:string"></span>PC</span> intel                        works predictions: PHY PFAM kernels i7CORE, 8MB <span id='am-843' about='obo:IMR_0700725' typeof='owl:Thing'><span id='am-844' property="rdfs:label" content="HsT18676" datatype="xsd:string"></span><span id='am-845' property="rdfs:label" content="RAB27A" datatype="xsd:string"></span><span id='am-846' property="oboInOwl:hasExactSynonym" content="MGC117246" datatype="xsd:string"></span><span id='am-847' property="oboInOwl:hasExactSynonym" content="Ras-related protein Rab-27A" datatype="xsd:string"></span><span id='am-848' property="rdfs:label" content="RAS-RELATED GENE FROM MEGAKARYOCYTE" datatype="xsd:string"></span><span id='am-849' property="oboInOwl:hasExactSynonym" content="GS2" datatype="xsd:string"></span><span id='am-850' property="oboInOwl:hasExactSynonym" content="RAB27A, member RAS oncogene family" datatype="xsd:string"></span><span id='am-851' property="oboInOwl:hasExactSynonym" content="RAM" datatype="xsd:string"></span><span id='am-852' property="rdfs:label" content="GTP-binding protein Ram" datatype="xsd:string"></span><span id='am-853' property="oboInOwl:hasExactSynonym" content="RAB27" datatype="xsd:string"></span><span id='am-854' property="rdfs:label" content="RAS-ASSOCIATED PROTEIN RAB27A" datatype="xsd:string"></span><span id='am-855' property="rdfs:label" content="RAM" datatype="xsd:string"></span><span id='am-856' property="oboInOwl:hasDbXref" content="UniProt:P51159" datatype="xsd:string"></span><span id='am-857' property="rdfs:label" content="RB27A_HUMAN" datatype="xsd:string"></span><span id='am-858' property="oboInOwl:hasExactSynonym" content="RAB27A" datatype="xsd:string"></span><span id='am-859' property="oboInOwl:hasExactSynonym" content="Rab-27" datatype="xsd:string"></span><span id='am-860' property="rdfs:label" content="RAB27A, member RAS oncogene family" datatype="xsd:string"></span><span id='am-861' property="rdfs:label" content="RAB27" datatype="xsd:string"></span><span id='am-862' property="rdfs:label" content="Rab-27" datatype="xsd:string"></span><span id='am-863' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-864' property="oboInOwl:hasExactSynonym" content="RAS-RELATED GENE FROM MEGAKARYOCYTE" datatype="xsd:string"></span><span id='am-865' property="rdfs:label" content="MGC117246" datatype="xsd:string"></span><span id='am-866' property="oboInOwl:id" content="IMR:0700725" datatype="xsd:string"></span><span id='am-867' property="oboInOwl:hasExactSynonym" content="HsT18676" datatype="xsd:string"></span><span id='am-868' property="rdfs:label" content="GS2" datatype="xsd:string"></span><span id='am-869' property="oboInOwl:hasExactSynonym" content="RAS-ASSOCIATED PROTEIN RAB27A" datatype="xsd:string"></span><span id='am-870' property="rdfs:label" content="Ras-related protein Rab-27A" datatype="xsd:string"></span><span id='am-871' property="oboInOwl:hasExactSynonym" content="GTP-binding protein Ram" datatype="xsd:string"></span>RAM</span>. validate model, used                       included (Experiments II III, respectively), accuracies <br /> 10-<span id='am-961' about='obo:IMR_0703347' typeof='owl:Thing'><span id='am-962' property="oboInOwl:hasExactSynonym" content="Argininosuccinate synthase" datatype="xsd:string"></span><span id='am-963' property="rdfs:label" content="Citrulline--aspartate ligase" datatype="xsd:string"></span><span id='am-964' property="rdfs:label" content="Ass-1" datatype="xsd:string"></span><span id='am-965' property="oboInOwl:hasExactSynonym" content="Ass1" datatype="xsd:string"></span><span id='am-966' property="oboInOwl:hasDbXref" content="UniProt:P16460" datatype="xsd:string"></span><span id='am-967' property="rdfs:label" content="ASSY_MOUSE" datatype="xsd:string"></span><span id='am-968' property="oboInOwl:hasExactSynonym" content="Ass-1" datatype="xsd:string"></span><span id='am-969' property="oboInOwl:id" content="IMR:0703347" datatype="xsd:string"></span><span id='am-970' property="rdfs:label" content="fold" datatype="xsd:string"></span><span id='am-971' property="oboInOwl:hasExactSynonym" content="EC 6.3.4.5" datatype="xsd:string"></span><span id='am-972' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-973' property="rdfs:label" content="Argininosuccinate synthase" datatype="xsd:string"></span><span id='am-974' property="oboInOwl:hasExactSynonym" content="argininosuccinate synthetase 1" datatype="xsd:string"></span><span id='am-975' property="rdfs:label" content="Ass1" datatype="xsd:string"></span><span id='am-976' property="oboInOwl:hasExactSynonym" content="Citrulline--aspartate ligase" datatype="xsd:string"></span><span id='am-977' property="rdfs:label" content="argininosuccinate synthetase 1" datatype="xsd:string"></span><span id='am-978' property="rdfs:label" content="EC 6.3.4.5" datatype="xsd:string"></span><span id='am-979' property="oboInOwl:hasExactSynonym" content="fold" datatype="xsd:string"></span>fold</span> cross validation method measured average                  improved maintaining adequate processing <br /> Area Curve Receiver Operating Characteris-                   times. best accuracy value obtained com- <br /> tic (AUC <span id='am-1194' about='obo:IMR_0010509' typeof='obo:IMR_0010024 owl:Thing obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000925'><span id='am-1195' property="oboInOwl:hasExactSynonym" content="hnRNP C1 / hnRNP C2" datatype="xsd:string"></span><span id='am-1196' property="oboInOwl:hasExactSynonym" content="hnRNP C1/C2" datatype="xsd:string"></span><span id='am-1197' property="rdfs:label" content="hnRNP C1/C2" datatype="xsd:string"></span><span id='am-1198' property="rdfs:label" content="ROC" datatype="xsd:string"></span><span id='am-1199' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1200' property="rdfs:label" content="Heterogeneous nuclear ribonucleoproteins C1/C2" datatype="xsd:string"></span><span id='am-1201' property="oboInOwl:hasExactSynonym" content="ROC" datatype="xsd:string"></span><span id='am-1202' property="rdfs:label" content="hnRNP C1 / hnRNP C2" datatype="xsd:string"></span><span id='am-1203' property="oboInOwl:id" content="IMR:0010509" datatype="xsd:string"></span>ROC</span>) score.                                                      bining PRK-Metric-3gram PFAM kernels (aver- <br />    Cross-validation method suitable approach val-                 <span id='am-337' about='obo:IMR_0200587' typeof='owl:Thing'><span id='am-338' property="oboInOwl:hasExactSynonym" content="RNBP" datatype="xsd:string"></span><span id='am-339' property="oboInOwl:hasExactSynonym" content="AGE" datatype="xsd:string"></span><span id='am-340' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-341' property="oboInOwl:hasExactSynonym" content="Renin-binding protein" datatype="xsd:string"></span><span id='am-342' property="oboInOwl:hasExactSynonym" content="renin binding protein" datatype="xsd:string"></span><span id='am-343' property="rdfs:label" content="N-acylglucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-344' property="rdfs:label" content="RNBP" datatype="xsd:string"></span><span id='am-345' property="oboInOwl:hasExactSynonym" content="RnBP" datatype="xsd:string"></span><span id='am-346' property="rdfs:label" content="Renin-binding protein" datatype="xsd:string"></span><span id='am-347' property="oboInOwl:hasDbXref" content="UniProt:P51606" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasExactSynonym" content="GlcNAc 2-epimerase" datatype="xsd:string"></span><span id='am-349' property="rdfs:label" content="AGE" datatype="xsd:string"></span><span id='am-350' property="rdfs:label" content="N-acetyl-D-glucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-351' property="rdfs:label" content="EC 5.1.3.8" datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasExactSynonym" content="N-acetyl-D-glucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasExactSynonym" content="RENBP" datatype="xsd:string"></span><span id='am-354' property="oboInOwl:hasExactSynonym" content="EC 5.1.3.8" datatype="xsd:string"></span><span id='am-355' property="rdfs:label" content="RENBP_HUMAN" datatype="xsd:string"></span><span id='am-356' property="rdfs:label" content="RnBP" datatype="xsd:string"></span><span id='am-357' property="rdfs:label" content="RBP" datatype="xsd:string"></span><span id='am-358' property="oboInOwl:id" content="IMR:0200587" datatype="xsd:string"></span><span id='am-359' property="rdfs:label" content="GlcNAc 2-epimerase" datatype="xsd:string"></span><span id='am-360' property="rdfs:label" content="renin binding protein" datatype="xsd:string"></span><span id='am-361' property="rdfs:label" content="RENBP" datatype="xsd:string"></span><span id='am-362' property="oboInOwl:hasExactSynonym" content="N-acylglucosamine 2-epimerase" datatype="xsd:string"></span><span id='am-363' property="oboInOwl:hasExactSynonym" content="RBP" datatype="xsd:string"></span>age</span> AUC=0.844). papers used similar kernel <br /> idate performance predictive models. k-fold cross-                  combinations improve prediction biological <span id='am-1147' about='obo:IMR_0001104' typeof='owl:Thing obo:IMR_0000397 obo:IMR_0000949 obo:IMR_0000369 obo:IMR_0000368 obo:IMR_0000001'><span id='am-1148' property="oboInOwl:hasExactSynonym" content="Sap-2" datatype="xsd:string"></span><span id='am-1149' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1150' property="rdfs:label" content="Elk-3" datatype="xsd:string"></span><span id='am-1151' property="rdfs:label" content="Net" datatype="xsd:string"></span><span id='am-1152' property="oboInOwl:hasExactSynonym" content="Elk-3" datatype="xsd:string"></span><span id='am-1153' property="oboInOwl:id" content="IMR:0001104" datatype="xsd:string"></span><span id='am-1154' property="rdfs:label" content="Sap-2" datatype="xsd:string"></span>net</span>- <br /> validation, original dataset randomly partitioned                  works, Ben-Hur et al. [22] Yamanishi [9]. k equal-sized subsets.  model trained k                   rational kernels used previous <br /> times. time, k subsets reserved                    research. <br /> testing remaining k  8722; 1 subsets used                    Ben-Hur et al. [22] report average AUC value training. final value obtained average k             0.78 PFAM kernels, Yamanishi [9] reports results  Kohavi et al. [42] details).                        average AUC 0.77 PHY kernel predicting <br />    Receiver Operating Characteristic (ROC) curve                   Saccharomyces cerevisiae metabolic pathways. plot True Positive Rate (TPR) versus False Pos-                previously developed similar experiments using SVM <br /> itive Rate (FPR) different possible cut-offs binary              methods [7]. result, obtain AUC values 0.92 <br /> classifier  cut defines level discriminat-             PFAM kernel 0.80 PHY kernel, execution <br /> ing positive negative categories. ROC curve analysis                  times 12060 7980 seconds, respectively.  used assess overall discriminatory ability               cases random selection negative posi- <br /> SVM binary classifiers. area curve (aver-                   tive training data used. noted Yu et al. [29], <br /> age AUC score) used metric evaluate                  average AUC values obtained random selection strength classification.                                           data training machine learning tools results bias <br />    addition, 95% Confidence Intervals (<span id='am-1271' about='obo:IMR_0000460' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000428'><span id='am-1272' property="rdfs:label" content="STAT induced STAT inhibitor" datatype="xsd:string"></span><span id='am-1273' property="rdfs:label" content="Cytokine-inducible SH2 protein" datatype="xsd:string"></span><span id='am-1274' property="oboInOwl:id" content="IMR:0000460" datatype="xsd:string"></span><span id='am-1275' property="rdfs:label" content="SSI" datatype="xsd:string"></span><span id='am-1276' property="rdfs:label" content="SOCS" datatype="xsd:string"></span><span id='am-1277' property="oboInOwl:hasExactSynonym" content="Cytokine-inducible SH2 protein" datatype="xsd:string"></span><span id='am-1278' property="rdfs:label" content="Suppressor of cytokine signaling" datatype="xsd:string"></span><span id='am-1279' property="oboInOwl:hasExactSynonym" content="SSI" datatype="xsd:string"></span><span id='am-1280' property="oboInOwl:hasExactSynonym" content="STAT induced STAT inhibitor" datatype="xsd:string"></span><span id='am-1281' property="oboInOwl:hasExactSynonym" content="CIS" datatype="xsd:string"></span><span id='am-1282' property="oboInOwl:hasExactSynonym" content="Suppressor of cytokine signaling" datatype="xsd:string"></span><span id='am-1283' property="rdfs:label" content="CIS" datatype="xsd:string"></span><span id='am-1284' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CIs</span>)                        genes  proteins) large numbers inter- computed, following method described                        actions.  high AUC results previous Cortes Mohri [43]. authors provide                           works directly compared results distribution-independent technique compute confi-                      paper. employed balanced sampling tech- <br /> dence intervals average AUC values. variance                      niques suggested Yu et al. [29] combat bias depends number positive negative examples                     training set. results, average AUC values (2575 2574 cases) number classifica-                range 0.5-0.844, comparable exceed cases tion errors, ranging 889 1912 cases.                   results obtained Yu et al. [29] balanced sampling, <br />                                                                           range 0.5-0.75 different kernels <br /> Results discussion                                                    protein interaction problems. obtained <br /> Table 2 shows SVM performance, execution times                    results execution times 15-140 seconds. 95% CIs grouped kernels mentioned                      exception direct sum kernel,   experiments using PRK best                 fidence intervals behaviour random <br /> execution times (Exp.  transducer representations                classifier. algorithms speed processing.                         developed experiment PFAM <br />                                                                           kernel simple kernel Pairwise Tensor Product <br /> Table 1 Groups PRK pairwise kernel comparison                     (Kpfam ) using balanced sampling suggested Yu <br />                                                                           et al. [29]. Note PRK; regular <br /> Group             PRKs 1                              Pairwise Kernel 2 <br />                                                                           pairwise kernel using PFAM simple kernel, similar <br /> N-GRAM            KPRKT 8722;3gram                         KT 8722;3gram            example Using pairwise kernel SVM <br /> PHY               KPRKT 8722;3gram +Kphy                   KT 8722;3gram + Kphy     predict metabolic networks Section. result, PFAM              KPRKT 8722;3gram +Kpfam                  KT 8722;3gram + Kpfam    average AUC 0.61 execution time 122 <br /> 1 <br />   Kernels taken Table 2. <br />                                                                           seconds. compare values results <br /> 2 <br />   Computed Tensor Product Pairwise Kernel.                       Table 2 Exp.  kernels KPRKM 8722;3gram <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                           Page 11 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />  <br /> Table 2 Average AUC ROC scores processing times various PRKs <br /> Exp        Type kernels      Kernel                                         Average AUC score   Runtime (sec)    Confidence intervals <br />            Pairwise Rational    PRK-Direct-Sum (KPRKDS 8722;3gram )                 0.499               15.0             [0.486, 0.512] <br />            Kernels (PRK)        PRK-Tensor-Product (KPRKT 8722;3gram )              0.597               16.2             [0.589, 0.605]            (3-gram)             PRK-Metric-Learning (KPRKM 8722;3gram )             0.641               17.4             [0.633, 0.648] <br />                                 PRK-Cartesian (KPRKC 8722;3gram )                   0.640               15.0             [0.632, 0.647] <br />            PRKs combined        PRK-Direct-Sum+Phy (KPRKDS 8722;3gram + Kphy )      0.425               136.2            [0.411, 0.438] <br />            phylogenetic    PRK-Tensor+Phy (KPRKT 8722;3gram +Kphy )            0.733               135.6            [0.725, 0.741] <br /> II <br />            data (Kphy Non-      PRK-Metric+Phy (KPRKM 8722;3gram +Kphy )            0.761               139.2            [0.753, 0.768] <br />            sequence kernel)     PRK-Cartesian+Phy (KPRKC 8722;3gram +Kphy )         0.742               132.6            [0.734, 0.749] <br />            PRKs combined        PRK-D-Sum+PFAM (KPRKDS 8722;3gram +Kpfam )          0.493               136.2            [0.480, 0.506] <br />            PFAM data       PRK-Tensor+PFAM (KPRKT 8722;3gram +Kpfam )          0.827               136.8            [0.819, 0.834] <br /> III <br />            (Kpfam               PRK-Metric+PFAM (KPRKM 8722;3gram +Kpfam )          0.844               140.4            [0.837, 0.850] <br />            Sequence kernel)     PRK-Cartesian+PFAM (KPRKC 8722;3gram +Kpfam )       0.842               132.0            [0.835, 0.849] <br />  <br />  <br />  KPRKC 8722;3gram better average accuracy  e., 0.641                   classification methods. McNemar 8217;s test defines 0.640, respectively) lesser average execution                      score, calculated  <br /> times (17.4 15.0 seconds, respectively). addition,                                                 <br />                                                                                        |Nsf  8722; Nfs |  8722; 1 Pairwise Rational Kernel 3-gram combined                           =                                              (2) <br />                                                                                                        PFAM kernel Exp. III,  e., Tensor                                        Nsf + Nfs <br /> Product Pairwise Rational Kernel - KPRKT 8722;3gram +Kpfam ), average accuracy value (average AUC=0.827)                          Nfs number times Algorithm failed <br /> better Pairwise Tensor Product (Kpfam ),                     Algorithm B succeeded, Nsf number execution time just increased 14.8 seconds  e.,                   times Algorithm succeeded Algorithm B failed. 122 seconds, using Kpfam , 134.8 seconds, using                     equal 0, algorithms similar <br /> KPRKT 8722;3gram +Kpfam ).                                                       performance. Additionally, Nfs larger Nsf   order statistically compares theses results,                     Algorithm B performs better Algorithm  vice <br /> applied McNemar 8217;s non-parametric statistical test                       versa. computed scores considering Algorithm [44]. McNemar 8217;s tests recently used Bostanci                   SVM algorithm using Pairwise Tensor Product <br /> et al. [45] prove significant statistical differences                    (Kpfam ) different Algorithm Bs, using SVM <br />  <br />  <br />  <br />  <br />      Figure 4 Comparison pairwise rational kernels pairwise kernels grouped kernel types (N-GRAM group, PHY group      PFAM group). <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                             Page 12 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br />  <br />  <br />  different PRKs Table 2  e., KPRKM 8722;3gram ,   used similar types stochiometric data, KPRKC 8722;3gram KPRKT 8722;3gram +Kpfam mentioned .          converted kernels consider PRKs. cases, obtained scores greater 0  e., <br /> 4.73, 4.54, 7.51), mean PRKs performed better.      Conclusion score proved difference statis-     paper, introduced new framework called <br /> tically significant confidence level 99% (based     Pairwise Rational Kernels, pairwise kernels tailed Prediction Confidence Levels described          obtained based transducer representations, e., ratio- [45]).                                                     nal kernels. defined framework, developed general <br />    Cartesian Kernel widely used        algorithms tested pairwise Support Vector defined Kashima et al. [10]. Kashima et al. [10]    Machine method predict metabolic networks. <br /> used Expression, Localization, Chemical Phylogenetic        used dataset yeast Saccharomyces cere- <br /> kernels predict metabolic networks.          visiae validate compare proposal similar non-sequence kernels. current experiments          models using data species. obtained computed, time, pairwise Cartesian ker-      better execution times models, nel rational kernel (sequence kernel) repre-        maintaining adequate accuracy values.  PRKs <br /> sent sequence data metabolic network prediction.          improved performance pairwise-SVM algo- <br /> Cartesian kernels [10] defined alternative    rithm used training process supervised improve Tensor Product Pairwise Kernel [22] com-       network inference methods. <br /> putation performance. experiments shown         methods, learning process executed <br /> Table 2, confirmed definition, obtained    obtain decision function. decision func- <br /> better accuracy execution times used          tion used times necessary predict <br /> Cartesian Pairwise Rational Kernel (KPRKC 8722;3gram )      interaction sequences species Tensor Product Rational Kernel (KPRKT 8722;3gram ).       predict metabolic pathways. <br /> Comparing results Kashima et al. [10],              methods research used sequence data <br /> obtained better average AUC values  e., 0.844 vs 0.79),     (e.g., nucleotide sequences) predict interactions. approximately average execution           Genes need correctly annotated raw <br /> times  e., 93 seconds). Kashima et al. [10] used non-       sequences used.  methods sequence data random selection positive nega-      able avoid error accumulation wrong gene <br /> tive data training.                                       annotations. <br />    Figure 4 shows results experiments compar-        future work, proposal used produce ing PRK framework pairwise kernels.        set candidate interactions pathways comparative groups described Table 1 used.      species, experimentally validated. seen, execution times better       pairwise rational kernels developed <br /> PRKs used groups. proves PRKs      using finite-state transducers operations. <br /> compute faster rational kernels use finite-state <br /> transducer operations representations, improving      Competing interests <br />                                                               authors declare competing interests. <br /> performance. <br />    power using kernels sort         Authors 8217; contributions data represented using kernels.           ARL implemented algorithms developed experiments. ARL, MD <br />                                                               BF contributed equally drafting manuscript. authors completely disparate types data combined         reviewed approved final version manuscript. <br /> add power kernel-based machine learning methods <br /> [8]. example, coefficients describing relative amounts    Acknowledgements <br />                                                               work funded Natural Sciences Engineering Research Council of metabolites involved biochemical reaction  e.,      Canada (NSERC) Microbial Genomics Biofuels Products stochiometric data) represented ker-           Biorefining Processes (MGCB2 project). <br /> nels added strength predicting model. example, reaction catalyzed fructose-bisphosphate      Author details <br />                                                               1 Department Science, University Manitoba, Winnipeg, <br /> aldolase [<span id='am-604' about='obo:IMR_0702342' typeof='owl:Thing'><span id='am-605' property="rdfs:label" content="ALDOC_HUMAN" datatype="xsd:string"></span><span id='am-606' property="oboInOwl:hasExactSynonym" content="aldolase C, fructose-bisphosphate" datatype="xsd:string"></span><span id='am-607' property="oboInOwl:hasExactSynonym" content="ALDC" datatype="xsd:string"></span><span id='am-608' property="rdfs:label" content="FRUCTOALDOLASE C" datatype="xsd:string"></span><span id='am-609' property="rdfs:label" content="Brain-type aldolase" datatype="xsd:string"></span><span id='am-610' property="rdfs:label" content="Fructose-bisphosphate aldolase C" datatype="xsd:string"></span><span id='am-611' property="rdfs:label" content="ALDOC" datatype="xsd:string"></span><span id='am-612' property="rdfs:label" content="ALDOLASE 3" datatype="xsd:string"></span><span id='am-613' property="oboInOwl:id" content="IMR:0702342" datatype="xsd:string"></span><span id='am-614' property="oboInOwl:hasExactSynonym" content="Fructose-bisphosphate aldolase C" datatype="xsd:string"></span><span id='am-615' property="oboInOwl:hasExactSynonym" content="ALDOC" datatype="xsd:string"></span><span id='am-616' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-617' property="oboInOwl:hasExactSynonym" content="FRUCTOALDOLASE C" datatype="xsd:string"></span><span id='am-618' property="rdfs:label" content="aldolase C, fructose-bisphosphate" datatype="xsd:string"></span><span id='am-619' property="oboInOwl:hasExactSynonym" content="ALDOLASE 3" datatype="xsd:string"></span><span id='am-620' property="oboInOwl:hasDbXref" content="UniProt:P09972" datatype="xsd:string"></span><span id='am-621' property="oboInOwl:hasExactSynonym" content="Brain-type aldolase" datatype="xsd:string"></span><span id='am-622' property="rdfs:label" content="ALDC" datatype="xsd:string"></span><span id='am-623' property="rdfs:label" content="EC 4.1.2.13" datatype="xsd:string"></span><span id='am-624' property="oboInOwl:hasExactSynonym" content="EC 4.1.2.13" datatype="xsd:string"></span>EC 4.1.2.13</span>] splits 1 molecule fructose          Manitoba, Canada. 2 Department Plant Science, University Manitoba, R3T <br /> 1,6-bisphosphate 2 molecules glyceraldehyde           2N2 Winnipeg, Manitoba, Canada. <br /> 3-phosphate, relative amounts substrate <br />                                                               Received: 11 April 2014 Accepted: 23 September 2014 product represented coefficients 1         Published: 26 September 2014 <br /> 2, respectively. stoichiometric kernel encode coefficients substrates products,    References <br />                                                               1. Faust K, Helden J: Predicting metabolic pathways sub-network <br /> enzymes interact stoichiometric            extraction. Bacterial Molecular Networks. Methods Molecular Biology. <br /> coefficients 0. authors [46-48] defined         Springer: New York; 2012:107 8211;130. <br />  Roche-Lima et al. BMC Bioinformatics 2014, 15:318                                                                                                   Page 13 13 <br /> http://www.biomedcentral.com/1471-2105/15/318 <br />  <br /> </body></html>