<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Predicting internal cell fluxes sub-optimal <br /> growth <br />  <br />  <br />  <br />   Abstract <br />   Background: Flux Balance Analysis (FBA) widely used tool model metabolic behavior cellular function. <br />   Applications FBA span breadth research synthetic engineering biofuels understanding evolutionary <br />   adaptations. FBA predicts metabolic reaction fluxes optimize given objective. objective generally defined <br />   unicellular organisms theoretical reaction simulates biomass production. FBA extremely <br />   successful predicting E. coli growth rates different media gene essentiality, things.   order improve predictions, additional constraints coupled optimization biomass function. Studies <br />   suggested,  unicellular organisms - like multicellular organisms - grow optimal rates.   improve FBA predictions, particularly internal cell fluxes, new techniques explore sub-optimal <br />   solution space need developed. <br />   Results: present innovative FBA method called corsoFBA based optimization <span id='am-1058' about='obo:IMR_0000949' typeof='owl:Thing obo:IMR_0000001'><span id='am-1059' property="rdfs:label" content="protein" datatype="xsd:string"></span><span id='am-1060' property="oboInOwl:id" content="IMR:0000949" datatype="xsd:string"></span><span id='am-1061' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>protein</span> cost   sub-optimal objective levels. method shows good agreement experimental data E. coli grown different <br />   dilution rates. Maintaining objective function close maximum value predicts metabolic states closely <br />   resemble low dilution rates; higher dilution rates mirrored lowering biomass production value.   using modified version Extreme Pathways, able quantify energy production overall protein <br />   cost possible pathways central carbon metabolism. <br />   Conclusion: Metabolic flux distributions optimal objective substantially different near-optimal <br />   distributions. Importantly, behavior E. coli central carbon metabolism better predicted exploring   sub-optimal FBA solution space. corsoFBA method presented able predict behavior <span id='am-821' about='obo:IMR_0704046' typeof='owl:Thing'><span id='am-822' property="rdfs:label" content="PROTEIN TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE, 22" datatype="xsd:string"></span><span id='am-823' property="oboInOwl:hasExactSynonym" content="PROTEIN TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE, 22" datatype="xsd:string"></span><span id='am-824' property="rdfs:label" content="Hematopoietic cell protein-tyrosine phosphatase 70Z-PEP" datatype="xsd:string"></span><span id='am-825' property="oboInOwl:hasExactSynonym" content="LyP" datatype="xsd:string"></span><span id='am-826' property="oboInOwl:hasExactSynonym" content="PEP" datatype="xsd:string"></span><span id='am-827' property="rdfs:label" content="PEP" datatype="xsd:string"></span><span id='am-828' property="oboInOwl:hasDbXref" content="UniProt:Q9Y2R2" datatype="xsd:string"></span><span id='am-829' property="rdfs:label" content="Lyp2" datatype="xsd:string"></span><span id='am-830' property="rdfs:label" content="PROTEIN-TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE 8, FORMERLY" datatype="xsd:string"></span><span id='am-831' property="rdfs:label" content="LyP" datatype="xsd:string"></span><span id='am-832' property="rdfs:label" content="PTN22_HUMAN" datatype="xsd:string"></span><span id='am-833' property="rdfs:label" content="protein tyrosine phosphatase, non-receptor type 22 (lymphoid)" datatype="xsd:string"></span><span id='am-834' property="oboInOwl:hasExactSynonym" content="PROTEIN-TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE 8, FORMERLY" datatype="xsd:string"></span><span id='am-835' property="oboInOwl:hasExactSynonym" content="LYP" datatype="xsd:string"></span><span id='am-836' property="oboInOwl:hasExactSynonym" content="LYP1" datatype="xsd:string"></span><span id='am-837' property="oboInOwl:hasExactSynonym" content="Hematopoietic cell protein-tyrosine phosphatase 70Z-PEP" datatype="xsd:string"></span><span id='am-838' property="rdfs:label" content="PTPN8" datatype="xsd:string"></span><span id='am-839' property="rdfs:label" content="PTPN8, FORMERLY" datatype="xsd:string"></span><span id='am-840' property="oboInOwl:hasExactSynonym" content="Lymphoid phosphatase" datatype="xsd:string"></span><span id='am-841' property="oboInOwl:hasExactSynonym" content="PTPN8" datatype="xsd:string"></span><span id='am-842' property="oboInOwl:hasExactSynonym" content="Lyp" datatype="xsd:string"></span><span id='am-843' property="oboInOwl:id" content="IMR:0704046" datatype="xsd:string"></span><span id='am-844' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein phosphatase non-receptor type 22" datatype="xsd:string"></span><span id='am-845' property="rdfs:label" content="Lyp1" datatype="xsd:string"></span><span id='am-846' property="oboInOwl:hasExactSynonym" content="PTPN22" datatype="xsd:string"></span><span id='am-847' property="rdfs:label" content="Lyp" datatype="xsd:string"></span><span id='am-848' property="rdfs:label" content="LYP" datatype="xsd:string"></span><span id='am-849' property="rdfs:label" content="LYP2" datatype="xsd:string"></span><span id='am-850' property="rdfs:label" content="Tyrosine-protein phosphatase non-receptor type 22" datatype="xsd:string"></span><span id='am-851' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-852' property="oboInOwl:hasExactSynonym" content="LYP2" datatype="xsd:string"></span><span id='am-853' property="oboInOwl:hasExactSynonym" content="Lyp1" datatype="xsd:string"></span><span id='am-854' property="oboInOwl:hasExactSynonym" content="protein tyrosine phosphatase, non-receptor type 22 (lymphoid)" datatype="xsd:string"></span><span id='am-855' property="rdfs:label" content="PEST-DOMAIN PHOSPHATASE" datatype="xsd:string"></span><span id='am-856' property="oboInOwl:hasExactSynonym" content="PEST-DOMAIN PHOSPHATASE" datatype="xsd:string"></span><span id='am-857' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-858' property="rdfs:label" content="PTPN22" datatype="xsd:string"></span><span id='am-859' property="rdfs:label" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-860' property="rdfs:label" content="Lymphoid phosphatase" datatype="xsd:string"></span><span id='am-861' property="rdfs:label" content="LYP1" datatype="xsd:string"></span><span id='am-862' property="oboInOwl:hasExactSynonym" content="Lyp2" datatype="xsd:string"></span><span id='am-863' property="oboInOwl:hasExactSynonym" content="PTPN8, FORMERLY" datatype="xsd:string"></span>PEP</span> <br />   <span id='am-1330' about='obo:IMR_0010046' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0010053 obo:IMR_0000001 obo:IMR_0000207'><span id='am-1331' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1332' property="oboInOwl:hasDbXref" content="EC:6.4.-.-" datatype="xsd:string"></span><span id='am-1333' property="oboInOwl:id" content="IMR:0010046" datatype="xsd:string"></span><span id='am-1334' property="rdfs:label" content="carboxylase" datatype="xsd:string"></span>Carboxylase</span>, glyoxylate shunt Entner-Doudoroff pathway different glucose levels, behavior   predicted minimization metabolic steps FBA  technique used better predict internal <br />   cell fluxes different conditions, corsoFBA great help study cells multicellular <br />   organisms using Flux Balance Analysis. <br />   Keywords: Flux balance analysis, COBRA, BiGG, Genome-wide metabolic reconstructions, Constraint based models, <br />   Extreme pathways, Sub-optimal growth, Protein cost <br />  <br />  <br /> Background                                                                          formulation reconstructions called Constraint <br /> Genome-wide metabolic reconstructions provide pow-                                Based Models. models defined core erful platform analysis metabolic pathways.                              sparse stoichiometric matrix, column repre- <br /> Reconstructions different species                           sents reaction, row metabolite, entry spanning fifty different genera, including humans                               corresponding stoichiometric coefficient. <br /> [1,2], available today [3]. reconstructions                             vast array computational methods analysis used successfully fields research,                               Constraint Based Models available, num- <br /> including metabolic engineering, evolutionary analysis                              ber continues grow [5,6]. widely used metabolic network properties [4]. mathematical                              methods Flux Balance Analysis (FBA). FBA <br />                                                                                     returns flux distribution entire metabolism <br /> *Correspondence: as86@rice.edu                                                      optimizes (minimizes maximizes) given objec- <br /> Department Bioengineering, Rice University, 6500 Main Street, Houston, USA <br />                                                                                     tive function reaction, ATP biomass <br />  <br />                                          169; 2015 Schultz Qutub; licensee BioMed Central. Open Access article distributed terms Creative <br />                                         Commons Attribution License (http://creativecommons.org/licenses 4.0), permits unrestricted use, distribution,                                         reproduction medium, provided original work properly credited. Creative Commons Public Domain Dedication <br />                                         waiver (http://creativecommons.org/publicdomain/zero/1.0/) applies data available article, unless                                         stated. <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                   Page 2 12 <br />  <br />  <br />  <br />  <br /> production. flux distribution satisfies steady-state      thermodynamic cost, miss key biological <br /> assumption, <span id='am-593' about='obo:IMR_0702135' typeof='owl:Thing'><span id='am-594' property="rdfs:label" content="NET" datatype="xsd:string"></span><span id='am-595' property="oboInOwl:hasExactSynonym" content="ERP" datatype="xsd:string"></span><span id='am-596' property="rdfs:label" content="ELK3, ETS-domain protein (SRF accessory protein 2)" datatype="xsd:string"></span><span id='am-597' property="oboInOwl:hasExactSynonym" content="SAP-2" datatype="xsd:string"></span><span id='am-598' property="oboInOwl:id" content="IMR:0702135" datatype="xsd:string"></span><span id='am-599' property="rdfs:label" content="Serum response factor accessory protein 2" datatype="xsd:string"></span><span id='am-600' property="oboInOwl:hasExactSynonym" content="SAP2" datatype="xsd:string"></span><span id='am-601' property="rdfs:label" content="SAP2" datatype="xsd:string"></span><span id='am-602' property="oboInOwl:hasExactSynonym" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-603' property="oboInOwl:hasExactSynonym" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-604' property="oboInOwl:hasExactSynonym" content="ELK3, ETS-domain protein (SRF accessory protein 2)" datatype="xsd:string"></span><span id='am-605' property="rdfs:label" content="ELK3_HUMAN" datatype="xsd:string"></span><span id='am-606' property="oboInOwl:hasExactSynonym" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-607' property="rdfs:label" content="ELK3" datatype="xsd:string"></span><span id='am-608' property="oboInOwl:hasDbXref" content="UniProt:P41970" datatype="xsd:string"></span><span id='am-609' property="rdfs:label" content="ERP" datatype="xsd:string"></span><span id='am-610' property="oboInOwl:hasExactSynonym" content="ELK3, ETS-DOMAIN PROTEIN" datatype="xsd:string"></span><span id='am-611' property="rdfs:label" content="SRF accessory protein 2" datatype="xsd:string"></span><span id='am-612' property="rdfs:label" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-613' property="rdfs:label" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-614' property="rdfs:label" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-615' property="oboInOwl:hasExactSynonym" content="NET" datatype="xsd:string"></span><span id='am-616' property="oboInOwl:hasExactSynonym" content="Serum response factor accessory protein 2" datatype="xsd:string"></span><span id='am-617' property="rdfs:label" content="ELK3, ETS-DOMAIN PROTEIN" datatype="xsd:string"></span><span id='am-618' property="oboInOwl:hasExactSynonym" content="ELK3" datatype="xsd:string"></span><span id='am-619' property="rdfs:label" content="SAP-2" datatype="xsd:string"></span><span id='am-620' property="oboInOwl:hasExactSynonym" content="SRF accessory protein 2" datatype="xsd:string"></span><span id='am-621' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-622' property="oboInOwl:hasExactSynonym" content="ETS-RELATED PROTEIN" datatype="xsd:string"></span><span id='am-623' property="rdfs:label" content="ETS-RELATED PROTEIN" datatype="xsd:string"></span>net</span> production          factors govern metabolism [16,28-33]. <br /> sumption metabolite. pre-defined set                Significant efforts directed inte- <br /> straints, upper lower bounds reaction       grating omics data FBA [34-38], generally low substrate availability, defined [7].                 correlation gene expression associated <br />   technique commonly applied metabolic           reaction flux makes challenging field [34].  <br /> reconstructions unicellular organisms opti-           predictive power integrated omics-FBA <br /> mization theoretical Biomass reaction. reaction        models limited, experimental data needed consumes resources amino acids, nucleotides          predict fluxes. <br /> ATP rate given organism need order              recent studies relied theory E. grow multiply [8,9]. FBA predicted           coli chooses pathways based minimization E. coli uptake release rates certain metabolites      protein cost [39,40] number metabolic steps [41]. <br /> [10-12], cell growth rate different environmental          successful methods based assumptions <br /> conditions [10,11] gene essentiality [10] great         proposed, include minimization <span id='am-624' about='obo:IMR_0702136' typeof='owl:Thing'><span id='am-625' property="rdfs:label" content="Elk3" datatype="xsd:string"></span><span id='am-626' property="oboInOwl:hasExactSynonym" content="Net" datatype="xsd:string"></span><span id='am-627' property="rdfs:label" content="ELK3, member of ETS oncogene family" datatype="xsd:string"></span><span id='am-628' property="rdfs:label" content="Erp" datatype="xsd:string"></span><span id='am-629' property="rdfs:label" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-630' property="oboInOwl:id" content="IMR:0702136" datatype="xsd:string"></span><span id='am-631' property="rdfs:label" content="ELK3_MOUSE" datatype="xsd:string"></span><span id='am-632' property="rdfs:label" content="Net" datatype="xsd:string"></span><span id='am-633' property="oboInOwl:hasExactSynonym" content="Sap-2" datatype="xsd:string"></span><span id='am-634' property="rdfs:label" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-635' property="rdfs:label" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-636' property="oboInOwl:hasDbXref" content="UniProt:P41971" datatype="xsd:string"></span><span id='am-637' property="oboInOwl:hasExactSynonym" content="ETS-related protein NET" datatype="xsd:string"></span><span id='am-638' property="oboInOwl:hasExactSynonym" content="ETS domain-containing protein Elk-3" datatype="xsd:string"></span><span id='am-639' property="oboInOwl:hasExactSynonym" content="Erp" datatype="xsd:string"></span><span id='am-640' property="oboInOwl:hasExactSynonym" content="ETS-related protein ERP" datatype="xsd:string"></span><span id='am-641' property="rdfs:label" content="Sap-2" datatype="xsd:string"></span><span id='am-642' property="oboInOwl:hasExactSynonym" content="Elk3" datatype="xsd:string"></span><span id='am-643' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-644' property="oboInOwl:hasExactSynonym" content="ELK3, member of ETS oncogene family" datatype="xsd:string"></span>net</span> <br /> success.  prediction internal cell fluxes         metabolic flux [12], minimization number steps <br /> remains challenge [13], mainly reasons:            metabolism [28,42,43] enzymatic level  <br />                                                                  straints [44-49]. alternative method utilizes Elemen- <br />   1. FBA solution unique.           tary Modes pre-define directionality reactions, <br />      fluxes model defined,      reducing FBA solution space [50]. <br />      exist certain bounds objective            methods successful improving <br />      function optimized [7]. fluctuations         predictions growth rate, substrate usage internal <br />      define FBA solution space,            cell fluxes unicellular organisms. studies      single, unique output.                                      suggested,  unicellular organisms reality <br />   2. Organisms operating maximum                 grow rates lower predicted FBA      capacity [14-18]. case, objective function      [14-17]. mentioned methods <br />      fully optimized, instead           predict growth rates lower standard FBA,      near-optimal sub-optimal state. Furthermore, Flux        rely implementing additional constraints opti- <br />      Variability Analysis shows FBA solution            mization objective function. Furthermore,      space increases drastically considering              approach successfully implemented      near-optimal optimal state [18], exacerbating        study cancer cells [49], objective function      possibility multiple FBA solutions.                      likely hold true analysis healthy multicellular <br />   3. observed metabolic state unique.       organisms, like mammals, healthy cells systems <br />       single flux distribution explain       complex objectives simply grow      flux states observed experiments, variation      multiply. <br />      occurs bacterial population. Wintermute             Additionally, <span id='am-289' about='obo:IMR_0703879' typeof='owl:Thing'><span id='am-290' property="rdfs:label" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-291' property="rdfs:label" content="PTN5_RAT" datatype="xsd:string"></span><span id='am-292' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-293' property="oboInOwl:hasExactSynonym" content="STEP" datatype="xsd:string"></span><span id='am-294' property="oboInOwl:hasExactSynonym" content="Ptpn5" datatype="xsd:string"></span><span id='am-295' property="rdfs:label" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-296' property="rdfs:label" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-297' property="oboInOwl:hasExactSynonym" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-298' property="rdfs:label" content="STEP" datatype="xsd:string"></span><span id='am-299' property="oboInOwl:hasDbXref" content="UniProt:P35234" datatype="xsd:string"></span><span id='am-300' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-301' property="rdfs:label" content="Ptpn5" datatype="xsd:string"></span><span id='am-302' property="oboInOwl:hasExactSynonym" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-303' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-304' property="oboInOwl:id" content="IMR:0703879" datatype="xsd:string"></span><span id='am-305' property="oboInOwl:hasExactSynonym" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-306' property="rdfs:label" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-307' property="rdfs:label" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span>step</span> optimization methods      et al. [19] proposes cloud theory metabolic regu-     additional constraints return single flux distribution, <br />      lation, bacteria allowed fluctuate      unable explore near-optimal solution <br />      near-optimal solution space. study demon-        space. limitation significant given recent study <br />      strates variability fluxes region   suggested flux distribution E. coli vary freely <br />      matches observed variability data.           near-optimal space [19].  order   4. Thermodynamically                                           improve FBA predictions, especially field <br />      infeasible loops appear FBA output.        expands include multicellular organisms, new tech- <br />      sets pathways termed  8220;type III pathways  65533;?            niques explore sub-optimal solution space need <br />      [20], composed internal reactions              developed. <br />      combined yield steady-state              address need, propose <span id='am-308' about='obo:IMR_0703880' typeof='owl:Thing'><span id='am-309' property="oboInOwl:hasExactSynonym" content="PROTEIN-TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE, 5" datatype="xsd:string"></span><span id='am-310' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-311' property="rdfs:label" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-312' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-313' property="rdfs:label" content="PTPN5" datatype="xsd:string"></span><span id='am-314' property="rdfs:label" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-315' property="rdfs:label" content="PTPN, STRIATUM-ENRICHED" datatype="xsd:string"></span><span id='am-316' property="rdfs:label" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-317' property="rdfs:label" content="STEP" datatype="xsd:string"></span><span id='am-318' property="rdfs:label" content="PROTEIN-TYROSINE PHOSPHATASE, NONRECEPTOR-TYPE, 5" datatype="xsd:string"></span><span id='am-319' property="oboInOwl:hasExactSynonym" content="PTPSTEP" datatype="xsd:string"></span><span id='am-320' property="oboInOwl:hasExactSynonym" content="PTPN5" datatype="xsd:string"></span><span id='am-321' property="oboInOwl:hasExactSynonym" content="STEP" datatype="xsd:string"></span><span id='am-322' property="oboInOwl:hasExactSynonym" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-323' property="rdfs:label" content="PTPSTEP" datatype="xsd:string"></span><span id='am-324' property="oboInOwl:hasExactSynonym" content="FLJ14427" datatype="xsd:string"></span><span id='am-325' property="rdfs:label" content="PTN5_HUMAN" datatype="xsd:string"></span><span id='am-326' property="oboInOwl:hasExactSynonym" content="protein tyrosine phosphatase, non-receptor type 5 (striatum-enriched)" datatype="xsd:string"></span><span id='am-327' property="rdfs:label" content="protein tyrosine phosphatase, non-receptor type 5 (striatum-enriched)" datatype="xsd:string"></span><span id='am-328' property="oboInOwl:id" content="IMR:0703880" datatype="xsd:string"></span><span id='am-329' property="oboInOwl:hasExactSynonym" content="PTPN, STRIATUM-ENRICHED" datatype="xsd:string"></span><span id='am-330' property="oboInOwl:hasExactSynonym" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-331' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-332' property="rdfs:label" content="FLJ14427" datatype="xsd:string"></span><span id='am-333' property="rdfs:label" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-334' property="oboInOwl:hasDbXref" content="UniProt:P54829" datatype="xsd:string"></span>step</span> optimiza- <br />      <span id='am-736' about='obo:IMR_0001104' typeof='owl:Thing obo:IMR_0000397 obo:IMR_0000949 obo:IMR_0000369 obo:IMR_0000368 obo:IMR_0000001'><span id='am-737' property="rdfs:label" content="Elk-3" datatype="xsd:string"></span><span id='am-738' property="oboInOwl:hasExactSynonym" content="Elk-3" datatype="xsd:string"></span><span id='am-739' property="rdfs:label" content="Sap-2" datatype="xsd:string"></span><span id='am-740' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-741' property="oboInOwl:id" content="IMR:0001104" datatype="xsd:string"></span><span id='am-742' property="oboInOwl:hasExactSynonym" content="Sap-2" datatype="xsd:string"></span><span id='am-743' property="rdfs:label" content="Net" datatype="xsd:string"></span>net</span> input output. cycles clutter        tion FBA method predicting internal fluxes, termed <br />      FBA output, hindering subsequent analysis [5].          COst Reduced Sub-Optimal FBA (corsoFBA),                                                                  suitable sub-optimal objectives. impos- <br />   better predict internal fluxes, studies           ing additional constraints optimizing objective <br /> relied additional thermodynamic constraints [21].         function, <span id='am-173' about='obo:IMR_0200603' typeof='owl:Thing'><span id='am-174' property="rdfs:label" content="AK3L1" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:hasExactSynonym" content="EC 2.7.4.10" datatype="xsd:string"></span><span id='am-176' property="rdfs:label" content="EC 2.7.4.10" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasExactSynonym" content="Adenylate kinase 3 alpha-like 1" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasExactSynonym" content="ADENYLATE KINASE 3-LIKE 1, FORMERLY" datatype="xsd:string"></span><span id='am-179' property="rdfs:label" content="Adenylate kinase 3 alpha-like 1" datatype="xsd:string"></span><span id='am-180' property="rdfs:label" content="adenylate kinase 3" datatype="xsd:string"></span><span id='am-181' property="rdfs:label" content="ADENYLATE KINASE 3-LIKE 1, FORMERLY" datatype="xsd:string"></span><span id='am-182' property="oboInOwl:hasExactSynonym" content="AK6" datatype="xsd:string"></span><span id='am-183' property="rdfs:label" content="FIX" datatype="xsd:string"></span><span id='am-184' property="rdfs:label" content="Adenylate kinase 3" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasExactSynonym" content="AK3L1" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasExactSynonym" content="AKL3L1" datatype="xsd:string"></span><span id='am-187' property="rdfs:label" content="GTP:AMP phosphotransferase, mitochondrial" datatype="xsd:string"></span><span id='am-188' property="oboInOwl:hasExactSynonym" content="FIX" datatype="xsd:string"></span><span id='am-189' property="oboInOwl:hasExactSynonym" content="adenylate kinase 3" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-191' property="rdfs:label" content="AKL3L1" datatype="xsd:string"></span><span id='am-192' property="oboInOwl:id" content="IMR:0200603" datatype="xsd:string"></span><span id='am-193' property="rdfs:label" content="AKL3L" datatype="xsd:string"></span><span id='am-194' property="rdfs:label" content="AK3" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasDbXref" content="UniProt:Q9UIJ7" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:hasExactSynonym" content="Adenylate kinase 3" datatype="xsd:string"></span><span id='am-197' property="rdfs:label" content="AK6" datatype="xsd:string"></span><span id='am-198' property="oboInOwl:hasExactSynonym" content="AK 3" datatype="xsd:string"></span><span id='am-199' property="rdfs:label" content="KAD3_HUMAN" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasExactSynonym" content="GTP:AMP phosphotransferase, mitochondrial" datatype="xsd:string"></span><span id='am-201' property="rdfs:label" content="AK 3" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasExactSynonym" content="AKL3L" datatype="xsd:string"></span><span id='am-203' property="oboInOwl:hasExactSynonym" content="AK3" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasExactSynonym" content="AK3L1, FORMERLY" datatype="xsd:string"></span><span id='am-205' property="rdfs:label" content="AK3L1, FORMERLY" datatype="xsd:string"></span>fix</span> objective predefined value. <br /> eral studies successfully reduce FBA solution       estimated protein cost metabolism <br /> space, leave variety states         minimized order predict internal cell <br /> thermodynamically feasible, physiologically improba-         fluxes. varying biomass production value, ble [22-27]. studies rely optimization         able predict changes pathway usage depend <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                   Page 3 12 <br />  <br />  <br />  <br />  changes conditions, particularly glucose                            temperature. molecular weight term included availability. method suited                       represent resources needed order pro- <br /> predicting growth rates, able predict metabolic                       duce sufficient enzymes maintain asso- <br /> flux distributions near-optimal solution space.                        ciated reaction flux. thermodynamic penalty, applied <br /> Furthermore, decomposing model using adapted                          reversible reactions, represents cost associ- <br /> version Extreme Pathway analysis, able                         ated change concentration metabolites <br /> break energy production pathways                           required reaction flow desired direc- <br /> understand model behavior glucose concentrations                         tion.  reaction positive  r G o require <br /> change. validate methods using E. coli                            large disparity concentration associated <br /> model organism.                                                                 metabolites, cost term multiplied positive <br />                                                                                 number larger  hand,  r G o Methods                                                                         negative, reaction favorable, cost <br /> Cost calculation implementation                                             term multiplied positive number zero methods successful predicting inter-                         given little metabolic adjustment <span id='am-1150' about='obo:IMR_0001797' typeof='owl:Thing'><span id='am-1151' property="oboInOwl:hasExactSynonym" content="necrotic" datatype="xsd:string"></span><span id='am-1152' property="oboInOwl:hasDbXref" content="UniProt:Q9U5W7" datatype="xsd:string"></span><span id='am-1153' property="rdfs:label" content="Serine protease inhibitor 3" datatype="xsd:string"></span><span id='am-1154' property="rdfs:label" content="Serpin, putative" datatype="xsd:string"></span><span id='am-1155' property="rdfs:label" content="Ser3" datatype="xsd:string"></span><span id='am-1156' property="rdfs:label" content="SER3" datatype="xsd:string"></span><span id='am-1157' property="oboInOwl:hasExactSynonym" content="SER3" datatype="xsd:string"></span><span id='am-1158' property="oboInOwl:hasExactSynonym" content="Ser3" datatype="xsd:string"></span><span id='am-1159' property="rdfs:label" content="Spn43Ac" datatype="xsd:string"></span><span id='am-1160' property="rdfs:label" content="CG1857" datatype="xsd:string"></span><span id='am-1161' property="oboInOwl:id" content="IMR:0001797" datatype="xsd:string"></span><span id='am-1162' property="oboInOwl:hasExactSynonym" content="anon-WO0118057.3" datatype="xsd:string"></span><span id='am-1163' property="oboInOwl:hasExactSynonym" content="Serpin, putative" datatype="xsd:string"></span><span id='am-1164' property="rdfs:label" content="anon-WO0118057.3" datatype="xsd:string"></span><span id='am-1165' property="rdfs:label" content="necrotic" datatype="xsd:string"></span><span id='am-1166' property="oboInOwl:hasExactSynonym" content="Spn43Ac" datatype="xsd:string"></span><span id='am-1167' property="rdfs:label" content="Q9U5W7_DROME" datatype="xsd:string"></span><span id='am-1168' property="rdfs:label" content="nec" datatype="xsd:string"></span><span id='am-1169' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1170' property="oboInOwl:hasExactSynonym" content="nec" datatype="xsd:string"></span><span id='am-1171' property="oboInOwl:hasExactSynonym" content="serpin43Ac" datatype="xsd:string"></span><span id='am-1172' property="oboInOwl:hasExactSynonym" content="CG1857" datatype="xsd:string"></span><span id='am-1173' property="rdfs:label" content="serpin43Ac" datatype="xsd:string"></span><span id='am-1174' property="oboInOwl:hasExactSynonym" content="Serine protease inhibitor 3" datatype="xsd:string"></span>nec</span>- <br /> nal cell fluxes based theory E. coli                      essary. worth noting thermodynamic cost subject protein cost constraint [44-49].                        does directly represent change <span id='am-122' about='obo:IMR_0000207' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001'><span id='am-123' property="obo:IAO_0000115" content="Protein that catalyze a specific chemical reaction." datatype="xsd:string"></span><span id='am-124' property="oboInOwl:id" content="IMR:0000207" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasDbXref" content="GO:0003824" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="enzyme" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>enzyme</span> concentra- methods available literature, cor-                       tion, increase decrease overall cost <br /> soFBA, summarized Table 1. explore                             associated reaction based standard Gibbs <br /> similar principle, calculate protein cost reac-                    free energy. Furthermore, term applied separately <br /> tion based net flux reaction (J),                        forward backward directions reversible reac- <br /> enzyme molecular weight (MW ), thermodynamic                              tions, favoring direction negative standard Gibbs <br /> penalty reversible reactions. cost term used                         free energy. similar thermodynamic cost, used  <br /> corsoFBA defined                                                          molecular weight considerations, applied                                <br />                    945;  183;  r G o                                                    Holzh 252;tter [28], weighing fluxes directed     J 183;MW  183;exp                                                                   thermodynamically unfavorable direction. <br />                     R 183;T                                                                                                                 mol <br />                                                                                   Finally, parameter  945; chosen 0.02RT kcal      , <br />    r G o standard Gibbs free energy                         making final thermodynamic cost simply: <br /> given reaction, R ideal <span id='am-752' about='obo:IMR_0002300' typeof='owl:Thing'><span id='am-753' property="rdfs:label" content="Gastrin-34" datatype="xsd:string"></span><span id='am-754' property="oboInOwl:hasExactSynonym" content="GAS" datatype="xsd:string"></span><span id='am-755' property="oboInOwl:hasExactSynonym" content="G71" datatype="xsd:string"></span><span id='am-756' property="rdfs:label" content="G34" datatype="xsd:string"></span><span id='am-757' property="rdfs:label" content="Gas" datatype="xsd:string"></span><span id='am-758' property="rdfs:label" content="GAST_MOUSE" datatype="xsd:string"></span><span id='am-759' property="oboInOwl:id" content="IMR:0002300" datatype="xsd:string"></span><span id='am-760' property="oboInOwl:hasExactSynonym" content="Gastrin-34" datatype="xsd:string"></span><span id='am-761' property="oboInOwl:hasExactSynonym" content="G34" datatype="xsd:string"></span><span id='am-762' property="rdfs:label" content="Gastrin-71" datatype="xsd:string"></span><span id='am-763' property="oboInOwl:hasDbXref" content="UniProt:P48757" datatype="xsd:string"></span><span id='am-764' property="oboInOwl:hasExactSynonym" content="Gas" datatype="xsd:string"></span><span id='am-765' property="oboInOwl:hasExactSynonym" content="gastrin" datatype="xsd:string"></span><span id='am-766' property="oboInOwl:hasExactSynonym" content="Gastrin-71" datatype="xsd:string"></span><span id='am-767' property="oboInOwl:hasExactSynonym" content="Big gastrin" datatype="xsd:string"></span><span id='am-768' property="oboInOwl:hasExactSynonym" content="Gastrin" datatype="xsd:string"></span><span id='am-769' property="rdfs:label" content="Gast" datatype="xsd:string"></span><span id='am-770' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-771' property="oboInOwl:hasExactSynonym" content="Gast" datatype="xsd:string"></span><span id='am-772' property="rdfs:label" content="GAS" datatype="xsd:string"></span><span id='am-773' property="rdfs:label" content="G71" datatype="xsd:string"></span><span id='am-774' property="rdfs:label" content="Gastrin" datatype="xsd:string"></span><span id='am-775' property="rdfs:label" content="Big gastrin" datatype="xsd:string"></span><span id='am-776' property="rdfs:label" content="gastrin" datatype="xsd:string"></span>gas</span> constant T                                                        <br />                                                                                                        o mol <br />                                                                                     exp 0.02  183;  r G             . <br />                                                                                                          kcal <br /> Table 1 Optimization methods additional constraints <br /> Method                             Enzymatic cost                  Constraint <br />                                                                                   value chosen order balance contribution <br />                                                                                 molecular weight thermodynamic penalty, yield- <br /> FBAwMC [44]                          <span id='am-670' about='obo:IMR_0703320' typeof='owl:Thing'><span id='am-671' property="oboInOwl:id" content="IMR:0703320" datatype="xsd:string"></span><span id='am-672' property="oboInOwl:hasExactSynonym" content="Type I arginase" datatype="xsd:string"></span><span id='am-673' property="oboInOwl:hasExactSynonym" content="Arg-1" datatype="xsd:string"></span><span id='am-674' property="oboInOwl:hasExactSynonym" content="arginase, liver" datatype="xsd:string"></span><span id='am-675' property="rdfs:label" content="Liver-type arginase" datatype="xsd:string"></span><span id='am-676' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-677' property="rdfs:label" content="Arginase-1" datatype="xsd:string"></span><span id='am-678' property="rdfs:label" content="arginase, liver" datatype="xsd:string"></span><span id='am-679' property="rdfs:label" content="Type I arginase" datatype="xsd:string"></span><span id='am-680' property="rdfs:label" content="PGIF" datatype="xsd:string"></span><span id='am-681' property="oboInOwl:hasExactSynonym" content="EC 3.5.3.1" datatype="xsd:string"></span><span id='am-682' property="oboInOwl:hasExactSynonym" content="AI" datatype="xsd:string"></span><span id='am-683' property="rdfs:label" content="Arg1" datatype="xsd:string"></span><span id='am-684' property="rdfs:label" content="ARGI1_MOUSE" datatype="xsd:string"></span><span id='am-685' property="rdfs:label" content="Arg-1" datatype="xsd:string"></span><span id='am-686' property="oboInOwl:hasExactSynonym" content="Liver-type arginase" datatype="xsd:string"></span><span id='am-687' property="oboInOwl:hasDbXref" content="UniProt:Q61176" datatype="xsd:string"></span><span id='am-688' property="oboInOwl:hasExactSynonym" content="PGIF" datatype="xsd:string"></span><span id='am-689' property="oboInOwl:hasExactSynonym" content="Arginase-1" datatype="xsd:string"></span><span id='am-690' property="oboInOwl:hasExactSynonym" content="Arg1" datatype="xsd:string"></span><span id='am-691' property="rdfs:label" content="EC 3.5.3.1" datatype="xsd:string"></span><span id='am-692' property="rdfs:label" content="AI" datatype="xsd:string"></span>ai</span> <span id='am-1375' about='obo:IMR_0701306' typeof='owl:Thing'><span id='am-1376' property="oboInOwl:hasExactSynonym" content="GMEM" datatype="xsd:string"></span><span id='am-1377' property="rdfs:label" content="TNC" datatype="xsd:string"></span><span id='am-1378' property="oboInOwl:hasExactSynonym" content="tenascin C" datatype="xsd:string"></span><span id='am-1379' property="oboInOwl:hasDbXref" content="UniProt:P24821" datatype="xsd:string"></span><span id='am-1380' property="rdfs:label" content="GMEM" datatype="xsd:string"></span><span id='am-1381' property="rdfs:label" content="Myotendinous antigen" datatype="xsd:string"></span><span id='am-1382' property="rdfs:label" content="GP" datatype="xsd:string"></span><span id='am-1383' property="rdfs:label" content="Tenascin-C" datatype="xsd:string"></span><span id='am-1384' property="oboInOwl:hasExactSynonym" content="GP 150-225" datatype="xsd:string"></span><span id='am-1385' property="rdfs:label" content="Cytotactin" datatype="xsd:string"></span><span id='am-1386' property="rdfs:label" content="MGC167029" datatype="xsd:string"></span><span id='am-1387' property="oboInOwl:hasExactSynonym" content="150-225" datatype="xsd:string"></span><span id='am-1388' property="rdfs:label" content="TN-C" datatype="xsd:string"></span><span id='am-1389' property="rdfs:label" content="GP 150-225" datatype="xsd:string"></span><span id='am-1390' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1391' property="oboInOwl:hasExactSynonym" content="TNC" datatype="xsd:string"></span><span id='am-1392' property="oboInOwl:hasExactSynonym" content="GP" datatype="xsd:string"></span><span id='am-1393' property="oboInOwl:hasExactSynonym" content="Cytotactin" datatype="xsd:string"></span><span id='am-1394' property="oboInOwl:hasExactSynonym" content="Tenascin-C" datatype="xsd:string"></span><span id='am-1395' property="rdfs:label" content="Glioma-associated-extracellular matrix antigen" datatype="xsd:string"></span><span id='am-1396' property="oboInOwl:hasExactSynonym" content="Hexabrachion" datatype="xsd:string"></span><span id='am-1397' property="oboInOwl:hasExactSynonym" content="Neuronectin" datatype="xsd:string"></span><span id='am-1398' property="rdfs:label" content="tenascin C" datatype="xsd:string"></span><span id='am-1399' property="rdfs:label" content="TN" datatype="xsd:string"></span><span id='am-1400' property="oboInOwl:hasExactSynonym" content="Myotendinous antigen" datatype="xsd:string"></span><span id='am-1401' property="rdfs:label" content="JI" datatype="xsd:string"></span><span id='am-1402' property="oboInOwl:hasExactSynonym" content="Glioma-associated-extracellular matrix antigen" datatype="xsd:string"></span><span id='am-1403' property="oboInOwl:id" content="IMR:0701306" datatype="xsd:string"></span><span id='am-1404' property="rdfs:label" content="Neuronectin" datatype="xsd:string"></span><span id='am-1405' property="oboInOwl:hasExactSynonym" content="TN-C" datatype="xsd:string"></span><span id='am-1406' property="oboInOwl:hasExactSynonym" content="MGC167029" datatype="xsd:string"></span><span id='am-1407' property="rdfs:label" content="150-225" datatype="xsd:string"></span><span id='am-1408' property="rdfs:label" content="HXB" datatype="xsd:string"></span><span id='am-1409' property="oboInOwl:hasExactSynonym" content="HXB" datatype="xsd:string"></span><span id='am-1410' property="oboInOwl:hasExactSynonym" content="JI" datatype="xsd:string"></span><span id='am-1411' property="rdfs:label" content="Hexabrachion" datatype="xsd:string"></span><span id='am-1412' property="rdfs:label" content="TENA_HUMAN" datatype="xsd:string"></span><span id='am-1413' property="oboInOwl:hasExactSynonym" content="Tenascin" datatype="xsd:string"></span><span id='am-1414' property="rdfs:label" content="Tenascin" datatype="xsd:string"></span><span id='am-1415' property="oboInOwl:hasExactSynonym" content="TN" datatype="xsd:string"></span>Ji</span>                          65533;? <br />                                      <br />                                                                                 ing prediction considering costs (Additional file 1: <br /> MOMENT [46]                          <span id='am-907' about='obo:IMR_0704094' typeof='owl:Thing'><span id='am-908' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-909' property="rdfs:label" content="GNAI1_HUMAN" datatype="xsd:string"></span><span id='am-910' property="rdfs:label" content="GNAI1" datatype="xsd:string"></span><span id='am-911' property="rdfs:label" content="ADENYLATE CYCLASE INHIBITORY PROTEIN" datatype="xsd:string"></span><span id='am-912' property="oboInOwl:hasExactSynonym" content="G PROTEIN, ALPHA-INHIBITING 1" datatype="xsd:string"></span><span id='am-913' property="oboInOwl:hasExactSynonym" content="Adenylate cyclase-inhibiting G alpha protein" datatype="xsd:string"></span><span id='am-914' property="rdfs:label" content="GUANINE NUCLEOTIDE-BINDING PROTEIN, ALPHA-INHIBITING ACTIVITY POLYPEPTIDE 1" datatype="xsd:string"></span><span id='am-915' property="oboInOwl:hasExactSynonym" content="Gi" datatype="xsd:string"></span><span id='am-916' property="oboInOwl:hasExactSynonym" content="INHIBITORY G PROTEIN" datatype="xsd:string"></span><span id='am-917' property="rdfs:label" content="guanine nucleotide binding protein (G protein), alpha inhibiting activity polypeptide 1" datatype="xsd:string"></span><span id='am-918' property="oboInOwl:hasExactSynonym" content="ADENYLATE CYCLASE INHIBITORY PROTEIN" datatype="xsd:string"></span><span id='am-919' property="oboInOwl:hasDbXref" content="UniProt:P63096" datatype="xsd:string"></span><span id='am-920' property="oboInOwl:hasExactSynonym" content="GNAI1" datatype="xsd:string"></span><span id='am-921' property="rdfs:label" content="Gi" datatype="xsd:string"></span><span id='am-922' property="oboInOwl:hasExactSynonym" content="guanine nucleotide binding protein (G protein), alpha inhibiting activity polypeptide 1" datatype="xsd:string"></span><span id='am-923' property="rdfs:label" content="INHIBITORY G PROTEIN" datatype="xsd:string"></span><span id='am-924' property="rdfs:label" content="G PROTEIN, ALPHA-INHIBITING 1" datatype="xsd:string"></span><span id='am-925' property="rdfs:label" content="Guanine nucleotide-binding protein G(i) subunit alpha-1" datatype="xsd:string"></span><span id='am-926' property="oboInOwl:hasExactSynonym" content="Guanine nucleotide-binding protein G(i) subunit alpha-1" datatype="xsd:string"></span><span id='am-927' property="oboInOwl:hasExactSynonym" content="GUANINE NUCLEOTIDE-BINDING PROTEIN, ALPHA-INHIBITING ACTIVITY POLYPEPTIDE 1" datatype="xsd:string"></span><span id='am-928' property="rdfs:label" content="Adenylate cyclase-inhibiting G alpha protein" datatype="xsd:string"></span><span id='am-929' property="oboInOwl:id" content="IMR:0704094" datatype="xsd:string"></span>gi</span>  183; MWi                       8804;C           Figure <span id='am-1297' about='obo:IMR_0702938' typeof='owl:Thing'><span id='am-1298' property="oboInOwl:hasExactSynonym" content="MGC133041" datatype="xsd:string"></span><span id='am-1299' property="oboInOwl:hasExactSynonym" content="MGC133040" datatype="xsd:string"></span><span id='am-1300' property="rdfs:label" content="26S proteasome regulatory subunit RPN2" datatype="xsd:string"></span><span id='am-1301' property="oboInOwl:hasDbXref" content="UniProt:Q99460" datatype="xsd:string"></span><span id='am-1302' property="oboInOwl:hasExactSynonym" content="PSMD1" datatype="xsd:string"></span><span id='am-1303' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S1" datatype="xsd:string"></span><span id='am-1304' property="oboInOwl:id" content="IMR:0702938" datatype="xsd:string"></span><span id='am-1305' property="rdfs:label" content="Rpn2" datatype="xsd:string"></span><span id='am-1306' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 1" datatype="xsd:string"></span><span id='am-1307' property="rdfs:label" content="MGC133040" datatype="xsd:string"></span><span id='am-1308' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 1" datatype="xsd:string"></span><span id='am-1309' property="oboInOwl:hasExactSynonym" content="26S proteasome subunit p112" datatype="xsd:string"></span><span id='am-1310' property="oboInOwl:hasExactSynonym" content="P112" datatype="xsd:string"></span><span id='am-1311' property="oboInOwl:hasExactSynonym" content="S1" datatype="xsd:string"></span><span id='am-1312' property="rdfs:label" content="26S proteasome subunit p112" datatype="xsd:string"></span><span id='am-1313' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 1" datatype="xsd:string"></span><span id='am-1314' property="rdfs:label" content="PSMD1" datatype="xsd:string"></span><span id='am-1315' property="rdfs:label" content="P112" datatype="xsd:string"></span><span id='am-1316' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN2" datatype="xsd:string"></span><span id='am-1317' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1318' property="rdfs:label" content="MGC133041" datatype="xsd:string"></span><span id='am-1319' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 1" datatype="xsd:string"></span><span id='am-1320' property="rdfs:label" content="S1" datatype="xsd:string"></span><span id='am-1321' property="oboInOwl:hasExactSynonym" content="Rpn2" datatype="xsd:string"></span><span id='am-1322' property="rdfs:label" content="26S proteasome regulatory subunit S1" datatype="xsd:string"></span><span id='am-1323' property="rdfs:label" content="PSMD1_HUMAN" datatype="xsd:string"></span>S1</span>). parameter, exponen- <br />                                                 <br /> Tepper et al. [47]                   Mi +  948;  183; gi                   minimize     tial nature thermodynamic penalty, term                                      MWi Ji                                     quickly approach extremely large numbers  positive <br /> Shlomi et al. [49]                                                  8804;C <br />                                                                                 values  r G o ) zero  negative values  r G o ),                                         kcati <br />                                      <br /> Holzh 252;tter [28]                        Ji++ keqi  183; Ji 65533;?            minimize <br />                                                                                 thermodynamic cost essentially consid- <br />                                                        945; 183; r G o <br /> corsoFBA                             Ji  183;MWi  183;exp       R 183;T <br />                                                                                                                                minimize     ered. case, reaction directionality essentially <br />                                  Variables                                      predetermined based solely standard Gibbs free - crowding coefficient                                                        energy, problematic. <br />                                                                                   Gibbs free energy reactions estimated J - flux reaction <br />                                                                                 MetaCyc database [51] change Gibbs energy <br /> g - enzyme concentration                                                        formation reaction compounds. Molecular <br /> MW - molecular weight associated enzyme                                      weight values obtained 506 523 (96.75%) <br /> M - metabolite levels                                                           enzyme catalyzed reactions E. coli iJR904 recon- <br />  948; - model parameter                                                             struction [52]. values extracted kcat - turnover number <br />                                                                                 BRENDA database [53] available. E. coli mea- <br />                                                                                 surements available, extracted values <br /> keq - thermodynamic equilibrium constant <br />                                                                                 organism closely related E. coli. small <br /> C - fraction dry weight mass associated proteins                        number molecular weights estimated methods calculate overall cost metabolism summing       EcoCyc database [54], remaining 17 values individual costs associated reaction. previous methods used constrain FBA solution, corsoFBA optimizes cost   defined median calculated values. Additional <br /> given growth rate.                                                              information concerning standard Gibbs free energy <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                 Page 4 12 <br />  <br />  <br />  <br />  reactions, protein molecular weights protein cost         applied small networks. number calculations supplemental information        pathways obtained methods, (Additional file 2, Additional file 3 Additional file 4:     computational cost analyses, increase expo- <br /> Table S1).                                                       nentially size reconstruction [5,59].   protein cost term scales linearly           large-scale metabolic reconstructions, decomposi- <br /> flux associated reaction, incorporated        tions quickly impractical tools. propose <br /> molecular weight thermodynamic cost directly            modified version Extreme Pathways yields model. enzyme associated reactions <span id='am-530' about='obo:IMR_0100713' typeof='owl:Thing'><span id='am-531' property="rdfs:label" content="notch" datatype="xsd:string"></span><span id='am-532' property="rdfs:label" content="anon-EST:Liang-1.12" datatype="xsd:string"></span><span id='am-533' property="oboInOwl:hasExactSynonym" content="Neurogenic locus Notch protein" datatype="xsd:string"></span><span id='am-534' property="oboInOwl:hasExactSynonym" content="Chopped" datatype="xsd:string"></span><span id='am-535' property="oboInOwl:hasExactSynonym" content="Abruptex" datatype="xsd:string"></span><span id='am-536' property="rdfs:label" content="Processed neurogenic locus Notch protein" datatype="xsd:string"></span><span id='am-537' property="oboInOwl:hasExactSynonym" content="clone 1.12" datatype="xsd:string"></span><span id='am-538' property="rdfs:label" content="strawberry" datatype="xsd:string"></span><span id='am-539' property="rdfs:label" content="Chp" datatype="xsd:string"></span><span id='am-540' property="rdfs:label" content="facet" datatype="xsd:string"></span><span id='am-541' property="oboInOwl:hasExactSynonym" content="swb" datatype="xsd:string"></span><span id='am-542' property="oboInOwl:hasExactSynonym" content="Notch" datatype="xsd:string"></span><span id='am-543' property="rdfs:label" content="16-55" datatype="xsd:string"></span><span id='am-544' property="rdfs:label" content="Abruptex" datatype="xsd:string"></span><span id='am-545' property="oboInOwl:hasExactSynonym" content="16-178" datatype="xsd:string"></span><span id='am-546' property="oboInOwl:hasExactSynonym" content="CG3936" datatype="xsd:string"></span><span id='am-547' property="oboInOwl:hasExactSynonym" content="l(1)Ax" datatype="xsd:string"></span><span id='am-548' property="rdfs:label" content="l(1)3Cb" datatype="xsd:string"></span><span id='am-549' property="oboInOwl:hasExactSynonym" content="l(1)3Cb" datatype="xsd:string"></span><span id='am-550' property="oboInOwl:hasExactSynonym" content="Confluens" datatype="xsd:string"></span><span id='am-551' property="rdfs:label" content="split" datatype="xsd:string"></span><span id='am-552' property="oboInOwl:hasExactSynonym" content="CT13012" datatype="xsd:string"></span><span id='am-553' property="rdfs:label" content="l(1)N" datatype="xsd:string"></span><span id='am-554' property="rdfs:label" content="Notch" datatype="xsd:string"></span><span id='am-555' property="rdfs:label" content="Confluens" datatype="xsd:string"></span><span id='am-556' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-557' property="rdfs:label" content="EG:140G11.1" datatype="xsd:string"></span><span id='am-558' property="oboInOwl:hasExactSynonym" content="Nintra" datatype="xsd:string"></span><span id='am-559' property="rdfs:label" content="notchoid" datatype="xsd:string"></span><span id='am-560' property="rdfs:label" content="EG:163A10.2" datatype="xsd:string"></span><span id='am-561' property="rdfs:label" content="n[fah]" datatype="xsd:string"></span><span id='am-562' property="rdfs:label" content="CG3936" datatype="xsd:string"></span><span id='am-563' property="rdfs:label" content="Nintra" datatype="xsd:string"></span><span id='am-564' property="rdfs:label" content="shd" datatype="xsd:string"></span><span id='am-565' property="oboInOwl:hasExactSynonym" content="Chp" datatype="xsd:string"></span><span id='am-566' property="oboInOwl:hasExactSynonym" content="strawberry" datatype="xsd:string"></span><span id='am-567' property="oboInOwl:hasExactSynonym" content="notch" datatype="xsd:string"></span><span id='am-568' property="oboInOwl:hasDbXref" content="UniProt:P07207" datatype="xsd:string"></span><span id='am-569' property="oboInOwl:hasExactSynonym" content="16-55" datatype="xsd:string"></span><span id='am-570' property="oboInOwl:hasExactSynonym" content="Processed neurogenic locus Notch protein" datatype="xsd:string"></span><span id='am-571' property="rdfs:label" content="dNotch" datatype="xsd:string"></span><span id='am-572' property="rdfs:label" content="CT13012" datatype="xsd:string"></span><span id='am-573' property="oboInOwl:hasExactSynonym" content="dNotch" datatype="xsd:string"></span><span id='am-574' property="oboInOwl:hasExactSynonym" content="EG:163A10.2" datatype="xsd:string"></span><span id='am-575' property="rdfs:label" content="l(1)Ax" datatype="xsd:string"></span><span id='am-576' property="oboInOwl:id" content="IMR:0100713" datatype="xsd:string"></span><span id='am-577' property="oboInOwl:hasExactSynonym" content="split" datatype="xsd:string"></span><span id='am-578' property="oboInOwl:hasExactSynonym" content="notchoid" datatype="xsd:string"></span><span id='am-579' property="oboInOwl:hasExactSynonym" content="shd" datatype="xsd:string"></span><span id='am-580' property="oboInOwl:hasExactSynonym" content="facet" datatype="xsd:string"></span><span id='am-581' property="rdfs:label" content="clone 1.12" datatype="xsd:string"></span><span id='am-582' property="oboInOwl:hasExactSynonym" content="spl" datatype="xsd:string"></span><span id='am-583' property="rdfs:label" content="16-178" datatype="xsd:string"></span><span id='am-584' property="oboInOwl:hasExactSynonym" content="l(1)N" datatype="xsd:string"></span><span id='am-585' property="oboInOwl:hasExactSynonym" content="n[fah]" datatype="xsd:string"></span><span id='am-586' property="rdfs:label" content="Neurogenic locus Notch protein" datatype="xsd:string"></span><span id='am-587' property="rdfs:label" content="NOTCH_DROME" datatype="xsd:string"></span><span id='am-588' property="rdfs:label" content="Chopped" datatype="xsd:string"></span><span id='am-589' property="oboInOwl:hasExactSynonym" content="anon-EST:Liang-1.12" datatype="xsd:string"></span><span id='am-590' property="rdfs:label" content="swb" datatype="xsd:string"></span><span id='am-591' property="rdfs:label" content="spl" datatype="xsd:string"></span><span id='am-592' property="oboInOwl:hasExactSynonym" content="EG:140G11.1" datatype="xsd:string"></span>split</span>       significantly lower number pathways decom- <br /> forward backward parts, cost added          position. pathways associated pro- <br /> produced metabolite:                                             tein cost ATP production potential, helps <br />                                                                  elucidate model behavior value objective <br />    Initial Reaction <br />                                                                  function varies. <br />    rxn : + B  65533;? 65533;?C + D                                             large number pathways stems par- <br />    Reactions cost                                           tially need balance pathway steady- <br />                                                        <br />                                         945;  183;  r G orxn             state, metabolite net production    rxnf : + B  65533;?C + D + MWrxn  183;exp                             consumption rate. Certain metabolites,  known <br />                                            R 183;T <br />                                                                  currency metabolites, responsible basic cell <br />                                         8722; 65533;? 183;  r G orxn <br />    rxnb : C + D  65533; + B + MWrxn  183;exp                             functions occur numerous reactions model. <br />                                             R 183;T                  Standard reactions involving metabolites,    reaction consumes produced metabolite            ATP hydrolysis, obtained different com- added model. original reac-          binations model reactions. combinations, <br /> tion model produces cost rate relative   loops, used multiple times absolute flux, added reaction       purpose, increasing number pathways consume cost. Minimizing flux reac-        essentially core reactions. <br /> tion return flux distribution lowest         phenomenon illustrated Extreme Path- <br /> enzyme associated cost achieve predefined biomass         ways analysis applied pathway set reac- <br /> production value. Reactions associated enzyme            tions presented Figure 1. pathway depicted given cost zero.                                       Figure 1A production energy  <br />    methods applied E. coli iJR904 recon-       version glucose lactate, produces mols <br /> struction [52]. simulations glucose uptake        ATP mol glucose. Reaction names bound set arbitrary value. Uptake bounds           extracted E. coli reconstruction Orth et al. <br /> CO2 , Fe2 , H2 O, H, Na, NH4 , O2 , Phosphate, SO4      [60]. Figure 1B illustrates loops reactions present set arbitrarily large values, larger <span id='am-1335' about='obo:IMR_0002833' typeof='owl:Thing'><span id='am-1336' property="rdfs:label" content="Glucose transporter 1" datatype="xsd:string"></span><span id='am-1337' property="oboInOwl:hasExactSynonym" content="Dmglut1" datatype="xsd:string"></span><span id='am-1338' property="oboInOwl:hasExactSynonym" content="Glut1" datatype="xsd:string"></span><span id='am-1339' property="oboInOwl:hasExactSynonym" content="Glucose transporter 1" datatype="xsd:string"></span><span id='am-1340' property="oboInOwl:id" content="IMR:0002833" datatype="xsd:string"></span><span id='am-1341' property="rdfs:label" content="CG1086" datatype="xsd:string"></span><span id='am-1342' property="rdfs:label" content="Glucose transporter type 1" datatype="xsd:string"></span><span id='am-1343' property="rdfs:label" content="GTR1_DROME" datatype="xsd:string"></span><span id='am-1344' property="oboInOwl:hasDbXref" content="UniProt:Q8IRI6" datatype="xsd:string"></span><span id='am-1345' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1346' property="rdfs:label" content="glut1" datatype="xsd:string"></span><span id='am-1347' property="oboInOwl:hasExactSynonym" content="glut1" datatype="xsd:string"></span><span id='am-1348' property="oboInOwl:hasExactSynonym" content="glucose transporter 1" datatype="xsd:string"></span><span id='am-1349' property="rdfs:label" content="glucose transporter 1" datatype="xsd:string"></span><span id='am-1350' property="rdfs:label" content="Dmglut1" datatype="xsd:string"></span><span id='am-1351' property="oboInOwl:hasExactSynonym" content="GLU" datatype="xsd:string"></span><span id='am-1352' property="rdfs:label" content="GLU" datatype="xsd:string"></span><span id='am-1353' property="oboInOwl:hasExactSynonym" content="CG1086" datatype="xsd:string"></span><span id='am-1354' property="rdfs:label" content="Glut1" datatype="xsd:string"></span><span id='am-1355' property="oboInOwl:hasExactSynonym" content="Glucose transporter type 1" datatype="xsd:string"></span>glu</span>-       reconstruction hydrolyze ATP, cose uptake bound, considered present             combined reactions Figure 1A pro- excess. exchange reactions release bounds set        duce pathway satisfying ATP steady-state condition. arbitrarily large values glucose, formate      Furthermore, ATPS4r introduces proton imbalance <br /> O2 , set zero. Predicted flux values          hydrolyzing ATP. imbalance cor- normalized flux glucose glucose-        rected proton transport loops presented <br /> 6-phosphate overall flux enzyme associated            Figure 1B. combination pathway reactions.                                                       given loops multiple Extreme Pathways    worth noting protein cost term does       essentially core reactions; represent account enzyme turnover rates substrate affin-        phenomenon - conversion glucose lactate <br /> ity. opted cost term kinetic          glycolysis. <br /> parameters enzymes extremely difficult obtain             order extract core reactions path- <br /> experimentally, vary greatly based pH temper-         way, remove selected set currency metabo- <br /> ature values. Furthermore, simplification constant      lites reconstruction. removal reduces turnover rate substrate affinity enzymes         number pathways obtain Extreme Pathway shown yield significant results [39].                    analysis. metabolite removal performed                                                                  certain considerations: <br /> Fundamental pathways analysis <br /> Different methods proposed characterize               1. Metabolites removed completely deleted decompose FBA solutions space [5,55], includ-                 reactions compartments model. <br /> ing Elementary Modes [56,57], Extreme Pathways [20]                   Removing metabolite reaction Minimal Generators [58].  tools                block certain reactions, fewer <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                         Page 5 12 <br />  <br />  <br />  <br />  <br />  Figure 1 Metabolic loops increase pathway multiplicity.   Pathway converting 1 mol glucose 2 mols lactate producing 2 mols  ATP. Reaction fluxes represented parentheses. Extracellular metabolites defined [e]. metabolites cytosolic space. <br />  (B) Network loops reactions hydrolyze ATP transport protons cell membrane. reactions loops used  balance reactions  , yielding multiple balanced pathways set backbone reactions. <br />  <br />  <br />  <br />  <br />      options balance metabolite present                     removal currency metabolites reconstruc- <br />      reconstruction.                                                  tion. reactions   yield ATP, <br />   2. reaction sink source                           ADP, phosphate, proton water imbalances orig- <br />      metabolites. Allowing reactions sinks                      inal model. Fundamental Pathways analysis,      sources introduce exchange reactions                          resulting fundamental pathway associated      certain metabolites, affecting model                                imbalance, used analyze      decomposition.                                                          pathway. study, imbalance used analyze <br />   3. Reactions removed reconstruction                        ATP production capacity pathway obtained. <br />      contain metabolite      removed reconstruction. removal                         Results discussion <br />      reactions remove pathway                          Comparison different cost functions <br />      uses reactions balance metabolite                 Cost-optimal flux simulations performed using      model.                                                              different types cost: (1) molecular weights  (2) <br />                                                                              thermodynamic penalties  (3) previously men- <br />   aware considerations                      tioned combination molecular weights thermo- <br /> modifying model, breaking specifications                    dynamic penalties (4) uniform costs. use invalidate Fundamental Pathway analy-                         uniform costs gives simple minimization  <br /> <span id='am-940' about='obo:IMR_0704301' typeof='owl:Thing'><span id='am-941' property="oboInOwl:hasExactSynonym" content="Pdgfb" datatype="xsd:string"></span><span id='am-942' property="rdfs:label" content="Platelet-derived growth factor B chain" datatype="xsd:string"></span><span id='am-943' property="rdfs:label" content="platelet derived growth factor, B polypeptide" datatype="xsd:string"></span><span id='am-944' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-945' property="oboInOwl:hasDbXref" content="UniProt:P31240" datatype="xsd:string"></span><span id='am-946' property="oboInOwl:hasExactSynonym" content="PDGF-2" datatype="xsd:string"></span><span id='am-947' property="oboInOwl:hasExactSynonym" content="Platelet-derived growth factor B chain" datatype="xsd:string"></span><span id='am-948' property="rdfs:label" content="PDGF subunit B" datatype="xsd:string"></span><span id='am-949' property="oboInOwl:hasExactSynonym" content="PDGF subunit B" datatype="xsd:string"></span><span id='am-950' property="oboInOwl:hasExactSynonym" content="Proto-oncogene c-Sis" datatype="xsd:string"></span><span id='am-951' property="oboInOwl:hasExactSynonym" content="PDGF-B" datatype="xsd:string"></span><span id='am-952' property="oboInOwl:hasExactSynonym" content="Platelet-derived growth factor subunit B" datatype="xsd:string"></span><span id='am-953' property="oboInOwl:id" content="IMR:0704301" datatype="xsd:string"></span><span id='am-954' property="rdfs:label" content="Proto-oncogene c-Sis" datatype="xsd:string"></span><span id='am-955' property="rdfs:label" content="PDGFB_MOUSE" datatype="xsd:string"></span><span id='am-956' property="rdfs:label" content="Sis" datatype="xsd:string"></span><span id='am-957' property="oboInOwl:hasExactSynonym" content="platelet derived growth factor, B polypeptide" datatype="xsd:string"></span><span id='am-958' property="rdfs:label" content="Platelet-derived growth factor subunit B" datatype="xsd:string"></span><span id='am-959' property="rdfs:label" content="PDGF-B" datatype="xsd:string"></span><span id='am-960' property="oboInOwl:hasExactSynonym" content="Platelet-derived growth factor beta polypeptide" datatype="xsd:string"></span><span id='am-961' property="oboInOwl:hasExactSynonym" content="Sis" datatype="xsd:string"></span><span id='am-962' property="rdfs:label" content="PDGF-2" datatype="xsd:string"></span><span id='am-963' property="rdfs:label" content="Platelet-derived growth factor beta polypeptide" datatype="xsd:string"></span><span id='am-964' property="rdfs:label" content="Pdgfb" datatype="xsd:string"></span>sis</span>. model decomposition performed,                       flux enzyme associated reactions. limitations considered.                         minimization strategy modified version widely illustrated Fundamental Pathways method using                      regarded <span id='am-335' about='obo:IMR_0703881' typeof='owl:Thing'><span id='am-336' property="oboInOwl:hasExactSynonym" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-337' property="rdfs:label" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-338' property="oboInOwl:hasExactSynonym" content="STEP" datatype="xsd:string"></span><span id='am-339' property="oboInOwl:hasExactSynonym" content="EC 3.1.3.48" datatype="xsd:string"></span><span id='am-340' property="rdfs:label" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-341' property="oboInOwl:hasExactSynonym" content="Neural-specific protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-342' property="rdfs:label" content="PTN5_MOUSE" datatype="xsd:string"></span><span id='am-343' property="rdfs:label" content="Tyrosine-protein phosphatase non-receptor type 5" datatype="xsd:string"></span><span id='am-344' property="rdfs:label" content="Ptpn5" datatype="xsd:string"></span><span id='am-345' property="oboInOwl:id" content="IMR:0703881" datatype="xsd:string"></span><span id='am-346' property="rdfs:label" content="STEP" datatype="xsd:string"></span><span id='am-347' property="rdfs:label" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-348' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-349' property="oboInOwl:hasExactSynonym" content="Ptpn5" datatype="xsd:string"></span><span id='am-350' property="oboInOwl:hasExactSynonym" content="Striatum-enriched protein-tyrosine phosphatase" datatype="xsd:string"></span><span id='am-351' property="oboInOwl:hasExactSynonym" content="protein tyrosine phosphatase, non-receptor type 5" datatype="xsd:string"></span><span id='am-352' property="oboInOwl:hasExactSynonym" content="Step" datatype="xsd:string"></span><span id='am-353' property="oboInOwl:hasDbXref" content="UniProt:P54830" datatype="xsd:string"></span><span id='am-354' property="rdfs:label" content="Step" datatype="xsd:string"></span><span id='am-355' property="rdfs:label" content="EC 3.1.3.48" datatype="xsd:string"></span>step</span> optimization method called pFBA [12]. E. coli model Orth et al. [60], demon-                       pFBA previously used predict unique flux <br /> strated original pathways recovered                       distributions predicted FBA optima. <br /> adding identified loops. analysis                         Simulations comparing costs  supplemental information (Additional file 5                     formed growth rates ranging 50 100% Additional file 6: Table <span id='am-1258' about='obo:IMR_0702943' typeof='owl:Thing'><span id='am-1259' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S, SUBUNIT 2" datatype="xsd:string"></span><span id='am-1260' property="rdfs:label" content="TNFR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-1261' property="rdfs:label" content="26S proteasome regulatory subunit S2" datatype="xsd:string"></span><span id='am-1262' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S SUBUNIT, NON-ATPASE, 2" datatype="xsd:string"></span><span id='am-1263' property="oboInOwl:id" content="IMR:0702943" datatype="xsd:string"></span><span id='am-1264' property="rdfs:label" content="PROTEASOME 26S, SUBUNIT 2" datatype="xsd:string"></span><span id='am-1265' property="oboInOwl:hasExactSynonym" content="P97" datatype="xsd:string"></span><span id='am-1266' property="oboInOwl:hasExactSynonym" content="TUMOR NECROSIS FACTOR RECEPTOR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-1267' property="rdfs:label" content="TRAP2" datatype="xsd:string"></span><span id='am-1268' property="rdfs:label" content="PSMD2_HUMAN" datatype="xsd:string"></span><span id='am-1269' property="oboInOwl:hasExactSynonym" content="TRAP2" datatype="xsd:string"></span><span id='am-1270' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 2" datatype="xsd:string"></span><span id='am-1271' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S2" datatype="xsd:string"></span><span id='am-1272' property="oboInOwl:hasExactSynonym" content="MGC14274" datatype="xsd:string"></span><span id='am-1273' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 2" datatype="xsd:string"></span><span id='am-1274' property="rdfs:label" content="26S proteasome regulatory subunit RPN1" datatype="xsd:string"></span><span id='am-1275' property="rdfs:label" content="P97" datatype="xsd:string"></span><span id='am-1276' property="oboInOwl:hasExactSynonym" content="Protein 55.11" datatype="xsd:string"></span><span id='am-1277' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 2" datatype="xsd:string"></span><span id='am-1278' property="rdfs:label" content="PSMD2" datatype="xsd:string"></span><span id='am-1279' property="rdfs:label" content="TUMOR NECROSIS FACTOR RECEPTOR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span><span id='am-1280' property="rdfs:label" content="PROTEASOME 26S SUBUNIT, NON-ATPASE, 2" datatype="xsd:string"></span><span id='am-1281' property="oboInOwl:hasExactSynonym" content="26S proteasome subunit p97" datatype="xsd:string"></span><span id='am-1282' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN1" datatype="xsd:string"></span><span id='am-1283' property="rdfs:label" content="MGC14274" datatype="xsd:string"></span><span id='am-1284' property="rdfs:label" content="Tumor necrosis factor type 1 receptor-associated protein 2" datatype="xsd:string"></span><span id='am-1285' property="oboInOwl:hasExactSynonym" content="Rpn1" datatype="xsd:string"></span><span id='am-1286' property="rdfs:label" content="Rpn1" datatype="xsd:string"></span><span id='am-1287' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 2" datatype="xsd:string"></span><span id='am-1288' property="oboInOwl:hasExactSynonym" content="S2" datatype="xsd:string"></span><span id='am-1289' property="rdfs:label" content="Protein 55.11" datatype="xsd:string"></span><span id='am-1290' property="oboInOwl:hasExactSynonym" content="Tumor necrosis factor type 1 receptor-associated protein 2" datatype="xsd:string"></span><span id='am-1291' property="oboInOwl:hasExactSynonym" content="PSMD2" datatype="xsd:string"></span><span id='am-1292' property="rdfs:label" content="S2" datatype="xsd:string"></span><span id='am-1293' property="oboInOwl:hasDbXref" content="UniProt:Q13200" datatype="xsd:string"></span><span id='am-1294' property="rdfs:label" content="26S proteasome subunit p97" datatype="xsd:string"></span><span id='am-1295' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1296' property="oboInOwl:hasExactSynonym" content="TNFR-ASSOCIATED PROTEIN 2" datatype="xsd:string"></span>S2</span>).                                            predicted optima. Flux distributions normalized   obtain fewer pathways tailored recon-                     flux glucose glucose-6-phosphate, results <br /> struction, set reactions associated                       central carbon metabolism reactions imbalance original model. example, instead                        supplemental information (Additional file 7: multiple Extreme Pathways, set reactions                        Figure S2). simulations using molecular weights, <br /> Figure 1A yield single fundamental pathway                     thermodynamic costs, combination present <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                 Page 6 12 <br />  <br />  <br />  <br />  <br /> considerably different flux distributions different sub-      [16,63]. ATP production optimization lead optimal values, simulations using uniform costs quickly          higher fluxes <span id='am-1062' about='obo:IMR_0002518' typeof='owl:Thing'><span id='am-1063' property="oboInOwl:id" content="IMR:0002518" datatype="xsd:string"></span><span id='am-1064' property="oboInOwl:hasExactSynonym" content="HLA-6.0" datatype="xsd:string"></span><span id='am-1065' property="oboInOwl:hasExactSynonym" content="HLA-G HISTOCOMPATIBILITY ANTIGEN, CLASS I" datatype="xsd:string"></span><span id='am-1066' property="rdfs:label" content="TCA" datatype="xsd:string"></span><span id='am-1067' property="rdfs:label" content="MHC class I antigen G" datatype="xsd:string"></span><span id='am-1068' property="rdfs:label" content="HLA class I histocompatibility antigen, alpha chain G" datatype="xsd:string"></span><span id='am-1069' property="oboInOwl:hasExactSynonym" content="TCA" datatype="xsd:string"></span><span id='am-1070' property="rdfs:label" content="HLAG_HUMAN" datatype="xsd:string"></span><span id='am-1071' property="rdfs:label" content="MHC-G" datatype="xsd:string"></span><span id='am-1072' property="oboInOwl:hasExactSynonym" content="major histocompatibility complex, class I, G" datatype="xsd:string"></span><span id='am-1073' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1074' property="rdfs:label" content="major histocompatibility complex, class I, G" datatype="xsd:string"></span><span id='am-1075' property="rdfs:label" content="HLA60" datatype="xsd:string"></span><span id='am-1076' property="rdfs:label" content="HLA G antigen" datatype="xsd:string"></span><span id='am-1077' property="rdfs:label" content="HLA-6.0" datatype="xsd:string"></span><span id='am-1078' property="oboInOwl:hasExactSynonym" content="HLA class I histocompatibility antigen, alpha chain G" datatype="xsd:string"></span><span id='am-1079' property="oboInOwl:hasExactSynonym" content="HLA-G" datatype="xsd:string"></span><span id='am-1080' property="rdfs:label" content="T-CELL A LOCUS" datatype="xsd:string"></span><span id='am-1081' property="oboInOwl:hasExactSynonym" content="HLAG" datatype="xsd:string"></span><span id='am-1082' property="rdfs:label" content="HLAG" datatype="xsd:string"></span><span id='am-1083' property="oboInOwl:hasExactSynonym" content="T-CELL A LOCUS" datatype="xsd:string"></span><span id='am-1084' property="rdfs:label" content="HLA-G" datatype="xsd:string"></span><span id='am-1085' property="oboInOwl:hasDbXref" content="UniProt:P17693" datatype="xsd:string"></span><span id='am-1086' property="oboInOwl:hasExactSynonym" content="MHC-G" datatype="xsd:string"></span><span id='am-1087' property="oboInOwl:hasExactSynonym" content="MHC class I antigen G" datatype="xsd:string"></span><span id='am-1088' property="oboInOwl:hasExactSynonym" content="HLA G antigen" datatype="xsd:string"></span><span id='am-1089' property="oboInOwl:hasExactSynonym" content="HLA60" datatype="xsd:string"></span><span id='am-1090' property="rdfs:label" content="HLA-G HISTOCOMPATIBILITY ANTIGEN, CLASS I" datatype="xsd:string"></span>TCA</span> cycle lower PPP fluxes. <br /> converge unique flux distribution objective          worth noting  fluxes value lowered.                                                TCA cycle predicted according Figure 2, <br />    set simulations compared          MFA experimental results inconsistent, eral data points Metabolic Flux Analysis (MFA)          predicted TCA fluxes good agreement experiments: Ishii et al. [61] Yao et al. [62]. Correla-     MFA studies [64,65]. <br /> tion sum squared error simulated             simulations mirror behavior known  <br /> experimental flux distributions central carbon       flow metabolism [66], E. coli, high growth rates, <br /> metabolism reactions calculated. results          moves away oxidation glucose presented supplemental information (Additional            TCA cycle uses ATP efficient pathways, <br /> file 8: Figure <span id='am-1231' about='obo:IMR_0702945' typeof='owl:Thing'><span id='am-1232' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1233' property="oboInOwl:hasExactSynonym" content="TSTA2" datatype="xsd:string"></span><span id='am-1234' property="rdfs:label" content="TSTA2" datatype="xsd:string"></span><span id='am-1235' property="oboInOwl:hasExactSynonym" content="S3" datatype="xsd:string"></span><span id='am-1236' property="oboInOwl:hasExactSynonym" content="Rpn3" datatype="xsd:string"></span><span id='am-1237' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit S3" datatype="xsd:string"></span><span id='am-1238' property="rdfs:label" content="P58" datatype="xsd:string"></span><span id='am-1239' property="oboInOwl:hasDbXref" content="UniProt:O43242" datatype="xsd:string"></span><span id='am-1240' property="rdfs:label" content="PSMD3_HUMAN" datatype="xsd:string"></span><span id='am-1241' property="rdfs:label" content="Rpn3" datatype="xsd:string"></span><span id='am-1242' property="oboInOwl:hasExactSynonym" content="RPN3" datatype="xsd:string"></span><span id='am-1243' property="oboInOwl:hasExactSynonym" content="P58" datatype="xsd:string"></span><span id='am-1244' property="oboInOwl:hasExactSynonym" content="PSMD3" datatype="xsd:string"></span><span id='am-1245' property="rdfs:label" content="26S proteasome regulatory subunit S3" datatype="xsd:string"></span><span id='am-1246' property="oboInOwl:hasExactSynonym" content="26S proteasome regulatory subunit RPN3" datatype="xsd:string"></span><span id='am-1247' property="oboInOwl:id" content="IMR:0702945" datatype="xsd:string"></span><span id='am-1248' property="rdfs:label" content="26S proteasome regulatory subunit RPN3" datatype="xsd:string"></span><span id='am-1249' property="rdfs:label" content="RPN3" datatype="xsd:string"></span><span id='am-1250' property="rdfs:label" content="Proteasome subunit p58" datatype="xsd:string"></span><span id='am-1251' property="rdfs:label" content="S3" datatype="xsd:string"></span><span id='am-1252' property="oboInOwl:hasExactSynonym" content="Proteasome subunit p58" datatype="xsd:string"></span><span id='am-1253' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 3" datatype="xsd:string"></span><span id='am-1254' property="rdfs:label" content="26S proteasome non-ATPase regulatory subunit 3" datatype="xsd:string"></span><span id='am-1255' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, non-ATPase, 3" datatype="xsd:string"></span><span id='am-1256' property="oboInOwl:hasExactSynonym" content="26S proteasome non-ATPase regulatory subunit 3" datatype="xsd:string"></span><span id='am-1257' property="rdfs:label" content="PSMD3" datatype="xsd:string"></span>S3</span>). similar method used             releasing acetate instead CO2 . simulations support <br /> Holzh 252;tter [28] included Additional file 8:          hypothesis behavior stems trade- <br /> Figure S3. pFBA simulations yield highest cor-         enzyme cost energy yield [40].  <br /> relation values suboptimal space compared         glucose available, E. coli uses higher flux Ishii et al. dataset, simulations yield      pathways enzymatically costly, highest sum squared error. Furthermore,          produce fewer ATP mmol glucose. <br /> objective function lowered 100% 65%             Predicted fluxes using molecular weights optima, corsoFBA yields lowest sum squared error          combined cost excellent agreement data points (Ishii et al. dilution    experimental values reactions PPC (<span id='am-1130' about='obo:IMR_0100342' typeof='owl:Thing obo:IMR_0100341 obo:IMR_0000272 obo:IMR_0000949 obo:IMR_0010044 obo:IMR_0000273 obo:IMR_0100337 obo:IMR_0000001 obo:IMR_0000517 obo:IMR_0000207'><span id='am-1131' property="oboInOwl:id" content="IMR:0100342" datatype="xsd:string"></span><span id='am-1132' property="rdfs:label" content="PEP" datatype="xsd:string"></span><span id='am-1133' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>PEP</span> Carboxy- <br /> 0.7h-1 ), highest correlation values higher      lase) ICL (Isocitrate <span id='am-1324' about='obo:IMR_0010054' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000001 obo:IMR_0000207'><span id='am-1325' property="oboInOwl:hasDbXref" content="GO:0016829" datatype="xsd:string"></span><span id='am-1326' property="oboInOwl:hasDbXref" content="EC:4.-.-.-" datatype="xsd:string"></span><span id='am-1327' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1328' property="oboInOwl:id" content="IMR:0010054" datatype="xsd:string"></span><span id='am-1329' property="rdfs:label" content="lyase" datatype="xsd:string"></span>Lyase</span>)[61,62]. experiments <br /> dilution rates Yao et al. data (dilution = 0.4, 0.6   decrease ICL flux increase PPC <br /> 0.7h-1 ).                                                        flux, near-optimal conditions growth rate 0.5 <br />                                                                  h-1 . result supported experimental <br /> Comparison metabolic flux analysis                   results Nanchen et al. [67], lower flux <br /> simulated fluxes                                                 ICL growth rate approximately 0.05 order visualize trend different fluxes          h-1 compared growth rate 0.1 h-1 , suggest- <br /> change sub-optimal space, simulations utilizing           ing transient response pathway glucose <br /> molecular weight costs, thermodynamic costs com-           concentration increases. transient response bination  normalized flux glucose        simulations. results suggest E. glucose-6-phosphate, plotted alongside exper-        coli utilizes anaplerotic reactions (reactions respon- <br /> imental values MFA experiments. Fluxes              sible forming intermediates metabolic pathways) plotted starting 100% 75% pre-            relieve enzymatic cost, considering dicted optima, results central <span id='am-140' about='obo:IMR_0704802' typeof='owl:Thing'><span id='am-141' property="rdfs:label" content="TSE1" datatype="xsd:string"></span><span id='am-142' property="rdfs:label" content="PROTEIN KINASE, cAMP-DEPENDENT, REGULATORY, TYPE I, ALPHA" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="UniProt:P10644" datatype="xsd:string"></span><span id='am-144' property="rdfs:label" content="PRKAR1A" datatype="xsd:string"></span><span id='am-145' property="rdfs:label" content="protein kinase, cAMP-dependent, regulatory, type I, alpha (tissue specific extinguisher 1)" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasExactSynonym" content="PKR1" datatype="xsd:string"></span><span id='am-148' property="rdfs:label" content="CNC1" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasExactSynonym" content="Tissue-specific extinguisher 1" datatype="xsd:string"></span><span id='am-150' property="rdfs:label" content="PRKAR1" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:hasExactSynonym" content="PROTEIN KINASE, cAMP-DEPENDENT, REGULATORY, TYPE I, ALPHA" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="DKFZp779L0468" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:id" content="IMR:0704802" datatype="xsd:string"></span><span id='am-154' property="rdfs:label" content="PPNAD1" datatype="xsd:string"></span><span id='am-155' property="rdfs:label" content="KAP0_HUMAN" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasExactSynonym" content="CNC1" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasExactSynonym" content="protein kinase, cAMP-dependent, regulatory, type I, alpha (tissue specific extinguisher 1)" datatype="xsd:string"></span><span id='am-158' property="rdfs:label" content="CAR" datatype="xsd:string"></span><span id='am-159' property="rdfs:label" content="PKR1" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasExactSynonym" content="cAMP-dependent protein kinase type I-alpha regulatory subunit" datatype="xsd:string"></span><span id='am-161' property="rdfs:label" content="Tissue-specific extinguisher 1" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasExactSynonym" content="CAR" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasExactSynonym" content="PRKAR1" datatype="xsd:string"></span><span id='am-164' property="rdfs:label" content="MGC17251" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:hasExactSynonym" content="TSE1" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasExactSynonym" content="DKFZp779L0468" datatype="xsd:string"></span><span id='am-167' property="rdfs:label" content="CNC" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasExactSynonym" content="PPNAD1" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasExactSynonym" content="CNC" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:hasExactSynonym" content="MGC17251" datatype="xsd:string"></span><span id='am-171' property="oboInOwl:hasExactSynonym" content="PRKAR1A" datatype="xsd:string"></span><span id='am-172' property="rdfs:label" content="cAMP-dependent protein kinase type I-alpha regulatory subunit" datatype="xsd:string"></span>car</span>-          costs FBA increase internal flux predictions <br /> bon metabolism reactions presented Figure 2.          comparing minimization metabolic steps <br /> threshold 75% chosen order match trend           <br /> observed simulations experimental data                Simulation results suggest optimization [61,62].                                                         biomass function yield fluxes fundamen- <br />   objective function value decreases, corsoFBA            tally different near-optimal conditions. flux distributions close agreement MFA data            optimal growth, predicted flux higher growth rates (Figure 2), particularly simu-       glyoxylate shunt, considerably lower fluxes lations considering thermodynamic molecular             TCA cycle PPP, flux PPC reaction <br /> weight costs. Fluxes Pentose Phosphate Path-         present near-optimal conditions. com- <br /> way (PPP) Glycolysis remain relatively constant,       parisons experimental data simulated flux flux TCA cycle gradually decreases.          distributions highest correlation low- <br /> decrease TCA cycle usage leads higher levels            <span id='am-128' about='obo:IMR_0000233' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000229 obo:IMR_0000512 obo:IMR_0000209 obo:IMR_0010050 obo:IMR_0000001 obo:IMR_0000207 obo:IMR_0000223 obo:IMR_0000681'><span id='am-129' property="rdfs:label" content="Est" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasExactSynonym" content="cancer Osaka thyroid oncogene" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="Cot" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasExactSynonym" content="Cot" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasExactSynonym" content="Tumor progression locus 2" datatype="xsd:string"></span><span id='am-134' property="rdfs:label" content="Tumor progression locus 2" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:id" content="IMR:0000233" datatype="xsd:string"></span><span id='am-136' property="rdfs:label" content="cancer Osaka thyroid oncogene" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasExactSynonym" content="Est" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="Tpl2" datatype="xsd:string"></span>est</span> error optimal growth, simulations <br /> acetate release. Quantitatively, fluxes glycol-      indicate implementation costs yield <br /> ysis TCA cycle slightly predicted           comparable results sub-optimal space (Additional simulations, flux PPP gener-        file 8: Figure S3).  suboptimal corsoFBA <br /> ally predicted. Similar patterns observed         approach better predict fluxes certain reac- FBA method approaches [33,45,48],              tions, PPC ICL. previous studies arise optimization biomass func-           shown FBA solution space increases drastically <br /> tion  optimization ATP production             considering objective function near-optimal biomass shown yield better predictions          optimal values [18], optimization <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                       Page 7 12 <br />  <br />  <br />  <br />  <br />  Figure 2 Comparison simulations MFA experiments. Comparison selected simulated fluxes using molecular weights, <br />  thermodynamic penalties, combination costs, Metabolic Flux Analysis (MFA) data. Fluxes normalized glucose  glucose-6-phosphate conversion rates. X-axes values kept constant plots. Reaction names taken Ecoli iJR904 model [52]. <br />  Yellow boxes indicate Pentose Phosphate Pathway (PPP), blue indicates glycolysis, green indicates TCA cycle red indicates reactions. <br />  <br />  <br />  enzymatic cost near-optimal conditions yields                     (Additional file 9: Figure <span id='am-930' about='obo:IMR_0010555' typeof='owl:Thing obo:IMR_0000949 obo:IMR_0000896 obo:IMR_0100710 obo:IMR_0000001 obo:IMR_0000950'><span id='am-931' property="rdfs:label" content="40S ribosomal protein S2" datatype="xsd:string"></span><span id='am-932' property="oboInOwl:hasExactSynonym" content="40S small ribosomal protein 2" datatype="xsd:string"></span><span id='am-933' property="rdfs:label" content="S4" datatype="xsd:string"></span><span id='am-934' property="oboInOwl:hasExactSynonym" content="S4" datatype="xsd:string"></span><span id='am-935' property="rdfs:label" content="LLREP3 protein" datatype="xsd:string"></span><span id='am-936' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-937' property="oboInOwl:hasExactSynonym" content="LLREP3 protein" datatype="xsd:string"></span><span id='am-938' property="rdfs:label" content="40S small ribosomal protein 2" datatype="xsd:string"></span><span id='am-939' property="oboInOwl:id" content="IMR:0010555" datatype="xsd:string"></span>S4</span>). Predictions using uniform <br /> results consistent experimental data                     costs quickly diverge experimental data. sim- certain reactions. result strongly suggests                   ulations support  8220;cloud theory 65533;? proposed exploring FBA solution space near-optimal opti-                   Wintermute et al. [19], suggest experimental <br /> <span id='am-260' about='obo:IMR_0703972' typeof='owl:Thing'><span id='am-261' property="oboInOwl:hasExactSynonym" content="MAL" datatype="xsd:string"></span><span id='am-262' property="rdfs:label" content="CD8" datatype="xsd:string"></span><span id='am-263' property="rdfs:label" content="T-cell surface glycoprotein CD8 alpha chain" datatype="xsd:string"></span><span id='am-264' property="rdfs:label" content="T-lymphocyte differentiation antigen T8/Leu-2" datatype="xsd:string"></span><span id='am-265' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-266' property="oboInOwl:hasExactSynonym" content="T-cell surface glycoprotein CD8 alpha chain" datatype="xsd:string"></span><span id='am-267' property="oboInOwl:hasExactSynonym" content="CD8" datatype="xsd:string"></span><span id='am-268' property="oboInOwl:hasExactSynonym" content="CD8A" datatype="xsd:string"></span><span id='am-269' property="oboInOwl:hasExactSynonym" content="OKT8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-270' property="rdfs:label" content="Leu2" datatype="xsd:string"></span><span id='am-271' property="oboInOwl:hasExactSynonym" content="p32" datatype="xsd:string"></span><span id='am-272' property="oboInOwl:hasExactSynonym" content="Leu2" datatype="xsd:string"></span><span id='am-273' property="oboInOwl:id" content="IMR:0703972" datatype="xsd:string"></span><span id='am-274' property="rdfs:label" content="CD8a molecule" datatype="xsd:string"></span><span id='am-275' property="rdfs:label" content="CD8 ANTIGEN, ALPHA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-276' property="rdfs:label" content="MAL" datatype="xsd:string"></span><span id='am-277' property="rdfs:label" content="CD8A" datatype="xsd:string"></span><span id='am-278' property="rdfs:label" content="OKT8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:hasExactSynonym" content="CD8a" datatype="xsd:string"></span><span id='am-280' property="oboInOwl:hasExactSynonym" content="CD8 ANTIGEN, ALPHA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-281' property="rdfs:label" content="CD8a" datatype="xsd:string"></span><span id='am-282' property="rdfs:label" content="CD8A_HUMAN" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasDbXref" content="UniProt:P01732" datatype="xsd:string"></span><span id='am-284' property="rdfs:label" content="p32" datatype="xsd:string"></span><span id='am-285' property="oboInOwl:hasExactSynonym" content="T-lymphocyte differentiation antigen T8/Leu-2" datatype="xsd:string"></span><span id='am-286' property="oboInOwl:hasExactSynonym" content="CD8a molecule" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:hasExactSynonym" content="T8 T-CELL ANTIGEN" datatype="xsd:string"></span><span id='am-288' property="rdfs:label" content="T8 T-CELL ANTIGEN" datatype="xsd:string"></span>mal</span> objective values increase predictive accuracy                   data good agreement near-optimal flux internal cell fluxes.                                                    distributions. <br />   near optimal solution space analyzed knockout strains. Flux distributions E. coli knock-                 Comparison gene expression data simulated strains reported Ishii et al. [61] compared                  fluxes <br /> cost-optimal simulations using combined cost func-                      validate analysis, simulations using tion. Results cost-optimal flux distribution                      combined cost function compared gene 95% optimal yield similar better predictions                     expression values reported MFA studies [61,62]. <br /> according correlation sum square error                       overall correlation reaction flux  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                    Page 8 12 <br />  <br />  <br />  <br />  <br /> gene expression moderate best [34], increased                       decreased flux TCA cycle, supported expression genes participating particular path-               relative gene expression associated path- <br /> way considered good indication increased flux                 ways [68]. <br /> [68]. Simulations performed                          Simulated values good agreement fluxes <br /> ditions  utilizing combination molecular                 Entner-Doudoroff (ED) pathway. Genes edd <br /> weights thermodynamic penalty, time flux                 eda, associated pathway, long distribution normalized overall flux                   known exist E. coli [69], activity enzyme associated reactions. Comparisons                      associated reactions observed mainly simulated values gene expression data shown                    growth gluconate, glucuronate, methyl-beta-D- <br /> Figure 3.                                                                 glucuronide; phosphate limitation; carbon starvation <br />   gene expression data inconsistent                  [70]. fact, reactions generally experiments, reactions associated                    included MFA experiment networks. contrast with multiple genes, comparison supports                     Murray et al., simulations using molecular weight qualitative results presented previous section.                combined costs predict use ED pathway normalizing predicted fluxes overall flux                 excess glucose conditions, starvation (Additional enzyme associated reactions, increased                     file 7: Figure S2). predictions supported relative uptake glucose observed objective                   genetic data presented MFA studies [61,62], <br /> value decreases. result, increased relative flux                  increase edd eda expression high glycolysis predicted. Furthermore,                    glucose concentrations, MFA experimental results relative flux TCA cycle decreases                    Harcombe et al. [71], fluxes ED path- growth rate increases. increase relative <span id='am-1356' about='obo:IMR_0002834' typeof='owl:Thing'><span id='am-1357' property="oboInOwl:id" content="IMR:0002834" datatype="xsd:string"></span><span id='am-1358' property="rdfs:label" content="Glut3" datatype="xsd:string"></span><span id='am-1359' property="oboInOwl:hasExactSynonym" content="Glucose transporter-like" datatype="xsd:string"></span><span id='am-1360' property="rdfs:label" content="Glucose transporter type 3" datatype="xsd:string"></span><span id='am-1361' property="rdfs:label" content="CG3853" datatype="xsd:string"></span><span id='am-1362' property="rdfs:label" content="Glucose transporter-like" datatype="xsd:string"></span><span id='am-1363' property="oboInOwl:hasExactSynonym" content="Glucose transporter type 3" datatype="xsd:string"></span><span id='am-1364' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1365' property="oboInOwl:hasDbXref" content="UniProt:P53403" datatype="xsd:string"></span><span id='am-1366' property="oboInOwl:hasExactSynonym" content="GLU" datatype="xsd:string"></span><span id='am-1367' property="rdfs:label" content="GTR3_DROME" datatype="xsd:string"></span><span id='am-1368' property="rdfs:label" content="GLU" datatype="xsd:string"></span><span id='am-1369' property="rdfs:label" content="glut-l" datatype="xsd:string"></span><span id='am-1370' property="oboInOwl:hasExactSynonym" content="glut-l" datatype="xsd:string"></span><span id='am-1371' property="oboInOwl:hasExactSynonym" content="Glucose transporter-like protein" datatype="xsd:string"></span><span id='am-1372' property="rdfs:label" content="Glucose transporter-like protein" datatype="xsd:string"></span><span id='am-1373' property="oboInOwl:hasExactSynonym" content="Glut3" datatype="xsd:string"></span><span id='am-1374' property="oboInOwl:hasExactSynonym" content="CG3853" datatype="xsd:string"></span>glu</span>-                  way measured bacteria growing growth rates <br /> cose uptake flux glycolysis,                   1 h-1 . <br />  <br />  <br />  <br />  <br />  Figure 3 Comparison simulations gene expression data. Comparison selected simulated fluxes, using combination  molecular weights thermodynamic penalties, associated gene expression. Plot axes ones defined ptsG plot <br />  unless specified. Multiple genes associated reaction included box. Reaction names taken E. <br />  coli iJR904 reconstruction [52]. Yellow boxes indicate Pentose Phosphate Pathway, blue indicates glycolysis, green indicates TCA cycle red <br />  indicates reactions. <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                       Page 9 12 <br />  <br />  <br />  <br />  <br /> Fundamental pathways analysis                                               total cost associated pathway. final ATP better understand transition metabolic                       production enzymatic cost compared states value objective function decreased,                 potential ATP production mmol glucose (Figure 4). energy producing pathways E. coli iJR904 recon-                    accordance general E. coli metabolism knowl- <br /> struction decomposed using Fundamental Path-                       edge, fundamental pathway analysis opti- <br /> ways analysis described Methods section. Details                     mal pathways (OP), according ATP production reactions calculated                         mmol glucose protein cost. energy pro- supplemental information (Additional file 5                      ductive pathway oxidation glucose Additional file 6: Table S2). Briefly, starting ATP                TCA cycle (<span id='am-206' about='obo:IMR_0700101' typeof='owl:Thing'><span id='am-207' property="oboInOwl:hasDbXref" content="UniProt:P34821" datatype="xsd:string"></span><span id='am-208' property="rdfs:label" content="Bmp-8" datatype="xsd:string"></span><span id='am-209' property="oboInOwl:hasExactSynonym" content="Bmp8a" datatype="xsd:string"></span><span id='am-210' property="oboInOwl:hasExactSynonym" content="Bmp-8" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasExactSynonym" content="BMP-8A" datatype="xsd:string"></span><span id='am-212' property="oboInOwl:hasExactSynonym" content="Bone morphogenetic protein 8A" datatype="xsd:string"></span><span id='am-213' property="rdfs:label" content="Bmp8a" datatype="xsd:string"></span><span id='am-214' property="oboInOwl:hasExactSynonym" content="osteogenic protein 2" datatype="xsd:string"></span><span id='am-215' property="rdfs:label" content="Osteogenic protein 2" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:hasExactSynonym" content="OP-2" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:id" content="IMR:0700101" datatype="xsd:string"></span><span id='am-218' property="oboInOwl:hasExactSynonym" content="Osteogenic protein 2" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasExactSynonym" content="OP2" datatype="xsd:string"></span><span id='am-220' property="rdfs:label" content="OP2" datatype="xsd:string"></span><span id='am-221' property="rdfs:label" content="BMP8A_MOUSE" datatype="xsd:string"></span><span id='am-222' property="rdfs:label" content="osteogenic protein 2" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasExactSynonym" content="Bmp8" datatype="xsd:string"></span><span id='am-224' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-225' property="rdfs:label" content="BMP-8A" datatype="xsd:string"></span><span id='am-226' property="oboInOwl:hasExactSynonym" content="Bmp7r1" datatype="xsd:string"></span><span id='am-227' property="oboInOwl:hasExactSynonym" content="bone morphogenetic protein 8a" datatype="xsd:string"></span><span id='am-228' property="rdfs:label" content="Bmp7r1" datatype="xsd:string"></span><span id='am-229' property="rdfs:label" content="bone morphogenetic protein 8a" datatype="xsd:string"></span><span id='am-230' property="rdfs:label" content="OP-2" datatype="xsd:string"></span><span id='am-231' property="rdfs:label" content="Bmp8" datatype="xsd:string"></span><span id='am-232' property="rdfs:label" content="Bone morphogenetic protein 8A" datatype="xsd:string"></span>OP2</span> ). cost efficient pathway imbalance associated fundamental pathway,                     production acetate glycoly- <br /> total ATP potential pathway estimated based                     <span id='am-965' about='obo:IMR_0704300' typeof='owl:Thing'><span id='am-966' property="rdfs:label" content="Platelet-derived growth factor subunit B" datatype="xsd:string"></span><span id='am-967' property="oboInOwl:hasExactSynonym" content="PDGF2" datatype="xsd:string"></span><span id='am-968' property="oboInOwl:hasExactSynonym" content="SSV" datatype="xsd:string"></span><span id='am-969' property="oboInOwl:hasExactSynonym" content="PLATELET-DERIVED GROWTH FACTOR, BETA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-970' property="oboInOwl:hasExactSynonym" content="Platelet-derived growth factor beta polypeptide" datatype="xsd:string"></span><span id='am-971' property="oboInOwl:hasExactSynonym" content="Platelet-derived growth factor subunit B" datatype="xsd:string"></span><span id='am-972' property="rdfs:label" content="SIS" datatype="xsd:string"></span><span id='am-973' property="rdfs:label" content="Becaplermin" datatype="xsd:string"></span><span id='am-974' property="oboInOwl:hasExactSynonym" content="PDGF-2" datatype="xsd:string"></span><span id='am-975' property="oboInOwl:hasExactSynonym" content="FLJ12858" datatype="xsd:string"></span><span id='am-976' property="rdfs:label" content="FLJ12858" datatype="xsd:string"></span><span id='am-977' property="rdfs:label" content="PDGF subunit B" datatype="xsd:string"></span><span id='am-978' property="oboInOwl:hasExactSynonym" content="platelet-derived growth factor beta polypeptide (simian sarcoma viral (v-sis) oncogene homolog)" datatype="xsd:string"></span><span id='am-979' property="rdfs:label" content="PLATELET-DERIVED GROWTH FACTOR, B CHAIN" datatype="xsd:string"></span><span id='am-980' property="oboInOwl:hasExactSynonym" content="c-sis" datatype="xsd:string"></span><span id='am-981' property="oboInOwl:hasExactSynonym" content="Platelet-derived growth factor B chain" datatype="xsd:string"></span><span id='am-982' property="rdfs:label" content="PLATELET-DERIVED GROWTH FACTOR, BETA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-983' property="oboInOwl:hasExactSynonym" content="PDGF subunit B" datatype="xsd:string"></span><span id='am-984' property="rdfs:label" content="PDGF, B CHAIN" datatype="xsd:string"></span><span id='am-985' property="oboInOwl:hasExactSynonym" content="SIMIAN SARCOMA VIRAL ONCOGENE HOMOLOG" datatype="xsd:string"></span><span id='am-986' property="rdfs:label" content="Proto-oncogene c-Sis" datatype="xsd:string"></span><span id='am-987' property="rdfs:label" content="V-SIS PLATELET-DERIVED GROWTH FACTOR, BETA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-988' property="oboInOwl:hasExactSynonym" content="V-SIS PLATELET-DERIVED GROWTH FACTOR, BETA POLYPEPTIDE" datatype="xsd:string"></span><span id='am-989' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-990' property="oboInOwl:id" content="IMR:0704300" datatype="xsd:string"></span><span id='am-991' property="rdfs:label" content="SIMIAN SARCOMA VIRAL ONCOGENE HOMOLOG" datatype="xsd:string"></span><span id='am-992' property="oboInOwl:hasExactSynonym" content="SIS" datatype="xsd:string"></span><span id='am-993' property="rdfs:label" content="platelet-derived growth factor beta polypeptide (simian sarcoma viral (v-sis) oncogene homolog)" datatype="xsd:string"></span><span id='am-994' property="oboInOwl:hasExactSynonym" content="Becaplermin" datatype="xsd:string"></span><span id='am-995' property="rdfs:label" content="PDGF2" datatype="xsd:string"></span><span id='am-996' property="oboInOwl:hasExactSynonym" content="PDGFB" datatype="xsd:string"></span><span id='am-997' property="rdfs:label" content="PDGFB_HUMAN" datatype="xsd:string"></span><span id='am-998' property="rdfs:label" content="PDGF-2" datatype="xsd:string"></span><span id='am-999' property="oboInOwl:hasExactSynonym" content="ONCOGENE SIS" datatype="xsd:string"></span><span id='am-1000' property="oboInOwl:hasExactSynonym" content="PLATELET-DERIVED GROWTH FACTOR, B CHAIN" datatype="xsd:string"></span><span id='am-1001' property="rdfs:label" content="ONCOGENE SIS" datatype="xsd:string"></span><span id='am-1002' property="oboInOwl:hasDbXref" content="UniProt:P01127" datatype="xsd:string"></span><span id='am-1003' property="oboInOwl:hasExactSynonym" content="Proto-oncogene c-Sis" datatype="xsd:string"></span><span id='am-1004' property="rdfs:label" content="Platelet-derived growth factor beta polypeptide" datatype="xsd:string"></span><span id='am-1005' property="rdfs:label" content="c-sis" datatype="xsd:string"></span><span id='am-1006' property="rdfs:label" content="Platelet-derived growth factor B chain" datatype="xsd:string"></span><span id='am-1007' property="rdfs:label" content="PDGFB" datatype="xsd:string"></span><span id='am-1008' property="oboInOwl:hasExactSynonym" content="PDGF, B CHAIN" datatype="xsd:string"></span><span id='am-1009' property="rdfs:label" content="SSV" datatype="xsd:string"></span>sis</span> (<span id='am-377' about='obo:IMR_0700099' typeof='owl:Thing'><span id='am-378' property="oboInOwl:hasExactSynonym" content="Op1" datatype="xsd:string"></span><span id='am-379' property="oboInOwl:id" content="IMR:0700099" datatype="xsd:string"></span><span id='am-380' property="rdfs:label" content="Bmp7" datatype="xsd:string"></span><span id='am-381' property="oboInOwl:hasExactSynonym" content="OP-1" datatype="xsd:string"></span><span id='am-382' property="rdfs:label" content="OP1" datatype="xsd:string"></span><span id='am-383' property="oboInOwl:hasExactSynonym" content="Bmp-7" datatype="xsd:string"></span><span id='am-384' property="rdfs:label" content="Op1" datatype="xsd:string"></span><span id='am-385' property="rdfs:label" content="OP-1" datatype="xsd:string"></span><span id='am-386' property="oboInOwl:hasDbXref" content="UniProt:P23359" datatype="xsd:string"></span><span id='am-387' property="oboInOwl:hasExactSynonym" content="Bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-388' property="oboInOwl:hasExactSynonym" content="BMP-7" datatype="xsd:string"></span><span id='am-389' property="oboInOwl:hasExactSynonym" content="osteogenic protein 1" datatype="xsd:string"></span><span id='am-390' property="rdfs:label" content="Osteogenic protein 1" datatype="xsd:string"></span><span id='am-391' property="oboInOwl:hasExactSynonym" content="Osteogenic protein 1" datatype="xsd:string"></span><span id='am-392' property="oboInOwl:hasExactSynonym" content="OP1" datatype="xsd:string"></span><span id='am-393' property="rdfs:label" content="Bmp-7" datatype="xsd:string"></span><span id='am-394' property="rdfs:label" content="osteogenic protein 1" datatype="xsd:string"></span><span id='am-395' property="oboInOwl:hasExactSynonym" content="bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-396' property="rdfs:label" content="BMP7_MOUSE" datatype="xsd:string"></span><span id='am-397' property="rdfs:label" content="BMP-7" datatype="xsd:string"></span><span id='am-398' property="oboInOwl:hasExactSynonym" content="Bmp7" datatype="xsd:string"></span><span id='am-399' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-400' property="rdfs:label" content="bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-401' property="rdfs:label" content="Bone morphogenetic protein 7" datatype="xsd:string"></span>OP1</span> ). pathway potentially produces imbalance energy generating metabolites,                    half ATP <span id='am-233' about='obo:IMR_0700100' typeof='owl:Thing'><span id='am-234' property="oboInOwl:hasExactSynonym" content="BMP-8" datatype="xsd:string"></span><span id='am-235' property="rdfs:label" content="bone morphogenetic protein 8b" datatype="xsd:string"></span><span id='am-236' property="rdfs:label" content="BMP8" datatype="xsd:string"></span><span id='am-237' property="oboInOwl:hasExactSynonym" content="BMP8" datatype="xsd:string"></span><span id='am-238' property="rdfs:label" content="OP2" datatype="xsd:string"></span><span id='am-239' property="oboInOwl:hasExactSynonym" content="BMP-8B" datatype="xsd:string"></span><span id='am-240' property="rdfs:label" content="OP-2" datatype="xsd:string"></span><span id='am-241' property="oboInOwl:hasExactSynonym" content="MGC131757" datatype="xsd:string"></span><span id='am-242' property="rdfs:label" content="BONE MORPHOGENETIC PROTEIN 8" datatype="xsd:string"></span><span id='am-243' property="rdfs:label" content="MGC131757" datatype="xsd:string"></span><span id='am-244' property="rdfs:label" content="BMP-8B" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasExactSynonym" content="Bone morphogenetic protein 8B" datatype="xsd:string"></span><span id='am-246' property="oboInOwl:hasExactSynonym" content="Osteogenic protein 2" datatype="xsd:string"></span><span id='am-247' property="rdfs:label" content="Bone morphogenetic protein 8B" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasDbXref" content="UniProt:P34820" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:id" content="IMR:0700100" datatype="xsd:string"></span><span id='am-250' property="oboInOwl:hasExactSynonym" content="bone morphogenetic protein 8b" datatype="xsd:string"></span><span id='am-251' property="rdfs:label" content="BMP-8" datatype="xsd:string"></span><span id='am-252' property="rdfs:label" content="Osteogenic protein 2" datatype="xsd:string"></span><span id='am-253' property="oboInOwl:hasExactSynonym" content="OP2" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasExactSynonym" content="OP-2" datatype="xsd:string"></span><span id='am-255' property="rdfs:label" content="BMP8B_HUMAN" datatype="xsd:string"></span><span id='am-256' property="rdfs:label" content="BMP8B" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasExactSynonym" content="BMP8B" datatype="xsd:string"></span><span id='am-258' property="oboInOwl:hasExactSynonym" content="BONE MORPHOGENETIC PROTEIN 8" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>OP2</span> , total enzymatic NADH Ubiquinol-8. lowest cost                        cost mmol ATP lower. agreement verting metabolites ATP added                        general knowledge, analysis predicted release  <br />  <br />  <br />  <br />  Figure 4 Fundamental pathway analysis. ATP production potential energy producing fundamental pathways, compared associated <br />  protein cost mmol ATP. Optimal (OP) Near Optimal (NOP) Pathways highlighted scatter plot. <span id='am-448' about='obo:IMR_0701940' typeof='owl:Thing'><span id='am-449' property="rdfs:label" content="LRP-5" datatype="xsd:string"></span><span id='am-450' property="rdfs:label" content="OPTA1" datatype="xsd:string"></span><span id='am-451' property="oboInOwl:hasExactSynonym" content="LRP7" datatype="xsd:string"></span><span id='am-452' property="oboInOwl:hasExactSynonym" content="LR3" datatype="xsd:string"></span><span id='am-453' property="oboInOwl:hasExactSynonym" content="HBM" datatype="xsd:string"></span><span id='am-454' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-455' property="oboInOwl:hasExactSynonym" content="BMND1" datatype="xsd:string"></span><span id='am-456' property="rdfs:label" content="HBM" datatype="xsd:string"></span><span id='am-457' property="rdfs:label" content="OPS" datatype="xsd:string"></span><span id='am-458' property="rdfs:label" content="low density lipoprotein receptor-related protein 5" datatype="xsd:string"></span><span id='am-459' property="oboInOwl:hasExactSynonym" content="LRP-5" datatype="xsd:string"></span><span id='am-460' property="oboInOwl:hasExactSynonym" content="LOW DENSITY LIPOPROTEIN RECEPTOR-RELATED PROTEIN 7" datatype="xsd:string"></span><span id='am-461' property="oboInOwl:hasExactSynonym" content="VBCH2" datatype="xsd:string"></span><span id='am-462' property="rdfs:label" content="VBCH2" datatype="xsd:string"></span><span id='am-463' property="rdfs:label" content="Low-density lipoprotein receptor-related protein 5" datatype="xsd:string"></span><span id='am-464' property="oboInOwl:hasExactSynonym" content="EVR1" datatype="xsd:string"></span><span id='am-465' property="rdfs:label" content="LRP5_HUMAN" datatype="xsd:string"></span><span id='am-466' property="rdfs:label" content="LOW DENSITY LIPOPROTEIN RECEPTOR-RELATED PROTEIN 7" datatype="xsd:string"></span><span id='am-467' property="oboInOwl:hasExactSynonym" content="EVR4" datatype="xsd:string"></span><span id='am-468' property="oboInOwl:hasDbXref" content="UniProt:O75197" datatype="xsd:string"></span><span id='am-469' property="rdfs:label" content="EVR4" datatype="xsd:string"></span><span id='am-470' property="oboInOwl:hasExactSynonym" content="OPPG" datatype="xsd:string"></span><span id='am-471' property="oboInOwl:hasExactSynonym" content="low density lipoprotein receptor-related protein 5" datatype="xsd:string"></span><span id='am-472' property="oboInOwl:id" content="IMR:0701940" datatype="xsd:string"></span><span id='am-473' property="oboInOwl:hasExactSynonym" content="OPTA1" datatype="xsd:string"></span><span id='am-474' property="rdfs:label" content="OPPG" datatype="xsd:string"></span><span id='am-475' property="rdfs:label" content="LRP7" datatype="xsd:string"></span><span id='am-476' property="rdfs:label" content="BMND1" datatype="xsd:string"></span><span id='am-477' property="oboInOwl:hasExactSynonym" content="LRP5" datatype="xsd:string"></span><span id='am-478' property="rdfs:label" content="EVR1" datatype="xsd:string"></span><span id='am-479' property="oboInOwl:hasExactSynonym" content="Low-density lipoprotein receptor-related protein 5" datatype="xsd:string"></span><span id='am-480' property="rdfs:label" content="LRP5" datatype="xsd:string"></span><span id='am-481' property="oboInOwl:hasExactSynonym" content="OPS" datatype="xsd:string"></span><span id='am-482' property="rdfs:label" content="LR3" datatype="xsd:string"></span>OPs</span> plotted blue  NOPs red central carbon metabolism diagrams. <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                  Page 10 12 <br />  <br />  <br />  <br />  <br /> acetate efficient release lactate       Conclusion ethanol.                                                        present corsoFBA, method analyzing    simulations presented previous sections,             predicting metabolic flux distributions sub-optimal <br /> gradual transition OP2 <span id='am-402' about='obo:IMR_0700099' typeof='owl:Thing'><span id='am-403' property="rdfs:label" content="bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-404' property="oboInOwl:hasExactSynonym" content="Bmp-7" datatype="xsd:string"></span><span id='am-405' property="rdfs:label" content="BMP-7" datatype="xsd:string"></span><span id='am-406' property="rdfs:label" content="Op1" datatype="xsd:string"></span><span id='am-407' property="oboInOwl:hasDbXref" content="UniProt:P23359" datatype="xsd:string"></span><span id='am-408' property="oboInOwl:hasExactSynonym" content="OP-1" datatype="xsd:string"></span><span id='am-409' property="rdfs:label" content="OP-1" datatype="xsd:string"></span><span id='am-410' property="oboInOwl:hasExactSynonym" content="Bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-411' property="oboInOwl:hasExactSynonym" content="bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-412' property="oboInOwl:hasExactSynonym" content="Osteogenic protein 1" datatype="xsd:string"></span><span id='am-413' property="oboInOwl:hasExactSynonym" content="Bmp7" datatype="xsd:string"></span><span id='am-414' property="rdfs:label" content="OP1" datatype="xsd:string"></span><span id='am-415' property="rdfs:label" content="Osteogenic protein 1" datatype="xsd:string"></span><span id='am-416' property="oboInOwl:hasExactSynonym" content="Op1" datatype="xsd:string"></span><span id='am-417' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-418' property="rdfs:label" content="Bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-419' property="rdfs:label" content="osteogenic protein 1" datatype="xsd:string"></span><span id='am-420' property="oboInOwl:hasExactSynonym" content="OP1" datatype="xsd:string"></span><span id='am-421' property="oboInOwl:hasExactSynonym" content="BMP-7" datatype="xsd:string"></span><span id='am-422' property="oboInOwl:hasExactSynonym" content="osteogenic protein 1" datatype="xsd:string"></span><span id='am-423' property="oboInOwl:id" content="IMR:0700099" datatype="xsd:string"></span><span id='am-424' property="rdfs:label" content="Bmp-7" datatype="xsd:string"></span><span id='am-425' property="rdfs:label" content="Bmp7" datatype="xsd:string"></span><span id='am-426' property="rdfs:label" content="BMP7_MOUSE" datatype="xsd:string"></span>OP1</span> observed               states based protein thermodynamic cost mini- <br /> objective function value decreases. trend sup-            mization. CorsoFBA demonstrates optimization <br /> ports idea <span id='am-483' about='obo:IMR_0100719' typeof='owl:Thing'><span id='am-484' property="rdfs:label" content="1423/11" datatype="xsd:string"></span><span id='am-485' property="oboInOwl:hasExactSynonym" content="delta" datatype="xsd:string"></span><span id='am-486' property="rdfs:label" content="l(3)05151" datatype="xsd:string"></span><span id='am-487' property="oboInOwl:hasExactSynonym" content="1119/09" datatype="xsd:string"></span><span id='am-488' property="rdfs:label" content="delta" datatype="xsd:string"></span><span id='am-489' property="rdfs:label" content="1304/03" datatype="xsd:string"></span><span id='am-490' property="oboInOwl:hasExactSynonym" content="1304/03" datatype="xsd:string"></span><span id='am-491' property="oboInOwl:hasExactSynonym" content="0926/11" datatype="xsd:string"></span><span id='am-492' property="oboInOwl:hasExactSynonym" content="1423/11" datatype="xsd:string"></span><span id='am-493' property="oboInOwl:hasExactSynonym" content="Overflow" datatype="xsd:string"></span><span id='am-494' property="oboInOwl:hasExactSynonym" content="delta D1" datatype="xsd:string"></span><span id='am-495' property="oboInOwl:hasExactSynonym" content="Complementation group 1" datatype="xsd:string"></span><span id='am-496' property="oboInOwl:hasExactSynonym" content="1053/14" datatype="xsd:string"></span><span id='am-497' property="rdfs:label" content="Neurogenic locus protein delta" datatype="xsd:string"></span><span id='am-498' property="oboInOwl:hasExactSynonym" content="l(3)j8C3" datatype="xsd:string"></span><span id='am-499' property="oboInOwl:id" content="IMR:0100719" datatype="xsd:string"></span><span id='am-500' property="oboInOwl:hasExactSynonym" content="dmDelta" datatype="xsd:string"></span><span id='am-501' property="rdfs:label" content="1119/09" datatype="xsd:string"></span><span id='am-502' property="rdfs:label" content="delta D1" datatype="xsd:string"></span><span id='am-503' property="rdfs:label" content="dmDelta" datatype="xsd:string"></span><span id='am-504' property="oboInOwl:hasExactSynonym" content="l(3)05151" datatype="xsd:string"></span><span id='am-505' property="rdfs:label" content="DL_DROME" datatype="xsd:string"></span><span id='am-506' property="oboInOwl:hasExactSynonym" content="1440/11" datatype="xsd:string"></span><span id='am-507' property="rdfs:label" content="1440/11" datatype="xsd:string"></span><span id='am-508' property="rdfs:label" content="Complementation group 1" datatype="xsd:string"></span><span id='am-509' property="oboInOwl:hasExactSynonym" content="CT12133" datatype="xsd:string"></span><span id='am-510' property="rdfs:label" content="1053/14" datatype="xsd:string"></span><span id='am-511' property="rdfs:label" content="l(3)92Ab" datatype="xsd:string"></span><span id='am-512' property="oboInOwl:hasExactSynonym" content="anon-WO0118547.269" datatype="xsd:string"></span><span id='am-513' property="rdfs:label" content="anon-WO0118547.269" datatype="xsd:string"></span><span id='am-514' property="oboInOwl:hasExactSynonym" content="1485/04" datatype="xsd:string"></span><span id='am-515' property="rdfs:label" content="CT12133" datatype="xsd:string"></span><span id='am-516' property="rdfs:label" content="CG3619" datatype="xsd:string"></span><span id='am-517' property="oboInOwl:hasExactSynonym" content="CG3619" datatype="xsd:string"></span><span id='am-518' property="rdfs:label" content="Delta" datatype="xsd:string"></span><span id='am-519' property="rdfs:label" content="0495/20" datatype="xsd:string"></span><span id='am-520' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-521' property="oboInOwl:hasExactSynonym" content="Neurogenic locus protein delta" datatype="xsd:string"></span><span id='am-522' property="oboInOwl:hasDbXref" content="UniProt:P10041" datatype="xsd:string"></span><span id='am-523' property="rdfs:label" content="0926/11" datatype="xsd:string"></span><span id='am-524' property="oboInOwl:hasExactSynonym" content="0495/20" datatype="xsd:string"></span><span id='am-525' property="rdfs:label" content="Overflow" datatype="xsd:string"></span><span id='am-526' property="rdfs:label" content="l(3)j8C3" datatype="xsd:string"></span><span id='am-527' property="oboInOwl:hasExactSynonym" content="Delta" datatype="xsd:string"></span><span id='am-528' property="oboInOwl:hasExactSynonym" content="l(3)92Ab" datatype="xsd:string"></span><span id='am-529' property="rdfs:label" content="1485/04" datatype="xsd:string"></span>overflow</span> metabolism takes place              protein cost near-optimal states produce sig- <br /> order alleviate enzymatic cost. low glucose            nificantly different results optimal <br /> ditions, E. coli need extract glucose             objective. Furthermore, correlation error <br /> largest possible energy order sustain               calculations indicate better predictions optimal <br /> growth. glucose available,                  solution, protein-optimal results sub-optimal objec- bacteria afford consume substrate              tives better agreement MFA experiments produce energy cheaper pathways.                            gene expression profiles anaplerotic reactions.    interesting observation analysis            results suggest important explore sub-optimal <br /> existence Near Optimal Pathways (NOP), com-                FBA space order better predict internal cell fluxes. <br /> bine previously described optimal pathways             method described suited pre- <br /> PPP ED pathways. NOP2 NOP3 produce                      dicting growth rates, provides platform analyzing ATP mmol glucose <span id='am-427' about='obo:IMR_0700093' typeof='owl:Thing'><span id='am-428' property="oboInOwl:id" content="IMR:0700093" datatype="xsd:string"></span><span id='am-429' property="oboInOwl:hasExactSynonym" content="BMP-7" datatype="xsd:string"></span><span id='am-430' property="rdfs:label" content="BMP7" datatype="xsd:string"></span><span id='am-431' property="oboInOwl:hasExactSynonym" content="BMP7" datatype="xsd:string"></span><span id='am-432' property="rdfs:label" content="Osteogenic protein 1" datatype="xsd:string"></span><span id='am-433' property="oboInOwl:hasExactSynonym" content="bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-434' property="rdfs:label" content="Eptotermin alfa" datatype="xsd:string"></span><span id='am-435' property="rdfs:label" content="BMP-7" datatype="xsd:string"></span><span id='am-436' property="oboInOwl:hasDbXref" content="UniProt:P18075" datatype="xsd:string"></span><span id='am-437' property="rdfs:label" content="Bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-438' property="oboInOwl:hasExactSynonym" content="Osteogenic protein 1" datatype="xsd:string"></span><span id='am-439' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-440' property="oboInOwl:hasExactSynonym" content="OP-1" datatype="xsd:string"></span><span id='am-441' property="rdfs:label" content="BMP7_HUMAN" datatype="xsd:string"></span><span id='am-442' property="rdfs:label" content="OP1" datatype="xsd:string"></span><span id='am-443' property="oboInOwl:hasExactSynonym" content="OP1" datatype="xsd:string"></span><span id='am-444' property="oboInOwl:hasExactSynonym" content="Eptotermin alfa" datatype="xsd:string"></span><span id='am-445' property="oboInOwl:hasExactSynonym" content="Bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-446' property="rdfs:label" content="bone morphogenetic protein 7" datatype="xsd:string"></span><span id='am-447' property="rdfs:label" content="OP-1" datatype="xsd:string"></span>OP1</span>                 internal cell fluxes sub-optimal space. <br /> OP2 , remaining cost efficient OP1                believe corsoFBA particularly useful given <br /> slightly OP2 . NOP2 NOP3 reduce                recent studies suggested E. coli exist freely flux upper glycolysis partially routing           near-optimal space [19]. believe method <br /> flux PPP, exhibit flux              useful predicting fluxes healthy, multicellu- <br /> <span id='am-40' about='obo:IMR_0702468' typeof='owl:Thing'><span id='am-41' property="rdfs:label" content="Phosphoglucose isomerase" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="Neuroleukin" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="UniProt:P06744" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="EC 5.3.1.9" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="G6PI_HUMAN" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="GNPI" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="PGI" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="PHI" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="SA36" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="GNPI" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="GPI" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="GLUCOSE PHOSPHATE ISOMERASE" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="Glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="GLUCOSE PHOSPHATE ISOMERASE" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="Phosphohexose isomerase" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Autocrine motility factor" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Autocrine motility factor" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasExactSynonym" content="Sperm antigen 36" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="NLK" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="Phosphoglucose isomerase" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="SA-36" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="NLK" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="DKFZp686C13233" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="AMF" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="AMF" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="PHI" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="GPI" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="SA-36" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="EC 5.3.1.9" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Neuroleukin" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Sperm antigen 36" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:id" content="IMR:0702468" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="SA36" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Phosphohexose isomerase" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="PGI" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="DKFZp686C13233" datatype="xsd:string"></span>Phosphoglucose isomerase</span> (<span id='am-81' about='obo:IMR_0702468' typeof='owl:Thing'><span id='am-82' property="rdfs:label" content="glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="Phosphohexose isomerase" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="PGI" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasExactSynonym" content="Glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="GLUCOSE PHOSPHATE ISOMERASE" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="PHI" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="GNPI" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="NLK" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="Phosphoglucose isomerase" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="AMF" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Phosphohexose isomerase" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="PGI" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="PHI" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="DKFZp686C13233" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="UniProt:P06744" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="SA36" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="G6PI_HUMAN" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:id" content="IMR:0702468" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="SA-36" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Sperm antigen 36" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="GPI" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasExactSynonym" content="Sperm antigen 36" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasExactSynonym" content="GPI" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasExactSynonym" content="GNPI" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasExactSynonym" content="SA-36" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasExactSynonym" content="SA36" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasExactSynonym" content="NLK" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="EC 5.3.1.9" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="GLUCOSE PHOSPHATE ISOMERASE" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Neuroleukin" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="AMF" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-114' property="rdfs:label" content="DKFZp686C13233" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Phosphoglucose isomerase" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasExactSynonym" content="Autocrine motility factor" datatype="xsd:string"></span><span id='am-117' property="rdfs:label" content="Glucose-6-phosphate isomerase" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasExactSynonym" content="Neuroleukin" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="Autocrine motility factor" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasExactSynonym" content="EC 5.3.1.9" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasExactSynonym" content="glucose-6-phosphate isomerase" datatype="xsd:string"></span>pgi</span>). Near Optimal                lar organisms, complex objectives Pathway, termed NOP1 , skips upper glycolysis           production biomass. Future studies merited ED pathway. pathway cost energy           explore predictive power methods effective OP1 .                                                optimize objective function enzymatic level <br />    use near optimal pathways explain             constraints, MOMENT [46] FBAwMC [44], inconsistent PPP fluxes reported MFA experi-             benefit exploring sub-optimal solu- <br /> ments considered  short recovery simulated            tion space. results suggest significant conclusions <br /> PPP flux near growth rate 0.65 h 65533;? (Figure 2) demon-           drawn adapting methods optimize strates pathway viable option certain dilu-       enzymatic cost implement near-optimal space, <br /> tion rates, coupling PPP biomass            using simply model constraint. <br /> production. demonstrated  <span id='am-1' about='obo:IMR_0702517' typeof='owl:Thing'><span id='am-2' property="oboInOwl:hasDbXref" content="UniProt:P08237" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="PFK, MUSCLE TYPE" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="PFK1" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="PFK-A" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="GSD7" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="Phosphofructokinase 1" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="MGC8699" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0702517" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="Phosphofructokinase-M" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="MGC8699" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="K6PF_HUMAN" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="PHOSPHOFRUCTOKINASE, MUSCLE TYPE" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="GSD7" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="6-phosphofructokinase, muscle type" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="EC 2.7.1.11" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="6-phosphofructokinase, muscle type" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="PFK-1" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="PFK, MUSCLE TYPE" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="PFKA" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="PFKM" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Phosphofructokinase-M" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Phosphofructokinase 1" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="PFKM" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="PFK-A" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Phosphofructo-1-kinase isozyme A" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="PFK-1" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="EC 2.7.1.11" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Phosphohexokinase" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="PFK1" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="PFKA" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="PFKX" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="PFKX" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Phosphofructo-1-kinase isozyme A" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="phosphofructokinase, muscle" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Phosphohexokinase" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="PHOSPHOFRUCTOKINASE, MUSCLE TYPE" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="phosphofructokinase, muscle" datatype="xsd:string"></span>pfkA</span> <br /> deficient E. coli strains reduce flux Phosphofruc- <br /> tokinase 1 diverting fluxes PPP [72],          Additional files <br /> using NOP2 NOP3 instead OP1 OP2 cost upper glycolysis increased.                          Additional file 1: Figure S1. Sensitivity analysis parameter  945;. Model <br />                                                                      simulations ranging 100 75% optima varying values  945;. <br />    Fundamental Pathway analysis elucidates               Simulations performed using thermodynamic costs expect model behave terms energy gen-             combined costs. <br /> eration optimize protein cost sub-optimal               Additional file 2: Gibbs free energy change distribution. File showing <br /> states. Based analysis, order fulfill catabolic    plot distribution Gibbs free energy change reactions <br />                                                                      considered. <br /> needs, model transitions OP2 OP1                                                                      Additional file 3: Molecular weight cost calculation. Details away optimal growth. fact               Molecular weight values obtained final costs calculated. <br /> general trend observe, anaplerotic needs need             Molecular weights final costs distribution plotted. considered. needs addressed efficiently            Additional file 4: Table S1. Thermodynamic, molecular weight final reactions OP1 OP2 , PEP carboxy-                cost values. Table includes values used simulations: Gibbs energy <br />                                                                      formation metabolites, calculated Gibbs energy change lase glyoxylate shunt. Interestingly, model              reactions based metabolite energy formation molecular weights predicts use ED pathway anaplerotic             obtained databases. <br /> purposes high growth rates. pathway                Additional file 5: Fundamental pathway analysis. Example studied catabolic activity  mobilis         demonstrating recovery Extreme Pathways adding reactions loops <br />                                                                      Fundamental Pathways. Details fundamental Pseudomonas species, simulations pre-               pathway analysis presented manuscript performed dict pathway cheapest way glucose              available. <br /> shuttled TCA cycle production building             Additional file 6: Table S2. Model decomposition. Extreme Pathways blocks, giving little efficiency energy pathways         Fundamental Pathways calculated analysis presented                                                                      manuscript pathway recovery analysis Additional file 5. NOP1 . <br />  Schultz Qutub BMC Systems Biology (2015) 9:18                                                                                                      Page 11 12 <br />  <br />  <br />  <br />  <br />                                                                                      <br />  Additional file 7: Figure S2. Cost values comparison. Cost-optimal <br />                                                                                         <br />  simulations using molecular weights  thermodynamic penalties <br />                                                                                       <br />   combination uniform costs presented major <br />                                                                                     <br />  central carbon metabolism reactions range 100 50%                                                                                     <br />  maximal growth. <br />                                                                                 <br />  Additional file 8: Figure S3. Error correlation calculations.                       <br />  Correlation sum squared error calculated simulated         <br />  experimental flux distributions major central carbon metabolism                   <br />  reactions.                                                                             <br />  Additional file 9: Figure <span id='am-1175' about='obo:IMR_0702929' typeof='owl:Thing'><span id='am-1176' property="oboInOwl:hasExactSynonym" content="Proteasome 26S subunit ATPase 1" datatype="xsd:string"></span><span id='am-1177' property="rdfs:label" content="PSMC1" datatype="xsd:string"></span><span id='am-1178' property="rdfs:label" content="MGC8541" datatype="xsd:string"></span><span id='am-1179' property="oboInOwl:hasExactSynonym" content="MGC24583" datatype="xsd:string"></span><span id='am-1180' property="rdfs:label" content="proteasome (prosome, macropain) 26S subunit, ATPase, 1" datatype="xsd:string"></span><span id='am-1181' property="rdfs:label" content="P26s4" datatype="xsd:string"></span><span id='am-1182' property="oboInOwl:id" content="IMR:0702929" datatype="xsd:string"></span><span id='am-1183' property="oboInOwl:hasExactSynonym" content="26S protease regulatory subunit 4" datatype="xsd:string"></span><span id='am-1184' property="oboInOwl:hasExactSynonym" content="PROTEASE 26S, SUBUNIT 4" datatype="xsd:string"></span><span id='am-1185' property="rdfs:label" content="PROTEASOME 26S SUBUNIT, ATPase, 1" datatype="xsd:string"></span><span id='am-1186' property="rdfs:label" content="MGC24583" datatype="xsd:string"></span><span id='am-1187' property="oboInOwl:hasExactSynonym" content="P26S4" datatype="xsd:string"></span><span id='am-1188' property="oboInOwl:hasExactSynonym" content="proteasome (prosome, macropain) 26S subunit, ATPase, 1" datatype="xsd:string"></span><span id='am-1189' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1190' property="rdfs:label" content="Proteasome 26S subunit ATPase 1" datatype="xsd:string"></span><span id='am-1191' property="oboInOwl:hasExactSynonym" content="PSMC1" datatype="xsd:string"></span><span id='am-1192' property="oboInOwl:hasExactSynonym" content="26S proteasome AAA-ATPase subunit RPT2" datatype="xsd:string"></span><span id='am-1193' property="rdfs:label" content="PRS4_HUMAN" datatype="xsd:string"></span><span id='am-1194' property="oboInOwl:hasExactSynonym" content="S4" datatype="xsd:string"></span><span id='am-1195' property="oboInOwl:hasDbXref" content="UniProt:P62191" datatype="xsd:string"></span><span id='am-1196' property="rdfs:label" content="PROTEASE 26S, SUBUNIT 4" datatype="xsd:string"></span><span id='am-1197' property="rdfs:label" content="26S proteasome AAA-ATPase subunit RPT2" datatype="xsd:string"></span><span id='am-1198' property="oboInOwl:hasExactSynonym" content="MGC8541" datatype="xsd:string"></span><span id='am-1199' property="oboInOwl:hasExactSynonym" content="PROTEASOME 26S SUBUNIT, ATPase, 1" datatype="xsd:string"></span><span id='am-1200' property="rdfs:label" content="S4" datatype="xsd:string"></span><span id='am-1201' property="rdfs:label" content="26S protease regulatory subunit 4" datatype="xsd:string"></span><span id='am-1202' property="rdfs:label" content="p56" datatype="xsd:string"></span><span id='am-1203' property="rdfs:label" content="P26S4" datatype="xsd:string"></span><span id='am-1204' property="oboInOwl:hasExactSynonym" content="p56" datatype="xsd:string"></span><span id='am-1205' property="oboInOwl:hasExactSynonym" content="P26s4" datatype="xsd:string"></span>S4</span>. Error correlation calculations               <br />  knockout strains. Correlation sum squared error calculated                   <br />  MFA experimental data simulated flux distributions using                 <br />  combined cost function knockout strains.                             <br /> TCA: Tricarboxylic acid; FBA: Flux balance analysis; PPP: Pentose phosphate                 <br /> pathway; MFA: Metabolic flux analysis; ED: Entner-Doudoroff; glc: glucose; g6p:     <br /> glucose-6-phosphate; f6p: fructose-6-phosphate; fdp: fructose-1,6-                       <br /> biphosphate; dhap: Dihydroxyacetone phosphate; g3p: Glyceraldehyde                      <br /> 3-phosphate; 13dpg: 3-Phospho-D-glyceroyl phosphate; 3pg: 3-Phospho-D-             <br /> glycerate; 2pg: D-Glycerate 2-phosphate; pep: Phosphoenolpyruvate; <span id='am-1493' about='obo:IMR_0002995' typeof='owl:Thing'><span id='am-1494' property="oboInOwl:hasDbXref" content="UniProt:B9ZW35" datatype="xsd:string"></span><span id='am-1495' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1496' property="rdfs:label" content="B9ZW35_DROME" datatype="xsd:string"></span><span id='am-1497' property="rdfs:label" content="FGF8-like2" datatype="xsd:string"></span><span id='am-1498' property="rdfs:label" content="DFGF8-2" datatype="xsd:string"></span><span id='am-1499' property="rdfs:label" content="CG13194" datatype="xsd:string"></span><span id='am-1500' property="rdfs:label" content="Pyramus" datatype="xsd:string"></span><span id='am-1501' property="oboInOwl:id" content="IMR:0002995" datatype="xsd:string"></span><span id='am-1502' property="rdfs:label" content="fgf8-like" datatype="xsd:string"></span><span id='am-1503' property="oboInOwl:hasExactSynonym" content="FGF8-like2" datatype="xsd:string"></span><span id='am-1504' property="oboInOwl:hasExactSynonym" content="CG13194" datatype="xsd:string"></span><span id='am-1505' property="oboInOwl:hasExactSynonym" content="Pyramus" datatype="xsd:string"></span><span id='am-1506' property="oboInOwl:hasExactSynonym" content="fgf8-like" datatype="xsd:string"></span><span id='am-1507' property="rdfs:label" content="pyr" datatype="xsd:string"></span><span id='am-1508' property="obo:IAO_0000115" content="ths and pyr are two closely linked genes with homology to the FGF8/17/18 subfamily in vertebrates." datatype="xsd:string"></span><span id='am-1509' property="oboInOwl:hasExactSynonym" content="DFGF8-2" datatype="xsd:string"></span><span id='am-1510' property="oboInOwl:hasExactSynonym" content="pyr" datatype="xsd:string"></span><span id='am-1511' property="oboInOwl:hasExactSynonym" content="pyramus" datatype="xsd:string"></span><span id='am-1512' property="rdfs:label" content="pyramus" datatype="xsd:string"></span>pyr</span>:                  <br /> Pyruvate; accoa: Acetyl-CoA; actp: Acetyl phosphate; ac: Acetate; cit: Citrate;         <br /> icit: Isocitrate; akg: 2-Oxoglutarate; succoa: Succinyl-CoA; succ: Succinate; fum:    <br /> Fumarate; mal-L: L-Malate; oaa: Oxaloacetate; <span id='am-1091' about='obo:IMR_0200874' typeof='owl:Thing'><span id='am-1092' property="rdfs:label" content="EC 3.1.1.31" datatype="xsd:string"></span><span id='am-1093' property="rdfs:label" content="Hexose-6-phosphate dehydrogenase" datatype="xsd:string"></span><span id='am-1094' property="oboInOwl:hasExactSynonym" content="GDH" datatype="xsd:string"></span><span id='am-1095' property="oboInOwl:hasExactSynonym" content="6PGL" datatype="xsd:string"></span><span id='am-1096' property="rdfs:label" content="GDH" datatype="xsd:string"></span><span id='am-1097' property="rdfs:label" content="H6PD" datatype="xsd:string"></span><span id='am-1098' property="rdfs:label" content="GDH/6PGL endoplasmic bifunctional protein" datatype="xsd:string"></span><span id='am-1099' property="oboInOwl:hasExactSynonym" content="EC 1.1.1.47" datatype="xsd:string"></span><span id='am-1100' property="rdfs:label" content="G6PD, H FORM" datatype="xsd:string"></span><span id='am-1101' property="rdfs:label" content="G6PE_HUMAN" datatype="xsd:string"></span><span id='am-1102' property="oboInOwl:hasExactSynonym" content="Glucose 1-dehydrogenase" datatype="xsd:string"></span><span id='am-1103' property="rdfs:label" content="EC 1.1.1.47" datatype="xsd:string"></span><span id='am-1104' property="rdfs:label" content="GLUCOSE DEHYDROGENASE" datatype="xsd:string"></span><span id='am-1105' property="oboInOwl:hasExactSynonym" content="GLUCOSE-6-PHOSPHATE DEHYDROGENASE, SALIVARY" datatype="xsd:string"></span><span id='am-1106' property="rdfs:label" content="hexose-6-phosphate dehydrogenase (glucose 1-dehydrogenase)" datatype="xsd:string"></span><span id='am-1107' property="oboInOwl:hasExactSynonym" content="G6PD, H FORM" datatype="xsd:string"></span><span id='am-1108' property="oboInOwl:hasExactSynonym" content="6-phosphogluconolactonase" datatype="xsd:string"></span><span id='am-1109' property="oboInOwl:hasExactSynonym" content="GDH/6PGL endoplasmic bifunctional protein" datatype="xsd:string"></span><span id='am-1110' property="oboInOwl:hasExactSynonym" content="H6PD" datatype="xsd:string"></span><span id='am-1111' property="rdfs:label" content="Glucose 1-dehydrogenase" datatype="xsd:string"></span><span id='am-1112' property="rdfs:label" content="DKFZp686A01246" datatype="xsd:string"></span><span id='am-1113' property="oboInOwl:hasExactSynonym" content="DKFZp686A01246" datatype="xsd:string"></span><span id='am-1114' property="rdfs:label" content="G6PDH" datatype="xsd:string"></span><span id='am-1115' property="oboInOwl:hasExactSynonym" content="Hexose-6-phosphate dehydrogenase" datatype="xsd:string"></span><span id='am-1116' property="oboInOwl:id" content="IMR:0200874" datatype="xsd:string"></span><span id='am-1117' property="oboInOwl:hasExactSynonym" content="EC 3.1.1.31" datatype="xsd:string"></span><span id='am-1118' property="rdfs:label" content="HEXOSE-6-PHOSPHATE DEHYDROGENASE PRECURSOR" datatype="xsd:string"></span><span id='am-1119' property="rdfs:label" content="GLUCOSE-6-PHOSPHATE DEHYDROGENASE, SALIVARY" datatype="xsd:string"></span><span id='am-1120' property="rdfs:label" content="6PGL" datatype="xsd:string"></span><span id='am-1121' property="rdfs:label" content="MGC87643" datatype="xsd:string"></span><span id='am-1122' property="oboInOwl:hasExactSynonym" content="HEXOSE-6-PHOSPHATE DEHYDROGENASE PRECURSOR" datatype="xsd:string"></span><span id='am-1123' property="rdfs:label" content="6-phosphogluconolactonase" datatype="xsd:string"></span><span id='am-1124' property="oboInOwl:hasDbXref" content="UniProt:O95479" datatype="xsd:string"></span><span id='am-1125' property="oboInOwl:hasExactSynonym" content="MGC87643" datatype="xsd:string"></span><span id='am-1126' property="oboInOwl:hasExactSynonym" content="GLUCOSE DEHYDROGENASE" datatype="xsd:string"></span><span id='am-1127' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-1128' property="oboInOwl:hasExactSynonym" content="hexose-6-phosphate dehydrogenase (glucose 1-dehydrogenase)" datatype="xsd:string"></span><span id='am-1129' property="oboInOwl:hasExactSynonym" content="G6PDH" datatype="xsd:string"></span>6pgl</span>:                                        <br /> 6-phospho-D-glucono-1,5-lactone; 6pgc: 6-Phospho-D-gluconate; ru5p-D:                     <br /> D-Ribulose 5-phosphate; r5p: alpha-D-Ribose 5-phosphate; xu5p-D:                     <br /> D-Xylulose 5-phosphate; s7p: Sedoheptulose 7-phosphate; e4p: D-Erythrose                   <br /> 4-phosphate; 2ddg6p: 2-Dehydro-3-deoxy-D-gluconate 6-phosphate.                            <br />   <br /> </body></html>